{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Cyanobiome phylogenomics"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "import glob\n",
    "import os\n",
    "import pandas as pd\n",
    "from Bio import SeqIO\n",
    "from natsort import natsorted\n",
    "import subprocess\n",
    "import numpy as np\n",
    "import re\n",
    "from Bio.Seq import Seq\n",
    "from Bio import SeqIO\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "from Bio.Alphabet import IUPAC\n",
    "from Bio import Phylo\n",
    "import matplotlib.pyplot as plt\n",
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir ./pnas-phylogenomics-top_100/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Renaming genomes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "def split_string_at(s, c, n):\n",
    "    words = s.split(c)\n",
    "    return c.join(words[:n]), c.join(words[n:])\n",
    "\n",
    "def extract_ncbi_taxa(inputs_folder):\n",
    "    input_list = glob.glob(\"%s*.fasta\"%inputs_folder)\n",
    "    taxa = {}\n",
    "    for item in input_list:\n",
    "        records = list(SeqIO.parse(item, \"fasta\"))\n",
    "        strain = os.path.splitext(os.path.basename(item))[0]\n",
    "        if records:\n",
    "            if \"TPA_asm\" in str(records[0]) or \"MAG\" in str(records[0]):\n",
    "                genera = split_string_at(genera,\":\",1)[0]\n",
    "            else:\n",
    "                genera = split_string_at(records[0].description,\" \",1)[1]\n",
    "            if \"Candidatus\" in str(genera):\n",
    "                taxa[strain] = str(genera).partition(' ')[2].partition(' ')[0]\n",
    "            else:\n",
    "                taxa[strain] = str(genera).partition(' ')[0].lstrip(\"[\").rstrip(\"]\")\n",
    "    return taxa\n",
    "            \n",
    "taxa_dict = extract_ncbi_taxa(\"./top_100/\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'NAPS01': 'Westiellopsis',\n",
       " 'JAHHHK01': 'Westiellopsis',\n",
       " 'NMQE01': 'Fischerella',\n",
       " 'AJLJ01': 'Fischerella',\n",
       " 'NQMD01': 'Fischerella',\n",
       " 'JACYLX01': 'Fischerella',\n",
       " 'MRCA01': 'Fischerella',\n",
       " 'JADEXT01': 'Nostoc',\n",
       " 'NHAE01': 'Fischerella',\n",
       " 'WBKL01': 'Nostoc',\n",
       " 'LAHF01': 'Nostoc',\n",
       " 'JAHHHW01': 'Nostoc',\n",
       " 'JACYLY01': 'Fischerella',\n",
       " 'NGZZ01': 'Fischerella',\n",
       " 'NHAD01': 'Fischerella',\n",
       " 'LIRN01': 'Hapalosiphon',\n",
       " 'ALVS01': 'Fischerella',\n",
       " 'LAHG01': 'Nostoc',\n",
       " 'AGIZ01': 'Fischerella',\n",
       " 'NGZS01': 'Fischerella',\n",
       " 'JACKZU01': 'Nostoc',\n",
       " 'NQME01': 'Fischerella',\n",
       " 'AJLK01': 'Fischerella',\n",
       " 'NMQD01': 'Fischerella',\n",
       " 'NHAF01': 'Fischerella',\n",
       " 'MNPM02': 'Mastigocladus',\n",
       " 'NGAA01': 'Fischerella',\n",
       " 'NGZX01': 'Fischerella',\n",
       " 'JAECZC01': 'Amazonocrinis',\n",
       " 'LAHE01': 'Nostoc',\n",
       " 'JACVRH01': 'Nostoc',\n",
       " 'LXQE01': 'Nostoc',\n",
       " 'ALVX01': 'Fischerella',\n",
       " 'NMQF01': 'Fischerella',\n",
       " 'NMQG01': 'Fischerella',\n",
       " 'NQMF01': 'Fischerella',\n",
       " 'JABXKH01': 'Nostoc',\n",
       " 'JACJRZ01': 'Fischerella',\n",
       " 'JADEXV01': 'Nostocales',\n",
       " 'NHAG01': 'Fischerella',\n",
       " 'NGZY01': 'Fischerella',\n",
       " 'LAHD01': 'Nostoc',\n",
       " 'QLKN01': 'Hapalosiphonaceae',\n",
       " 'LAHA01': 'Nostoc',\n",
       " 'NOLK01': 'Nostoc',\n",
       " 'NMQK01': 'Fischerella',\n",
       " 'SZNH01': 'Nostoc',\n",
       " 'NHAB01': 'Fischerella',\n",
       " 'NGJL01': 'Mastigocladus',\n",
       " 'LWTK01': 'Nostoc',\n",
       " 'NMQB01': 'Fischerella',\n",
       " 'JACJSJ01': 'Nostoc',\n",
       " 'LAHH01': 'Nostoc',\n",
       " 'NGZU01': 'Fischerella',\n",
       " 'JACKZS01': 'Nostoc',\n",
       " 'LAGY01': 'Nostoc',\n",
       " 'NHAK01': 'Fischerella',\n",
       " 'AJLL01': 'Fischerella',\n",
       " 'JAHHIM01': 'Fischerella',\n",
       " 'NMQC01': 'Fischerella',\n",
       " 'LAHI01': 'Nostoc',\n",
       " 'JABXKL01': 'Nostoc',\n",
       " 'NGZT01': 'Fischerella',\n",
       " 'JACKZR01': 'Nostoc',\n",
       " 'LAGX01': 'Nostoc',\n",
       " 'NHAJ01': 'Fischerella',\n",
       " 'JAIVFR01': 'Nostoc',\n",
       " 'JACJTU01': 'Nostoc',\n",
       " 'NMQJ01': 'Fischerella',\n",
       " 'NHAC01': 'Fischerella',\n",
       " 'JACKZN01': 'Nostoc',\n",
       " 'VRZE01': 'Nostoc',\n",
       " 'ALVT01': 'Fischerella',\n",
       " 'JACJQM01': 'Nostoc',\n",
       " 'JACJTK01': 'Nostoc',\n",
       " 'JABXKN01': 'Nostoc',\n",
       " 'WVID01': 'Nostoc',\n",
       " 'JAIVFY01': 'Nostoc',\n",
       " 'NMQA01': 'Fischerella',\n",
       " 'NHAH01': 'Fischerella',\n",
       " 'LAGZ01': 'Nostoc',\n",
       " 'NGZV01': 'Fischerella',\n",
       " 'NMQH01': 'Fischerella',\n",
       " 'JAHHHS01': 'Fischerella',\n",
       " 'LAHB01': 'Nostoc',\n",
       " 'JABXKG01': 'Nostoc',\n",
       " 'NHAA01': 'Fischerella',\n",
       " 'DSPK01': 'Fischerella',\n",
       " 'NMQI01': 'Fischerella',\n",
       " 'NRQW01': 'Fischerella',\n",
       " 'JABXKF01': 'Nostoc',\n",
       " 'LAHC01': 'Nostoc',\n",
       " 'JACJTC01': 'Nostoc',\n",
       " 'JACERF01': 'Desmonostoc',\n",
       " 'LAHJ01': 'Nostoc',\n",
       " 'JABXKO01': 'Nostoc',\n",
       " 'JACJSH01': 'Nostoc',\n",
       " 'NHAI01': 'Fischerella',\n",
       " 'JACYMA01': 'Fischerella',\n",
       " 'NGZW01': 'Fischerella'}"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "taxa_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mv ./top_100/NAPS01.fasta ./top_100/Westiellopsis_NAPS01.fasta\n",
      "mv ./top_100/JAHHHK01.fasta ./top_100/Westiellopsis_JAHHHK01.fasta\n",
      "mv ./top_100/NMQE01.fasta ./top_100/Fischerella_NMQE01.fasta\n",
      "mv ./top_100/AJLJ01.fasta ./top_100/Fischerella_AJLJ01.fasta\n",
      "mv ./top_100/NQMD01.fasta ./top_100/Fischerella_NQMD01.fasta\n",
      "mv ./top_100/JACYLX01.fasta ./top_100/Fischerella_JACYLX01.fasta\n",
      "mv ./top_100/MRCA01.fasta ./top_100/Fischerella_MRCA01.fasta\n",
      "mv ./top_100/JADEXT01.fasta ./top_100/Nostoc_JADEXT01.fasta\n",
      "mv ./top_100/NHAE01.fasta ./top_100/Fischerella_NHAE01.fasta\n",
      "mv ./top_100/WBKL01.fasta ./top_100/Nostoc_WBKL01.fasta\n",
      "mv ./top_100/LAHF01.fasta ./top_100/Nostoc_LAHF01.fasta\n",
      "mv ./top_100/JAHHHW01.fasta ./top_100/Nostoc_JAHHHW01.fasta\n",
      "mv ./top_100/JACYLY01.fasta ./top_100/Fischerella_JACYLY01.fasta\n",
      "mv ./top_100/NGZZ01.fasta ./top_100/Fischerella_NGZZ01.fasta\n",
      "mv ./top_100/NHAD01.fasta ./top_100/Fischerella_NHAD01.fasta\n",
      "mv ./top_100/LIRN01.fasta ./top_100/Hapalosiphon_LIRN01.fasta\n",
      "mv ./top_100/ALVS01.fasta ./top_100/Fischerella_ALVS01.fasta\n",
      "mv ./top_100/LAHG01.fasta ./top_100/Nostoc_LAHG01.fasta\n",
      "mv ./top_100/AGIZ01.fasta ./top_100/Fischerella_AGIZ01.fasta\n",
      "mv ./top_100/NGZS01.fasta ./top_100/Fischerella_NGZS01.fasta\n",
      "mv ./top_100/JACKZU01.fasta ./top_100/Nostoc_JACKZU01.fasta\n",
      "mv ./top_100/NQME01.fasta ./top_100/Fischerella_NQME01.fasta\n",
      "mv ./top_100/AJLK01.fasta ./top_100/Fischerella_AJLK01.fasta\n",
      "mv ./top_100/NMQD01.fasta ./top_100/Fischerella_NMQD01.fasta\n",
      "mv ./top_100/NHAF01.fasta ./top_100/Fischerella_NHAF01.fasta\n",
      "mv ./top_100/MNPM02.fasta ./top_100/Mastigocladus_MNPM02.fasta\n",
      "mv ./top_100/NGAA01.fasta ./top_100/Fischerella_NGAA01.fasta\n",
      "mv ./top_100/NGZX01.fasta ./top_100/Fischerella_NGZX01.fasta\n",
      "mv ./top_100/JAECZC01.fasta ./top_100/Amazonocrinis_JAECZC01.fasta\n",
      "mv ./top_100/LAHE01.fasta ./top_100/Nostoc_LAHE01.fasta\n",
      "mv ./top_100/JACVRH01.fasta ./top_100/Nostoc_JACVRH01.fasta\n",
      "mv ./top_100/LXQE01.fasta ./top_100/Nostoc_LXQE01.fasta\n",
      "mv ./top_100/ALVX01.fasta ./top_100/Fischerella_ALVX01.fasta\n",
      "mv ./top_100/NMQF01.fasta ./top_100/Fischerella_NMQF01.fasta\n",
      "mv ./top_100/NMQG01.fasta ./top_100/Fischerella_NMQG01.fasta\n",
      "mv ./top_100/NQMF01.fasta ./top_100/Fischerella_NQMF01.fasta\n",
      "mv ./top_100/JABXKH01.fasta ./top_100/Nostoc_JABXKH01.fasta\n",
      "mv ./top_100/JACJRZ01.fasta ./top_100/Fischerella_JACJRZ01.fasta\n",
      "mv ./top_100/JADEXV01.fasta ./top_100/Nostocales_JADEXV01.fasta\n",
      "mv ./top_100/NHAG01.fasta ./top_100/Fischerella_NHAG01.fasta\n",
      "mv ./top_100/NGZY01.fasta ./top_100/Fischerella_NGZY01.fasta\n",
      "mv ./top_100/LAHD01.fasta ./top_100/Nostoc_LAHD01.fasta\n",
      "mv ./top_100/QLKN01.fasta ./top_100/Hapalosiphonaceae_QLKN01.fasta\n",
      "mv ./top_100/LAHA01.fasta ./top_100/Nostoc_LAHA01.fasta\n",
      "mv ./top_100/NOLK01.fasta ./top_100/Nostoc_NOLK01.fasta\n",
      "mv ./top_100/NMQK01.fasta ./top_100/Fischerella_NMQK01.fasta\n",
      "mv ./top_100/SZNH01.fasta ./top_100/Nostoc_SZNH01.fasta\n",
      "mv ./top_100/NHAB01.fasta ./top_100/Fischerella_NHAB01.fasta\n",
      "mv ./top_100/NGJL01.fasta ./top_100/Mastigocladus_NGJL01.fasta\n",
      "mv ./top_100/LWTK01.fasta ./top_100/Nostoc_LWTK01.fasta\n",
      "mv ./top_100/NMQB01.fasta ./top_100/Fischerella_NMQB01.fasta\n",
      "mv ./top_100/JACJSJ01.fasta ./top_100/Nostoc_JACJSJ01.fasta\n",
      "mv ./top_100/LAHH01.fasta ./top_100/Nostoc_LAHH01.fasta\n",
      "mv ./top_100/NGZU01.fasta ./top_100/Fischerella_NGZU01.fasta\n",
      "mv ./top_100/JACKZS01.fasta ./top_100/Nostoc_JACKZS01.fasta\n",
      "mv ./top_100/LAGY01.fasta ./top_100/Nostoc_LAGY01.fasta\n",
      "mv ./top_100/NHAK01.fasta ./top_100/Fischerella_NHAK01.fasta\n",
      "mv ./top_100/AJLL01.fasta ./top_100/Fischerella_AJLL01.fasta\n",
      "mv ./top_100/JAHHIM01.fasta ./top_100/Fischerella_JAHHIM01.fasta\n",
      "mv ./top_100/NMQC01.fasta ./top_100/Fischerella_NMQC01.fasta\n",
      "mv ./top_100/LAHI01.fasta ./top_100/Nostoc_LAHI01.fasta\n",
      "mv ./top_100/JABXKL01.fasta ./top_100/Nostoc_JABXKL01.fasta\n",
      "mv ./top_100/NGZT01.fasta ./top_100/Fischerella_NGZT01.fasta\n",
      "mv ./top_100/JACKZR01.fasta ./top_100/Nostoc_JACKZR01.fasta\n",
      "mv ./top_100/LAGX01.fasta ./top_100/Nostoc_LAGX01.fasta\n",
      "mv ./top_100/NHAJ01.fasta ./top_100/Fischerella_NHAJ01.fasta\n",
      "mv ./top_100/JAIVFR01.fasta ./top_100/Nostoc_JAIVFR01.fasta\n",
      "mv ./top_100/JACJTU01.fasta ./top_100/Nostoc_JACJTU01.fasta\n",
      "mv ./top_100/NMQJ01.fasta ./top_100/Fischerella_NMQJ01.fasta\n",
      "mv ./top_100/NHAC01.fasta ./top_100/Fischerella_NHAC01.fasta\n",
      "mv ./top_100/JACKZN01.fasta ./top_100/Nostoc_JACKZN01.fasta\n",
      "mv ./top_100/VRZE01.fasta ./top_100/Nostoc_VRZE01.fasta\n",
      "mv ./top_100/ALVT01.fasta ./top_100/Fischerella_ALVT01.fasta\n",
      "mv ./top_100/JACJQM01.fasta ./top_100/Nostoc_JACJQM01.fasta\n",
      "mv ./top_100/JACJTK01.fasta ./top_100/Nostoc_JACJTK01.fasta\n",
      "mv ./top_100/JABXKN01.fasta ./top_100/Nostoc_JABXKN01.fasta\n",
      "mv ./top_100/WVID01.fasta ./top_100/Nostoc_WVID01.fasta\n",
      "mv ./top_100/JAIVFY01.fasta ./top_100/Nostoc_JAIVFY01.fasta\n",
      "mv ./top_100/NMQA01.fasta ./top_100/Fischerella_NMQA01.fasta\n",
      "mv ./top_100/NHAH01.fasta ./top_100/Fischerella_NHAH01.fasta\n",
      "mv ./top_100/LAGZ01.fasta ./top_100/Nostoc_LAGZ01.fasta\n",
      "mv ./top_100/NGZV01.fasta ./top_100/Fischerella_NGZV01.fasta\n",
      "mv ./top_100/NMQH01.fasta ./top_100/Fischerella_NMQH01.fasta\n",
      "mv ./top_100/JAHHHS01.fasta ./top_100/Fischerella_JAHHHS01.fasta\n",
      "mv ./top_100/LAHB01.fasta ./top_100/Nostoc_LAHB01.fasta\n",
      "mv ./top_100/JABXKG01.fasta ./top_100/Nostoc_JABXKG01.fasta\n",
      "mv ./top_100/NHAA01.fasta ./top_100/Fischerella_NHAA01.fasta\n",
      "mv ./top_100/DSPK01.fasta ./top_100/Fischerella_DSPK01.fasta\n",
      "mv ./top_100/NMQI01.fasta ./top_100/Fischerella_NMQI01.fasta\n",
      "mv ./top_100/NRQW01.fasta ./top_100/Fischerella_NRQW01.fasta\n",
      "mv ./top_100/JABXKF01.fasta ./top_100/Nostoc_JABXKF01.fasta\n",
      "mv ./top_100/LAHC01.fasta ./top_100/Nostoc_LAHC01.fasta\n",
      "mv ./top_100/JACJTC01.fasta ./top_100/Nostoc_JACJTC01.fasta\n",
      "mv ./top_100/JACERF01.fasta ./top_100/Desmonostoc_JACERF01.fasta\n",
      "mv ./top_100/LAHJ01.fasta ./top_100/Nostoc_LAHJ01.fasta\n",
      "mv ./top_100/JABXKO01.fasta ./top_100/Nostoc_JABXKO01.fasta\n",
      "mv ./top_100/JACJSH01.fasta ./top_100/Nostoc_JACJSH01.fasta\n",
      "mv ./top_100/NHAI01.fasta ./top_100/Fischerella_NHAI01.fasta\n",
      "mv ./top_100/JACYMA01.fasta ./top_100/Fischerella_JACYMA01.fasta\n",
      "mv ./top_100/NGZW01.fasta ./top_100/Fischerella_NGZW01.fasta\n"
     ]
    }
   ],
   "source": [
    "fasta_glob = glob.glob(\"./top_100/*.fasta\")\n",
    "\n",
    "for fasta_path in fasta_glob:\n",
    "    strain = os.path.basename(fasta_path.split('.fasta')[0])\n",
    "    print('mv %s ./top_100/%s_%s.fasta'%(fasta_path,taxa_dict[strain],strain))\n",
    "    subprocess.call('mv %s ./top_100/%s_%s.fasta'%(fasta_path,taxa_dict[strain],strain),shell=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mv /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/Fischerella_CENA71.fasta ./top_100/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Parsing input and generating reference (M. producens PAL)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Input: 21 housekeeping genes from complete genomes published here, [*Moorea* PNAS paper](http://www.pnas.org/content/114/12/3198.short)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "      21\r\n"
     ]
    }
   ],
   "source": [
    "!cp ./inputs/hk_genes_calteau_pnas/* ./pnas-phylogenomics-top_100/\n",
    "!ls ./pnas-phylogenomics-top_100/*.faa | wc -l"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "input_list = glob.glob(\"/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/*.faa\")\n",
    "output_handle = open(\"/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/PAL_core_genes.faa\", \"w\")\n",
    "\n",
    "count = 0\n",
    "for item in natsorted(input_list):\n",
    "    input_handle  = open(item,\"r\")\n",
    "    new_id = os.path.basename(item).split(\"_scaffold\")[0]\n",
    "    for seq_record in SeqIO.parse(input_handle, \"fasta\"):\n",
    "        if \"Moorea_producens_PAL_15AUG08-1\" in seq_record.id:\n",
    "            count += 1\n",
    "            new_id = str(os.path.basename(item).split(\".faa\")[0]) + str(\"_Moorea_producens_PAL15AUG081\")\n",
    "            output_handle.write(\">%s %s\\n%s\\n\" % (\n",
    "               new_id,\n",
    "               \"\",\n",
    "               seq_record.seq))\n",
    "    input_handle.close()\n",
    "\n",
    "output_handle.close()\n",
    "\n",
    "assert(len(input_list) == count)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[31m./pnas-phylogenomics-top_100/10_rpsB.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/11_rpsC.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/12_rpsD.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/13_rpsE.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/14_rpsG.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/15_rpsH.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/16_rpsI.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/17_rpsK.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/18_rpsL.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/19_rpsM.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/1_rplA.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/20_rpsO.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/21_rpsQ.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/2_rplC.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/3_rplE.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/4_rplF.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/5_rplK.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/6_rplM.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/7_rplN.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/8_rplR.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m./pnas-phylogenomics-top_100/9_rplV.faa\u001b[m\u001b[m\r\n",
      "./pnas-phylogenomics-top_100/PAL_core_genes.faa\r\n"
     ]
    }
   ],
   "source": [
    "!ls ./pnas-phylogenomics-top_100/*"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Annotating genomes with Prokka"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir ./prokka"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [],
   "source": [
    "#conda install -c bioconda prokka"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:02:10] This is prokka 1.14.6\\n[10:02:10] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:02:10] Homepage is https://github.com/tseemann/prokka\\n[10:02:10] Local time is Mon Feb 21 10:02:10 2022\\n[10:02:10] You are tiagoferreiraleao\\n[10:02:10] Operating system is darwin\\n[10:02:10] You have BioPerl 1.007002\\n[10:02:10] System has 4 cores.\\n[10:02:10] Option --cpu asked for 12 cores, but system only has 4\\n[10:02:10] Will use maximum of 4 cores.\\n[10:02:10] Annotating as >>> Bacteria <<<\\n[10:02:10] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQG01.fasta' contents.\\n[10:02:11] Setting --locustag KLMOHPHK from MD5 45681914da976b78ddc3663dbebd6d48\\n[10:02:11] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQG01/\\n[10:02:11] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQG01\\\\/\\n[10:02:11] Using filename prefix: Fischerella_NMQG01.XXX\\n[10:02:11] Setting HMMER_NCPU=1\\n[10:02:11] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQG01//Fischerella_NMQG01.log\\n[10:02:11] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQG01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQG01/ --prefix Fischerella_NMQG01 --cpus 12\\n[10:02:11] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:02:11] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:02:11] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:02:11] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:02:11] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:02:11] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:02:11] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:02:11] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:02:11] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:02:11] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:02:11] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:02:11] Looking for 'egrep' - found /usr/bin/egrep\\n[10:02:11] Looking for 'find' - found /usr/bin/find\\n[10:02:11] Looking for 'grep' - found /usr/bin/grep\\n[10:02:11] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:02:11] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:02:11] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:02:11] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:02:11] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:02:11] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:02:11] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:02:11] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:02:11] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:02:11] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:02:12] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:02:12] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:02:12] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:02:12] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:02:12] Looking for 'sed' - found /usr/bin/sed\\n[10:02:12] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:02:12] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:02:12] Using genetic code table 11.\\n[10:02:12] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQG01.fasta\\n[10:02:12] Wrote 364 contigs totalling 5321124 bp.\\n[10:02:12] Predicting tRNAs and tmRNAs\\n[10:02:12] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQG01\\\\/\\\\/Fischerella_NMQG01\\\\.fna\\n[10:02:14] 1 tRNA-Leu [8476,8558] 35 (cag)\\n[10:02:14] 1 tRNA-Phe c[41576,41648] 34 (gaa)\\n[10:02:14] 2 tRNA-Val [49046,49118] 34 (tac)\\n[10:02:14] 1 tRNA-Met c[11863,11938] 36 (cat)\\n[10:02:14] 1 tRNA-Met c[3794,3866] 34 (cat)\\n[10:02:14] 1 tRNA-Leu [15859,15942] 36 (tag)\\n[10:02:14] 1 tRNA-Pro [10176,10249] 35 (ggg)\\n[10:02:14] 1 tRNA-Thr [43151,43222] 33 (tgt)\\n[10:02:14] 1 tRNA-Asn c[20413,20486] 34 (gtt)\\n[10:02:14] 1 tRNA-Leu [18821,18905] 35 (caa)\\n[10:02:14] 1 tRNA-Trp c[6388,6461] 34 (cca)\\n[10:02:14] 1 tmRNA c[1308,1694] 100,150 ANNIVKFARKEALVAA*\\n[10:02:14] 1 tRNA-Gly c[42142,42213] 33 (gcc)\\n[10:02:14] 1 tRNA-Gly [10569,10640] 33 (ccc)\\n[10:02:14] 1 tRNA-Gln c[5754,5825] 33 (ttg)\\n[10:02:14] 1 tRNA-Met [1675,1751] 35 (cat)\\n[10:02:14] 1 tRNA-Arg [5945,6018] 35 (ccg)\\n[10:02:14] 1 tRNA-His [1959,2032] 34 (gtg)\\n[10:02:14] 1 tRNA-Ser c[43588,43674] 36 (cga)\\n[10:02:14] 1 tRNA-Ser c[16651,16744] 36 (gct)\\n[10:02:14] 1 tRNA-Asp c[46177,46250] 35 (gtc)\\n[10:02:14] 1 tRNA-Ala [10098,10172] 35 (cgc)\\n[10:02:14] 1 tRNA-Leu c[32756,32838] 36 (gag)\\n[10:02:14] 1 tRNA-Gly c[4393,4463] 33 (tcc)\\n[10:02:14] 1 tRNA-Arg [18685,18760] 36 (cct)\\n[10:02:14] 1 tRNA-Ser [15516,15602] 36 (tga)\\n[10:02:14] 1 tRNA-Pro c[331,405] 35 (tgg)\\n[10:02:14] 1 tRNA-Arg [837,909] 34 (tcg)\\n[10:02:14] 2 tRNA-Asn [1581,1654] 34 (att)\\n[10:02:14] 1 tRNA-Arg c[43107,43180] 35 (acg)\\n[10:02:14] 2 tRNA-Ala c[48072,48145] 34 (ggc)\\n[10:02:14] 3 tRNA-Thr c[50195,50266] 33 (cgt)\\n[10:02:14] 1 tRNA-Ser [21772,21860] 36 (gga)\\n[10:02:14] 1 tRNA-Cys c[99,170] 33 (gca)\\n[10:02:14] 1 tRNA-Arg [10501,10575] 35 (tct)\\n[10:02:14] 1 tRNA-Tyr [13166,13250] 35 (gta)\\n[10:02:14] 2 tRNA-Thr [13314,13385] 33 (ggt)\\n[10:02:14] 1 tRNA-Glu [7556,7628] 35 (ttc)\\n[10:02:14] 1 tRNA-Val c[40558,40631] 35 (gac)\\n[10:02:14] 1 tRNA-Gln c[18259,18329] 32 (ttg)\\n[10:02:14] 1 tRNA-Pro c[4792,4865] 35 (cgg)\\n[10:02:14] 1 tRNA-Lys [23714,23787] 34 (ttt)\\n[10:02:14] Found 42 tRNAs\\n[10:02:14] Predicting Ribosomal RNAs\\n[10:02:14] Running Barrnap with 4 threads\\n[10:02:15] 1 NMQG01000015.1 4 5S ribosomal RNA\\n[10:02:15] 2 NMQG01000015.1 233 23S ribosomal RNA\\n[10:02:15] 3 NMQG01000039.1 5 16S ribosomal RNA\\n[10:02:15] Found 3 rRNAs\\n[10:02:15] Skipping ncRNA search, enable with --rfam if desired.\\n[10:02:15] Total of 44 tRNA + rRNA features\\n[10:02:15] Searching for CRISPR repeats\\n[10:02:15] CRISPR1 NMQG01000013.1 15436 with 3 spacers\\n[10:02:15] CRISPR2 NMQG01000022.1 39 with 118 spacers\\n[10:02:15] CRISPR3 NMQG01000054.1 19672 with 35 spacers\\n[10:02:15] CRISPR4 NMQG01000054.1 22535 with 42 spacers\\n[10:02:15] CRISPR5 NMQG01000136.1 42 with 28 spacers\\n[10:02:15] CRISPR6 NMQG01000140.1 19188 with 255 spacers\\n[10:02:16] CRISPR7 NMQG01000158.1 1711 with 123 spacers\\n[10:02:16] Found 7 CRISPRs\\n[10:02:16] Predicting coding sequences\\n[10:02:16] Contigs total 5321124 bp, so using single mode\\n[10:02:16] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQG01\\\\/\\\\/Fischerella_NMQG01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:02:20] Excluding CDS which overlaps existing RNA (rRNA) at NMQG01000015.1:518..694 on + strand\\n[10:02:20] Excluding CDS which overlaps existing RNA (rRNA) at NMQG01000015.1:2480..2659 on - strand\\n[10:02:20] Excluding CDS which overlaps existing RNA (repeat_region) at NMQG01000022.1:2418..2741 on + strand\\n[10:02:20] Excluding CDS which overlaps existing RNA (tRNA) at NMQG01000040.1:19173..20612 on - strand\\n[10:02:21] Excluding CDS which overlaps existing RNA (tRNA) at NMQG01000092.1:43558..43686 on - strand\\n[10:02:21] Excluding CDS which overlaps existing RNA (tRNA) at NMQG01000117.1:46212..46607 on + strand\\n[10:02:21] Excluding CDS which overlaps existing RNA (repeat_region) at NMQG01000140.1:21051..21239 on - strand\\n[10:02:21] Excluding CDS which overlaps existing RNA (repeat_region) at NMQG01000140.1:27272..27373 on - strand\\n[10:02:21] Excluding CDS which overlaps existing RNA (tRNA) at NMQG01000233.1:43064..43735 on + strand\\n[10:02:22] Found 4504 CDS\\n[10:02:22] Connecting features back to sequences\\n[10:02:22] Not using genus-specific database. Try --usegenus to enable it.\\n[10:02:22] Annotating CDS, please be patient.\\n[10:02:22] Will use 4 CPUs for similarity searching.\\n[10:02:23] There are still 4504 unannotated CDS left (started with 4504)\\n[10:02:23] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:02:23] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQG01\\\\/\\\\/Fischerella_NMQG01\\\\.IS\\\\.tmp\\\\.96377\\\\.faa | parallel --gnu --plain -j 4 --block 176575 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQG01\\\\/\\\\/Fischerella_NMQG01\\\\.IS\\\\.tmp\\\\.96377\\\\.blast 2> /dev/null\\n[10:02:40] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQG01\\\\/\\\\/Fischerella_NMQG01\\\\.IS\\\\.tmp\\\\.96377\\\\.faa | parallel --gnu --plain -j 4 --block 176575 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQG01\\\\/\\\\/Fischerella_NMQG01\\\\.IS\\\\.tmp\\\\.96377\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:02:41] This is prokka 1.14.6\\n[10:02:41] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:02:41] Homepage is https://github.com/tseemann/prokka\\n[10:02:41] Local time is Mon Feb 21 10:02:41 2022\\n[10:02:41] You are tiagoferreiraleao\\n[10:02:41] Operating system is darwin\\n[10:02:41] You have BioPerl 1.007002\\n[10:02:41] System has 4 cores.\\n[10:02:41] Option --cpu asked for 12 cores, but system only has 4\\n[10:02:41] Will use maximum of 4 cores.\\n[10:02:41] Annotating as >>> Bacteria <<<\\n[10:02:41] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NQMF01.fasta' contents.\\n[10:02:41] Setting --locustag NOGBNIPK from MD5 780b729497921e37e556a305abdebbb3\\n[10:02:41] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NQMF01/\\n[10:02:41] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQMF01\\\\/\\n[10:02:41] Using filename prefix: Fischerella_NQMF01.XXX\\n[10:02:41] Setting HMMER_NCPU=1\\n[10:02:41] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NQMF01//Fischerella_NQMF01.log\\n[10:02:41] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NQMF01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NQMF01/ --prefix Fischerella_NQMF01 --cpus 12\\n[10:02:41] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:02:41] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:02:41] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:02:41] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:02:41] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:02:41] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:02:41] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:02:41] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:02:41] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:02:41] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:02:41] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:02:41] Looking for 'egrep' - found /usr/bin/egrep\\n[10:02:41] Looking for 'find' - found /usr/bin/find\\n[10:02:41] Looking for 'grep' - found /usr/bin/grep\\n[10:02:41] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:02:41] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:02:41] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:02:41] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:02:41] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:02:41] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:02:42] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:02:42] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:02:42] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:02:42] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:02:42] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:02:42] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:02:42] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:02:42] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:02:42] Looking for 'sed' - found /usr/bin/sed\\n[10:02:42] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:02:42] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:02:42] Using genetic code table 11.\\n[10:02:42] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NQMF01.fasta\\n[10:02:43] Wrote 1628 contigs totalling 4837129 bp.\\n[10:02:43] Predicting tRNAs and tmRNAs\\n[10:02:43] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQMF01\\\\/\\\\/Fischerella_NQMF01\\\\.fna\\n[10:02:43] 1 tRNA-Ala [2321,2395] 35 (cgc)\\n[10:02:43] 1 tRNA-Ser [2728,2821] 36 (gct)\\n[10:02:43] 1 tRNA-Pro c[574,647] 35 (cgg)\\n[10:02:43] 1 tRNA-Val [1263,1335] 34 (tac)\\n[10:02:43] 1 tRNA-Leu c[2303,2385] 36 (gag)\\n[10:02:43] 1 tRNA-Thr [1237,1308] 33 (tgt)\\n[10:02:43] 1 tRNA-Gln [3643,3714] 33 (ttg)\\n[10:02:43] 1 tRNA-Ser c[1673,1761] 36 (gga)\\n[10:02:43] 1 tRNA-Asn c[4149,4222] 34 (att)\\n[10:02:43] 2 tRNA-Arg c[4893,4965] 34 (tcg)\\n[10:02:43] 1 tRNA-Ser [5667,5753] 36 (tga)\\n[10:02:43] 1 tRNA-Phe [5700,5772] 34 (gaa)\\n[10:02:43] 1 tRNA-Pro [524,597] 35 (ggg)\\n[10:02:43] 1 tRNA-Arg c[3660,3735] 36 (cct)\\n[10:02:43] 1 tRNA-Leu [2825,2907] 35 (cag)\\n[10:02:43] 1 tRNA-Met c[3356,3432] 35 (cat)\\n[10:02:43] 2 tRNA-Val [6377,6450] 35 (gac)\\n[10:02:43] 1 tRNA-Met c[3388,3463] 36 (cat)\\n[10:02:43] 1 tRNA-Arg c[2652,2725] 35 (ccg)\\n[10:02:43] 1 tRNA-Gly c[4385,4455] 33 (tcc)\\n[10:02:43] 1 tRNA-Ala c[3009,3082] 34 (ggc)\\n[10:02:43] 2 tRNA-Thr c[5114,5185] 33 (cgt)\\n[10:02:43] 1 tRNA-Asn [2390,2463] 34 (gtt)\\n[10:02:43] 1 tRNA-Asp [1340,1413] 35 (gtc)\\n[10:02:44] 1 tRNA-Thr c[2029,2100] 33 (ggt)\\n[10:02:44] 1 tRNA-Trp [1874,1947] 34 (cca)\\n[10:02:44] 1 tRNA-Leu [2533,2617] 35 (caa)\\n[10:02:44] 1 tRNA-Arg c[6327,6400] 35 (acg)\\n[10:02:44] 1 tRNA-Gly c[307,378] 33 (gcc)\\n[10:02:44] 1 tRNA-Gly c[1518,1589] 33 (ccc)\\n[10:02:44] 1 tRNA-Lys c[2624,2697] 34 (ttt)\\n[10:02:44] 1 tRNA-Arg [589,663] 35 (tct)\\n[10:02:44] 1 tRNA-Cys [3962,4033] 33 (gca)\\n[10:02:44] 1 tRNA-Pro c[394,468] 35 (tgg)\\n[10:02:44] 1 tRNA-His c[966,1039] 34 (gtg)\\n[10:02:44] 1 tmRNA c[248,634] 100,150 ANNIVKFARKEALVAA*\\n[10:02:44] 1 tRNA-Glu [4276,4348] 35 (ttc)\\n[10:02:44] 1 tRNA-Leu [460,543] 36 (tag)\\n[10:02:44] Found 38 tRNAs\\n[10:02:44] Predicting Ribosomal RNAs\\n[10:02:44] Running Barrnap with 4 threads\\n[10:02:45] 1 NQMF01000635.1 1 23S ribosomal RNA (partial)\\n[10:02:45] 2 NQMF01000635.1 1101 5S ribosomal RNA\\n[10:02:45] Found 2 rRNAs\\n[10:02:45] Skipping ncRNA search, enable with --rfam if desired.\\n[10:02:45] Total of 39 tRNA + rRNA features\\n[10:02:45] Searching for CRISPR repeats\\n[10:02:46] CRISPR1 NQMF01000265.1 3526 with 7 spacers\\n[10:02:46] CRISPR2 NQMF01000336.1 39 with 13 spacers\\n[10:02:46] CRISPR3 NQMF01000407.1 2088 with 7 spacers\\n[10:02:46] CRISPR4 NQMF01000770.1 61 with 15 spacers\\n[10:02:46] CRISPR5 NQMF01000770.1 1230 with 18 spacers\\n[10:02:46] CRISPR6 NQMF01001440.1 34 with 3 spacers\\n[10:02:46] Found 6 CRISPRs\\n[10:02:46] Predicting coding sequences\\n[10:02:46] Contigs total 4837129 bp, so using single mode\\n[10:02:46] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQMF01\\\\/\\\\/Fischerella_NQMF01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:02:50] Excluding CDS which overlaps existing RNA (repeat_region) at NQMF01000265.1:3346..3570 on - strand\\n[10:02:50] Excluding CDS which overlaps existing RNA (tRNA) at NQMF01000473.1:2264..3703 on + strand\\n[10:02:50] Excluding CDS which overlaps existing RNA (rRNA) at NQMF01000635.1:522..698 on - strand\\n[10:02:50] Excluding CDS which overlaps existing RNA (tRNA) at NQMF01000820.1:6284..6955 on + strand\\n[10:02:51] Found 3987 CDS\\n[10:02:51] Connecting features back to sequences\\n[10:02:51] Not using genus-specific database. Try --usegenus to enable it.\\n[10:02:51] Annotating CDS, please be patient.\\n[10:02:51] Will use 4 CPUs for similarity searching.\\n[10:02:52] There are still 3987 unannotated CDS left (started with 3987)\\n[10:02:52] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:02:52] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQMF01\\\\/\\\\/Fischerella_NQMF01\\\\.IS\\\\.tmp\\\\.96489\\\\.faa | parallel --gnu --plain -j 4 --block 142999 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQMF01\\\\/\\\\/Fischerella_NQMF01\\\\.IS\\\\.tmp\\\\.96489\\\\.blast 2> /dev/null\\n[10:03:07] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQMF01\\\\/\\\\/Fischerella_NQMF01\\\\.IS\\\\.tmp\\\\.96489\\\\.faa | parallel --gnu --plain -j 4 --block 142999 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQMF01\\\\/\\\\/Fischerella_NQMF01\\\\.IS\\\\.tmp\\\\.96489\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:03:07] This is prokka 1.14.6\\n[10:03:07] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:03:07] Homepage is https://github.com/tseemann/prokka\\n[10:03:07] Local time is Mon Feb 21 10:03:07 2022\\n[10:03:07] You are tiagoferreiraleao\\n[10:03:07] Operating system is darwin\\n[10:03:07] You have BioPerl 1.007002\\n[10:03:08] System has 4 cores.\\n[10:03:08] Option --cpu asked for 12 cores, but system only has 4\\n[10:03:08] Will use maximum of 4 cores.\\n[10:03:08] Annotating as >>> Bacteria <<<\\n[10:03:08] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostocales_JADEXV01.fasta' contents.\\n[10:03:08] Setting --locustag BMHCBODI from MD5 b61cb8d2e719a07ffcdca4ea4cdda202\\n[10:03:08] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostocales_JADEXV01/\\n[10:03:08] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostocales_JADEXV01\\\\/\\n[10:03:08] Using filename prefix: Nostocales_JADEXV01.XXX\\n[10:03:08] Setting HMMER_NCPU=1\\n[10:03:08] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostocales_JADEXV01//Nostocales_JADEXV01.log\\n[10:03:08] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostocales_JADEXV01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostocales_JADEXV01/ --prefix Nostocales_JADEXV01 --cpus 12\\n[10:03:08] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:03:08] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:03:08] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:03:08] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:03:08] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:03:08] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:03:08] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:03:08] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:03:08] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:03:08] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:03:08] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:03:08] Looking for 'egrep' - found /usr/bin/egrep\\n[10:03:08] Looking for 'find' - found /usr/bin/find\\n[10:03:08] Looking for 'grep' - found /usr/bin/grep\\n[10:03:08] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:03:08] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:03:08] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:03:08] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:03:08] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:03:08] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:03:08] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:03:08] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:03:08] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:03:08] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:03:09] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:03:09] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:03:09] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:03:09] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:03:09] Looking for 'sed' - found /usr/bin/sed\\n[10:03:09] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:03:09] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:03:09] Using genetic code table 11.\\n[10:03:09] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostocales_JADEXV01.fasta\\n[10:03:09] Wrote 552 contigs totalling 8381711 bp.\\n[10:03:09] Predicting tRNAs and tmRNAs\\n[10:03:09] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostocales_JADEXV01\\\\/\\\\/Nostocales_JADEXV01\\\\.fna\\n[10:03:12] 1 tRNA-His c[9627,9700] 34 (gtg)\\n[10:03:12] 2 tRNA-Met c[45068,45141] 33 (cat)\\n[10:03:12] 1 tRNA-Gly [54118,54189] 33 (gcc)\\n[10:03:12] 1 tRNA-Gly c[59980,60051] 33 (ccc)\\n[10:03:12] 1 tRNA-Met [31067,31141] 35 (cat)\\n[10:03:12] 2 tRNA-Ala [32885,32959] 35 (ggc)\\n[10:03:12] 3 tRNA-Arg c[34064,34136] 34 (ccg)\\n[10:03:12] 1 tRNA-Lys [15510,15582] 33 (ctt)\\n[10:03:12] 2 tRNA-Asp c[40299,40372] 35 (gtc)\\n[10:03:12] 1 tRNA-Cys [54691,54761] 33 (gca)\\n[10:03:12] 1 tmRNA c[1930,2317] 100,150 ANNIVNFARKDALVAA*\\n[10:03:12] 2 tRNA-Phe [27283,27355] 34 (gaa)\\n[10:03:12] 3 tRNA-Leu [41827,41907] 34 (cag)\\n[10:03:12] 1 tRNA-Val [4424,4496] 33 (cac)\\n[10:03:12] 1 tRNA-Pro [1286,1359] 35 (tgg)\\n[10:03:12] 1 tRNA-Ser c[59,145] 36 (tga)\\n[10:03:12] 2 tRNA-Tyr [2445,2530] 35 (gta)\\n[10:03:12] 3 tRNA-Thr [2618,2689] 33 (ggt)\\n[10:03:12] 4 tRNA-Arg c[3542,3616] 35 (tct)\\n[10:03:12] 1 tRNA-Val c[15743,15816] 35 (gac)\\n[10:03:12] 1 tRNA-Thr [18100,18171] 33 (cgt)\\n[10:03:12] 1 tRNA-Gly c[2041,2111] 33 (tcc)\\n[10:03:12] 1 tRNA-Lys [7288,7359] 33 (ttt)\\n[10:03:12] 2 tRNA-Arg c[13724,13799] 36 (cct)\\n[10:03:12] 1 tRNA-Trp [2059,2132] 34 (cca)\\n[10:03:12] 1 tRNA-Glu [2252,2325] 35 (ttc)\\n[10:03:12] 2 tRNA-Asp [8771,8846] 36 (gtc)\\n[10:03:12] 3 tRNA-Cys [8975,9047] 35 (gca)\\n[10:03:12] 4 tRNA-Met [9058,9133] 34 (cat)\\n[10:03:12] tRNA [9136,9210] is a pseudo/wacky gene - skipping.\\n[10:03:12] 6 tRNA-Glu [9273,9346] 35 (ttc)\\n[10:03:12] 7 tRNA-Ser [9458,9549] 36 (gct)\\n[10:03:12] 8 tRNA-Arg [9552,9628] 36 (acg)\\n[10:03:12] 9 tRNA-Asn [9813,9887] 33 (gtt)\\n[10:03:12] 10 tRNA-Gln [9892,9966] 35 (ctg)\\n[10:03:12] 11 tRNA-Gln [9969,10044] 34 (ttg)\\n[10:03:12] 12 tRNA-Leu [10160,10237] 35 (taa)\\n[10:03:12] 13 tRNA-Pro [10239,10311] 34 (tgg)\\n[10:03:12] 14 tRNA-Leu [10315,10389] 35 (cag)\\n[10:03:12] 15 tRNA-Leu [10393,10466] 34 (gag)\\n[10:03:12] 16 tRNA-Pro [10476,10548] 34 (cgg)\\n[10:03:12] 17 tRNA-Leu [10551,10625] 35 (tag)\\n[10:03:12] 18 tRNA-Met [10635,10711] 34 (cat)\\n[10:03:12] 19 tRNA-Thr [10720,10792] 34 (tgt)\\n[10:03:12] 20 tRNA-Gly [10797,10868] 33 (tcc)\\n[10:03:12] 21 tRNA-Val [10954,11029] 34 (tac)\\n[10:03:12] 22 tRNA-Arg [11049,11124] 35 (tct)\\n[10:03:12] 23 tRNA-Ala [11129,11201] 34 (ggc)\\n[10:03:12] 24 tRNA-Ala [11211,11285] 35 (tgc)\\n[10:03:12] 25 tRNA-His [11291,11366] 35 (gtg)\\n[10:03:12] 26 tRNA-Ser [11372,11455] 34 (tga)\\n[10:03:12] 27 tRNA-Lys [11610,11681] 33 (ttt)\\n[10:03:12] 28 tRNA-Ser [15787,15878] 35 (gct)\\n[10:03:12] 1 tRNA-Thr c[4770,4841] 33 (tgt)\\n[10:03:12] 1 tRNA-Arg [12616,12689] 35 (acg)\\n[10:03:12] 1 tRNA-Leu [6386,6468] 35 (tag)\\n[10:03:12] 1 tRNA-Pro [6856,6929] 35 (cgg)\\n[10:03:12] 1 tRNA-Ser [8217,8303] 36 (cga)\\n[10:03:12] 1 tRNA-Ala [5599,5673] 35 (tgc)\\n[10:03:12] 1 tRNA-Met [4310,4386] 35 (cat)\\n[10:03:12] 1 tRNA-Met [2065,2140] 34 (cat)\\n[10:03:12] 2 tRNA-Met [2141,2212] 32 (cat)\\n[10:03:12] 1 tRNA-Ser c[5810,5896] 36 (gga)\\n[10:03:12] 1 tRNA-Leu [2567,2651] 35 (caa)\\n[10:03:12] 2 tRNA-Gln [4467,4540] 34 (ttg)\\n[10:03:12] 1 tRNA-Ala c[956,1028] 34 (cgc)\\n[10:03:12] 1 tRNA-Asn [2696,2767] 33 (gtt)\\n[10:03:12] 1 tRNA-Leu [1851,1933] 36 (gag)\\n[10:03:12] 1 tRNA-Val [1658,1730] 34 (tac)\\n[10:03:12] 1 tRNA-Pro c[201,274] 35 (ggg)\\n[10:03:12] Found 69 tRNAs\\n[10:03:12] Predicting Ribosomal RNAs\\n[10:03:12] Running Barrnap with 4 threads\\n[10:03:14] 1 JADEXV010000552.1 5 16S ribosomal RNA\\n[10:03:14] Found 1 rRNAs\\n[10:03:14] Skipping ncRNA search, enable with --rfam if desired.\\n[10:03:14] Total of 69 tRNA + rRNA features\\n[10:03:14] Searching for CRISPR repeats\\n[10:03:14] CRISPR1 JADEXV010000001.1 20354 with 23 spacers\\n[10:03:14] CRISPR2 JADEXV010000027.1 2142 with 21 spacers\\n[10:03:14] CRISPR3 JADEXV010000077.1 20731 with 9 spacers\\n[10:03:14] CRISPR4 JADEXV010000145.1 7037 with 9 spacers\\n[10:03:14] CRISPR5 JADEXV010000172.1 6571 with 5 spacers\\n[10:03:14] CRISPR6 JADEXV010000208.1 8160 with 19 spacers\\n[10:03:14] CRISPR7 JADEXV010000250.1 152 with 6 spacers\\n[10:03:15] CRISPR8 JADEXV010000395.1 2751 with 5 spacers\\n[10:03:15] Found 8 CRISPRs\\n[10:03:15] Predicting coding sequences\\n[10:03:15] Contigs total 8381711 bp, so using single mode\\n[10:03:15] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostocales_JADEXV01\\\\/\\\\/Nostocales_JADEXV01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:03:21] Excluding CDS which overlaps existing RNA (tRNA) at JADEXV010000001.1:9655..10209 on + strand\\n[10:03:21] Excluding CDS which overlaps existing RNA (repeat_region) at JADEXV010000001.1:21098..21277 on + strand\\n[10:03:22] Excluding CDS which overlaps existing RNA (repeat_region) at JADEXV010000027.1:3240..3533 on - strand\\n[10:03:23] Excluding CDS which overlaps existing RNA (tRNA) at JADEXV010000097.1:9873..10280 on - strand\\n[10:03:23] Excluding CDS which overlaps existing RNA (tRNA) at JADEXV010000097.1:10986..11276 on - strand\\n[10:03:24] Excluding CDS which overlaps existing RNA (repeat_region) at JADEXV010000172.1:6366..6920 on + strand\\n[10:03:24] Excluding CDS which overlaps existing RNA (repeat_region) at JADEXV010000395.1:1024..2784 on + strand\\n[10:03:25] Excluding CDS which overlaps existing RNA (rRNA) at JADEXV010000552.1:1136..1450 on + strand\\n[10:03:25] Found 7011 CDS\\n[10:03:25] Connecting features back to sequences\\n[10:03:25] Not using genus-specific database. Try --usegenus to enable it.\\n[10:03:25] Annotating CDS, please be patient.\\n[10:03:25] Will use 4 CPUs for similarity searching.\\n[10:03:26] There are still 7011 unannotated CDS left (started with 7011)\\n[10:03:26] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:03:26] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostocales_JADEXV01\\\\/\\\\/Nostocales_JADEXV01\\\\.IS\\\\.tmp\\\\.96601\\\\.faa | parallel --gnu --plain -j 4 --block 279701 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostocales_JADEXV01\\\\/\\\\/Nostocales_JADEXV01\\\\.IS\\\\.tmp\\\\.96601\\\\.blast 2> /dev/null\\n[10:03:36] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostocales_JADEXV01\\\\/\\\\/Nostocales_JADEXV01\\\\.IS\\\\.tmp\\\\.96601\\\\.faa | parallel --gnu --plain -j 4 --block 279701 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostocales_JADEXV01\\\\/\\\\/Nostocales_JADEXV01\\\\.IS\\\\.tmp\\\\.96601\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:03:36] This is prokka 1.14.6\\n[10:03:36] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:03:36] Homepage is https://github.com/tseemann/prokka\\n[10:03:36] Local time is Mon Feb 21 10:03:36 2022\\n[10:03:36] You are tiagoferreiraleao\\n[10:03:36] Operating system is darwin\\n[10:03:36] You have BioPerl 1.007002\\n[10:03:36] System has 4 cores.\\n[10:03:36] Option --cpu asked for 12 cores, but system only has 4\\n[10:03:36] Will use maximum of 4 cores.\\n[10:03:36] Annotating as >>> Bacteria <<<\\n[10:03:36] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JACJRZ01.fasta' contents.\\n[10:03:36] Setting --locustag COMBBOFO from MD5 c86bb8f8aa0885c0dc9d51118a17400a\\n[10:03:36] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JACJRZ01/\\n[10:03:36] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACJRZ01\\\\/\\n[10:03:36] Using filename prefix: Fischerella_JACJRZ01.XXX\\n[10:03:36] Setting HMMER_NCPU=1\\n[10:03:36] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JACJRZ01//Fischerella_JACJRZ01.log\\n[10:03:36] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JACJRZ01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JACJRZ01/ --prefix Fischerella_JACJRZ01 --cpus 12\\n[10:03:36] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:03:36] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:03:36] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:03:36] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:03:36] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:03:36] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:03:37] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:03:37] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:03:37] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:03:37] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:03:37] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:03:37] Looking for 'egrep' - found /usr/bin/egrep\\n[10:03:37] Looking for 'find' - found /usr/bin/find\\n[10:03:37] Looking for 'grep' - found /usr/bin/grep\\n[10:03:37] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:03:37] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:03:37] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:03:37] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:03:37] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:03:37] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:03:37] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:03:37] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:03:37] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:03:37] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:03:37] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:03:37] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:03:37] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:03:37] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:03:37] Looking for 'sed' - found /usr/bin/sed\\n[10:03:37] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:03:37] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:03:37] Using genetic code table 11.\\n[10:03:37] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JACJRZ01.fasta\\n[10:03:38] Wrote 132 contigs totalling 6910048 bp.\\n[10:03:38] Predicting tRNAs and tmRNAs\\n[10:03:38] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACJRZ01\\\\/\\\\/Fischerella_JACJRZ01\\\\.fna\\n[10:03:40] 1 tRNA-Val [316262,316336] 35 (tac)\\n[10:03:40] 1 tRNA-Gly [99494,99563] 32 (tcc)\\n[10:03:40] 2 tRNA-His c[227741,227814] 34 (gtg)\\n[10:03:40] 1 tRNA-Ser [19140,19228] 36 (gga)\\n[10:03:40] 2 tRNA-Leu c[84009,84091] 36 (gag)\\n[10:03:40] 3 tRNA-Trp c[233252,233325] 34 (cca)\\n[10:03:40] 1 tRNA-Leu c[34557,34639] 35 (tag)\\n[10:03:40] 1 tRNA-Thr [83524,83595] 33 (cgt)\\n[10:03:40] 2 tRNA-Ala [84937,85009] 34 (ggc)\\n[10:03:40] 3 tRNA-Arg [92257,92330] 35 (acg)\\n[10:03:40] 4 tRNA-Ser c[102175,102268] 36 (gct)\\n[10:03:40] 1 tRNA-Thr c[73262,73333] 33 (tgt)\\n[10:03:40] 1 tRNA-Glu [92901,92973] 35 (ttc)\\n[10:03:40] 2 tRNA-Gly [114646,114719] 35 (tcc)\\n[10:03:40] 3 tRNA-Asn [115187,115260] 34 (att)\\n[10:03:40] 1 tRNA-Ser [4635,4721] 36 (cga)\\n[10:03:40] 2 tRNA-Gly [13318,13389] 33 (gcc)\\n[10:03:40] 1 tRNA-Arg c[72923,72997] 35 (tct)\\n[10:03:40] 1 tmRNA [84133,84519] 100,153 ANNIVKFARKEALVAA**\\n[10:03:40] 1 tRNA-Gln c[19769,19840] 33 (ttg)\\n[10:03:40] 2 tRNA-Leu c[58106,58190] 35 (caa)\\n[10:03:40] 1 tRNA-Tyr [52183,52267] 35 (gta)\\n[10:03:40] 2 tRNA-Thr [52331,52402] 33 (ggt)\\n[10:03:40] 3 tRNA-Asp [91319,91395] 37 (gtc)\\n[10:03:40] 4 tRNA-Glu [91398,91474] 35 (ctc)\\n[10:03:40] 5 tRNA-Phe [91558,91635] 38 (gaa)\\n[10:03:40] 6 tRNA-Asn [91644,91719] 34 (gtt)\\n[10:03:40] 7 tRNA-Gln [91810,91884] 35 (ttg)\\n[10:03:40] 8 tRNA-Leu [91889,91966] 35 (caa)\\n[10:03:40] 9 tRNA-Pro [91968,92044] 35 (tgg)\\n[10:03:40] 10 tRNA-Leu [92048,92122] 35 (tag)\\n[10:03:40] 11 tRNA-Leu [92127,92204] 35 (gag)\\n[10:03:40] 12 tRNA-Lys [92209,92282] 34 (ttt)\\n[10:03:40] 13 tRNA-Lys [92286,92360] 34 (ctt)\\n[10:03:40] 14 tRNA-Leu [92363,92437] 35 (taa)\\n[10:03:40] 15 tRNA-Met [92439,92516] 35 (cat)\\n[10:03:40] 16 tRNA-Thr [92520,92593] 35 (tgt)\\n[10:03:40] 17 tRNA-Ala [92877,92949] 34 (ggc)\\n[10:03:40] 18 tRNA-Ala [92959,93033] 35 (tgc)\\n[10:03:40] 1 tRNA-Pro c[5820,5912] 34 (tgg)\\n[10:03:40] 1 tRNA-Pro c[27430,27504] 35 (tgg)\\n[10:03:40] 2 tRNA-Met c[67848,67924] 35 (cat)\\n[10:03:40] 3 tRNA-Val [70995,71068] 35 (gac)\\n[10:03:40] 4 tRNA-Met c[89836,89910] 35 (cat)\\n[10:03:40] 1 tRNA-Arg [44200,44273] 35 (ccg)\\n[10:03:40] 1 tRNA-Gly c[73461,73531] 33 (tcc)\\n[10:03:40] 1 tRNA-Trp [9867,9938] 34 (cca)\\n[10:03:40] 1 tRNA-Leu c[51893,51975] 35 (cag)\\n[10:03:40] 1 tRNA-Asp c[57326,57399] 35 (gtc)\\n[10:03:40] 1 tRNA-Cys [23517,23588] 33 (gca)\\n[10:03:40] 1 tRNA-Pro c[15490,15563] 35 (ggg)\\n[10:03:40] 1 tRNA-Lys c[26747,26818] 33 (ttt)\\n[10:03:40] 1 tRNA-Phe c[19626,19698] 34 (gaa)\\n[10:03:40] 1 tRNA-Met c[34063,34135] 34 (cat)\\n[10:03:40] 1 tRNA-Pro c[22918,22991] 35 (cgg)\\n[10:03:40] 1 tRNA-Gly c[756,827] 33 (ccc)\\n[10:03:40] 1 tRNA-Ala [8551,8625] 35 (cgc)\\n[10:03:40] 1 tRNA-Ser [1568,1653] 35 (tga)\\n[10:03:40] 1 tRNA-Asn [2619,2692] 34 (gtt)\\n[10:03:40] Found 59 tRNAs\\n[10:03:40] Predicting Ribosomal RNAs\\n[10:03:40] Running Barrnap with 4 threads\\n[10:03:41] 1 JACJRZ010000131.1 6 16S ribosomal RNA\\n[10:03:41] 2 JACJRZ010000132.1 4 5S ribosomal RNA\\n[10:03:41] 3 JACJRZ010000132.1 248 23S ribosomal RNA\\n[10:03:41] Found 3 rRNAs\\n[10:03:41] Skipping ncRNA search, enable with --rfam if desired.\\n[10:03:41] Total of 61 tRNA + rRNA features\\n[10:03:41] Searching for CRISPR repeats\\n[10:03:42] CRISPR1 JACJRZ010000009.1 118256 with 83 spacers\\n[10:03:42] CRISPR2 JACJRZ010000010.1 5337 with 30 spacers\\n[10:03:42] CRISPR3 JACJRZ010000013.1 204 with 19 spacers\\n[10:03:42] CRISPR4 JACJRZ010000026.1 12538 with 70 spacers\\n[10:03:42] CRISPR5 JACJRZ010000036.1 60720 with 67 spacers\\n[10:03:42] CRISPR6 JACJRZ010000065.1 20866 with 74 spacers\\n[10:03:42] CRISPR7 JACJRZ010000107.1 116 with 4 spacers\\n[10:03:42] CRISPR8 JACJRZ010000108.1 7430 with 9 spacers\\n[10:03:42] Found 8 CRISPRs\\n[10:03:42] Predicting coding sequences\\n[10:03:42] Contigs total 6910048 bp, so using single mode\\n[10:03:42] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACJRZ01\\\\/\\\\/Fischerella_JACJRZ01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:03:48] Excluding CDS which overlaps existing RNA (tRNA) at JACJRZ010000002.1:227750..227995 on + strand\\n[10:03:48] Excluding CDS which overlaps existing RNA (tRNA) at JACJRZ010000006.1:91711..92373 on - strand\\n[10:03:48] Excluding CDS which overlaps existing RNA (tRNA) at JACJRZ010000009.1:115198..115419 on - strand\\n[10:03:48] Excluding CDS which overlaps existing RNA (repeat_region) at JACJRZ010000009.1:124228..124428 on - strand\\n[10:03:49] Excluding CDS which overlaps existing RNA (repeat_region) at JACJRZ010000010.1:5191..5370 on + strand\\n[10:03:50] Excluding CDS which overlaps existing RNA (repeat_region) at JACJRZ010000026.1:13776..14144 on - strand\\n[10:03:50] Excluding CDS which overlaps existing RNA (repeat_region) at JACJRZ010000036.1:60499..60753 on + strand\\n[10:03:51] Excluding CDS which overlaps existing RNA (repeat_region) at JACJRZ010000108.1:7277..7582 on + strand\\n[10:03:51] Excluding CDS which overlaps existing RNA (repeat_region) at JACJRZ010000108.1:7561..7770 on + strand\\n[10:03:51] Excluding CDS which overlaps existing RNA (rRNA) at JACJRZ010000132.1:2544..2936 on + strand\\n[10:03:51] Found 5975 CDS\\n[10:03:51] Connecting features back to sequences\\n[10:03:51] Not using genus-specific database. Try --usegenus to enable it.\\n[10:03:51] Annotating CDS, please be patient.\\n[10:03:51] Will use 4 CPUs for similarity searching.\\n[10:03:53] There are still 5975 unannotated CDS left (started with 5975)\\n[10:03:53] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:03:53] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACJRZ01\\\\/\\\\/Fischerella_JACJRZ01\\\\.IS\\\\.tmp\\\\.96722\\\\.faa | parallel --gnu --plain -j 4 --block 231214 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACJRZ01\\\\/\\\\/Fischerella_JACJRZ01\\\\.IS\\\\.tmp\\\\.96722\\\\.blast 2> /dev/null\\n[10:04:05] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACJRZ01\\\\/\\\\/Fischerella_JACJRZ01\\\\.IS\\\\.tmp\\\\.96722\\\\.faa | parallel --gnu --plain -j 4 --block 231214 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACJRZ01\\\\/\\\\/Fischerella_JACJRZ01\\\\.IS\\\\.tmp\\\\.96722\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:04:05] This is prokka 1.14.6\\n[10:04:05] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:04:05] Homepage is https://github.com/tseemann/prokka\\n[10:04:05] Local time is Mon Feb 21 10:04:05 2022\\n[10:04:05] You are tiagoferreiraleao\\n[10:04:05] Operating system is darwin\\n[10:04:05] You have BioPerl 1.007002\\n[10:04:06] System has 4 cores.\\n[10:04:06] Option --cpu asked for 12 cores, but system only has 4\\n[10:04:06] Will use maximum of 4 cores.\\n[10:04:06] Annotating as >>> Bacteria <<<\\n[10:04:06] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACVRH01.fasta' contents.\\n[10:04:06] Setting --locustag PLFJKCIB from MD5 95f34c2bc6f5610527e051db8da17888\\n[10:04:06] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACVRH01/\\n[10:04:06] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACVRH01\\\\/\\n[10:04:06] Using filename prefix: Nostoc_JACVRH01.XXX\\n[10:04:06] Setting HMMER_NCPU=1\\n[10:04:06] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACVRH01//Nostoc_JACVRH01.log\\n[10:04:06] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACVRH01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACVRH01/ --prefix Nostoc_JACVRH01 --cpus 12\\n[10:04:06] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:04:06] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:04:06] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:04:06] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:04:06] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:04:06] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:04:06] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:04:06] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:04:06] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:04:06] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:04:06] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:04:06] Looking for 'egrep' - found /usr/bin/egrep\\n[10:04:06] Looking for 'find' - found /usr/bin/find\\n[10:04:06] Looking for 'grep' - found /usr/bin/grep\\n[10:04:06] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:04:06] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:04:06] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:04:06] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:04:06] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:04:06] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:04:06] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:04:06] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:04:06] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:04:06] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:04:06] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:04:06] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:04:06] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:04:06] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:04:06] Looking for 'sed' - found /usr/bin/sed\\n[10:04:06] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:04:07] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:04:07] Using genetic code table 11.\\n[10:04:07] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACVRH01.fasta\\n[10:04:07] Wrote 1528 contigs totalling 6228448 bp.\\n[10:04:07] Predicting tRNAs and tmRNAs\\n[10:04:07] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACVRH01\\\\/\\\\/Nostoc_JACVRH01\\\\.fna\\n[10:04:08] 1 tRNA-Leu c[13286,13369] 36 (tag)\\n[10:04:08] 1 tRNA-Val c[1019,1091] 34 (tac)\\n[10:04:08] 1 tRNA-Glu [7881,7954] 35 (ttc)\\n[10:04:08] 1 tRNA-Gly [6157,6228] 33 (gcc)\\n[10:04:08] 1 tRNA-Pro c[3706,3779] 35 (tgg)\\n[10:04:08] 1 tRNA-Ser c[8275,8363] 36 (gga)\\n[10:04:08] 1 tRNA-Asp [5187,5260] 35 (gtc)\\n[10:04:08] 1 tRNA-Ser c[7148,7234] 36 (cga)\\n[10:04:08] 1 tRNA-Pro [3836,3909] 35 (ggg)\\n[10:04:08] 1 tRNA-Met [4195,4269] 35 (cat)\\n[10:04:08] 1 tRNA-Gln [301,374] 34 (ttg)\\n[10:04:08] 1 tRNA-Leu c[3096,3180] 35 (caa)\\n[10:04:08] 1 tRNA-Trp [5098,5171] 34 (cca)\\n[10:04:08] 1 tRNA-Leu [3036,3117] 35 (cag)\\n[10:04:08] 1 tRNA-Asn [960,1031] 33 (gtt)\\n[10:04:08] 1 tRNA-Met c[1993,2066] 33 (cat)\\n[10:04:08] 1 tRNA-Lys [3159,3230] 33 (ctt)\\n[10:04:09] 1 tRNA-Gly [4161,4232] 33 (ccc)\\n[10:04:09] 1 tRNA-Ala c[234,308] 35 (tgc)\\n[10:04:09] 1 tRNA-Ser [1122,1210] 36 (gga)\\n[10:04:09] 1 tRNA-Cys [1130,1200] 33 (gca)\\n[10:04:09] 1 tRNA-Phe [4189,4261] 34 (gaa)\\n[10:04:09] 1 tRNA-Pro c[2819,2892] 35 (cgg)\\n[10:04:09] 1 tRNA-Thr c[741,812] 33 (ggt)\\n[10:04:09] 2 tRNA-Tyr c[900,985] 35 (gta)\\n[10:04:09] 3 tRNA-Ser [3263,3349] 36 (tga)\\n[10:04:09] 1 tRNA-Arg [3630,3702] 34 (ccg)\\n[10:04:09] 1 tRNA-Thr [566,637] 33 (cgt)\\n[10:04:09] 1 tRNA-Arg [701,777] 36 (acg)\\n[10:04:09] 2 tRNA-Asn [832,904] 33 (gtt)\\n[10:04:09] 1 tRNA-Ala c[3157,3229] 34 (ggc)\\n[10:04:09] 1 tRNA-His c[2709,2782] 34 (gtg)\\n[10:04:09] 1 tmRNA [837,1224] 101,151 ANNIVKFARREALVAA*\\n[10:04:09] 1 tRNA-Ser c[2734,2827] 36 (gct)\\n[10:04:09] 1 tRNA-Leu c[1068,1150] 36 (gag)\\n[10:04:09] 1 tRNA-Lys c[2588,2660] 33 (ctt)\\n[10:04:09] 1 tRNA-Arg [209,284] 36 (cct)\\n[10:04:09] 1 tRNA-Ala [2210,2284] 35 (cgc)\\n[10:04:09] 1 tRNA-Val c[837,910] 35 (gac)\\n[10:04:09] 1 tRNA-Lys [1647,1719] 33 (ctt)\\n[10:04:09] 1 tRNA-Ala [244,316] 34 (cgc)\\n[10:04:09] Found 41 tRNAs\\n[10:04:09] Predicting Ribosomal RNAs\\n[10:04:09] Running Barrnap with 4 threads\\n[10:04:10] Found 0 rRNAs\\n[10:04:10] Skipping ncRNA search, enable with --rfam if desired.\\n[10:04:10] Total of 40 tRNA + rRNA features\\n[10:04:10] Searching for CRISPR repeats\\n[10:04:11] CRISPR1 JACVRH010000527.1 3866 with 5 spacers\\n[10:04:11] CRISPR2 JACVRH010000892.1 2682 with 4 spacers\\n[10:04:11] Found 2 CRISPRs\\n[10:04:11] Predicting coding sequences\\n[10:04:11] Contigs total 6228448 bp, so using single mode\\n[10:04:11] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACVRH01\\\\/\\\\/Nostoc_JACVRH01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:04:16] Excluding CDS which overlaps existing RNA (tRNA) at JACVRH010000070.1:6144..6440 on + strand\\n[10:04:17] Excluding CDS which overlaps existing RNA (tRNA) at JACVRH010000328.1:5074..5220 on + strand\\n[10:04:17] Excluding CDS which overlaps existing RNA (tRNA) at JACVRH010000738.1:846..1016 on + strand\\n[10:04:18] Found 5399 CDS\\n[10:04:18] Connecting features back to sequences\\n[10:04:18] Not using genus-specific database. Try --usegenus to enable it.\\n[10:04:18] Annotating CDS, please be patient.\\n[10:04:18] Will use 4 CPUs for similarity searching.\\n[10:04:19] There are still 5399 unannotated CDS left (started with 5399)\\n[10:04:19] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:04:19] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACVRH01\\\\/\\\\/Nostoc_JACVRH01\\\\.IS\\\\.tmp\\\\.96835\\\\.faa | parallel --gnu --plain -j 4 --block 176475 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACVRH01\\\\/\\\\/Nostoc_JACVRH01\\\\.IS\\\\.tmp\\\\.96835\\\\.blast 2> /dev/null\\n[10:04:32] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACVRH01\\\\/\\\\/Nostoc_JACVRH01\\\\.IS\\\\.tmp\\\\.96835\\\\.faa | parallel --gnu --plain -j 4 --block 176475 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACVRH01\\\\/\\\\/Nostoc_JACVRH01\\\\.IS\\\\.tmp\\\\.96835\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:04:32] This is prokka 1.14.6\\n[10:04:32] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:04:32] Homepage is https://github.com/tseemann/prokka\\n[10:04:32] Local time is Mon Feb 21 10:04:32 2022\\n[10:04:32] You are tiagoferreiraleao\\n[10:04:32] Operating system is darwin\\n[10:04:32] You have BioPerl 1.007002\\n[10:04:32] System has 4 cores.\\n[10:04:32] Option --cpu asked for 12 cores, but system only has 4\\n[10:04:32] Will use maximum of 4 cores.\\n[10:04:32] Annotating as >>> Bacteria <<<\\n[10:04:32] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHD01.fasta' contents.\\n[10:04:32] Setting --locustag FABLFGPB from MD5 fab5f09b70eb34fc47676146e845c655\\n[10:04:32] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHD01/\\n[10:04:32] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHD01\\\\/\\n[10:04:32] Using filename prefix: Nostoc_LAHD01.XXX\\n[10:04:32] Setting HMMER_NCPU=1\\n[10:04:32] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHD01//Nostoc_LAHD01.log\\n[10:04:32] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHD01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHD01/ --prefix Nostoc_LAHD01 --cpus 12\\n[10:04:32] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:04:32] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:04:32] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:04:32] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:04:32] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:04:32] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:04:32] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:04:32] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:04:32] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:04:32] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:04:32] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:04:32] Looking for 'egrep' - found /usr/bin/egrep\\n[10:04:32] Looking for 'find' - found /usr/bin/find\\n[10:04:32] Looking for 'grep' - found /usr/bin/grep\\n[10:04:32] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:04:32] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:04:32] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:04:32] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:04:32] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:04:32] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:04:33] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:04:33] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:04:33] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:04:33] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:04:33] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:04:33] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:04:33] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:04:33] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:04:33] Looking for 'sed' - found /usr/bin/sed\\n[10:04:33] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:04:33] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:04:33] Using genetic code table 11.\\n[10:04:33] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHD01.fasta\\n[10:04:33] Wrote 307 contigs totalling 9054479 bp.\\n[10:04:33] Predicting tRNAs and tmRNAs\\n[10:04:33] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHD01\\\\/\\\\/Nostoc_LAHD01\\\\.fna\\n[10:04:36] 1 tRNA-Thr [118847,118918] 33 (cgt)\\n[10:04:36] 2 tRNA-Trp [164520,164593] 34 (cca)\\n[10:04:36] 1 tRNA-Cys c[131053,131124] 33 (gca)\\n[10:04:36] 1 tRNA-Gly c[125847,125917] 33 (tcc)\\n[10:04:36] 1 tRNA-Gly [42692,42765] 34 (ccc)\\n[10:04:36] 2 tRNA-Ser c[50910,50997] 35 (gga)\\n[10:04:36] 1 tRNA-Ser [60696,60782] 36 (cga)\\n[10:04:36] 1 tRNA-Leu c[46490,46573] 36 (tag)\\n[10:04:36] 2 tRNA-Ala c[80905,80979] 34 (tgc)\\n[10:04:36] 3 tRNA-Ile c[81063,81138] 35 (gat)\\n[10:04:36] 1 tmRNA c[75839,76226] 101,151 ANNIVKFARREALVAA*\\n[10:04:36] 1 tRNA-Ala c[29370,29442] 34 (cgc)\\n[10:04:36] 2 tRNA-Leu [78144,78228] 35 (caa)\\n[10:04:36] 3 tRNA-Gln [82982,83055] 34 (ttg)\\n[10:04:36] 1 tRNA-Ala c[14794,14866] 34 (ggc)\\n[10:04:36] 1 tRNA-Leu c[66589,66670] 35 (cag)\\n[10:04:36] 1 tRNA-Pro c[43228,43301] 35 (ggg)\\n[10:04:36] 1 tRNA-Ser c[7722,7815] 36 (gct)\\n[10:04:36] 2 tRNA-Lys c[13490,13560] 33 (ttt)\\n[10:04:36] 3 tRNA-Ser c[13819,13902] 34 (tga)\\n[10:04:36] 4 tRNA-His c[13907,13982] 35 (gtg)\\n[10:04:36] 5 tRNA-Ala c[13988,14062] 35 (tgc)\\n[10:04:36] 6 tRNA-Arg c[14154,14229] 35 (tct)\\n[10:04:36] 7 tRNA-Val c[14236,14311] 34 (tac)\\n[10:04:36] 8 tRNA-Thr c[14393,14465] 34 (tgt)\\n[10:04:36] 9 tRNA-Leu c[14760,14834] 35 (tag)\\n[10:04:36] 10 tRNA-Leu c[14946,15022] 34 (gag)\\n[10:04:36] 11 tRNA-Pro c[15130,15203] 35 (tgg)\\n[10:04:36] 12 tRNA-Leu c[15205,15282] 35 (taa)\\n[10:04:36] 13 tRNA-Leu c[15303,15380] 36 (caa)\\n[10:04:36] 14 tRNA-Gln c[15406,15481] 35 (ttg)\\n[10:04:36] 15 tRNA-Gln c[16356,16430] 35 (ctg)\\n[10:04:36] 16 tRNA-Asn c[16441,16515] 33 (gtt)\\n[10:04:36] 17 tRNA-Phe c[16521,16596] 36 (gaa)\\n[10:04:36] 18 tRNA-Arg c[16684,16759] 35 (acg)\\n[10:04:36] 19 tRNA-Ser c[17521,17607] 35 (gct)\\n[10:04:36] 20 tRNA-Tyr c[17611,17692] 35 (gta)\\n[10:04:36] 21 tRNA-Glu c[17703,17777] 35 (ttc)\\n[10:04:36] tRNA c[17865,17939] is a pseudo/wacky gene - skipping.\\n[10:04:36] 23 tRNA-Met c[17942,18015] 32 (cat)\\n[10:04:36] 24 tRNA-Glu c[18125,18201] 35 (ctc)\\n[10:04:36] 25 tRNA-Asp c[18205,18280] 36 (gtc)\\n[10:04:36] 1 tRNA-Met [29763,29838] 34 (cat)\\n[10:04:36] 2 tRNA-Met [29840,29915] 34 (cat)\\n[10:04:36] 3 tRNA-Ile [31521,31596] 34 (tat)\\n[10:04:36] 1 tRNA-Asn c[53377,53450] 34 (gtt)\\n[10:04:36] 1 tRNA-Met c[40785,40859] 35 (cat)\\n[10:04:36] 1 tRNA-Asp [25380,25457] 38 (gtc)\\n[10:04:36] 2 tRNA-Glu [25460,25536] 35 (ctc)\\n[10:04:36] 3 tRNA-Cys [25541,25616] 34 (gca)\\n[10:04:36] 4 tRNA-Met [25622,25696] 33 (cat)\\n[10:04:36] 5 tRNA-Trp [25699,25771] 34 (cca)\\n[10:04:36] 6 tRNA-Pyl [25796,25867] 33 (cta)\\n[10:04:36] 7 tRNA-Glu [25898,25973] 34 (ttc)\\n[10:04:36] 8 tRNA-Tyr [25977,26060] 36 (gta)\\n[10:04:36] 9 tRNA-Ser [26064,26153] 33 (gct)\\n[10:04:36] 10 tRNA-Arg [26158,26233] 35 (acg)\\n[10:04:36] 11 tRNA-Phe [26238,26312] 35 (gaa)\\n[10:04:36] 12 tRNA-Asn [26321,26395] 33 (gtt)\\n[10:04:36] 13 tRNA-Gln [26401,26476] 35 (ttg)\\n[10:04:36] 14 tRNA-Leu [26484,26561] 35 (caa)\\n[10:04:36] 15 tRNA-Pro [26566,26642] 35 (tgg)\\n[10:04:36] 16 tRNA-Leu [26646,26720] 35 (tag)\\n[10:04:36] 17 tRNA-Leu [26740,26817] 35 (gag)\\n[10:04:36] 18 tRNA-Lys [26821,26897] 34 (ttt)\\n[10:04:36] 19 tRNA-Lys [26903,26977] 33 (ctt)\\n[10:04:36] 20 tRNA-Leu [26983,27058] 36 (taa)\\n[10:04:36] 21 tRNA-Cys [27068,27145] 34 (gca)\\n[10:04:36] 22 tRNA-Thr [27153,27225] 34 (tgt)\\n[10:04:36] 23 tRNA-Gly [27231,27301] 33 (tcc)\\n[10:04:36] 24 tRNA-Val [27307,27382] 34 (tac)\\n[10:04:36] 25 tRNA-Arg [27389,27463] 35 (tct)\\n[10:04:36] 26 tRNA-Ala [27468,27540] 34 (tgc)\\n[10:04:36] 27 tRNA-His [27548,27624] 35 (gtg)\\n[10:04:36] 28 tRNA-Ser [27635,27719] 34 (tga)\\n[10:04:36] 1 tRNA-Lys c[10747,10818] 33 (ttt)\\n[10:04:36] 1 tRNA-Phe c[8546,8619] 34 (gaa)\\n[10:04:36] 1 tRNA-Gly c[18260,18331] 33 (gcc)\\n[10:04:36] 2 tRNA-Met c[25141,25214] 33 (cat)\\n[10:04:36] 1 tRNA-Thr c[19458,19529] 33 (ggt)\\n[10:04:36] 2 tRNA-Tyr c[19605,19689] 35 (gta)\\n[10:04:36] 3 tRNA-Ser [22140,22226] 36 (tga)\\n[10:04:36] 4 tRNA-Pro [30908,30981] 35 (tgg)\\n[10:04:36] 1 tRNA-Thr [35832,35906] 33 (agt)\\n[10:04:36] 2 tRNA-Leu [37297,37371] 35 (caa)\\n[10:04:36] 1 tRNA-Pro c[15059,15132] 35 (cgg)\\n[10:04:36] 1 tRNA-Lys c[3831,3903] 33 (ctt)\\n[10:04:36] 1 tRNA-Arg [4882,4954] 34 (ccg)\\n[10:04:36] 1 tRNA-Ala [19167,19239] 34 (tgc)\\n[10:04:36] 1 tRNA-Val [13862,13934] 34 (tac)\\n[10:04:36] 1 tRNA-Glu c[3267,3340] 35 (ttc)\\n[10:04:36] 1 tRNA-His c[6005,6078] 34 (gtg)\\n[10:04:36] 1 tRNA-Thr c[22818,22890] 33 (tgt)\\n[10:04:36] 1 tRNA-Leu [61554,61634] 35 (gag)\\n[10:04:36] 1 tRNA-Arg [1090,1163] 35 (acg)\\n[10:04:36] 1 tRNA-Asp [175,248] 35 (gtc)\\n[10:04:36] 1 tRNA-Ala c[3738,3824] 29 (cgc)\\n[10:04:36] 1 tRNA-Arg c[3738,3812] 35 (tct)\\n[10:04:36] 1 tRNA-Val [9634,9707] 35 (gac)\\n[10:04:36] 1 tRNA-Ile [2588,2664] 35 (gat)\\n[10:04:36] 1 tRNA-Ile [164,240] 35 (gat)\\n[10:04:36] tRNA [341,416] is a pseudo/wacky gene - skipping.\\n[10:04:36] 1 tRNA-Ile [590,666] 35 (gat)\\n[10:04:36] tRNA [721,796] is a pseudo/wacky gene - skipping.\\n[10:04:36] 1 tRNA-Thr [1143,1217] 33 (ggt)\\n[10:04:36] Found 102 tRNAs\\n[10:04:36] Predicting Ribosomal RNAs\\n[10:04:36] Running Barrnap with 4 threads\\n[10:04:38] 1 LAHD01000012.1 77909 23S ribosomal RNA\\n[10:04:38] 2 LAHD01000012.1 81260 16S ribosomal RNA\\n[10:04:38] 3 LAHD01000261.1 899 16S ribosomal RNA\\n[10:04:38] 4 LAHD01000263.1 703 23S ribosomal RNA (partial)\\n[10:04:38] 5 LAHD01000268.1 1312 23S ribosomal RNA (partial)\\n[10:04:38] Found 5 rRNAs\\n[10:04:38] Skipping ncRNA search, enable with --rfam if desired.\\n[10:04:38] Total of 106 tRNA + rRNA features\\n[10:04:38] Searching for CRISPR repeats\\n[10:04:39] CRISPR1 LAHD01000001.1 68191 with 24 spacers\\n[10:04:39] CRISPR2 LAHD01000006.1 63957 with 7 spacers\\n[10:04:39] CRISPR3 LAHD01000013.1 51211 with 14 spacers\\n[10:04:39] CRISPR4 LAHD01000019.1 41344 with 17 spacers\\n[10:04:39] CRISPR5 LAHD01000027.1 64424 with 6 spacers\\n[10:04:39] CRISPR6 LAHD01000034.1 46073 with 20 spacers\\n[10:04:39] CRISPR7 LAHD01000047.1 23937 with 15 spacers\\n[10:04:39] CRISPR8 LAHD01000051.1 27778 with 10 spacers\\n[10:04:39] CRISPR9 LAHD01000054.1 4853 with 43 spacers\\n[10:04:39] CRISPR10 LAHD01000054.1 19430 with 27 spacers\\n[10:04:39] CRISPR11 LAHD01000162.1 710 with 24 spacers\\n[10:04:39] CRISPR12 LAHD01000163.1 14187 with 7 spacers\\n[10:04:39] CRISPR13 LAHD01000179.1 2773 with 18 spacers\\n[10:04:39] Found 13 CRISPRs\\n[10:04:39] Predicting coding sequences\\n[10:04:39] Contigs total 9054479 bp, so using single mode\\n[10:04:39] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHD01\\\\/\\\\/Nostoc_LAHD01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:04:48] Excluding CDS which overlaps existing RNA (repeat_region) at LAHD01000013.1:51927..52043 on + strand\\n[10:04:48] Excluding CDS which overlaps existing RNA (repeat_region) at LAHD01000027.1:63488..64447 on + strand\\n[10:04:49] Excluding CDS which overlaps existing RNA (repeat_region) at LAHD01000051.1:28109..28261 on - strand\\n[10:04:49] Excluding CDS which overlaps existing RNA (repeat_region) at LAHD01000054.1:7791..7955 on + strand\\n[10:04:51] Excluding CDS which overlaps existing RNA (tRNA) at LAHD01000127.1:6014..6157 on + strand\\n[10:04:51] Excluding CDS which overlaps existing RNA (tRNA) at LAHD01000157.1:3273..4643 on - strand\\n[10:04:51] Excluding CDS which overlaps existing RNA (repeat_region) at LAHD01000162.1:2316..2594 on - strand\\n[10:04:51] Excluding CDS which overlaps existing RNA (repeat_region) at LAHD01000163.1:14215..14340 on - strand\\n[10:04:52] Excluding CDS which overlaps existing RNA (rRNA) at LAHD01000263.1:294..947 on - strand\\n[10:04:52] Found 7622 CDS\\n[10:04:52] Connecting features back to sequences\\n[10:04:52] Not using genus-specific database. Try --usegenus to enable it.\\n[10:04:52] Annotating CDS, please be patient.\\n[10:04:52] Will use 4 CPUs for similarity searching.\\n[10:04:53] There are still 7622 unannotated CDS left (started with 7622)\\n[10:04:53] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:04:53] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHD01\\\\/\\\\/Nostoc_LAHD01\\\\.IS\\\\.tmp\\\\.96947\\\\.faa | parallel --gnu --plain -j 4 --block 305101 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHD01\\\\/\\\\/Nostoc_LAHD01\\\\.IS\\\\.tmp\\\\.96947\\\\.blast 2> /dev/null\\n[10:05:15] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHD01\\\\/\\\\/Nostoc_LAHD01\\\\.IS\\\\.tmp\\\\.96947\\\\.faa | parallel --gnu --plain -j 4 --block 305101 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHD01\\\\/\\\\/Nostoc_LAHD01\\\\.IS\\\\.tmp\\\\.96947\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:05:15] This is prokka 1.14.6\\n[10:05:15] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:05:15] Homepage is https://github.com/tseemann/prokka\\n[10:05:15] Local time is Mon Feb 21 10:05:15 2022\\n[10:05:15] You are tiagoferreiraleao\\n[10:05:15] Operating system is darwin\\n[10:05:15] You have BioPerl 1.007002\\n[10:05:15] System has 4 cores.\\n[10:05:15] Option --cpu asked for 12 cores, but system only has 4\\n[10:05:15] Will use maximum of 4 cores.\\n[10:05:15] Annotating as >>> Bacteria <<<\\n[10:05:15] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAG01.fasta' contents.\\n[10:05:15] Setting --locustag EOFNLION from MD5 e8f752871f229fa32f7003376ba10f0c\\n[10:05:15] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAG01/\\n[10:05:15] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAG01\\\\/\\n[10:05:15] Using filename prefix: Fischerella_NHAG01.XXX\\n[10:05:15] Setting HMMER_NCPU=1\\n[10:05:15] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAG01//Fischerella_NHAG01.log\\n[10:05:15] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAG01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAG01/ --prefix Fischerella_NHAG01 --cpus 12\\n[10:05:15] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:05:15] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:05:15] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:05:15] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:05:16] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:05:16] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:05:16] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:05:16] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:05:16] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:05:16] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:05:16] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:05:16] Looking for 'egrep' - found /usr/bin/egrep\\n[10:05:16] Looking for 'find' - found /usr/bin/find\\n[10:05:16] Looking for 'grep' - found /usr/bin/grep\\n[10:05:16] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:05:16] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:05:16] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:05:16] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:05:16] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:05:16] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:05:16] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:05:16] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:05:16] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:05:16] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:05:16] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:05:16] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:05:16] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:05:16] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:05:16] Looking for 'sed' - found /usr/bin/sed\\n[10:05:16] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:05:17] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:05:17] Using genetic code table 11.\\n[10:05:17] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAG01.fasta\\n[10:05:17] Wrote 131 contigs totalling 5327607 bp.\\n[10:05:17] Predicting tRNAs and tmRNAs\\n[10:05:17] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAG01\\\\/\\\\/Fischerella_NHAG01\\\\.fna\\n[10:05:19] 1 tRNA-Thr c[10147,10218] 33 (ggt)\\n[10:05:19] 1 tRNA-Arg [42846,42919] 35 (ccg)\\n[10:05:19] 1 tRNA-Asn c[10494,10567] 34 (gtt)\\n[10:05:19] 2 tRNA-Gly [27506,27577] 33 (gcc)\\n[10:05:19] 1 tRNA-Asn c[3511,3584] 34 (att)\\n[10:05:19] 2 tRNA-Arg c[4255,4327] 34 (tcg)\\n[10:05:19] 3 tRNA-Glu c[22240,22312] 35 (ttc)\\n[10:05:19] 1 tRNA-Trp c[50660,50733] 34 (cca)\\n[10:05:19] 1 tRNA-Arg c[5209,5283] 35 (tct)\\n[10:05:19] 1 tRNA-Met [14870,14945] 36 (cat)\\n[10:05:19] 1 tRNA-Val [1637,1710] 35 (gac)\\n[10:05:19] 1 tRNA-Met c[11882,11958] 35 (cat)\\n[10:05:19] 1 tRNA-Met [9123,9195] 34 (cat)\\n[10:05:19] 1 tRNA-Arg c[29155,29228] 35 (acg)\\n[10:05:19] 2 tRNA-Ala c[34120,34193] 34 (ggc)\\n[10:05:19] 1 tRNA-Gln c[23455,23525] 32 (ttg)\\n[10:05:19] 1 tRNA-Leu [27543,27625] 36 (gag)\\n[10:05:19] 1 tRNA-Val c[38104,38176] 34 (tac)\\n[10:05:19] 2 tRNA-Phe [45575,45647] 34 (gaa)\\n[10:05:19] 3 tRNA-Leu [93032,93115] 36 (tag)\\n[10:05:19] 1 tRNA-Gly [14336,14407] 33 (ccc)\\n[10:05:19] 1 tRNA-Asp [15986,16059] 35 (gtc)\\n[10:05:19] 1 tRNA-Pro c[99738,99811] 35 (cgg)\\n[10:05:19] 1 tRNA-Gly [12370,12440] 33 (tcc)\\n[10:05:19] 1 tRNA-Pro c[110391,110465] 35 (tgg)\\n[10:05:19] 1 tRNA-Leu [13747,13829] 35 (cag)\\n[10:05:19] 1 tRNA-Ala [2887,2961] 35 (cgc)\\n[10:05:19] 1 tmRNA [15148,15534] 100,150 ANNIVKFARKEALVAA*\\n[10:05:19] 1 tRNA-His c[51927,52000] 34 (gtg)\\n[10:05:19] 1 tRNA-Ser [4571,4657] 36 (cga)\\n[10:05:19] 1 tRNA-Thr [2098,2169] 33 (tgt)\\n[10:05:19] 1 tRNA-Ser c[28386,28472] 36 (tga)\\n[10:05:19] 1 tRNA-Arg [4648,4723] 36 (cct)\\n[10:05:19] 1 tRNA-Ser [61275,61368] 36 (gct)\\n[10:05:19] 1 tRNA-Lys [34762,34835] 34 (ttt)\\n[10:05:19] 1 tRNA-Thr [183919,183990] 33 (cgt)\\n[10:05:19] 1 tRNA-Pro [13697,13770] 35 (ggg)\\n[10:05:19] 1 tRNA-Ser [77990,78078] 36 (gga)\\n[10:05:19] 1 tRNA-Gln [35397,35468] 33 (ttg)\\n[10:05:19] 1 tRNA-Leu [66886,66970] 35 (caa)\\n[10:05:19] Found 40 tRNAs\\n[10:05:19] Predicting Ribosomal RNAs\\n[10:05:19] Running Barrnap with 4 threads\\n[10:05:20] Found 0 rRNAs\\n[10:05:20] Skipping ncRNA search, enable with --rfam if desired.\\n[10:05:20] Total of 39 tRNA + rRNA features\\n[10:05:20] Searching for CRISPR repeats\\n[10:05:20] CRISPR1 NHAG01000046.1 92609 with 3 spacers\\n[10:05:20] CRISPR2 NHAG01000090.1 19 with 3 spacers\\n[10:05:20] Found 2 CRISPRs\\n[10:05:20] Predicting coding sequences\\n[10:05:20] Contigs total 5327607 bp, so using single mode\\n[10:05:20] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAG01\\\\/\\\\/Fischerella_NHAG01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:05:25] Excluding CDS which overlaps existing RNA (tRNA) at NHAG01000011.1:9254..10693 on - strand\\n[10:05:26] Excluding CDS which overlaps existing RNA (tRNA) at NHAG01000037.1:29112..29783 on + strand\\n[10:05:28] Found 4504 CDS\\n[10:05:28] Connecting features back to sequences\\n[10:05:28] Not using genus-specific database. Try --usegenus to enable it.\\n[10:05:28] Annotating CDS, please be patient.\\n[10:05:28] Will use 4 CPUs for similarity searching.\\n[10:05:30] There are still 4504 unannotated CDS left (started with 4504)\\n[10:05:30] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:05:30] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAG01\\\\/\\\\/Fischerella_NHAG01\\\\.IS\\\\.tmp\\\\.97062\\\\.faa | parallel --gnu --plain -j 4 --block 172354 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAG01\\\\/\\\\/Fischerella_NHAG01\\\\.IS\\\\.tmp\\\\.97062\\\\.blast 2> /dev/null\\n[10:05:52] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAG01\\\\/\\\\/Fischerella_NHAG01\\\\.IS\\\\.tmp\\\\.97062\\\\.faa | parallel --gnu --plain -j 4 --block 172354 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAG01\\\\/\\\\/Fischerella_NHAG01\\\\.IS\\\\.tmp\\\\.97062\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:05:52] This is prokka 1.14.6\\n[10:05:52] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:05:52] Homepage is https://github.com/tseemann/prokka\\n[10:05:52] Local time is Mon Feb 21 10:05:52 2022\\n[10:05:52] You are tiagoferreiraleao\\n[10:05:52] Operating system is darwin\\n[10:05:52] You have BioPerl 1.007002\\n[10:05:52] System has 4 cores.\\n[10:05:52] Option --cpu asked for 12 cores, but system only has 4\\n[10:05:52] Will use maximum of 4 cores.\\n[10:05:52] Annotating as >>> Bacteria <<<\\n[10:05:52] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZY01.fasta' contents.\\n[10:05:52] Setting --locustag MLHOLIEI from MD5 651852e2e709a5d8db26d58ac0038172\\n[10:05:52] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZY01/\\n[10:05:52] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZY01\\\\/\\n[10:05:52] Using filename prefix: Fischerella_NGZY01.XXX\\n[10:05:52] Setting HMMER_NCPU=1\\n[10:05:52] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZY01//Fischerella_NGZY01.log\\n[10:05:52] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZY01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZY01/ --prefix Fischerella_NGZY01 --cpus 12\\n[10:05:52] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:05:52] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:05:52] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:05:52] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:05:52] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:05:52] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:05:52] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:05:52] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:05:52] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:05:52] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:05:52] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:05:52] Looking for 'egrep' - found /usr/bin/egrep\\n[10:05:52] Looking for 'find' - found /usr/bin/find\\n[10:05:52] Looking for 'grep' - found /usr/bin/grep\\n[10:05:52] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:05:52] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:05:52] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:05:52] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:05:52] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:05:52] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:05:53] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:05:53] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:05:53] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:05:53] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:05:53] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:05:53] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:05:53] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:05:53] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:05:53] Looking for 'sed' - found /usr/bin/sed\\n[10:05:53] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:05:53] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:05:53] Using genetic code table 11.\\n[10:05:53] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZY01.fasta\\n[10:05:53] Wrote 236 contigs totalling 5282445 bp.\\n[10:05:53] Predicting tRNAs and tmRNAs\\n[10:05:53] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZY01\\\\/\\\\/Fischerella_NGZY01\\\\.fna\\n[10:05:55] 1 tRNA-Thr c[10193,10264] 33 (ggt)\\n[10:05:55] 2 tRNA-Tyr c[10328,10412] 35 (gta)\\n[10:05:55] 1 tRNA-Arg [42722,42795] 35 (ccg)\\n[10:05:55] 1 tRNA-His [26441,26514] 34 (gtg)\\n[10:05:55] 1 tRNA-Asn c[26072,26145] 34 (gtt)\\n[10:05:55] 2 tRNA-Gly [43197,43268] 33 (gcc)\\n[10:05:55] 1 tRNA-Pro [18873,18947] 35 (tgg)\\n[10:05:55] 1 tRNA-Glu c[15856,15928] 35 (ttc)\\n[10:05:55] 1 tRNA-Asn c[3512,3585] 34 (att)\\n[10:05:55] 2 tRNA-Arg c[4256,4328] 34 (tcg)\\n[10:05:55] 1 tRNA-Trp c[20400,20473] 34 (cca)\\n[10:05:55] 1 tRNA-Arg [11157,11231] 35 (tct)\\n[10:05:55] 1 tRNA-Met c[26798,26873] 36 (cat)\\n[10:05:55] 1 tRNA-Cys c[69,140] 33 (gca)\\n[10:05:55] 1 tRNA-Met c[26694,26770] 35 (cat)\\n[10:05:55] 2 tRNA-Val [29849,29922] 35 (gac)\\n[10:05:55] 1 tRNA-Phe c[24268,24340] 34 (gaa)\\n[10:05:55] 2 tRNA-Val [31680,31752] 34 (tac)\\n[10:05:55] 1 tRNA-Leu [6691,6774] 36 (tag)\\n[10:05:55] 1 tRNA-Met [9095,9167] 34 (cat)\\n[10:05:55] 1 tRNA-Arg c[45644,45717] 35 (acg)\\n[10:05:55] 2 tRNA-Ala c[50609,50682] 34 (ggc)\\n[10:05:55] 1 tRNA-Gln c[22878,22948] 32 (ttg)\\n[10:05:55] 1 tRNA-Leu [71739,71821] 36 (gag)\\n[10:05:55] 1 tRNA-Gly [14363,14434] 33 (ccc)\\n[10:05:55] 1 tRNA-Pro c[13997,14070] 35 (cgg)\\n[10:05:55] 1 tRNA-Ser [47471,47559] 36 (gga)\\n[10:05:55] 1 tRNA-Gly [12296,12366] 33 (tcc)\\n[10:05:55] 1 tRNA-Leu [12985,13067] 35 (cag)\\n[10:05:55] 1 tRNA-Ala [28445,28519] 35 (cgc)\\n[10:05:55] 1 tmRNA [26433,26819] 100,150 ANNIVKFARKEALVAA*\\n[10:05:55] 1 tRNA-Ser [4824,4910] 36 (cga)\\n[10:05:55] 1 tRNA-Thr [36296,36367] 33 (tgt)\\n[10:05:55] 1 tRNA-Ser c[70068,70154] 36 (tga)\\n[10:05:55] 1 tRNA-Arg [30553,30628] 36 (cct)\\n[10:05:55] 1 tRNA-Leu c[108,192] 35 (caa)\\n[10:05:55] 1 tRNA-Ser [7629,7722] 36 (gct)\\n[10:05:55] 1 tRNA-Lys [1062,1135] 34 (ttt)\\n[10:05:55] 1 tRNA-Thr [10087,10158] 33 (cgt)\\n[10:05:55] 1 tRNA-Pro [13225,13298] 35 (ggg)\\n[10:05:55] 1 tRNA-Asp c[34321,34394] 35 (gtc)\\n[10:05:55] 1 tRNA-Gln [17228,17299] 33 (ttg)\\n[10:05:55] Found 42 tRNAs\\n[10:05:55] Predicting Ribosomal RNAs\\n[10:05:55] Running Barrnap with 4 threads\\n[10:05:56] Found 0 rRNAs\\n[10:05:56] Skipping ncRNA search, enable with --rfam if desired.\\n[10:05:56] Total of 41 tRNA + rRNA features\\n[10:05:56] Searching for CRISPR repeats\\n[10:05:57] CRISPR1 NGZY01000051.1 6259 with 3 spacers\\n[10:05:57] CRISPR2 NGZY01000131.1 3825 with 4 spacers\\n[10:05:57] CRISPR3 NGZY01000220.1 20432 with 4 spacers\\n[10:05:57] Found 3 CRISPRs\\n[10:05:57] Predicting coding sequences\\n[10:05:57] Contigs total 5282445 bp, so using single mode\\n[10:05:57] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZY01\\\\/\\\\/Fischerella_NGZY01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:06:01] Excluding CDS which overlaps existing RNA (tRNA) at NGZY01000015.1:24832..26271 on - strand\\n[10:06:02] Excluding CDS which overlaps existing RNA (tRNA) at NGZY01000061.1:45601..46272 on + strand\\n[10:06:04] Excluding CDS which overlaps existing RNA (tRNA) at NGZY01000224.1:34356..34751 on + strand\\n[10:06:04] Found 4491 CDS\\n[10:06:04] Connecting features back to sequences\\n[10:06:04] Not using genus-specific database. Try --usegenus to enable it.\\n[10:06:04] Annotating CDS, please be patient.\\n[10:06:04] Will use 4 CPUs for similarity searching.\\n[10:06:05] There are still 4491 unannotated CDS left (started with 4491)\\n[10:06:05] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:06:05] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZY01\\\\/\\\\/Fischerella_NGZY01\\\\.IS\\\\.tmp\\\\.97175\\\\.faa | parallel --gnu --plain -j 4 --block 174656 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZY01\\\\/\\\\/Fischerella_NGZY01\\\\.IS\\\\.tmp\\\\.97175\\\\.blast 2> /dev/null\\n[10:06:25] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZY01\\\\/\\\\/Fischerella_NGZY01\\\\.IS\\\\.tmp\\\\.97175\\\\.faa | parallel --gnu --plain -j 4 --block 174656 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZY01\\\\/\\\\/Fischerella_NGZY01\\\\.IS\\\\.tmp\\\\.97175\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:06:26] This is prokka 1.14.6\\n[10:06:26] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:06:26] Homepage is https://github.com/tseemann/prokka\\n[10:06:26] Local time is Mon Feb 21 10:06:26 2022\\n[10:06:26] You are tiagoferreiraleao\\n[10:06:26] Operating system is darwin\\n[10:06:26] You have BioPerl 1.007002\\n[10:06:26] System has 4 cores.\\n[10:06:26] Option --cpu asked for 12 cores, but system only has 4\\n[10:06:26] Will use maximum of 4 cores.\\n[10:06:26] Annotating as >>> Bacteria <<<\\n[10:06:26] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LXQE01.fasta' contents.\\n[10:06:26] Setting --locustag PAFIKHEE from MD5 9af241ee7b5099a873c4470eebd8f120\\n[10:06:26] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LXQE01/\\n[10:06:26] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LXQE01\\\\/\\n[10:06:26] Using filename prefix: Nostoc_LXQE01.XXX\\n[10:06:26] Setting HMMER_NCPU=1\\n[10:06:26] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LXQE01//Nostoc_LXQE01.log\\n[10:06:26] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LXQE01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LXQE01/ --prefix Nostoc_LXQE01 --cpus 12\\n[10:06:26] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:06:26] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:06:26] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:06:26] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:06:26] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:06:26] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:06:26] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:06:26] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:06:26] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:06:26] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:06:26] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:06:26] Looking for 'egrep' - found /usr/bin/egrep\\n[10:06:26] Looking for 'find' - found /usr/bin/find\\n[10:06:26] Looking for 'grep' - found /usr/bin/grep\\n[10:06:26] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:06:26] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:06:26] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:06:26] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:06:26] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:06:26] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:06:26] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:06:26] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:06:27] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:06:27] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:06:27] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:06:27] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:06:27] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:06:27] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:06:27] Looking for 'sed' - found /usr/bin/sed\\n[10:06:27] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:06:27] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:06:27] Using genetic code table 11.\\n[10:06:27] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LXQE01.fasta\\n[10:06:28] Wrote 205 contigs totalling 9204130 bp.\\n[10:06:28] Predicting tRNAs and tmRNAs\\n[10:06:28] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LXQE01\\\\/\\\\/Nostoc_LXQE01\\\\.fna\\n[10:06:31] 1 tRNA-Pro c[500756,500829] 35 (tgg)\\n[10:06:31] 1 tRNA-Ala c[3284,3358] 34 (tgc)\\n[10:06:31] 2 tRNA-Ile c[3434,3509] 35 (gat)\\n[10:06:31] 3 tRNA-Leu c[174462,174544] 36 (gag)\\n[10:06:31] 4 tRNA-Gly c[286659,286729] 33 (tcc)\\n[10:06:31] 1 tRNA-Cys [243085,243155] 33 (gca)\\n[10:06:31] 2 tmRNA c[307364,307752] 101,151 ANNIVSFARKDALVAA*\\n[10:06:31] 3 tRNA-Phe [332948,333020] 34 (gaa)\\n[10:06:31] 1 tRNA-Asn c[48316,48387] 33 (gtt)\\n[10:06:31] 2 tRNA-His c[187529,187602] 34 (gtg)\\n[10:06:31] 1 tRNA-Asp c[18620,18693] 35 (gtc)\\n[10:06:31] 1 tRNA-Arg [110248,110322] 35 (tct)\\n[10:06:31] 2 tRNA-Thr c[132953,133024] 33 (ggt)\\n[10:06:31] 3 tRNA-Tyr c[133112,133197] 35 (gta)\\n[10:06:31] 4 tRNA-Ser [135498,135584] 36 (tga)\\n[10:06:31] 5 tRNA-Val [158412,158484] 34 (tac)\\n[10:06:31] 6 tRNA-Met c[217231,217300] 32 (cat)\\n[10:06:31] 7 tRNA-Met c[217301,217376] 34 (cat)\\n[10:06:31] 8 tRNA-Gly c[285688,285759] 33 (ccc)\\n[10:06:31] 1 tRNA-Pro [112916,112989] 35 (ggg)\\n[10:06:31] 2 tRNA-Met c[508362,508435] 33 (cat)\\n[10:06:31] 1 tRNA-Glu [160239,160312] 35 (ttc)\\n[10:06:31] 2 tRNA-Asp [166776,166851] 36 (gtc)\\n[10:06:31] 3 tRNA-Glu [166855,166931] 35 (ctc)\\n[10:06:31] 4 tRNA-Cys [166935,167008] 35 (gca)\\n[10:06:31] 5 tRNA-Met [167015,167088] 32 (cat)\\n[10:06:31] tRNA [167091,167165] is a pseudo/wacky gene - skipping.\\n[10:06:31] 7 tRNA-Glu [167227,167300] 35 (ttc)\\n[10:06:31] 8 tRNA-Ser [167412,167503] 36 (gct)\\n[10:06:31] 9 tRNA-Arg [167506,167582] 36 (acg)\\n[10:06:31] 10 tRNA-Phe [167663,167738] 36 (gaa)\\n[10:06:31] 11 tRNA-Asn [167744,167818] 33 (gtt)\\n[10:06:31] 12 tRNA-Gln [167956,168028] 34 (ctg)\\n[10:06:31] 13 tRNA-Gln [168032,168107] 34 (ttg)\\n[10:06:31] 14 tRNA-Leu [168222,168299] 35 (taa)\\n[10:06:31] 15 tRNA-Pro [168301,168373] 34 (tgg)\\n[10:06:31] 16 tRNA-Leu [168377,168451] 35 (cag)\\n[10:06:31] 17 tRNA-Leu [168455,168531] 34 (gag)\\n[10:06:31] 18 tRNA-Pro [168538,168612] 34 (cgg)\\n[10:06:31] 19 tRNA-Leu [168614,168688] 35 (tag)\\n[10:06:31] 20 tRNA-Met [168698,168774] 34 (cat)\\n[10:06:31] 21 tRNA-Thr [168783,168855] 34 (tgt)\\n[10:06:31] 22 tRNA-Gly [168860,168931] 33 (tcc)\\n[10:06:31] 23 tRNA-Gly [168939,169010] 34 (gcc)\\n[10:06:31] 24 tRNA-Val [169016,169091] 34 (tac)\\n[10:06:31] 25 tRNA-Arg [169098,169173] 35 (tct)\\n[10:06:31] 26 tRNA-Ala [169270,169344] 35 (tgc)\\n[10:06:31] 27 tRNA-His [169353,169428] 35 (gtg)\\n[10:06:31] 28 tRNA-Lys [169691,169762] 33 (ttt)\\n[10:06:31] 29 tRNA-Ser [173917,174008] 35 (gct)\\n[10:06:31] 1 tRNA-Ala [66218,66292] 35 (tgc)\\n[10:06:31] 2 tRNA-Ala c[70058,70132] 35 (ggc)\\n[10:06:31] 1 tRNA-Ala [44445,44517] 34 (cgc)\\n[10:06:31] 2 tRNA-Pyl c[83611,83684] 34 (cta)\\n[10:06:31] 3 tRNA-Met c[208239,208313] 35 (cat)\\n[10:06:31] 1 tRNA-Trp [104876,104949] 34 (cca)\\n[10:06:31] 2 tRNA-Thr c[160600,160671] 33 (tgt)\\n[10:06:31] 1 tRNA-Lys [74786,74857] 33 (ttt)\\n[10:06:31] 2 tRNA-Arg c[81151,81226] 36 (cct)\\n[10:06:31] 3 tRNA-Ser [98534,98620] 36 (cga)\\n[10:06:31] 1 tRNA-Lys [137627,137699] 33 (ctt)\\n[10:06:31] 1 tRNA-Met c[65995,66071] 35 (cat)\\n[10:06:31] 2 tRNA-Leu c[138513,138595] 35 (tag)\\n[10:06:31] 1 tRNA-Thr [147609,147680] 33 (cgt)\\n[10:06:31] 1 tRNA-Val c[98752,98824] 33 (cac)\\n[10:06:31] 1 tRNA-Arg [46141,46213] 34 (ccg)\\n[10:06:31] 1 tRNA-Gln c[64768,64841] 34 (ttg)\\n[10:06:31] 2 tRNA-Leu c[66662,66746] 35 (caa)\\n[10:06:31] 3 tRNA-Pro c[111718,111791] 35 (cgg)\\n[10:06:31] 1 tRNA-Ser c[66213,66299] 36 (gga)\\n[10:06:31] 1 tRNA-Ser c[22638,22721] 34 (tga)\\n[10:06:31] 2 tRNA-Ala c[22809,22883] 35 (tgc)\\n[10:06:31] 3 tRNA-Arg c[22888,22961] 34 (tct)\\n[10:06:31] 4 tRNA-Val c[22968,23043] 34 (tac)\\n[10:06:31] 5 tRNA-Gly c[23049,23119] 33 (tcc)\\n[10:06:31] 6 tRNA-Thr c[23125,23200] 35 (tgt)\\n[10:06:31] 7 tRNA-Met c[23206,23281] 35 (cat)\\n[10:06:31] 8 tRNA-Leu c[23283,23357] 35 (taa)\\n[10:06:31] 9 tRNA-Lys c[23407,23480] 33 (ctt)\\n[10:06:31] 10 tRNA-Lys c[24026,24101] 34 (ttt)\\n[10:06:31] 11 tRNA-Pro c[24106,24181] 36 (tgg)\\n[10:06:31] 12 tRNA-Leu c[24184,24260] 37 (tag)\\n[10:06:31] 13 tRNA-Leu c[24264,24341] 35 (caa)\\n[10:06:31] 14 tRNA-Gln c[24345,24420] 35 (ttg)\\n[10:06:31] 15 tRNA-Asn c[24426,24500] 33 (gtt)\\n[10:06:31] 16 tRNA-Phe c[24509,24583] 35 (gaa)\\n[10:06:31] 17 tRNA-Arg c[24588,24663] 35 (acg)\\n[10:06:31] 18 tRNA-Ser c[24667,24753] 33 (gct)\\n[10:06:31] 19 tRNA-Tyr c[24757,24839] 36 (gta)\\n[10:06:31] 20 tRNA-Glu c[24846,24922] 34 (ttc)\\n[10:06:31] 21 tRNA-Pyl c[24953,25024] 33 (cta)\\n[10:06:31] 22 tRNA-Trp c[25049,25121] 34 (cca)\\n[10:06:31] 23 tRNA-Met c[25124,25198] 33 (cat)\\n[10:06:31] 24 tRNA-Cys c[25204,25277] 33 (gca)\\n[10:06:31] 25 tRNA-Arg c[25283,25358] 36 (ccg)\\n[10:06:31] 26 tRNA-Glu c[25361,25437] 35 (ctc)\\n[10:06:31] 27 tRNA-Asp c[25440,25517] 38 (gtc)\\n[10:06:31] 1 tRNA-Val [23833,23906] 35 (gac)\\n[10:06:31] 1 tRNA-Leu c[6780,6860] 34 (cag)\\n[10:06:31] 2 tRNA-Gly [42315,42386] 33 (gcc)\\n[10:06:31] 1 tRNA-Arg [3531,3604] 35 (acg)\\n[10:06:31] Found 100 tRNAs\\n[10:06:31] Predicting Ribosomal RNAs\\n[10:06:31] Running Barrnap with 4 threads\\n[10:06:33] 1 LXQE01000075.1 201 23S ribosomal RNA\\n[10:06:33] 2 LXQE01000075.1 3642 16S ribosomal RNA\\n[10:06:33] Found 2 rRNAs\\n[10:06:33] Skipping ncRNA search, enable with --rfam if desired.\\n[10:06:33] Total of 101 tRNA + rRNA features\\n[10:06:33] Searching for CRISPR repeats\\n[10:06:33] CRISPR1 LXQE01000096.1 202109 with 21 spacers\\n[10:06:33] CRISPR2 LXQE01000096.1 243982 with 3 spacers\\n[10:06:33] CRISPR3 LXQE01000096.1 370312 with 12 spacers\\n[10:06:33] CRISPR4 LXQE01000147.1 97950 with 16 spacers\\n[10:06:33] CRISPR5 LXQE01000160.1 77797 with 3 spacers\\n[10:06:33] CRISPR6 LXQE01000162.1 86878 with 15 spacers\\n[10:06:33] CRISPR7 LXQE01000162.1 111283 with 3 spacers\\n[10:06:34] CRISPR8 LXQE01000172.1 41953 with 16 spacers\\n[10:06:34] CRISPR9 LXQE01000172.1 47623 with 15 spacers\\n[10:06:34] CRISPR10 LXQE01000172.1 52399 with 11 spacers\\n[10:06:34] CRISPR11 LXQE01000172.1 57344 with 15 spacers\\n[10:06:34] CRISPR12 LXQE01000188.1 29992 with 20 spacers\\n[10:06:34] Found 12 CRISPRs\\n[10:06:34] Predicting coding sequences\\n[10:06:34] Contigs total 9204130 bp, so using single mode\\n[10:06:34] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LXQE01\\\\/\\\\/Nostoc_LXQE01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:06:43] Excluding CDS which overlaps existing RNA (tRNA) at LXQE01000096.1:187538..187672 on + strand\\n[10:06:43] Excluding CDS which overlaps existing RNA (repeat_region) at LXQE01000096.1:203148..203351 on + strand\\n[10:06:45] Excluding CDS which overlaps existing RNA (tRNA) at LXQE01000146.1:168328..168624 on + strand\\n[10:06:46] Excluding CDS which overlaps existing RNA (tRNA) at LXQE01000153.1:74754..75077 on - strand\\n[10:06:47] Excluding CDS which overlaps existing RNA (repeat_region) at LXQE01000162.1:86650..86892 on + strand\\n[10:06:47] Excluding CDS which overlaps existing RNA (repeat_region) at LXQE01000162.1:86810..87289 on - strand\\n[10:06:47] Excluding CDS which overlaps existing RNA (repeat_region) at LXQE01000162.1:87334..87522 on - strand\\n[10:06:47] Excluding CDS which overlaps existing RNA (tRNA) at LXQE01000169.1:23471..23920 on - strand\\n[10:06:47] Excluding CDS which overlaps existing RNA (repeat_region) at LXQE01000172.1:42447..42968 on - strand\\n[10:06:47] Excluding CDS which overlaps existing RNA (repeat_region) at LXQE01000172.1:52785..53012 on + strand\\n[10:06:48] Excluding CDS which overlaps existing RNA (repeat_region) at LXQE01000188.1:30657..31124 on + strand\\n[10:06:48] Found 8116 CDS\\n[10:06:48] Connecting features back to sequences\\n[10:06:48] Not using genus-specific database. Try --usegenus to enable it.\\n[10:06:48] Annotating CDS, please be patient.\\n[10:06:48] Will use 4 CPUs for similarity searching.\\n[10:06:50] There are still 8116 unannotated CDS left (started with 8116)\\n[10:06:50] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:06:50] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LXQE01\\\\/\\\\/Nostoc_LXQE01\\\\.IS\\\\.tmp\\\\.97289\\\\.faa | parallel --gnu --plain -j 4 --block 310597 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LXQE01\\\\/\\\\/Nostoc_LXQE01\\\\.IS\\\\.tmp\\\\.97289\\\\.blast 2> /dev/null\\n[10:07:06] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LXQE01\\\\/\\\\/Nostoc_LXQE01\\\\.IS\\\\.tmp\\\\.97289\\\\.faa | parallel --gnu --plain -j 4 --block 310597 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LXQE01\\\\/\\\\/Nostoc_LXQE01\\\\.IS\\\\.tmp\\\\.97289\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:07:07] This is prokka 1.14.6\\n[10:07:07] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:07:07] Homepage is https://github.com/tseemann/prokka\\n[10:07:07] Local time is Mon Feb 21 10:07:07 2022\\n[10:07:07] You are tiagoferreiraleao\\n[10:07:07] Operating system is darwin\\n[10:07:07] You have BioPerl 1.007002\\n[10:07:07] System has 4 cores.\\n[10:07:07] Option --cpu asked for 12 cores, but system only has 4\\n[10:07:07] Will use maximum of 4 cores.\\n[10:07:07] Annotating as >>> Bacteria <<<\\n[10:07:07] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHE01.fasta' contents.\\n[10:07:07] Setting --locustag KKDAJDGK from MD5 44da3d04704c52a17d58f20c236eee14\\n[10:07:07] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHE01/\\n[10:07:07] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHE01\\\\/\\n[10:07:07] Using filename prefix: Nostoc_LAHE01.XXX\\n[10:07:07] Setting HMMER_NCPU=1\\n[10:07:07] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHE01//Nostoc_LAHE01.log\\n[10:07:07] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHE01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHE01/ --prefix Nostoc_LAHE01 --cpus 12\\n[10:07:07] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:07:07] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:07:07] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:07:07] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:07:07] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:07:07] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:07:07] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:07:07] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:07:07] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:07:07] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:07:07] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:07:07] Looking for 'egrep' - found /usr/bin/egrep\\n[10:07:07] Looking for 'find' - found /usr/bin/find\\n[10:07:07] Looking for 'grep' - found /usr/bin/grep\\n[10:07:07] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:07:07] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:07:07] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:07:07] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:07:07] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:07:07] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:07:08] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:07:08] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:07:08] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:07:08] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:07:08] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:07:08] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:07:08] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:07:08] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:07:08] Looking for 'sed' - found /usr/bin/sed\\n[10:07:08] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:07:08] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:07:08] Using genetic code table 11.\\n[10:07:08] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHE01.fasta\\n[10:07:09] Wrote 325 contigs totalling 8951761 bp.\\n[10:07:09] Predicting tRNAs and tmRNAs\\n[10:07:09] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHE01\\\\/\\\\/Nostoc_LAHE01\\\\.fna\\n[10:07:12] 1 tRNA-Cys c[130805,130876] 33 (gca)\\n[10:07:12] 1 tRNA-Arg c[72565,72639] 35 (tct)\\n[10:07:12] 1 tmRNA c[60750,61137] 101,151 ANNIVKFARREALVAA*\\n[10:07:12] 1 tRNA-Thr [107373,107444] 33 (cgt)\\n[10:07:12] 2 tRNA-Trp [153074,153147] 34 (cca)\\n[10:07:12] 1 tRNA-Leu c[84419,84500] 35 (cag)\\n[10:07:12] 1 tRNA-Ala c[860,932] 34 (cgc)\\n[10:07:12] 2 tRNA-Leu [49634,49718] 35 (caa)\\n[10:07:12] 3 tRNA-Gln [54472,54545] 34 (ttg)\\n[10:07:12] 1 tRNA-Asn c[53403,53476] 34 (gtt)\\n[10:07:12] 1 tRNA-Ser [35377,35463] 36 (cga)\\n[10:07:12] 1 tRNA-Pro c[62823,62896] 35 (cgg)\\n[10:07:12] 1 tRNA-Thr c[19450,19521] 33 (ggt)\\n[10:07:12] 2 tRNA-Tyr c[19597,19681] 35 (gta)\\n[10:07:12] 3 tRNA-Ser [22132,22218] 36 (tga)\\n[10:07:12] 4 tRNA-Pro [30900,30973] 35 (tgg)\\n[10:07:12] 1 tRNA-Ile [104384,104459] 35 (gat)\\n[10:07:12] 2 tRNA-Ala [104543,104617] 34 (tgc)\\n[10:07:12] 1 tRNA-Met [29810,29885] 34 (cat)\\n[10:07:12] 2 tRNA-Met [29887,29962] 34 (cat)\\n[10:07:12] 3 tRNA-Ile [31568,31643] 34 (tat)\\n[10:07:12] 1 tRNA-Leu c[42764,42847] 36 (tag)\\n[10:07:12] 1 tRNA-Val [9564,9637] 35 (gac)\\n[10:07:12] 1 tRNA-Gly [7625,7698] 34 (ccc)\\n[10:07:12] 2 tRNA-Ser c[15843,15930] 35 (gga)\\n[10:07:12] 1 tRNA-Asp [26562,26639] 38 (gtc)\\n[10:07:12] 2 tRNA-Glu [26642,26718] 35 (ctc)\\n[10:07:12] 3 tRNA-Cys [26723,26798] 34 (gca)\\n[10:07:12] 4 tRNA-Met [26804,26878] 33 (cat)\\n[10:07:12] 5 tRNA-Trp [26881,26953] 34 (cca)\\n[10:07:12] 6 tRNA-Pyl [26978,27049] 33 (cta)\\n[10:07:12] 7 tRNA-Glu [27080,27155] 34 (ttc)\\n[10:07:12] 8 tRNA-Tyr [27159,27242] 36 (gta)\\n[10:07:12] 9 tRNA-Ser [27246,27335] 33 (gct)\\n[10:07:12] 10 tRNA-Arg [27340,27415] 35 (acg)\\n[10:07:12] 11 tRNA-Phe [27420,27494] 35 (gaa)\\n[10:07:12] 12 tRNA-Asn [27503,27577] 33 (gtt)\\n[10:07:12] 13 tRNA-Gln [27583,27658] 35 (ttg)\\n[10:07:12] 14 tRNA-Leu [27666,27743] 35 (caa)\\n[10:07:12] 15 tRNA-Pro [27748,27824] 35 (tgg)\\n[10:07:12] 16 tRNA-Leu [27828,27902] 35 (tag)\\n[10:07:12] 17 tRNA-Leu [27922,27999] 35 (gag)\\n[10:07:12] 18 tRNA-Lys [28003,28079] 34 (ttt)\\n[10:07:12] 19 tRNA-Lys [28085,28159] 33 (ctt)\\n[10:07:12] 20 tRNA-Leu [28165,28240] 36 (taa)\\n[10:07:12] 21 tRNA-Cys [28250,28327] 34 (gca)\\n[10:07:12] 22 tRNA-Thr [28335,28407] 34 (tgt)\\n[10:07:12] 23 tRNA-Gly [28413,28483] 33 (tcc)\\n[10:07:12] 24 tRNA-Val [28489,28564] 34 (tac)\\n[10:07:12] 25 tRNA-Arg [28571,28645] 35 (tct)\\n[10:07:12] 26 tRNA-Ala [28650,28722] 34 (tgc)\\n[10:07:12] 27 tRNA-His [28730,28806] 35 (gtg)\\n[10:07:12] 28 tRNA-Ser [28817,28901] 34 (tga)\\n[10:07:12] 1 tRNA-Lys c[10128,10199] 33 (ttt)\\n[10:07:12] 1 tRNA-Lys c[21283,21355] 33 (ctt)\\n[10:07:12] 1 tRNA-Pro c[43658,43731] 35 (ggg)\\n[10:07:12] 1 tRNA-Ala c[14898,14970] 34 (ggc)\\n[10:07:12] 1 tRNA-Gly c[28501,28572] 33 (gcc)\\n[10:07:12] 2 tRNA-Met c[35382,35455] 33 (cat)\\n[10:07:12] 1 tRNA-Asp [24873,24948] 36 (gtc)\\n[10:07:12] 2 tRNA-Glu [24952,25028] 35 (ctc)\\n[10:07:12] 3 tRNA-Met [25138,25211] 32 (cat)\\n[10:07:12] tRNA [25214,25288] is a pseudo/wacky gene - skipping.\\n[10:07:12] 5 tRNA-Glu [25376,25450] 35 (ttc)\\n[10:07:12] 6 tRNA-Tyr [25461,25542] 35 (gta)\\n[10:07:12] 7 tRNA-Ser [25546,25632] 35 (gct)\\n[10:07:12] 8 tRNA-Arg [26394,26469] 35 (acg)\\n[10:07:12] 9 tRNA-Phe [26557,26632] 36 (gaa)\\n[10:07:12] 10 tRNA-Asn [26638,26712] 33 (gtt)\\n[10:07:12] 11 tRNA-Gln [26723,26797] 35 (ctg)\\n[10:07:12] 12 tRNA-Gln [27672,27747] 35 (ttg)\\n[10:07:12] 13 tRNA-Leu [27773,27850] 36 (caa)\\n[10:07:12] 14 tRNA-Leu [27871,27948] 35 (taa)\\n[10:07:12] 15 tRNA-Pro [27950,28023] 35 (tgg)\\n[10:07:12] 16 tRNA-Leu [28131,28207] 34 (gag)\\n[10:07:12] 17 tRNA-Leu [28319,28393] 35 (tag)\\n[10:07:12] 18 tRNA-Thr [28688,28760] 34 (tgt)\\n[10:07:12] 19 tRNA-Val [28842,28917] 34 (tac)\\n[10:07:12] 20 tRNA-Arg [28924,28999] 35 (tct)\\n[10:07:12] 21 tRNA-Ala [29091,29165] 35 (tgc)\\n[10:07:12] 22 tRNA-His [29171,29246] 35 (gtg)\\n[10:07:12] 23 tRNA-Ser [29251,29334] 34 (tga)\\n[10:07:12] 24 tRNA-Lys [29593,29663] 33 (ttt)\\n[10:07:12] 25 tRNA-Ser [35338,35431] 36 (gct)\\n[10:07:12] 1 tRNA-Leu c[3505,3585] 35 (gag)\\n[10:07:12] 1 tRNA-Ala [19173,19245] 34 (tgc)\\n[10:07:12] 1 tRNA-Met c[16545,16619] 35 (cat)\\n[10:07:12] 1 tRNA-Glu c[3258,3331] 35 (ttc)\\n[10:07:12] 1 tRNA-His c[6000,6073] 34 (gtg)\\n[10:07:12] 1 tRNA-Thr [20609,20683] 33 (agt)\\n[10:07:12] 2 tRNA-Leu [22074,22148] 35 (caa)\\n[10:07:12] 1 tRNA-Arg [941,1014] 35 (acg)\\n[10:07:12] 1 tRNA-Val [5373,5445] 34 (tac)\\n[10:07:12] 1 tRNA-Ala c[3748,3834] 29 (cgc)\\n[10:07:12] 1 tRNA-Asp [174,247] 35 (gtc)\\n[10:07:12] 1 tRNA-Gly c[8502,8572] 33 (tcc)\\n[10:07:12] 1 tRNA-Phe c[723,796] 34 (gaa)\\n[10:07:12] 1 tRNA-Arg c[4692,4764] 34 (ccg)\\n[10:07:12] 1 tRNA-Ile [1575,1651] 35 (gat)\\n[10:07:12] tRNA [1706,1781] is a pseudo/wacky gene - skipping.\\n[10:07:12] 1 tRNA-Thr [2277,2349] 33 (tgt)\\n[10:07:12] 1 tRNA-Ile [743,819] 35 (gat)\\n[10:07:12] 2 tRNA-Ala [846,921] 34 (tgc)\\n[10:07:12] 1 tmRNA* [1043,1394] 220,261 ANDNNAKGYALAA*\\n[10:07:12] Found 102 tRNAs\\n[10:07:12] Predicting Ribosomal RNAs\\n[10:07:12] Running Barrnap with 4 threads\\n[10:07:14] 1 LAHE01000027.1 102777 16S ribosomal RNA\\n[10:07:14] 2 LAHE01000027.1 104790 23S ribosomal RNA\\n[10:07:14] 3 LAHE01000265.1 1 16S ribosomal RNA (partial)\\n[10:07:14] 4 LAHE01000265.1 2295 23S ribosomal RNA (partial)\\n[10:07:14] 5 LAHE01000271.1 1 16S ribosomal RNA (partial)\\n[10:07:14] 6 LAHE01000271.1 1224 23S ribosomal RNA (partial)\\n[10:07:14] 7 LAHE01000296.1 254 5S ribosomal RNA\\n[10:07:14] 8 LAHE01000296.1 591 23S ribosomal RNA (partial)\\n[10:07:14] Found 8 rRNAs\\n[10:07:14] Skipping ncRNA search, enable with --rfam if desired.\\n[10:07:14] Total of 108 tRNA + rRNA features\\n[10:07:14] Searching for CRISPR repeats\\n[10:07:15] CRISPR1 LAHE01000007.1 36122 with 14 spacers\\n[10:07:15] CRISPR2 LAHE01000008.1 56717 with 24 spacers\\n[10:07:15] CRISPR3 LAHE01000009.1 92596 with 7 spacers\\n[10:07:15] CRISPR4 LAHE01000015.1 41246 with 18 spacers\\n[10:07:15] CRISPR5 LAHE01000019.1 103145 with 15 spacers\\n[10:07:15] CRISPR6 LAHE01000022.1 4863 with 43 spacers\\n[10:07:15] CRISPR7 LAHE01000022.1 19440 with 27 spacers\\n[10:07:15] CRISPR8 LAHE01000023.1 19028 with 7 spacers\\n[10:07:15] CRISPR9 LAHE01000026.1 53111 with 10 spacers\\n[10:07:15] CRISPR10 LAHE01000029.1 46120 with 20 spacers\\n[10:07:15] CRISPR11 LAHE01000162.1 86 with 22 spacers\\n[10:07:15] CRISPR12 LAHE01000181.1 2768 with 18 spacers\\n[10:07:15] CRISPR13 LAHE01000207.1 4833 with 6 spacers\\n[10:07:15] Found 13 CRISPRs\\n[10:07:15] Predicting coding sequences\\n[10:07:15] Contigs total 8951761 bp, so using single mode\\n[10:07:15] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHE01\\\\/\\\\/Nostoc_LAHE01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:07:24] Excluding CDS which overlaps existing RNA (repeat_region) at LAHE01000007.1:36838..36954 on + strand\\n[10:07:24] Excluding CDS which overlaps existing RNA (repeat_region) at LAHE01000009.1:92624..92749 on - strand\\n[10:07:25] Excluding CDS which overlaps existing RNA (repeat_region) at LAHE01000022.1:7801..7965 on + strand\\n[10:07:25] Excluding CDS which overlaps existing RNA (repeat_region) at LAHE01000026.1:53320..53472 on + strand\\n[10:07:28] Excluding CDS which overlaps existing RNA (tRNA) at LAHE01000121.1:6009..6152 on + strand\\n[10:07:28] Excluding CDS which overlaps existing RNA (tRNA) at LAHE01000148.1:3283..4653 on - strand\\n[10:07:28] Excluding CDS which overlaps existing RNA (repeat_region) at LAHE01000162.1:1540..1818 on - strand\\n[10:07:29] Excluding CDS which overlaps existing RNA (repeat_region) at LAHE01000207.1:3897..4856 on + strand\\n[10:07:29] Excluding CDS which overlaps existing RNA (rRNA) at LAHE01000265.1:1849..2325 on - strand\\n[10:07:29] Excluding CDS which overlaps existing RNA (rRNA) at LAHE01000271.1:1080..1466 on - strand\\n[10:07:29] Excluding CDS which overlaps existing RNA (rRNA) at LAHE01000296.1:724..1182 on + strand\\n[10:07:29] Found 7486 CDS\\n[10:07:29] Connecting features back to sequences\\n[10:07:29] Not using genus-specific database. Try --usegenus to enable it.\\n[10:07:29] Annotating CDS, please be patient.\\n[10:07:29] Will use 4 CPUs for similarity searching.\\n[10:07:31] There are still 7486 unannotated CDS left (started with 7486)\\n[10:07:31] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:07:31] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHE01\\\\/\\\\/Nostoc_LAHE01\\\\.IS\\\\.tmp\\\\.97402\\\\.faa | parallel --gnu --plain -j 4 --block 300568 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHE01\\\\/\\\\/Nostoc_LAHE01\\\\.IS\\\\.tmp\\\\.97402\\\\.blast 2> /dev/null\\n[10:07:56] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHE01\\\\/\\\\/Nostoc_LAHE01\\\\.IS\\\\.tmp\\\\.97402\\\\.faa | parallel --gnu --plain -j 4 --block 300568 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHE01\\\\/\\\\/Nostoc_LAHE01\\\\.IS\\\\.tmp\\\\.97402\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:07:56] This is prokka 1.14.6\\n[10:07:56] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:07:56] Homepage is https://github.com/tseemann/prokka\\n[10:07:56] Local time is Mon Feb 21 10:07:56 2022\\n[10:07:56] You are tiagoferreiraleao\\n[10:07:56] Operating system is darwin\\n[10:07:56] You have BioPerl 1.007002\\n[10:07:57] System has 4 cores.\\n[10:07:57] Option --cpu asked for 12 cores, but system only has 4\\n[10:07:57] Will use maximum of 4 cores.\\n[10:07:57] Annotating as >>> Bacteria <<<\\n[10:07:57] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAF01.fasta' contents.\\n[10:07:57] Setting --locustag POJCIEIG from MD5 983c2e202bd20156d8235fc70194722e\\n[10:07:57] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAF01/\\n[10:07:57] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAF01\\\\/\\n[10:07:57] Using filename prefix: Fischerella_NHAF01.XXX\\n[10:07:57] Setting HMMER_NCPU=1\\n[10:07:57] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAF01//Fischerella_NHAF01.log\\n[10:07:57] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAF01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAF01/ --prefix Fischerella_NHAF01 --cpus 12\\n[10:07:57] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:07:57] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:07:57] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:07:57] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:07:57] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:07:57] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:07:57] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:07:57] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:07:57] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:07:57] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:07:57] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:07:57] Looking for 'egrep' - found /usr/bin/egrep\\n[10:07:57] Looking for 'find' - found /usr/bin/find\\n[10:07:57] Looking for 'grep' - found /usr/bin/grep\\n[10:07:57] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:07:57] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:07:57] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:07:57] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:07:57] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:07:57] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:07:57] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:07:57] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:07:57] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:07:57] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:07:57] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:07:57] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:07:57] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:07:57] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:07:57] Looking for 'sed' - found /usr/bin/sed\\n[10:07:57] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:07:58] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:07:58] Using genetic code table 11.\\n[10:07:58] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAF01.fasta\\n[10:07:58] Wrote 126 contigs totalling 5426915 bp.\\n[10:07:58] Predicting tRNAs and tmRNAs\\n[10:07:58] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAF01\\\\/\\\\/Fischerella_NHAF01\\\\.fna\\n[10:08:00] 1 tRNA-His [1358,1431] 34 (gtg)\\n[10:08:00] 1 tRNA-Arg c[36129,36204] 36 (cct)\\n[10:08:00] 1 tRNA-Thr [110222,110293] 33 (cgt)\\n[10:08:00] 1 tRNA-Ser [3168,3254] 36 (tga)\\n[10:08:00] 1 tRNA-Leu [65997,66081] 35 (caa)\\n[10:08:00] 1 tRNA-Gly [12147,12217] 33 (tcc)\\n[10:08:00] 1 tRNA-Met [69,141] 34 (cat)\\n[10:08:00] 1 tRNA-Arg c[28278,28351] 35 (ccg)\\n[10:08:00] 1 tRNA-Ile [24311,24386] 35 (gat)\\n[10:08:00] 2 tRNA-Ala [24469,24543] 34 (tgc)\\n[10:08:00] 1 tRNA-Gly [39108,39179] 33 (ccc)\\n[10:08:00] 1 tRNA-Phe c[7387,7459] 34 (gaa)\\n[10:08:00] 2 tRNA-Val [14800,14872] 34 (tac)\\n[10:08:00] 1 tRNA-Ser c[60038,60124] 36 (cga)\\n[10:08:00] 1 tRNA-Asp c[54093,54166] 35 (gtc)\\n[10:08:00] 1 tRNA-Gln [34308,34379] 33 (ttg)\\n[10:08:00] 1 tRNA-Met c[11616,11692] 35 (cat)\\n[10:08:00] 2 tRNA-Val [14638,14711] 35 (gac)\\n[10:08:00] 1 tmRNA [12008,12394] 100,150 ANNIVKFARKEALVAA*\\n[10:08:00] 1 tRNA-Met c[27062,27137] 36 (cat)\\n[10:08:00] 1 tRNA-Cys c[87,158] 33 (gca)\\n[10:08:00] 1 tRNA-Pro c[52909,52982] 35 (ggg)\\n[10:08:00] 1 tRNA-Ala [639,712] 34 (ggc)\\n[10:08:00] 2 tRNA-Arg [5604,5677] 35 (acg)\\n[10:08:00] 1 tRNA-Glu [7743,7815] 35 (ttc)\\n[10:08:00] 2 tRNA-Arg [25727,25799] 34 (tcg)\\n[10:08:00] 3 tRNA-Asn [26470,26543] 34 (att)\\n[10:08:00] 1 tRNA-Lys c[69449,69522] 34 (ttt)\\n[10:08:00] 1 tRNA-Thr c[8767,8838] 33 (ggt)\\n[10:08:00] 2 tRNA-Tyr c[8902,8986] 35 (gta)\\n[10:08:00] 3 tRNA-Leu [67868,67951] 36 (tag)\\n[10:08:00] 1 tRNA-Trp c[128012,128085] 34 (cca)\\n[10:08:00] 1 tRNA-Arg c[183,257] 35 (tct)\\n[10:08:00] 1 tRNA-Gly [12764,12835] 33 (gcc)\\n[10:08:00] 1 tRNA-Ala [86719,86793] 35 (cgc)\\n[10:08:00] 2 tRNA-Leu [111478,111560] 35 (cag)\\n[10:08:00] 1 tRNA-Gln [46375,46445] 32 (ttg)\\n[10:08:00] 1 tRNA-Ser [45522,45610] 36 (gga)\\n[10:08:00] 1 tRNA-Pro [19764,19838] 35 (tgg)\\n[10:08:00] 1 tRNA-Thr [2098,2169] 33 (tgt)\\n[10:08:00] 2 tRNA-Asn c[102080,102153] 34 (gtt)\\n[10:08:00] 1 tRNA-Leu [27775,27857] 36 (gag)\\n[10:08:00] 1 tRNA-Ser c[216792,216885] 36 (gct)\\n[10:08:00] 1 tRNA-Pro [18017,18090] 35 (cgg)\\n[10:08:00] Found 44 tRNAs\\n[10:08:00] Predicting Ribosomal RNAs\\n[10:08:00] Running Barrnap with 4 threads\\n[10:08:01] 1 NHAF01000042.1 22686 16S ribosomal RNA\\n[10:08:01] 2 NHAF01000042.1 24841 23S ribosomal RNA\\n[10:08:01] 3 NHAF01000042.1 27780 5S ribosomal RNA\\n[10:08:01] Found 3 rRNAs\\n[10:08:01] Skipping ncRNA search, enable with --rfam if desired.\\n[10:08:01] Total of 46 tRNA + rRNA features\\n[10:08:01] Searching for CRISPR repeats\\n[10:08:02] CRISPR1 NHAF01000089.1 55075 with 3 spacers\\n[10:08:02] CRISPR2 NHAF01000105.1 1 with 8 spacers\\n[10:08:02] CRISPR3 NHAF01000108.1 279 with 5 spacers\\n[10:08:02] CRISPR4 NHAF01000109.1 73 with 45 spacers\\n[10:08:02] CRISPR5 NHAF01000121.1 73 with 25 spacers\\n[10:08:02] CRISPR6 NHAF01000121.1 2125 with 8 spacers\\n[10:08:02] CRISPR7 NHAF01000121.1 2786 with 24 spacers\\n[10:08:02] CRISPR8 NHAF01000121.1 4552 with 38 spacers\\n[10:08:02] CRISPR9 NHAF01000121.1 7392 with 5 spacers\\n[10:08:02] CRISPR10 NHAF01000121.1 7971 with 11 spacers\\n[10:08:02] CRISPR11 NHAF01000121.1 8905 with 7 spacers\\n[10:08:02] CRISPR12 NHAF01000121.1 9646 with 11 spacers\\n[10:08:02] CRISPR13 NHAF01000122.1 2 with 10 spacers\\n[10:08:02] CRISPR14 NHAF01000123.1 136555 with 9 spacers\\n[10:08:02] CRISPR15 NHAF01000123.1 140063 with 18 spacers\\n[10:08:02] CRISPR16 NHAF01000123.1 141650 with 13 spacers\\n[10:08:02] CRISPR17 NHAF01000123.1 142718 with 15 spacers\\n[10:08:02] CRISPR18 NHAF01000123.1 143875 with 24 spacers\\n[10:08:02] CRISPR19 NHAF01000123.1 145770 with 8 spacers\\n[10:08:02] CRISPR20 NHAF01000123.1 146765 with 3 spacers\\n[10:08:02] CRISPR21 NHAF01000123.1 147200 with 13 spacers\\n[10:08:02] CRISPR22 NHAF01000123.1 148429 with 4 spacers\\n[10:08:02] CRISPR23 NHAF01000123.1 148745 with 10 spacers\\n[10:08:02] CRISPR24 NHAF01000124.1 76 with 6 spacers\\n[10:08:02] CRISPR25 NHAF01000124.1 1097 with 4 spacers\\n[10:08:02] CRISPR26 NHAF01000124.1 1542 with 15 spacers\\n[10:08:02] CRISPR27 NHAF01000124.1 2780 with 18 spacers\\n[10:08:02] CRISPR28 NHAF01000124.1 4314 with 11 spacers\\n[10:08:02] CRISPR29 NHAF01000124.1 5878 with 13 spacers\\n[10:08:02] CRISPR30 NHAF01000125.1 86082 with 43 spacers\\n[10:08:02] CRISPR31 NHAF01000125.1 89710 with 50 spacers\\n[10:08:02] CRISPR32 NHAF01000125.1 93525 with 24 spacers\\n[10:08:02] CRISPR33 NHAF01000125.1 95601 with 21 spacers\\n[10:08:02] Found 33 CRISPRs\\n[10:08:02] Predicting coding sequences\\n[10:08:02] Contigs total 5426915 bp, so using single mode\\n[10:08:02] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAF01\\\\/\\\\/Fischerella_NHAF01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:08:08] Excluding CDS which overlaps existing RNA (rRNA) at NHAF01000042.1:25236..25415 on + strand\\n[10:08:08] Excluding CDS which overlaps existing RNA (rRNA) at NHAF01000042.1:27201..27377 on - strand\\n[10:08:09] Excluding CDS which overlaps existing RNA (tRNA) at NHAF01000079.1:5049..5720 on - strand\\n[10:08:10] Excluding CDS which overlaps existing RNA (repeat_region) at NHAF01000109.1:1934..2194 on - strand\\n[10:08:10] Excluding CDS which overlaps existing RNA (tRNA) at NHAF01000113.1:100840..102279 on - strand\\n[10:08:11] Excluding CDS which overlaps existing RNA (repeat_region) at NHAF01000125.1:87169..87549 on + strand\\n[10:08:11] Excluding CDS which overlaps existing RNA (repeat_region) at NHAF01000125.1:91887..92078 on + strand\\n[10:08:11] Excluding CDS which overlaps existing RNA (repeat_region) at NHAF01000125.1:93228..93362 on + strand\\n[10:08:11] Found 4620 CDS\\n[10:08:11] Connecting features back to sequences\\n[10:08:11] Not using genus-specific database. Try --usegenus to enable it.\\n[10:08:11] Annotating CDS, please be patient.\\n[10:08:11] Will use 4 CPUs for similarity searching.\\n[10:08:13] There are still 4620 unannotated CDS left (started with 4620)\\n[10:08:13] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:08:13] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAF01\\\\/\\\\/Fischerella_NHAF01\\\\.IS\\\\.tmp\\\\.97520\\\\.faa | parallel --gnu --plain -j 4 --block 178716 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAF01\\\\/\\\\/Fischerella_NHAF01\\\\.IS\\\\.tmp\\\\.97520\\\\.blast 2> /dev/null\\n[10:08:40] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAF01\\\\/\\\\/Fischerella_NHAF01\\\\.IS\\\\.tmp\\\\.97520\\\\.faa | parallel --gnu --plain -j 4 --block 178716 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAF01\\\\/\\\\/Fischerella_NHAF01\\\\.IS\\\\.tmp\\\\.97520\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:08:40] This is prokka 1.14.6\\n[10:08:40] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:08:40] Homepage is https://github.com/tseemann/prokka\\n[10:08:40] Local time is Mon Feb 21 10:08:40 2022\\n[10:08:40] You are tiagoferreiraleao\\n[10:08:40] Operating system is darwin\\n[10:08:40] You have BioPerl 1.007002\\n[10:08:40] System has 4 cores.\\n[10:08:40] Option --cpu asked for 12 cores, but system only has 4\\n[10:08:40] Will use maximum of 4 cores.\\n[10:08:40] Annotating as >>> Bacteria <<<\\n[10:08:40] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGAA01.fasta' contents.\\n[10:08:41] Setting --locustag HMJHPBMD from MD5 16319b6d7253e518fa56134725a03720\\n[10:08:41] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGAA01/\\n[10:08:41] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGAA01\\\\/\\n[10:08:41] Using filename prefix: Fischerella_NGAA01.XXX\\n[10:08:41] Setting HMMER_NCPU=1\\n[10:08:41] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGAA01//Fischerella_NGAA01.log\\n[10:08:41] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGAA01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGAA01/ --prefix Fischerella_NGAA01 --cpus 12\\n[10:08:41] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:08:41] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:08:41] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:08:41] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:08:41] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:08:41] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:08:41] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:08:41] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:08:41] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:08:41] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:08:41] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:08:41] Looking for 'egrep' - found /usr/bin/egrep\\n[10:08:41] Looking for 'find' - found /usr/bin/find\\n[10:08:41] Looking for 'grep' - found /usr/bin/grep\\n[10:08:41] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:08:41] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:08:41] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:08:41] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:08:41] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:08:41] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:08:41] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:08:41] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:08:41] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:08:41] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:08:41] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:08:41] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:08:41] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:08:41] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:08:41] Looking for 'sed' - found /usr/bin/sed\\n[10:08:41] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:08:42] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:08:42] Using genetic code table 11.\\n[10:08:42] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGAA01.fasta\\n[10:08:42] Wrote 100 contigs totalling 5452329 bp.\\n[10:08:42] Predicting tRNAs and tmRNAs\\n[10:08:42] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGAA01\\\\/\\\\/Fischerella_NGAA01\\\\.fna\\n[10:08:44] 1 tRNA-Tyr [13458,13542] 35 (gta)\\n[10:08:44] 2 tRNA-Thr [13606,13677] 33 (ggt)\\n[10:08:44] 1 tRNA-Arg c[28018,28091] 35 (ccg)\\n[10:08:44] 1 tRNA-Gly c[2561,2632] 33 (gcc)\\n[10:08:44] 2 tRNA-Asn [19236,19309] 34 (gtt)\\n[10:08:44] 3 tRNA-Pro c[102604,102678] 35 (tgg)\\n[10:08:44] 1 tRNA-Phe c[1419,1491] 34 (gaa)\\n[10:08:44] 2 tRNA-Val [8949,9021] 34 (tac)\\n[10:08:44] 1 tRNA-Glu [15364,15436] 35 (ttc)\\n[10:08:44] 2 tRNA-Arg [33389,33461] 34 (tcg)\\n[10:08:44] 3 tRNA-Asn [34132,34205] 34 (att)\\n[10:08:44] 1 tRNA-Trp [16238,16311] 34 (cca)\\n[10:08:44] 1 tRNA-Arg [11344,11418] 35 (tct)\\n[10:08:44] 2 tRNA-Met [36485,36560] 36 (cat)\\n[10:08:44] 1 tRNA-Cys [48454,48525] 33 (gca)\\n[10:08:44] 1 tRNA-Val c[27019,27092] 35 (gac)\\n[10:08:44] 2 tRNA-Met [29982,30058] 35 (cat)\\n[10:08:44] 1 tRNA-Leu c[23111,23194] 36 (tag)\\n[10:08:44] 1 tRNA-Met c[68151,68223] 34 (cat)\\n[10:08:44] 1 tRNA-Ala [581,654] 34 (ggc)\\n[10:08:44] 2 tRNA-Arg [5546,5619] 35 (acg)\\n[10:08:44] 1 tRNA-Gln [5765,5835] 32 (ttg)\\n[10:08:44] 1 tRNA-Leu c[1230,1312] 36 (gag)\\n[10:08:44] 1 tRNA-Gly c[7552,7623] 33 (ccc)\\n[10:08:44] 1 tRNA-Pro [58901,58974] 35 (cgg)\\n[10:08:44] 1 tRNA-Gly c[10938,11008] 33 (tcc)\\n[10:08:44] 1 tRNA-Leu c[123974,124056] 35 (cag)\\n[10:08:44] 2 tRNA-Ala c[148642,148716] 35 (cgc)\\n[10:08:44] 1 tmRNA c[184,570] 100,150 ANNIVKFARKEALVAA*\\n[10:08:44] 1 tRNA-His [26244,26317] 34 (gtg)\\n[10:08:44] 1 tRNA-Ser c[70350,70436] 36 (cga)\\n[10:08:44] 1 tRNA-Thr c[8080,8151] 33 (tgt)\\n[10:08:44] 1 tRNA-Ser [30277,30363] 36 (tga)\\n[10:08:44] 1 tRNA-Arg c[36420,36495] 36 (cct)\\n[10:08:44] 1 tRNA-Ser c[53541,53634] 36 (gct)\\n[10:08:44] 2 tRNA-Lys c[209530,209603] 34 (ttt)\\n[10:08:44] 1 tRNA-Thr c[1603,1674] 33 (cgt)\\n[10:08:44] 2 tRNA-Pro c[123715,123788] 35 (ggg)\\n[10:08:44] 1 tRNA-Ser c[4163,4251] 36 (gga)\\n[10:08:44] 1 tRNA-Asp [15778,15851] 35 (gtc)\\n[10:08:44] 1 tRNA-Gln c[23185,23256] 33 (ttg)\\n[10:08:44] 2 tRNA-Leu c[60790,60874] 35 (caa)\\n[10:08:44] Found 42 tRNAs\\n[10:08:44] Predicting Ribosomal RNAs\\n[10:08:44] Running Barrnap with 4 threads\\n[10:08:45] Found 0 rRNAs\\n[10:08:45] Skipping ncRNA search, enable with --rfam if desired.\\n[10:08:45] Total of 41 tRNA + rRNA features\\n[10:08:45] Searching for CRISPR repeats\\n[10:08:45] CRISPR1 NGAA01000041.1 299 with 3 spacers\\n[10:08:45] CRISPR2 NGAA01000041.1 475 with 7 spacers\\n[10:08:46] CRISPR3 NGAA01000077.1 69366 with 4 spacers\\n[10:08:46] CRISPR4 NGAA01000092.1 82409 with 3 spacers\\n[10:08:46] CRISPR5 NGAA01000092.1 82738 with 16 spacers\\n[10:08:46] CRISPR6 NGAA01000096.1 57 with 4 spacers\\n[10:08:46] CRISPR7 NGAA01000096.1 437 with 3 spacers\\n[10:08:46] CRISPR8 NGAA01000096.1 748 with 12 spacers\\n[10:08:46] CRISPR9 NGAA01000099.1 10247 with 3 spacers\\n[10:08:46] CRISPR10 NGAA01000100.1 2677 with 7 spacers\\n[10:08:46] Found 10 CRISPRs\\n[10:08:46] Predicting coding sequences\\n[10:08:46] Contigs total 5452329 bp, so using single mode\\n[10:08:46] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGAA01\\\\/\\\\/Fischerella_NGAA01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:08:50] Excluding CDS which overlaps existing RNA (tRNA) at NGAA01000013.1:19110..20549 on + strand\\n[10:08:51] Excluding CDS which overlaps existing RNA (tRNA) at NGAA01000043.1:4991..5662 on - strand\\n[10:08:54] Excluding CDS which overlaps existing RNA (tRNA) at NGAA01000094.1:15421..15816 on - strand\\n[10:08:54] Excluding CDS which overlaps existing RNA (repeat_region) at NGAA01000096.1:1389..1523 on - strand\\n[10:08:54] Found 4728 CDS\\n[10:08:54] Connecting features back to sequences\\n[10:08:54] Not using genus-specific database. Try --usegenus to enable it.\\n[10:08:54] Annotating CDS, please be patient.\\n[10:08:54] Will use 4 CPUs for similarity searching.\\n[10:08:55] There are still 4728 unannotated CDS left (started with 4728)\\n[10:08:55] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:08:55] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGAA01\\\\/\\\\/Fischerella_NGAA01\\\\.IS\\\\.tmp\\\\.97640\\\\.faa | parallel --gnu --plain -j 4 --block 183708 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGAA01\\\\/\\\\/Fischerella_NGAA01\\\\.IS\\\\.tmp\\\\.97640\\\\.blast 2> /dev/null\\n[10:09:16] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGAA01\\\\/\\\\/Fischerella_NGAA01\\\\.IS\\\\.tmp\\\\.97640\\\\.faa | parallel --gnu --plain -j 4 --block 183708 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGAA01\\\\/\\\\/Fischerella_NGAA01\\\\.IS\\\\.tmp\\\\.97640\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:09:17] This is prokka 1.14.6\\n[10:09:17] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:09:17] Homepage is https://github.com/tseemann/prokka\\n[10:09:17] Local time is Mon Feb 21 10:09:17 2022\\n[10:09:17] You are tiagoferreiraleao\\n[10:09:17] Operating system is darwin\\n[10:09:17] You have BioPerl 1.007002\\n[10:09:17] System has 4 cores.\\n[10:09:17] Option --cpu asked for 12 cores, but system only has 4\\n[10:09:17] Will use maximum of 4 cores.\\n[10:09:17] Annotating as >>> Bacteria <<<\\n[10:09:17] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZX01.fasta' contents.\\n[10:09:17] Setting --locustag GDFCJOHP from MD5 0dfc3819c9aaeede4600f7415b076acb\\n[10:09:17] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZX01/\\n[10:09:17] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZX01\\\\/\\n[10:09:17] Using filename prefix: Fischerella_NGZX01.XXX\\n[10:09:17] Setting HMMER_NCPU=1\\n[10:09:17] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZX01//Fischerella_NGZX01.log\\n[10:09:17] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZX01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZX01/ --prefix Fischerella_NGZX01 --cpus 12\\n[10:09:17] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:09:17] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:09:17] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:09:17] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:09:17] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:09:17] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:09:17] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:09:17] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:09:17] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:09:17] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:09:17] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:09:17] Looking for 'egrep' - found /usr/bin/egrep\\n[10:09:17] Looking for 'find' - found /usr/bin/find\\n[10:09:17] Looking for 'grep' - found /usr/bin/grep\\n[10:09:17] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:09:17] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:09:17] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:09:17] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:09:17] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:09:17] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:09:17] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:09:17] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:09:17] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:09:17] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:09:18] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:09:18] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:09:18] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:09:18] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:09:18] Looking for 'sed' - found /usr/bin/sed\\n[10:09:18] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:09:18] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:09:18] Using genetic code table 11.\\n[10:09:18] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZX01.fasta\\n[10:09:18] Wrote 315 contigs totalling 5270127 bp.\\n[10:09:18] Predicting tRNAs and tmRNAs\\n[10:09:18] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZX01\\\\/\\\\/Fischerella_NGZX01\\\\.fna\\n[10:09:20] 1 tRNA-Thr c[2066,2139] 34 (ggt)\\n[10:09:20] 2 tRNA-Tyr c[2293,2377] 35 (gta)\\n[10:09:20] 1 tRNA-Arg [7807,7880] 35 (ccg)\\n[10:09:20] 1 tRNA-His [26482,26555] 34 (gtg)\\n[10:09:20] 1 tRNA-Glu c[11305,11377] 35 (ttc)\\n[10:09:20] 1 tRNA-Asn c[3509,3582] 34 (att)\\n[10:09:20] 2 tRNA-Arg c[4253,4325] 34 (tcg)\\n[10:09:20] 1 tRNA-Cys c[63,134] 33 (gca)\\n[10:09:20] 1 tRNA-Met c[8859,8935] 35 (cat)\\n[10:09:20] 2 tRNA-Val [11983,12056] 35 (gac)\\n[10:09:20] 1 tmRNA [27012,27398] 100,150 ANNIVKFARKEALVAA*\\n[10:09:20] 1 tRNA-Phe c[24214,24286] 34 (gaa)\\n[10:09:20] 2 tRNA-Val [31627,31699] 34 (tac)\\n[10:09:20] 1 tRNA-Leu [6533,6616] 36 (tag)\\n[10:09:20] 1 tRNA-Met [8684,8756] 34 (cat)\\n[10:09:20] 1 tRNA-Arg c[11003,11076] 35 (acg)\\n[10:09:20] 2 tRNA-Ala c[15968,16041] 34 (ggc)\\n[10:09:20] 1 tRNA-Gln c[6874,6944] 32 (ttg)\\n[10:09:20] 1 tRNA-Leu [35515,35597] 36 (gag)\\n[10:09:20] 1 tRNA-Gly [13713,13784] 33 (ccc)\\n[10:09:20] 1 tRNA-Ser [36372,36465] 36 (gct)\\n[10:09:20] 1 tRNA-Arg [12814,12888] 35 (tct)\\n[10:09:20] 1 tRNA-Pro c[25486,25559] 35 (cgg)\\n[10:09:20] 1 tRNA-Gly [6640,6710] 33 (tcc)\\n[10:09:20] 1 tRNA-Trp [16368,16441] 34 (cca)\\n[10:09:20] 1 tRNA-Pro [27904,27978] 35 (tgg)\\n[10:09:20] 1 tRNA-Leu [13041,13123] 35 (cag)\\n[10:09:20] 1 tRNA-Ala [18967,19041] 35 (cgc)\\n[10:09:20] 1 tRNA-Met [14991,15066] 36 (cat)\\n[10:09:20] 1 tRNA-Ser [4818,4904] 36 (cga)\\n[10:09:20] 1 tRNA-Thr [12901,12972] 33 (tgt)\\n[10:09:20] 1 tRNA-Ser c[10428,10514] 36 (tga)\\n[10:09:20] 1 tRNA-Arg [4270,4345] 36 (cct)\\n[10:09:20] 1 tRNA-Lys [34870,34943] 34 (ttt)\\n[10:09:20] 1 tRNA-Thr [13891,13962] 33 (cgt)\\n[10:09:20] 1 tRNA-Gly c[342,413] 33 (gcc)\\n[10:09:20] 2 tRNA-Asn [17324,17397] 34 (gtt)\\n[10:09:20] 1 tRNA-Pro [13680,13753] 35 (ggg)\\n[10:09:20] 1 tRNA-Ser [20412,20500] 36 (gga)\\n[10:09:20] 1 tRNA-Asp c[8647,8720] 35 (gtc)\\n[10:09:20] 1 tRNA-Gln [32294,32365] 33 (ttg)\\n[10:09:20] 1 tRNA-Leu [48868,48952] 35 (caa)\\n[10:09:20] Found 42 tRNAs\\n[10:09:20] Predicting Ribosomal RNAs\\n[10:09:20] Running Barrnap with 4 threads\\n[10:09:21] Found 0 rRNAs\\n[10:09:21] Skipping ncRNA search, enable with --rfam if desired.\\n[10:09:21] Total of 41 tRNA + rRNA features\\n[10:09:21] Searching for CRISPR repeats\\n[10:09:21] CRISPR1 NGZX01000064.1 1960 with 5 spacers\\n[10:09:21] Found 1 CRISPRs\\n[10:09:21] Predicting coding sequences\\n[10:09:21] Contigs total 5270127 bp, so using single mode\\n[10:09:21] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZX01\\\\/\\\\/Fischerella_NGZX01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:09:25] Excluding CDS which overlaps existing RNA (tRNA) at NGZX01000051.1:31546..31707 on + strand\\n[10:09:25] Excluding CDS which overlaps existing RNA (repeat_region) at NGZX01000064.1:1946..2194 on + strand\\n[10:09:26] Excluding CDS which overlaps existing RNA (tRNA) at NGZX01000066.1:10960..11631 on + strand\\n[10:09:27] Excluding CDS which overlaps existing RNA (tRNA) at NGZX01000294.1:17198..18637 on + strand\\n[10:09:27] Excluding CDS which overlaps existing RNA (tRNA) at NGZX01000307.1:8682..9077 on + strand\\n[10:09:27] Found 4427 CDS\\n[10:09:27] Connecting features back to sequences\\n[10:09:27] Not using genus-specific database. Try --usegenus to enable it.\\n[10:09:27] Annotating CDS, please be patient.\\n[10:09:27] Will use 4 CPUs for similarity searching.\\n[10:09:28] There are still 4427 unannotated CDS left (started with 4427)\\n[10:09:28] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:09:28] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZX01\\\\/\\\\/Fischerella_NGZX01\\\\.IS\\\\.tmp\\\\.97753\\\\.faa | parallel --gnu --plain -j 4 --block 169140 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZX01\\\\/\\\\/Fischerella_NGZX01\\\\.IS\\\\.tmp\\\\.97753\\\\.blast 2> /dev/null\\n[10:09:46] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZX01\\\\/\\\\/Fischerella_NGZX01\\\\.IS\\\\.tmp\\\\.97753\\\\.faa | parallel --gnu --plain -j 4 --block 169140 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZX01\\\\/\\\\/Fischerella_NGZX01\\\\.IS\\\\.tmp\\\\.97753\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:09:46] This is prokka 1.14.6\\n[10:09:46] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:09:46] Homepage is https://github.com/tseemann/prokka\\n[10:09:46] Local time is Mon Feb 21 10:09:46 2022\\n[10:09:46] You are tiagoferreiraleao\\n[10:09:46] Operating system is darwin\\n[10:09:46] You have BioPerl 1.007002\\n[10:09:46] System has 4 cores.\\n[10:09:46] Option --cpu asked for 12 cores, but system only has 4\\n[10:09:46] Will use maximum of 4 cores.\\n[10:09:46] Annotating as >>> Bacteria <<<\\n[10:09:46] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQF01.fasta' contents.\\n[10:09:46] Setting --locustag GBNHMPAN from MD5 0b7169a72010d8be6f828ecb7fb165fd\\n[10:09:46] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQF01/\\n[10:09:46] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQF01\\\\/\\n[10:09:46] Using filename prefix: Fischerella_NMQF01.XXX\\n[10:09:46] Setting HMMER_NCPU=1\\n[10:09:46] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQF01//Fischerella_NMQF01.log\\n[10:09:46] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQF01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQF01/ --prefix Fischerella_NMQF01 --cpus 12\\n[10:09:46] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:09:46] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:09:46] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:09:46] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:09:46] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:09:46] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:09:46] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:09:46] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:09:46] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:09:46] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:09:46] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:09:46] Looking for 'egrep' - found /usr/bin/egrep\\n[10:09:47] Looking for 'find' - found /usr/bin/find\\n[10:09:47] Looking for 'grep' - found /usr/bin/grep\\n[10:09:47] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:09:47] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:09:47] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:09:47] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:09:47] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:09:47] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:09:47] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:09:47] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:09:47] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:09:47] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:09:47] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:09:47] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:09:47] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:09:47] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:09:47] Looking for 'sed' - found /usr/bin/sed\\n[10:09:47] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:09:47] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:09:47] Using genetic code table 11.\\n[10:09:47] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQF01.fasta\\n[10:09:48] Wrote 922 contigs totalling 5074053 bp.\\n[10:09:48] Predicting tRNAs and tmRNAs\\n[10:09:48] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQF01\\\\/\\\\/Fischerella_NMQF01\\\\.fna\\n[10:09:49] 1 tRNA-Leu [9512,9594] 36 (gag)\\n[10:09:49] 1 tRNA-Arg c[14793,14867] 35 (tct)\\n[10:09:49] 1 tRNA-Val c[1414,1487] 35 (gac)\\n[10:09:49] 2 tRNA-Met [4458,4534] 35 (cat)\\n[10:09:49] 1 tRNA-Phe [3300,3372] 34 (gaa)\\n[10:09:49] 1 tRNA-Gly [6460,6530] 33 (tcc)\\n[10:09:49] 1 tRNA-Arg [449,521] 34 (tcg)\\n[10:09:49] 2 tRNA-Asn [1247,1320] 34 (att)\\n[10:09:49] 1 tRNA-Gly [3861,3932] 33 (gcc)\\n[10:09:49] 1 tRNA-Arg [4182,4257] 36 (cct)\\n[10:09:49] 1 tRNA-Val [4078,4150] 34 (tac)\\n[10:09:49] 1 tRNA-Glu [8816,8888] 35 (ttc)\\n[10:09:49] 1 tRNA-Gly [11326,11397] 33 (ccc)\\n[10:09:49] 1 tRNA-Ser c[13986,14079] 36 (gct)\\n[10:09:49] 1 tRNA-Arg [1471,1544] 35 (acg)\\n[10:09:49] 1 tRNA-Thr c[3703,3774] 33 (tgt)\\n[10:09:49] 1 tRNA-Asn c[9410,9483] 34 (gtt)\\n[10:09:49] 1 tRNA-Ser [21029,21114] 35 (tga)\\n[10:09:49] 1 tRNA-Pro c[335,409] 35 (tgg)\\n[10:09:49] 1 tRNA-Lys [13063,13136] 34 (ttt)\\n[10:09:49] 1 tRNA-Trp c[4282,4355] 34 (cca)\\n[10:09:49] 1 tRNA-Gln [7114,7185] 33 (ttg)\\n[10:09:49] 1 tRNA-Ala c[2468,2541] 34 (ggc)\\n[10:09:49] 2 tRNA-Thr c[3931,4002] 33 (cgt)\\n[10:09:49] 1 tRNA-Ile [732,808] 35 (gat)\\n[10:09:49] 2 tRNA-Ala [825,900] 34 (tgc)\\n[10:09:49] 1 tRNA-Pro c[2976,3049] 35 (ggg)\\n[10:09:49] 1 tRNA-Leu c[1058,1140] 35 (cag)\\n[10:09:49] tRNA [7067,7138] is a pseudo/wacky gene - skipping.\\n[10:09:49] 1 tRNA-Ala c[2310,2384] 35 (cgc)\\n[10:09:49] 1 tRNA-Leu c[381,465] 35 (caa)\\n[10:09:49] 1 tRNA-Arg c[6106,6179] 35 (ccg)\\n[10:09:49] 1 tRNA-Asp [4061,4134] 35 (gtc)\\n[10:09:49] 1 tRNA-Met [3885,3960] 36 (cat)\\n[10:09:49] 1 tRNA-Ser c[142,228] 36 (cga)\\n[10:09:49] 1 tRNA-Ser [12418,12506] 36 (gga)\\n[10:09:49] 1 tRNA-Cys c[541,612] 33 (gca)\\n[10:09:49] 1 tRNA-His [729,802] 34 (gtg)\\n[10:09:49] 1 tRNA-Met [3349,3421] 34 (cat)\\n[10:09:49] 1 tRNA-Leu [4516,4599] 36 (tag)\\n[10:09:49] 1 tRNA-Thr c[2615,2686] 33 (ggt)\\n[10:09:49] 1 tRNA-Pro c[943,1016] 35 (cgg)\\n[10:09:49] 1 tmRNA [1135,1521] 100,150 ANNIVKFARKEALVAA*\\n[10:09:49] Found 42 tRNAs\\n[10:09:49] Predicting Ribosomal RNAs\\n[10:09:49] Running Barrnap with 4 threads\\n[10:09:50] 1 NMQF01000166.1 4 23S ribosomal RNA (partial)\\n[10:09:50] 2 NMQF01000241.1 1 16S ribosomal RNA (partial)\\n[10:09:50] 3 NMQF01000252.1 82 23S ribosomal RNA (partial)\\n[10:09:50] 4 NMQF01000452.1 1 16S ribosomal RNA (partial)\\n[10:09:50] 5 NMQF01000611.1 3 16S ribosomal RNA (partial)\\n[10:09:50] 6 NMQF01000615.1 4 23S ribosomal RNA (partial)\\n[10:09:50] 7 NMQF01000627.1 13 5S ribosomal RNA\\n[10:09:50] 8 NMQF01000841.1 120 16S ribosomal RNA (partial)\\n[10:09:50] Found 8 rRNAs\\n[10:09:50] Skipping ncRNA search, enable with --rfam if desired.\\n[10:09:50] Total of 49 tRNA + rRNA features\\n[10:09:50] Searching for CRISPR repeats\\n[10:09:51] CRISPR1 NMQF01000656.1 2 with 13 spacers\\n[10:09:51] CRISPR2 NMQF01000674.1 5 with 12 spacers\\n[10:09:51] CRISPR3 NMQF01000850.1 3 with 3 spacers\\n[10:09:51] CRISPR4 NMQF01000889.1 70 with 7 spacers\\n[10:09:51] CRISPR5 NMQF01000892.1 75 with 7 spacers\\n[10:09:51] CRISPR6 NMQF01000896.1 74 with 14 spacers\\n[10:09:51] Found 6 CRISPRs\\n[10:09:51] Predicting coding sequences\\n[10:09:51] Contigs total 5074053 bp, so using single mode\\n[10:09:51] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQF01\\\\/\\\\/Fischerella_NMQF01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:09:55] Excluding CDS which overlaps existing RNA (tRNA) at NMQF01000122.1:263..550 on + strand\\n[10:09:55] Excluding CDS which overlaps existing RNA (tRNA) at NMQF01000206.1:916..1587 on - strand\\n[10:09:55] Excluding CDS which overlaps existing RNA (tRNA) at NMQF01000251.1:8170..9609 on - strand\\n[10:09:55] Excluding CDS which overlaps existing RNA (rRNA) at NMQF01000252.1:1139..1345 on + strand\\n[10:09:56] Excluding CDS which overlaps existing RNA (rRNA) at NMQF01000615.1:612..791 on - strand\\n[10:09:56] Excluding CDS which overlaps existing RNA (tRNA) at NMQF01000617.1:97..294 on - strand\\n[10:09:56] Excluding CDS which overlaps existing RNA (repeat_region) at NMQF01000656.1:574..801 on - strand\\n[10:09:56] Found 4304 CDS\\n[10:09:56] Connecting features back to sequences\\n[10:09:56] Not using genus-specific database. Try --usegenus to enable it.\\n[10:09:56] Annotating CDS, please be patient.\\n[10:09:56] Will use 4 CPUs for similarity searching.\\n[10:09:57] There are still 4304 unannotated CDS left (started with 4304)\\n[10:09:57] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:09:57] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQF01\\\\/\\\\/Fischerella_NMQF01\\\\.IS\\\\.tmp\\\\.97867\\\\.faa | parallel --gnu --plain -j 4 --block 166982 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQF01\\\\/\\\\/Fischerella_NMQF01\\\\.IS\\\\.tmp\\\\.97867\\\\.blast 2> /dev/null\\n[10:10:12] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQF01\\\\/\\\\/Fischerella_NMQF01\\\\.IS\\\\.tmp\\\\.97867\\\\.faa | parallel --gnu --plain -j 4 --block 166982 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQF01\\\\/\\\\/Fischerella_NMQF01\\\\.IS\\\\.tmp\\\\.97867\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:10:12] This is prokka 1.14.6\\n[10:10:12] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:10:12] Homepage is https://github.com/tseemann/prokka\\n[10:10:12] Local time is Mon Feb 21 10:10:12 2022\\n[10:10:12] You are tiagoferreiraleao\\n[10:10:12] Operating system is darwin\\n[10:10:12] You have BioPerl 1.007002\\n[10:10:12] System has 4 cores.\\n[10:10:12] Option --cpu asked for 12 cores, but system only has 4\\n[10:10:12] Will use maximum of 4 cores.\\n[10:10:12] Annotating as >>> Bacteria <<<\\n[10:10:12] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Mastigocladus_NGJL01.fasta' contents.\\n[10:10:12] Setting --locustag FHDPKJBC from MD5 f1d943bcff7364560b2317fb9700d00d\\n[10:10:12] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Mastigocladus_NGJL01/\\n[10:10:12] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Mastigocladus_NGJL01\\\\/\\n[10:10:12] Using filename prefix: Mastigocladus_NGJL01.XXX\\n[10:10:12] Setting HMMER_NCPU=1\\n[10:10:12] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Mastigocladus_NGJL01//Mastigocladus_NGJL01.log\\n[10:10:12] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Mastigocladus_NGJL01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Mastigocladus_NGJL01/ --prefix Mastigocladus_NGJL01 --cpus 12\\n[10:10:12] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:10:12] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:10:12] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:10:12] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:10:12] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:10:12] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:10:13] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:10:13] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:10:13] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:10:13] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:10:13] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:10:13] Looking for 'egrep' - found /usr/bin/egrep\\n[10:10:13] Looking for 'find' - found /usr/bin/find\\n[10:10:13] Looking for 'grep' - found /usr/bin/grep\\n[10:10:13] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:10:13] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:10:13] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:10:13] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:10:13] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:10:13] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:10:13] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:10:13] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:10:13] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:10:13] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:10:13] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:10:13] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:10:13] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:10:13] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:10:13] Looking for 'sed' - found /usr/bin/sed\\n[10:10:13] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:10:14] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:10:14] Using genetic code table 11.\\n[10:10:14] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Mastigocladus_NGJL01.fasta\\n[10:10:14] Wrote 264 contigs totalling 5289159 bp.\\n[10:10:14] Predicting tRNAs and tmRNAs\\n[10:10:14] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Mastigocladus_NGJL01\\\\/\\\\/Mastigocladus_NGJL01\\\\.fna\\n[10:10:16] 1 tRNA-Trp [14800,14873] 34 (cca)\\n[10:10:16] 1 tRNA-Asn c[1277,1356] 40 (gtt)\\n[10:10:16] 1 tRNA-Pro [10599,10673] 35 (tgg)\\n[10:10:16] 1 tRNA-Asn c[3439,3512] 34 (att)\\n[10:10:16] 1 tRNA-Val c[2713,2785] 34 (tac)\\n[10:10:16] 2 tRNA-Phe [10288,10360] 34 (gaa)\\n[10:10:16] 1 tRNA-Ala c[1687,1760] 34 (ggc)\\n[10:10:16] 1 tRNA-Gln [5709,5779] 32 (ttg)\\n[10:10:16] 1 tRNA-Leu c[22613,22696] 36 (tag)\\n[10:10:16] 1 tRNA-Arg [15667,15741] 35 (tct)\\n[10:10:16] 1 tRNA-Gly [14384,14455] 33 (ccc)\\n[10:10:16] 1 tmRNA [2506,2907] 100,150 ANNIVKFARKEALVAA*\\n[10:10:16] 1 tRNA-Met [19940,20016] 35 (cat)\\n[10:10:16] 1 tRNA-Gly [17783,17853] 33 (tcc)\\n[10:10:16] 2 tRNA-His c[45491,45564] 34 (gtg)\\n[10:10:16] 1 tRNA-Thr [2129,2200] 33 (tgt)\\n[10:10:16] 1 tRNA-Pro c[25723,25796] 35 (cgg)\\n[10:10:16] 1 tRNA-Met c[35124,35199] 36 (cat)\\n[10:10:16] 1 tRNA-Thr [5040,5120] 33 (cgt)\\n[10:10:16] 1 tRNA-Pro [61782,61855] 35 (ggg)\\n[10:10:16] 1 tRNA-Ser c[18442,18530] 36 (gga)\\n[10:10:16] 1 tRNA-Lys c[20652,20725] 34 (ttt)\\n[10:10:16] 1 tRNA-Gln [16983,17054] 33 (ttg)\\n[10:10:16] 1 tRNA-Leu [12537,12621] 35 (caa)\\n[10:10:16] 1 tRNA-Ser [14001,14090] 39 (cga)\\n[10:10:16] 1 tRNA-Asp c[109304,109377] 35 (gtc)\\n[10:10:16] Found 26 tRNAs\\n[10:10:16] Predicting Ribosomal RNAs\\n[10:10:16] Running Barrnap with 4 threads\\n[10:10:16] Found 0 rRNAs\\n[10:10:16] Skipping ncRNA search, enable with --rfam if desired.\\n[10:10:16] Total of 25 tRNA + rRNA features\\n[10:10:16] Searching for CRISPR repeats\\n[10:10:17] Found 0 CRISPRs\\n[10:10:17] Predicting coding sequences\\n[10:10:17] Contigs total 5289159 bp, so using single mode\\n[10:10:17] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Mastigocladus_NGJL01\\\\/\\\\/Mastigocladus_NGJL01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:10:21] Excluding CDS which overlaps existing RNA (tRNA) at NGJL01000015.1:988..1308 on - strand\\n[10:10:21] Excluding CDS which overlaps existing RNA (tRNA) at NGJL01000050.1:2705..2866 on - strand\\n[10:10:24] Found 8451 CDS\\n[10:10:24] Connecting features back to sequences\\n[10:10:24] Not using genus-specific database. Try --usegenus to enable it.\\n[10:10:24] Annotating CDS, please be patient.\\n[10:10:24] Will use 4 CPUs for similarity searching.\\n[10:10:27] There are still 8451 unannotated CDS left (started with 8451)\\n[10:10:27] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:10:27] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Mastigocladus_NGJL01\\\\/\\\\/Mastigocladus_NGJL01\\\\.IS\\\\.tmp\\\\.97980\\\\.faa | parallel --gnu --plain -j 4 --block 160777 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Mastigocladus_NGJL01\\\\/\\\\/Mastigocladus_NGJL01\\\\.IS\\\\.tmp\\\\.97980\\\\.blast 2> /dev/null\\n[10:10:44] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Mastigocladus_NGJL01\\\\/\\\\/Mastigocladus_NGJL01\\\\.IS\\\\.tmp\\\\.97980\\\\.faa | parallel --gnu --plain -j 4 --block 160777 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Mastigocladus_NGJL01\\\\/\\\\/Mastigocladus_NGJL01\\\\.IS\\\\.tmp\\\\.97980\\\\.blast 2> /dev/null\\n\"\n",
      "b\"[10:10:45] This is prokka 1.14.6\\n[10:10:45] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:10:45] Homepage is https://github.com/tseemann/prokka\\n[10:10:45] Local time is Mon Feb 21 10:10:45 2022\\n[10:10:45] You are tiagoferreiraleao\\n[10:10:45] Operating system is darwin\\n[10:10:45] You have BioPerl 1.007002\\n[10:10:45] System has 4 cores.\\n[10:10:45] Option --cpu asked for 12 cores, but system only has 4\\n[10:10:45] Will use maximum of 4 cores.\\n[10:10:45] Annotating as >>> Bacteria <<<\\n[10:10:45] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKH01.fasta' contents.\\n[10:10:45] Setting --locustag PKMMFJPH from MD5 9466f39111cd657fab10f9f98b1d96ba\\n[10:10:45] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKH01/\\n[10:10:45] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKH01\\\\/\\n[10:10:45] Using filename prefix: Nostoc_JABXKH01.XXX\\n[10:10:45] Setting HMMER_NCPU=1\\n[10:10:45] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKH01//Nostoc_JABXKH01.log\\n[10:10:45] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKH01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKH01/ --prefix Nostoc_JABXKH01 --cpus 12\\n[10:10:45] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:10:45] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:10:45] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:10:45] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:10:45] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:10:45] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:10:45] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:10:45] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:10:45] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:10:45] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:10:45] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:10:45] Looking for 'egrep' - found /usr/bin/egrep\\n[10:10:45] Looking for 'find' - found /usr/bin/find\\n[10:10:45] Looking for 'grep' - found /usr/bin/grep\\n[10:10:45] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:10:45] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:10:45] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:10:45] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:10:45] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:10:45] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:10:45] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:10:45] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:10:46] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:10:46] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:10:46] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:10:46] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:10:46] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:10:46] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:10:46] Looking for 'sed' - found /usr/bin/sed\\n[10:10:46] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:10:46] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:10:46] Using genetic code table 11.\\n[10:10:46] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKH01.fasta\\n[10:10:46] Wrote 183 contigs totalling 7571466 bp.\\n[10:10:46] Predicting tRNAs and tmRNAs\\n[10:10:46] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKH01\\\\/\\\\/Nostoc_JABXKH01\\\\.fna\\n[10:10:49] 1 tRNA-Phe c[28274,28346] 34 (gaa)\\n[10:10:49] 1 tRNA-Arg c[24321,24397] 36 (cct)\\n[10:10:49] 1 tRNA-Asp [20512,20585] 35 (gtc)\\n[10:10:49] 1 tRNA-Val [40580,40652] 34 (tac)\\n[10:10:49] 2 tRNA-Ser c[105397,105490] 36 (gct)\\n[10:10:49] 3 tRNA-Lys c[110950,111020] 33 (ttt)\\n[10:10:49] 1 tRNA-Arg c[105832,105904] 34 (ccg)\\n[10:10:49] 2 tRNA-Pro c[132455,132528] 35 (ggg)\\n[10:10:49] 3 tRNA-Met [145135,145209] 35 (cat)\\n[10:10:49] 4 tRNA-Ser [153539,153625] 36 (cga)\\n[10:10:49] 1 tRNA-Gly c[28830,28901] 33 (gcc)\\n[10:10:49] 1 tRNA-Arg [90468,90542] 35 (tct)\\n[10:10:49] 1 tmRNA [96415,96802] 101,151 ANNIVNFARKDALVAA*\\n[10:10:49] 1 tRNA-Trp [52761,52834] 34 (cca)\\n[10:10:49] 1 tRNA-Arg [35052,35125] 35 (acg)\\n[10:10:49] 2 tRNA-Cys [133482,133552] 33 (gca)\\n[10:10:49] 1 tRNA-Pro [43718,43791] 35 (cgg)\\n[10:10:49] 1 tRNA-Thr c[13882,13953] 33 (ggt)\\n[10:10:49] 2 tRNA-Tyr c[14041,14126] 35 (gta)\\n[10:10:49] 3 tRNA-Ser [16425,16511] 36 (tga)\\n[10:10:49] 1 tRNA-Ala [31118,31190] 34 (ggc)\\n[10:10:49] 1 tRNA-Val [4870,4943] 35 (gac)\\n[10:10:49] 1 tRNA-Met [3491,3566] 34 (cat)\\n[10:10:49] 2 tRNA-Met [3566,3640] 33 (cat)\\n[10:10:49] 1 tRNA-Gly c[5783,5853] 33 (tcc)\\n[10:10:49] 2 tRNA-Ser [28427,28513] 36 (gga)\\n[10:10:49] 1 tRNA-Gln c[13992,14063] 33 (ttg)\\n[10:10:49] 2 tRNA-Leu c[15739,15823] 35 (caa)\\n[10:10:49] 1 tRNA-Asn [2409,2480] 33 (gtt)\\n[10:10:49] 1 tRNA-His c[305,379] 35 (gtg)\\n[10:10:49] 2 tRNA-Ala c[385,459] 35 (tgc)\\n[10:10:49] 3 tRNA-Arg c[496,571] 35 (tct)\\n[10:10:49] 4 tRNA-Val c[578,653] 34 (tac)\\n[10:10:49] 5 tRNA-Gly c[659,730] 34 (gcc)\\n[10:10:49] 6 tRNA-Thr c[734,808] 35 (tgt)\\n[10:10:49] 7 tRNA-Met c[816,892] 34 (cat)\\n[10:10:49] 8 tRNA-Leu c[902,976] 35 (tag)\\n[10:10:49] 9 tRNA-Pro c[979,1052] 34 (tgg)\\n[10:10:49] 10 tRNA-Leu c[1234,1311] 35 (taa)\\n[10:10:49] 11 tRNA-Leu c[1318,1394] 35 (caa)\\n[10:10:49] 12 tRNA-Gln c[1401,1476] 34 (ttg)\\n[10:10:49] 13 tRNA-Gln c[1480,1552] 34 (ctg)\\n[10:10:49] 14 tRNA-Asn c[1703,1777] 33 (gtt)\\n[10:10:49] 15 tRNA-Phe c[1783,1858] 36 (gaa)\\n[10:10:49] 16 tRNA-Arg c[1879,1955] 36 (acg)\\n[10:10:49] 17 tRNA-Ser c[1958,2049] 36 (gct)\\n[10:10:49] 18 tRNA-Tyr c[2052,2135] 37 (gta)\\n[10:10:49] 19 tRNA-Glu c[2143,2216] 35 (ttc)\\n[10:10:49] tRNA c[2334,2408] is a pseudo/wacky gene - skipping.\\n[10:10:49] 21 tRNA-Met c[2411,2485] 33 (cat)\\n[10:10:49] 22 tRNA-Cys c[2496,2568] 35 (gca)\\n[10:10:49] 23 tRNA-Glu c[9837,9910] 35 (ttc)\\n[10:10:49] 1 tRNA-Pro [94560,94633] 35 (tgg)\\n[10:10:49] tRNA c[111179,111251] is a pseudo/wacky gene - skipping.\\n[10:10:49] 1 tRNA-Met [31230,31303] 33 (cat)\\n[10:10:49] 2 tRNA-Thr c[33434,33505] 33 (tgt)\\n[10:10:49] 1 tRNA-Leu [5412,5492] 34 (cag)\\n[10:10:49] 1 tRNA-His c[1868,1941] 34 (gtg)\\n[10:10:49] tRNA [16945,17016] is a pseudo/wacky gene - skipping.\\n[10:10:49] 2 tRNA-Ser [18378,18450] 33 (tga)\\n[10:10:49] 1 tRNA-Leu [65392,65475] 36 (tag)\\n[10:10:49] 1 tRNA-Gly [7658,7729] 33 (ccc)\\n[10:10:49] 1 tRNA-Thr c[88303,88374] 33 (cgt)\\n[10:10:49] 1 tRNA-Ala [111336,111410] 35 (tgc)\\n[10:10:49] 1 tRNA-Lys [15145,15216] 33 (ttt)\\n[10:10:49] 1 tRNA-Lys c[2722,2794] 33 (ctt)\\n[10:10:49] 1 tRNA-Ala c[7378,7452] 35 (cgc)\\n[10:10:49] 1 tRNA-Leu c[4415,4497] 36 (gag)\\n[10:10:49] Found 65 tRNAs\\n[10:10:49] Predicting Ribosomal RNAs\\n[10:10:49] Running Barrnap with 4 threads\\n[10:10:50] Found 0 rRNAs\\n[10:10:50] Skipping ncRNA search, enable with --rfam if desired.\\n[10:10:50] Total of 64 tRNA + rRNA features\\n[10:10:50] Searching for CRISPR repeats\\n[10:10:50] CRISPR1 JABXKH010000082.1 21822 with 23 spacers\\n[10:10:51] CRISPR2 JABXKH010000117.1 18238 with 16 spacers\\n[10:10:51] CRISPR3 JABXKH010000137.1 91629 with 12 spacers\\n[10:10:51] CRISPR4 JABXKH010000160.1 67713 with 5 spacers\\n[10:10:51] CRISPR5 JABXKH010000162.1 31168 with 15 spacers\\n[10:10:51] Found 5 CRISPRs\\n[10:10:51] Predicting coding sequences\\n[10:10:51] Contigs total 7571466 bp, so using single mode\\n[10:10:51] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKH01\\\\/\\\\/Nostoc_JABXKH01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:10:59] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKH010000117.1:18737..18874 on + strand\\n[10:10:59] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKH010000117.1:19337..19579 on - strand\\n[10:10:59] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKH010000137.1:91540..94434 on + strand\\n[10:11:00] Excluding CDS which overlaps existing RNA (tRNA) at JABXKH010000171.1:15113..15346 on - strand\\n[10:11:00] Found 6732 CDS\\n[10:11:00] Connecting features back to sequences\\n[10:11:00] Not using genus-specific database. Try --usegenus to enable it.\\n[10:11:00] Annotating CDS, please be patient.\\n[10:11:00] Will use 4 CPUs for similarity searching.\\n[10:11:02] There are still 6732 unannotated CDS left (started with 6732)\\n[10:11:02] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:11:02] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKH01\\\\/\\\\/Nostoc_JABXKH01\\\\.IS\\\\.tmp\\\\.98092\\\\.faa | parallel --gnu --plain -j 4 --block 255155 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKH01\\\\/\\\\/Nostoc_JABXKH01\\\\.IS\\\\.tmp\\\\.98092\\\\.blast 2> /dev/null\\n[10:11:08] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKH01\\\\/\\\\/Nostoc_JABXKH01\\\\.IS\\\\.tmp\\\\.98092\\\\.faa | parallel --gnu --plain -j 4 --block 255155 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKH01\\\\/\\\\/Nostoc_JABXKH01\\\\.IS\\\\.tmp\\\\.98092\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:11:08] This is prokka 1.14.6\\n[10:11:08] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:11:08] Homepage is https://github.com/tseemann/prokka\\n[10:11:08] Local time is Mon Feb 21 10:11:08 2022\\n[10:11:08] You are tiagoferreiraleao\\n[10:11:08] Operating system is darwin\\n[10:11:08] You have BioPerl 1.007002\\n[10:11:08] System has 4 cores.\\n[10:11:08] Option --cpu asked for 12 cores, but system only has 4\\n[10:11:08] Will use maximum of 4 cores.\\n[10:11:08] Annotating as >>> Bacteria <<<\\n[10:11:08] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_ALVX01.fasta' contents.\\n[10:11:08] Setting --locustag EIHKBLEB from MD5 e214b5ebe2716239d0318b3afb8468b8\\n[10:11:08] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_ALVX01/\\n[10:11:08] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVX01\\\\/\\n[10:11:08] Using filename prefix: Fischerella_ALVX01.XXX\\n[10:11:08] Setting HMMER_NCPU=1\\n[10:11:08] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_ALVX01//Fischerella_ALVX01.log\\n[10:11:08] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_ALVX01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_ALVX01/ --prefix Fischerella_ALVX01 --cpus 12\\n[10:11:08] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:11:08] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:11:08] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:11:08] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:11:08] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:11:08] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:11:09] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:11:09] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:11:09] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:11:09] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:11:09] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:11:09] Looking for 'egrep' - found /usr/bin/egrep\\n[10:11:09] Looking for 'find' - found /usr/bin/find\\n[10:11:09] Looking for 'grep' - found /usr/bin/grep\\n[10:11:09] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:11:09] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:11:09] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:11:09] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:11:09] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:11:09] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:11:09] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:11:09] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:11:09] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:11:09] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:11:09] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:11:09] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:11:09] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:11:09] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:11:09] Looking for 'sed' - found /usr/bin/sed\\n[10:11:09] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:11:09] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:11:09] Using genetic code table 11.\\n[10:11:09] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_ALVX01.fasta\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687436|gb|ALVX01000001.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687435|gb|ALVX01000002.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687434|gb|ALVX01000003.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687433|gb|ALVX01000004.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687432|gb|ALVX01000005.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687431|gb|ALVX01000006.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687430|gb|ALVX01000007.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687429|gb|ALVX01000008.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687428|gb|ALVX01000009.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687427|gb|ALVX01000010.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687426|gb|ALVX01000011.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687425|gb|ALVX01000012.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687424|gb|ALVX01000013.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687423|gb|ALVX01000014.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687422|gb|ALVX01000015.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687421|gb|ALVX01000016.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687420|gb|ALVX01000017.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687419|gb|ALVX01000018.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687418|gb|ALVX01000019.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687417|gb|ALVX01000020.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687416|gb|ALVX01000021.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687415|gb|ALVX01000022.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687414|gb|ALVX01000023.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687413|gb|ALVX01000024.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687412|gb|ALVX01000025.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687411|gb|ALVX01000026.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687410|gb|ALVX01000027.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687409|gb|ALVX01000028.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687408|gb|ALVX01000029.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687407|gb|ALVX01000030.1|\\n[10:11:09] Changing illegal '|' to '_' in sequence name: gi|530687406|gb|ALVX01000031.1|\\n[10:11:10] Changing illegal '|' to '_' in sequence name: gi|530687405|gb|ALVX01000032.1|\\n[10:11:10] Changing illegal '|' to '_' in sequence name: gi|530687404|gb|ALVX01000033.1|\\n[10:11:10] Changing illegal '|' to '_' in sequence name: gi|530687403|gb|ALVX01000034.1|\\n[10:11:10] Changing illegal '|' to '_' in sequence name: gi|530687402|gb|ALVX01000035.1|\\n[10:11:10] Changing illegal '|' to '_' in sequence name: gi|530687401|gb|ALVX01000036.1|\\n[10:11:10] Changing illegal '|' to '_' in sequence name: gi|530687400|gb|ALVX01000037.1|\\n[10:11:10] Changing illegal '|' to '_' in sequence name: gi|530687399|gb|ALVX01000038.1|\\n[10:11:10] Changing illegal '|' to '_' in sequence name: gi|530687398|gb|ALVX01000039.1|\\n[10:11:10] Changing illegal '|' to '_' in sequence name: gi|530687397|gb|ALVX01000040.1|\\n[10:11:10] Changing illegal '|' to '_' in sequence name: gi|530687396|gb|ALVX01000041.1|\\n[10:11:10] Changing illegal '|' to '_' in sequence name: gi|530687395|gb|ALVX01000042.1|\\n[10:11:10] Changing illegal '|' to '_' in sequence name: gi|530687394|gb|ALVX01000043.1|\\n[10:11:10] Wrote 43 contigs totalling 7164046 bp.\\n[10:11:10] Predicting tRNAs and tmRNAs\\n[10:11:10] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVX01\\\\/\\\\/Fischerella_ALVX01\\\\.fna\\n[10:11:12] 1 tRNA-Ala c[55734,55806] 34 (tgc)\\n[10:11:12] 2 tRNA-Arg c[55811,55884] 34 (tct)\\n[10:11:12] 3 tRNA-Gly c[55998,56068] 33 (tcc)\\n[10:11:12] 4 tRNA-Thr c[56074,56147] 34 (tgt)\\n[10:11:12] 5 tRNA-Leu c[56157,56233] 35 (taa)\\n[10:11:12] 6 tRNA-Ile c[56236,56311] 33 (tat)\\n[10:11:12] 7 tRNA-Lys c[56314,56388] 34 (ttt)\\n[10:11:12] 8 tRNA-Pro c[56393,56468] 36 (tgg)\\n[10:11:12] 9 tRNA-Gln c[56473,56547] 35 (ttg)\\n[10:11:12] 10 tRNA-Asn c[56552,56627] 34 (gtt)\\n[10:11:12] 11 tRNA-Phe c[56636,56710] 35 (gaa)\\n[10:11:12] 12 tRNA-Arg c[56715,56791] 36 (acg)\\n[10:11:12] 13 tRNA-Ser c[56796,56887] 35 (gct)\\n[10:11:12] 14 tRNA-Glu c[56995,57070] 35 (ttc)\\n[10:11:12] 15 tRNA-Trp c[57237,57309] 34 (cca)\\n[10:11:12] 16 tRNA-Met c[57312,57386] 33 (cat)\\n[10:11:12] 17 tRNA-Glu c[57392,57472] 39 (ctc)\\n[10:11:12] 18 tRNA-Asp c[57475,57552] 38 (gtc)\\n[10:11:12] 1 tRNA-Met c[229554,229627] 34 (cat)\\n[10:11:12] 2 tRNA-Pro [553894,553988] 35 (tgg)\\n[10:11:12] 3 tRNA-Ser [554020,554114] 35 (tga)\\n[10:11:12] 4 tRNA-Ser c[561320,561414] 36 (tga)\\n[10:11:12] 5 tRNA-Ser c[561383,561477] 36 (tga)\\n[10:11:12] 1 tRNA-Gly [198782,198852] 33 (tcc)\\n[10:11:12] 2 tRNA-Leu [321920,322002] 36 (gag)\\n[10:11:12] 1 tRNA-Arg [6137,6210] 35 (acg)\\n[10:11:12] 2 tRNA-Ser c[15947,16040] 36 (gct)\\n[10:11:12] 3 tRNA-Trp [111017,111090] 34 (cca)\\n[10:11:12] 4 tRNA-Phe c[306474,306546] 34 (gaa)\\n[10:11:12] 5 tRNA-Pro c[339987,340061] 35 (tgg)\\n[10:11:12] 6 tRNA-Leu c[419769,419851] 35 (tag)\\n[10:11:12] 1 tRNA-Ile [77,152] 35 (gat)\\n[10:11:12] 2 tRNA-Ala [240,314] 34 (tgc)\\n[10:11:12] 3 tRNA-Asp c[89663,89736] 35 (gtc)\\n[10:11:12] 4 tRNA-Lys [282144,282216] 33 (ctt)\\n[10:11:12] 5 tRNA-Lys [292848,292919] 33 (ttt)\\n[10:11:12] 1 tRNA-Pro [47751,47824] 35 (ggg)\\n[10:11:12] 2 tRNA-Thr c[68907,68978] 33 (tgt)\\n[10:11:12] 3 tRNA-Met [293942,294017] 36 (cat)\\n[10:11:12] 4 tRNA-Asn [404443,404516] 34 (gtt)\\n[10:11:12] 5 tRNA-Thr [478457,478528] 33 (cgt)\\n[10:11:12] 1 tRNA-Ala c[192148,192222] 34 (tgc)\\n[10:11:12] 1 tmRNA c[195752,196139] 101,154 ANNIVKFARKEALVAA**\\n[10:11:12] 2 tRNA-Gly c[244331,244402] 33 (ccc)\\n[10:11:12] 3 tRNA-Val [251518,251590] 34 (tac)\\n[10:11:12] 4 tRNA-Arg c[456500,456576] 36 (cct)\\n[10:11:12] 5 tRNA-Thr c[733718,733789] 33 (ggt)\\n[10:11:12] 6 tRNA-Tyr c[733866,733950] 35 (gta)\\n[10:11:12] 7 tRNA-Ser [782644,782730] 36 (cga)\\n[10:11:12] 1 tRNA-Cys c[20396,20467] 33 (gca)\\n[10:11:12] 2 tRNA-His [98263,98336] 34 (gtg)\\n[10:11:12] 3 tRNA-Arg [312192,312266] 35 (tct)\\n[10:11:12] 4 tRNA-Gln c[644700,644771] 33 (ttg)\\n[10:11:12] 5 tRNA-Leu c[676194,676278] 35 (caa)\\n[10:11:12] 1 tRNA-Pro [12842,12917] 36 (cgg)\\n[10:11:12] 2 tRNA-Trp c[86670,86763] 34 (cca)\\n[10:11:12] 1 tRNA-Ala [94738,94812] 35 (cgc)\\n[10:11:12] 1 tRNA-Ser [2511,2596] 35 (tga)\\n[10:11:12] 1 tRNA-Met c[121793,121869] 35 (cat)\\n[10:11:12] 2 tRNA-Val [124890,124963] 35 (gac)\\n[10:11:12] 3 tRNA-Glu c[152916,152988] 35 (ttc)\\n[10:11:12] 1 tRNA-Cys [20603,20675] 34 (gca)\\n[10:11:12] 2 tRNA-Lys [21386,21459] 34 (ctt)\\n[10:11:12] 1 tRNA-Arg [188461,188533] 34 (ccg)\\n[10:11:12] 2 tRNA-Leu c[431701,431783] 35 (cag)\\n[10:11:12] 1 tRNA-Gly c[14435,14506] 33 (gcc)\\n[10:11:12] 2 tRNA-Ser [96352,96440] 36 (gga)\\n[10:11:12] 3 tRNA-Ala c[128192,128264] 34 (ggc)\\n[10:11:12] 1 tRNA-Trp c[22691,22765] 35 (cca)\\n[10:11:12] 2 tRNA-Trp c[24989,25063] 35 (cca)\\n[10:11:12] 1 tRNA-Ala c[68720,68792] 34 (tgc)\\n[10:11:12] 2 tRNA-Arg c[68797,68870] 34 (tct)\\n[10:11:12] 1 tRNA-Asn c[122,197] 34 (gtt)\\n[10:11:12] 2 tRNA-Phe c[206,280] 35 (gaa)\\n[10:11:12] 3 tRNA-Arg c[285,379] 35 (acg)\\n[10:11:12] 4 tRNA-Ser c[384,466] 35 (gct)\\n[10:11:12] 5 tRNA-Lys c[697,768] 33 (ctt)\\n[10:11:12] Found 77 tRNAs\\n[10:11:12] Predicting Ribosomal RNAs\\n[10:11:12] Running Barrnap with 4 threads\\n[10:11:14] 1 gi_530687417_gb_ALVX01000020.1_ 114357 16S ribosomal RNA\\n[10:11:14] 2 gi_530687417_gb_ALVX01000020.1_ 116187 23S ribosomal RNA\\n[10:11:14] 3 gi_530687418_gb_ALVX01000019.1_ 58 16S ribosomal RNA\\n[10:11:14] 4 gi_530687419_gb_ALVX01000018.1_ 189119 23S ribosomal RNA\\n[10:11:14] 5 gi_530687422_gb_ALVX01000015.1_ 521 23S ribosomal RNA\\n[10:11:14] 6 gi_530687427_gb_ALVX01000010.1_ 239697 16S ribosomal RNA\\n[10:11:14] 7 gi_530687427_gb_ALVX01000010.1_ 241527 23S ribosomal RNA\\n[10:11:14] 8 gi_530687430_gb_ALVX01000007.1_ 29678 23S ribosomal RNA (partial)\\n[10:11:14] Found 8 rRNAs\\n[10:11:14] Skipping ncRNA search, enable with --rfam if desired.\\n[10:11:14] Total of 84 tRNA + rRNA features\\n[10:11:14] Searching for CRISPR repeats\\n[10:11:14] CRISPR1 gi_530687433_gb_ALVX01000004.1_ 60335 with 11 spacers\\n[10:11:14] CRISPR2 gi_530687433_gb_ALVX01000004.1_ 75284 with 36 spacers\\n[10:11:14] CRISPR3 gi_530687433_gb_ALVX01000004.1_ 164237 with 13 spacers\\n[10:11:14] CRISPR4 gi_530687430_gb_ALVX01000007.1_ 3657 with 62 spacers\\n[10:11:14] CRISPR5 gi_530687417_gb_ALVX01000020.1_ 604439 with 19 spacers\\n[10:11:15] CRISPR6 gi_530687410_gb_ALVX01000027.1_ 3479 with 4 spacers\\n[10:11:15] CRISPR7 gi_530687409_gb_ALVX01000028.1_ 1217 with 18 spacers\\n[10:11:15] CRISPR8 gi_530687403_gb_ALVX01000034.1_ 163762 with 5 spacers\\n[10:11:15] Found 8 CRISPRs\\n[10:11:15] Predicting coding sequences\\n[10:11:15] Contigs total 7164046 bp, so using single mode\\n[10:11:15] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVX01\\\\/\\\\/Fischerella_ALVX01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:11:21] Excluding CDS which overlaps existing RNA (tRNA) at gi_530687430_gb_ALVX01000007.1_:56551..56943 on + strand\\n[10:11:21] Excluding CDS which overlaps existing RNA (rRNA) at gi_530687427_gb_ALVX01000010.1_:243888..244064 on - strand\\n[10:11:21] Excluding CDS which overlaps existing RNA (tRNA) at gi_530687427_gb_ALVX01000010.1_:561238..561459 on - strand\\n[10:11:22] Excluding CDS which overlaps existing RNA (tRNA) at gi_530687423_gb_ALVX01000014.1_:413950..419778 on + strand\\n[10:11:22] Excluding CDS which overlaps existing RNA (tRNA) at gi_530687423_gb_ALVX01000014.1_:419747..419872 on - strand\\n[10:11:22] Excluding CDS which overlaps existing RNA (rRNA) at gi_530687422_gb_ALVX01000015.1_:2882..3058 on - strand\\n[10:11:23] Excluding CDS which overlaps existing RNA (tRNA) at gi_530687420_gb_ALVX01000017.1_:294017..296218 on - strand\\n[10:11:23] Excluding CDS which overlaps existing RNA (rRNA) at gi_530687419_gb_ALVX01000018.1_:189404..189580 on + strand\\n[10:11:23] Excluding CDS which overlaps existing RNA (rRNA) at gi_530687417_gb_ALVX01000020.1_:118548..118724 on - strand\\n[10:11:23] Excluding CDS which overlaps existing RNA (repeat_region) at gi_530687417_gb_ALVX01000020.1_:605379..605756 on + strand\\n[10:11:25] Excluding CDS which overlaps existing RNA (tRNA) at gi_530687400_gb_ALVX01000037.1_:68856..69023 on - strand\\n[10:11:25] Excluding CDS which overlaps existing RNA (tRNA) at gi_530687399_gb_ALVX01000038.1_:121..477 on + strand\\n[10:11:25] Found 6006 CDS\\n[10:11:25] Connecting features back to sequences\\n[10:11:25] Not using genus-specific database. Try --usegenus to enable it.\\n[10:11:25] Annotating CDS, please be patient.\\n[10:11:25] Will use 4 CPUs for similarity searching.\\n[10:11:27] There are still 6006 unannotated CDS left (started with 6006)\\n[10:11:27] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:11:27] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVX01\\\\/\\\\/Fischerella_ALVX01\\\\.IS\\\\.tmp\\\\.98204\\\\.faa | parallel --gnu --plain -j 4 --block 244360 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVX01\\\\/\\\\/Fischerella_ALVX01\\\\.IS\\\\.tmp\\\\.98204\\\\.blast 2> /dev/null\\n[10:11:33] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVX01\\\\/\\\\/Fischerella_ALVX01\\\\.IS\\\\.tmp\\\\.98204\\\\.faa | parallel --gnu --plain -j 4 --block 244360 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVX01\\\\/\\\\/Fischerella_ALVX01\\\\.IS\\\\.tmp\\\\.98204\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:11:33] This is prokka 1.14.6\\n[10:11:33] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:11:33] Homepage is https://github.com/tseemann/prokka\\n[10:11:33] Local time is Mon Feb 21 10:11:33 2022\\n[10:11:33] You are tiagoferreiraleao\\n[10:11:33] Operating system is darwin\\n[10:11:33] You have BioPerl 1.007002\\n[10:11:33] System has 4 cores.\\n[10:11:33] Option --cpu asked for 12 cores, but system only has 4\\n[10:11:33] Will use maximum of 4 cores.\\n[10:11:33] Annotating as >>> Bacteria <<<\\n[10:11:33] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JACYLY01.fasta' contents.\\n[10:11:33] Setting --locustag FKHAMMLE from MD5 f41a665ea95feadb4c8702f9a0dedd4e\\n[10:11:33] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JACYLY01/\\n[10:11:33] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYLY01\\\\/\\n[10:11:33] Using filename prefix: Fischerella_JACYLY01.XXX\\n[10:11:33] Setting HMMER_NCPU=1\\n[10:11:33] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JACYLY01//Fischerella_JACYLY01.log\\n[10:11:33] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JACYLY01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JACYLY01/ --prefix Fischerella_JACYLY01 --cpus 12\\n[10:11:33] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:11:33] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:11:33] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:11:33] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:11:33] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:11:33] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:11:33] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:11:33] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:11:33] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:11:33] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:11:33] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:11:33] Looking for 'egrep' - found /usr/bin/egrep\\n[10:11:33] Looking for 'find' - found /usr/bin/find\\n[10:11:33] Looking for 'grep' - found /usr/bin/grep\\n[10:11:33] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:11:33] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:11:33] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:11:33] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:11:33] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:11:33] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:11:34] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:11:34] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:11:34] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:11:34] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:11:34] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:11:34] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:11:34] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:11:34] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:11:34] Looking for 'sed' - found /usr/bin/sed\\n[10:11:34] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:11:34] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:11:34] Using genetic code table 11.\\n[10:11:34] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JACYLY01.fasta\\n[10:11:35] Wrote 271 contigs totalling 5203860 bp.\\n[10:11:35] Predicting tRNAs and tmRNAs\\n[10:11:35] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYLY01\\\\/\\\\/Fischerella_JACYLY01\\\\.fna\\n[10:11:37] 1 tRNA-Pro c[20710,20784] 35 (tgg)\\n[10:11:37] 1 tRNA-Ser [6760,6853] 36 (gct)\\n[10:11:37] 1 tRNA-Met [95,170] 36 (cat)\\n[10:11:37] 1 tRNA-Lys c[23081,23154] 34 (ttt)\\n[10:11:37] 1 tRNA-Arg [17798,17871] 35 (ccg)\\n[10:11:37] 1 tRNA-Arg c[47284,47357] 35 (acg)\\n[10:11:37] 2 tRNA-Ala c[52229,52302] 34 (ggc)\\n[10:11:37] 3 tRNA-Thr c[54123,54194] 33 (cgt)\\n[10:11:37] 1 tRNA-Gly c[22259,22330] 33 (gcc)\\n[10:11:37] 1 tRNA-Arg c[24729,24803] 35 (tct)\\n[10:11:37] 1 tRNA-Val c[8321,8393] 34 (tac)\\n[10:11:37] 2 tRNA-Phe [15743,15815] 34 (gaa)\\n[10:11:37] 1 tRNA-Met c[25964,26036] 34 (cat)\\n[10:11:37] 1 tRNA-Thr [52855,52926] 33 (tgt)\\n[10:11:37] 1 tRNA-Ser c[33205,33291] 36 (cga)\\n[10:11:37] 1 tRNA-Asn c[7462,7535] 34 (att)\\n[10:11:37] 2 tRNA-Arg c[8206,8278] 34 (tcg)\\n[10:11:37] 1 tRNA-His c[30978,31051] 34 (gtg)\\n[10:11:37] 1 tRNA-Ser c[48441,48529] 36 (gga)\\n[10:11:37] 1 tRNA-Asn [2675,2748] 34 (gtt)\\n[10:11:37] 1 tmRNA c[10071,10457] 100,153 ANNIVKFARKEALVAA**\\n[10:11:37] 1 tRNA-Gly c[180,251] 33 (ccc)\\n[10:11:37] 1 tRNA-Pro [15656,15729] 35 (ggg)\\n[10:11:37] 1 tRNA-Cys c[22732,22803] 33 (gca)\\n[10:11:37] 2 tRNA-Gly c[56135,56205] 33 (tcc)\\n[10:11:37] 1 tRNA-Trp c[6419,6492] 34 (cca)\\n[10:11:37] 1 tRNA-Ala [15549,15624] 35 (cgc)\\n[10:11:37] 2 tRNA-Leu [41275,41357] 35 (cag)\\n[10:11:37] 1 tRNA-Leu [53761,53843] 36 (gag)\\n[10:11:37] 1 tRNA-Glu [5851,5923] 35 (ttc)\\n[10:11:37] 1 tRNA-Asp [17561,17634] 35 (gtc)\\n[10:11:37] 1 tRNA-Met c[27429,27505] 35 (cat)\\n[10:11:37] 2 tRNA-Val [30476,30549] 35 (gac)\\n[10:11:37] 1 tRNA-Gln [31050,31120] 32 (ttg)\\n[10:11:37] 2 tRNA-Gly [31123,31192] 32 (tcc)\\n[10:11:37] 1 tRNA-Leu [143,226] 36 (tag)\\n[10:11:37] 1 tRNA-Leu [824,908] 35 (caa)\\n[10:11:37] 2 tRNA-Gln [33819,33890] 33 (ttg)\\n[10:11:37] 1 tRNA-Ser c[25427,25513] 36 (tga)\\n[10:11:37] Found 39 tRNAs\\n[10:11:37] Predicting Ribosomal RNAs\\n[10:11:37] Running Barrnap with 4 threads\\n[10:11:38] Found 0 rRNAs\\n[10:11:38] Skipping ncRNA search, enable with --rfam if desired.\\n[10:11:38] Total of 38 tRNA + rRNA features\\n[10:11:38] Searching for CRISPR repeats\\n[10:11:38] CRISPR1 JACYLY010000046.1 3321 with 3 spacers\\n[10:11:38] CRISPR2 JACYLY010000059.1 11004 with 20 spacers\\n[10:11:39] CRISPR3 JACYLY010000197.1 20599 with 21 spacers\\n[10:11:39] Found 3 CRISPRs\\n[10:11:39] Predicting coding sequences\\n[10:11:39] Contigs total 5203860 bp, so using single mode\\n[10:11:39] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYLY01\\\\/\\\\/Fischerella_JACYLY01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:11:44] Excluding CDS which overlaps existing RNA (tRNA) at JACYLY010000111.1:2690..3988 on + strand\\n[10:11:46] Found 4381 CDS\\n[10:11:46] Connecting features back to sequences\\n[10:11:46] Not using genus-specific database. Try --usegenus to enable it.\\n[10:11:46] Annotating CDS, please be patient.\\n[10:11:46] Will use 4 CPUs for similarity searching.\\n[10:11:47] There are still 4381 unannotated CDS left (started with 4381)\\n[10:11:47] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:11:47] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYLY01\\\\/\\\\/Fischerella_JACYLY01\\\\.IS\\\\.tmp\\\\.98317\\\\.faa | parallel --gnu --plain -j 4 --block 172442 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYLY01\\\\/\\\\/Fischerella_JACYLY01\\\\.IS\\\\.tmp\\\\.98317\\\\.blast 2> /dev/null\\n[10:12:04] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYLY01\\\\/\\\\/Fischerella_JACYLY01\\\\.IS\\\\.tmp\\\\.98317\\\\.faa | parallel --gnu --plain -j 4 --block 172442 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYLY01\\\\/\\\\/Fischerella_JACYLY01\\\\.IS\\\\.tmp\\\\.98317\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:12:05] This is prokka 1.14.6\\n[10:12:05] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:12:05] Homepage is https://github.com/tseemann/prokka\\n[10:12:05] Local time is Mon Feb 21 10:12:05 2022\\n[10:12:05] You are tiagoferreiraleao\\n[10:12:05] Operating system is darwin\\n[10:12:05] You have BioPerl 1.007002\\n[10:12:05] System has 4 cores.\\n[10:12:05] Option --cpu asked for 12 cores, but system only has 4\\n[10:12:05] Will use maximum of 4 cores.\\n[10:12:05] Annotating as >>> Bacteria <<<\\n[10:12:05] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHG01.fasta' contents.\\n[10:12:05] Setting --locustag MMDPNNOM from MD5 66d9778620943a71208cd5a9daff19ad\\n[10:12:05] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHG01/\\n[10:12:05] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHG01\\\\/\\n[10:12:05] Using filename prefix: Nostoc_LAHG01.XXX\\n[10:12:05] Setting HMMER_NCPU=1\\n[10:12:05] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHG01//Nostoc_LAHG01.log\\n[10:12:05] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHG01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHG01/ --prefix Nostoc_LAHG01 --cpus 12\\n[10:12:05] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:12:05] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:12:05] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:12:05] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:12:05] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:12:05] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:12:05] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:12:05] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:12:05] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:12:05] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:12:05] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:12:05] Looking for 'egrep' - found /usr/bin/egrep\\n[10:12:05] Looking for 'find' - found /usr/bin/find\\n[10:12:05] Looking for 'grep' - found /usr/bin/grep\\n[10:12:05] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:12:05] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:12:05] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:12:05] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:12:05] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:12:05] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:12:05] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:12:05] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:12:06] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:12:06] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:12:06] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:12:06] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:12:06] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:12:06] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:12:06] Looking for 'sed' - found /usr/bin/sed\\n[10:12:06] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:12:06] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:12:06] Using genetic code table 11.\\n[10:12:06] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHG01.fasta\\n[10:12:06] Wrote 345 contigs totalling 8997043 bp.\\n[10:12:06] Predicting tRNAs and tmRNAs\\n[10:12:06] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHG01\\\\/\\\\/Nostoc_LAHG01\\\\.fna\\n[10:12:09] 1 tRNA-Cys c[130753,130824] 33 (gca)\\n[10:12:09] 1 tRNA-Arg [29911,29985] 35 (tct)\\n[10:12:09] 1 tRNA-Ile [69064,69139] 35 (gat)\\n[10:12:09] 2 tRNA-Ala [69223,69297] 34 (tgc)\\n[10:12:09] 1 tRNA-Ala c[791,863] 34 (cgc)\\n[10:12:09] 2 tRNA-Leu [49565,49649] 35 (caa)\\n[10:12:09] 3 tRNA-Gln [54403,54476] 34 (ttg)\\n[10:12:09] 1 tmRNA [18277,18664] 101,151 ANNIVKFARREALVAA*\\n[10:12:09] 1 tRNA-Ser [35411,35497] 36 (cga)\\n[10:12:09] 1 tRNA-Leu c[42817,42900] 36 (tag)\\n[10:12:09] 1 tRNA-Pro [25155,25228] 35 (ggg)\\n[10:12:09] 1 tRNA-Ile c[32311,32386] 34 (tat)\\n[10:12:09] 2 tRNA-Met c[33992,34067] 34 (cat)\\n[10:12:09] 3 tRNA-Met c[34069,34144] 34 (cat)\\n[10:12:09] 1 tRNA-Ala c[800,872] 34 (ggc)\\n[10:12:09] 1 tRNA-Thr c[44933,45004] 33 (cgt)\\n[10:12:09] 1 tRNA-Gly c[28331,28402] 33 (gcc)\\n[10:12:09] 2 tRNA-Met c[35212,35285] 33 (cat)\\n[10:12:09] 1 tRNA-Pro [48002,48075] 35 (cgg)\\n[10:12:09] 1 tRNA-Ser c[15639,15732] 36 (gct)\\n[10:12:09] 2 tRNA-Lys c[21407,21477] 33 (ttt)\\n[10:12:09] 3 tRNA-Ser c[21736,21819] 34 (tga)\\n[10:12:09] 4 tRNA-His c[21824,21899] 35 (gtg)\\n[10:12:09] 5 tRNA-Ala c[21905,21979] 35 (tgc)\\n[10:12:09] 6 tRNA-Arg c[22071,22146] 35 (tct)\\n[10:12:09] 7 tRNA-Val c[22153,22228] 34 (tac)\\n[10:12:09] 8 tRNA-Thr c[22310,22382] 34 (tgt)\\n[10:12:09] 9 tRNA-Leu c[22677,22751] 35 (tag)\\n[10:12:09] 10 tRNA-Leu c[22863,22939] 34 (gag)\\n[10:12:09] 11 tRNA-Pro c[23047,23120] 35 (tgg)\\n[10:12:09] 12 tRNA-Leu c[23122,23199] 35 (taa)\\n[10:12:09] 13 tRNA-Leu c[23220,23297] 36 (caa)\\n[10:12:09] 14 tRNA-Gln c[23323,23398] 35 (ttg)\\n[10:12:09] 15 tRNA-Gln c[24273,24347] 35 (ctg)\\n[10:12:09] 16 tRNA-Asn c[24358,24432] 33 (gtt)\\n[10:12:09] 17 tRNA-Phe c[24438,24513] 36 (gaa)\\n[10:12:09] 18 tRNA-Arg c[24601,24676] 35 (acg)\\n[10:12:09] 19 tRNA-Ser c[25438,25524] 35 (gct)\\n[10:12:09] 20 tRNA-Tyr c[25528,25609] 35 (gta)\\n[10:12:09] 21 tRNA-Glu c[25620,25694] 35 (ttc)\\n[10:12:09] tRNA c[25782,25856] is a pseudo/wacky gene - skipping.\\n[10:12:09] 23 tRNA-Met c[25859,25932] 32 (cat)\\n[10:12:09] 24 tRNA-Glu c[26042,26118] 35 (ctc)\\n[10:12:09] 25 tRNA-Asp c[26122,26197] 36 (gtc)\\n[10:12:09] 1 tRNA-Asp [25602,25679] 38 (gtc)\\n[10:12:09] 2 tRNA-Glu [25682,25758] 35 (ctc)\\n[10:12:09] 3 tRNA-Cys [25763,25838] 34 (gca)\\n[10:12:09] 4 tRNA-Met [25844,25918] 33 (cat)\\n[10:12:09] 5 tRNA-Trp [25921,25993] 34 (cca)\\n[10:12:09] 6 tRNA-Pyl [26018,26089] 33 (cta)\\n[10:12:09] 7 tRNA-Glu [26120,26195] 34 (ttc)\\n[10:12:09] 8 tRNA-Tyr [26199,26282] 36 (gta)\\n[10:12:09] 9 tRNA-Ser [26286,26375] 33 (gct)\\n[10:12:09] 10 tRNA-Arg [26380,26455] 35 (acg)\\n[10:12:09] 11 tRNA-Phe [26460,26534] 35 (gaa)\\n[10:12:09] 12 tRNA-Asn [26543,26617] 33 (gtt)\\n[10:12:09] 13 tRNA-Gln [26623,26698] 35 (ttg)\\n[10:12:09] 14 tRNA-Leu [26706,26783] 35 (caa)\\n[10:12:09] 15 tRNA-Pro [26788,26864] 35 (tgg)\\n[10:12:09] 16 tRNA-Leu [26868,26942] 35 (tag)\\n[10:12:09] 17 tRNA-Leu [26962,27039] 35 (gag)\\n[10:12:09] 18 tRNA-Lys [27043,27119] 34 (ttt)\\n[10:12:09] 19 tRNA-Lys [27125,27199] 33 (ctt)\\n[10:12:09] 20 tRNA-Leu [27205,27280] 36 (taa)\\n[10:12:09] 21 tRNA-Cys [27290,27367] 34 (gca)\\n[10:12:09] 22 tRNA-Thr [27375,27447] 34 (tgt)\\n[10:12:09] 23 tRNA-Gly [27453,27523] 33 (tcc)\\n[10:12:09] 24 tRNA-Val [27529,27604] 34 (tac)\\n[10:12:09] 25 tRNA-Arg [27611,27685] 35 (tct)\\n[10:12:09] 26 tRNA-Ala [27690,27762] 34 (tgc)\\n[10:12:09] 27 tRNA-His [27770,27846] 35 (gtg)\\n[10:12:09] 28 tRNA-Ser [27857,27941] 34 (tga)\\n[10:12:09] 1 tRNA-Asp [30272,30345] 35 (gtc)\\n[10:12:09] 1 tRNA-Pro c[11710,11783] 35 (tgg)\\n[10:12:09] 2 tRNA-Ser c[20465,20551] 36 (tga)\\n[10:12:09] 3 tRNA-Tyr [23002,23086] 35 (gta)\\n[10:12:09] 4 tRNA-Thr [23162,23233] 33 (ggt)\\n[10:12:09] 1 tRNA-Gly [22177,22250] 34 (ccc)\\n[10:12:09] 2 tRNA-Ser c[30395,30482] 35 (gga)\\n[10:12:09] 1 tRNA-Trp c[67816,67889] 34 (cca)\\n[10:12:09] 1 tRNA-Leu [30678,30758] 35 (gag)\\n[10:12:09] 1 tRNA-Lys [21435,21506] 33 (ttt)\\n[10:12:09] 1 tRNA-Arg [4882,4954] 34 (ccg)\\n[10:12:09] 1 tRNA-Met c[16740,16814] 35 (cat)\\n[10:12:09] 1 tRNA-Val c[14143,14215] 34 (tac)\\n[10:12:09] 1 tRNA-Glu [22627,22700] 35 (ttc)\\n[10:12:09] 1 tRNA-Gly [17362,17432] 33 (tcc)\\n[10:12:09] 1 tRNA-Lys [21919,21991] 33 (ctt)\\n[10:12:09] 1 tRNA-Thr [2553,2625] 33 (tgt)\\n[10:12:09] 1 tRNA-His c[5985,6058] 34 (gtg)\\n[10:12:09] 1 tRNA-Thr [20584,20658] 33 (agt)\\n[10:12:09] 2 tRNA-Leu [22049,22123] 35 (caa)\\n[10:12:09] 1 tRNA-Phe c[179,252] 34 (gaa)\\n[10:12:09] 1 tRNA-Arg [1102,1175] 35 (acg)\\n[10:12:09] 1 tRNA-Ala c[3726,3812] 29 (cgc)\\n[10:12:09] 1 tRNA-Ala c[9909,9981] 34 (tgc)\\n[10:12:09] 1 tRNA-Asn c[13670,13743] 34 (gtt)\\n[10:12:09] 1 tRNA-Leu [10529,10610] 35 (cag)\\n[10:12:09] 1 tRNA-Val c[2639,2712] 35 (gac)\\n[10:12:09] tRNA c[3733,3808] is a pseudo/wacky gene - skipping.\\n[10:12:09] 2 tRNA-Ile c[3835,3911] 35 (gat)\\n[10:12:09] Found 99 tRNAs\\n[10:12:09] Predicting Ribosomal RNAs\\n[10:12:09] Running Barrnap with 4 threads\\n[10:12:11] 1 LAHG01000010.1 67457 16S ribosomal RNA\\n[10:12:11] 2 LAHG01000010.1 69470 23S ribosomal RNA\\n[10:12:11] 3 LAHG01000264.1 230 5S ribosomal RNA\\n[10:12:11] 4 LAHG01000264.1 541 23S ribosomal RNA\\n[10:12:11] 5 LAHG01000264.1 4046 16S ribosomal RNA\\n[10:12:11] 6 LAHG01000306.1 221 23S ribosomal RNA\\n[10:12:11] 7 LAHG01000306.1 3130 5S ribosomal RNA\\n[10:12:11] 8 LAHG01000340.1 2 23S ribosomal RNA (partial)\\n[10:12:11] Found 8 rRNAs\\n[10:12:11] Skipping ncRNA search, enable with --rfam if desired.\\n[10:12:11] Total of 106 tRNA + rRNA features\\n[10:12:11] Searching for CRISPR repeats\\n[10:12:12] CRISPR1 LAHG01000004.1 88922 with 15 spacers\\n[10:12:12] CRISPR2 LAHG01000014.1 41393 with 12 spacers\\n[10:12:12] CRISPR3 LAHG01000015.1 42289 with 14 spacers\\n[10:12:12] CRISPR4 LAHG01000022.1 50647 with 7 spacers\\n[10:12:12] CRISPR5 LAHG01000027.1 16433 with 20 spacers\\n[10:12:12] CRISPR6 LAHG01000053.1 10177 with 24 spacers\\n[10:12:12] CRISPR7 LAHG01000077.1 21304 with 10 spacers\\n[10:12:12] CRISPR8 LAHG01000135.1 4863 with 43 spacers\\n[10:12:12] CRISPR9 LAHG01000135.1 19440 with 27 spacers\\n[10:12:12] CRISPR10 LAHG01000177.1 2719 with 7 spacers\\n[10:12:12] CRISPR11 LAHG01000200.1 2792 with 18 spacers\\n[10:12:12] CRISPR12 LAHG01000235.1 4995 with 6 spacers\\n[10:12:12] CRISPR13 LAHG01000275.1 86 with 22 spacers\\n[10:12:12] Found 13 CRISPRs\\n[10:12:12] Predicting coding sequences\\n[10:12:12] Contigs total 8997043 bp, so using single mode\\n[10:12:12] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHG01\\\\/\\\\/Nostoc_LAHG01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:12:20] Excluding CDS which overlaps existing RNA (repeat_region) at LAHG01000015.1:42460..42576 on - strand\\n[10:12:22] Excluding CDS which overlaps existing RNA (repeat_region) at LAHG01000077.1:21513..21665 on + strand\\n[10:12:22] Excluding CDS which overlaps existing RNA (tRNA) at LAHG01000128.1:5994..6137 on + strand\\n[10:12:22] Excluding CDS which overlaps existing RNA (repeat_region) at LAHG01000135.1:7801..7965 on + strand\\n[10:12:23] Excluding CDS which overlaps existing RNA (tRNA) at LAHG01000159.1:3261..4631 on - strand\\n[10:12:23] Excluding CDS which overlaps existing RNA (repeat_region) at LAHG01000177.1:3039..3164 on + strand\\n[10:12:23] Excluding CDS which overlaps existing RNA (repeat_region) at LAHG01000235.1:5387..6346 on - strand\\n[10:12:23] Excluding CDS which overlaps existing RNA (rRNA) at LAHG01000264.1:689..1393 on + strand\\n[10:12:23] Excluding CDS which overlaps existing RNA (rRNA) at LAHG01000264.1:4146..4400 on - strand\\n[10:12:23] Excluding CDS which overlaps existing RNA (repeat_region) at LAHG01000275.1:1540..1818 on - strand\\n[10:12:23] Excluding CDS which overlaps existing RNA (rRNA) at LAHG01000306.1:1436..1912 on - strand\\n[10:12:23] Found 7512 CDS\\n[10:12:23] Connecting features back to sequences\\n[10:12:23] Not using genus-specific database. Try --usegenus to enable it.\\n[10:12:23] Annotating CDS, please be patient.\\n[10:12:23] Will use 4 CPUs for similarity searching.\\n[10:12:25] There are still 7512 unannotated CDS left (started with 7512)\\n[10:12:25] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:12:25] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHG01\\\\/\\\\/Nostoc_LAHG01\\\\.IS\\\\.tmp\\\\.98433\\\\.faa | parallel --gnu --plain -j 4 --block 302207 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHG01\\\\/\\\\/Nostoc_LAHG01\\\\.IS\\\\.tmp\\\\.98433\\\\.blast 2> /dev/null\\n[10:12:47] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHG01\\\\/\\\\/Nostoc_LAHG01\\\\.IS\\\\.tmp\\\\.98433\\\\.faa | parallel --gnu --plain -j 4 --block 302207 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHG01\\\\/\\\\/Nostoc_LAHG01\\\\.IS\\\\.tmp\\\\.98433\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:12:48] This is prokka 1.14.6\\n[10:12:48] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:12:48] Homepage is https://github.com/tseemann/prokka\\n[10:12:48] Local time is Mon Feb 21 10:12:48 2022\\n[10:12:48] You are tiagoferreiraleao\\n[10:12:48] Operating system is darwin\\n[10:12:48] You have BioPerl 1.007002\\n[10:12:48] System has 4 cores.\\n[10:12:48] Option --cpu asked for 12 cores, but system only has 4\\n[10:12:48] Will use maximum of 4 cores.\\n[10:12:48] Annotating as >>> Bacteria <<<\\n[10:12:48] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JAHHHW01.fasta' contents.\\n[10:12:48] Setting --locustag KALHFLNI from MD5 4a51f572ac1a11407b79324138125db3\\n[10:12:48] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JAHHHW01/\\n[10:12:48] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAHHHW01\\\\/\\n[10:12:48] Using filename prefix: Nostoc_JAHHHW01.XXX\\n[10:12:48] Setting HMMER_NCPU=1\\n[10:12:48] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JAHHHW01//Nostoc_JAHHHW01.log\\n[10:12:48] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JAHHHW01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JAHHHW01/ --prefix Nostoc_JAHHHW01 --cpus 12\\n[10:12:48] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:12:48] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:12:48] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:12:48] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:12:48] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:12:48] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:12:48] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:12:48] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:12:48] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:12:48] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:12:48] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:12:48] Looking for 'egrep' - found /usr/bin/egrep\\n[10:12:48] Looking for 'find' - found /usr/bin/find\\n[10:12:48] Looking for 'grep' - found /usr/bin/grep\\n[10:12:48] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:12:48] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:12:48] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:12:48] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:12:48] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:12:48] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:12:48] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:12:48] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:12:48] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:12:48] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:12:49] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:12:49] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:12:49] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:12:49] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:12:49] Looking for 'sed' - found /usr/bin/sed\\n[10:12:49] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:12:49] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:12:49] Using genetic code table 11.\\n[10:12:49] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JAHHHW01.fasta\\n[10:12:49] Wrote 167 contigs totalling 6704691 bp.\\n[10:12:49] Predicting tRNAs and tmRNAs\\n[10:12:49] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAHHHW01\\\\/\\\\/Nostoc_JAHHHW01\\\\.fna\\n[10:12:51] 1 tRNA-Arg [20113,20186] 35 (tcg)\\n[10:12:51] 1 tRNA-Ser c[25829,25915] 36 (cga)\\n[10:12:51] 1 tRNA-His c[12652,12745] 35 (atg)\\n[10:12:51] 1 tRNA-Gly c[3393,3464] 33 (gcc)\\n[10:12:51] 1 tRNA-Met [10164,10238] 35 (cat)\\n[10:12:51] 1 tRNA-Gly [17309,17380] 33 (ccc)\\n[10:12:51] 1 tRNA-Val c[13174,13246] 34 (tac)\\n[10:12:51] 1 tRNA-Lys [10468,10541] 34 (ctt)\\n[10:12:51] 1 tRNA-Asp [54396,54469] 35 (gtc)\\n[10:12:51] 1 tRNA-Asn [9670,9743] 34 (gtt)\\n[10:12:51] 1 tRNA-Phe c[3231,3304] 34 (gaa)\\n[10:12:51] 1 tRNA-Leu c[3874,3957] 36 (tag)\\n[10:12:51] 1 tRNA-Met [63244,63316] 34 (cat)\\n[10:12:51] 1 tRNA-Leu [6697,6780] 35 (caa)\\n[10:12:51] 2 tRNA-Gln [40602,40673] 33 (ttg)\\n[10:12:51] 1 tRNA-Glu c[50069,50141] 35 (ttc)\\n[10:12:51] 1 tRNA-Ser [70818,70906] 36 (gga)\\n[10:12:51] 2 tRNA-Arg c[104675,104749] 35 (tct)\\n[10:12:51] 1 tRNA-Trp [23406,23479] 34 (cca)\\n[10:12:51] 1 tRNA-Thr c[235,306] 33 (cgt)\\n[10:12:51] 1 tRNA-Arg c[138594,138670] 36 (cct)\\n[10:12:51] 1 tRNA-Leu [4830,4912] 35 (cag)\\n[10:12:51] 1 tRNA-Asn [858,930] 34 (att)\\n[10:12:51] 1 tmRNA [20622,21009] 101,154 ANNIVKFARKEALVAA**\\n[10:12:51] 1 tRNA-Thr c[58475,58546] 33 (ggt)\\n[10:12:51] 2 tRNA-Tyr c[58624,58708] 35 (gta)\\n[10:12:51] 1 tRNA-Ala [1292,1364] 34 (ggc)\\n[10:12:51] 2 tRNA-Arg [7362,7435] 35 (acg)\\n[10:12:51] 3 tRNA-Ser c[17560,17653] 36 (gct)\\n[10:12:51] 1 tRNA-Pro [40641,40714] 35 (cgg)\\n[10:12:51] 1 tRNA-Arg c[42762,42834] 34 (ccg)\\n[10:12:51] 1 tRNA-Val c[4035,4108] 35 (gac)\\n[10:12:51] 2 tRNA-Met [7100,7176] 35 (cat)\\n[10:12:51] 3 tRNA-Pro c[41012,41085] 35 (ggg)\\n[10:12:51] 1 tRNA-Pro c[13556,13629] 35 (tgg)\\n[10:12:51] 1 tRNA-Trp [60143,60217] 35 (cca)\\n[10:12:51] 2 tRNA-Trp [62449,62523] 35 (cca)\\n[10:12:51] 1 tRNA-His [36098,36171] 34 (gtg)\\n[10:12:51] 1 tRNA-Leu c[20724,20806] 36 (gag)\\n[10:12:51] 1 tRNA-Ala [30680,30754] 35 (cgc)\\n[10:12:51] 1 tRNA-Thr [12265,12336] 33 (tgt)\\n[10:12:51] 2 tRNA-Gly [20603,20673] 33 (tcc)\\n[10:12:51] 1 tRNA-Ser [52183,52268] 35 (tga)\\n[10:12:51] 1 tRNA-Ser c[25335,25427] 34 (tga)\\n[10:12:51] 2 tRNA-Ser c[25461,25553] 34 (tga)\\n[10:12:51] 3 tRNA-Ser c[25545,25637] 34 (tga)\\n[10:12:51] Found 46 tRNAs\\n[10:12:51] Predicting Ribosomal RNAs\\n[10:12:51] Running Barrnap with 4 threads\\n[10:12:52] Found 0 rRNAs\\n[10:12:52] Skipping ncRNA search, enable with --rfam if desired.\\n[10:12:52] Total of 45 tRNA + rRNA features\\n[10:12:52] Searching for CRISPR repeats\\n[10:12:53] CRISPR1 JAHHHW010000021.1 9542 with 20 spacers\\n[10:12:53] CRISPR2 JAHHHW010000093.1 56336 with 7 spacers\\n[10:12:53] CRISPR3 JAHHHW010000095.1 54728 with 3 spacers\\n[10:12:53] CRISPR4 JAHHHW010000167.1 41192 with 19 spacers\\n[10:12:53] Found 4 CRISPRs\\n[10:12:53] Predicting coding sequences\\n[10:12:53] Contigs total 6704691 bp, so using single mode\\n[10:12:53] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAHHHW01\\\\/\\\\/Nostoc_JAHHHW01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:12:58] Excluding CDS which overlaps existing RNA (tRNA) at JAHHHW010000017.1:12149..13111 on - strand\\n[10:13:00] Excluding CDS which overlaps existing RNA (repeat_region) at JAHHHW010000093.1:56190..56369 on + strand\\n[10:13:01] Excluding CDS which overlaps existing RNA (tRNA) at JAHHHW010000158.1:25163..25651 on + strand\\n[10:13:01] Found 5628 CDS\\n[10:13:01] Connecting features back to sequences\\n[10:13:01] Not using genus-specific database. Try --usegenus to enable it.\\n[10:13:01] Annotating CDS, please be patient.\\n[10:13:01] Will use 4 CPUs for similarity searching.\\n[10:13:03] There are still 5628 unannotated CDS left (started with 5628)\\n[10:13:03] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:13:03] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAHHHW01\\\\/\\\\/Nostoc_JAHHHW01\\\\.IS\\\\.tmp\\\\.98545\\\\.faa | parallel --gnu --plain -j 4 --block 229282 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAHHHW01\\\\/\\\\/Nostoc_JAHHHW01\\\\.IS\\\\.tmp\\\\.98545\\\\.blast 2> /dev/null\\n[10:13:15] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAHHHW01\\\\/\\\\/Nostoc_JAHHHW01\\\\.IS\\\\.tmp\\\\.98545\\\\.faa | parallel --gnu --plain -j 4 --block 229282 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAHHHW01\\\\/\\\\/Nostoc_JAHHHW01\\\\.IS\\\\.tmp\\\\.98545\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:13:16] This is prokka 1.14.6\\n[10:13:16] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:13:16] Homepage is https://github.com/tseemann/prokka\\n[10:13:16] Local time is Mon Feb 21 10:13:16 2022\\n[10:13:16] You are tiagoferreiraleao\\n[10:13:16] Operating system is darwin\\n[10:13:16] You have BioPerl 1.007002\\n[10:13:16] System has 4 cores.\\n[10:13:16] Option --cpu asked for 12 cores, but system only has 4\\n[10:13:16] Will use maximum of 4 cores.\\n[10:13:16] Annotating as >>> Bacteria <<<\\n[10:13:16] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZZ01.fasta' contents.\\n[10:13:16] Setting --locustag COEKAEEN from MD5 c8e4aee761e4f18db00001369b70f4a3\\n[10:13:16] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZZ01/\\n[10:13:16] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZZ01\\\\/\\n[10:13:16] Using filename prefix: Fischerella_NGZZ01.XXX\\n[10:13:16] Setting HMMER_NCPU=1\\n[10:13:16] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZZ01//Fischerella_NGZZ01.log\\n[10:13:16] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZZ01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZZ01/ --prefix Fischerella_NGZZ01 --cpus 12\\n[10:13:16] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:13:16] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:13:16] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:13:16] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:13:16] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:13:16] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:13:16] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:13:16] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:13:16] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:13:16] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:13:16] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:13:16] Looking for 'egrep' - found /usr/bin/egrep\\n[10:13:16] Looking for 'find' - found /usr/bin/find\\n[10:13:16] Looking for 'grep' - found /usr/bin/grep\\n[10:13:16] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:13:16] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:13:16] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:13:16] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:13:16] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:13:16] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:13:16] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:13:16] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:13:17] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:13:17] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:13:17] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:13:17] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:13:17] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:13:17] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:13:17] Looking for 'sed' - found /usr/bin/sed\\n[10:13:17] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:13:17] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:13:17] Using genetic code table 11.\\n[10:13:17] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZZ01.fasta\\n[10:13:18] Wrote 263 contigs totalling 5493527 bp.\\n[10:13:18] Predicting tRNAs and tmRNAs\\n[10:13:18] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZZ01\\\\/\\\\/Fischerella_NGZZ01\\\\.fna\\n[10:13:20] 1 tRNA-Thr c[10367,10438] 33 (ggt)\\n[10:13:20] 1 tRNA-Arg [43703,43776] 35 (ccg)\\n[10:13:20] 1 tRNA-Asn c[53246,53319] 34 (gtt)\\n[10:13:20] 2 tRNA-Gly [71369,71440] 33 (gcc)\\n[10:13:20] 1 tRNA-Pro [21014,21088] 35 (tgg)\\n[10:13:20] 1 tRNA-Asn c[3796,3869] 34 (att)\\n[10:13:20] 2 tRNA-Arg c[4540,4612] 34 (tcg)\\n[10:13:20] 1 tRNA-Trp c[7161,7234] 34 (cca)\\n[10:13:20] 1 tRNA-Met c[27614,27689] 36 (cat)\\n[10:13:20] 2 tRNA-Arg c[53315,53389] 35 (tct)\\n[10:13:20] 1 tRNA-Cys c[29,100] 33 (gca)\\n[10:13:20] 1 tRNA-Met c[8663,8739] 35 (cat)\\n[10:13:20] 2 tRNA-Val [11967,12040] 35 (gac)\\n[10:13:20] 1 tmRNA c[179,565] 100,150 ANNIVKFARKEALVAA*\\n[10:13:20] 1 tRNA-Leu [7055,7138] 36 (tag)\\n[10:13:20] 1 tRNA-Met [8658,8730] 34 (cat)\\n[10:13:20] 1 tRNA-Arg c[6618,6691] 35 (acg)\\n[10:13:20] 2 tRNA-Ala c[11583,11656] 34 (ggc)\\n[10:13:20] 1 tRNA-Phe c[32967,33039] 34 (gaa)\\n[10:13:20] 2 tRNA-Val [40380,40452] 34 (tac)\\n[10:13:20] 1 tRNA-Gln c[7079,7149] 32 (ttg)\\n[10:13:20] 1 tRNA-Leu [36732,36814] 36 (gag)\\n[10:13:20] 1 tRNA-Pro c[77924,77997] 35 (ggg)\\n[10:13:20] 1 tRNA-Gly [14746,14817] 33 (ccc)\\n[10:13:20] 1 tRNA-Pro c[26058,26131] 35 (cgg)\\n[10:13:20] 1 tRNA-Asp c[48593,48666] 35 (gtc)\\n[10:13:20] 1 tRNA-Gly [12397,12467] 33 (tcc)\\n[10:13:20] 1 tRNA-Ala [19472,19546] 35 (cgc)\\n[10:13:20] 1 tRNA-His c[29088,29161] 34 (gtg)\\n[10:13:20] 1 tRNA-Ser [5124,5210] 36 (cga)\\n[10:13:20] 1 tRNA-Thr c[8367,8438] 33 (tgt)\\n[10:13:20] 1 tRNA-Glu c[12516,12588] 35 (ttc)\\n[10:13:20] 2 tRNA-Ser c[25466,25552] 36 (tga)\\n[10:13:20] 1 tRNA-Arg [2588,2663] 36 (cct)\\n[10:13:20] 1 tRNA-Ser [23957,24050] 36 (gct)\\n[10:13:20] 1 tRNA-Lys [36265,36338] 34 (ttt)\\n[10:13:20] 1 tRNA-Thr [14160,14231] 33 (cgt)\\n[10:13:20] 1 tRNA-Ser [21390,21478] 36 (gga)\\n[10:13:20] 1 tRNA-Gln [17597,17668] 33 (ttg)\\n[10:13:20] Found 39 tRNAs\\n[10:13:20] Predicting Ribosomal RNAs\\n[10:13:20] Running Barrnap with 4 threads\\n[10:13:20] Found 0 rRNAs\\n[10:13:20] Skipping ncRNA search, enable with --rfam if desired.\\n[10:13:20] Total of 38 tRNA + rRNA features\\n[10:13:20] Searching for CRISPR repeats\\n[10:13:21] CRISPR1 NGZZ01000062.1 216 with 3 spacers\\n[10:13:21] Found 1 CRISPRs\\n[10:13:21] Predicting coding sequences\\n[10:13:21] Contigs total 5493527 bp, so using single mode\\n[10:13:21] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZZ01\\\\/\\\\/Fischerella_NGZZ01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:13:26] Excluding CDS which overlaps existing RNA (tRNA) at NGZZ01000064.1:6575..7246 on + strand\\n[10:13:26] Excluding CDS which overlaps existing RNA (tRNA) at NGZZ01000065.1:40299..40460 on + strand\\n[10:13:26] Excluding CDS which overlaps existing RNA (tRNA) at NGZZ01000115.1:48628..49023 on + strand\\n[10:13:27] Excluding CDS which overlaps existing RNA (tRNA) at NGZZ01000199.1:29097..29276 on + strand\\n[10:13:27] Found 4674 CDS\\n[10:13:27] Connecting features back to sequences\\n[10:13:27] Not using genus-specific database. Try --usegenus to enable it.\\n[10:13:27] Annotating CDS, please be patient.\\n[10:13:27] Will use 4 CPUs for similarity searching.\\n[10:13:29] There are still 4674 unannotated CDS left (started with 4674)\\n[10:13:29] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:13:29] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZZ01\\\\/\\\\/Fischerella_NGZZ01\\\\.IS\\\\.tmp\\\\.98657\\\\.faa | parallel --gnu --plain -j 4 --block 158415 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZZ01\\\\/\\\\/Fischerella_NGZZ01\\\\.IS\\\\.tmp\\\\.98657\\\\.blast 2> /dev/null\\n[10:13:42] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZZ01\\\\/\\\\/Fischerella_NGZZ01\\\\.IS\\\\.tmp\\\\.98657\\\\.faa | parallel --gnu --plain -j 4 --block 158415 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZZ01\\\\/\\\\/Fischerella_NGZZ01\\\\.IS\\\\.tmp\\\\.98657\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:13:42] This is prokka 1.14.6\\n[10:13:42] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:13:42] Homepage is https://github.com/tseemann/prokka\\n[10:13:42] Local time is Mon Feb 21 10:13:42 2022\\n[10:13:42] You are tiagoferreiraleao\\n[10:13:42] Operating system is darwin\\n[10:13:42] You have BioPerl 1.007002\\n[10:13:42] System has 4 cores.\\n[10:13:42] Option --cpu asked for 12 cores, but system only has 4\\n[10:13:42] Will use maximum of 4 cores.\\n[10:13:42] Annotating as >>> Bacteria <<<\\n[10:13:42] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAD01.fasta' contents.\\n[10:13:42] Setting --locustag BMBDAHDL from MD5 b6bda1d50346e9ac3bdb356bec613c6b\\n[10:13:42] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAD01/\\n[10:13:42] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAD01\\\\/\\n[10:13:42] Using filename prefix: Fischerella_NHAD01.XXX\\n[10:13:42] Setting HMMER_NCPU=1\\n[10:13:42] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAD01//Fischerella_NHAD01.log\\n[10:13:42] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAD01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAD01/ --prefix Fischerella_NHAD01 --cpus 12\\n[10:13:42] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:13:42] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:13:42] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:13:42] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:13:42] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:13:42] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:13:43] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:13:43] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:13:43] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:13:43] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:13:43] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:13:43] Looking for 'egrep' - found /usr/bin/egrep\\n[10:13:43] Looking for 'find' - found /usr/bin/find\\n[10:13:43] Looking for 'grep' - found /usr/bin/grep\\n[10:13:43] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:13:43] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:13:43] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:13:43] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:13:43] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:13:43] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:13:43] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:13:43] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:13:43] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:13:43] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:13:43] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:13:43] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:13:43] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:13:43] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:13:43] Looking for 'sed' - found /usr/bin/sed\\n[10:13:43] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:13:44] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:13:44] Using genetic code table 11.\\n[10:13:44] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAD01.fasta\\n[10:13:44] Wrote 101 contigs totalling 5506947 bp.\\n[10:13:44] Predicting tRNAs and tmRNAs\\n[10:13:44] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAD01\\\\/\\\\/Fischerella_NHAD01\\\\.fna\\n[10:13:46] 1 tRNA-Arg c[7800,7873] 35 (acg)\\n[10:13:46] 2 tRNA-Ala c[12765,12838] 34 (ggc)\\n[10:13:46] 1 tRNA-Gly [46243,46313] 33 (tcc)\\n[10:13:46] 2 tRNA-Cys [97529,97600] 33 (gca)\\n[10:13:46] 1 tRNA-Ser [4838,4924] 36 (cga)\\n[10:13:46] 1 tRNA-Leu [49231,49314] 36 (tag)\\n[10:13:46] 1 tRNA-Thr [2274,2345] 33 (tgt)\\n[10:13:46] 1 tRNA-Phe c[946,1018] 34 (gaa)\\n[10:13:46] 2 tRNA-Val [8417,8489] 34 (tac)\\n[10:13:46] 1 tRNA-Val c[68381,68454] 35 (gac)\\n[10:13:46] 2 tRNA-Met [71400,71476] 35 (cat)\\n[10:13:46] 1 tRNA-Arg [30796,30871] 36 (cct)\\n[10:13:46] 1 tRNA-Asn c[22016,22089] 34 (gtt)\\n[10:13:46] 1 tRNA-Arg [43225,43298] 35 (ccg)\\n[10:13:46] 1 tRNA-Gln c[30003,30074] 33 (ttg)\\n[10:13:46] 1 tRNA-Asp c[60975,61048] 35 (gtc)\\n[10:13:46] 2 tRNA-Gly [91362,91433] 33 (gcc)\\n[10:13:46] 1 tRNA-Arg [11817,11891] 35 (tct)\\n[10:13:46] 1 tRNA-Pro [18303,18376] 35 (cgg)\\n[10:13:46] 1 tRNA-Gln c[59218,59288] 32 (ttg)\\n[10:13:46] 1 tRNA-Asn c[41847,41920] 34 (att)\\n[10:13:46] 2 tRNA-Arg c[42591,42663] 34 (tcg)\\n[10:13:46] 3 tRNA-Glu c[61317,61389] 35 (ttc)\\n[10:13:46] 1 tRNA-Ser [8755,8848] 36 (gct)\\n[10:13:46] 1 tRNA-Ala c[46384,46458] 34 (tgc)\\n[10:13:46] 2 tRNA-Ile c[46541,46616] 35 (gat)\\n[10:13:46] 3 tRNA-Met [57427,57499] 34 (cat)\\n[10:13:46] 1 tRNA-Ser c[62858,62944] 36 (tga)\\n[10:13:46] 1 tRNA-Leu [35162,35244] 36 (gag)\\n[10:13:46] 1 tRNA-Leu [71718,71802] 35 (caa)\\n[10:13:46] 1 tRNA-Leu c[51247,51329] 35 (cag)\\n[10:13:46] 1 tRNA-Ser c[4478,4566] 36 (gga)\\n[10:13:46] 1 tRNA-Pro [19687,19761] 35 (tgg)\\n[10:13:46] 1 tmRNA [67417,67803] 100,150 ANNIVKFARKEALVAA*\\n[10:13:46] 1 tRNA-Pro [56497,56565] 30 (ggg)\\n[10:13:46] 2 tRNA-Thr [178977,179048] 33 (cgt)\\n[10:13:46] 1 tRNA-Trp [16500,16573] 34 (cca)\\n[10:13:46] 1 tRNA-Tyr [14269,14353] 35 (gta)\\n[10:13:46] 2 tRNA-Thr [14417,14488] 33 (ggt)\\n[10:13:46] 3 tRNA-Gly c[32888,32959] 33 (ccc)\\n[10:13:46] 1 tRNA-Ala c[47810,47884] 35 (cgc)\\n[10:13:46] 1 tRNA-Lys [199025,199098] 34 (ttt)\\n[10:13:46] 2 tRNA-His [477582,477655] 34 (gtg)\\n[10:13:46] Found 43 tRNAs\\n[10:13:46] Predicting Ribosomal RNAs\\n[10:13:46] Running Barrnap with 4 threads\\n[10:13:48] 1 NHAD01000068.1 43036 5S ribosomal RNA\\n[10:13:48] 2 NHAD01000068.1 43265 23S ribosomal RNA\\n[10:13:48] 3 NHAD01000068.1 46855 16S ribosomal RNA\\n[10:13:48] 4 NHAD01000098.1 1 16S ribosomal RNA\\n[10:13:48] 5 NHAD01000098.1 3171 23S ribosomal RNA (partial)\\n[10:13:48] 6 NHAD01000098.1 4911 23S ribosomal RNA (partial)\\n[10:13:48] Found 6 rRNAs\\n[10:13:48] Skipping ncRNA search, enable with --rfam if desired.\\n[10:13:48] Total of 48 tRNA + rRNA features\\n[10:13:48] Searching for CRISPR repeats\\n[10:13:48] CRISPR1 NHAD01000021.1 48808 with 3 spacers\\n[10:13:48] CRISPR2 NHAD01000051.1 39659 with 56 spacers\\n[10:13:48] CRISPR3 NHAD01000065.1 20176 with 16 spacers\\n[10:13:48] CRISPR4 NHAD01000065.1 107029 with 16 spacers\\n[10:13:48] CRISPR5 NHAD01000065.1 108259 with 63 spacers\\n[10:13:48] CRISPR6 NHAD01000065.1 112998 with 18 spacers\\n[10:13:48] CRISPR7 NHAD01000065.1 114467 with 41 spacers\\n[10:13:48] CRISPR8 NHAD01000082.1 84897 with 71 spacers\\n[10:13:48] CRISPR9 NHAD01000101.1 148158 with 95 spacers\\n[10:13:48] Found 9 CRISPRs\\n[10:13:48] Predicting coding sequences\\n[10:13:48] Contigs total 5506947 bp, so using single mode\\n[10:13:48] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAD01\\\\/\\\\/Fischerella_NHAD01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:13:52] Excluding CDS which overlaps existing RNA (tRNA) at NHAD01000007.1:7757..8428 on + strand\\n[10:13:53] Excluding CDS which overlaps existing RNA (tRNA) at NHAD01000028.1:8336..8497 on + strand\\n[10:13:53] Excluding CDS which overlaps existing RNA (tRNA) at NHAD01000045.1:20776..22215 on - strand\\n[10:13:54] Excluding CDS which overlaps existing RNA (rRNA) at NHAD01000068.1:43550..43726 on + strand\\n[10:13:54] Excluding CDS which overlaps existing RNA (rRNA) at NHAD01000068.1:45512..45691 on - strand\\n[10:13:54] Excluding CDS which overlaps existing RNA (rRNA) at NHAD01000068.1:46897..47211 on - strand\\n[10:13:55] Excluding CDS which overlaps existing RNA (rRNA) at NHAD01000098.1:540..1034 on - strand\\n[10:13:55] Excluding CDS which overlaps existing RNA (rRNA) at NHAD01000098.1:3569..3775 on + strand\\n[10:13:55] Excluding CDS which overlaps existing RNA (repeat_region) at NHAD01000101.1:154317..154493 on - strand\\n[10:13:55] Found 4677 CDS\\n[10:13:55] Connecting features back to sequences\\n[10:13:55] Not using genus-specific database. Try --usegenus to enable it.\\n[10:13:55] Annotating CDS, please be patient.\\n[10:13:55] Will use 4 CPUs for similarity searching.\\n[10:13:56] There are still 4677 unannotated CDS left (started with 4677)\\n[10:13:56] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:13:56] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAD01\\\\/\\\\/Fischerella_NHAD01\\\\.IS\\\\.tmp\\\\.98770\\\\.faa | parallel --gnu --plain -j 4 --block 167364 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAD01\\\\/\\\\/Fischerella_NHAD01\\\\.IS\\\\.tmp\\\\.98770\\\\.blast 2> /dev/null\\n[10:14:09] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAD01\\\\/\\\\/Fischerella_NHAD01\\\\.IS\\\\.tmp\\\\.98770\\\\.faa | parallel --gnu --plain -j 4 --block 167364 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAD01\\\\/\\\\/Fischerella_NHAD01\\\\.IS\\\\.tmp\\\\.98770\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:14:09] This is prokka 1.14.6\\n[10:14:09] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:14:09] Homepage is https://github.com/tseemann/prokka\\n[10:14:09] Local time is Mon Feb 21 10:14:09 2022\\n[10:14:09] You are tiagoferreiraleao\\n[10:14:09] Operating system is darwin\\n[10:14:09] You have BioPerl 1.007002\\n[10:14:09] System has 4 cores.\\n[10:14:09] Option --cpu asked for 12 cores, but system only has 4\\n[10:14:09] Will use maximum of 4 cores.\\n[10:14:09] Annotating as >>> Bacteria <<<\\n[10:14:09] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_ALVS01.fasta' contents.\\n[10:14:09] Setting --locustag GMEDPCCD from MD5 06ed9ccd0e72945da4b7907c6596ddff\\n[10:14:09] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_ALVS01/\\n[10:14:09] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVS01\\\\/\\n[10:14:09] Using filename prefix: Fischerella_ALVS01.XXX\\n[10:14:09] Setting HMMER_NCPU=1\\n[10:14:09] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_ALVS01//Fischerella_ALVS01.log\\n[10:14:09] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_ALVS01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_ALVS01/ --prefix Fischerella_ALVS01 --cpus 12\\n[10:14:09] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:14:09] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:14:09] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:14:09] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:14:09] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:14:09] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:14:10] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:14:10] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:14:10] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:14:10] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:14:10] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:14:10] Looking for 'egrep' - found /usr/bin/egrep\\n[10:14:10] Looking for 'find' - found /usr/bin/find\\n[10:14:10] Looking for 'grep' - found /usr/bin/grep\\n[10:14:10] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:14:10] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:14:10] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:14:10] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:14:10] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:14:10] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:14:10] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:14:10] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:14:10] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:14:10] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:14:10] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:14:10] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:14:10] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:14:10] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:14:10] Looking for 'sed' - found /usr/bin/sed\\n[10:14:10] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:14:11] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:14:11] Using genetic code table 11.\\n[10:14:11] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_ALVS01.fasta\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733471|gb|ALVS01000001.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733470|gb|ALVS01000002.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733469|gb|ALVS01000003.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733468|gb|ALVS01000004.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733467|gb|ALVS01000005.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733466|gb|ALVS01000006.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733465|gb|ALVS01000007.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733464|gb|ALVS01000008.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733463|gb|ALVS01000009.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733462|gb|ALVS01000010.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733461|gb|ALVS01000011.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733460|gb|ALVS01000012.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733459|gb|ALVS01000013.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733458|gb|ALVS01000014.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733457|gb|ALVS01000015.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733456|gb|ALVS01000016.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733455|gb|ALVS01000017.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733454|gb|ALVS01000018.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733453|gb|ALVS01000019.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733452|gb|ALVS01000020.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733451|gb|ALVS01000021.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733450|gb|ALVS01000022.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733449|gb|ALVS01000023.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733448|gb|ALVS01000024.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733447|gb|ALVS01000025.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733446|gb|ALVS01000026.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733445|gb|ALVS01000027.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733444|gb|ALVS01000028.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733443|gb|ALVS01000029.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733442|gb|ALVS01000030.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733441|gb|ALVS01000031.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733440|gb|ALVS01000032.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733439|gb|ALVS01000033.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733438|gb|ALVS01000034.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733437|gb|ALVS01000035.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733436|gb|ALVS01000036.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733435|gb|ALVS01000037.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733434|gb|ALVS01000038.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733433|gb|ALVS01000039.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733432|gb|ALVS01000040.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733431|gb|ALVS01000041.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733430|gb|ALVS01000042.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733429|gb|ALVS01000043.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733428|gb|ALVS01000044.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733427|gb|ALVS01000045.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733426|gb|ALVS01000046.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733425|gb|ALVS01000047.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733424|gb|ALVS01000048.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733423|gb|ALVS01000049.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733422|gb|ALVS01000050.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733421|gb|ALVS01000051.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733420|gb|ALVS01000052.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733419|gb|ALVS01000053.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733418|gb|ALVS01000054.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733417|gb|ALVS01000055.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733416|gb|ALVS01000056.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733415|gb|ALVS01000057.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733414|gb|ALVS01000058.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733413|gb|ALVS01000059.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733412|gb|ALVS01000060.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733411|gb|ALVS01000061.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733410|gb|ALVS01000062.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733409|gb|ALVS01000063.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733408|gb|ALVS01000064.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733407|gb|ALVS01000065.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733406|gb|ALVS01000066.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733405|gb|ALVS01000067.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733404|gb|ALVS01000068.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733403|gb|ALVS01000069.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733402|gb|ALVS01000070.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733401|gb|ALVS01000071.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733400|gb|ALVS01000072.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733399|gb|ALVS01000073.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733398|gb|ALVS01000074.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733397|gb|ALVS01000075.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733396|gb|ALVS01000076.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733395|gb|ALVS01000077.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733394|gb|ALVS01000078.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733393|gb|ALVS01000079.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733392|gb|ALVS01000080.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733391|gb|ALVS01000081.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733390|gb|ALVS01000082.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733389|gb|ALVS01000083.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733388|gb|ALVS01000084.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733387|gb|ALVS01000085.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733386|gb|ALVS01000086.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733385|gb|ALVS01000087.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733384|gb|ALVS01000088.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733383|gb|ALVS01000089.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733382|gb|ALVS01000090.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733381|gb|ALVS01000091.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733380|gb|ALVS01000092.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733379|gb|ALVS01000093.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733378|gb|ALVS01000094.1|\\n[10:14:11] Changing illegal '|' to '_' in sequence name: gi|425733377|gb|ALVS01000095.1|\\n[10:14:11] Wrote 95 contigs totalling 8000057 bp.\\n[10:14:11] Predicting tRNAs and tmRNAs\\n[10:14:11] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVS01\\\\/\\\\/Fischerella_ALVS01\\\\.fna\\n[10:14:14] 1 tRNA-Trp c[80433,80507] 35 (cca)\\n[10:14:14] 1 tRNA-Thr c[100024,100095] 33 (ggt)\\n[10:14:14] 2 tRNA-Tyr c[100173,100257] 35 (gta)\\n[10:14:14] 1 tRNA-Gln c[211986,212057] 33 (ttg)\\n[10:14:14] 2 tRNA-Leu c[246921,247005] 35 (caa)\\n[10:14:14] 3 tRNA-Ala c[313897,313969] 34 (cgc)\\n[10:14:14] 1 tRNA-Phe [126319,126392] 35 (gaa)\\n[10:14:14] 2 tRNA-Asn [126763,126836] 34 (att)\\n[10:14:14] 1 tRNA-Ser [116281,116367] 36 (cga)\\n[10:14:14] 2 tRNA-Pro c[284442,284515] 35 (cgg)\\n[10:14:14] 1 tRNA-Asp c[91874,91947] 35 (gtc)\\n[10:14:14] 2 tRNA-Ser c[94977,95065] 36 (gga)\\n[10:14:14] 1 tRNA-Pro [105591,105664] 35 (tgg)\\n[10:14:14] 2 tRNA-Phe [137022,137094] 34 (gaa)\\n[10:14:14] 1 tRNA-Ser c[115019,115104] 35 (tga)\\n[10:14:14] 2 tRNA-His [494748,494821] 34 (gtg)\\n[10:14:14] 3 tRNA-Arg c[630687,630763] 36 (cct)\\n[10:14:14] 4 tRNA-Met [714100,714175] 36 (cat)\\n[10:14:14] 5 tRNA-Asn c[827477,827550] 34 (gtt)\\n[10:14:14] 1 tRNA-Val c[99091,99163] 34 (tac)\\n[10:14:14] 2 tRNA-Val c[207614,207687] 35 (gac)\\n[10:14:14] 3 tRNA-Met [210708,210784] 35 (cat)\\n[10:14:14] 4 tmRNA [269170,269556] 100,153 ANNIVKFARKEALVAA**\\n[10:14:14] 5 tRNA-Gly c[290926,290997] 33 (ccc)\\n[10:14:14] 1 tRNA-Lys [112789,112861] 33 (ctt)\\n[10:14:14] 2 tRNA-Arg [236084,236157] 35 (acg)\\n[10:14:14] 3 tRNA-Ser c[246486,246579] 36 (gct)\\n[10:14:14] 4 tRNA-Cys c[304214,304285] 33 (gca)\\n[10:14:14] 5 tRNA-Ile [311821,311896] 35 (gat)\\n[10:14:14] 6 tRNA-Ala [311983,312057] 34 (tgc)\\n[10:14:14] 1 tRNA-Arg [108143,108217] 35 (tct)\\n[10:14:14] 2 tRNA-Ala c[146795,146869] 34 (tgc)\\n[10:14:14] 3 tRNA-Ile c[146956,147031] 35 (gat)\\n[10:14:14] 4 tRNA-Met [157586,157659] 34 (cat)\\n[10:14:14] 5 tRNA-Pro [403786,403859] 35 (ggg)\\n[10:14:14] 1 tRNA-Thr [64322,64393] 33 (tgt)\\n[10:14:14] 1 tRNA-Ala [10996,11070] 35 (ggc)\\n[10:14:14] 2 tRNA-Gly [60312,60382] 33 (tcc)\\n[10:14:14] 3 tRNA-Lys c[248836,248907] 33 (ttt)\\n[10:14:14] 1 tRNA-Gly c[60165,60236] 33 (gcc)\\n[10:14:14] 1 tRNA-Ala c[44722,44796] 34 (tgc)\\n[10:14:14] 2 tRNA-Ile c[44883,44958] 35 (gat)\\n[10:14:14] 3 tRNA-Thr c[179214,179285] 33 (cgt)\\n[10:14:14] 4 tRNA-Leu c[305137,305219] 36 (gag)\\n[10:14:14] 5 tRNA-Arg [489652,489724] 34 (ccg)\\n[10:14:14] 1 tRNA-Ala c[11587,11661] 34 (tgc)\\n[10:14:14] 2 tRNA-Ile c[11748,11823] 35 (gat)\\n[10:14:14] 3 tRNA-Glu c[56357,56429] 35 (ttc)\\n[10:14:14] 1 tRNA-Leu [3304,3387] 36 (tag)\\n[10:14:14] 1 tRNA-Leu [11101,11183] 35 (cag)\\n[10:14:14] 1 tRNA-Trp [135883,135956] 34 (cca)\\n[10:14:14] Found 51 tRNAs\\n[10:14:14] Predicting Ribosomal RNAs\\n[10:14:14] Running Barrnap with 4 threads\\n[10:14:17] 1 gi_425733382_gb_ALVS01000090.1_ 8496 23S ribosomal RNA\\n[10:14:17] 2 gi_425733382_gb_ALVS01000090.1_ 11952 16S ribosomal RNA\\n[10:14:17] 3 gi_425733384_gb_ALVS01000088.1_ 41631 23S ribosomal RNA\\n[10:14:17] 4 gi_425733384_gb_ALVS01000088.1_ 45087 16S ribosomal RNA\\n[10:14:17] 5 gi_425733394_gb_ALVS01000078.1_ 143704 23S ribosomal RNA\\n[10:14:17] 6 gi_425733394_gb_ALVS01000078.1_ 147160 16S ribosomal RNA\\n[10:14:17] 7 gi_425733403_gb_ALVS01000069.1_ 310203 16S ribosomal RNA\\n[10:14:17] 8 gi_425733403_gb_ALVS01000069.1_ 312263 23S ribosomal RNA\\n[10:14:17] Found 8 rRNAs\\n[10:14:17] Skipping ncRNA search, enable with --rfam if desired.\\n[10:14:17] Total of 58 tRNA + rRNA features\\n[10:14:17] Searching for CRISPR repeats\\n[10:14:17] CRISPR1 gi_425733468_gb_ALVS01000004.1_ 18347 with 16 spacers\\n[10:14:17] CRISPR2 gi_425733456_gb_ALVS01000016.1_ 41671 with 12 spacers\\n[10:14:17] CRISPR3 gi_425733452_gb_ALVS01000020.1_ 83169 with 8 spacers\\n[10:14:17] CRISPR4 gi_425733452_gb_ALVS01000020.1_ 90035 with 10 spacers\\n[10:14:17] CRISPR5 gi_425733439_gb_ALVS01000033.1_ 21842 with 4 spacers\\n[10:14:17] CRISPR6 gi_425733439_gb_ALVS01000033.1_ 22133 with 3 spacers\\n[10:14:17] CRISPR7 gi_425733403_gb_ALVS01000069.1_ 41720 with 17 spacers\\n[10:14:17] CRISPR8 gi_425733403_gb_ALVS01000069.1_ 192884 with 10 spacers\\n[10:14:17] CRISPR9 gi_425733403_gb_ALVS01000069.1_ 199210 with 19 spacers\\n[10:14:17] CRISPR10 gi_425733402_gb_ALVS01000070.1_ 2617 with 15 spacers\\n[10:14:17] CRISPR11 gi_425733389_gb_ALVS01000083.1_ 2580 with 25 spacers\\n[10:14:17] CRISPR12 gi_425733389_gb_ALVS01000083.1_ 8188 with 9 spacers\\n[10:14:17] CRISPR13 gi_425733387_gb_ALVS01000085.1_ 21069 with 13 spacers\\n[10:14:17] CRISPR14 gi_425733386_gb_ALVS01000086.1_ 1485 with 13 spacers\\n[10:14:18] CRISPR15 gi_425733384_gb_ALVS01000088.1_ 11162 with 8 spacers\\n[10:14:18] Found 15 CRISPRs\\n[10:14:18] Predicting coding sequences\\n[10:14:18] Contigs total 8000057 bp, so using single mode\\n[10:14:18] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVS01\\\\/\\\\/Fischerella_ALVS01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:14:24] Excluding CDS which overlaps existing RNA (repeat_region) at gi_425733456_gb_ALVS01000016.1_:41523..41699 on + strand\\n[10:14:25] Excluding CDS which overlaps existing RNA (tRNA) at gi_425733413_gb_ALVS01000059.1_:211897..212175 on - strand\\n[10:14:25] Excluding CDS which overlaps existing RNA (tRNA) at gi_425733413_gb_ALVS01000059.1_:313203..313958 on + strand\\n[10:14:27] Excluding CDS which overlaps existing RNA (rRNA) at gi_425733403_gb_ALVS01000069.1_:314687..314863 on - strand\\n[10:14:27] Excluding CDS which overlaps existing RNA (repeat_region) at gi_425733402_gb_ALVS01000070.1_:3639..3818 on - strand\\n[10:14:27] Excluding CDS which overlaps existing RNA (rRNA) at gi_425733394_gb_ALVS01000078.1_:143989..144165 on + strand\\n[10:14:27] Excluding CDS which overlaps existing RNA (repeat_region) at gi_425733389_gb_ALVS01000083.1_:2395..2613 on + strand\\n[10:14:27] Excluding CDS which overlaps existing RNA (repeat_region) at gi_425733387_gb_ALVS01000085.1_:21770..22027 on + strand\\n[10:14:28] Excluding CDS which overlaps existing RNA (repeat_region) at gi_425733386_gb_ALVS01000086.1_:2362..2541 on - strand\\n[10:14:28] Excluding CDS which overlaps existing RNA (rRNA) at gi_425733384_gb_ALVS01000088.1_:41916..42092 on + strand\\n[10:14:28] Excluding CDS which overlaps existing RNA (rRNA) at gi_425733382_gb_ALVS01000090.1_:8781..8957 on + strand\\n[10:14:28] Excluding CDS which overlaps existing RNA (tRNA) at gi_425733381_gb_ALVS01000091.1_:3284..3460 on + strand\\n[10:14:28] Found 6661 CDS\\n[10:14:28] Connecting features back to sequences\\n[10:14:28] Not using genus-specific database. Try --usegenus to enable it.\\n[10:14:28] Annotating CDS, please be patient.\\n[10:14:28] Will use 4 CPUs for similarity searching.\\n[10:14:30] There are still 6661 unannotated CDS left (started with 6661)\\n[10:14:30] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:14:30] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVS01\\\\/\\\\/Fischerella_ALVS01\\\\.IS\\\\.tmp\\\\.98884\\\\.faa | parallel --gnu --plain -j 4 --block 271660 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVS01\\\\/\\\\/Fischerella_ALVS01\\\\.IS\\\\.tmp\\\\.98884\\\\.blast 2> /dev/null\\n[10:14:39] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVS01\\\\/\\\\/Fischerella_ALVS01\\\\.IS\\\\.tmp\\\\.98884\\\\.faa | parallel --gnu --plain -j 4 --block 271660 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVS01\\\\/\\\\/Fischerella_ALVS01\\\\.IS\\\\.tmp\\\\.98884\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:14:39] This is prokka 1.14.6\\n[10:14:39] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:14:39] Homepage is https://github.com/tseemann/prokka\\n[10:14:39] Local time is Mon Feb 21 10:14:39 2022\\n[10:14:39] You are tiagoferreiraleao\\n[10:14:39] Operating system is darwin\\n[10:14:39] You have BioPerl 1.007002\\n[10:14:40] System has 4 cores.\\n[10:14:40] Option --cpu asked for 12 cores, but system only has 4\\n[10:14:40] Will use maximum of 4 cores.\\n[10:14:40] Annotating as >>> Bacteria <<<\\n[10:14:40] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JADEXT01.fasta' contents.\\n[10:14:40] Setting --locustag PPNIEDCH from MD5 9972edc1135ef7899796bf96df5b9622\\n[10:14:40] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JADEXT01/\\n[10:14:40] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JADEXT01\\\\/\\n[10:14:40] Using filename prefix: Nostoc_JADEXT01.XXX\\n[10:14:40] Setting HMMER_NCPU=1\\n[10:14:40] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JADEXT01//Nostoc_JADEXT01.log\\n[10:14:40] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JADEXT01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JADEXT01/ --prefix Nostoc_JADEXT01 --cpus 12\\n[10:14:40] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:14:40] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:14:40] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:14:40] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:14:40] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:14:40] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:14:40] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:14:40] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:14:40] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:14:40] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:14:40] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:14:40] Looking for 'egrep' - found /usr/bin/egrep\\n[10:14:40] Looking for 'find' - found /usr/bin/find\\n[10:14:40] Looking for 'grep' - found /usr/bin/grep\\n[10:14:40] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:14:40] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:14:40] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:14:40] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:14:40] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:14:40] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:14:40] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:14:40] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:14:41] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:14:41] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:14:41] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:14:41] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:14:41] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:14:41] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:14:41] Looking for 'sed' - found /usr/bin/sed\\n[10:14:41] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:14:41] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:14:41] Using genetic code table 11.\\n[10:14:41] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JADEXT01.fasta\\n[10:14:41] Wrote 109 contigs totalling 8559644 bp.\\n[10:14:41] Predicting tRNAs and tmRNAs\\n[10:14:41] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JADEXT01\\\\/\\\\/Nostoc_JADEXT01\\\\.fna\\n[10:14:45] 1 tRNA-Phe [44781,44853] 34 (gaa)\\n[10:14:45] 2 tRNA-Val c[463704,463776] 34 (tac)\\n[10:14:45] 1 tRNA-Ala c[273478,273552] 35 (tgc)\\n[10:14:45] 2 tRNA-Met c[485530,485605] 36 (cat)\\n[10:14:45] 3 tRNA-Pro [497774,497847] 35 (ggg)\\n[10:14:45] 1 tRNA-Ala [370199,370271] 34 (ggc)\\n[10:14:45] 2 tRNA-Met [444707,444783] 35 (cat)\\n[10:14:45] 1 tRNA-Cys c[160415,160485] 33 (gca)\\n[10:14:45] 2 tRNA-Arg c[254085,254158] 35 (acg)\\n[10:14:45] 1 tRNA-Asp [60172,60245] 35 (gtc)\\n[10:14:45] 1 tRNA-Ser c[49981,50067] 36 (tga)\\n[10:14:45] 2 tRNA-Tyr [52367,52452] 35 (gta)\\n[10:14:45] 3 tRNA-Thr [52540,52611] 33 (ggt)\\n[10:14:45] 4 tRNA-Arg c[54192,54266] 35 (tct)\\n[10:14:45] 5 tRNA-Gly c[142899,142969] 33 (tcc)\\n[10:14:45] 1 tRNA-Gly c[171570,171641] 33 (gcc)\\n[10:14:45] 1 tRNA-Ala [12157,12229] 34 (cgc)\\n[10:14:45] 2 tRNA-Pro [183218,183291] 35 (cgg)\\n[10:14:45] 1 tRNA-Pro c[131185,131258] 35 (tgg)\\n[10:14:45] 1 tRNA-Leu c[126673,126755] 36 (gag)\\n[10:14:45] 2 tRNA-Lys [154761,154833] 33 (ctt)\\n[10:14:45] 1 tRNA-Thr [149199,149270] 33 (cgt)\\n[10:14:45] 1 tRNA-Lys [40247,40318] 33 (ttt)\\n[10:14:45] 2 tRNA-Leu [99799,99882] 36 (tag)\\n[10:14:45] 3 tRNA-Val [130545,130618] 35 (gac)\\n[10:14:45] 1 tRNA-Thr [72275,72346] 33 (tgt)\\n[10:14:45] 2 tRNA-Met c[79588,79661] 33 (cat)\\n[10:14:45] 1 tRNA-Trp c[70516,70589] 34 (cca)\\n[10:14:45] 1 tRNA-Arg [22625,22697] 34 (ccg)\\n[10:14:45] tRNA c[22132,22204] is a pseudo/wacky gene - skipping.\\n[10:14:45] 1 tRNA-Ser c[62047,62140] 36 (gct)\\n[10:14:45] 2 tRNA-Lys c[66167,66238] 33 (ttt)\\n[10:14:45] 3 tRNA-Ser c[66392,66475] 34 (tga)\\n[10:14:45] 4 tRNA-His c[66482,66556] 35 (gtg)\\n[10:14:45] 5 tRNA-Ala c[66562,66636] 35 (tgc)\\n[10:14:45] 6 tRNA-Ala c[66646,66718] 34 (ggc)\\n[10:14:45] 7 tRNA-Arg c[66722,66797] 35 (tct)\\n[10:14:45] 8 tRNA-Val c[66804,66879] 34 (tac)\\n[10:14:45] 9 tRNA-Thr c[66983,67055] 34 (tgt)\\n[10:14:45] 10 tRNA-Met c[67063,67139] 34 (cat)\\n[10:14:45] 11 tRNA-Leu c[67149,67223] 35 (tag)\\n[10:14:45] 12 tRNA-Leu c[67269,67345] 34 (gag)\\n[10:14:45] 13 tRNA-Leu c[67349,67423] 35 (cag)\\n[10:14:45] 14 tRNA-Pro c[67427,67499] 34 (tgg)\\n[10:14:45] 15 tRNA-Leu c[67501,67578] 35 (taa)\\n[10:14:45] 16 tRNA-Gln c[67694,67769] 34 (ttg)\\n[10:14:45] 17 tRNA-Gln c[67773,67845] 34 (ctg)\\n[10:14:45] 18 tRNA-Asn c[67988,68062] 33 (gtt)\\n[10:14:45] 19 tRNA-Arg c[68182,68258] 36 (acg)\\n[10:14:45] 20 tRNA-Ser c[68261,68352] 36 (gct)\\n[10:14:45] 21 tRNA-Glu c[68464,68537] 35 (ttc)\\n[10:14:45] tRNA c[68590,68664] is a pseudo/wacky gene - skipping.\\n[10:14:45] 23 tRNA-Met c[68667,68742] 34 (cat)\\n[10:14:45] 24 tRNA-Cys c[68752,68825] 35 (gca)\\n[10:14:45] 25 tRNA-Glu c[68829,68905] 35 (ctc)\\n[10:14:45] 26 tRNA-Asp c[68909,68984] 36 (gtc)\\n[10:14:45] 27 tRNA-Glu c[75684,75757] 35 (ttc)\\n[10:14:45] 1 tRNA-Ile c[30628,30703] 34 (tat)\\n[10:14:45] 2 tRNA-Met c[32594,32665] 32 (cat)\\n[10:14:45] 3 tRNA-Met c[32666,32741] 34 (cat)\\n[10:14:45] 1 tRNA-Phe c[12654,12726] 33 (gaa)\\n[10:14:45] 2 tRNA-Pyl c[14269,14341] 34 (cta)\\n[10:14:45] 3 tRNA-Ser [74246,74332] 36 (gga)\\n[10:14:45] 1 tmRNA [56685,57072] 101,151 ANNIVNFARKDALVAA*\\n[10:14:45] 2 tRNA-Gly c[80861,80932] 33 (ccc)\\n[10:14:45] 1 tRNA-Arg c[17851,17926] 36 (cct)\\n[10:14:45] 1 tRNA-His [5512,5585] 34 (gtg)\\n[10:14:45] 1 tRNA-Leu c[39567,39648] 35 (cag)\\n[10:14:45] 2 tRNA-Leu [44231,44315] 35 (caa)\\n[10:14:45] 3 tRNA-Gln [46283,46356] 34 (ttg)\\n[10:14:45] 1 tRNA-Ser c[2116,2202] 36 (cga)\\n[10:14:45] 1 tRNA-Asn [2549,2620] 33 (gtt)\\n[10:14:45] Found 70 tRNAs\\n[10:14:45] Predicting Ribosomal RNAs\\n[10:14:45] Running Barrnap with 4 threads\\n[10:14:48] 1 JADEXT010000062.1 16392 23S ribosomal RNA\\n[10:14:48] 2 JADEXT010000062.1 23331 16S ribosomal RNA (partial)\\n[10:14:48] 3 JADEXT010000109.1 5 16S ribosomal RNA\\n[10:14:48] 4 JADEXT010000109.1 1891 23S ribosomal RNA\\n[10:14:48] Found 4 rRNAs\\n[10:14:48] Skipping ncRNA search, enable with --rfam if desired.\\n[10:14:48] Total of 73 tRNA + rRNA features\\n[10:14:48] Searching for CRISPR repeats\\n[10:14:48] CRISPR1 JADEXT010000005.1 104535 with 32 spacers\\n[10:14:48] CRISPR2 JADEXT010000007.1 148476 with 24 spacers\\n[10:14:49] CRISPR3 JADEXT010000039.1 15756 with 24 spacers\\n[10:14:49] CRISPR4 JADEXT010000078.1 9584 with 8 spacers\\n[10:14:49] Found 4 CRISPRs\\n[10:14:49] Predicting coding sequences\\n[10:14:49] Contigs total 8559644 bp, so using single mode\\n[10:14:49] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JADEXT01\\\\/\\\\/Nostoc_JADEXT01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:15:00] Excluding CDS which overlaps existing RNA (tRNA) at JADEXT010000025.1:67458..67874 on + strand\\n[10:15:01] Excluding CDS which overlaps existing RNA (repeat_region) at JADEXT010000039.1:15657..15875 on - strand\\n[10:15:01] Excluding CDS which overlaps existing RNA (repeat_region) at JADEXT010000039.1:17230..17424 on + strand\\n[10:15:01] Excluding CDS which overlaps existing RNA (rRNA) at JADEXT010000062.1:17234..17776 on + strand\\n[10:15:01] Excluding CDS which overlaps existing RNA (rRNA) at JADEXT010000062.1:18203..18697 on + strand\\n[10:15:01] Excluding CDS which overlaps existing RNA (rRNA) at JADEXT010000062.1:18710..19153 on + strand\\n[10:15:01] Excluding CDS which overlaps existing RNA (rRNA) at JADEXT010000062.1:19272..19775 on - strand\\n[10:15:01] Excluding CDS which overlaps existing RNA (rRNA) at JADEXT010000062.1:20588..21241 on - strand\\n[10:15:02] Excluding CDS which overlaps existing RNA (repeat_region) at JADEXT010000078.1:10077..10358 on - strand\\n[10:15:02] Excluding CDS which overlaps existing RNA (rRNA) at JADEXT010000109.1:1134..1448 on + strand\\n[10:15:02] Found 7232 CDS\\n[10:15:02] Connecting features back to sequences\\n[10:15:02] Not using genus-specific database. Try --usegenus to enable it.\\n[10:15:02] Annotating CDS, please be patient.\\n[10:15:02] Will use 4 CPUs for similarity searching.\\n[10:15:04] There are still 7232 unannotated CDS left (started with 7232)\\n[10:15:04] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:15:04] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JADEXT01\\\\/\\\\/Nostoc_JADEXT01\\\\.IS\\\\.tmp\\\\.98997\\\\.faa | parallel --gnu --plain -j 4 --block 292890 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JADEXT01\\\\/\\\\/Nostoc_JADEXT01\\\\.IS\\\\.tmp\\\\.98997\\\\.blast 2> /dev/null\\n[10:15:25] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JADEXT01\\\\/\\\\/Nostoc_JADEXT01\\\\.IS\\\\.tmp\\\\.98997\\\\.faa | parallel --gnu --plain -j 4 --block 292890 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JADEXT01\\\\/\\\\/Nostoc_JADEXT01\\\\.IS\\\\.tmp\\\\.98997\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:15:26] This is prokka 1.14.6\\n[10:15:26] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:15:26] Homepage is https://github.com/tseemann/prokka\\n[10:15:26] Local time is Mon Feb 21 10:15:26 2022\\n[10:15:26] You are tiagoferreiraleao\\n[10:15:26] Operating system is darwin\\n[10:15:26] You have BioPerl 1.007002\\n[10:15:26] System has 4 cores.\\n[10:15:26] Option --cpu asked for 12 cores, but system only has 4\\n[10:15:26] Will use maximum of 4 cores.\\n[10:15:26] Annotating as >>> Bacteria <<<\\n[10:15:26] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_AJLK01.fasta' contents.\\n[10:15:26] Setting --locustag FHNHHBBJ from MD5 f1711bb38e27b4530279617e0de1f811\\n[10:15:26] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_AJLK01/\\n[10:15:26] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLK01\\\\/\\n[10:15:26] Using filename prefix: Fischerella_AJLK01.XXX\\n[10:15:26] Setting HMMER_NCPU=1\\n[10:15:26] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_AJLK01//Fischerella_AJLK01.log\\n[10:15:26] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_AJLK01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_AJLK01/ --prefix Fischerella_AJLK01 --cpus 12\\n[10:15:26] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:15:26] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:15:26] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:15:26] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:15:26] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:15:26] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:15:26] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:15:26] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:15:26] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:15:26] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:15:26] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:15:26] Looking for 'egrep' - found /usr/bin/egrep\\n[10:15:26] Looking for 'find' - found /usr/bin/find\\n[10:15:26] Looking for 'grep' - found /usr/bin/grep\\n[10:15:26] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:15:26] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:15:26] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:15:26] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:15:26] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:15:26] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:15:26] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:15:26] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:15:26] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:15:26] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:15:27] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:15:27] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:15:27] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:15:27] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:15:27] Looking for 'sed' - found /usr/bin/sed\\n[10:15:27] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:15:27] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:15:27] Using genetic code table 11.\\n[10:15:27] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_AJLK01.fasta\\n[10:15:27] Wrote 271 contigs totalling 6902954 bp.\\n[10:15:27] Predicting tRNAs and tmRNAs\\n[10:15:27] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLK01\\\\/\\\\/Fischerella_AJLK01\\\\.fna\\n[10:15:30] 1 tRNA-Gly [3341,3410] 32 (tcc)\\n[10:15:30] 1 tRNA-His c[16376,16449] 34 (gtg)\\n[10:15:30] 1 tRNA-Thr [377,448] 33 (cgt)\\n[10:15:30] 2 tRNA-Ala [1790,1862] 34 (ggc)\\n[10:15:30] 3 tRNA-Arg [9110,9183] 35 (acg)\\n[10:15:30] 4 tRNA-Ser c[19028,19121] 36 (gct)\\n[10:15:30] 1 tRNA-Asn c[45244,45317] 34 (att)\\n[10:15:30] 2 tRNA-Gly c[45785,45858] 35 (tcc)\\n[10:15:30] 1 tRNA-Glu c[19015,19087] 35 (ttc)\\n[10:15:30] 1 tRNA-Arg [1189,1263] 35 (tct)\\n[10:15:30] 1 tRNA-Leu [143,225] 35 (cag)\\n[10:15:30] 1 tRNA-Ser c[2830,2915] 35 (tga)\\n[10:15:30] 1 tmRNA c[12577,12963] 100,153 ANNIVKFARKEALVAA**\\n[10:15:30] 1 tRNA-Thr [84836,84907] 33 (tgt)\\n[10:15:30] 1 tRNA-Arg [97257,97330] 35 (ccg)\\n[10:15:30] 1 tRNA-Pro c[6014,6088] 35 (tgg)\\n[10:15:30] 1 tRNA-Met c[1444,1520] 35 (cat)\\n[10:15:30] 2 tRNA-Val [4591,4664] 35 (gac)\\n[10:15:30] 3 tRNA-Met c[23432,23506] 35 (cat)\\n[10:15:30] 1 tRNA-Met c[15328,15400] 34 (cat)\\n[10:15:30] 1 tRNA-Pro [180695,180787] 34 (tgg)\\n[10:15:30] 1 tRNA-Val c[34142,34216] 35 (tac)\\n[10:15:30] 1 tRNA-Gly [23979,24049] 33 (tcc)\\n[10:15:30] 1 tRNA-Trp c[82226,82297] 34 (cca)\\n[10:15:30] 1 tRNA-Ala c[18442,18516] 35 (tgc)\\n[10:15:30] 2 tRNA-Ala c[18526,18598] 34 (ggc)\\n[10:15:30] 3 tRNA-Thr c[18882,18955] 35 (tgt)\\n[10:15:30] 4 tRNA-Met c[18959,19036] 35 (cat)\\n[10:15:30] 5 tRNA-Leu c[19038,19112] 35 (taa)\\n[10:15:30] 6 tRNA-Lys c[19115,19189] 34 (ctt)\\n[10:15:30] 7 tRNA-Lys c[19193,19266] 34 (ttt)\\n[10:15:30] 8 tRNA-Leu c[19271,19348] 35 (gag)\\n[10:15:30] 9 tRNA-Leu c[19353,19427] 35 (tag)\\n[10:15:30] 10 tRNA-Pro c[19431,19507] 35 (tgg)\\n[10:15:30] 11 tRNA-Leu c[19509,19586] 35 (caa)\\n[10:15:30] 12 tRNA-Gln c[19591,19665] 35 (ttg)\\n[10:15:30] 13 tRNA-Asn c[19756,19831] 34 (gtt)\\n[10:15:30] 14 tRNA-Phe c[19840,19917] 38 (gaa)\\n[10:15:30] 15 tRNA-Glu c[20001,20077] 35 (ctc)\\n[10:15:30] 16 tRNA-Asp c[20080,20156] 37 (gtc)\\n[10:15:30] 17 tRNA-Thr c[59071,59142] 33 (ggt)\\n[10:15:30] 18 tRNA-Tyr c[59206,59290] 35 (gta)\\n[10:15:30] 1 tRNA-Cys [7773,7844] 33 (gca)\\n[10:15:30] 1 tRNA-Pro c[15389,15462] 35 (ggg)\\n[10:15:30] 1 tRNA-Gly [11869,11940] 33 (ccc)\\n[10:15:30] 1 tRNA-Lys [19627,19698] 33 (ttt)\\n[10:15:30] 1 tRNA-Leu [8018,8102] 35 (caa)\\n[10:15:30] 2 tRNA-Gln [46368,46439] 33 (ttg)\\n[10:15:30] 1 tRNA-Leu c[46521,46603] 35 (tag)\\n[10:15:30] 1 tRNA-Trp [2027,2100] 34 (cca)\\n[10:15:30] 1 tRNA-Leu [46790,46872] 36 (gag)\\n[10:15:30] 1 tRNA-Ser c[22816,22904] 36 (gga)\\n[10:15:30] 1 tRNA-Asp [11211,11284] 35 (gtc)\\n[10:15:30] 1 tRNA-Phe c[19524,19596] 34 (gaa)\\n[10:15:30] 1 tRNA-Asn c[1367,1440] 34 (gtt)\\n[10:15:30] 1 tRNA-Gly c[86846,86917] 33 (gcc)\\n[10:15:30] 2 tRNA-Ser c[95514,95600] 36 (cga)\\n[10:15:30] 1 tRNA-Pro [8352,8425] 35 (cgg)\\n[10:15:30] 1 tRNA-Ala c[1265,1339] 35 (cgc)\\n[10:15:30] 1 tRNA-Ala c[153,227] 34 (tgc)\\n[10:15:30] 2 tRNA-Ile c[314,389] 35 (gat)\\n[10:15:30] Found 61 tRNAs\\n[10:15:30] Predicting Ribosomal RNAs\\n[10:15:30] Running Barrnap with 4 threads\\n[10:15:31] 1 AJLK01000210.1 4 16S ribosomal RNA\\n[10:15:31] 2 AJLK01000211.1 161 23S ribosomal RNA\\n[10:15:31] 3 AJLK01000211.1 3119 5S ribosomal RNA\\n[10:15:31] Found 3 rRNAs\\n[10:15:31] Skipping ncRNA search, enable with --rfam if desired.\\n[10:15:31] Total of 63 tRNA + rRNA features\\n[10:15:31] Searching for CRISPR repeats\\n[10:15:32] CRISPR1 AJLK01000028.1 36243 with 83 spacers\\n[10:15:32] CRISPR2 AJLK01000058.1 13774 with 71 spacers\\n[10:15:32] CRISPR3 AJLK01000074.1 26120 with 8 spacers\\n[10:15:32] CRISPR4 AJLK01000075.1 51 with 65 spacers\\n[10:15:32] CRISPR5 AJLK01000131.1 74741 with 18 spacers\\n[10:15:32] CRISPR6 AJLK01000132.1 36 with 11 spacers\\n[10:15:32] CRISPR7 AJLK01000155.1 71 with 8 spacers\\n[10:15:32] CRISPR8 AJLK01000166.1 31 with 67 spacers\\n[10:15:32] CRISPR9 AJLK01000189.1 92739 with 30 spacers\\n[10:15:32] CRISPR10 AJLK01000204.1 7987 with 4 spacers\\n[10:15:32] CRISPR11 AJLK01000268.1 1 with 3 spacers\\n[10:15:32] Found 11 CRISPRs\\n[10:15:32] Predicting coding sequences\\n[10:15:32] Contigs total 6902954 bp, so using single mode\\n[10:15:32] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLK01\\\\/\\\\/Fischerella_AJLK01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:15:38] Excluding CDS which overlaps existing RNA (tRNA) at AJLK01000008.1:16385..16630 on + strand\\n[10:15:38] Excluding CDS which overlaps existing RNA (repeat_region) at AJLK01000028.1:36076..36276 on + strand\\n[10:15:38] Excluding CDS which overlaps existing RNA (tRNA) at AJLK01000028.1:45085..45306 on + strand\\n[10:15:38] Excluding CDS which overlaps existing RNA (repeat_region) at AJLK01000058.1:15012..15380 on - strand\\n[10:15:40] Excluding CDS which overlaps existing RNA (repeat_region) at AJLK01000155.1:271..480 on - strand\\n[10:15:40] Excluding CDS which overlaps existing RNA (repeat_region) at AJLK01000155.1:459..764 on - strand\\n[10:15:40] Excluding CDS which overlaps existing RNA (repeat_region) at AJLK01000166.1:4836..5090 on - strand\\n[10:15:41] Excluding CDS which overlaps existing RNA (repeat_region) at AJLK01000189.1:94865..95044 on - strand\\n[10:15:41] Excluding CDS which overlaps existing RNA (rRNA) at AJLK01000211.1:298..690 on - strand\\n[10:15:41] Found 5953 CDS\\n[10:15:41] Connecting features back to sequences\\n[10:15:41] Not using genus-specific database. Try --usegenus to enable it.\\n[10:15:41] Annotating CDS, please be patient.\\n[10:15:41] Will use 4 CPUs for similarity searching.\\n[10:15:42] There are still 5953 unannotated CDS left (started with 5953)\\n[10:15:42] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:15:42] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLK01\\\\/\\\\/Fischerella_AJLK01\\\\.IS\\\\.tmp\\\\.99111\\\\.faa | parallel --gnu --plain -j 4 --block 230059 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLK01\\\\/\\\\/Fischerella_AJLK01\\\\.IS\\\\.tmp\\\\.99111\\\\.blast 2> /dev/null\\n[10:15:57] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLK01\\\\/\\\\/Fischerella_AJLK01\\\\.IS\\\\.tmp\\\\.99111\\\\.faa | parallel --gnu --plain -j 4 --block 230059 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLK01\\\\/\\\\/Fischerella_AJLK01\\\\.IS\\\\.tmp\\\\.99111\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:15:58] This is prokka 1.14.6\\n[10:15:58] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:15:58] Homepage is https://github.com/tseemann/prokka\\n[10:15:58] Local time is Mon Feb 21 10:15:58 2022\\n[10:15:58] You are tiagoferreiraleao\\n[10:15:58] Operating system is darwin\\n[10:15:58] You have BioPerl 1.007002\\n[10:15:58] System has 4 cores.\\n[10:15:58] Option --cpu asked for 12 cores, but system only has 4\\n[10:15:58] Will use maximum of 4 cores.\\n[10:15:58] Annotating as >>> Bacteria <<<\\n[10:15:58] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NQME01.fasta' contents.\\n[10:15:58] Setting --locustag FMBCNBMO from MD5 f6bc7b687f8ac174f4c05ced15098cde\\n[10:15:58] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NQME01/\\n[10:15:58] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQME01\\\\/\\n[10:15:58] Using filename prefix: Fischerella_NQME01.XXX\\n[10:15:58] Setting HMMER_NCPU=1\\n[10:15:58] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NQME01//Fischerella_NQME01.log\\n[10:15:58] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NQME01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NQME01/ --prefix Fischerella_NQME01 --cpus 12\\n[10:15:58] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:15:58] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:15:58] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:15:58] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:15:58] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:15:58] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:15:58] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:15:58] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:15:58] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:15:58] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:15:58] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:15:58] Looking for 'egrep' - found /usr/bin/egrep\\n[10:15:58] Looking for 'find' - found /usr/bin/find\\n[10:15:58] Looking for 'grep' - found /usr/bin/grep\\n[10:15:58] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:15:58] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:15:58] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:15:58] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:15:58] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:15:58] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:15:58] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:15:58] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:15:59] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:15:59] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:15:59] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:15:59] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:15:59] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:15:59] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:15:59] Looking for 'sed' - found /usr/bin/sed\\n[10:15:59] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:15:59] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:15:59] Using genetic code table 11.\\n[10:15:59] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NQME01.fasta\\n[10:15:59] Wrote 604 contigs totalling 5168941 bp.\\n[10:15:59] Predicting tRNAs and tmRNAs\\n[10:15:59] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQME01\\\\/\\\\/Fischerella_NQME01\\\\.fna\\n[10:16:01] 1 tRNA-Gln c[1798,1868] 32 (ttg)\\n[10:16:01] 1 tRNA-Ser c[310,396] 36 (tga)\\n[10:16:01] 1 tRNA-Leu [82,165] 36 (tag)\\n[10:16:01] 1 tRNA-Thr [34217,34288] 33 (cgt)\\n[10:16:01] 1 tRNA-Lys c[6961,7034] 34 (ttt)\\n[10:16:01] 1 tRNA-Arg c[7186,7259] 35 (ccg)\\n[10:16:01] 1 tRNA-Ser c[3291,3379] 36 (gga)\\n[10:16:01] 1 tRNA-Met c[3711,3783] 34 (cat)\\n[10:16:01] 1 tRNA-Trp c[4010,4083] 34 (cca)\\n[10:16:01] 1 tRNA-Asp [2175,2248] 35 (gtc)\\n[10:16:01] 1 tRNA-Pro c[324,398] 35 (tgg)\\n[10:16:01] 1 tRNA-Met [3308,3383] 36 (cat)\\n[10:16:01] 1 tRNA-Cys [14769,14840] 33 (gca)\\n[10:16:01] 1 tmRNA c[5899,6285] 100,153 ANNIVKFARKEALVAA**\\n[10:16:01] 1 tRNA-Leu [557,641] 35 (caa)\\n[10:16:01] 1 tRNA-Ser c[2733,2819] 36 (cga)\\n[10:16:01] 1 tRNA-Thr [9378,9449] 33 (tgt)\\n[10:16:01] 1 tRNA-Asn [2663,2736] 34 (gtt)\\n[10:16:01] 1 tRNA-Phe c[1228,1302] 35 (gaa)\\n[10:16:01] 2 tRNA-Val [8652,8724] 34 (tac)\\n[10:16:01] 1 tRNA-Gly [7472,7543] 33 (gcc)\\n[10:16:01] 1 tRNA-Ala c[3070,3143] 34 (ggc)\\n[10:16:01] 1 tRNA-Arg c[1203,1277] 35 (tct)\\n[10:16:01] 1 tRNA-Thr c[174,245] 33 (ggt)\\n[10:16:01] 2 tRNA-Tyr c[309,393] 35 (gta)\\n[10:16:01] 1 tRNA-His c[9676,9749] 34 (gtg)\\n[10:16:01] 1 tRNA-Gln c[5300,5371] 33 (ttg)\\n[10:16:01] 1 tRNA-Gly c[127,198] 33 (ccc)\\n[10:16:01] 1 tRNA-Ser [6737,6830] 36 (gct)\\n[10:16:01] 1 tRNA-Pro [12959,13032] 35 (cgg)\\n[10:16:01] 1 tRNA-Ala c[1087,1162] 35 (cgc)\\n[10:16:01] 1 tRNA-Arg [1843,1916] 35 (acg)\\n[10:16:01] 1 tRNA-Gly [6693,6763] 33 (tcc)\\n[10:16:01] 1 tRNA-Leu [7483,7565] 36 (gag)\\n[10:16:01] 1 tRNA-Met c[10966,11042] 35 (cat)\\n[10:16:01] 2 tRNA-Val [14026,14099] 35 (gac)\\n[10:16:01] 1 tRNA-Arg [829,901] 34 (tcg)\\n[10:16:01] 1 tRNA-Leu [14314,14396] 35 (cag)\\n[10:16:01] 1 tRNA-Pro [3776,3849] 35 (ggg)\\n[10:16:01] 1 tRNA-Glu [7208,7280] 35 (ttc)\\n[10:16:01] Found 40 tRNAs\\n[10:16:01] Predicting Ribosomal RNAs\\n[10:16:01] Running Barrnap with 4 threads\\n[10:16:02] Found 0 rRNAs\\n[10:16:02] Skipping ncRNA search, enable with --rfam if desired.\\n[10:16:02] Total of 39 tRNA + rRNA features\\n[10:16:02] Searching for CRISPR repeats\\n[10:16:03] CRISPR1 NQME01000116.1 65 with 8 spacers\\n[10:16:03] CRISPR2 NQME01000226.1 798 with 4 spacers\\n[10:16:03] CRISPR3 NQME01000481.1 65 with 9 spacers\\n[10:16:03] CRISPR4 NQME01000546.1 4063 with 12 spacers\\n[10:16:03] CRISPR5 NQME01000574.1 68 with 5 spacers\\n[10:16:03] Found 5 CRISPRs\\n[10:16:03] Predicting coding sequences\\n[10:16:03] Contigs total 5168941 bp, so using single mode\\n[10:16:03] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQME01\\\\/\\\\/Fischerella_NQME01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:16:08] Excluding CDS which overlaps existing RNA (tRNA) at NQME01000311.1:2537..3976 on + strand\\n[10:16:09] Found 4389 CDS\\n[10:16:09] Connecting features back to sequences\\n[10:16:09] Not using genus-specific database. Try --usegenus to enable it.\\n[10:16:09] Annotating CDS, please be patient.\\n[10:16:09] Will use 4 CPUs for similarity searching.\\n[10:16:10] There are still 4389 unannotated CDS left (started with 4389)\\n[10:16:10] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:16:10] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQME01\\\\/\\\\/Fischerella_NQME01\\\\.IS\\\\.tmp\\\\.99225\\\\.faa | parallel --gnu --plain -j 4 --block 171405 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQME01\\\\/\\\\/Fischerella_NQME01\\\\.IS\\\\.tmp\\\\.99225\\\\.blast 2> /dev/null\\n[10:16:21] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQME01\\\\/\\\\/Fischerella_NQME01\\\\.IS\\\\.tmp\\\\.99225\\\\.faa | parallel --gnu --plain -j 4 --block 171405 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQME01\\\\/\\\\/Fischerella_NQME01\\\\.IS\\\\.tmp\\\\.99225\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:16:21] This is prokka 1.14.6\\n[10:16:21] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:16:21] Homepage is https://github.com/tseemann/prokka\\n[10:16:21] Local time is Mon Feb 21 10:16:21 2022\\n[10:16:21] You are tiagoferreiraleao\\n[10:16:21] Operating system is darwin\\n[10:16:21] You have BioPerl 1.007002\\n[10:16:21] System has 4 cores.\\n[10:16:21] Option --cpu asked for 12 cores, but system only has 4\\n[10:16:21] Will use maximum of 4 cores.\\n[10:16:21] Annotating as >>> Bacteria <<<\\n[10:16:21] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQD01.fasta' contents.\\n[10:16:21] Setting --locustag BHKEEFPD from MD5 b14eef9dc89f18f2784676bc1027c318\\n[10:16:21] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQD01/\\n[10:16:21] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQD01\\\\/\\n[10:16:21] Using filename prefix: Fischerella_NMQD01.XXX\\n[10:16:21] Setting HMMER_NCPU=1\\n[10:16:21] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQD01//Fischerella_NMQD01.log\\n[10:16:21] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQD01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQD01/ --prefix Fischerella_NMQD01 --cpus 12\\n[10:16:21] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:16:21] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:16:21] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:16:21] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:16:21] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:16:21] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:16:21] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:16:21] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:16:21] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:16:21] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:16:21] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:16:21] Looking for 'egrep' - found /usr/bin/egrep\\n[10:16:21] Looking for 'find' - found /usr/bin/find\\n[10:16:21] Looking for 'grep' - found /usr/bin/grep\\n[10:16:21] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:16:21] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:16:21] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:16:21] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:16:21] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:16:21] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:16:22] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:16:22] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:16:22] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:16:22] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:16:22] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:16:22] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:16:22] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:16:22] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:16:22] Looking for 'sed' - found /usr/bin/sed\\n[10:16:22] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:16:23] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:16:23] Using genetic code table 11.\\n[10:16:23] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQD01.fasta\\n[10:16:23] Wrote 2176 contigs totalling 6252881 bp.\\n[10:16:23] Predicting tRNAs and tmRNAs\\n[10:16:23] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQD01\\\\/\\\\/Fischerella_NMQD01\\\\.fna\\n[10:16:24] 1 tRNA-Ser [126,219] 36 (gct)\\n[10:16:24] 1 tmRNA [1113,1498] 99,152 ANNIVKFARKEALVAA**\\n[10:16:24] 1 tRNA-Val [3622,3694] 34 (tac)\\n[10:16:24] 1 tRNA-Asp c[3375,3448] 35 (gtc)\\n[10:16:24] 1 tRNA-Leu [970,1052] 36 (gag)\\n[10:16:24] 1 tRNA-Pyl [320,407] 30 (cta)\\n[10:16:24] 1 tRNA-Ala c[327,399] 34 (ggc)\\n[10:16:24] 2 tRNA-Thr c[1715,1786] 33 (cgt)\\n[10:16:24] 1 tRNA-Lys [1057,1130] 34 (ttt)\\n[10:16:24] 1 tRNA-Ala c[1669,1743] 35 (cgc)\\n[10:16:24] 1 tRNA-Cys c[4176,4247] 33 (gca)\\n[10:16:24] 1 tRNA-Gln c[3504,3575] 33 (ttg)\\n[10:16:24] 1 tRNA-Arg [12065,12137] 34 (cct)\\n[10:16:25] 1 tRNA-Thr [7618,7691] 34 (tgt)\\n[10:16:25] 2 tRNA-Tyr [7716,7797] 35 (gta)\\n[10:16:25] 3 tRNA-Gly [7808,7883] 34 (tcc)\\n[10:16:25] 4 tRNA-Thr [7889,7961] 33 (ggt)\\n[10:16:25] 5 tRNA-Trp [9252,9323] 33 (cca)\\n[10:16:25] 1 tRNA-Met c[521,594] 34 (cat)\\n[10:16:25] 1 tRNA-Ser c[647,735] 36 (gga)\\n[10:16:25] 1 tRNA-Leu c[5016,5102] 35 (taa)\\n[10:16:25] 1 tRNA-Pro c[8197,8270] 35 (cgg)\\n[10:16:25] 1 tRNA-Asp c[2112,2186] 35 (gtc)\\n[10:16:25] 1 tRNA-Met [2813,2885] 34 (cat)\\n[10:16:25] 1 tRNA-Leu c[927,1009] 35 (cag)\\n[10:16:25] 1 tRNA-Tyr [15,99] 35 (gta)\\n[10:16:25] 2 tRNA-Thr [164,235] 33 (ggt)\\n[10:16:25] 1 tRNA-Cys [1924,1995] 33 (gca)\\n[10:16:25] 1 tRNA-Gly c[782,853] 33 (gcc)\\n[10:16:25] 1 tRNA-Asn c[12536,12610] 35 (gtt)\\n[10:16:25] 1 tRNA-Leu [1075,1161] 34 (gag)\\n[10:16:25] 1 tRNA-Asp c[129,204] 34 (gtc)\\n[10:16:25] 1 tRNA-Ser c[503,589] 36 (cga)\\n[10:16:25] 1 tRNA-Ser [46,134] 35 (tga)\\n[10:16:25] 1 tRNA-Arg [3604,3676] 34 (ccg)\\n[10:16:25] 1 tRNA-Met c[146,222] 35 (cat)\\n[10:16:25] 2 tRNA-Val [3193,3266] 35 (gac)\\n[10:16:25] 1 tRNA-Gly [1744,1815] 33 (ccc)\\n[10:16:25] 1 tRNA-Ser c[19668,19757] 35 (gct)\\n[10:16:25] 1 tRNA-Gln [15642,15713] 33 (ttg)\\n[10:16:25] 1 tRNA-Gly c[319,391] 33 (ccc)\\n[10:16:26] 1 tRNA-Asn c[1733,1806] 34 (att)\\n[10:16:26] 2 tRNA-Arg c[2202,2274] 34 (tcg)\\n[10:16:26] 1 tRNA-Ile c[130,204] 35 (gat)\\n[10:16:26] 1 tRNA-Arg c[617,691] 35 (acg)\\n[10:16:26] 1 tRNA-Asn [2587,2660] 34 (gtt)\\n[10:16:26] 1 tRNA-Arg c[2320,2394] 35 (tct)\\n[10:16:26] 1 tRNA-Met [104,179] 36 (cat)\\n[10:16:26] 1 tRNA-Ser [1383,1469] 36 (tga)\\n[10:16:26] 1 tRNA-Pro c[167,240] 35 (ggg)\\n[10:16:26] 1 tRNA-Pro [4239,4313] 35 (tgg)\\n[10:16:26] 1 tRNA-Phe [7981,8055] 34 (gaa)\\n[10:16:26] 1 tRNA-Arg c[2565,2638] 35 (ccg)\\n[10:16:26] 1 tRNA-Arg c[1859,1932] 35 (acg)\\n[10:16:26] 1 tRNA-Leu c[1067,1151] 35 (caa)\\n[10:16:26] 1 tRNA-His [1774,1847] 34 (gtg)\\n[10:16:26] 1 tRNA-Glu c[652,724] 35 (ttc)\\n[10:16:26] Found 57 tRNAs\\n[10:16:26] Predicting Ribosomal RNAs\\n[10:16:26] Running Barrnap with 4 threads\\n[10:16:27] 1 NMQD01000199.1 2 23S ribosomal RNA (partial)\\n[10:16:27] 2 NMQD01000617.1 2 23S ribosomal RNA (partial)\\n[10:16:27] 3 NMQD01000930.1 420 5S ribosomal RNA\\n[10:16:27] 4 NMQD01001062.1 2 16S ribosomal RNA (partial)\\n[10:16:27] Found 4 rRNAs\\n[10:16:27] Skipping ncRNA search, enable with --rfam if desired.\\n[10:16:27] Total of 60 tRNA + rRNA features\\n[10:16:27] Searching for CRISPR repeats\\n[10:16:28] Found 0 CRISPRs\\n[10:16:28] Predicting coding sequences\\n[10:16:28] Contigs total 6252881 bp, so using single mode\\n[10:16:28] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQD01\\\\/\\\\/Fischerella_NMQD01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:16:33] Excluding CDS which overlaps existing RNA (rRNA) at NMQD01000199.1:1204..1383 on - strand\\n[10:16:33] Excluding CDS which overlaps existing RNA (tRNA) at NMQD01000399.1:4003..4194 on + strand\\n[10:16:34] Excluding CDS which overlaps existing RNA (tRNA) at NMQD01000952.1:773..1231 on + strand\\n[10:16:34] Excluding CDS which overlaps existing RNA (tRNA) at NMQD01000985.1:190..579 on - strand\\n[10:16:34] Excluding CDS which overlaps existing RNA (tRNA) at NMQD01001429.1:39..377 on - strand\\n[10:16:34] Excluding CDS which overlaps existing RNA (tRNA) at NMQD01001614.1:2173..2460 on - strand\\n[10:16:34] Excluding CDS which overlaps existing RNA (tRNA) at NMQD01001710.1:2602..3900 on + strand\\n[10:16:35] Excluding CDS which overlaps existing RNA (tRNA) at NMQD01001984.1:1816..2487 on + strand\\n[10:16:35] Found 5122 CDS\\n[10:16:35] Connecting features back to sequences\\n[10:16:35] Not using genus-specific database. Try --usegenus to enable it.\\n[10:16:35] Annotating CDS, please be patient.\\n[10:16:35] Will use 4 CPUs for similarity searching.\\n[10:16:36] There are still 5122 unannotated CDS left (started with 5122)\\n[10:16:36] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:16:36] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQD01\\\\/\\\\/Fischerella_NMQD01\\\\.IS\\\\.tmp\\\\.99337\\\\.faa | parallel --gnu --plain -j 4 --block 188550 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQD01\\\\/\\\\/Fischerella_NMQD01\\\\.IS\\\\.tmp\\\\.99337\\\\.blast 2> /dev/null\\n[10:16:47] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQD01\\\\/\\\\/Fischerella_NMQD01\\\\.IS\\\\.tmp\\\\.99337\\\\.faa | parallel --gnu --plain -j 4 --block 188550 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQD01\\\\/\\\\/Fischerella_NMQD01\\\\.IS\\\\.tmp\\\\.99337\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:16:48] This is prokka 1.14.6\\n[10:16:48] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:16:48] Homepage is https://github.com/tseemann/prokka\\n[10:16:48] Local time is Mon Feb 21 10:16:48 2022\\n[10:16:48] You are tiagoferreiraleao\\n[10:16:48] Operating system is darwin\\n[10:16:48] You have BioPerl 1.007002\\n[10:16:48] System has 4 cores.\\n[10:16:48] Option --cpu asked for 12 cores, but system only has 4\\n[10:16:48] Will use maximum of 4 cores.\\n[10:16:48] Annotating as >>> Bacteria <<<\\n[10:16:48] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_AGIZ01.fasta' contents.\\n[10:16:48] Setting --locustag KNAPIPKK from MD5 47a92944c84d843a876b5072d3b643ff\\n[10:16:48] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_AGIZ01/\\n[10:16:48] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AGIZ01\\\\/\\n[10:16:48] Using filename prefix: Fischerella_AGIZ01.XXX\\n[10:16:48] Setting HMMER_NCPU=1\\n[10:16:48] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_AGIZ01//Fischerella_AGIZ01.log\\n[10:16:48] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_AGIZ01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_AGIZ01/ --prefix Fischerella_AGIZ01 --cpus 12\\n[10:16:48] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:16:48] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:16:48] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:16:48] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:16:48] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:16:48] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:16:48] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:16:48] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:16:48] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:16:48] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:16:48] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:16:48] Looking for 'egrep' - found /usr/bin/egrep\\n[10:16:48] Looking for 'find' - found /usr/bin/find\\n[10:16:48] Looking for 'grep' - found /usr/bin/grep\\n[10:16:48] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:16:48] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:16:48] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:16:48] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:16:48] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:16:48] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:16:48] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:16:48] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:16:49] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:16:49] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:16:49] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:16:49] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:16:49] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:16:49] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:16:49] Looking for 'sed' - found /usr/bin/sed\\n[10:16:49] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:16:49] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:16:49] Using genetic code table 11.\\n[10:16:49] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_AGIZ01.fasta\\n[10:16:49] Wrote 34 contigs totalling 5380000 bp.\\n[10:16:49] Predicting tRNAs and tmRNAs\\n[10:16:49] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AGIZ01\\\\/\\\\/Fischerella_AGIZ01\\\\.fna\\n[10:16:52] 1 tRNA-Glu [64800,64872] 35 (ttc)\\n[10:16:52] 2 tRNA-Arg [82783,82855] 34 (tcg)\\n[10:16:52] 3 tRNA-Asn [83526,83599] 34 (att)\\n[10:16:52] 4 tRNA-Leu c[496535,496617] 36 (gag)\\n[10:16:52] 1 tRNA-Thr [57830,57901] 33 (tgt)\\n[10:16:52] 2 tRNA-His [368906,368979] 34 (gtg)\\n[10:16:52] 1 tRNA-Arg [87095,87169] 35 (tct)\\n[10:16:52] 2 tRNA-Met [111973,112048] 36 (cat)\\n[10:16:52] 3 tRNA-Phe c[150881,150953] 34 (gaa)\\n[10:16:52] 4 tRNA-Val [158356,158428] 34 (tac)\\n[10:16:52] 5 tRNA-Val c[357451,357524] 35 (gac)\\n[10:16:52] 6 tRNA-Met [362204,362280] 35 (cat)\\n[10:16:52] 1 tRNA-Arg c[37072,37147] 36 (cct)\\n[10:16:52] 2 tRNA-Ser c[102963,103051] 36 (gga)\\n[10:16:52] 3 tRNA-Lys [391337,391410] 34 (ttt)\\n[10:16:52] 1 tRNA-Gly [91435,91505] 33 (tcc)\\n[10:16:52] 2 tRNA-Cys [142957,143028] 33 (gca)\\n[10:16:52] 3 tRNA-Gln c[167171,167242] 33 (ttg)\\n[10:16:52] 4 tRNA-Leu c[200614,200698] 35 (caa)\\n[10:16:52] 1 tRNA-Pro [155158,155231] 35 (ggg)\\n[10:16:52] 2 tRNA-Thr [276781,276852] 33 (cgt)\\n[10:16:52] 3 tRNA-Ala [278974,279047] 34 (ggc)\\n[10:16:52] 4 tRNA-Arg [283939,284012] 35 (acg)\\n[10:16:52] 5 tmRNA c[377057,377443] 100,150 ANNIVKFARKEALVAA*\\n[10:16:52] 1 tRNA-Leu c[23943,24026] 36 (tag)\\n[10:16:52] 2 tRNA-Ala [177246,177320] 35 (cgc)\\n[10:16:52] 3 tRNA-Leu [203120,203202] 35 (cag)\\n[10:16:52] 1 tRNA-Arg [178955,179028] 35 (ccg)\\n[10:16:52] 2 tRNA-Trp [223485,223558] 34 (cca)\\n[10:16:52] 1 tRNA-Ser [82772,82858] 36 (tga)\\n[10:16:52] 2 tRNA-Ser c[223442,223528] 36 (cga)\\n[10:16:52] 1 tRNA-Gln [37718,37788] 32 (ttg)\\n[10:16:52] 2 tRNA-Ser c[199737,199830] 36 (gct)\\n[10:16:52] 1 tRNA-Pro [97354,97427] 35 (cgg)\\n[10:16:52] 1 tRNA-Asn [111578,111651] 34 (gtt)\\n[10:16:52] 2 tRNA-Pro c[196078,196152] 35 (tgg)\\n[10:16:52] 1 tRNA-Tyr [9191,9275] 35 (gta)\\n[10:16:52] 2 tRNA-Thr [9339,9410] 33 (ggt)\\n[10:16:52] 3 tRNA-Gly c[28129,28200] 33 (ccc)\\n[10:16:52] 1 tRNA-Gly c[156656,156727] 33 (gcc)\\n[10:16:52] 1 tRNA-Met c[111714,111786] 34 (cat)\\n[10:16:52] 1 tRNA-Asp [15829,15902] 35 (gtc)\\n[10:16:52] 1 tRNA-Ala c[378,452] 34 (tgc)\\n[10:16:52] 2 tRNA-Ile c[535,610] 35 (gat)\\n[10:16:52] Found 44 tRNAs\\n[10:16:52] Predicting Ribosomal RNAs\\n[10:16:52] Running Barrnap with 4 threads\\n[10:16:53] 1 AGIZ01000009.1 228141 16S ribosomal RNA\\n[10:16:53] 2 AGIZ01000009.1 229980 23S ribosomal RNA\\n[10:16:53] 3 AGIZ01000009.1 232919 5S ribosomal RNA\\n[10:16:53] 4 AGIZ01000012.1 215210 5S ribosomal RNA\\n[10:16:53] 5 AGIZ01000019.1 60031 5S ribosomal RNA\\n[10:16:53] Found 5 rRNAs\\n[10:16:53] Skipping ncRNA search, enable with --rfam if desired.\\n[10:16:53] Total of 48 tRNA + rRNA features\\n[10:16:53] Searching for CRISPR repeats\\n[10:16:53] CRISPR1 AGIZ01000001.1 249682 with 61 spacers\\n[10:16:54] CRISPR2 AGIZ01000004.1 180673 with 44 spacers\\n[10:16:54] CRISPR3 AGIZ01000004.1 184800 with 14 spacers\\n[10:16:54] CRISPR4 AGIZ01000007.1 24268 with 4 spacers\\n[10:16:54] CRISPR5 AGIZ01000007.1 352970 with 16 spacers\\n[10:16:54] CRISPR6 AGIZ01000007.1 354817 with 24 spacers\\n[10:16:54] CRISPR7 AGIZ01000011.1 213533 with 86 spacers\\n[10:16:54] CRISPR8 AGIZ01000012.1 132 with 24 spacers\\n[10:16:54] Found 8 CRISPRs\\n[10:16:54] Predicting coding sequences\\n[10:16:54] Contigs total 5380000 bp, so using single mode\\n[10:16:54] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AGIZ01\\\\/\\\\/Fischerella_AGIZ01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:16:58] Excluding CDS which overlaps existing RNA (repeat_region) at AGIZ01000001.1:250976..251152 on - strand\\n[10:17:00] Excluding CDS which overlaps existing RNA (tRNA) at AGIZ01000006.1:283384..284055 on - strand\\n[10:17:00] Excluding CDS which overlaps existing RNA (tRNA) at AGIZ01000009.1:223412..223540 on - strand\\n[10:17:00] Excluding CDS which overlaps existing RNA (rRNA) at AGIZ01000009.1:230375..230554 on + strand\\n[10:17:01] Excluding CDS which overlaps existing RNA (tRNA) at AGIZ01000012.1:111452..112891 on + strand\\n[10:17:01] Found 4623 CDS\\n[10:17:01] Connecting features back to sequences\\n[10:17:01] Not using genus-specific database. Try --usegenus to enable it.\\n[10:17:01] Annotating CDS, please be patient.\\n[10:17:01] Will use 4 CPUs for similarity searching.\\n[10:17:02] There are still 4623 unannotated CDS left (started with 4623)\\n[10:17:02] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:17:02] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AGIZ01\\\\/\\\\/Fischerella_AGIZ01\\\\.IS\\\\.tmp\\\\.99467\\\\.faa | parallel --gnu --plain -j 4 --block 181156 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AGIZ01\\\\/\\\\/Fischerella_AGIZ01\\\\.IS\\\\.tmp\\\\.99467\\\\.blast 2> /dev/null\\n[10:17:06] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AGIZ01\\\\/\\\\/Fischerella_AGIZ01\\\\.IS\\\\.tmp\\\\.99467\\\\.faa | parallel --gnu --plain -j 4 --block 181156 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AGIZ01\\\\/\\\\/Fischerella_AGIZ01\\\\.IS\\\\.tmp\\\\.99467\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:17:06] This is prokka 1.14.6\\n[10:17:06] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:17:06] Homepage is https://github.com/tseemann/prokka\\n[10:17:06] Local time is Mon Feb 21 10:17:06 2022\\n[10:17:06] You are tiagoferreiraleao\\n[10:17:06] Operating system is darwin\\n[10:17:06] You have BioPerl 1.007002\\n[10:17:07] System has 4 cores.\\n[10:17:07] Option --cpu asked for 12 cores, but system only has 4\\n[10:17:07] Will use maximum of 4 cores.\\n[10:17:07] Annotating as >>> Bacteria <<<\\n[10:17:07] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZS01.fasta' contents.\\n[10:17:07] Setting --locustag DMLEMEPC from MD5 d65e6e9cde1678e1c30c7c10b8cf972e\\n[10:17:07] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZS01/\\n[10:17:07] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZS01\\\\/\\n[10:17:07] Using filename prefix: Fischerella_NGZS01.XXX\\n[10:17:07] Setting HMMER_NCPU=1\\n[10:17:07] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZS01//Fischerella_NGZS01.log\\n[10:17:07] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZS01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZS01/ --prefix Fischerella_NGZS01 --cpus 12\\n[10:17:07] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:17:07] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:17:07] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:17:07] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:17:07] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:17:07] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:17:07] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:17:07] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:17:07] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:17:07] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:17:07] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:17:07] Looking for 'egrep' - found /usr/bin/egrep\\n[10:17:07] Looking for 'find' - found /usr/bin/find\\n[10:17:07] Looking for 'grep' - found /usr/bin/grep\\n[10:17:07] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:17:07] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:17:07] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:17:07] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:17:07] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:17:07] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:17:07] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:17:07] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:17:07] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:17:07] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:17:07] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:17:07] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:17:07] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:17:07] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:17:07] Looking for 'sed' - found /usr/bin/sed\\n[10:17:07] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:17:08] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:17:08] Using genetic code table 11.\\n[10:17:08] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZS01.fasta\\n[10:17:08] Wrote 128 contigs totalling 5399690 bp.\\n[10:17:08] Predicting tRNAs and tmRNAs\\n[10:17:08] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZS01\\\\/\\\\/Fischerella_NGZS01\\\\.fna\\n[10:17:10] 1 tRNA-Gln [34308,34379] 33 (ttg)\\n[10:17:10] 1 tRNA-Arg c[36321,36396] 36 (cct)\\n[10:17:10] 1 tRNA-Pro c[1735,1809] 35 (tgg)\\n[10:17:10] 1 tRNA-Cys c[90,161] 33 (gca)\\n[10:17:10] 1 tRNA-Gln c[17757,17827] 32 (ttg)\\n[10:17:10] 1 tRNA-Met c[20772,20844] 34 (cat)\\n[10:17:10] 1 tRNA-Leu [43063,43147] 35 (caa)\\n[10:17:10] 1 tRNA-Thr [49959,50030] 33 (cgt)\\n[10:17:10] 1 tRNA-Ser [4597,4683] 36 (cga)\\n[10:17:10] 1 tRNA-Leu [20858,20940] 35 (cag)\\n[10:17:10] 1 tRNA-Arg [1525,1598] 35 (ccg)\\n[10:17:10] 1 tRNA-Asp [16036,16109] 35 (gtc)\\n[10:17:10] 1 tRNA-Ser [15940,16026] 36 (tga)\\n[10:17:10] 1 tRNA-Ser [22527,22620] 36 (gct)\\n[10:17:10] 1 tRNA-Trp [4112,4185] 34 (cca)\\n[10:17:10] 1 tRNA-Ala [641,714] 34 (ggc)\\n[10:17:10] 2 tRNA-Arg [5606,5679] 35 (acg)\\n[10:17:10] 1 tRNA-Gly [16388,16459] 33 (ccc)\\n[10:17:10] 2 tRNA-Thr c[34714,34785] 33 (ggt)\\n[10:17:10] 3 tRNA-Tyr c[34849,34933] 35 (gta)\\n[10:17:10] 1 tRNA-Gly c[35234,35305] 33 (gcc)\\n[10:17:10] 2 tRNA-Asn [51885,51958] 34 (gtt)\\n[10:17:10] 1 tRNA-Val c[44800,44873] 35 (gac)\\n[10:17:10] 2 tRNA-Met [47818,47894] 35 (cat)\\n[10:17:10] 1 tRNA-Thr c[4363,4434] 33 (tgt)\\n[10:17:10] 1 tRNA-Leu c[56420,56502] 36 (gag)\\n[10:17:10] 1 tRNA-Glu [39349,39421] 35 (ttc)\\n[10:17:10] 2 tRNA-Arg [57334,57406] 34 (tcg)\\n[10:17:10] 3 tRNA-Asn [58077,58150] 34 (att)\\n[10:17:10] 1 tRNA-Phe c[1673,1745] 34 (gaa)\\n[10:17:10] 2 tRNA-Val [9128,9200] 34 (tac)\\n[10:17:10] 1 tRNA-Ser c[4210,4298] 36 (gga)\\n[10:17:10] 1 tRNA-Gly [66034,66104] 33 (tcc)\\n[10:17:10] 1 tRNA-Ala c[37073,37147] 34 (tgc)\\n[10:17:10] 2 tRNA-Ile c[37230,37305] 35 (gat)\\n[10:17:10] 1 tRNA-Ala [87139,87213] 35 (cgc)\\n[10:17:10] 1 tRNA-His c[57637,57710] 34 (gtg)\\n[10:17:10] 2 tRNA-Arg [96063,96137] 35 (tct)\\n[10:17:10] 3 tRNA-Met [120913,120988] 36 (cat)\\n[10:17:10] 1 tRNA-Lys c[259267,259340] 34 (ttt)\\n[10:17:10] 1 tRNA-Pro [12606,12679] 35 (cgg)\\n[10:17:10] 2 tmRNA c[92328,92713] 100,150 ANNIVKFARKEALVAA*\\n[10:17:10] 1 tRNA-Leu c[26235,26318] 36 (tag)\\n[10:17:10] 1 tRNA-Pro [49215,49288] 35 (ggg)\\n[10:17:10] Found 44 tRNAs\\n[10:17:10] Predicting Ribosomal RNAs\\n[10:17:10] Running Barrnap with 4 threads\\n[10:17:11] 1 NGZS01000114.1 33488 5S ribosomal RNA\\n[10:17:11] 2 NGZS01000114.1 33717 23S ribosomal RNA\\n[10:17:11] 3 NGZS01000114.1 37512 16S ribosomal RNA\\n[10:17:11] Found 3 rRNAs\\n[10:17:11] Skipping ncRNA search, enable with --rfam if desired.\\n[10:17:11] Total of 46 tRNA + rRNA features\\n[10:17:11] Searching for CRISPR repeats\\n[10:17:11] CRISPR1 NGZS01000104.1 34720 with 45 spacers\\n[10:17:11] CRISPR2 NGZS01000117.1 5 with 11 spacers\\n[10:17:11] CRISPR3 NGZS01000117.1 975 with 14 spacers\\n[10:17:11] CRISPR4 NGZS01000121.1 115568 with 11 spacers\\n[10:17:11] CRISPR5 NGZS01000122.1 142011 with 47 spacers\\n[10:17:11] CRISPR6 NGZS01000122.1 145660 with 6 spacers\\n[10:17:11] CRISPR7 NGZS01000123.1 26561 with 3 spacers\\n[10:17:11] CRISPR8 NGZS01000124.1 25240 with 83 spacers\\n[10:17:11] CRISPR9 NGZS01000128.1 2899 with 66 spacers\\n[10:17:11] Found 9 CRISPRs\\n[10:17:11] Predicting coding sequences\\n[10:17:11] Contigs total 5399690 bp, so using single mode\\n[10:17:11] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZS01\\\\/\\\\/Fischerella_NGZS01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:17:17] Excluding CDS which overlaps existing RNA (tRNA) at NGZS01000077.1:5051..5722 on - strand\\n[10:17:17] Excluding CDS which overlaps existing RNA (tRNA) at NGZS01000080.1:51759..53198 on + strand\\n[10:17:18] Excluding CDS which overlaps existing RNA (rRNA) at NGZS01000114.1:34002..34178 on + strand\\n[10:17:18] Excluding CDS which overlaps existing RNA (rRNA) at NGZS01000114.1:35964..36143 on - strand\\n[10:17:18] Excluding CDS which overlaps existing RNA (repeat_region) at NGZS01000124.1:26092..26352 on + strand\\n[10:17:18] Found 4621 CDS\\n[10:17:18] Connecting features back to sequences\\n[10:17:18] Not using genus-specific database. Try --usegenus to enable it.\\n[10:17:18] Annotating CDS, please be patient.\\n[10:17:18] Will use 4 CPUs for similarity searching.\\n[10:17:19] There are still 4621 unannotated CDS left (started with 4621)\\n[10:17:19] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:17:19] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZS01\\\\/\\\\/Fischerella_NGZS01\\\\.IS\\\\.tmp\\\\.99580\\\\.faa | parallel --gnu --plain -j 4 --block 180438 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZS01\\\\/\\\\/Fischerella_NGZS01\\\\.IS\\\\.tmp\\\\.99580\\\\.blast 2> /dev/null\\n[10:17:35] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZS01\\\\/\\\\/Fischerella_NGZS01\\\\.IS\\\\.tmp\\\\.99580\\\\.faa | parallel --gnu --plain -j 4 --block 180438 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZS01\\\\/\\\\/Fischerella_NGZS01\\\\.IS\\\\.tmp\\\\.99580\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:17:35] This is prokka 1.14.6\\n[10:17:35] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:17:35] Homepage is https://github.com/tseemann/prokka\\n[10:17:35] Local time is Mon Feb 21 10:17:35 2022\\n[10:17:35] You are tiagoferreiraleao\\n[10:17:35] Operating system is darwin\\n[10:17:35] You have BioPerl 1.007002\\n[10:17:36] System has 4 cores.\\n[10:17:36] Option --cpu asked for 12 cores, but system only has 4\\n[10:17:36] Will use maximum of 4 cores.\\n[10:17:36] Annotating as >>> Bacteria <<<\\n[10:17:36] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQE01.fasta' contents.\\n[10:17:36] Setting --locustag LCFJLOHB from MD5 5cf3581b3293c817858a5c5a3e141592\\n[10:17:36] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQE01/\\n[10:17:36] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQE01\\\\/\\n[10:17:36] Using filename prefix: Fischerella_NMQE01.XXX\\n[10:17:36] Setting HMMER_NCPU=1\\n[10:17:36] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQE01//Fischerella_NMQE01.log\\n[10:17:36] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQE01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQE01/ --prefix Fischerella_NMQE01 --cpus 12\\n[10:17:36] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:17:36] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:17:36] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:17:36] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:17:36] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:17:36] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:17:36] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:17:36] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:17:36] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:17:36] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:17:36] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:17:36] Looking for 'egrep' - found /usr/bin/egrep\\n[10:17:36] Looking for 'find' - found /usr/bin/find\\n[10:17:36] Looking for 'grep' - found /usr/bin/grep\\n[10:17:36] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:17:36] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:17:36] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:17:36] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:17:36] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:17:36] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:17:36] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:17:36] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:17:36] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:17:36] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:17:37] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:17:37] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:17:37] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:17:37] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:17:37] Looking for 'sed' - found /usr/bin/sed\\n[10:17:37] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:17:37] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:17:37] Using genetic code table 11.\\n[10:17:37] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQE01.fasta\\n[10:17:37] Wrote 999 contigs totalling 6197867 bp.\\n[10:17:37] Predicting tRNAs and tmRNAs\\n[10:17:37] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQE01\\\\/\\\\/Fischerella_NMQE01\\\\.fna\\n[10:17:39] 1 tmRNA [19222,19608] 100,153 ANNIVKFARKEALVAA**\\n[10:17:39] 1 tRNA-Asp c[10580,10653] 35 (gtc)\\n[10:17:39] 1 tRNA-Gly [5569,5640] 33 (gcc)\\n[10:17:39] 1 tRNA-Arg c[2616,2689] 35 (ccg)\\n[10:17:39] 1 tRNA-Gln c[5981,6052] 33 (ttg)\\n[10:17:39] 1 tRNA-Leu [268,351] 36 (tag)\\n[10:17:39] 1 tRNA-Thr [8875,8946] 33 (cgt)\\n[10:17:39] 2 tRNA-Ala [10262,10334] 34 (ggc)\\n[10:17:39] 1 tRNA-Gly [14355,14426] 33 (ccc)\\n[10:17:39] 1 tRNA-Pro [7013,7086] 35 (cgg)\\n[10:17:39] 1 tRNA-Arg [2227,2299] 34 (tcg)\\n[10:17:39] 2 tRNA-Asn [2695,2768] 34 (att)\\n[10:17:39] 1 tRNA-Ser c[1575,1663] 36 (gga)\\n[10:17:39] 1 tRNA-Ser [9858,9944] 36 (tga)\\n[10:17:39] 1 tRNA-Cys c[9304,9375] 33 (gca)\\n[10:17:39] 1 tRNA-Glu [5879,5951] 35 (ttc)\\n[10:17:39] 1 tRNA-Val c[8269,8342] 35 (gac)\\n[10:17:39] 2 tRNA-Met [11313,11389] 35 (cat)\\n[10:17:39] 1 tRNA-Arg c[8355,8428] 35 (acg)\\n[10:17:39] 1 tRNA-Met c[3853,3925] 34 (cat)\\n[10:17:39] 1 tRNA-Leu c[3447,3529] 36 (gag)\\n[10:17:39] 1 tRNA-Ser c[10664,10757] 36 (gct)\\n[10:17:39] 1 tRNA-Met [10253,10328] 36 (cat)\\n[10:17:39] 1 tRNA-Ser [4319,4405] 36 (cga)\\n[10:17:40] 1 tRNA-Val c[4443,4515] 34 (tac)\\n[10:17:40] 1 tRNA-Gly c[6175,6245] 33 (tcc)\\n[10:17:40] 1 tRNA-Thr c[8806,8877] 33 (tgt)\\n[10:17:40] 1 tRNA-Phe c[1318,1390] 34 (gaa)\\n[10:17:40] 1 tRNA-Asn c[6461,6534] 34 (gtt)\\n[10:17:40] 1 tRNA-Lys c[20673,20746] 34 (ttt)\\n[10:17:40] 1 tRNA-Pro [644,721] 36 (ggg)\\n[10:17:40] 2 tRNA-Arg [725,801] 35 (tcg)\\n[10:17:40] 1 tRNA-His c[250,323] 34 (gtg)\\n[10:17:40] 1 tRNA-Leu c[4745,4829] 35 (caa)\\n[10:17:40] 1 tRNA-Thr c[2794,2865] 33 (ggt)\\n[10:17:40] 2 tRNA-Tyr c[2930,3014] 35 (gta)\\n[10:17:40] 1 tRNA-Trp [1922,1995] 34 (cca)\\n[10:17:40] 1 tRNA-Pro [514,587] 35 (ggg)\\n[10:17:40] 1 tRNA-Leu c[11168,11250] 35 (cag)\\n[10:17:40] 1 tRNA-Arg c[6518,6592] 35 (tct)\\n[10:17:40] 1 tRNA-Ile [70,145] 35 (gat)\\n[10:17:40] 2 tRNA-Ala [233,307] 34 (tgc)\\n[10:17:40] 1 tRNA-Pro c[697,771] 35 (tgg)\\n[10:17:40] 1 tRNA-Ala [3275,3348] 34 (cgc)\\n[10:17:40] Found 44 tRNAs\\n[10:17:40] Predicting Ribosomal RNAs\\n[10:17:40] Running Barrnap with 4 threads\\n[10:17:41] 1 NMQE01000142.1 80 16S ribosomal RNA (partial)\\n[10:17:41] 2 NMQE01000261.1 94 23S ribosomal RNA\\n[10:17:41] 3 NMQE01000887.1 420 5S ribosomal RNA\\n[10:17:41] 4 NMQE01000938.1 5 16S ribosomal RNA (partial)\\n[10:17:41] 5 NMQE01000987.1 1 23S ribosomal RNA (partial)\\n[10:17:41] Found 5 rRNAs\\n[10:17:41] Skipping ncRNA search, enable with --rfam if desired.\\n[10:17:41] Total of 48 tRNA + rRNA features\\n[10:17:41] Searching for CRISPR repeats\\n[10:17:42] CRISPR1 NMQE01000052.1 7061 with 9 spacers\\n[10:17:42] CRISPR2 NMQE01000366.1 1706 with 13 spacers\\n[10:17:42] CRISPR3 NMQE01000434.1 5885 with 9 spacers\\n[10:17:42] CRISPR4 NMQE01000586.1 3 with 6 spacers\\n[10:17:42] CRISPR5 NMQE01000660.1 37 with 5 spacers\\n[10:17:42] CRISPR6 NMQE01000705.1 9771 with 22 spacers\\n[10:17:42] CRISPR7 NMQE01000935.1 42 with 10 spacers\\n[10:17:42] Found 7 CRISPRs\\n[10:17:42] Predicting coding sequences\\n[10:17:42] Contigs total 6197867 bp, so using single mode\\n[10:17:42] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQE01\\\\/\\\\/Fischerella_NMQE01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:17:47] Excluding CDS which overlaps existing RNA (tRNA) at NMQE01000209.1:2041..2328 on + strand\\n[10:17:48] Excluding CDS which overlaps existing RNA (rRNA) at NMQE01000261.1:380..556 on + strand\\n[10:17:48] Excluding CDS which overlaps existing RNA (rRNA) at NMQE01000261.1:2346..2525 on - strand\\n[10:17:48] Excluding CDS which overlaps existing RNA (tRNA) at NMQE01000296.1:9131..9322 on + strand\\n[10:17:48] Excluding CDS which overlaps existing RNA (tRNA) at NMQE01000350.1:8312..8983 on + strand\\n[10:17:48] Excluding CDS which overlaps existing RNA (tRNA) at NMQE01000616.1:5221..6519 on - strand\\n[10:17:49] Excluding CDS which overlaps existing RNA (repeat_region) at NMQE01000935.1:712..1314 on + strand\\n[10:17:49] Found 5073 CDS\\n[10:17:49] Connecting features back to sequences\\n[10:17:49] Not using genus-specific database. Try --usegenus to enable it.\\n[10:17:49] Annotating CDS, please be patient.\\n[10:17:49] Will use 4 CPUs for similarity searching.\\n[10:17:50] There are still 5073 unannotated CDS left (started with 5073)\\n[10:17:50] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:17:50] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQE01\\\\/\\\\/Fischerella_NMQE01\\\\.IS\\\\.tmp\\\\.99694\\\\.faa | parallel --gnu --plain -j 4 --block 201479 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQE01\\\\/\\\\/Fischerella_NMQE01\\\\.IS\\\\.tmp\\\\.99694\\\\.blast 2> /dev/null\\n[10:18:04] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQE01\\\\/\\\\/Fischerella_NMQE01\\\\.IS\\\\.tmp\\\\.99694\\\\.faa | parallel --gnu --plain -j 4 --block 201479 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQE01\\\\/\\\\/Fischerella_NMQE01\\\\.IS\\\\.tmp\\\\.99694\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:18:05] This is prokka 1.14.6\\n[10:18:05] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:18:05] Homepage is https://github.com/tseemann/prokka\\n[10:18:05] Local time is Mon Feb 21 10:18:05 2022\\n[10:18:05] You are tiagoferreiraleao\\n[10:18:05] Operating system is darwin\\n[10:18:05] You have BioPerl 1.007002\\n[10:18:05] System has 4 cores.\\n[10:18:05] Option --cpu asked for 12 cores, but system only has 4\\n[10:18:05] Will use maximum of 4 cores.\\n[10:18:05] Annotating as >>> Bacteria <<<\\n[10:18:05] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NQMD01.fasta' contents.\\n[10:18:05] Setting --locustag MMKIJDAE from MD5 66423dae2407a444470ccb93e563be2e\\n[10:18:05] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NQMD01/\\n[10:18:05] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQMD01\\\\/\\n[10:18:05] Using filename prefix: Fischerella_NQMD01.XXX\\n[10:18:05] Setting HMMER_NCPU=1\\n[10:18:05] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NQMD01//Fischerella_NQMD01.log\\n[10:18:05] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NQMD01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NQMD01/ --prefix Fischerella_NQMD01 --cpus 12\\n[10:18:05] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:18:05] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:18:05] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:18:05] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:18:05] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:18:05] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:18:05] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:18:05] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:18:05] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:18:05] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:18:05] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:18:05] Looking for 'egrep' - found /usr/bin/egrep\\n[10:18:05] Looking for 'find' - found /usr/bin/find\\n[10:18:05] Looking for 'grep' - found /usr/bin/grep\\n[10:18:05] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:18:05] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:18:05] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:18:05] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:18:05] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:18:05] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:18:05] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:18:05] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:18:06] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:18:06] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:18:06] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:18:06] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:18:06] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:18:06] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:18:06] Looking for 'sed' - found /usr/bin/sed\\n[10:18:06] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:18:06] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:18:06] Using genetic code table 11.\\n[10:18:06] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NQMD01.fasta\\n[10:18:07] Wrote 1820 contigs totalling 4995964 bp.\\n[10:18:07] Predicting tRNAs and tmRNAs\\n[10:18:07] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQMD01\\\\/\\\\/Fischerella_NQMD01\\\\.fna\\n[10:18:07] 1 tRNA-Gly [383,454] 33 (gcc)\\n[10:18:07] 1 tRNA-Leu [8683,8765] 36 (gag)\\n[10:18:07] 1 tRNA-Ser c[484,572] 36 (gga)\\n[10:18:07] 1 tRNA-Asp c[3388,3461] 35 (gtc)\\n[10:18:07] 1 tRNA-Thr c[2468,2539] 33 (tgt)\\n[10:18:07] 1 tRNA-Arg c[6060,6133] 35 (ccg)\\n[10:18:07] 1 tRNA-Pro c[309,383] 35 (tgg)\\n[10:18:07] 1 tRNA-Ser [3416,3509] 36 (gct)\\n[10:18:08] 1 tRNA-Arg c[188,263] 36 (cct)\\n[10:18:08] 1 tRNA-Met c[3364,3439] 36 (cat)\\n[10:18:08] 1 tRNA-Arg [1500,1573] 35 (acg)\\n[10:18:08] 1 tRNA-Trp [648,721] 34 (cca)\\n[10:18:08] 1 tRNA-Leu c[578,661] 36 (tag)\\n[10:18:08] 1 tRNA-Lys [6652,6725] 34 (ttt)\\n[10:18:08] 1 tRNA-Cys [143,214] 33 (gca)\\n[10:18:08] 1 tRNA-Thr [2580,2651] 33 (cgt)\\n[10:18:08] 1 tmRNA c[33,419] 100,150 ANNIVKFARKEALVAA*\\n[10:18:08] 1 tRNA-Arg [824,896] 34 (tcg)\\n[10:18:08] 2 tRNA-Asn [1622,1695] 34 (att)\\n[10:18:08] 1 tRNA-Gly [5779,5850] 33 (ccc)\\n[10:18:08] 1 tRNA-Gly [6481,6551] 33 (tcc)\\n[10:18:08] 1 tRNA-Ala c[2458,2531] 34 (ggc)\\n[10:18:08] 1 tRNA-Glu [5738,5810] 35 (ttc)\\n[10:18:08] 1 tRNA-His [719,792] 34 (gtg)\\n[10:18:08] 1 tRNA-Ser c[37,122] 35 (tga)\\n[10:18:08] 1 tRNA-Met [3111,3183] 34 (cat)\\n[10:18:08] 1 tRNA-Pro [1069,1142] 35 (cgg)\\n[10:18:08] 1 tRNA-Pro [1464,1537] 35 (ggg)\\n[10:18:08] 1 tRNA-Asn [1071,1144] 34 (gtt)\\n[10:18:08] 1 tRNA-Val [1364,1436] 34 (tac)\\n[10:18:08] 1 tRNA-Arg [1039,1113] 35 (tct)\\n[10:18:08] 1 tRNA-Gln c[3415,3486] 33 (ttg)\\n[10:18:08] 1 tRNA-Leu [519,603] 35 (caa)\\n[10:18:08] 1 tRNA-Leu [8129,8211] 35 (cag)\\n[10:18:08] tRNA [7066,7137] is a pseudo/wacky gene - skipping.\\n[10:18:08] 1 tRNA-Val c[3762,3835] 35 (gac)\\n[10:18:08] 2 tRNA-Met [6806,6891] 35 (cat)\\n[10:18:08] Found 36 tRNAs\\n[10:18:08] Predicting Ribosomal RNAs\\n[10:18:08] Running Barrnap with 4 threads\\n[10:18:09] Found 0 rRNAs\\n[10:18:09] Skipping ncRNA search, enable with --rfam if desired.\\n[10:18:09] Total of 35 tRNA + rRNA features\\n[10:18:09] Searching for CRISPR repeats\\n[10:18:10] Found 0 CRISPRs\\n[10:18:10] Predicting coding sequences\\n[10:18:10] Contigs total 4995964 bp, so using single mode\\n[10:18:10] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQMD01\\\\/\\\\/Fischerella_NQMD01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:18:14] Excluding CDS which overlaps existing RNA (tRNA) at NQMD01000790.1:945..1616 on - strand\\n[10:18:14] Excluding CDS which overlaps existing RNA (tRNA) at NQMD01000950.1:638..925 on + strand\\n[10:18:15] Excluding CDS which overlaps existing RNA (tRNA) at NQMD01001603.1:945..2384 on + strand\\n[10:18:15] Excluding CDS which overlaps existing RNA (tRNA) at NQMD01001674.1:8081..8224 on + strand\\n[10:18:15] Found 4262 CDS\\n[10:18:15] Connecting features back to sequences\\n[10:18:15] Not using genus-specific database. Try --usegenus to enable it.\\n[10:18:15] Annotating CDS, please be patient.\\n[10:18:15] Will use 4 CPUs for similarity searching.\\n[10:18:16] There are still 4262 unannotated CDS left (started with 4262)\\n[10:18:16] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:18:16] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQMD01\\\\/\\\\/Fischerella_NQMD01\\\\.IS\\\\.tmp\\\\.99806\\\\.faa | parallel --gnu --plain -j 4 --block 152074 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQMD01\\\\/\\\\/Fischerella_NQMD01\\\\.IS\\\\.tmp\\\\.99806\\\\.blast 2> /dev/null\\n[10:18:29] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQMD01\\\\/\\\\/Fischerella_NQMD01\\\\.IS\\\\.tmp\\\\.99806\\\\.faa | parallel --gnu --plain -j 4 --block 152074 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NQMD01\\\\/\\\\/Fischerella_NQMD01\\\\.IS\\\\.tmp\\\\.99806\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:18:29] This is prokka 1.14.6\\n[10:18:29] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:18:29] Homepage is https://github.com/tseemann/prokka\\n[10:18:29] Local time is Mon Feb 21 10:18:29 2022\\n[10:18:29] You are tiagoferreiraleao\\n[10:18:29] Operating system is darwin\\n[10:18:29] You have BioPerl 1.007002\\n[10:18:29] System has 4 cores.\\n[10:18:29] Option --cpu asked for 12 cores, but system only has 4\\n[10:18:29] Will use maximum of 4 cores.\\n[10:18:29] Annotating as >>> Bacteria <<<\\n[10:18:29] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_AJLJ01.fasta' contents.\\n[10:18:29] Setting --locustag INMOIMCP from MD5 276826c9449faa15b6d6ed1896dac067\\n[10:18:29] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_AJLJ01/\\n[10:18:29] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLJ01\\\\/\\n[10:18:29] Using filename prefix: Fischerella_AJLJ01.XXX\\n[10:18:29] Setting HMMER_NCPU=1\\n[10:18:29] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_AJLJ01//Fischerella_AJLJ01.log\\n[10:18:29] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_AJLJ01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_AJLJ01/ --prefix Fischerella_AJLJ01 --cpus 12\\n[10:18:29] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:18:29] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:18:29] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:18:29] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:18:29] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:18:29] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:18:30] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:18:30] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:18:30] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:18:30] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:18:30] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:18:30] Looking for 'egrep' - found /usr/bin/egrep\\n[10:18:30] Looking for 'find' - found /usr/bin/find\\n[10:18:30] Looking for 'grep' - found /usr/bin/grep\\n[10:18:30] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:18:30] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:18:30] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:18:30] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:18:30] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:18:30] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:18:30] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:18:30] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:18:30] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:18:30] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:18:30] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:18:30] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:18:30] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:18:30] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:18:30] Looking for 'sed' - found /usr/bin/sed\\n[10:18:30] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:18:31] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:18:31] Using genetic code table 11.\\n[10:18:31] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_AJLJ01.fasta\\n[10:18:31] Wrote 524 contigs totalling 7360256 bp.\\n[10:18:31] Predicting tRNAs and tmRNAs\\n[10:18:31] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLJ01\\\\/\\\\/Fischerella_AJLJ01\\\\.fna\\n[10:18:34] 1 tRNA-Ala c[60,134] 34 (tgc)\\n[10:18:34] 2 tRNA-Ile c[222,297] 35 (gat)\\n[10:18:34] 1 tRNA-Leu c[5379,5462] 36 (tag)\\n[10:18:34] 1 tRNA-Asn c[100274,100347] 34 (gtt)\\n[10:18:34] 1 tRNA-Leu c[80440,80522] 35 (cag)\\n[10:18:34] 1 tRNA-Thr [66807,66878] 33 (cgt)\\n[10:18:34] 1 tRNA-Arg [59571,59645] 35 (tct)\\n[10:18:34] 1 tRNA-Ile c[108398,108471] 34 (tat)\\n[10:18:34] 2 tRNA-Cys c[109185,109258] 35 (gca)\\n[10:18:34] 1 tRNA-Ala [3639,3713] 35 (cgc)\\n[10:18:34] 1 tRNA-Lys [20485,20556] 33 (ttt)\\n[10:18:34] 1 tRNA-Trp [7367,7440] 34 (cca)\\n[10:18:34] 1 tRNA-Gly c[4177,4248] 33 (gcc)\\n[10:18:34] 1 tRNA-Gly [4671,4741] 33 (tcc)\\n[10:18:34] 1 tRNA-Thr c[13473,13544] 33 (tgt)\\n[10:18:34] 1 tRNA-His [1748,1821] 34 (gtg)\\n[10:18:34] 1 tRNA-Ser c[2842,2930] 36 (gga)\\n[10:18:34] 1 tRNA-Ser [7998,8091] 36 (gct)\\n[10:18:34] 2 tRNA-Arg c[17703,17776] 35 (acg)\\n[10:18:34] 3 tRNA-Ala c[34130,34202] 34 (ggc)\\n[10:18:34] 1 tRNA-Pro c[1519,1592] 35 (ggg)\\n[10:18:34] 1 tRNA-Glu [9533,9605] 35 (ttc)\\n[10:18:34] 1 tRNA-Leu [2025,2109] 35 (caa)\\n[10:18:34] 2 tRNA-Gln [38526,38597] 33 (ttg)\\n[10:18:34] 1 tRNA-Tyr [23359,23443] 35 (gta)\\n[10:18:34] 2 tRNA-Thr [23521,23592] 33 (ggt)\\n[10:18:34] 1 tRNA-Met c[18094,18170] 35 (cat)\\n[10:18:34] 2 tRNA-Val [21215,21288] 35 (gac)\\n[10:18:34] 1 tRNA-Lys [35346,35418] 33 (ctt)\\n[10:18:34] 1 tRNA-Leu [1809,1891] 36 (gag)\\n[10:18:34] 1 tRNA-Met c[3910,3984] 35 (cat)\\n[10:18:34] 1 tRNA-Met c[6541,6614] 34 (cat)\\n[10:18:34] 1 tRNA-Trp [35052,35126] 35 (cca)\\n[10:18:34] 2 tRNA-Trp [37352,37426] 35 (cca)\\n[10:18:34] 1 tRNA-Pro c[48,121] 35 (tgg)\\n[10:18:34] 1 tRNA-Ser c[885,970] 35 (tga)\\n[10:18:34] 1 tRNA-Cys c[6665,6736] 33 (gca)\\n[10:18:34] 1 tRNA-Arg [13541,13613] 34 (ccg)\\n[10:18:34] 1 tRNA-Gly c[5391,5462] 33 (ccc)\\n[10:18:34] 1 tmRNA [34901,35288] 101,154 ANNIVKFARKEALVAA**\\n[10:18:34] 1 tRNA-Val c[54463,54535] 34 (tac)\\n[10:18:34] 1 tRNA-Asp [84667,84740] 35 (gtc)\\n[10:18:34] 1 tRNA-Pro [54437,54510] 35 (cgg)\\n[10:18:34] 1 tRNA-Ser c[2316,2402] 36 (cga)\\n[10:18:34] 1 tRNA-Phe c[8570,8642] 34 (gaa)\\n[10:18:34] Found 45 tRNAs\\n[10:18:34] Predicting Ribosomal RNAs\\n[10:18:34] Running Barrnap with 4 threads\\n[10:18:35] 1 AJLJ01000084.1 304 23S ribosomal RNA\\n[10:18:35] 2 AJLJ01000085.1 83 16S ribosomal RNA\\n[10:18:35] Found 2 rRNAs\\n[10:18:35] Skipping ncRNA search, enable with --rfam if desired.\\n[10:18:35] Total of 46 tRNA + rRNA features\\n[10:18:35] Searching for CRISPR repeats\\n[10:18:36] CRISPR1 AJLJ01000072.1 33630 with 7 spacers\\n[10:18:36] CRISPR2 AJLJ01000079.1 9012 with 7 spacers\\n[10:18:36] CRISPR3 AJLJ01000126.1 10003 with 10 spacers\\n[10:18:36] CRISPR4 AJLJ01000177.1 3435 with 13 spacers\\n[10:18:36] CRISPR5 AJLJ01000363.1 24680 with 15 spacers\\n[10:18:36] CRISPR6 AJLJ01000522.1 12285 with 6 spacers\\n[10:18:36] CRISPR7 AJLJ01000523.1 646 with 11 spacers\\n[10:18:36] Found 7 CRISPRs\\n[10:18:36] Predicting coding sequences\\n[10:18:36] Contigs total 7360256 bp, so using single mode\\n[10:18:36] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLJ01\\\\/\\\\/Fischerella_AJLJ01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:18:42] Excluding CDS which overlaps existing RNA (rRNA) at AJLJ01000084.1:590..766 on + strand\\n[10:18:43] Excluding CDS which overlaps existing RNA (tRNA) at AJLJ01000190.1:33932..34138 on + strand\\n[10:18:44] Excluding CDS which overlaps existing RNA (tRNA) at AJLJ01000237.1:35028..35177 on - strand\\n[10:18:45] Found 6254 CDS\\n[10:18:45] Connecting features back to sequences\\n[10:18:45] Not using genus-specific database. Try --usegenus to enable it.\\n[10:18:45] Annotating CDS, please be patient.\\n[10:18:45] Will use 4 CPUs for similarity searching.\\n[10:18:46] There are still 6254 unannotated CDS left (started with 6254)\\n[10:18:46] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:18:46] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLJ01\\\\/\\\\/Fischerella_AJLJ01\\\\.IS\\\\.tmp\\\\.99918\\\\.faa | parallel --gnu --plain -j 4 --block 245246 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLJ01\\\\/\\\\/Fischerella_AJLJ01\\\\.IS\\\\.tmp\\\\.99918\\\\.blast 2> /dev/null\\n[10:18:50] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLJ01\\\\/\\\\/Fischerella_AJLJ01\\\\.IS\\\\.tmp\\\\.99918\\\\.faa | parallel --gnu --plain -j 4 --block 245246 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLJ01\\\\/\\\\/Fischerella_AJLJ01\\\\.IS\\\\.tmp\\\\.99918\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:18:51] This is prokka 1.14.6\\n[10:18:51] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:18:51] Homepage is https://github.com/tseemann/prokka\\n[10:18:51] Local time is Mon Feb 21 10:18:51 2022\\n[10:18:51] You are tiagoferreiraleao\\n[10:18:51] Operating system is darwin\\n[10:18:51] You have BioPerl 1.007002\\n[10:18:51] System has 4 cores.\\n[10:18:51] Option --cpu asked for 12 cores, but system only has 4\\n[10:18:51] Will use maximum of 4 cores.\\n[10:18:51] Annotating as >>> Bacteria <<<\\n[10:18:51] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACKZU01.fasta' contents.\\n[10:18:51] Setting --locustag BNMAGGBH from MD5 b76a00b17ffe223fe7e66dde391a3cc8\\n[10:18:51] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACKZU01/\\n[10:18:51] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZU01\\\\/\\n[10:18:51] Using filename prefix: Nostoc_JACKZU01.XXX\\n[10:18:51] Setting HMMER_NCPU=1\\n[10:18:51] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACKZU01//Nostoc_JACKZU01.log\\n[10:18:51] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACKZU01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACKZU01/ --prefix Nostoc_JACKZU01 --cpus 12\\n[10:18:51] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:18:51] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:18:51] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:18:51] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:18:51] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:18:51] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:18:51] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:18:51] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:18:51] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:18:51] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:18:51] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:18:51] Looking for 'egrep' - found /usr/bin/egrep\\n[10:18:51] Looking for 'find' - found /usr/bin/find\\n[10:18:51] Looking for 'grep' - found /usr/bin/grep\\n[10:18:51] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:18:51] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:18:51] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:18:51] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:18:51] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:18:51] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:18:52] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:18:52] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:18:52] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:18:52] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:18:52] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:18:52] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:18:52] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:18:52] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:18:52] Looking for 'sed' - found /usr/bin/sed\\n[10:18:52] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:18:52] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:18:52] Using genetic code table 11.\\n[10:18:52] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACKZU01.fasta\\n[10:18:53] Wrote 869 contigs totalling 8684590 bp.\\n[10:18:53] Predicting tRNAs and tmRNAs\\n[10:18:53] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZU01\\\\/\\\\/Nostoc_JACKZU01\\\\.fna\\n[10:18:55] 1 tRNA-Lys [53075,53146] 33 (ttt)\\n[10:18:55] 1 tRNA-Met c[48943,49017] 35 (cat)\\n[10:18:55] 1 tRNA-Met [7709,7784] 34 (cat)\\n[10:18:55] 2 tRNA-Met [7790,7865] 34 (cat)\\n[10:18:55] 3 tRNA-Ile [9471,9546] 34 (tat)\\n[10:18:55] 1 tRNA-Cys [27238,27309] 33 (gca)\\n[10:18:55] 1 tRNA-Met [17000,17073] 33 (cat)\\n[10:18:55] 2 tRNA-Gly [24091,24162] 33 (gcc)\\n[10:18:55] 1 tRNA-Ala c[8819,8891] 34 (ggc)\\n[10:18:55] 1 tRNA-Pro [15697,15770] 35 (ggg)\\n[10:18:55] 1 tRNA-Leu [4363,4443] 35 (gag)\\n[10:18:55] 1 tRNA-Arg [11512,11586] 35 (tct)\\n[10:18:55] 1 tRNA-Thr [21267,21339] 33 (tgt)\\n[10:18:55] 2 tRNA-Thr c[22260,22331] 33 (ggt)\\n[10:18:55] 3 tRNA-Tyr c[22407,22491] 35 (gta)\\n[10:18:55] 4 tRNA-Ser [24942,25028] 36 (tga)\\n[10:18:55] 1 tRNA-Asn c[23981,24054] 34 (gtt)\\n[10:18:55] 1 tRNA-His [10842,10915] 34 (gtg)\\n[10:18:55] 1 tRNA-Phe [9693,9765] 34 (gaa)\\n[10:18:55] 1 tRNA-Gly [22273,22346] 34 (ccc)\\n[10:18:55] 1 tmRNA c[1464,1851] 101,151 ANNIVKFARREALVAA*\\n[10:18:55] 1 tRNA-Ala [9499,9571] 34 (tgc)\\n[10:18:55] 1 tRNA-Ser c[13637,13730] 36 (gct)\\n[10:18:55] 1 tRNA-Leu c[1468,1551] 36 (tag)\\n[10:18:55] 1 tRNA-Thr [9277,9348] 33 (cgt)\\n[10:18:55] 1 tRNA-Thr [12550,12624] 33 (agt)\\n[10:18:55] 2 tRNA-Leu [14015,14089] 35 (caa)\\n[10:18:55] 1 tRNA-Leu [10487,10568] 35 (cag)\\n[10:18:55] 1 tRNA-Arg c[9208,9280] 34 (ccg)\\n[10:18:55] 1 tRNA-Val [12642,12715] 35 (gac)\\n[10:18:55] 1 tRNA-Ser [4606,4692] 36 (cga)\\n[10:18:55] 1 tRNA-Gly c[8293,8363] 33 (tcc)\\n[10:18:55] 1 tRNA-Gln c[4544,4617] 34 (ttg)\\n[10:18:55] 2 tRNA-Leu c[9638,9722] 35 (caa)\\n[10:18:55] 1 tRNA-Ser [4553,4640] 35 (gga)\\n[10:18:55] 1 tRNA-Asp [3944,4021] 37 (gtc)\\n[10:18:55] 2 tRNA-Glu [4026,4102] 35 (ctc)\\n[10:18:55] 3 tRNA-Cys [4108,4182] 33 (gca)\\n[10:18:55] 4 tRNA-Met [4187,4261] 34 (cat)\\n[10:18:55] 5 tRNA-Trp [4265,4338] 35 (cca)\\n[10:18:55] 6 tRNA-Glu [4348,4422] 35 (ttc)\\n[10:18:55] 7 tRNA-Tyr [4431,4515] 36 (gta)\\n[10:18:55] 8 tRNA-Ser [4518,4607] 33 (gct)\\n[10:18:55] 9 tRNA-Arg [4609,4684] 35 (acg)\\n[10:18:55] 10 tRNA-Phe [4689,4766] 38 (gaa)\\n[10:18:55] 11 tRNA-Asn [4775,4849] 33 (gtt)\\n[10:18:55] 12 tRNA-Gln [4855,4931] 35 (ctg)\\n[10:18:55] 13 tRNA-Gln [4937,5012] 35 (ttg)\\n[10:18:55] 14 tRNA-Leu [5016,5091] 35 (caa)\\n[10:18:55] 15 tRNA-Pro [5095,5168] 35 (tgg)\\n[10:18:55] 16 tRNA-Leu [5175,5249] 35 (tag)\\n[10:18:55] 17 tmRNA* [5318,5926] 450,518 ATPEYAKLQMRVQLSSSAFIAL*\\n[10:18:55] tRNA/tmRNA [5318,5926] is too big (>500bp) - skipping.\\n[10:18:55] 18 tRNA-Lys [5336,5410] 33 (ttt)\\n[10:18:55] 19 tRNA-Leu [5416,5492] 36 (taa)\\n[10:18:55] 20 tRNA-Met [5492,5567] 35 (cat)\\n[10:18:55] 21 tRNA-Thr [5573,5646] 35 (tgt)\\n[10:18:55] 22 tRNA-Gly [5655,5727] 34 (tcc)\\n[10:18:55] 23 tRNA-Gly [5750,5820] 33 (gcc)\\n[10:18:55] 24 tRNA-Arg [5908,5982] 33 (tct)\\n[10:18:55] 25 tRNA-Ala [5986,6061] 36 (tgc)\\n[10:18:55] 26 tRNA-His [6067,6142] 35 (gtg)\\n[10:18:55] 27 tRNA-Ser [6148,6232] 34 (tga)\\n[10:18:55] 1 tRNA-Trp c[4030,4103] 34 (cca)\\n[10:18:55] 1 tRNA-Pro [6769,6842] 35 (cgg)\\n[10:18:55] 1 tRNA-Arg c[7418,7491] 35 (acg)\\n[10:18:55] 1 tRNA-Lys c[1173,1243] 33 (ttt)\\n[10:18:55] 2 tRNA-Ser c[1515,1598] 34 (tga)\\n[10:18:55] 3 tRNA-Arg c[1604,1678] 34 (tct)\\n[10:18:55] 4 tRNA-Val c[1686,1761] 34 (tac)\\n[10:18:55] 5 tRNA-Thr c[1942,2014] 34 (tgt)\\n[10:18:55] 6 tRNA-Met c[2023,2099] 34 (cat)\\n[10:18:55] 7 tRNA-Leu c[2109,2183] 35 (tag)\\n[10:18:55] 8 tRNA-Leu c[2295,2371] 34 (gag)\\n[10:18:55] 9 tRNA-Pro c[2475,2547] 34 (tgg)\\n[10:18:55] 10 tRNA-Leu c[2549,2626] 35 (taa)\\n[10:18:56] 11 tRNA-Leu c[2633,2709] 35 (caa)\\n[10:18:56] 12 tRNA-Gln c[2736,2811] 35 (ttg)\\n[10:18:56] 13 tRNA-Gln c[3686,3760] 35 (ctg)\\n[10:18:56] 14 tRNA-Asn c[3771,3845] 33 (gtt)\\n[10:18:56] 15 tRNA-Arg c[4030,4106] 36 (acg)\\n[10:18:56] 16 tRNA-Ser c[4108,4194] 35 (gct)\\n[10:18:56] 17 tRNA-Tyr c[4198,4279] 35 (gta)\\n[10:18:56] 18 tRNA-Glu c[4290,4364] 35 (ttc)\\n[10:18:56] tRNA c[4452,4526] is a pseudo/wacky gene - skipping.\\n[10:18:56] 20 tRNA-Met c[4528,4601] 32 (cat)\\n[10:18:56] 21 tRNA-Cys c[4608,4681] 35 (gca)\\n[10:18:56] 22 tRNA-Glu c[4685,4761] 35 (ctc)\\n[10:18:56] 1 tRNA-Lys c[3396,3468] 33 (ctt)\\n[10:18:56] 1 tRNA-Pro [3534,3607] 35 (tgg)\\n[10:18:56] 1 tRNA-Glu [3024,3097] 35 (ttc)\\n[10:18:56] 1 tRNA-Val c[100,172] 34 (tac)\\n[10:18:56] 1 tRNA-Ala c[755,827] 34 (cgc)\\n[10:18:56] 1 tRNA-Asp c[2344,2417] 35 (gtc)\\n[10:18:56] 1 tRNA-Ile [123,198] 35 (gat)\\n[10:18:56] 2 tRNA-Ala [282,356] 34 (tgc)\\n[10:18:56] 1 tRNA-Pro c[17,91] 35 (tgg)\\n[10:18:56] 1 tRNA-Gly c[145,218] 33 (tcc)\\n[10:18:56] Found 95 tRNAs\\n[10:18:56] Predicting Ribosomal RNAs\\n[10:18:56] Running Barrnap with 4 threads\\n[10:18:58] 1 JACKZU010000572.1 268 23S ribosomal RNA\\n[10:18:58] 2 JACKZU010000682.1 77 16S ribosomal RNA\\n[10:18:58] 3 JACKZU010000807.1 24 5S ribosomal RNA (partial)\\n[10:18:58] Found 3 rRNAs\\n[10:18:58] Skipping ncRNA search, enable with --rfam if desired.\\n[10:18:58] Total of 97 tRNA + rRNA features\\n[10:18:58] Searching for CRISPR repeats\\n[10:18:58] CRISPR1 JACKZU010000006.1 14830 with 14 spacers\\n[10:18:58] CRISPR2 JACKZU010000014.1 31057 with 18 spacers\\n[10:18:58] CRISPR3 JACKZU010000021.1 30225 with 23 spacers\\n[10:18:58] CRISPR4 JACKZU010000056.1 16139 with 6 spacers\\n[10:18:59] CRISPR5 JACKZU010000155.1 4564 with 9 spacers\\n[10:18:59] CRISPR6 JACKZU010000182.1 12023 with 21 spacers\\n[10:18:59] CRISPR7 JACKZU010000193.1 8658 with 6 spacers\\n[10:18:59] CRISPR8 JACKZU010000401.1 6161 with 10 spacers\\n[10:18:59] CRISPR9 JACKZU010000422.1 1919 with 11 spacers\\n[10:18:59] Found 9 CRISPRs\\n[10:18:59] Predicting coding sequences\\n[10:18:59] Contigs total 8684590 bp, so using single mode\\n[10:18:59] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZU01\\\\/\\\\/Nostoc_JACKZU01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:19:07] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZU010000021.1:29956..30387 on + strand\\n[10:19:09] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZU010000182.1:12747..12890 on + strand\\n[10:19:09] Excluding CDS which overlaps existing RNA (tRNA) at JACKZU010000342.1:4789..4974 on + strand\\n[10:19:09] Excluding CDS which overlaps existing RNA (tRNA) at JACKZU010000342.1:6122..6457 on - strand\\n[10:19:09] Excluding CDS which overlaps existing RNA (tRNA) at JACKZU010000372.1:2758..2904 on - strand\\n[10:19:09] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZU010000401.1:6703..6798 on - strand\\n[10:19:09] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZU010000401.1:6811..7026 on - strand\\n[10:19:09] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZU010000422.1:2706..3665 on - strand\\n[10:19:10] Found 6927 CDS\\n[10:19:10] Connecting features back to sequences\\n[10:19:10] Not using genus-specific database. Try --usegenus to enable it.\\n[10:19:10] Annotating CDS, please be patient.\\n[10:19:10] Will use 4 CPUs for similarity searching.\\n[10:19:11] There are still 6927 unannotated CDS left (started with 6927)\\n[10:19:11] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:19:11] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZU01\\\\/\\\\/Nostoc_JACKZU01\\\\.IS\\\\.tmp\\\\.142\\\\.faa | parallel --gnu --plain -j 4 --block 287446 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZU01\\\\/\\\\/Nostoc_JACKZU01\\\\.IS\\\\.tmp\\\\.142\\\\.blast 2> /dev/null\\n[10:19:29] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZU01\\\\/\\\\/Nostoc_JACKZU01\\\\.IS\\\\.tmp\\\\.142\\\\.faa | parallel --gnu --plain -j 4 --block 287446 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZU01\\\\/\\\\/Nostoc_JACKZU01\\\\.IS\\\\.tmp\\\\.142\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:19:30] This is prokka 1.14.6\\n[10:19:30] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:19:30] Homepage is https://github.com/tseemann/prokka\\n[10:19:30] Local time is Mon Feb 21 10:19:30 2022\\n[10:19:30] You are tiagoferreiraleao\\n[10:19:30] Operating system is darwin\\n[10:19:30] You have BioPerl 1.007002\\n[10:19:30] System has 4 cores.\\n[10:19:30] Option --cpu asked for 12 cores, but system only has 4\\n[10:19:30] Will use maximum of 4 cores.\\n[10:19:30] Annotating as >>> Bacteria <<<\\n[10:19:30] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHF01.fasta' contents.\\n[10:19:30] Setting --locustag EILCIFPB from MD5 e25c2f9b94a9064e60fccb9f75746506\\n[10:19:30] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHF01/\\n[10:19:30] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHF01\\\\/\\n[10:19:30] Using filename prefix: Nostoc_LAHF01.XXX\\n[10:19:30] Setting HMMER_NCPU=1\\n[10:19:30] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHF01//Nostoc_LAHF01.log\\n[10:19:30] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHF01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHF01/ --prefix Nostoc_LAHF01 --cpus 12\\n[10:19:30] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:19:30] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:19:30] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:19:30] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:19:30] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:19:30] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:19:30] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:19:30] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:19:30] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:19:30] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:19:31] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:19:31] Looking for 'egrep' - found /usr/bin/egrep\\n[10:19:31] Looking for 'find' - found /usr/bin/find\\n[10:19:31] Looking for 'grep' - found /usr/bin/grep\\n[10:19:31] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:19:31] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:19:31] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:19:31] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:19:31] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:19:31] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:19:31] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:19:31] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:19:31] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:19:31] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:19:31] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:19:31] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:19:31] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:19:31] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:19:31] Looking for 'sed' - found /usr/bin/sed\\n[10:19:31] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:19:32] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:19:32] Using genetic code table 11.\\n[10:19:32] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHF01.fasta\\n[10:19:32] Wrote 566 contigs totalling 9177463 bp.\\n[10:19:32] Predicting tRNAs and tmRNAs\\n[10:19:32] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHF01\\\\/\\\\/Nostoc_LAHF01\\\\.fna\\n[10:19:35] 1 tRNA-Pro c[100302,100375] 35 (tgg)\\n[10:19:35] 2 tRNA-Ser c[109057,109143] 36 (tga)\\n[10:19:35] 3 tRNA-Tyr [111594,111678] 35 (gta)\\n[10:19:35] 4 tRNA-Thr [111754,111825] 33 (ggt)\\n[10:19:35] 1 tRNA-Ala c[27517,27589] 34 (cgc)\\n[10:19:35] 2 tRNA-Leu [76291,76375] 35 (caa)\\n[10:19:35] 3 tRNA-Gln [81129,81202] 34 (ttg)\\n[10:19:35] 1 tRNA-Ile [61861,61936] 35 (gat)\\n[10:19:35] 2 tRNA-Ala [62020,62094] 34 (tgc)\\n[10:19:35] 1 tRNA-Ile c[31750,31825] 34 (tat)\\n[10:19:35] 2 tRNA-Met c[33431,33506] 34 (cat)\\n[10:19:35] 3 tRNA-Met c[33508,33583] 34 (cat)\\n[10:19:35] 1 tRNA-Pro [17632,17705] 35 (ggg)\\n[10:19:35] 1 tmRNA [18265,18652] 101,151 ANNIVKFARREALVAA*\\n[10:19:35] 1 tRNA-Lys [46341,46412] 33 (ttt)\\n[10:19:35] 1 tRNA-Gly c[18368,18439] 33 (gcc)\\n[10:19:35] 2 tRNA-Met c[25249,25322] 33 (cat)\\n[10:19:35] 1 tRNA-Asp [25364,25441] 38 (gtc)\\n[10:19:35] 2 tRNA-Glu [25444,25520] 35 (ctc)\\n[10:19:35] 3 tRNA-Cys [25525,25600] 34 (gca)\\n[10:19:35] 4 tRNA-Met [25606,25680] 33 (cat)\\n[10:19:35] 5 tRNA-Trp [25683,25755] 34 (cca)\\n[10:19:35] 6 tRNA-Pyl [25780,25851] 33 (cta)\\n[10:19:35] 7 tRNA-Glu [25882,25957] 34 (ttc)\\n[10:19:35] 8 tRNA-Tyr [25961,26044] 36 (gta)\\n[10:19:35] 9 tRNA-Ser [26048,26137] 33 (gct)\\n[10:19:35] 10 tRNA-Arg [26142,26217] 35 (acg)\\n[10:19:35] 11 tRNA-Phe [26222,26296] 35 (gaa)\\n[10:19:35] 12 tRNA-Asn [26305,26379] 33 (gtt)\\n[10:19:35] 13 tRNA-Gln [26385,26460] 35 (ttg)\\n[10:19:35] 14 tRNA-Leu [26468,26545] 35 (caa)\\n[10:19:35] 15 tRNA-Pro [26550,26626] 35 (tgg)\\n[10:19:35] 16 tRNA-Leu [26630,26704] 35 (tag)\\n[10:19:35] 17 tRNA-Leu [26724,26801] 35 (gag)\\n[10:19:35] 18 tRNA-Lys [26805,26881] 34 (ttt)\\n[10:19:35] 19 tRNA-Lys [26887,26961] 33 (ctt)\\n[10:19:35] 20 tRNA-Leu [26967,27042] 36 (taa)\\n[10:19:35] 21 tRNA-Cys [27052,27129] 34 (gca)\\n[10:19:35] 22 tRNA-Thr [27137,27209] 34 (tgt)\\n[10:19:35] 23 tRNA-Gly [27215,27285] 33 (tcc)\\n[10:19:35] 24 tRNA-Val [27291,27366] 34 (tac)\\n[10:19:35] 25 tRNA-Arg [27373,27447] 35 (tct)\\n[10:19:35] 26 tRNA-Ala [27452,27524] 34 (tgc)\\n[10:19:35] 27 tRNA-His [27532,27608] 35 (gtg)\\n[10:19:35] 28 tRNA-Ser [27619,27703] 34 (tga)\\n[10:19:35] 1 tRNA-Ser [24235,24321] 36 (cga)\\n[10:19:35] 1 tRNA-Thr [20576,20650] 33 (agt)\\n[10:19:35] 2 tRNA-Leu [22041,22115] 35 (caa)\\n[10:19:35] 1 tRNA-Ala c[1013,1085] 34 (ggc)\\n[10:19:35] 1 tRNA-His c[22957,23030] 34 (gtg)\\n[10:19:35] 1 tRNA-Ser c[7684,7777] 36 (gct)\\n[10:19:35] 2 tRNA-Lys c[13452,13522] 33 (ttt)\\n[10:19:35] 3 tRNA-Ser c[13781,13864] 34 (tga)\\n[10:19:35] 4 tRNA-His c[13869,13944] 35 (gtg)\\n[10:19:35] 5 tRNA-Ala c[13950,14024] 35 (tgc)\\n[10:19:35] 6 tRNA-Arg c[14116,14191] 35 (tct)\\n[10:19:35] 7 tRNA-Val c[14198,14273] 34 (tac)\\n[10:19:35] 8 tRNA-Thr c[14355,14427] 34 (tgt)\\n[10:19:35] 9 tRNA-Leu c[14722,14796] 35 (tag)\\n[10:19:35] 10 tRNA-Leu c[14908,14984] 34 (gag)\\n[10:19:35] 11 tRNA-Pro c[15092,15165] 35 (tgg)\\n[10:19:35] 12 tRNA-Leu c[15167,15244] 35 (taa)\\n[10:19:35] 13 tRNA-Leu c[15265,15342] 36 (caa)\\n[10:19:35] 14 tRNA-Gln c[15368,15443] 35 (ttg)\\n[10:19:35] 15 tRNA-Gln c[16318,16392] 35 (ctg)\\n[10:19:35] 16 tRNA-Asn c[16403,16477] 33 (gtt)\\n[10:19:35] 17 tRNA-Phe c[16483,16558] 36 (gaa)\\n[10:19:35] 18 tRNA-Arg c[16646,16721] 35 (acg)\\n[10:19:35] 19 tRNA-Ser c[17483,17569] 35 (gct)\\n[10:19:35] 20 tRNA-Tyr c[17573,17654] 35 (gta)\\n[10:19:35] 21 tRNA-Glu c[17665,17739] 35 (ttc)\\n[10:19:35] tRNA c[17827,17901] is a pseudo/wacky gene - skipping.\\n[10:19:35] 23 tRNA-Met c[17904,17977] 32 (cat)\\n[10:19:35] 24 tRNA-Glu c[18087,18163] 35 (ctc)\\n[10:19:35] 25 tRNA-Asp c[18167,18242] 36 (gtc)\\n[10:19:35] 1 tRNA-Gly [22169,22242] 34 (ccc)\\n[10:19:35] 2 tRNA-Ser c[30387,30474] 35 (gga)\\n[10:19:35] 1 tRNA-Lys [27679,27751] 33 (ctt)\\n[10:19:35] 1 tRNA-Arg [27197,27271] 35 (tct)\\n[10:19:35] 1 tRNA-Val c[14387,14459] 34 (tac)\\n[10:19:35] 1 tRNA-Glu [22586,22659] 35 (ttc)\\n[10:19:35] 1 tRNA-Pro [7709,7782] 35 (cgg)\\n[10:19:35] 1 tRNA-Leu c[8469,8552] 36 (tag)\\n[10:19:35] 1 tRNA-Arg [959,1032] 35 (acg)\\n[10:19:35] 1 tRNA-Asn [3121,3194] 34 (gtt)\\n[10:19:35] 1 tRNA-Met [11529,11603] 35 (cat)\\n[10:19:35] 1 tRNA-Ala c[9801,9873] 34 (tgc)\\n[10:19:35] 1 tRNA-Leu [14275,14355] 35 (gag)\\n[10:19:35] 1 tRNA-Gly c[9485,9555] 33 (tcc)\\n[10:19:35] 1 tRNA-Val c[6982,7055] 35 (gac)\\n[10:19:35] 1 tRNA-Leu [10393,10474] 35 (cag)\\n[10:19:35] 1 tRNA-Cys c[6892,6963] 33 (gca)\\n[10:19:35] 1 tRNA-Phe c[176,249] 34 (gaa)\\n[10:19:35] 1 tRNA-Arg c[4642,4714] 34 (ccg)\\n[10:19:35] 1 tRNA-Trp c[7576,7649] 34 (cca)\\n[10:19:35] 1 tRNA-Thr [393,464] 33 (cgt)\\n[10:19:35] 1 tRNA-Ala c[3767,3853] 29 (cgc)\\n[10:19:35] 1 tRNA-Met c[450,526] 35 (cat)\\n[10:19:35] 2 tRNA-Ala c[3952,4027] 34 (tgc)\\n[10:19:35] 3 tRNA-Ile c[4050,4126] 35 (gat)\\n[10:19:35] 1 tRNA-Ile [1915,1989] 35 (gat)\\n[10:19:35] 2 tRNA-Ala [2100,2174] 35 (tgc)\\n[10:19:35] 1 tRNA-Thr c[956,1028] 33 (tgt)\\n[10:19:35] 1 tRNA-Asp [513,586] 35 (gtc)\\n[10:19:35] 1 tRNA-Met c[123,199] 35 (cat)\\n[10:19:35] 1 tRNA-Ile [654,730] 35 (gat)\\n[10:19:35] 2 tRNA-Ala [785,860] 34 (tgc)\\n[10:19:35] 1 tmRNA c[3,371] 97,126 GENNYALAA*\\n[10:19:35] Found 107 tRNAs\\n[10:19:35] Predicting Ribosomal RNAs\\n[10:19:35] Running Barrnap with 4 threads\\n[10:19:38] 1 LAHF01000011.1 60594 16S ribosomal RNA (partial)\\n[10:19:38] 2 LAHF01000011.1 62267 23S ribosomal RNA\\n[10:19:38] 3 LAHF01000384.1 641 5S ribosomal RNA\\n[10:19:38] 4 LAHF01000384.1 897 23S ribosomal RNA\\n[10:19:38] 5 LAHF01000386.1 227 16S ribosomal RNA\\n[10:19:38] 6 LAHF01000386.1 2289 23S ribosomal RNA (partial)\\n[10:19:38] 7 LAHF01000446.1 179 23S ribosomal RNA (partial)\\n[10:19:38] 8 LAHF01000475.1 288 5S ribosomal RNA\\n[10:19:38] 9 LAHF01000475.1 507 23S ribosomal RNA (partial)\\n[10:19:38] 10 LAHF01000493.1 93 16S ribosomal RNA\\n[10:19:38] 11 LAHF01000526.1 15 23S ribosomal RNA (partial)\\n[10:19:38] 12 LAHF01000526.1 1519 5S ribosomal RNA\\n[10:19:38] Found 12 rRNAs\\n[10:19:38] Skipping ncRNA search, enable with --rfam if desired.\\n[10:19:38] Total of 117 tRNA + rRNA features\\n[10:19:38] Searching for CRISPR repeats\\n[10:19:38] CRISPR1 LAHF01000014.1 15872 with 20 spacers\\n[10:19:38] CRISPR2 LAHF01000017.1 42277 with 14 spacers\\n[10:19:38] CRISPR3 LAHF01000036.1 63966 with 7 spacers\\n[10:19:38] CRISPR4 LAHF01000049.1 38348 with 14 spacers\\n[10:19:38] CRISPR5 LAHF01000058.1 8850 with 15 spacers\\n[10:19:38] CRISPR6 LAHF01000064.1 39 with 4 spacers\\n[10:19:38] CRISPR7 LAHF01000078.1 52280 with 10 spacers\\n[10:19:38] CRISPR8 LAHF01000084.1 5259 with 24 spacers\\n[10:19:39] CRISPR9 LAHF01000198.1 10660 with 27 spacers\\n[10:19:39] CRISPR10 LAHF01000214.1 10363 with 7 spacers\\n[10:19:39] CRISPR11 LAHF01000252.1 297 with 18 spacers\\n[10:19:39] CRISPR12 LAHF01000290.1 4928 with 6 spacers\\n[10:19:39] CRISPR13 LAHF01000301.1 4853 with 43 spacers\\n[10:19:39] CRISPR14 LAHF01000361.1 86 with 22 spacers\\n[10:19:39] Found 14 CRISPRs\\n[10:19:39] Predicting coding sequences\\n[10:19:39] Contigs total 9177463 bp, so using single mode\\n[10:19:39] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHF01\\\\/\\\\/Nostoc_LAHF01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:19:48] Excluding CDS which overlaps existing RNA (rRNA) at LAHF01000011.1:60288..60698 on - strand\\n[10:19:49] Excluding CDS which overlaps existing RNA (repeat_region) at LAHF01000017.1:42448..42564 on - strand\\n[10:19:50] Excluding CDS which overlaps existing RNA (tRNA) at LAHF01000044.1:22966..23109 on + strand\\n[10:19:50] Excluding CDS which overlaps existing RNA (repeat_region) at LAHF01000078.1:52489..52641 on + strand\\n[10:19:52] Excluding CDS which overlaps existing RNA (repeat_region) at LAHF01000214.1:10391..10516 on - strand\\n[10:19:52] Excluding CDS which overlaps existing RNA (repeat_region) at LAHF01000290.1:5320..6279 on - strand\\n[10:19:52] Excluding CDS which overlaps existing RNA (repeat_region) at LAHF01000301.1:7791..7955 on + strand\\n[10:19:52] Excluding CDS which overlaps existing RNA (tRNA) at LAHF01000335.1:3302..4672 on - strand\\n[10:19:52] Excluding CDS which overlaps existing RNA (repeat_region) at LAHF01000361.1:1540..1818 on - strand\\n[10:19:52] Excluding CDS which overlaps existing RNA (rRNA) at LAHF01000384.1:715..1047 on + strand\\n[10:19:52] Excluding CDS which overlaps existing RNA (tRNA) at LAHF01000384.1:4049..4444 on + strand\\n[10:19:52] Excluding CDS which overlaps existing RNA (rRNA) at LAHF01000386.1:2356..2823 on - strand\\n[10:19:52] Excluding CDS which overlaps existing RNA (rRNA) at LAHF01000446.1:2025..2273 on - strand\\n[10:19:52] Excluding CDS which overlaps existing RNA (rRNA) at LAHF01000475.1:298..501 on - strand\\n[10:19:52] Excluding CDS which overlaps existing RNA (rRNA) at LAHF01000475.1:639..1097 on + strand\\n[10:19:53] Excluding CDS which overlaps existing RNA (rRNA) at LAHF01000493.1:4..393 on + strand\\n[10:19:53] Excluding CDS which overlaps existing RNA (rRNA) at LAHF01000526.1:693..1151 on - strand\\n[10:19:53] Excluding CDS which overlaps existing RNA (tRNA) at LAHF01000549.1:767..955 on - strand\\n[10:19:53] Found 7637 CDS\\n[10:19:53] Connecting features back to sequences\\n[10:19:53] Not using genus-specific database. Try --usegenus to enable it.\\n[10:19:53] Annotating CDS, please be patient.\\n[10:19:53] Will use 4 CPUs for similarity searching.\\n[10:19:55] There are still 7637 unannotated CDS left (started with 7637)\\n[10:19:55] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:19:55] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHF01\\\\/\\\\/Nostoc_LAHF01\\\\.IS\\\\.tmp\\\\.282\\\\.faa | parallel --gnu --plain -j 4 --block 303373 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHF01\\\\/\\\\/Nostoc_LAHF01\\\\.IS\\\\.tmp\\\\.282\\\\.blast 2> /dev/null\\n[10:20:14] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHF01\\\\/\\\\/Nostoc_LAHF01\\\\.IS\\\\.tmp\\\\.282\\\\.faa | parallel --gnu --plain -j 4 --block 303373 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHF01\\\\/\\\\/Nostoc_LAHF01\\\\.IS\\\\.tmp\\\\.282\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:20:14] This is prokka 1.14.6\\n[10:20:14] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:20:14] Homepage is https://github.com/tseemann/prokka\\n[10:20:14] Local time is Mon Feb 21 10:20:14 2022\\n[10:20:14] You are tiagoferreiraleao\\n[10:20:14] Operating system is darwin\\n[10:20:14] You have BioPerl 1.007002\\n[10:20:14] System has 4 cores.\\n[10:20:14] Option --cpu asked for 12 cores, but system only has 4\\n[10:20:14] Will use maximum of 4 cores.\\n[10:20:14] Annotating as >>> Bacteria <<<\\n[10:20:14] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JACYLX01.fasta' contents.\\n[10:20:15] Setting --locustag MCCJIOFK from MD5 6cc328f43b95573ec12a8babc45de7fa\\n[10:20:15] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JACYLX01/\\n[10:20:15] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYLX01\\\\/\\n[10:20:15] Using filename prefix: Fischerella_JACYLX01.XXX\\n[10:20:15] Setting HMMER_NCPU=1\\n[10:20:15] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JACYLX01//Fischerella_JACYLX01.log\\n[10:20:15] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JACYLX01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JACYLX01/ --prefix Fischerella_JACYLX01 --cpus 12\\n[10:20:15] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:20:15] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:20:15] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:20:15] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:20:15] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:20:15] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:20:15] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:20:15] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:20:15] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:20:15] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:20:15] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:20:15] Looking for 'egrep' - found /usr/bin/egrep\\n[10:20:15] Looking for 'find' - found /usr/bin/find\\n[10:20:15] Looking for 'grep' - found /usr/bin/grep\\n[10:20:15] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:20:15] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:20:15] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:20:15] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:20:15] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:20:15] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:20:15] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:20:15] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:20:15] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:20:15] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:20:15] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:20:15] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:20:15] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:20:15] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:20:15] Looking for 'sed' - found /usr/bin/sed\\n[10:20:15] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:20:16] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:20:16] Using genetic code table 11.\\n[10:20:16] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JACYLX01.fasta\\n[10:20:16] Wrote 281 contigs totalling 5141023 bp.\\n[10:20:16] Predicting tRNAs and tmRNAs\\n[10:20:16] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYLX01\\\\/\\\\/Fischerella_JACYLX01\\\\.fna\\n[10:20:18] 1 tRNA-Arg c[5863,5936] 35 (ccg)\\n[10:20:18] 1 tRNA-Glu [5950,6022] 35 (ttc)\\n[10:20:18] 1 tRNA-Pro c[11646,11719] 35 (ggg)\\n[10:20:18] 1 tRNA-Leu [107,190] 36 (tag)\\n[10:20:18] 1 tRNA-Asp c[43976,44049] 35 (gtc)\\n[10:20:18] 1 tRNA-Ser c[22090,22176] 36 (tga)\\n[10:20:18] 1 tRNA-Leu c[35491,35573] 35 (cag)\\n[10:20:18] 1 tRNA-Asn c[3893,3966] 34 (att)\\n[10:20:18] 2 tRNA-Arg c[4637,4709] 34 (tcg)\\n[10:20:18] 1 tRNA-Pro c[20711,20785] 35 (tgg)\\n[10:20:18] 1 tRNA-Arg [10446,10520] 35 (tct)\\n[10:20:18] 1 tRNA-Gln c[1817,1888] 33 (ttg)\\n[10:20:18] 2 tRNA-Leu c[34791,34875] 35 (caa)\\n[10:20:18] 1 tRNA-Leu [53750,53832] 36 (gag)\\n[10:20:18] 1 tmRNA [1123,1509] 100,153 ANNIVKFARKEALVAA**\\n[10:20:18] 1 tRNA-Ser c[33205,33291] 36 (cga)\\n[10:20:18] 1 tRNA-Gly c[22509,22580] 33 (gcc)\\n[10:20:18] 1 tRNA-Ser [20588,20681] 36 (gct)\\n[10:20:18] 1 tRNA-Ser c[94116,94204] 36 (gga)\\n[10:20:18] 1 tRNA-Arg c[32305,32378] 35 (acg)\\n[10:20:18] 2 tRNA-Ala c[37243,37316] 34 (ggc)\\n[10:20:18] 3 tRNA-Thr c[39137,39208] 33 (cgt)\\n[10:20:18] 1 tRNA-Trp [28666,28739] 34 (cca)\\n[10:20:18] 1 tRNA-Val c[15932,16004] 34 (tac)\\n[10:20:18] 2 tRNA-Phe [23354,23426] 34 (gaa)\\n[10:20:18] 1 tRNA-Asn c[62344,62417] 34 (gtt)\\n[10:20:18] 1 tRNA-Gly [24747,24817] 33 (tcc)\\n[10:20:18] 1 tRNA-Thr c[89,160] 33 (tgt)\\n[10:20:18] 1 tRNA-Cys [7922,7993] 33 (gca)\\n[10:20:18] 1 tRNA-Met c[18908,18983] 36 (cat)\\n[10:20:18] 1 tRNA-Val c[2234,2307] 35 (gac)\\n[10:20:18] 2 tRNA-Met [5278,5354] 35 (cat)\\n[10:20:18] 1 tRNA-Gly c[180,251] 33 (ccc)\\n[10:20:18] 1 tRNA-His c[593,666] 34 (gtg)\\n[10:20:18] 1 tRNA-Lys c[23081,23154] 34 (ttt)\\n[10:20:18] 1 tRNA-Met [7692,7764] 34 (cat)\\n[10:20:18] 1 tRNA-Ala c[12846,12921] 35 (cgc)\\n[10:20:18] 1 tRNA-Gln c[30,100] 32 (ttg)\\n[10:20:18] Found 38 tRNAs\\n[10:20:18] Predicting Ribosomal RNAs\\n[10:20:18] Running Barrnap with 4 threads\\n[10:20:19] Found 0 rRNAs\\n[10:20:19] Skipping ncRNA search, enable with --rfam if desired.\\n[10:20:19] Total of 37 tRNA + rRNA features\\n[10:20:19] Searching for CRISPR repeats\\n[10:20:19] CRISPR1 JACYLX010000069.1 19 with 10 spacers\\n[10:20:19] CRISPR2 JACYLX010000098.1 45 with 20 spacers\\n[10:20:19] CRISPR3 JACYLX010000119.1 19 with 6 spacers\\n[10:20:19] CRISPR4 JACYLX010000244.1 9596 with 3 spacers\\n[10:20:19] Found 4 CRISPRs\\n[10:20:19] Predicting coding sequences\\n[10:20:19] Contigs total 5141023 bp, so using single mode\\n[10:20:19] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYLX01\\\\/\\\\/Fischerella_JACYLX01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:20:25] Excluding CDS which overlaps existing RNA (tRNA) at JACYLX010000135.1:61104..62402 on - strand\\n[10:20:26] Found 4325 CDS\\n[10:20:26] Connecting features back to sequences\\n[10:20:26] Not using genus-specific database. Try --usegenus to enable it.\\n[10:20:26] Annotating CDS, please be patient.\\n[10:20:26] Will use 4 CPUs for similarity searching.\\n[10:20:27] There are still 4325 unannotated CDS left (started with 4325)\\n[10:20:27] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:20:27] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYLX01\\\\/\\\\/Fischerella_JACYLX01\\\\.IS\\\\.tmp\\\\.434\\\\.faa | parallel --gnu --plain -j 4 --block 170907 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYLX01\\\\/\\\\/Fischerella_JACYLX01\\\\.IS\\\\.tmp\\\\.434\\\\.blast 2> /dev/null\\n[10:20:48] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYLX01\\\\/\\\\/Fischerella_JACYLX01\\\\.IS\\\\.tmp\\\\.434\\\\.faa | parallel --gnu --plain -j 4 --block 170907 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYLX01\\\\/\\\\/Fischerella_JACYLX01\\\\.IS\\\\.tmp\\\\.434\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:20:49] This is prokka 1.14.6\\n[10:20:49] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:20:49] Homepage is https://github.com/tseemann/prokka\\n[10:20:49] Local time is Mon Feb 21 10:20:49 2022\\n[10:20:49] You are tiagoferreiraleao\\n[10:20:49] Operating system is darwin\\n[10:20:49] You have BioPerl 1.007002\\n[10:20:49] System has 4 cores.\\n[10:20:49] Option --cpu asked for 12 cores, but system only has 4\\n[10:20:49] Will use maximum of 4 cores.\\n[10:20:49] Annotating as >>> Bacteria <<<\\n[10:20:49] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_WBKL01.fasta' contents.\\n[10:20:49] Setting --locustag KDJBJOGE from MD5 4d3b380eb8cd18adf13253ec060a5f51\\n[10:20:49] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_WBKL01/\\n[10:20:49] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_WBKL01\\\\/\\n[10:20:49] Using filename prefix: Nostoc_WBKL01.XXX\\n[10:20:49] Setting HMMER_NCPU=1\\n[10:20:49] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_WBKL01//Nostoc_WBKL01.log\\n[10:20:49] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_WBKL01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_WBKL01/ --prefix Nostoc_WBKL01 --cpus 12\\n[10:20:49] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:20:49] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:20:49] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:20:49] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:20:49] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:20:49] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:20:50] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:20:50] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:20:50] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:20:50] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:20:50] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:20:50] Looking for 'egrep' - found /usr/bin/egrep\\n[10:20:50] Looking for 'find' - found /usr/bin/find\\n[10:20:50] Looking for 'grep' - found /usr/bin/grep\\n[10:20:50] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:20:50] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:20:50] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:20:50] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:20:50] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:20:50] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:20:50] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:20:50] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:20:50] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:20:50] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:20:50] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:20:50] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:20:51] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:20:51] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:20:51] Looking for 'sed' - found /usr/bin/sed\\n[10:20:51] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:20:51] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:20:51] Using genetic code table 11.\\n[10:20:51] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_WBKL01.fasta\\n[10:20:51] Wrote 217 contigs totalling 7473008 bp.\\n[10:20:51] Predicting tRNAs and tmRNAs\\n[10:20:51] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_WBKL01\\\\/\\\\/Nostoc_WBKL01\\\\.fna\\n[10:20:55] 1 tRNA-His c[8685,8758] 34 (gtg)\\n[10:20:55] 1 tRNA-Met c[77638,77714] 35 (cat)\\n[10:20:55] 1 tRNA-Leu c[332,416] 35 (caa)\\n[10:20:55] 2 tRNA-Leu [4567,4648] 35 (cag)\\n[10:20:55] 3 tRNA-Val [31675,31747] 33 (cac)\\n[10:20:55] 1 tRNA-Ser [64261,64347] 36 (gga)\\n[10:20:55] 1 tRNA-Val c[4032,4104] 34 (tac)\\n[10:20:55] 2 tRNA-Ser c[23562,23648] 36 (tga)\\n[10:20:55] 3 tRNA-Tyr [25948,26033] 35 (gta)\\n[10:20:55] 4 tRNA-Thr [26121,26192] 33 (ggt)\\n[10:20:55] 5 tRNA-Arg c[34805,34879] 35 (tct)\\n[10:20:55] 1 tRNA-Phe [32011,32083] 34 (gaa)\\n[10:20:55] 1 tRNA-Phe c[14502,14574] 33 (gaa)\\n[10:20:55] 2 tRNA-Pyl c[16089,16163] 33 (cta)\\n[10:20:55] 1 tRNA-Thr c[2052,2123] 33 (tgt)\\n[10:20:55] 1 tRNA-Leu c[114691,114773] 35 (tag)\\n[10:20:55] 1 tRNA-Pro [6586,6659] 35 (cgg)\\n[10:20:55] 1 tRNA-Lys [32295,32366] 33 (ttt)\\n[10:20:55] 1 tRNA-Arg c[24883,24956] 35 (acg)\\n[10:20:55] 2 tRNA-Met c[42533,42606] 33 (cat)\\n[10:20:55] 1 tRNA-Ala [2663,2735] 34 (ggc)\\n[10:20:55] 1 tRNA-Pro c[42398,42471] 35 (tgg)\\n[10:20:55] 1 tRNA-Arg c[3163,3235] 34 (ccg)\\n[10:20:55] 1 tRNA-Leu [29698,29780] 36 (gag)\\n[10:20:55] 1 tRNA-Trp [42889,42962] 34 (cca)\\n[10:20:55] 2 tRNA-Ala [68969,69041] 34 (cgc)\\n[10:20:55] 1 tRNA-Gly [11770,11840] 33 (tcc)\\n[10:20:55] 1 tRNA-Gly [168,239] 33 (gcc)\\n[10:20:55] 1 tRNA-Lys c[66207,66279] 33 (ctt)\\n[10:20:55] 1 tRNA-Val [24704,24777] 35 (gac)\\n[10:20:55] 1 tRNA-Ser c[74794,74880] 36 (cga)\\n[10:20:55] 1 tRNA-Asp c[34874,34947] 35 (gtc)\\n[10:20:55] 1 tRNA-Thr c[14520,14591] 33 (cgt)\\n[10:20:55] 1 tRNA-Pro [19734,19807] 35 (ggg)\\n[10:20:55] 1 tRNA-Cys [33347,33417] 33 (gca)\\n[10:20:55] 1 tRNA-Glu [77699,77772] 35 (ttc)\\n[10:20:55] 2 tRNA-Glu [84882,84958] 35 (ctc)\\n[10:20:55] 3 tRNA-Cys [84962,85035] 35 (gca)\\n[10:20:55] tRNA [85120,85194] is a pseudo/wacky gene - skipping.\\n[10:20:55] 5 tRNA-Glu [85247,85320] 35 (ttc)\\n[10:20:55] 6 tRNA-Ser [85462,85552] 36 (gct)\\n[10:20:55] 7 tRNA-Arg [85555,85631] 36 (acg)\\n[10:20:55] 8 tRNA-Asn [85816,85890] 33 (gtt)\\n[10:20:55] 9 tRNA-Gln [85897,85968] 33 (ctg)\\n[10:20:55] 10 tRNA-Ala [86058,86132] 35 (tgc)\\n[10:20:55] 11 tRNA-Ser [90876,90969] 36 (gct)\\n[10:20:55] 1 tRNA-Ala c[3346,3420] 34 (tgc)\\n[10:20:55] 2 tRNA-Ile c[3499,3574] 35 (gat)\\n[10:20:55] 1 tRNA-Gly [4533,4604] 33 (ccc)\\n[10:20:55] 2 tmRNA c[27995,28382] 101,151 ANNIVNFARKDALVAA*\\n[10:20:55] 1 tRNA-Asn [2623,2694] 33 (gtt)\\n[10:20:55] 1 tRNA-Ala c[4688,4762] 35 (tgc)\\n[10:20:55] 1 tRNA-Arg [9008,9083] 36 (cct)\\n[10:20:55] 1 tRNA-Gln c[11051,11124] 34 (ttg)\\n[10:20:55] 1 tRNA-Met [45242,45316] 35 (cat)\\n[10:20:55] Found 54 tRNAs\\n[10:20:55] Predicting Ribosomal RNAs\\n[10:20:55] Running Barrnap with 4 threads\\n[10:20:57] 1 WBKL01000170.1 270 23S ribosomal RNA\\n[10:20:57] 2 WBKL01000170.1 3704 16S ribosomal RNA\\n[10:20:57] Found 2 rRNAs\\n[10:20:57] Skipping ncRNA search, enable with --rfam if desired.\\n[10:20:57] Total of 55 tRNA + rRNA features\\n[10:20:57] Searching for CRISPR repeats\\n[10:20:57] CRISPR1 WBKL01000032.1 46846 with 12 spacers\\n[10:20:57] CRISPR2 WBKL01000038.1 29093 with 10 spacers\\n[10:20:57] CRISPR3 WBKL01000083.1 10893 with 18 spacers\\n[10:20:57] CRISPR4 WBKL01000116.1 49028 with 12 spacers\\n[10:20:57] CRISPR5 WBKL01000132.1 73 with 27 spacers\\n[10:20:58] CRISPR6 WBKL01000208.1 77894 with 5 spacers\\n[10:20:58] CRISPR7 WBKL01000210.1 45813 with 5 spacers\\n[10:20:58] Found 7 CRISPRs\\n[10:20:58] Predicting coding sequences\\n[10:20:58] Contigs total 7473008 bp, so using single mode\\n[10:20:58] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_WBKL01\\\\/\\\\/Nostoc_WBKL01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:21:07] Excluding CDS which overlaps existing RNA (repeat_region) at WBKL01000032.1:46726..46872 on + strand\\n[10:21:07] Excluding CDS which overlaps existing RNA (repeat_region) at WBKL01000038.1:28865..29107 on + strand\\n[10:21:07] Excluding CDS which overlaps existing RNA (tRNA) at WBKL01000064.1:6571..6795 on + strand\\n[10:21:07] Excluding CDS which overlaps existing RNA (tRNA) at WBKL01000065.1:32263..33063 on - strand\\n[10:21:08] Excluding CDS which overlaps existing RNA (repeat_region) at WBKL01000083.1:11867..12310 on + strand\\n[10:21:10] Excluding CDS which overlaps existing RNA (tRNA) at WBKL01000167.1:77697..77849 on + strand\\n[10:21:10] Excluding CDS which overlaps existing RNA (tRNA) at WBKL01000167.1:90875..91057 on + strand\\n[10:21:10] Excluding CDS which overlaps existing RNA (rRNA) at WBKL01000170.1:3746..4060 on - strand\\n[10:21:11] Found 6331 CDS\\n[10:21:11] Connecting features back to sequences\\n[10:21:11] Not using genus-specific database. Try --usegenus to enable it.\\n[10:21:11] Annotating CDS, please be patient.\\n[10:21:11] Will use 4 CPUs for similarity searching.\\n[10:21:13] There are still 6331 unannotated CDS left (started with 6331)\\n[10:21:13] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:21:13] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_WBKL01\\\\/\\\\/Nostoc_WBKL01\\\\.IS\\\\.tmp\\\\.555\\\\.faa | parallel --gnu --plain -j 4 --block 254328 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_WBKL01\\\\/\\\\/Nostoc_WBKL01\\\\.IS\\\\.tmp\\\\.555\\\\.blast 2> /dev/null\\n[10:21:23] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_WBKL01\\\\/\\\\/Nostoc_WBKL01\\\\.IS\\\\.tmp\\\\.555\\\\.faa | parallel --gnu --plain -j 4 --block 254328 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_WBKL01\\\\/\\\\/Nostoc_WBKL01\\\\.IS\\\\.tmp\\\\.555\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:21:23] This is prokka 1.14.6\\n[10:21:23] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:21:23] Homepage is https://github.com/tseemann/prokka\\n[10:21:23] Local time is Mon Feb 21 10:21:23 2022\\n[10:21:23] You are tiagoferreiraleao\\n[10:21:23] Operating system is darwin\\n[10:21:23] You have BioPerl 1.007002\\n[10:21:23] System has 4 cores.\\n[10:21:23] Option --cpu asked for 12 cores, but system only has 4\\n[10:21:23] Will use maximum of 4 cores.\\n[10:21:23] Annotating as >>> Bacteria <<<\\n[10:21:23] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Westiellopsis_NAPS01.fasta' contents.\\n[10:21:23] Setting --locustag BIADDOPG from MD5 b2add8904ccff5020a095d68fd0b67bc\\n[10:21:23] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Westiellopsis_NAPS01/\\n[10:21:23] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Westiellopsis_NAPS01\\\\/\\n[10:21:23] Using filename prefix: Westiellopsis_NAPS01.XXX\\n[10:21:23] Setting HMMER_NCPU=1\\n[10:21:23] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Westiellopsis_NAPS01//Westiellopsis_NAPS01.log\\n[10:21:23] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Westiellopsis_NAPS01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Westiellopsis_NAPS01/ --prefix Westiellopsis_NAPS01 --cpus 12\\n[10:21:23] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:21:23] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:21:23] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:21:23] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:21:24] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:21:24] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:21:24] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:21:24] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:21:24] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:21:24] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:21:24] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:21:24] Looking for 'egrep' - found /usr/bin/egrep\\n[10:21:24] Looking for 'find' - found /usr/bin/find\\n[10:21:24] Looking for 'grep' - found /usr/bin/grep\\n[10:21:24] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:21:24] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:21:24] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:21:24] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:21:24] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:21:24] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:21:24] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:21:24] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:21:25] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:21:25] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:21:25] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:21:25] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:21:25] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:21:25] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:21:25] Looking for 'sed' - found /usr/bin/sed\\n[10:21:25] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:21:26] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:21:26] Using genetic code table 11.\\n[10:21:26] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Westiellopsis_NAPS01.fasta\\n[10:21:26] Wrote 29 contigs totalling 7313429 bp.\\n[10:21:26] Predicting tRNAs and tmRNAs\\n[10:21:26] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Westiellopsis_NAPS01\\\\/\\\\/Westiellopsis_NAPS01\\\\.fna\\n[10:21:30] 1 tRNA-Gly c[123404,123474] 33 (tcc)\\n[10:21:30] 2 tRNA-Ala c[174632,174706] 35 (ggc)\\n[10:21:30] 3 tRNA-Thr c[188606,188677] 33 (tgt)\\n[10:21:30] 4 tRNA-Pro [312245,312318] 35 (ggg)\\n[10:21:30] 5 tRNA-Met c[547622,547695] 34 (cat)\\n[10:21:30] 6 tRNA-Ser c[881934,882020] 36 (cga)\\n[10:21:30] 7 tRNA-Asn c[1075407,1075480] 34 (att)\\n[10:21:30] 8 tRNA-Phe c[1075851,1075924] 35 (gaa)\\n[10:21:30] 9 tRNA-Ala [1278371,1278443] 34 (cgc)\\n[10:21:30] 10 tRNA-Leu [1336508,1336592] 35 (caa)\\n[10:21:30] 11 tRNA-Gln [1373927,1373998] 33 (ttg)\\n[10:21:30] 12 tRNA-Thr c[1880608,1880679] 33 (ggt)\\n[10:21:30] 13 tRNA-Tyr c[1880757,1880841] 35 (gta)\\n[10:21:30] 14 tRNA-Trp c[2051842,2051915] 34 (cca)\\n[10:21:30] 15 tRNA-Leu c[2216989,2217071] 35 (cag)\\n[10:21:30] 16 tRNA-Leu c[2444934,2445017] 36 (tag)\\n[10:21:30] 17 tRNA-Glu [2603942,2604014] 35 (ttc)\\n[10:21:30] 18 tRNA-Arg c[2688833,2688905] 34 (ccg)\\n[10:21:30] 19 tRNA-Leu [2864026,2864108] 36 (gag)\\n[10:21:30] 20 tRNA-Thr [2989149,2989220] 33 (cgt)\\n[10:21:30] 21 tRNA-Gly [3298686,3298757] 33 (gcc)\\n[10:21:30] 22 tRNA-Lys [3392369,3392440] 33 (ttt)\\n[10:21:30] 1 tRNA-Pro c[95802,95875] 35 (cgg)\\n[10:21:30] 2 tRNA-Asp c[197854,197927] 35 (gtc)\\n[10:21:30] 3 tRNA-Ser c[201699,201787] 36 (gga)\\n[10:21:30] 4 tRNA-Pro [389659,389733] 35 (tgg)\\n[10:21:30] 5 tRNA-Phe [425412,425484] 34 (gaa)\\n[10:21:30] 6 tRNA-Ser c[615385,615470] 35 (tga)\\n[10:21:30] 7 tRNA-His [988748,988821] 34 (gtg)\\n[10:21:30] 8 tRNA-Arg c[1129884,1129959] 36 (cct)\\n[10:21:30] 9 tRNA-Met [1202590,1202665] 36 (cat)\\n[10:21:30] 10 tRNA-Asn c[1325396,1325469] 34 (gtt)\\n[10:21:30] 11 tRNA-Val c[1426789,1426861] 34 (tac)\\n[10:21:30] 12 tRNA-Val c[1542785,1542858] 35 (gac)\\n[10:21:30] 13 tRNA-Met [1545892,1545968] 35 (cat)\\n[10:21:30] 14 tmRNA [1601612,1601999] 100,153 ANNIVKFARKEALVAA**\\n[10:21:30] 15 tRNA-Gly c[1618719,1618790] 33 (ccc)\\n[10:21:30] 16 tRNA-Lys [2105008,2105080] 33 (ctt)\\n[10:21:30] 17 tRNA-Arg [2223431,2223504] 35 (acg)\\n[10:21:30] 18 tRNA-Ser c[2233731,2233824] 36 (gct)\\n[10:21:30] 19 tRNA-Cys c[2308555,2308626] 33 (gca)\\n[10:21:30] 20 tRNA-Arg c[2370350,2370424] 35 (tct)\\n[10:21:30] 1 tRNA-Trp [493416,493490] 35 (cca)\\n[10:21:30] Found 43 tRNAs\\n[10:21:30] Predicting Ribosomal RNAs\\n[10:21:30] Running Barrnap with 4 threads\\n[10:21:33] 1 NAPS01000001.1 556707 16S ribosomal RNA (partial)\\n[10:21:33] 2 NAPS01000001.1 2648497 16S ribosomal RNA (partial)\\n[10:21:33] 3 NAPS01000001.1 3127700 16S ribosomal RNA (partial)\\n[10:21:33] 4 NAPS01000002.1 2328425 16S ribosomal RNA (partial)\\n[10:21:33] Found 4 rRNAs\\n[10:21:33] Skipping ncRNA search, enable with --rfam if desired.\\n[10:21:33] Total of 46 tRNA + rRNA features\\n[10:21:33] Searching for CRISPR repeats\\n[10:21:34] CRISPR1 NAPS01000001.1 616708 with 6 spacers\\n[10:21:34] CRISPR2 NAPS01000001.1 2345651 with 5 spacers\\n[10:21:34] CRISPR3 NAPS01000001.1 2590860 with 4 spacers\\n[10:21:34] CRISPR4 NAPS01000001.1 3161560 with 23 spacers\\n[10:21:34] CRISPR5 NAPS01000001.1 3254465 with 9 spacers\\n[10:21:34] CRISPR6 NAPS01000001.1 3290867 with 17 spacers\\n[10:21:34] CRISPR7 NAPS01000001.1 3338867 with 10 spacers\\n[10:21:34] CRISPR8 NAPS01000001.1 3376473 with 5 spacers\\n[10:21:34] CRISPR9 NAPS01000001.1 3379998 with 19 spacers\\n[10:21:34] CRISPR10 NAPS01000002.1 300206 with 9 spacers\\n[10:21:34] CRISPR11 NAPS01000002.1 2039671 with 17 spacers\\n[10:21:34] CRISPR12 NAPS01000002.1 2180558 with 6 spacers\\n[10:21:34] CRISPR13 NAPS01000002.1 2187642 with 4 spacers\\n[10:21:34] CRISPR14 NAPS01000002.1 2353185 with 6 spacers\\n[10:21:34] CRISPR15 NAPS01000005.1 7778 with 8 spacers\\n[10:21:34] CRISPR16 NAPS01000005.1 13344 with 16 spacers\\n[10:21:34] CRISPR17 NAPS01000007.1 10562 with 6 spacers\\n[10:21:34] CRISPR18 NAPS01000007.1 14411 with 13 spacers\\n[10:21:34] CRISPR19 NAPS01000013.1 2983 with 17 spacers\\n[10:21:34] CRISPR20 NAPS01000020.1 1694 with 3 spacers\\n[10:21:34] Found 20 CRISPRs\\n[10:21:34] Predicting coding sequences\\n[10:21:34] Contigs total 7313429 bp, so using single mode\\n[10:21:34] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Westiellopsis_NAPS01\\\\/\\\\/Westiellopsis_NAPS01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:21:45] Excluding CDS which overlaps existing RNA (rRNA) at NAPS01000001.1:556920..557153 on - strand\\n[10:21:45] Excluding CDS which overlaps existing RNA (repeat_region) at NAPS01000001.1:616746..617042 on + strand\\n[10:21:45] Excluding CDS which overlaps existing RNA (repeat_region) at NAPS01000001.1:617074..617253 on - strand\\n[10:21:46] Excluding CDS which overlaps existing RNA (tRNA) at NAPS01000001.1:1278382..1279137 on - strand\\n[10:21:46] Excluding CDS which overlaps existing RNA (tRNA) at NAPS01000001.1:1373809..1374087 on + strand\\n[10:21:46] Excluding CDS which overlaps existing RNA (tRNA) at NAPS01000001.1:2444861..2445037 on - strand\\n[10:21:46] Excluding CDS which overlaps existing RNA (rRNA) at NAPS01000001.1:2648710..2648943 on - strand\\n[10:21:47] Excluding CDS which overlaps existing RNA (rRNA) at NAPS01000001.1:3127913..3128146 on - strand\\n[10:21:47] Excluding CDS which overlaps existing RNA (repeat_region) at NAPS01000001.1:3254663..3254965 on + strand\\n[10:21:47] Excluding CDS which overlaps existing RNA (repeat_region) at NAPS01000001.1:3290721..3290900 on + strand\\n[10:21:47] Excluding CDS which overlaps existing RNA (repeat_region) at NAPS01000001.1:3338666..3338986 on - strand\\n[10:21:47] Excluding CDS which overlaps existing RNA (repeat_region) at NAPS01000001.1:3381292..3381510 on - strand\\n[10:21:48] Excluding CDS which overlaps existing RNA (rRNA) at NAPS01000002.1:2328638..2328871 on - strand\\n[10:21:48] Excluding CDS which overlaps existing RNA (repeat_region) at NAPS01000002.1:2353281..2353610 on + strand\\n[10:21:48] Excluding CDS which overlaps existing RNA (repeat_region) at NAPS01000013.1:3333..3581 on - strand\\n[10:21:48] Excluding CDS which overlaps existing RNA (repeat_region) at NAPS01000013.1:3862..4113 on + strand\\n[10:21:48] Excluding CDS which overlaps existing RNA (repeat_region) at NAPS01000013.1:4103..4366 on + strand\\n[10:21:48] Found 6023 CDS\\n[10:21:48] Connecting features back to sequences\\n[10:21:48] Not using genus-specific database. Try --usegenus to enable it.\\n[10:21:48] Annotating CDS, please be patient.\\n[10:21:48] Will use 4 CPUs for similarity searching.\\n[10:21:50] There are still 6023 unannotated CDS left (started with 6023)\\n[10:21:50] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:21:50] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Westiellopsis_NAPS01\\\\/\\\\/Westiellopsis_NAPS01\\\\.IS\\\\.tmp\\\\.668\\\\.faa | parallel --gnu --plain -j 4 --block 243734 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Westiellopsis_NAPS01\\\\/\\\\/Westiellopsis_NAPS01\\\\.IS\\\\.tmp\\\\.668\\\\.blast 2> /dev/null\\n[10:21:59] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Westiellopsis_NAPS01\\\\/\\\\/Westiellopsis_NAPS01\\\\.IS\\\\.tmp\\\\.668\\\\.faa | parallel --gnu --plain -j 4 --block 243734 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Westiellopsis_NAPS01\\\\/\\\\/Westiellopsis_NAPS01\\\\.IS\\\\.tmp\\\\.668\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:21:59] This is prokka 1.14.6\\n[10:21:59] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:21:59] Homepage is https://github.com/tseemann/prokka\\n[10:21:59] Local time is Mon Feb 21 10:21:59 2022\\n[10:21:59] You are tiagoferreiraleao\\n[10:21:59] Operating system is darwin\\n[10:21:59] You have BioPerl 1.007002\\n[10:22:00] System has 4 cores.\\n[10:22:00] Option --cpu asked for 12 cores, but system only has 4\\n[10:22:00] Will use maximum of 4 cores.\\n[10:22:00] Annotating as >>> Bacteria <<<\\n[10:22:00] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_MRCA01.fasta' contents.\\n[10:22:00] Setting --locustag EHPLAAMF from MD5 e195aa6f4eb3aade861bafd57560d34d\\n[10:22:00] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_MRCA01/\\n[10:22:00] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_MRCA01\\\\/\\n[10:22:00] Using filename prefix: Fischerella_MRCA01.XXX\\n[10:22:00] Setting HMMER_NCPU=1\\n[10:22:00] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_MRCA01//Fischerella_MRCA01.log\\n[10:22:00] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_MRCA01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_MRCA01/ --prefix Fischerella_MRCA01 --cpus 12\\n[10:22:00] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:22:00] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:22:00] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:22:00] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:22:00] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:22:00] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:22:00] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:22:00] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:22:00] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:22:00] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:22:00] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:22:00] Looking for 'egrep' - found /usr/bin/egrep\\n[10:22:00] Looking for 'find' - found /usr/bin/find\\n[10:22:00] Looking for 'grep' - found /usr/bin/grep\\n[10:22:00] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:22:00] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:22:00] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:22:00] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:22:00] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:22:00] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:22:00] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:22:00] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:22:01] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:22:01] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:22:01] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:22:01] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:22:01] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:22:01] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:22:01] Looking for 'sed' - found /usr/bin/sed\\n[10:22:01] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:22:01] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:22:01] Using genetic code table 11.\\n[10:22:01] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_MRCA01.fasta\\n[10:22:02] Wrote 34 contigs totalling 5639496 bp.\\n[10:22:02] Predicting tRNAs and tmRNAs\\n[10:22:02] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_MRCA01\\\\/\\\\/Fischerella_MRCA01\\\\.fna\\n[10:22:05] 1 tRNA-Arg [142691,142765] 35 (tct)\\n[10:22:05] 2 tRNA-Met [168538,168614] 36 (cat)\\n[10:22:05] 3 tRNA-Val c[213572,213644] 34 (tac)\\n[10:22:05] 4 tRNA-Phe [220971,221043] 34 (gaa)\\n[10:22:05] 5 tRNA-Ser c[562368,562461] 36 (gct)\\n[10:22:05] 6 tRNA-Lys c[714424,714497] 34 (ttt)\\n[10:22:05] 1 tRNA-Arg c[35651,35726] 36 (cct)\\n[10:22:05] 2 tRNA-Ser c[101358,101446] 36 (gga)\\n[10:22:05] 3 tRNA-Pro [451864,451937] 35 (ggg)\\n[10:22:05] 4 tRNA-Arg [649596,649669] 35 (ccg)\\n[10:22:05] 5 tRNA-Trp [693128,693201] 34 (cca)\\n[10:22:05] 1 tRNA-Gly [104146,104216] 33 (tcc)\\n[10:22:05] 2 tRNA-Cys [177755,177826] 33 (gca)\\n[10:22:05] 3 tRNA-Gln c[289857,289928] 33 (ttg)\\n[10:22:05] 4 tRNA-Leu c[324895,324979] 35 (caa)\\n[10:22:05] 1 tRNA-Met [8387,8459] 34 (cat)\\n[10:22:05] 2 tRNA-Thr [257938,258009] 33 (cgt)\\n[10:22:05] 3 tRNA-Ala [259417,259490] 34 (ggc)\\n[10:22:05] 4 tRNA-Arg [264164,264237] 35 (acg)\\n[10:22:05] 5 tmRNA c[361010,361396] 100,153 ANNIVKFARKEALVAA**\\n[10:22:05] 1 tRNA-Asn c[382733,382806] 34 (att)\\n[10:22:05] 2 tRNA-Arg c[383509,383581] 34 (tcg)\\n[10:22:05] 1 tRNA-Thr [70221,70292] 33 (tgt)\\n[10:22:05] 1 tRNA-Leu [33052,33135] 36 (tag)\\n[10:22:05] 2 tRNA-Ala [177262,177336] 35 (cgc)\\n[10:22:05] 3 tRNA-Leu [194379,194461] 35 (cag)\\n[10:22:05] 1 tRNA-Gly [160535,160606] 33 (ccc)\\n[10:22:05] 2 tRNA-Thr c[174589,174660] 33 (ggt)\\n[10:22:05] 3 tRNA-Tyr c[174723,174807] 35 (gta)\\n[10:22:05] 1 tRNA-Pro [150686,150759] 35 (cgg)\\n[10:22:05] 1 tRNA-Ser [4611,4697] 36 (cga)\\n[10:22:05] 2 tRNA-Ser c[147515,147600] 35 (tga)\\n[10:22:05] 1 tRNA-Val c[53685,53758] 35 (gac)\\n[10:22:05] 2 tRNA-Met [56729,56805] 35 (cat)\\n[10:22:05] 1 tRNA-Asp c[103323,103396] 35 (gtc)\\n[10:22:05] 1 tRNA-Asn [2676,2749] 34 (gtt)\\n[10:22:05] 2 tRNA-Pro c[85961,86035] 35 (tgg)\\n[10:22:05] 1 tRNA-His [26842,26915] 34 (gtg)\\n[10:22:05] 1 tRNA-Glu c[5660,5732] 35 (ttc)\\n[10:22:05] 1 tRNA-Leu [33573,33655] 36 (gag)\\n[10:22:05] 1 tRNA-Gly [12491,12562] 33 (gcc)\\n[10:22:05] Found 41 tRNAs\\n[10:22:05] Predicting Ribosomal RNAs\\n[10:22:05] Running Barrnap with 4 threads\\n[10:22:07] 1 MRCA01000032.1 5 16S ribosomal RNA\\n[10:22:07] 2 MRCA01000032.1 1888 23S ribosomal RNA\\n[10:22:07] 3 MRCA01000032.1 4841 5S ribosomal RNA\\n[10:22:07] Found 3 rRNAs\\n[10:22:07] Skipping ncRNA search, enable with --rfam if desired.\\n[10:22:07] Total of 43 tRNA + rRNA features\\n[10:22:07] Searching for CRISPR repeats\\n[10:22:07] CRISPR1 MRCA01000001.1 256326 with 9 spacers\\n[10:22:07] CRISPR2 MRCA01000001.1 264540 with 8 spacers\\n[10:22:07] CRISPR3 MRCA01000026.1 20 with 64 spacers\\n[10:22:07] CRISPR4 MRCA01000027.1 28074 with 42 spacers\\n[10:22:07] Found 4 CRISPRs\\n[10:22:07] Predicting coding sequences\\n[10:22:07] Contigs total 5639496 bp, so using single mode\\n[10:22:07] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_MRCA01\\\\/\\\\/Fischerella_MRCA01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:22:14] Excluding CDS which overlaps existing RNA (repeat_region) at MRCA01000001.1:264408..264608 on + strand\\n[10:22:15] Excluding CDS which overlaps existing RNA (tRNA) at MRCA01000004.1:263609..264280 on - strand\\n[10:22:17] Excluding CDS which overlaps existing RNA (tRNA) at MRCA01000016.1:2550..3989 on + strand\\n[10:22:18] Excluding CDS which overlaps existing RNA (rRNA) at MRCA01000032.1:2283..2462 on + strand\\n[10:22:18] Found 4795 CDS\\n[10:22:18] Connecting features back to sequences\\n[10:22:18] Not using genus-specific database. Try --usegenus to enable it.\\n[10:22:18] Annotating CDS, please be patient.\\n[10:22:18] Will use 4 CPUs for similarity searching.\\n[10:22:21] There are still 4795 unannotated CDS left (started with 4795)\\n[10:22:21] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:22:21] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_MRCA01\\\\/\\\\/Fischerella_MRCA01\\\\.IS\\\\.tmp\\\\.784\\\\.faa | parallel --gnu --plain -j 4 --block 191186 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_MRCA01\\\\/\\\\/Fischerella_MRCA01\\\\.IS\\\\.tmp\\\\.784\\\\.blast 2> /dev/null\\n[10:22:56] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_MRCA01\\\\/\\\\/Fischerella_MRCA01\\\\.IS\\\\.tmp\\\\.784\\\\.faa | parallel --gnu --plain -j 4 --block 191186 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_MRCA01\\\\/\\\\/Fischerella_MRCA01\\\\.IS\\\\.tmp\\\\.784\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:22:56] This is prokka 1.14.6\\n[10:22:56] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:22:56] Homepage is https://github.com/tseemann/prokka\\n[10:22:56] Local time is Mon Feb 21 10:22:56 2022\\n[10:22:56] You are tiagoferreiraleao\\n[10:22:56] Operating system is darwin\\n[10:22:56] You have BioPerl 1.007002\\n[10:22:56] System has 4 cores.\\n[10:22:56] Option --cpu asked for 12 cores, but system only has 4\\n[10:22:56] Will use maximum of 4 cores.\\n[10:22:56] Annotating as >>> Bacteria <<<\\n[10:22:56] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAE01.fasta' contents.\\n[10:22:56] Setting --locustag GEPBFEFA from MD5 0e9bfefad05bd1dd469931490fd0cef2\\n[10:22:56] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAE01/\\n[10:22:56] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAE01\\\\/\\n[10:22:56] Using filename prefix: Fischerella_NHAE01.XXX\\n[10:22:56] Setting HMMER_NCPU=1\\n[10:22:56] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAE01//Fischerella_NHAE01.log\\n[10:22:56] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAE01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAE01/ --prefix Fischerella_NHAE01 --cpus 12\\n[10:22:56] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:22:56] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:22:56] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:22:56] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:22:57] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:22:57] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:22:57] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:22:57] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:22:57] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:22:57] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:22:57] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:22:57] Looking for 'egrep' - found /usr/bin/egrep\\n[10:22:57] Looking for 'find' - found /usr/bin/find\\n[10:22:57] Looking for 'grep' - found /usr/bin/grep\\n[10:22:57] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:22:57] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:22:57] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:22:57] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:22:57] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:22:57] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:22:57] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:22:57] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:22:57] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:22:57] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:22:58] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:22:58] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:22:58] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:22:58] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:22:58] Looking for 'sed' - found /usr/bin/sed\\n[10:22:58] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:22:58] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:22:58] Using genetic code table 11.\\n[10:22:58] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAE01.fasta\\n[10:22:58] Wrote 199 contigs totalling 5336250 bp.\\n[10:22:58] Predicting tRNAs and tmRNAs\\n[10:22:58] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAE01\\\\/\\\\/Fischerella_NHAE01\\\\.fna\\n[10:23:01] 1 tRNA-Asp [4041,4114] 35 (gtc)\\n[10:23:01] 1 tRNA-Thr c[1507,1578] 33 (cgt)\\n[10:23:01] 1 tRNA-Met c[26566,26642] 35 (cat)\\n[10:23:01] 2 tRNA-Val [29590,29663] 35 (gac)\\n[10:23:01] 1 tmRNA [26332,26718] 100,150 ANNIVKFARKEALVAA*\\n[10:23:01] 1 tRNA-Leu [71582,71664] 36 (gag)\\n[10:23:01] 1 tRNA-Trp [16213,16286] 34 (cca)\\n[10:23:01] 1 tRNA-Pro [18762,18836] 35 (tgg)\\n[10:23:01] 1 tRNA-Gln [17132,17203] 33 (ttg)\\n[10:23:01] 1 tRNA-Leu [12854,12936] 35 (cag)\\n[10:23:01] 1 tRNA-Leu c[119,203] 35 (caa)\\n[10:23:01] 1 tRNA-Cys [34690,34761] 33 (gca)\\n[10:23:01] 1 tRNA-Pro [13236,13309] 35 (ggg)\\n[10:23:01] 1 tRNA-Thr c[10308,10379] 33 (tgt)\\n[10:23:01] 1 tRNA-Leu c[696,779] 36 (tag)\\n[10:23:01] 1 tRNA-Asn c[26082,26155] 34 (gtt)\\n[10:23:01] 2 tRNA-Gly [46066,46137] 33 (gcc)\\n[10:23:01] 1 tRNA-Ser c[4208,4296] 36 (gga)\\n[10:23:01] 1 tRNA-Lys c[12802,12875] 34 (ttt)\\n[10:23:01] 1 tRNA-Met c[45413,45485] 34 (cat)\\n[10:23:01] 1 tRNA-Asn c[3585,3658] 34 (att)\\n[10:23:01] 2 tRNA-Arg c[4329,4401] 34 (tcg)\\n[10:23:01] 1 tRNA-Ala [26648,26722] 35 (cgc)\\n[10:23:01] 1 tRNA-Arg [11385,11459] 35 (tct)\\n[10:23:01] 1 tRNA-Thr c[296,367] 33 (ggt)\\n[10:23:01] 2 tRNA-Tyr c[431,515] 35 (gta)\\n[10:23:01] 1 tRNA-Gly [14366,14437] 33 (ccc)\\n[10:23:01] 1 tRNA-Pro c[14133,14206] 35 (cgg)\\n[10:23:01] 1 tRNA-Arg [42794,42867] 35 (ccg)\\n[10:23:01] 1 tRNA-Met [14982,15057] 36 (cat)\\n[10:23:01] 1 tRNA-Gly c[5037,5107] 33 (tcc)\\n[10:23:01] 1 tRNA-Ser [80292,80378] 36 (tga)\\n[10:23:01] 2 tRNA-Glu [88304,88376] 35 (ttc)\\n[10:23:01] 1 tRNA-Ala [639,712] 34 (ggc)\\n[10:23:01] 2 tRNA-Arg [5612,5685] 35 (acg)\\n[10:23:01] 1 tRNA-Phe c[40173,40245] 34 (gaa)\\n[10:23:01] 2 tRNA-Val [47585,47657] 34 (tac)\\n[10:23:01] 1 tRNA-Ser [26342,26435] 36 (gct)\\n[10:23:01] 2 tRNA-Gln c[187296,187366] 32 (ttg)\\n[10:23:01] 1 tRNA-Ala c[9013,9087] 34 (tgc)\\n[10:23:01] 2 tRNA-Ile c[9170,9245] 35 (gat)\\n[10:23:01] 3 tRNA-Ser [23503,23589] 36 (cga)\\n[10:23:01] 1 tRNA-His [61852,61925] 34 (gtg)\\n[10:23:01] 1 tRNA-Arg c[3697,3772] 36 (cct)\\n[10:23:01] Found 44 tRNAs\\n[10:23:01] Predicting Ribosomal RNAs\\n[10:23:01] Running Barrnap with 4 threads\\n[10:23:03] 1 NHAE01000179.1 56763 5S ribosomal RNA\\n[10:23:03] 2 NHAE01000187.1 5894 23S ribosomal RNA\\n[10:23:03] 3 NHAE01000187.1 9366 16S ribosomal RNA\\n[10:23:03] Found 3 rRNAs\\n[10:23:03] Skipping ncRNA search, enable with --rfam if desired.\\n[10:23:03] Total of 46 tRNA + rRNA features\\n[10:23:03] Searching for CRISPR repeats\\n[10:23:04] CRISPR1 NHAE01000092.1 4496 with 16 spacers\\n[10:23:04] CRISPR2 NHAE01000186.1 75 with 11 spacers\\n[10:23:04] CRISPR3 NHAE01000186.1 1129 with 17 spacers\\n[10:23:04] CRISPR4 NHAE01000188.1 72 with 14 spacers\\n[10:23:04] CRISPR5 NHAE01000188.1 1280 with 7 spacers\\n[10:23:04] CRISPR6 NHAE01000192.1 73 with 16 spacers\\n[10:23:04] CRISPR7 NHAE01000195.1 3 with 8 spacers\\n[10:23:04] CRISPR8 NHAE01000195.1 682 with 40 spacers\\n[10:23:04] CRISPR9 NHAE01000195.1 3800 with 14 spacers\\n[10:23:04] CRISPR10 NHAE01000195.1 4988 with 3 spacers\\n[10:23:04] CRISPR11 NHAE01000195.1 5280 with 15 spacers\\n[10:23:04] CRISPR12 NHAE01000197.1 8430 with 10 spacers\\n[10:23:04] CRISPR13 NHAE01000197.1 9319 with 6 spacers\\n[10:23:04] CRISPR14 NHAE01000197.1 10202 with 7 spacers\\n[10:23:04] CRISPR15 NHAE01000197.1 10903 with 5 spacers\\n[10:23:04] CRISPR16 NHAE01000197.1 11417 with 4 spacers\\n[10:23:04] CRISPR17 NHAE01000197.1 11846 with 5 spacers\\n[10:23:04] CRISPR18 NHAE01000197.1 12352 with 9 spacers\\n[10:23:04] CRISPR19 NHAE01000197.1 13123 with 7 spacers\\n[10:23:04] CRISPR20 NHAE01000197.1 13793 with 11 spacers\\n[10:23:04] CRISPR21 NHAE01000197.1 14701 with 11 spacers\\n[10:23:04] CRISPR22 NHAE01000197.1 15572 with 5 spacers\\n[10:23:04] CRISPR23 NHAE01000197.1 16077 with 8 spacers\\n[10:23:04] CRISPR24 NHAE01000198.1 143224 with 10 spacers\\n[10:23:04] CRISPR25 NHAE01000198.1 144168 with 3 spacers\\n[10:23:04] CRISPR26 NHAE01000198.1 144467 with 16 spacers\\n[10:23:04] CRISPR27 NHAE01000198.1 145765 with 3 spacers\\n[10:23:04] CRISPR28 NHAE01000198.1 146086 with 10 spacers\\n[10:23:04] CRISPR29 NHAE01000198.1 146930 with 25 spacers\\n[10:23:04] CRISPR30 NHAE01000198.1 148873 with 10 spacers\\n[10:23:04] CRISPR31 NHAE01000198.1 149767 with 3 spacers\\n[10:23:04] CRISPR32 NHAE01000198.1 150085 with 19 spacers\\n[10:23:04] CRISPR33 NHAE01000198.1 151577 with 25 spacers\\n[10:23:04] CRISPR34 NHAE01000199.1 73 with 12 spacers\\n[10:23:04] CRISPR35 NHAE01000199.1 1006 with 7 spacers\\n[10:23:04] CRISPR36 NHAE01000199.1 1628 with 6 spacers\\n[10:23:04] Found 36 CRISPRs\\n[10:23:04] Predicting coding sequences\\n[10:23:04] Contigs total 5336250 bp, so using single mode\\n[10:23:04] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAE01\\\\/\\\\/Fischerella_NHAE01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:23:10] Excluding CDS which overlaps existing RNA (tRNA) at NHAE01000003.1:3684..4079 on - strand\\n[10:23:12] Excluding CDS which overlaps existing RNA (tRNA) at NHAE01000098.1:24842..26281 on - strand\\n[10:23:13] Excluding CDS which overlaps existing RNA (tRNA) at NHAE01000171.1:5057..5728 on - strand\\n[10:23:14] Excluding CDS which overlaps existing RNA (rRNA) at NHAE01000187.1:6179..6355 on + strand\\n[10:23:14] Excluding CDS which overlaps existing RNA (rRNA) at NHAE01000187.1:8141..8320 on - strand\\n[10:23:15] Found 4534 CDS\\n[10:23:15] Connecting features back to sequences\\n[10:23:15] Not using genus-specific database. Try --usegenus to enable it.\\n[10:23:15] Annotating CDS, please be patient.\\n[10:23:15] Will use 4 CPUs for similarity searching.\\n[10:23:16] There are still 4534 unannotated CDS left (started with 4534)\\n[10:23:16] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:23:16] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAE01\\\\/\\\\/Fischerella_NHAE01\\\\.IS\\\\.tmp\\\\.913\\\\.faa | parallel --gnu --plain -j 4 --block 177589 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAE01\\\\/\\\\/Fischerella_NHAE01\\\\.IS\\\\.tmp\\\\.913\\\\.blast 2> /dev/null\\n[10:23:41] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAE01\\\\/\\\\/Fischerella_NHAE01\\\\.IS\\\\.tmp\\\\.913\\\\.faa | parallel --gnu --plain -j 4 --block 177589 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAE01\\\\/\\\\/Fischerella_NHAE01\\\\.IS\\\\.tmp\\\\.913\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:23:41] This is prokka 1.14.6\\n[10:23:41] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:23:41] Homepage is https://github.com/tseemann/prokka\\n[10:23:41] Local time is Mon Feb 21 10:23:41 2022\\n[10:23:41] You are tiagoferreiraleao\\n[10:23:41] Operating system is darwin\\n[10:23:41] You have BioPerl 1.007002\\n[10:23:42] System has 4 cores.\\n[10:23:42] Option --cpu asked for 12 cores, but system only has 4\\n[10:23:42] Will use maximum of 4 cores.\\n[10:23:42] Annotating as >>> Bacteria <<<\\n[10:23:42] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHC01.fasta' contents.\\n[10:23:42] Setting --locustag EKCOEFKG from MD5 e4c8ef4038309b2d37cc50786af493b1\\n[10:23:42] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHC01/\\n[10:23:42] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHC01\\\\/\\n[10:23:42] Using filename prefix: Nostoc_LAHC01.XXX\\n[10:23:42] Setting HMMER_NCPU=1\\n[10:23:42] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHC01//Nostoc_LAHC01.log\\n[10:23:42] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHC01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHC01/ --prefix Nostoc_LAHC01 --cpus 12\\n[10:23:42] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:23:42] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:23:42] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:23:42] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:23:42] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:23:42] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:23:42] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:23:42] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:23:42] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:23:42] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:23:42] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:23:42] Looking for 'egrep' - found /usr/bin/egrep\\n[10:23:42] Looking for 'find' - found /usr/bin/find\\n[10:23:42] Looking for 'grep' - found /usr/bin/grep\\n[10:23:42] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:23:42] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:23:42] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:23:42] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:23:42] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:23:42] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:23:42] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:23:42] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:23:43] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:23:43] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:23:43] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:23:43] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:23:43] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:23:43] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:23:43] Looking for 'sed' - found /usr/bin/sed\\n[10:23:43] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:23:43] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:23:43] Using genetic code table 11.\\n[10:23:43] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHC01.fasta\\n[10:23:44] Wrote 262 contigs totalling 9038860 bp.\\n[10:23:44] Predicting tRNAs and tmRNAs\\n[10:23:44] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHC01\\\\/\\\\/Nostoc_LAHC01\\\\.fna\\n[10:23:48] 1 tRNA-Cys [96547,96618] 33 (gca)\\n[10:23:48] 2 tRNA-His [246560,246633] 34 (gtg)\\n[10:23:48] 1 tRNA-Pro c[47065,47138] 35 (cgg)\\n[10:23:48] 1 tRNA-Gly c[75229,75299] 33 (tcc)\\n[10:23:48] 1 tRNA-Ala c[828,900] 34 (cgc)\\n[10:23:48] 2 tRNA-Leu [49602,49686] 35 (caa)\\n[10:23:48] 3 tRNA-Gln [54440,54513] 34 (ttg)\\n[10:23:48] 1 tRNA-Ala c[62667,62739] 34 (tgc)\\n[10:23:48] 1 tmRNA c[60441,60828] 101,151 ANNIVKFARREALVAA*\\n[10:23:48] 1 tRNA-Val c[14131,14203] 34 (tac)\\n[10:23:48] 1 tRNA-Leu [31467,31550] 36 (tag)\\n[10:23:48] 1 tRNA-Ala c[14795,14867] 34 (ggc)\\n[10:23:48] 1 tRNA-Leu c[66610,66691] 35 (cag)\\n[10:23:48] 1 tRNA-Ile c[45560,45635] 34 (tat)\\n[10:23:48] 2 tRNA-Met c[47241,47316] 34 (cat)\\n[10:23:48] 3 tRNA-Met c[47318,47393] 34 (cat)\\n[10:23:48] 1 tRNA-Ser c[28915,29008] 36 (gct)\\n[10:23:48] 2 tRNA-Lys c[34683,34753] 33 (ttt)\\n[10:23:48] 3 tRNA-Ser c[35012,35095] 34 (tga)\\n[10:23:48] 4 tRNA-His c[35100,35175] 35 (gtg)\\n[10:23:48] 5 tRNA-Ala c[35181,35255] 35 (tgc)\\n[10:23:48] 6 tRNA-Arg c[35347,35422] 35 (tct)\\n[10:23:48] 7 tRNA-Val c[35429,35504] 34 (tac)\\n[10:23:48] 8 tRNA-Thr c[35586,35658] 34 (tgt)\\n[10:23:48] 9 tRNA-Leu c[35953,36027] 35 (tag)\\n[10:23:48] 10 tRNA-Leu c[36139,36215] 34 (gag)\\n[10:23:48] 11 tRNA-Pro c[36323,36396] 35 (tgg)\\n[10:23:48] 12 tRNA-Leu c[36398,36475] 35 (taa)\\n[10:23:48] 13 tRNA-Leu c[36496,36573] 36 (caa)\\n[10:23:48] 14 tRNA-Gln c[36599,36674] 35 (ttg)\\n[10:23:48] 15 tRNA-Gln c[37549,37623] 35 (ctg)\\n[10:23:48] 16 tRNA-Asn c[37634,37708] 33 (gtt)\\n[10:23:48] 17 tRNA-Phe c[37714,37789] 36 (gaa)\\n[10:23:48] 18 tRNA-Arg c[37877,37952] 35 (acg)\\n[10:23:48] 19 tRNA-Ser c[38714,38800] 35 (gct)\\n[10:23:48] 20 tRNA-Tyr c[38804,38885] 35 (gta)\\n[10:23:48] 21 tRNA-Glu c[38896,38970] 35 (ttc)\\n[10:23:48] tRNA c[39058,39132] is a pseudo/wacky gene - skipping.\\n[10:23:48] 23 tRNA-Met c[39135,39208] 32 (cat)\\n[10:23:48] 24 tRNA-Glu c[39318,39394] 35 (ctc)\\n[10:23:48] 25 tRNA-Asp c[39398,39473] 36 (gtc)\\n[10:23:48] 1 tRNA-Ala c[78392,78466] 34 (tgc)\\n[10:23:48] 2 tRNA-Ile c[78550,78625] 35 (gat)\\n[10:23:48] 1 tRNA-Pro c[43704,43777] 35 (ggg)\\n[10:23:48] 1 tRNA-Ser [28823,28909] 36 (cga)\\n[10:23:48] 1 tRNA-Leu [61851,61931] 35 (gag)\\n[10:23:48] 1 tRNA-Thr [32094,32168] 33 (agt)\\n[10:23:48] 2 tRNA-Leu [33559,33633] 35 (caa)\\n[10:23:48] 1 tRNA-Lys c[11794,11865] 33 (ttt)\\n[10:23:48] 1 tRNA-Asn c[53386,53459] 34 (gtt)\\n[10:23:48] 1 tRNA-Thr [11579,11650] 33 (cgt)\\n[10:23:48] 2 tRNA-Trp [57259,57332] 34 (cca)\\n[10:23:48] 1 tRNA-Ser c[24552,24636] 34 (tga)\\n[10:23:48] 2 tRNA-His c[24647,24723] 35 (gtg)\\n[10:23:48] 3 tRNA-Ala c[24731,24803] 34 (tgc)\\n[10:23:48] 4 tRNA-Arg c[24808,24882] 35 (tct)\\n[10:23:48] 5 tRNA-Val c[24889,24964] 34 (tac)\\n[10:23:48] 6 tRNA-Gly c[24970,25040] 33 (tcc)\\n[10:23:48] 7 tRNA-Thr c[25046,25118] 34 (tgt)\\n[10:23:48] 8 tRNA-Cys c[25126,25203] 34 (gca)\\n[10:23:48] 9 tRNA-Leu c[25213,25288] 36 (taa)\\n[10:23:48] 10 tRNA-Lys c[25294,25368] 33 (ctt)\\n[10:23:48] 11 tRNA-Lys c[25374,25450] 34 (ttt)\\n[10:23:48] 12 tRNA-Leu c[25454,25531] 35 (gag)\\n[10:23:48] 13 tRNA-Leu c[25551,25625] 35 (tag)\\n[10:23:48] 14 tRNA-Pro c[25629,25705] 35 (tgg)\\n[10:23:48] 15 tRNA-Leu c[25710,25787] 35 (caa)\\n[10:23:48] 16 tRNA-Gln c[25795,25870] 35 (ttg)\\n[10:23:48] 17 tRNA-Asn c[25876,25950] 33 (gtt)\\n[10:23:48] 18 tRNA-Phe c[25959,26033] 35 (gaa)\\n[10:23:48] 19 tRNA-Arg c[26038,26113] 35 (acg)\\n[10:23:48] 20 tRNA-Ser c[26118,26207] 33 (gct)\\n[10:23:48] 21 tRNA-Tyr c[26211,26294] 36 (gta)\\n[10:23:48] 22 tRNA-Glu c[26298,26373] 34 (ttc)\\n[10:23:48] 23 tRNA-Pyl c[26404,26475] 33 (cta)\\n[10:23:48] 24 tRNA-Trp c[26500,26572] 34 (cca)\\n[10:23:48] 25 tRNA-Met c[26575,26649] 33 (cat)\\n[10:23:48] 26 tRNA-Cys c[26655,26730] 34 (gca)\\n[10:23:48] 27 tRNA-Glu c[26735,26811] 35 (ctc)\\n[10:23:48] 28 tRNA-Asp c[26814,26891] 38 (gtc)\\n[10:23:48] 1 tRNA-Ser [53941,54028] 35 (gga)\\n[10:23:48] 2 tRNA-Gly c[62173,62246] 34 (ccc)\\n[10:23:48] 1 tRNA-Gly c[28486,28557] 33 (gcc)\\n[10:23:48] 2 tRNA-Met c[35367,35440] 33 (cat)\\n[10:23:48] 1 tRNA-Thr c[19450,19521] 33 (ggt)\\n[10:23:48] 2 tRNA-Tyr c[19597,19681] 35 (gta)\\n[10:23:48] 3 tRNA-Ser [22132,22218] 36 (tga)\\n[10:23:48] 4 tRNA-Pro [30900,30973] 35 (tgg)\\n[10:23:48] 1 tRNA-Lys [27702,27774] 33 (ctt)\\n[10:23:48] 1 tRNA-Thr [30170,30242] 33 (tgt)\\n[10:23:48] 1 tRNA-Arg c[25324,25396] 34 (ccg)\\n[10:23:48] 1 tRNA-Met [11453,11527] 35 (cat)\\n[10:23:48] 1 tRNA-Glu c[3232,3305] 35 (ttc)\\n[10:23:48] 1 tRNA-Arg [1086,1159] 35 (acg)\\n[10:23:48] 1 tRNA-Asp [165,238] 35 (gtc)\\n[10:23:48] 1 tRNA-Arg c[5602,5676] 35 (tct)\\n[10:23:48] 1 tRNA-Val [9482,9555] 35 (gac)\\n[10:23:48] 1 tRNA-Ile [360,436] 35 (gat)\\n[10:23:48] 2 tRNA-Ala [457,532] 34 (tgc)\\n[10:23:48] 1 tRNA-Ala [3167,3253] 29 (cgc)\\n[10:23:48] Found 99 tRNAs\\n[10:23:48] Predicting Ribosomal RNAs\\n[10:23:48] Running Barrnap with 4 threads\\n[10:23:50] 1 LAHC01000033.1 75396 23S ribosomal RNA\\n[10:23:50] 2 LAHC01000033.1 78747 16S ribosomal RNA\\n[10:23:50] 3 LAHC01000242.1 29 5S ribosomal RNA\\n[10:23:50] 4 LAHC01000242.1 224 23S ribosomal RNA\\n[10:23:50] 5 LAHC01000244.1 862 23S ribosomal RNA (partial)\\n[10:23:50] Found 5 rRNAs\\n[10:23:50] Skipping ncRNA search, enable with --rfam if desired.\\n[10:23:50] Total of 103 tRNA + rRNA features\\n[10:23:50] Searching for CRISPR repeats\\n[10:23:51] CRISPR1 LAHC01000003.1 83391 with 7 spacers\\n[10:23:51] CRISPR2 LAHC01000016.1 55037 with 18 spacers\\n[10:23:51] CRISPR3 LAHC01000020.1 53201 with 10 spacers\\n[10:23:51] CRISPR4 LAHC01000023.1 35813 with 14 spacers\\n[10:23:51] CRISPR5 LAHC01000025.1 26529 with 15 spacers\\n[10:23:51] CRISPR6 LAHC01000029.1 29682 with 20 spacers\\n[10:23:51] CRISPR7 LAHC01000054.1 15850 with 24 spacers\\n[10:23:51] CRISPR8 LAHC01000123.1 1425 with 28 spacers\\n[10:23:51] CRISPR9 LAHC01000123.1 14859 with 43 spacers\\n[10:23:51] CRISPR10 LAHC01000160.1 10445 with 7 spacers\\n[10:23:51] CRISPR11 LAHC01000161.1 9912 with 18 spacers\\n[10:23:51] CRISPR12 LAHC01000193.1 3938 with 6 spacers\\n[10:23:51] CRISPR13 LAHC01000221.1 3917 with 23 spacers\\n[10:23:51] Found 13 CRISPRs\\n[10:23:51] Predicting coding sequences\\n[10:23:51] Contigs total 9038860 bp, so using single mode\\n[10:23:51] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHC01\\\\/\\\\/Nostoc_LAHC01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:24:02] Excluding CDS which overlaps existing RNA (tRNA) at LAHC01000001.1:246481..246624 on - strand\\n[10:24:03] Excluding CDS which overlaps existing RNA (repeat_region) at LAHC01000020.1:53410..53562 on + strand\\n[10:24:04] Excluding CDS which overlaps existing RNA (repeat_region) at LAHC01000023.1:36529..36645 on + strand\\n[10:24:07] Excluding CDS which overlaps existing RNA (repeat_region) at LAHC01000123.1:14900..15064 on - strand\\n[10:24:07] Excluding CDS which overlaps existing RNA (repeat_region) at LAHC01000160.1:10473..10598 on - strand\\n[10:24:07] Excluding CDS which overlaps existing RNA (repeat_region) at LAHC01000193.1:3002..3961 on + strand\\n[10:24:08] Excluding CDS which overlaps existing RNA (repeat_region) at LAHC01000221.1:3729..4007 on + strand\\n[10:24:08] Excluding CDS which overlaps existing RNA (rRNA) at LAHC01000242.1:355..813 on + strand\\n[10:24:08] Excluding CDS which overlaps existing RNA (rRNA) at LAHC01000242.1:2512..2853 on - strand\\n[10:24:08] Excluding CDS which overlaps existing RNA (tRNA) at LAHC01000244.1:379..744 on - strand\\n[10:24:08] Excluding CDS which overlaps existing RNA (tRNA) at LAHC01000262.1:2348..3718 on + strand\\n[10:24:08] Found 7624 CDS\\n[10:24:08] Connecting features back to sequences\\n[10:24:08] Not using genus-specific database. Try --usegenus to enable it.\\n[10:24:08] Annotating CDS, please be patient.\\n[10:24:08] Will use 4 CPUs for similarity searching.\\n[10:24:10] There are still 7624 unannotated CDS left (started with 7624)\\n[10:24:10] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:24:10] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHC01\\\\/\\\\/Nostoc_LAHC01\\\\.IS\\\\.tmp\\\\.1027\\\\.faa | parallel --gnu --plain -j 4 --block 304977 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHC01\\\\/\\\\/Nostoc_LAHC01\\\\.IS\\\\.tmp\\\\.1027\\\\.blast 2> /dev/null\\n[10:24:30] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHC01\\\\/\\\\/Nostoc_LAHC01\\\\.IS\\\\.tmp\\\\.1027\\\\.faa | parallel --gnu --plain -j 4 --block 304977 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHC01\\\\/\\\\/Nostoc_LAHC01\\\\.IS\\\\.tmp\\\\.1027\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:24:30] This is prokka 1.14.6\\n[10:24:30] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:24:30] Homepage is https://github.com/tseemann/prokka\\n[10:24:30] Local time is Mon Feb 21 10:24:30 2022\\n[10:24:30] You are tiagoferreiraleao\\n[10:24:30] Operating system is darwin\\n[10:24:30] You have BioPerl 1.007002\\n[10:24:30] System has 4 cores.\\n[10:24:30] Option --cpu asked for 12 cores, but system only has 4\\n[10:24:30] Will use maximum of 4 cores.\\n[10:24:30] Annotating as >>> Bacteria <<<\\n[10:24:30] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NRQW01.fasta' contents.\\n[10:24:30] Setting --locustag DLAOILMK from MD5 d5a8256499f3726cfacfa38219dd7d62\\n[10:24:30] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NRQW01/\\n[10:24:30] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NRQW01\\\\/\\n[10:24:30] Using filename prefix: Fischerella_NRQW01.XXX\\n[10:24:30] Setting HMMER_NCPU=1\\n[10:24:30] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NRQW01//Fischerella_NRQW01.log\\n[10:24:30] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NRQW01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NRQW01/ --prefix Fischerella_NRQW01 --cpus 12\\n[10:24:30] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:24:30] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:24:30] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:24:30] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:24:30] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:24:30] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:24:30] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:24:30] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:24:30] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:24:30] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:24:30] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:24:30] Looking for 'egrep' - found /usr/bin/egrep\\n[10:24:30] Looking for 'find' - found /usr/bin/find\\n[10:24:30] Looking for 'grep' - found /usr/bin/grep\\n[10:24:30] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:24:30] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:24:30] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:24:30] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:24:30] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:24:30] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:24:31] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:24:31] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:24:31] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:24:31] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:24:31] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:24:31] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:24:31] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:24:31] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:24:31] Looking for 'sed' - found /usr/bin/sed\\n[10:24:31] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:24:32] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:24:32] Using genetic code table 11.\\n[10:24:32] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NRQW01.fasta\\n[10:24:32] Wrote 753 contigs totalling 6758464 bp.\\n[10:24:32] Predicting tRNAs and tmRNAs\\n[10:24:32] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NRQW01\\\\/\\\\/Fischerella_NRQW01\\\\.fna\\n[10:24:34] 1 tRNA-Leu c[5390,5474] 35 (caa)\\n[10:24:34] 1 tRNA-Ser [10251,10344] 36 (gct)\\n[10:24:34] 1 tRNA-Asp c[14511,14584] 35 (gtc)\\n[10:24:34] 1 tRNA-Leu [17961,18043] 35 (cag)\\n[10:24:34] 1 tRNA-Gly c[3181,3252] 33 (gcc)\\n[10:24:34] 1 tmRNA c[11586,11972] 100,153 ANNIVKFARKEALVAA**\\n[10:24:34] 1 tRNA-Ser c[8528,8614] 36 (cga)\\n[10:24:34] 1 tRNA-Phe c[20081,20153] 34 (gaa)\\n[10:24:34] 1 tRNA-Pro [13855,13928] 35 (ggg)\\n[10:24:34] 1 tRNA-Cys c[15587,15658] 33 (gca)\\n[10:24:34] 1 tRNA-Thr c[2561,2632] 33 (tgt)\\n[10:24:34] 1 tRNA-Gln c[2913,2984] 33 (ttg)\\n[10:24:34] 1 tRNA-Arg [9090,9164] 35 (tct)\\n[10:24:34] 1 tRNA-Pro c[35,128] 34 (tgg)\\n[10:24:34] 1 tRNA-Ser c[4467,4552] 35 (tga)\\n[10:24:34] 1 tRNA-Leu c[952,1034] 35 (tag)\\n[10:24:34] 1 tRNA-Gly [3389,3458] 32 (tcc)\\n[10:24:34] 1 tRNA-Met [3205,3279] 35 (cat)\\n[10:24:34] 2 tRNA-Val c[22104,22177] 35 (gac)\\n[10:24:34] 3 tRNA-Met [25255,25331] 35 (cat)\\n[10:24:34] 1 tRNA-Val c[8458,8532] 35 (tac)\\n[10:24:34] 1 tRNA-Gly c[6170,6240] 33 (tcc)\\n[10:24:34] 1 tRNA-Glu c[3576,3648] 35 (ttc)\\n[10:24:34] 1 tRNA-Asn c[1411,1484] 34 (gtt)\\n[10:24:34] 1 tRNA-Gly [11923,11994] 33 (ccc)\\n[10:24:34] 1 tRNA-Leu [2256,2338] 36 (gag)\\n[10:24:34] 1 tRNA-Thr c[2825,2896] 33 (ggt)\\n[10:24:34] 2 tRNA-Tyr c[2960,3044] 35 (gta)\\n[10:24:34] 1 tRNA-Ala c[8463,8537] 35 (tgc)\\n[10:24:34] 2 tRNA-Ala c[8547,8619] 34 (ggc)\\n[10:24:34] 3 tRNA-Thr c[8903,8976] 35 (tgt)\\n[10:24:34] 4 tRNA-Met c[8980,9057] 35 (cat)\\n[10:24:34] 5 tRNA-Leu c[9059,9133] 35 (taa)\\n[10:24:34] 6 tRNA-Lys c[9136,9210] 34 (ctt)\\n[10:24:34] 7 tRNA-Lys c[9214,9287] 34 (ttt)\\n[10:24:34] 8 tRNA-Leu c[9292,9369] 35 (gag)\\n[10:24:34] 9 tRNA-Leu c[9374,9448] 35 (tag)\\n[10:24:34] 10 tRNA-Pro c[9452,9528] 35 (tgg)\\n[10:24:34] 11 tRNA-Leu c[9530,9607] 35 (caa)\\n[10:24:34] 12 tRNA-Gln c[9612,9686] 35 (ttg)\\n[10:24:34] 13 tRNA-Asn c[9777,9852] 34 (gtt)\\n[10:24:34] 14 tRNA-Phe c[9861,9938] 38 (gaa)\\n[10:24:34] 15 tRNA-Glu c[10022,10098] 35 (ctc)\\n[10:24:34] 16 tRNA-Asp c[10101,10177] 37 (gtc)\\n[10:24:35] 1 tRNA-Ala [5132,5206] 35 (cgc)\\n[10:24:35] 1 tRNA-Pro c[5975,6049] 35 (tgg)\\n[10:24:35] 1 tRNA-His c[13856,13929] 34 (gtg)\\n[10:24:35] 1 tRNA-Pro c[1723,1796] 35 (cgg)\\n[10:24:35] 1 tRNA-Trp [3732,3803] 34 (cca)\\n[10:24:35] 1 tRNA-Arg c[21870,21943] 35 (ccg)\\n[10:24:35] 1 tRNA-Trp [2079,2152] 34 (cca)\\n[10:24:35] 1 tRNA-Lys c[1507,1578] 33 (ttt)\\n[10:24:35] 1 tRNA-Ser c[23264,23352] 36 (gga)\\n[10:24:35] 1 tRNA-Ala c[207,281] 34 (tgc)\\n[10:24:35] 2 tRNA-Ile c[368,443] 35 (gat)\\n[10:24:35] 1 tRNA-Met [3503,3575] 34 (cat)\\n[10:24:35] 1 tRNA-Gly [694,767] 35 (tcc)\\n[10:24:35] 2 tRNA-Asn [1235,1308] 34 (att)\\n[10:24:35] 1 tRNA-Thr [18012,18083] 33 (cgt)\\n[10:24:35] 2 tRNA-Ala [19425,19497] 34 (ggc)\\n[10:24:35] 3 tRNA-Arg [26738,26811] 35 (acg)\\n[10:24:35] Found 61 tRNAs\\n[10:24:35] Predicting Ribosomal RNAs\\n[10:24:35] Running Barrnap with 4 threads\\n[10:24:36] 1 NRQW01000025.1 118 16S ribosomal RNA\\n[10:24:36] 2 NRQW01000645.1 4 5S ribosomal RNA\\n[10:24:36] 3 NRQW01000645.1 248 23S ribosomal RNA\\n[10:24:36] Found 3 rRNAs\\n[10:24:36] Skipping ncRNA search, enable with --rfam if desired.\\n[10:24:36] Total of 63 tRNA + rRNA features\\n[10:24:36] Searching for CRISPR repeats\\n[10:24:37] CRISPR1 NRQW01000024.1 11 with 77 spacers\\n[10:24:37] CRISPR2 NRQW01000080.1 9074 with 7 spacers\\n[10:24:37] CRISPR3 NRQW01000103.1 50 with 109 spacers\\n[10:24:37] CRISPR4 NRQW01000104.1 2694 with 7 spacers\\n[10:24:37] CRISPR5 NRQW01000106.1 3617 with 48 spacers\\n[10:24:37] CRISPR6 NRQW01000152.1 51 with 8 spacers\\n[10:24:37] CRISPR7 NRQW01000642.1 4304 with 90 spacers\\n[10:24:37] CRISPR8 NRQW01000716.1 2066 with 41 spacers\\n[10:24:37] Found 8 CRISPRs\\n[10:24:37] Predicting coding sequences\\n[10:24:37] Contigs total 6758464 bp, so using single mode\\n[10:24:37] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NRQW01\\\\/\\\\/Fischerella_NRQW01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:24:45] Excluding CDS which overlaps existing RNA (repeat_region) at NRQW01000103.1:7879..8058 on - strand\\n[10:24:45] Excluding CDS which overlaps existing RNA (repeat_region) at NRQW01000106.1:3396..3650 on + strand\\n[10:24:45] Excluding CDS which overlaps existing RNA (repeat_region) at NRQW01000152.1:324..533 on - strand\\n[10:24:47] Excluding CDS which overlaps existing RNA (tRNA) at NRQW01000496.1:13865..14110 on + strand\\n[10:24:47] Excluding CDS which overlaps existing RNA (tRNA) at NRQW01000642.1:1246..1467 on - strand\\n[10:24:47] Excluding CDS which overlaps existing RNA (repeat_region) at NRQW01000642.1:6054..6191 on - strand\\n[10:24:47] Excluding CDS which overlaps existing RNA (repeat_region) at NRQW01000642.1:7874..8062 on - strand\\n[10:24:47] Excluding CDS which overlaps existing RNA (repeat_region) at NRQW01000642.1:9319..9702 on + strand\\n[10:24:47] Excluding CDS which overlaps existing RNA (repeat_region) at NRQW01000642.1:10779..10979 on - strand\\n[10:24:48] Excluding CDS which overlaps existing RNA (rRNA) at NRQW01000645.1:2544..2936 on + strand\\n[10:24:48] Excluding CDS which overlaps existing RNA (tRNA) at NRQW01000695.1:26192..26854 on - strand\\n[10:24:48] Excluding CDS which overlaps existing RNA (repeat_region) at NRQW01000716.1:2089..2514 on + strand\\n[10:24:48] Found 5803 CDS\\n[10:24:48] Connecting features back to sequences\\n[10:24:48] Not using genus-specific database. Try --usegenus to enable it.\\n[10:24:48] Annotating CDS, please be patient.\\n[10:24:48] Will use 4 CPUs for similarity searching.\\n[10:24:49] There are still 5803 unannotated CDS left (started with 5803)\\n[10:24:49] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:24:49] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NRQW01\\\\/\\\\/Fischerella_NRQW01\\\\.IS\\\\.tmp\\\\.1139\\\\.faa | parallel --gnu --plain -j 4 --block 223005 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NRQW01\\\\/\\\\/Fischerella_NRQW01\\\\.IS\\\\.tmp\\\\.1139\\\\.blast 2> /dev/null\\n[10:25:09] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NRQW01\\\\/\\\\/Fischerella_NRQW01\\\\.IS\\\\.tmp\\\\.1139\\\\.faa | parallel --gnu --plain -j 4 --block 223005 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NRQW01\\\\/\\\\/Fischerella_NRQW01\\\\.IS\\\\.tmp\\\\.1139\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:25:09] This is prokka 1.14.6\\n[10:25:09] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:25:09] Homepage is https://github.com/tseemann/prokka\\n[10:25:09] Local time is Mon Feb 21 10:25:09 2022\\n[10:25:09] You are tiagoferreiraleao\\n[10:25:09] Operating system is darwin\\n[10:25:09] You have BioPerl 1.007002\\n[10:25:09] System has 4 cores.\\n[10:25:09] Option --cpu asked for 12 cores, but system only has 4\\n[10:25:09] Will use maximum of 4 cores.\\n[10:25:09] Annotating as >>> Bacteria <<<\\n[10:25:09] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQI01.fasta' contents.\\n[10:25:09] Setting --locustag DNBLIMHD from MD5 d7b5261d684da372932a1d58ccdb3aec\\n[10:25:09] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQI01/\\n[10:25:09] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQI01\\\\/\\n[10:25:09] Using filename prefix: Fischerella_NMQI01.XXX\\n[10:25:09] Setting HMMER_NCPU=1\\n[10:25:09] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQI01//Fischerella_NMQI01.log\\n[10:25:09] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQI01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQI01/ --prefix Fischerella_NMQI01 --cpus 12\\n[10:25:09] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:25:09] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:25:09] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:25:09] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:25:10] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:25:10] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:25:10] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:25:10] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:25:10] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:25:10] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:25:10] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:25:10] Looking for 'egrep' - found /usr/bin/egrep\\n[10:25:10] Looking for 'find' - found /usr/bin/find\\n[10:25:10] Looking for 'grep' - found /usr/bin/grep\\n[10:25:10] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:25:10] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:25:10] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:25:10] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:25:10] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:25:10] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:25:10] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:25:10] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:25:10] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:25:10] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:25:11] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:25:11] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:25:11] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:25:11] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:25:11] Looking for 'sed' - found /usr/bin/sed\\n[10:25:11] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:25:11] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:25:11] Using genetic code table 11.\\n[10:25:11] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQI01.fasta\\n[10:25:11] Wrote 745 contigs totalling 5725024 bp.\\n[10:25:11] Predicting tRNAs and tmRNAs\\n[10:25:11] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQI01\\\\/\\\\/Fischerella_NMQI01\\\\.fna\\n[10:25:13] 1 tRNA-Met c[17351,17427] 35 (cat)\\n[10:25:13] 2 tRNA-Met c[17433,17505] 33 (cat)\\n[10:25:13] 1 tRNA-Leu [146,228] 35 (cag)\\n[10:25:13] 1 tmRNA c[105,491] 100,150 ANNIVKFARKEALVAA*\\n[10:25:13] 1 tRNA-Met c[3406,3478] 34 (cat)\\n[10:25:13] 1 tRNA-Ser [4502,4587] 35 (tga)\\n[10:25:13] 1 tRNA-Thr [2183,2254] 33 (tgt)\\n[10:25:13] 1 tRNA-Arg [907,979] 34 (tcg)\\n[10:25:13] 2 tRNA-Asn [1679,1752] 34 (att)\\n[10:25:13] 1 tRNA-Arg [3699,3772] 35 (ccg)\\n[10:25:13] 1 tRNA-Leu c[554,638] 35 (caa)\\n[10:25:13] 1 tRNA-Ser [4596,4682] 36 (cga)\\n[10:25:13] 1 tRNA-Gly [3300,3371] 33 (ccc)\\n[10:25:13] 1 tRNA-Trp c[6331,6404] 34 (cca)\\n[10:25:13] 1 tRNA-Val [6346,6418] 34 (tac)\\n[10:25:13] 1 tRNA-Pro c[6262,6335] 35 (ggg)\\n[10:25:13] 1 tRNA-Glu [3204,3276] 35 (ttc)\\n[10:25:13] 1 tRNA-Gly c[392,463] 33 (gcc)\\n[10:25:13] 1 tRNA-Thr [20885,20956] 33 (cgt)\\n[10:25:13] 2 tRNA-Ala [22328,22401] 34 (ggc)\\n[10:25:13] 1 tRNA-Arg c[8222,8295] 35 (acg)\\n[10:25:13] 1 tRNA-Ser [1192,1280] 36 (gga)\\n[10:25:13] 1 tRNA-Asn c[25713,25786] 34 (gtt)\\n[10:25:13] 1 tRNA-Arg [1158,1232] 35 (tct)\\n[10:25:13] 1 tRNA-His c[1211,1284] 34 (gtg)\\n[10:25:13] 1 tRNA-Cys c[4421,4492] 33 (gca)\\n[10:25:13] 1 tRNA-Gln [10760,10831] 33 (ttg)\\n[10:25:13] 1 tRNA-Leu [4842,4924] 36 (gag)\\n[10:25:13] 1 tRNA-Met [5540,5615] 36 (cat)\\n[10:25:13] 1 tRNA-Thr c[5522,5593] 33 (ggt)\\n[10:25:13] 2 tRNA-Tyr c[5656,5740] 35 (gta)\\n[10:25:13] 1 tRNA-Leu c[28521,28604] 36 (tag)\\n[10:25:13] 1 tRNA-Gly [6586,6656] 33 (tcc)\\n[10:25:13] 1 tRNA-Arg [2164,2239] 36 (cct)\\n[10:25:14] 1 tRNA-Ala c[249,323] 35 (cgc)\\n[10:25:14] 1 tRNA-Asp c[3459,3532] 35 (gtc)\\n[10:25:14] 1 tRNA-Pro [5362,5436] 35 (tgg)\\n[10:25:14] 1 tRNA-Met c[124,200] 35 (cat)\\n[10:25:14] 2 tRNA-Val [3180,3253] 35 (gac)\\n[10:25:14] 1 tRNA-Ser [2652,2745] 36 (gct)\\n[10:25:14] 1 tRNA-Lys c[8030,8103] 34 (ttt)\\n[10:25:14] 1 tRNA-Pro [12826,12899] 35 (cgg)\\n[10:25:14] Found 42 tRNAs\\n[10:25:14] Predicting Ribosomal RNAs\\n[10:25:14] Running Barrnap with 4 threads\\n[10:25:15] Found 0 rRNAs\\n[10:25:15] Skipping ncRNA search, enable with --rfam if desired.\\n[10:25:15] Total of 41 tRNA + rRNA features\\n[10:25:15] Searching for CRISPR repeats\\n[10:25:15] CRISPR1 NMQI01000249.1 151 with 10 spacers\\n[10:25:16] Found 1 CRISPRs\\n[10:25:16] Predicting coding sequences\\n[10:25:16] Contigs total 5725024 bp, so using single mode\\n[10:25:16] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQI01\\\\/\\\\/Fischerella_NMQI01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:25:21] Excluding CDS which overlaps existing RNA (tRNA) at NMQI01000100.1:721..1008 on + strand\\n[10:25:22] Excluding CDS which overlaps existing RNA (tRNA) at NMQI01000186.1:6265..6426 on + strand\\n[10:25:22] Excluding CDS which overlaps existing RNA (tRNA) at NMQI01000243.1:24473..25912 on - strand\\n[10:25:23] Found 4831 CDS\\n[10:25:23] Connecting features back to sequences\\n[10:25:23] Not using genus-specific database. Try --usegenus to enable it.\\n[10:25:23] Annotating CDS, please be patient.\\n[10:25:23] Will use 4 CPUs for similarity searching.\\n[10:25:24] There are still 4831 unannotated CDS left (started with 4831)\\n[10:25:24] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:25:24] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQI01\\\\/\\\\/Fischerella_NMQI01\\\\.IS\\\\.tmp\\\\.1251\\\\.faa | parallel --gnu --plain -j 4 --block 190227 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQI01\\\\/\\\\/Fischerella_NMQI01\\\\.IS\\\\.tmp\\\\.1251\\\\.blast 2> /dev/null\\n[10:25:51] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQI01\\\\/\\\\/Fischerella_NMQI01\\\\.IS\\\\.tmp\\\\.1251\\\\.faa | parallel --gnu --plain -j 4 --block 190227 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQI01\\\\/\\\\/Fischerella_NMQI01\\\\.IS\\\\.tmp\\\\.1251\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:25:51] This is prokka 1.14.6\\n[10:25:51] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:25:51] Homepage is https://github.com/tseemann/prokka\\n[10:25:51] Local time is Mon Feb 21 10:25:51 2022\\n[10:25:51] You are tiagoferreiraleao\\n[10:25:51] Operating system is darwin\\n[10:25:51] You have BioPerl 1.007002\\n[10:25:51] System has 4 cores.\\n[10:25:51] Option --cpu asked for 12 cores, but system only has 4\\n[10:25:51] Will use maximum of 4 cores.\\n[10:25:51] Annotating as >>> Bacteria <<<\\n[10:25:51] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Amazonocrinis_JAECZC01.fasta' contents.\\n[10:25:52] Setting --locustag GBGLGPDC from MD5 0b0509dcc23269104a913a2664cdded7\\n[10:25:52] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Amazonocrinis_JAECZC01/\\n[10:25:52] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Amazonocrinis_JAECZC01\\\\/\\n[10:25:52] Using filename prefix: Amazonocrinis_JAECZC01.XXX\\n[10:25:52] Setting HMMER_NCPU=1\\n[10:25:52] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Amazonocrinis_JAECZC01//Amazonocrinis_JAECZC01.log\\n[10:25:52] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Amazonocrinis_JAECZC01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Amazonocrinis_JAECZC01/ --prefix Amazonocrinis_JAECZC01 --cpus 12\\n[10:25:52] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:25:52] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:25:52] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:25:52] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:25:52] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:25:52] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:25:52] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:25:52] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:25:52] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:25:52] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:25:52] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:25:52] Looking for 'egrep' - found /usr/bin/egrep\\n[10:25:52] Looking for 'find' - found /usr/bin/find\\n[10:25:52] Looking for 'grep' - found /usr/bin/grep\\n[10:25:52] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:25:52] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:25:52] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:25:52] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:25:52] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:25:52] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:25:52] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:25:52] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:25:52] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:25:52] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:25:52] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:25:52] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:25:52] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:25:52] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:25:52] Looking for 'sed' - found /usr/bin/sed\\n[10:25:52] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:25:53] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:25:53] Using genetic code table 11.\\n[10:25:53] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Amazonocrinis_JAECZC01.fasta\\n[10:25:53] Wrote 114 contigs totalling 8256472 bp.\\n[10:25:53] Predicting tRNAs and tmRNAs\\n[10:25:53] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Amazonocrinis_JAECZC01\\\\/\\\\/Amazonocrinis_JAECZC01\\\\.fna\\n[10:25:56] 1 tRNA-Val c[197180,197253] 35 (gac)\\n[10:25:56] 2 tRNA-Lys c[253348,253419] 33 (ttt)\\n[10:25:56] 3 tRNA-Lys c[318556,318627] 33 (ttt)\\n[10:25:56] 4 tRNA-Lys c[318635,318708] 35 (ctt)\\n[10:25:56] 5 tRNA-Ser c[318838,318921] 34 (tga)\\n[10:25:56] 6 tRNA-His c[318927,319002] 35 (gtg)\\n[10:25:56] 7 tRNA-Ala c[319008,319082] 35 (tgc)\\n[10:25:56] 8 tRNA-Arg c[319194,319269] 35 (tct)\\n[10:25:56] 9 tRNA-Val c[319278,319350] 33 (tac)\\n[10:25:56] 10 tRNA-Gly c[319540,319610] 33 (tcc)\\n[10:25:56] 11 tRNA-Asn c[319831,319904] 34 (gtt)\\n[10:25:56] 12 tRNA-Phe c[319910,319985] 36 (gaa)\\n[10:25:56] 13 tRNA-Arg c[320065,320141] 36 (acg)\\n[10:25:56] 14 tRNA-Ser c[320147,320234] 35 (gct)\\n[10:25:56] 15 tRNA-Tyr c[320238,320321] 37 (gta)\\n[10:25:56] 16 tRNA-Glu c[320327,320400] 35 (ttc)\\n[10:25:56] tRNA c[320404,320477] is a pseudo/wacky gene - skipping.\\n[10:25:56] 18 tRNA-Met c[320479,320554] 34 (cat)\\n[10:25:56] 19 tRNA-Glu c[320668,320744] 35 (ctc)\\n[10:25:56] 20 tRNA-Asp c[320747,320822] 36 (gtc)\\n[10:25:56] 21 tRNA-Glu c[327593,327666] 35 (ttc)\\n[10:25:56] 1 tRNA-Ser [280025,280111] 36 (cga)\\n[10:25:56] 1 tRNA-Asn c[14027,14116] 35 (gtt)\\n[10:25:56] 2 tRNA-Gly c[92922,92992] 33 (tcc)\\n[10:25:56] 1 tRNA-Ser [198197,198281] 34 (tga)\\n[10:25:56] 1 tRNA-Trp [25156,25229] 34 (cca)\\n[10:25:56] 2 tRNA-Met [66399,66475] 35 (cat)\\n[10:25:56] 1 tRNA-Arg c[54603,54678] 36 (cct)\\n[10:25:56] 1 tRNA-Met c[127061,127135] 35 (cat)\\n[10:25:56] 1 tRNA-Ala c[119606,119678] 34 (tgc)\\n[10:25:56] 1 tRNA-Gln c[111709,111782] 34 (ttg)\\n[10:25:56] 2 tRNA-Leu c[112349,112433] 35 (caa)\\n[10:25:56] 1 tRNA-Arg [79865,79937] 34 (ccg)\\n[10:25:56] 1 tRNA-Gly [54623,54694] 33 (ccc)\\n[10:25:56] 1 tRNA-Thr c[928,1020] 33 (cgt)\\n[10:25:56] 2 tRNA-Ser [5987,6075] 36 (gga)\\n[10:25:56] 1 tRNA-Leu c[11656,11735] 34 (aag)\\n[10:25:56] 1 tRNA-Thr c[22275,22346] 33 (cgt)\\n[10:25:56] 1 tRNA-Ala [19356,19428] 34 (cgc)\\n[10:25:56] 2 tRNA-Lys c[27412,27484] 33 (ctt)\\n[10:25:56] 1 tRNA-Asn [14834,14926] 37 (att)\\n[10:25:56] 1 tRNA-Met [15155,15228] 33 (cat)\\n[10:25:56] 1 tRNA-Leu [36966,37048] 36 (gag)\\n[10:25:56] 1 tRNA-Leu c[178888,178971] 36 (tag)\\n[10:25:56] 2 tRNA-Thr c[208398,208469] 33 (ggt)\\n[10:25:56] 3 tRNA-Tyr c[208523,208605] 35 (gta)\\n[10:25:56] 1 tRNA-Arg [23443,23516] 35 (acg)\\n[10:25:56] 2 tRNA-Ser c[30533,30624] 35 (gct)\\n[10:25:56] 1 tRNA-Ser c[74094,74180] 36 (tga)\\n[10:25:56] 2 tRNA-Ala c[83379,83453] 34 (tgc)\\n[10:25:56] 3 tRNA-Ile c[83537,83612] 35 (gat)\\n[10:25:56] 1 tRNA-Val [66763,66835] 34 (tac)\\n[10:25:56] 1 tRNA-Asp c[14860,14933] 35 (gtc)\\n[10:25:56] 1 tmRNA [14419,14805] 101,154 ANNIVKFARKEALVAA**\\n[10:25:56] 2 tRNA-Arg [29477,29551] 35 (tct)\\n[10:25:56] 1 tRNA-Leu c[34277,34351] 35 (taa)\\n[10:25:56] 2 tRNA-Pyl c[36003,36076] 33 (cta)\\n[10:25:56] 1 tRNA-Asn [2126,2197] 33 (gtt)\\n[10:25:56] 1 tRNA-Thr c[843,914] 33 (tgt)\\n[10:25:56] 1 tRNA-His c[334,407] 34 (gtg)\\n[10:25:56] 1 tRNA-Val c[35411,35482] 33 (cac)\\n[10:25:56] 1 tRNA-Pro c[41230,41303] 35 (tgg)\\n[10:25:56] 2 tRNA-Ala c[90742,90814] 34 (ggc)\\n[10:25:56] 3 tRNA-Gly c[176691,176762] 33 (gcc)\\n[10:25:56] 4 tRNA-Pro [222030,222103] 35 (ggg)\\n[10:25:56] 1 tRNA-Ala c[2247,2321] 34 (tgc)\\n[10:25:56] 2 tRNA-Ile c[2404,2479] 35 (gat)\\n[10:25:56] 1 tRNA-Phe c[49360,49432] 34 (gaa)\\n[10:25:56] 1 tRNA-Cys [68946,69019] 34 (gca)\\n[10:25:56] 2 tRNA-Leu c[201153,201234] 35 (cag)\\n[10:25:56] 1 tRNA-Pro [6663,6736] 35 (cgg)\\n[10:25:56] Found 70 tRNAs\\n[10:25:56] Predicting Ribosomal RNAs\\n[10:25:56] Running Barrnap with 4 threads\\n[10:25:59] 1 JAECZC010000059.1 80320 23S ribosomal RNA\\n[10:25:59] 2 JAECZC010000059.1 83733 16S ribosomal RNA\\n[10:25:59] 3 JAECZC010000091.1 2599 16S ribosomal RNA (partial)\\n[10:25:59] Found 3 rRNAs\\n[10:25:59] Skipping ncRNA search, enable with --rfam if desired.\\n[10:25:59] Total of 72 tRNA + rRNA features\\n[10:25:59] Searching for CRISPR repeats\\n[10:26:00] CRISPR1 JAECZC010000008.1 35588 with 24 spacers\\n[10:26:00] CRISPR2 JAECZC010000023.1 17112 with 3 spacers\\n[10:26:00] CRISPR3 JAECZC010000026.1 53252 with 7 spacers\\n[10:26:00] CRISPR4 JAECZC010000032.1 48461 with 27 spacers\\n[10:26:00] CRISPR5 JAECZC010000050.1 51244 with 17 spacers\\n[10:26:00] CRISPR6 JAECZC010000055.1 27067 with 26 spacers\\n[10:26:00] CRISPR7 JAECZC010000055.1 41837 with 31 spacers\\n[10:26:00] CRISPR8 JAECZC010000057.1 23437 with 31 spacers\\n[10:26:00] CRISPR9 JAECZC010000067.1 19819 with 42 spacers\\n[10:26:00] CRISPR10 JAECZC010000077.1 7099 with 8 spacers\\n[10:26:00] CRISPR11 JAECZC010000094.1 1065 with 8 spacers\\n[10:26:00] CRISPR12 JAECZC010000108.1 126596 with 4 spacers\\n[10:26:00] CRISPR13 JAECZC010000111.1 137845 with 18 spacers\\n[10:26:00] Found 13 CRISPRs\\n[10:26:00] Predicting coding sequences\\n[10:26:00] Contigs total 8256472 bp, so using single mode\\n[10:26:00] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Amazonocrinis_JAECZC01\\\\/\\\\/Amazonocrinis_JAECZC01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:26:09] Excluding CDS which overlaps existing RNA (tRNA) at JAECZC010000001.1:320540..320806 on - strand\\n[10:26:10] Excluding CDS which overlaps existing RNA (tRNA) at JAECZC010000003.1:13804..14967 on - strand\\n[10:26:10] Excluding CDS which overlaps existing RNA (tRNA) at JAECZC010000007.1:198161..198346 on - strand\\n[10:26:12] Excluding CDS which overlaps existing RNA (repeat_region) at JAECZC010000032.1:48581..49225 on + strand\\n[10:26:12] Excluding CDS which overlaps existing RNA (repeat_region) at JAECZC010000032.1:50122..50580 on + strand\\n[10:26:13] Excluding CDS which overlaps existing RNA (tRNA) at JAECZC010000037.1:11646..12329 on + strand\\n[10:26:13] Excluding CDS which overlaps existing RNA (repeat_region) at JAECZC010000050.1:51735..52031 on - strand\\n[10:26:13] Excluding CDS which overlaps existing RNA (repeat_region) at JAECZC010000055.1:43494..43622 on - strand\\n[10:26:13] Excluding CDS which overlaps existing RNA (tRNA) at JAECZC010000055.1:178888..179169 on + strand\\n[10:26:14] Excluding CDS which overlaps existing RNA (rRNA) at JAECZC010000059.1:82186..82590 on + strand\\n[10:26:14] Excluding CDS which overlaps existing RNA (rRNA) at JAECZC010000059.1:84589..84798 on - strand\\n[10:26:14] Excluding CDS which overlaps existing RNA (repeat_region) at JAECZC010000067.1:21208..21549 on + strand\\n[10:26:14] Excluding CDS which overlaps existing RNA (repeat_region) at JAECZC010000067.1:22244..22441 on + strand\\n[10:26:14] Excluding CDS which overlaps existing RNA (repeat_region) at JAECZC010000067.1:22534..22923 on + strand\\n[10:26:14] Excluding CDS which overlaps existing RNA (tRNA) at JAECZC010000072.1:326..541 on + strand\\n[10:26:14] Excluding CDS which overlaps existing RNA (repeat_region) at JAECZC010000094.1:1076..1222 on - strand\\n[10:26:14] Excluding CDS which overlaps existing RNA (repeat_region) at JAECZC010000094.1:1477..1656 on - strand\\n[10:26:15] Found 6889 CDS\\n[10:26:15] Connecting features back to sequences\\n[10:26:15] Not using genus-specific database. Try --usegenus to enable it.\\n[10:26:15] Annotating CDS, please be patient.\\n[10:26:15] Will use 4 CPUs for similarity searching.\\n[10:26:17] There are still 6889 unannotated CDS left (started with 6889)\\n[10:26:17] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:26:17] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Amazonocrinis_JAECZC01\\\\/\\\\/Amazonocrinis_JAECZC01\\\\.IS\\\\.tmp\\\\.1363\\\\.faa | parallel --gnu --plain -j 4 --block 273887 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Amazonocrinis_JAECZC01\\\\/\\\\/Amazonocrinis_JAECZC01\\\\.IS\\\\.tmp\\\\.1363\\\\.blast 2> /dev/null\\n[10:26:26] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Amazonocrinis_JAECZC01\\\\/\\\\/Amazonocrinis_JAECZC01\\\\.IS\\\\.tmp\\\\.1363\\\\.faa | parallel --gnu --plain -j 4 --block 273887 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Amazonocrinis_JAECZC01\\\\/\\\\/Amazonocrinis_JAECZC01\\\\.IS\\\\.tmp\\\\.1363\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:26:26] This is prokka 1.14.6\\n[10:26:26] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:26:26] Homepage is https://github.com/tseemann/prokka\\n[10:26:26] Local time is Mon Feb 21 10:26:26 2022\\n[10:26:26] You are tiagoferreiraleao\\n[10:26:26] Operating system is darwin\\n[10:26:26] You have BioPerl 1.007002\\n[10:26:26] System has 4 cores.\\n[10:26:26] Option --cpu asked for 12 cores, but system only has 4\\n[10:26:26] Will use maximum of 4 cores.\\n[10:26:26] Annotating as >>> Bacteria <<<\\n[10:26:26] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKG01.fasta' contents.\\n[10:26:26] Setting --locustag OLFONEGP from MD5 85f87e09219b4d57ef72a451ef2db248\\n[10:26:26] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKG01/\\n[10:26:26] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKG01\\\\/\\n[10:26:26] Using filename prefix: Nostoc_JABXKG01.XXX\\n[10:26:26] Setting HMMER_NCPU=1\\n[10:26:26] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKG01//Nostoc_JABXKG01.log\\n[10:26:26] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKG01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKG01/ --prefix Nostoc_JABXKG01 --cpus 12\\n[10:26:26] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:26:26] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:26:26] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:26:26] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:26:26] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:26:26] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:26:27] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:26:27] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:26:27] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:26:27] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:26:27] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:26:27] Looking for 'egrep' - found /usr/bin/egrep\\n[10:26:27] Looking for 'find' - found /usr/bin/find\\n[10:26:27] Looking for 'grep' - found /usr/bin/grep\\n[10:26:27] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:26:27] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:26:27] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:26:27] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:26:27] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:26:27] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:26:27] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:26:27] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:26:27] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:26:27] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:26:27] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:26:27] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:26:27] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:26:27] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:26:27] Looking for 'sed' - found /usr/bin/sed\\n[10:26:27] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:26:28] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:26:28] Using genetic code table 11.\\n[10:26:28] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKG01.fasta\\n[10:26:28] Wrote 204 contigs totalling 7881170 bp.\\n[10:26:28] Predicting tRNAs and tmRNAs\\n[10:26:28] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKG01\\\\/\\\\/Nostoc_JABXKG01\\\\.fna\\n[10:26:32] 1 tRNA-Leu [5412,5492] 34 (cag)\\n[10:26:32] 1 tRNA-Phe c[28284,28356] 34 (gaa)\\n[10:26:32] 1 tRNA-Gln c[13992,14063] 33 (ttg)\\n[10:26:32] 2 tRNA-Leu c[15739,15823] 35 (caa)\\n[10:26:32] 1 tRNA-Arg c[31171,31247] 36 (cct)\\n[10:26:32] 2 tRNA-Val c[50444,50517] 35 (gac)\\n[10:26:32] 1 tRNA-Arg [34780,34853] 35 (acg)\\n[10:26:32] 2 tRNA-Cys [133194,133264] 33 (gca)\\n[10:26:32] 1 tRNA-His c[305,379] 35 (gtg)\\n[10:26:32] 2 tRNA-Ala c[385,459] 35 (tgc)\\n[10:26:32] 3 tRNA-Arg c[496,571] 35 (tct)\\n[10:26:32] 4 tRNA-Val c[578,653] 34 (tac)\\n[10:26:32] 5 tRNA-Gly c[659,730] 34 (gcc)\\n[10:26:32] 6 tRNA-Thr c[734,808] 35 (tgt)\\n[10:26:32] 7 tRNA-Met c[816,892] 34 (cat)\\n[10:26:32] 8 tRNA-Leu c[902,976] 35 (tag)\\n[10:26:32] 9 tRNA-Pro c[979,1052] 34 (tgg)\\n[10:26:32] 10 tRNA-Leu c[1234,1311] 35 (taa)\\n[10:26:32] 11 tRNA-Leu c[1318,1394] 35 (caa)\\n[10:26:32] 12 tRNA-Gln c[1401,1476] 34 (ttg)\\n[10:26:32] 13 tRNA-Gln c[1480,1552] 34 (ctg)\\n[10:26:32] 14 tRNA-Asn c[1703,1777] 33 (gtt)\\n[10:26:32] 15 tRNA-Phe c[1783,1858] 36 (gaa)\\n[10:26:32] 16 tRNA-Arg c[1879,1955] 36 (acg)\\n[10:26:32] 17 tRNA-Ser c[1958,2049] 36 (gct)\\n[10:26:32] 18 tRNA-Tyr c[2052,2135] 37 (gta)\\n[10:26:32] 19 tRNA-Glu c[2143,2216] 35 (ttc)\\n[10:26:32] tRNA c[2334,2408] is a pseudo/wacky gene - skipping.\\n[10:26:32] 21 tRNA-Met c[2411,2485] 33 (cat)\\n[10:26:32] 22 tRNA-Cys c[2496,2568] 35 (gca)\\n[10:26:32] 23 tRNA-Glu c[9837,9910] 35 (ttc)\\n[10:26:32] 1 tRNA-Ala c[92807,92881] 35 (ggc)\\n[10:26:32] 1 tRNA-His c[1833,1906] 34 (gtg)\\n[10:26:32] 1 tRNA-Gly c[33922,33992] 33 (tcc)\\n[10:26:32] 2 tRNA-Ser [56758,56844] 36 (gga)\\n[10:26:32] 1 tRNA-Val [40634,40706] 34 (tac)\\n[10:26:32] 2 tRNA-Ser c[105304,105397] 36 (gct)\\n[10:26:32] 3 tRNA-Lys c[110857,110927] 33 (ttt)\\n[10:26:32] 1 tRNA-Gly [8508,8579] 33 (ccc)\\n[10:26:32] 1 tRNA-Asn [2409,2480] 33 (gtt)\\n[10:26:32] 1 tRNA-Lys c[2722,2794] 33 (ctt)\\n[10:26:32] 1 tRNA-Met [968,1043] 34 (cat)\\n[10:26:32] 2 tRNA-Met [1043,1117] 33 (cat)\\n[10:26:32] 1 tmRNA [95913,96300] 101,151 ANNIVNFARKDALVAA*\\n[10:26:32] tRNA [16944,17015] is a pseudo/wacky gene - skipping.\\n[10:26:32] 2 tRNA-Ser [18377,18449] 33 (tga)\\n[10:26:32] 1 tRNA-Thr c[21732,21803] 33 (ggt)\\n[10:26:32] 2 tRNA-Tyr c[21891,21976] 35 (gta)\\n[10:26:32] 3 tRNA-Ser [24275,24361] 36 (tga)\\n[10:26:32] 1 tRNA-Met [31238,31311] 33 (cat)\\n[10:26:32] 2 tRNA-Thr c[33442,33513] 33 (tgt)\\n[10:26:32] 1 tRNA-Leu [65392,65475] 36 (tag)\\n[10:26:32] 1 tRNA-Thr [45392,45463] 33 (cgt)\\n[10:26:32] 1 tRNA-Pro [94517,94590] 35 (tgg)\\n[10:26:32] 1 tRNA-Arg c[37953,38025] 34 (ccg)\\n[10:26:32] 1 tRNA-Pro [43715,43788] 35 (cgg)\\n[10:26:32] 1 tRNA-Gly [127504,127575] 33 (gcc)\\n[10:26:32] tRNA c[100959,101031] is a pseudo/wacky gene - skipping.\\n[10:26:32] 1 tRNA-Lys [15145,15216] 33 (ttt)\\n[10:26:32] 1 tRNA-Leu [15564,15646] 36 (gag)\\n[10:26:32] 1 tRNA-Asp [20420,20493] 35 (gtc)\\n[10:26:32] 1 tRNA-Trp [52388,52461] 34 (cca)\\n[10:26:32] 1 tRNA-Ala [11311,11385] 35 (tgc)\\n[10:26:32] 1 tRNA-Arg [62678,62752] 35 (tct)\\n[10:26:32] 1 tRNA-Ser c[39014,39100] 36 (cga)\\n[10:26:32] 2 tRNA-Met c[47430,47504] 35 (cat)\\n[10:26:32] 3 tRNA-Pro [60102,60175] 35 (ggg)\\n[10:26:32] 1 tRNA-Ala c[7378,7452] 35 (cgc)\\n[10:26:32] Found 65 tRNAs\\n[10:26:32] Predicting Ribosomal RNAs\\n[10:26:32] Running Barrnap with 4 threads\\n[10:26:34] 1 JABXKG010000082.1 1 16S ribosomal RNA (partial)\\n[10:26:34] Found 1 rRNAs\\n[10:26:34] Skipping ncRNA search, enable with --rfam if desired.\\n[10:26:34] Total of 65 tRNA + rRNA features\\n[10:26:34] Searching for CRISPR repeats\\n[10:26:34] CRISPR1 JABXKG010000016.1 21823 with 13 spacers\\n[10:26:35] CRISPR2 JABXKG010000146.1 59963 with 11 spacers\\n[10:26:35] CRISPR3 JABXKG010000169.1 31168 with 15 spacers\\n[10:26:35] CRISPR4 JABXKG010000184.1 11707 with 6 spacers\\n[10:26:35] Found 4 CRISPRs\\n[10:26:35] Predicting coding sequences\\n[10:26:35] Contigs total 7881170 bp, so using single mode\\n[10:26:35] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKG01\\\\/\\\\/Nostoc_JABXKG01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:26:42] Excluding CDS which overlaps existing RNA (tRNA) at JABXKG010000005.1:5426..5605 on + strand\\n[10:26:46] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKG010000146.1:60462..60599 on + strand\\n[10:26:46] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKG010000146.1:60702..60944 on - strand\\n[10:26:46] Excluding CDS which overlaps existing RNA (tRNA) at JABXKG010000160.1:15113..15346 on - strand\\n[10:26:47] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKG010000184.1:10438..12750 on + strand\\n[10:26:47] Found 6992 CDS\\n[10:26:47] Connecting features back to sequences\\n[10:26:47] Not using genus-specific database. Try --usegenus to enable it.\\n[10:26:47] Annotating CDS, please be patient.\\n[10:26:47] Will use 4 CPUs for similarity searching.\\n[10:26:49] There are still 6992 unannotated CDS left (started with 6992)\\n[10:26:49] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:26:49] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKG01\\\\/\\\\/Nostoc_JABXKG01\\\\.IS\\\\.tmp\\\\.1475\\\\.faa | parallel --gnu --plain -j 4 --block 265781 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKG01\\\\/\\\\/Nostoc_JABXKG01\\\\.IS\\\\.tmp\\\\.1475\\\\.blast 2> /dev/null\\n[10:26:59] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKG01\\\\/\\\\/Nostoc_JABXKG01\\\\.IS\\\\.tmp\\\\.1475\\\\.faa | parallel --gnu --plain -j 4 --block 265781 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKG01\\\\/\\\\/Nostoc_JABXKG01\\\\.IS\\\\.tmp\\\\.1475\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:27:00] This is prokka 1.14.6\\n[10:27:00] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:27:00] Homepage is https://github.com/tseemann/prokka\\n[10:27:00] Local time is Mon Feb 21 10:27:00 2022\\n[10:27:00] You are tiagoferreiraleao\\n[10:27:00] Operating system is darwin\\n[10:27:00] You have BioPerl 1.007002\\n[10:27:00] System has 4 cores.\\n[10:27:00] Option --cpu asked for 12 cores, but system only has 4\\n[10:27:00] Will use maximum of 4 cores.\\n[10:27:00] Annotating as >>> Bacteria <<<\\n[10:27:00] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAI01.fasta' contents.\\n[10:27:00] Setting --locustag IFMKCMOH from MD5 2f64c68175490d5ae200f0847b53ffab\\n[10:27:00] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAI01/\\n[10:27:00] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAI01\\\\/\\n[10:27:00] Using filename prefix: Fischerella_NHAI01.XXX\\n[10:27:00] Setting HMMER_NCPU=1\\n[10:27:00] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAI01//Fischerella_NHAI01.log\\n[10:27:00] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAI01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAI01/ --prefix Fischerella_NHAI01 --cpus 12\\n[10:27:00] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:27:00] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:27:00] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:27:00] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:27:00] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:27:00] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:27:00] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:27:00] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:27:00] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:27:00] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:27:00] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:27:00] Looking for 'egrep' - found /usr/bin/egrep\\n[10:27:00] Looking for 'find' - found /usr/bin/find\\n[10:27:00] Looking for 'grep' - found /usr/bin/grep\\n[10:27:00] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:27:00] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:27:00] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:27:00] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:27:00] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:27:00] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:27:01] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:27:01] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:27:01] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:27:01] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:27:01] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:27:01] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:27:01] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:27:01] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:27:01] Looking for 'sed' - found /usr/bin/sed\\n[10:27:01] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:27:02] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:27:02] Using genetic code table 11.\\n[10:27:02] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAI01.fasta\\n[10:27:02] Wrote 119 contigs totalling 5304471 bp.\\n[10:27:02] Predicting tRNAs and tmRNAs\\n[10:27:02] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAI01\\\\/\\\\/Fischerella_NHAI01\\\\.fna\\n[10:27:05] 1 tRNA-Arg [6841,6913] 34 (tcg)\\n[10:27:05] 2 tRNA-Asn [7584,7657] 34 (att)\\n[10:27:05] 1 tRNA-Ala [942,1015] 34 (ggc)\\n[10:27:05] 2 tRNA-Arg [5907,5980] 35 (acg)\\n[10:27:05] 1 tRNA-Gln [5692,5762] 32 (ttg)\\n[10:27:05] 1 tRNA-Met c[45325,45397] 34 (cat)\\n[10:27:05] 1 tRNA-Phe c[1441,1513] 34 (gaa)\\n[10:27:05] 2 tRNA-Val [8912,8984] 34 (tac)\\n[10:27:05] 1 tRNA-Arg c[35881,35956] 36 (cct)\\n[10:27:05] 1 tRNA-Gly c[92989,93060] 33 (gcc)\\n[10:27:05] 1 tRNA-Gly [14333,14404] 33 (ccc)\\n[10:27:05] 2 tRNA-Thr c[32400,32471] 33 (ggt)\\n[10:27:05] 3 tRNA-Tyr c[32535,32619] 35 (gta)\\n[10:27:05] 1 tRNA-His c[51187,51260] 34 (gtg)\\n[10:27:05] 1 tRNA-Pro c[81336,81409] 35 (cgg)\\n[10:27:05] 1 tRNA-Gln c[23383,23454] 33 (ttg)\\n[10:27:05] 1 tRNA-Pro c[60860,60933] 35 (ggg)\\n[10:27:05] 1 tRNA-Arg [11429,11503] 35 (tct)\\n[10:27:05] 2 tRNA-Met [36390,36465] 36 (cat)\\n[10:27:05] 1 tRNA-Ser c[43979,44065] 36 (cga)\\n[10:27:05] 1 tRNA-Ser [15774,15860] 36 (tga)\\n[10:27:05] 1 tRNA-Thr [58239,58310] 33 (tgt)\\n[10:27:05] 1 tRNA-Thr [62211,62282] 33 (cgt)\\n[10:27:05] 1 tRNA-Asn [2873,2946] 34 (gtt)\\n[10:27:05] 1 tRNA-Pro c[23409,23483] 35 (tgg)\\n[10:27:05] 1 tRNA-Met c[11623,11699] 35 (cat)\\n[10:27:05] 2 tRNA-Val [14647,14720] 35 (gac)\\n[10:27:05] 1 tmRNA c[233,619] 100,150 ANNIVKFARKEALVAA*\\n[10:27:05] 1 tRNA-Leu [43541,43624] 36 (tag)\\n[10:27:05] 1 tRNA-Trp c[6310,6383] 34 (cca)\\n[10:27:05] 2 tRNA-Arg c[50538,50611] 35 (ccg)\\n[10:27:05] 1 tRNA-Asp [16022,16095] 35 (gtc)\\n[10:27:05] 1 tRNA-Gly [12149,12219] 33 (tcc)\\n[10:27:05] 2 tRNA-Cys [62885,62956] 33 (gca)\\n[10:27:05] 1 tRNA-Lys [21325,21398] 34 (ttt)\\n[10:27:05] 1 tRNA-Leu c[31375,31457] 36 (gag)\\n[10:27:05] 1 tRNA-Ala [78883,78957] 35 (cgc)\\n[10:27:05] 2 tRNA-Leu [103443,103525] 35 (cag)\\n[10:27:05] 1 tRNA-Leu c[1047,1131] 35 (caa)\\n[10:27:05] 1 tRNA-Glu [99630,99702] 35 (ttc)\\n[10:27:05] 1 tRNA-Ser c[53532,53625] 36 (gct)\\n[10:27:05] 1 tRNA-Ser [91055,91143] 36 (gga)\\n[10:27:05] Found 42 tRNAs\\n[10:27:05] Predicting Ribosomal RNAs\\n[10:27:05] Running Barrnap with 4 threads\\n[10:27:07] 1 NHAI01000019.1 116 16S ribosomal RNA\\n[10:27:07] 2 NHAI01000105.1 84 23S ribosomal RNA\\n[10:27:07] Found 2 rRNAs\\n[10:27:07] Skipping ncRNA search, enable with --rfam if desired.\\n[10:27:07] Total of 43 tRNA + rRNA features\\n[10:27:07] Searching for CRISPR repeats\\n[10:27:07] CRISPR1 NHAI01000078.1 43133 with 3 spacers\\n[10:27:08] CRISPR2 NHAI01000101.1 2926 with 3 spacers\\n[10:27:08] CRISPR3 NHAI01000101.1 2926 with 20 spacers\\n[10:27:08] CRISPR4 NHAI01000109.1 8168 with 8 spacers\\n[10:27:08] CRISPR5 NHAI01000109.1 8168 with 16 spacers\\n[10:27:08] CRISPR6 NHAI01000109.1 9569 with 12 spacers\\n[10:27:08] CRISPR7 NHAI01000109.1 10460 with 3 spacers\\n[10:27:08] CRISPR8 NHAI01000109.1 10772 with 12 spacers\\n[10:27:08] CRISPR9 NHAI01000109.1 11753 with 26 spacers\\n[10:27:08] CRISPR10 NHAI01000115.1 9382 with 8 spacers\\n[10:27:08] CRISPR11 NHAI01000115.1 10029 with 14 spacers\\n[10:27:08] CRISPR12 NHAI01000116.1 29662 with 26 spacers\\n[10:27:08] CRISPR13 NHAI01000116.1 31842 with 12 spacers\\n[10:27:08] CRISPR14 NHAI01000116.1 32790 with 12 spacers\\n[10:27:08] CRISPR15 NHAI01000117.1 2889 with 5 spacers\\n[10:27:08] CRISPR16 NHAI01000117.1 3290 with 12 spacers\\n[10:27:08] CRISPR17 NHAI01000117.1 3292 with 37 spacers\\n[10:27:08] CRISPR18 NHAI01000117.1 6028 with 33 spacers\\n[10:27:08] CRISPR19 NHAI01000117.1 8624 with 6 spacers\\n[10:27:08] CRISPR20 NHAI01000118.1 1004 with 4 spacers\\n[10:27:08] Found 20 CRISPRs\\n[10:27:08] Predicting coding sequences\\n[10:27:08] Contigs total 5304471 bp, so using single mode\\n[10:27:08] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAI01\\\\/\\\\/Fischerella_NHAI01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:27:14] Excluding CDS which overlaps existing RNA (tRNA) at NHAI01000020.1:5352..6023 on - strand\\n[10:27:15] Excluding CDS which overlaps existing RNA (tRNA) at NHAI01000065.1:2747..4186 on + strand\\n[10:27:17] Excluding CDS which overlaps existing RNA (rRNA) at NHAI01000105.1:2331..2510 on - strand\\n[10:27:17] Excluding CDS which overlaps existing RNA (repeat_region) at NHAI01000109.1:7969..8184 on + strand\\n[10:27:17] Found 4507 CDS\\n[10:27:17] Connecting features back to sequences\\n[10:27:17] Not using genus-specific database. Try --usegenus to enable it.\\n[10:27:17] Annotating CDS, please be patient.\\n[10:27:17] Will use 4 CPUs for similarity searching.\\n[10:27:19] There are still 4507 unannotated CDS left (started with 4507)\\n[10:27:19] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:27:19] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAI01\\\\/\\\\/Fischerella_NHAI01\\\\.IS\\\\.tmp\\\\.1589\\\\.faa | parallel --gnu --plain -j 4 --block 177716 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAI01\\\\/\\\\/Fischerella_NHAI01\\\\.IS\\\\.tmp\\\\.1589\\\\.blast 2> /dev/null\\n[10:27:43] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAI01\\\\/\\\\/Fischerella_NHAI01\\\\.IS\\\\.tmp\\\\.1589\\\\.faa | parallel --gnu --plain -j 4 --block 177716 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAI01\\\\/\\\\/Fischerella_NHAI01\\\\.IS\\\\.tmp\\\\.1589\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:27:43] This is prokka 1.14.6\\n[10:27:43] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:27:43] Homepage is https://github.com/tseemann/prokka\\n[10:27:43] Local time is Mon Feb 21 10:27:43 2022\\n[10:27:43] You are tiagoferreiraleao\\n[10:27:43] Operating system is darwin\\n[10:27:43] You have BioPerl 1.007002\\n[10:27:44] System has 4 cores.\\n[10:27:44] Option --cpu asked for 12 cores, but system only has 4\\n[10:27:44] Will use maximum of 4 cores.\\n[10:27:44] Annotating as >>> Bacteria <<<\\n[10:27:44] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZW01.fasta' contents.\\n[10:27:44] Setting --locustag PHFDAFDI from MD5 91fdafd29ef94918d411357ad7bf8ff0\\n[10:27:44] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZW01/\\n[10:27:44] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZW01\\\\/\\n[10:27:44] Using filename prefix: Fischerella_NGZW01.XXX\\n[10:27:44] Setting HMMER_NCPU=1\\n[10:27:44] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZW01//Fischerella_NGZW01.log\\n[10:27:44] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZW01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZW01/ --prefix Fischerella_NGZW01 --cpus 12\\n[10:27:44] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:27:44] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:27:44] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:27:44] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:27:44] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:27:44] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:27:44] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:27:44] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:27:44] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:27:44] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:27:44] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:27:44] Looking for 'egrep' - found /usr/bin/egrep\\n[10:27:44] Looking for 'find' - found /usr/bin/find\\n[10:27:44] Looking for 'grep' - found /usr/bin/grep\\n[10:27:44] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:27:44] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:27:44] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:27:44] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:27:44] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:27:44] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:27:44] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:27:44] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:27:44] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:27:44] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:27:45] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:27:45] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:27:45] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:27:45] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:27:45] Looking for 'sed' - found /usr/bin/sed\\n[10:27:45] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:27:45] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:27:45] Using genetic code table 11.\\n[10:27:45] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZW01.fasta\\n[10:27:45] Wrote 197 contigs totalling 5398744 bp.\\n[10:27:45] Predicting tRNAs and tmRNAs\\n[10:27:45] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZW01\\\\/\\\\/Fischerella_NGZW01\\\\.fna\\n[10:27:47] 1 tRNA-Gln [34451,34522] 33 (ttg)\\n[10:27:47] 1 tRNA-Arg [30384,30459] 36 (cct)\\n[10:27:47] 1 tRNA-Cys c[87,158] 33 (gca)\\n[10:27:47] 1 tRNA-Met [9107,9179] 34 (cat)\\n[10:27:47] 1 tmRNA c[243,629] 100,150 ANNIVKFARKEALVAA*\\n[10:27:47] 1 tRNA-Asn c[17806,17879] 34 (gtt)\\n[10:27:47] 1 tRNA-Leu c[23166,23249] 36 (tag)\\n[10:27:47] 2 tRNA-Ala c[72590,72664] 34 (tgc)\\n[10:27:47] 3 tRNA-Ile c[72747,72822] 35 (gat)\\n[10:27:47] 4 tRNA-Ser [79243,79329] 36 (cga)\\n[10:27:47] 1 tRNA-Gln c[36358,36428] 32 (ttg)\\n[10:27:47] 1 tRNA-Ala [18635,18709] 35 (cgc)\\n[10:27:47] 1 tRNA-Pro [49051,49124] 35 (ggg)\\n[10:27:47] 1 tRNA-Leu c[447,531] 35 (caa)\\n[10:27:47] 1 tRNA-Ser [51470,51563] 36 (gct)\\n[10:27:47] 1 tRNA-Asp [16034,16107] 35 (gtc)\\n[10:27:47] 1 tRNA-Val c[15953,16026] 35 (gac)\\n[10:27:47] 2 tRNA-Met [18973,19049] 35 (cat)\\n[10:27:47] 1 tRNA-Thr c[10910,10981] 33 (tgt)\\n[10:27:47] 1 tRNA-Leu [33109,33191] 36 (gag)\\n[10:27:47] 1 tRNA-Arg c[28276,28349] 35 (ccg)\\n[10:27:47] 1 tRNA-Arg c[9940,10014] 35 (tct)\\n[10:27:47] 1 tRNA-Leu c[2228,2310] 35 (cag)\\n[10:27:47] 1 tRNA-Ser c[23720,23806] 36 (tga)\\n[10:27:47] 1 tRNA-Arg [6787,6859] 34 (tcg)\\n[10:27:47] 2 tRNA-Asn [7530,7603] 34 (att)\\n[10:27:47] 1 tRNA-Lys c[90424,90497] 34 (ttt)\\n[10:27:47] 1 tRNA-Thr [6818,6889] 33 (cgt)\\n[10:27:47] 1 tRNA-Pro c[2194,2267] 35 (cgg)\\n[10:27:47] 1 tRNA-Ser c[4218,4306] 36 (gga)\\n[10:27:47] 1 tRNA-Val c[11713,11785] 34 (tac)\\n[10:27:47] 2 tRNA-Phe [19125,19197] 34 (gaa)\\n[10:27:47] 1 tRNA-Gly [453,524] 33 (gcc)\\n[10:27:47] 1 tRNA-Trp [16305,16378] 34 (cca)\\n[10:27:47] 1 tRNA-Glu [11393,11465] 35 (ttc)\\n[10:27:47] 1 tRNA-Gly [35493,35564] 33 (ccc)\\n[10:27:47] 1 tRNA-Met c[36691,36766] 36 (cat)\\n[10:27:47] 1 tRNA-Gly c[11231,11301] 33 (tcc)\\n[10:27:47] 1 tRNA-Pro [19756,19830] 35 (tgg)\\n[10:27:47] 1 tRNA-Ala [641,714] 34 (ggc)\\n[10:27:47] 2 tRNA-Arg [5764,5837] 35 (acg)\\n[10:27:47] 1 tRNA-His c[91853,91926] 34 (gtg)\\n[10:27:47] 2 tRNA-Tyr [154930,155014] 35 (gta)\\n[10:27:47] 3 tRNA-Thr [155239,155310] 33 (ggt)\\n[10:27:47] Found 44 tRNAs\\n[10:27:47] Predicting Ribosomal RNAs\\n[10:27:47] Running Barrnap with 4 threads\\n[10:27:48] 1 NGZW01000017.1 69242 5S ribosomal RNA\\n[10:27:48] 2 NGZW01000017.1 69471 23S ribosomal RNA\\n[10:27:48] 3 NGZW01000017.1 72943 16S ribosomal RNA\\n[10:27:48] Found 3 rRNAs\\n[10:27:48] Skipping ncRNA search, enable with --rfam if desired.\\n[10:27:48] Total of 46 tRNA + rRNA features\\n[10:27:48] Searching for CRISPR repeats\\n[10:27:49] CRISPR1 NGZW01000017.1 23492 with 3 spacers\\n[10:27:49] CRISPR2 NGZW01000176.1 3053 with 10 spacers\\n[10:27:49] CRISPR3 NGZW01000176.1 4067 with 19 spacers\\n[10:27:49] CRISPR4 NGZW01000176.1 5701 with 18 spacers\\n[10:27:49] CRISPR5 NGZW01000176.1 7143 with 4 spacers\\n[10:27:49] CRISPR6 NGZW01000176.1 7717 with 4 spacers\\n[10:27:49] CRISPR7 NGZW01000176.1 8077 with 23 spacers\\n[10:27:49] CRISPR8 NGZW01000176.1 10068 with 5 spacers\\n[10:27:49] CRISPR9 NGZW01000176.1 10923 with 14 spacers\\n[10:27:49] CRISPR10 NGZW01000176.1 12084 with 3 spacers\\n[10:27:49] CRISPR11 NGZW01000177.1 6 with 9 spacers\\n[10:27:49] CRISPR12 NGZW01000177.1 818 with 10 spacers\\n[10:27:49] CRISPR13 NGZW01000177.1 1753 with 7 spacers\\n[10:27:49] CRISPR14 NGZW01000177.1 2370 with 27 spacers\\n[10:27:49] CRISPR15 NGZW01000177.1 4515 with 16 spacers\\n[10:27:49] CRISPR16 NGZW01000177.1 5747 with 5 spacers\\n[10:27:49] CRISPR17 NGZW01000177.1 6172 with 14 spacers\\n[10:27:49] CRISPR18 NGZW01000181.1 1849 with 7 spacers\\n[10:27:49] CRISPR19 NGZW01000181.1 2526 with 3 spacers\\n[10:27:49] CRISPR20 NGZW01000181.1 2921 with 6 spacers\\n[10:27:49] CRISPR21 NGZW01000184.1 1 with 4 spacers\\n[10:27:49] CRISPR22 NGZW01000185.1 20092 with 21 spacers\\n[10:27:49] CRISPR23 NGZW01000185.1 21811 with 49 spacers\\n[10:27:49] CRISPR24 NGZW01000185.1 25490 with 7 spacers\\n[10:27:49] CRISPR25 NGZW01000185.1 26178 with 4 spacers\\n[10:27:49] CRISPR26 NGZW01000185.1 26559 with 9 spacers\\n[10:27:49] CRISPR27 NGZW01000192.1 2297 with 14 spacers\\n[10:27:49] CRISPR28 NGZW01000192.1 3423 with 13 spacers\\n[10:27:49] CRISPR29 NGZW01000192.1 4606 with 17 spacers\\n[10:27:49] CRISPR30 NGZW01000192.1 6010 with 25 spacers\\n[10:27:49] CRISPR31 NGZW01000192.1 7980 with 7 spacers\\n[10:27:49] CRISPR32 NGZW01000192.1 8664 with 5 spacers\\n[10:27:49] CRISPR33 NGZW01000193.1 13894 with 12 spacers\\n[10:27:49] CRISPR34 NGZW01000193.1 14928 with 14 spacers\\n[10:27:49] CRISPR35 NGZW01000193.1 14928 with 27 spacers\\n[10:27:49] CRISPR36 NGZW01000193.1 17100 with 33 spacers\\n[10:27:49] CRISPR37 NGZW01000193.1 19718 with 4 spacers\\n[10:27:49] CRISPR38 NGZW01000193.1 20158 with 13 spacers\\n[10:27:49] CRISPR39 NGZW01000193.1 21251 with 4 spacers\\n[10:27:49] CRISPR40 NGZW01000193.1 21740 with 5 spacers\\n[10:27:49] CRISPR41 NGZW01000193.1 22219 with 12 spacers\\n[10:27:49] CRISPR42 NGZW01000193.1 23321 with 22 spacers\\n[10:27:49] CRISPR43 NGZW01000193.1 25069 with 7 spacers\\n[10:27:49] Found 43 CRISPRs\\n[10:27:49] Predicting coding sequences\\n[10:27:49] Contigs total 5398744 bp, so using single mode\\n[10:27:49] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZW01\\\\/\\\\/Fischerella_NGZW01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:27:54] Excluding CDS which overlaps existing RNA (tRNA) at NGZW01000016.1:16566..18005 on - strand\\n[10:27:54] Excluding CDS which overlaps existing RNA (rRNA) at NGZW01000017.1:69756..69932 on + strand\\n[10:27:54] Excluding CDS which overlaps existing RNA (rRNA) at NGZW01000017.1:71718..71897 on - strand\\n[10:27:56] Excluding CDS which overlaps existing RNA (repeat_region) at NGZW01000193.1:18048..18641 on - strand\\n[10:27:56] Excluding CDS which overlaps existing RNA (tRNA) at NGZW01000194.1:5209..5880 on - strand\\n[10:27:57] Found 4567 CDS\\n[10:27:57] Connecting features back to sequences\\n[10:27:57] Not using genus-specific database. Try --usegenus to enable it.\\n[10:27:57] Annotating CDS, please be patient.\\n[10:27:57] Will use 4 CPUs for similarity searching.\\n[10:27:58] There are still 4567 unannotated CDS left (started with 4567)\\n[10:27:58] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:27:58] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZW01\\\\/\\\\/Fischerella_NGZW01\\\\.IS\\\\.tmp\\\\.1703\\\\.faa | parallel --gnu --plain -j 4 --block 177683 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZW01\\\\/\\\\/Fischerella_NGZW01\\\\.IS\\\\.tmp\\\\.1703\\\\.blast 2> /dev/null\\n[10:28:10] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZW01\\\\/\\\\/Fischerella_NGZW01\\\\.IS\\\\.tmp\\\\.1703\\\\.faa | parallel --gnu --plain -j 4 --block 177683 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZW01\\\\/\\\\/Fischerella_NGZW01\\\\.IS\\\\.tmp\\\\.1703\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:28:11] This is prokka 1.14.6\\n[10:28:11] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:28:11] Homepage is https://github.com/tseemann/prokka\\n[10:28:11] Local time is Mon Feb 21 10:28:11 2022\\n[10:28:11] You are tiagoferreiraleao\\n[10:28:11] Operating system is darwin\\n[10:28:11] You have BioPerl 1.007002\\n[10:28:11] System has 4 cores.\\n[10:28:11] Option --cpu asked for 12 cores, but system only has 4\\n[10:28:11] Will use maximum of 4 cores.\\n[10:28:11] Annotating as >>> Bacteria <<<\\n[10:28:11] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKN01.fasta' contents.\\n[10:28:11] Setting --locustag DMNEBBKJ from MD5 d67ebb43082f4ed423196a80d136f573\\n[10:28:11] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKN01/\\n[10:28:11] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKN01\\\\/\\n[10:28:11] Using filename prefix: Nostoc_JABXKN01.XXX\\n[10:28:11] Setting HMMER_NCPU=1\\n[10:28:11] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKN01//Nostoc_JABXKN01.log\\n[10:28:11] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKN01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKN01/ --prefix Nostoc_JABXKN01 --cpus 12\\n[10:28:11] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:28:11] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:28:11] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:28:11] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:28:11] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:28:11] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:28:11] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:28:11] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:28:11] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:28:11] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:28:11] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:28:11] Looking for 'egrep' - found /usr/bin/egrep\\n[10:28:11] Looking for 'find' - found /usr/bin/find\\n[10:28:11] Looking for 'grep' - found /usr/bin/grep\\n[10:28:11] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:28:11] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:28:11] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:28:11] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:28:11] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:28:11] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:28:12] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:28:12] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:28:12] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:28:12] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:28:12] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:28:12] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:28:12] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:28:12] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:28:12] Looking for 'sed' - found /usr/bin/sed\\n[10:28:12] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:28:12] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:28:12] Using genetic code table 11.\\n[10:28:12] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKN01.fasta\\n[10:28:13] Wrote 157 contigs totalling 8059889 bp.\\n[10:28:13] Predicting tRNAs and tmRNAs\\n[10:28:13] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKN01\\\\/\\\\/Nostoc_JABXKN01\\\\.fna\\n[10:28:16] 1 tRNA-Ala [5497,5571] 35 (ggc)\\n[10:28:16] 2 tRNA-Ala c[9337,9411] 35 (tgc)\\n[10:28:16] 1 tRNA-Val c[107745,107817] 34 (tac)\\n[10:28:16] 2 tRNA-Ser c[128115,128201] 36 (tga)\\n[10:28:16] 3 tRNA-Tyr [130502,130587] 35 (gta)\\n[10:28:16] 4 tRNA-Thr [130675,130746] 33 (ggt)\\n[10:28:16] 5 tRNA-Arg c[141364,141438] 35 (tct)\\n[10:28:16] 1 tRNA-Trp [57456,57529] 34 (cca)\\n[10:28:16] 2 tRNA-Thr c[114091,114162] 33 (tgt)\\n[10:28:16] 1 tRNA-Arg c[27348,27421] 35 (acg)\\n[10:28:16] 1 tRNA-Val c[120571,120643] 33 (cac)\\n[10:28:16] 1 tRNA-Met [17382,17457] 34 (cat)\\n[10:28:16] 2 tRNA-Met [17458,17529] 32 (cat)\\n[10:28:16] 1 tRNA-Met c[504,578] 35 (cat)\\n[10:28:16] 1 tRNA-Asp c[14229,14302] 35 (gtc)\\n[10:28:16] 1 tRNA-Ala c[24594,24666] 34 (cgc)\\n[10:28:16] 1 tRNA-Phe c[152733,152805] 34 (gaa)\\n[10:28:16] 2 tmRNA [179763,180150] 101,151 ANNIVNFARKDALVAA*\\n[10:28:16] 1 tRNA-Gly c[10228,10298] 33 (tcc)\\n[10:28:16] 1 tRNA-Met c[213389,213462] 33 (cat)\\n[10:28:16] 1 tRNA-Arg c[143560,143632] 34 (ccg)\\n[10:28:16] 1 tRNA-Leu [23667,23751] 35 (caa)\\n[10:28:16] 2 tRNA-Gln [25572,25645] 34 (ttg)\\n[10:28:16] 1 tRNA-Thr c[95497,95568] 33 (cgt)\\n[10:28:16] 1 tRNA-Gly c[99356,99427] 33 (gcc)\\n[10:28:16] 2 tRNA-Leu [136131,136211] 34 (cag)\\n[10:28:16] 1 tRNA-Pro [48609,48682] 35 (tgg)\\n[10:28:16] 1 tRNA-Val [9122,9195] 35 (gac)\\n[10:28:16] 1 tRNA-Pyl c[30944,31017] 34 (cta)\\n[10:28:16] 1 tRNA-Ser [32463,32549] 36 (gga)\\n[10:28:16] 1 tRNA-Ser c[116806,116892] 36 (cga)\\n[10:28:16] 2 tRNA-Arg [133751,133826] 36 (cct)\\n[10:28:16] 3 tRNA-Lys c[140128,140199] 33 (ttt)\\n[10:28:16] 1 tRNA-Glu [57340,57413] 35 (ttc)\\n[10:28:16] 2 tRNA-Asp [63877,63952] 36 (gtc)\\n[10:28:16] 3 tRNA-Glu [63956,64032] 35 (ctc)\\n[10:28:16] 4 tRNA-Cys [64036,64109] 35 (gca)\\n[10:28:16] 5 tRNA-Met [64116,64189] 32 (cat)\\n[10:28:16] tRNA [64192,64266] is a pseudo/wacky gene - skipping.\\n[10:28:16] 7 tRNA-Glu [64328,64401] 35 (ttc)\\n[10:28:16] 8 tRNA-Ser [64513,64604] 36 (gct)\\n[10:28:16] 9 tRNA-Arg [64607,64683] 36 (acg)\\n[10:28:16] 10 tRNA-Phe [64704,64779] 36 (gaa)\\n[10:28:16] 11 tRNA-Asn [64785,64859] 33 (gtt)\\n[10:28:16] 12 tRNA-Gln [64864,64938] 35 (ctg)\\n[10:28:16] 13 tRNA-Gln [64941,65016] 34 (ttg)\\n[10:28:16] 14 tRNA-Leu [65132,65209] 36 (taa)\\n[10:28:16] 15 tRNA-Pro [65212,65284] 34 (tgg)\\n[10:28:16] 16 tRNA-Leu [65288,65362] 35 (cag)\\n[10:28:16] 17 tRNA-Leu [65366,65442] 34 (gag)\\n[10:28:16] 18 tRNA-Leu [65521,65595] 35 (tag)\\n[10:28:16] 19 tRNA-Met [65605,65681] 34 (cat)\\n[10:28:16] 20 tRNA-Thr [65689,65761] 34 (tgt)\\n[10:28:16] 21 tRNA-Val [65843,65918] 34 (tac)\\n[10:28:16] 22 tRNA-Arg [65925,65999] 35 (tct)\\n[10:28:16] 23 tRNA-Ala [66097,66171] 35 (tgc)\\n[10:28:16] 24 tRNA-His [66180,66255] 35 (gtg)\\n[10:28:16] 25 tRNA-Lys [66518,66589] 33 (ttt)\\n[10:28:16] 26 tRNA-Ser [70698,70789] 35 (gct)\\n[10:28:16] 1 tRNA-Cys c[36180,36250] 33 (gca)\\n[10:28:16] 1 tRNA-Met c[17879,17955] 35 (cat)\\n[10:28:16] 2 tRNA-Leu c[90149,90231] 35 (tag)\\n[10:28:16] 3 tRNA-Pro [161767,161840] 35 (cgg)\\n[10:28:16] 1 tRNA-Leu [17195,17277] 36 (gag)\\n[10:28:16] 1 tRNA-Gly c[47637,47708] 33 (ccc)\\n[10:28:16] 1 tRNA-His [116602,116675] 34 (gtg)\\n[10:28:16] 2 tRNA-Asn [250584,250655] 33 (gtt)\\n[10:28:16] 1 tRNA-Lys c[7391,7463] 33 (ctt)\\n[10:28:16] 1 tRNA-Pro [6424,6497] 35 (ggg)\\n[10:28:16] Found 68 tRNAs\\n[10:28:16] Predicting Ribosomal RNAs\\n[10:28:16] Running Barrnap with 4 threads\\n[10:28:18] Found 0 rRNAs\\n[10:28:18] Skipping ncRNA search, enable with --rfam if desired.\\n[10:28:18] Total of 67 tRNA + rRNA features\\n[10:28:18] Searching for CRISPR repeats\\n[10:28:18] CRISPR1 JABXKN010000047.1 23834 with 5 spacers\\n[10:28:18] CRISPR2 JABXKN010000048.1 10211 with 12 spacers\\n[10:28:18] CRISPR3 JABXKN010000108.1 57249 with 5 spacers\\n[10:28:18] CRISPR4 JABXKN010000146.1 32230 with 6 spacers\\n[10:28:18] CRISPR5 JABXKN010000155.1 6596 with 5 spacers\\n[10:28:18] Found 5 CRISPRs\\n[10:28:18] Predicting coding sequences\\n[10:28:18] Contigs total 8059889 bp, so using single mode\\n[10:28:18] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKN01\\\\/\\\\/Nostoc_JABXKN01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:28:26] Excluding CDS which overlaps existing RNA (tRNA) at JABXKN010000035.1:17488..17703 on - strand\\n[10:28:26] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKN010000047.1:23227..24099 on - strand\\n[10:28:26] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKN010000048.1:10331..10597 on + strand\\n[10:28:26] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKN010000048.1:10566..10826 on - strand\\n[10:28:28] Excluding CDS which overlaps existing RNA (tRNA) at JABXKN010000124.1:32509..32847 on - strand\\n[10:28:28] Excluding CDS which overlaps existing RNA (tRNA) at JABXKN010000126.1:139908..140231 on + strand\\n[10:28:28] Excluding CDS which overlaps existing RNA (tRNA) at JABXKN010000128.1:64799..65032 on + strand\\n[10:28:28] Excluding CDS which overlaps existing RNA (tRNA) at JABXKN010000128.1:65239..65601 on + strand\\n[10:28:29] Excluding CDS which overlaps existing RNA (tRNA) at JABXKN010000143.1:161687..161869 on + strand\\n[10:28:29] Found 7016 CDS\\n[10:28:29] Connecting features back to sequences\\n[10:28:29] Not using genus-specific database. Try --usegenus to enable it.\\n[10:28:29] Annotating CDS, please be patient.\\n[10:28:29] Will use 4 CPUs for similarity searching.\\n[10:28:31] There are still 7016 unannotated CDS left (started with 7016)\\n[10:28:31] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:28:31] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKN01\\\\/\\\\/Nostoc_JABXKN01\\\\.IS\\\\.tmp\\\\.1823\\\\.faa | parallel --gnu --plain -j 4 --block 275248 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKN01\\\\/\\\\/Nostoc_JABXKN01\\\\.IS\\\\.tmp\\\\.1823\\\\.blast 2> /dev/null\\n[10:28:50] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKN01\\\\/\\\\/Nostoc_JABXKN01\\\\.IS\\\\.tmp\\\\.1823\\\\.faa | parallel --gnu --plain -j 4 --block 275248 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKN01\\\\/\\\\/Nostoc_JABXKN01\\\\.IS\\\\.tmp\\\\.1823\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:28:50] This is prokka 1.14.6\\n[10:28:50] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:28:50] Homepage is https://github.com/tseemann/prokka\\n[10:28:50] Local time is Mon Feb 21 10:28:50 2022\\n[10:28:50] You are tiagoferreiraleao\\n[10:28:50] Operating system is darwin\\n[10:28:50] You have BioPerl 1.007002\\n[10:28:51] System has 4 cores.\\n[10:28:51] Option --cpu asked for 12 cores, but system only has 4\\n[10:28:51] Will use maximum of 4 cores.\\n[10:28:51] Annotating as >>> Bacteria <<<\\n[10:28:51] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJTK01.fasta' contents.\\n[10:28:51] Setting --locustag MGJBGHKN from MD5 603b014749ece7d8bf753b00678ba153\\n[10:28:51] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJTK01/\\n[10:28:51] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTK01\\\\/\\n[10:28:51] Using filename prefix: Nostoc_JACJTK01.XXX\\n[10:28:51] Setting HMMER_NCPU=1\\n[10:28:51] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJTK01//Nostoc_JACJTK01.log\\n[10:28:51] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJTK01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJTK01/ --prefix Nostoc_JACJTK01 --cpus 12\\n[10:28:51] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:28:51] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:28:51] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:28:51] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:28:51] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:28:51] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:28:51] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:28:51] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:28:51] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:28:51] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:28:51] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:28:51] Looking for 'egrep' - found /usr/bin/egrep\\n[10:28:51] Looking for 'find' - found /usr/bin/find\\n[10:28:51] Looking for 'grep' - found /usr/bin/grep\\n[10:28:51] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:28:51] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:28:51] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:28:51] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:28:51] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:28:51] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:28:51] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:28:51] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:28:52] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:28:52] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:28:52] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:28:52] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:28:52] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:28:52] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:28:52] Looking for 'sed' - found /usr/bin/sed\\n[10:28:52] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:28:52] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:28:52] Using genetic code table 11.\\n[10:28:52] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJTK01.fasta\\n[10:28:52] Wrote 165 contigs totalling 9241482 bp.\\n[10:28:52] Predicting tRNAs and tmRNAs\\n[10:28:52] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTK01\\\\/\\\\/Nostoc_JACJTK01\\\\.fna\\n[10:28:56] 1 tRNA-Arg [263896,263969] 35 (acg)\\n[10:28:56] 2 tRNA-Ser c[353870,353944] 35 (tga)\\n[10:28:56] 3 tRNA-Lys c[355338,355412] 33 (ctt)\\n[10:28:56] 1 tRNA-Val c[23950,24022] 34 (tac)\\n[10:28:56] 1 tRNA-Pro c[248874,248947] 35 (ggg)\\n[10:28:56] 1 tRNA-His [57067,57140] 34 (gtg)\\n[10:28:56] 2 tRNA-Thr c[230504,230575] 33 (tgt)\\n[10:28:56] 1 tRNA-Met c[147400,147476] 35 (cat)\\n[10:28:56] 2 tRNA-Stop [270785,270874] 31 (tta)\\n[10:28:56] 1 tRNA-Asp c[107796,107869] 35 (gtc)\\n[10:28:56] 1 tRNA-Gln c[93592,93665] 34 (ttg)\\n[10:28:56] 2 tRNA-Leu c[98335,98419] 35 (caa)\\n[10:28:56] 3 tRNA-Glu c[230156,230230] 36 (ttc)\\n[10:28:56] 1 tRNA-Leu [72568,72648] 35 (gag)\\n[10:28:56] 2 tRNA-Ser c[219906,219999] 36 (gct)\\n[10:28:56] 3 tRNA-Lys c[225720,225790] 33 (ttt)\\n[10:28:56] 4 tRNA-His c[226053,226128] 35 (gtg)\\n[10:28:56] 5 tRNA-Ala c[226134,226208] 35 (tgc)\\n[10:28:56] 6 tRNA-Arg c[226238,226313] 35 (tct)\\n[10:28:56] 7 tRNA-Val c[226320,226395] 34 (tac)\\n[10:28:56] 8 tRNA-Gly c[226481,226552] 33 (tcc)\\n[10:28:56] 9 tRNA-Thr c[226557,226629] 34 (tgt)\\n[10:28:56] 10 tRNA-Met c[226638,226714] 34 (cat)\\n[10:28:56] 11 tRNA-Leu c[226725,226797] 34 (tag)\\n[10:28:56] 12 tRNA-Pro c[226801,226874] 34 (tgg)\\n[10:28:56] 13 tRNA-Leu c[226961,227036] 35 (taa)\\n[10:28:56] 14 tRNA-Gln c[227045,227117] 34 (ttg)\\n[10:28:56] 15 tRNA-Gln c[227121,227193] 34 (ctg)\\n[10:28:56] 16 tRNA-Asn c[227344,227418] 33 (gtt)\\n[10:28:56] 17 tRNA-Phe c[227424,227499] 36 (gaa)\\n[10:28:56] 18 tRNA-Arg c[227580,227656] 36 (acg)\\n[10:28:56] 19 tRNA-Ser c[227658,227749] 35 (gct)\\n[10:28:56] 20 tRNA-Glu c[227861,227936] 35 (ttc)\\n[10:28:56] tRNA c[227992,228066] is a pseudo/wacky gene - skipping.\\n[10:28:56] 22 tRNA-Met c[228068,228142] 33 (cat)\\n[10:28:56] 23 tRNA-Cys c[228150,228222] 35 (gca)\\n[10:28:56] 24 tRNA-Glu c[228226,228302] 35 (ctc)\\n[10:28:56] 1 tRNA-Ala [145718,145790] 34 (ggc)\\n[10:28:56] 1 tRNA-Met [14949,15024] 36 (cat)\\n[10:28:56] 1 tRNA-Phe c[74185,74257] 34 (gaa)\\n[10:28:56] 2 tRNA-Pro c[118188,118261] 35 (tgg)\\n[10:28:56] 1 tRNA-Leu [150147,150228] 35 (cag)\\n[10:28:56] 1 tRNA-Ala [82159,82231] 34 (cgc)\\n[10:28:56] 2 tRNA-Trp c[97716,97789] 34 (cca)\\n[10:28:56] 3 tRNA-Thr c[145795,145866] 33 (cgt)\\n[10:28:56] 1 tmRNA [131681,132068] 101,151 ANNIVKFARREALVAA*\\n[10:28:56] 1 tRNA-Gly [12169,12239] 33 (tcc)\\n[10:28:56] 1 tRNA-Lys [108752,108823] 33 (ttt)\\n[10:28:56] 1 tRNA-Ala [20127,20199] 34 (tgc)\\n[10:28:56] 1 tRNA-Gly c[61990,62061] 33 (gcc)\\n[10:28:56] 2 tRNA-Met c[69116,69189] 33 (cat)\\n[10:28:56] 1 tRNA-Arg c[5469,5543] 35 (tct)\\n[10:28:56] 1 tRNA-Leu c[37949,38032] 36 (tag)\\n[10:28:56] 1 tRNA-Arg c[44035,44107] 34 (ccg)\\n[10:28:56] 1 tRNA-Asn [2430,2503] 34 (gtt)\\n[10:28:56] 1 tRNA-Val [71169,71242] 35 (gac)\\n[10:28:56] 1 tRNA-Met [66868,66943] 34 (cat)\\n[10:28:56] 2 tRNA-Met [66945,67020] 34 (cat)\\n[10:28:56] 1 tRNA-Thr c[55559,55630] 33 (ggt)\\n[10:28:56] 2 tRNA-Tyr c[55711,55795] 35 (gta)\\n[10:28:56] 3 tRNA-Ser [58050,58136] 36 (tga)\\n[10:28:56] 1 tRNA-Ser c[18327,18413] 36 (cga)\\n[10:28:56] 2 tRNA-Pro c[18591,18664] 35 (cgg)\\n[10:28:56] 1 tRNA-Gly [44971,45042] 33 (ccc)\\n[10:28:56] 1 tRNA-Lys c[10272,10344] 33 (ctt)\\n[10:28:56] 1 tRNA-Ser c[20434,20521] 35 (gga)\\n[10:28:56] 1 tRNA-Gly [21066,21137] 33 (tcc)\\n[10:28:56] 1 tRNA-Val c[7429,7502] 34 (cac)\\n[10:28:56] 1 tRNA-Cys [6952,7023] 33 (gca)\\n[10:28:56] Found 68 tRNAs\\n[10:28:56] Predicting Ribosomal RNAs\\n[10:28:56] Running Barrnap with 4 threads\\n[10:28:57] 1 JACJTK010000164.1 1 16S ribosomal RNA\\n[10:28:57] 2 JACJTK010000165.1 230 23S ribosomal RNA\\n[10:28:57] Found 2 rRNAs\\n[10:28:57] Skipping ncRNA search, enable with --rfam if desired.\\n[10:28:57] Total of 69 tRNA + rRNA features\\n[10:28:57] Searching for CRISPR repeats\\n[10:28:58] CRISPR1 JACJTK010000001.1 393768 with 10 spacers\\n[10:28:58] CRISPR2 JACJTK010000005.1 16272 with 11 spacers\\n[10:28:58] CRISPR3 JACJTK010000008.1 211348 with 21 spacers\\n[10:28:58] CRISPR4 JACJTK010000015.1 55577 with 19 spacers\\n[10:28:58] CRISPR5 JACJTK010000016.1 9652 with 14 spacers\\n[10:28:58] CRISPR6 JACJTK010000021.1 10917 with 20 spacers\\n[10:28:58] CRISPR7 JACJTK010000023.1 18184 with 15 spacers\\n[10:28:58] CRISPR8 JACJTK010000029.1 63324 with 25 spacers\\n[10:28:58] CRISPR9 JACJTK010000029.1 79603 with 40 spacers\\n[10:28:58] CRISPR10 JACJTK010000053.1 42724 with 6 spacers\\n[10:28:58] CRISPR11 JACJTK010000112.1 4875 with 21 spacers\\n[10:28:58] Found 11 CRISPRs\\n[10:28:58] Predicting coding sequences\\n[10:28:58] Contigs total 9241482 bp, so using single mode\\n[10:28:58] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTK01\\\\/\\\\/Nostoc_JACJTK01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:29:08] Excluding CDS which overlaps existing RNA (repeat_region) at JACJTK010000005.1:16463..16936 on - strand\\n[10:29:08] Excluding CDS which overlaps existing RNA (tRNA) at JACJTK010000007.1:270236..271543 on + strand\\n[10:29:08] Excluding CDS which overlaps existing RNA (tRNA) at JACJTK010000009.1:93574..93759 on + strand\\n[10:29:08] Excluding CDS which overlaps existing RNA (tRNA) at JACJTK010000009.1:98310..98558 on + strand\\n[10:29:10] Excluding CDS which overlaps existing RNA (tRNA) at JACJTK010000027.1:37921..38178 on - strand\\n[10:29:10] Excluding CDS which overlaps existing RNA (repeat_region) at JACJTK010000029.1:63189..63464 on + strand\\n[10:29:11] Excluding CDS which overlaps existing RNA (repeat_region) at JACJTK010000053.1:43083..43301 on - strand\\n[10:29:12] Found 7761 CDS\\n[10:29:12] Connecting features back to sequences\\n[10:29:12] Not using genus-specific database. Try --usegenus to enable it.\\n[10:29:12] Annotating CDS, please be patient.\\n[10:29:12] Will use 4 CPUs for similarity searching.\\n[10:29:14] There are still 7761 unannotated CDS left (started with 7761)\\n[10:29:14] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:29:14] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTK01\\\\/\\\\/Nostoc_JACJTK01\\\\.IS\\\\.tmp\\\\.1939\\\\.faa | parallel --gnu --plain -j 4 --block 313879 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTK01\\\\/\\\\/Nostoc_JACJTK01\\\\.IS\\\\.tmp\\\\.1939\\\\.blast 2> /dev/null\\n[10:29:39] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTK01\\\\/\\\\/Nostoc_JACJTK01\\\\.IS\\\\.tmp\\\\.1939\\\\.faa | parallel --gnu --plain -j 4 --block 313879 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTK01\\\\/\\\\/Nostoc_JACJTK01\\\\.IS\\\\.tmp\\\\.1939\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:29:39] This is prokka 1.14.6\\n[10:29:39] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:29:39] Homepage is https://github.com/tseemann/prokka\\n[10:29:39] Local time is Mon Feb 21 10:29:39 2022\\n[10:29:39] You are tiagoferreiraleao\\n[10:29:39] Operating system is darwin\\n[10:29:39] You have BioPerl 1.007002\\n[10:29:39] System has 4 cores.\\n[10:29:39] Option --cpu asked for 12 cores, but system only has 4\\n[10:29:39] Will use maximum of 4 cores.\\n[10:29:39] Annotating as >>> Bacteria <<<\\n[10:29:39] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JACYMA01.fasta' contents.\\n[10:29:40] Setting --locustag HAANIKHG from MD5 1aa7241013631e3d15dfefbeb31cd4be\\n[10:29:40] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JACYMA01/\\n[10:29:40] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYMA01\\\\/\\n[10:29:40] Using filename prefix: Fischerella_JACYMA01.XXX\\n[10:29:40] Setting HMMER_NCPU=1\\n[10:29:40] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JACYMA01//Fischerella_JACYMA01.log\\n[10:29:40] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JACYMA01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JACYMA01/ --prefix Fischerella_JACYMA01 --cpus 12\\n[10:29:40] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:29:40] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:29:40] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:29:40] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:29:40] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:29:40] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:29:40] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:29:40] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:29:40] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:29:40] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:29:40] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:29:40] Looking for 'egrep' - found /usr/bin/egrep\\n[10:29:40] Looking for 'find' - found /usr/bin/find\\n[10:29:40] Looking for 'grep' - found /usr/bin/grep\\n[10:29:40] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:29:40] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:29:40] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:29:40] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:29:40] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:29:40] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:29:40] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:29:40] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:29:40] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:29:40] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:29:40] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:29:40] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:29:40] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:29:40] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:29:40] Looking for 'sed' - found /usr/bin/sed\\n[10:29:40] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:29:41] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:29:41] Using genetic code table 11.\\n[10:29:41] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JACYMA01.fasta\\n[10:29:41] Wrote 463 contigs totalling 5271540 bp.\\n[10:29:41] Predicting tRNAs and tmRNAs\\n[10:29:41] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYMA01\\\\/\\\\/Fischerella_JACYMA01\\\\.fna\\n[10:29:43] 1 tRNA-Arg c[28401,28474] 35 (acg)\\n[10:29:43] 2 tRNA-Ala c[33349,33422] 34 (ggc)\\n[10:29:43] 3 tRNA-Thr c[35243,35314] 33 (cgt)\\n[10:29:43] 1 tRNA-Leu [6,89] 36 (tag)\\n[10:29:43] 1 tRNA-Met [95,170] 36 (cat)\\n[10:29:43] 1 tRNA-Pro [19098,19172] 35 (tgg)\\n[10:29:43] 1 tRNA-Glu [3184,3256] 35 (ttc)\\n[10:29:43] 1 tRNA-Ser c[8702,8790] 36 (gga)\\n[10:29:43] 1 tRNA-Ser [14903,14989] 36 (tga)\\n[10:29:43] 1 tRNA-Val c[2,74] 34 (tac)\\n[10:29:43] 2 tRNA-Phe [7425,7497] 34 (gaa)\\n[10:29:43] 1 tRNA-Pro [1011,1084] 35 (cgg)\\n[10:29:43] 1 tRNA-Ala [2104,2179] 35 (cgc)\\n[10:29:43] 1 tRNA-Ser [352,438] 36 (cga)\\n[10:29:43] 1 tRNA-Pro [4071,4144] 35 (ggg)\\n[10:29:43] 1 tRNA-Gly [19366,19437] 33 (ccc)\\n[10:29:43] 1 tRNA-Ser [6858,6951] 36 (gct)\\n[10:29:43] 1 tRNA-Arg c[5799,5872] 35 (ccg)\\n[10:29:43] 1 tRNA-Lys [20812,20885] 34 (ttt)\\n[10:29:43] 1 tRNA-His c[30978,31051] 34 (gtg)\\n[10:29:43] 1 tRNA-Gly c[4336,4406] 33 (tcc)\\n[10:29:43] 1 tRNA-Asn [2675,2748] 34 (gtt)\\n[10:29:43] 1 tRNA-Gln c[17024,17095] 33 (ttg)\\n[10:29:43] 1 tRNA-Arg c[19463,19537] 35 (tct)\\n[10:29:43] 1 tRNA-Leu c[552,634] 35 (cag)\\n[10:29:43] 1 tRNA-Gln [34795,34865] 32 (ttg)\\n[10:29:43] 2 tRNA-Gly [34868,34937] 32 (tcc)\\n[10:29:43] 1 tRNA-Thr c[5,76] 33 (tgt)\\n[10:29:43] 1 tRNA-Met c[4067,4143] 35 (cat)\\n[10:29:43] 2 tRNA-Val [7114,7187] 35 (gac)\\n[10:29:43] 1 tRNA-Asp [2335,2408] 35 (gtc)\\n[10:29:43] 1 tRNA-Asn c[7517,7590] 34 (att)\\n[10:29:43] 2 tRNA-Arg c[8261,8333] 34 (tcg)\\n[10:29:43] 1 tRNA-Met [8708,8780] 34 (cat)\\n[10:29:43] 1 tRNA-Gly [7502,7573] 33 (gcc)\\n[10:29:43] 1 tRNA-Cys [797,868] 33 (gca)\\n[10:29:43] 1 tRNA-Trp [28860,28933] 34 (cca)\\n[10:29:43] Found 37 tRNAs\\n[10:29:43] Predicting Ribosomal RNAs\\n[10:29:43] Running Barrnap with 4 threads\\n[10:29:44] Found 0 rRNAs\\n[10:29:44] Skipping ncRNA search, enable with --rfam if desired.\\n[10:29:44] Total of 37 tRNA + rRNA features\\n[10:29:44] Searching for CRISPR repeats\\n[10:29:44] CRISPR1 JACYMA010000285.1 12049 with 3 spacers\\n[10:29:44] CRISPR2 JACYMA010000293.1 13266 with 21 spacers\\n[10:29:44] CRISPR3 JACYMA010000356.1 17390 with 56 spacers\\n[10:29:45] CRISPR4 JACYMA010000446.1 44 with 20 spacers\\n[10:29:45] Found 4 CRISPRs\\n[10:29:45] Predicting coding sequences\\n[10:29:45] Contigs total 5271540 bp, so using single mode\\n[10:29:45] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYMA01\\\\/\\\\/Fischerella_JACYMA01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:29:51] Excluding CDS which overlaps existing RNA (tRNA) at JACYMA010000259.1:2690..3988 on + strand\\n[10:29:52] Found 4368 CDS\\n[10:29:52] Connecting features back to sequences\\n[10:29:52] Not using genus-specific database. Try --usegenus to enable it.\\n[10:29:52] Annotating CDS, please be patient.\\n[10:29:52] Will use 4 CPUs for similarity searching.\\n[10:29:53] There are still 4368 unannotated CDS left (started with 4368)\\n[10:29:53] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:29:53] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYMA01\\\\/\\\\/Fischerella_JACYMA01\\\\.IS\\\\.tmp\\\\.2055\\\\.faa | parallel --gnu --plain -j 4 --block 171406 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYMA01\\\\/\\\\/Fischerella_JACYMA01\\\\.IS\\\\.tmp\\\\.2055\\\\.blast 2> /dev/null\\n[10:30:07] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYMA01\\\\/\\\\/Fischerella_JACYMA01\\\\.IS\\\\.tmp\\\\.2055\\\\.faa | parallel --gnu --plain -j 4 --block 171406 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JACYMA01\\\\/\\\\/Fischerella_JACYMA01\\\\.IS\\\\.tmp\\\\.2055\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:30:07] This is prokka 1.14.6\\n[10:30:07] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:30:07] Homepage is https://github.com/tseemann/prokka\\n[10:30:07] Local time is Mon Feb 21 10:30:07 2022\\n[10:30:07] You are tiagoferreiraleao\\n[10:30:07] Operating system is darwin\\n[10:30:07] You have BioPerl 1.007002\\n[10:30:08] System has 4 cores.\\n[10:30:08] Option --cpu asked for 12 cores, but system only has 4\\n[10:30:08] Will use maximum of 4 cores.\\n[10:30:08] Annotating as >>> Bacteria <<<\\n[10:30:08] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHJ01.fasta' contents.\\n[10:30:08] Setting --locustag CIFAKDII from MD5 c2fa4d22ed60a6a45c06c4307421d96c\\n[10:30:08] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHJ01/\\n[10:30:08] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHJ01\\\\/\\n[10:30:08] Using filename prefix: Nostoc_LAHJ01.XXX\\n[10:30:08] Setting HMMER_NCPU=1\\n[10:30:08] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHJ01//Nostoc_LAHJ01.log\\n[10:30:08] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHJ01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHJ01/ --prefix Nostoc_LAHJ01 --cpus 12\\n[10:30:08] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:30:08] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:30:08] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:30:08] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:30:08] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:30:08] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:30:08] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:30:08] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:30:08] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:30:08] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:30:08] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:30:08] Looking for 'egrep' - found /usr/bin/egrep\\n[10:30:08] Looking for 'find' - found /usr/bin/find\\n[10:30:08] Looking for 'grep' - found /usr/bin/grep\\n[10:30:08] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:30:08] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:30:08] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:30:08] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:30:08] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:30:08] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:30:08] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:30:08] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:30:08] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:30:08] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:30:08] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:30:08] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:30:08] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:30:08] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:30:08] Looking for 'sed' - found /usr/bin/sed\\n[10:30:08] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:30:09] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:30:09] Using genetic code table 11.\\n[10:30:09] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHJ01.fasta\\n[10:30:09] Wrote 255 contigs totalling 8898113 bp.\\n[10:30:09] Predicting tRNAs and tmRNAs\\n[10:30:09] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHJ01\\\\/\\\\/Nostoc_LAHJ01\\\\.fna\\n[10:30:12] 1 tRNA-Thr c[44961,45032] 33 (cgt)\\n[10:30:12] 1 tRNA-Gly [17226,17296] 33 (tcc)\\n[10:30:12] 1 tRNA-Cys [90239,90310] 33 (gca)\\n[10:30:12] 1 tmRNA [33507,33894] 101,151 ANNIVKFARREALVAA*\\n[10:30:12] 1 tRNA-Gly [42864,42937] 34 (ccc)\\n[10:30:12] 2 tRNA-Ser c[51082,51169] 35 (gga)\\n[10:30:12] 1 tRNA-Ala c[1010,1082] 34 (cgc)\\n[10:30:12] 2 tRNA-Leu [49784,49868] 35 (caa)\\n[10:30:12] 3 tRNA-Gln [54622,54695] 34 (ttg)\\n[10:30:12] 1 tRNA-Asn [21520,21593] 34 (gtt)\\n[10:30:12] 1 tRNA-Ser [92063,92149] 36 (cga)\\n[10:30:12] 1 tRNA-Leu c[42743,42826] 36 (tag)\\n[10:30:12] 1 tRNA-Pro c[39475,39548] 35 (cgg)\\n[10:30:12] 1 tRNA-Pro c[39909,39982] 35 (tgg)\\n[10:30:12] 2 tRNA-Ser c[48664,48750] 36 (tga)\\n[10:30:12] 3 tRNA-Tyr [51201,51285] 35 (gta)\\n[10:30:12] 4 tRNA-Thr [51361,51432] 33 (ggt)\\n[10:30:12] 1 tRNA-Pro [25153,25226] 35 (ggg)\\n[10:30:12] 1 tRNA-Ile c[33465,33540] 34 (tat)\\n[10:30:12] 2 tRNA-Met c[35146,35221] 34 (cat)\\n[10:30:12] 3 tRNA-Met c[35223,35298] 34 (cat)\\n[10:30:12] 1 tRNA-Ala c[771,843] 34 (ggc)\\n[10:30:12] 1 tRNA-Ala c[29396,29470] 34 (tgc)\\n[10:30:12] 2 tRNA-Ile c[29554,29629] 35 (gat)\\n[10:30:12] 1 tRNA-Lys [80097,80168] 33 (ttt)\\n[10:30:12] 1 tRNA-Gly c[28514,28585] 33 (gcc)\\n[10:30:12] 2 tRNA-Met c[35395,35468] 33 (cat)\\n[10:30:12] 1 tRNA-Ser c[7798,7891] 36 (gct)\\n[10:30:12] 2 tRNA-Lys c[13566,13636] 33 (ttt)\\n[10:30:12] 3 tRNA-Ser c[13895,13978] 34 (tga)\\n[10:30:12] 4 tRNA-His c[13983,14058] 35 (gtg)\\n[10:30:12] 5 tRNA-Ala c[14064,14138] 35 (tgc)\\n[10:30:12] 6 tRNA-Arg c[14230,14305] 35 (tct)\\n[10:30:12] 7 tRNA-Val c[14312,14387] 34 (tac)\\n[10:30:12] 8 tRNA-Thr c[14469,14541] 34 (tgt)\\n[10:30:12] 9 tRNA-Leu c[14836,14910] 35 (tag)\\n[10:30:12] 10 tRNA-Leu c[15022,15098] 34 (gag)\\n[10:30:12] 11 tRNA-Pro c[15206,15279] 35 (tgg)\\n[10:30:12] 12 tRNA-Leu c[15281,15358] 35 (taa)\\n[10:30:12] 13 tRNA-Leu c[15379,15456] 36 (caa)\\n[10:30:12] 14 tRNA-Gln c[15482,15557] 35 (ttg)\\n[10:30:12] 15 tRNA-Gln c[16432,16506] 35 (ctg)\\n[10:30:12] 16 tRNA-Asn c[16517,16591] 33 (gtt)\\n[10:30:12] 17 tRNA-Phe c[16597,16672] 36 (gaa)\\n[10:30:12] 18 tRNA-Arg c[16760,16835] 35 (acg)\\n[10:30:12] 19 tRNA-Ser c[17597,17683] 35 (gct)\\n[10:30:12] 20 tRNA-Tyr c[17687,17768] 35 (gta)\\n[10:30:12] 21 tRNA-Glu c[17779,17853] 35 (ttc)\\n[10:30:12] tRNA c[17941,18015] is a pseudo/wacky gene - skipping.\\n[10:30:12] 23 tRNA-Met c[18018,18091] 32 (cat)\\n[10:30:12] 24 tRNA-Glu c[18201,18277] 35 (ctc)\\n[10:30:12] 25 tRNA-Asp c[18281,18356] 36 (gtc)\\n[10:30:12] 1 tRNA-Asp [55002,55079] 38 (gtc)\\n[10:30:12] 2 tRNA-Glu [55082,55158] 35 (ctc)\\n[10:30:12] 3 tRNA-Cys [55163,55238] 34 (gca)\\n[10:30:12] 4 tRNA-Met [55244,55318] 33 (cat)\\n[10:30:12] 5 tRNA-Trp [55321,55393] 34 (cca)\\n[10:30:12] 6 tRNA-Pyl [55418,55489] 33 (cta)\\n[10:30:12] 7 tRNA-Glu [55520,55595] 34 (ttc)\\n[10:30:12] 8 tRNA-Tyr [55599,55682] 36 (gta)\\n[10:30:12] 9 tRNA-Ser [55686,55775] 33 (gct)\\n[10:30:12] 10 tRNA-Arg [55780,55855] 35 (acg)\\n[10:30:12] 11 tRNA-Phe [55860,55934] 35 (gaa)\\n[10:30:12] 12 tRNA-Asn [55943,56017] 33 (gtt)\\n[10:30:12] 13 tRNA-Gln [56023,56098] 35 (ttg)\\n[10:30:12] 14 tRNA-Leu [56106,56183] 35 (caa)\\n[10:30:12] 15 tRNA-Pro [56188,56264] 35 (tgg)\\n[10:30:12] 16 tRNA-Leu [56268,56342] 35 (tag)\\n[10:30:12] 17 tRNA-Leu [56362,56439] 35 (gag)\\n[10:30:12] 18 tRNA-Lys [56443,56519] 34 (ttt)\\n[10:30:12] 19 tRNA-Lys [56525,56599] 33 (ctt)\\n[10:30:12] 20 tRNA-Leu [56605,56680] 36 (taa)\\n[10:30:12] 21 tRNA-Cys [56690,56767] 34 (gca)\\n[10:30:12] 22 tRNA-Thr [56775,56847] 34 (tgt)\\n[10:30:12] 23 tRNA-Gly [56853,56923] 33 (tcc)\\n[10:30:12] 24 tRNA-Val [56929,57004] 34 (tac)\\n[10:30:12] 25 tRNA-Arg [57011,57085] 35 (tct)\\n[10:30:12] 26 tRNA-Ala [57090,57162] 34 (tgc)\\n[10:30:12] 27 tRNA-His [57170,57246] 35 (gtg)\\n[10:30:12] 28 tRNA-Ser [57257,57341] 34 (tga)\\n[10:30:12] 1 tRNA-Arg [29911,29985] 35 (tct)\\n[10:30:12] 1 tRNA-Lys [27700,27772] 33 (ctt)\\n[10:30:12] 1 tRNA-Asp [30552,30625] 35 (gtc)\\n[10:30:12] 1 tRNA-Arg [4885,4957] 34 (ccg)\\n[10:30:12] 1 tRNA-Leu c[30450,30524] 35 (caa)\\n[10:30:12] 2 tRNA-Thr c[31915,31989] 33 (agt)\\n[10:30:12] 1 tRNA-Leu [61482,61562] 35 (gag)\\n[10:30:12] tRNA c[16271,16345] is a pseudo/wacky gene - skipping.\\n[10:30:12] 1 tRNA-Trp [684,757] 34 (cca)\\n[10:30:12] 1 tRNA-Glu [22611,22684] 35 (ttc)\\n[10:30:12] 1 tRNA-His c[6168,6241] 34 (gtg)\\n[10:30:12] 1 tRNA-Arg c[19830,19903] 35 (acg)\\n[10:30:12] 1 tRNA-Ala c[17003,17089] 29 (cgc)\\n[10:30:12] 1 tRNA-Ala c[9864,9936] 34 (tgc)\\n[10:30:12] 1 tRNA-Val c[14097,14169] 34 (tac)\\n[10:30:12] 1 tRNA-Leu [10509,10590] 35 (cag)\\n[10:30:12] 1 tRNA-Thr [2244,2316] 33 (tgt)\\n[10:30:12] 1 tRNA-Val c[1435,1508] 35 (gac)\\n[10:30:12] Found 96 tRNAs\\n[10:30:12] Predicting Ribosomal RNAs\\n[10:30:12] Running Barrnap with 4 threads\\n[10:30:14] 1 LAHJ01000037.1 26400 23S ribosomal RNA\\n[10:30:14] 2 LAHJ01000037.1 29751 16S ribosomal RNA\\n[10:30:14] 3 LAHJ01000209.1 401 16S ribosomal RNA\\n[10:30:14] 4 LAHJ01000209.1 2365 23S ribosomal RNA\\n[10:30:14] 5 LAHJ01000209.1 5670 5S ribosomal RNA\\n[10:30:14] Found 5 rRNAs\\n[10:30:14] Skipping ncRNA search, enable with --rfam if desired.\\n[10:30:14] Total of 100 tRNA + rRNA features\\n[10:30:14] Searching for CRISPR repeats\\n[10:30:15] CRISPR1 LAHJ01000002.1 94001 with 24 spacers\\n[10:30:15] CRISPR2 LAHJ01000007.1 57519 with 14 spacers\\n[10:30:15] CRISPR3 LAHJ01000009.1 74115 with 15 spacers\\n[10:30:15] CRISPR4 LAHJ01000016.1 38383 with 18 spacers\\n[10:30:15] CRISPR5 LAHJ01000023.1 50468 with 7 spacers\\n[10:30:15] CRISPR6 LAHJ01000028.1 41492 with 10 spacers\\n[10:30:15] CRISPR7 LAHJ01000032.1 17587 with 20 spacers\\n[10:30:15] CRISPR8 LAHJ01000125.1 4887 with 43 spacers\\n[10:30:15] CRISPR9 LAHJ01000125.1 19464 with 27 spacers\\n[10:30:15] CRISPR10 LAHJ01000149.1 86 with 22 spacers\\n[10:30:15] CRISPR11 LAHJ01000161.1 14006 with 7 spacers\\n[10:30:15] CRISPR12 LAHJ01000164.1 2792 with 18 spacers\\n[10:30:15] CRISPR13 LAHJ01000190.1 5013 with 6 spacers\\n[10:30:15] Found 13 CRISPRs\\n[10:30:15] Predicting coding sequences\\n[10:30:15] Contigs total 8898113 bp, so using single mode\\n[10:30:15] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHJ01\\\\/\\\\/Nostoc_LAHJ01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:30:24] Excluding CDS which overlaps existing RNA (repeat_region) at LAHJ01000007.1:57690..57806 on - strand\\n[10:30:25] Excluding CDS which overlaps existing RNA (repeat_region) at LAHJ01000028.1:41701..41853 on + strand\\n[10:30:28] Excluding CDS which overlaps existing RNA (tRNA) at LAHJ01000116.1:6177..6320 on + strand\\n[10:30:28] Excluding CDS which overlaps existing RNA (repeat_region) at LAHJ01000125.1:7825..7989 on + strand\\n[10:30:28] Excluding CDS which overlaps existing RNA (tRNA) at LAHJ01000139.1:16538..17908 on - strand\\n[10:30:28] Excluding CDS which overlaps existing RNA (repeat_region) at LAHJ01000149.1:1540..1818 on - strand\\n[10:30:28] Excluding CDS which overlaps existing RNA (repeat_region) at LAHJ01000161.1:14034..14159 on - strand\\n[10:30:28] Excluding CDS which overlaps existing RNA (repeat_region) at LAHJ01000190.1:5405..6364 on - strand\\n[10:30:29] Excluding CDS which overlaps existing RNA (rRNA) at LAHJ01000209.1:1560..1820 on + strand\\n[10:30:29] Excluding CDS which overlaps existing RNA (rRNA) at LAHJ01000209.1:2187..2642 on - strand\\n[10:30:29] Excluding CDS which overlaps existing RNA (rRNA) at LAHJ01000209.1:2875..3291 on - strand\\n[10:30:29] Excluding CDS which overlaps existing RNA (rRNA) at LAHJ01000209.1:4837..5061 on + strand\\n[10:30:29] Excluding CDS which overlaps existing RNA (rRNA) at LAHJ01000209.1:5542..6366 on + strand\\n[10:30:29] Found 7407 CDS\\n[10:30:29] Connecting features back to sequences\\n[10:30:29] Not using genus-specific database. Try --usegenus to enable it.\\n[10:30:29] Annotating CDS, please be patient.\\n[10:30:29] Will use 4 CPUs for similarity searching.\\n[10:30:31] There are still 7407 unannotated CDS left (started with 7407)\\n[10:30:31] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:30:31] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHJ01\\\\/\\\\/Nostoc_LAHJ01\\\\.IS\\\\.tmp\\\\.2169\\\\.faa | parallel --gnu --plain -j 4 --block 299286 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHJ01\\\\/\\\\/Nostoc_LAHJ01\\\\.IS\\\\.tmp\\\\.2169\\\\.blast 2> /dev/null\\n[10:30:51] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHJ01\\\\/\\\\/Nostoc_LAHJ01\\\\.IS\\\\.tmp\\\\.2169\\\\.faa | parallel --gnu --plain -j 4 --block 299286 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHJ01\\\\/\\\\/Nostoc_LAHJ01\\\\.IS\\\\.tmp\\\\.2169\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:30:51] This is prokka 1.14.6\\n[10:30:51] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:30:51] Homepage is https://github.com/tseemann/prokka\\n[10:30:51] Local time is Mon Feb 21 10:30:51 2022\\n[10:30:51] You are tiagoferreiraleao\\n[10:30:51] Operating system is darwin\\n[10:30:51] You have BioPerl 1.007002\\n[10:30:52] System has 4 cores.\\n[10:30:52] Option --cpu asked for 12 cores, but system only has 4\\n[10:30:52] Will use maximum of 4 cores.\\n[10:30:52] Annotating as >>> Bacteria <<<\\n[10:30:52] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JAIVFY01.fasta' contents.\\n[10:30:52] Setting --locustag MDJGMCFE from MD5 6d306cfebd17449403eb17af1c148ebd\\n[10:30:52] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JAIVFY01/\\n[10:30:52] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAIVFY01\\\\/\\n[10:30:52] Using filename prefix: Nostoc_JAIVFY01.XXX\\n[10:30:52] Setting HMMER_NCPU=1\\n[10:30:52] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JAIVFY01//Nostoc_JAIVFY01.log\\n[10:30:52] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JAIVFY01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JAIVFY01/ --prefix Nostoc_JAIVFY01 --cpus 12\\n[10:30:52] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:30:52] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:30:52] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:30:52] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:30:52] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:30:52] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:30:52] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:30:52] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:30:52] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:30:52] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:30:52] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:30:52] Looking for 'egrep' - found /usr/bin/egrep\\n[10:30:52] Looking for 'find' - found /usr/bin/find\\n[10:30:52] Looking for 'grep' - found /usr/bin/grep\\n[10:30:52] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:30:52] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:30:52] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:30:52] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:30:52] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:30:52] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:30:52] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:30:52] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:30:52] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:30:52] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:30:52] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:30:52] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:30:52] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:30:52] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:30:52] Looking for 'sed' - found /usr/bin/sed\\n[10:30:52] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:30:53] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:30:53] Using genetic code table 11.\\n[10:30:53] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JAIVFY01.fasta\\n[10:30:53] Wrote 211 contigs totalling 8483635 bp.\\n[10:30:53] Predicting tRNAs and tmRNAs\\n[10:30:53] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAIVFY01\\\\/\\\\/Nostoc_JAIVFY01\\\\.fna\\n[10:30:56] 1 tRNA-Gln c[10618,10691] 34 (ttg)\\n[10:30:56] 2 tRNA-Leu c[17491,17575] 35 (caa)\\n[10:30:56] 3 tRNA-Leu [22852,22933] 35 (cag)\\n[10:30:56] 1 tRNA-Gly c[17704,17775] 33 (gcc)\\n[10:30:56] 2 tRNA-Ala [207320,207392] 34 (cgc)\\n[10:30:56] 3 tRNA-Trp c[211922,211995] 34 (cca)\\n[10:30:56] 4 tRNA-Thr [290043,290114] 33 (cgt)\\n[10:30:56] 1 tRNA-Glu [201443,201516] 35 (ttc)\\n[10:30:56] 2 tRNA-Lys [206719,206789] 33 (ctt)\\n[10:30:56] 3 tRNA-Ser [210939,211032] 36 (gct)\\n[10:30:56] 4 tRNA-Ala [285162,285234] 34 (ggc)\\n[10:30:56] 5 tRNA-Met [342253,342329] 35 (cat)\\n[10:30:56] 1 tRNA-Lys c[110284,110355] 33 (ttt)\\n[10:30:56] 2 tRNA-Cys c[118948,119018] 33 (gca)\\n[10:30:56] 3 tRNA-Val c[226269,226342] 35 (gac)\\n[10:30:56] 1 tRNA-Met c[4798,4871] 33 (cat)\\n[10:30:56] 2 tRNA-Pro c[105790,105863] 35 (tgg)\\n[10:30:56] 1 tRNA-Phe c[187414,187486] 34 (gaa)\\n[10:30:56] 1 tRNA-Ser [33838,33924] 36 (cga)\\n[10:30:56] 1 tRNA-His [205870,205943] 34 (gtg)\\n[10:30:56] 1 tRNA-Val c[13170,13242] 34 (tac)\\n[10:30:56] 2 tRNA-Ser c[31210,31296] 36 (tga)\\n[10:30:56] 3 tRNA-Tyr [33596,33681] 35 (gta)\\n[10:30:56] 4 tRNA-Thr [33769,33840] 33 (ggt)\\n[10:30:56] 5 tRNA-Arg c[35636,35710] 35 (tct)\\n[10:30:56] 6 tRNA-Gly c[125644,125714] 33 (tcc)\\n[10:30:56] 1 tRNA-Leu [75271,75353] 36 (gag)\\n[10:30:56] 2 tRNA-Lys [167206,167278] 33 (ctt)\\n[10:30:56] 1 tRNA-Pro c[142822,142895] 35 (cgg)\\n[10:30:56] 1 tRNA-Pro [1731,1804] 35 (ggg)\\n[10:30:56] 2 tRNA-Arg c[39768,39841] 35 (acg)\\n[10:30:56] 3 tRNA-Met c[91569,91643] 35 (cat)\\n[10:30:56] 1 tRNA-Asn c[71292,71363] 33 (gtt)\\n[10:30:56] 1 tRNA-Trp [42297,42371] 35 (cca)\\n[10:30:56] 1 tRNA-Leu c[19294,19376] 35 (tag)\\n[10:30:56] 2 tRNA-Thr [44353,44446] 34 (cgt)\\n[10:30:56] 3 tRNA-Ser [115533,115621] 36 (gga)\\n[10:30:56] 1 tRNA-Ala c[6425,6517] 33 (ggc)\\n[10:30:56] 1 tRNA-Gly [4424,4495] 33 (ccc)\\n[10:30:56] 2 tmRNA c[37428,37818] 101,154 ANNIVSFARKPALVAAA*\\n[10:30:56] 1 tRNA-Asp [30396,30469] 35 (gtc)\\n[10:30:56] 1 tRNA-Arg c[4419,4491] 34 (ccg)\\n[10:30:56] 1 tRNA-Arg c[55943,56018] 36 (cct)\\n[10:30:56] 1 tRNA-Ala c[2776,2850] 35 (tgc)\\n[10:30:56] 1 tRNA-Trp c[19352,19423] 33 (cca)\\n[10:30:56] 2 tRNA-Thr c[20731,20803] 33 (ggt)\\n[10:30:56] 1 tRNA-Thr [10030,10101] 33 (tgt)\\n[10:30:56] 1 tRNA-Ile [1861,1935] 35 (gat)\\n[10:30:56] 2 tRNA-Ala [2005,2079] 35 (tgc)\\n[10:30:56] 1 tRNA-Phe [129,204] 34 (gaa)\\n[10:30:56] Found 50 tRNAs\\n[10:30:56] Predicting Ribosomal RNAs\\n[10:30:56] Running Barrnap with 4 threads\\n[10:30:59] 1 JAIVFY010000153.1 255 16S ribosomal RNA\\n[10:30:59] 2 JAIVFY010000153.1 2183 23S ribosomal RNA\\n[10:30:59] 3 JAIVFY010000153.1 5191 5S ribosomal RNA\\n[10:30:59] 4 JAIVFY010000164.1 5 16S ribosomal RNA\\n[10:30:59] 5 JAIVFY010000164.1 1718 23S ribosomal RNA (partial)\\n[10:30:59] 6 JAIVFY010000164.1 2915 23S ribosomal RNA (partial)\\n[10:30:59] 7 JAIVFY010000181.1 121 5S ribosomal RNA (partial)\\n[10:30:59] 8 JAIVFY010000181.1 312 23S ribosomal RNA\\n[10:30:59] 9 JAIVFY010000192.1 1 23S ribosomal RNA (partial)\\n[10:30:59] Found 9 rRNAs\\n[10:30:59] Skipping ncRNA search, enable with --rfam if desired.\\n[10:30:59] Total of 58 tRNA + rRNA features\\n[10:30:59] Searching for CRISPR repeats\\n[10:31:00] CRISPR1 JAIVFY010000045.1 20621 with 7 spacers\\n[10:31:00] CRISPR2 JAIVFY010000046.1 6593 with 41 spacers\\n[10:31:00] CRISPR3 JAIVFY010000046.1 11136 with 47 spacers\\n[10:31:00] CRISPR4 JAIVFY010000046.1 36508 with 18 spacers\\n[10:31:00] CRISPR5 JAIVFY010000057.1 13435 with 6 spacers\\n[10:31:00] CRISPR6 JAIVFY010000065.1 6963 with 25 spacers\\n[10:31:00] CRISPR7 JAIVFY010000065.1 19771 with 16 spacers\\n[10:31:01] CRISPR8 JAIVFY010000160.1 789 with 4 spacers\\n[10:31:01] Found 8 CRISPRs\\n[10:31:01] Predicting coding sequences\\n[10:31:01] Contigs total 8483635 bp, so using single mode\\n[10:31:01] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAIVFY01\\\\/\\\\/Nostoc_JAIVFY01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:31:10] Excluding CDS which overlaps existing RNA (tRNA) at JAIVFY010000001.1:10600..10764 on + strand\\n[10:31:10] Excluding CDS which overlaps existing RNA (tRNA) at JAIVFY010000002.1:17474..17788 on - strand\\n[10:31:10] Excluding CDS which overlaps existing RNA (tRNA) at JAIVFY010000002.1:211873..212019 on - strand\\n[10:31:11] Excluding CDS which overlaps existing RNA (tRNA) at JAIVFY010000004.1:110202..110387 on + strand\\n[10:31:12] Excluding CDS which overlaps existing RNA (tRNA) at JAIVFY010000010.1:30902..31246 on + strand\\n[10:31:13] Excluding CDS which overlaps existing RNA (tRNA) at JAIVFY010000021.1:43889..44365 on + strand\\n[10:31:14] Excluding CDS which overlaps existing RNA (repeat_region) at JAIVFY010000045.1:20171..20686 on - strand\\n[10:31:14] Excluding CDS which overlaps existing RNA (repeat_region) at JAIVFY010000046.1:7330..7482 on - strand\\n[10:31:14] Excluding CDS which overlaps existing RNA (repeat_region) at JAIVFY010000046.1:7708..8199 on - strand\\n[10:31:14] Excluding CDS which overlaps existing RNA (repeat_region) at JAIVFY010000046.1:8712..8915 on + strand\\n[10:31:14] Excluding CDS which overlaps existing RNA (repeat_region) at JAIVFY010000046.1:9085..9345 on + strand\\n[10:31:14] Excluding CDS which overlaps existing RNA (repeat_region) at JAIVFY010000057.1:13609..13905 on + strand\\n[10:31:14] Excluding CDS which overlaps existing RNA (repeat_region) at JAIVFY010000057.1:13823..14065 on - strand\\n[10:31:14] Excluding CDS which overlaps existing RNA (repeat_region) at JAIVFY010000065.1:7672..7881 on + strand\\n[10:31:14] Excluding CDS which overlaps existing RNA (repeat_region) at JAIVFY010000065.1:8027..8242 on + strand\\n[10:31:15] Excluding CDS which overlaps existing RNA (rRNA) at JAIVFY010000153.1:2622..3026 on - strand\\n[10:31:15] Excluding CDS which overlaps existing RNA (rRNA) at JAIVFY010000153.1:3067..3195 on - strand\\n[10:31:15] Excluding CDS which overlaps existing RNA (rRNA) at JAIVFY010000153.1:4464..4922 on - strand\\n[10:31:15] Excluding CDS which overlaps existing RNA (repeat_region) at JAIVFY010000160.1:1015..1545 on - strand\\n[10:31:15] Excluding CDS which overlaps existing RNA (rRNA) at JAIVFY010000164.1:1136..1450 on + strand\\n[10:31:15] Excluding CDS which overlaps existing RNA (rRNA) at JAIVFY010000181.1:90..299 on - strand\\n[10:31:15] Excluding CDS which overlaps existing RNA (rRNA) at JAIVFY010000181.1:444..902 on + strand\\n[10:31:15] Excluding CDS which overlaps existing RNA (rRNA) at JAIVFY010000192.1:2397..2573 on - strand\\n[10:31:15] Found 7500 CDS\\n[10:31:15] Connecting features back to sequences\\n[10:31:15] Not using genus-specific database. Try --usegenus to enable it.\\n[10:31:15] Annotating CDS, please be patient.\\n[10:31:15] Will use 4 CPUs for similarity searching.\\n[10:31:17] There are still 7500 unannotated CDS left (started with 7500)\\n[10:31:17] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:31:17] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAIVFY01\\\\/\\\\/Nostoc_JAIVFY01\\\\.IS\\\\.tmp\\\\.2281\\\\.faa | parallel --gnu --plain -j 4 --block 285353 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAIVFY01\\\\/\\\\/Nostoc_JAIVFY01\\\\.IS\\\\.tmp\\\\.2281\\\\.blast 2> /dev/null\\n[10:31:36] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAIVFY01\\\\/\\\\/Nostoc_JAIVFY01\\\\.IS\\\\.tmp\\\\.2281\\\\.faa | parallel --gnu --plain -j 4 --block 285353 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAIVFY01\\\\/\\\\/Nostoc_JAIVFY01\\\\.IS\\\\.tmp\\\\.2281\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:31:37] This is prokka 1.14.6\\n[10:31:37] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:31:37] Homepage is https://github.com/tseemann/prokka\\n[10:31:37] Local time is Mon Feb 21 10:31:37 2022\\n[10:31:37] You are tiagoferreiraleao\\n[10:31:37] Operating system is darwin\\n[10:31:37] You have BioPerl 1.007002\\n[10:31:37] System has 4 cores.\\n[10:31:37] Option --cpu asked for 12 cores, but system only has 4\\n[10:31:37] Will use maximum of 4 cores.\\n[10:31:37] Annotating as >>> Bacteria <<<\\n[10:31:37] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAH01.fasta' contents.\\n[10:31:37] Setting --locustag ALGNOIJN from MD5 a5078237810c69164b6b85a65fb85206\\n[10:31:37] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAH01/\\n[10:31:37] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAH01\\\\/\\n[10:31:37] Using filename prefix: Fischerella_NHAH01.XXX\\n[10:31:37] Setting HMMER_NCPU=1\\n[10:31:37] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAH01//Fischerella_NHAH01.log\\n[10:31:37] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAH01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAH01/ --prefix Fischerella_NHAH01 --cpus 12\\n[10:31:37] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:31:37] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:31:37] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:31:37] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:31:37] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:31:37] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:31:37] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:31:37] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:31:37] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:31:37] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:31:37] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:31:37] Looking for 'egrep' - found /usr/bin/egrep\\n[10:31:37] Looking for 'find' - found /usr/bin/find\\n[10:31:37] Looking for 'grep' - found /usr/bin/grep\\n[10:31:37] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:31:37] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:31:37] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:31:37] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:31:37] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:31:37] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:31:37] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:31:37] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:31:38] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:31:38] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:31:38] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:31:38] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:31:38] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:31:38] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:31:38] Looking for 'sed' - found /usr/bin/sed\\n[10:31:38] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:31:38] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:31:38] Using genetic code table 11.\\n[10:31:38] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAH01.fasta\\n[10:31:38] Wrote 106 contigs totalling 5369433 bp.\\n[10:31:38] Predicting tRNAs and tmRNAs\\n[10:31:38] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAH01\\\\/\\\\/Fischerella_NHAH01\\\\.fna\\n[10:31:40] 1 tRNA-Met [9109,9181] 34 (cat)\\n[10:31:40] 1 tRNA-Gln [34607,34678] 33 (ttg)\\n[10:31:40] 1 tRNA-Trp c[55418,55491] 34 (cca)\\n[10:31:40] 1 tRNA-Arg [6855,6927] 34 (tcg)\\n[10:31:40] 2 tRNA-Asn [7598,7671] 34 (att)\\n[10:31:40] 1 tRNA-Phe c[374,446] 34 (gaa)\\n[10:31:40] 2 tRNA-Val [7845,7917] 34 (tac)\\n[10:31:40] 1 tRNA-Gly [14576,14647] 33 (ccc)\\n[10:31:40] 2 tRNA-Thr c[32673,32744] 33 (ggt)\\n[10:31:40] 3 tRNA-Tyr c[32808,32892] 35 (gta)\\n[10:31:40] 1 tRNA-Ala [56638,56712] 35 (cgc)\\n[10:31:40] 2 tRNA-Leu [86260,86342] 35 (cag)\\n[10:31:40] 1 tRNA-Val c[45288,45361] 35 (gac)\\n[10:31:40] 1 tRNA-Asp c[60341,60414] 35 (gtc)\\n[10:31:40] 1 tRNA-Glu [7677,7749] 35 (ttc)\\n[10:31:40] 1 tRNA-Gln [5706,5776] 32 (ttg)\\n[10:31:40] 1 tRNA-Cys c[92,163] 33 (gca)\\n[10:31:40] 1 tRNA-Leu c[76176,76259] 36 (tag)\\n[10:31:40] 1 tRNA-Met [19826,19901] 36 (cat)\\n[10:31:40] 1 tRNA-Arg [42842,42915] 35 (ccg)\\n[10:31:40] 1 tRNA-Ser c[4398,4486] 36 (gga)\\n[10:31:40] 1 tRNA-Ser c[321,407] 36 (tga)\\n[10:31:40] 1 tRNA-Arg [17133,17207] 35 (tct)\\n[10:31:40] 1 tRNA-Pro [59276,59349] 35 (cgg)\\n[10:31:40] 2 tmRNA c[141057,141443] 100,150 ANNIVKFARKEALVAA*\\n[10:31:40] 1 tRNA-Gly c[5804,5874] 33 (tcc)\\n[10:31:40] 1 tRNA-His c[90406,90479] 34 (gtg)\\n[10:31:40] 1 tRNA-Arg [4134,4209] 36 (cct)\\n[10:31:40] 1 tRNA-Ser c[44072,44158] 36 (cga)\\n[10:31:40] 2 tRNA-Ile [50581,50656] 35 (gat)\\n[10:31:40] 3 tRNA-Ala [50739,50813] 34 (tgc)\\n[10:31:40] 4 tRNA-Pro [73328,73402] 35 (tgg)\\n[10:31:40] 5 tRNA-Thr [76349,76420] 33 (tgt)\\n[10:31:40] 6 tRNA-Asn c[177385,177458] 34 (gtt)\\n[10:31:40] 7 tRNA-Gly [194052,194123] 33 (gcc)\\n[10:31:40] 8 tRNA-Ser c[312286,312379] 36 (gct)\\n[10:31:40] 9 tRNA-Lys c[464950,465023] 34 (ttt)\\n[10:31:40] 1 tRNA-Pro [13875,13948] 35 (ggg)\\n[10:31:40] 2 tRNA-Thr [138639,138710] 33 (cgt)\\n[10:31:40] 3 tRNA-Leu [207795,207879] 35 (caa)\\n[10:31:40] 1 tRNA-Leu [28208,28290] 36 (gag)\\n[10:31:40] 1 tRNA-Met c[86576,86652] 35 (cat)\\n[10:31:40] 1 tRNA-Arg c[86793,86866] 35 (acg)\\n[10:31:40] 2 tRNA-Ala c[91758,91831] 34 (ggc)\\n[10:31:40] Found 44 tRNAs\\n[10:31:40] Predicting Ribosomal RNAs\\n[10:31:40] Running Barrnap with 4 threads\\n[10:31:42] 1 NHAH01000093.1 48956 16S ribosomal RNA\\n[10:31:42] 2 NHAH01000093.1 51229 23S ribosomal RNA\\n[10:31:42] 3 NHAH01000093.1 54168 5S ribosomal RNA\\n[10:31:42] Found 3 rRNAs\\n[10:31:42] Skipping ncRNA search, enable with --rfam if desired.\\n[10:31:42] Total of 46 tRNA + rRNA features\\n[10:31:42] Searching for CRISPR repeats\\n[10:31:42] CRISPR1 NHAH01000066.1 76502 with 3 spacers\\n[10:31:42] CRISPR2 NHAH01000070.1 1851 with 99 spacers\\n[10:31:42] CRISPR3 NHAH01000074.1 2901 with 64 spacers\\n[10:31:42] CRISPR4 NHAH01000077.1 82240 with 108 spacers\\n[10:31:42] CRISPR5 NHAH01000088.1 49780 with 16 spacers\\n[10:31:42] CRISPR6 NHAH01000100.1 29832 with 99 spacers\\n[10:31:42] Found 6 CRISPRs\\n[10:31:42] Predicting coding sequences\\n[10:31:42] Contigs total 5369433 bp, so using single mode\\n[10:31:42] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAH01\\\\/\\\\/Fischerella_NHAH01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:31:49] Excluding CDS which overlaps existing RNA (rRNA) at NHAH01000093.1:51624..51803 on + strand\\n[10:31:49] Excluding CDS which overlaps existing RNA (rRNA) at NHAH01000093.1:53589..53765 on - strand\\n[10:31:49] Excluding CDS which overlaps existing RNA (tRNA) at NHAH01000093.1:176145..177584 on - strand\\n[10:31:49] Excluding CDS which overlaps existing RNA (repeat_region) at NHAH01000100.1:31196..31351 on + strand\\n[10:31:49] Excluding CDS which overlaps existing RNA (tRNA) at NHAH01000102.1:86750..87421 on + strand\\n[10:31:49] Found 4558 CDS\\n[10:31:49] Connecting features back to sequences\\n[10:31:49] Not using genus-specific database. Try --usegenus to enable it.\\n[10:31:49] Annotating CDS, please be patient.\\n[10:31:49] Will use 4 CPUs for similarity searching.\\n[10:31:51] There are still 4558 unannotated CDS left (started with 4558)\\n[10:31:51] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:31:51] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAH01\\\\/\\\\/Fischerella_NHAH01\\\\.IS\\\\.tmp\\\\.2393\\\\.faa | parallel --gnu --plain -j 4 --block 174639 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAH01\\\\/\\\\/Fischerella_NHAH01\\\\.IS\\\\.tmp\\\\.2393\\\\.blast 2> /dev/null\\n[10:32:13] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAH01\\\\/\\\\/Fischerella_NHAH01\\\\.IS\\\\.tmp\\\\.2393\\\\.faa | parallel --gnu --plain -j 4 --block 174639 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAH01\\\\/\\\\/Fischerella_NHAH01\\\\.IS\\\\.tmp\\\\.2393\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:32:14] This is prokka 1.14.6\\n[10:32:14] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:32:14] Homepage is https://github.com/tseemann/prokka\\n[10:32:14] Local time is Mon Feb 21 10:32:14 2022\\n[10:32:14] You are tiagoferreiraleao\\n[10:32:14] Operating system is darwin\\n[10:32:14] You have BioPerl 1.007002\\n[10:32:14] System has 4 cores.\\n[10:32:14] Option --cpu asked for 12 cores, but system only has 4\\n[10:32:14] Will use maximum of 4 cores.\\n[10:32:14] Annotating as >>> Bacteria <<<\\n[10:32:14] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAGZ01.fasta' contents.\\n[10:32:14] Setting --locustag INFIMJHD from MD5 27f2631d82225585dc7833054636743a\\n[10:32:14] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAGZ01/\\n[10:32:14] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGZ01\\\\/\\n[10:32:14] Using filename prefix: Nostoc_LAGZ01.XXX\\n[10:32:14] Setting HMMER_NCPU=1\\n[10:32:14] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAGZ01//Nostoc_LAGZ01.log\\n[10:32:14] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAGZ01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAGZ01/ --prefix Nostoc_LAGZ01 --cpus 12\\n[10:32:14] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:32:14] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:32:14] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:32:14] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:32:14] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:32:14] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:32:14] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:32:14] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:32:14] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:32:14] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:32:14] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:32:14] Looking for 'egrep' - found /usr/bin/egrep\\n[10:32:14] Looking for 'find' - found /usr/bin/find\\n[10:32:14] Looking for 'grep' - found /usr/bin/grep\\n[10:32:14] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:32:14] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:32:14] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:32:14] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:32:14] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:32:14] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:32:15] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:32:15] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:32:15] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:32:15] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:32:15] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:32:15] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:32:15] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:32:15] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:32:15] Looking for 'sed' - found /usr/bin/sed\\n[10:32:15] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:32:15] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:32:15] Using genetic code table 11.\\n[10:32:15] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAGZ01.fasta\\n[10:32:16] Wrote 134 contigs totalling 8984433 bp.\\n[10:32:16] Predicting tRNAs and tmRNAs\\n[10:32:16] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGZ01\\\\/\\\\/Nostoc_LAGZ01\\\\.fna\\n[10:32:19] 1 tRNA-Gly c[230867,230937] 33 (tcc)\\n[10:32:19] 1 tRNA-Cys [96377,96448] 33 (gca)\\n[10:32:19] 2 tRNA-His [246305,246378] 34 (gtg)\\n[10:32:19] 1 tRNA-Thr [162453,162524] 33 (cgt)\\n[10:32:19] 2 tRNA-Trp [208154,208227] 34 (cca)\\n[10:32:19] 1 tmRNA c[193958,194345] 101,151 ANNIVKFARREALVAA*\\n[10:32:19] 1 tRNA-Ala c[125002,125074] 34 (tgc)\\n[10:32:19] 2 tRNA-Ser c[241821,241914] 36 (gct)\\n[10:32:19] 3 tRNA-Lys c[247589,247659] 33 (ttt)\\n[10:32:19] 4 tRNA-Ser c[247918,248001] 34 (tga)\\n[10:32:19] 5 tRNA-His c[248006,248081] 35 (gtg)\\n[10:32:19] 6 tRNA-Ala c[248087,248161] 35 (tgc)\\n[10:32:19] 7 tRNA-Arg c[248253,248328] 35 (tct)\\n[10:32:19] 8 tRNA-Val c[248335,248410] 34 (tac)\\n[10:32:19] 9 tRNA-Thr c[248492,248564] 34 (tgt)\\n[10:32:19] 10 tRNA-Leu c[248859,248933] 35 (tag)\\n[10:32:19] 11 tRNA-Leu c[249045,249121] 34 (gag)\\n[10:32:19] 12 tRNA-Pro c[249229,249302] 35 (tgg)\\n[10:32:19] 13 tRNA-Leu c[249304,249381] 35 (taa)\\n[10:32:19] 14 tRNA-Leu c[249402,249479] 36 (caa)\\n[10:32:19] 15 tRNA-Gln c[249505,249580] 35 (ttg)\\n[10:32:19] 16 tRNA-Gln c[250455,250529] 35 (ctg)\\n[10:32:19] 17 tRNA-Asn c[250540,250614] 33 (gtt)\\n[10:32:19] 18 tRNA-Phe c[250620,250695] 36 (gaa)\\n[10:32:19] 19 tRNA-Arg c[250783,250858] 35 (acg)\\n[10:32:19] 20 tRNA-Ser c[251620,251706] 35 (gct)\\n[10:32:19] 21 tRNA-Tyr c[251710,251791] 35 (gta)\\n[10:32:19] 22 tRNA-Glu c[251802,251876] 35 (ttc)\\n[10:32:19] tRNA c[251964,252038] is a pseudo/wacky gene - skipping.\\n[10:32:19] 24 tRNA-Met c[252041,252114] 32 (cat)\\n[10:32:19] 25 tRNA-Glu c[252224,252300] 35 (ctc)\\n[10:32:19] 26 tRNA-Asp c[252304,252379] 36 (gtc)\\n[10:32:19] 27 tRNA-Glu [326027,326100] 35 (ttc)\\n[10:32:19] 1 tRNA-Asn [28794,28867] 34 (gtt)\\n[10:32:19] 2 tRNA-Leu c[128466,128549] 36 (tag)\\n[10:32:19] 1 tRNA-Asp c[60227,60300] 35 (gtc)\\n[10:32:19] 1 tRNA-Gln c[36045,36118] 34 (ttg)\\n[10:32:19] 2 tRNA-Leu c[40872,40956] 35 (caa)\\n[10:32:19] 3 tRNA-Ala [89658,89730] 34 (cgc)\\n[10:32:19] 1 tRNA-Ser [56572,56659] 35 (gga)\\n[10:32:19] 2 tRNA-Gly c[64804,64877] 34 (ccc)\\n[10:32:19] 1 tRNA-Ala [166145,166217] 34 (ggc)\\n[10:32:19] 1 tRNA-Ala c[3146,3232] 29 (cgc)\\n[10:32:19] 2 tRNA-Ser [117998,118084] 36 (cga)\\n[10:32:19] 1 tRNA-Pro c[43705,43778] 35 (ggg)\\n[10:32:19] 1 tRNA-Lys [80170,80241] 33 (ttt)\\n[10:32:19] 1 tRNA-Lys c[21226,21298] 33 (ctt)\\n[10:32:19] 1 tRNA-Val [64629,64701] 34 (tac)\\n[10:32:19] 1 tRNA-Leu [11044,11125] 35 (cag)\\n[10:32:19] 1 tRNA-Pro [56888,56961] 35 (cgg)\\n[10:32:19] 1 tRNA-Thr [101827,101899] 33 (tgt)\\n[10:32:19] 1 tRNA-Thr [37448,37522] 33 (agt)\\n[10:32:19] 2 tRNA-Leu [38913,38987] 35 (caa)\\n[10:32:19] 1 tRNA-Pro c[38453,38526] 35 (tgg)\\n[10:32:19] 2 tRNA-Ser c[47208,47294] 36 (tga)\\n[10:32:19] 3 tRNA-Tyr [49745,49829] 35 (gta)\\n[10:32:19] 4 tRNA-Thr [49905,49976] 33 (ggt)\\n[10:32:19] 1 tRNA-Leu [61580,61660] 35 (gag)\\n[10:32:19] 1 tRNA-Ile c[31791,31866] 34 (tat)\\n[10:32:19] 2 tRNA-Met c[33472,33547] 34 (cat)\\n[10:32:19] 3 tRNA-Met c[33549,33624] 34 (cat)\\n[10:32:19] 1 tRNA-Arg c[3738,3812] 35 (tct)\\n[10:32:19] 1 tRNA-Met [18633,18706] 33 (cat)\\n[10:32:19] 2 tRNA-Gly [25516,25587] 33 (gcc)\\n[10:32:19] 1 tRNA-Asp [25382,25459] 38 (gtc)\\n[10:32:19] 2 tRNA-Glu [25462,25538] 35 (ctc)\\n[10:32:19] 3 tRNA-Cys [25543,25618] 34 (gca)\\n[10:32:19] 4 tRNA-Met [25624,25698] 33 (cat)\\n[10:32:19] 5 tRNA-Trp [25701,25773] 34 (cca)\\n[10:32:19] 6 tRNA-Pyl [25798,25869] 33 (cta)\\n[10:32:19] 7 tRNA-Glu [25900,25975] 34 (ttc)\\n[10:32:19] 8 tRNA-Tyr [25979,26062] 36 (gta)\\n[10:32:19] 9 tRNA-Ser [26066,26155] 33 (gct)\\n[10:32:19] 10 tRNA-Arg [26160,26235] 35 (acg)\\n[10:32:19] 11 tRNA-Phe [26240,26314] 35 (gaa)\\n[10:32:19] 12 tRNA-Asn [26323,26397] 33 (gtt)\\n[10:32:19] 13 tRNA-Gln [26403,26478] 35 (ttg)\\n[10:32:19] 14 tRNA-Leu [26486,26563] 35 (caa)\\n[10:32:19] 15 tRNA-Pro [26568,26644] 35 (tgg)\\n[10:32:19] 16 tRNA-Leu [26648,26722] 35 (tag)\\n[10:32:19] 17 tRNA-Leu [26742,26819] 35 (gag)\\n[10:32:19] 18 tRNA-Lys [26823,26899] 34 (ttt)\\n[10:32:19] 19 tRNA-Lys [26905,26979] 33 (ctt)\\n[10:32:19] 20 tRNA-Leu [26985,27060] 36 (taa)\\n[10:32:19] 21 tRNA-Cys [27070,27147] 34 (gca)\\n[10:32:19] 22 tRNA-Thr [27155,27227] 34 (tgt)\\n[10:32:19] 23 tRNA-Gly [27233,27303] 33 (tcc)\\n[10:32:19] 24 tRNA-Val [27309,27384] 34 (tac)\\n[10:32:19] 25 tRNA-Arg [27391,27465] 35 (tct)\\n[10:32:19] 26 tRNA-Ala [27470,27542] 34 (tgc)\\n[10:32:19] 27 tRNA-His [27550,27626] 35 (gtg)\\n[10:32:19] 28 tRNA-Ser [27637,27721] 34 (tga)\\n[10:32:19] 1 tRNA-Arg [4913,4985] 34 (ccg)\\n[10:32:19] 1 tRNA-Met c[16793,16867] 35 (cat)\\n[10:32:19] 1 tRNA-Arg c[19851,19924] 35 (acg)\\n[10:32:19] 1 tRNA-Ile [14294,14369] 35 (gat)\\n[10:32:19] 2 tRNA-Ala [14453,14527] 34 (tgc)\\n[10:32:19] 1 tRNA-Val [14069,14142] 35 (gac)\\n[10:32:19] Found 97 tRNAs\\n[10:32:19] Predicting Ribosomal RNAs\\n[10:32:19] Running Barrnap with 4 threads\\n[10:32:21] 1 LAGZ01000104.1 12687 16S ribosomal RNA\\n[10:32:21] 2 LAGZ01000104.1 14700 23S ribosomal RNA\\n[10:32:21] Found 2 rRNAs\\n[10:32:21] Skipping ncRNA search, enable with --rfam if desired.\\n[10:32:21] Total of 98 tRNA + rRNA features\\n[10:32:21] Searching for CRISPR repeats\\n[10:32:21] CRISPR1 LAGZ01000003.1 111797 with 24 spacers\\n[10:32:21] CRISPR2 LAGZ01000004.1 169330 with 14 spacers\\n[10:32:21] CRISPR3 LAGZ01000005.1 103199 with 15 spacers\\n[10:32:21] CRISPR4 LAGZ01000010.1 151828 with 18 spacers\\n[10:32:21] CRISPR5 LAGZ01000016.1 63989 with 7 spacers\\n[10:32:21] CRISPR6 LAGZ01000017.1 86 with 22 spacers\\n[10:32:21] CRISPR7 LAGZ01000017.1 83435 with 10 spacers\\n[10:32:21] CRISPR8 LAGZ01000023.1 8681 with 7 spacers\\n[10:32:22] CRISPR9 LAGZ01000031.1 104742 with 17 spacers\\n[10:32:22] CRISPR10 LAGZ01000049.1 15913 with 20 spacers\\n[10:32:22] CRISPR11 LAGZ01000064.1 26335 with 28 spacers\\n[10:32:22] CRISPR12 LAGZ01000064.1 39769 with 43 spacers\\n[10:32:22] CRISPR13 LAGZ01000119.1 4982 with 6 spacers\\n[10:32:22] Found 13 CRISPRs\\n[10:32:22] Predicting coding sequences\\n[10:32:22] Contigs total 8984433 bp, so using single mode\\n[10:32:22] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGZ01\\\\/\\\\/Nostoc_LAGZ01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:32:30] Excluding CDS which overlaps existing RNA (tRNA) at LAGZ01000002.1:246226..246369 on - strand\\n[10:32:30] Excluding CDS which overlaps existing RNA (repeat_region) at LAGZ01000004.1:170046..170162 on + strand\\n[10:32:32] Excluding CDS which overlaps existing RNA (repeat_region) at LAGZ01000017.1:1540..1818 on - strand\\n[10:32:32] Excluding CDS which overlaps existing RNA (repeat_region) at LAGZ01000017.1:83644..83796 on + strand\\n[10:32:32] Excluding CDS which overlaps existing RNA (repeat_region) at LAGZ01000023.1:9001..9126 on + strand\\n[10:32:33] Excluding CDS which overlaps existing RNA (tRNA) at LAGZ01000026.1:2681..4051 on - strand\\n[10:32:34] Excluding CDS which overlaps existing RNA (repeat_region) at LAGZ01000064.1:39810..39974 on - strand\\n[10:32:35] Excluding CDS which overlaps existing RNA (repeat_region) at LAGZ01000119.1:5374..6333 on - strand\\n[10:32:35] Found 7590 CDS\\n[10:32:35] Connecting features back to sequences\\n[10:32:35] Not using genus-specific database. Try --usegenus to enable it.\\n[10:32:35] Annotating CDS, please be patient.\\n[10:32:35] Will use 4 CPUs for similarity searching.\\n[10:32:37] There are still 7590 unannotated CDS left (started with 7590)\\n[10:32:37] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:32:37] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGZ01\\\\/\\\\/Nostoc_LAGZ01\\\\.IS\\\\.tmp\\\\.2508\\\\.faa | parallel --gnu --plain -j 4 --block 304097 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGZ01\\\\/\\\\/Nostoc_LAGZ01\\\\.IS\\\\.tmp\\\\.2508\\\\.blast 2> /dev/null\\n[10:32:59] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGZ01\\\\/\\\\/Nostoc_LAGZ01\\\\.IS\\\\.tmp\\\\.2508\\\\.faa | parallel --gnu --plain -j 4 --block 304097 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGZ01\\\\/\\\\/Nostoc_LAGZ01\\\\.IS\\\\.tmp\\\\.2508\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:32:59] This is prokka 1.14.6\\n[10:32:59] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:32:59] Homepage is https://github.com/tseemann/prokka\\n[10:32:59] Local time is Mon Feb 21 10:32:59 2022\\n[10:32:59] You are tiagoferreiraleao\\n[10:32:59] Operating system is darwin\\n[10:32:59] You have BioPerl 1.007002\\n[10:33:00] System has 4 cores.\\n[10:33:00] Option --cpu asked for 12 cores, but system only has 4\\n[10:33:00] Will use maximum of 4 cores.\\n[10:33:00] Annotating as >>> Bacteria <<<\\n[10:33:00] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZV01.fasta' contents.\\n[10:33:00] Setting --locustag PBJJKLMB from MD5 9b33456bb9e5020bddf9cb939de7b9fc\\n[10:33:00] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZV01/\\n[10:33:00] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZV01\\\\/\\n[10:33:00] Using filename prefix: Fischerella_NGZV01.XXX\\n[10:33:00] Setting HMMER_NCPU=1\\n[10:33:00] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZV01//Fischerella_NGZV01.log\\n[10:33:00] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZV01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZV01/ --prefix Fischerella_NGZV01 --cpus 12\\n[10:33:00] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:33:00] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:33:00] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:33:00] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:33:00] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:33:00] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:33:00] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:33:00] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:33:00] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:33:00] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:33:00] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:33:00] Looking for 'egrep' - found /usr/bin/egrep\\n[10:33:00] Looking for 'find' - found /usr/bin/find\\n[10:33:00] Looking for 'grep' - found /usr/bin/grep\\n[10:33:00] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:33:00] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:33:00] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:33:00] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:33:00] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:33:00] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:33:00] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:33:00] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:33:01] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:33:01] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:33:01] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:33:01] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:33:01] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:33:01] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:33:01] Looking for 'sed' - found /usr/bin/sed\\n[10:33:01] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:33:01] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:33:01] Using genetic code table 11.\\n[10:33:01] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZV01.fasta\\n[10:33:01] Wrote 109 contigs totalling 5375759 bp.\\n[10:33:01] Predicting tRNAs and tmRNAs\\n[10:33:01] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZV01\\\\/\\\\/Fischerella_NGZV01\\\\.fna\\n[10:33:04] 1 tRNA-Glu [7511,7583] 35 (ttc)\\n[10:33:04] 1 tRNA-Gly [12157,12227] 33 (tcc)\\n[10:33:04] 1 tRNA-Arg [33962,34037] 36 (cct)\\n[10:33:04] 1 tRNA-Gly c[82833,82904] 33 (gcc)\\n[10:33:04] 1 tRNA-Met [1633,1709] 35 (cat)\\n[10:33:04] 2 tRNA-Lys c[104723,104796] 34 (ttt)\\n[10:33:04] 1 tRNA-Asn c[3583,3656] 34 (att)\\n[10:33:04] 2 tRNA-Arg c[4327,4399] 34 (tcg)\\n[10:33:04] 1 tRNA-Val c[2777,2849] 34 (tac)\\n[10:33:04] 2 tRNA-Phe [10248,10320] 34 (gaa)\\n[10:33:04] 1 tRNA-Ser c[53442,53535] 36 (gct)\\n[10:33:04] 1 tRNA-Asn [109274,109347] 34 (gtt)\\n[10:33:04] 1 tRNA-Ile [2139,2213] 35 (gat)\\n[10:33:04] 2 tmRNA* [2200,2682] 351,392 EDPQMSEWGNPAG*\\n[10:33:04] 3 tRNA-Ala [2219,2293] 35 (tgc)\\n[10:33:04] 1 tRNA-Arg [42655,42728] 35 (ccg)\\n[10:33:04] 2 tRNA-Trp [87162,87235] 34 (cca)\\n[10:33:04] 1 tRNA-Cys [4017,4088] 33 (gca)\\n[10:33:04] 1 tRNA-His c[91817,91890] 34 (gtg)\\n[10:33:04] 1 tRNA-Gln c[23526,23597] 33 (ttg)\\n[10:33:04] 2 tRNA-Leu c[59769,59853] 35 (caa)\\n[10:33:04] 1 tRNA-Pro [57683,57756] 35 (cgg)\\n[10:33:04] 2 tmRNA c[137611,137997] 100,150 ANNIVKFARKEALVAA*\\n[10:33:04] 1 tRNA-Ser c[28228,28314] 36 (tga)\\n[10:33:04] 1 tRNA-Ala c[45432,45506] 35 (cgc)\\n[10:33:04] 1 tRNA-Ser [73079,73167] 36 (gga)\\n[10:33:04] 1 tRNA-Met c[20823,20895] 34 (cat)\\n[10:33:04] 2 tRNA-Pro [54091,54165] 35 (tgg)\\n[10:33:04] 1 tRNA-Leu [27552,27634] 36 (gag)\\n[10:33:04] 1 tRNA-Pro [49300,49373] 35 (ggg)\\n[10:33:04] 1 tRNA-Val [1389,1462] 35 (gac)\\n[10:33:04] 1 tRNA-Arg [72903,72977] 35 (tct)\\n[10:33:04] 2 tRNA-Met [97864,97939] 36 (cat)\\n[10:33:04] 1 tRNA-Gln c[106404,106474] 32 (ttg)\\n[10:33:04] 1 tRNA-Thr [7614,7685] 33 (tgt)\\n[10:33:04] 2 tRNA-Arg c[186914,186987] 35 (acg)\\n[10:33:04] 3 tRNA-Ala c[191888,191961] 34 (ggc)\\n[10:33:04] 4 tRNA-Thr c[261281,261352] 33 (cgt)\\n[10:33:04] 1 tRNA-Asp c[60078,60151] 35 (gtc)\\n[10:33:04] 1 tRNA-Tyr [13383,13467] 35 (gta)\\n[10:33:04] 2 tRNA-Thr [13531,13602] 33 (ggt)\\n[10:33:04] 3 tRNA-Gly c[31389,31460] 33 (ccc)\\n[10:33:04] 4 tRNA-Leu [110799,110881] 35 (cag)\\n[10:33:04] 1 tRNA-Leu c[20478,20561] 36 (tag)\\n[10:33:04] 2 tRNA-Ser c[112216,112302] 36 (cga)\\n[10:33:04] Found 45 tRNAs\\n[10:33:04] Predicting Ribosomal RNAs\\n[10:33:04] Running Barrnap with 4 threads\\n[10:33:05] 1 NGZV01000052.1 490 16S ribosomal RNA\\n[10:33:05] 2 NGZV01000052.1 2462 23S ribosomal RNA\\n[10:33:05] 3 NGZV01000052.1 5414 5S ribosomal RNA\\n[10:33:05] 4 NGZV01000079.1 29964 16S ribosomal RNA\\n[10:33:05] 5 NGZV01000079.1 31811 23S ribosomal RNA (partial)\\n[10:33:05] 6 NGZV01000079.1 34931 5S ribosomal RNA\\n[10:33:05] 7 NGZV01000103.1 133 16S ribosomal RNA\\n[10:33:05] 8 NGZV01000103.1 2043 23S ribosomal RNA (partial)\\n[10:33:05] 9 NGZV01000103.1 5235 5S ribosomal RNA\\n[10:33:05] Found 9 rRNAs\\n[10:33:05] Skipping ncRNA search, enable with --rfam if desired.\\n[10:33:05] Total of 52 tRNA + rRNA features\\n[10:33:05] Searching for CRISPR repeats\\n[10:33:06] CRISPR1 NGZV01000081.1 1338 with 6 spacers\\n[10:33:06] CRISPR2 NGZV01000090.1 77 with 4 spacers\\n[10:33:06] CRISPR3 NGZV01000090.1 588 with 19 spacers\\n[10:33:06] CRISPR4 NGZV01000090.1 2094 with 16 spacers\\n[10:33:06] CRISPR5 NGZV01000090.1 3706 with 5 spacers\\n[10:33:06] CRISPR6 NGZV01000090.1 4932 with 4 spacers\\n[10:33:06] CRISPR7 NGZV01000095.1 32 with 5 spacers\\n[10:33:06] CRISPR8 NGZV01000095.1 453 with 5 spacers\\n[10:33:06] CRISPR9 NGZV01000095.1 948 with 3 spacers\\n[10:33:06] CRISPR10 NGZV01000099.1 2907 with 4 spacers\\n[10:33:06] CRISPR11 NGZV01000099.1 3356 with 3 spacers\\n[10:33:06] CRISPR12 NGZV01000099.1 4134 with 5 spacers\\n[10:33:06] CRISPR13 NGZV01000099.1 4661 with 9 spacers\\n[10:33:06] CRISPR14 NGZV01000099.1 5391 with 7 spacers\\n[10:33:06] CRISPR15 NGZV01000099.1 6076 with 3 spacers\\n[10:33:06] CRISPR16 NGZV01000099.1 6304 with 5 spacers\\n[10:33:06] CRISPR17 NGZV01000099.1 6975 with 4 spacers\\n[10:33:06] CRISPR18 NGZV01000099.1 7327 with 5 spacers\\n[10:33:06] CRISPR19 NGZV01000099.1 7782 with 6 spacers\\n[10:33:06] CRISPR20 NGZV01000101.1 20804 with 3 spacers\\n[10:33:06] CRISPR21 NGZV01000102.1 101 with 5 spacers\\n[10:33:06] CRISPR22 NGZV01000102.1 653 with 4 spacers\\n[10:33:06] CRISPR23 NGZV01000102.1 1265 with 6 spacers\\n[10:33:06] CRISPR24 NGZV01000102.1 1911 with 3 spacers\\n[10:33:06] CRISPR25 NGZV01000102.1 2143 with 9 spacers\\n[10:33:06] CRISPR26 NGZV01000102.1 3369 with 3 spacers\\n[10:33:06] CRISPR27 NGZV01000108.1 19785 with 3 spacers\\n[10:33:06] CRISPR28 NGZV01000108.1 20165 with 3 spacers\\n[10:33:06] CRISPR29 NGZV01000108.1 20531 with 4 spacers\\n[10:33:06] CRISPR30 NGZV01000108.1 21016 with 7 spacers\\n[10:33:06] CRISPR31 NGZV01000108.1 21808 with 9 spacers\\n[10:33:06] CRISPR32 NGZV01000108.1 22758 with 6 spacers\\n[10:33:06] CRISPR33 NGZV01000108.1 23352 with 5 spacers\\n[10:33:06] CRISPR34 NGZV01000108.1 23855 with 5 spacers\\n[10:33:06] Found 34 CRISPRs\\n[10:33:06] Predicting coding sequences\\n[10:33:06] Contigs total 5375759 bp, so using single mode\\n[10:33:06] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZV01\\\\/\\\\/Fischerella_NGZV01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:33:12] Excluding CDS which overlaps existing RNA (tRNA) at NGZV01000050.1:109148..110587 on + strand\\n[10:33:13] Excluding CDS which overlaps existing RNA (rRNA) at NGZV01000079.1:32206..32385 on + strand\\n[10:33:14] Excluding CDS which overlaps existing RNA (tRNA) at NGZV01000093.1:186871..187542 on + strand\\n[10:33:14] Excluding CDS which overlaps existing RNA (rRNA) at NGZV01000103.1:699..1193 on - strand\\n[10:33:14] Found 4564 CDS\\n[10:33:14] Connecting features back to sequences\\n[10:33:14] Not using genus-specific database. Try --usegenus to enable it.\\n[10:33:14] Annotating CDS, please be patient.\\n[10:33:14] Will use 4 CPUs for similarity searching.\\n[10:33:15] There are still 4564 unannotated CDS left (started with 4564)\\n[10:33:15] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:33:15] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZV01\\\\/\\\\/Fischerella_NGZV01\\\\.IS\\\\.tmp\\\\.2620\\\\.faa | parallel --gnu --plain -j 4 --block 179217 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZV01\\\\/\\\\/Fischerella_NGZV01\\\\.IS\\\\.tmp\\\\.2620\\\\.blast 2> /dev/null\\n[10:33:22] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZV01\\\\/\\\\/Fischerella_NGZV01\\\\.IS\\\\.tmp\\\\.2620\\\\.faa | parallel --gnu --plain -j 4 --block 179217 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZV01\\\\/\\\\/Fischerella_NGZV01\\\\.IS\\\\.tmp\\\\.2620\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:33:22] This is prokka 1.14.6\\n[10:33:22] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:33:22] Homepage is https://github.com/tseemann/prokka\\n[10:33:22] Local time is Mon Feb 21 10:33:22 2022\\n[10:33:22] You are tiagoferreiraleao\\n[10:33:22] Operating system is darwin\\n[10:33:22] You have BioPerl 1.007002\\n[10:33:22] System has 4 cores.\\n[10:33:22] Option --cpu asked for 12 cores, but system only has 4\\n[10:33:22] Will use maximum of 4 cores.\\n[10:33:22] Annotating as >>> Bacteria <<<\\n[10:33:22] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKO01.fasta' contents.\\n[10:33:22] Setting --locustag IIGODLAO from MD5 2208d5a81b6ee6c3da591f0ce4465497\\n[10:33:22] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKO01/\\n[10:33:22] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKO01\\\\/\\n[10:33:22] Using filename prefix: Nostoc_JABXKO01.XXX\\n[10:33:22] Setting HMMER_NCPU=1\\n[10:33:22] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKO01//Nostoc_JABXKO01.log\\n[10:33:22] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKO01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKO01/ --prefix Nostoc_JABXKO01 --cpus 12\\n[10:33:22] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:33:22] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:33:22] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:33:22] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:33:22] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:33:22] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:33:23] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:33:23] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:33:23] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:33:23] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:33:23] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:33:23] Looking for 'egrep' - found /usr/bin/egrep\\n[10:33:23] Looking for 'find' - found /usr/bin/find\\n[10:33:23] Looking for 'grep' - found /usr/bin/grep\\n[10:33:23] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:33:23] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:33:23] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:33:23] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:33:23] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:33:23] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:33:23] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:33:23] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:33:23] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:33:23] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:33:23] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:33:23] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:33:23] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:33:23] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:33:23] Looking for 'sed' - found /usr/bin/sed\\n[10:33:23] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:33:24] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:33:24] Using genetic code table 11.\\n[10:33:24] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKO01.fasta\\n[10:33:24] Wrote 123 contigs totalling 7630187 bp.\\n[10:33:24] Predicting tRNAs and tmRNAs\\n[10:33:24] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKO01\\\\/\\\\/Nostoc_JABXKO01\\\\.fna\\n[10:33:27] 1 tRNA-Ala [5770,5844] 35 (tgc)\\n[10:33:27] 1 tRNA-Phe c[23724,23796] 34 (gaa)\\n[10:33:27] 2 tmRNA [48921,49309] 101,151 ANNIVSFARKDALVAV*\\n[10:33:27] 1 tRNA-Met [61537,61613] 35 (cat)\\n[10:33:27] 1 tRNA-Val [6179,6252] 35 (gac)\\n[10:33:27] 1 tRNA-Ser c[103826,103912] 36 (gga)\\n[10:33:27] 2 tRNA-Arg [158640,158712] 34 (ccg)\\n[10:33:27] 1 tRNA-Pro c[8602,8675] 35 (ggg)\\n[10:33:27] 1 tRNA-His [31764,31837] 34 (gtg)\\n[10:33:27] 1 tRNA-Arg [254350,254423] 35 (acg)\\n[10:33:27] 1 tRNA-Met [14987,15062] 34 (cat)\\n[10:33:27] 2 tRNA-Met [15063,15134] 32 (cat)\\n[10:33:27] 3 tRNA-Val c[54496,54568] 34 (tac)\\n[10:33:27] 4 tRNA-Ser c[77277,77363] 36 (tga)\\n[10:33:27] 5 tRNA-Tyr [79664,79749] 35 (gta)\\n[10:33:27] 6 tRNA-Thr [79837,79908] 33 (ggt)\\n[10:33:27] 7 tRNA-Arg c[98700,98774] 35 (tct)\\n[10:33:27] 1 tRNA-Gly c[37954,38025] 33 (gcc)\\n[10:33:27] 1 tRNA-Val [46906,46978] 33 (cac)\\n[10:33:27] 1 tRNA-Thr c[1041,1112] 33 (cgt)\\n[10:33:27] 1 tRNA-Trp [7390,7463] 34 (cca)\\n[10:33:27] 2 tRNA-Thr c[62743,62814] 33 (tgt)\\n[10:33:27] 1 tRNA-Lys c[22714,22786] 33 (ctt)\\n[10:33:27] 1 tRNA-Ala c[3822,3896] 35 (ggc)\\n[10:33:27] 1 tRNA-Ala [72629,72701] 34 (cgc)\\n[10:33:27] 2 tRNA-Phe c[134360,134432] 33 (gaa)\\n[10:33:27] 1 tRNA-Leu c[6343,6423] 34 (cag)\\n[10:33:27] 1 tRNA-Ser c[12502,12588] 36 (cga)\\n[10:33:27] 2 tRNA-Arg [30409,30484] 36 (cct)\\n[10:33:27] 3 tRNA-Lys c[36779,36850] 33 (ttt)\\n[10:33:27] 1 tRNA-Asp c[65822,65895] 35 (gtc)\\n[10:33:27] 1 tRNA-Pro c[35105,35178] 35 (cgg)\\n[10:33:27] 1 tRNA-Asn c[234,305] 33 (gtt)\\n[10:33:27] 1 tRNA-Gly [55376,55446] 33 (tcc)\\n[10:33:27] 2 tRNA-Leu [170435,170517] 36 (gag)\\n[10:33:27] 1 tRNA-Met c[3634,3708] 35 (cat)\\n[10:33:27] 1 tRNA-Gly c[50550,50621] 33 (ccc)\\n[10:33:27] 1 tRNA-Cys c[68015,68085] 33 (gca)\\n[10:33:27] 1 tRNA-Pro [80835,80908] 35 (tgg)\\n[10:33:27] 1 tRNA-Leu [16434,16518] 35 (caa)\\n[10:33:27] 2 tRNA-Gln [18339,18412] 34 (ttg)\\n[10:33:27] 1 tRNA-Glu [165176,165249] 35 (ttc)\\n[10:33:27] 2 tRNA-Asp [171714,171789] 36 (gtc)\\n[10:33:27] 3 tRNA-Glu [171793,171869] 35 (ctc)\\n[10:33:27] 4 tRNA-Cys [171873,171946] 35 (gca)\\n[10:33:27] 5 tRNA-Met [171953,172026] 32 (cat)\\n[10:33:27] tRNA [172029,172103] is a pseudo/wacky gene - skipping.\\n[10:33:27] 7 tRNA-Glu [172165,172238] 35 (ttc)\\n[10:33:27] 8 tRNA-Ser [172350,172441] 36 (gct)\\n[10:33:27] 9 tRNA-Arg [172444,172520] 36 (acg)\\n[10:33:27] 10 tRNA-Phe [172541,172616] 36 (gaa)\\n[10:33:27] 11 tRNA-Leu [172824,172898] 35 (tag)\\n[10:33:27] 12 tRNA-Met [172908,172984] 34 (cat)\\n[10:33:27] 13 tRNA-Thr [172992,173064] 34 (tgt)\\n[10:33:27] 14 tRNA-Val [173146,173221] 34 (tac)\\n[10:33:27] 15 tRNA-Arg [173228,173303] 35 (tct)\\n[10:33:27] 16 tRNA-Ala [173386,173460] 35 (tgc)\\n[10:33:27] 17 tRNA-His [173469,173544] 35 (gtg)\\n[10:33:27] 18 tRNA-Lys [173807,173878] 33 (ttt)\\n[10:33:27] 19 tRNA-Ser [177994,178085] 35 (gct)\\n[10:33:27] 1 tRNA-Leu [27655,27737] 35 (tag)\\n[10:33:27] 1 tRNA-Met c[112356,112429] 33 (cat)\\n[10:33:27] Found 61 tRNAs\\n[10:33:27] Predicting Ribosomal RNAs\\n[10:33:27] Running Barrnap with 4 threads\\n[10:33:28] Found 0 rRNAs\\n[10:33:28] Skipping ncRNA search, enable with --rfam if desired.\\n[10:33:28] Total of 60 tRNA + rRNA features\\n[10:33:28] Searching for CRISPR repeats\\n[10:33:28] CRISPR1 JABXKO010000010.1 22776 with 4 spacers\\n[10:33:28] CRISPR2 JABXKO010000022.1 159061 with 4 spacers\\n[10:33:28] CRISPR3 JABXKO010000031.1 18935 with 10 spacers\\n[10:33:28] CRISPR4 JABXKO010000068.1 72984 with 5 spacers\\n[10:33:28] CRISPR5 JABXKO010000068.1 95721 with 8 spacers\\n[10:33:28] CRISPR6 JABXKO010000078.1 70570 with 5 spacers\\n[10:33:28] CRISPR7 JABXKO010000088.1 22180 with 4 spacers\\n[10:33:28] CRISPR8 JABXKO010000104.1 3707 with 6 spacers\\n[10:33:28] CRISPR9 JABXKO010000121.1 4160 with 7 spacers\\n[10:33:28] Found 9 CRISPRs\\n[10:33:28] Predicting coding sequences\\n[10:33:28] Contigs total 7630187 bp, so using single mode\\n[10:33:28] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKO01\\\\/\\\\/Nostoc_JABXKO01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:33:35] Excluding CDS which overlaps existing RNA (tRNA) at JABXKO010000022.1:103528..103866 on + strand\\n[10:33:35] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKO010000031.1:18897..19091 on - strand\\n[10:33:35] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKO010000031.1:19200..19499 on - strand\\n[10:33:35] Excluding CDS which overlaps existing RNA (tRNA) at JABXKO010000042.1:15093..15308 on - strand\\n[10:33:36] Excluding CDS which overlaps existing RNA (tRNA) at JABXKO010000068.1:72603..72836 on - strand\\n[10:33:36] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKO010000068.1:95794..96306 on - strand\\n[10:33:36] Excluding CDS which overlaps existing RNA (tRNA) at JABXKO010000075.1:6342..6503 on + strand\\n[10:33:36] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKO010000078.1:70581..70694 on - strand\\n[10:33:36] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKO010000078.1:70817..71035 on + strand\\n[10:33:37] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKO010000104.1:3709..3867 on - strand\\n[10:33:37] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKO010000104.1:3904..4110 on - strand\\n[10:33:38] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKO010000121.1:3932..4174 on + strand\\n[10:33:38] Found 6615 CDS\\n[10:33:38] Connecting features back to sequences\\n[10:33:38] Not using genus-specific database. Try --usegenus to enable it.\\n[10:33:38] Annotating CDS, please be patient.\\n[10:33:38] Will use 4 CPUs for similarity searching.\\n[10:33:40] There are still 6615 unannotated CDS left (started with 6615)\\n[10:33:40] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:33:40] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKO01\\\\/\\\\/Nostoc_JABXKO01\\\\.IS\\\\.tmp\\\\.2732\\\\.faa | parallel --gnu --plain -j 4 --block 260029 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKO01\\\\/\\\\/Nostoc_JABXKO01\\\\.IS\\\\.tmp\\\\.2732\\\\.blast 2> /dev/null\\n[10:33:58] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKO01\\\\/\\\\/Nostoc_JABXKO01\\\\.IS\\\\.tmp\\\\.2732\\\\.faa | parallel --gnu --plain -j 4 --block 260029 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKO01\\\\/\\\\/Nostoc_JABXKO01\\\\.IS\\\\.tmp\\\\.2732\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:33:58] This is prokka 1.14.6\\n[10:33:58] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:33:58] Homepage is https://github.com/tseemann/prokka\\n[10:33:58] Local time is Mon Feb 21 10:33:58 2022\\n[10:33:58] You are tiagoferreiraleao\\n[10:33:58] Operating system is darwin\\n[10:33:58] You have BioPerl 1.007002\\n[10:33:59] System has 4 cores.\\n[10:33:59] Option --cpu asked for 12 cores, but system only has 4\\n[10:33:59] Will use maximum of 4 cores.\\n[10:33:59] Annotating as >>> Bacteria <<<\\n[10:33:59] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJSH01.fasta' contents.\\n[10:33:59] Setting --locustag DPNIJIDD from MD5 d97232dde413171bcf8fee0bf26a6b6a\\n[10:33:59] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJSH01/\\n[10:33:59] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJSH01\\\\/\\n[10:33:59] Using filename prefix: Nostoc_JACJSH01.XXX\\n[10:33:59] Setting HMMER_NCPU=1\\n[10:33:59] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJSH01//Nostoc_JACJSH01.log\\n[10:33:59] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJSH01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJSH01/ --prefix Nostoc_JACJSH01 --cpus 12\\n[10:33:59] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:33:59] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:33:59] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:33:59] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:33:59] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:33:59] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:33:59] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:33:59] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:33:59] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:33:59] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:33:59] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:33:59] Looking for 'egrep' - found /usr/bin/egrep\\n[10:33:59] Looking for 'find' - found /usr/bin/find\\n[10:33:59] Looking for 'grep' - found /usr/bin/grep\\n[10:33:59] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:33:59] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:33:59] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:33:59] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:33:59] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:33:59] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:33:59] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:33:59] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:34:00] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:34:00] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:34:00] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:34:00] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:34:00] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:34:00] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:34:00] Looking for 'sed' - found /usr/bin/sed\\n[10:34:00] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:34:00] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:34:00] Using genetic code table 11.\\n[10:34:00] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJSH01.fasta\\n[10:34:00] Wrote 160 contigs totalling 8861200 bp.\\n[10:34:00] Predicting tRNAs and tmRNAs\\n[10:34:00] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJSH01\\\\/\\\\/Nostoc_JACJSH01\\\\.fna\\n[10:34:03] 1 tRNA-Met [95205,95280] 34 (cat)\\n[10:34:03] 2 tRNA-Met [95281,95352] 32 (cat)\\n[10:34:03] 3 tRNA-Ile [97052,97127] 34 (tat)\\n[10:34:03] 4 tRNA-Ser c[386396,386482] 36 (tga)\\n[10:34:03] 5 tRNA-Tyr [388782,388867] 35 (gta)\\n[10:34:03] 6 tRNA-Thr [388955,389026] 33 (ggt)\\n[10:34:03] 7 tRNA-Arg c[389782,389856] 35 (tct)\\n[10:34:03] 8 tRNA-Gly c[484048,484118] 33 (tcc)\\n[10:34:03] 1 tRNA-Pro [341814,341887] 35 (cgg)\\n[10:34:03] 1 tRNA-Asp c[215235,215308] 35 (gtc)\\n[10:34:03] 1 tRNA-Lys [5735,5806] 33 (ttt)\\n[10:34:03] 2 tRNA-Leu [64365,64448] 36 (tag)\\n[10:34:03] 3 tRNA-Val [162039,162112] 35 (gac)\\n[10:34:03] 4 tRNA-Pro c[212951,213024] 35 (ggg)\\n[10:34:03] 5 tRNA-Met [225117,225192] 36 (cat)\\n[10:34:03] 1 tRNA-Pro [105860,105933] 35 (tgg)\\n[10:34:03] 1 tRNA-Gly [4523,4594] 33 (ccc)\\n[10:34:03] 2 tmRNA c[28017,28404] 101,151 ANNIVNFARKDALVAA*\\n[10:34:03] 1 tRNA-Arg c[161806,161881] 36 (cct)\\n[10:34:03] 1 tRNA-Arg [84732,84804] 34 (ccg)\\n[10:34:03] 1 tRNA-Ser c[36796,36882] 36 (cga)\\n[10:34:03] 2 tRNA-Phe c[138569,138641] 34 (gaa)\\n[10:34:03] 1 tRNA-Thr [149387,149458] 33 (cgt)\\n[10:34:03] 1 tRNA-Trp [43313,43386] 34 (cca)\\n[10:34:03] 1 tRNA-Ser c[13681,13767] 36 (gga)\\n[10:34:03] 2 tRNA-Pyl [73480,73553] 34 (cta)\\n[10:34:03] 1 tRNA-Ser c[41138,41229] 35 (gct)\\n[10:34:03] 2 tRNA-Lys c[45289,45360] 33 (ttt)\\n[10:34:03] 3 tRNA-His c[45620,45694] 35 (gtg)\\n[10:34:03] 4 tRNA-Ala c[45700,45774] 35 (tgc)\\n[10:34:03] 5 tRNA-Ala c[45784,45856] 34 (ggc)\\n[10:34:03] 6 tRNA-Arg c[45860,45935] 35 (tct)\\n[10:34:03] 7 tRNA-Val c[45942,46017] 34 (tac)\\n[10:34:03] 8 tRNA-Thr c[46099,46171] 34 (tgt)\\n[10:34:03] 9 tRNA-Met c[46179,46255] 34 (cat)\\n[10:34:03] 10 tRNA-Leu c[46265,46340] 36 (cag)\\n[10:34:03] 11 tRNA-Pro c[46343,46415] 34 (tgg)\\n[10:34:03] 12 tRNA-Leu c[46417,46494] 35 (taa)\\n[10:34:03] 13 tRNA-Gln c[46501,46576] 34 (ttg)\\n[10:34:03] 14 tRNA-Gln c[46580,46652] 34 (ctg)\\n[10:34:03] 15 tRNA-Asn c[46803,46877] 33 (gtt)\\n[10:34:03] 16 tRNA-Phe c[46883,46958] 36 (gaa)\\n[10:34:03] 17 tRNA-Arg c[46983,47058] 35 (acg)\\n[10:34:03] 18 tRNA-Ser c[47062,47153] 36 (gct)\\n[10:34:03] 19 tRNA-Tyr c[47156,47239] 37 (gta)\\n[10:34:03] 20 tRNA-Glu c[47245,47320] 35 (ttc)\\n[10:34:03] tRNA c[47499,47571] is a pseudo/wacky gene - skipping.\\n[10:34:03] 22 tRNA-Met c[47575,47650] 34 (cat)\\n[10:34:03] 23 tRNA-Cys c[47660,47733] 35 (gca)\\n[10:34:03] 24 tRNA-Glu c[47737,47813] 35 (ctc)\\n[10:34:03] 25 tRNA-Glu c[54613,54686] 35 (ttc)\\n[10:34:03] 1 tRNA-Ala [58117,58189] 34 (ggc)\\n[10:34:03] 2 tRNA-Met [110463,110539] 35 (cat)\\n[10:34:03] 1 tRNA-Ala [124006,124078] 34 (cgc)\\n[10:34:03] 1 tRNA-His [87535,87608] 34 (gtg)\\n[10:34:03] 1 tRNA-Cys c[4091,4161] 33 (gca)\\n[10:34:03] 2 tRNA-Arg c[91911,91984] 35 (acg)\\n[10:34:03] 1 tRNA-Thr [47099,47170] 33 (tgt)\\n[10:34:03] 2 tRNA-Met c[48554,48627] 33 (cat)\\n[10:34:03] 1 tRNA-Val [62577,62649] 34 (tac)\\n[10:34:03] 1 tRNA-Asp [49979,50056] 38 (gtc)\\n[10:34:03] 2 tRNA-Glu [50059,50135] 35 (ctc)\\n[10:34:03] 3 tRNA-Arg [50138,50213] 36 (ccg)\\n[10:34:03] 4 tRNA-Met [50321,50397] 35 (cat)\\n[10:34:03] 5 tRNA-Trp [50400,50472] 34 (cca)\\n[10:34:03] 6 tRNA-Lys [50497,50568] 33 (ctt)\\n[10:34:03] 7 tRNA-Glu [50599,50674] 34 (ttc)\\n[10:34:03] 8 tRNA-Tyr [50678,50760] 36 (gta)\\n[10:34:03] 9 tRNA-Ser [50764,50853] 33 (gct)\\n[10:34:03] 10 tRNA-Arg [50858,50933] 35 (acg)\\n[10:34:03] 11 tRNA-Phe [50938,51012] 35 (gaa)\\n[10:34:03] 12 tRNA-Asn [51021,51095] 33 (gtt)\\n[10:34:03] 13 tRNA-Gln [51183,51257] 35 (ttg)\\n[10:34:03] 14 tRNA-Pro [51262,51337] 36 (tgg)\\n[10:34:03] 15 tRNA-Lys [51342,51416] 34 (ttt)\\n[10:34:03] 16 tRNA-Leu [51419,51493] 35 (taa)\\n[10:34:03] 17 tRNA-Thr [51503,51576] 35 (tgt)\\n[10:34:03] 18 tRNA-Gly [51582,51653] 33 (tcc)\\n[10:34:03] 19 tRNA-Val [51658,51733] 34 (tac)\\n[10:34:03] 20 tRNA-Arg [51740,51813] 34 (tct)\\n[10:34:03] 21 tRNA-Ala [51818,51892] 35 (tgc)\\n[10:34:03] 22 tRNA-His [51898,51974] 35 (gtg)\\n[10:34:03] 23 tRNA-Ser [51985,52069] 34 (tga)\\n[10:34:03] 1 tRNA-Ala c[64005,64079] 35 (tgc)\\n[10:34:03] 1 tRNA-Gly c[24383,24454] 33 (gcc)\\n[10:34:03] 1 tRNA-Gln c[7607,7680] 34 (ttg)\\n[10:34:03] 2 tRNA-Leu c[9500,9584] 35 (caa)\\n[10:34:03] 3 tRNA-Leu [13925,14006] 35 (cag)\\n[10:34:03] 1 tRNA-Leu c[9109,9191] 36 (gag)\\n[10:34:03] 2 tRNA-Lys [37361,37433] 33 (ctt)\\n[10:34:03] 1 tRNA-Asn c[44193,44264] 33 (gtt)\\n[10:34:03] Found 90 tRNAs\\n[10:34:03] Predicting Ribosomal RNAs\\n[10:34:03] Running Barrnap with 4 threads\\n[10:34:05] 1 JACJSH010000159.1 5 16S ribosomal RNA\\n[10:34:05] 2 JACJSH010000160.1 179 23S ribosomal RNA\\n[10:34:05] Found 2 rRNAs\\n[10:34:05] Skipping ncRNA search, enable with --rfam if desired.\\n[10:34:05] Total of 91 tRNA + rRNA features\\n[10:34:05] Searching for CRISPR repeats\\n[10:34:05] CRISPR1 JACJSH010000002.1 342218 with 6 spacers\\n[10:34:05] CRISPR2 JACJSH010000003.1 142776 with 37 spacers\\n[10:34:05] CRISPR3 JACJSH010000003.1 305027 with 20 spacers\\n[10:34:05] CRISPR4 JACJSH010000004.1 175687 with 39 spacers\\n[10:34:05] CRISPR5 JACJSH010000010.1 12556 with 83 spacers\\n[10:34:05] CRISPR6 JACJSH010000034.1 70395 with 13 spacers\\n[10:34:06] Found 6 CRISPRs\\n[10:34:06] Predicting coding sequences\\n[10:34:06] Contigs total 8861200 bp, so using single mode\\n[10:34:06] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJSH01\\\\/\\\\/Nostoc_JACJSH01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:34:13] Excluding CDS which overlaps existing RNA (repeat_region) at JACJSH010000004.1:176516..176710 on + strand\\n[10:34:13] Excluding CDS which overlaps existing RNA (repeat_region) at JACJSH010000004.1:178463..178594 on + strand\\n[10:34:14] Excluding CDS which overlaps existing RNA (repeat_region) at JACJSH010000010.1:13122..13415 on + strand\\n[10:34:14] Excluding CDS which overlaps existing RNA (repeat_region) at JACJSH010000010.1:13573..13758 on - strand\\n[10:34:14] Excluding CDS which overlaps existing RNA (repeat_region) at JACJSH010000010.1:14162..14587 on + strand\\n[10:34:14] Excluding CDS which overlaps existing RNA (repeat_region) at JACJSH010000010.1:15107..15310 on + strand\\n[10:34:14] Excluding CDS which overlaps existing RNA (repeat_region) at JACJSH010000010.1:18482..18736 on - strand\\n[10:34:17] Excluding CDS which overlaps existing RNA (rRNA) at JACJSH010000159.1:1134..1448 on + strand\\n[10:34:17] Found 7475 CDS\\n[10:34:17] Connecting features back to sequences\\n[10:34:17] Not using genus-specific database. Try --usegenus to enable it.\\n[10:34:17] Annotating CDS, please be patient.\\n[10:34:17] Will use 4 CPUs for similarity searching.\\n[10:34:19] There are still 7475 unannotated CDS left (started with 7475)\\n[10:34:19] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:34:19] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJSH01\\\\/\\\\/Nostoc_JACJSH01\\\\.IS\\\\.tmp\\\\.2872\\\\.faa | parallel --gnu --plain -j 4 --block 302431 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJSH01\\\\/\\\\/Nostoc_JACJSH01\\\\.IS\\\\.tmp\\\\.2872\\\\.blast 2> /dev/null\\n[10:34:35] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJSH01\\\\/\\\\/Nostoc_JACJSH01\\\\.IS\\\\.tmp\\\\.2872\\\\.faa | parallel --gnu --plain -j 4 --block 302431 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJSH01\\\\/\\\\/Nostoc_JACJSH01\\\\.IS\\\\.tmp\\\\.2872\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:34:35] This is prokka 1.14.6\\n[10:34:35] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:34:35] Homepage is https://github.com/tseemann/prokka\\n[10:34:35] Local time is Mon Feb 21 10:34:35 2022\\n[10:34:35] You are tiagoferreiraleao\\n[10:34:35] Operating system is darwin\\n[10:34:35] You have BioPerl 1.007002\\n[10:34:35] System has 4 cores.\\n[10:34:35] Option --cpu asked for 12 cores, but system only has 4\\n[10:34:35] Will use maximum of 4 cores.\\n[10:34:35] Annotating as >>> Bacteria <<<\\n[10:34:35] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_WVID01.fasta' contents.\\n[10:34:35] Setting --locustag CCIEEPLJ from MD5 cc2ee953a3303b60c4c09de260fc2133\\n[10:34:35] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_WVID01/\\n[10:34:35] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_WVID01\\\\/\\n[10:34:35] Using filename prefix: Nostoc_WVID01.XXX\\n[10:34:35] Setting HMMER_NCPU=1\\n[10:34:35] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_WVID01//Nostoc_WVID01.log\\n[10:34:35] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_WVID01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_WVID01/ --prefix Nostoc_WVID01 --cpus 12\\n[10:34:35] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:34:35] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:34:35] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:34:35] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:34:36] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:34:36] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:34:36] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:34:36] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:34:36] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:34:36] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:34:36] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:34:36] Looking for 'egrep' - found /usr/bin/egrep\\n[10:34:36] Looking for 'find' - found /usr/bin/find\\n[10:34:36] Looking for 'grep' - found /usr/bin/grep\\n[10:34:36] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:34:36] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:34:36] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:34:36] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:34:36] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:34:36] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:34:36] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:34:36] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:34:36] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:34:36] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:34:36] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:34:36] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:34:36] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:34:36] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:34:36] Looking for 'sed' - found /usr/bin/sed\\n[10:34:36] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:34:36] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:34:36] Using genetic code table 11.\\n[10:34:36] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_WVID01.fasta\\n[10:34:37] Wrote 109 contigs totalling 7426239 bp.\\n[10:34:37] Predicting tRNAs and tmRNAs\\n[10:34:37] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_WVID01\\\\/\\\\/Nostoc_WVID01\\\\.fna\\n[10:34:39] 1 tRNA-Ala c[170721,170793] 34 (ggc)\\n[10:34:39] 2 tRNA-Met [272079,272154] 36 (cat)\\n[10:34:39] 3 tRNA-Pro [286460,286533] 35 (ggg)\\n[10:34:39] 1 tRNA-Ser c[101164,101250] 36 (tga)\\n[10:34:39] 2 tRNA-Gly c[104950,105020] 33 (tcc)\\n[10:34:39] 1 tRNA-Val [14384,14456] 34 (tac)\\n[10:34:39] 1 tRNA-Met [50928,51001] 33 (cat)\\n[10:34:39] 2 tRNA-Pro [160648,160721] 35 (tgg)\\n[10:34:39] 3 tRNA-Phe c[288282,288354] 34 (gaa)\\n[10:34:39] 1 tRNA-Lys [86597,86669] 33 (ctt)\\n[10:34:39] 2 tRNA-Pro [199782,199855] 35 (cgg)\\n[10:34:39] 3 tmRNA c[295020,295407] 101,151 ANNIVKFARKDALVAA*\\n[10:34:39] 1 tRNA-His c[21910,21983] 34 (gtg)\\n[10:34:39] 1 tRNA-Arg c[107927,108003] 36 (cct)\\n[10:34:39] 1 tRNA-Thr [188811,188882] 33 (tgt)\\n[10:34:39] 1 tRNA-Met [18168,18244] 35 (cat)\\n[10:34:39] 1 tRNA-Glu c[20151,20224] 35 (ttc)\\n[10:34:39] 2 tRNA-Leu [74045,74127] 35 (tag)\\n[10:34:39] 3 tRNA-Thr [102836,102907] 33 (cgt)\\n[10:34:39] 1 tRNA-Asp [33322,33395] 35 (gtc)\\n[10:34:39] 2 tRNA-Cys [58096,58167] 33 (gca)\\n[10:34:39] 3 tRNA-Leu c[120439,120521] 36 (gag)\\n[10:34:39] 1 tRNA-Leu c[10994,11075] 35 (cag)\\n[10:34:39] 1 tRNA-Gly [45285,45356] 33 (ccc)\\n[10:34:39] 2 tRNA-Gln c[151632,151705] 34 (ttg)\\n[10:34:39] 3 tRNA-Leu c[169965,170049] 35 (caa)\\n[10:34:39] 1 tRNA-Asn [2351,2422] 33 (gtt)\\n[10:34:39] 2 tRNA-Ser [112629,112722] 36 (gct)\\n[10:34:39] 3 tRNA-Arg c[121880,121953] 35 (acg)\\n[10:34:39] 1 tRNA-Val c[30637,30710] 35 (gac)\\n[10:34:39] 1 tRNA-Arg [61014,61088] 35 (tct)\\n[10:34:39] 2 tRNA-Thr c[61298,61369] 33 (ggt)\\n[10:34:39] 3 tRNA-Tyr c[61437,61521] 35 (gta)\\n[10:34:39] 1 tRNA-Gly c[61370,61441] 33 (gcc)\\n[10:34:39] 1 tRNA-Arg c[66025,66097] 34 (ccg)\\n[10:34:39] 1 tRNA-Ala c[21156,21230] 35 (tgc)\\n[10:34:39] 2 tRNA-Val [25197,25270] 34 (cac)\\n[10:34:39] 1 tRNA-Ser c[63493,63579] 36 (cga)\\n[10:34:39] 1 tRNA-Lys [23871,23942] 33 (ttt)\\n[10:34:39] 1 tRNA-Asp [27308,27384] 37 (gtc)\\n[10:34:39] 2 tRNA-Glu [27387,27463] 35 (ctc)\\n[10:34:39] 3 tRNA-Cys [27469,27544] 33 (gca)\\n[10:34:39] 4 tRNA-Met [27550,27624] 33 (cat)\\n[10:34:39] 5 tRNA-Lys [27754,27825] 33 (ctt)\\n[10:34:39] 6 tRNA-Glu [27856,27932] 34 (ttc)\\n[10:34:39] 7 tRNA-Tyr [27939,28022] 36 (gta)\\n[10:34:39] 8 tRNA-Met [28023,28095] 33 (cat)\\n[10:34:39] 9 tRNA-Ser [28099,28190] 36 (gct)\\n[10:34:39] 10 tRNA-Arg [28194,28269] 35 (acg)\\n[10:34:39] 11 tRNA-Asn [28381,28456] 34 (gtt)\\n[10:34:39] 12 tRNA-Gln [28462,28537] 34 (ctg)\\n[10:34:39] 13 tRNA-Gln [28546,28622] 35 (ttg)\\n[10:34:39] 14 tRNA-Leu [28628,28705] 35 (caa)\\n[10:34:39] 15 tRNA-Pro [28707,28780] 35 (tgg)\\n[10:34:39] 16 tRNA-Leu [28786,28862] 36 (tag)\\n[10:34:39] 17 tRNA-Met [28862,28939] 35 (cat)\\n[10:34:39] 18 tRNA-Thr [28943,29016] 35 (tgt)\\n[10:34:39] 19 tRNA-Gly [29025,29096] 33 (tcc)\\n[10:34:39] 20 tRNA-Val [29182,29257] 34 (tac)\\n[10:34:39] 21 tRNA-Arg [29264,29339] 35 (tct)\\n[10:34:39] 22 tRNA-Ala [29391,29465] 35 (tgc)\\n[10:34:39] 1 tRNA-Trp [2351,2424] 34 (cca)\\n[10:34:39] 2 tRNA-Ala c[6665,6737] 34 (cgc)\\n[10:34:39] 1 tRNA-Ser c[8089,8172] 33 (tga)\\n[10:34:39] 2 tRNA-His c[8179,8254] 35 (gtg)\\n[10:34:39] 3 tRNA-Lys c[8267,8342] 33 (ctt)\\n[10:34:39] 4 tRNA-Lys c[8348,8423] 34 (ttt)\\n[10:34:39] 5 tRNA-Pro c[8428,8503] 36 (tgg)\\n[10:34:39] 6 tRNA-Gln c[8591,8664] 34 (ttg)\\n[10:34:39] 7 tRNA-Gln c[8670,8746] 35 (ctg)\\n[10:34:39] 8 tRNA-Asn c[8757,8831] 33 (gtt)\\n[10:34:39] 9 tRNA-Phe c[8844,8918] 35 (gaa)\\n[10:34:39] 10 tRNA-Arg c[8924,8998] 35 (acg)\\n[10:34:39] 11 tRNA-Ser c[9002,9091] 33 (gct)\\n[10:34:39] 12 tRNA-Tyr c[9095,9178] 36 (gta)\\n[10:34:39] 13 tRNA-Glu c[9183,9257] 35 (ttc)\\n[10:34:39] 14 tRNA-Trp c[9273,9345] 34 (cca)\\n[10:34:39] 15 tRNA-Met c[9347,9423] 35 (cat)\\n[10:34:39] 16 tRNA-Glu c[9534,9610] 35 (ctc)\\n[10:34:39] 17 tRNA-Asp c[9615,9692] 37 (gtc)\\n[10:34:39] 1 tRNA-Ser c[6313,6401] 36 (gga)\\n[10:34:39] 1 tRNA-Ala c[724,798] 34 (tgc)\\n[10:34:39] 2 tRNA-Ile c[878,953] 35 (gat)\\n[10:34:39] 1 tRNA-Ala c[327,400] 34 (tgc)\\n[10:34:39] 2 tRNA-Ile c[411,486] 35 (gat)\\n[10:34:39] 1 tRNA-Gly [23,97] 33 (gcc)\\n[10:34:39] 2 tRNA-Cys [201,274] 33 (gca)\\n[10:34:39] Found 87 tRNAs\\n[10:34:39] Predicting Ribosomal RNAs\\n[10:34:39] Running Barrnap with 4 threads\\n[10:34:40] Found 0 rRNAs\\n[10:34:40] Skipping ncRNA search, enable with --rfam if desired.\\n[10:34:40] Total of 86 tRNA + rRNA features\\n[10:34:40] Searching for CRISPR repeats\\n[10:34:41] CRISPR1 WVID01000001.1 111116 with 36 spacers\\n[10:34:41] CRISPR2 WVID01000002.1 217919 with 31 spacers\\n[10:34:41] CRISPR3 WVID01000008.1 62027 with 66 spacers\\n[10:34:41] CRISPR4 WVID01000009.1 180809 with 26 spacers\\n[10:34:41] CRISPR5 WVID01000012.1 214961 with 4 spacers\\n[10:34:41] CRISPR6 WVID01000021.1 32036 with 30 spacers\\n[10:34:41] Found 6 CRISPRs\\n[10:34:41] Predicting coding sequences\\n[10:34:41] Contigs total 7426239 bp, so using single mode\\n[10:34:41] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_WVID01\\\\/\\\\/Nostoc_WVID01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:34:47] Excluding CDS which overlaps existing RNA (repeat_region) at WVID01000001.1:111637..111846 on + strand\\n[10:34:47] Excluding CDS which overlaps existing RNA (repeat_region) at WVID01000001.1:113164..113271 on + strand\\n[10:34:47] Excluding CDS which overlaps existing RNA (repeat_region) at WVID01000001.1:113445..113534 on - strand\\n[10:34:47] Excluding CDS which overlaps existing RNA (repeat_region) at WVID01000002.1:218359..218631 on - strand\\n[10:34:47] Excluding CDS which overlaps existing RNA (repeat_region) at WVID01000002.1:220084..220296 on - strand\\n[10:34:47] Excluding CDS which overlaps existing RNA (tRNA) at WVID01000004.1:14342..14515 on + strand\\n[10:34:48] Excluding CDS which overlaps existing RNA (repeat_region) at WVID01000008.1:64837..64971 on + strand\\n[10:34:48] Excluding CDS which overlaps existing RNA (repeat_region) at WVID01000009.1:181036..181170 on - strand\\n[10:34:48] Excluding CDS which overlaps existing RNA (repeat_region) at WVID01000009.1:181977..182225 on + strand\\n[10:34:49] Excluding CDS which overlaps existing RNA (repeat_region) at WVID01000012.1:214856..215188 on + strand\\n[10:34:49] Excluding CDS which overlaps existing RNA (tRNA) at WVID01000018.1:112547..112744 on - strand\\n[10:34:50] Excluding CDS which overlaps existing RNA (tRNA) at WVID01000037.1:27465..27905 on + strand\\n[10:34:50] Excluding CDS which overlaps existing RNA (tRNA) at WVID01000038.1:2327..2452 on + strand\\n[10:34:50] Excluding CDS which overlaps existing RNA (tRNA) at WVID01000040.1:9140..9532 on - strand\\n[10:34:51] Found 6570 CDS\\n[10:34:51] Connecting features back to sequences\\n[10:34:51] Not using genus-specific database. Try --usegenus to enable it.\\n[10:34:51] Annotating CDS, please be patient.\\n[10:34:51] Will use 4 CPUs for similarity searching.\\n[10:34:52] There are still 6570 unannotated CDS left (started with 6570)\\n[10:34:52] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:34:52] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_WVID01\\\\/\\\\/Nostoc_WVID01\\\\.IS\\\\.tmp\\\\.2984\\\\.faa | parallel --gnu --plain -j 4 --block 254671 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_WVID01\\\\/\\\\/Nostoc_WVID01\\\\.IS\\\\.tmp\\\\.2984\\\\.blast 2> /dev/null\\n[10:35:02] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_WVID01\\\\/\\\\/Nostoc_WVID01\\\\.IS\\\\.tmp\\\\.2984\\\\.faa | parallel --gnu --plain -j 4 --block 254671 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_WVID01\\\\/\\\\/Nostoc_WVID01\\\\.IS\\\\.tmp\\\\.2984\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:35:02] This is prokka 1.14.6\\n[10:35:02] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:35:02] Homepage is https://github.com/tseemann/prokka\\n[10:35:02] Local time is Mon Feb 21 10:35:02 2022\\n[10:35:02] You are tiagoferreiraleao\\n[10:35:02] Operating system is darwin\\n[10:35:02] You have BioPerl 1.007002\\n[10:35:02] System has 4 cores.\\n[10:35:02] Option --cpu asked for 12 cores, but system only has 4\\n[10:35:02] Will use maximum of 4 cores.\\n[10:35:02] Annotating as >>> Bacteria <<<\\n[10:35:02] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQA01.fasta' contents.\\n[10:35:02] Setting --locustag NICHKEGN from MD5 72c14e07dafc9db898c86e716bef1e78\\n[10:35:02] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQA01/\\n[10:35:02] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQA01\\\\/\\n[10:35:02] Using filename prefix: Fischerella_NMQA01.XXX\\n[10:35:02] Setting HMMER_NCPU=1\\n[10:35:02] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQA01//Fischerella_NMQA01.log\\n[10:35:02] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQA01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQA01/ --prefix Fischerella_NMQA01 --cpus 12\\n[10:35:02] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:35:02] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:35:02] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:35:02] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:35:03] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:35:03] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:35:03] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:35:03] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:35:03] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:35:03] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:35:03] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:35:03] Looking for 'egrep' - found /usr/bin/egrep\\n[10:35:03] Looking for 'find' - found /usr/bin/find\\n[10:35:03] Looking for 'grep' - found /usr/bin/grep\\n[10:35:03] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:35:03] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:35:03] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:35:03] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:35:03] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:35:03] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:35:03] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:35:03] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:35:03] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:35:03] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:35:03] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:35:03] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:35:03] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:35:03] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:35:03] Looking for 'sed' - found /usr/bin/sed\\n[10:35:03] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:35:04] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:35:04] Using genetic code table 11.\\n[10:35:04] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQA01.fasta\\n[10:35:04] Wrote 383 contigs totalling 5913697 bp.\\n[10:35:04] Predicting tRNAs and tmRNAs\\n[10:35:04] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQA01\\\\/\\\\/Fischerella_NMQA01\\\\.fna\\n[10:35:06] 1 tRNA-Pro c[354,428] 35 (tgg)\\n[10:35:06] 1 tRNA-His c[68362,68435] 34 (gtg)\\n[10:35:06] 1 tRNA-Pro c[12283,12356] 35 (cgg)\\n[10:35:06] 2 tRNA-Val [22081,22155] 37 (tac)\\n[10:35:06] 1 tRNA-Ser c[62301,62387] 36 (cga)\\n[10:35:06] 1 tRNA-Asn c[304,377] 34 (gtt)\\n[10:35:06] 1 tRNA-Gly [12739,12810] 33 (gcc)\\n[10:35:06] 1 tRNA-Arg c[3275,3349] 35 (tct)\\n[10:35:06] 1 tRNA-Arg c[6069,6142] 35 (ccg)\\n[10:35:06] 1 tRNA-Met c[30260,30335] 36 (cat)\\n[10:35:06] 1 tRNA-Asp [15297,15370] 35 (gtc)\\n[10:35:06] 1 tRNA-Thr c[8147,8218] 33 (ggt)\\n[10:35:06] 2 tRNA-Tyr c[8282,8366] 35 (gta)\\n[10:35:06] 1 tRNA-Ser [11057,11143] 36 (tga)\\n[10:35:06] 1 tRNA-Gly [8021,8091] 33 (tcc)\\n[10:35:06] 1 tRNA-Leu [159,241] 35 (cag)\\n[10:35:06] 1 tRNA-Met [8760,8832] 34 (cat)\\n[10:35:06] 1 tRNA-Ser [8296,8389] 36 (gct)\\n[10:35:06] 1 tRNA-Pro c[3604,3677] 35 (ggg)\\n[10:35:06] 1 tRNA-Glu [5740,5812] 35 (ttc)\\n[10:35:06] 1 tRNA-Gly c[9514,9585] 33 (ccc)\\n[10:35:06] 1 tRNA-Thr c[2528,2599] 33 (tgt)\\n[10:35:06] 1 tRNA-Cys c[9206,9277] 33 (gca)\\n[10:35:06] 1 tRNA-Ser [8481,8569] 36 (gga)\\n[10:35:06] 1 tRNA-Met c[15126,15202] 35 (cat)\\n[10:35:06] 2 tRNA-Val [18173,18246] 35 (gac)\\n[10:35:06] 1 tRNA-Leu c[1009,1093] 35 (caa)\\n[10:35:06] 1 tRNA-Leu c[414,496] 36 (gag)\\n[10:35:06] 1 tRNA-Gln [27468,27539] 33 (ttg)\\n[10:35:06] 1 tRNA-Arg c[7441,7514] 35 (acg)\\n[10:35:06] 2 tRNA-Ala c[12632,12704] 34 (ggc)\\n[10:35:06] 3 tRNA-Thr c[14146,14217] 33 (cgt)\\n[10:35:06] 1 tmRNA c[3418,3804] 100,153 ANNIVKFARKEALVAA**\\n[10:35:06] 1 tRNA-Trp c[4058,4131] 34 (cca)\\n[10:35:06] 1 tRNA-Ala c[274,348] 35 (cgc)\\n[10:35:06] 1 tRNA-Val c[12096,12168] 34 (tac)\\n[10:35:06] 2 tRNA-Phe [19593,19667] 35 (gaa)\\n[10:35:06] 1 tRNA-Lys [21085,21158] 34 (ttt)\\n[10:35:06] 1 tRNA-Leu c[195,278] 36 (tag)\\n[10:35:06] 1 tRNA-Asn c[5157,5230] 34 (att)\\n[10:35:06] 2 tRNA-Arg c[5932,6004] 34 (tcg)\\n[10:35:06] Found 41 tRNAs\\n[10:35:06] Predicting Ribosomal RNAs\\n[10:35:06] Running Barrnap with 4 threads\\n[10:35:08] 1 NMQA01000259.1 158 23S ribosomal RNA\\n[10:35:08] 2 NMQA01000259.1 3073 5S ribosomal RNA\\n[10:35:08] 3 NMQA01000260.1 5 16S ribosomal RNA\\n[10:35:08] Found 3 rRNAs\\n[10:35:08] Skipping ncRNA search, enable with --rfam if desired.\\n[10:35:08] Total of 43 tRNA + rRNA features\\n[10:35:08] Searching for CRISPR repeats\\n[10:35:08] CRISPR1 NMQA01000018.1 27 with 4 spacers\\n[10:35:08] CRISPR2 NMQA01000053.1 3506 with 3 spacers\\n[10:35:08] CRISPR3 NMQA01000067.1 18804 with 5 spacers\\n[10:35:08] CRISPR4 NMQA01000080.1 137 with 46 spacers\\n[10:35:08] CRISPR5 NMQA01000238.1 35 with 5 spacers\\n[10:35:08] CRISPR6 NMQA01000238.1 8431 with 5 spacers\\n[10:35:08] CRISPR7 NMQA01000253.1 38 with 12 spacers\\n[10:35:08] CRISPR8 NMQA01000305.1 268 with 6 spacers\\n[10:35:08] Found 8 CRISPRs\\n[10:35:08] Predicting coding sequences\\n[10:35:08] Contigs total 5913697 bp, so using single mode\\n[10:35:08] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQA01\\\\/\\\\/Fischerella_NMQA01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:35:13] Excluding CDS which overlaps existing RNA (tRNA) at NMQA01000017.1:18334..22170 on - strand\\n[10:35:14] Excluding CDS which overlaps existing RNA (tRNA) at NMQA01000090.1:6032..6181 on - strand\\n[10:35:15] Excluding CDS which overlaps existing RNA (rRNA) at NMQA01000259.1:553..732 on + strand\\n[10:35:15] Excluding CDS which overlaps existing RNA (rRNA) at NMQA01000260.1:1134..1400 on + strand\\n[10:35:15] Excluding CDS which overlaps existing RNA (tRNA) at NMQA01000378.1:5903..6199 on - strand\\n[10:35:15] Found 4834 CDS\\n[10:35:15] Connecting features back to sequences\\n[10:35:15] Not using genus-specific database. Try --usegenus to enable it.\\n[10:35:15] Annotating CDS, please be patient.\\n[10:35:15] Will use 4 CPUs for similarity searching.\\n[10:35:16] There are still 4834 unannotated CDS left (started with 4834)\\n[10:35:16] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:35:16] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQA01\\\\/\\\\/Fischerella_NMQA01\\\\.IS\\\\.tmp\\\\.3098\\\\.faa | parallel --gnu --plain -j 4 --block 196740 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQA01\\\\/\\\\/Fischerella_NMQA01\\\\.IS\\\\.tmp\\\\.3098\\\\.blast 2> /dev/null\\n[10:35:38] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQA01\\\\/\\\\/Fischerella_NMQA01\\\\.IS\\\\.tmp\\\\.3098\\\\.faa | parallel --gnu --plain -j 4 --block 196740 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQA01\\\\/\\\\/Fischerella_NMQA01\\\\.IS\\\\.tmp\\\\.3098\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:35:38] This is prokka 1.14.6\\n[10:35:38] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:35:38] Homepage is https://github.com/tseemann/prokka\\n[10:35:38] Local time is Mon Feb 21 10:35:38 2022\\n[10:35:38] You are tiagoferreiraleao\\n[10:35:38] Operating system is darwin\\n[10:35:38] You have BioPerl 1.007002\\n[10:35:38] System has 4 cores.\\n[10:35:38] Option --cpu asked for 12 cores, but system only has 4\\n[10:35:38] Will use maximum of 4 cores.\\n[10:35:38] Annotating as >>> Bacteria <<<\\n[10:35:38] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAA01.fasta' contents.\\n[10:35:38] Setting --locustag GMBKGEHD from MD5 06b40e1dca0865c1b7c7c403eb786c1b\\n[10:35:38] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAA01/\\n[10:35:38] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAA01\\\\/\\n[10:35:38] Using filename prefix: Fischerella_NHAA01.XXX\\n[10:35:38] Setting HMMER_NCPU=1\\n[10:35:38] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAA01//Fischerella_NHAA01.log\\n[10:35:38] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAA01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAA01/ --prefix Fischerella_NHAA01 --cpus 12\\n[10:35:38] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:35:38] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:35:38] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:35:38] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:35:38] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:35:38] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:35:38] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:35:38] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:35:38] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:35:38] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:35:38] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:35:38] Looking for 'egrep' - found /usr/bin/egrep\\n[10:35:38] Looking for 'find' - found /usr/bin/find\\n[10:35:38] Looking for 'grep' - found /usr/bin/grep\\n[10:35:38] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:35:38] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:35:38] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:35:38] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:35:38] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:35:38] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:35:39] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:35:39] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:35:39] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:35:39] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:35:39] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:35:39] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:35:39] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:35:39] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:35:39] Looking for 'sed' - found /usr/bin/sed\\n[10:35:39] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:35:39] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:35:39] Using genetic code table 11.\\n[10:35:39] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAA01.fasta\\n[10:35:39] Wrote 123 contigs totalling 5411116 bp.\\n[10:35:39] Predicting tRNAs and tmRNAs\\n[10:35:39] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAA01\\\\/\\\\/Fischerella_NHAA01\\\\.fna\\n[10:35:41] 1 tRNA-Thr c[4365,4436] 33 (tgt)\\n[10:35:41] 1 tRNA-Arg [41085,41160] 36 (cct)\\n[10:35:41] 1 tRNA-Leu [43065,43149] 35 (caa)\\n[10:35:41] 1 tRNA-Ser c[4295,4383] 36 (gga)\\n[10:35:41] 1 tRNA-Met c[26810,26885] 36 (cat)\\n[10:35:41] 2 tRNA-Arg c[51661,51735] 35 (tct)\\n[10:35:41] 1 tRNA-Met c[20774,20846] 34 (cat)\\n[10:35:41] 2 tRNA-Ile [31657,31732] 35 (gat)\\n[10:35:41] 3 tRNA-Ala [31930,32004] 34 (tgc)\\n[10:35:41] 1 tRNA-Gln c[23565,23636] 33 (ttg)\\n[10:35:41] 1 tRNA-Arg c[28558,28631] 35 (ccg)\\n[10:35:41] 1 tRNA-Leu [11069,11151] 36 (gag)\\n[10:35:41] 2 tRNA-Gln c[85245,85315] 32 (ttg)\\n[10:35:41] 1 tRNA-Trp [4114,4187] 34 (cca)\\n[10:35:41] 1 tRNA-Leu c[23167,23250] 36 (tag)\\n[10:35:41] 1 tRNA-Ser [30602,30688] 36 (tga)\\n[10:35:41] 1 tRNA-Asp c[60883,60956] 35 (gtc)\\n[10:35:41] 1 tRNA-Pro c[77980,78053] 35 (cgg)\\n[10:35:41] 1 tmRNA c[246,631] 100,150 ANNIVKFARKEALVAA*\\n[10:35:41] 1 tRNA-Cys [4023,4094] 33 (gca)\\n[10:35:41] 1 tRNA-Leu [20860,20942] 35 (cag)\\n[10:35:41] 1 tRNA-Asn c[3589,3662] 34 (att)\\n[10:35:41] 2 tRNA-Arg c[4333,4405] 34 (tcg)\\n[10:35:41] 3 tRNA-Glu c[22318,22390] 35 (ttc)\\n[10:35:41] 1 tRNA-Ala c[3903,3977] 35 (cgc)\\n[10:35:41] 1 tRNA-Ser [4836,4922] 36 (cga)\\n[10:35:41] 1 tRNA-Asn [109264,109337] 34 (gtt)\\n[10:35:41] 1 tRNA-Val c[11720,11792] 34 (tac)\\n[10:35:41] 2 tRNA-Phe [19175,19247] 34 (gaa)\\n[10:35:41] 3 tRNA-Pro c[91777,91850] 35 (ggg)\\n[10:35:41] 1 tRNA-Pro [19080,19154] 35 (tgg)\\n[10:35:41] 2 tRNA-Arg c[182331,182404] 35 (acg)\\n[10:35:41] 3 tRNA-Ala c[187296,187369] 34 (ggc)\\n[10:35:41] 1 tRNA-Gly [16153,16224] 33 (ccc)\\n[10:35:41] 2 tRNA-Thr c[34250,34321] 33 (ggt)\\n[10:35:41] 3 tRNA-Tyr c[34385,34469] 35 (gta)\\n[10:35:41] 1 tRNA-Thr [49961,50032] 33 (cgt)\\n[10:35:41] 2 tRNA-Lys [94184,94257] 34 (ttt)\\n[10:35:41] 3 tRNA-His c[301091,301164] 34 (gtg)\\n[10:35:41] 1 tRNA-Val c[45064,45137] 35 (gac)\\n[10:35:41] 2 tRNA-Met [48082,48158] 35 (cat)\\n[10:35:41] 1 tRNA-Gly c[35469,35540] 33 (gcc)\\n[10:35:41] 1 tRNA-Gly c[5906,5976] 33 (tcc)\\n[10:35:41] 1 tRNA-Ser [77361,77454] 36 (gct)\\n[10:35:41] Found 44 tRNAs\\n[10:35:41] Predicting Ribosomal RNAs\\n[10:35:41] Running Barrnap with 4 threads\\n[10:35:42] 1 NHAA01000045.1 29914 16S ribosomal RNA\\n[10:35:42] 2 NHAA01000045.1 32492 23S ribosomal RNA\\n[10:35:42] 3 NHAA01000045.1 35431 5S ribosomal RNA\\n[10:35:42] Found 3 rRNAs\\n[10:35:42] Skipping ncRNA search, enable with --rfam if desired.\\n[10:35:42] Total of 46 tRNA + rRNA features\\n[10:35:42] Searching for CRISPR repeats\\n[10:35:43] CRISPR1 NHAA01000020.1 34956 with 43 spacers\\n[10:35:43] CRISPR2 NHAA01000065.1 23493 with 3 spacers\\n[10:35:43] CRISPR3 NHAA01000084.1 29 with 50 spacers\\n[10:35:43] CRISPR4 NHAA01000088.1 24461 with 79 spacers\\n[10:35:43] CRISPR5 NHAA01000119.1 4019 with 11 spacers\\n[10:35:43] CRISPR6 NHAA01000119.1 7662 with 88 spacers\\n[10:35:43] Found 6 CRISPRs\\n[10:35:43] Predicting coding sequences\\n[10:35:43] Contigs total 5411116 bp, so using single mode\\n[10:35:43] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAA01\\\\/\\\\/Fischerella_NHAA01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:35:47] Excluding CDS which overlaps existing RNA (rRNA) at NHAA01000045.1:32887..33066 on + strand\\n[10:35:47] Excluding CDS which overlaps existing RNA (rRNA) at NHAA01000045.1:34852..35028 on - strand\\n[10:35:49] Excluding CDS which overlaps existing RNA (tRNA) at NHAA01000099.1:109138..110577 on + strand\\n[10:35:49] Excluding CDS which overlaps existing RNA (tRNA) at NHAA01000103.1:182288..182959 on + strand\\n[10:35:50] Found 4634 CDS\\n[10:35:50] Connecting features back to sequences\\n[10:35:50] Not using genus-specific database. Try --usegenus to enable it.\\n[10:35:50] Annotating CDS, please be patient.\\n[10:35:50] Will use 4 CPUs for similarity searching.\\n[10:35:51] There are still 4634 unannotated CDS left (started with 4634)\\n[10:35:51] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:35:51] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAA01\\\\/\\\\/Fischerella_NHAA01\\\\.IS\\\\.tmp\\\\.3210\\\\.faa | parallel --gnu --plain -j 4 --block 179591 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAA01\\\\/\\\\/Fischerella_NHAA01\\\\.IS\\\\.tmp\\\\.3210\\\\.blast 2> /dev/null\\n[10:36:00] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAA01\\\\/\\\\/Fischerella_NHAA01\\\\.IS\\\\.tmp\\\\.3210\\\\.faa | parallel --gnu --plain -j 4 --block 179591 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAA01\\\\/\\\\/Fischerella_NHAA01\\\\.IS\\\\.tmp\\\\.3210\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:36:00] This is prokka 1.14.6\\n[10:36:00] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:36:00] Homepage is https://github.com/tseemann/prokka\\n[10:36:00] Local time is Mon Feb 21 10:36:00 2022\\n[10:36:00] You are tiagoferreiraleao\\n[10:36:00] Operating system is darwin\\n[10:36:00] You have BioPerl 1.007002\\n[10:36:00] System has 4 cores.\\n[10:36:00] Option --cpu asked for 12 cores, but system only has 4\\n[10:36:00] Will use maximum of 4 cores.\\n[10:36:00] Annotating as >>> Bacteria <<<\\n[10:36:00] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_DSPK01.fasta' contents.\\n[10:36:00] Setting --locustag BKGEEFED from MD5 b40eefedb14cb5ee121484b5b0bdf1cc\\n[10:36:00] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_DSPK01/\\n[10:36:00] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_DSPK01\\\\/\\n[10:36:00] Using filename prefix: Fischerella_DSPK01.XXX\\n[10:36:00] Setting HMMER_NCPU=1\\n[10:36:00] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_DSPK01//Fischerella_DSPK01.log\\n[10:36:00] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_DSPK01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_DSPK01/ --prefix Fischerella_DSPK01 --cpus 12\\n[10:36:00] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:36:00] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:36:00] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:36:00] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:36:00] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:36:00] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:36:00] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:36:00] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:36:00] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:36:00] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:36:00] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:36:00] Looking for 'egrep' - found /usr/bin/egrep\\n[10:36:00] Looking for 'find' - found /usr/bin/find\\n[10:36:00] Looking for 'grep' - found /usr/bin/grep\\n[10:36:00] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:36:00] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:36:00] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:36:00] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:36:00] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:36:00] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:36:01] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:36:01] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:36:01] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:36:01] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:36:01] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:36:01] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:36:01] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:36:01] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:36:01] Looking for 'sed' - found /usr/bin/sed\\n[10:36:01] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:36:01] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:36:01] Using genetic code table 11.\\n[10:36:01] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_DSPK01.fasta\\n[10:36:02] Wrote 777 contigs totalling 4863947 bp.\\n[10:36:02] Predicting tRNAs and tmRNAs\\n[10:36:02] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_DSPK01\\\\/\\\\/Fischerella_DSPK01\\\\.fna\\n[10:36:03] 1 tRNA-Gly c[4895,4966] 33 (ccc)\\n[10:36:03] 1 tRNA-Leu c[394,476] 36 (gag)\\n[10:36:03] 1 tRNA-Asn [536,609] 34 (gtt)\\n[10:36:03] 1 tRNA-Pro [524,597] 35 (ggg)\\n[10:36:03] 1 tRNA-Val c[4789,4861] 34 (tac)\\n[10:36:03] 1 tRNA-Gly [4175,4246] 33 (gcc)\\n[10:36:03] 1 tRNA-Val c[4777,4850] 35 (gac)\\n[10:36:03] 1 tRNA-Met c[5996,6071] 36 (cat)\\n[10:36:03] 1 tRNA-Gly c[2964,3034] 33 (tcc)\\n[10:36:03] 1 tRNA-Gln [2671,2741] 32 (ttg)\\n[10:36:03] 1 tRNA-Met [3990,4062] 34 (cat)\\n[10:36:03] 1 tRNA-Ser [2025,2118] 36 (gct)\\n[10:36:03] 1 tRNA-Ala c[2706,2779] 34 (ggc)\\n[10:36:03] 1 tRNA-Ile c[1269,1345] 35 (gat)\\n[10:36:03] 2 tRNA-Ala c[1364,1440] 35 (tgc)\\n[10:36:03] 1 tRNA-Asp [2483,2558] 36 (gtc)\\n[10:36:03] 2 tRNA-Glu [2561,2637] 35 (ctc)\\n[10:36:03] 3 tRNA-Trp [2640,2712] 34 (cca)\\n[10:36:03] 4 tRNA-Tyr [2823,2906] 36 (gta)\\n[10:36:03] 5 tRNA-Ser [2910,2999] 33 (gct)\\n[10:36:03] 6 tRNA-Phe [3004,3078] 35 (gaa)\\n[10:36:03] 7 tRNA-Asn [3087,3161] 33 (gtt)\\n[10:36:03] 8 tRNA-Pro [3369,3445] 35 (tgg)\\n[10:36:03] 9 tRNA-Leu [3449,3523] 35 (tag)\\n[10:36:03] 10 tRNA-Leu [3528,3605] 35 (gag)\\n[10:36:03] 11 tRNA-Lys [3610,3684] 33 (ttt)\\n[10:36:03] 12 tRNA-Gly [3775,3847] 34 (tcc)\\n[10:36:03] 13 tRNA-Val [3969,4043] 34 (tac)\\n[10:36:03] 14 tRNA-Arg [4051,4125] 33 (tct)\\n[10:36:03] 15 tRNA-Ser [4319,4403] 34 (tga)\\n[10:36:03] 1 tRNA-Glu [1935,2007] 35 (ttc)\\n[10:36:03] 1 tRNA-Thr [12398,12469] 33 (cgt)\\n[10:36:03] 1 tRNA-Gln c[2742,2813] 33 (ttg)\\n[10:36:03] 1 tRNA-His [2229,2302] 34 (gtg)\\n[10:36:03] 1 tRNA-Ser [6939,7025] 36 (tga)\\n[10:36:03] 1 tRNA-Leu c[5614,5698] 35 (caa)\\n[10:36:03] 1 tRNA-Arg c[4680,4755] 36 (cct)\\n[10:36:03] 1 tRNA-Ala c[348,422] 35 (cgc)\\n[10:36:03] 1 tRNA-Leu c[5754,5836] 35 (cag)\\n[10:36:03] 1 tRNA-Asp c[3220,3293] 35 (gtc)\\n[10:36:03] 1 tRNA-Trp [1959,2032] 34 (cca)\\n[10:36:03] 1 tRNA-Ser [4411,4497] 36 (cga)\\n[10:36:03] 1 tRNA-Arg [5719,5793] 35 (tct)\\n[10:36:03] 1 tRNA-Phe c[361,433] 34 (gaa)\\n[10:36:03] 1 tRNA-Arg [67,140] 35 (acg)\\n[10:36:03] 1 tRNA-Thr [156,227] 33 (tgt)\\n[10:36:03] 1 tRNA-Arg c[6138,6211] 35 (ccg)\\n[10:36:03] 1 tRNA-Met [3275,3351] 35 (cat)\\n[10:36:03] 1 tRNA-Leu c[2125,2208] 36 (tag)\\n[10:36:03] Found 49 tRNAs\\n[10:36:03] Predicting Ribosomal RNAs\\n[10:36:03] Running Barrnap with 4 threads\\n[10:36:04] 1 DSPK01000337.1 2312 23S ribosomal RNA (partial)\\n[10:36:04] 2 DSPK01000393.1 7586 5S ribosomal RNA\\n[10:36:04] Found 2 rRNAs\\n[10:36:04] Skipping ncRNA search, enable with --rfam if desired.\\n[10:36:04] Total of 51 tRNA + rRNA features\\n[10:36:04] Searching for CRISPR repeats\\n[10:36:05] CRISPR1 DSPK01000719.1 111 with 28 spacers\\n[10:36:05] CRISPR2 DSPK01000749.1 8109 with 24 spacers\\n[10:36:05] CRISPR3 DSPK01000765.1 2451 with 3 spacers\\n[10:36:05] Found 3 CRISPRs\\n[10:36:05] Predicting coding sequences\\n[10:36:05] Contigs total 4863947 bp, so using single mode\\n[10:36:05] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_DSPK01\\\\/\\\\/Fischerella_DSPK01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:36:09] Excluding CDS which overlaps existing RNA (tRNA) at DSPK01000104.1:410..1849 on + strand\\n[10:36:10] Excluding CDS which overlaps existing RNA (tRNA) at DSPK01000565.1:3255..3650 on + strand\\n[10:36:10] Excluding CDS which overlaps existing RNA (tRNA) at DSPK01000613.1:4399..4527 on + strand\\n[10:36:10] Found 4122 CDS\\n[10:36:10] Connecting features back to sequences\\n[10:36:10] Not using genus-specific database. Try --usegenus to enable it.\\n[10:36:10] Annotating CDS, please be patient.\\n[10:36:10] Will use 4 CPUs for similarity searching.\\n[10:36:11] There are still 4122 unannotated CDS left (started with 4122)\\n[10:36:11] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:36:11] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_DSPK01\\\\/\\\\/Fischerella_DSPK01\\\\.IS\\\\.tmp\\\\.3322\\\\.faa | parallel --gnu --plain -j 4 --block 148347 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_DSPK01\\\\/\\\\/Fischerella_DSPK01\\\\.IS\\\\.tmp\\\\.3322\\\\.blast 2> /dev/null\\n[10:36:21] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_DSPK01\\\\/\\\\/Fischerella_DSPK01\\\\.IS\\\\.tmp\\\\.3322\\\\.faa | parallel --gnu --plain -j 4 --block 148347 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_DSPK01\\\\/\\\\/Fischerella_DSPK01\\\\.IS\\\\.tmp\\\\.3322\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:36:21] This is prokka 1.14.6\\n[10:36:21] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:36:21] Homepage is https://github.com/tseemann/prokka\\n[10:36:21] Local time is Mon Feb 21 10:36:21 2022\\n[10:36:21] You are tiagoferreiraleao\\n[10:36:21] Operating system is darwin\\n[10:36:21] You have BioPerl 1.007002\\n[10:36:21] System has 4 cores.\\n[10:36:21] Option --cpu asked for 12 cores, but system only has 4\\n[10:36:21] Will use maximum of 4 cores.\\n[10:36:21] Annotating as >>> Bacteria <<<\\n[10:36:21] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_CENA71.fasta' contents.\\n[10:36:21] Setting --locustag OHEOOBLK from MD5 81e88b5453244827aeebe1dc72283261\\n[10:36:21] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_CENA71/\\n[10:36:21] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_CENA71\\\\/\\n[10:36:21] Using filename prefix: Fischerella_CENA71.XXX\\n[10:36:21] Setting HMMER_NCPU=1\\n[10:36:21] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_CENA71//Fischerella_CENA71.log\\n[10:36:21] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_CENA71.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_CENA71/ --prefix Fischerella_CENA71 --cpus 12\\n[10:36:21] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:36:21] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:36:21] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:36:21] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:36:21] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:36:21] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:36:22] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:36:22] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:36:22] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:36:22] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:36:22] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:36:22] Looking for 'egrep' - found /usr/bin/egrep\\n[10:36:22] Looking for 'find' - found /usr/bin/find\\n[10:36:22] Looking for 'grep' - found /usr/bin/grep\\n[10:36:22] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:36:22] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:36:22] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:36:22] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:36:22] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:36:22] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:36:22] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:36:22] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:36:22] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:36:22] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:36:22] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:36:22] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:36:22] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:36:22] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:36:22] Looking for 'sed' - found /usr/bin/sed\\n[10:36:22] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:36:23] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:36:23] Using genetic code table 11.\\n[10:36:23] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_CENA71.fasta\\n[10:36:23] Wrote 152 contigs totalling 7466050 bp.\\n[10:36:23] Predicting tRNAs and tmRNAs\\n[10:36:23] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_CENA71\\\\/\\\\/Fischerella_CENA71\\\\.fna\\n[10:36:26] 1 tRNA-Ser c[52937,53023] 36 (tga)\\n[10:36:26] 2 tRNA-Leu c[68024,68106] 36 (gag)\\n[10:36:26] 3 tRNA-Thr c[309663,309734] 33 (cgt)\\n[10:36:26] 1 tRNA-Pro c[36951,37024] 35 (cgg)\\n[10:36:26] 2 tRNA-Leu c[295594,295676] 35 (tag)\\n[10:36:26] 1 tRNA-Lys [262569,262641] 33 (ctt)\\n[10:36:26] 1 tRNA-Asn [2428,2501] 34 (gtt)\\n[10:36:26] 2 tRNA-Asp c[141874,141947] 35 (gtc)\\n[10:36:26] 1 tRNA-Arg [172468,172541] 35 (acg)\\n[10:36:26] 2 tRNA-Ser c[182182,182275] 36 (gct)\\n[10:36:26] 1 tRNA-Gly c[166940,167011] 33 (gcc)\\n[10:36:26] 1 tRNA-Met [37362,37438] 36 (cat)\\n[10:36:26] 1 tRNA-Ala c[108270,108344] 35 (ggc)\\n[10:36:26] 2 tRNA-Pro c[108798,108871] 35 (ggg)\\n[10:36:26] 1 tRNA-Val c[9273,9346] 35 (gac)\\n[10:36:26] 2 tRNA-Met [12383,12459] 35 (cat)\\n[10:36:26] 1 tRNA-Leu c[76305,76387] 35 (cag)\\n[10:36:26] 1 tRNA-Lys c[102721,102792] 33 (ttt)\\n[10:36:26] 1 tRNA-Gln c[79892,79963] 33 (ttg)\\n[10:36:26] 2 tRNA-Leu c[111120,111203] 35 (caa)\\n[10:36:26] 1 tmRNA c[2363,2749] 100,153 ANNIVKFARKEALVAA**\\n[10:36:26] 2 tRNA-Trp [75591,75664] 34 (cca)\\n[10:36:26] 1 tRNA-Gly c[68724,68794] 33 (tcc)\\n[10:36:26] 1 tRNA-Ser c[14255,14341] 36 (cga)\\n[10:36:26] 1 tRNA-Ser [85976,86064] 36 (gga)\\n[10:36:26] 1 tRNA-His c[2540,2613] 34 (gtg)\\n[10:36:26] 1 tRNA-Gly c[3583,3654] 33 (ccc)\\n[10:36:26] 1 tRNA-Ile [73702,73777] 35 (gat)\\n[10:36:26] 2 tRNA-Ala [73855,73929] 34 (tgc)\\n[10:36:26] 1 tRNA-Arg [9077,9149] 34 (acg)\\n[10:36:26] 2 tRNA-Asn [9845,9918] 34 (att)\\n[10:36:26] 1 tRNA-Trp [47777,47851] 35 (cca)\\n[10:36:26] 2 tRNA-Trp [50081,50155] 35 (cca)\\n[10:36:26] 1 tRNA-Pro [54580,54653] 35 (tgg)\\n[10:36:26] 1 tRNA-Thr c[34118,34189] 33 (ggt)\\n[10:36:26] 2 tRNA-Tyr c[34267,34351] 35 (gta)\\n[10:36:26] 1 tRNA-Phe [28663,28735] 34 (gaa)\\n[10:36:26] 1 tRNA-Arg c[1372,1446] 35 (tct)\\n[10:36:26] 1 tRNA-Cys [6172,6243] 33 (gca)\\n[10:36:26] 1 tRNA-Glu [15238,15310] 35 (ttc)\\n[10:36:26] 2 tRNA-Thr c[27122,27193] 33 (tgt)\\n[10:36:26] 1 tRNA-Ala c[35414,35486] 34 (cgc)\\n[10:36:26] 1 tRNA-Val c[32585,32657] 34 (tac)\\n[10:36:26] 1 tRNA-Arg [29926,29998] 34 (ccg)\\n[10:36:26] 1 tRNA-Met c[16402,16474] 34 (cat)\\n[10:36:26] Found 45 tRNAs\\n[10:36:26] Predicting Ribosomal RNAs\\n[10:36:26] Running Barrnap with 4 threads\\n[10:36:28] 1 NODE_315_length_77219_cov_102.249630 72100 16S ribosomal RNA\\n[10:36:28] 2 NODE_315_length_77219_cov_102.249630 74141 23S ribosomal RNA\\n[10:36:28] Found 2 rRNAs\\n[10:36:28] Skipping ncRNA search, enable with --rfam if desired.\\n[10:36:28] Total of 46 tRNA + rRNA features\\n[10:36:28] Searching for CRISPR repeats\\n[10:36:28] CRISPR1 NODE_134_length_164225_cov_95.631149 160386 with 10 spacers\\n[10:36:28] CRISPR2 NODE_142_length_159900_cov_96.346479 8832 with 4 spacers\\n[10:36:28] CRISPR3 NODE_176_length_137694_cov_93.343694 113801 with 8 spacers\\n[10:36:28] CRISPR4 NODE_217_length_113081_cov_91.885845 167 with 8 spacers\\n[10:36:28] CRISPR5 NODE_217_length_113081_cov_91.885845 12077 with 18 spacers\\n[10:36:29] CRISPR6 NODE_463_length_44371_cov_82.057876 10393 with 26 spacers\\n[10:36:29] CRISPR7 NODE_463_length_44371_cov_82.057876 22598 with 12 spacers\\n[10:36:29] CRISPR8 NODE_463_length_44371_cov_82.057876 24095 with 10 spacers\\n[10:36:29] CRISPR9 NODE_463_length_44371_cov_82.057876 36871 with 10 spacers\\n[10:36:29] CRISPR10 NODE_463_length_44371_cov_82.057876 37839 with 11 spacers\\n[10:36:29] CRISPR11 NODE_516_length_37350_cov_91.571597 30322 with 8 spacers\\n[10:36:29] CRISPR12 NODE_572_length_31127_cov_87.203283 16162 with 15 spacers\\n[10:36:29] CRISPR13 NODE_572_length_31127_cov_87.203283 24549 with 14 spacers\\n[10:36:29] CRISPR14 NODE_594_length_29279_cov_68.207902 3882 with 8 spacers\\n[10:36:29] CRISPR15 NODE_594_length_29279_cov_68.207902 11957 with 17 spacers\\n[10:36:29] CRISPR16 NODE_594_length_29279_cov_68.207902 13579 with 10 spacers\\n[10:36:29] CRISPR17 NODE_838_length_14806_cov_85.747770 3344 with 8 spacers\\n[10:36:29] CRISPR18 NODE_1425_length_5694_cov_78.907231 83 with 3 spacers\\n[10:36:29] Found 18 CRISPRs\\n[10:36:29] Predicting coding sequences\\n[10:36:29] Contigs total 7466050 bp, so using single mode\\n[10:36:29] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_CENA71\\\\/\\\\/Fischerella_CENA71\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:36:35] Excluding CDS which overlaps existing RNA (tRNA) at NODE_59_length_276955_cov_95.191823:141864..142127 on - strand\\n[10:36:37] Excluding CDS which overlaps existing RNA (rRNA) at NODE_315_length_77219_cov_102.249630:74200..74670 on - strand\\n[10:36:37] Excluding CDS which overlaps existing RNA (rRNA) at NODE_315_length_77219_cov_102.249630:75586..75780 on + strand\\n[10:36:37] Excluding CDS which overlaps existing RNA (tRNA) at NODE_355_length_63269_cov_82.640163:47753..47902 on - strand\\n[10:36:37] Excluding CDS which overlaps existing RNA (tRNA) at NODE_355_length_63269_cov_82.640163:50069..50188 on - strand\\n[10:36:38] Excluding CDS which overlaps existing RNA (repeat_region) at NODE_463_length_44371_cov_82.057876:10528..10638 on - strand\\n[10:36:38] Excluding CDS which overlaps existing RNA (repeat_region) at NODE_594_length_29279_cov_68.207902:4413..4598 on - strand\\n[10:36:38] Excluding CDS which overlaps existing RNA (repeat_region) at NODE_594_length_29279_cov_68.207902:13608..13763 on + strand\\n[10:36:38] Found 6318 CDS\\n[10:36:38] Connecting features back to sequences\\n[10:36:38] Not using genus-specific database. Try --usegenus to enable it.\\n[10:36:38] Annotating CDS, please be patient.\\n[10:36:38] Will use 4 CPUs for similarity searching.\\n[10:36:40] There are still 6318 unannotated CDS left (started with 6318)\\n[10:36:40] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:36:40] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_CENA71\\\\/\\\\/Fischerella_CENA71\\\\.IS\\\\.tmp\\\\.3434\\\\.faa | parallel --gnu --plain -j 4 --block 254174 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_CENA71\\\\/\\\\/Fischerella_CENA71\\\\.IS\\\\.tmp\\\\.3434\\\\.blast 2> /dev/null\\n[10:36:50] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_CENA71\\\\/\\\\/Fischerella_CENA71\\\\.IS\\\\.tmp\\\\.3434\\\\.faa | parallel --gnu --plain -j 4 --block 254174 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_CENA71\\\\/\\\\/Fischerella_CENA71\\\\.IS\\\\.tmp\\\\.3434\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:36:50] This is prokka 1.14.6\\n[10:36:50] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:36:50] Homepage is https://github.com/tseemann/prokka\\n[10:36:50] Local time is Mon Feb 21 10:36:50 2022\\n[10:36:50] You are tiagoferreiraleao\\n[10:36:50] Operating system is darwin\\n[10:36:50] You have BioPerl 1.007002\\n[10:36:50] System has 4 cores.\\n[10:36:50] Option --cpu asked for 12 cores, but system only has 4\\n[10:36:50] Will use maximum of 4 cores.\\n[10:36:50] Annotating as >>> Bacteria <<<\\n[10:36:50] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JAHHHS01.fasta' contents.\\n[10:36:50] Setting --locustag FMDKCKMF from MD5 f6d4c46fc360ccf1e71d991eae7902c4\\n[10:36:50] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JAHHHS01/\\n[10:36:50] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JAHHHS01\\\\/\\n[10:36:50] Using filename prefix: Fischerella_JAHHHS01.XXX\\n[10:36:50] Setting HMMER_NCPU=1\\n[10:36:50] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JAHHHS01//Fischerella_JAHHHS01.log\\n[10:36:50] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JAHHHS01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JAHHHS01/ --prefix Fischerella_JAHHHS01 --cpus 12\\n[10:36:50] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:36:50] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:36:50] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:36:50] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:36:50] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:36:50] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:36:51] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:36:51] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:36:51] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:36:51] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:36:51] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:36:51] Looking for 'egrep' - found /usr/bin/egrep\\n[10:36:51] Looking for 'find' - found /usr/bin/find\\n[10:36:51] Looking for 'grep' - found /usr/bin/grep\\n[10:36:51] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:36:51] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:36:51] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:36:51] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:36:51] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:36:51] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:36:51] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:36:51] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:36:51] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:36:51] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:36:51] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:36:51] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:36:51] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:36:51] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:36:51] Looking for 'sed' - found /usr/bin/sed\\n[10:36:51] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:36:52] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:36:52] Using genetic code table 11.\\n[10:36:52] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JAHHHS01.fasta\\n[10:36:52] Wrote 253 contigs totalling 8868606 bp.\\n[10:36:52] Predicting tRNAs and tmRNAs\\n[10:36:52] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JAHHHS01\\\\/\\\\/Fischerella_JAHHHS01\\\\.fna\\n[10:36:56] 1 tRNA-Leu c[60826,60908] 35 (tag)\\n[10:36:56] 2 tRNA-Ser [152947,153035] 36 (gga)\\n[10:36:56] 1 tRNA-Pro [88715,88788] 35 (tgg)\\n[10:36:56] 1 tRNA-Arg [57812,57884] 34 (ccg)\\n[10:36:56] 1 tmRNA [122066,122456] 101,154 ANNIVSFARKPALVAAA*\\n[10:36:56] 2 tRNA-Gly c[161097,161168] 33 (ccc)\\n[10:36:56] 1 tRNA-Leu [116153,116235] 36 (gag)\\n[10:36:56] 1 tRNA-Glu [95205,95278] 35 (ttc)\\n[10:36:56] 1 tRNA-Lys c[18210,18282] 33 (ctt)\\n[10:36:56] 1 tRNA-Thr [57696,57767] 33 (cgt)\\n[10:36:56] 1 tRNA-Pro [58962,59035] 35 (ggg)\\n[10:36:56] 1 tRNA-Gly [64473,64544] 33 (gcc)\\n[10:36:56] 1 tRNA-Ala c[82781,82873] 33 (ggc)\\n[10:36:56] 1 tRNA-Met [8289,8362] 33 (cat)\\n[10:36:56] 2 tRNA-Thr c[18939,19010] 33 (tgt)\\n[10:36:56] 3 tRNA-Asn c[78218,78289] 33 (gtt)\\n[10:36:56] 1 tRNA-Ser [4661,4754] 36 (gct)\\n[10:36:56] 2 tRNA-Ala [74744,74816] 34 (ggc)\\n[10:36:56] 1 tRNA-Ala [37190,37264] 35 (tgc)\\n[10:36:56] 2 tRNA-Val c[58283,58355] 34 (tac)\\n[10:36:56] 1 tRNA-Arg c[1840,1914] 35 (tct)\\n[10:36:56] 1 tRNA-Pro [32779,32852] 35 (cgg)\\n[10:36:56] 1 tRNA-Cys [34128,34198] 33 (gca)\\n[10:36:56] 2 tRNA-Lys [43010,43081] 33 (ttt)\\n[10:36:56] 1 tRNA-Met c[8188,8262] 35 (cat)\\n[10:36:56] 2 tRNA-Arg [48124,48199] 36 (cct)\\n[10:36:56] 1 tRNA-Arg [45970,46043] 35 (acg)\\n[10:36:56] 1 tRNA-Gly c[23995,24065] 33 (tcc)\\n[10:36:56] 1 tRNA-Met c[42559,42635] 35 (cat)\\n[10:36:56] 1 tRNA-Trp [17797,17870] 34 (cca)\\n[10:36:56] 2 tRNA-Ala c[22414,22486] 34 (cgc)\\n[10:36:56] 1 tRNA-Asp c[12308,12381] 35 (gtc)\\n[10:36:56] 1 tRNA-Phe c[9603,9675] 34 (gaa)\\n[10:36:56] 1 tRNA-Gln c[8936,9009] 34 (ttg)\\n[10:36:56] 2 tRNA-Leu c[15811,15895] 35 (caa)\\n[10:36:56] 3 tRNA-Leu [21221,21302] 35 (cag)\\n[10:36:56] 1 tRNA-Val c[5464,5537] 35 (gac)\\n[10:36:56] 1 tRNA-Ile [9249,9337] 31 (tat)\\n[10:36:56] 1 tRNA-His [14600,14673] 34 (gtg)\\n[10:36:56] 1 tRNA-Ser [12962,13048] 36 (cga)\\n[10:36:56] Found 40 tRNAs\\n[10:36:56] Predicting Ribosomal RNAs\\n[10:36:56] Running Barrnap with 4 threads\\n[10:36:58] Found 0 rRNAs\\n[10:36:58] Skipping ncRNA search, enable with --rfam if desired.\\n[10:36:58] Total of 39 tRNA + rRNA features\\n[10:36:58] Searching for CRISPR repeats\\n[10:36:58] CRISPR1 JAHHHS010000002.1 134653 with 5 spacers\\n[10:36:58] CRISPR2 JAHHHS010000033.1 57363 with 4 spacers\\n[10:36:58] CRISPR3 JAHHHS010000086.1 26397 with 3 spacers\\n[10:36:59] CRISPR4 JAHHHS010000203.1 6455 with 5 spacers\\n[10:36:59] Found 4 CRISPRs\\n[10:36:59] Predicting coding sequences\\n[10:36:59] Contigs total 8868606 bp, so using single mode\\n[10:36:59] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JAHHHS01\\\\/\\\\/Fischerella_JAHHHS01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:37:07] Excluding CDS which overlaps existing RNA (tRNA) at JAHHHS010000023.1:64460..64765 on + strand\\n[10:37:08] Excluding CDS which overlaps existing RNA (repeat_region) at JAHHHS010000033.1:56768..57406 on + strand\\n[10:37:08] Excluding CDS which overlaps existing RNA (tRNA) at JAHHHS010000060.1:17773..17919 on + strand\\n[10:37:09] Excluding CDS which overlaps existing RNA (tRNA) at JAHHHS010000077.1:8918..9082 on + strand\\n[10:37:10] Excluding CDS which overlaps existing RNA (repeat_region) at JAHHHS010000203.1:6639..6824 on + strand\\n[10:37:10] Found 8431 CDS\\n[10:37:10] Connecting features back to sequences\\n[10:37:10] Not using genus-specific database. Try --usegenus to enable it.\\n[10:37:10] Annotating CDS, please be patient.\\n[10:37:10] Will use 4 CPUs for similarity searching.\\n[10:37:12] There are still 8431 unannotated CDS left (started with 8431)\\n[10:37:12] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:37:12] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JAHHHS01\\\\/\\\\/Fischerella_JAHHHS01\\\\.IS\\\\.tmp\\\\.3546\\\\.faa | parallel --gnu --plain -j 4 --block 295205 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JAHHHS01\\\\/\\\\/Fischerella_JAHHHS01\\\\.IS\\\\.tmp\\\\.3546\\\\.blast 2> /dev/null\\n[10:37:16] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JAHHHS01\\\\/\\\\/Fischerella_JAHHHS01\\\\.IS\\\\.tmp\\\\.3546\\\\.faa | parallel --gnu --plain -j 4 --block 295205 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JAHHHS01\\\\/\\\\/Fischerella_JAHHHS01\\\\.IS\\\\.tmp\\\\.3546\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:37:16] This is prokka 1.14.6\\n[10:37:16] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:37:16] Homepage is https://github.com/tseemann/prokka\\n[10:37:16] Local time is Mon Feb 21 10:37:16 2022\\n[10:37:16] You are tiagoferreiraleao\\n[10:37:16] Operating system is darwin\\n[10:37:16] You have BioPerl 1.007002\\n[10:37:16] System has 4 cores.\\n[10:37:16] Option --cpu asked for 12 cores, but system only has 4\\n[10:37:16] Will use maximum of 4 cores.\\n[10:37:16] Annotating as >>> Bacteria <<<\\n[10:37:16] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHB01.fasta' contents.\\n[10:37:16] Setting --locustag EBGHLNOM from MD5 eb015786a3a3ad3bcb1ab86265e57e14\\n[10:37:16] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHB01/\\n[10:37:16] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHB01\\\\/\\n[10:37:16] Using filename prefix: Nostoc_LAHB01.XXX\\n[10:37:16] Setting HMMER_NCPU=1\\n[10:37:16] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHB01//Nostoc_LAHB01.log\\n[10:37:16] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHB01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHB01/ --prefix Nostoc_LAHB01 --cpus 12\\n[10:37:16] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:37:16] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:37:16] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:37:16] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:37:16] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:37:16] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:37:16] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:37:16] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:37:16] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:37:16] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:37:17] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:37:17] Looking for 'egrep' - found /usr/bin/egrep\\n[10:37:17] Looking for 'find' - found /usr/bin/find\\n[10:37:17] Looking for 'grep' - found /usr/bin/grep\\n[10:37:17] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:37:17] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:37:17] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:37:17] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:37:17] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:37:17] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:37:17] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:37:17] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:37:17] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:37:17] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:37:17] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:37:17] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:37:17] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:37:17] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:37:17] Looking for 'sed' - found /usr/bin/sed\\n[10:37:17] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:37:17] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:37:17] Using genetic code table 11.\\n[10:37:17] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHB01.fasta\\n[10:37:18] Wrote 138 contigs totalling 9014207 bp.\\n[10:37:18] Predicting tRNAs and tmRNAs\\n[10:37:18] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHB01\\\\/\\\\/Nostoc_LAHB01\\\\.fna\\n[10:37:22] 1 tRNA-Gly c[310042,310112] 33 (tcc)\\n[10:37:22] 1 tRNA-Cys [90219,90290] 33 (gca)\\n[10:37:22] 2 tRNA-His [240118,240191] 34 (gtg)\\n[10:37:22] 1 tmRNA c[194000,194387] 101,151 ANNIVKFARREALVAA*\\n[10:37:22] 1 tRNA-Lys [186768,186839] 33 (ttt)\\n[10:37:22] 1 tRNA-Thr [161137,161208] 33 (cgt)\\n[10:37:22] 2 tRNA-Trp [206957,207030] 34 (cca)\\n[10:37:22] 1 tRNA-Arg c[3538,3612] 35 (tct)\\n[10:37:22] 1 tRNA-Asn [28828,28901] 34 (gtt)\\n[10:37:22] 2 tRNA-Leu c[128500,128583] 36 (tag)\\n[10:37:22] 3 tRNA-Ala c[162915,162989] 34 (tgc)\\n[10:37:22] 4 tRNA-Ile c[163073,163148] 35 (gat)\\n[10:37:22] 1 tRNA-Ala c[229316,229388] 34 (tgc)\\n[10:37:22] 1 tRNA-Ala c[34958,35044] 29 (cgc)\\n[10:37:22] 2 tRNA-Ser [149804,149890] 36 (cga)\\n[10:37:22] 1 tRNA-Pro [112544,112617] 35 (cgg)\\n[10:37:22] 1 tRNA-Thr [101712,101784] 33 (tgt)\\n[10:37:22] 1 tRNA-Gln c[29258,29331] 34 (ttg)\\n[10:37:22] 2 tRNA-Leu c[34085,34169] 35 (caa)\\n[10:37:22] 3 tRNA-Ala [82871,82943] 34 (cgc)\\n[10:37:22] 1 tRNA-Ser [53909,53996] 35 (gga)\\n[10:37:22] 2 tRNA-Gly c[62141,62214] 34 (ccc)\\n[10:37:22] 1 tRNA-Asp c[59509,59582] 35 (gtc)\\n[10:37:22] 1 tRNA-Met [11590,11664] 35 (cat)\\n[10:37:22] 1 tRNA-Leu [11952,12033] 35 (cag)\\n[10:37:22] 1 tRNA-Thr [46572,46646] 33 (agt)\\n[10:37:22] 2 tRNA-Leu [48037,48111] 35 (caa)\\n[10:37:22] 1 tRNA-Ala c[14793,14865] 34 (ggc)\\n[10:37:22] 1 tRNA-Ser c[15538,15631] 36 (gct)\\n[10:37:22] 2 tRNA-Lys c[21306,21376] 33 (ttt)\\n[10:37:22] 3 tRNA-Ser c[21635,21718] 34 (tga)\\n[10:37:22] 4 tRNA-His c[21723,21798] 35 (gtg)\\n[10:37:22] 5 tRNA-Ala c[21804,21878] 35 (tgc)\\n[10:37:22] 6 tRNA-Arg c[21970,22045] 35 (tct)\\n[10:37:22] 7 tRNA-Val c[22052,22127] 34 (tac)\\n[10:37:22] 8 tRNA-Thr c[22209,22281] 34 (tgt)\\n[10:37:22] 9 tRNA-Leu c[22576,22650] 35 (tag)\\n[10:37:22] 10 tRNA-Leu c[22762,22838] 34 (gag)\\n[10:37:22] 11 tRNA-Pro c[22946,23019] 35 (tgg)\\n[10:37:22] 12 tRNA-Leu c[23021,23098] 35 (taa)\\n[10:37:22] 13 tRNA-Leu c[23119,23196] 36 (caa)\\n[10:37:22] 14 tRNA-Gln c[23222,23297] 35 (ttg)\\n[10:37:22] 15 tRNA-Gln c[24172,24246] 35 (ctg)\\n[10:37:22] 16 tRNA-Asn c[24257,24331] 33 (gtt)\\n[10:37:22] 17 tRNA-Phe c[24337,24412] 36 (gaa)\\n[10:37:22] 18 tRNA-Arg c[24500,24575] 35 (acg)\\n[10:37:22] 19 tRNA-Ser c[25337,25423] 35 (gct)\\n[10:37:22] 20 tRNA-Tyr c[25427,25508] 35 (gta)\\n[10:37:22] 21 tRNA-Glu c[25519,25593] 35 (ttc)\\n[10:37:22] tRNA c[25681,25755] is a pseudo/wacky gene - skipping.\\n[10:37:22] 23 tRNA-Met c[25758,25831] 32 (cat)\\n[10:37:22] 24 tRNA-Glu c[25941,26017] 35 (ctc)\\n[10:37:22] 25 tRNA-Asp c[26021,26096] 36 (gtc)\\n[10:37:22] 1 tRNA-Pro c[38341,38414] 35 (tgg)\\n[10:37:22] 2 tRNA-Ser c[47096,47182] 36 (tga)\\n[10:37:22] 3 tRNA-Tyr [49633,49717] 35 (gta)\\n[10:37:22] 4 tRNA-Thr [49793,49864] 33 (ggt)\\n[10:37:22] 1 tRNA-Pro c[43709,43782] 35 (ggg)\\n[10:37:22] 1 tRNA-Ile c[31798,31873] 34 (tat)\\n[10:37:22] 2 tRNA-Met c[33479,33554] 34 (cat)\\n[10:37:22] 3 tRNA-Met c[33556,33631] 34 (cat)\\n[10:37:22] 1 tRNA-Met [18619,18692] 33 (cat)\\n[10:37:22] 2 tRNA-Gly [25502,25573] 33 (gcc)\\n[10:37:22] 1 tRNA-Asp [25373,25450] 38 (gtc)\\n[10:37:22] 2 tRNA-Glu [25453,25529] 35 (ctc)\\n[10:37:22] 3 tRNA-Cys [25534,25609] 34 (gca)\\n[10:37:22] 4 tRNA-Met [25615,25689] 33 (cat)\\n[10:37:22] 5 tRNA-Trp [25692,25764] 34 (cca)\\n[10:37:22] 6 tRNA-Pyl [25789,25860] 33 (cta)\\n[10:37:22] 7 tRNA-Glu [25891,25966] 34 (ttc)\\n[10:37:22] 8 tRNA-Tyr [25970,26053] 36 (gta)\\n[10:37:22] 9 tRNA-Ser [26057,26146] 33 (gct)\\n[10:37:22] 10 tRNA-Arg [26151,26226] 35 (acg)\\n[10:37:22] 11 tRNA-Phe [26231,26305] 35 (gaa)\\n[10:37:22] 12 tRNA-Asn [26314,26388] 33 (gtt)\\n[10:37:22] 13 tRNA-Gln [26394,26469] 35 (ttg)\\n[10:37:22] 14 tRNA-Leu [26477,26554] 35 (caa)\\n[10:37:22] 15 tRNA-Pro [26559,26635] 35 (tgg)\\n[10:37:22] 16 tRNA-Leu [26639,26713] 35 (tag)\\n[10:37:22] 17 tRNA-Leu [26733,26810] 35 (gag)\\n[10:37:22] 18 tRNA-Lys [26814,26890] 34 (ttt)\\n[10:37:22] 19 tRNA-Lys [26896,26970] 33 (ctt)\\n[10:37:22] 20 tRNA-Leu [26976,27051] 36 (taa)\\n[10:37:22] 21 tRNA-Cys [27061,27138] 34 (gca)\\n[10:37:22] 22 tRNA-Thr [27146,27218] 34 (tgt)\\n[10:37:22] 23 tRNA-Gly [27224,27294] 33 (tcc)\\n[10:37:22] 24 tRNA-Val [27300,27375] 34 (tac)\\n[10:37:22] 25 tRNA-Arg [27382,27456] 35 (tct)\\n[10:37:22] 26 tRNA-Ala [27461,27533] 34 (tgc)\\n[10:37:22] 27 tRNA-His [27541,27617] 35 (gtg)\\n[10:37:22] 28 tRNA-Ser [27628,27712] 34 (tga)\\n[10:37:22] 1 tRNA-Phe c[873,946] 34 (gaa)\\n[10:37:22] 1 tRNA-Arg c[43920,43993] 35 (acg)\\n[10:37:22] 1 tRNA-Leu c[3546,3626] 35 (gag)\\n[10:37:22] 1 tRNA-Lys c[3705,3777] 33 (ctt)\\n[10:37:22] 1 tRNA-Arg [4901,4973] 34 (ccg)\\n[10:37:22] 1 tRNA-Glu [22623,22696] 35 (ttc)\\n[10:37:22] 1 tRNA-Val c[14182,14254] 34 (tac)\\n[10:37:22] 1 tRNA-Val [14317,14390] 35 (gac)\\n[10:37:22] Found 98 tRNAs\\n[10:37:22] Predicting Ribosomal RNAs\\n[10:37:22] Running Barrnap with 4 threads\\n[10:37:24] 1 LAHB01000008.1 159919 23S ribosomal RNA\\n[10:37:24] 2 LAHB01000008.1 163270 16S ribosomal RNA\\n[10:37:24] Found 2 rRNAs\\n[10:37:24] Skipping ncRNA search, enable with --rfam if desired.\\n[10:37:24] Total of 99 tRNA + rRNA features\\n[10:37:24] Searching for CRISPR repeats\\n[10:37:24] CRISPR1 LAHB01000003.1 169372 with 14 spacers\\n[10:37:24] CRISPR2 LAHB01000006.1 110481 with 24 spacers\\n[10:37:25] CRISPR3 LAHB01000013.1 27628 with 15 spacers\\n[10:37:25] CRISPR4 LAHB01000019.1 63973 with 7 spacers\\n[10:37:25] CRISPR5 LAHB01000025.1 8572 with 7 spacers\\n[10:37:25] CRISPR6 LAHB01000026.1 66096 with 24 spacers\\n[10:37:25] CRISPR7 LAHB01000026.1 149796 with 10 spacers\\n[10:37:25] CRISPR8 LAHB01000033.1 51638 with 17 spacers\\n[10:37:25] CRISPR9 LAHB01000047.1 15920 with 20 spacers\\n[10:37:25] CRISPR10 LAHB01000051.1 14103 with 6 spacers\\n[10:37:25] CRISPR11 LAHB01000061.1 26583 with 28 spacers\\n[10:37:25] CRISPR12 LAHB01000061.1 40017 with 43 spacers\\n[10:37:25] CRISPR13 LAHB01000082.1 36169 with 18 spacers\\n[10:37:25] Found 13 CRISPRs\\n[10:37:25] Predicting coding sequences\\n[10:37:25] Contigs total 9014207 bp, so using single mode\\n[10:37:25] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHB01\\\\/\\\\/Nostoc_LAHB01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:37:33] Excluding CDS which overlaps existing RNA (tRNA) at LAHB01000002.1:240039..240182 on - strand\\n[10:37:33] Excluding CDS which overlaps existing RNA (repeat_region) at LAHB01000003.1:170088..170204 on + strand\\n[10:37:34] Excluding CDS which overlaps existing RNA (tRNA) at LAHB01000011.1:34493..35863 on - strand\\n[10:37:35] Excluding CDS which overlaps existing RNA (repeat_region) at LAHB01000025.1:8892..9017 on + strand\\n[10:37:35] Excluding CDS which overlaps existing RNA (repeat_region) at LAHB01000026.1:67712..67990 on - strand\\n[10:37:35] Excluding CDS which overlaps existing RNA (repeat_region) at LAHB01000026.1:150005..150157 on + strand\\n[10:37:36] Excluding CDS which overlaps existing RNA (repeat_region) at LAHB01000051.1:14495..15454 on - strand\\n[10:37:36] Excluding CDS which overlaps existing RNA (repeat_region) at LAHB01000061.1:40058..40222 on - strand\\n[10:37:37] Found 7597 CDS\\n[10:37:37] Connecting features back to sequences\\n[10:37:37] Not using genus-specific database. Try --usegenus to enable it.\\n[10:37:37] Annotating CDS, please be patient.\\n[10:37:37] Will use 4 CPUs for similarity searching.\\n[10:37:39] There are still 7597 unannotated CDS left (started with 7597)\\n[10:37:39] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:37:39] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHB01\\\\/\\\\/Nostoc_LAHB01\\\\.IS\\\\.tmp\\\\.3682\\\\.faa | parallel --gnu --plain -j 4 --block 305369 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHB01\\\\/\\\\/Nostoc_LAHB01\\\\.IS\\\\.tmp\\\\.3682\\\\.blast 2> /dev/null\\n[10:38:03] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHB01\\\\/\\\\/Nostoc_LAHB01\\\\.IS\\\\.tmp\\\\.3682\\\\.faa | parallel --gnu --plain -j 4 --block 305369 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHB01\\\\/\\\\/Nostoc_LAHB01\\\\.IS\\\\.tmp\\\\.3682\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:38:04] This is prokka 1.14.6\\n[10:38:04] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:38:04] Homepage is https://github.com/tseemann/prokka\\n[10:38:04] Local time is Mon Feb 21 10:38:04 2022\\n[10:38:04] You are tiagoferreiraleao\\n[10:38:04] Operating system is darwin\\n[10:38:04] You have BioPerl 1.007002\\n[10:38:04] System has 4 cores.\\n[10:38:04] Option --cpu asked for 12 cores, but system only has 4\\n[10:38:04] Will use maximum of 4 cores.\\n[10:38:04] Annotating as >>> Bacteria <<<\\n[10:38:04] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQH01.fasta' contents.\\n[10:38:04] Setting --locustag LGNKBADH from MD5 5074bad1dbfa49ba841821f5dc469fdf\\n[10:38:04] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQH01/\\n[10:38:04] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQH01\\\\/\\n[10:38:04] Using filename prefix: Fischerella_NMQH01.XXX\\n[10:38:04] Setting HMMER_NCPU=1\\n[10:38:04] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQH01//Fischerella_NMQH01.log\\n[10:38:04] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQH01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQH01/ --prefix Fischerella_NMQH01 --cpus 12\\n[10:38:04] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:38:04] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:38:04] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:38:04] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:38:04] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:38:04] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:38:04] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:38:04] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:38:04] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:38:04] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:38:04] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:38:04] Looking for 'egrep' - found /usr/bin/egrep\\n[10:38:04] Looking for 'find' - found /usr/bin/find\\n[10:38:04] Looking for 'grep' - found /usr/bin/grep\\n[10:38:04] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:38:04] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:38:04] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:38:04] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:38:04] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:38:04] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:38:04] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:38:04] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:38:04] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:38:04] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:38:04] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:38:04] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:38:04] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:38:04] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:38:04] Looking for 'sed' - found /usr/bin/sed\\n[10:38:04] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:38:05] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:38:05] Using genetic code table 11.\\n[10:38:05] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQH01.fasta\\n[10:38:05] Wrote 403 contigs totalling 5356884 bp.\\n[10:38:05] Predicting tRNAs and tmRNAs\\n[10:38:05] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQH01\\\\/\\\\/Fischerella_NMQH01\\\\.fna\\n[10:38:07] 1 tRNA-Gln [27511,27582] 33 (ttg)\\n[10:38:07] 1 tRNA-His c[9770,9843] 34 (gtg)\\n[10:38:07] 1 tRNA-Leu [15440,15522] 35 (cag)\\n[10:38:07] 1 tRNA-Ser [9348,9441] 36 (gct)\\n[10:38:07] 1 tRNA-Gly c[1639,1710] 33 (ccc)\\n[10:38:07] 1 tRNA-Leu [16763,16846] 36 (tag)\\n[10:38:07] 1 tRNA-Lys c[21973,22046] 34 (ttt)\\n[10:38:07] 1 tRNA-Ser c[7521,7607] 36 (cga)\\n[10:38:07] 1 tRNA-Gly [3679,3750] 33 (gcc)\\n[10:38:07] 1 tRNA-Ser c[4191,4279] 36 (gga)\\n[10:38:07] 1 tRNA-Val c[12579,12651] 34 (tac)\\n[10:38:07] 2 tRNA-Phe [20003,20075] 34 (gaa)\\n[10:38:07] 1 tRNA-Thr [36137,36208] 33 (cgt)\\n[10:38:07] 1 tRNA-Ala [1465,1538] 34 (ggc)\\n[10:38:07] 2 tRNA-Arg [6430,6503] 35 (acg)\\n[10:38:07] 1 tRNA-Ser c[323,409] 36 (tga)\\n[10:38:07] 1 tRNA-Thr c[8112,8183] 33 (tgt)\\n[10:38:07] 1 tRNA-Leu c[541,625] 35 (caa)\\n[10:38:07] 1 tRNA-Ala [10090,10165] 35 (cgc)\\n[10:38:07] 1 tRNA-Gln [7076,7146] 32 (ttg)\\n[10:38:07] 1 tRNA-Trp [1889,1962] 34 (cca)\\n[10:38:07] 1 tRNA-Met [8704,8776] 34 (cat)\\n[10:38:07] 1 tRNA-Asn [2676,2749] 34 (gtt)\\n[10:38:07] 1 tRNA-Cys c[94,165] 33 (gca)\\n[10:38:07] 1 tRNA-Arg [12947,13022] 36 (cct)\\n[10:38:07] 1 tRNA-Arg [7499,7572] 35 (ccg)\\n[10:38:07] 1 tRNA-Asp c[17758,17831] 35 (gtc)\\n[10:38:07] 1 tRNA-Pro c[1739,1813] 35 (tgg)\\n[10:38:07] 1 tRNA-Glu [7519,7591] 35 (ttc)\\n[10:38:07] 1 tRNA-Pro [2148,2221] 35 (cgg)\\n[10:38:07] 1 tRNA-Arg c[16228,16302] 35 (tct)\\n[10:38:07] 1 tRNA-Met c[11864,11939] 36 (cat)\\n[10:38:07] 1 tRNA-Pro c[3332,3405] 35 (ggg)\\n[10:38:07] 1 tRNA-Thr c[17077,17148] 33 (ggt)\\n[10:38:07] 2 tRNA-Tyr c[17212,17296] 35 (gta)\\n[10:38:07] 1 tRNA-Met c[26571,26647] 35 (cat)\\n[10:38:07] 2 tRNA-Val [29593,29666] 35 (gac)\\n[10:38:07] 1 tmRNA c[248,634] 100,150 ANNIVKFARKEALVAA*\\n[10:38:07] 1 tRNA-Gly [8021,8091] 33 (tcc)\\n[10:38:07] 1 tRNA-Leu c[3910,3992] 36 (gag)\\n[10:38:07] 1 tRNA-Arg [839,911] 34 (tcg)\\n[10:38:07] 2 tRNA-Asn [1582,1655] 34 (att)\\n[10:38:07] Found 42 tRNAs\\n[10:38:07] Predicting Ribosomal RNAs\\n[10:38:07] Running Barrnap with 4 threads\\n[10:38:08] 1 NMQH01000004.1 4 5S ribosomal RNA\\n[10:38:08] 2 NMQH01000004.1 233 23S ribosomal RNA\\n[10:38:08] 3 NMQH01000239.1 116 16S ribosomal RNA\\n[10:38:08] Found 3 rRNAs\\n[10:38:08] Skipping ncRNA search, enable with --rfam if desired.\\n[10:38:08] Total of 44 tRNA + rRNA features\\n[10:38:08] Searching for CRISPR repeats\\n[10:38:08] CRISPR1 NMQH01000003.1 4538 with 73 spacers\\n[10:38:08] CRISPR2 NMQH01000008.1 15876 with 75 spacers\\n[10:38:08] CRISPR3 NMQH01000036.1 16251 with 4 spacers\\n[10:38:08] CRISPR4 NMQH01000142.1 9109 with 68 spacers\\n[10:38:08] CRISPR5 NMQH01000291.1 17104 with 87 spacers\\n[10:38:08] Found 5 CRISPRs\\n[10:38:08] Predicting coding sequences\\n[10:38:08] Contigs total 5356884 bp, so using single mode\\n[10:38:08] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQH01\\\\/\\\\/Fischerella_NMQH01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:38:13] Excluding CDS which overlaps existing RNA (rRNA) at NMQH01000004.1:2480..2659 on - strand\\n[10:38:13] Excluding CDS which overlaps existing RNA (repeat_region) at NMQH01000008.1:17360..17617 on + strand\\n[10:38:13] Excluding CDS which overlaps existing RNA (tRNA) at NMQH01000040.1:7491..7619 on - strand\\n[10:38:13] Excluding CDS which overlaps existing RNA (tRNA) at NMQH01000103.1:5875..6546 on - strand\\n[10:38:14] Excluding CDS which overlaps existing RNA (tRNA) at NMQH01000172.1:2550..3989 on + strand\\n[10:38:15] Found 4530 CDS\\n[10:38:15] Connecting features back to sequences\\n[10:38:15] Not using genus-specific database. Try --usegenus to enable it.\\n[10:38:15] Annotating CDS, please be patient.\\n[10:38:15] Will use 4 CPUs for similarity searching.\\n[10:38:16] There are still 4530 unannotated CDS left (started with 4530)\\n[10:38:16] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:38:16] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQH01\\\\/\\\\/Fischerella_NMQH01\\\\.IS\\\\.tmp\\\\.3794\\\\.faa | parallel --gnu --plain -j 4 --block 178597 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQH01\\\\/\\\\/Fischerella_NMQH01\\\\.IS\\\\.tmp\\\\.3794\\\\.blast 2> /dev/null\\n[10:38:36] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQH01\\\\/\\\\/Fischerella_NMQH01\\\\.IS\\\\.tmp\\\\.3794\\\\.faa | parallel --gnu --plain -j 4 --block 178597 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQH01\\\\/\\\\/Fischerella_NMQH01\\\\.IS\\\\.tmp\\\\.3794\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:38:36] This is prokka 1.14.6\\n[10:38:36] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:38:36] Homepage is https://github.com/tseemann/prokka\\n[10:38:36] Local time is Mon Feb 21 10:38:36 2022\\n[10:38:36] You are tiagoferreiraleao\\n[10:38:36] Operating system is darwin\\n[10:38:36] You have BioPerl 1.007002\\n[10:38:37] System has 4 cores.\\n[10:38:37] Option --cpu asked for 12 cores, but system only has 4\\n[10:38:37] Will use maximum of 4 cores.\\n[10:38:37] Annotating as >>> Bacteria <<<\\n[10:38:37] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJTC01.fasta' contents.\\n[10:38:37] Setting --locustag AFMFHHLJ from MD5 af6f115391bae7d678fb2c9411b7dba5\\n[10:38:37] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJTC01/\\n[10:38:37] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTC01\\\\/\\n[10:38:37] Using filename prefix: Nostoc_JACJTC01.XXX\\n[10:38:37] Setting HMMER_NCPU=1\\n[10:38:37] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJTC01//Nostoc_JACJTC01.log\\n[10:38:37] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJTC01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJTC01/ --prefix Nostoc_JACJTC01 --cpus 12\\n[10:38:37] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:38:37] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:38:37] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:38:37] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:38:37] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:38:37] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:38:37] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:38:37] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:38:37] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:38:37] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:38:37] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:38:37] Looking for 'egrep' - found /usr/bin/egrep\\n[10:38:37] Looking for 'find' - found /usr/bin/find\\n[10:38:37] Looking for 'grep' - found /usr/bin/grep\\n[10:38:37] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:38:37] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:38:37] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:38:37] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:38:37] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:38:37] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:38:37] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:38:37] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:38:37] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:38:37] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:38:37] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:38:37] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:38:37] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:38:37] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:38:37] Looking for 'sed' - found /usr/bin/sed\\n[10:38:37] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:38:38] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:38:38] Using genetic code table 11.\\n[10:38:38] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJTC01.fasta\\n[10:38:38] Wrote 77 contigs totalling 8954766 bp.\\n[10:38:38] Predicting tRNAs and tmRNAs\\n[10:38:38] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTC01\\\\/\\\\/Nostoc_JACJTC01\\\\.fna\\n[10:38:41] 1 tRNA-Val c[349955,350028] 35 (gac)\\n[10:38:41] 2 tRNA-Leu [414057,414137] 35 (gag)\\n[10:38:41] 1 tRNA-Gln c[397911,397984] 34 (ttg)\\n[10:38:41] 2 tRNA-Leu c[402738,402822] 35 (caa)\\n[10:38:41] 3 tRNA-Ala [458363,458435] 34 (cgc)\\n[10:38:41] 1 tRNA-Trp c[161982,162055] 34 (cca)\\n[10:38:41] 2 tRNA-Thr c[202678,202749] 33 (cgt)\\n[10:38:41] 1 tRNA-Val [82933,83005] 34 (tac)\\n[10:38:41] 2 tRNA-Arg c[100165,100239] 35 (tct)\\n[10:38:41] 1 tRNA-Arg c[32366,32438] 34 (ccg)\\n[10:38:41] 2 tRNA-Cys [269243,269314] 33 (gca)\\n[10:38:41] 3 tRNA-His [295359,295432] 34 (gtg)\\n[10:38:41] 1 tRNA-Phe [119111,119183] 34 (gaa)\\n[10:38:41] 2 tRNA-Ser [167242,167332] 36 (gga)\\n[10:38:41] 3 tRNA-Gly c[204881,204954] 34 (ccc)\\n[10:38:41] 1 tRNA-Met c[197044,197118] 35 (cat)\\n[10:38:41] 1 tRNA-Leu c[44396,44470] 35 (caa)\\n[10:38:41] 2 tRNA-Asn c[46047,46121] 33 (att)\\n[10:38:41] 1 tRNA-Leu c[222662,222743] 35 (cag)\\n[10:38:41] 1 tRNA-Ala [32038,32110] 34 (tgc)\\n[10:38:41] 2 tRNA-Thr c[276335,276406] 33 (ggt)\\n[10:38:41] 3 tRNA-Tyr c[276482,276566] 35 (gta)\\n[10:38:41] 4 tRNA-Ser [279017,279103] 36 (tga)\\n[10:38:41] 5 tRNA-Pro [287762,287835] 35 (tgg)\\n[10:38:41] 1 tRNA-Gly c[273308,273378] 33 (tcc)\\n[10:38:41] 1 tRNA-Lys [157626,157698] 33 (ctt)\\n[10:38:41] 2 tRNA-Asp [192286,192359] 35 (gtc)\\n[10:38:41] 1 tRNA-Met [82711,82784] 33 (cat)\\n[10:38:41] 2 tRNA-Gly [89716,89787] 33 (gcc)\\n[10:38:41] 3 tRNA-Thr [210559,210631] 33 (tgt)\\n[10:38:41] 1 tRNA-Pro [190492,190565] 35 (cgg)\\n[10:38:41] 1 tRNA-Ser c[106427,106513] 36 (cga)\\n[10:38:41] 1 tRNA-Glu [15624,15697] 35 (ttc)\\n[10:38:41] 2 tRNA-Met c[81269,81345] 35 (cat)\\n[10:38:41] 1 tRNA-Asp [60670,60745] 36 (gtc)\\n[10:38:41] 2 tRNA-Glu [60749,60825] 35 (ctc)\\n[10:38:41] 3 tRNA-Cys [60829,60902] 35 (gca)\\n[10:38:41] 4 tRNA-Met [60909,60982] 32 (cat)\\n[10:38:41] tRNA [60985,61059] is a pseudo/wacky gene - skipping.\\n[10:38:41] 6 tRNA-Glu [61133,61207] 35 (ttc)\\n[10:38:41] 7 tRNA-Tyr [61218,61299] 35 (gta)\\n[10:38:41] 8 tRNA-Ser [61303,61389] 35 (gct)\\n[10:38:41] 9 tRNA-Arg [61391,61467] 36 (acg)\\n[10:38:41] 10 tRNA-Phe [61549,61624] 36 (gaa)\\n[10:38:41] 11 tRNA-Asn [61630,61704] 33 (gtt)\\n[10:38:41] 12 tRNA-Gln [61831,61905] 35 (ctg)\\n[10:38:41] 13 tRNA-Leu [62008,62084] 35 (caa)\\n[10:38:41] 14 tRNA-Leu [62091,62168] 35 (taa)\\n[10:38:41] 15 tRNA-Pro [62170,62243] 35 (tgg)\\n[10:38:41] 16 tRNA-Leu [62351,62427] 34 (gag)\\n[10:38:41] 17 tRNA-Leu [62539,62613] 35 (tag)\\n[10:38:41] 18 tRNA-Met [62623,62699] 34 (cat)\\n[10:38:41] 19 tRNA-Thr [62708,62780] 34 (tgt)\\n[10:38:41] 20 tRNA-Val [62961,63036] 34 (tac)\\n[10:38:41] 21 tRNA-Arg [63043,63118] 35 (tct)\\n[10:38:41] 22 tRNA-Ala [63209,63283] 35 (tgc)\\n[10:38:41] 23 tRNA-His [63289,63364] 35 (gtg)\\n[10:38:41] 24 tRNA-Ser [63370,63453] 34 (tga)\\n[10:38:41] 25 tRNA-Lys [63697,63767] 33 (ttt)\\n[10:38:41] 26 tRNA-Ser [69397,69490] 36 (gct)\\n[10:38:41] 1 tRNA-Asn [2580,2653] 34 (gtt)\\n[10:38:41] 2 tRNA-Leu c[105466,105549] 36 (tag)\\n[10:38:41] 1 tmRNA [123135,123522] 101,151 ANNIVKFARREALVAA*\\n[10:38:41] 1 tRNA-Arg c[85788,85861] 35 (acg)\\n[10:38:41] 1 tRNA-Lys [34716,34787] 33 (ttt)\\n[10:38:41] 1 tRNA-Ala c[18560,18632] 34 (ggc)\\n[10:38:41] 1 tRNA-Pro c[26396,26469] 35 (ggg)\\n[10:38:41] 1 tRNA-Ile c[32761,32836] 34 (tat)\\n[10:38:41] Found 67 tRNAs\\n[10:38:41] Predicting Ribosomal RNAs\\n[10:38:41] Running Barrnap with 4 threads\\n[10:38:42] 1 JACJTC010000076.1 7 16S ribosomal RNA\\n[10:38:42] 2 JACJTC010000077.1 268 23S ribosomal RNA\\n[10:38:42] Found 2 rRNAs\\n[10:38:42] Skipping ncRNA search, enable with --rfam if desired.\\n[10:38:42] Total of 68 tRNA + rRNA features\\n[10:38:42] Searching for CRISPR repeats\\n[10:38:43] CRISPR1 JACJTC010000001.1 136794 with 14 spacers\\n[10:38:43] CRISPR2 JACJTC010000001.1 400539 with 13 spacers\\n[10:38:43] CRISPR3 JACJTC010000003.1 250409 with 38 spacers\\n[10:38:43] CRISPR4 JACJTC010000004.1 12434 with 17 spacers\\n[10:38:43] CRISPR5 JACJTC010000005.1 85863 with 30 spacers\\n[10:38:43] CRISPR6 JACJTC010000006.1 204215 with 6 spacers\\n[10:38:43] CRISPR7 JACJTC010000008.1 322550 with 14 spacers\\n[10:38:43] CRISPR8 JACJTC010000011.1 278541 with 15 spacers\\n[10:38:43] CRISPR9 JACJTC010000018.1 11864 with 33 spacers\\n[10:38:43] CRISPR10 JACJTC010000021.1 67175 with 5 spacers\\n[10:38:43] CRISPR11 JACJTC010000021.1 67576 with 7 spacers\\n[10:38:43] CRISPR12 JACJTC010000024.1 69635 with 17 spacers\\n[10:38:43] CRISPR13 JACJTC010000045.1 16633 with 12 spacers\\n[10:38:43] CRISPR14 JACJTC010000048.1 26064 with 20 spacers\\n[10:38:43] CRISPR15 JACJTC010000059.1 2124 with 14 spacers\\n[10:38:43] Found 15 CRISPRs\\n[10:38:43] Predicting coding sequences\\n[10:38:43] Contigs total 8954766 bp, so using single mode\\n[10:38:43] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTC01\\\\/\\\\/Nostoc_JACJTC01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:38:50] Excluding CDS which overlaps existing RNA (repeat_region) at JACJTC010000001.1:135858..136817 on + strand\\n[10:38:50] Excluding CDS which overlaps existing RNA (repeat_region) at JACJTC010000001.1:400395..403418 on + strand\\n[10:38:52] Excluding CDS which overlaps existing RNA (repeat_region) at JACJTC010000004.1:12246..12461 on + strand\\n[10:38:52] Excluding CDS which overlaps existing RNA (repeat_region) at JACJTC010000004.1:12458..12595 on + strand\\n[10:38:53] Excluding CDS which overlaps existing RNA (repeat_region) at JACJTC010000011.1:278539..278826 on + strand\\n[10:38:55] Excluding CDS which overlaps existing RNA (repeat_region) at JACJTC010000021.1:66987..67376 on + strand\\n[10:38:55] Excluding CDS which overlaps existing RNA (repeat_region) at JACJTC010000021.1:67713..67889 on + strand\\n[10:38:55] Excluding CDS which overlaps existing RNA (repeat_region) at JACJTC010000021.1:67974..68309 on + strand\\n[10:38:55] Excluding CDS which overlaps existing RNA (repeat_region) at JACJTC010000024.1:69345..69725 on + strand\\n[10:38:56] Excluding CDS which overlaps existing RNA (repeat_region) at JACJTC010000059.1:409..5892 on + strand\\n[10:38:56] Found 7195 CDS\\n[10:38:56] Connecting features back to sequences\\n[10:38:56] Not using genus-specific database. Try --usegenus to enable it.\\n[10:38:56] Annotating CDS, please be patient.\\n[10:38:56] Will use 4 CPUs for similarity searching.\\n[10:38:58] There are still 7195 unannotated CDS left (started with 7195)\\n[10:38:58] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:38:58] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTC01\\\\/\\\\/Nostoc_JACJTC01\\\\.IS\\\\.tmp\\\\.3906\\\\.faa | parallel --gnu --plain -j 4 --block 303095 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTC01\\\\/\\\\/Nostoc_JACJTC01\\\\.IS\\\\.tmp\\\\.3906\\\\.blast 2> /dev/null\\n[10:39:17] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTC01\\\\/\\\\/Nostoc_JACJTC01\\\\.IS\\\\.tmp\\\\.3906\\\\.faa | parallel --gnu --plain -j 4 --block 303095 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTC01\\\\/\\\\/Nostoc_JACJTC01\\\\.IS\\\\.tmp\\\\.3906\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:39:17] This is prokka 1.14.6\\n[10:39:17] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:39:17] Homepage is https://github.com/tseemann/prokka\\n[10:39:17] Local time is Mon Feb 21 10:39:17 2022\\n[10:39:17] You are tiagoferreiraleao\\n[10:39:17] Operating system is darwin\\n[10:39:17] You have BioPerl 1.007002\\n[10:39:17] System has 4 cores.\\n[10:39:17] Option --cpu asked for 12 cores, but system only has 4\\n[10:39:17] Will use maximum of 4 cores.\\n[10:39:17] Annotating as >>> Bacteria <<<\\n[10:39:17] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKF01.fasta' contents.\\n[10:39:17] Setting --locustag GEAKOJJH from MD5 0ea48331088458ec3b8750356c728031\\n[10:39:17] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKF01/\\n[10:39:17] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKF01\\\\/\\n[10:39:17] Using filename prefix: Nostoc_JABXKF01.XXX\\n[10:39:17] Setting HMMER_NCPU=1\\n[10:39:17] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKF01//Nostoc_JABXKF01.log\\n[10:39:17] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKF01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKF01/ --prefix Nostoc_JABXKF01 --cpus 12\\n[10:39:17] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:39:17] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:39:17] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:39:17] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:39:17] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:39:17] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:39:18] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:39:18] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:39:18] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:39:18] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:39:18] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:39:18] Looking for 'egrep' - found /usr/bin/egrep\\n[10:39:18] Looking for 'find' - found /usr/bin/find\\n[10:39:18] Looking for 'grep' - found /usr/bin/grep\\n[10:39:18] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:39:18] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:39:18] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:39:18] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:39:18] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:39:18] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:39:18] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:39:18] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:39:18] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:39:18] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:39:18] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:39:18] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:39:18] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:39:18] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:39:18] Looking for 'sed' - found /usr/bin/sed\\n[10:39:18] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:39:18] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:39:18] Using genetic code table 11.\\n[10:39:18] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKF01.fasta\\n[10:39:19] Wrote 190 contigs totalling 8613793 bp.\\n[10:39:19] Predicting tRNAs and tmRNAs\\n[10:39:19] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKF01\\\\/\\\\/Nostoc_JABXKF01\\\\.fna\\n[10:39:22] 1 tRNA-Val c[1981,2053] 34 (tac)\\n[10:39:22] 2 tRNA-Ser c[23188,23274] 36 (tga)\\n[10:39:22] 3 tRNA-Tyr [25573,25658] 35 (gta)\\n[10:39:22] 4 tRNA-Thr [25746,25817] 33 (ggt)\\n[10:39:22] 1 tRNA-Pro c[6025,6098] 35 (ggg)\\n[10:39:22] 1 tRNA-Asn c[23510,23581] 33 (gtt)\\n[10:39:22] 1 tRNA-Arg [28810,28885] 36 (cct)\\n[10:39:22] 2 tRNA-Lys c[37130,37201] 33 (ttt)\\n[10:39:22] 1 tRNA-Pro [64662,64735] 35 (tgg)\\n[10:39:22] 1 tRNA-Leu [33472,33556] 35 (caa)\\n[10:39:22] 2 tRNA-Gln [36522,36595] 34 (ttg)\\n[10:39:22] 1 tRNA-Arg [57354,57427] 35 (acg)\\n[10:39:22] 1 tRNA-Trp c[23084,23157] 34 (cca)\\n[10:39:22] 1 tRNA-Phe [1776,1848] 34 (gaa)\\n[10:39:22] 1 tRNA-Gly c[92553,92624] 33 (ccc)\\n[10:39:22] 1 tRNA-Leu c[26883,26965] 36 (gag)\\n[10:39:22] 1 tRNA-Lys c[18157,18228] 33 (ctt)\\n[10:39:22] 1 tRNA-Gly [13611,13682] 33 (gcc)\\n[10:39:22] 1 tRNA-Arg [1773,1845] 34 (ccg)\\n[10:39:22] 1 tRNA-Asp [24821,24903] 36 (gtc)\\n[10:39:22] 1 tRNA-Met [69096,69169] 33 (cat)\\n[10:39:22] 2 tRNA-Thr c[82753,82824] 33 (tgt)\\n[10:39:22] 1 tRNA-His [24452,24525] 34 (gtg)\\n[10:39:22] 1 tRNA-Thr [203550,203621] 33 (cgt)\\n[10:39:22] 1 tRNA-Val [1908,1981] 35 (gac)\\n[10:39:22] 1 tRNA-Ser [71662,71748] 36 (gga)\\n[10:39:22] 1 tRNA-Arg [179596,179670] 35 (tct)\\n[10:39:22] 1 tRNA-Ala [12868,12940] 34 (cgc)\\n[10:39:22] 1 tRNA-Ser [147306,147395] 36 (cga)\\n[10:39:22] 1 tRNA-Pro c[56753,56826] 35 (cgg)\\n[10:39:22] 1 tRNA-Leu c[27059,27141] 35 (tag)\\n[10:39:22] 1 tRNA-Gly [39160,39230] 33 (ccc)\\n[10:39:22] 1 tRNA-Gly [65158,65228] 33 (tcc)\\n[10:39:22] 1 tRNA-Leu [25486,25567] 35 (cag)\\n[10:39:22] 1 tRNA-Pyl [9870,9944] 33 (cta)\\n[10:39:22] 2 tRNA-Ser [11675,11747] 33 (tga)\\n[10:39:22] 1 tRNA-Asp c[21802,21875] 35 (gtc)\\n[10:39:22] 1 tRNA-Val [14634,14706] 33 (cac)\\n[10:39:22] 1 tRNA-Met c[46896,46972] 35 (cat)\\n[10:39:22] 1 tmRNA [9199,9587] 101,151 ANNIVNFARKDALVAA*\\n[10:39:22] 1 tRNA-Glu [51302,51375] 35 (ttc)\\n[10:39:22] 2 tRNA-Asp [58006,58081] 36 (gtc)\\n[10:39:22] 3 tRNA-Cys [58241,58314] 35 (gca)\\n[10:39:22] tRNA [58399,58473] is a pseudo/wacky gene - skipping.\\n[10:39:22] 5 tRNA-Glu [58526,58599] 35 (ttc)\\n[10:39:22] 6 tRNA-Ser [58711,58802] 36 (gct)\\n[10:39:22] 7 tRNA-Arg [58805,58881] 36 (acg)\\n[10:39:22] 8 tRNA-Phe [58902,58977] 36 (gaa)\\n[10:39:22] 9 tRNA-Asn [58983,59057] 33 (gtt)\\n[10:39:22] 10 tRNA-Gln [59062,59136] 35 (ctg)\\n[10:39:22] 11 tRNA-Gln [59139,59214] 34 (ttg)\\n[10:39:22] 12 tRNA-Leu [59330,59407] 35 (taa)\\n[10:39:22] 13 tRNA-Pro [59409,59481] 34 (tgg)\\n[10:39:22] 14 tRNA-Leu [59484,59559] 36 (cag)\\n[10:39:22] 15 tRNA-Met [59570,59641] 33 (cat)\\n[10:39:22] 16 tRNA-Thr [59654,59726] 34 (tgt)\\n[10:39:22] 17 tRNA-Gly [59731,59802] 33 (tcc)\\n[10:39:22] 18 tRNA-Val [59910,59985] 34 (tac)\\n[10:39:22] 19 tRNA-Arg [59992,60067] 35 (tct)\\n[10:39:22] 20 tRNA-Ala [60165,60239] 35 (tgc)\\n[10:39:22] 21 tRNA-His [60245,60320] 35 (gtg)\\n[10:39:22] 22 tRNA-Lys [60582,60653] 33 (ttt)\\n[10:39:22] 23 tRNA-Ser [64792,64883] 35 (gct)\\n[10:39:22] 24 tRNA-Met [105480,105554] 35 (cat)\\n[10:39:22] 25 tRNA-Ala [107993,108067] 35 (ggc)\\n[10:39:22] 26 tRNA-Ala c[110876,110950] 35 (tgc)\\n[10:39:22] Found 65 tRNAs\\n[10:39:22] Predicting Ribosomal RNAs\\n[10:39:22] Running Barrnap with 4 threads\\n[10:39:23] Found 0 rRNAs\\n[10:39:23] Skipping ncRNA search, enable with --rfam if desired.\\n[10:39:23] Total of 64 tRNA + rRNA features\\n[10:39:23] Searching for CRISPR repeats\\n[10:39:23] CRISPR1 JABXKF010000024.1 25625 with 20 spacers\\n[10:39:23] CRISPR2 JABXKF010000029.1 19111 with 14 spacers\\n[10:39:23] CRISPR3 JABXKF010000033.1 13458 with 19 spacers\\n[10:39:23] CRISPR4 JABXKF010000034.1 14475 with 11 spacers\\n[10:39:23] CRISPR5 JABXKF010000058.1 49180 with 23 spacers\\n[10:39:23] CRISPR6 JABXKF010000092.1 251 with 3 spacers\\n[10:39:23] CRISPR7 JABXKF010000141.1 54608 with 28 spacers\\n[10:39:24] Found 7 CRISPRs\\n[10:39:24] Predicting coding sequences\\n[10:39:24] Contigs total 8613793 bp, so using single mode\\n[10:39:24] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKF01\\\\/\\\\/Nostoc_JABXKF01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:39:31] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKF010000024.1:26206..26388 on + strand\\n[10:39:31] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKF010000024.1:26550..26774 on + strand\\n[10:39:31] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKF010000033.1:13483..13749 on - strand\\n[10:39:31] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKF010000033.1:13817..14080 on + strand\\n[10:39:31] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKF010000033.1:14785..15024 on - strand\\n[10:39:31] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKF010000034.1:14704..15120 on + strand\\n[10:39:32] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKF010000058.1:50311..50475 on - strand\\n[10:39:32] Excluding CDS which overlaps existing RNA (tRNA) at JABXKF010000081.1:20359..26070 on - strand\\n[10:39:34] Excluding CDS which overlaps existing RNA (tRNA) at JABXKF010000160.1:38502..39722 on - strand\\n[10:39:35] Excluding CDS which overlaps existing RNA (tRNA) at JABXKF010000184.1:58997..59230 on + strand\\n[10:39:35] Found 7461 CDS\\n[10:39:35] Connecting features back to sequences\\n[10:39:35] Not using genus-specific database. Try --usegenus to enable it.\\n[10:39:35] Annotating CDS, please be patient.\\n[10:39:35] Will use 4 CPUs for similarity searching.\\n[10:39:36] There are still 7461 unannotated CDS left (started with 7461)\\n[10:39:36] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:39:36] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKF01\\\\/\\\\/Nostoc_JABXKF01\\\\.IS\\\\.tmp\\\\.4018\\\\.faa | parallel --gnu --plain -j 4 --block 289816 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKF01\\\\/\\\\/Nostoc_JABXKF01\\\\.IS\\\\.tmp\\\\.4018\\\\.blast 2> /dev/null\\n[10:39:55] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKF01\\\\/\\\\/Nostoc_JABXKF01\\\\.IS\\\\.tmp\\\\.4018\\\\.faa | parallel --gnu --plain -j 4 --block 289816 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKF01\\\\/\\\\/Nostoc_JABXKF01\\\\.IS\\\\.tmp\\\\.4018\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:39:55] This is prokka 1.14.6\\n[10:39:55] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:39:55] Homepage is https://github.com/tseemann/prokka\\n[10:39:55] Local time is Mon Feb 21 10:39:55 2022\\n[10:39:55] You are tiagoferreiraleao\\n[10:39:55] Operating system is darwin\\n[10:39:55] You have BioPerl 1.007002\\n[10:39:55] System has 4 cores.\\n[10:39:55] Option --cpu asked for 12 cores, but system only has 4\\n[10:39:55] Will use maximum of 4 cores.\\n[10:39:55] Annotating as >>> Bacteria <<<\\n[10:39:55] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAGX01.fasta' contents.\\n[10:39:55] Setting --locustag EGLEOLHL from MD5 e05e8515a9c8a0cdb24149eaccdd9f91\\n[10:39:55] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAGX01/\\n[10:39:55] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGX01\\\\/\\n[10:39:55] Using filename prefix: Nostoc_LAGX01.XXX\\n[10:39:55] Setting HMMER_NCPU=1\\n[10:39:55] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAGX01//Nostoc_LAGX01.log\\n[10:39:55] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAGX01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAGX01/ --prefix Nostoc_LAGX01 --cpus 12\\n[10:39:55] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:39:55] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:39:55] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:39:55] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:39:55] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:39:55] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:39:56] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:39:56] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:39:56] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:39:56] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:39:56] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:39:56] Looking for 'egrep' - found /usr/bin/egrep\\n[10:39:56] Looking for 'find' - found /usr/bin/find\\n[10:39:56] Looking for 'grep' - found /usr/bin/grep\\n[10:39:56] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:39:56] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:39:56] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:39:56] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:39:56] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:39:56] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:39:56] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:39:56] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:39:56] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:39:56] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:39:56] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:39:56] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:39:56] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:39:56] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:39:56] Looking for 'sed' - found /usr/bin/sed\\n[10:39:56] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:39:56] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:39:56] Using genetic code table 11.\\n[10:39:56] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAGX01.fasta\\n[10:39:57] Wrote 187 contigs totalling 9033951 bp.\\n[10:39:57] Predicting tRNAs and tmRNAs\\n[10:39:57] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGX01\\\\/\\\\/Nostoc_LAGX01\\\\.fna\\n[10:40:00] 1 tRNA-Cys [96448,96519] 33 (gca)\\n[10:40:00] 2 tRNA-His [246376,246449] 34 (gtg)\\n[10:40:00] 1 tRNA-Lys [231560,231631] 33 (ttt)\\n[10:40:00] 1 tRNA-Gly c[230834,230904] 33 (tcc)\\n[10:40:00] 1 tRNA-Thr [161115,161186] 33 (cgt)\\n[10:40:00] 2 tRNA-Trp [206806,206879] 34 (cca)\\n[10:40:00] 1 tmRNA c[194049,194436] 101,151 ANNIVKFARREALVAA*\\n[10:40:00] 1 tRNA-Ala [53923,54009] 29 (cgc)\\n[10:40:00] 2 tRNA-Pro c[110702,110775] 35 (tgg)\\n[10:40:00] 3 tRNA-Ser c[119457,119543] 36 (tga)\\n[10:40:00] 4 tRNA-Tyr [121994,122078] 35 (gta)\\n[10:40:00] 5 tRNA-Thr [122154,122225] 33 (ggt)\\n[10:40:00] 1 tRNA-Asn [2942,3015] 34 (gtt)\\n[10:40:00] 2 tRNA-Leu c[102607,102690] 36 (tag)\\n[10:40:00] 1 tRNA-Ala c[83304,83376] 34 (tgc)\\n[10:40:00] 1 tRNA-Glu [22620,22693] 35 (ttc)\\n[10:40:00] 2 tRNA-Ala [192103,192175] 34 (ggc)\\n[10:40:00] 1 tRNA-Pro [111936,112009] 35 (cgg)\\n[10:40:00] 1 tRNA-Asp [95812,95885] 35 (gtc)\\n[10:40:00] 1 tRNA-Gln c[29366,29439] 34 (ttg)\\n[10:40:00] 2 tRNA-Leu c[34193,34277] 35 (caa)\\n[10:40:00] 3 tRNA-Ala [82979,83051] 34 (cgc)\\n[10:40:00] 1 tRNA-Ser [53936,54023] 35 (gga)\\n[10:40:00] 2 tRNA-Gly c[62168,62241] 34 (ccc)\\n[10:40:00] 1 tRNA-Arg [4897,4969] 34 (ccg)\\n[10:40:00] 2 tRNA-Pro c[73980,74053] 35 (ggg)\\n[10:40:00] 1 tRNA-Leu [10492,10573] 35 (cag)\\n[10:40:00] 1 tRNA-Ser c[15666,15759] 36 (gct)\\n[10:40:00] 2 tRNA-Lys c[21434,21504] 33 (ttt)\\n[10:40:00] 3 tRNA-Ser c[21763,21846] 34 (tga)\\n[10:40:00] 4 tRNA-His c[21851,21926] 35 (gtg)\\n[10:40:00] 5 tRNA-Ala c[21932,22006] 35 (tgc)\\n[10:40:00] 6 tRNA-Arg c[22098,22173] 35 (tct)\\n[10:40:00] 7 tRNA-Val c[22180,22255] 34 (tac)\\n[10:40:00] 8 tRNA-Thr c[22337,22409] 34 (tgt)\\n[10:40:00] 9 tRNA-Leu c[22704,22778] 35 (tag)\\n[10:40:00] 10 tRNA-Leu c[22890,22966] 34 (gag)\\n[10:40:00] 11 tRNA-Pro c[23074,23147] 35 (tgg)\\n[10:40:00] 12 tRNA-Leu c[23149,23226] 35 (taa)\\n[10:40:00] 13 tRNA-Leu c[23247,23324] 36 (caa)\\n[10:40:00] 14 tRNA-Gln c[23350,23425] 35 (ttg)\\n[10:40:00] 15 tRNA-Gln c[24300,24374] 35 (ctg)\\n[10:40:00] 16 tRNA-Asn c[24385,24459] 33 (gtt)\\n[10:40:00] 17 tRNA-Phe c[24465,24540] 36 (gaa)\\n[10:40:00] 18 tRNA-Arg c[24628,24703] 35 (acg)\\n[10:40:00] 19 tRNA-Ser c[25465,25551] 35 (gct)\\n[10:40:00] 20 tRNA-Tyr c[25555,25636] 35 (gta)\\n[10:40:00] 21 tRNA-Glu c[25647,25721] 35 (ttc)\\n[10:40:00] tRNA c[25809,25883] is a pseudo/wacky gene - skipping.\\n[10:40:00] 23 tRNA-Met c[25886,25959] 32 (cat)\\n[10:40:00] 24 tRNA-Glu c[26069,26145] 35 (ctc)\\n[10:40:00] 25 tRNA-Asp c[26149,26224] 36 (gtc)\\n[10:40:00] 1 tRNA-Ile c[44890,44965] 34 (tat)\\n[10:40:00] 2 tRNA-Met c[46571,46646] 34 (cat)\\n[10:40:00] 3 tRNA-Met c[46648,46723] 34 (cat)\\n[10:40:00] 1 tRNA-Ser [28823,28909] 36 (cga)\\n[10:40:00] 1 tRNA-Met [32963,33036] 33 (cat)\\n[10:40:00] 2 tRNA-Gly [39846,39917] 33 (gcc)\\n[10:40:00] 1 tRNA-Val c[81541,81613] 34 (tac)\\n[10:40:00] 1 tRNA-Leu [61423,61503] 35 (gag)\\n[10:40:00] 1 tRNA-Ile [32744,32819] 35 (gat)\\n[10:40:00] 2 tRNA-Ala [32903,32977] 34 (tgc)\\n[10:40:00] 1 tRNA-Arg c[5908,5982] 35 (tct)\\n[10:40:00] 1 tRNA-Leu c[30432,30506] 35 (caa)\\n[10:40:00] 2 tRNA-Thr c[31897,31971] 33 (agt)\\n[10:40:00] 1 tRNA-Asp [25381,25458] 38 (gtc)\\n[10:40:00] 2 tRNA-Glu [25461,25537] 35 (ctc)\\n[10:40:00] 3 tRNA-Cys [25542,25617] 34 (gca)\\n[10:40:00] 4 tRNA-Met [25623,25697] 33 (cat)\\n[10:40:00] 5 tRNA-Trp [25700,25772] 34 (cca)\\n[10:40:00] 6 tRNA-Pyl [25797,25868] 33 (cta)\\n[10:40:00] 7 tRNA-Glu [25899,25974] 34 (ttc)\\n[10:40:00] 8 tRNA-Tyr [25978,26061] 36 (gta)\\n[10:40:00] 9 tRNA-Ser [26065,26154] 33 (gct)\\n[10:40:00] 10 tRNA-Arg [26159,26234] 35 (acg)\\n[10:40:00] 11 tRNA-Phe [26239,26313] 35 (gaa)\\n[10:40:00] 12 tRNA-Asn [26322,26396] 33 (gtt)\\n[10:40:00] 13 tRNA-Gln [26402,26477] 35 (ttg)\\n[10:40:00] 14 tRNA-Leu [26485,26562] 35 (caa)\\n[10:40:00] 15 tRNA-Pro [26567,26643] 35 (tgg)\\n[10:40:00] 16 tRNA-Leu [26647,26721] 35 (tag)\\n[10:40:00] 17 tRNA-Leu [26741,26818] 35 (gag)\\n[10:40:00] 18 tRNA-Lys [26822,26898] 34 (ttt)\\n[10:40:00] 19 tRNA-Lys [26904,26978] 33 (ctt)\\n[10:40:00] 20 tRNA-Leu [26984,27059] 36 (taa)\\n[10:40:00] 21 tRNA-Cys [27069,27146] 34 (gca)\\n[10:40:00] 22 tRNA-Thr [27154,27226] 34 (tgt)\\n[10:40:00] 23 tRNA-Gly [27232,27302] 33 (tcc)\\n[10:40:00] 24 tRNA-Val [27308,27383] 34 (tac)\\n[10:40:00] 25 tRNA-Arg [27390,27464] 35 (tct)\\n[10:40:00] 26 tRNA-Ala [27469,27541] 34 (tgc)\\n[10:40:00] 27 tRNA-His [27549,27625] 35 (gtg)\\n[10:40:00] 28 tRNA-Ser [27636,27720] 34 (tga)\\n[10:40:00] 1 tRNA-Met c[40602,40676] 35 (cat)\\n[10:40:00] 1 tRNA-Phe c[728,801] 34 (gaa)\\n[10:40:00] 1 tRNA-Lys c[21235,21307] 33 (ctt)\\n[10:40:00] 1 tRNA-Thr [2238,2310] 33 (tgt)\\n[10:40:00] 1 tRNA-Arg c[20015,20088] 35 (acg)\\n[10:40:00] 1 tRNA-Val [14062,14135] 35 (gac)\\n[10:40:00] 1 tRNA-Ile [1810,1886] 35 (gat)\\n[10:40:00] 2 tRNA-Ala [1941,2016] 34 (tgc)\\n[10:40:00] 1 tRNA-Ala c[2187,2262] 34 (tgc)\\n[10:40:00] 1 tRNA-Met [1575,1651] 35 (cat)\\n[10:40:00] Found 102 tRNAs\\n[10:40:00] Predicting Ribosomal RNAs\\n[10:40:00] Running Barrnap with 4 threads\\n[10:40:02] 1 LAGX01000054.1 31137 16S ribosomal RNA\\n[10:40:02] 2 LAGX01000054.1 33150 23S ribosomal RNA\\n[10:40:02] 3 LAGX01000155.1 1 16S ribosomal RNA\\n[10:40:02] 4 LAGX01000155.1 2530 23S ribosomal RNA (partial)\\n[10:40:02] 5 LAGX01000165.1 476 23S ribosomal RNA (partial)\\n[10:40:02] 6 LAGX01000170.1 2 23S ribosomal RNA (partial)\\n[10:40:02] 7 LAGX01000177.1 1 23S ribosomal RNA (partial)\\n[10:40:02] 8 LAGX01000177.1 1168 5S ribosomal RNA\\n[10:40:02] 9 LAGX01000187.1 309 23S ribosomal RNA (partial)\\n[10:40:02] Found 9 rRNAs\\n[10:40:02] Skipping ncRNA search, enable with --rfam if desired.\\n[10:40:02] Total of 110 tRNA + rRNA features\\n[10:40:02] Searching for CRISPR repeats\\n[10:40:02] CRISPR1 LAGX01000004.1 110459 with 24 spacers\\n[10:40:02] CRISPR2 LAGX01000005.1 169421 with 14 spacers\\n[10:40:02] CRISPR3 LAGX01000018.1 2845 with 18 spacers\\n[10:40:02] CRISPR4 LAGX01000019.1 63986 with 7 spacers\\n[10:40:02] CRISPR5 LAGX01000024.1 26479 with 15 spacers\\n[10:40:02] CRISPR6 LAGX01000029.1 67280 with 10 spacers\\n[10:40:02] CRISPR7 LAGX01000034.1 39657 with 17 spacers\\n[10:40:02] CRISPR8 LAGX01000041.1 29012 with 20 spacers\\n[10:40:02] CRISPR9 LAGX01000070.1 26334 with 28 spacers\\n[10:40:02] CRISPR10 LAGX01000070.1 39768 with 43 spacers\\n[10:40:03] CRISPR11 LAGX01000107.1 13974 with 7 spacers\\n[10:40:03] CRISPR12 LAGX01000117.1 94 with 22 spacers\\n[10:40:03] CRISPR13 LAGX01000142.1 5057 with 6 spacers\\n[10:40:03] Found 13 CRISPRs\\n[10:40:03] Predicting coding sequences\\n[10:40:03] Contigs total 9033951 bp, so using single mode\\n[10:40:03] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGX01\\\\/\\\\/Nostoc_LAGX01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:40:10] Excluding CDS which overlaps existing RNA (tRNA) at LAGX01000001.1:246297..246440 on - strand\\n[10:40:11] Excluding CDS which overlaps existing RNA (repeat_region) at LAGX01000005.1:170137..170253 on + strand\\n[10:40:11] Excluding CDS which overlaps existing RNA (tRNA) at LAGX01000008.1:53104..54474 on + strand\\n[10:40:13] Excluding CDS which overlaps existing RNA (repeat_region) at LAGX01000029.1:67489..67641 on + strand\\n[10:40:15] Excluding CDS which overlaps existing RNA (repeat_region) at LAGX01000070.1:39809..39973 on - strand\\n[10:40:16] Excluding CDS which overlaps existing RNA (repeat_region) at LAGX01000107.1:14002..14127 on - strand\\n[10:40:16] Excluding CDS which overlaps existing RNA (repeat_region) at LAGX01000117.1:1548..1826 on - strand\\n[10:40:16] Excluding CDS which overlaps existing RNA (repeat_region) at LAGX01000142.1:5449..6408 on - strand\\n[10:40:16] Excluding CDS which overlaps existing RNA (rRNA) at LAGX01000155.1:404..532 on + strand\\n[10:40:16] Excluding CDS which overlaps existing RNA (tRNA) at LAGX01000155.1:1923..2111 on - strand\\n[10:40:16] Excluding CDS which overlaps existing RNA (rRNA) at LAGX01000155.1:2084..2560 on - strand\\n[10:40:16] Excluding CDS which overlaps existing RNA (rRNA) at LAGX01000165.1:753..878 on - strand\\n[10:40:16] Excluding CDS which overlaps existing RNA (tRNA) at LAGX01000170.1:1656..2309 on + strand\\n[10:40:16] Excluding CDS which overlaps existing RNA (rRNA) at LAGX01000177.1:492..950 on - strand\\n[10:40:16] Excluding CDS which overlaps existing RNA (rRNA) at LAGX01000187.1:441..899 on + strand\\n[10:40:16] Found 7620 CDS\\n[10:40:16] Connecting features back to sequences\\n[10:40:16] Not using genus-specific database. Try --usegenus to enable it.\\n[10:40:16] Annotating CDS, please be patient.\\n[10:40:16] Will use 4 CPUs for similarity searching.\\n[10:40:18] There are still 7620 unannotated CDS left (started with 7620)\\n[10:40:18] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:40:18] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGX01\\\\/\\\\/Nostoc_LAGX01\\\\.IS\\\\.tmp\\\\.4130\\\\.faa | parallel --gnu --plain -j 4 --block 304901 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGX01\\\\/\\\\/Nostoc_LAGX01\\\\.IS\\\\.tmp\\\\.4130\\\\.blast 2> /dev/null\\n[10:40:36] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGX01\\\\/\\\\/Nostoc_LAGX01\\\\.IS\\\\.tmp\\\\.4130\\\\.faa | parallel --gnu --plain -j 4 --block 304901 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGX01\\\\/\\\\/Nostoc_LAGX01\\\\.IS\\\\.tmp\\\\.4130\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:40:36] This is prokka 1.14.6\\n[10:40:36] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:40:36] Homepage is https://github.com/tseemann/prokka\\n[10:40:36] Local time is Mon Feb 21 10:40:36 2022\\n[10:40:36] You are tiagoferreiraleao\\n[10:40:36] Operating system is darwin\\n[10:40:36] You have BioPerl 1.007002\\n[10:40:37] System has 4 cores.\\n[10:40:37] Option --cpu asked for 12 cores, but system only has 4\\n[10:40:37] Will use maximum of 4 cores.\\n[10:40:37] Annotating as >>> Bacteria <<<\\n[10:40:37] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JAHHIM01.fasta' contents.\\n[10:40:37] Setting --locustag GCMPHOCE from MD5 0c6918ce3ae88a904d29d90529f0ab78\\n[10:40:37] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JAHHIM01/\\n[10:40:37] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JAHHIM01\\\\/\\n[10:40:37] Using filename prefix: Fischerella_JAHHIM01.XXX\\n[10:40:37] Setting HMMER_NCPU=1\\n[10:40:37] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JAHHIM01//Fischerella_JAHHIM01.log\\n[10:40:37] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JAHHIM01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_JAHHIM01/ --prefix Fischerella_JAHHIM01 --cpus 12\\n[10:40:37] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:40:37] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:40:37] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:40:37] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:40:37] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:40:37] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:40:37] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:40:37] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:40:37] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:40:37] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:40:37] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:40:37] Looking for 'egrep' - found /usr/bin/egrep\\n[10:40:37] Looking for 'find' - found /usr/bin/find\\n[10:40:37] Looking for 'grep' - found /usr/bin/grep\\n[10:40:37] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:40:37] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:40:37] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:40:37] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:40:37] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:40:37] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:40:37] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:40:37] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:40:37] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:40:37] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:40:38] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:40:38] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:40:38] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:40:38] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:40:38] Looking for 'sed' - found /usr/bin/sed\\n[10:40:38] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:40:38] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:40:38] Using genetic code table 11.\\n[10:40:38] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_JAHHIM01.fasta\\n[10:40:38] Wrote 73 contigs totalling 5970731 bp.\\n[10:40:38] Predicting tRNAs and tmRNAs\\n[10:40:38] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JAHHIM01\\\\/\\\\/Fischerella_JAHHIM01\\\\.fna\\n[10:40:41] 1 tRNA-Trp c[9608,9681] 34 (cca)\\n[10:40:41] 2 tRNA-Thr c[102508,102579] 33 (cgt)\\n[10:40:41] 3 tRNA-Leu [296033,296115] 35 (tag)\\n[10:40:41] 1 tRNA-Gly c[23396,23467] 33 (ccc)\\n[10:40:41] 2 tRNA-Thr c[39010,39081] 33 (tgt)\\n[10:40:41] 3 tRNA-Phe c[48695,48767] 34 (gaa)\\n[10:40:41] 4 tRNA-Pro c[83112,83185] 35 (tgg)\\n[10:40:41] 1 tRNA-Arg [28497,28569] 34 (ccg)\\n[10:40:41] 2 tRNA-Val c[82877,82950] 34 (tac)\\n[10:40:41] 3 tRNA-Pro c[104890,104963] 35 (ggg)\\n[10:40:41] 1 tRNA-Ser [121690,121783] 36 (gct)\\n[10:40:41] 2 tRNA-Asp [169080,169153] 35 (gtc)\\n[10:40:41] 1 tRNA-Val [139514,139587] 35 (gac)\\n[10:40:41] 1 tRNA-Ala [58650,58723] 34 (tgc)\\n[10:40:41] 2 tRNA-Ser [121033,121121] 36 (gga)\\n[10:40:41] 1 tRNA-Ala c[45487,45559] 34 (ggc)\\n[10:40:41] 2 tRNA-Met c[113621,113693] 34 (cat)\\n[10:40:41] 1 tRNA-Arg c[10036,10113] 36 (cct)\\n[10:40:41] 1 tRNA-Asn c[74295,74368] 34 (gtt)\\n[10:40:41] 1 tRNA-Lys [38715,38810] 33 (ttt)\\n[10:40:41] 2 tRNA-His [51526,51599] 34 (gtg)\\n[10:40:41] 1 tRNA-Cys c[32153,32224] 33 (gca)\\n[10:40:41] 1 tRNA-Gly c[12469,12539] 33 (tcc)\\n[10:40:41] 1 tRNA-Gln c[124827,124900] 34 (ttg)\\n[10:40:41] 2 tRNA-Leu c[155912,155996] 35 (caa)\\n[10:40:41] 3 tRNA-Leu [159766,159847] 35 (cag)\\n[10:40:41] 4 tRNA-Arg c[331066,331140] 35 (tct)\\n[10:40:41] 1 tRNA-Lys c[13059,13130] 33 (ttt)\\n[10:40:41] 1 tmRNA [17673,18061] 101,151 ANNIVKFARREALVAA*\\n[10:40:41] 1 tRNA-Ser c[41437,41523] 36 (cga)\\n[10:40:41] 2 tRNA-Glu c[66252,66325] 35 (ttc)\\n[10:40:41] 3 tRNA-Ser [75591,75677] 36 (tga)\\n[10:40:41] 4 tRNA-Ala [85301,85373] 34 (cgc)\\n[10:40:41] 5 tRNA-Gly c[179754,179825] 33 (gcc)\\n[10:40:41] 6 tRNA-Arg c[351179,351252] 35 (acg)\\n[10:40:41] 1 tRNA-Phe c[6018,6090] 35 (gaa)\\n[10:40:41] 2 tRNA-His c[7337,7411] 33 (atg)\\n[10:40:41] 1 tRNA-Met [97383,97458] 36 (cat)\\n[10:40:41] 2 tRNA-Leu c[194979,195059] 35 (gag)\\n[10:40:41] 3 tRNA-Pro c[331846,331919] 35 (cgg)\\n[10:40:41] 1 tRNA-Thr c[293885,293956] 33 (ggt)\\n[10:40:41] 2 tRNA-Tyr c[294044,294129] 35 (gta)\\n[10:40:41] 3 tRNA-Lys [346228,346300] 33 (ctt)\\n[10:40:41] Found 43 tRNAs\\n[10:40:41] Predicting Ribosomal RNAs\\n[10:40:41] Running Barrnap with 4 threads\\n[10:40:42] Found 0 rRNAs\\n[10:40:42] Skipping ncRNA search, enable with --rfam if desired.\\n[10:40:42] Total of 42 tRNA + rRNA features\\n[10:40:42] Searching for CRISPR repeats\\n[10:40:42] CRISPR1 JAHHIM010000010.1 20501 with 6 spacers\\n[10:40:42] CRISPR2 JAHHIM010000010.1 22803 with 3 spacers\\n[10:40:42] CRISPR3 JAHHIM010000010.1 34895 with 6 spacers\\n[10:40:42] CRISPR4 JAHHIM010000011.1 64834 with 15 spacers\\n[10:40:42] CRISPR5 JAHHIM010000018.1 49478 with 11 spacers\\n[10:40:42] CRISPR6 JAHHIM010000060.1 10605 with 16 spacers\\n[10:40:43] CRISPR7 JAHHIM010000068.1 1018 with 33 spacers\\n[10:40:43] CRISPR8 JAHHIM010000073.1 195334 with 11 spacers\\n[10:40:43] Found 8 CRISPRs\\n[10:40:43] Predicting coding sequences\\n[10:40:43] Contigs total 5970731 bp, so using single mode\\n[10:40:43] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JAHHIM01\\\\/\\\\/Fischerella_JAHHIM01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:40:48] Excluding CDS which overlaps existing RNA (tRNA) at JAHHIM010000001.1:295951..296196 on - strand\\n[10:40:48] Excluding CDS which overlaps existing RNA (tRNA) at JAHHIM010000002.1:83163..83294 on - strand\\n[10:40:48] Excluding CDS which overlaps existing RNA (tRNA) at JAHHIM010000006.1:58713..59348 on - strand\\n[10:40:50] Excluding CDS which overlaps existing RNA (tRNA) at JAHHIM010000036.1:329873..331105 on + strand\\n[10:40:51] Excluding CDS which overlaps existing RNA (repeat_region) at JAHHIM010000068.1:1784..2101 on - strand\\n[10:40:51] Found 5414 CDS\\n[10:40:51] Connecting features back to sequences\\n[10:40:51] Not using genus-specific database. Try --usegenus to enable it.\\n[10:40:51] Annotating CDS, please be patient.\\n[10:40:51] Will use 4 CPUs for similarity searching.\\n[10:40:53] There are still 5414 unannotated CDS left (started with 5414)\\n[10:40:53] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:40:53] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JAHHIM01\\\\/\\\\/Fischerella_JAHHIM01\\\\.IS\\\\.tmp\\\\.4242\\\\.faa | parallel --gnu --plain -j 4 --block 207294 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JAHHIM01\\\\/\\\\/Fischerella_JAHHIM01\\\\.IS\\\\.tmp\\\\.4242\\\\.blast 2> /dev/null\\n[10:41:07] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JAHHIM01\\\\/\\\\/Fischerella_JAHHIM01\\\\.IS\\\\.tmp\\\\.4242\\\\.faa | parallel --gnu --plain -j 4 --block 207294 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_JAHHIM01\\\\/\\\\/Fischerella_JAHHIM01\\\\.IS\\\\.tmp\\\\.4242\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:41:07] This is prokka 1.14.6\\n[10:41:07] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:41:07] Homepage is https://github.com/tseemann/prokka\\n[10:41:07] Local time is Mon Feb 21 10:41:07 2022\\n[10:41:07] You are tiagoferreiraleao\\n[10:41:07] Operating system is darwin\\n[10:41:07] You have BioPerl 1.007002\\n[10:41:07] System has 4 cores.\\n[10:41:07] Option --cpu asked for 12 cores, but system only has 4\\n[10:41:07] Will use maximum of 4 cores.\\n[10:41:07] Annotating as >>> Bacteria <<<\\n[10:41:07] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZT01.fasta' contents.\\n[10:41:07] Setting --locustag EFLMAJOF from MD5 ef56a38f9f7d9782e6b7ce441725bf7f\\n[10:41:07] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZT01/\\n[10:41:07] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZT01\\\\/\\n[10:41:07] Using filename prefix: Fischerella_NGZT01.XXX\\n[10:41:07] Setting HMMER_NCPU=1\\n[10:41:07] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZT01//Fischerella_NGZT01.log\\n[10:41:07] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZT01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZT01/ --prefix Fischerella_NGZT01 --cpus 12\\n[10:41:07] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:41:07] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:41:07] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:41:07] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:41:07] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:41:07] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:41:07] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:41:07] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:41:07] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:41:07] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:41:07] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:41:07] Looking for 'egrep' - found /usr/bin/egrep\\n[10:41:07] Looking for 'find' - found /usr/bin/find\\n[10:41:07] Looking for 'grep' - found /usr/bin/grep\\n[10:41:07] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:41:07] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:41:07] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:41:07] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:41:07] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:41:07] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:41:08] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:41:08] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:41:08] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:41:08] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:41:08] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:41:08] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:41:08] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:41:08] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:41:08] Looking for 'sed' - found /usr/bin/sed\\n[10:41:08] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:41:08] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:41:08] Using genetic code table 11.\\n[10:41:08] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZT01.fasta\\n[10:41:09] Wrote 73 contigs totalling 5343510 bp.\\n[10:41:09] Predicting tRNAs and tmRNAs\\n[10:41:09] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZT01\\\\/\\\\/Fischerella_NGZT01\\\\.fna\\n[10:41:11] 1 tRNA-Met c[45329,45401] 34 (cat)\\n[10:41:11] 1 tRNA-Met c[26841,26916] 36 (cat)\\n[10:41:11] 1 tRNA-Ser [4633,4719] 36 (cga)\\n[10:41:11] 1 tRNA-Leu c[31402,31484] 36 (gag)\\n[10:41:11] 1 tRNA-Gly [25647,25717] 33 (tcc)\\n[10:41:11] 2 tRNA-Cys [76532,76603] 33 (gca)\\n[10:41:11] 1 tRNA-Lys [34435,34508] 34 (ttt)\\n[10:41:11] 1 tRNA-Ser [84095,84183] 36 (gga)\\n[10:41:11] 1 tRNA-Ser [11289,11382] 36 (gct)\\n[10:41:11] 1 tRNA-Gln [97798,97868] 32 (ttg)\\n[10:41:11] 1 tRNA-Ser [122902,122988] 36 (tga)\\n[10:41:11] 2 tRNA-Glu [130927,130999] 35 (ttc)\\n[10:41:11] 3 tRNA-Arg [149069,149141] 34 (tcg)\\n[10:41:11] 4 tRNA-Asn [149812,149885] 34 (att)\\n[10:41:11] 1 tRNA-Arg c[114349,114422] 35 (acg)\\n[10:41:11] 2 tRNA-Ala c[119477,119550] 34 (ggc)\\n[10:41:11] 3 tRNA-Thr c[123460,123531] 33 (cgt)\\n[10:41:11] 1 tRNA-Arg [42434,42509] 36 (cct)\\n[10:41:11] 2 tRNA-Leu [129850,129933] 36 (tag)\\n[10:41:11] 1 tRNA-Ala [144816,144890] 35 (cgc)\\n[10:41:11] 1 tRNA-Asp [16038,16111] 35 (gtc)\\n[10:41:11] 1 tRNA-Arg c[28149,28222] 35 (ccg)\\n[10:41:11] 1 tRNA-Met c[11644,11720] 35 (cat)\\n[10:41:11] 2 tRNA-Val [14668,14741] 35 (gac)\\n[10:41:11] 1 tRNA-Pro [49461,49534] 35 (ggg)\\n[10:41:11] 2 tRNA-Phe c[115859,115931] 34 (gaa)\\n[10:41:11] 3 tRNA-Val [123330,123402] 34 (tac)\\n[10:41:11] 1 tRNA-Gly c[126268,126339] 33 (gcc)\\n[10:41:11] 2 tRNA-Asn [143151,143224] 34 (gtt)\\n[10:41:11] 1 tRNA-Leu [24154,24236] 35 (cag)\\n[10:41:11] 1 tRNA-His c[51527,51600] 34 (gtg)\\n[10:41:11] 2 tRNA-Arg [89965,90039] 35 (tct)\\n[10:41:11] 3 tRNA-Gly [447572,447643] 33 (ccc)\\n[10:41:11] 4 tRNA-Thr c[465789,465860] 33 (ggt)\\n[10:41:11] 5 tRNA-Tyr c[465924,466008] 35 (gta)\\n[10:41:11] 1 tRNA-Gln c[23527,23598] 33 (ttg)\\n[10:41:11] 2 tRNA-Leu c[59781,59865] 35 (caa)\\n[10:41:11] 1 tRNA-Pro c[105736,105810] 35 (tgg)\\n[10:41:11] 2 tRNA-Thr c[142156,142227] 33 (tgt)\\n[10:41:11] 1 tRNA-Trp c[98122,98195] 34 (cca)\\n[10:41:11] 1 tRNA-Pro [58940,59013] 35 (cgg)\\n[10:41:11] 2 tmRNA c[140520,140906] 100,150 ANNIVKFARKEALVAA*\\n[10:41:11] Found 42 tRNAs\\n[10:41:11] Predicting Ribosomal RNAs\\n[10:41:11] Running Barrnap with 4 threads\\n[10:41:12] 1 NGZT01000039.1 78908 16S ribosomal RNA\\n[10:41:12] 2 NGZT01000039.1 80747 23S ribosomal RNA\\n[10:41:12] 3 NGZT01000039.1 83686 5S ribosomal RNA\\n[10:41:12] Found 3 rRNAs\\n[10:41:12] Skipping ncRNA search, enable with --rfam if desired.\\n[10:41:12] Total of 44 tRNA + rRNA features\\n[10:41:12] Searching for CRISPR repeats\\n[10:41:12] CRISPR1 NGZT01000019.1 103 with 90 spacers\\n[10:41:12] CRISPR2 NGZT01000039.1 129427 with 3 spacers\\n[10:41:12] CRISPR3 NGZT01000046.1 19472 with 66 spacers\\n[10:41:12] CRISPR4 NGZT01000051.1 90 with 9 spacers\\n[10:41:12] CRISPR5 NGZT01000064.1 300475 with 78 spacers\\n[10:41:12] CRISPR6 NGZT01000068.1 9 with 8 spacers\\n[10:41:12] CRISPR7 NGZT01000068.1 2404 with 4 spacers\\n[10:41:12] CRISPR8 NGZT01000068.1 3552 with 40 spacers\\n[10:41:12] Found 8 CRISPRs\\n[10:41:12] Predicting coding sequences\\n[10:41:12] Contigs total 5343510 bp, so using single mode\\n[10:41:12] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZT01\\\\/\\\\/Fischerella_NGZT01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:41:18] Excluding CDS which overlaps existing RNA (tRNA) at NGZT01000038.1:114306..114977 on + strand\\n[10:41:18] Excluding CDS which overlaps existing RNA (rRNA) at NGZT01000039.1:81142..81321 on + strand\\n[10:41:18] Excluding CDS which overlaps existing RNA (rRNA) at NGZT01000039.1:83107..83283 on - strand\\n[10:41:19] Excluding CDS which overlaps existing RNA (repeat_region) at NGZT01000046.1:22471..22668 on - strand\\n[10:41:19] Excluding CDS which overlaps existing RNA (tRNA) at NGZT01000061.1:143025..144464 on + strand\\n[10:41:20] Found 4540 CDS\\n[10:41:20] Connecting features back to sequences\\n[10:41:20] Not using genus-specific database. Try --usegenus to enable it.\\n[10:41:20] Annotating CDS, please be patient.\\n[10:41:20] Will use 4 CPUs for similarity searching.\\n[10:41:21] There are still 4540 unannotated CDS left (started with 4540)\\n[10:41:21] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:41:21] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZT01\\\\/\\\\/Fischerella_NGZT01\\\\.IS\\\\.tmp\\\\.4355\\\\.faa | parallel --gnu --plain -j 4 --block 178579 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZT01\\\\/\\\\/Fischerella_NGZT01\\\\.IS\\\\.tmp\\\\.4355\\\\.blast 2> /dev/null\\n[10:41:37] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZT01\\\\/\\\\/Fischerella_NGZT01\\\\.IS\\\\.tmp\\\\.4355\\\\.faa | parallel --gnu --plain -j 4 --block 178579 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZT01\\\\/\\\\/Fischerella_NGZT01\\\\.IS\\\\.tmp\\\\.4355\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:41:38] This is prokka 1.14.6\\n[10:41:38] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:41:38] Homepage is https://github.com/tseemann/prokka\\n[10:41:38] Local time is Mon Feb 21 10:41:38 2022\\n[10:41:38] You are tiagoferreiraleao\\n[10:41:38] Operating system is darwin\\n[10:41:38] You have BioPerl 1.007002\\n[10:41:38] System has 4 cores.\\n[10:41:38] Option --cpu asked for 12 cores, but system only has 4\\n[10:41:38] Will use maximum of 4 cores.\\n[10:41:38] Annotating as >>> Bacteria <<<\\n[10:41:38] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACKZS01.fasta' contents.\\n[10:41:38] Setting --locustag IHNFGFMF from MD5 217f0f6f04171853ef5b2835350427bc\\n[10:41:38] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACKZS01/\\n[10:41:38] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZS01\\\\/\\n[10:41:38] Using filename prefix: Nostoc_JACKZS01.XXX\\n[10:41:38] Setting HMMER_NCPU=1\\n[10:41:38] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACKZS01//Nostoc_JACKZS01.log\\n[10:41:38] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACKZS01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACKZS01/ --prefix Nostoc_JACKZS01 --cpus 12\\n[10:41:38] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:41:38] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:41:38] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:41:38] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:41:38] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:41:38] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:41:38] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:41:38] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:41:38] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:41:38] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:41:38] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:41:38] Looking for 'egrep' - found /usr/bin/egrep\\n[10:41:38] Looking for 'find' - found /usr/bin/find\\n[10:41:38] Looking for 'grep' - found /usr/bin/grep\\n[10:41:38] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:41:38] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:41:38] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:41:38] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:41:38] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:41:38] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:41:38] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:41:38] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:41:39] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:41:39] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:41:39] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:41:39] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:41:39] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:41:39] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:41:39] Looking for 'sed' - found /usr/bin/sed\\n[10:41:39] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:41:39] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:41:39] Using genetic code table 11.\\n[10:41:39] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACKZS01.fasta\\n[10:41:39] Wrote 631 contigs totalling 8499034 bp.\\n[10:41:39] Predicting tRNAs and tmRNAs\\n[10:41:39] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZS01\\\\/\\\\/Nostoc_JACKZS01\\\\.fna\\n[10:41:42] 1 tRNA-Lys [36299,36370] 33 (ttt)\\n[10:41:42] 2 tRNA-Leu [86832,86915] 36 (tag)\\n[10:41:42] 1 tRNA-Arg [64351,64425] 35 (tct)\\n[10:41:42] 2 tRNA-Thr c[65932,66003] 33 (ggt)\\n[10:41:42] 3 tRNA-Tyr c[66091,66176] 35 (gta)\\n[10:41:42] 4 tRNA-Ser [68476,68562] 36 (tga)\\n[10:41:42] 1 tRNA-Phe c[36018,36091] 34 (gaa)\\n[10:41:42] 2 tRNA-Pyl c[37679,37751] 34 (cta)\\n[10:41:42] 1 tRNA-Ala [65832,65906] 35 (tgc)\\n[10:41:42] 1 tRNA-Gly c[46660,46731] 33 (gcc)\\n[10:41:42] 1 tRNA-Pro c[29191,29264] 35 (cgg)\\n[10:41:42] 1 tRNA-Ala [1311,1383] 34 (ggc)\\n[10:41:42] 1 tRNA-Pro [18555,18628] 35 (tgg)\\n[10:41:42] 1 tRNA-His [54271,54344] 34 (gtg)\\n[10:41:42] 1 tRNA-Ala c[16015,16087] 34 (cgc)\\n[10:41:42] 1 tRNA-Met [26748,26823] 34 (cat)\\n[10:41:42] 2 tRNA-Met [26823,26899] 35 (cat)\\n[10:41:42] 1 tRNA-Leu c[30452,30533] 35 (cag)\\n[10:41:42] 2 tRNA-Leu [34846,34930] 35 (caa)\\n[10:41:42] 3 tRNA-Gln [36749,36822] 34 (ttg)\\n[10:41:42] 1 tRNA-Val c[36742,36814] 34 (tac)\\n[10:41:42] 1 tRNA-Phe c[26771,26843] 34 (gaa)\\n[10:41:42] 1 tRNA-Cys c[32686,32756] 33 (gca)\\n[10:41:42] 1 tRNA-Ser c[12480,12571] 35 (gct)\\n[10:41:42] 2 tRNA-Lys c[16769,16840] 33 (ttt)\\n[10:41:42] 3 tRNA-His c[17208,17283] 35 (gtg)\\n[10:41:42] 4 tRNA-Ala c[17289,17363] 35 (tgc)\\n[10:41:42] 5 tRNA-Ala c[17373,17445] 34 (ggc)\\n[10:41:42] 6 tRNA-Arg c[17450,17525] 35 (tct)\\n[10:41:42] 7 tRNA-Gln c[18577,18652] 34 (ctg)\\n[10:41:42] 8 tRNA-Asn c[18658,18732] 33 (gtt)\\n[10:41:42] 9 tRNA-Phe c[18738,18813] 36 (gaa)\\n[10:41:42] 10 tRNA-Arg c[18832,18928] 36 (acg)\\n[10:41:42] 11 tRNA-Ser c[18931,19022] 36 (gct)\\n[10:41:42] 12 tRNA-Glu c[19155,19228] 35 (ttc)\\n[10:41:42] 13 tRNA-Trp c[19316,19390] 35 (cca)\\n[10:41:42] 14 tRNA-Met c[19393,19467] 33 (cat)\\n[10:41:42] 15 tRNA-Cys c[19477,19551] 35 (gca)\\n[10:41:42] 16 tRNA-Glu c[19555,19631] 35 (ctc)\\n[10:41:42] 17 tRNA-Asp c[19635,19710] 36 (gtc)\\n[10:41:42] 18 tRNA-Glu c[26411,26484] 35 (ttc)\\n[10:41:42] 1 tRNA-Trp [31524,31597] 34 (cca)\\n[10:41:42] 1 tRNA-Lys c[27411,27484] 33 (ctt)\\n[10:41:42] 1 tRNA-Asp [8119,8196] 38 (gtc)\\n[10:41:42] 2 tRNA-Glu [8199,8273] 35 (ctc)\\n[10:41:42] 3 tRNA-Cys [8281,8355] 33 (gca)\\n[10:41:42] 4 tRNA-Met [8361,8435] 33 (cat)\\n[10:41:42] 5 tRNA-Trp [8437,8509] 34 (cca)\\n[10:41:42] 6 tRNA-Tyr [8744,8826] 36 (gta)\\n[10:41:42] 7 tRNA-Ser [8830,8916] 33 (gct)\\n[10:41:42] 8 tRNA-Arg [8920,8995] 35 (acg)\\n[10:41:42] 9 tRNA-Phe [9000,9074] 35 (gaa)\\n[10:41:42] 10 tRNA-Asn [9083,9157] 33 (gtt)\\n[10:41:42] 11 tRNA-Gln [9245,9319] 35 (ttg)\\n[10:41:42] 12 tRNA-Pro [9324,9399] 36 (tgg)\\n[10:41:42] 13 tRNA-Lys [9404,9478] 34 (ttt)\\n[10:41:42] 14 tRNA-Leu [9481,9555] 35 (taa)\\n[10:41:42] 15 tRNA-Thr [9565,9637] 34 (tgt)\\n[10:41:42] 16 tRNA-Gly [9643,9714] 33 (tcc)\\n[10:41:42] 17 tRNA-Val [9719,9794] 34 (tac)\\n[10:41:42] 18 tRNA-Arg [9801,9876] 34 (tct)\\n[10:41:42] 19 tRNA-Ala [9879,9954] 36 (tgc)\\n[10:41:42] 20 tRNA-His [9960,10035] 35 (gtg)\\n[10:41:42] 21 tRNA-Ser [10042,10125] 34 (tga)\\n[10:41:42] 1 tRNA-Met [30912,30985] 33 (cat)\\n[10:41:42] 2 tRNA-Thr c[32369,32440] 33 (tgt)\\n[10:41:42] 1 tRNA-Ser [4203,4289] 36 (cga)\\n[10:41:42] 1 tRNA-Thr [375,446] 33 (cgt)\\n[10:41:42] 1 tRNA-Arg c[20273,20349] 36 (cct)\\n[10:41:42] 1 tRNA-Leu c[17989,18073] 36 (gag)\\n[10:41:42] 1 tRNA-Gly [6838,6908] 33 (tcc)\\n[10:41:42] 1 tRNA-Gly [3154,3225] 33 (ccc)\\n[10:41:42] 1 tRNA-Arg [1062,1135] 35 (acg)\\n[10:41:42] 1 tmRNA c[1595,1982] 101,151 ANNIVSFARKDALVAA*\\n[10:41:42] 1 tRNA-Asp c[2355,2428] 35 (gtc)\\n[10:41:42] 1 tRNA-Val c[12128,12201] 35 (gac)\\n[10:41:42] 1 tRNA-Pro c[15933,16006] 35 (ggg)\\n[10:41:42] 1 tRNA-Asn [2483,2554] 33 (gtt)\\n[10:41:42] 1 tRNA-Trp c[8063,8136] 36 (cca)\\n[10:41:42] 1 tRNA-Met [4159,4233] 35 (cat)\\n[10:41:42] 1 tRNA-Arg [1792,1864] 34 (ccg)\\n[10:41:42] 1 tRNA-Ile [1621,1696] 35 (gat)\\n[10:41:42] 2 tRNA-Ala [1778,1852] 34 (tgc)\\n[10:41:42] 1 tRNA-Ser [519,605] 36 (gga)\\n[10:41:42] Found 84 tRNAs\\n[10:41:42] Predicting Ribosomal RNAs\\n[10:41:42] Running Barrnap with 4 threads\\n[10:41:44] 1 JACKZS010000336.1 4 16S ribosomal RNA\\n[10:41:44] 2 JACKZS010000336.1 2100 23S ribosomal RNA\\n[10:41:44] 3 JACKZS010000522.1 63 16S ribosomal RNA (partial)\\n[10:41:44] Found 3 rRNAs\\n[10:41:44] Skipping ncRNA search, enable with --rfam if desired.\\n[10:41:44] Total of 86 tRNA + rRNA features\\n[10:41:44] Searching for CRISPR repeats\\n[10:41:44] CRISPR1 JACKZS010000003.1 93080 with 3 spacers\\n[10:41:45] CRISPR2 JACKZS010000107.1 384 with 11 spacers\\n[10:41:45] CRISPR3 JACKZS010000107.1 1437 with 6 spacers\\n[10:41:45] CRISPR4 JACKZS010000107.1 2567 with 3 spacers\\n[10:41:45] CRISPR5 JACKZS010000107.1 2776 with 8 spacers\\n[10:41:45] CRISPR6 JACKZS010000107.1 3369 with 10 spacers\\n[10:41:45] CRISPR7 JACKZS010000107.1 5316 with 9 spacers\\n[10:41:45] CRISPR8 JACKZS010000107.1 13879 with 3 spacers\\n[10:41:45] CRISPR9 JACKZS010000107.1 16331 with 6 spacers\\n[10:41:45] CRISPR10 JACKZS010000107.1 17431 with 3 spacers\\n[10:41:45] CRISPR11 JACKZS010000127.1 2668 with 7 spacers\\n[10:41:45] CRISPR12 JACKZS010000192.1 9754 with 12 spacers\\n[10:41:45] CRISPR13 JACKZS010000249.1 1977 with 6 spacers\\n[10:41:45] CRISPR14 JACKZS010000249.1 3873 with 10 spacers\\n[10:41:45] CRISPR15 JACKZS010000457.1 291 with 7 spacers\\n[10:41:45] Found 15 CRISPRs\\n[10:41:45] Predicting coding sequences\\n[10:41:45] Contigs total 8499034 bp, so using single mode\\n[10:41:45] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZS01\\\\/\\\\/Nostoc_JACKZS01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:41:52] Excluding CDS which overlaps existing RNA (tRNA) at JACKZS010000015.1:1248..1388 on - strand\\n[10:41:52] Excluding CDS which overlaps existing RNA (tRNA) at JACKZS010000048.1:17013..17249 on - strand\\n[10:41:52] Excluding CDS which overlaps existing RNA (tRNA) at JACKZS010000048.1:17298..17537 on + strand\\n[10:41:53] Excluding CDS which overlaps existing RNA (tRNA) at JACKZS010000064.1:30798..30992 on + strand\\n[10:41:53] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZS010000107.1:559..831 on - strand\\n[10:41:53] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZS010000107.1:1283..1444 on - strand\\n[10:41:53] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZS010000107.1:1623..1817 on + strand\\n[10:41:53] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZS010000107.1:3591..4022 on + strand\\n[10:41:53] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZS010000107.1:5852..5974 on + strand\\n[10:41:53] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZS010000107.1:16330..16713 on - strand\\n[10:41:54] Excluding CDS which overlaps existing RNA (tRNA) at JACKZS010000134.1:11988..12182 on + strand\\n[10:41:54] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZS010000249.1:1907..2092 on + strand\\n[10:41:54] Excluding CDS which overlaps existing RNA (tRNA) at JACKZS010000332.1:1806..2138 on + strand\\n[10:41:54] Excluding CDS which overlaps existing RNA (rRNA) at JACKZS010000336.1:1133..1447 on + strand\\n[10:41:55] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZS010000457.1:121..390 on + strand\\n[10:41:55] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZS010000457.1:613..810 on + strand\\n[10:41:55] Found 7236 CDS\\n[10:41:55] Connecting features back to sequences\\n[10:41:55] Not using genus-specific database. Try --usegenus to enable it.\\n[10:41:55] Annotating CDS, please be patient.\\n[10:41:55] Will use 4 CPUs for similarity searching.\\n[10:41:56] There are still 7236 unannotated CDS left (started with 7236)\\n[10:41:56] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:41:56] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZS01\\\\/\\\\/Nostoc_JACKZS01\\\\.IS\\\\.tmp\\\\.4467\\\\.faa | parallel --gnu --plain -j 4 --block 284801 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZS01\\\\/\\\\/Nostoc_JACKZS01\\\\.IS\\\\.tmp\\\\.4467\\\\.blast 2> /dev/null\\n[10:42:15] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZS01\\\\/\\\\/Nostoc_JACKZS01\\\\.IS\\\\.tmp\\\\.4467\\\\.faa | parallel --gnu --plain -j 4 --block 284801 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZS01\\\\/\\\\/Nostoc_JACKZS01\\\\.IS\\\\.tmp\\\\.4467\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:42:15] This is prokka 1.14.6\\n[10:42:15] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:42:15] Homepage is https://github.com/tseemann/prokka\\n[10:42:15] Local time is Mon Feb 21 10:42:15 2022\\n[10:42:15] You are tiagoferreiraleao\\n[10:42:15] Operating system is darwin\\n[10:42:15] You have BioPerl 1.007002\\n[10:42:15] System has 4 cores.\\n[10:42:15] Option --cpu asked for 12 cores, but system only has 4\\n[10:42:15] Will use maximum of 4 cores.\\n[10:42:15] Annotating as >>> Bacteria <<<\\n[10:42:15] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAJ01.fasta' contents.\\n[10:42:15] Setting --locustag AEEAKPJB from MD5 aeea493b5dae95ed87130190cd5c5c67\\n[10:42:15] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01/\\n[10:42:15] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAJ01\\\\/\\n[10:42:16] Using filename prefix: Fischerella_NHAJ01.XXX\\n[10:42:16] Setting HMMER_NCPU=1\\n[10:42:16] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.log\\n[10:42:16] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAJ01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01/ --prefix Fischerella_NHAJ01 --cpus 12\\n[10:42:16] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:42:16] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:42:16] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:42:16] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:42:16] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:42:16] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:42:16] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:42:16] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:42:16] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:42:16] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:42:16] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:42:16] Looking for 'egrep' - found /usr/bin/egrep\\n[10:42:16] Looking for 'find' - found /usr/bin/find\\n[10:42:16] Looking for 'grep' - found /usr/bin/grep\\n[10:42:16] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:42:16] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:42:16] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:42:16] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:42:16] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:42:16] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:42:16] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:42:16] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:42:16] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:42:16] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:42:16] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:42:16] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:42:16] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:42:16] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:42:16] Looking for 'sed' - found /usr/bin/sed\\n[10:42:16] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:42:17] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:42:17] Using genetic code table 11.\\n[10:42:17] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAJ01.fasta\\n[10:42:17] Wrote 270 contigs totalling 5672409 bp.\\n[10:42:17] Predicting tRNAs and tmRNAs\\n[10:42:17] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAJ01\\\\/\\\\/Fischerella_NHAJ01\\\\.fna\\n[10:42:19] 1 tRNA-Thr c[600,671] 33 (ggt)\\n[10:42:19] 1 tRNA-Val c[1167,1239] 34 (tac)\\n[10:42:19] 1 tRNA-Asn c[3933,4006] 34 (att)\\n[10:42:19] 2 tRNA-Arg c[4677,4749] 34 (tcg)\\n[10:42:19] 1 tRNA-Val [1368,1441] 35 (gac)\\n[10:42:19] 1 tRNA-Met c[8783,8859] 35 (cat)\\n[10:42:19] 1 tRNA-Trp c[22662,22735] 34 (cca)\\n[10:42:19] 1 tRNA-Pro c[4864,4937] 35 (cgg)\\n[10:42:19] 1 tRNA-Ala c[11027,11100] 34 (ggc)\\n[10:42:19] 1 tRNA-Gly [8159,8230] 33 (ccc)\\n[10:42:19] 1 tRNA-Arg [917,992] 36 (cct)\\n[10:42:19] 1 tRNA-Gly [13795,13865] 33 (tcc)\\n[10:42:19] 1 tRNA-Lys [5852,5925] 34 (ttt)\\n[10:42:19] 1 tRNA-Asn c[28546,28619] 34 (gtt)\\n[10:42:19] 1 tRNA-Ala [32645,32719] 35 (cgc)\\n[10:42:19] 1 tRNA-Ser c[5379,5467] 36 (gga)\\n[10:42:19] 1 tRNA-Asp c[3033,3106] 35 (gtc)\\n[10:42:19] 1 tmRNA [9250,9689] 100,150 ANNIVKFARKEALVAA*\\n[10:42:19] 1 tRNA-Thr [796,867] 33 (tgt)\\n[10:42:19] 1 tRNA-Leu c[49573,49655] 36 (gag)\\n[10:42:19] 1 tRNA-Gln [6490,6560] 33 (ttg)\\n[10:42:19] 2 tRNA-His [66686,66759] 34 (gtg)\\n[10:42:19] 1 tRNA-Thr [11788,11859] 33 (cgt)\\n[10:42:19] 1 tRNA-Arg c[37863,37936] 35 (ccg)\\n[10:42:19] 1 tRNA-Leu [158928,159009] 35 (tag)\\n[10:42:19] 1 tRNA-Pro [57425,57499] 37 (ggg)\\n[10:42:19] 1 tRNA-Met c[94231,94303] 34 (cat)\\n[10:42:19] 1 tRNA-Gln [36902,36973] 33 (ttg)\\n[10:42:19] 1 tRNA-Leu [25080,25164] 35 (caa)\\n[10:42:19] 1 tRNA-Met c[40592,40667] 36 (cat)\\n[10:42:19] Found 30 tRNAs\\n[10:42:19] Predicting Ribosomal RNAs\\n[10:42:19] Running Barrnap with 4 threads\\n[10:42:19] Found 0 rRNAs\\n[10:42:19] Skipping ncRNA search, enable with --rfam if desired.\\n[10:42:19] Total of 29 tRNA + rRNA features\\n[10:42:19] Searching for CRISPR repeats\\n[10:42:20] CRISPR1 NHAJ01000245.1 15454 with 5 spacers\\n[10:42:20] CRISPR2 NHAJ01000245.1 15808 with 3 spacers\\n[10:42:20] CRISPR3 NHAJ01000245.1 16897 with 4 spacers\\n[10:42:20] Found 3 CRISPRs\\n[10:42:20] Predicting coding sequences\\n[10:42:20] Contigs total 5672409 bp, so using single mode\\n[10:42:20] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAJ01\\\\/\\\\/Fischerella_NHAJ01\\\\.fna -c -m -g 11 -p single -f sco -q\\nError: saw too many regions of 'N''s in the sequence.\\n[10:42:20] Found 0 CDS\\n[10:42:20] Connecting features back to sequences\\n[10:42:20] Not using genus-specific database. Try --usegenus to enable it.\\n[10:42:20] Annotating CDS, please be patient.\\n[10:42:20] Will use 4 CPUs for similarity searching.\\n[10:42:20] Found 0 unique /gene codes.\\n[10:42:20] Fixed 0 colliding /gene names.\\n[10:42:20] Adding /locus_tag identifiers\\n[10:42:20] Assigned 30 locus_tags to CDS and RNA features.\\n[10:42:20] Writing outputs to /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//\\n[10:42:20] Generating annotation statistics file\\n[10:42:20] Generating Genbank and Sequin files\\n[10:42:20] Running: tbl2asn -V b -a r10k -l paired-ends -M n -N 1 -y 'Annotated using prokka 1.14.6 from https://github.com/tseemann/prokka' -Z \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAJ01\\\\/\\\\/Fischerella_NHAJ01\\\\.err -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAJ01\\\\/\\\\/Fischerella_NHAJ01\\\\.fsa 2> /dev/null\\n[tbl2asn-forever] Found Prokka input, correcting dates in .gbf|.sqn files.\\n[tbl2asn-forever] Correcting dates in /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.gbf\\n[tbl2asn-forever] Correcting dates in /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.sqn\\n[tbl2asn-forever] Dates changed from 01-JAN-2019 to 21-FEB-2022\\n[10:42:26] Deleting unwanted file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//errorsummary.val\\n[10:42:26] Deleting unwanted file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.dr\\n[10:42:26] Deleting unwanted file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.fixedproducts\\n[10:42:26] Deleting unwanted file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.ecn\\n[10:42:26] Deleting unwanted file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.val\\n[10:42:26] Repairing broken .GBK output that tbl2asn produces...\\n[10:42:26] Running: sed 's/COORDINATES: profile/COORDINATES:profile/' < \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAJ01\\\\/\\\\/Fischerella_NHAJ01\\\\.gbf > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAJ01\\\\/\\\\/Fischerella_NHAJ01\\\\.gbk\\n[10:42:26] Deleting unwanted file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.gbf\\n[10:42:26] Output files:\\n[10:42:26] /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.log\\n[10:42:26] /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.ffn\\n[10:42:26] /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.sprot.tmp.4579.faa\\n[10:42:26] /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.gbk\\n[10:42:26] /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.fsa\\n[10:42:26] /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.faa\\n[10:42:26] /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.txt\\n[10:42:26] /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.gbf-r\\n[10:42:26] /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.AMR.tmp.4579.faa\\n[10:42:26] /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.gff\\n[10:42:26] /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.HAMAP.hmm.tmp.4579.faa\\n[10:42:26] /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.tbl\\n[10:42:26] /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.sqn\\n[10:42:26] /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.tsv\\n[10:42:26] /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.IS.tmp.4579.faa\\n[10:42:26] /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.fna\\n[10:42:26] /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAJ01//Fischerella_NHAJ01.err\\n[10:42:26] Annotation finished successfully.\\n[10:42:26] Walltime used: 0.18 minutes\\n[10:42:26] If you use this result please cite the Prokka paper:\\n[10:42:26] Seemann T (2014) Prokka: rapid prokaryotic genome annotation. Bioinformatics. 30(14):2068-9.\\n[10:42:26] Type 'prokka --citation' for more details.\\n[10:42:26] Share and enjoy!\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:42:26] This is prokka 1.14.6\\n[10:42:26] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:42:26] Homepage is https://github.com/tseemann/prokka\\n[10:42:26] Local time is Mon Feb 21 10:42:26 2022\\n[10:42:26] You are tiagoferreiraleao\\n[10:42:26] Operating system is darwin\\n[10:42:26] You have BioPerl 1.007002\\n[10:42:26] System has 4 cores.\\n[10:42:26] Option --cpu asked for 12 cores, but system only has 4\\n[10:42:26] Will use maximum of 4 cores.\\n[10:42:26] Annotating as >>> Bacteria <<<\\n[10:42:26] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_AJLL01.fasta' contents.\\n[10:42:27] Setting --locustag PDCDDDMF from MD5 9dcddd6f202bcf4875c9d05138c7b462\\n[10:42:27] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_AJLL01/\\n[10:42:27] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLL01\\\\/\\n[10:42:27] Using filename prefix: Fischerella_AJLL01.XXX\\n[10:42:27] Setting HMMER_NCPU=1\\n[10:42:27] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_AJLL01//Fischerella_AJLL01.log\\n[10:42:27] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_AJLL01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_AJLL01/ --prefix Fischerella_AJLL01 --cpus 12\\n[10:42:27] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:42:27] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:42:27] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:42:27] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:42:27] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:42:27] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:42:27] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:42:27] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:42:27] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:42:27] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:42:27] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:42:27] Looking for 'egrep' - found /usr/bin/egrep\\n[10:42:27] Looking for 'find' - found /usr/bin/find\\n[10:42:27] Looking for 'grep' - found /usr/bin/grep\\n[10:42:27] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:42:27] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:42:27] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:42:27] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:42:27] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:42:27] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:42:27] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:42:27] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:42:27] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:42:27] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:42:27] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:42:27] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:42:27] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:42:27] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:42:27] Looking for 'sed' - found /usr/bin/sed\\n[10:42:27] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:42:28] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:42:28] Using genetic code table 11.\\n[10:42:28] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_AJLL01.fasta\\n[10:42:28] Wrote 141 contigs totalling 5438272 bp.\\n[10:42:28] Predicting tRNAs and tmRNAs\\n[10:42:28] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLL01\\\\/\\\\/Fischerella_AJLL01\\\\.fna\\n[10:42:30] 1 tRNA-Arg [220197,220271] 35 (tct)\\n[10:42:30] 2 tRNA-Met [245074,245149] 36 (cat)\\n[10:42:30] 1 tRNA-Val c[12844,12916] 34 (tac)\\n[10:42:30] 2 tRNA-Phe [20314,20386] 34 (gaa)\\n[10:42:30] 1 tRNA-Val c[44769,44842] 35 (gac)\\n[10:42:30] 2 tRNA-Met [47814,47890] 35 (cat)\\n[10:42:30] 1 tRNA-Trp c[6337,6410] 34 (cca)\\n[10:42:30] 1 tRNA-Arg c[5754,5827] 35 (ccg)\\n[10:42:30] 1 tRNA-Met c[32397,32469] 34 (cat)\\n[10:42:30] 1 tRNA-Ile [89,164] 35 (gat)\\n[10:42:30] 2 tRNA-Ala [247,321] 34 (tgc)\\n[10:42:30] 1 tRNA-Ser [4371,4457] 36 (cga)\\n[10:42:30] 1 tRNA-Ser c[70043,70129] 36 (tga)\\n[10:42:30] 1 tRNA-Pro [12726,12799] 35 (cgg)\\n[10:42:30] 1 tRNA-Gln [5610,5680] 32 (ttg)\\n[10:42:30] 2 tRNA-Ser c[165479,165572] 36 (gct)\\n[10:42:30] 1 tRNA-Lys c[47817,47890] 34 (ttt)\\n[10:42:30] 1 tRNA-Ser [58001,58089] 36 (gga)\\n[10:42:30] 1 tRNA-Arg [60989,61064] 36 (cct)\\n[10:42:30] 1 tRNA-Asp c[67682,67755] 35 (gtc)\\n[10:42:30] 1 tRNA-Gly [12710,12781] 33 (gcc)\\n[10:42:30] 1 tRNA-Pro [19056,19130] 35 (tgg)\\n[10:42:30] 1 tRNA-Asn c[42494,42567] 34 (gtt)\\n[10:42:30] 1 tRNA-Leu c[149258,149340] 35 (cag)\\n[10:42:30] 1 tRNA-Ala c[18784,18859] 35 (cgc)\\n[10:42:30] 1 tRNA-Leu [34246,34329] 36 (tag)\\n[10:42:30] 1 tRNA-Thr [56633,56704] 33 (tgt)\\n[10:42:30] 1 tRNA-His [26221,26294] 34 (gtg)\\n[10:42:30] 1 tRNA-Tyr [9134,9218] 35 (gta)\\n[10:42:30] 2 tRNA-Thr [9282,9353] 33 (ggt)\\n[10:42:30] 3 tRNA-Gly c[28071,28142] 33 (ccc)\\n[10:42:30] 1 tRNA-Glu [189036,189108] 35 (ttc)\\n[10:42:30] 1 tRNA-Arg [8492,8564] 34 (tcg)\\n[10:42:30] 2 tRNA-Asn [9236,9309] 34 (att)\\n[10:42:30] 1 tRNA-Leu c[139948,140030] 36 (gag)\\n[10:42:30] 1 tRNA-Gly [12118,12188] 33 (tcc)\\n[10:42:30] 2 tRNA-Cys [71487,71558] 33 (gca)\\n[10:42:30] 1 tRNA-Gln c[23333,23404] 33 (ttg)\\n[10:42:30] 2 tRNA-Leu c[56776,56860] 35 (caa)\\n[10:42:30] 1 tmRNA [14424,14810] 100,150 ANNIVKFARKEALVAA*\\n[10:42:30] 1 tRNA-Arg c[6394,6467] 35 (acg)\\n[10:42:30] 2 tRNA-Ala c[11359,11432] 34 (ggc)\\n[10:42:30] 3 tRNA-Thr c[13518,13589] 33 (cgt)\\n[10:42:30] 1 tRNA-Pro c[11024,11097] 35 (ggg)\\n[10:42:30] Found 44 tRNAs\\n[10:42:30] Predicting Ribosomal RNAs\\n[10:42:30] Running Barrnap with 4 threads\\n[10:42:30] 1 AJLL01000019.1 4 16S ribosomal RNA\\n[10:42:30] 2 AJLL01000042.1 159 23S ribosomal RNA\\n[10:42:30] 3 AJLL01000042.1 3098 5S ribosomal RNA\\n[10:42:30] Found 3 rRNAs\\n[10:42:30] Skipping ncRNA search, enable with --rfam if desired.\\n[10:42:30] Total of 46 tRNA + rRNA features\\n[10:42:30] Searching for CRISPR repeats\\n[10:42:31] CRISPR1 AJLL01000030.1 18640 with 39 spacers\\n[10:42:31] CRISPR2 AJLL01000031.1 38 with 17 spacers\\n[10:42:31] CRISPR3 AJLL01000037.1 76471 with 74 spacers\\n[10:42:31] CRISPR4 AJLL01000044.1 34 with 4 spacers\\n[10:42:31] CRISPR5 AJLL01000060.1 73847 with 57 spacers\\n[10:42:31] CRISPR6 AJLL01000066.1 33734 with 4 spacers\\n[10:42:31] CRISPR7 AJLL01000089.1 72303 with 46 spacers\\n[10:42:31] Found 7 CRISPRs\\n[10:42:31] Predicting coding sequences\\n[10:42:31] Contigs total 5438272 bp, so using single mode\\n[10:42:31] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLL01\\\\/\\\\/Fischerella_AJLL01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:42:36] Excluding CDS which overlaps existing RNA (tRNA) at AJLL01000022.1:4359..4487 on + strand\\n[10:42:36] Excluding CDS which overlaps existing RNA (rRNA) at AJLL01000042.1:554..733 on + strand\\n[10:42:36] Excluding CDS which overlaps existing RNA (tRNA) at AJLL01000046.1:67717..68112 on + strand\\n[10:42:37] Excluding CDS which overlaps existing RNA (tRNA) at AJLL01000058.1:41254..42693 on - strand\\n[10:42:38] Excluding CDS which overlaps existing RNA (tRNA) at AJLL01000103.1:6351..7022 on + strand\\n[10:42:38] Found 4595 CDS\\n[10:42:38] Connecting features back to sequences\\n[10:42:38] Not using genus-specific database. Try --usegenus to enable it.\\n[10:42:38] Annotating CDS, please be patient.\\n[10:42:38] Will use 4 CPUs for similarity searching.\\n[10:42:39] There are still 4595 unannotated CDS left (started with 4595)\\n[10:42:39] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:42:39] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLL01\\\\/\\\\/Fischerella_AJLL01\\\\.IS\\\\.tmp\\\\.4670\\\\.faa | parallel --gnu --plain -j 4 --block 182485 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLL01\\\\/\\\\/Fischerella_AJLL01\\\\.IS\\\\.tmp\\\\.4670\\\\.blast 2> /dev/null\\n[10:42:54] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLL01\\\\/\\\\/Fischerella_AJLL01\\\\.IS\\\\.tmp\\\\.4670\\\\.faa | parallel --gnu --plain -j 4 --block 182485 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_AJLL01\\\\/\\\\/Fischerella_AJLL01\\\\.IS\\\\.tmp\\\\.4670\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:42:54] This is prokka 1.14.6\\n[10:42:54] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:42:54] Homepage is https://github.com/tseemann/prokka\\n[10:42:54] Local time is Mon Feb 21 10:42:54 2022\\n[10:42:54] You are tiagoferreiraleao\\n[10:42:54] Operating system is darwin\\n[10:42:54] You have BioPerl 1.007002\\n[10:42:55] System has 4 cores.\\n[10:42:55] Option --cpu asked for 12 cores, but system only has 4\\n[10:42:55] Will use maximum of 4 cores.\\n[10:42:55] Annotating as >>> Bacteria <<<\\n[10:42:55] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Hapalosiphonaceae_QLKN01.fasta' contents.\\n[10:42:55] Setting --locustag NJPJBMMK from MD5 7393b6642d0d66bb4c00b34b32d3748d\\n[10:42:55] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Hapalosiphonaceae_QLKN01/\\n[10:42:55] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Hapalosiphonaceae_QLKN01\\\\/\\n[10:42:55] Using filename prefix: Hapalosiphonaceae_QLKN01.XXX\\n[10:42:55] Setting HMMER_NCPU=1\\n[10:42:55] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Hapalosiphonaceae_QLKN01//Hapalosiphonaceae_QLKN01.log\\n[10:42:55] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Hapalosiphonaceae_QLKN01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Hapalosiphonaceae_QLKN01/ --prefix Hapalosiphonaceae_QLKN01 --cpus 12\\n[10:42:55] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:42:55] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:42:55] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:42:55] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:42:55] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:42:55] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:42:55] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:42:55] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:42:55] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:42:55] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:42:55] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:42:55] Looking for 'egrep' - found /usr/bin/egrep\\n[10:42:55] Looking for 'find' - found /usr/bin/find\\n[10:42:55] Looking for 'grep' - found /usr/bin/grep\\n[10:42:55] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:42:55] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:42:55] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:42:55] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:42:55] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:42:55] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:42:55] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:42:55] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:42:56] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:42:56] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:42:56] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:42:56] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:42:56] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:42:56] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:42:56] Looking for 'sed' - found /usr/bin/sed\\n[10:42:56] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:42:56] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:42:56] Using genetic code table 11.\\n[10:42:56] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Hapalosiphonaceae_QLKN01.fasta\\n[10:42:56] Wrote 209 contigs totalling 7145071 bp.\\n[10:42:56] Predicting tRNAs and tmRNAs\\n[10:42:56] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Hapalosiphonaceae_QLKN01\\\\/\\\\/Hapalosiphonaceae_QLKN01\\\\.fna\\n[10:42:59] 1 tRNA-Met c[35350,35425] 36 (cat)\\n[10:42:59] 1 tRNA-Gly c[28900,28970] 33 (tcc)\\n[10:42:59] 1 tRNA-Val c[37623,37696] 35 (gac)\\n[10:42:59] 2 tRNA-Met [40731,40807] 35 (cat)\\n[10:42:59] 1 tRNA-Ser c[62681,62766] 35 (tga)\\n[10:42:59] 1 tRNA-Asn c[36789,36862] 34 (att)\\n[10:42:59] 2 tRNA-Arg c[37451,37523] 34 (acg)\\n[10:42:59] 1 tRNA-Pro c[55280,55353] 35 (cgg)\\n[10:42:59] 1 tRNA-Thr c[86146,86217] 33 (tgt)\\n[10:42:59] 1 tRNA-Met [9131,9204] 34 (cat)\\n[10:42:59] 1 tRNA-Leu [111115,111197] 35 (cag)\\n[10:42:59] 1 tRNA-Ala [31414,31486] 34 (cgc)\\n[10:42:59] 1 tRNA-Thr [27898,27969] 33 (cgt)\\n[10:42:59] 1 tRNA-Leu [18836,18919] 36 (tag)\\n[10:42:59] 1 tRNA-Gln c[49759,49830] 33 (ttg)\\n[10:42:59] 2 tRNA-Leu c[95034,95118] 35 (caa)\\n[10:42:59] 1 tRNA-Pro c[9589,9662] 35 (ggg)\\n[10:42:59] 1 tRNA-Gly [31338,31409] 33 (ccc)\\n[10:42:59] 2 tmRNA [81279,81665] 100,150 ANNIVKFARKEALVAA*\\n[10:42:59] 3 tRNA-Trp c[336781,336854] 34 (cca)\\n[10:42:59] 4 tRNA-Thr c[353609,353680] 33 (ggt)\\n[10:42:59] 5 tRNA-Val [533689,533761] 34 (tac)\\n[10:42:59] 1 tRNA-Lys [60346,60417] 33 (ttt)\\n[10:42:59] 1 tRNA-Glu c[59061,59133] 35 (ttc)\\n[10:42:59] 2 tRNA-Asp [67573,67646] 35 (gtc)\\n[10:42:59] 1 tRNA-Arg [41935,42008] 35 (acg)\\n[10:42:59] 2 tRNA-Ser c[51842,51935] 36 (gct)\\n[10:42:59] 1 tRNA-Lys c[169002,169074] 33 (ctt)\\n[10:42:59] 1 tRNA-Ser [4824,4912] 36 (gga)\\n[10:42:59] 1 tRNA-Pro [61821,61894] 35 (tgg)\\n[10:42:59] 2 tRNA-Phe [95755,95827] 34 (gaa)\\n[10:42:59] 3 tRNA-His c[279741,279814] 34 (gtg)\\n[10:42:59] 1 tRNA-Asn c[25581,25654] 34 (gtt)\\n[10:42:59] 1 tRNA-Trp c[37134,37208] 35 (cca)\\n[10:42:59] 2 tRNA-Cys [128308,128379] 33 (gca)\\n[10:42:59] 1 tRNA-Ala [2248,2320] 34 (ggc)\\n[10:42:59] 1 tRNA-Arg c[36566,36640] 35 (tct)\\n[10:42:59] 1 tRNA-Gly c[40620,40691] 33 (gcc)\\n[10:42:59] 1 tRNA-Arg c[12048,12124] 36 (cct)\\n[10:42:59] 1 tRNA-Ser c[7186,7272] 36 (cga)\\n[10:42:59] 1 tRNA-Ile [479,555] 35 (gat)\\n[10:42:59] 2 tRNA-Ala [623,698] 34 (tgc)\\n[10:42:59] 1 tRNA-Met [1045,1121] 35 (cat)\\n[10:42:59] Found 43 tRNAs\\n[10:42:59] Predicting Ribosomal RNAs\\n[10:42:59] Running Barrnap with 4 threads\\n[10:43:00] 1 QLKN01000121.1 2 16S ribosomal RNA (partial)\\n[10:43:00] 2 QLKN01000142.1 2 23S ribosomal RNA (partial)\\n[10:43:00] 3 QLKN01000197.1 716 5S ribosomal RNA\\n[10:43:00] Found 3 rRNAs\\n[10:43:00] Skipping ncRNA search, enable with --rfam if desired.\\n[10:43:00] Total of 45 tRNA + rRNA features\\n[10:43:00] Searching for CRISPR repeats\\n[10:43:01] CRISPR1 QLKN01000016.1 581 with 12 spacers\\n[10:43:01] CRISPR2 QLKN01000022.1 5872 with 26 spacers\\n[10:43:01] CRISPR3 QLKN01000022.1 20036 with 11 spacers\\n[10:43:01] CRISPR4 QLKN01000023.1 29980 with 21 spacers\\n[10:43:01] CRISPR5 QLKN01000040.1 2555 with 5 spacers\\n[10:43:01] CRISPR6 QLKN01000063.1 49593 with 22 spacers\\n[10:43:01] CRISPR7 QLKN01000073.1 28790 with 17 spacers\\n[10:43:01] CRISPR8 QLKN01000073.1 34765 with 24 spacers\\n[10:43:01] CRISPR9 QLKN01000099.1 27170 with 9 spacers\\n[10:43:01] CRISPR10 QLKN01000104.1 26470 with 8 spacers\\n[10:43:01] CRISPR11 QLKN01000163.1 31 with 8 spacers\\n[10:43:01] Found 11 CRISPRs\\n[10:43:01] Predicting coding sequences\\n[10:43:01] Contigs total 7145071 bp, so using single mode\\n[10:43:01] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Hapalosiphonaceae_QLKN01\\\\/\\\\/Hapalosiphonaceae_QLKN01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:43:06] Excluding CDS which overlaps existing RNA (tRNA) at QLKN01000013.1:37422..37586 on - strand\\n[10:43:07] Excluding CDS which overlaps existing RNA (repeat_region) at QLKN01000022.1:20400..20531 on - strand\\n[10:43:07] Excluding CDS which overlaps existing RNA (repeat_region) at QLKN01000022.1:20615..20827 on - strand\\n[10:43:07] Excluding CDS which overlaps existing RNA (repeat_region) at QLKN01000023.1:30018..30245 on + strand\\n[10:43:07] Excluding CDS which overlaps existing RNA (repeat_region) at QLKN01000040.1:2599..2757 on - strand\\n[10:43:07] Excluding CDS which overlaps existing RNA (tRNA) at QLKN01000048.1:49721..50002 on - strand\\n[10:43:09] Excluding CDS which overlaps existing RNA (repeat_region) at QLKN01000073.1:34897..35040 on - strand\\n[10:43:09] Excluding CDS which overlaps existing RNA (repeat_region) at QLKN01000073.1:35098..35283 on - strand\\n[10:43:09] Excluding CDS which overlaps existing RNA (repeat_region) at QLKN01000099.1:27305..27442 on + strand\\n[10:43:10] Excluding CDS which overlaps existing RNA (rRNA) at QLKN01000142.1:446..907 on - strand\\n[10:43:10] Found 5949 CDS\\n[10:43:10] Connecting features back to sequences\\n[10:43:10] Not using genus-specific database. Try --usegenus to enable it.\\n[10:43:10] Annotating CDS, please be patient.\\n[10:43:10] Will use 4 CPUs for similarity searching.\\n[10:43:11] There are still 5949 unannotated CDS left (started with 5949)\\n[10:43:11] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:43:11] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Hapalosiphonaceae_QLKN01\\\\/\\\\/Hapalosiphonaceae_QLKN01\\\\.IS\\\\.tmp\\\\.4782\\\\.faa | parallel --gnu --plain -j 4 --block 236739 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Hapalosiphonaceae_QLKN01\\\\/\\\\/Hapalosiphonaceae_QLKN01\\\\.IS\\\\.tmp\\\\.4782\\\\.blast 2> /dev/null\\n[10:43:15] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Hapalosiphonaceae_QLKN01\\\\/\\\\/Hapalosiphonaceae_QLKN01\\\\.IS\\\\.tmp\\\\.4782\\\\.faa | parallel --gnu --plain -j 4 --block 236739 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Hapalosiphonaceae_QLKN01\\\\/\\\\/Hapalosiphonaceae_QLKN01\\\\.IS\\\\.tmp\\\\.4782\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:43:16] This is prokka 1.14.6\\n[10:43:16] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:43:16] Homepage is https://github.com/tseemann/prokka\\n[10:43:16] Local time is Mon Feb 21 10:43:16 2022\\n[10:43:16] You are tiagoferreiraleao\\n[10:43:16] Operating system is darwin\\n[10:43:16] You have BioPerl 1.007002\\n[10:43:16] System has 4 cores.\\n[10:43:16] Option --cpu asked for 12 cores, but system only has 4\\n[10:43:16] Will use maximum of 4 cores.\\n[10:43:16] Annotating as >>> Bacteria <<<\\n[10:43:16] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJSJ01.fasta' contents.\\n[10:43:16] Setting --locustag BLOJDBLO from MD5 b583db58b8457c359586645814b85600\\n[10:43:16] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJSJ01/\\n[10:43:16] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJSJ01\\\\/\\n[10:43:16] Using filename prefix: Nostoc_JACJSJ01.XXX\\n[10:43:16] Setting HMMER_NCPU=1\\n[10:43:16] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJSJ01//Nostoc_JACJSJ01.log\\n[10:43:16] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJSJ01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJSJ01/ --prefix Nostoc_JACJSJ01 --cpus 12\\n[10:43:16] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:43:16] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:43:16] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:43:16] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:43:16] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:43:16] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:43:16] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:43:16] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:43:16] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:43:16] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:43:16] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:43:16] Looking for 'egrep' - found /usr/bin/egrep\\n[10:43:16] Looking for 'find' - found /usr/bin/find\\n[10:43:16] Looking for 'grep' - found /usr/bin/grep\\n[10:43:16] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:43:16] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:43:16] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:43:16] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:43:16] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:43:16] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:43:16] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:43:16] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:43:17] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:43:17] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:43:17] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:43:17] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:43:17] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:43:17] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:43:17] Looking for 'sed' - found /usr/bin/sed\\n[10:43:17] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:43:17] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:43:17] Using genetic code table 11.\\n[10:43:17] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJSJ01.fasta\\n[10:43:17] Wrote 263 contigs totalling 9526178 bp.\\n[10:43:17] Predicting tRNAs and tmRNAs\\n[10:43:17] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJSJ01\\\\/\\\\/Nostoc_JACJSJ01\\\\.fna\\n[10:43:20] 1 tRNA-Val [104794,104867] 34 (cac)\\n[10:43:20] 2 tRNA-Ala c[126446,126518] 34 (cgc)\\n[10:43:20] 1 tRNA-Ala [7923,7995] 34 (tgc)\\n[10:43:20] 2 tRNA-Trp [121681,121774] 38 (cca)\\n[10:43:20] 1 tRNA-Gly [85282,85353] 33 (ccc)\\n[10:43:20] 1 tmRNA [139202,139589] 101,151 ANNIVKFARKDALVAA*\\n[10:43:20] 1 tRNA-Gly c[93920,93991] 33 (gcc)\\n[10:43:20] 1 tRNA-Lys c[16680,16753] 33 (ctt)\\n[10:43:20] 1 tRNA-Asn c[99717,99788] 33 (gtt)\\n[10:43:20] 1 tRNA-Val c[3388,3461] 35 (gac)\\n[10:43:20] 1 tRNA-His c[44614,44687] 34 (gtg)\\n[10:43:20] 1 tRNA-Pyl [73574,73646] 34 (cta)\\n[10:43:20] 2 tRNA-Leu [75045,75118] 34 (caa)\\n[10:43:20] 1 tRNA-Glu [16601,16672] 33 (ttc)\\n[10:43:20] 2 tRNA-Gly [16873,16944] 33 (tcc)\\n[10:43:20] 3 tRNA-Ser c[30360,30446] 36 (tga)\\n[10:43:20] 4 tRNA-Tyr [32700,32784] 35 (gta)\\n[10:43:20] 5 tRNA-Thr [32868,32939] 33 (ggt)\\n[10:43:20] 6 tRNA-Lys c[87271,87342] 33 (ttt)\\n[10:43:20] 1 tRNA-Leu [4019,4103] 35 (caa)\\n[10:43:20] 2 tRNA-Gln [4665,4738] 34 (ttg)\\n[10:43:20] 1 tRNA-Pro c[24711,24784] 35 (tgg)\\n[10:43:20] 1 tRNA-Trp c[40955,41028] 34 (cca)\\n[10:43:20] 1 tRNA-Met [57605,57679] 35 (cat)\\n[10:43:20] 1 tRNA-Ser [56176,56264] 36 (gga)\\n[10:43:20] 2 tRNA-Phe [57723,57796] 34 (gaa)\\n[10:43:20] 1 tRNA-Ala c[61411,61483] 34 (ggc)\\n[10:43:20] 1 tRNA-Ser c[11188,11279] 35 (gct)\\n[10:43:20] 2 tRNA-Lys c[15950,16020] 33 (ttt)\\n[10:43:20] 3 tRNA-Ser c[16152,16235] 34 (tga)\\n[10:43:20] 4 tRNA-His c[16241,16316] 35 (gtg)\\n[10:43:20] 5 tRNA-Ala c[16322,16396] 35 (tgc)\\n[10:43:20] 6 tRNA-Ala c[16406,16478] 34 (ggc)\\n[10:43:20] 7 tRNA-Arg c[16482,16557] 35 (tct)\\n[10:43:20] 8 tRNA-Val c[16564,16639] 34 (tac)\\n[10:43:20] 9 tRNA-Thr c[16820,16892] 34 (tgt)\\n[10:43:20] 10 tRNA-Met c[17067,17143] 34 (cat)\\n[10:43:20] 11 tRNA-Leu c[17153,17227] 35 (tag)\\n[10:43:20] 12 tRNA-Leu c[17340,17413] 34 (gag)\\n[10:43:20] 13 tRNA-Leu c[17417,17491] 35 (cag)\\n[10:43:20] 14 tRNA-Pro c[17495,17567] 34 (tgg)\\n[10:43:20] 15 tRNA-Leu c[17569,17646] 35 (taa)\\n[10:43:20] 16 tRNA-Leu c[17653,17729] 35 (caa)\\n[10:43:20] 17 tRNA-Gln c[17736,17811] 34 (ttg)\\n[10:43:20] 18 tRNA-Gln c[17815,17887] 34 (ctg)\\n[10:43:20] 19 tRNA-Asn c[18045,18119] 33 (gtt)\\n[10:43:20] 20 tRNA-Phe c[18125,18200] 36 (gaa)\\n[10:43:20] 21 tRNA-Arg c[18281,18357] 36 (acg)\\n[10:43:20] 22 tRNA-Ser c[18359,18450] 35 (gct)\\n[10:43:20] 23 tRNA-Tyr c[18454,18537] 37 (gta)\\n[10:43:20] 24 tRNA-Glu c[18589,18663] 35 (ttc)\\n[10:43:20] tRNA c[18751,18825] is a pseudo/wacky gene - skipping.\\n[10:43:20] 26 tRNA-Met c[18828,18901] 32 (cat)\\n[10:43:20] 27 tRNA-Cys c[18908,18981] 35 (gca)\\n[10:43:20] 28 tRNA-Glu c[18985,19061] 35 (ctc)\\n[10:43:20] 29 tRNA-Asp c[19065,19140] 36 (gtc)\\n[10:43:20] 1 tRNA-Arg c[19838,19912] 35 (tct)\\n[10:43:20] 1 tRNA-Glu c[48788,48861] 35 (ttc)\\n[10:43:20] 1 tRNA-Gly c[38180,38250] 33 (tcc)\\n[10:43:20] 1 tRNA-Arg c[36400,36472] 34 (ccg)\\n[10:43:20] 1 tRNA-Thr [29289,29360] 33 (cgt)\\n[10:43:20] 1 tRNA-Ser c[11642,11728] 36 (cga)\\n[10:43:20] 1 tRNA-Pro c[21353,21428] 36 (ggg)\\n[10:43:20] 1 tRNA-Arg c[36624,36697] 35 (acg)\\n[10:43:20] 1 tRNA-Met c[1992,2065] 33 (cat)\\n[10:43:20] 1 tRNA-Met [15148,15223] 34 (cat)\\n[10:43:20] 2 tRNA-Met [15225,15301] 35 (cat)\\n[10:43:20] 1 tRNA-Pro [16487,16560] 35 (cgg)\\n[10:43:20] 1 tRNA-Thr [31337,31408] 33 (tgt)\\n[10:43:20] 1 tRNA-Gln [423,498] 34 (ttg)\\n[10:43:20] 1 tRNA-Leu c[14449,14531] 35 (tag)\\n[10:43:20] 1 tRNA-Leu [24181,24261] 35 (gag)\\n[10:43:20] 1 tRNA-Val c[10315,10387] 34 (tac)\\n[10:43:20] 1 tRNA-Leu [5729,5810] 35 (cag)\\n[10:43:20] 1 tRNA-Leu c[170,238] 31 (caa)\\n[10:43:20] 1 tRNA-Asp [463,536] 35 (gtc)\\n[10:43:20] 1 tRNA-Cys [2644,2715] 33 (gca)\\n[10:43:20] 1 tRNA-Ile [1614,1689] 35 (gat)\\n[10:43:20] 2 tRNA-Ala [1767,1841] 34 (tgc)\\n[10:43:20] Found 78 tRNAs\\n[10:43:20] Predicting Ribosomal RNAs\\n[10:43:20] Running Barrnap with 4 threads\\n[10:43:22] 1 JACJSJ010000263.1 5 16S ribosomal RNA\\n[10:43:22] 2 JACJSJ010000263.1 2094 23S ribosomal RNA\\n[10:43:22] Found 2 rRNAs\\n[10:43:22] Skipping ncRNA search, enable with --rfam if desired.\\n[10:43:22] Total of 79 tRNA + rRNA features\\n[10:43:22] Searching for CRISPR repeats\\n[10:43:22] CRISPR1 JACJSJ010000001.1 126278 with 45 spacers\\n[10:43:22] CRISPR2 JACJSJ010000001.1 207753 with 16 spacers\\n[10:43:22] CRISPR3 JACJSJ010000003.1 125755 with 4 spacers\\n[10:43:23] CRISPR4 JACJSJ010000063.1 35896 with 3 spacers\\n[10:43:23] CRISPR5 JACJSJ010000081.1 30477 with 13 spacers\\n[10:43:23] CRISPR6 JACJSJ010000089.1 2307 with 24 spacers\\n[10:43:23] CRISPR7 JACJSJ010000092.1 16940 with 4 spacers\\n[10:43:23] CRISPR8 JACJSJ010000093.1 12278 with 10 spacers\\n[10:43:23] CRISPR9 JACJSJ010000095.1 9952 with 28 spacers\\n[10:43:23] CRISPR10 JACJSJ010000143.1 2588 with 19 spacers\\n[10:43:23] Found 10 CRISPRs\\n[10:43:23] Predicting coding sequences\\n[10:43:23] Contigs total 9526178 bp, so using single mode\\n[10:43:23] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJSJ01\\\\/\\\\/Nostoc_JACJSJ01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:43:31] Excluding CDS which overlaps existing RNA (repeat_region) at JACJSJ010000003.1:125719..126051 on - strand\\n[10:43:31] Excluding CDS which overlaps existing RNA (tRNA) at JACJSJ010000004.1:120934..122247 on - strand\\n[10:43:32] Excluding CDS which overlaps existing RNA (tRNA) at JACJSJ010000026.1:4547..4801 on - strand\\n[10:43:32] Excluding CDS which overlaps existing RNA (tRNA) at JACJSJ010000029.1:40807..41052 on - strand\\n[10:43:33] Excluding CDS which overlaps existing RNA (tRNA) at JACJSJ010000047.1:16040..16495 on + strand\\n[10:43:33] Excluding CDS which overlaps existing RNA (tRNA) at JACJSJ010000047.1:17292..17540 on - strand\\n[10:43:34] Excluding CDS which overlaps existing RNA (repeat_region) at JACJSJ010000081.1:29422..30477 on - strand\\n[10:43:35] Excluding CDS which overlaps existing RNA (rRNA) at JACJSJ010000263.1:425..634 on + strand\\n[10:43:35] Found 7831 CDS\\n[10:43:35] Connecting features back to sequences\\n[10:43:35] Not using genus-specific database. Try --usegenus to enable it.\\n[10:43:35] Annotating CDS, please be patient.\\n[10:43:35] Will use 4 CPUs for similarity searching.\\n[10:43:37] There are still 7831 unannotated CDS left (started with 7831)\\n[10:43:37] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:43:37] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJSJ01\\\\/\\\\/Nostoc_JACJSJ01\\\\.IS\\\\.tmp\\\\.4894\\\\.faa | parallel --gnu --plain -j 4 --block 323715 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJSJ01\\\\/\\\\/Nostoc_JACJSJ01\\\\.IS\\\\.tmp\\\\.4894\\\\.blast 2> /dev/null\\n[10:43:46] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJSJ01\\\\/\\\\/Nostoc_JACJSJ01\\\\.IS\\\\.tmp\\\\.4894\\\\.faa | parallel --gnu --plain -j 4 --block 323715 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJSJ01\\\\/\\\\/Nostoc_JACJSJ01\\\\.IS\\\\.tmp\\\\.4894\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:43:47] This is prokka 1.14.6\\n[10:43:47] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:43:47] Homepage is https://github.com/tseemann/prokka\\n[10:43:47] Local time is Mon Feb 21 10:43:47 2022\\n[10:43:47] You are tiagoferreiraleao\\n[10:43:47] Operating system is darwin\\n[10:43:47] You have BioPerl 1.007002\\n[10:43:47] System has 4 cores.\\n[10:43:47] Option --cpu asked for 12 cores, but system only has 4\\n[10:43:47] Will use maximum of 4 cores.\\n[10:43:47] Annotating as >>> Bacteria <<<\\n[10:43:47] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQC01.fasta' contents.\\n[10:43:47] Setting --locustag FLBJOEBE from MD5 f5b38ebe0002d5dcd10eef383f319ed8\\n[10:43:47] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQC01/\\n[10:43:47] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQC01\\\\/\\n[10:43:47] Using filename prefix: Fischerella_NMQC01.XXX\\n[10:43:47] Setting HMMER_NCPU=1\\n[10:43:47] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQC01//Fischerella_NMQC01.log\\n[10:43:47] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQC01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQC01/ --prefix Fischerella_NMQC01 --cpus 12\\n[10:43:47] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:43:47] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:43:47] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:43:47] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:43:47] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:43:47] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:43:47] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:43:47] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:43:47] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:43:47] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:43:47] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:43:47] Looking for 'egrep' - found /usr/bin/egrep\\n[10:43:47] Looking for 'find' - found /usr/bin/find\\n[10:43:47] Looking for 'grep' - found /usr/bin/grep\\n[10:43:47] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:43:47] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:43:47] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:43:47] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:43:47] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:43:47] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:43:47] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:43:47] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:43:48] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:43:48] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:43:48] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:43:48] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:43:48] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:43:48] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:43:48] Looking for 'sed' - found /usr/bin/sed\\n[10:43:48] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:43:48] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:43:48] Using genetic code table 11.\\n[10:43:48] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQC01.fasta\\n[10:43:49] Wrote 535 contigs totalling 5341442 bp.\\n[10:43:49] Predicting tRNAs and tmRNAs\\n[10:43:49] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQC01\\\\/\\\\/Fischerella_NMQC01\\\\.fna\\n[10:43:51] 1 tRNA-Glu c[17073,17145] 35 (ttc)\\n[10:43:51] 1 tRNA-Trp [16234,16307] 34 (cca)\\n[10:43:51] 1 tRNA-Ser c[3319,3407] 36 (gga)\\n[10:43:51] 1 tRNA-Arg c[12835,12908] 35 (acg)\\n[10:43:51] 1 tRNA-Val c[1166,1238] 34 (tac)\\n[10:43:51] 2 tRNA-Phe [8644,8716] 34 (gaa)\\n[10:43:51] 1 tRNA-Ser c[21048,21134] 36 (tga)\\n[10:43:51] 1 tRNA-Leu c[19184,19267] 36 (tag)\\n[10:43:51] 1 tRNA-Asp c[8337,8410] 35 (gtc)\\n[10:43:51] 1 tRNA-Lys [14279,14352] 34 (ttt)\\n[10:43:51] 1 tmRNA [14901,15287] 100,150 ANNIVKFARKEALVAA*\\n[10:43:51] 1 tRNA-Gly [24019,24090] 33 (ccc)\\n[10:43:51] 1 tRNA-Pro [1116,1189] 35 (cgg)\\n[10:43:51] 1 tRNA-Leu [13955,14037] 35 (cag)\\n[10:43:51] 1 tRNA-Asn c[19374,19447] 34 (gtt)\\n[10:43:51] 2 tRNA-Gly [35815,35886] 33 (gcc)\\n[10:43:51] 1 tRNA-Arg c[2634,2707] 35 (ccg)\\n[10:43:51] 1 tRNA-Tyr [3655,3739] 35 (gta)\\n[10:43:51] 2 tRNA-Thr [3803,3874] 33 (ggt)\\n[10:43:51] 1 tRNA-His [10998,11071] 34 (gtg)\\n[10:43:51] 1 tRNA-Thr [43860,43931] 33 (cgt)\\n[10:43:51] 2 tRNA-Ala [46520,46593] 34 (ggc)\\n[10:43:51] 1 tRNA-Met [7589,7664] 36 (cat)\\n[10:43:51] 1 tRNA-Ala [20947,21022] 35 (cgc)\\n[10:43:51] 1 tRNA-Cys c[64,135] 33 (gca)\\n[10:43:51] 1 tRNA-Arg c[17239,17313] 35 (tct)\\n[10:43:51] 1 tRNA-Pro [20645,20718] 35 (ggg)\\n[10:43:51] 1 tRNA-Gly [5869,5939] 33 (tcc)\\n[10:43:51] 1 tRNA-Met c[8628,8704] 35 (cat)\\n[10:43:51] 2 tRNA-Val [11649,11722] 35 (gac)\\n[10:43:51] 1 tRNA-Asn [648,721] 34 (att)\\n[10:43:51] 1 tRNA-Gln [7251,7321] 32 (ttg)\\n[10:43:51] 1 tRNA-Thr [2066,2137] 33 (tgt)\\n[10:43:51] 1 tRNA-Gln c[12283,12354] 33 (ttg)\\n[10:43:51] 1 tRNA-Leu c[6964,7046] 36 (gag)\\n[10:43:51] 1 tRNA-Ser c[2707,2800] 36 (gct)\\n[10:43:51] 1 tRNA-Met [8481,8553] 34 (cat)\\n[10:43:51] 1 tRNA-Ala c[162,236] 34 (tgc)\\n[10:43:51] 2 tRNA-Ile c[333,408] 35 (gat)\\n[10:43:51] 1 tRNA-Leu [181,265] 35 (caa)\\n[10:43:51] 1 tRNA-Ser c[618,704] 36 (cga)\\n[10:43:51] Found 41 tRNAs\\n[10:43:51] Predicting Ribosomal RNAs\\n[10:43:51] Running Barrnap with 4 threads\\n[10:43:52] 1 NMQC01000159.1 4 5S ribosomal RNA\\n[10:43:52] 2 NMQC01000159.1 233 23S ribosomal RNA (partial)\\n[10:43:52] 3 NMQC01000160.1 116 16S ribosomal RNA\\n[10:43:52] Found 3 rRNAs\\n[10:43:52] Skipping ncRNA search, enable with --rfam if desired.\\n[10:43:52] Total of 43 tRNA + rRNA features\\n[10:43:52] Searching for CRISPR repeats\\n[10:43:52] CRISPR1 NMQC01000015.1 14675 with 9 spacers\\n[10:43:53] CRISPR2 NMQC01000052.1 59 with 15 spacers\\n[10:43:53] CRISPR3 NMQC01000053.1 6260 with 8 spacers\\n[10:43:53] CRISPR4 NMQC01000131.1 31992 with 3 spacers\\n[10:43:53] CRISPR5 NMQC01000360.1 52 with 7 spacers\\n[10:43:53] CRISPR6 NMQC01000405.1 2207 with 7 spacers\\n[10:43:53] Found 6 CRISPRs\\n[10:43:53] Predicting coding sequences\\n[10:43:53] Contigs total 5341442 bp, so using single mode\\n[10:43:53] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQC01\\\\/\\\\/Fischerella_NMQC01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:43:57] Excluding CDS which overlaps existing RNA (tRNA) at NMQC01000038.1:12792..13463 on + strand\\n[10:43:58] Excluding CDS which overlaps existing RNA (tRNA) at NMQC01000109.1:18134..19573 on - strand\\n[10:43:58] Excluding CDS which overlaps existing RNA (repeat_region) at NMQC01000131.1:31540..32007 on + strand\\n[10:43:58] Excluding CDS which overlaps existing RNA (rRNA) at NMQC01000159.1:2480..2695 on - strand\\n[10:43:59] Excluding CDS which overlaps existing RNA (tRNA) at NMQC01000508.1:588..716 on - strand\\n[10:43:59] Found 4544 CDS\\n[10:43:59] Connecting features back to sequences\\n[10:43:59] Not using genus-specific database. Try --usegenus to enable it.\\n[10:43:59] Annotating CDS, please be patient.\\n[10:43:59] Will use 4 CPUs for similarity searching.\\n[10:44:00] There are still 4544 unannotated CDS left (started with 4544)\\n[10:44:00] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:44:00] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQC01\\\\/\\\\/Fischerella_NMQC01\\\\.IS\\\\.tmp\\\\.5008\\\\.faa | parallel --gnu --plain -j 4 --block 175565 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQC01\\\\/\\\\/Fischerella_NMQC01\\\\.IS\\\\.tmp\\\\.5008\\\\.blast 2> /dev/null\\n[10:44:16] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQC01\\\\/\\\\/Fischerella_NMQC01\\\\.IS\\\\.tmp\\\\.5008\\\\.faa | parallel --gnu --plain -j 4 --block 175565 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQC01\\\\/\\\\/Fischerella_NMQC01\\\\.IS\\\\.tmp\\\\.5008\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:44:17] This is prokka 1.14.6\\n[10:44:17] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:44:17] Homepage is https://github.com/tseemann/prokka\\n[10:44:17] Local time is Mon Feb 21 10:44:17 2022\\n[10:44:17] You are tiagoferreiraleao\\n[10:44:17] Operating system is darwin\\n[10:44:17] You have BioPerl 1.007002\\n[10:44:17] System has 4 cores.\\n[10:44:17] Option --cpu asked for 12 cores, but system only has 4\\n[10:44:17] Will use maximum of 4 cores.\\n[10:44:17] Annotating as >>> Bacteria <<<\\n[10:44:17] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Desmonostoc_JACERF01.fasta' contents.\\n[10:44:17] Setting --locustag KKNKGAJO from MD5 44740a389543113ce92898ae98c07fa4\\n[10:44:17] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Desmonostoc_JACERF01/\\n[10:44:17] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Desmonostoc_JACERF01\\\\/\\n[10:44:17] Using filename prefix: Desmonostoc_JACERF01.XXX\\n[10:44:17] Setting HMMER_NCPU=1\\n[10:44:17] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Desmonostoc_JACERF01//Desmonostoc_JACERF01.log\\n[10:44:17] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Desmonostoc_JACERF01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Desmonostoc_JACERF01/ --prefix Desmonostoc_JACERF01 --cpus 12\\n[10:44:17] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:44:17] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:44:17] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:44:17] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:44:17] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:44:17] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:44:17] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:44:17] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:44:17] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:44:17] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:44:17] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:44:17] Looking for 'egrep' - found /usr/bin/egrep\\n[10:44:17] Looking for 'find' - found /usr/bin/find\\n[10:44:17] Looking for 'grep' - found /usr/bin/grep\\n[10:44:17] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:44:17] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:44:17] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:44:17] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:44:17] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:44:17] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:44:17] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:44:17] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:44:17] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:44:17] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:44:17] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:44:17] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:44:18] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:44:18] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:44:18] Looking for 'sed' - found /usr/bin/sed\\n[10:44:18] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:44:18] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:44:18] Using genetic code table 11.\\n[10:44:18] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Desmonostoc_JACERF01.fasta\\n[10:44:18] Wrote 151 contigs totalling 8982067 bp.\\n[10:44:18] Predicting tRNAs and tmRNAs\\n[10:44:18] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Desmonostoc_JACERF01\\\\/\\\\/Desmonostoc_JACERF01\\\\.fna\\n[10:44:21] 1 tRNA-Glu c[50498,50571] 35 (ttc)\\n[10:44:21] 2 tRNA-Asp [73595,73670] 36 (gtc)\\n[10:44:21] 3 tRNA-Glu [73674,73750] 35 (ctc)\\n[10:44:21] 4 tRNA-Cys [73754,73827] 35 (gca)\\n[10:44:21] 5 tRNA-Met [73834,73907] 32 (cat)\\n[10:44:21] tRNA [73910,73984] is a pseudo/wacky gene - skipping.\\n[10:44:21] 7 tRNA-Glu [74072,74146] 35 (ttc)\\n[10:44:21] 8 tRNA-Tyr [74198,74281] 37 (gta)\\n[10:44:21] 9 tRNA-Ser [74285,74376] 35 (gct)\\n[10:44:21] 10 tRNA-Arg [74378,74454] 36 (acg)\\n[10:44:21] 11 tRNA-Phe [74535,74610] 36 (gaa)\\n[10:44:21] 12 tRNA-Asn [74616,74690] 33 (gtt)\\n[10:44:21] 13 tRNA-Gln [74848,74920] 34 (ctg)\\n[10:44:21] 14 tRNA-Gln [74924,74999] 34 (ttg)\\n[10:44:21] 15 tRNA-Leu [75006,75082] 35 (caa)\\n[10:44:21] 16 tRNA-Leu [75089,75166] 35 (taa)\\n[10:44:21] 17 tRNA-Pro [75168,75240] 34 (tgg)\\n[10:44:21] 18 tRNA-Leu [75244,75318] 35 (cag)\\n[10:44:21] 19 tRNA-Leu [75322,75395] 34 (gag)\\n[10:44:21] 20 tRNA-Leu [75508,75582] 35 (tag)\\n[10:44:21] 21 tRNA-Met [75592,75668] 34 (cat)\\n[10:44:21] 22 tRNA-Thr [75843,75915] 34 (tgt)\\n[10:44:21] 23 tRNA-Val [76096,76171] 34 (tac)\\n[10:44:21] 24 tRNA-Arg [76178,76253] 35 (tct)\\n[10:44:21] 25 tRNA-Ala [76257,76329] 34 (ggc)\\n[10:44:21] 26 tRNA-Ala [76339,76413] 35 (tgc)\\n[10:44:21] 27 tRNA-His [76419,76494] 35 (gtg)\\n[10:44:21] 28 tRNA-Ser [76500,76583] 34 (tga)\\n[10:44:21] 29 tRNA-Lys [76715,76785] 33 (ttt)\\n[10:44:21] 30 tRNA-Ser [81455,81546] 35 (gct)\\n[10:44:21] 1 tRNA-Thr [123778,123849] 33 (tgt)\\n[10:44:21] 2 tRNA-Met c[126654,126727] 33 (cat)\\n[10:44:21] 3 tRNA-Arg [183973,184046] 35 (acg)\\n[10:44:21] 1 tRNA-Pro c[66387,66460] 35 (tgg)\\n[10:44:21] 1 tRNA-Gly [225154,225225] 33 (ccc)\\n[10:44:21] 1 tRNA-His [65874,65947] 34 (gtg)\\n[10:44:21] 1 tRNA-Lys [76173,76246] 33 (ctt)\\n[10:44:21] 1 tRNA-Ala c[128635,128707] 34 (ggc)\\n[10:44:21] 1 tRNA-Phe [60620,60715] 34 (gaa)\\n[10:44:21] 2 tRNA-Gly c[101995,102065] 33 (tcc)\\n[10:44:21] 1 tRNA-Leu c[3221,3302] 35 (cag)\\n[10:44:21] 1 tRNA-Leu [12768,12848] 35 (gag)\\n[10:44:21] 1 tRNA-Cys [67106,67177] 33 (gca)\\n[10:44:21] 1 tRNA-His c[89499,89594] 36 (gtg)\\n[10:44:21] 1 tRNA-Met c[31837,31913] 35 (cat)\\n[10:44:21] 2 tRNA-Met c[31915,31990] 34 (cat)\\n[10:44:21] 1 tRNA-Ala c[57799,57871] 34 (tgc)\\n[10:44:21] 1 tRNA-Ser c[30860,30946] 36 (cga)\\n[10:44:21] 1 tRNA-Gly [18153,18224] 33 (gcc)\\n[10:44:21] 2 tRNA-Arg c[139716,139788] 34 (ccg)\\n[10:44:21] 1 tRNA-Pyl [23210,23282] 34 (cta)\\n[10:44:21] 1 tRNA-Trp [37764,37837] 34 (cca)\\n[10:44:21] 1 tRNA-Asp [10056,10129] 35 (gtc)\\n[10:44:21] 2 tRNA-Leu [36745,36827] 35 (tag)\\n[10:44:21] 3 tRNA-Gln c[217079,217154] 34 (ttg)\\n[10:44:21] 1 tRNA-Val c[8092,8165] 35 (gac)\\n[10:44:21] 1 tRNA-Val [81418,81491] 34 (cac)\\n[10:44:21] 1 tRNA-Arg [59047,59121] 35 (tct)\\n[10:44:21] 2 tRNA-Val c[87391,87463] 34 (tac)\\n[10:44:21] 1 tRNA-Ser [29591,29679] 36 (gga)\\n[10:44:21] 2 tRNA-Phe [31138,31210] 34 (gaa)\\n[10:44:21] 1 tRNA-Asn [80971,81042] 33 (gtt)\\n[10:44:21] 1 tRNA-Pro c[114132,114207] 36 (ggg)\\n[10:44:21] 1 tRNA-Met c[37970,38046] 35 (cat)\\n[10:44:21] 1 tRNA-Lys [59107,59178] 33 (ttt)\\n[10:44:21] 2 tRNA-Thr c[113587,113658] 33 (ggt)\\n[10:44:21] 3 tRNA-Tyr c[113742,113826] 35 (gta)\\n[10:44:21] 4 tRNA-Ser [116079,116167] 36 (tga)\\n[10:44:21] 5 tRNA-Gly c[129399,129470] 33 (tcc)\\n[10:44:21] 6 tRNA-Glu c[129671,129742] 33 (ttc)\\n[10:44:21] 1 tmRNA c[4445,4832] 101,151 ANNIVKFARKDALVAA*\\n[10:44:21] 1 tRNA-Met [360,434] 35 (cat)\\n[10:44:21] 1 tRNA-Thr c[16935,17006] 33 (cgt)\\n[10:44:21] 1 tRNA-Pro c[37458,37531] 35 (cgg)\\n[10:44:21] 1 tRNA-Gln c[4047,4120] 34 (ttg)\\n[10:44:21] 2 tRNA-Leu c[4682,4766] 35 (caa)\\n[10:44:21] 1 tRNA-Ala c[3678,3750] 34 (cgc)\\n[10:44:21] Found 76 tRNAs\\n[10:44:21] Predicting Ribosomal RNAs\\n[10:44:21] Running Barrnap with 4 threads\\n[10:44:22] Found 0 rRNAs\\n[10:44:22] Skipping ncRNA search, enable with --rfam if desired.\\n[10:44:22] Total of 75 tRNA + rRNA features\\n[10:44:22] Searching for CRISPR repeats\\n[10:44:23] CRISPR1 JACERF010000002.1 16207 with 11 spacers\\n[10:44:23] CRISPR2 JACERF010000021.1 51667 with 18 spacers\\n[10:44:23] CRISPR3 JACERF010000022.1 43312 with 3 spacers\\n[10:44:23] CRISPR4 JACERF010000048.1 5646 with 20 spacers\\n[10:44:23] CRISPR5 JACERF010000057.1 18361 with 25 spacers\\n[10:44:23] CRISPR6 JACERF010000059.1 69740 with 19 spacers\\n[10:44:23] CRISPR7 JACERF010000082.1 13161 with 40 spacers\\n[10:44:23] CRISPR8 JACERF010000082.1 46352 with 46 spacers\\n[10:44:23] CRISPR9 JACERF010000115.1 36742 with 5 spacers\\n[10:44:23] Found 9 CRISPRs\\n[10:44:23] Predicting coding sequences\\n[10:44:23] Contigs total 8982067 bp, so using single mode\\n[10:44:23] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Desmonostoc_JACERF01\\\\/\\\\/Desmonostoc_JACERF01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:44:33] Excluding CDS which overlaps existing RNA (tRNA) at JACERF010000004.1:75195..75443 on + strand\\n[10:44:33] Excluding CDS which overlaps existing RNA (tRNA) at JACERF010000004.1:76240..76695 on - strand\\n[10:44:33] Excluding CDS which overlaps existing RNA (tRNA) at JACERF010000007.1:126477..126683 on + strand\\n[10:44:34] Excluding CDS which overlaps existing RNA (tRNA) at JACERF010000016.1:60356..61747 on + strand\\n[10:44:34] Excluding CDS which overlaps existing RNA (tRNA) at JACERF010000024.1:89059..90372 on + strand\\n[10:44:35] Excluding CDS which overlaps existing RNA (tRNA) at JACERF010000040.1:139698..139865 on + strand\\n[10:44:35] Excluding CDS which overlaps existing RNA (tRNA) at JACERF010000047.1:37740..37949 on + strand\\n[10:44:36] Excluding CDS which overlaps existing RNA (tRNA) at JACERF010000049.1:36751..36912 on + strand\\n[10:44:36] Excluding CDS which overlaps existing RNA (tRNA) at JACERF010000049.1:215410..218724 on + strand\\n[10:44:36] Excluding CDS which overlaps existing RNA (repeat_region) at JACERF010000057.1:18133..18375 on + strand\\n[10:44:37] Excluding CDS which overlaps existing RNA (repeat_region) at JACERF010000082.1:15375..15644 on - strand\\n[10:44:38] Found 7289 CDS\\n[10:44:38] Connecting features back to sequences\\n[10:44:38] Not using genus-specific database. Try --usegenus to enable it.\\n[10:44:38] Annotating CDS, please be patient.\\n[10:44:38] Will use 4 CPUs for similarity searching.\\n[10:44:40] There are still 7289 unannotated CDS left (started with 7289)\\n[10:44:40] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:44:40] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Desmonostoc_JACERF01\\\\/\\\\/Desmonostoc_JACERF01\\\\.IS\\\\.tmp\\\\.5121\\\\.faa | parallel --gnu --plain -j 4 --block 303046 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Desmonostoc_JACERF01\\\\/\\\\/Desmonostoc_JACERF01\\\\.IS\\\\.tmp\\\\.5121\\\\.blast 2> /dev/null\\n[10:44:48] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Desmonostoc_JACERF01\\\\/\\\\/Desmonostoc_JACERF01\\\\.IS\\\\.tmp\\\\.5121\\\\.faa | parallel --gnu --plain -j 4 --block 303046 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Desmonostoc_JACERF01\\\\/\\\\/Desmonostoc_JACERF01\\\\.IS\\\\.tmp\\\\.5121\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:44:49] This is prokka 1.14.6\\n[10:44:49] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:44:49] Homepage is https://github.com/tseemann/prokka\\n[10:44:49] Local time is Mon Feb 21 10:44:49 2022\\n[10:44:49] You are tiagoferreiraleao\\n[10:44:49] Operating system is darwin\\n[10:44:49] You have BioPerl 1.007002\\n[10:44:49] System has 4 cores.\\n[10:44:49] Option --cpu asked for 12 cores, but system only has 4\\n[10:44:49] Will use maximum of 4 cores.\\n[10:44:49] Annotating as >>> Bacteria <<<\\n[10:44:49] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHI01.fasta' contents.\\n[10:44:49] Setting --locustag HAPHPNHM from MD5 1a919716be0941f603a359a87a9daca9\\n[10:44:49] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHI01/\\n[10:44:49] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHI01\\\\/\\n[10:44:49] Using filename prefix: Nostoc_LAHI01.XXX\\n[10:44:49] Setting HMMER_NCPU=1\\n[10:44:49] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHI01//Nostoc_LAHI01.log\\n[10:44:49] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHI01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHI01/ --prefix Nostoc_LAHI01 --cpus 12\\n[10:44:49] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:44:49] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:44:49] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:44:49] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:44:49] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:44:49] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:44:49] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:44:49] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:44:49] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:44:49] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:44:49] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:44:49] Looking for 'egrep' - found /usr/bin/egrep\\n[10:44:49] Looking for 'find' - found /usr/bin/find\\n[10:44:49] Looking for 'grep' - found /usr/bin/grep\\n[10:44:49] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:44:49] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:44:49] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:44:49] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:44:49] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:44:49] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:44:50] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:44:50] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:44:50] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:44:50] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:44:50] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:44:50] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:44:50] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:44:50] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:44:50] Looking for 'sed' - found /usr/bin/sed\\n[10:44:50] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:44:51] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:44:51] Using genetic code table 11.\\n[10:44:51] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHI01.fasta\\n[10:44:51] Wrote 712 contigs totalling 9936114 bp.\\n[10:44:51] Predicting tRNAs and tmRNAs\\n[10:44:51] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHI01\\\\/\\\\/Nostoc_LAHI01\\\\.fna\\n[10:44:56] 1 tRNA-Pro c[100522,100595] 35 (tgg)\\n[10:44:56] 2 tRNA-Ser c[109277,109363] 36 (tga)\\n[10:44:56] 3 tRNA-Tyr [111814,111898] 35 (gta)\\n[10:44:56] 4 tRNA-Thr [111974,112045] 33 (ggt)\\n[10:44:56] 1 tRNA-Gly [17223,17293] 33 (tcc)\\n[10:44:56] 1 tRNA-Ser [100394,100480] 36 (cga)\\n[10:44:56] 1 tRNA-Gln c[29234,29307] 34 (ttg)\\n[10:44:56] 2 tRNA-Leu c[34061,34145] 35 (caa)\\n[10:44:56] 3 tRNA-Ala [82847,82919] 34 (cgc)\\n[10:44:56] 1 tRNA-Asn [28798,28871] 34 (gtt)\\n[10:44:56] 1 tmRNA [18288,18675] 101,151 ANNIVKFARREALVAA*\\n[10:44:56] 1 tRNA-Cys [25023,25094] 33 (gca)\\n[10:44:56] 1 tRNA-Ile c[45557,45632] 34 (tat)\\n[10:44:56] 2 tRNA-Met c[47238,47313] 34 (cat)\\n[10:44:56] 3 tRNA-Met c[47315,47390] 34 (cat)\\n[10:44:56] 1 tRNA-Ala c[59446,59518] 34 (tgc)\\n[10:44:56] 1 tRNA-Pro [56844,56917] 35 (cgg)\\n[10:44:56] 1 tRNA-Leu c[46466,46549] 36 (tag)\\n[10:44:56] 1 tRNA-Pro c[43823,43896] 35 (ggg)\\n[10:44:56] 1 tRNA-Met [32595,32668] 33 (cat)\\n[10:44:56] 2 tRNA-Gly [39478,39549] 33 (gcc)\\n[10:44:56] 1 tRNA-Trp c[17971,18044] 34 (cca)\\n[10:44:56] 2 tRNA-Thr c[63653,63724] 33 (cgt)\\n[10:44:56] 1 tRNA-Ala c[14843,14915] 34 (ggc)\\n[10:44:56] 1 tRNA-Ser c[10015,10108] 36 (gct)\\n[10:44:56] 2 tRNA-Lys c[15783,15853] 33 (ttt)\\n[10:44:56] 3 tRNA-Ser c[16112,16195] 34 (tga)\\n[10:44:56] 4 tRNA-His c[16200,16275] 35 (gtg)\\n[10:44:56] 5 tRNA-Ala c[16281,16355] 35 (tgc)\\n[10:44:56] 6 tRNA-Arg c[16447,16522] 35 (tct)\\n[10:44:56] 7 tRNA-Val c[16529,16604] 34 (tac)\\n[10:44:56] 8 tRNA-Thr c[16686,16758] 34 (tgt)\\n[10:44:56] 9 tRNA-Leu c[17053,17127] 35 (tag)\\n[10:44:56] 10 tRNA-Leu c[17239,17315] 34 (gag)\\n[10:44:56] 11 tRNA-Pro c[17423,17496] 35 (tgg)\\n[10:44:56] 12 tRNA-Leu c[17498,17575] 35 (taa)\\n[10:44:56] 13 tRNA-Leu c[17596,17673] 36 (caa)\\n[10:44:56] 14 tRNA-Gln c[17699,17774] 35 (ttg)\\n[10:44:56] 15 tRNA-Gln c[18649,18723] 35 (ctg)\\n[10:44:56] 16 tRNA-Asn c[18734,18808] 33 (gtt)\\n[10:44:56] 17 tRNA-Phe c[18814,18889] 36 (gaa)\\n[10:44:56] 18 tRNA-Arg c[18977,19052] 35 (acg)\\n[10:44:56] 19 tRNA-Ser c[19814,19900] 35 (gct)\\n[10:44:56] 20 tRNA-Tyr c[19904,19985] 35 (gta)\\n[10:44:56] 21 tRNA-Glu c[19996,20070] 35 (ttc)\\n[10:44:56] tRNA c[20158,20232] is a pseudo/wacky gene - skipping.\\n[10:44:56] 23 tRNA-Met c[20235,20308] 32 (cat)\\n[10:44:56] 24 tRNA-Glu c[20418,20494] 35 (ctc)\\n[10:44:56] 25 tRNA-Asp c[20498,20573] 36 (gtc)\\n[10:44:56] 1 tRNA-Leu c[30427,30501] 35 (caa)\\n[10:44:56] 2 tRNA-Thr c[31892,31966] 33 (agt)\\n[10:44:56] 1 tRNA-Asp [25365,25442] 38 (gtc)\\n[10:44:56] 2 tRNA-Glu [25445,25521] 35 (ctc)\\n[10:44:56] 3 tRNA-Cys [25526,25601] 34 (gca)\\n[10:44:56] 4 tRNA-Met [25607,25681] 33 (cat)\\n[10:44:56] 5 tRNA-Trp [25684,25756] 34 (cca)\\n[10:44:56] 6 tRNA-Pyl [25781,25852] 33 (cta)\\n[10:44:56] 7 tRNA-Glu [25883,25958] 34 (ttc)\\n[10:44:56] 8 tRNA-Tyr [25962,26045] 36 (gta)\\n[10:44:56] 9 tRNA-Ser [26049,26138] 33 (gct)\\n[10:44:56] 10 tRNA-Arg [26143,26218] 35 (acg)\\n[10:44:56] 11 tRNA-Phe [26223,26297] 35 (gaa)\\n[10:44:56] 12 tRNA-Asn [26306,26380] 33 (gtt)\\n[10:44:56] 13 tRNA-Gln [26386,26461] 35 (ttg)\\n[10:44:56] 14 tRNA-Leu [26469,26546] 35 (caa)\\n[10:44:56] 15 tRNA-Pro [26551,26627] 35 (tgg)\\n[10:44:56] 16 tRNA-Leu [26631,26705] 35 (tag)\\n[10:44:56] 17 tRNA-Leu [26725,26802] 35 (gag)\\n[10:44:56] 18 tRNA-Lys [26806,26882] 34 (ttt)\\n[10:44:56] 19 tRNA-Lys [26888,26962] 33 (ctt)\\n[10:44:56] 20 tRNA-Leu [26968,27043] 36 (taa)\\n[10:44:56] 21 tRNA-Cys [27053,27130] 34 (gca)\\n[10:44:56] 22 tRNA-Thr [27138,27210] 34 (tgt)\\n[10:44:56] 23 tRNA-Gly [27216,27286] 33 (tcc)\\n[10:44:56] 24 tRNA-Val [27292,27367] 34 (tac)\\n[10:44:56] 25 tRNA-Arg [27374,27448] 35 (tct)\\n[10:44:56] 26 tRNA-Ala [27453,27525] 34 (tgc)\\n[10:44:56] 27 tRNA-His [27533,27609] 35 (gtg)\\n[10:44:56] 28 tRNA-Ser [27620,27704] 34 (tga)\\n[10:44:56] 1 tRNA-Lys [46183,46254] 33 (ttt)\\n[10:44:56] 1 tRNA-Ile [39726,39801] 35 (gat)\\n[10:44:56] 2 tRNA-Ala [39885,39959] 34 (tgc)\\n[10:44:56] 1 tRNA-Val c[88895,88967] 34 (tac)\\n[10:44:56] 1 tRNA-His [18876,18949] 34 (gtg)\\n[10:44:56] 1 tRNA-Glu [35934,36007] 35 (ttc)\\n[10:44:56] 1 tRNA-Thr [29951,30023] 33 (tgt)\\n[10:44:56] 1 tRNA-Ser [4757,4844] 35 (gga)\\n[10:44:56] 2 tRNA-Gly c[12989,13062] 34 (ccc)\\n[10:44:56] 1 tRNA-Arg c[3738,3812] 35 (tct)\\n[10:44:56] 1 tRNA-Lys c[3692,3764] 33 (ctt)\\n[10:44:56] 1 tRNA-Arg [4192,4264] 34 (ccg)\\n[10:44:56] 1 tRNA-Arg c[19822,19895] 35 (acg)\\n[10:44:56] 1 tRNA-Leu c[3542,3622] 35 (gag)\\n[10:44:56] 1 tRNA-Asp [151,224] 35 (gtc)\\n[10:44:56] 1 tRNA-Leu [10506,10587] 35 (cag)\\n[10:44:56] 1 tRNA-Val [9574,9647] 35 (gac)\\n[10:44:56] 1 tRNA-Ala [2200,2286] 29 (cgc)\\n[10:44:56] 1 tRNA-Ala c[3427,3501] 35 (tgc)\\n[10:44:56] 2 tRNA-Ile c[3608,3682] 35 (gat)\\n[10:44:56] 1 tRNA-Tyr [66,149] 35 (gta)\\n[10:44:56] 2 tRNA-Thr [239,311] 33 (ggt)\\n[10:44:56] 3 tRNA-Trp [1620,1691] 33 (cca)\\n[10:44:56] 1 tRNA-Asn c[2140,2214] 35 (gtt)\\n[10:44:56] 1 tRNA-Arg c[2120,2194] 35 (tct)\\n[10:44:56] 2 tRNA-Pro c[2240,2315] 36 (tgg)\\n[10:44:56] 3 tRNA-Ser c[2322,2406] 33 (gct)\\n[10:44:56] 1 tRNA-Arg c[227,301] 35 (tct)\\n[10:44:56] 1 tRNA-Leu c[372,452] 35 (caa)\\n[10:44:56] 1 tRNA-Phe c[240,313] 34 (gaa)\\n[10:44:56] 1 tRNA-Leu c[1535,1621] 35 (taa)\\n[10:44:56] 2 tRNA-Gly c[1660,1735] 34 (gcc)\\n[10:44:56] 1 tRNA-Pro c[1993,2067] 35 (ggg)\\n[10:44:56] Found 111 tRNAs\\n[10:44:56] Predicting Ribosomal RNAs\\n[10:44:56] Running Barrnap with 4 threads\\n[10:44:59] 1 LAHI01000063.1 38194 16S ribosomal RNA\\n[10:44:59] 2 LAHI01000063.1 40132 23S ribosomal RNA\\n[10:44:59] 3 LAHI01000250.1 56 5S ribosomal RNA\\n[10:44:59] 4 LAHI01000250.1 350 23S ribosomal RNA\\n[10:44:59] 5 LAHI01000250.1 3862 16S ribosomal RNA\\n[10:44:59] Found 5 rRNAs\\n[10:44:59] Skipping ncRNA search, enable with --rfam if desired.\\n[10:44:59] Total of 115 tRNA + rRNA features\\n[10:44:59] Searching for CRISPR repeats\\n[10:44:59] CRISPR1 LAHI01000006.1 77811 with 10 spacers\\n[10:44:59] CRISPR2 LAHI01000007.1 4841 with 43 spacers\\n[10:44:59] CRISPR3 LAHI01000007.1 19418 with 27 spacers\\n[10:44:59] CRISPR4 LAHI01000009.1 39577 with 3 spacers\\n[10:44:59] CRISPR5 LAHI01000014.1 13133 with 15 spacers\\n[10:44:59] CRISPR6 LAHI01000017.1 42300 with 14 spacers\\n[10:44:59] CRISPR7 LAHI01000018.1 38405 with 5 spacers\\n[10:44:59] CRISPR8 LAHI01000018.1 38811 with 9 spacers\\n[10:44:59] CRISPR9 LAHI01000022.1 29679 with 20 spacers\\n[10:44:59] CRISPR10 LAHI01000037.1 38293 with 7 spacers\\n[10:44:59] CRISPR11 LAHI01000048.1 27565 with 24 spacers\\n[10:45:00] CRISPR12 LAHI01000170.1 9919 with 18 spacers\\n[10:45:00] CRISPR13 LAHI01000172.1 3027 with 7 spacers\\n[10:45:00] CRISPR14 LAHI01000205.1 5056 with 6 spacers\\n[10:45:00] CRISPR15 LAHI01000245.1 4064 with 23 spacers\\n[10:45:00] Found 15 CRISPRs\\n[10:45:00] Predicting coding sequences\\n[10:45:00] Contigs total 9936114 bp, so using single mode\\n[10:45:00] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHI01\\\\/\\\\/Nostoc_LAHI01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:45:09] Excluding CDS which overlaps existing RNA (repeat_region) at LAHI01000006.1:78020..78172 on + strand\\n[10:45:09] Excluding CDS which overlaps existing RNA (repeat_region) at LAHI01000007.1:7779..7943 on + strand\\n[10:45:10] Excluding CDS which overlaps existing RNA (repeat_region) at LAHI01000009.1:39496..39762 on - strand\\n[10:45:10] Excluding CDS which overlaps existing RNA (repeat_region) at LAHI01000017.1:42471..42587 on - strand\\n[10:45:13] Excluding CDS which overlaps existing RNA (tRNA) at LAHI01000075.1:18797..18940 on - strand\\n[10:45:14] Excluding CDS which overlaps existing RNA (repeat_region) at LAHI01000172.1:3347..3472 on + strand\\n[10:45:14] Excluding CDS which overlaps existing RNA (repeat_region) at LAHI01000205.1:5448..6407 on - strand\\n[10:45:14] Excluding CDS which overlaps existing RNA (tRNA) at LAHI01000241.1:1381..2751 on + strand\\n[10:45:14] Excluding CDS which overlaps existing RNA (repeat_region) at LAHI01000245.1:3876..4154 on + strand\\n[10:45:14] Excluding CDS which overlaps existing RNA (rRNA) at LAHI01000250.1:2204..2332 on + strand\\n[10:45:14] Excluding CDS which overlaps existing RNA (rRNA) at LAHI01000250.1:2370..2774 on + strand\\n[10:45:14] Excluding CDS which overlaps existing RNA (rRNA) at LAHI01000250.1:3049..3195 on - strand\\n[10:45:14] Excluding CDS which overlaps existing RNA (tRNA) at LAHI01000324.1:1664..1906 on + strand\\n[10:45:15] Found 8386 CDS\\n[10:45:15] Connecting features back to sequences\\n[10:45:15] Not using genus-specific database. Try --usegenus to enable it.\\n[10:45:15] Annotating CDS, please be patient.\\n[10:45:15] Will use 4 CPUs for similarity searching.\\n[10:45:17] There are still 8386 unannotated CDS left (started with 8386)\\n[10:45:17] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:45:17] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHI01\\\\/\\\\/Nostoc_LAHI01\\\\.IS\\\\.tmp\\\\.5243\\\\.faa | parallel --gnu --plain -j 4 --block 328229 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHI01\\\\/\\\\/Nostoc_LAHI01\\\\.IS\\\\.tmp\\\\.5243\\\\.blast 2> /dev/null\\n[10:45:39] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHI01\\\\/\\\\/Nostoc_LAHI01\\\\.IS\\\\.tmp\\\\.5243\\\\.faa | parallel --gnu --plain -j 4 --block 328229 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHI01\\\\/\\\\/Nostoc_LAHI01\\\\.IS\\\\.tmp\\\\.5243\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:45:39] This is prokka 1.14.6\\n[10:45:39] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:45:39] Homepage is https://github.com/tseemann/prokka\\n[10:45:39] Local time is Mon Feb 21 10:45:39 2022\\n[10:45:39] You are tiagoferreiraleao\\n[10:45:39] Operating system is darwin\\n[10:45:39] You have BioPerl 1.007002\\n[10:45:40] System has 4 cores.\\n[10:45:40] Option --cpu asked for 12 cores, but system only has 4\\n[10:45:40] Will use maximum of 4 cores.\\n[10:45:40] Annotating as >>> Bacteria <<<\\n[10:45:40] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAC01.fasta' contents.\\n[10:45:40] Setting --locustag GBJCLFHJ from MD5 0b3c5f13f652245280a553a9cebcd4bf\\n[10:45:40] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAC01/\\n[10:45:40] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAC01\\\\/\\n[10:45:40] Using filename prefix: Fischerella_NHAC01.XXX\\n[10:45:40] Setting HMMER_NCPU=1\\n[10:45:40] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAC01//Fischerella_NHAC01.log\\n[10:45:40] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAC01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAC01/ --prefix Fischerella_NHAC01 --cpus 12\\n[10:45:40] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:45:40] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:45:40] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:45:40] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:45:40] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:45:40] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:45:40] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:45:40] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:45:40] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:45:40] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:45:40] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:45:40] Looking for 'egrep' - found /usr/bin/egrep\\n[10:45:40] Looking for 'find' - found /usr/bin/find\\n[10:45:40] Looking for 'grep' - found /usr/bin/grep\\n[10:45:40] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:45:40] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:45:40] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:45:40] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:45:40] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:45:40] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:45:40] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:45:40] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:45:40] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:45:40] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:45:40] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:45:40] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:45:40] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:45:40] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:45:40] Looking for 'sed' - found /usr/bin/sed\\n[10:45:40] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:45:41] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:45:41] Using genetic code table 11.\\n[10:45:41] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAC01.fasta\\n[10:45:41] Wrote 188 contigs totalling 5311703 bp.\\n[10:45:41] Predicting tRNAs and tmRNAs\\n[10:45:41] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAC01\\\\/\\\\/Fischerella_NHAC01\\\\.fna\\n[10:45:43] 1 tRNA-Thr c[10307,10378] 33 (tgt)\\n[10:45:43] 1 tRNA-Thr c[1525,1596] 33 (cgt)\\n[10:45:43] 1 tRNA-Pro c[316,390] 35 (tgg)\\n[10:45:43] 1 tRNA-Cys [21042,21113] 33 (gca)\\n[10:45:43] 1 tRNA-Gln [123,193] 32 (ttg)\\n[10:45:43] 1 tRNA-Phe c[38726,38798] 34 (gaa)\\n[10:45:43] 2 tRNA-Val [46139,46211] 34 (tac)\\n[10:45:43] 1 tRNA-Ala [601,674] 34 (ggc)\\n[10:45:43] 2 tRNA-Arg [5566,5639] 35 (acg)\\n[10:45:43] 1 tRNA-Asn c[3585,3658] 34 (att)\\n[10:45:43] 2 tRNA-Arg c[4329,4401] 34 (tcg)\\n[10:45:43] 1 tRNA-Leu [6621,6704] 36 (tag)\\n[10:45:43] 1 tRNA-Asp [4041,4114] 35 (gtc)\\n[10:45:43] 1 tRNA-Leu c[1056,1140] 35 (caa)\\n[10:45:43] 1 tRNA-Leu [12854,12936] 35 (cag)\\n[10:45:43] 1 tRNA-Arg [42948,43021] 35 (ccg)\\n[10:45:43] 1 tRNA-Gln [34462,34533] 33 (ttg)\\n[10:45:43] 1 tRNA-Ser c[44419,44505] 36 (cga)\\n[10:45:43] 1 tRNA-Arg [456,530] 35 (tct)\\n[10:45:43] 2 tRNA-Met [25306,25381] 36 (cat)\\n[10:45:43] 1 tRNA-Leu [35932,36014] 36 (gag)\\n[10:45:43] 1 tRNA-Lys c[13104,13177] 34 (ttt)\\n[10:45:43] 1 tRNA-Ser c[60212,60298] 36 (tga)\\n[10:45:43] 2 tRNA-Glu c[83112,83184] 35 (ttc)\\n[10:45:43] 1 tRNA-Gly [13896,13967] 33 (ccc)\\n[10:45:43] 1 tRNA-Ala c[37015,37089] 35 (cgc)\\n[10:45:43] 1 tRNA-Ser c[4208,4296] 36 (gga)\\n[10:45:43] 1 tRNA-Val c[45101,45174] 35 (gac)\\n[10:45:43] 2 tRNA-Met [48122,48198] 35 (cat)\\n[10:45:43] 1 tRNA-Pro c[13927,14000] 35 (ggg)\\n[10:45:43] 1 tRNA-Pro c[23790,23863] 35 (cgg)\\n[10:45:43] 1 tRNA-Gly c[5057,5127] 33 (tcc)\\n[10:45:43] 1 tRNA-Thr c[221,292] 33 (ggt)\\n[10:45:43] 2 tRNA-Tyr c[356,440] 35 (gta)\\n[10:45:43] 1 tRNA-Asn c[89865,89938] 34 (gtt)\\n[10:45:43] 2 tRNA-Gly [106501,106572] 33 (gcc)\\n[10:45:43] 1 tRNA-Trp [16414,16487] 34 (cca)\\n[10:45:43] 1 tRNA-Met c[125932,126004] 34 (cat)\\n[10:45:43] 2 tRNA-Ile [136699,136774] 35 (gat)\\n[10:45:43] 3 tRNA-Ala [136857,136931] 34 (tgc)\\n[10:45:43] 1 tRNA-Ser c[68686,68779] 36 (gct)\\n[10:45:43] 1 tRNA-His c[9935,10008] 34 (gtg)\\n[10:45:43] 1 tmRNA [11998,12384] 100,150 ANNIVKFARKEALVAA*\\n[10:45:43] 1 tRNA-Arg [148260,148335] 36 (cct)\\n[10:45:43] Found 44 tRNAs\\n[10:45:43] Predicting Ribosomal RNAs\\n[10:45:43] Running Barrnap with 4 threads\\n[10:45:44] 1 NHAC01000163.1 135074 16S ribosomal RNA\\n[10:45:44] 2 NHAC01000163.1 137229 23S ribosomal RNA\\n[10:45:44] 3 NHAC01000163.1 140168 5S ribosomal RNA\\n[10:45:44] Found 3 rRNAs\\n[10:45:44] Skipping ncRNA search, enable with --rfam if desired.\\n[10:45:44] Total of 46 tRNA + rRNA features\\n[10:45:44] Searching for CRISPR repeats\\n[10:45:44] CRISPR1 NHAC01000028.1 73 with 9 spacers\\n[10:45:44] CRISPR2 NHAC01000028.1 943 with 15 spacers\\n[10:45:44] CRISPR3 NHAC01000028.1 2208 with 5 spacers\\n[10:45:44] CRISPR4 NHAC01000038.1 72 with 13 spacers\\n[10:45:45] CRISPR5 NHAC01000131.1 41771 with 14 spacers\\n[10:45:45] CRISPR6 NHAC01000147.1 1 with 3 spacers\\n[10:45:45] CRISPR7 NHAC01000178.1 72 with 7 spacers\\n[10:45:45] CRISPR8 NHAC01000179.1 73646 with 4 spacers\\n[10:45:45] CRISPR9 NHAC01000179.1 74084 with 3 spacers\\n[10:45:45] CRISPR10 NHAC01000179.1 74439 with 8 spacers\\n[10:45:45] CRISPR11 NHAC01000184.1 72 with 19 spacers\\n[10:45:45] CRISPR12 NHAC01000184.1 1660 with 14 spacers\\n[10:45:45] CRISPR13 NHAC01000184.1 2839 with 11 spacers\\n[10:45:45] CRISPR14 NHAC01000184.1 3709 with 23 spacers\\n[10:45:45] CRISPR15 NHAC01000185.1 2 with 18 spacers\\n[10:45:45] CRISPR16 NHAC01000185.1 1591 with 3 spacers\\n[10:45:45] CRISPR17 NHAC01000187.1 1 with 4 spacers\\n[10:45:45] CRISPR18 NHAC01000187.1 456 with 6 spacers\\n[10:45:45] CRISPR19 NHAC01000187.1 989 with 6 spacers\\n[10:45:45] CRISPR20 NHAC01000187.1 1551 with 10 spacers\\n[10:45:45] CRISPR21 NHAC01000187.1 2515 with 5 spacers\\n[10:45:45] CRISPR22 NHAC01000187.1 2987 with 7 spacers\\n[10:45:45] CRISPR23 NHAC01000187.1 3608 with 3 spacers\\n[10:45:45] CRISPR24 NHAC01000187.1 23136 with 10 spacers\\n[10:45:45] CRISPR25 NHAC01000187.1 24016 with 7 spacers\\n[10:45:45] CRISPR26 NHAC01000187.1 24587 with 17 spacers\\n[10:45:45] CRISPR27 NHAC01000187.1 25969 with 3 spacers\\n[10:45:45] CRISPR28 NHAC01000187.1 26352 with 16 spacers\\n[10:45:45] CRISPR29 NHAC01000187.1 27709 with 3 spacers\\n[10:45:45] CRISPR30 NHAC01000187.1 28100 with 3 spacers\\n[10:45:45] CRISPR31 NHAC01000188.1 38961 with 7 spacers\\n[10:45:45] CRISPR32 NHAC01000188.1 39614 with 5 spacers\\n[10:45:45] CRISPR33 NHAC01000188.1 39992 with 15 spacers\\n[10:45:45] CRISPR34 NHAC01000188.1 41419 with 4 spacers\\n[10:45:45] CRISPR35 NHAC01000188.1 41833 with 18 spacers\\n[10:45:45] CRISPR36 NHAC01000188.1 43397 with 16 spacers\\n[10:45:45] CRISPR37 NHAC01000188.1 45058 with 27 spacers\\n[10:45:45] CRISPR38 NHAC01000188.1 47201 with 11 spacers\\n[10:45:45] Found 38 CRISPRs\\n[10:45:45] Predicting coding sequences\\n[10:45:45] Contigs total 5311703 bp, so using single mode\\n[10:45:45] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAC01\\\\/\\\\/Fischerella_NHAC01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:45:50] Excluding CDS which overlaps existing RNA (tRNA) at NHAC01000030.1:5011..5682 on - strand\\n[10:45:50] Excluding CDS which overlaps existing RNA (tRNA) at NHAC01000058.1:3684..4079 on - strand\\n[10:45:52] Excluding CDS which overlaps existing RNA (tRNA) at NHAC01000154.1:88625..90064 on - strand\\n[10:45:52] Excluding CDS which overlaps existing RNA (rRNA) at NHAC01000163.1:137624..137803 on + strand\\n[10:45:52] Excluding CDS which overlaps existing RNA (rRNA) at NHAC01000163.1:139589..139765 on - strand\\n[10:45:53] Found 4500 CDS\\n[10:45:53] Connecting features back to sequences\\n[10:45:53] Not using genus-specific database. Try --usegenus to enable it.\\n[10:45:53] Annotating CDS, please be patient.\\n[10:45:53] Will use 4 CPUs for similarity searching.\\n[10:45:54] There are still 4500 unannotated CDS left (started with 4500)\\n[10:45:54] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:45:54] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAC01\\\\/\\\\/Fischerella_NHAC01\\\\.IS\\\\.tmp\\\\.5355\\\\.faa | parallel --gnu --plain -j 4 --block 172088 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAC01\\\\/\\\\/Fischerella_NHAC01\\\\.IS\\\\.tmp\\\\.5355\\\\.blast 2> /dev/null\\n[10:46:07] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAC01\\\\/\\\\/Fischerella_NHAC01\\\\.IS\\\\.tmp\\\\.5355\\\\.faa | parallel --gnu --plain -j 4 --block 172088 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAC01\\\\/\\\\/Fischerella_NHAC01\\\\.IS\\\\.tmp\\\\.5355\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:46:07] This is prokka 1.14.6\\n[10:46:07] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:46:07] Homepage is https://github.com/tseemann/prokka\\n[10:46:07] Local time is Mon Feb 21 10:46:07 2022\\n[10:46:07] You are tiagoferreiraleao\\n[10:46:07] Operating system is darwin\\n[10:46:07] You have BioPerl 1.007002\\n[10:46:07] System has 4 cores.\\n[10:46:07] Option --cpu asked for 12 cores, but system only has 4\\n[10:46:07] Will use maximum of 4 cores.\\n[10:46:07] Annotating as >>> Bacteria <<<\\n[10:46:07] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_ALVT01.fasta' contents.\\n[10:46:07] Setting --locustag DFNIDKJD from MD5 df72d43df24be41cda13b191b29a2f7f\\n[10:46:07] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_ALVT01/\\n[10:46:07] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVT01\\\\/\\n[10:46:07] Using filename prefix: Fischerella_ALVT01.XXX\\n[10:46:07] Setting HMMER_NCPU=1\\n[10:46:07] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_ALVT01//Fischerella_ALVT01.log\\n[10:46:07] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_ALVT01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_ALVT01/ --prefix Fischerella_ALVT01 --cpus 12\\n[10:46:07] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:46:07] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:46:07] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:46:07] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:46:07] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:46:07] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:46:08] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:46:08] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:46:08] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:46:08] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:46:08] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:46:08] Looking for 'egrep' - found /usr/bin/egrep\\n[10:46:08] Looking for 'find' - found /usr/bin/find\\n[10:46:08] Looking for 'grep' - found /usr/bin/grep\\n[10:46:08] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:46:08] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:46:08] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:46:08] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:46:08] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:46:08] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:46:08] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:46:08] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:46:08] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:46:08] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:46:08] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:46:08] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:46:08] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:46:08] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:46:08] Looking for 'sed' - found /usr/bin/sed\\n[10:46:08] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:46:09] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:46:09] Using genetic code table 11.\\n[10:46:09] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_ALVT01.fasta\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657830|gb|ALVT01000001.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657822|gb|ALVT01000002.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657818|gb|ALVT01000003.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657814|gb|ALVT01000004.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657809|gb|ALVT01000005.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657805|gb|ALVT01000006.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657801|gb|ALVT01000007.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657796|gb|ALVT01000008.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657792|gb|ALVT01000009.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657788|gb|ALVT01000010.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657784|gb|ALVT01000011.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657780|gb|ALVT01000012.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657775|gb|ALVT01000013.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657771|gb|ALVT01000014.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657767|gb|ALVT01000015.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657763|gb|ALVT01000016.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657760|gb|ALVT01000017.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657756|gb|ALVT01000018.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657752|gb|ALVT01000019.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657748|gb|ALVT01000020.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657743|gb|ALVT01000021.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657739|gb|ALVT01000022.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657735|gb|ALVT01000023.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657731|gb|ALVT01000024.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657727|gb|ALVT01000025.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657724|gb|ALVT01000026.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657718|gb|ALVT01000027.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657713|gb|ALVT01000028.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657709|gb|ALVT01000029.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657705|gb|ALVT01000030.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657701|gb|ALVT01000031.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657698|gb|ALVT01000032.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657694|gb|ALVT01000033.1|\\n[10:46:09] Changing illegal '|' to '_' in sequence name: gi|572657690|gb|ALVT01000034.1|\\n[10:46:09] Wrote 34 contigs totalling 8076981 bp.\\n[10:46:09] Predicting tRNAs and tmRNAs\\n[10:46:09] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVT01\\\\/\\\\/Fischerella_ALVT01\\\\.fna\\n[10:46:12] 1 tRNA-Gly [123595,123666] 33 (gcc)\\n[10:46:12] 2 tRNA-Gly [224995,225066] 33 (ccc)\\n[10:46:12] 3 tRNA-Ala c[250512,250586] 35 (cgc)\\n[10:46:12] 4 tRNA-Lys c[469632,469704] 33 (ctt)\\n[10:46:12] 5 tRNA-Arg c[578249,578323] 35 (tct)\\n[10:46:12] 6 tRNA-Thr [1073016,1073087] 33 (tgt)\\n[10:46:12] 7 tRNA-Lys [1349770,1349841] 33 (ttt)\\n[10:46:12] 8 tRNA-Gly [1356780,1356850] 33 (tcc)\\n[10:46:12] 9 tRNA-Arg c[1385566,1385641] 36 (cct)\\n[10:46:12] 10 tRNA-Cys c[1471753,1471824] 33 (gca)\\n[10:46:12] 11 tRNA-Ala c[1843353,1843427] 34 (tgc)\\n[10:46:12] 12 tRNA-Ile c[1843515,1843590] 35 (gat)\\n[10:46:12] 13 tRNA-Asn [1853625,1853698] 34 (gtt)\\n[10:46:12] 14 tRNA-Arg c[1978886,1978959] 35 (ccg)\\n[10:46:12] 15 tRNA-Met [2049364,2049439] 36 (cat)\\n[10:46:12] 16 tRNA-Ser c[2268604,2268690] 36 (cga)\\n[10:46:12] 1 tRNA-Thr [22941,23012] 33 (cgt)\\n[10:46:12] 2 tRNA-Asp [54110,54183] 35 (gtc)\\n[10:46:12] 3 tRNA-Leu c[215078,215160] 36 (gag)\\n[10:46:12] 4 tRNA-Pro [324287,324360] 35 (tgg)\\n[10:46:12] 5 tRNA-Phe [356241,356313] 34 (gaa)\\n[10:46:12] 1 tRNA-Glu c[69076,69149] 35 (ttc)\\n[10:46:12] 2 tRNA-Ser [126295,126381] 36 (tga)\\n[10:46:12] 3 tRNA-Ser [314270,314363] 36 (gct)\\n[10:46:12] 1 tRNA-Leu c[144171,144254] 36 (tag)\\n[10:46:12] 2 tRNA-Arg [186780,186853] 35 (acg)\\n[10:46:12] 1 tRNA-Met [168364,168440] 35 (cat)\\n[10:46:12] 1 tRNA-Val [217938,218011] 35 (gac)\\n[10:46:12] 2 tRNA-Met [255792,255864] 34 (cat)\\n[10:46:12] 1 tRNA-Ala [51565,51639] 35 (ggc)\\n[10:46:12] 1 tRNA-Ile [12503,12575] 34 (gat)\\n[10:46:12] 2 tRNA-Asn [13130,13199] 33 (gtt)\\n[10:46:12] 1 tRNA-Trp c[73216,73308] 36 (cca)\\n[10:46:12] 1 tRNA-Ser [15933,16021] 36 (gga)\\n[10:46:12] 2 tRNA-Pro c[119657,119730] 35 (cgg)\\n[10:46:12] 3 tRNA-Leu [214545,214626] 35 (cag)\\n[10:46:12] 4 tRNA-Pro [490117,490190] 35 (ggg)\\n[10:46:12] 5 tRNA-Phe [533292,533375] 34 (aaa)\\n[10:46:12] 1 tRNA-Val c[39226,39299] 34 (tac)\\n[10:46:12] 2 tRNA-Gln [119454,119525] 33 (ttg)\\n[10:46:12] 3 tRNA-Thr c[143443,143514] 33 (ggt)\\n[10:46:12] 4 tRNA-Tyr c[143538,143622] 35 (gta)\\n[10:46:12] 1 tRNA-Trp c[22363,22436] 34 (cca)\\n[10:46:12] 2 tRNA-Leu c[92690,92774] 35 (caa)\\n[10:46:12] 1 tRNA-His c[13535,13608] 34 (gtg)\\n[10:46:12] 1 tmRNA [254795,255181] 100,153 ANNIVKFARKEALVAA**\\n[10:46:12] Found 46 tRNAs\\n[10:46:12] Predicting Ribosomal RNAs\\n[10:46:12] Running Barrnap with 4 threads\\n[10:46:14] 1 gi_572657724_gb_ALVT01000026.1_ 93661 16S ribosomal RNA (partial)\\n[10:46:14] 2 gi_572657830_gb_ALVT01000001.1_ 1840209 23S ribosomal RNA\\n[10:46:14] 3 gi_572657830_gb_ALVT01000001.1_ 1843708 16S ribosomal RNA\\n[10:46:14] Found 3 rRNAs\\n[10:46:14] Skipping ncRNA search, enable with --rfam if desired.\\n[10:46:14] Total of 48 tRNA + rRNA features\\n[10:46:14] Searching for CRISPR repeats\\n[10:46:14] CRISPR1 gi_572657830_gb_ALVT01000001.1_ 2649 with 42 spacers\\n[10:46:15] CRISPR2 gi_572657814_gb_ALVT01000004.1_ 78844 with 38 spacers\\n[10:46:15] CRISPR3 gi_572657784_gb_ALVT01000011.1_ 158906 with 27 spacers\\n[10:46:15] CRISPR4 gi_572657771_gb_ALVT01000014.1_ 249784 with 26 spacers\\n[10:46:15] CRISPR5 gi_572657771_gb_ALVT01000014.1_ 259032 with 42 spacers\\n[10:46:15] CRISPR6 gi_572657771_gb_ALVT01000014.1_ 481815 with 33 spacers\\n[10:46:15] CRISPR7 gi_572657756_gb_ALVT01000018.1_ 4923 with 5 spacers\\n[10:46:15] Found 7 CRISPRs\\n[10:46:15] Predicting coding sequences\\n[10:46:15] Contigs total 8076981 bp, so using single mode\\n[10:46:15] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVT01\\\\/\\\\/Fischerella_ALVT01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:46:25] Excluding CDS which overlaps existing RNA (rRNA) at gi_572657830_gb_ALVT01000001.1_:1842467..1842637 on - strand\\n[10:46:25] Excluding CDS which overlaps existing RNA (rRNA) at gi_572657830_gb_ALVT01000001.1_:1843750..1844064 on - strand\\n[10:46:25] Excluding CDS which overlaps existing RNA (tRNA) at gi_572657814_gb_ALVT01000004.1_:68920..69114 on + strand\\n[10:46:25] Excluding CDS which overlaps existing RNA (repeat_region) at gi_572657814_gb_ALVT01000004.1_:78616..78870 on + strand\\n[10:46:27] Excluding CDS which overlaps existing RNA (tRNA) at gi_572657775_gb_ALVT01000013.1_:72177..74075 on - strand\\n[10:46:27] Excluding CDS which overlaps existing RNA (repeat_region) at gi_572657771_gb_ALVT01000014.1_:250283..250408 on - strand\\n[10:46:27] Excluding CDS which overlaps existing RNA (repeat_region) at gi_572657771_gb_ALVT01000014.1_:259641..259787 on + strand\\n[10:46:27] Excluding CDS which overlaps existing RNA (tRNA) at gi_572657771_gb_ALVT01000014.1_:532730..533965 on - strand\\n[10:46:29] Found 7041 CDS\\n[10:46:29] Connecting features back to sequences\\n[10:46:29] Not using genus-specific database. Try --usegenus to enable it.\\n[10:46:29] Annotating CDS, please be patient.\\n[10:46:29] Will use 4 CPUs for similarity searching.\\n[10:46:30] There are still 7041 unannotated CDS left (started with 7041)\\n[10:46:30] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:46:30] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVT01\\\\/\\\\/Fischerella_ALVT01\\\\.IS\\\\.tmp\\\\.5467\\\\.faa | parallel --gnu --plain -j 4 --block 272126 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVT01\\\\/\\\\/Fischerella_ALVT01\\\\.IS\\\\.tmp\\\\.5467\\\\.blast 2> /dev/null\\n[10:46:44] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVT01\\\\/\\\\/Fischerella_ALVT01\\\\.IS\\\\.tmp\\\\.5467\\\\.faa | parallel --gnu --plain -j 4 --block 272126 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_ALVT01\\\\/\\\\/Fischerella_ALVT01\\\\.IS\\\\.tmp\\\\.5467\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:46:45] This is prokka 1.14.6\\n[10:46:45] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:46:45] Homepage is https://github.com/tseemann/prokka\\n[10:46:45] Local time is Mon Feb 21 10:46:45 2022\\n[10:46:45] You are tiagoferreiraleao\\n[10:46:45] Operating system is darwin\\n[10:46:45] You have BioPerl 1.007002\\n[10:46:45] System has 4 cores.\\n[10:46:45] Option --cpu asked for 12 cores, but system only has 4\\n[10:46:45] Will use maximum of 4 cores.\\n[10:46:45] Annotating as >>> Bacteria <<<\\n[10:46:45] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_VRZE01.fasta' contents.\\n[10:46:45] Setting --locustag NOCCGOAJ from MD5 78cc08a3960ba952c38a234e071f1358\\n[10:46:45] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_VRZE01/\\n[10:46:45] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_VRZE01\\\\/\\n[10:46:45] Using filename prefix: Nostoc_VRZE01.XXX\\n[10:46:45] Setting HMMER_NCPU=1\\n[10:46:45] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_VRZE01//Nostoc_VRZE01.log\\n[10:46:45] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_VRZE01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_VRZE01/ --prefix Nostoc_VRZE01 --cpus 12\\n[10:46:45] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:46:45] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:46:45] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:46:45] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:46:45] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:46:45] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:46:45] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:46:45] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:46:45] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:46:45] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:46:45] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:46:45] Looking for 'egrep' - found /usr/bin/egrep\\n[10:46:45] Looking for 'find' - found /usr/bin/find\\n[10:46:45] Looking for 'grep' - found /usr/bin/grep\\n[10:46:45] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:46:45] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:46:45] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:46:45] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:46:45] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:46:45] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:46:45] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:46:45] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:46:45] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:46:45] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:46:46] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:46:46] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:46:46] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:46:46] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:46:46] Looking for 'sed' - found /usr/bin/sed\\n[10:46:46] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:46:46] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:46:46] Using genetic code table 11.\\n[10:46:46] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_VRZE01.fasta\\n[10:46:46] Wrote 135 contigs totalling 8131703 bp.\\n[10:46:46] Predicting tRNAs and tmRNAs\\n[10:46:46] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_VRZE01\\\\/\\\\/Nostoc_VRZE01\\\\.fna\\n[10:46:49] 1 tRNA-Met [230325,230400] 34 (cat)\\n[10:46:49] 2 tRNA-Met [230406,230481] 34 (cat)\\n[10:46:49] 1 tRNA-Thr c[260359,260431] 33 (tgt)\\n[10:46:49] 1 tRNA-Pro [262447,262520] 35 (ggg)\\n[10:46:49] 1 tRNA-Thr c[29637,29708] 33 (cgt)\\n[10:46:49] 1 tRNA-Gly c[183822,183892] 33 (tcc)\\n[10:46:49] 2 tRNA-Ala c[215861,215933] 34 (ggc)\\n[10:46:49] 1 tmRNA [100112,100499] 101,151 ANNIVKFARREALVAA*\\n[10:46:49] 1 tRNA-Asp [127304,127377] 35 (gtc)\\n[10:46:49] 1 tRNA-His c[23010,23083] 34 (gtg)\\n[10:46:49] 2 tRNA-Cys c[49181,49252] 33 (gca)\\n[10:46:49] 1 tRNA-Leu [73833,73913] 35 (gag)\\n[10:46:49] 1 tRNA-Phe [10748,10820] 34 (gaa)\\n[10:46:49] 2 tRNA-Ser [59153,59240] 35 (gga)\\n[10:46:49] 3 tRNA-Gly c[71198,71271] 34 (ccc)\\n[10:46:49] 1 tRNA-Glu [15699,15772] 35 (ttc)\\n[10:46:49] 2 tRNA-Met c[81358,81434] 35 (cat)\\n[10:46:49] 1 tRNA-Val [13364,13436] 34 (tac)\\n[10:46:49] 2 tRNA-Arg c[30646,30720] 35 (tct)\\n[10:46:49] 1 tRNA-Asn [2577,2650] 34 (gtt)\\n[10:46:49] 2 tRNA-Leu c[105659,105742] 36 (tag)\\n[10:46:49] 1 tRNA-Ala c[26640,26712] 34 (cgc)\\n[10:46:49] 2 tRNA-Leu [84153,84237] 35 (caa)\\n[10:46:49] 3 tRNA-Gln [88991,89064] 34 (ttg)\\n[10:46:49] 1 tRNA-Thr c[108775,108846] 33 (ggt)\\n[10:46:49] 2 tRNA-Tyr c[108922,109006] 35 (gta)\\n[10:46:49] 3 tRNA-Ser [111457,111543] 36 (tga)\\n[10:46:49] 4 tRNA-Pro [120227,120300] 35 (tgg)\\n[10:46:49] 1 tRNA-Ser c[46700,46786] 36 (cga)\\n[10:46:49] 1 tRNA-Trp [11061,11134] 34 (cca)\\n[10:46:49] 1 tRNA-Asp [59758,59833] 36 (gtc)\\n[10:46:49] 2 tRNA-Glu [59837,59913] 35 (ctc)\\n[10:46:49] 3 tRNA-Cys [59917,59990] 35 (gca)\\n[10:46:49] 4 tRNA-Met [59997,60070] 32 (cat)\\n[10:46:49] tRNA [60073,60147] is a pseudo/wacky gene - skipping.\\n[10:46:49] 6 tRNA-Glu [60221,60295] 35 (ttc)\\n[10:46:49] 7 tRNA-Tyr [60306,60387] 35 (gta)\\n[10:46:49] 8 tRNA-Ser [60391,60477] 35 (gct)\\n[10:46:49] 9 tRNA-Arg [60479,60555] 36 (acg)\\n[10:46:49] 10 tRNA-Phe [60637,60712] 36 (gaa)\\n[10:46:49] 11 tRNA-Asn [60718,60792] 33 (gtt)\\n[10:46:49] 12 tRNA-Gln [60933,61007] 35 (ctg)\\n[10:46:49] 13 tRNA-Leu [61110,61186] 35 (caa)\\n[10:46:49] 14 tRNA-Leu [61193,61270] 35 (taa)\\n[10:46:49] 15 tRNA-Pro [61272,61345] 35 (tgg)\\n[10:46:49] 16 tRNA-Leu [61453,61529] 34 (gag)\\n[10:46:49] 17 tRNA-Leu [61641,61715] 35 (tag)\\n[10:46:49] 18 tRNA-Met [61725,61801] 34 (cat)\\n[10:46:49] 19 tRNA-Thr [61810,61882] 34 (tgt)\\n[10:46:49] 20 tRNA-Val [62063,62138] 34 (tac)\\n[10:46:49] 21 tRNA-Arg [62145,62220] 35 (tct)\\n[10:46:49] 22 tRNA-Ala [62312,62386] 35 (tgc)\\n[10:46:49] 23 tRNA-His [62392,62467] 35 (gtg)\\n[10:46:49] 24 tRNA-Ser [62473,62556] 34 (tga)\\n[10:46:49] 25 tRNA-Lys [62822,62892] 33 (ttt)\\n[10:46:49] 26 tRNA-Ser [68522,68615] 36 (gct)\\n[10:46:49] 1 tRNA-Asn [69400,69474] 33 (att)\\n[10:46:49] 2 tRNA-Leu [70868,70942] 35 (caa)\\n[10:46:49] 1 tRNA-Arg c[18559,18631] 34 (ccg)\\n[10:46:49] 1 tRNA-Pro [90049,90122] 35 (cgg)\\n[10:46:49] 1 tRNA-Ala [35429,35501] 34 (tgc)\\n[10:46:49] 1 tRNA-Lys c[48176,48247] 33 (ttt)\\n[10:46:49] 1 tRNA-Arg c[67940,68013] 35 (acg)\\n[10:46:49] 1 tRNA-Gly c[22073,22144] 33 (gcc)\\n[10:46:49] 2 tRNA-Met c[29076,29149] 33 (cat)\\n[10:46:49] 1 tRNA-Met [15220,15294] 35 (cat)\\n[10:46:49] 1 tRNA-Lys [26761,26833] 33 (ctt)\\n[10:46:49] 2 tRNA-Lys [36691,36760] 32 (ctt)\\n[10:46:49] 1 tRNA-Leu [9872,9953] 35 (cag)\\n[10:46:49] 1 tRNA-Val c[1148,1221] 35 (gac)\\n[10:46:49] 1 tRNA-Ala c[287,361] 34 (tgc)\\n[10:46:49] 2 tRNA-Ile c[445,520] 35 (gat)\\n[10:46:49] Found 71 tRNAs\\n[10:46:49] Predicting Ribosomal RNAs\\n[10:46:49] Running Barrnap with 4 threads\\n[10:46:51] Found 0 rRNAs\\n[10:46:51] Skipping ncRNA search, enable with --rfam if desired.\\n[10:46:51] Total of 70 tRNA + rRNA features\\n[10:46:51] Searching for CRISPR repeats\\n[10:46:51] CRISPR1 VRZE01000001.1 250752 with 44 spacers\\n[10:46:51] CRISPR2 VRZE01000004.1 77435 with 39 spacers\\n[10:46:51] CRISPR3 VRZE01000005.1 132564 with 11 spacers\\n[10:46:51] CRISPR4 VRZE01000007.1 60188 with 11 spacers\\n[10:46:51] CRISPR5 VRZE01000007.1 139846 with 4 spacers\\n[10:46:51] CRISPR6 VRZE01000009.1 123980 with 28 spacers\\n[10:46:52] CRISPR7 VRZE01000014.1 99202 with 21 spacers\\n[10:46:52] CRISPR8 VRZE01000015.1 140124 with 15 spacers\\n[10:46:52] CRISPR9 VRZE01000030.1 73890 with 38 spacers\\n[10:46:52] CRISPR10 VRZE01000051.1 26063 with 37 spacers\\n[10:46:52] CRISPR11 VRZE01000051.1 39879 with 19 spacers\\n[10:46:52] Found 11 CRISPRs\\n[10:46:52] Predicting coding sequences\\n[10:46:52] Contigs total 8131703 bp, so using single mode\\n[10:46:52] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_VRZE01\\\\/\\\\/Nostoc_VRZE01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:47:01] Excluding CDS which overlaps existing RNA (repeat_region) at VRZE01000005.1:132697..132972 on - strand\\n[10:47:01] Excluding CDS which overlaps existing RNA (repeat_region) at VRZE01000005.1:133177..133338 on + strand\\n[10:47:02] Excluding CDS which overlaps existing RNA (repeat_region) at VRZE01000007.1:60919..61134 on - strand\\n[10:47:02] Excluding CDS which overlaps existing RNA (repeat_region) at VRZE01000007.1:140023..140358 on + strand\\n[10:47:02] Excluding CDS which overlaps existing RNA (repeat_region) at VRZE01000009.1:125536..125868 on - strand\\n[10:47:03] Excluding CDS which overlaps existing RNA (repeat_region) at VRZE01000014.1:100626..100766 on + strand\\n[10:47:03] Excluding CDS which overlaps existing RNA (repeat_region) at VRZE01000015.1:139188..140147 on + strand\\n[10:47:03] Excluding CDS which overlaps existing RNA (repeat_region) at VRZE01000015.1:140567..140731 on + strand\\n[10:47:03] Excluding CDS which overlaps existing RNA (tRNA) at VRZE01000019.1:105625..105885 on - strand\\n[10:47:06] Found 6510 CDS\\n[10:47:06] Connecting features back to sequences\\n[10:47:06] Not using genus-specific database. Try --usegenus to enable it.\\n[10:47:06] Annotating CDS, please be patient.\\n[10:47:06] Will use 4 CPUs for similarity searching.\\n[10:47:07] There are still 6510 unannotated CDS left (started with 6510)\\n[10:47:07] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:47:07] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_VRZE01\\\\/\\\\/Nostoc_VRZE01\\\\.IS\\\\.tmp\\\\.5579\\\\.faa | parallel --gnu --plain -j 4 --block 273434 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_VRZE01\\\\/\\\\/Nostoc_VRZE01\\\\.IS\\\\.tmp\\\\.5579\\\\.blast 2> /dev/null\\n[10:47:29] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_VRZE01\\\\/\\\\/Nostoc_VRZE01\\\\.IS\\\\.tmp\\\\.5579\\\\.faa | parallel --gnu --plain -j 4 --block 273434 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_VRZE01\\\\/\\\\/Nostoc_VRZE01\\\\.IS\\\\.tmp\\\\.5579\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:47:30] This is prokka 1.14.6\\n[10:47:30] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:47:30] Homepage is https://github.com/tseemann/prokka\\n[10:47:30] Local time is Mon Feb 21 10:47:30 2022\\n[10:47:30] You are tiagoferreiraleao\\n[10:47:30] Operating system is darwin\\n[10:47:30] You have BioPerl 1.007002\\n[10:47:30] System has 4 cores.\\n[10:47:30] Option --cpu asked for 12 cores, but system only has 4\\n[10:47:30] Will use maximum of 4 cores.\\n[10:47:30] Annotating as >>> Bacteria <<<\\n[10:47:30] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQJ01.fasta' contents.\\n[10:47:30] Setting --locustag FHJMCKOH from MD5 f136c4818d89e01459528080c1592350\\n[10:47:30] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQJ01/\\n[10:47:30] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQJ01\\\\/\\n[10:47:30] Using filename prefix: Fischerella_NMQJ01.XXX\\n[10:47:30] Setting HMMER_NCPU=1\\n[10:47:30] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQJ01//Fischerella_NMQJ01.log\\n[10:47:30] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQJ01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQJ01/ --prefix Fischerella_NMQJ01 --cpus 12\\n[10:47:30] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:47:30] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:47:30] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:47:30] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:47:30] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:47:30] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:47:30] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:47:30] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:47:30] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:47:30] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:47:30] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:47:30] Looking for 'egrep' - found /usr/bin/egrep\\n[10:47:30] Looking for 'find' - found /usr/bin/find\\n[10:47:30] Looking for 'grep' - found /usr/bin/grep\\n[10:47:30] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:47:30] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:47:30] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:47:30] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:47:30] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:47:30] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:47:30] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:47:30] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:47:31] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:47:31] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:47:31] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:47:31] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:47:31] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:47:31] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:47:31] Looking for 'sed' - found /usr/bin/sed\\n[10:47:31] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:47:31] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:47:31] Using genetic code table 11.\\n[10:47:31] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQJ01.fasta\\n[10:47:31] Wrote 873 contigs totalling 6241889 bp.\\n[10:47:31] Predicting tRNAs and tmRNAs\\n[10:47:31] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQJ01\\\\/\\\\/Fischerella_NMQJ01\\\\.fna\\n[10:47:33] 1 tRNA-Ser c[3041,3127] 36 (cga)\\n[10:47:33] 1 tRNA-Glu [7154,7226] 35 (ttc)\\n[10:47:33] 1 tRNA-Arg [3772,3845] 35 (ccg)\\n[10:47:33] 1 tRNA-Pyl [5310,5397] 30 (cta)\\n[10:47:33] 1 tRNA-Gln c[4867,4938] 33 (ttg)\\n[10:47:33] 1 tRNA-Gly c[4196,4266] 33 (tcc)\\n[10:47:33] 1 tRNA-Arg [10222,10296] 35 (tct)\\n[10:47:33] 1 tRNA-Val c[8785,8857] 34 (tac)\\n[10:47:33] 2 tRNA-Phe [16413,16485] 34 (gaa)\\n[10:47:33] 1 tRNA-Lys [5141,5214] 34 (ttt)\\n[10:47:33] 1 tRNA-Ser c[9834,9920] 36 (tga)\\n[10:47:33] 1 tRNA-Cys c[4125,4196] 33 (gca)\\n[10:47:33] 1 tRNA-Pro c[533,607] 35 (tgg)\\n[10:47:33] 1 tRNA-Met [7633,7705] 34 (cat)\\n[10:47:33] 1 tRNA-Leu [7129,7212] 36 (tag)\\n[10:47:33] 1 tRNA-Asn [2622,2695] 34 (gtt)\\n[10:47:33] 1 tRNA-Gly c[371,442] 33 (gcc)\\n[10:47:33] 1 tRNA-Gly [10417,10488] 33 (ccc)\\n[10:47:33] 1 tRNA-Leu c[512,596] 35 (caa)\\n[10:47:33] 1 tRNA-Leu [2116,2198] 36 (gag)\\n[10:47:33] 1 tRNA-Leu [10155,10237] 35 (cag)\\n[10:47:33] 1 tRNA-Asp [234,307] 35 (gtc)\\n[10:47:33] 1 tRNA-Trp c[17117,17190] 34 (cca)\\n[10:47:33] 1 tRNA-Pro [2185,2258] 35 (cgg)\\n[10:47:33] 1 tRNA-Met c[40890,40966] 35 (cat)\\n[10:47:33] 1 tRNA-Met [16416,16491] 36 (cat)\\n[10:47:33] 1 tRNA-Ser c[1500,1588] 36 (gga)\\n[10:47:33] 1 tRNA-Thr [381,452] 33 (cgt)\\n[10:47:33] 2 tRNA-Ala [1882,1954] 34 (ggc)\\n[10:47:33] 1 tRNA-Arg c[6253,6326] 35 (acg)\\n[10:47:34] 1 tRNA-Pro [636,709] 35 (ggg)\\n[10:47:34] 1 tRNA-Thr c[8307,8378] 33 (tgt)\\n[10:47:34] 1 tRNA-His [1787,1860] 34 (gtg)\\n[10:47:34] 1 tRNA-Val c[63628,63701] 35 (gac)\\n[10:47:34] 1 tRNA-Arg [827,899] 34 (tcg)\\n[10:47:34] 2 tRNA-Asn [1575,1648] 34 (att)\\n[10:47:34] 1 tRNA-Ser c[2635,2728] 36 (gct)\\n[10:47:34] 1 tRNA-Thr c[3308,3379] 33 (ggt)\\n[10:47:34] 2 tRNA-Tyr c[3445,3529] 35 (gta)\\n[10:47:34] 1 tmRNA c[157,543] 100,153 ANNIVKFARKEALVAA**\\n[10:47:34] Found 40 tRNAs\\n[10:47:34] Predicting Ribosomal RNAs\\n[10:47:34] Running Barrnap with 4 threads\\n[10:47:35] 1 NMQJ01000095.1 6 23S ribosomal RNA (partial)\\n[10:47:35] 2 NMQJ01000527.1 12 16S ribosomal RNA\\n[10:47:35] 3 NMQJ01000811.1 239 23S ribosomal RNA (partial)\\n[10:47:35] Found 3 rRNAs\\n[10:47:35] Skipping ncRNA search, enable with --rfam if desired.\\n[10:47:35] Total of 42 tRNA + rRNA features\\n[10:47:35] Searching for CRISPR repeats\\n[10:47:36] CRISPR1 NMQJ01000136.1 5864 with 15 spacers\\n[10:47:36] CRISPR2 NMQJ01000172.1 461 with 25 spacers\\n[10:47:36] CRISPR3 NMQJ01000190.1 6700 with 3 spacers\\n[10:47:36] CRISPR4 NMQJ01000279.1 1682 with 9 spacers\\n[10:47:36] CRISPR5 NMQJ01000315.1 3305 with 29 spacers\\n[10:47:36] CRISPR6 NMQJ01000530.1 1552 with 3 spacers\\n[10:47:36] Found 6 CRISPRs\\n[10:47:36] Predicting coding sequences\\n[10:47:36] Contigs total 6241889 bp, so using single mode\\n[10:47:36] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQJ01\\\\/\\\\/Fischerella_NMQJ01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:47:42] Excluding CDS which overlaps existing RNA (tRNA) at NMQJ01000061.1:4971..5462 on - strand\\n[10:47:42] Excluding CDS which overlaps existing RNA (rRNA) at NMQJ01000095.1:401..580 on + strand\\n[10:47:42] Excluding CDS which overlaps existing RNA (tRNA) at NMQJ01000176.1:4003..4143 on + strand\\n[10:47:43] Excluding CDS which overlaps existing RNA (tRNA) at NMQJ01000230.1:2637..3935 on + strand\\n[10:47:44] Excluding CDS which overlaps existing RNA (rRNA) at NMQJ01000527.1:102..368 on - strand\\n[10:47:44] Excluding CDS which overlaps existing RNA (rRNA) at NMQJ01000811.1:525..701 on + strand\\n[10:47:45] Found 5113 CDS\\n[10:47:45] Connecting features back to sequences\\n[10:47:45] Not using genus-specific database. Try --usegenus to enable it.\\n[10:47:45] Annotating CDS, please be patient.\\n[10:47:45] Will use 4 CPUs for similarity searching.\\n[10:47:46] There are still 5113 unannotated CDS left (started with 5113)\\n[10:47:46] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:47:46] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQJ01\\\\/\\\\/Fischerella_NMQJ01\\\\.IS\\\\.tmp\\\\.5691\\\\.faa | parallel --gnu --plain -j 4 --block 207520 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQJ01\\\\/\\\\/Fischerella_NMQJ01\\\\.IS\\\\.tmp\\\\.5691\\\\.blast 2> /dev/null\\n[10:48:03] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQJ01\\\\/\\\\/Fischerella_NMQJ01\\\\.IS\\\\.tmp\\\\.5691\\\\.faa | parallel --gnu --plain -j 4 --block 207520 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQJ01\\\\/\\\\/Fischerella_NMQJ01\\\\.IS\\\\.tmp\\\\.5691\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:48:03] This is prokka 1.14.6\\n[10:48:03] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:48:03] Homepage is https://github.com/tseemann/prokka\\n[10:48:03] Local time is Mon Feb 21 10:48:03 2022\\n[10:48:03] You are tiagoferreiraleao\\n[10:48:03] Operating system is darwin\\n[10:48:03] You have BioPerl 1.007002\\n[10:48:04] System has 4 cores.\\n[10:48:04] Option --cpu asked for 12 cores, but system only has 4\\n[10:48:04] Will use maximum of 4 cores.\\n[10:48:04] Annotating as >>> Bacteria <<<\\n[10:48:04] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACKZN01.fasta' contents.\\n[10:48:04] Setting --locustag PLOGILKO from MD5 958025486870e4a8c03f26cf79bee4fc\\n[10:48:04] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACKZN01/\\n[10:48:04] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZN01\\\\/\\n[10:48:04] Using filename prefix: Nostoc_JACKZN01.XXX\\n[10:48:04] Setting HMMER_NCPU=1\\n[10:48:04] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACKZN01//Nostoc_JACKZN01.log\\n[10:48:04] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACKZN01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACKZN01/ --prefix Nostoc_JACKZN01 --cpus 12\\n[10:48:04] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:48:04] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:48:04] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:48:04] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:48:04] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:48:04] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:48:04] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:48:04] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:48:04] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:48:04] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:48:04] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:48:04] Looking for 'egrep' - found /usr/bin/egrep\\n[10:48:04] Looking for 'find' - found /usr/bin/find\\n[10:48:04] Looking for 'grep' - found /usr/bin/grep\\n[10:48:04] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:48:04] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:48:04] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:48:04] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:48:04] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:48:04] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:48:04] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:48:04] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:48:04] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:48:04] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:48:04] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:48:04] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:48:04] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:48:04] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:48:04] Looking for 'sed' - found /usr/bin/sed\\n[10:48:04] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:48:05] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:48:05] Using genetic code table 11.\\n[10:48:05] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACKZN01.fasta\\n[10:48:05] Wrote 1244 contigs totalling 8288950 bp.\\n[10:48:05] Predicting tRNAs and tmRNAs\\n[10:48:05] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZN01\\\\/\\\\/Nostoc_JACKZN01\\\\.fna\\n[10:48:07] 1 tRNA-Asp [54312,54389] 38 (gtc)\\n[10:48:07] 2 tRNA-Glu [54392,54466] 35 (ctc)\\n[10:48:07] 3 tRNA-Cys [54474,54548] 33 (gca)\\n[10:48:07] 4 tRNA-Met [54554,54628] 33 (cat)\\n[10:48:07] 5 tRNA-Trp [54630,54702] 34 (cca)\\n[10:48:07] 6 tRNA-Tyr [54937,55019] 36 (gta)\\n[10:48:07] 7 tRNA-Ser [55023,55109] 33 (gct)\\n[10:48:07] 8 tRNA-Arg [55113,55188] 35 (acg)\\n[10:48:07] 9 tRNA-Phe [55193,55267] 35 (gaa)\\n[10:48:07] 10 tRNA-Asn [55276,55350] 33 (gtt)\\n[10:48:07] 11 tRNA-Gln [55438,55512] 35 (ttg)\\n[10:48:07] 12 tRNA-Pro [55517,55592] 36 (tgg)\\n[10:48:07] 13 tRNA-Lys [55597,55671] 34 (ttt)\\n[10:48:07] 14 tRNA-Leu [55674,55748] 35 (taa)\\n[10:48:07] 15 tRNA-Thr [55758,55830] 34 (tgt)\\n[10:48:07] 16 tRNA-Gly [55836,55907] 33 (tcc)\\n[10:48:07] 17 tRNA-Val [55912,55987] 34 (tac)\\n[10:48:07] 18 tRNA-Arg [55994,56069] 34 (tct)\\n[10:48:07] 19 tRNA-Ala [56072,56147] 36 (tgc)\\n[10:48:07] 20 tRNA-His [56153,56228] 35 (gtg)\\n[10:48:07] 21 tRNA-Ser [56235,56318] 34 (tga)\\n[10:48:07] 1 tRNA-Pro [3873,3946] 35 (ggg)\\n[10:48:07] 1 tRNA-Leu c[27804,27888] 36 (gag)\\n[10:48:07] 1 tRNA-Trp [31530,31603] 34 (cca)\\n[10:48:07] 1 tRNA-Val [9503,9576] 35 (gac)\\n[10:48:07] 1 tRNA-Lys c[27427,27500] 33 (ctt)\\n[10:48:07] 1 tRNA-Ala c[16044,16116] 34 (cgc)\\n[10:48:07] 1 tRNA-Val [5634,5706] 34 (tac)\\n[10:48:07] 1 tRNA-Gly c[19822,19893] 33 (gcc)\\n[10:48:07] 1 tRNA-Pro c[16942,17015] 35 (cgg)\\n[10:48:07] 1 tRNA-Pro c[7686,7759] 35 (tgg)\\n[10:48:07] 1 tRNA-Ser c[4577,4663] 36 (cga)\\n[10:48:07] 1 tRNA-Arg c[9874,9950] 36 (cct)\\n[10:48:07] 1 tRNA-Met c[20110,20186] 35 (cat)\\n[10:48:07] 2 tRNA-Met c[20186,20261] 34 (cat)\\n[10:48:07] 1 tRNA-Lys [9623,9694] 33 (ttt)\\n[10:48:07] 1 tRNA-Gly c[12766,12836] 33 (tcc)\\n[10:48:07] 1 tmRNA c[1538,1925] 101,151 ANNIVSFARKDALVAA*\\n[10:48:07] 1 tRNA-Phe c[9980,10053] 34 (gaa)\\n[10:48:07] 2 tRNA-Pyl c[11641,11713] 34 (cta)\\n[10:48:07] 1 tRNA-Met [12077,12150] 33 (cat)\\n[10:48:07] 2 tRNA-Thr c[13534,13605] 33 (tgt)\\n[10:48:07] 1 tRNA-Leu [216,297] 35 (cag)\\n[10:48:07] 1 tRNA-Asn [2483,2554] 33 (gtt)\\n[10:48:07] 1 tRNA-Leu [14623,14706] 36 (tag)\\n[10:48:07] 1 tRNA-Cys c[5761,5831] 33 (gca)\\n[10:48:07] 1 tRNA-Ala c[6723,6797] 34 (tgc)\\n[10:48:07] 2 tRNA-Ile c[6879,6954] 35 (gat)\\n[10:48:07] 1 tRNA-Asp c[9486,9559] 35 (gtc)\\n[10:48:07] 1 tRNA-Arg c[9802,9875] 35 (acg)\\n[10:48:07] 1 tRNA-Gly [2547,2618] 33 (ccc)\\n[10:48:07] 1 tRNA-Thr [498,569] 33 (cgt)\\n[10:48:07] 1 tRNA-Phe c[5462,5534] 34 (gaa)\\n[10:48:07] 1 tRNA-Ser c[5720,5811] 35 (gct)\\n[10:48:07] 1 tRNA-Trp [2101,2174] 36 (cca)\\n[10:48:07] 1 tRNA-Lys c[1520,1591] 33 (ttt)\\n[10:48:07] 2 tRNA-His c[1959,2034] 35 (gtg)\\n[10:48:07] 3 tRNA-Ala c[2040,2114] 35 (tgc)\\n[10:48:07] 4 tRNA-Ala c[2124,2196] 34 (ggc)\\n[10:48:07] 5 tRNA-Arg c[2201,2276] 35 (tct)\\n[10:48:07] 6 tRNA-Gln c[3328,3403] 34 (ctg)\\n[10:48:07] 7 tRNA-Asn c[3409,3483] 33 (gtt)\\n[10:48:07] 8 tRNA-Phe c[3489,3564] 36 (gaa)\\n[10:48:07] 9 tRNA-Arg c[3583,3679] 36 (acg)\\n[10:48:07] 10 tRNA-Ser c[3682,3773] 36 (gct)\\n[10:48:07] 11 tRNA-Glu c[3906,3979] 35 (ttc)\\n[10:48:07] 12 tRNA-Trp c[4067,4141] 35 (cca)\\n[10:48:07] 13 tRNA-Met c[4144,4218] 33 (cat)\\n[10:48:07] 14 tRNA-Cys c[4228,4302] 35 (gca)\\n[10:48:07] 15 tRNA-Glu c[4306,4382] 35 (ctc)\\n[10:48:07] 16 tRNA-Asp c[4386,4461] 36 (gtc)\\n[10:48:08] 1 tRNA-Glu c[3309,3382] 35 (ttc)\\n[10:48:08] 1 tRNA-Met c[1284,1358] 35 (cat)\\n[10:48:08] 1 tRNA-Arg c[3454,3526] 34 (ccg)\\n[10:48:08] 1 tRNA-His c[113,186] 34 (gtg)\\n[10:48:08] 1 tRNA-Gln c[2093,2166] 34 (ttg)\\n[10:48:08] 2 tRNA-Leu c[3985,4069] 35 (caa)\\n[10:48:08] 1 tRNA-Ala [3115,3189] 35 (tgc)\\n[10:48:08] 1 tRNA-Ser c[259,345] 36 (tga)\\n[10:48:08] 2 tRNA-Tyr [2645,2730] 35 (gta)\\n[10:48:08] 3 tRNA-Thr [2818,2889] 33 (ggt)\\n[10:48:08] 1 tRNA-Arg [1937,2011] 35 (tct)\\n[10:48:08] 1 tRNA-Ser [504,590] 36 (gga)\\n[10:48:08] 1 tRNA-Ala [1318,1390] 34 (ggc)\\n[10:48:08] Found 84 tRNAs\\n[10:48:08] Predicting Ribosomal RNAs\\n[10:48:08] Running Barrnap with 4 threads\\n[10:48:10] 1 JACKZN010000184.1 3587 23S ribosomal RNA\\n[10:48:10] 2 JACKZN010000184.1 7086 16S ribosomal RNA\\n[10:48:10] Found 2 rRNAs\\n[10:48:10] Skipping ncRNA search, enable with --rfam if desired.\\n[10:48:10] Total of 85 tRNA + rRNA features\\n[10:48:10] Searching for CRISPR repeats\\n[10:48:10] CRISPR1 JACKZN010000007.1 13487 with 3 spacers\\n[10:48:11] CRISPR2 JACKZN010000123.1 5149 with 3 spacers\\n[10:48:11] CRISPR3 JACKZN010000123.1 6018 with 6 spacers\\n[10:48:11] CRISPR4 JACKZN010000123.1 8700 with 3 spacers\\n[10:48:11] CRISPR5 JACKZN010000128.1 8040 with 6 spacers\\n[10:48:11] CRISPR6 JACKZN010000128.1 9936 with 10 spacers\\n[10:48:11] CRISPR7 JACKZN010000424.1 1552 with 12 spacers\\n[10:48:11] CRISPR8 JACKZN010000433.1 2624 with 7 spacers\\n[10:48:11] CRISPR9 JACKZN010000481.1 152 with 11 spacers\\n[10:48:11] CRISPR10 JACKZN010000481.1 1205 with 6 spacers\\n[10:48:11] CRISPR11 JACKZN010000481.1 2335 with 3 spacers\\n[10:48:11] CRISPR12 JACKZN010000481.1 2544 with 8 spacers\\n[10:48:11] CRISPR13 JACKZN010000481.1 3137 with 10 spacers\\n[10:48:11] CRISPR14 JACKZN010000481.1 5084 with 9 spacers\\n[10:48:11] CRISPR15 JACKZN010000851.1 292 with 7 spacers\\n[10:48:11] Found 15 CRISPRs\\n[10:48:11] Predicting coding sequences\\n[10:48:11] Contigs total 8288950 bp, so using single mode\\n[10:48:11] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZN01\\\\/\\\\/Nostoc_JACKZN01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:48:19] Excluding CDS which overlaps existing RNA (tRNA) at JACKZN010000012.1:9522..9716 on - strand\\n[10:48:20] Excluding CDS which overlaps existing RNA (tRNA) at JACKZN010000118.1:11963..12157 on + strand\\n[10:48:20] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZN010000123.1:6046..6429 on + strand\\n[10:48:20] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZN010000128.1:7970..8155 on + strand\\n[10:48:20] Excluding CDS which overlaps existing RNA (rRNA) at JACKZN010000184.1:7128..7442 on - strand\\n[10:48:21] Excluding CDS which overlaps existing RNA (tRNA) at JACKZN010000363.1:1764..2000 on - strand\\n[10:48:21] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZN010000481.1:327..599 on - strand\\n[10:48:21] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZN010000481.1:1051..1212 on - strand\\n[10:48:21] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZN010000481.1:1391..1585 on + strand\\n[10:48:21] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZN010000481.1:3359..3790 on + strand\\n[10:48:21] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZN010000481.1:5620..5742 on + strand\\n[10:48:21] Excluding CDS which overlaps existing RNA (tRNA) at JACKZN010000515.1:3180..3512 on - strand\\n[10:48:22] Excluding CDS which overlaps existing RNA (tRNA) at JACKZN010000746.1:1255..1395 on - strand\\n[10:48:22] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZN010000851.1:122..391 on + strand\\n[10:48:22] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZN010000851.1:614..811 on + strand\\n[10:48:22] Found 7004 CDS\\n[10:48:22] Connecting features back to sequences\\n[10:48:22] Not using genus-specific database. Try --usegenus to enable it.\\n[10:48:22] Annotating CDS, please be patient.\\n[10:48:22] Will use 4 CPUs for similarity searching.\\n[10:48:24] There are still 7004 unannotated CDS left (started with 7004)\\n[10:48:24] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:48:24] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZN01\\\\/\\\\/Nostoc_JACKZN01\\\\.IS\\\\.tmp\\\\.5803\\\\.faa | parallel --gnu --plain -j 4 --block 272317 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZN01\\\\/\\\\/Nostoc_JACKZN01\\\\.IS\\\\.tmp\\\\.5803\\\\.blast 2> /dev/null\\n[10:48:29] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZN01\\\\/\\\\/Nostoc_JACKZN01\\\\.IS\\\\.tmp\\\\.5803\\\\.faa | parallel --gnu --plain -j 4 --block 272317 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZN01\\\\/\\\\/Nostoc_JACKZN01\\\\.IS\\\\.tmp\\\\.5803\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:48:29] This is prokka 1.14.6\\n[10:48:29] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:48:29] Homepage is https://github.com/tseemann/prokka\\n[10:48:29] Local time is Mon Feb 21 10:48:29 2022\\n[10:48:29] You are tiagoferreiraleao\\n[10:48:29] Operating system is darwin\\n[10:48:29] You have BioPerl 1.007002\\n[10:48:29] System has 4 cores.\\n[10:48:29] Option --cpu asked for 12 cores, but system only has 4\\n[10:48:29] Will use maximum of 4 cores.\\n[10:48:29] Annotating as >>> Bacteria <<<\\n[10:48:29] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LWTK01.fasta' contents.\\n[10:48:29] Setting --locustag GEMLPBEK from MD5 0e659be48491850978789735f62ccd89\\n[10:48:29] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LWTK01/\\n[10:48:29] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LWTK01\\\\/\\n[10:48:29] Using filename prefix: Nostoc_LWTK01.XXX\\n[10:48:29] Setting HMMER_NCPU=1\\n[10:48:29] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LWTK01//Nostoc_LWTK01.log\\n[10:48:29] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LWTK01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LWTK01/ --prefix Nostoc_LWTK01 --cpus 12\\n[10:48:29] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:48:29] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:48:29] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:48:29] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:48:29] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:48:29] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:48:30] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:48:30] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:48:30] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:48:30] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:48:30] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:48:30] Looking for 'egrep' - found /usr/bin/egrep\\n[10:48:30] Looking for 'find' - found /usr/bin/find\\n[10:48:30] Looking for 'grep' - found /usr/bin/grep\\n[10:48:30] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:48:30] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:48:30] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:48:30] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:48:30] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:48:30] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:48:30] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:48:30] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:48:30] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:48:30] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:48:30] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:48:30] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:48:30] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:48:30] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:48:30] Looking for 'sed' - found /usr/bin/sed\\n[10:48:30] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:48:31] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:48:31] Using genetic code table 11.\\n[10:48:31] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LWTK01.fasta\\n[10:48:31] Wrote 113 contigs totalling 8458523 bp.\\n[10:48:31] Predicting tRNAs and tmRNAs\\n[10:48:31] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LWTK01\\\\/\\\\/Nostoc_LWTK01\\\\.fna\\n[10:48:34] 1 tRNA-Val c[19993,20065] 33 (cac)\\n[10:48:34] 2 tRNA-Arg [204034,204108] 35 (tct)\\n[10:48:34] 3 tRNA-Val c[229626,229698] 34 (tac)\\n[10:48:34] 1 tRNA-Ile [344,419] 35 (gat)\\n[10:48:34] 2 tRNA-Ala [511,585] 34 (tgc)\\n[10:48:34] 1 tRNA-Arg c[159183,159255] 34 (ccg)\\n[10:48:34] 2 tRNA-Val [248965,249038] 35 (gac)\\n[10:48:34] 1 tRNA-Leu c[137129,137210] 35 (cag)\\n[10:48:34] 1 tRNA-Thr c[102259,102330] 33 (tgt)\\n[10:48:34] 2 tRNA-Gly c[213651,213722] 33 (gcc)\\n[10:48:34] 3 tRNA-Met c[220567,220640] 33 (cat)\\n[10:48:34] 1 tRNA-Glu c[77266,77338] 35 (ttc)\\n[10:48:34] 2 tRNA-Asp [101424,101499] 36 (gtc)\\n[10:48:34] 3 tRNA-Glu [101503,101579] 35 (ctc)\\n[10:48:34] 4 tRNA-Cys [101583,101655] 35 (gca)\\n[10:48:34] 5 tRNA-Met [101664,101737] 32 (cat)\\n[10:48:34] tRNA [101741,101815] is a pseudo/wacky gene - skipping.\\n[10:48:34] 7 tRNA-Glu [101939,102014] 35 (ttc)\\n[10:48:34] 8 tRNA-Tyr [102020,102102] 36 (gta)\\n[10:48:34] 9 tRNA-Ser [102105,102192] 36 (gct)\\n[10:48:34] 10 tRNA-Arg [102194,102270] 36 (acg)\\n[10:48:34] 11 tRNA-Phe [102435,102510] 36 (gaa)\\n[10:48:34] 12 tRNA-Asn [102516,102588] 33 (gtt)\\n[10:48:34] 13 tRNA-Gln [102686,102760] 35 (ctg)\\n[10:48:34] 14 tRNA-Gln [102763,102838] 34 (ttg)\\n[10:48:34] 15 tRNA-Leu [102844,102920] 35 (caa)\\n[10:48:34] 16 tRNA-Leu [102927,103002] 35 (taa)\\n[10:48:34] 17 tRNA-Pro [103006,103079] 35 (tgg)\\n[10:48:34] 18 tRNA-Leu [103083,103157] 35 (cag)\\n[10:48:34] 19 tRNA-Pro [103245,103318] 34 (tgg)\\n[10:48:34] 20 tRNA-Leu [103322,103394] 34 (tag)\\n[10:48:34] 21 tRNA-Met [103405,103481] 34 (cat)\\n[10:48:34] 22 tRNA-Thr [103675,103747] 34 (tgt)\\n[10:48:34] 23 tRNA-Val [103829,103904] 34 (tac)\\n[10:48:34] 24 tRNA-Arg [103911,103986] 35 (tct)\\n[10:48:34] 25 tRNA-Ala [104030,104104] 35 (tgc)\\n[10:48:34] 26 tRNA-His [104110,104185] 35 (gtg)\\n[10:48:34] 27 tRNA-Ser [104191,104274] 34 (tga)\\n[10:48:34] 28 tRNA-Lys [104427,104497] 33 (ttt)\\n[10:48:34] 29 tRNA-Ser [108578,108671] 36 (gct)\\n[10:48:34] 1 tRNA-Thr [206809,206880] 33 (cgt)\\n[10:48:34] 1 tRNA-Leu [457477,457561] 35 (caa)\\n[10:48:34] 2 tRNA-Gln [462413,462486] 34 (ttg)\\n[10:48:34] 3 tRNA-Leu c[599114,599194] 35 (gag)\\n[10:48:34] 4 tRNA-Ile c[697215,697290] 34 (tat)\\n[10:48:34] 5 tRNA-Met c[698911,698987] 35 (cat)\\n[10:48:34] 6 tRNA-Met c[698989,699064] 34 (cat)\\n[10:48:34] 1 tRNA-Ala [70499,70571] 34 (ggc)\\n[10:48:34] 2 tRNA-Ala [168131,168203] 34 (tgc)\\n[10:48:34] 1 tRNA-Lys [123228,123299] 33 (ttt)\\n[10:48:34] 1 tRNA-Ser c[137765,137851] 36 (tga)\\n[10:48:34] 2 tRNA-Tyr [140106,140190] 35 (gta)\\n[10:48:34] 3 tRNA-Thr [140268,140339] 33 (ggt)\\n[10:48:34] 1 tRNA-Asp c[67468,67541] 35 (gtc)\\n[10:48:34] 1 tRNA-Phe c[65578,65650] 34 (gaa)\\n[10:48:34] 1 tRNA-Leu [31696,31779] 36 (tag)\\n[10:48:34] 2 tRNA-Ser c[115754,115841] 35 (gga)\\n[10:48:34] 1 tRNA-Pro [136746,136819] 35 (ggg)\\n[10:48:34] 1 tRNA-Gly c[71517,71587] 33 (tcc)\\n[10:48:34] 1 tRNA-Gly c[16357,16428] 33 (ccc)\\n[10:48:34] 2 tRNA-Cys c[107595,107666] 33 (gca)\\n[10:48:34] 1 tRNA-His [74000,74073] 34 (gtg)\\n[10:48:34] 1 tRNA-Met [21665,21740] 36 (cat)\\n[10:48:34] 1 tRNA-Ala [23846,23918] 34 (cgc)\\n[10:48:34] 2 tRNA-Trp c[39376,39449] 34 (cca)\\n[10:48:34] 1 tRNA-Pro c[11891,11964] 35 (tgg)\\n[10:48:34] 1 tmRNA c[248168,248555] 101,151 ANNIVKFARREALVAA*\\n[10:48:34] 1 tRNA-Pro [230888,230961] 35 (cgg)\\n[10:48:34] 2 tRNA-Ser [231115,231201] 36 (cga)\\n[10:48:34] 3 tRNA-Asn c[346356,346429] 34 (gtt)\\n[10:48:34] 1 tRNA-Ile [1775,1851] 35 (gat)\\n[10:48:34] 2 tRNA-Ala [1862,1937] 34 (tgc)\\n[10:48:34] 1 tRNA-Lys [323543,323615] 33 (ctt)\\n[10:48:34] 1 tRNA-Lys [204406,204480] 33 (ctt)\\n[10:48:34] 2 tRNA-SeC [205875,205947] 33 (tca)\\n[10:48:34] 3 tRNA-Arg c[306663,306736] 35 (acg)\\n[10:48:34] Found 75 tRNAs\\n[10:48:34] Predicting Ribosomal RNAs\\n[10:48:34] Running Barrnap with 4 threads\\n[10:48:37] 1 LWTK01000052.1 94378 16S ribosomal RNA (partial)\\n[10:48:37] 2 LWTK01000089.1 161 16S ribosomal RNA\\n[10:48:37] 3 LWTK01000089.1 2171 23S ribosomal RNA\\n[10:48:37] 4 LWTK01000089.1 5195 5S ribosomal RNA\\n[10:48:37] 5 LWTK01000093.1 128 23S ribosomal RNA\\n[10:48:37] 6 LWTK01000102.1 210 16S ribosomal RNA\\n[10:48:37] Found 6 rRNAs\\n[10:48:37] Skipping ncRNA search, enable with --rfam if desired.\\n[10:48:37] Total of 80 tRNA + rRNA features\\n[10:48:37] Searching for CRISPR repeats\\n[10:48:37] CRISPR1 LWTK01000030.1 150847 with 3 spacers\\n[10:48:37] CRISPR2 LWTK01000041.1 142771 with 5 spacers\\n[10:48:38] CRISPR3 LWTK01000053.1 80424 with 15 spacers\\n[10:48:38] CRISPR4 LWTK01000084.1 173111 with 8 spacers\\n[10:48:38] CRISPR5 LWTK01000105.1 154045 with 8 spacers\\n[10:48:38] Found 5 CRISPRs\\n[10:48:38] Predicting coding sequences\\n[10:48:38] Contigs total 8458523 bp, so using single mode\\n[10:48:38] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LWTK01\\\\/\\\\/Nostoc_LWTK01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:48:46] Excluding CDS which overlaps existing RNA (tRNA) at LWTK01000033.1:102877..103104 on - strand\\n[10:48:46] Excluding CDS which overlaps existing RNA (tRNA) at LWTK01000033.1:103973..104407 on - strand\\n[10:48:48] Excluding CDS which overlaps existing RNA (tRNA) at LWTK01000057.1:39261..39473 on - strand\\n[10:48:49] Excluding CDS which overlaps existing RNA (rRNA) at LWTK01000089.1:879..1031 on + strand\\n[10:48:49] Excluding CDS which overlaps existing RNA (rRNA) at LWTK01000089.1:1347..1607 on + strand\\n[10:48:49] Excluding CDS which overlaps existing RNA (rRNA) at LWTK01000102.1:1339..1620 on + strand\\n[10:48:50] Found 7077 CDS\\n[10:48:50] Connecting features back to sequences\\n[10:48:50] Not using genus-specific database. Try --usegenus to enable it.\\n[10:48:50] Annotating CDS, please be patient.\\n[10:48:50] Will use 4 CPUs for similarity searching.\\n[10:48:51] There are still 7077 unannotated CDS left (started with 7077)\\n[10:48:51] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:48:51] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LWTK01\\\\/\\\\/Nostoc_LWTK01\\\\.IS\\\\.tmp\\\\.5915\\\\.faa | parallel --gnu --plain -j 4 --block 285384 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LWTK01\\\\/\\\\/Nostoc_LWTK01\\\\.IS\\\\.tmp\\\\.5915\\\\.blast 2> /dev/null\\n[10:49:03] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LWTK01\\\\/\\\\/Nostoc_LWTK01\\\\.IS\\\\.tmp\\\\.5915\\\\.faa | parallel --gnu --plain -j 4 --block 285384 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LWTK01\\\\/\\\\/Nostoc_LWTK01\\\\.IS\\\\.tmp\\\\.5915\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:49:03] This is prokka 1.14.6\\n[10:49:03] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:49:03] Homepage is https://github.com/tseemann/prokka\\n[10:49:03] Local time is Mon Feb 21 10:49:03 2022\\n[10:49:03] You are tiagoferreiraleao\\n[10:49:03] Operating system is darwin\\n[10:49:03] You have BioPerl 1.007002\\n[10:49:03] System has 4 cores.\\n[10:49:03] Option --cpu asked for 12 cores, but system only has 4\\n[10:49:03] Will use maximum of 4 cores.\\n[10:49:03] Annotating as >>> Bacteria <<<\\n[10:49:03] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAB01.fasta' contents.\\n[10:49:03] Setting --locustag AEJPJAJK from MD5 ae393a347cc120a7cd8bb18e1e05a171\\n[10:49:03] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAB01/\\n[10:49:03] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAB01\\\\/\\n[10:49:03] Using filename prefix: Fischerella_NHAB01.XXX\\n[10:49:03] Setting HMMER_NCPU=1\\n[10:49:03] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAB01//Fischerella_NHAB01.log\\n[10:49:03] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAB01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAB01/ --prefix Fischerella_NHAB01 --cpus 12\\n[10:49:03] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:49:03] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:49:03] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:49:03] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:49:03] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:49:03] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:49:03] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:49:03] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:49:04] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:49:04] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:49:04] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:49:04] Looking for 'egrep' - found /usr/bin/egrep\\n[10:49:04] Looking for 'find' - found /usr/bin/find\\n[10:49:04] Looking for 'grep' - found /usr/bin/grep\\n[10:49:04] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:49:04] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:49:04] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:49:04] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:49:04] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:49:04] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:49:04] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:49:04] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:49:04] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:49:04] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:49:04] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:49:04] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:49:04] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:49:04] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:49:04] Looking for 'sed' - found /usr/bin/sed\\n[10:49:04] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:49:05] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:49:05] Using genetic code table 11.\\n[10:49:05] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAB01.fasta\\n[10:49:05] Wrote 128 contigs totalling 5405800 bp.\\n[10:49:05] Predicting tRNAs and tmRNAs\\n[10:49:05] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAB01\\\\/\\\\/Fischerella_NHAB01\\\\.fna\\n[10:49:07] 1 tRNA-Cys [4024,4095] 33 (gca)\\n[10:49:07] 1 tRNA-Arg [40853,40928] 36 (cct)\\n[10:49:07] 1 tRNA-Gln c[17757,17827] 32 (ttg)\\n[10:49:07] 1 tRNA-Thr [49961,50032] 33 (cgt)\\n[10:49:07] 2 tRNA-Lys [80553,80626] 34 (ttt)\\n[10:49:07] 1 tRNA-Gly [12754,12825] 33 (gcc)\\n[10:49:07] 1 tRNA-Leu c[57480,57562] 36 (gag)\\n[10:49:07] 1 tRNA-Arg [1527,1600] 35 (ccg)\\n[10:49:07] 1 tRNA-Asp c[60480,60553] 35 (gtc)\\n[10:49:07] 1 tRNA-Trp c[100405,100478] 34 (cca)\\n[10:49:07] 1 tRNA-Ser c[47160,47246] 36 (tga)\\n[10:49:07] 1 tRNA-Ala c[3808,3882] 35 (cgc)\\n[10:49:07] 2 tRNA-Met c[66784,66859] 36 (cat)\\n[10:49:07] 3 tRNA-Arg c[91635,91709] 35 (tct)\\n[10:49:07] 1 tRNA-Thr c[4365,4436] 33 (tgt)\\n[10:49:07] 1 tRNA-Ser c[16957,17050] 36 (gct)\\n[10:49:07] 1 tRNA-Leu c[145301,145383] 35 (cag)\\n[10:49:07] 1 tRNA-Asn [19721,19794] 34 (gtt)\\n[10:49:07] 1 tRNA-Ser [44582,44670] 36 (gga)\\n[10:49:07] 1 tRNA-Ala [643,716] 34 (ggc)\\n[10:49:07] 2 tRNA-Arg [5608,5681] 35 (acg)\\n[10:49:07] 1 tRNA-Ser [4599,4685] 36 (cga)\\n[10:49:07] 1 tRNA-Leu c[43621,43704] 36 (tag)\\n[10:49:07] 1 tRNA-Leu [43199,43283] 35 (caa)\\n[10:49:07] 1 tRNA-Gly c[5906,5976] 33 (tcc)\\n[10:49:07] 1 tRNA-Gln c[133094,133165] 33 (ttg)\\n[10:49:07] 1 tRNA-Met c[2001,2077] 35 (cat)\\n[10:49:07] 2 tRNA-Val [5022,5095] 35 (gac)\\n[10:49:07] 1 tRNA-Pro [12608,12681] 35 (cgg)\\n[10:49:07] 2 tmRNA c[92372,92757] 100,150 ANNIVKFARKEALVAA*\\n[10:49:07] 1 tRNA-Pro c[1737,1811] 35 (tgg)\\n[10:49:07] 1 tRNA-Phe c[1584,1656] 34 (gaa)\\n[10:49:07] 2 tRNA-Val [9039,9111] 34 (tac)\\n[10:49:07] 1 tRNA-Pro [49391,49464] 35 (ggg)\\n[10:49:07] 1 tRNA-Asn c[40867,40940] 34 (att)\\n[10:49:07] 2 tRNA-Arg c[41611,41683] 34 (tcg)\\n[10:49:07] 3 tRNA-Glu c[59596,59668] 35 (ttc)\\n[10:49:07] 1 tRNA-His c[11856,11929] 34 (gtg)\\n[10:49:07] 1 tRNA-Tyr [13635,13719] 35 (gta)\\n[10:49:07] 2 tRNA-Thr [13783,13854] 33 (ggt)\\n[10:49:07] 3 tRNA-Gly c[31880,31951] 33 (ccc)\\n[10:49:07] 1 tRNA-Met c[20774,20846] 34 (cat)\\n[10:49:07] 2 tRNA-Ile [31627,31702] 35 (gat)\\n[10:49:07] 3 tRNA-Ala [31785,31859] 34 (tgc)\\n[10:49:07] Found 44 tRNAs\\n[10:49:07] Predicting Ribosomal RNAs\\n[10:49:07] Running Barrnap with 4 threads\\n[10:49:09] 1 NHAB01000123.1 29914 16S ribosomal RNA\\n[10:49:09] 2 NHAB01000123.1 32398 23S ribosomal RNA\\n[10:49:09] 3 NHAB01000123.1 35337 5S ribosomal RNA\\n[10:49:09] 4 NHAB01000128.1 20 5S ribosomal RNA\\n[10:49:09] 5 NHAB01000128.1 5041 16S ribosomal RNA (partial)\\n[10:49:09] Found 5 rRNAs\\n[10:49:09] Skipping ncRNA search, enable with --rfam if desired.\\n[10:49:09] Total of 48 tRNA + rRNA features\\n[10:49:09] Searching for CRISPR repeats\\n[10:49:09] CRISPR1 NHAB01000038.1 17159 with 43 spacers\\n[10:49:09] CRISPR2 NHAB01000077.1 1684 with 88 spacers\\n[10:49:09] CRISPR3 NHAB01000078.1 14985 with 10 spacers\\n[10:49:09] CRISPR4 NHAB01000093.1 43947 with 3 spacers\\n[10:49:09] CRISPR5 NHAB01000097.1 31 with 51 spacers\\n[10:49:09] CRISPR6 NHAB01000105.1 39160 with 50 spacers\\n[10:49:09] Found 6 CRISPRs\\n[10:49:09] Predicting coding sequences\\n[10:49:09] Contigs total 5405800 bp, so using single mode\\n[10:49:09] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAB01\\\\/\\\\/Fischerella_NHAB01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:49:15] Excluding CDS which overlaps existing RNA (tRNA) at NHAB01000080.1:19595..21034 on + strand\\n[10:49:16] Excluding CDS which overlaps existing RNA (tRNA) at NHAB01000087.1:5053..5724 on - strand\\n[10:49:17] Excluding CDS which overlaps existing RNA (rRNA) at NHAB01000123.1:32793..32972 on + strand\\n[10:49:17] Excluding CDS which overlaps existing RNA (rRNA) at NHAB01000123.1:34758..34934 on - strand\\n[10:49:17] Found 4618 CDS\\n[10:49:17] Connecting features back to sequences\\n[10:49:17] Not using genus-specific database. Try --usegenus to enable it.\\n[10:49:17] Annotating CDS, please be patient.\\n[10:49:17] Will use 4 CPUs for similarity searching.\\n[10:49:18] There are still 4618 unannotated CDS left (started with 4618)\\n[10:49:18] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:49:18] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAB01\\\\/\\\\/Fischerella_NHAB01\\\\.IS\\\\.tmp\\\\.6027\\\\.faa | parallel --gnu --plain -j 4 --block 180268 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAB01\\\\/\\\\/Fischerella_NHAB01\\\\.IS\\\\.tmp\\\\.6027\\\\.blast 2> /dev/null\\n[10:49:31] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAB01\\\\/\\\\/Fischerella_NHAB01\\\\.IS\\\\.tmp\\\\.6027\\\\.faa | parallel --gnu --plain -j 4 --block 180268 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAB01\\\\/\\\\/Fischerella_NHAB01\\\\.IS\\\\.tmp\\\\.6027\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:49:31] This is prokka 1.14.6\\n[10:49:31] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:49:31] Homepage is https://github.com/tseemann/prokka\\n[10:49:31] Local time is Mon Feb 21 10:49:31 2022\\n[10:49:31] You are tiagoferreiraleao\\n[10:49:31] Operating system is darwin\\n[10:49:31] You have BioPerl 1.007002\\n[10:49:31] System has 4 cores.\\n[10:49:31] Option --cpu asked for 12 cores, but system only has 4\\n[10:49:31] Will use maximum of 4 cores.\\n[10:49:31] Annotating as >>> Bacteria <<<\\n[10:49:31] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Hapalosiphon_LIRN01.fasta' contents.\\n[10:49:31] Setting --locustag IBJEPLHO from MD5 2b3e95181b072be2d686b397ebabb1bc\\n[10:49:31] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Hapalosiphon_LIRN01/\\n[10:49:31] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Hapalosiphon_LIRN01\\\\/\\n[10:49:31] Using filename prefix: Hapalosiphon_LIRN01.XXX\\n[10:49:31] Setting HMMER_NCPU=1\\n[10:49:31] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Hapalosiphon_LIRN01//Hapalosiphon_LIRN01.log\\n[10:49:31] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Hapalosiphon_LIRN01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Hapalosiphon_LIRN01/ --prefix Hapalosiphon_LIRN01 --cpus 12\\n[10:49:31] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:49:31] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:49:31] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:49:31] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:49:31] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:49:31] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:49:32] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:49:32] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:49:32] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:49:32] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:49:32] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:49:32] Looking for 'egrep' - found /usr/bin/egrep\\n[10:49:32] Looking for 'find' - found /usr/bin/find\\n[10:49:32] Looking for 'grep' - found /usr/bin/grep\\n[10:49:32] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:49:32] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:49:32] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:49:32] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:49:32] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:49:32] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:49:32] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:49:32] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:49:32] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:49:32] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:49:32] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:49:32] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:49:32] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:49:32] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:49:32] Looking for 'sed' - found /usr/bin/sed\\n[10:49:32] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:49:33] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:49:33] Using genetic code table 11.\\n[10:49:33] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Hapalosiphon_LIRN01.fasta\\n[10:49:33] Wrote 115 contigs totalling 7429720 bp.\\n[10:49:33] Predicting tRNAs and tmRNAs\\n[10:49:33] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Hapalosiphon_LIRN01\\\\/\\\\/Hapalosiphon_LIRN01\\\\.fna\\n[10:49:36] 1 tRNA-Ala c[97060,97134] 35 (cgc)\\n[10:49:36] 1 tRNA-Glu [14496,14568] 35 (ttc)\\n[10:49:36] 2 tRNA-Val c[41169,41242] 35 (gac)\\n[10:49:36] 3 tRNA-Met [44270,44346] 35 (cat)\\n[10:49:36] 1 tRNA-Asp c[90152,90225] 35 (gtc)\\n[10:49:36] 1 tRNA-Asn c[31873,31946] 34 (att)\\n[10:49:36] 2 tRNA-Cys c[32684,32756] 34 (gca)\\n[10:49:36] 1 tRNA-Pro c[36465,36540] 36 (cgg)\\n[10:49:36] 2 tRNA-Phe c[72633,72705] 34 (gaa)\\n[10:49:36] 1 tRNA-Gly c[48147,48218] 33 (gcc)\\n[10:49:36] 1 tRNA-Met [10073,10146] 34 (cat)\\n[10:49:36] 1 tRNA-Arg [116239,116313] 35 (acg)\\n[10:49:36] 2 tRNA-Ser c[126363,126456] 36 (gct)\\n[10:49:36] 3 tRNA-Trp [229035,229108] 34 (cca)\\n[10:49:36] 4 tRNA-Pro [463832,463906] 35 (tgg)\\n[10:49:36] 1 tRNA-Trp c[15830,15904] 35 (cca)\\n[10:49:36] 2 tRNA-Trp c[18128,18202] 35 (cca)\\n[10:49:36] 1 tRNA-Ser c[16542,16630] 36 (gga)\\n[10:49:36] 1 tRNA-Leu c[26000,26082] 35 (tag)\\n[10:49:36] 1 tRNA-Lys [45464,45536] 33 (ctt)\\n[10:49:36] 2 tRNA-Lys [56173,56244] 33 (ttt)\\n[10:49:36] 1 tRNA-Met [135111,135186] 36 (cat)\\n[10:49:36] 2 tRNA-Asn [241685,241758] 34 (gtt)\\n[10:49:36] 3 tRNA-Thr [309473,309544] 33 (cgt)\\n[10:49:36] 1 tRNA-Leu c[119458,119540] 35 (cag)\\n[10:49:36] 1 tRNA-Cys [4504,4575] 33 (gca)\\n[10:49:36] 2 tRNA-Ser c[152062,152148] 36 (cga)\\n[10:49:36] 3 tRNA-Tyr [195819,195903] 35 (gta)\\n[10:49:36] 4 tRNA-Thr [195981,196052] 33 (ggt)\\n[10:49:36] 5 tRNA-Gln [259652,259729] 38 (ttg)\\n[10:49:36] 1 tRNA-Leu c[103946,104028] 36 (gag)\\n[10:49:36] 2 tRNA-Gly c[234345,234415] 33 (tcc)\\n[10:49:36] 3 tRNA-Ser c[503296,503390] 35 (tga)\\n[10:49:36] 1 tRNA-Val c[63645,63717] 34 (tac)\\n[10:49:36] 2 tRNA-Gly [70645,70716] 33 (ccc)\\n[10:49:36] 3 tmRNA [105331,105718] 101,154 ANNIVKFARKEALVAA**\\n[10:49:36] 1 tRNA-Pro [95026,95099] 35 (ggg)\\n[10:49:36] 2 tRNA-Thr c[114521,114592] 33 (tgt)\\n[10:49:36] 1 tRNA-Ser [58488,58573] 35 (tga)\\n[10:49:36] 1 tRNA-Arg [63812,63884] 34 (ccg)\\n[10:49:36] 1 tRNA-Arg [35579,35653] 35 (tct)\\n[10:49:36] 1 tRNA-Gln c[140745,140816] 33 (ttg)\\n[10:49:36] 2 tRNA-Leu c[172175,172259] 35 (caa)\\n[10:49:36] Found 43 tRNAs\\n[10:49:36] Predicting Ribosomal RNAs\\n[10:49:36] Running Barrnap with 4 threads\\n[10:49:38] 1 LIRN01000019.1 69 23S ribosomal RNA\\n[10:49:38] 2 LIRN01000019.1 3207 16S ribosomal RNA\\n[10:49:38] Found 2 rRNAs\\n[10:49:38] Skipping ncRNA search, enable with --rfam if desired.\\n[10:49:38] Total of 44 tRNA + rRNA features\\n[10:49:38] Searching for CRISPR repeats\\n[10:49:38] CRISPR1 LIRN01000001.1 46767 with 30 spacers\\n[10:49:38] CRISPR2 LIRN01000012.1 28010 with 14 spacers\\n[10:49:38] CRISPR3 LIRN01000028.1 5240 with 22 spacers\\n[10:49:38] CRISPR4 LIRN01000029.1 10544 with 24 spacers\\n[10:49:38] CRISPR5 LIRN01000074.1 81438 with 35 spacers\\n[10:49:38] CRISPR6 LIRN01000074.1 94428 with 21 spacers\\n[10:49:38] CRISPR7 LIRN01000084.1 29314 with 15 spacers\\n[10:49:38] Found 7 CRISPRs\\n[10:49:38] Predicting coding sequences\\n[10:49:38] Contigs total 7429720 bp, so using single mode\\n[10:49:38] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Hapalosiphon_LIRN01\\\\/\\\\/Hapalosiphon_LIRN01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:49:44] Excluding CDS which overlaps existing RNA (tRNA) at LIRN01000007.1:32655..32819 on - strand\\n[10:49:45] Excluding CDS which overlaps existing RNA (repeat_region) at LIRN01000012.1:27998..28444 on - strand\\n[10:49:45] Excluding CDS which overlaps existing RNA (rRNA) at LIRN01000019.1:3249..3563 on - strand\\n[10:49:45] Excluding CDS which overlaps existing RNA (repeat_region) at LIRN01000028.1:5339..5506 on + strand\\n[10:49:45] Excluding CDS which overlaps existing RNA (repeat_region) at LIRN01000029.1:10870..11076 on + strand\\n[10:49:45] Excluding CDS which overlaps existing RNA (repeat_region) at LIRN01000029.1:12070..12231 on + strand\\n[10:49:46] Excluding CDS which overlaps existing RNA (tRNA) at LIRN01000048.1:20181..26009 on + strand\\n[10:49:46] Excluding CDS which overlaps existing RNA (tRNA) at LIRN01000048.1:25978..26103 on - strand\\n[10:49:46] Excluding CDS which overlaps existing RNA (tRNA) at LIRN01000064.1:135186..137339 on - strand\\n[10:49:47] Excluding CDS which overlaps existing RNA (repeat_region) at LIRN01000074.1:82103..82345 on - strand\\n[10:49:47] Excluding CDS which overlaps existing RNA (repeat_region) at LIRN01000074.1:82974..83150 on - strand\\n[10:49:47] Excluding CDS which overlaps existing RNA (repeat_region) at LIRN01000074.1:83234..83401 on - strand\\n[10:49:48] Found 6278 CDS\\n[10:49:48] Connecting features back to sequences\\n[10:49:48] Not using genus-specific database. Try --usegenus to enable it.\\n[10:49:48] Annotating CDS, please be patient.\\n[10:49:48] Will use 4 CPUs for similarity searching.\\n[10:49:50] There are still 6278 unannotated CDS left (started with 6278)\\n[10:49:50] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:49:50] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Hapalosiphon_LIRN01\\\\/\\\\/Hapalosiphon_LIRN01\\\\.IS\\\\.tmp\\\\.6140\\\\.faa | parallel --gnu --plain -j 4 --block 252701 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Hapalosiphon_LIRN01\\\\/\\\\/Hapalosiphon_LIRN01\\\\.IS\\\\.tmp\\\\.6140\\\\.blast 2> /dev/null\\n[10:50:07] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Hapalosiphon_LIRN01\\\\/\\\\/Hapalosiphon_LIRN01\\\\.IS\\\\.tmp\\\\.6140\\\\.faa | parallel --gnu --plain -j 4 --block 252701 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Hapalosiphon_LIRN01\\\\/\\\\/Hapalosiphon_LIRN01\\\\.IS\\\\.tmp\\\\.6140\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:50:07] This is prokka 1.14.6\\n[10:50:07] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:50:07] Homepage is https://github.com/tseemann/prokka\\n[10:50:07] Local time is Mon Feb 21 10:50:07 2022\\n[10:50:07] You are tiagoferreiraleao\\n[10:50:07] Operating system is darwin\\n[10:50:07] You have BioPerl 1.007002\\n[10:50:07] System has 4 cores.\\n[10:50:07] Option --cpu asked for 12 cores, but system only has 4\\n[10:50:07] Will use maximum of 4 cores.\\n[10:50:07] Annotating as >>> Bacteria <<<\\n[10:50:07] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJQM01.fasta' contents.\\n[10:50:07] Setting --locustag GLGDHOPB from MD5 050d189bb9aa20bb72bdaced50bde404\\n[10:50:07] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJQM01/\\n[10:50:07] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJQM01\\\\/\\n[10:50:07] Using filename prefix: Nostoc_JACJQM01.XXX\\n[10:50:07] Setting HMMER_NCPU=1\\n[10:50:07] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJQM01//Nostoc_JACJQM01.log\\n[10:50:07] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJQM01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJQM01/ --prefix Nostoc_JACJQM01 --cpus 12\\n[10:50:07] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:50:07] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:50:07] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:50:07] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:50:07] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:50:07] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:50:07] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:50:07] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:50:07] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:50:07] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:50:07] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:50:07] Looking for 'egrep' - found /usr/bin/egrep\\n[10:50:07] Looking for 'find' - found /usr/bin/find\\n[10:50:07] Looking for 'grep' - found /usr/bin/grep\\n[10:50:07] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:50:07] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:50:07] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:50:07] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:50:07] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:50:07] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:50:08] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:50:08] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:50:08] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:50:08] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:50:08] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:50:08] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:50:08] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:50:08] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:50:08] Looking for 'sed' - found /usr/bin/sed\\n[10:50:08] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:50:08] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:50:08] Using genetic code table 11.\\n[10:50:08] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJQM01.fasta\\n[10:50:09] Wrote 430 contigs totalling 8988648 bp.\\n[10:50:09] Predicting tRNAs and tmRNAs\\n[10:50:09] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJQM01\\\\/\\\\/Nostoc_JACJQM01\\\\.fna\\n[10:50:12] 1 tRNA-Ala [227034,227126] 33 (ggc)\\n[10:50:12] 1 tRNA-Met c[17549,17623] 35 (cat)\\n[10:50:12] 2 tRNA-Arg [58618,58693] 36 (cct)\\n[10:50:12] 3 tRNA-Ala c[211160,211232] 34 (ggc)\\n[10:50:12] 1 tRNA-Ser [33010,33096] 36 (cga)\\n[10:50:12] 1 tRNA-Pro [15184,15257] 35 (tgg)\\n[10:50:12] 2 tRNA-Met [116210,116283] 33 (cat)\\n[10:50:12] 3 tRNA-Thr c[121224,121295] 33 (tgt)\\n[10:50:12] 1 tRNA-His c[41977,42050] 34 (gtg)\\n[10:50:12] 1 tRNA-Pro [116266,116339] 35 (ggg)\\n[10:50:12] 2 tRNA-Arg c[122893,122966] 35 (acg)\\n[10:50:12] 1 tRNA-Pro [22205,22278] 35 (cgg)\\n[10:50:12] 1 tRNA-Asp [104406,104479] 35 (gtc)\\n[10:50:12] 1 tRNA-Val c[3031,3104] 35 (gac)\\n[10:50:12] 1 tRNA-Glu [131502,131575] 35 (ttc)\\n[10:50:12] 1 tRNA-Leu c[101621,101703] 35 (tag)\\n[10:50:12] 1 tRNA-Lys c[20039,20111] 33 (ctt)\\n[10:50:12] 2 tRNA-Leu c[112086,112168] 36 (gag)\\n[10:50:12] 1 tmRNA [93246,93636] 101,154 ANNIVSFARKPALVAAA*\\n[10:50:12] 2 tRNA-Gly c[121344,121415] 33 (ccc)\\n[10:50:12] 1 tRNA-Gly c[34920,34990] 33 (tcc)\\n[10:50:12] 1 tRNA-Arg [50050,50124] 35 (tct)\\n[10:50:12] 2 tRNA-Thr c[62842,62913] 33 (ggt)\\n[10:50:12] 3 tRNA-Tyr c[63001,63086] 35 (gta)\\n[10:50:12] 4 tRNA-Ser [65368,65454] 36 (tga)\\n[10:50:12] 5 tRNA-Val [82385,82457] 34 (tac)\\n[10:50:12] 1 tRNA-Leu c[5383,5479] 37 (cag)\\n[10:50:12] 2 tRNA-Asp [40758,40834] 37 (gtc)\\n[10:50:12] 3 tRNA-Glu [40837,40913] 35 (ctc)\\n[10:50:12] 4 tRNA-Arg [40916,40991] 36 (ccg)\\n[10:50:12] 5 tRNA-Cys [40996,41070] 34 (gca)\\n[10:50:12] 6 tRNA-Gln [41076,41151] 35 (ttg)\\n[10:50:12] 7 tRNA-Leu [41155,41232] 35 (caa)\\n[10:50:12] 8 tRNA-Pro [41234,41307] 35 (tgg)\\n[10:50:12] 9 tRNA-Leu [41314,41388] 35 (tag)\\n[10:50:12] 10 tRNA-Leu [41408,41485] 35 (gag)\\n[10:50:12] 11 tRNA-Lys [41490,41564] 33 (ttt)\\n[10:50:12] 12 tRNA-Lys [41571,41645] 33 (ctt)\\n[10:50:12] 13 tRNA-Cys [41737,41809] 33 (gca)\\n[10:50:12] 14 tRNA-Thr [41822,41894] 34 (tgt)\\n[10:50:12] 15 tRNA-Gly [41900,41970] 33 (tcc)\\n[10:50:12] 16 tRNA-Val [41976,42051] 34 (tac)\\n[10:50:12] 17 tRNA-Arg [42058,42130] 33 (tct)\\n[10:50:12] 18 tRNA-Ala [42135,42207] 34 (tgc)\\n[10:50:12] 19 tRNA-His [42215,42291] 35 (gtg)\\n[10:50:12] 20 tRNA-Ser [42302,42386] 34 (tga)\\n[10:50:12] 1 tRNA-Ser c[75712,75800] 36 (gga)\\n[10:50:12] 1 tRNA-Thr c[16092,16163] 33 (cgt)\\n[10:50:12] 1 tRNA-Asn c[54153,54224] 33 (gtt)\\n[10:50:12] 1 tRNA-Cys [36187,36257] 33 (gca)\\n[10:50:12] 2 tRNA-Lys [44852,44923] 33 (ttt)\\n[10:50:12] 1 tRNA-Ser [6564,6657] 36 (gct)\\n[10:50:12] 1 tRNA-Gly [18351,18422] 33 (gcc)\\n[10:50:12] 1 tRNA-Leu c[8712,8793] 35 (cag)\\n[10:50:12] 2 tRNA-Leu [14650,14734] 35 (caa)\\n[10:50:12] 3 tRNA-Gln [21537,21610] 34 (ttg)\\n[10:50:12] 1 tRNA-Phe [16611,16683] 34 (gaa)\\n[10:50:12] 1 tRNA-Trp c[4416,4489] 34 (cca)\\n[10:50:12] 1 tRNA-Trp [14686,14760] 35 (cca)\\n[10:50:12] 1 tRNA-Ala c[351,423] 34 (cgc)\\n[10:50:12] 1 tRNA-Ala [8410,8484] 35 (tgc)\\n[10:50:12] 1 tRNA-Arg [3271,3343] 34 (ccg)\\n[10:50:12] 1 tRNA-Ile [1625,1700] 35 (gat)\\n[10:50:12] 2 tRNA-Ala [1787,1861] 34 (tgc)\\n[10:50:12] Found 64 tRNAs\\n[10:50:12] Predicting Ribosomal RNAs\\n[10:50:12] Running Barrnap with 4 threads\\n[10:50:14] 1 JACJQM010000430.1 5 16S ribosomal RNA\\n[10:50:14] 2 JACJQM010000430.1 2110 23S ribosomal RNA\\n[10:50:14] Found 2 rRNAs\\n[10:50:14] Skipping ncRNA search, enable with --rfam if desired.\\n[10:50:14] Total of 65 tRNA + rRNA features\\n[10:50:14] Searching for CRISPR repeats\\n[10:50:14] CRISPR1 JACJQM010000002.1 68258 with 4 spacers\\n[10:50:14] CRISPR2 JACJQM010000013.1 55585 with 5 spacers\\n[10:50:14] CRISPR3 JACJQM010000041.1 24446 with 8 spacers\\n[10:50:14] CRISPR4 JACJQM010000076.1 58 with 21 spacers\\n[10:50:14] CRISPR5 JACJQM010000076.1 13031 with 8 spacers\\n[10:50:14] CRISPR6 JACJQM010000087.1 12 with 6 spacers\\n[10:50:14] CRISPR7 JACJQM010000087.1 11928 with 6 spacers\\n[10:50:14] CRISPR8 JACJQM010000101.1 8868 with 9 spacers\\n[10:50:14] CRISPR9 JACJQM010000161.1 4473 with 19 spacers\\n[10:50:14] CRISPR10 JACJQM010000254.1 238 with 11 spacers\\n[10:50:14] CRISPR11 JACJQM010000254.1 2640 with 6 spacers\\n[10:50:14] CRISPR12 JACJQM010000265.1 1 with 4 spacers\\n[10:50:15] Found 12 CRISPRs\\n[10:50:15] Predicting coding sequences\\n[10:50:15] Contigs total 8988648 bp, so using single mode\\n[10:50:15] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJQM01\\\\/\\\\/Nostoc_JACJQM01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:50:24] Excluding CDS which overlaps existing RNA (tRNA) at JACJQM010000024.1:5098..5886 on + strand\\n[10:50:25] Excluding CDS which overlaps existing RNA (tRNA) at JACJQM010000043.1:44820..45005 on - strand\\n[10:50:26] Excluding CDS which overlaps existing RNA (tRNA) at JACJQM010000057.1:18338..18652 on + strand\\n[10:50:26] Excluding CDS which overlaps existing RNA (repeat_region) at JACJQM010000087.1:12079..12258 on - strand\\n[10:50:26] Excluding CDS which overlaps existing RNA (tRNA) at JACJQM010000089.1:4367..4513 on - strand\\n[10:50:27] Excluding CDS which overlaps existing RNA (repeat_region) at JACJQM010000254.1:2571..2759 on + strand\\n[10:50:27] Excluding CDS which overlaps existing RNA (rRNA) at JACJQM010000430.1:1136..1450 on + strand\\n[10:50:27] Found 8241 CDS\\n[10:50:27] Connecting features back to sequences\\n[10:50:27] Not using genus-specific database. Try --usegenus to enable it.\\n[10:50:27] Annotating CDS, please be patient.\\n[10:50:27] Will use 4 CPUs for similarity searching.\\n[10:50:29] There are still 8241 unannotated CDS left (started with 8241)\\n[10:50:29] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:50:29] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJQM01\\\\/\\\\/Nostoc_JACJQM01\\\\.IS\\\\.tmp\\\\.6252\\\\.faa | parallel --gnu --plain -j 4 --block 297085 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJQM01\\\\/\\\\/Nostoc_JACJQM01\\\\.IS\\\\.tmp\\\\.6252\\\\.blast 2> /dev/null\\n[10:50:43] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJQM01\\\\/\\\\/Nostoc_JACJQM01\\\\.IS\\\\.tmp\\\\.6252\\\\.faa | parallel --gnu --plain -j 4 --block 297085 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJQM01\\\\/\\\\/Nostoc_JACJQM01\\\\.IS\\\\.tmp\\\\.6252\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:50:43] This is prokka 1.14.6\\n[10:50:43] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:50:43] Homepage is https://github.com/tseemann/prokka\\n[10:50:43] Local time is Mon Feb 21 10:50:43 2022\\n[10:50:43] You are tiagoferreiraleao\\n[10:50:43] Operating system is darwin\\n[10:50:43] You have BioPerl 1.007002\\n[10:50:44] System has 4 cores.\\n[10:50:44] Option --cpu asked for 12 cores, but system only has 4\\n[10:50:44] Will use maximum of 4 cores.\\n[10:50:44] Annotating as >>> Bacteria <<<\\n[10:50:44] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_SZNH01.fasta' contents.\\n[10:50:44] Setting --locustag DGMMIEPI from MD5 d0662e925b9312bf17ccf22843b608a1\\n[10:50:44] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_SZNH01/\\n[10:50:44] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_SZNH01\\\\/\\n[10:50:44] Using filename prefix: Nostoc_SZNH01.XXX\\n[10:50:44] Setting HMMER_NCPU=1\\n[10:50:44] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_SZNH01//Nostoc_SZNH01.log\\n[10:50:44] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_SZNH01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_SZNH01/ --prefix Nostoc_SZNH01 --cpus 12\\n[10:50:44] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:50:44] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:50:44] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:50:44] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:50:44] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:50:44] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:50:44] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:50:44] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:50:44] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:50:44] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:50:44] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:50:44] Looking for 'egrep' - found /usr/bin/egrep\\n[10:50:44] Looking for 'find' - found /usr/bin/find\\n[10:50:44] Looking for 'grep' - found /usr/bin/grep\\n[10:50:44] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:50:44] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:50:44] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:50:44] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:50:44] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:50:44] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:50:44] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:50:44] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:50:44] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:50:44] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:50:45] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:50:45] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:50:45] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:50:45] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:50:45] Looking for 'sed' - found /usr/bin/sed\\n[10:50:45] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:50:45] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:50:45] Using genetic code table 11.\\n[10:50:45] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_SZNH01.fasta\\n[10:50:45] Wrote 106 contigs totalling 7618130 bp.\\n[10:50:45] Predicting tRNAs and tmRNAs\\n[10:50:45] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_SZNH01\\\\/\\\\/Nostoc_SZNH01\\\\.fna\\n[10:50:48] 1 tRNA-Trp c[9261,9334] 34 (cca)\\n[10:50:48] 2 tRNA-Pyl [52274,52346] 34 (cta)\\n[10:50:48] 3 tRNA-Cys [53097,53171] 35 (gca)\\n[10:50:48] 4 tRNA-Pro c[93816,93889] 35 (tgg)\\n[10:50:48] 5 tRNA-Met [316241,316314] 33 (cat)\\n[10:50:48] 6 tRNA-Ser [380548,380636] 36 (gga)\\n[10:50:48] 1 tRNA-Asp c[378428,378501] 35 (gtc)\\n[10:50:48] 1 tRNA-Ala c[61032,61104] 34 (ggc)\\n[10:50:48] 2 tRNA-Leu c[352462,352557] 36 (caa)\\n[10:50:48] 3 tRNA-Ile [374861,374936] 35 (gat)\\n[10:50:48] 4 tRNA-Ala [374998,375072] 34 (tgc)\\n[10:50:48] 1 tRNA-Ala [268939,269011] 34 (tgc)\\n[10:50:48] 2 tRNA-Asn [269229,269302] 34 (gtt)\\n[10:50:48] 1 tRNA-Glu c[47447,47520] 35 (ttc)\\n[10:50:48] 2 tRNA-Glu [72076,72152] 35 (ctc)\\n[10:50:48] 3 tRNA-Cys [72156,72229] 35 (gca)\\n[10:50:48] 4 tRNA-Met [72236,72309] 32 (cat)\\n[10:50:48] tRNA [72311,72385] is a pseudo/wacky gene - skipping.\\n[10:50:48] 6 tRNA-Tyr [72519,72602] 37 (gta)\\n[10:50:48] 7 tRNA-Ser [72606,72697] 35 (gct)\\n[10:50:48] 8 tRNA-Arg [72699,72776] 36 (acg)\\n[10:50:48] 9 tRNA-Phe [72856,72931] 36 (gaa)\\n[10:50:48] 10 tRNA-Asn [72937,73011] 33 (gtt)\\n[10:50:48] 11 tRNA-Gln [73059,73131] 34 (ctg)\\n[10:50:48] 12 tRNA-Gln [73135,73207] 34 (ttg)\\n[10:50:48] 13 tRNA-Leu [73410,73487] 35 (taa)\\n[10:50:48] 14 tRNA-Pro [73489,73562] 35 (tgg)\\n[10:50:48] 15 tRNA-Leu [73565,73639] 35 (tag)\\n[10:50:48] 16 tRNA-Met [73649,73725] 34 (cat)\\n[10:50:48] 17 tRNA-Thr [73734,73806] 34 (tgt)\\n[10:50:48] 18 tRNA-Gly [73922,73993] 34 (gcc)\\n[10:50:48] 19 tRNA-Val [73999,74074] 34 (tac)\\n[10:50:48] 20 tRNA-Arg [74081,74156] 35 (tct)\\n[10:50:48] 21 tRNA-Ala [74200,74274] 35 (tgc)\\n[10:50:48] 22 tRNA-His [74280,74355] 35 (gtg)\\n[10:50:48] 23 tRNA-Ser [74361,74444] 34 (tga)\\n[10:50:48] 24 tRNA-Lys [74599,74669] 33 (ttt)\\n[10:50:48] 25 tRNA-Ser [80116,80205] 36 (gct)\\n[10:50:48] 26 tRNA-Ser c[152204,152290] 36 (cga)\\n[10:50:48] 1 tRNA-Gly c[81495,81565] 33 (tcc)\\n[10:50:48] 2 tRNA-His [143113,143186] 34 (gtg)\\n[10:50:48] 1 tRNA-Leu [114837,114917] 35 (gag)\\n[10:50:48] 2 tRNA-Gly [254219,254290] 33 (gcc)\\n[10:50:48] 1 tRNA-Leu [24880,24959] 36 (gag)\\n[10:50:48] 2 tRNA-Lys [66653,66724] 33 (ttt)\\n[10:50:48] 3 tRNA-Thr c[177683,177754] 33 (cgt)\\n[10:50:48] 1 tRNA-Arg c[219740,219813] 35 (acg)\\n[10:50:48] 1 tRNA-Leu c[102466,102547] 35 (cag)\\n[10:50:48] 2 tRNA-Val c[116178,116251] 35 (gac)\\n[10:50:48] 1 tRNA-Cys c[179794,179865] 33 (gca)\\n[10:50:48] 1 tRNA-Arg [47886,47958] 34 (ccg)\\n[10:50:48] 2 tRNA-Phe c[145001,145073] 34 (gaa)\\n[10:50:48] 1 tRNA-Thr c[104675,104746] 33 (ggt)\\n[10:50:48] 2 tRNA-Tyr c[104827,104912] 35 (gta)\\n[10:50:48] 3 tRNA-Ser [107283,107369] 36 (tga)\\n[10:50:48] 4 tRNA-Arg [141431,141505] 35 (tct)\\n[10:50:48] 5 tRNA-Val c[163288,163360] 34 (tac)\\n[10:50:48] 1 tRNA-Pro c[64801,64876] 36 (ggg)\\n[10:50:48] 1 tRNA-Val [120951,121024] 34 (cac)\\n[10:50:48] 1 tRNA-Leu c[97421,97504] 35 (caa)\\n[10:50:48] 1 tRNA-Pro [115558,115631] 35 (cgg)\\n[10:50:48] 1 tRNA-Met [96410,96486] 35 (cat)\\n[10:50:48] 1 tRNA-Met c[69541,69615] 35 (cat)\\n[10:50:48] 2 tRNA-Gln c[141149,141223] 34 (ttg)\\n[10:50:48] 1 tRNA-Thr [9507,9578] 33 (tgt)\\n[10:50:48] 1 tmRNA c[43302,43689] 101,151 ANNIVKFARREALVAA*\\n[10:50:48] 1 tRNA-Ser c[62840,62924] 34 (tga)\\n[10:50:48] 2 tRNA-His c[62935,63011] 35 (gtg)\\n[10:50:48] 3 tRNA-Ala c[63017,63091] 35 (tgc)\\n[10:50:48] 4 tRNA-Arg c[63095,63170] 34 (tct)\\n[10:50:48] 5 tRNA-Val c[63177,63252] 34 (tac)\\n[10:50:48] 6 tRNA-Gly c[63258,63328] 33 (tcc)\\n[10:50:48] 7 tRNA-Thr c[63334,63407] 35 (tgt)\\n[10:50:48] 8 tRNA-Met c[63420,63492] 34 (cat)\\n[10:50:48] 9 tRNA-Leu c[63504,63578] 35 (taa)\\n[10:50:48] 10 tRNA-Lys c[63580,63654] 34 (ctt)\\n[10:50:48] 11 tRNA-Lys c[63664,63738] 33 (ttt)\\n[10:50:48] 12 tRNA-Leu c[63743,63820] 35 (gag)\\n[10:50:48] 13 tRNA-Leu c[63840,63915] 35 (tag)\\n[10:50:48] 14 tRNA-Pro c[63919,63995] 35 (tgg)\\n[10:50:48] 15 tRNA-Leu c[64000,64077] 35 (caa)\\n[10:50:48] 16 tRNA-Gln c[64105,64180] 36 (ttg)\\n[10:50:48] 17 tRNA-Asn c[64185,64259] 33 (gtt)\\n[10:50:48] 18 tRNA-Arg c[64390,64465] 35 (acg)\\n[10:50:48] 19 tRNA-Ser c[64471,64559] 33 (gct)\\n[10:50:48] 20 tRNA-Glu c[64653,64729] 34 (ttc)\\n[10:50:48] 21 tRNA-Pyl c[64760,64831] 33 (cta)\\n[10:50:48] 22 tRNA-Met c[64932,65006] 33 (cat)\\n[10:50:48] 23 tRNA-Glu c[65012,65088] 35 (ctc)\\n[10:50:48] 24 tRNA-Asp c[65091,65168] 38 (gtc)\\n[10:50:48] 1 tRNA-Leu [16271,16353] 35 (tag)\\n[10:50:48] 2 tRNA-Ala [55771,55843] 34 (cgc)\\n[10:50:48] 1 tRNA-Gly [52434,52507] 34 (ccc)\\n[10:50:48] 1 tRNA-Asp [21128,21210] 36 (gtc)\\n[10:50:48] 1 tRNA-Lys [28373,28446] 34 (ctt)\\n[10:50:48] Found 94 tRNAs\\n[10:50:48] Predicting Ribosomal RNAs\\n[10:50:48] Running Barrnap with 4 threads\\n[10:50:49] 1 SZNH01000003.1 373251 16S ribosomal RNA\\n[10:50:49] 2 SZNH01000003.1 375258 23S ribosomal RNA\\n[10:50:49] Found 2 rRNAs\\n[10:50:49] Skipping ncRNA search, enable with --rfam if desired.\\n[10:50:49] Total of 95 tRNA + rRNA features\\n[10:50:49] Searching for CRISPR repeats\\n[10:50:50] CRISPR1 SZNH01000008.1 42641 with 13 spacers\\n[10:50:50] CRISPR2 SZNH01000009.1 124959 with 6 spacers\\n[10:50:50] CRISPR3 SZNH01000038.1 52810 with 4 spacers\\n[10:50:50] Found 3 CRISPRs\\n[10:50:50] Predicting coding sequences\\n[10:50:50] Contigs total 7618130 bp, so using single mode\\n[10:50:50] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_SZNH01\\\\/\\\\/Nostoc_SZNH01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:50:57] Excluding CDS which overlaps existing RNA (tRNA) at SZNH01000005.1:152017..152268 on - strand\\n[10:50:58] Excluding CDS which overlaps existing RNA (tRNA) at SZNH01000008.1:24327..24890 on + strand\\n[10:50:59] Excluding CDS which overlaps existing RNA (tRNA) at SZNH01000015.1:64643..64897 on + strand\\n[10:50:59] Excluding CDS which overlaps existing RNA (tRNA) at SZNH01000017.1:97446..97637 on + strand\\n[10:51:00] Excluding CDS which overlaps existing RNA (tRNA) at SZNH01000039.1:16987..22386 on - strand\\n[10:51:01] Found 6337 CDS\\n[10:51:01] Connecting features back to sequences\\n[10:51:01] Not using genus-specific database. Try --usegenus to enable it.\\n[10:51:01] Annotating CDS, please be patient.\\n[10:51:01] Will use 4 CPUs for similarity searching.\\n[10:51:02] There are still 6337 unannotated CDS left (started with 6337)\\n[10:51:02] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:51:02] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_SZNH01\\\\/\\\\/Nostoc_SZNH01\\\\.IS\\\\.tmp\\\\.6364\\\\.faa | parallel --gnu --plain -j 4 --block 255303 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_SZNH01\\\\/\\\\/Nostoc_SZNH01\\\\.IS\\\\.tmp\\\\.6364\\\\.blast 2> /dev/null\\n[10:51:22] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_SZNH01\\\\/\\\\/Nostoc_SZNH01\\\\.IS\\\\.tmp\\\\.6364\\\\.faa | parallel --gnu --plain -j 4 --block 255303 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_SZNH01\\\\/\\\\/Nostoc_SZNH01\\\\.IS\\\\.tmp\\\\.6364\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:51:22] This is prokka 1.14.6\\n[10:51:22] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:51:22] Homepage is https://github.com/tseemann/prokka\\n[10:51:22] Local time is Mon Feb 21 10:51:22 2022\\n[10:51:22] You are tiagoferreiraleao\\n[10:51:22] Operating system is darwin\\n[10:51:22] You have BioPerl 1.007002\\n[10:51:22] System has 4 cores.\\n[10:51:22] Option --cpu asked for 12 cores, but system only has 4\\n[10:51:22] Will use maximum of 4 cores.\\n[10:51:22] Annotating as >>> Bacteria <<<\\n[10:51:22] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_NOLK01.fasta' contents.\\n[10:51:22] Setting --locustag FOJLKJOJ from MD5 f8354383ff00c9a9b6bcdb328812e2b3\\n[10:51:22] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_NOLK01/\\n[10:51:22] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_NOLK01\\\\/\\n[10:51:22] Using filename prefix: Nostoc_NOLK01.XXX\\n[10:51:22] Setting HMMER_NCPU=1\\n[10:51:22] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_NOLK01//Nostoc_NOLK01.log\\n[10:51:22] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_NOLK01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_NOLK01/ --prefix Nostoc_NOLK01 --cpus 12\\n[10:51:22] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:51:22] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:51:22] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:51:22] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:51:23] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:51:23] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:51:23] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:51:23] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:51:23] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:51:23] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:51:23] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:51:23] Looking for 'egrep' - found /usr/bin/egrep\\n[10:51:23] Looking for 'find' - found /usr/bin/find\\n[10:51:23] Looking for 'grep' - found /usr/bin/grep\\n[10:51:23] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:51:23] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:51:23] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:51:23] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:51:23] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:51:23] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:51:23] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:51:23] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:51:23] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:51:23] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:51:23] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:51:23] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:51:23] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:51:23] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:51:23] Looking for 'sed' - found /usr/bin/sed\\n[10:51:23] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:51:24] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:51:24] Using genetic code table 11.\\n[10:51:24] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_NOLK01.fasta\\n[10:51:24] Wrote 397 contigs totalling 9158766 bp.\\n[10:51:24] Predicting tRNAs and tmRNAs\\n[10:51:24] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_NOLK01\\\\/\\\\/Nostoc_NOLK01\\\\.fna\\n[10:51:27] 1 tRNA-Pro [180066,180139] 35 (tgg)\\n[10:51:27] 1 tRNA-Gly [96401,96472] 33 (ccc)\\n[10:51:27] 1 tRNA-Asp [24880,24953] 35 (gtc)\\n[10:51:27] 2 tRNA-Arg c[100766,100838] 34 (ccg)\\n[10:51:27] 1 tRNA-Gly [96623,96693] 33 (tcc)\\n[10:51:27] 1 tRNA-Glu [45263,45336] 35 (ttc)\\n[10:51:27] 2 tRNA-Asp [52041,52116] 36 (gtc)\\n[10:51:27] 3 tRNA-Glu [52120,52196] 35 (ctc)\\n[10:51:27] 4 tRNA-Cys [52200,52274] 35 (gca)\\n[10:51:27] 5 tRNA-Met [52284,52358] 33 (cat)\\n[10:51:27] tRNA [52361,52435] is a pseudo/wacky gene - skipping.\\n[10:51:27] 7 tRNA-Glu [52490,52563] 35 (ttc)\\n[10:51:27] 8 tRNA-Tyr [52571,52654] 37 (gta)\\n[10:51:27] 9 tRNA-Ser [52657,52746] 36 (gct)\\n[10:51:27] 10 tRNA-Arg [52750,52825] 35 (acg)\\n[10:51:27] 11 tRNA-Phe [52850,52925] 36 (gaa)\\n[10:51:27] 12 tRNA-Asn [52931,53005] 33 (gtt)\\n[10:51:27] 13 tRNA-Gln [53155,53227] 34 (ctg)\\n[10:51:27] 14 tRNA-Gln [53231,53306] 34 (ttg)\\n[10:51:27] 15 tRNA-Leu [53422,53499] 35 (taa)\\n[10:51:27] 16 tRNA-Pro [53500,53572] 34 (tgg)\\n[10:51:27] 17 tRNA-Leu [53576,53650] 35 (cag)\\n[10:51:27] 18 tRNA-Leu [53803,53877] 35 (tag)\\n[10:51:27] 19 tRNA-Met [53887,53963] 34 (cat)\\n[10:51:27] 20 tRNA-Thr [53971,54045] 35 (tgt)\\n[10:51:27] 21 tRNA-Gly [54049,54121] 34 (tcc)\\n[10:51:27] 22 tRNA-Val [54207,54282] 34 (tac)\\n[10:51:27] 23 tRNA-Arg [54685,54760] 35 (tct)\\n[10:51:27] 24 tRNA-Ala [54842,54916] 35 (tgc)\\n[10:51:27] 25 tRNA-His [54922,54997] 35 (gtg)\\n[10:51:27] 26 tRNA-Ser [55003,55086] 34 (tga)\\n[10:51:27] 27 tRNA-Lys [55342,55413] 33 (ttt)\\n[10:51:27] 28 tRNA-Ser [59461,59552] 35 (gct)\\n[10:51:27] 29 tRNA-Ala c[64843,64915] 34 (ggc)\\n[10:51:27] 1 tRNA-Leu [9415,9497] 35 (tag)\\n[10:51:27] 1 tRNA-Asn c[47725,47796] 33 (gtt)\\n[10:51:27] 1 tRNA-Phe c[61120,61192] 33 (gaa)\\n[10:51:27] 2 tRNA-Pyl c[63130,63203] 34 (cta)\\n[10:51:27] 1 tmRNA [80822,81209] 101,151 ANNIVSFARKDALVAA*\\n[10:51:27] 1 tRNA-Gln c[49867,49940] 34 (ttg)\\n[10:51:27] 2 tRNA-Leu c[51823,51907] 35 (caa)\\n[10:51:27] 3 tRNA-Leu [56402,56482] 34 (cag)\\n[10:51:27] 1 tRNA-Arg [52603,52678] 36 (cct)\\n[10:51:27] 1 tRNA-Thr [9837,9908] 33 (cgt)\\n[10:51:27] 2 tRNA-Thr c[20799,20870] 33 (ggt)\\n[10:51:27] 3 tRNA-Tyr c[20958,21043] 35 (gta)\\n[10:51:27] 4 tRNA-Ser [23341,23427] 36 (tga)\\n[10:51:27] 5 tRNA-Val [42481,42553] 34 (tac)\\n[10:51:27] 1 tRNA-Cys [12132,12202] 33 (gca)\\n[10:51:27] 1 tRNA-Met [45110,45186] 35 (cat)\\n[10:51:27] 1 tRNA-Ala [54610,54682] 34 (cgc)\\n[10:51:27] 1 tRNA-Ala c[70424,70498] 34 (tgc)\\n[10:51:27] 2 tRNA-Ile c[70577,70652] 35 (gat)\\n[10:51:27] 1 tRNA-Val c[610,683] 35 (gac)\\n[10:51:27] 1 tRNA-Arg c[16729,16803] 35 (tct)\\n[10:51:27] 1 tRNA-Met c[18693,18767] 35 (cat)\\n[10:51:27] 2 tRNA-Pro [24951,25024] 35 (ggg)\\n[10:51:27] 1 tRNA-Ser [14965,15051] 36 (cga)\\n[10:51:27] 1 tRNA-Arg [10901,10974] 35 (acg)\\n[10:51:27] 1 tRNA-Lys [13613,13690] 36 (ctt)\\n[10:51:27] 2 tRNA-Met [13690,13766] 35 (cat)\\n[10:51:27] 3 tRNA-Ile [15385,15460] 34 (tat)\\n[10:51:27] 1 tRNA-Lys c[25564,25636] 33 (ctt)\\n[10:51:27] 1 tRNA-Glu [21317,21390] 34 (ttc)\\n[10:51:27] 1 tRNA-Phe [4882,4954] 34 (gaa)\\n[10:51:27] 1 tRNA-Trp [8391,8464] 34 (cca)\\n[10:51:27] 2 tRNA-Ala c[14904,14978] 35 (tgc)\\n[10:51:27] 1 tRNA-Asp [18935,19012] 38 (gtc)\\n[10:51:27] 2 tRNA-Glu [19015,19091] 35 (ctc)\\n[10:51:27] 3 tRNA-Arg [19094,19169] 36 (ccg)\\n[10:51:27] 4 tRNA-Cys [19175,19248] 33 (gca)\\n[10:51:27] 5 tRNA-Met [19254,19330] 35 (cat)\\n[10:51:27] 6 tRNA-Trp [19333,19405] 34 (cca)\\n[10:51:27] 7 tRNA-Lys [19430,19501] 33 (ctt)\\n[10:51:27] 8 tRNA-Glu [19532,19607] 34 (ttc)\\n[10:51:27] 9 tRNA-Tyr [19611,19693] 36 (gta)\\n[10:51:27] 10 tRNA-Ser [19697,19786] 33 (gct)\\n[10:51:27] 11 tRNA-Arg [19791,19866] 35 (acg)\\n[10:51:27] 12 tRNA-Phe [19871,19945] 35 (gaa)\\n[10:51:27] 13 tRNA-Asn [19954,20028] 33 (gtt)\\n[10:51:27] 14 tRNA-Gln [20116,20190] 35 (ttg)\\n[10:51:27] 15 tRNA-Pro [20195,20270] 36 (tgg)\\n[10:51:27] 16 tRNA-Lys [20275,20349] 34 (ttt)\\n[10:51:27] 17 tRNA-Leu [20352,20426] 35 (taa)\\n[10:51:27] 18 tRNA-Thr [20436,20508] 34 (tgt)\\n[10:51:27] 19 tRNA-Gly [20514,20584] 33 (tcc)\\n[10:51:27] 20 tRNA-Val [20590,20665] 34 (tac)\\n[10:51:27] 21 tRNA-Arg [20672,20745] 34 (tct)\\n[10:51:27] 22 tRNA-Ala [20750,20824] 35 (tgc)\\n[10:51:27] 23 tRNA-His [20830,20905] 35 (gtg)\\n[10:51:27] 24 tRNA-Ser [20912,20996] 34 (tga)\\n[10:51:27] 1 tRNA-Ser [528,616] 36 (gga)\\n[10:51:27] 1 tRNA-His c[12096,12169] 34 (gtg)\\n[10:51:27] 1 tRNA-Thr [6899,6970] 33 (tgt)\\n[10:51:27] 2 tRNA-Met c[8352,8425] 33 (cat)\\n[10:51:27] 1 tRNA-Pro c[6751,6824] 35 (cgg)\\n[10:51:27] 1 tRNA-Lys [1088,1159] 33 (ttt)\\n[10:51:27] 1 tRNA-Val [8543,8615] 33 (cac)\\n[10:51:27] 1 tRNA-Gly [6762,6833] 33 (gcc)\\n[10:51:27] 1 tRNA-Leu c[4413,4497] 36 (gag)\\n[10:51:27] Found 99 tRNAs\\n[10:51:27] Predicting Ribosomal RNAs\\n[10:51:27] Running Barrnap with 4 threads\\n[10:51:29] 1 NOLK01000032.1 67344 23S ribosomal RNA\\n[10:51:29] 2 NOLK01000032.1 70786 16S ribosomal RNA\\n[10:51:29] Found 2 rRNAs\\n[10:51:29] Skipping ncRNA search, enable with --rfam if desired.\\n[10:51:29] Total of 100 tRNA + rRNA features\\n[10:51:29] Searching for CRISPR repeats\\n[10:51:29] CRISPR1 NOLK01000004.1 99951 with 7 spacers\\n[10:51:29] CRISPR2 NOLK01000013.1 47663 with 10 spacers\\n[10:51:29] CRISPR3 NOLK01000021.1 25867 with 6 spacers\\n[10:51:29] CRISPR4 NOLK01000041.1 15868 with 13 spacers\\n[10:51:29] CRISPR5 NOLK01000041.1 63607 with 7 spacers\\n[10:51:29] CRISPR6 NOLK01000046.1 58971 with 6 spacers\\n[10:51:29] CRISPR7 NOLK01000049.1 6839 with 5 spacers\\n[10:51:29] CRISPR8 NOLK01000049.1 16859 with 8 spacers\\n[10:51:29] CRISPR9 NOLK01000049.1 19714 with 6 spacers\\n[10:51:29] CRISPR10 NOLK01000049.1 27185 with 3 spacers\\n[10:51:29] CRISPR11 NOLK01000049.1 27389 with 11 spacers\\n[10:51:29] CRISPR12 NOLK01000049.1 41926 with 14 spacers\\n[10:51:29] CRISPR13 NOLK01000058.1 9250 with 13 spacers\\n[10:51:29] CRISPR14 NOLK01000067.1 17571 with 6 spacers\\n[10:51:29] CRISPR15 NOLK01000081.1 23529 with 10 spacers\\n[10:51:29] CRISPR16 NOLK01000118.1 11491 with 9 spacers\\n[10:51:30] CRISPR17 NOLK01000147.1 19514 with 6 spacers\\n[10:51:30] CRISPR18 NOLK01000151.1 193 with 6 spacers\\n[10:51:30] CRISPR19 NOLK01000198.1 12050 with 3 spacers\\n[10:51:30] CRISPR20 NOLK01000205.1 372 with 24 spacers\\n[10:51:30] CRISPR21 NOLK01000205.1 2222 with 6 spacers\\n[10:51:30] CRISPR22 NOLK01000210.1 9902 with 12 spacers\\n[10:51:30] CRISPR23 NOLK01000245.1 674 with 5 spacers\\n[10:51:30] CRISPR24 NOLK01000245.1 1284 with 22 spacers\\n[10:51:30] CRISPR25 NOLK01000281.1 2450 with 9 spacers\\n[10:51:30] CRISPR26 NOLK01000328.1 1046 with 6 spacers\\n[10:51:30] Found 26 CRISPRs\\n[10:51:30] Predicting coding sequences\\n[10:51:30] Contigs total 9158766 bp, so using single mode\\n[10:51:30] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_NOLK01\\\\/\\\\/Nostoc_NOLK01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:51:38] Excluding CDS which overlaps existing RNA (tRNA) at NOLK01000007.1:54956..55138 on + strand\\n[10:51:38] Excluding CDS which overlaps existing RNA (tRNA) at NOLK01000008.1:9421..9564 on + strand\\n[10:51:38] Excluding CDS which overlaps existing RNA (repeat_region) at NOLK01000021.1:25962..26195 on + strand\\n[10:51:39] Excluding CDS which overlaps existing RNA (rRNA) at NOLK01000032.1:70828..71142 on - strand\\n[10:51:39] Excluding CDS which overlaps existing RNA (repeat_region) at NOLK01000046.1:59142..59426 on + strand\\n[10:51:40] Excluding CDS which overlaps existing RNA (repeat_region) at NOLK01000049.1:17114..17236 on - strand\\n[10:51:40] Excluding CDS which overlaps existing RNA (repeat_region) at NOLK01000049.1:17221..17448 on - strand\\n[10:51:40] Excluding CDS which overlaps existing RNA (repeat_region) at NOLK01000058.1:10148..10390 on - strand\\n[10:51:40] Excluding CDS which overlaps existing RNA (repeat_region) at NOLK01000081.1:23301..23555 on + strand\\n[10:51:40] Excluding CDS which overlaps existing RNA (repeat_region) at NOLK01000081.1:23527..23787 on - strand\\n[10:51:41] Excluding CDS which overlaps existing RNA (tRNA) at NOLK01000099.1:8367..8513 on + strand\\n[10:51:41] Excluding CDS which overlaps existing RNA (repeat_region) at NOLK01000118.1:11428..11574 on - strand\\n[10:51:41] Excluding CDS which overlaps existing RNA (repeat_region) at NOLK01000147.1:19614..19913 on + strand\\n[10:51:41] Excluding CDS which overlaps existing RNA (repeat_region) at NOLK01000205.1:436..528 on + strand\\n[10:51:41] Excluding CDS which overlaps existing RNA (repeat_region) at NOLK01000210.1:10642..10857 on + strand\\n[10:51:42] Excluding CDS which overlaps existing RNA (repeat_region) at NOLK01000245.1:337..687 on + strand\\n[10:51:42] Excluding CDS which overlaps existing RNA (repeat_region) at NOLK01000245.1:1624..2151 on - strand\\n[10:51:42] Excluding CDS which overlaps existing RNA (repeat_region) at NOLK01000281.1:2827..3000 on - strand\\n[10:51:42] Found 8042 CDS\\n[10:51:42] Connecting features back to sequences\\n[10:51:42] Not using genus-specific database. Try --usegenus to enable it.\\n[10:51:42] Annotating CDS, please be patient.\\n[10:51:42] Will use 4 CPUs for similarity searching.\\n[10:51:44] There are still 8042 unannotated CDS left (started with 8042)\\n[10:51:44] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:51:44] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_NOLK01\\\\/\\\\/Nostoc_NOLK01\\\\.IS\\\\.tmp\\\\.6476\\\\.faa | parallel --gnu --plain -j 4 --block 310867 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_NOLK01\\\\/\\\\/Nostoc_NOLK01\\\\.IS\\\\.tmp\\\\.6476\\\\.blast 2> /dev/null\\n[10:51:56] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_NOLK01\\\\/\\\\/Nostoc_NOLK01\\\\.IS\\\\.tmp\\\\.6476\\\\.faa | parallel --gnu --plain -j 4 --block 310867 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_NOLK01\\\\/\\\\/Nostoc_NOLK01\\\\.IS\\\\.tmp\\\\.6476\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:51:57] This is prokka 1.14.6\\n[10:51:57] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:51:57] Homepage is https://github.com/tseemann/prokka\\n[10:51:57] Local time is Mon Feb 21 10:51:57 2022\\n[10:51:57] You are tiagoferreiraleao\\n[10:51:57] Operating system is darwin\\n[10:51:57] You have BioPerl 1.007002\\n[10:51:57] System has 4 cores.\\n[10:51:57] Option --cpu asked for 12 cores, but system only has 4\\n[10:51:57] Will use maximum of 4 cores.\\n[10:51:57] Annotating as >>> Bacteria <<<\\n[10:51:57] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQK01.fasta' contents.\\n[10:51:57] Setting --locustag HIDJNAKN from MD5 12d37a477b327e18f027a94c104938aa\\n[10:51:57] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQK01/\\n[10:51:57] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQK01\\\\/\\n[10:51:57] Using filename prefix: Fischerella_NMQK01.XXX\\n[10:51:57] Setting HMMER_NCPU=1\\n[10:51:57] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQK01//Fischerella_NMQK01.log\\n[10:51:57] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQK01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQK01/ --prefix Fischerella_NMQK01 --cpus 12\\n[10:51:57] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:51:57] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:51:57] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:51:57] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:51:57] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:51:57] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:51:57] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:51:57] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:51:57] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:51:57] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:51:57] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:51:57] Looking for 'egrep' - found /usr/bin/egrep\\n[10:51:57] Looking for 'find' - found /usr/bin/find\\n[10:51:57] Looking for 'grep' - found /usr/bin/grep\\n[10:51:57] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:51:57] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:51:57] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:51:57] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:51:57] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:51:57] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:51:57] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:51:57] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:51:58] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:51:58] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:51:58] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:51:58] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:51:58] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:51:58] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:51:58] Looking for 'sed' - found /usr/bin/sed\\n[10:51:58] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:51:58] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:51:58] Using genetic code table 11.\\n[10:51:58] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQK01.fasta\\n[10:51:58] Wrote 852 contigs totalling 6721469 bp.\\n[10:51:58] Predicting tRNAs and tmRNAs\\n[10:51:58] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQK01\\\\/\\\\/Fischerella_NMQK01\\\\.fna\\n[10:52:00] 1 tRNA-Gln c[3534,3605] 33 (ttg)\\n[10:52:00] 1 tRNA-Pro c[15019,15092] 35 (cgg)\\n[10:52:00] 1 tRNA-Leu c[532,616] 35 (caa)\\n[10:52:00] 1 tRNA-Leu c[9880,9962] 35 (cag)\\n[10:52:00] 1 tRNA-Met [8776,8848] 34 (cat)\\n[10:52:00] 1 tRNA-Ser [7414,7507] 36 (gct)\\n[10:52:00] 1 tRNA-Gly [6708,6778] 33 (tcc)\\n[10:52:00] 1 tRNA-His [5541,5614] 34 (gtg)\\n[10:52:00] 1 tRNA-Val c[6231,6303] 34 (tac)\\n[10:52:00] 1 tRNA-Arg c[13374,13448] 35 (tct)\\n[10:52:00] 1 tRNA-Asp c[7814,7887] 35 (gtc)\\n[10:52:00] 1 tRNA-Cys c[11906,11977] 33 (gca)\\n[10:52:00] 1 tRNA-Ala c[13193,13267] 35 (cgc)\\n[10:52:00] 1 tRNA-Gly [16345,16416] 33 (gcc)\\n[10:52:00] 1 tRNA-Met c[10093,10168] 36 (cat)\\n[10:52:00] 1 tRNA-Arg c[2953,3026] 35 (acg)\\n[10:52:00] 2 tRNA-Ala c[8115,8187] 34 (ggc)\\n[10:52:00] 3 tRNA-Thr c[9663,9734] 33 (cgt)\\n[10:52:00] 1 tRNA-Pro c[18945,19018] 35 (ggg)\\n[10:52:00] 1 tmRNA [2825,3211] 100,153 ANNIVKFARKEALVAA**\\n[10:52:00] 1 tRNA-Gly c[5791,5860] 32 (tcc)\\n[10:52:00] 2 tRNA-Gln c[5863,5933] 32 (ttg)\\n[10:52:00] 1 tRNA-Thr c[9253,9324] 33 (tgt)\\n[10:52:00] 1 tRNA-Leu c[1122,1204] 36 (gag)\\n[10:52:00] 1 tRNA-Gly c[388,459] 33 (ccc)\\n[10:52:00] 1 tRNA-Trp c[4287,4360] 34 (cca)\\n[10:52:00] 1 tRNA-Ser [24354,24440] 36 (tga)\\n[10:52:01] 1 tRNA-Phe [9105,9177] 34 (gaa)\\n[10:52:01] 1 tRNA-Pro c[544,618] 35 (tgg)\\n[10:52:01] 1 tRNA-Ser c[6100,6186] 36 (cga)\\n[10:52:01] 1 tRNA-Asn [2819,2892] 34 (gtt)\\n[10:52:01] 1 tRNA-Tyr [107,191] 35 (gta)\\n[10:52:01] 2 tRNA-Thr [254,325] 33 (ggt)\\n[10:52:01] 1 tRNA-Ser c[4395,4483] 36 (gga)\\n[10:52:01] 1 tRNA-Met c[157,233] 35 (cat)\\n[10:52:01] 2 tRNA-Val [3217,3290] 35 (gac)\\n[10:52:01] 1 tRNA-Lys [20338,20411] 34 (ttt)\\n[10:52:01] 1 tRNA-Arg [400,473] 35 (ccg)\\n[10:52:01] 1 tRNA-Arg [860,932] 34 (tcg)\\n[10:52:01] 2 tRNA-Asn [1658,1731] 34 (att)\\n[10:52:01] 1 tRNA-Leu c[978,1061] 36 (tag)\\n[10:52:01] 1 tRNA-Glu [7863,7935] 35 (ttc)\\n[10:52:01] Found 42 tRNAs\\n[10:52:01] Predicting Ribosomal RNAs\\n[10:52:01] Running Barrnap with 4 threads\\n[10:52:02] 1 NMQK01000076.1 5 16S ribosomal RNA\\n[10:52:02] 2 NMQK01000708.1 149 23S ribosomal RNA\\n[10:52:02] Found 2 rRNAs\\n[10:52:02] Skipping ncRNA search, enable with --rfam if desired.\\n[10:52:02] Total of 43 tRNA + rRNA features\\n[10:52:02] Searching for CRISPR repeats\\n[10:52:03] CRISPR1 NMQK01000142.1 232 with 12 spacers\\n[10:52:03] CRISPR2 NMQK01000192.1 1723 with 17 spacers\\n[10:52:03] CRISPR3 NMQK01000192.1 2935 with 3 spacers\\n[10:52:03] CRISPR4 NMQK01000420.1 2432 with 5 spacers\\n[10:52:03] CRISPR5 NMQK01000575.1 18974 with 26 spacers\\n[10:52:03] CRISPR6 NMQK01000582.1 16071 with 17 spacers\\n[10:52:03] CRISPR7 NMQK01000676.1 387 with 4 spacers\\n[10:52:03] CRISPR8 NMQK01000688.1 19576 with 3 spacers\\n[10:52:03] CRISPR9 NMQK01000771.1 1304 with 6 spacers\\n[10:52:03] CRISPR10 NMQK01000824.1 5655 with 9 spacers\\n[10:52:03] Found 10 CRISPRs\\n[10:52:03] Predicting coding sequences\\n[10:52:03] Contigs total 6721469 bp, so using single mode\\n[10:52:03] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQK01\\\\/\\\\/Fischerella_NMQK01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:52:09] Excluding CDS which overlaps existing RNA (tRNA) at NMQK01000163.1:6190..6384 on - strand\\n[10:52:09] Excluding CDS which overlaps existing RNA (tRNA) at NMQK01000236.1:2910..3581 on + strand\\n[10:52:10] Excluding CDS which overlaps existing RNA (repeat_region) at NMQK01000420.1:2231..2455 on + strand\\n[10:52:10] Excluding CDS which overlaps existing RNA (repeat_region) at NMQK01000676.1:175..396 on - strand\\n[10:52:10] Excluding CDS which overlaps existing RNA (repeat_region) at NMQK01000676.1:491..949 on + strand\\n[10:52:10] Excluding CDS which overlaps existing RNA (rRNA) at NMQK01000708.1:543..722 on + strand\\n[10:52:10] Excluding CDS which overlaps existing RNA (tRNA) at NMQK01000746.1:674..961 on + strand\\n[10:52:11] Found 5668 CDS\\n[10:52:11] Connecting features back to sequences\\n[10:52:11] Not using genus-specific database. Try --usegenus to enable it.\\n[10:52:11] Annotating CDS, please be patient.\\n[10:52:11] Will use 4 CPUs for similarity searching.\\n[10:52:12] There are still 5668 unannotated CDS left (started with 5668)\\n[10:52:12] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:52:12] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQK01\\\\/\\\\/Fischerella_NMQK01\\\\.IS\\\\.tmp\\\\.6590\\\\.faa | parallel --gnu --plain -j 4 --block 223179 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQK01\\\\/\\\\/Fischerella_NMQK01\\\\.IS\\\\.tmp\\\\.6590\\\\.blast 2> /dev/null\\n[10:52:30] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQK01\\\\/\\\\/Fischerella_NMQK01\\\\.IS\\\\.tmp\\\\.6590\\\\.faa | parallel --gnu --plain -j 4 --block 223179 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQK01\\\\/\\\\/Fischerella_NMQK01\\\\.IS\\\\.tmp\\\\.6590\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:52:30] This is prokka 1.14.6\\n[10:52:30] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:52:30] Homepage is https://github.com/tseemann/prokka\\n[10:52:30] Local time is Mon Feb 21 10:52:30 2022\\n[10:52:30] You are tiagoferreiraleao\\n[10:52:30] Operating system is darwin\\n[10:52:30] You have BioPerl 1.007002\\n[10:52:31] System has 4 cores.\\n[10:52:31] Option --cpu asked for 12 cores, but system only has 4\\n[10:52:31] Will use maximum of 4 cores.\\n[10:52:31] Annotating as >>> Bacteria <<<\\n[10:52:31] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHA01.fasta' contents.\\n[10:52:31] Setting --locustag JAOGCPMI from MD5 3a80c962e4d66485c94777386696af03\\n[10:52:31] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHA01/\\n[10:52:31] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHA01\\\\/\\n[10:52:31] Using filename prefix: Nostoc_LAHA01.XXX\\n[10:52:31] Setting HMMER_NCPU=1\\n[10:52:31] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHA01//Nostoc_LAHA01.log\\n[10:52:31] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHA01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHA01/ --prefix Nostoc_LAHA01 --cpus 12\\n[10:52:31] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:52:31] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:52:31] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:52:31] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:52:31] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:52:31] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:52:31] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:52:31] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:52:31] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:52:31] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:52:31] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:52:31] Looking for 'egrep' - found /usr/bin/egrep\\n[10:52:31] Looking for 'find' - found /usr/bin/find\\n[10:52:31] Looking for 'grep' - found /usr/bin/grep\\n[10:52:31] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:52:31] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:52:31] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:52:31] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:52:31] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:52:31] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:52:31] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:52:31] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:52:31] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:52:31] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:52:32] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:52:32] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:52:32] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:52:32] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:52:32] Looking for 'sed' - found /usr/bin/sed\\n[10:52:32] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:52:32] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:52:32] Using genetic code table 11.\\n[10:52:32] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHA01.fasta\\n[10:52:32] Wrote 131 contigs totalling 8997966 bp.\\n[10:52:32] Predicting tRNAs and tmRNAs\\n[10:52:32] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHA01\\\\/\\\\/Nostoc_LAHA01\\\\.fna\\n[10:52:35] 1 tRNA-Ala c[117189,117261] 34 (tgc)\\n[10:52:35] 2 tRNA-Ser c[233480,233573] 36 (gct)\\n[10:52:35] 3 tRNA-Lys c[239248,239318] 33 (ttt)\\n[10:52:35] 4 tRNA-Ser c[239577,239660] 34 (tga)\\n[10:52:35] 5 tRNA-His c[239665,239740] 35 (gtg)\\n[10:52:35] 6 tRNA-Ala c[239746,239820] 35 (tgc)\\n[10:52:35] 7 tRNA-Arg c[239912,239987] 35 (tct)\\n[10:52:35] 8 tRNA-Val c[239994,240069] 34 (tac)\\n[10:52:35] 9 tRNA-Thr c[240151,240223] 34 (tgt)\\n[10:52:35] 10 tRNA-Leu c[240518,240592] 35 (tag)\\n[10:52:35] 11 tRNA-Leu c[240704,240780] 34 (gag)\\n[10:52:35] 12 tRNA-Pro c[240888,240961] 35 (tgg)\\n[10:52:35] 13 tRNA-Leu c[240963,241040] 35 (taa)\\n[10:52:35] 14 tRNA-Leu c[241061,241138] 36 (caa)\\n[10:52:35] 15 tRNA-Gln c[241164,241239] 35 (ttg)\\n[10:52:35] 16 tRNA-Gln c[242114,242188] 35 (ctg)\\n[10:52:35] 17 tRNA-Asn c[242199,242273] 33 (gtt)\\n[10:52:35] 18 tRNA-Phe c[242279,242354] 36 (gaa)\\n[10:52:35] 19 tRNA-Arg c[242442,242517] 35 (acg)\\n[10:52:35] 20 tRNA-Ser c[243279,243365] 35 (gct)\\n[10:52:35] 21 tRNA-Tyr c[243369,243450] 35 (gta)\\n[10:52:35] 22 tRNA-Glu c[243461,243535] 35 (ttc)\\n[10:52:35] tRNA c[243623,243697] is a pseudo/wacky gene - skipping.\\n[10:52:35] 24 tRNA-Met c[243700,243773] 32 (cat)\\n[10:52:35] 25 tRNA-Glu c[243883,243959] 35 (ctc)\\n[10:52:35] 26 tRNA-Asp c[243963,244038] 36 (gtc)\\n[10:52:35] 27 tRNA-Glu [317682,317755] 35 (ttc)\\n[10:52:35] 28 tRNA-Ala [486870,486942] 34 (ggc)\\n[10:52:35] 1 tRNA-Thr [200657,200728] 33 (cgt)\\n[10:52:35] 2 tRNA-Trp [246287,246360] 34 (cca)\\n[10:52:35] 1 tRNA-Gly c[235647,235717] 33 (tcc)\\n[10:52:35] 1 tRNA-Arg c[25347,25419] 34 (ccg)\\n[10:52:35] 2 tRNA-Cys [261165,261236] 33 (gca)\\n[10:52:35] 1 tmRNA c[194102,194489] 101,151 ANNIVKFARREALVAA*\\n[10:52:35] 1 tRNA-Arg [143741,143815] 35 (tct)\\n[10:52:35] 1 tRNA-Asn [2942,3015] 34 (gtt)\\n[10:52:35] 2 tRNA-Leu c[102607,102690] 36 (tag)\\n[10:52:35] 3 tRNA-Ala c[137022,137096] 34 (tgc)\\n[10:52:35] 4 tRNA-Ile c[137180,137255] 35 (gat)\\n[10:52:35] 1 tRNA-Lys c[32111,32182] 33 (ttt)\\n[10:52:35] 1 tRNA-Val c[81545,81617] 34 (tac)\\n[10:52:35] 1 tRNA-Gln c[29313,29386] 34 (ttg)\\n[10:52:35] 2 tRNA-Leu c[34140,34224] 35 (caa)\\n[10:52:35] 3 tRNA-Ala [82926,82998] 34 (cgc)\\n[10:52:35] 1 tRNA-Ser [53919,54006] 35 (gga)\\n[10:52:35] 2 tRNA-Gly c[62151,62224] 34 (ccc)\\n[10:52:35] 1 tRNA-Pro [86227,86300] 35 (cgg)\\n[10:52:35] 1 tRNA-Ala c[16947,17033] 29 (cgc)\\n[10:52:35] 2 tRNA-Ser [131832,131918] 36 (cga)\\n[10:52:35] 1 tRNA-His [18843,18916] 34 (gtg)\\n[10:52:35] 1 tRNA-Leu c[96226,96306] 35 (gag)\\n[10:52:35] 1 tRNA-Leu [44347,44428] 35 (cag)\\n[10:52:35] 1 tRNA-Ile c[45740,45815] 34 (tat)\\n[10:52:35] 2 tRNA-Met c[47421,47496] 34 (cat)\\n[10:52:35] 3 tRNA-Met c[47498,47573] 34 (cat)\\n[10:52:35] 1 tRNA-Pro c[39908,39981] 35 (tgg)\\n[10:52:35] 2 tRNA-Ser c[48663,48749] 36 (tga)\\n[10:52:35] 3 tRNA-Tyr [51200,51284] 35 (gta)\\n[10:52:35] 4 tRNA-Thr [51360,51431] 33 (ggt)\\n[10:52:35] 1 tRNA-Pro c[43859,43932] 35 (ggg)\\n[10:52:35] 1 tRNA-Met [19108,19181] 33 (cat)\\n[10:52:35] 2 tRNA-Gly [25991,26062] 33 (gcc)\\n[10:52:35] 1 tRNA-Thr [29991,30063] 33 (tgt)\\n[10:52:35] 1 tRNA-Lys c[21227,21299] 33 (ctt)\\n[10:52:35] 1 tRNA-Leu c[63522,63596] 35 (caa)\\n[10:52:35] 2 tRNA-Thr c[64987,65061] 33 (agt)\\n[10:52:35] 1 tRNA-Asp [25380,25457] 38 (gtc)\\n[10:52:35] 2 tRNA-Glu [25460,25536] 35 (ctc)\\n[10:52:35] 3 tRNA-Cys [25541,25616] 34 (gca)\\n[10:52:35] 4 tRNA-Met [25622,25696] 33 (cat)\\n[10:52:35] 5 tRNA-Trp [25699,25771] 34 (cca)\\n[10:52:35] 6 tRNA-Pyl [25796,25867] 33 (cta)\\n[10:52:35] 7 tRNA-Glu [25898,25973] 34 (ttc)\\n[10:52:35] 8 tRNA-Tyr [25977,26060] 36 (gta)\\n[10:52:35] 9 tRNA-Ser [26064,26153] 33 (gct)\\n[10:52:35] 10 tRNA-Arg [26158,26233] 35 (acg)\\n[10:52:35] 11 tRNA-Phe [26238,26312] 35 (gaa)\\n[10:52:35] 12 tRNA-Asn [26321,26395] 33 (gtt)\\n[10:52:35] 13 tRNA-Gln [26401,26476] 35 (ttg)\\n[10:52:35] 14 tRNA-Leu [26484,26561] 35 (caa)\\n[10:52:35] 15 tRNA-Pro [26566,26642] 35 (tgg)\\n[10:52:35] 16 tRNA-Leu [26646,26720] 35 (tag)\\n[10:52:35] 17 tRNA-Leu [26740,26817] 35 (gag)\\n[10:52:35] 18 tRNA-Lys [26821,26897] 34 (ttt)\\n[10:52:35] 19 tRNA-Lys [26903,26977] 33 (ctt)\\n[10:52:35] 20 tRNA-Leu [26983,27058] 36 (taa)\\n[10:52:35] 21 tRNA-Cys [27068,27145] 34 (gca)\\n[10:52:35] 22 tRNA-Thr [27153,27225] 34 (tgt)\\n[10:52:35] 23 tRNA-Gly [27231,27301] 33 (tcc)\\n[10:52:35] 24 tRNA-Val [27307,27382] 34 (tac)\\n[10:52:35] 25 tRNA-Arg [27389,27463] 35 (tct)\\n[10:52:35] 26 tRNA-Ala [27468,27540] 34 (tgc)\\n[10:52:35] 27 tRNA-His [27548,27624] 35 (gtg)\\n[10:52:35] 28 tRNA-Ser [27635,27719] 34 (tga)\\n[10:52:35] 1 tRNA-Phe c[899,972] 34 (gaa)\\n[10:52:35] 1 tRNA-Asp c[60246,60319] 35 (gtc)\\n[10:52:35] 1 tRNA-Met c[40922,40996] 35 (cat)\\n[10:52:35] 1 tRNA-Arg c[44242,44315] 35 (acg)\\n[10:52:35] 1 tRNA-Val [14092,14165] 35 (gac)\\n[10:52:35] tRNA [22,97] is a pseudo/wacky gene - skipping.\\n[10:52:35] Found 98 tRNAs\\n[10:52:35] Predicting Ribosomal RNAs\\n[10:52:35] Running Barrnap with 4 threads\\n[10:52:37] 1 LAHA01000010.1 134026 23S ribosomal RNA\\n[10:52:37] 2 LAHA01000010.1 137377 16S ribosomal RNA\\n[10:52:37] 3 LAHA01000130.1 612 23S ribosomal RNA (partial)\\n[10:52:37] Found 3 rRNAs\\n[10:52:37] Skipping ncRNA search, enable with --rfam if desired.\\n[10:52:37] Total of 100 tRNA + rRNA features\\n[10:52:37] Searching for CRISPR repeats\\n[10:52:37] CRISPR1 LAHA01000002.1 150001 with 24 spacers\\n[10:52:37] CRISPR2 LAHA01000002.1 303983 with 18 spacers\\n[10:52:37] CRISPR3 LAHA01000004.1 80810 with 7 spacers\\n[10:52:37] CRISPR4 LAHA01000005.1 103120 with 15 spacers\\n[10:52:37] CRISPR5 LAHA01000006.1 169474 with 14 spacers\\n[10:52:37] CRISPR6 LAHA01000024.1 66898 with 10 spacers\\n[10:52:37] CRISPR7 LAHA01000035.1 29862 with 20 spacers\\n[10:52:38] CRISPR8 LAHA01000042.1 4935 with 6 spacers\\n[10:52:38] CRISPR9 LAHA01000062.1 26332 with 28 spacers\\n[10:52:38] CRISPR10 LAHA01000062.1 39765 with 43 spacers\\n[10:52:38] CRISPR11 LAHA01000095.1 10422 with 7 spacers\\n[10:52:38] CRISPR12 LAHA01000102.1 908 with 24 spacers\\n[10:52:38] CRISPR13 LAHA01000108.1 9923 with 18 spacers\\n[10:52:38] Found 13 CRISPRs\\n[10:52:38] Predicting coding sequences\\n[10:52:38] Contigs total 8997966 bp, so using single mode\\n[10:52:38] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHA01\\\\/\\\\/Nostoc_LAHA01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:52:46] Excluding CDS which overlaps existing RNA (repeat_region) at LAHA01000006.1:170190..170306 on + strand\\n[10:52:48] Excluding CDS which overlaps existing RNA (tRNA) at LAHA01000023.1:16482..17852 on - strand\\n[10:52:48] Excluding CDS which overlaps existing RNA (repeat_region) at LAHA01000024.1:67107..67259 on + strand\\n[10:52:48] Excluding CDS which overlaps existing RNA (tRNA) at LAHA01000025.1:18764..18907 on - strand\\n[10:52:49] Excluding CDS which overlaps existing RNA (repeat_region) at LAHA01000042.1:5327..6286 on - strand\\n[10:52:49] Excluding CDS which overlaps existing RNA (repeat_region) at LAHA01000062.1:39806..39970 on - strand\\n[10:52:50] Excluding CDS which overlaps existing RNA (repeat_region) at LAHA01000095.1:10450..10575 on - strand\\n[10:52:50] Excluding CDS which overlaps existing RNA (repeat_region) at LAHA01000102.1:2514..2792 on - strand\\n[10:52:50] Excluding CDS which overlaps existing RNA (rRNA) at LAHA01000130.1:165..641 on - strand\\n[10:52:50] Excluding CDS which overlaps existing RNA (rRNA) at LAHA01000130.1:1843..2415 on - strand\\n[10:52:50] Found 7604 CDS\\n[10:52:50] Connecting features back to sequences\\n[10:52:50] Not using genus-specific database. Try --usegenus to enable it.\\n[10:52:50] Annotating CDS, please be patient.\\n[10:52:50] Will use 4 CPUs for similarity searching.\\n[10:52:52] There are still 7604 unannotated CDS left (started with 7604)\\n[10:52:52] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:52:52] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHA01\\\\/\\\\/Nostoc_LAHA01\\\\.IS\\\\.tmp\\\\.6704\\\\.faa | parallel --gnu --plain -j 4 --block 304754 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHA01\\\\/\\\\/Nostoc_LAHA01\\\\.IS\\\\.tmp\\\\.6704\\\\.blast 2> /dev/null\\n[10:53:01] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHA01\\\\/\\\\/Nostoc_LAHA01\\\\.IS\\\\.tmp\\\\.6704\\\\.faa | parallel --gnu --plain -j 4 --block 304754 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHA01\\\\/\\\\/Nostoc_LAHA01\\\\.IS\\\\.tmp\\\\.6704\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:53:01] This is prokka 1.14.6\\n[10:53:01] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:53:01] Homepage is https://github.com/tseemann/prokka\\n[10:53:01] Local time is Mon Feb 21 10:53:01 2022\\n[10:53:01] You are tiagoferreiraleao\\n[10:53:01] Operating system is darwin\\n[10:53:01] You have BioPerl 1.007002\\n[10:53:02] System has 4 cores.\\n[10:53:02] Option --cpu asked for 12 cores, but system only has 4\\n[10:53:02] Will use maximum of 4 cores.\\n[10:53:02] Annotating as >>> Bacteria <<<\\n[10:53:02] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJTU01.fasta' contents.\\n[10:53:02] Setting --locustag JIIMNJOD from MD5 3226738da8c84da58eb885f4520e1250\\n[10:53:02] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJTU01/\\n[10:53:02] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTU01\\\\/\\n[10:53:02] Using filename prefix: Nostoc_JACJTU01.XXX\\n[10:53:02] Setting HMMER_NCPU=1\\n[10:53:02] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJTU01//Nostoc_JACJTU01.log\\n[10:53:02] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJTU01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACJTU01/ --prefix Nostoc_JACJTU01 --cpus 12\\n[10:53:02] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:53:02] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:53:02] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:53:02] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:53:02] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:53:02] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:53:02] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:53:02] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:53:02] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:53:02] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:53:02] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:53:02] Looking for 'egrep' - found /usr/bin/egrep\\n[10:53:02] Looking for 'find' - found /usr/bin/find\\n[10:53:02] Looking for 'grep' - found /usr/bin/grep\\n[10:53:02] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:53:02] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:53:02] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:53:02] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:53:02] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:53:02] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:53:02] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:53:02] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:53:03] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:53:03] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:53:03] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:53:03] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:53:03] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:53:03] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:53:03] Looking for 'sed' - found /usr/bin/sed\\n[10:53:03] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:53:03] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:53:03] Using genetic code table 11.\\n[10:53:03] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACJTU01.fasta\\n[10:53:03] Wrote 159 contigs totalling 9270090 bp.\\n[10:53:03] Predicting tRNAs and tmRNAs\\n[10:53:03] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTU01\\\\/\\\\/Nostoc_JACJTU01\\\\.fna\\n[10:53:07] 1 tRNA-Lys [229266,229340] 33 (ctt)\\n[10:53:07] 2 tRNA-Ser [230734,230808] 35 (tga)\\n[10:53:07] 3 tRNA-Arg c[320681,320754] 35 (acg)\\n[10:53:07] 1 tRNA-Val c[23950,24022] 34 (tac)\\n[10:53:07] 1 tRNA-Gln c[94010,94083] 34 (ttg)\\n[10:53:07] 2 tRNA-Leu c[98753,98837] 35 (caa)\\n[10:53:07] 3 tRNA-Glu c[230574,230648] 36 (ttc)\\n[10:53:07] 4 tRNA-Lys c[265659,265731] 33 (ctt)\\n[10:53:07] 1 tRNA-His [57067,57140] 34 (gtg)\\n[10:53:07] 2 tRNA-Thr c[230504,230575] 33 (tgt)\\n[10:53:07] 1 tRNA-Stop c[8322,8411] 31 (tta)\\n[10:53:07] 2 tRNA-Met [131741,131817] 35 (cat)\\n[10:53:07] 1 tRNA-Pro [25219,25292] 35 (ggg)\\n[10:53:07] 1 tRNA-Asp c[107796,107869] 35 (gtc)\\n[10:53:07] 1 tRNA-Leu [72568,72648] 35 (gag)\\n[10:53:07] 2 tRNA-Ser c[219990,220083] 36 (gct)\\n[10:53:07] 3 tRNA-Lys c[225804,225874] 33 (ttt)\\n[10:53:07] 4 tRNA-His c[226137,226212] 35 (gtg)\\n[10:53:07] 5 tRNA-Ala c[226218,226292] 35 (tgc)\\n[10:53:07] 6 tRNA-Arg c[226322,226397] 35 (tct)\\n[10:53:07] 7 tRNA-Val c[226404,226479] 34 (tac)\\n[10:53:07] 8 tRNA-Gly c[226565,226636] 33 (tcc)\\n[10:53:07] 9 tRNA-Thr c[226641,226713] 34 (tgt)\\n[10:53:07] 10 tRNA-Met c[226722,226798] 34 (cat)\\n[10:53:07] 11 tRNA-Leu c[226809,226881] 34 (tag)\\n[10:53:07] 12 tRNA-Pro c[226885,226958] 34 (tgg)\\n[10:53:07] 13 tRNA-Leu c[227045,227120] 35 (taa)\\n[10:53:07] 14 tRNA-Gln c[227129,227201] 34 (ttg)\\n[10:53:07] 15 tRNA-Gln c[227205,227277] 34 (ctg)\\n[10:53:07] 16 tRNA-Asn c[227428,227502] 33 (gtt)\\n[10:53:07] 17 tRNA-Phe c[227508,227583] 36 (gaa)\\n[10:53:07] 18 tRNA-Arg c[227664,227740] 36 (acg)\\n[10:53:07] 19 tRNA-Ser c[227742,227833] 35 (gct)\\n[10:53:07] 20 tRNA-Glu c[227945,228020] 35 (ttc)\\n[10:53:07] tRNA c[228076,228150] is a pseudo/wacky gene - skipping.\\n[10:53:07] 22 tRNA-Met c[228152,228226] 33 (cat)\\n[10:53:07] 23 tRNA-Cys c[228234,228306] 35 (gca)\\n[10:53:07] 24 tRNA-Glu c[228310,228386] 35 (ctc)\\n[10:53:07] 1 tRNA-Ala [145718,145790] 34 (ggc)\\n[10:53:07] 1 tRNA-Met [14949,15024] 36 (cat)\\n[10:53:07] 1 tRNA-Phe c[74185,74257] 34 (gaa)\\n[10:53:07] 2 tRNA-Pro c[118215,118288] 35 (tgg)\\n[10:53:07] 1 tRNA-Ala [82338,82410] 34 (cgc)\\n[10:53:07] 2 tRNA-Trp c[97895,97968] 34 (cca)\\n[10:53:07] 3 tRNA-Thr c[145974,146045] 33 (cgt)\\n[10:53:07] 1 tRNA-Ala [20126,20198] 34 (tgc)\\n[10:53:07] 1 tmRNA [131681,132068] 101,151 ANNIVKFARREALVAA*\\n[10:53:07] 1 tRNA-Gly [12169,12239] 33 (tcc)\\n[10:53:07] 1 tRNA-Lys c[41722,41793] 33 (ttt)\\n[10:53:07] 1 tRNA-Leu c[37949,38032] 36 (tag)\\n[10:53:07] 1 tRNA-Gly c[61990,62061] 33 (gcc)\\n[10:53:07] 2 tRNA-Met c[69116,69189] 33 (cat)\\n[10:53:07] 1 tRNA-Arg [93005,93079] 35 (tct)\\n[10:53:07] 1 tRNA-Arg c[44035,44107] 34 (ccg)\\n[10:53:07] 1 tRNA-Asn [2430,2503] 34 (gtt)\\n[10:53:07] 1 tRNA-Val [71190,71263] 35 (gac)\\n[10:53:07] 1 tRNA-Pro [46768,46841] 35 (cgg)\\n[10:53:07] 2 tRNA-Ser [47019,47105] 36 (cga)\\n[10:53:07] 1 tRNA-Met [66868,66943] 34 (cat)\\n[10:53:07] 2 tRNA-Met [66945,67020] 34 (cat)\\n[10:53:07] 1 tRNA-Thr c[55559,55630] 33 (ggt)\\n[10:53:07] 2 tRNA-Tyr c[55711,55795] 35 (gta)\\n[10:53:07] 3 tRNA-Ser [58050,58136] 36 (tga)\\n[10:53:07] 1 tRNA-Gly c[19857,19928] 33 (ccc)\\n[10:53:07] 1 tRNA-Ser [31762,31849] 35 (gga)\\n[10:53:07] 1 tRNA-Gly [21066,21137] 33 (tcc)\\n[10:53:07] 1 tRNA-Val [37360,37433] 34 (cac)\\n[10:53:07] 1 tRNA-Leu c[32730,32811] 35 (cag)\\n[10:53:07] 1 tRNA-Cys [6952,7023] 33 (gca)\\n[10:53:07] Found 68 tRNAs\\n[10:53:07] Predicting Ribosomal RNAs\\n[10:53:07] Running Barrnap with 4 threads\\n[10:53:09] 1 JACJTU010000158.1 1 16S ribosomal RNA\\n[10:53:09] 2 JACJTU010000159.1 230 23S ribosomal RNA\\n[10:53:09] Found 2 rRNAs\\n[10:53:09] Skipping ncRNA search, enable with --rfam if desired.\\n[10:53:09] Total of 69 tRNA + rRNA features\\n[10:53:09] Searching for CRISPR repeats\\n[10:53:10] CRISPR1 JACJTU010000001.1 190224 with 10 spacers\\n[10:53:10] CRISPR2 JACJTU010000007.1 257121 with 11 spacers\\n[10:53:10] CRISPR3 JACJTU010000008.1 211348 with 21 spacers\\n[10:53:10] CRISPR4 JACJTU010000016.1 9747 with 14 spacers\\n[10:53:10] CRISPR5 JACJTU010000021.1 138259 with 20 spacers\\n[10:53:10] CRISPR6 JACJTU010000024.1 18184 with 15 spacers\\n[10:53:10] CRISPR7 JACJTU010000029.1 83220 with 19 spacers\\n[10:53:10] CRISPR8 JACJTU010000031.1 63324 with 25 spacers\\n[10:53:10] CRISPR9 JACJTU010000031.1 79603 with 40 spacers\\n[10:53:10] CRISPR10 JACJTU010000052.1 42724 with 6 spacers\\n[10:53:10] CRISPR11 JACJTU010000086.1 13986 with 21 spacers\\n[10:53:10] Found 11 CRISPRs\\n[10:53:10] Predicting coding sequences\\n[10:53:10] Contigs total 9270090 bp, so using single mode\\n[10:53:10] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTU01\\\\/\\\\/Nostoc_JACJTU01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:53:18] Excluding CDS which overlaps existing RNA (tRNA) at JACJTU010000004.1:93992..94177 on + strand\\n[10:53:18] Excluding CDS which overlaps existing RNA (tRNA) at JACJTU010000004.1:98728..98976 on + strand\\n[10:53:18] Excluding CDS which overlaps existing RNA (tRNA) at JACJTU010000006.1:7653..8960 on - strand\\n[10:53:19] Excluding CDS which overlaps existing RNA (repeat_region) at JACJTU010000007.1:257230..257703 on + strand\\n[10:53:20] Excluding CDS which overlaps existing RNA (tRNA) at JACJTU010000022.1:37921..38178 on - strand\\n[10:53:21] Excluding CDS which overlaps existing RNA (repeat_region) at JACJTU010000031.1:63189..63464 on + strand\\n[10:53:22] Excluding CDS which overlaps existing RNA (repeat_region) at JACJTU010000052.1:43083..43301 on - strand\\n[10:53:22] Found 7803 CDS\\n[10:53:22] Connecting features back to sequences\\n[10:53:22] Not using genus-specific database. Try --usegenus to enable it.\\n[10:53:22] Annotating CDS, please be patient.\\n[10:53:22] Will use 4 CPUs for similarity searching.\\n[10:53:24] There are still 7803 unannotated CDS left (started with 7803)\\n[10:53:24] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:53:24] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTU01\\\\/\\\\/Nostoc_JACJTU01\\\\.IS\\\\.tmp\\\\.6816\\\\.faa | parallel --gnu --plain -j 4 --block 315376 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTU01\\\\/\\\\/Nostoc_JACJTU01\\\\.IS\\\\.tmp\\\\.6816\\\\.blast 2> /dev/null\\n[10:53:46] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTU01\\\\/\\\\/Nostoc_JACJTU01\\\\.IS\\\\.tmp\\\\.6816\\\\.faa | parallel --gnu --plain -j 4 --block 315376 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACJTU01\\\\/\\\\/Nostoc_JACJTU01\\\\.IS\\\\.tmp\\\\.6816\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:53:46] This is prokka 1.14.6\\n[10:53:46] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:53:46] Homepage is https://github.com/tseemann/prokka\\n[10:53:46] Local time is Mon Feb 21 10:53:46 2022\\n[10:53:46] You are tiagoferreiraleao\\n[10:53:46] Operating system is darwin\\n[10:53:46] You have BioPerl 1.007002\\n[10:53:47] System has 4 cores.\\n[10:53:47] Option --cpu asked for 12 cores, but system only has 4\\n[10:53:47] Will use maximum of 4 cores.\\n[10:53:47] Annotating as >>> Bacteria <<<\\n[10:53:47] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JAIVFR01.fasta' contents.\\n[10:53:47] Setting --locustag EFJIOHFJ from MD5 ef3281f342eb2356f7816afa4c746649\\n[10:53:47] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JAIVFR01/\\n[10:53:47] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAIVFR01\\\\/\\n[10:53:47] Using filename prefix: Nostoc_JAIVFR01.XXX\\n[10:53:47] Setting HMMER_NCPU=1\\n[10:53:47] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JAIVFR01//Nostoc_JAIVFR01.log\\n[10:53:47] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JAIVFR01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JAIVFR01/ --prefix Nostoc_JAIVFR01 --cpus 12\\n[10:53:47] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:53:47] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:53:47] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:53:47] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:53:47] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:53:47] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:53:47] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:53:47] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:53:47] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:53:47] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:53:47] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:53:47] Looking for 'egrep' - found /usr/bin/egrep\\n[10:53:47] Looking for 'find' - found /usr/bin/find\\n[10:53:47] Looking for 'grep' - found /usr/bin/grep\\n[10:53:47] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:53:47] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:53:47] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:53:47] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:53:47] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:53:47] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:53:47] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:53:47] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:53:47] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:53:47] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:53:47] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:53:47] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:53:47] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:53:47] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:53:47] Looking for 'sed' - found /usr/bin/sed\\n[10:53:47] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:53:48] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:53:48] Using genetic code table 11.\\n[10:53:48] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JAIVFR01.fasta\\n[10:53:48] Wrote 1200 contigs totalling 7052049 bp.\\n[10:53:48] Predicting tRNAs and tmRNAs\\n[10:53:48] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAIVFR01\\\\/\\\\/Nostoc_JAIVFR01\\\\.fna\\n[10:53:49] 1 tRNA-Ser c[347,433] 36 (gga)\\n[10:53:49] 1 tRNA-Ala [8129,8203] 35 (cgc)\\n[10:53:49] 2 tRNA-Trp c[14247,14320] 34 (cca)\\n[10:53:49] 1 tRNA-Pyl [3771,3843] 34 (cta)\\n[10:53:49] 2 tRNA-Pyl [4550,4624] 35 (cta)\\n[10:53:49] 1 tRNA-Leu c[6008,6089] 35 (cag)\\n[10:53:49] 1 tRNA-Ser c[5291,5384] 36 (gct)\\n[10:53:49] 2 tRNA-Lys c[11113,11184] 33 (ttt)\\n[10:53:49] 3 tRNA-His c[11450,11524] 35 (gtg)\\n[10:53:49] 4 tRNA-Ala c[11530,11604] 35 (tgc)\\n[10:53:49] 5 tRNA-Arg c[11696,11771] 35 (tct)\\n[10:53:49] 6 tRNA-Val c[11778,11852] 33 (tac)\\n[10:53:49] 7 tRNA-Gly c[11938,12009] 33 (tcc)\\n[10:53:49] 8 tRNA-Thr c[12014,12086] 34 (tgt)\\n[10:53:49] 9 tRNA-Met c[12095,12171] 34 (cat)\\n[10:53:49] 10 tRNA-Pro c[12284,12357] 34 (tgg)\\n[10:53:49] 11 tRNA-Leu c[12515,12592] 36 (taa)\\n[10:53:49] 12 tRNA-Leu c[12598,12674] 35 (caa)\\n[10:53:49] 13 tRNA-Gln c[12681,12756] 34 (ttg)\\n[10:53:49] 14 tRNA-Gln c[12759,12833] 35 (ctg)\\n[10:53:49] 15 tRNA-Asn c[12838,12912] 33 (gtt)\\n[10:53:49] 16 tRNA-Phe c[12918,12993] 36 (gaa)\\n[10:53:49] 17 tRNA-Arg c[13014,13090] 36 (acg)\\n[10:53:49] 18 tRNA-Ser c[13093,13180] 35 (gct)\\n[10:53:49] 19 tRNA-Tyr c[13184,13268] 38 (gta)\\n[10:53:49] 20 tRNA-Glu c[13274,13347] 35 (ttc)\\n[10:53:49] tRNA c[13414,13488] is a pseudo/wacky gene - skipping.\\n[10:53:49] 22 tRNA-Met c[13491,13566] 34 (cat)\\n[10:53:49] 23 tRNA-Cys c[13577,13650] 35 (gca)\\n[10:53:49] 24 tRNA-Asp c[13722,13797] 36 (gtc)\\n[10:53:49] 1 tRNA-Arg [6917,6992] 36 (cct)\\n[10:53:49] 1 tRNA-Cys c[10694,10764] 33 (gca)\\n[10:53:49] 1 tRNA-Met c[5182,5258] 35 (cat)\\n[10:53:49] 1 tRNA-Asn c[1372,1443] 33 (gtt)\\n[10:53:49] 1 tRNA-Gln c[6050,6123] 34 (ttg)\\n[10:53:49] 1 tRNA-Arg [2365,2441] 35 (cct)\\n[10:53:49] 1 tRNA-Leu c[789,872] 35 (gag)\\n[10:53:49] 1 tRNA-Gly [9679,9749] 33 (tcc)\\n[10:53:49] 1 tRNA-Pro c[2789,2862] 35 (tgg)\\n[10:53:49] 1 tRNA-Pro [6880,6953] 35 (ggg)\\n[10:53:49] 1 tRNA-Leu c[6701,6784] 36 (tag)\\n[10:53:49] 1 tRNA-Met [5991,6065] 35 (cat)\\n[10:53:49] 1 tRNA-Leu [5998,6082] 36 (gag)\\n[10:53:49] 1 tRNA-Val [2678,2750] 34 (tac)\\n[10:53:49] 1 tRNA-Phe [4459,4531] 34 (gaa)\\n[10:53:49] 1 tRNA-Thr c[1885,1956] 33 (cgt)\\n[10:53:49] 1 tRNA-Arg [1254,1326] 34 (ccg)\\n[10:53:49] 1 tRNA-Asp [5071,5144] 35 (gtc)\\n[10:53:50] 1 tRNA-Ala c[1270,1344] 34 (tgc)\\n[10:53:50] 2 tRNA-Ile c[1428,1503] 35 (gat)\\n[10:53:50] 1 tRNA-Thr [5459,5530] 33 (tgt)\\n[10:53:50] 1 tRNA-Gly [1048,1119] 33 (gcc)\\n[10:53:50] 1 tRNA-Ile c[1564,1643] 37 (aat)\\n[10:53:50] 2 tRNA-Val c[1606,1685] 37 (aac)\\n[10:53:50] 1 tRNA-Arg c[4516,4589] 35 (acg)\\n[10:53:50] 1 tRNA-Ser c[683,769] 36 (tga)\\n[10:53:50] 2 tRNA-Tyr [3069,3154] 35 (gta)\\n[10:53:50] 3 tRNA-Thr [3242,3313] 33 (ggt)\\n[10:53:50] 1 tRNA-Lys c[2803,2875] 33 (ctt)\\n[10:53:50] 1 tRNA-Ala c[2063,2135] 34 (ggc)\\n[10:53:50] 1 tRNA-Gly c[680,751] 33 (ccc)\\n[10:53:50] 1 tRNA-Ser c[3535,3621] 36 (cga)\\n[10:53:50] 1 tRNA-Met c[914,987] 33 (cat)\\n[10:53:50] 1 tRNA-Ala c[677,752] 34 (tgc)\\n[10:53:50] 2 tRNA-Ile c[819,895] 35 (gat)\\n[10:53:50] 1 tRNA-Pro [2093,2166] 35 (cgg)\\n[10:53:50] 1 tRNA-Ile [1879,1955] 35 (gat)\\n[10:53:50] 2 tRNA-Ala [1970,2044] 35 (tgc)\\n[10:53:50] 1 tRNA-Ile [1909,1985] 35 (gat)\\n[10:53:50] 2 tRNA-Ala [2002,2077] 34 (tgc)\\n[10:53:50] 1 tRNA-Ala c[179,254] 34 (tgc)\\n[10:53:50] 2 tRNA-Ile c[345,421] 35 (gat)\\n[10:53:50] 1 tRNA-Leu [348,432] 35 (caa)\\n[10:53:50] 1 tRNA-Ser [568,658] 35 (gga)\\n[10:53:50] 1 tRNA-Ala c[197,272] 34 (tgc)\\n[10:53:50] 2 tRNA-Ile c[302,378] 35 (gat)\\n[10:53:50] 1 tRNA-Arg c[343,417] 35 (tct)\\n[10:53:50] 1 tRNA-Thr c[1908,1979] 33 (ggt)\\n[10:53:50] 1 tRNA-Ile [1907,1983] 35 (gat)\\n[10:53:50] 2 tRNA-Ala [2010,2085] 34 (tgc)\\n[10:53:50] Found 79 tRNAs\\n[10:53:50] Predicting Ribosomal RNAs\\n[10:53:50] Running Barrnap with 4 threads\\n[10:53:52] 1 JAIVFR010000429.1 1 23S ribosomal RNA (partial)\\n[10:53:52] 2 JAIVFR010000429.1 1637 16S ribosomal RNA\\n[10:53:52] 3 JAIVFR010000975.1 1063 16S ribosomal RNA\\n[10:53:52] 4 JAIVFR010001026.1 227 16S ribosomal RNA\\n[10:53:52] 5 JAIVFR010001066.1 283 16S ribosomal RNA\\n[10:53:52] 6 JAIVFR010001076.1 657 16S ribosomal RNA\\n[10:53:52] 7 JAIVFR010001104.1 584 16S ribosomal RNA\\n[10:53:52] 8 JAIVFR010001142.1 234 16S ribosomal RNA\\n[10:53:52] Found 8 rRNAs\\n[10:53:52] Skipping ncRNA search, enable with --rfam if desired.\\n[10:53:52] Total of 87 tRNA + rRNA features\\n[10:53:52] Searching for CRISPR repeats\\n[10:53:52] CRISPR1 JAIVFR010000142.1 4224 with 4 spacers\\n[10:53:52] CRISPR2 JAIVFR010000232.1 3386 with 6 spacers\\n[10:53:52] CRISPR3 JAIVFR010000472.1 335 with 9 spacers\\n[10:53:52] Found 3 CRISPRs\\n[10:53:52] Predicting coding sequences\\n[10:53:52] Contigs total 7052049 bp, so using single mode\\n[10:53:52] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAIVFR01\\\\/\\\\/Nostoc_JAIVFR01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:53:58] Excluding CDS which overlaps existing RNA (tRNA) at JAIVFR010000032.1:12458..12619 on + strand\\n[10:53:58] Excluding CDS which overlaps existing RNA (tRNA) at JAIVFR010000032.1:12665..12898 on - strand\\n[10:53:58] Excluding CDS which overlaps existing RNA (tRNA) at JAIVFR010000133.1:2414..2776 on + strand\\n[10:53:59] Excluding CDS which overlaps existing RNA (tRNA) at JAIVFR010000229.1:6644..6826 on + strand\\n[10:53:59] Excluding CDS which overlaps existing RNA (repeat_region) at JAIVFR010000232.1:2758..3399 on + strand\\n[10:53:59] Excluding CDS which overlaps existing RNA (rRNA) at JAIVFR010000429.1:1679..1993 on - strand\\n[10:53:59] Excluding CDS which overlaps existing RNA (rRNA) at JAIVFR010000429.1:2678..2911 on + strand\\n[10:53:59] Excluding CDS which overlaps existing RNA (repeat_region) at JAIVFR010000472.1:725..970 on + strand\\n[10:53:59] Excluding CDS which overlaps existing RNA (tRNA) at JAIVFR010000508.1:168..2231 on + strand\\n[10:54:00] Excluding CDS which overlaps existing RNA (tRNA) at JAIVFR010000952.1:953..1129 on + strand\\n[10:54:00] Excluding CDS which overlaps existing RNA (tRNA) at JAIVFR010000975.1:587..976 on + strand\\n[10:54:00] Excluding CDS which overlaps existing RNA (rRNA) at JAIVFR010000975.1:1974..2102 on - strand\\n[10:54:00] Excluding CDS which overlaps existing RNA (rRNA) at JAIVFR010001066.1:994..1119 on + strand\\n[10:54:00] Excluding CDS which overlaps existing RNA (rRNA) at JAIVFR010001066.1:1091..1315 on + strand\\n[10:54:00] Excluding CDS which overlaps existing RNA (tRNA) at JAIVFR010001102.1:477..689 on + strand\\n[10:54:00] Excluding CDS which overlaps existing RNA (rRNA) at JAIVFR010001104.1:1518..1646 on - strand\\n[10:54:00] Excluding CDS which overlaps existing RNA (tRNA) at JAIVFR010001134.1:1916..2200 on + strand\\n[10:54:00] Found 6773 CDS\\n[10:54:00] Connecting features back to sequences\\n[10:54:00] Not using genus-specific database. Try --usegenus to enable it.\\n[10:54:00] Annotating CDS, please be patient.\\n[10:54:00] Will use 4 CPUs for similarity searching.\\n[10:54:02] There are still 6773 unannotated CDS left (started with 6773)\\n[10:54:02] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:54:02] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAIVFR01\\\\/\\\\/Nostoc_JAIVFR01\\\\.IS\\\\.tmp\\\\.6928\\\\.faa | parallel --gnu --plain -j 4 --block 210215 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAIVFR01\\\\/\\\\/Nostoc_JAIVFR01\\\\.IS\\\\.tmp\\\\.6928\\\\.blast 2> /dev/null\\n[10:54:07] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAIVFR01\\\\/\\\\/Nostoc_JAIVFR01\\\\.IS\\\\.tmp\\\\.6928\\\\.faa | parallel --gnu --plain -j 4 --block 210215 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JAIVFR01\\\\/\\\\/Nostoc_JAIVFR01\\\\.IS\\\\.tmp\\\\.6928\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:54:07] This is prokka 1.14.6\\n[10:54:07] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:54:07] Homepage is https://github.com/tseemann/prokka\\n[10:54:07] Local time is Mon Feb 21 10:54:07 2022\\n[10:54:07] You are tiagoferreiraleao\\n[10:54:07] Operating system is darwin\\n[10:54:07] You have BioPerl 1.007002\\n[10:54:08] System has 4 cores.\\n[10:54:08] Option --cpu asked for 12 cores, but system only has 4\\n[10:54:08] Will use maximum of 4 cores.\\n[10:54:08] Annotating as >>> Bacteria <<<\\n[10:54:08] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Westiellopsis_JAHHHK01.fasta' contents.\\n[10:54:08] Setting --locustag MILODFIA from MD5 6258df2a59bff479935a62c9b196fc52\\n[10:54:08] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Westiellopsis_JAHHHK01/\\n[10:54:08] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Westiellopsis_JAHHHK01\\\\/\\n[10:54:08] Using filename prefix: Westiellopsis_JAHHHK01.XXX\\n[10:54:08] Setting HMMER_NCPU=1\\n[10:54:08] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Westiellopsis_JAHHHK01//Westiellopsis_JAHHHK01.log\\n[10:54:08] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Westiellopsis_JAHHHK01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Westiellopsis_JAHHHK01/ --prefix Westiellopsis_JAHHHK01 --cpus 12\\n[10:54:08] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:54:08] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:54:08] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:54:08] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:54:08] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:54:08] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:54:08] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:54:08] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:54:08] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:54:08] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:54:08] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:54:08] Looking for 'egrep' - found /usr/bin/egrep\\n[10:54:08] Looking for 'find' - found /usr/bin/find\\n[10:54:08] Looking for 'grep' - found /usr/bin/grep\\n[10:54:08] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:54:08] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:54:08] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:54:08] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:54:08] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:54:08] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:54:08] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:54:08] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:54:08] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:54:08] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:54:09] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:54:09] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:54:09] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:54:09] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:54:09] Looking for 'sed' - found /usr/bin/sed\\n[10:54:09] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:54:09] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:54:09] Using genetic code table 11.\\n[10:54:09] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Westiellopsis_JAHHHK01.fasta\\n[10:54:09] Wrote 80 contigs totalling 7783819 bp.\\n[10:54:09] Predicting tRNAs and tmRNAs\\n[10:54:09] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Westiellopsis_JAHHHK01\\\\/\\\\/Westiellopsis_JAHHHK01\\\\.fna\\n[10:54:12] 1 tRNA-Asn [196049,196120] 33 (gtt)\\n[10:54:12] 1 tRNA-Asp c[319063,319136] 35 (gtc)\\n[10:54:12] 1 tRNA-His [50906,50979] 34 (gtg)\\n[10:54:12] 2 tRNA-Thr [300147,300218] 33 (tgt)\\n[10:54:12] 1 tRNA-Leu [25078,25160] 36 (gag)\\n[10:54:12] 1 tRNA-Met c[13550,13626] 35 (cat)\\n[10:54:12] 2 tRNA-Pro c[295485,295558] 35 (tgg)\\n[10:54:12] 1 tRNA-Cys c[197097,197168] 33 (gca)\\n[10:54:12] 1 tRNA-Pro c[85871,85944] 35 (ggg)\\n[10:54:12] 2 tRNA-Met c[99886,99961] 36 (cat)\\n[10:54:12] 3 tRNA-Val [117384,117456] 34 (tac)\\n[10:54:12] 1 tRNA-Leu [50457,50540] 35 (caa)\\n[10:54:12] 2 tRNA-Gln [69543,69616] 34 (ttg)\\n[10:54:12] 3 tRNA-Gly c[174768,174839] 33 (ccc)\\n[10:54:12] 4 tRNA-Val c[187522,187595] 34 (cac)\\n[10:54:12] 5 tRNA-Ala [192742,192815] 34 (tgc)\\n[10:54:12] 1 tRNA-Met [142312,142385] 33 (cat)\\n[10:54:12] 1 tRNA-Thr [39088,39159] 33 (cgt)\\n[10:54:12] 1 tRNA-Arg [11163,11235] 34 (ccg)\\n[10:54:12] 2 tRNA-Glu c[114642,114715] 35 (ttc)\\n[10:54:12] 3 tRNA-Leu [168781,168863] 35 (tag)\\n[10:54:12] 1 tRNA-Trp [99838,99911] 34 (cca)\\n[10:54:12] 2 tRNA-Ala c[110779,110851] 34 (cgc)\\n[10:54:12] 3 tRNA-Ser [181780,181868] 36 (gga)\\n[10:54:12] 1 tRNA-Val [122752,122825] 35 (gac)\\n[10:54:12] 2 tRNA-Phe c[160979,161051] 34 (gaa)\\n[10:54:12] 1 tRNA-Thr [88392,88486] 35 (ggt)\\n[10:54:12] 2 tRNA-Lys [91464,91536] 33 (ctt)\\n[10:54:12] 1 tRNA-Gly [119936,120006] 33 (tcc)\\n[10:54:12] 2 tRNA-Ser [123838,123924] 36 (tga)\\n[10:54:12] 1 tRNA-Phe [3037,3116] 38 (gaa)\\n[10:54:12] 2 tRNA-Pro c[43483,43556] 35 (cgg)\\n[10:54:12] 1 tRNA-Ser [39168,39261] 36 (gct)\\n[10:54:12] 2 tRNA-Arg c[46481,46554] 35 (acg)\\n[10:54:12] 1 tmRNA [49698,50085] 101,151 ANNIVKFARKDALVAA*\\n[10:54:12] 1 tRNA-Lys c[4010,4081] 33 (ttt)\\n[10:54:12] 2 tRNA-Ala c[24235,24307] 34 (ggc)\\n[10:54:12] 1 tRNA-Glu [34885,34966] 32 (ttc)\\n[10:54:12] 2 tRNA-Glu [36394,36475] 32 (ttc)\\n[10:54:12] 1 tRNA-Arg [22327,22401] 35 (tct)\\n[10:54:12] 2 tRNA-Thr c[22512,22583] 33 (ggt)\\n[10:54:12] 3 tRNA-Tyr c[22661,22745] 35 (gta)\\n[10:54:12] 4 tRNA-Arg c[48273,48348] 36 (cct)\\n[10:54:12] 1 tRNA-Ser c[46311,46397] 36 (cga)\\n[10:54:12] 1 tRNA-Gly [26613,26684] 33 (gcc)\\n[10:54:12] Found 45 tRNAs\\n[10:54:12] Predicting Ribosomal RNAs\\n[10:54:12] Running Barrnap with 4 threads\\n[10:54:14] Found 0 rRNAs\\n[10:54:14] Skipping ncRNA search, enable with --rfam if desired.\\n[10:54:14] Total of 44 tRNA + rRNA features\\n[10:54:14] Searching for CRISPR repeats\\n[10:54:14] CRISPR1 JAHHHK010000001.1 103569 with 27 spacers\\n[10:54:15] CRISPR2 JAHHHK010000020.1 39810 with 25 spacers\\n[10:54:15] CRISPR3 JAHHHK010000027.1 77176 with 22 spacers\\n[10:54:15] Found 3 CRISPRs\\n[10:54:15] Predicting coding sequences\\n[10:54:15] Contigs total 7783819 bp, so using single mode\\n[10:54:15] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Westiellopsis_JAHHHK01\\\\/\\\\/Westiellopsis_JAHHHK01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:54:21] Excluding CDS which overlaps existing RNA (repeat_region) at JAHHHK010000001.1:104707..104937 on + strand\\n[10:54:22] Excluding CDS which overlaps existing RNA (tRNA) at JAHHHK010000009.1:117342..117512 on + strand\\n[10:54:22] Excluding CDS which overlaps existing RNA (tRNA) at JAHHHK010000010.1:187133..187555 on - strand\\n[10:54:23] Excluding CDS which overlaps existing RNA (tRNA) at JAHHHK010000014.1:99814..99939 on + strand\\n[10:54:23] Excluding CDS which overlaps existing RNA (tRNA) at JAHHHK010000016.1:88180..88749 on + strand\\n[10:54:24] Excluding CDS which overlaps existing RNA (tRNA) at JAHHHK010000025.1:2939..6580 on + strand\\n[10:54:24] Excluding CDS which overlaps existing RNA (tRNA) at JAHHHK010000028.1:39152..39283 on - strand\\n[10:54:25] Found 6885 CDS\\n[10:54:25] Connecting features back to sequences\\n[10:54:25] Not using genus-specific database. Try --usegenus to enable it.\\n[10:54:25] Annotating CDS, please be patient.\\n[10:54:25] Will use 4 CPUs for similarity searching.\\n[10:54:27] There are still 6885 unannotated CDS left (started with 6885)\\n[10:54:27] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:54:27] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Westiellopsis_JAHHHK01\\\\/\\\\/Westiellopsis_JAHHHK01\\\\.IS\\\\.tmp\\\\.7040\\\\.faa | parallel --gnu --plain -j 4 --block 265346 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Westiellopsis_JAHHHK01\\\\/\\\\/Westiellopsis_JAHHHK01\\\\.IS\\\\.tmp\\\\.7040\\\\.blast 2> /dev/null\\n[10:54:45] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Westiellopsis_JAHHHK01\\\\/\\\\/Westiellopsis_JAHHHK01\\\\.IS\\\\.tmp\\\\.7040\\\\.faa | parallel --gnu --plain -j 4 --block 265346 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Westiellopsis_JAHHHK01\\\\/\\\\/Westiellopsis_JAHHHK01\\\\.IS\\\\.tmp\\\\.7040\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:54:46] This is prokka 1.14.6\\n[10:54:46] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:54:46] Homepage is https://github.com/tseemann/prokka\\n[10:54:46] Local time is Mon Feb 21 10:54:46 2022\\n[10:54:46] You are tiagoferreiraleao\\n[10:54:46] Operating system is darwin\\n[10:54:46] You have BioPerl 1.007002\\n[10:54:46] System has 4 cores.\\n[10:54:46] Option --cpu asked for 12 cores, but system only has 4\\n[10:54:46] Will use maximum of 4 cores.\\n[10:54:46] Annotating as >>> Bacteria <<<\\n[10:54:46] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZU01.fasta' contents.\\n[10:54:46] Setting --locustag AIHLLJPK from MD5 a21553941bd0e10b56c32444c7ba4ba0\\n[10:54:46] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZU01/\\n[10:54:46] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZU01\\\\/\\n[10:54:46] Using filename prefix: Fischerella_NGZU01.XXX\\n[10:54:46] Setting HMMER_NCPU=1\\n[10:54:46] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZU01//Fischerella_NGZU01.log\\n[10:54:46] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZU01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NGZU01/ --prefix Fischerella_NGZU01 --cpus 12\\n[10:54:46] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:54:46] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:54:46] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:54:46] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:54:46] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:54:46] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:54:46] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:54:46] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:54:46] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:54:46] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:54:46] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:54:46] Looking for 'egrep' - found /usr/bin/egrep\\n[10:54:46] Looking for 'find' - found /usr/bin/find\\n[10:54:46] Looking for 'grep' - found /usr/bin/grep\\n[10:54:46] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:54:46] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:54:46] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:54:46] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:54:46] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:54:46] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:54:46] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:54:46] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:54:46] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:54:46] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:54:47] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:54:47] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:54:47] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:54:47] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:54:47] Looking for 'sed' - found /usr/bin/sed\\n[10:54:47] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:54:47] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:54:47] Using genetic code table 11.\\n[10:54:47] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NGZU01.fasta\\n[10:54:47] Wrote 76 contigs totalling 5332626 bp.\\n[10:54:47] Predicting tRNAs and tmRNAs\\n[10:54:47] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZU01\\\\/\\\\/Fischerella_NGZU01\\\\.fna\\n[10:54:49] 1 tRNA-Thr c[1545,1616] 33 (cgt)\\n[10:54:49] 1 tRNA-Met c[45383,45455] 34 (cat)\\n[10:54:49] 1 tRNA-Asn c[3587,3660] 34 (att)\\n[10:54:49] 2 tRNA-Arg c[4331,4403] 34 (tcg)\\n[10:54:49] 1 tRNA-Thr [2112,2183] 33 (tgt)\\n[10:54:49] 1 tRNA-Met c[26420,26495] 36 (cat)\\n[10:54:49] 1 tRNA-Pro [59308,59381] 35 (cgg)\\n[10:54:49] 1 tRNA-Arg [33890,33965] 36 (cct)\\n[10:54:49] 1 tRNA-Gly [12161,12231] 33 (tcc)\\n[10:54:49] 1 tRNA-Pro c[13930,14003] 35 (ggg)\\n[10:54:49] 1 tRNA-Glu c[11139,11211] 35 (ttc)\\n[10:54:49] 1 tRNA-Gly c[82806,82877] 33 (gcc)\\n[10:54:49] 1 tRNA-Met c[11635,11711] 35 (cat)\\n[10:54:49] 1 tRNA-Lys c[91227,91300] 34 (ttt)\\n[10:54:49] 1 tRNA-Ser c[69942,70035] 36 (gct)\\n[10:54:49] 1 tRNA-Trp c[138532,138605] 34 (cca)\\n[10:54:49] 1 tRNA-Ser [30537,30623] 36 (tga)\\n[10:54:49] 1 tRNA-Asn c[82077,82150] 34 (gtt)\\n[10:54:49] 1 tRNA-Ser c[44223,44309] 36 (cga)\\n[10:54:49] 2 tRNA-Ile [50848,50923] 35 (gat)\\n[10:54:49] 3 tRNA-Ala [51006,51080] 34 (tgc)\\n[10:54:49] 1 tRNA-Cys c[90,161] 33 (gca)\\n[10:54:49] 1 tRNA-Arg c[27977,28050] 35 (ccg)\\n[10:54:49] 1 tRNA-Ala [271681,271755] 35 (cgc)\\n[10:54:49] 2 tRNA-Leu [296525,296607] 35 (cag)\\n[10:54:49] 1 tRNA-Gln [126444,126514] 32 (ttg)\\n[10:54:49] 2 tRNA-Val c[187204,187277] 35 (gac)\\n[10:54:49] 1 tRNA-Leu [27704,27786] 36 (gag)\\n[10:54:49] 1 tRNA-Gly [14361,14432] 33 (ccc)\\n[10:54:49] 2 tRNA-Thr c[32227,32298] 33 (ggt)\\n[10:54:49] 3 tRNA-Tyr c[32362,32446] 35 (gta)\\n[10:54:49] 1 tRNA-Ser [83898,83986] 36 (gga)\\n[10:54:49] 1 tRNA-Arg c[115472,115546] 35 (tct)\\n[10:54:49] 2 tRNA-His [158746,158819] 34 (gtg)\\n[10:54:49] 1 tRNA-Arg [1856,1929] 35 (acg)\\n[10:54:49] 2 tRNA-Pro c[245457,245531] 35 (tgg)\\n[10:54:49] 1 tmRNA c[243,629] 100,150 ANNIVKFARKEALVAA*\\n[10:54:49] 2 tRNA-Gln c[65520,65591] 33 (ttg)\\n[10:54:49] 3 tRNA-Leu c[101763,101847] 35 (caa)\\n[10:54:49] 4 tRNA-Ala c[235589,235662] 34 (ggc)\\n[10:54:49] 1 tRNA-Val c[2781,2853] 34 (tac)\\n[10:54:49] 2 tRNA-Phe [10252,10324] 34 (gaa)\\n[10:54:49] 1 tRNA-Asp c[59898,59971] 35 (gtc)\\n[10:54:49] 1 tRNA-Leu [44560,44643] 36 (tag)\\n[10:54:49] Found 44 tRNAs\\n[10:54:49] Predicting Ribosomal RNAs\\n[10:54:49] Running Barrnap with 4 threads\\n[10:54:50] 1 NGZU01000043.1 49107 16S ribosomal RNA\\n[10:54:50] 2 NGZU01000043.1 51378 23S ribosomal RNA\\n[10:54:50] 3 NGZU01000043.1 54317 5S ribosomal RNA\\n[10:54:50] Found 3 rRNAs\\n[10:54:50] Skipping ncRNA search, enable with --rfam if desired.\\n[10:54:50] Total of 46 tRNA + rRNA features\\n[10:54:50] Searching for CRISPR repeats\\n[10:54:50] CRISPR1 NGZU01000055.1 2911 with 69 spacers\\n[10:54:50] CRISPR2 NGZU01000058.1 2758 with 49 spacers\\n[10:54:50] CRISPR3 NGZU01000059.1 19323 with 49 spacers\\n[10:54:50] CRISPR4 NGZU01000064.1 16 with 10 spacers\\n[10:54:50] CRISPR5 NGZU01000068.1 32399 with 16 spacers\\n[10:54:50] CRISPR6 NGZU01000068.1 33715 with 23 spacers\\n[10:54:50] CRISPR7 NGZU01000068.1 36244 with 4 spacers\\n[10:54:50] CRISPR8 NGZU01000076.1 44137 with 3 spacers\\n[10:54:50] Found 8 CRISPRs\\n[10:54:50] Predicting coding sequences\\n[10:54:50] Contigs total 5332626 bp, so using single mode\\n[10:54:50] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZU01\\\\/\\\\/Fischerella_NGZU01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:54:56] Excluding CDS which overlaps existing RNA (tRNA) at NGZU01000039.1:80837..82276 on - strand\\n[10:54:56] Excluding CDS which overlaps existing RNA (rRNA) at NGZU01000043.1:51773..51952 on + strand\\n[10:54:56] Excluding CDS which overlaps existing RNA (rRNA) at NGZU01000043.1:53738..53914 on - strand\\n[10:54:56] Excluding CDS which overlaps existing RNA (repeat_region) at NGZU01000059.1:21134..21271 on - strand\\n[10:54:57] Excluding CDS which overlaps existing RNA (tRNA) at NGZU01000067.1:1301..1972 on - strand\\n[10:54:57] Found 4535 CDS\\n[10:54:57] Connecting features back to sequences\\n[10:54:57] Not using genus-specific database. Try --usegenus to enable it.\\n[10:54:57] Annotating CDS, please be patient.\\n[10:54:57] Will use 4 CPUs for similarity searching.\\n[10:54:58] There are still 4535 unannotated CDS left (started with 4535)\\n[10:54:58] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:54:58] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZU01\\\\/\\\\/Fischerella_NGZU01\\\\.IS\\\\.tmp\\\\.7153\\\\.faa | parallel --gnu --plain -j 4 --block 178446 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZU01\\\\/\\\\/Fischerella_NGZU01\\\\.IS\\\\.tmp\\\\.7153\\\\.blast 2> /dev/null\\n[10:55:17] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZU01\\\\/\\\\/Fischerella_NGZU01\\\\.IS\\\\.tmp\\\\.7153\\\\.faa | parallel --gnu --plain -j 4 --block 178446 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NGZU01\\\\/\\\\/Fischerella_NGZU01\\\\.IS\\\\.tmp\\\\.7153\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:55:17] This is prokka 1.14.6\\n[10:55:17] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:55:17] Homepage is https://github.com/tseemann/prokka\\n[10:55:17] Local time is Mon Feb 21 10:55:17 2022\\n[10:55:17] You are tiagoferreiraleao\\n[10:55:17] Operating system is darwin\\n[10:55:17] You have BioPerl 1.007002\\n[10:55:18] System has 4 cores.\\n[10:55:18] Option --cpu asked for 12 cores, but system only has 4\\n[10:55:18] Will use maximum of 4 cores.\\n[10:55:18] Annotating as >>> Bacteria <<<\\n[10:55:18] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACKZR01.fasta' contents.\\n[10:55:18] Setting --locustag FFPPMMDO from MD5 ff9966d89fe91ecf95509f42669c6ddd\\n[10:55:18] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACKZR01/\\n[10:55:18] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZR01\\\\/\\n[10:55:18] Using filename prefix: Nostoc_JACKZR01.XXX\\n[10:55:18] Setting HMMER_NCPU=1\\n[10:55:18] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACKZR01//Nostoc_JACKZR01.log\\n[10:55:18] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACKZR01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JACKZR01/ --prefix Nostoc_JACKZR01 --cpus 12\\n[10:55:18] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:55:18] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:55:18] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:55:18] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:55:18] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:55:18] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:55:18] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:55:18] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:55:18] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:55:18] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:55:18] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:55:18] Looking for 'egrep' - found /usr/bin/egrep\\n[10:55:18] Looking for 'find' - found /usr/bin/find\\n[10:55:18] Looking for 'grep' - found /usr/bin/grep\\n[10:55:18] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:55:18] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:55:18] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:55:18] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:55:18] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:55:18] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:55:18] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:55:18] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:55:18] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:55:18] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:55:18] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:55:18] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:55:18] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:55:18] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:55:18] Looking for 'sed' - found /usr/bin/sed\\n[10:55:18] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:55:19] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:55:19] Using genetic code table 11.\\n[10:55:19] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JACKZR01.fasta\\n[10:55:19] Wrote 1143 contigs totalling 8411387 bp.\\n[10:55:19] Predicting tRNAs and tmRNAs\\n[10:55:19] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZR01\\\\/\\\\/Nostoc_JACKZR01\\\\.fna\\n[10:55:20] 1 tRNA-Asp [9184,9261] 38 (gtc)\\n[10:55:20] 2 tRNA-Glu [9264,9338] 35 (ctc)\\n[10:55:20] 3 tRNA-Cys [9346,9420] 33 (gca)\\n[10:55:20] 4 tRNA-Met [9426,9500] 33 (cat)\\n[10:55:20] 5 tRNA-Trp [9502,9574] 34 (cca)\\n[10:55:20] 6 tRNA-Tyr [9809,9891] 36 (gta)\\n[10:55:20] 7 tRNA-Ser [9895,9981] 33 (gct)\\n[10:55:20] 8 tRNA-Arg [9985,10060] 35 (acg)\\n[10:55:20] 9 tRNA-Phe [10065,10139] 35 (gaa)\\n[10:55:20] 10 tRNA-Asn [10148,10222] 33 (gtt)\\n[10:55:20] 11 tRNA-Gln [10310,10384] 35 (ttg)\\n[10:55:20] 12 tRNA-Pro [10389,10464] 36 (tgg)\\n[10:55:20] 13 tRNA-Lys [10469,10543] 34 (ttt)\\n[10:55:20] 14 tRNA-Leu [10546,10620] 35 (taa)\\n[10:55:20] 15 tRNA-Thr [10630,10702] 34 (tgt)\\n[10:55:20] 16 tRNA-Gly [10708,10779] 33 (tcc)\\n[10:55:20] 17 tRNA-Val [10784,10859] 34 (tac)\\n[10:55:20] 18 tRNA-Arg [10866,10941] 34 (tct)\\n[10:55:20] 19 tRNA-Ala [10944,11019] 36 (tgc)\\n[10:55:20] 20 tRNA-His [11025,11100] 35 (gtg)\\n[10:55:20] 21 tRNA-Ser [11107,11190] 34 (tga)\\n[10:55:20] 1 tRNA-Ile [131,206] 35 (gat)\\n[10:55:20] 2 tRNA-Ala [288,362] 34 (tgc)\\n[10:55:20] 1 tRNA-Ser [8575,8661] 36 (gga)\\n[10:55:20] 1 tRNA-Leu c[5901,5985] 36 (gag)\\n[10:55:20] 1 tRNA-Met [13106,13179] 33 (cat)\\n[10:55:20] 2 tRNA-Thr c[14563,14634] 33 (tgt)\\n[10:55:20] 1 tRNA-Gln [1453,1526] 34 (ttg)\\n[10:55:20] 1 tRNA-Gly [2891,2962] 33 (ccc)\\n[10:55:20] 1 tRNA-Trp [465,538] 36 (cca)\\n[10:55:20] 1 tRNA-Asp [3352,3427] 36 (gtc)\\n[10:55:20] 2 tRNA-Glu [3431,3507] 35 (ctc)\\n[10:55:20] 3 tRNA-Cys [3511,3585] 35 (gca)\\n[10:55:20] 4 tRNA-Met [3595,3669] 33 (cat)\\n[10:55:20] 5 tRNA-Trp [3672,3746] 35 (cca)\\n[10:55:20] 1 tRNA-Gly c[4662,4732] 33 (tcc)\\n[10:55:20] 1 tRNA-Arg [937,1010] 35 (acg)\\n[10:55:20] 1 tRNA-Met c[1279,1353] 35 (cat)\\n[10:55:20] 1 tRNA-Pro [9639,9712] 35 (cgg)\\n[10:55:21] 1 tRNA-Thr c[8378,8449] 33 (cgt)\\n[10:55:21] 1 tRNA-His [537,610] 34 (gtg)\\n[10:55:21] 1 tRNA-Lys [6675,6748] 33 (ctt)\\n[10:55:21] 1 tRNA-Asn [2475,2546] 33 (gtt)\\n[10:55:21] 1 tRNA-Val c[15792,15865] 35 (gac)\\n[10:55:21] 1 tRNA-Val [7732,7804] 34 (tac)\\n[10:55:21] 1 tRNA-Phe c[5083,5155] 34 (gaa)\\n[10:55:21] 1 tRNA-Ala [1292,1364] 34 (ggc)\\n[10:55:21] 1 tRNA-Ser c[1367,1453] 36 (cga)\\n[10:55:21] 1 tRNA-Glu [9,82] 35 (ttc)\\n[10:55:21] 2 tRNA-Ser [215,306] 36 (gct)\\n[10:55:21] 3 tRNA-Arg [309,405] 36 (acg)\\n[10:55:21] 4 tRNA-Phe [424,499] 36 (gaa)\\n[10:55:21] 5 tRNA-Asn [505,579] 33 (gtt)\\n[10:55:21] 6 tRNA-Gln [585,660] 34 (ctg)\\n[10:55:21] 7 tRNA-Arg [1712,1787] 35 (tct)\\n[10:55:21] 8 tRNA-Ala [1792,1864] 34 (ggc)\\n[10:55:21] 9 tRNA-Ala [1874,1948] 35 (tgc)\\n[10:55:21] 10 tRNA-His [1954,2029] 35 (gtg)\\n[10:55:21] 11 tRNA-Lys [2397,2468] 33 (ttt)\\n[10:55:21] 12 tRNA-Ser [6666,6757] 35 (gct)\\n[10:55:21] 1 tRNA-Ser c[6562,6648] 36 (tga)\\n[10:55:21] 2 tRNA-Tyr [8948,9033] 35 (gta)\\n[10:55:21] 3 tRNA-Thr [9121,9192] 33 (ggt)\\n[10:55:21] 4 tRNA-Arg c[10699,10773] 35 (tct)\\n[10:55:21] 1 tRNA-Arg c[9818,9894] 36 (cct)\\n[10:55:21] 1 tRNA-Ala [8629,8703] 35 (tgc)\\n[10:55:21] 1 tRNA-Glu [11080,11153] 35 (ttc)\\n[10:55:21] 1 tRNA-Trp [9110,9183] 34 (cca)\\n[10:55:21] 1 tRNA-Ala c[16002,16074] 34 (cgc)\\n[10:55:21] 1 tRNA-Leu c[435,518] 36 (tag)\\n[10:55:21] 1 tRNA-Pro [16290,16363] 35 (tgg)\\n[10:55:21] 1 tRNA-Gly c[5045,5116] 33 (gcc)\\n[10:55:21] 1 tRNA-Pro c[16023,16096] 35 (ggg)\\n[10:55:21] 1 tRNA-Pyl [3064,3136] 34 (cta)\\n[10:55:21] 2 tRNA-Phe [4724,4797] 34 (gaa)\\n[10:55:21] 1 tRNA-Lys c[10495,10566] 33 (ttt)\\n[10:55:22] 1 tRNA-Leu c[351,435] 35 (caa)\\n[10:55:22] 2 tRNA-Leu [4748,4829] 35 (cag)\\n[10:55:22] 1 tmRNA c[1132,1519] 101,151 ANNIVSFARKDALVAA*\\n[10:55:22] 1 tRNA-Met c[18582,18658] 35 (cat)\\n[10:55:22] 2 tRNA-Met c[18658,18733] 34 (cat)\\n[10:55:22] 1 tRNA-Cys [8112,8182] 33 (gca)\\n[10:55:22] 1 tRNA-Arg c[3396,3468] 34 (ccg)\\n[10:55:22] 1 tRNA-Asp [3367,3440] 35 (gtc)\\n[10:55:22] Found 84 tRNAs\\n[10:55:22] Predicting Ribosomal RNAs\\n[10:55:22] Running Barrnap with 4 threads\\n[10:55:23] Found 0 rRNAs\\n[10:55:23] Skipping ncRNA search, enable with --rfam if desired.\\n[10:55:23] Total of 83 tRNA + rRNA features\\n[10:55:23] Searching for CRISPR repeats\\n[10:55:23] CRISPR1 JACKZR010000047.1 77 with 10 spacers\\n[10:55:23] CRISPR2 JACKZR010000136.1 1552 with 12 spacers\\n[10:55:24] CRISPR3 JACKZR010000563.1 351 with 7 spacers\\n[10:55:24] CRISPR4 JACKZR010000580.1 7338 with 6 spacers\\n[10:55:24] CRISPR5 JACKZR010000580.1 9234 with 10 spacers\\n[10:55:24] CRISPR6 JACKZR010000629.1 663 with 6 spacers\\n[10:55:24] CRISPR7 JACKZR010000629.1 1354 with 11 spacers\\n[10:55:24] CRISPR8 JACKZR010000667.1 24547 with 3 spacers\\n[10:55:24] CRISPR9 JACKZR010000741.1 6623 with 3 spacers\\n[10:55:24] CRISPR10 JACKZR010000741.1 9081 with 6 spacers\\n[10:55:24] CRISPR11 JACKZR010000741.1 10181 with 3 spacers\\n[10:55:24] CRISPR12 JACKZR010000834.1 3580 with 7 spacers\\n[10:55:24] CRISPR13 JACKZR010000928.1 609 with 6 spacers\\n[10:55:24] CRISPR14 JACKZR010000928.1 2492 with 10 spacers\\n[10:55:24] CRISPR15 JACKZR010000928.1 3241 with 8 spacers\\n[10:55:24] CRISPR16 JACKZR010000928.1 3840 with 3 spacers\\n[10:55:24] Found 16 CRISPRs\\n[10:55:24] Predicting coding sequences\\n[10:55:24] Contigs total 8411387 bp, so using single mode\\n[10:55:24] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZR01\\\\/\\\\/Nostoc_JACKZR01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:55:30] Excluding CDS which overlaps existing RNA (tRNA) at JACKZR010000164.1:12992..13186 on + strand\\n[10:55:31] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZR010000563.1:181..450 on + strand\\n[10:55:31] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZR010000563.1:673..870 on + strand\\n[10:55:31] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZR010000580.1:7268..7453 on + strand\\n[10:55:31] Excluding CDS which overlaps existing RNA (tRNA) at JACKZR010000615.1:15652..15846 on + strand\\n[10:55:31] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZR010000629.1:688..882 on - strand\\n[10:55:31] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZR010000629.1:1061..1222 on + strand\\n[10:55:31] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZR010000629.1:1674..1946 on + strand\\n[10:55:32] Excluding CDS which overlaps existing RNA (tRNA) at JACKZR010000672.1:1229..1369 on - strand\\n[10:55:32] Excluding CDS which overlaps existing RNA (tRNA) at JACKZR010000721.1:519..701 on + strand\\n[10:55:32] Excluding CDS which overlaps existing RNA (tRNA) at JACKZR010000721.1:1988..2224 on + strand\\n[10:55:32] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZR010000741.1:9080..9463 on - strand\\n[10:55:33] Excluding CDS which overlaps existing RNA (tRNA) at JACKZR010000914.1:3122..3454 on - strand\\n[10:55:33] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZR010000928.1:597..719 on - strand\\n[10:55:33] Excluding CDS which overlaps existing RNA (repeat_region) at JACKZR010000928.1:2549..2980 on - strand\\n[10:55:33] Found 7177 CDS\\n[10:55:33] Connecting features back to sequences\\n[10:55:33] Not using genus-specific database. Try --usegenus to enable it.\\n[10:55:33] Annotating CDS, please be patient.\\n[10:55:33] Will use 4 CPUs for similarity searching.\\n[10:55:35] There are still 7177 unannotated CDS left (started with 7177)\\n[10:55:35] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:55:35] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZR01\\\\/\\\\/Nostoc_JACKZR01\\\\.IS\\\\.tmp\\\\.7265\\\\.faa | parallel --gnu --plain -j 4 --block 279651 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZR01\\\\/\\\\/Nostoc_JACKZR01\\\\.IS\\\\.tmp\\\\.7265\\\\.blast 2> /dev/null\\n[10:55:55] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZR01\\\\/\\\\/Nostoc_JACKZR01\\\\.IS\\\\.tmp\\\\.7265\\\\.faa | parallel --gnu --plain -j 4 --block 279651 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JACKZR01\\\\/\\\\/Nostoc_JACKZR01\\\\.IS\\\\.tmp\\\\.7265\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:55:55] This is prokka 1.14.6\\n[10:55:55] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:55:55] Homepage is https://github.com/tseemann/prokka\\n[10:55:55] Local time is Mon Feb 21 10:55:55 2022\\n[10:55:55] You are tiagoferreiraleao\\n[10:55:55] Operating system is darwin\\n[10:55:55] You have BioPerl 1.007002\\n[10:55:55] System has 4 cores.\\n[10:55:55] Option --cpu asked for 12 cores, but system only has 4\\n[10:55:55] Will use maximum of 4 cores.\\n[10:55:55] Annotating as >>> Bacteria <<<\\n[10:55:55] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAGY01.fasta' contents.\\n[10:55:55] Setting --locustag OHFEBJEF from MD5 81feb3ef5134c8940c506d6fd421ff3a\\n[10:55:55] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAGY01/\\n[10:55:55] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGY01\\\\/\\n[10:55:55] Using filename prefix: Nostoc_LAGY01.XXX\\n[10:55:55] Setting HMMER_NCPU=1\\n[10:55:55] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAGY01//Nostoc_LAGY01.log\\n[10:55:55] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAGY01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAGY01/ --prefix Nostoc_LAGY01 --cpus 12\\n[10:55:55] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:55:55] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:55:55] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:55:55] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:55:55] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:55:55] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:55:55] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:55:55] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:55:55] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:55:55] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:55:55] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:55:55] Looking for 'egrep' - found /usr/bin/egrep\\n[10:55:55] Looking for 'find' - found /usr/bin/find\\n[10:55:55] Looking for 'grep' - found /usr/bin/grep\\n[10:55:55] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:55:55] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:55:55] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:55:55] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:55:55] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:55:55] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:55:55] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:55:55] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:55:56] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:55:56] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:55:56] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:55:56] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:55:56] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:55:56] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:55:56] Looking for 'sed' - found /usr/bin/sed\\n[10:55:56] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:55:56] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:55:56] Using genetic code table 11.\\n[10:55:56] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAGY01.fasta\\n[10:55:56] Wrote 178 contigs totalling 9011414 bp.\\n[10:55:56] Predicting tRNAs and tmRNAs\\n[10:55:56] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGY01\\\\/\\\\/Nostoc_LAGY01\\\\.fna\\n[10:55:59] 1 tRNA-Cys [96248,96319] 33 (gca)\\n[10:55:59] 2 tRNA-His [246147,246220] 34 (gtg)\\n[10:55:59] 1 tRNA-Gly c[231434,231504] 33 (tcc)\\n[10:55:59] 1 tRNA-Ala c[90498,90570] 34 (tgc)\\n[10:55:59] 1 tRNA-Trp c[19699,19772] 34 (cca)\\n[10:55:59] 2 tRNA-Thr c[65373,65444] 33 (cgt)\\n[10:55:59] 1 tRNA-Pro [111945,112018] 35 (cgg)\\n[10:55:59] 1 tRNA-Arg [119815,119887] 34 (ccg)\\n[10:55:59] 1 tRNA-Asn [2742,2815] 34 (gtt)\\n[10:55:59] 2 tRNA-Leu c[102414,102497] 36 (tag)\\n[10:55:59] 1 tRNA-Gln c[29230,29303] 34 (ttg)\\n[10:55:59] 2 tRNA-Leu c[34057,34141] 35 (caa)\\n[10:55:59] 3 tRNA-Ala [82843,82915] 34 (cgc)\\n[10:55:59] 1 tRNA-Ala [166022,166094] 34 (ggc)\\n[10:55:59] 1 tRNA-Asp [80899,80972] 35 (gtc)\\n[10:55:59] 1 tRNA-Lys [80206,80277] 33 (ttt)\\n[10:55:59] 1 tmRNA [18403,18790] 101,151 ANNIVKFARREALVAA*\\n[10:55:59] 1 tRNA-Ser [56515,56602] 35 (gga)\\n[10:55:59] 2 tRNA-Gly c[64747,64820] 34 (ccc)\\n[10:55:59] 1 tRNA-Thr [46572,46646] 33 (agt)\\n[10:55:59] 2 tRNA-Leu [48037,48111] 35 (caa)\\n[10:55:59] 1 tRNA-Leu [10524,10605] 35 (cag)\\n[10:55:59] 1 tRNA-Ile c[45433,45508] 34 (tat)\\n[10:55:59] 2 tRNA-Met c[47114,47189] 34 (cat)\\n[10:55:59] 3 tRNA-Met c[47191,47266] 34 (cat)\\n[10:55:59] 1 tRNA-Arg [55707,55780] 35 (acg)\\n[10:55:59] 1 tRNA-Pro c[38314,38387] 35 (tgg)\\n[10:55:59] 2 tRNA-Ser c[47069,47155] 36 (tga)\\n[10:55:59] 3 tRNA-Tyr [49606,49690] 35 (gta)\\n[10:55:59] 4 tRNA-Thr [49766,49837] 33 (ggt)\\n[10:55:59] 1 tRNA-Pro c[43680,43753] 35 (ggg)\\n[10:55:59] 1 tRNA-Ser [28751,28837] 36 (cga)\\n[10:55:59] 1 tRNA-Leu [61613,61693] 35 (gag)\\n[10:55:59] 1 tRNA-Ser c[15630,15723] 36 (gct)\\n[10:55:59] 2 tRNA-Lys c[21398,21468] 33 (ttt)\\n[10:55:59] 3 tRNA-Ser c[21727,21810] 34 (tga)\\n[10:55:59] 4 tRNA-His c[21815,21890] 35 (gtg)\\n[10:55:59] 5 tRNA-Ala c[21896,21970] 35 (tgc)\\n[10:55:59] 6 tRNA-Arg c[22062,22137] 35 (tct)\\n[10:55:59] 7 tRNA-Val c[22144,22219] 34 (tac)\\n[10:55:59] 8 tRNA-Thr c[22301,22373] 34 (tgt)\\n[10:55:59] 9 tRNA-Leu c[22668,22742] 35 (tag)\\n[10:55:59] 10 tRNA-Leu c[22854,22930] 34 (gag)\\n[10:55:59] 11 tRNA-Pro c[23038,23111] 35 (tgg)\\n[10:55:59] 12 tRNA-Leu c[23113,23190] 35 (taa)\\n[10:55:59] 13 tRNA-Leu c[23211,23288] 36 (caa)\\n[10:55:59] 14 tRNA-Gln c[23314,23389] 35 (ttg)\\n[10:55:59] 15 tRNA-Gln c[24264,24338] 35 (ctg)\\n[10:55:59] 16 tRNA-Asn c[24349,24423] 33 (gtt)\\n[10:55:59] 17 tRNA-Phe c[24429,24504] 36 (gaa)\\n[10:55:59] 18 tRNA-Arg c[24592,24667] 35 (acg)\\n[10:55:59] 19 tRNA-Ser c[25429,25515] 35 (gct)\\n[10:55:59] 20 tRNA-Tyr c[25519,25600] 35 (gta)\\n[10:55:59] 21 tRNA-Glu c[25611,25685] 35 (ttc)\\n[10:55:59] tRNA c[25773,25847] is a pseudo/wacky gene - skipping.\\n[10:55:59] 23 tRNA-Met c[25850,25923] 32 (cat)\\n[10:55:59] 24 tRNA-Glu c[26033,26109] 35 (ctc)\\n[10:55:59] 25 tRNA-Asp c[26113,26188] 36 (gtc)\\n[10:55:59] 1 tRNA-Met [18731,18804] 33 (cat)\\n[10:55:59] 2 tRNA-Gly [25614,25685] 33 (gcc)\\n[10:55:59] 1 tRNA-Thr [30023,30095] 33 (tgt)\\n[10:55:59] 1 tRNA-Met c[41067,41141] 35 (cat)\\n[10:55:59] 1 tRNA-Asp [25382,25459] 38 (gtc)\\n[10:55:59] 2 tRNA-Glu [25462,25538] 35 (ctc)\\n[10:55:59] 3 tRNA-Cys [25543,25618] 34 (gca)\\n[10:55:59] 4 tRNA-Met [25624,25698] 33 (cat)\\n[10:55:59] 5 tRNA-Trp [25701,25773] 34 (cca)\\n[10:55:59] 6 tRNA-Pyl [25798,25869] 33 (cta)\\n[10:55:59] 7 tRNA-Glu [25900,25975] 34 (ttc)\\n[10:55:59] 8 tRNA-Tyr [25979,26062] 36 (gta)\\n[10:55:59] 9 tRNA-Ser [26066,26155] 33 (gct)\\n[10:55:59] 10 tRNA-Arg [26160,26235] 35 (acg)\\n[10:55:59] 11 tRNA-Phe [26240,26314] 35 (gaa)\\n[10:55:59] 12 tRNA-Asn [26323,26397] 33 (gtt)\\n[10:55:59] 13 tRNA-Gln [26403,26478] 35 (ttg)\\n[10:55:59] 14 tRNA-Leu [26486,26563] 35 (caa)\\n[10:55:59] 15 tRNA-Pro [26568,26644] 35 (tgg)\\n[10:55:59] 16 tRNA-Leu [26648,26722] 35 (tag)\\n[10:55:59] 17 tRNA-Leu [26742,26819] 35 (gag)\\n[10:55:59] 18 tRNA-Lys [26823,26899] 34 (ttt)\\n[10:55:59] 19 tRNA-Lys [26905,26979] 33 (ctt)\\n[10:55:59] 20 tRNA-Leu [26985,27060] 36 (taa)\\n[10:55:59] 21 tRNA-Cys [27070,27147] 34 (gca)\\n[10:55:59] 22 tRNA-Thr [27155,27227] 34 (tgt)\\n[10:55:59] 23 tRNA-Gly [27233,27303] 33 (tcc)\\n[10:55:59] 24 tRNA-Val [27309,27384] 34 (tac)\\n[10:55:59] 25 tRNA-Arg [27391,27465] 35 (tct)\\n[10:55:59] 26 tRNA-Ala [27470,27542] 34 (tgc)\\n[10:55:59] 27 tRNA-His [27550,27626] 35 (gtg)\\n[10:55:59] 28 tRNA-Ser [27637,27721] 34 (tga)\\n[10:55:59] 1 tRNA-Ala [54011,54097] 29 (cgc)\\n[10:55:59] 1 tRNA-Phe c[895,968] 34 (gaa)\\n[10:55:59] 1 tRNA-Arg c[3738,3812] 35 (tct)\\n[10:55:59] 1 tRNA-Lys c[21229,21301] 33 (ctt)\\n[10:55:59] 1 tRNA-Ala c[29310,29384] 34 (tgc)\\n[10:55:59] 2 tRNA-Ile c[29468,29543] 35 (gat)\\n[10:55:59] 1 tRNA-Val c[14604,14676] 34 (tac)\\n[10:55:59] 1 tRNA-Glu [22513,22586] 35 (ttc)\\n[10:55:59] 1 tRNA-Val [14294,14367] 35 (gac)\\n[10:55:59] Found 98 tRNAs\\n[10:55:59] Predicting Ribosomal RNAs\\n[10:55:59] Running Barrnap with 4 threads\\n[10:56:01] 1 LAGY01000095.1 26329 23S ribosomal RNA\\n[10:56:01] 2 LAGY01000095.1 29665 16S ribosomal RNA\\n[10:56:01] Found 2 rRNAs\\n[10:56:01] Skipping ncRNA search, enable with --rfam if desired.\\n[10:56:01] Total of 99 tRNA + rRNA features\\n[10:56:01] Searching for CRISPR repeats\\n[10:56:01] CRISPR1 LAGY01000005.1 114413 with 24 spacers\\n[10:56:02] CRISPR2 LAGY01000007.1 153401 with 28 spacers\\n[10:56:02] CRISPR3 LAGY01000007.1 166835 with 43 spacers\\n[10:56:02] CRISPR4 LAGY01000012.1 63985 with 7 spacers\\n[10:56:02] CRISPR5 LAGY01000013.1 27645 with 15 spacers\\n[10:56:02] CRISPR6 LAGY01000021.1 8770 with 7 spacers\\n[10:56:02] CRISPR7 LAGY01000026.1 67093 with 10 spacers\\n[10:56:02] CRISPR8 LAGY01000027.1 42415 with 14 spacers\\n[10:56:02] CRISPR9 LAGY01000033.1 39713 with 17 spacers\\n[10:56:02] CRISPR10 LAGY01000037.1 29555 with 20 spacers\\n[10:56:02] CRISPR11 LAGY01000047.1 5052 with 6 spacers\\n[10:56:02] CRISPR12 LAGY01000124.1 209 with 12 spacers\\n[10:56:02] CRISPR13 LAGY01000135.1 9924 with 18 spacers\\n[10:56:02] Found 13 CRISPRs\\n[10:56:02] Predicting coding sequences\\n[10:56:02] Contigs total 9011414 bp, so using single mode\\n[10:56:02] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGY01\\\\/\\\\/Nostoc_LAGY01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:56:09] Excluding CDS which overlaps existing RNA (tRNA) at LAGY01000001.1:246068..246211 on - strand\\n[10:56:10] Excluding CDS which overlaps existing RNA (repeat_region) at LAGY01000007.1:166876..167040 on - strand\\n[10:56:11] Excluding CDS which overlaps existing RNA (repeat_region) at LAGY01000021.1:9090..9215 on + strand\\n[10:56:12] Excluding CDS which overlaps existing RNA (repeat_region) at LAGY01000026.1:67302..67454 on + strand\\n[10:56:12] Excluding CDS which overlaps existing RNA (repeat_region) at LAGY01000027.1:42586..42702 on - strand\\n[10:56:12] Excluding CDS which overlaps existing RNA (repeat_region) at LAGY01000047.1:5444..6403 on - strand\\n[10:56:13] Excluding CDS which overlaps existing RNA (tRNA) at LAGY01000065.1:53192..54562 on + strand\\n[10:56:14] Excluding CDS which overlaps existing RNA (repeat_region) at LAGY01000124.1:947..1225 on - strand\\n[10:56:14] Found 7623 CDS\\n[10:56:14] Connecting features back to sequences\\n[10:56:14] Not using genus-specific database. Try --usegenus to enable it.\\n[10:56:14] Annotating CDS, please be patient.\\n[10:56:14] Will use 4 CPUs for similarity searching.\\n[10:56:16] There are still 7623 unannotated CDS left (started with 7623)\\n[10:56:16] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:56:16] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGY01\\\\/\\\\/Nostoc_LAGY01\\\\.IS\\\\.tmp\\\\.7377\\\\.faa | parallel --gnu --plain -j 4 --block 304569 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGY01\\\\/\\\\/Nostoc_LAGY01\\\\.IS\\\\.tmp\\\\.7377\\\\.blast 2> /dev/null\\n[10:56:22] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGY01\\\\/\\\\/Nostoc_LAGY01\\\\.IS\\\\.tmp\\\\.7377\\\\.faa | parallel --gnu --plain -j 4 --block 304569 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAGY01\\\\/\\\\/Nostoc_LAGY01\\\\.IS\\\\.tmp\\\\.7377\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:56:23] This is prokka 1.14.6\\n[10:56:23] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:56:23] Homepage is https://github.com/tseemann/prokka\\n[10:56:23] Local time is Mon Feb 21 10:56:23 2022\\n[10:56:23] You are tiagoferreiraleao\\n[10:56:23] Operating system is darwin\\n[10:56:23] You have BioPerl 1.007002\\n[10:56:23] System has 4 cores.\\n[10:56:23] Option --cpu asked for 12 cores, but system only has 4\\n[10:56:23] Will use maximum of 4 cores.\\n[10:56:23] Annotating as >>> Bacteria <<<\\n[10:56:23] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAK01.fasta' contents.\\n[10:56:23] Setting --locustag LHKAMFKC from MD5 514a6f4c51372f49561ae50dea9cedac\\n[10:56:23] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAK01/\\n[10:56:23] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAK01\\\\/\\n[10:56:23] Using filename prefix: Fischerella_NHAK01.XXX\\n[10:56:23] Setting HMMER_NCPU=1\\n[10:56:23] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAK01//Fischerella_NHAK01.log\\n[10:56:23] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAK01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NHAK01/ --prefix Fischerella_NHAK01 --cpus 12\\n[10:56:23] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:56:23] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:56:23] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:56:23] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:56:23] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:56:23] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:56:23] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:56:23] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:56:23] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:56:23] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:56:23] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:56:23] Looking for 'egrep' - found /usr/bin/egrep\\n[10:56:23] Looking for 'find' - found /usr/bin/find\\n[10:56:23] Looking for 'grep' - found /usr/bin/grep\\n[10:56:23] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:56:23] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:56:23] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:56:23] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:56:23] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:56:23] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:56:24] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:56:24] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:56:24] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:56:24] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:56:24] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:56:24] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:56:24] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:56:24] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:56:24] Looking for 'sed' - found /usr/bin/sed\\n[10:56:24] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:56:24] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:56:24] Using genetic code table 11.\\n[10:56:24] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NHAK01.fasta\\n[10:56:25] Wrote 187 contigs totalling 5351095 bp.\\n[10:56:25] Predicting tRNAs and tmRNAs\\n[10:56:25] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAK01\\\\/\\\\/Fischerella_NHAK01\\\\.fna\\n[10:56:27] 1 tRNA-Gln [34451,34522] 33 (ttg)\\n[10:56:27] 1 tRNA-Arg [30380,30455] 36 (cct)\\n[10:56:27] 1 tRNA-Pro [18762,18836] 35 (tgg)\\n[10:56:27] 1 tRNA-Asp c[8227,8300] 35 (gtc)\\n[10:56:27] 1 tRNA-Pro [7322,7395] 35 (cgg)\\n[10:56:27] 1 tRNA-Met [8699,8771] 34 (cat)\\n[10:56:27] 1 tRNA-Leu c[17053,17135] 35 (cag)\\n[10:56:27] 1 tRNA-Arg [2117,2189] 34 (tcg)\\n[10:56:27] 2 tRNA-Asn [2860,2933] 34 (att)\\n[10:56:27] 1 tRNA-Gly c[2104,2175] 33 (gcc)\\n[10:56:27] 1 tRNA-Arg [11137,11211] 35 (tct)\\n[10:56:27] 2 tRNA-Met [35987,36062] 36 (cat)\\n[10:56:27] 3 tRNA-Ala [82154,82228] 35 (cgc)\\n[10:56:27] 1 tRNA-Leu [6621,6704] 36 (tag)\\n[10:56:27] 1 tRNA-Val c[44596,44669] 35 (gac)\\n[10:56:27] 2 tRNA-Met [47617,47693] 35 (cat)\\n[10:56:27] 1 tRNA-Cys c[88,159] 33 (gca)\\n[10:56:27] 1 tRNA-His c[32154,32227] 34 (gtg)\\n[10:56:27] 2 tRNA-Ser c[150814,150907] 36 (gct)\\n[10:56:27] 1 tRNA-Thr [2097,2168] 33 (tgt)\\n[10:56:27] 1 tRNA-Trp c[6612,6685] 34 (cca)\\n[10:56:27] 1 tRNA-Gly c[1312,1383] 33 (ccc)\\n[10:56:27] 1 tRNA-Pro c[9476,9549] 35 (ggg)\\n[10:56:27] 1 tRNA-Ser [30739,30825] 36 (tga)\\n[10:56:27] 1 tRNA-Phe c[83208,83280] 34 (gaa)\\n[10:56:27] 2 tRNA-Val [90621,90693] 34 (tac)\\n[10:56:27] 1 tRNA-Tyr [13790,13874] 35 (gta)\\n[10:56:27] 2 tRNA-Thr [13938,14009] 33 (ggt)\\n[10:56:27] 1 tRNA-Arg c[82924,82997] 35 (acg)\\n[10:56:27] 1 tRNA-Arg [8325,8398] 35 (ccg)\\n[10:56:27] 1 tRNA-Thr [14451,14522] 33 (cgt)\\n[10:56:27] 1 tRNA-Leu [35644,35726] 36 (gag)\\n[10:56:27] 1 tmRNA c[66574,66960] 100,150 ANNIVKFARKEALVAA*\\n[10:56:27] 1 tRNA-Leu [19122,19206] 35 (caa)\\n[10:56:27] 1 tRNA-Ser c[4208,4296] 36 (gga)\\n[10:56:27] 2 tRNA-Ala [50953,51026] 34 (ggc)\\n[10:56:27] 3 tRNA-Glu [94560,94632] 35 (ttc)\\n[10:56:27] 1 tRNA-Gln c[78574,78644] 32 (ttg)\\n[10:56:27] 1 tRNA-Lys [70952,71025] 34 (ttt)\\n[10:56:27] 1 tRNA-Gly c[5058,5128] 33 (tcc)\\n[10:56:27] 1 tRNA-Ser [4627,4713] 36 (cga)\\n[10:56:27] 1 tRNA-Asn c[92019,92092] 34 (gtt)\\n[10:56:27] Found 42 tRNAs\\n[10:56:27] Predicting Ribosomal RNAs\\n[10:56:27] Running Barrnap with 4 threads\\n[10:56:28] 1 NHAK01000038.1 18990 16S ribosomal RNA\\n[10:56:28] 2 NHAK01000038.1 20836 23S ribosomal RNA\\n[10:56:28] 3 NHAK01000038.1 23775 5S ribosomal RNA\\n[10:56:28] Found 3 rRNAs\\n[10:56:28] Skipping ncRNA search, enable with --rfam if desired.\\n[10:56:28] Total of 44 tRNA + rRNA features\\n[10:56:28] Searching for CRISPR repeats\\n[10:56:29] CRISPR1 NHAK01000175.1 1798 with 12 spacers\\n[10:56:29] CRISPR2 NHAK01000175.1 2888 with 31 spacers\\n[10:56:29] CRISPR3 NHAK01000175.1 5384 with 28 spacers\\n[10:56:29] CRISPR4 NHAK01000175.1 7518 with 14 spacers\\n[10:56:29] CRISPR5 NHAK01000175.1 8820 with 6 spacers\\n[10:56:29] CRISPR6 NHAK01000178.1 73476 with 5 spacers\\n[10:56:29] CRISPR7 NHAK01000178.1 73854 with 4 spacers\\n[10:56:29] CRISPR8 NHAK01000178.1 74150 with 15 spacers\\n[10:56:29] CRISPR9 NHAK01000179.1 65413 with 47 spacers\\n[10:56:29] CRISPR10 NHAK01000179.1 69131 with 9 spacers\\n[10:56:29] CRISPR11 NHAK01000179.1 70083 with 13 spacers\\n[10:56:29] CRISPR12 NHAK01000182.1 8721 with 3 spacers\\n[10:56:29] CRISPR13 NHAK01000182.1 8992 with 123 spacers\\n[10:56:29] CRISPR14 NHAK01000184.1 1 with 4 spacers\\n[10:56:29] CRISPR15 NHAK01000184.1 357 with 35 spacers\\n[10:56:29] CRISPR16 NHAK01000184.1 2994 with 3 spacers\\n[10:56:29] CRISPR17 NHAK01000184.1 3455 with 27 spacers\\n[10:56:29] CRISPR18 NHAK01000184.1 5717 with 70 spacers\\n[10:56:29] CRISPR19 NHAK01000184.1 11139 with 7 spacers\\n[10:56:29] CRISPR20 NHAK01000186.1 20061 with 4 spacers\\n[10:56:29] CRISPR21 NHAK01000186.1 20634 with 154 spacers\\n[10:56:29] Found 21 CRISPRs\\n[10:56:29] Predicting coding sequences\\n[10:56:29] Contigs total 5351095 bp, so using single mode\\n[10:56:29] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAK01\\\\/\\\\/Fischerella_NHAK01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:56:33] Excluding CDS which overlaps existing RNA (tRNA) at NHAK01000026.1:8262..8657 on + strand\\n[10:56:33] Excluding CDS which overlaps existing RNA (rRNA) at NHAK01000038.1:20145..20459 on + strand\\n[10:56:33] Excluding CDS which overlaps existing RNA (rRNA) at NHAK01000038.1:21231..21410 on + strand\\n[10:56:33] Excluding CDS which overlaps existing RNA (rRNA) at NHAK01000038.1:23196..23372 on - strand\\n[10:56:34] Excluding CDS which overlaps existing RNA (tRNA) at NHAK01000134.1:82881..83552 on + strand\\n[10:56:35] Excluding CDS which overlaps existing RNA (repeat_region) at NHAK01000182.1:10507..10662 on + strand\\n[10:56:35] Excluding CDS which overlaps existing RNA (repeat_region) at NHAK01000182.1:13723..13938 on + strand\\n[10:56:35] Excluding CDS which overlaps existing RNA (tRNA) at NHAK01000185.1:90779..92218 on - strand\\n[10:56:36] Found 4518 CDS\\n[10:56:36] Connecting features back to sequences\\n[10:56:36] Not using genus-specific database. Try --usegenus to enable it.\\n[10:56:36] Annotating CDS, please be patient.\\n[10:56:36] Will use 4 CPUs for similarity searching.\\n[10:56:36] There are still 4518 unannotated CDS left (started with 4518)\\n[10:56:36] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:56:36] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAK01\\\\/\\\\/Fischerella_NHAK01\\\\.IS\\\\.tmp\\\\.7490\\\\.faa | parallel --gnu --plain -j 4 --block 170749 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAK01\\\\/\\\\/Fischerella_NHAK01\\\\.IS\\\\.tmp\\\\.7490\\\\.blast 2> /dev/null\\n[10:56:54] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAK01\\\\/\\\\/Fischerella_NHAK01\\\\.IS\\\\.tmp\\\\.7490\\\\.faa | parallel --gnu --plain -j 4 --block 170749 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NHAK01\\\\/\\\\/Fischerella_NHAK01\\\\.IS\\\\.tmp\\\\.7490\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:56:54] This is prokka 1.14.6\\n[10:56:54] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:56:54] Homepage is https://github.com/tseemann/prokka\\n[10:56:54] Local time is Mon Feb 21 10:56:54 2022\\n[10:56:54] You are tiagoferreiraleao\\n[10:56:54] Operating system is darwin\\n[10:56:54] You have BioPerl 1.007002\\n[10:56:55] System has 4 cores.\\n[10:56:55] Option --cpu asked for 12 cores, but system only has 4\\n[10:56:55] Will use maximum of 4 cores.\\n[10:56:55] Annotating as >>> Bacteria <<<\\n[10:56:55] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQB01.fasta' contents.\\n[10:56:55] Setting --locustag JGICIIKI from MD5 302c2242ac780daa824fbf206ad0c845\\n[10:56:55] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQB01/\\n[10:56:55] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQB01\\\\/\\n[10:56:55] Using filename prefix: Fischerella_NMQB01.XXX\\n[10:56:55] Setting HMMER_NCPU=1\\n[10:56:55] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQB01//Fischerella_NMQB01.log\\n[10:56:55] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQB01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Fischerella_NMQB01/ --prefix Fischerella_NMQB01 --cpus 12\\n[10:56:55] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:56:55] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:56:55] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:56:55] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:56:55] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:56:55] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:56:55] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:56:55] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:56:55] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:56:55] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:56:55] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:56:55] Looking for 'egrep' - found /usr/bin/egrep\\n[10:56:55] Looking for 'find' - found /usr/bin/find\\n[10:56:55] Looking for 'grep' - found /usr/bin/grep\\n[10:56:55] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:56:55] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:56:55] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:56:55] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:56:55] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:56:55] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:56:55] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:56:55] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:56:55] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:56:55] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:56:56] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:56:56] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:56:56] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:56:56] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:56:56] Looking for 'sed' - found /usr/bin/sed\\n[10:56:56] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:56:56] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:56:56] Using genetic code table 11.\\n[10:56:56] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Fischerella_NMQB01.fasta\\n[10:56:56] Wrote 937 contigs totalling 6104484 bp.\\n[10:56:56] Predicting tRNAs and tmRNAs\\n[10:56:56] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQB01\\\\/\\\\/Fischerella_NMQB01\\\\.fna\\n[10:56:57] 1 tRNA-Met c[6969,7044] 36 (cat)\\n[10:56:57] 1 tRNA-Met c[3735,3807] 34 (cat)\\n[10:56:57] 1 tRNA-Gln c[15047,15118] 33 (ttg)\\n[10:56:57] 1 tRNA-Gln c[64,138] 33 (ttg)\\n[10:56:57] 2 tRNA-Tyr c[147,231] 35 (gta)\\n[10:56:57] 3 tRNA-Val c[237,312] 34 (tac)\\n[10:56:57] 4 tRNA-Glu c[316,391] 34 (ttc)\\n[10:56:57] 5 tRNA-Thr c[396,471] 34 (ggt)\\n[10:56:57] 6 tRNA-Asn c[475,549] 33 (gtt)\\n[10:56:57] 1 tRNA-Val c[1137,1209] 34 (tac)\\n[10:56:57] 1 tRNA-Leu [5788,5871] 36 (tag)\\n[10:56:57] 1 tRNA-Gly c[345,415] 33 (tcc)\\n[10:56:57] 1 tRNA-Ser c[7973,8066] 36 (gct)\\n[10:56:57] 1 tRNA-Pro c[6300,6373] 35 (cgg)\\n[10:56:57] 1 tRNA-Gly c[377,448] 33 (gcc)\\n[10:56:57] 1 tRNA-Lys [22106,22179] 34 (ttt)\\n[10:56:57] 1 tRNA-Leu c[5827,5909] 35 (cag)\\n[10:56:57] 1 tRNA-Arg c[6321,6394] 35 (acg)\\n[10:56:57] 1 tRNA-Gly [1729,1800] 33 (ccc)\\n[10:56:57] 1 tRNA-Arg c[521,595] 35 (tct)\\n[10:56:57] 1 tRNA-Gln [4444,4514] 32 (ttg)\\n[10:56:57] 1 tRNA-Leu c[3315,3397] 36 (gag)\\n[10:56:57] 1 tRNA-His c[14552,14625] 34 (gtg)\\n[10:56:57] 1 tRNA-Val [311,384] 35 (gac)\\n[10:56:57] 1 tRNA-Asn c[9040,9113] 34 (att)\\n[10:56:57] 1 tRNA-Ala c[257,331] 35 (cgc)\\n[10:56:57] 1 tRNA-Ile [732,808] 35 (gat)\\n[10:56:57] 2 tRNA-Ala [825,900] 34 (tgc)\\n[10:56:57] 1 tRNA-Ala c[2998,3071] 34 (ggc)\\n[10:56:57] 1 tRNA-Glu c[5776,5848] 35 (ttc)\\n[10:56:57] 1 tRNA-Trp [7428,7501] 34 (cca)\\n[10:56:57] 1 tRNA-Pro c[14656,14729] 35 (ggg)\\n[10:56:58] 1 tRNA-Ser [4411,4497] 36 (cga)\\n[10:56:58] 1 tRNA-Met c[166,242] 35 (cat)\\n[10:56:58] 1 tRNA-Thr [17719,17790] 33 (cgt)\\n[10:56:58] 1 tRNA-Phe c[384,456] 34 (gaa)\\n[10:56:58] 1 tRNA-Ser [491,579] 36 (gga)\\n[10:56:58] 1 tRNA-Thr [9297,9368] 33 (tgt)\\n[10:56:58] 1 tRNA-Ser c[320,406] 36 (tga)\\n[10:56:58] 1 tRNA-Asn [2883,2956] 34 (gtt)\\n[10:56:58] 1 tRNA-Arg [1980,2055] 36 (cct)\\n[10:56:58] 1 tRNA-Arg c[2592,2665] 35 (ccg)\\n[10:56:58] 1 tRNA-Asp c[8744,8817] 35 (gtc)\\n[10:56:58] 1 tRNA-Pro [11756,11830] 35 (tgg)\\n[10:56:58] 1 tRNA-Ser [351,445] 36 (gga)\\n[10:56:58] 2 tRNA-Phe [500,575] 34 (gaa)\\n[10:56:58] 3 tRNA-Gly [590,664] 33 (gcc)\\n[10:56:58] 4 tRNA-Leu [677,765] 35 (taa)\\n[10:56:58] 1 tRNA-Cys [9218,9289] 33 (gca)\\n[10:56:58] 1 tRNA-Leu [2105,2189] 35 (caa)\\n[10:56:58] 1 tRNA-Arg [840,912] 34 (tcg)\\n[10:56:58] Found 51 tRNAs\\n[10:56:58] Predicting Ribosomal RNAs\\n[10:56:58] Running Barrnap with 4 threads\\n[10:56:59] 1 NMQB01000006.1 7 5S ribosomal RNA\\n[10:56:59] 2 NMQB01000341.1 4 5S ribosomal RNA\\n[10:56:59] 3 NMQB01000377.1 175 16S ribosomal RNA (partial)\\n[10:56:59] 4 NMQB01000398.1 1 16S ribosomal RNA (partial)\\n[10:56:59] 5 NMQB01000398.1 1005 23S ribosomal RNA (partial)\\n[10:56:59] 6 NMQB01000433.1 5 16S ribosomal RNA (partial)\\n[10:56:59] 7 NMQB01000442.1 1 23S ribosomal RNA (partial)\\n[10:56:59] 8 NMQB01000721.1 116 16S ribosomal RNA (partial)\\n[10:56:59] Found 8 rRNAs\\n[10:56:59] Skipping ncRNA search, enable with --rfam if desired.\\n[10:56:59] Total of 59 tRNA + rRNA features\\n[10:56:59] Searching for CRISPR repeats\\n[10:57:00] CRISPR1 NMQB01000134.1 5362 with 3 spacers\\n[10:57:00] CRISPR2 NMQB01000190.1 4614 with 4 spacers\\n[10:57:00] CRISPR3 NMQB01000190.1 5122 with 5 spacers\\n[10:57:00] CRISPR4 NMQB01000471.1 5786 with 5 spacers\\n[10:57:00] Found 4 CRISPRs\\n[10:57:00] Predicting coding sequences\\n[10:57:00] Contigs total 6104484 bp, so using single mode\\n[10:57:00] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQB01\\\\/\\\\/Fischerella_NMQB01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:57:05] Excluding CDS which overlaps existing RNA (repeat_region) at NMQB01000190.1:4702..5172 on - strand\\n[10:57:06] Excluding CDS which overlaps existing RNA (tRNA) at NMQB01000259.1:6278..6949 on + strand\\n[10:57:06] Excluding CDS which overlaps existing RNA (rRNA) at NMQB01000398.1:2062..2268 on + strand\\n[10:57:06] Excluding CDS which overlaps existing RNA (rRNA) at NMQB01000442.1:1288..1467 on - strand\\n[10:57:06] Excluding CDS which overlaps existing RNA (tRNA) at NMQB01000504.1:4399..4527 on + strand\\n[10:57:07] Excluding CDS which overlaps existing RNA (rRNA) at NMQB01000721.1:158..472 on - strand\\n[10:57:07] Excluding CDS which overlaps existing RNA (tRNA) at NMQB01000732.1:2757..4196 on + strand\\n[10:57:07] Found 5200 CDS\\n[10:57:07] Connecting features back to sequences\\n[10:57:07] Not using genus-specific database. Try --usegenus to enable it.\\n[10:57:07] Annotating CDS, please be patient.\\n[10:57:07] Will use 4 CPUs for similarity searching.\\n[10:57:08] There are still 5200 unannotated CDS left (started with 5200)\\n[10:57:08] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:57:08] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQB01\\\\/\\\\/Fischerella_NMQB01\\\\.IS\\\\.tmp\\\\.7603\\\\.faa | parallel --gnu --plain -j 4 --block 194848 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQB01\\\\/\\\\/Fischerella_NMQB01\\\\.IS\\\\.tmp\\\\.7603\\\\.blast 2> /dev/null\\n[10:57:16] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQB01\\\\/\\\\/Fischerella_NMQB01\\\\.IS\\\\.tmp\\\\.7603\\\\.faa | parallel --gnu --plain -j 4 --block 194848 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Fischerella_NMQB01\\\\/\\\\/Fischerella_NMQB01\\\\.IS\\\\.tmp\\\\.7603\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:57:16] This is prokka 1.14.6\\n[10:57:16] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:57:16] Homepage is https://github.com/tseemann/prokka\\n[10:57:16] Local time is Mon Feb 21 10:57:16 2022\\n[10:57:16] You are tiagoferreiraleao\\n[10:57:16] Operating system is darwin\\n[10:57:16] You have BioPerl 1.007002\\n[10:57:16] System has 4 cores.\\n[10:57:16] Option --cpu asked for 12 cores, but system only has 4\\n[10:57:16] Will use maximum of 4 cores.\\n[10:57:16] Annotating as >>> Bacteria <<<\\n[10:57:16] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Mastigocladus_MNPM02.fasta' contents.\\n[10:57:16] Setting --locustag PGJICEHD from MD5 9032ce1d991eb6ed691aad33cbfe3e25\\n[10:57:16] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Mastigocladus_MNPM02/\\n[10:57:16] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Mastigocladus_MNPM02\\\\/\\n[10:57:16] Using filename prefix: Mastigocladus_MNPM02.XXX\\n[10:57:16] Setting HMMER_NCPU=1\\n[10:57:16] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Mastigocladus_MNPM02//Mastigocladus_MNPM02.log\\n[10:57:16] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Mastigocladus_MNPM02.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Mastigocladus_MNPM02/ --prefix Mastigocladus_MNPM02 --cpus 12\\n[10:57:16] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:57:16] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:57:16] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:57:16] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:57:16] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:57:16] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:57:16] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:57:16] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:57:16] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:57:16] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:57:16] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:57:16] Looking for 'egrep' - found /usr/bin/egrep\\n[10:57:16] Looking for 'find' - found /usr/bin/find\\n[10:57:16] Looking for 'grep' - found /usr/bin/grep\\n[10:57:16] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:57:17] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:57:17] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:57:17] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:57:17] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:57:17] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:57:17] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:57:17] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:57:17] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:57:17] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:57:17] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:57:17] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:57:17] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:57:17] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:57:17] Looking for 'sed' - found /usr/bin/sed\\n[10:57:17] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:57:17] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:57:17] Using genetic code table 11.\\n[10:57:17] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Mastigocladus_MNPM02.fasta\\n[10:57:18] Wrote 52 contigs totalling 6911113 bp.\\n[10:57:18] Predicting tRNAs and tmRNAs\\n[10:57:18] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Mastigocladus_MNPM02\\\\/\\\\/Mastigocladus_MNPM02\\\\.fna\\n[10:57:20] 1 tRNA-Leu c[269899,269982] 36 (tag)\\n[10:57:20] 2 tRNA-Trp c[448551,448624] 34 (cca)\\n[10:57:20] 3 tRNA-Leu c[610542,610624] 36 (gag)\\n[10:57:20] 1 tmRNA [14822,15209] 101,154 ANNIVKFARKEALVAA**\\n[10:57:20] 2 tRNA-Gly c[32594,32665] 33 (ccc)\\n[10:57:20] 3 tRNA-Arg c[99542,99615] 35 (acg)\\n[10:57:20] 4 tRNA-Lys c[226363,226436] 34 (ctt)\\n[10:57:20] 1 tRNA-Leu c[83855,83937] 35 (cag)\\n[10:57:20] 2 tRNA-Gly c[153555,153625] 33 (tcc)\\n[10:57:20] 3 tRNA-Ala c[198311,198385] 35 (ggc)\\n[10:57:20] 1 tRNA-Leu [46619,46703] 35 (caa)\\n[10:57:20] 2 tRNA-Gln [82176,82247] 33 (ttg)\\n[10:57:20] 1 tRNA-Asn c[72053,72126] 34 (att)\\n[10:57:20] 2 tRNA-Phe c[72497,72570] 35 (gaa)\\n[10:57:20] 3 tRNA-Thr c[123099,123170] 33 (cgt)\\n[10:57:20] 1 tRNA-Met c[93546,93622] 35 (cat)\\n[10:57:20] 2 tRNA-Val [96656,96729] 35 (gac)\\n[10:57:20] 3 tRNA-Val [210576,210648] 34 (tac)\\n[10:57:20] 1 tRNA-Arg [63784,63856] 34 (ccg)\\n[10:57:20] 1 tRNA-Phe [30772,30844] 34 (gaa)\\n[10:57:20] 1 tRNA-Asn c[16071,16144] 34 (gtt)\\n[10:57:20] 1 tRNA-Ile [396876,396951] 35 (gat)\\n[10:57:20] 2 tRNA-Ala [397038,397112] 34 (tgc)\\n[10:57:20] 1 tRNA-Gly c[241897,241968] 33 (gcc)\\n[10:57:20] 2 tRNA-Thr [399737,399808] 33 (tgt)\\n[10:57:20] 3 tRNA-Ser c[503241,503326] 35 (tga)\\n[10:57:20] 1 tRNA-Ser [31372,31460] 36 (gga)\\n[10:57:20] 2 tRNA-Asp [34491,34564] 35 (gtc)\\n[10:57:20] 3 tRNA-Met c[168088,168163] 36 (cat)\\n[10:57:20] 4 tRNA-Arg [243153,243229] 36 (cct)\\n[10:57:20] 5 tRNA-His c[378056,378129] 34 (gtg)\\n[10:57:20] 1 tRNA-Met [5002,5075] 34 (cat)\\n[10:57:20] 2 tRNA-Pro c[169621,169694] 35 (cgg)\\n[10:57:20] 1 tRNA-Ala c[43274,43346] 34 (cgc)\\n[10:57:20] 1 tRNA-Ser c[52663,52756] 36 (gct)\\n[10:57:20] 2 tRNA-Cys c[108300,108371] 33 (gca)\\n[10:57:20] 1 tRNA-Pro [26063,26136] 35 (tgg)\\n[10:57:20] 2 tRNA-Glu [69575,69647] 35 (ttc)\\n[10:57:20] 1 tRNA-Arg [38275,38349] 35 (tct)\\n[10:57:20] 1 tRNA-Trp c[59995,60069] 35 (cca)\\n[10:57:20] 1 tRNA-Lys [15842,15913] 33 (ttt)\\n[10:57:20] 1 tRNA-Ser [24649,24735] 36 (cga)\\n[10:57:20] 1 tRNA-Thr c[2402,2473] 33 (ggt)\\n[10:57:20] 2 tRNA-Tyr c[2551,2635] 35 (gta)\\n[10:57:20] Found 44 tRNAs\\n[10:57:20] Predicting Ribosomal RNAs\\n[10:57:20] Running Barrnap with 4 threads\\n[10:57:22] 1 MNPM02000014.1 395258 16S ribosomal RNA\\n[10:57:22] 2 MNPM02000014.1 397318 23S ribosomal RNA\\n[10:57:22] Found 2 rRNAs\\n[10:57:22] Skipping ncRNA search, enable with --rfam if desired.\\n[10:57:22] Total of 45 tRNA + rRNA features\\n[10:57:22] Searching for CRISPR repeats\\n[10:57:22] CRISPR1 MNPM02000002.1 135770 with 19 spacers\\n[10:57:22] CRISPR2 MNPM02000002.1 142708 with 12 spacers\\n[10:57:22] CRISPR3 MNPM02000002.1 290139 with 11 spacers\\n[10:57:22] CRISPR4 MNPM02000005.1 33564 with 12 spacers\\n[10:57:22] CRISPR5 MNPM02000006.1 162255 with 18 spacers\\n[10:57:22] CRISPR6 MNPM02000006.1 170930 with 14 spacers\\n[10:57:23] CRISPR7 MNPM02000014.1 429046 with 4 spacers\\n[10:57:23] CRISPR8 MNPM02000015.1 95455 with 11 spacers\\n[10:57:23] CRISPR9 MNPM02000015.1 161343 with 5 spacers\\n[10:57:23] CRISPR10 MNPM02000015.1 200756 with 7 spacers\\n[10:57:23] CRISPR11 MNPM02000015.1 201646 with 4 spacers\\n[10:57:23] CRISPR12 MNPM02000017.1 330541 with 5 spacers\\n[10:57:23] CRISPR13 MNPM02000018.1 194647 with 12 spacers\\n[10:57:23] CRISPR14 MNPM02000018.1 196674 with 6 spacers\\n[10:57:23] CRISPR15 MNPM02000022.1 38981 with 7 spacers\\n[10:57:23] CRISPR16 MNPM02000035.1 3140 with 19 spacers\\n[10:57:23] Found 16 CRISPRs\\n[10:57:23] Predicting coding sequences\\n[10:57:23] Contigs total 6911113 bp, so using single mode\\n[10:57:23] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Mastigocladus_MNPM02\\\\/\\\\/Mastigocladus_MNPM02\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:57:29] Excluding CDS which overlaps existing RNA (tRNA) at MNPM02000001.1:269826..270002 on - strand\\n[10:57:29] Excluding CDS which overlaps existing RNA (repeat_region) at MNPM02000002.1:142628..142765 on - strand\\n[10:57:30] Excluding CDS which overlaps existing RNA (tRNA) at MNPM02000004.1:81905..82285 on + strand\\n[10:57:30] Excluding CDS which overlaps existing RNA (repeat_region) at MNPM02000005.1:33708..33986 on + strand\\n[10:57:31] Excluding CDS which overlaps existing RNA (repeat_region) at MNPM02000015.1:96192..96371 on - strand\\n[10:57:32] Excluding CDS which overlaps existing RNA (tRNA) at MNPM02000020.1:43105..43335 on + strand\\n[10:57:33] Found 5785 CDS\\n[10:57:33] Connecting features back to sequences\\n[10:57:33] Not using genus-specific database. Try --usegenus to enable it.\\n[10:57:33] Annotating CDS, please be patient.\\n[10:57:33] Will use 4 CPUs for similarity searching.\\n[10:57:35] There are still 5785 unannotated CDS left (started with 5785)\\n[10:57:35] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:57:35] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Mastigocladus_MNPM02\\\\/\\\\/Mastigocladus_MNPM02\\\\.IS\\\\.tmp\\\\.7715\\\\.faa | parallel --gnu --plain -j 4 --block 234387 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Mastigocladus_MNPM02\\\\/\\\\/Mastigocladus_MNPM02\\\\.IS\\\\.tmp\\\\.7715\\\\.blast 2> /dev/null\\n[10:57:39] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Mastigocladus_MNPM02\\\\/\\\\/Mastigocladus_MNPM02\\\\.IS\\\\.tmp\\\\.7715\\\\.faa | parallel --gnu --plain -j 4 --block 234387 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Mastigocladus_MNPM02\\\\/\\\\/Mastigocladus_MNPM02\\\\.IS\\\\.tmp\\\\.7715\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:57:40] This is prokka 1.14.6\\n[10:57:40] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:57:40] Homepage is https://github.com/tseemann/prokka\\n[10:57:40] Local time is Mon Feb 21 10:57:40 2022\\n[10:57:40] You are tiagoferreiraleao\\n[10:57:40] Operating system is darwin\\n[10:57:40] You have BioPerl 1.007002\\n[10:57:40] System has 4 cores.\\n[10:57:40] Option --cpu asked for 12 cores, but system only has 4\\n[10:57:40] Will use maximum of 4 cores.\\n[10:57:40] Annotating as >>> Bacteria <<<\\n[10:57:40] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHH01.fasta' contents.\\n[10:57:40] Setting --locustag HIMFHEKA from MD5 126f1e4a4415e2f3e4ff00d6eb1b335d\\n[10:57:40] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHH01/\\n[10:57:40] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHH01\\\\/\\n[10:57:40] Using filename prefix: Nostoc_LAHH01.XXX\\n[10:57:40] Setting HMMER_NCPU=1\\n[10:57:40] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHH01//Nostoc_LAHH01.log\\n[10:57:40] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHH01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_LAHH01/ --prefix Nostoc_LAHH01 --cpus 12\\n[10:57:40] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:57:40] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:57:40] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:57:40] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:57:40] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:57:40] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:57:40] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:57:40] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:57:40] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:57:40] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:57:40] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:57:40] Looking for 'egrep' - found /usr/bin/egrep\\n[10:57:40] Looking for 'find' - found /usr/bin/find\\n[10:57:40] Looking for 'grep' - found /usr/bin/grep\\n[10:57:40] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:57:40] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:57:40] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:57:40] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:57:40] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:57:40] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:57:41] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:57:41] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:57:41] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:57:41] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:57:41] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:57:41] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:57:41] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:57:41] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:57:41] Looking for 'sed' - found /usr/bin/sed\\n[10:57:41] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:57:41] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:57:41] Using genetic code table 11.\\n[10:57:41] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_LAHH01.fasta\\n[10:57:42] Wrote 789 contigs totalling 9597019 bp.\\n[10:57:42] Predicting tRNAs and tmRNAs\\n[10:57:42] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHH01\\\\/\\\\/Nostoc_LAHH01\\\\.fna\\n[10:57:45] 1 tRNA-Ala c[53644,53716] 34 (tgc)\\n[10:57:45] 1 tRNA-Ser [44721,44807] 36 (cga)\\n[10:57:45] 1 tRNA-Asn c[53260,53333] 34 (gtt)\\n[10:57:45] 1 tRNA-Asp [27285,27362] 38 (gtc)\\n[10:57:45] 2 tRNA-Glu [27365,27441] 35 (ctc)\\n[10:57:45] 3 tRNA-Cys [27446,27521] 34 (gca)\\n[10:57:45] 4 tRNA-Met [27527,27601] 33 (cat)\\n[10:57:45] 5 tRNA-Trp [27604,27676] 34 (cca)\\n[10:57:45] 6 tRNA-Pyl [27701,27772] 33 (cta)\\n[10:57:45] 7 tRNA-Glu [27803,27878] 34 (ttc)\\n[10:57:45] 8 tRNA-Tyr [27882,27965] 36 (gta)\\n[10:57:45] 9 tRNA-Ser [27969,28058] 33 (gct)\\n[10:57:45] 10 tRNA-Arg [28063,28138] 35 (acg)\\n[10:57:45] 11 tRNA-Phe [28143,28217] 35 (gaa)\\n[10:57:45] 12 tRNA-Asn [28226,28300] 33 (gtt)\\n[10:57:45] 13 tRNA-Gln [28306,28381] 35 (ttg)\\n[10:57:45] 14 tRNA-Leu [28389,28466] 35 (caa)\\n[10:57:45] 15 tRNA-Pro [28471,28547] 35 (tgg)\\n[10:57:45] 16 tRNA-Leu [28551,28625] 35 (tag)\\n[10:57:45] 17 tRNA-Leu [28645,28722] 35 (gag)\\n[10:57:45] 18 tRNA-Lys [28726,28802] 34 (ttt)\\n[10:57:45] 19 tRNA-Lys [28808,28882] 33 (ctt)\\n[10:57:45] 20 tRNA-Leu [28888,28963] 36 (taa)\\n[10:57:45] 21 tRNA-Cys [28973,29050] 34 (gca)\\n[10:57:45] 22 tRNA-Thr [29058,29130] 34 (tgt)\\n[10:57:45] 23 tRNA-Gly [29136,29206] 33 (tcc)\\n[10:57:45] 24 tRNA-Val [29212,29287] 34 (tac)\\n[10:57:45] 25 tRNA-Arg [29294,29368] 35 (tct)\\n[10:57:45] 26 tRNA-Ala [29373,29445] 34 (tgc)\\n[10:57:45] 27 tRNA-His [29453,29529] 35 (gtg)\\n[10:57:45] 28 tRNA-Ser [29540,29624] 34 (tga)\\n[10:57:45] 1 tRNA-His [72462,72535] 34 (gtg)\\n[10:57:45] 1 tRNA-Pro c[43239,43312] 35 (ggg)\\n[10:57:45] 1 tRNA-Ile c[13306,13381] 34 (tat)\\n[10:57:45] 2 tRNA-Met c[14987,15062] 34 (cat)\\n[10:57:45] 3 tRNA-Met c[15064,15139] 34 (cat)\\n[10:57:45] 1 tRNA-Gly c[28539,28610] 33 (gcc)\\n[10:57:45] 2 tRNA-Met c[35420,35493] 33 (cat)\\n[10:57:45] 1 tRNA-Pro c[11698,11771] 35 (tgg)\\n[10:57:45] 2 tRNA-Ser c[20453,20539] 36 (tga)\\n[10:57:45] 3 tRNA-Tyr [22990,23074] 35 (gta)\\n[10:57:45] 4 tRNA-Thr [23150,23221] 33 (ggt)\\n[10:57:45] 1 tRNA-Lys c[1179,1249] 33 (ttt)\\n[10:57:45] 2 tRNA-Ser c[1508,1591] 34 (tga)\\n[10:57:45] 3 tRNA-His c[1596,1671] 35 (gtg)\\n[10:57:45] 4 tRNA-Ala c[1677,1751] 35 (tgc)\\n[10:57:45] 5 tRNA-Arg c[1843,1918] 35 (tct)\\n[10:57:45] 6 tRNA-Val c[1925,2000] 34 (tac)\\n[10:57:45] 7 tRNA-Thr c[2082,2154] 34 (tgt)\\n[10:57:45] 8 tRNA-Leu c[2449,2523] 35 (tag)\\n[10:57:45] 9 tRNA-Leu c[2635,2711] 34 (gag)\\n[10:57:45] 10 tRNA-Pro c[2819,2892] 35 (tgg)\\n[10:57:45] 11 tRNA-Leu c[2894,2971] 35 (taa)\\n[10:57:45] 12 tRNA-Leu c[2992,3069] 36 (caa)\\n[10:57:45] 13 tRNA-Gln c[3095,3170] 35 (ttg)\\n[10:57:45] 14 tRNA-Gln c[4045,4119] 35 (ctg)\\n[10:57:45] 15 tRNA-Asn c[4130,4204] 33 (gtt)\\n[10:57:45] 16 tRNA-Phe c[4210,4285] 36 (gaa)\\n[10:57:45] 17 tRNA-Arg c[4373,4448] 35 (acg)\\n[10:57:45] 18 tRNA-Ser c[5210,5296] 35 (gct)\\n[10:57:45] 19 tRNA-Tyr c[5300,5381] 35 (gta)\\n[10:57:45] 20 tRNA-Glu c[5392,5466] 35 (ttc)\\n[10:57:45] tRNA c[5554,5628] is a pseudo/wacky gene - skipping.\\n[10:57:45] 22 tRNA-Met c[5631,5704] 32 (cat)\\n[10:57:45] 23 tRNA-Glu c[5814,5890] 35 (ctc)\\n[10:57:45] 24 tRNA-Asp c[5894,5969] 36 (gtc)\\n[10:57:45] 1 tRNA-Leu c[25647,25730] 36 (tag)\\n[10:57:45] 1 tRNA-Ala c[790,862] 34 (ggc)\\n[10:57:45] 1 tRNA-Thr [10198,10269] 33 (cgt)\\n[10:57:45] 1 tRNA-Cys c[7306,7377] 33 (gca)\\n[10:57:45] 1 tRNA-Ile [27820,27895] 35 (gat)\\n[10:57:45] 2 tRNA-Ala [27979,28053] 34 (tgc)\\n[10:57:45] 1 tmRNA [18319,18706] 101,151 ANNIVKFARREALVAA*\\n[10:57:45] 1 tRNA-Ser [4761,4848] 35 (gga)\\n[10:57:45] 2 tRNA-Gly c[12993,13066] 34 (ccc)\\n[10:57:45] 1 tRNA-Met c[16545,16619] 35 (cat)\\n[10:57:45] 1 tRNA-Glu [22542,22615] 35 (ttc)\\n[10:57:45] 1 tRNA-Lys c[3689,3761] 33 (ctt)\\n[10:57:45] 1 tRNA-Thr [2122,2194] 33 (tgt)\\n[10:57:45] 1 tRNA-Leu [30874,30958] 35 (caa)\\n[10:57:45] 2 tRNA-Gln [35712,35785] 34 (ttg)\\n[10:57:45] 1 tRNA-Leu c[664,738] 35 (caa)\\n[10:57:45] 2 tRNA-Thr c[2129,2203] 33 (agt)\\n[10:57:45] 1 tRNA-Trp c[11586,11659] 34 (cca)\\n[10:57:45] 1 tRNA-Ala [22871,22957] 29 (cgc)\\n[10:57:45] 1 tRNA-Asp [241,314] 35 (gtc)\\n[10:57:45] 1 tRNA-Ala [18353,18425] 34 (cgc)\\n[10:57:45] 1 tRNA-Val c[14177,14249] 34 (tac)\\n[10:57:45] 1 tRNA-Leu c[3508,3588] 35 (gag)\\n[10:57:45] 1 tRNA-Val [9420,9493] 35 (gac)\\n[10:57:45] 1 tRNA-Arg c[3738,3812] 35 (tct)\\n[10:57:45] 1 tRNA-Gly [5763,5833] 33 (tcc)\\n[10:57:45] 1 tRNA-Leu [7210,7291] 35 (cag)\\n[10:57:45] 1 tRNA-Ser c[7733,7826] 36 (gct)\\n[10:57:45] 1 tRNA-Arg c[5017,5089] 34 (ccg)\\n[10:57:45] 1 tRNA-Phe c[193,266] 34 (gaa)\\n[10:57:46] 1 tRNA-Lys [2648,2719] 33 (ttt)\\n[10:57:46] 1 tRNA-Pro [4920,4993] 35 (cgg)\\n[10:57:46] 1 tRNA-Arg [928,1001] 35 (acg)\\n[10:57:46] 1 tRNA-Ala c[2414,2488] 35 (tgc)\\n[10:57:46] 2 tRNA-Ile c[2595,2669] 35 (gat)\\n[10:57:46] 1 tRNA-Met [2667,2741] 35 (cat)\\n[10:57:46] 1 tRNA-Arg c[1850,1922] 34 (cct)\\n[10:57:46] 1 tRNA-Asp c[85,161] 35 (gtc)\\n[10:57:46] 1 tRNA-Trp [5,76] 33 (cca)\\n[10:57:46] 1 tRNA-Arg c[34,108] 35 (acg)\\n[10:57:46] Found 105 tRNAs\\n[10:57:46] Predicting Ribosomal RNAs\\n[10:57:46] Running Barrnap with 4 threads\\n[10:57:48] 1 LAHH01000071.1 26213 16S ribosomal RNA\\n[10:57:48] 2 LAHH01000071.1 28226 23S ribosomal RNA\\n[10:57:48] 3 LAHH01000440.1 65 23S ribosomal RNA (partial)\\n[10:57:48] 4 LAHH01000440.1 2849 16S ribosomal RNA\\n[10:57:48] 5 LAHH01000580.1 2335 5S ribosomal RNA\\n[10:57:48] Found 5 rRNAs\\n[10:57:48] Skipping ncRNA search, enable with --rfam if desired.\\n[10:57:48] Total of 109 tRNA + rRNA features\\n[10:57:48] Searching for CRISPR repeats\\n[10:57:49] CRISPR1 LAHH01000007.1 19302 with 7 spacers\\n[10:57:49] CRISPR2 LAHH01000017.1 19368 with 15 spacers\\n[10:57:49] CRISPR3 LAHH01000046.1 31704 with 5 spacers\\n[10:57:49] CRISPR4 LAHH01000046.1 32039 with 10 spacers\\n[10:57:49] CRISPR5 LAHH01000069.1 1017 with 20 spacers\\n[10:57:49] CRISPR6 LAHH01000082.1 27424 with 12 spacers\\n[10:57:49] CRISPR7 LAHH01000084.1 6229 with 10 spacers\\n[10:57:49] CRISPR8 LAHH01000116.1 9953 with 24 spacers\\n[10:57:49] CRISPR9 LAHH01000192.1 12156 with 7 spacers\\n[10:57:49] CRISPR10 LAHH01000217.1 58 with 16 spacers\\n[10:57:49] CRISPR11 LAHH01000217.1 12613 with 17 spacers\\n[10:57:49] CRISPR12 LAHH01000231.1 8691 with 18 spacers\\n[10:57:49] CRISPR13 LAHH01000246.1 40 with 3 spacers\\n[10:57:49] CRISPR14 LAHH01000314.1 5057 with 6 spacers\\n[10:57:49] CRISPR15 LAHH01000366.1 37 with 28 spacers\\n[10:57:49] CRISPR16 LAHH01000407.1 3655 with 23 spacers\\n[10:57:49] CRISPR17 LAHH01000479.1 8 with 16 spacers\\n[10:57:49] Found 17 CRISPRs\\n[10:57:49] Predicting coding sequences\\n[10:57:49] Contigs total 9597019 bp, so using single mode\\n[10:57:49] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHH01\\\\/\\\\/Nostoc_LAHH01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:57:59] Excluding CDS which overlaps existing RNA (tRNA) at LAHH01000020.1:72383..72526 on - strand\\n[10:58:00] Excluding CDS which overlaps existing RNA (repeat_region) at LAHH01000082.1:28140..28256 on + strand\\n[10:58:00] Excluding CDS which overlaps existing RNA (repeat_region) at LAHH01000084.1:6560..6712 on - strand\\n[10:58:01] Excluding CDS which overlaps existing RNA (tRNA) at LAHH01000145.1:22052..23422 on + strand\\n[10:58:01] Excluding CDS which overlaps existing RNA (repeat_region) at LAHH01000192.1:12184..12309 on - strand\\n[10:58:02] Excluding CDS which overlaps existing RNA (repeat_region) at LAHH01000217.1:12655..12819 on - strand\\n[10:58:02] Excluding CDS which overlaps existing RNA (repeat_region) at LAHH01000314.1:5449..6408 on - strand\\n[10:58:02] Excluding CDS which overlaps existing RNA (repeat_region) at LAHH01000407.1:3467..3745 on + strand\\n[10:58:02] Excluding CDS which overlaps existing RNA (rRNA) at LAHH01000440.1:1191..1319 on + strand\\n[10:58:02] Excluding CDS which overlaps existing RNA (rRNA) at LAHH01000440.1:1357..1761 on + strand\\n[10:58:02] Excluding CDS which overlaps existing RNA (rRNA) at LAHH01000440.1:2036..2182 on - strand\\n[10:58:03] Excluding CDS which overlaps existing RNA (tRNA) at LAHH01000664.1:49..291 on + strand\\n[10:58:03] Found 8068 CDS\\n[10:58:03] Connecting features back to sequences\\n[10:58:03] Not using genus-specific database. Try --usegenus to enable it.\\n[10:58:03] Annotating CDS, please be patient.\\n[10:58:03] Will use 4 CPUs for similarity searching.\\n[10:58:05] There are still 8068 unannotated CDS left (started with 8068)\\n[10:58:05] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:58:05] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHH01\\\\/\\\\/Nostoc_LAHH01\\\\.IS\\\\.tmp\\\\.7827\\\\.faa | parallel --gnu --plain -j 4 --block 316107 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHH01\\\\/\\\\/Nostoc_LAHH01\\\\.IS\\\\.tmp\\\\.7827\\\\.blast 2> /dev/null\\n[10:58:28] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHH01\\\\/\\\\/Nostoc_LAHH01\\\\.IS\\\\.tmp\\\\.7827\\\\.faa | parallel --gnu --plain -j 4 --block 316107 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_LAHH01\\\\/\\\\/Nostoc_LAHH01\\\\.IS\\\\.tmp\\\\.7827\\\\.blast 2> /dev/null\\n\"\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b\"[10:58:28] This is prokka 1.14.6\\n[10:58:28] Written by Torsten Seemann <torsten.seemann@gmail.com>\\n[10:58:28] Homepage is https://github.com/tseemann/prokka\\n[10:58:28] Local time is Mon Feb 21 10:58:28 2022\\n[10:58:28] You are tiagoferreiraleao\\n[10:58:28] Operating system is darwin\\n[10:58:28] You have BioPerl 1.007002\\n[10:58:28] System has 4 cores.\\n[10:58:28] Option --cpu asked for 12 cores, but system only has 4\\n[10:58:28] Will use maximum of 4 cores.\\n[10:58:28] Annotating as >>> Bacteria <<<\\n[10:58:28] Generating locus_tag from '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKL01.fasta' contents.\\n[10:58:28] Setting --locustag GPGBLMAP from MD5 090b56a9325beed67f4908813cc36b74\\n[10:58:28] Creating new output folder: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKL01/\\n[10:58:28] Running: mkdir -p \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKL01\\\\/\\n[10:58:28] Using filename prefix: Nostoc_JABXKL01.XXX\\n[10:58:28] Setting HMMER_NCPU=1\\n[10:58:28] Writing log to: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKL01//Nostoc_JABXKL01.log\\n[10:58:28] Command: /Users/tiagoferreiraleao/anaconda3/bin/prokka /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKL01.fasta --outdir /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/Nostoc_JABXKL01/ --prefix Nostoc_JABXKL01 --cpus 12\\n[10:58:28] Appending to PATH: /Users/tiagoferreiraleao/anaconda3/bin\\n[10:58:28] Looking for 'aragorn' - found /Users/tiagoferreiraleao/anaconda3/bin/aragorn\\n[10:58:28] Determined aragorn version is 001002 from 'ARAGORN v1.2.38 Dean Laslett'\\n[10:58:28] Looking for 'barrnap' - found /Users/tiagoferreiraleao/anaconda3/bin/barrnap\\n[10:58:28] Determined barrnap version is 000009 from 'barrnap 0.9'\\n[10:58:28] Looking for 'blastp' - found /Users/tiagoferreiraleao/anaconda3/bin/blastp\\n[10:58:28] Determined blastp version is 002009 from 'blastp: 2.9.0+'\\n[10:58:28] Looking for 'cmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/cmpress\\n[10:58:28] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:58:28] Looking for 'cmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/cmscan\\n[10:58:28] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\\n[10:58:28] Looking for 'egrep' - found /usr/bin/egrep\\n[10:58:28] Looking for 'find' - found /usr/bin/find\\n[10:58:28] Looking for 'grep' - found /usr/bin/grep\\n[10:58:28] Looking for 'hmmpress' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmpress\\n[10:58:28] Determined hmmpress version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:58:28] Looking for 'hmmscan' - found /Users/tiagoferreiraleao/anaconda3/bin/hmmscan\\n[10:58:28] Determined hmmscan version is 003003 from '# HMMER 3.3 (Nov 2019); http://hmmer.org/'\\n[10:58:28] Looking for 'java' - found /Users/tiagoferreiraleao/anaconda3/bin/java\\n[10:58:28] Looking for 'makeblastdb' - found /Users/tiagoferreiraleao/anaconda3/bin/makeblastdb\\n[10:58:29] Determined makeblastdb version is 002009 from 'makeblastdb: 2.9.0+'\\n[10:58:29] Looking for 'minced' - found /Users/tiagoferreiraleao/anaconda3/bin/minced\\n[10:58:29] Determined minced version is 004002 from 'minced 0.4.2'\\n[10:58:29] Looking for 'parallel' - found /Users/tiagoferreiraleao/anaconda3/bin/parallel\\n[10:58:29] Determined parallel version is 20211222 from 'GNU parallel 20211222'\\n[10:58:29] Looking for 'prodigal' - found /Users/tiagoferreiraleao/anaconda3/bin/prodigal\\n[10:58:29] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\\n[10:58:29] Looking for 'prokka-genbank_to_fasta_db' - found /Users/tiagoferreiraleao/anaconda3/bin/prokka-genbank_to_fasta_db\\n[10:58:29] Looking for 'sed' - found /usr/bin/sed\\n[10:58:29] Looking for 'tbl2asn' - found /Users/tiagoferreiraleao/anaconda3/bin/tbl2asn\\n[10:58:29] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\\n[10:58:29] Using genetic code table 11.\\n[10:58:29] Loading and checking input file: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/Nostoc_JABXKL01.fasta\\n[10:58:29] Wrote 110 contigs totalling 8088017 bp.\\n[10:58:29] Predicting tRNAs and tmRNAs\\n[10:58:29] Running: aragorn -l -gc11  -w \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKL01\\\\/\\\\/Nostoc_JABXKL01\\\\.fna\\n[10:58:32] 1 tRNA-Gly c[59136,59206] 33 (tcc)\\n[10:58:32] 1 tRNA-Met c[30265,30336] 32 (cat)\\n[10:58:32] 2 tRNA-Met c[30337,30412] 34 (cat)\\n[10:58:32] 3 tRNA-Gly c[99024,99095] 33 (ccc)\\n[10:58:32] 1 tRNA-Met c[207710,207786] 35 (cat)\\n[10:58:32] 1 tRNA-Leu c[712,795] 36 (tag)\\n[10:58:32] 2 tRNA-Pro [73125,73198] 35 (cgg)\\n[10:58:32] 1 tRNA-Arg [53082,53156] 35 (tct)\\n[10:58:32] 2 tRNA-Thr c[54836,54907] 33 (ggt)\\n[10:58:32] 3 tRNA-Tyr c[54995,55080] 35 (gta)\\n[10:58:32] 4 tRNA-Ser [57380,57466] 36 (tga)\\n[10:58:32] 1 tRNA-Arg c[14337,14410] 35 (acg)\\n[10:58:32] 1 tRNA-Lys c[73471,73542] 33 (ctt)\\n[10:58:32] 1 tRNA-Pro c[81824,81897] 35 (tgg)\\n[10:58:32] 2 tRNA-Leu [255922,256006] 35 (caa)\\n[10:58:32] 1 tRNA-Pro c[59165,59238] 35 (ggg)\\n[10:58:32] 1 tRNA-Gln c[40977,41050] 34 (ttg)\\n[10:58:32] 1 tRNA-Thr c[13861,13932] 33 (tgt)\\n[10:58:32] 2 tRNA-Asn c[97293,97364] 33 (gtt)\\n[10:58:32] 1 tRNA-Ser [22820,22906] 36 (gga)\\n[10:58:32] 1 tRNA-Asp c[226427,226500] 35 (gtc)\\n[10:58:32] 1 tRNA-Leu c[7549,7630] 35 (cag)\\n[10:58:32] 1 tRNA-Ser c[18616,18702] 36 (cga)\\n[10:58:32] 1 tRNA-Gly [35751,35822] 33 (gcc)\\n[10:58:32] 1 tRNA-Val c[28129,28201] 33 (cac)\\n[10:58:32] 2 tRNA-Leu c[82372,82467] 36 (caa)\\n[10:58:32] 3 tRNA-Leu c[82435,82531] 37 (caa)\\n[10:58:32] 1 tRNA-Trp c[35445,35518] 34 (cca)\\n[10:58:32] 1 tRNA-Ala c[34207,34279] 34 (cgc)\\n[10:58:32] 1 tRNA-Arg c[11845,11917] 34 (ccg)\\n[10:58:32] 1 tRNA-Phe c[59936,60009] 34 (gaa)\\n[10:58:32] 2 tmRNA [88434,88821] 101,151 ANNIVSFARKGALAAA*\\n[10:58:32] 1 tRNA-Phe c[128623,128695] 33 (gaa)\\n[10:58:32] 2 tRNA-Pyl c[130244,130316] 34 (cta)\\n[10:58:32] 1 tRNA-Leu [38622,38719] 37 (caa)\\n[10:58:32] 1 tRNA-Ala [114122,114196] 35 (tgc)\\n[10:58:32] 2 tRNA-Ala c[118097,118171] 35 (ggc)\\n[10:58:32] 1 tRNA-Val [4838,4911] 35 (gac)\\n[10:58:32] 1 tRNA-Glu [92925,92998] 35 (ttc)\\n[10:58:32] 2 tRNA-Asp [99782,99857] 36 (gtc)\\n[10:58:32] 3 tRNA-Glu [99861,99937] 35 (ctc)\\n[10:58:32] 4 tRNA-Cys [99941,100014] 35 (gca)\\n[10:58:32] tRNA [100095,100169] is a pseudo/wacky gene - skipping.\\n[10:58:32] 6 tRNA-Glu [100175,100250] 35 (ttc)\\n[10:58:32] 7 tRNA-Tyr [100256,100339] 37 (gta)\\n[10:58:32] 8 tRNA-Ser [100343,100434] 35 (gct)\\n[10:58:32] 9 tRNA-Arg [100436,100512] 36 (acg)\\n[10:58:32] 10 tRNA-Phe [100593,100668] 36 (gaa)\\n[10:58:32] 11 tRNA-Asn [100674,100748] 33 (gtt)\\n[10:58:32] 12 tRNA-Gln [100753,100827] 35 (ctg)\\n[10:58:32] 13 tRNA-Gln [100830,100905] 34 (ttg)\\n[10:58:32] 14 tRNA-Leu [100911,100987] 36 (caa)\\n[10:58:32] 15 tRNA-Leu [100995,101072] 35 (taa)\\n[10:58:32] 16 tRNA-Pro [101074,101146] 34 (tgg)\\n[10:58:32] 17 tRNA-Leu [101172,101246] 35 (tag)\\n[10:58:32] tRNA [101256,101333] is a pseudo/wacky gene - skipping.\\n[10:58:32] 19 tRNA-Thr [101342,101414] 34 (tgt)\\n[10:58:32] 20 tRNA-Gly [101419,101490] 33 (tcc)\\n[10:58:32] 21 tRNA-Val [101598,101673] 34 (tac)\\n[10:58:32] 22 tRNA-Arg [101680,101755] 35 (tct)\\n[10:58:32] 23 tRNA-Ala [101860,101934] 35 (tgc)\\n[10:58:32] 24 tRNA-His [101940,102015] 35 (gtg)\\n[10:58:32] 25 tRNA-Ser [102021,102105] 34 (tga)\\n[10:58:32] 26 tRNA-Lys [102262,102333] 33 (ttt)\\n[10:58:32] 27 tRNA-Ser [106458,106549] 35 (gct)\\n[10:58:32] 1 tRNA-Thr [26889,26960] 33 (cgt)\\n[10:58:32] 1 tRNA-Lys [30190,30261] 33 (ttt)\\n[10:58:32] 2 tRNA-Arg c[36442,36517] 36 (cct)\\n[10:58:32] 1 tRNA-Met [41370,41444] 35 (cat)\\n[10:58:32] 1 tRNA-Met [86083,86156] 33 (cat)\\n[10:58:32] 1 tRNA-Leu [15681,15763] 36 (gag)\\n[10:58:32] 1 tRNA-Cys c[74410,74480] 33 (gca)\\n[10:58:32] 1 tRNA-His c[14108,14181] 34 (gtg)\\n[10:58:32] Found 71 tRNAs\\n[10:58:32] Predicting Ribosomal RNAs\\n[10:58:32] Running Barrnap with 4 threads\\n[10:58:34] Found 0 rRNAs\\n[10:58:34] Skipping ncRNA search, enable with --rfam if desired.\\n[10:58:34] Total of 70 tRNA + rRNA features\\n[10:58:34] Searching for CRISPR repeats\\n[10:58:34] CRISPR1 JABXKL010000061.1 33667 with 4 spacers\\n[10:58:35] CRISPR2 JABXKL010000085.1 10676 with 4 spacers\\n[10:58:35] CRISPR3 JABXKL010000102.1 3714 with 10 spacers\\n[10:58:35] CRISPR4 JABXKL010000102.1 12628 with 9 spacers\\n[10:58:35] CRISPR5 JABXKL010000102.1 16291 with 16 spacers\\n[10:58:35] Found 5 CRISPRs\\n[10:58:35] Predicting coding sequences\\n[10:58:35] Contigs total 8088017 bp, so using single mode\\n[10:58:35] Running: prodigal -i \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKL01\\\\/\\\\/Nostoc_JABXKL01\\\\.fna -c -m -g 11 -p single -f sco -q\\n[10:58:43] Excluding CDS which overlaps existing RNA (tRNA) at JABXKL010000021.1:40959..41192 on + strand\\n[10:58:45] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKL010000061.1:33657..33872 on - strand\\n[10:58:46] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKL010000085.1:10640..10894 on + strand\\n[10:58:47] Excluding CDS which overlaps existing RNA (tRNA) at JABXKL010000091.1:100172..100417 on - strand\\n[10:58:47] Excluding CDS which overlaps existing RNA (tRNA) at JABXKL010000091.1:100688..100921 on + strand\\n[10:58:47] Excluding CDS which overlaps existing RNA (repeat_region) at JABXKL010000102.1:4142..4357 on - strand\\n[10:58:47] Found 7036 CDS\\n[10:58:47] Connecting features back to sequences\\n[10:58:47] Not using genus-specific database. Try --usegenus to enable it.\\n[10:58:47] Annotating CDS, please be patient.\\n[10:58:47] Will use 4 CPUs for similarity searching.\\n[10:58:49] There are still 7036 unannotated CDS left (started with 7036)\\n[10:58:49] Will use blast to search against /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS with 4 CPUs\\n[10:58:49] Running: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKL01\\\\/\\\\/Nostoc_JABXKL01\\\\.IS\\\\.tmp\\\\.7939\\\\.faa | parallel --gnu --plain -j 4 --block 274596 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKL01\\\\/\\\\/Nostoc_JABXKL01\\\\.IS\\\\.tmp\\\\.7939\\\\.blast 2> /dev/null\\n[10:59:04] Could not run command: cat \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKL01\\\\/\\\\/Nostoc_JABXKL01\\\\.IS\\\\.tmp\\\\.7939\\\\.faa | parallel --gnu --plain -j 4 --block 274596 --recstart '>' --pipe blastp -query - -db /Users/tiagoferreiraleao/anaconda3/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \\\\/Users\\\\/tiagoferreiraleao\\\\/Dropbox\\\\/tiago\\\\-NAS\\\\/schneider\\\\/prokka\\\\/Nostoc_JABXKL01\\\\/\\\\/Nostoc_JABXKL01\\\\.IS\\\\.tmp\\\\.7939\\\\.blast 2> /dev/null\\n\"\n"
     ]
    }
   ],
   "source": [
    "def run_prokka(input_fasta,prokka_folder):\n",
    "    fasta_glob = glob.glob(\"%s*.fasta\"%input_fasta)\n",
    "    for fasta_path in fasta_glob:\n",
    "        strain = os.path.basename(fasta_path).split('.')[0]\n",
    "        output_folder = '%s%s/'%(prokka_folder,strain)\n",
    "        prokka_cmd = \"prokka %s --outdir %s --prefix %s --cpus 12\"%(fasta_path,output_folder,strain)\n",
    "        p = subprocess.Popen(prokka_cmd, shell=True, stdin=subprocess.PIPE, stdout=subprocess.PIPE,\n",
    "                             stderr=subprocess.STDOUT, close_fds=True,cwd=\"./\")\n",
    "        output = p.stdout.read()\n",
    "        print(output)\n",
    "        subprocess.call('mv %s%s/%s.*.faa %s%s/%s.faa'%\n",
    "                        (prokka_folder,strain,strain,prokka_folder,strain,strain),shell=True)\n",
    "    \n",
    "run_prokka(\"/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/\",\n",
    "           \"/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Identify housekeeping genes in prokka annotation for final scaffolds"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "#conda install -c bioconda diamond"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir ./pnas-phylogenomics-top_100/diamond"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\r\n",
      "Licensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\r\n",
      "Check http://github.com/bbuchfink/diamond for updates.\r\n",
      "\r\n",
      "#CPU threads: 8\r\n",
      "Scoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\r\n",
      "Database file: ./pnas-phylogenomics-top_100/PAL_core_genes.faa\r\n",
      "Opening the database file...  [0.001633s]\r\n",
      "Loading sequences...  [0.00035s]\r\n",
      "Masking sequences...  [0.000751s]\r\n",
      "Writing sequences...  [0.000106s]\r\n",
      "Hashing sequences...  [8e-06s]\r\n",
      "Loading sequences...  [9e-06s]\r\n",
      "Writing trailer...  [4e-06s]\r\n",
      "Closing the input file...  [2.1e-05s]\r\n",
      "Closing the database file...  [0.00102s]\r\n",
      "Database hash = cf7016f4bf64255b40ca9dd62558c2d2\r\n",
      "Processed 21 sequences, 3334 letters.\r\n",
      "Total time = 0.003997s\r\n"
     ]
    }
   ],
   "source": [
    "!diamond makedb --in ./pnas-phylogenomics-top_100/PAL_core_genes.faa -d ./pnas-phylogenomics-top_100/diamond/PAL_hkg"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000416s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000312s]\\nOpening the output file...  [6.1e-05s]\\nLoading query sequences...  [0.013829s]\\nMasking queries...  [0.089515s]\\nBuilding query seed set...  [0.018152s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.014318s]\\nAllocating buffers...  [2.3e-05s]\\nLoading reference sequences...  [0.000439s]\\nBuilding reference histograms...  [0.000306s]\\nAllocating buffers...  [3.2e-05s]\\nInitializing temporary storage...  [0.044864s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.000996s]\\nBuilding query index...  [0.011088s]\\nBuilding seed filter...  [0.000862s]\\nSearching alignments...  [0.001061s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000423s]\\nBuilding query index...  [0.01443s]\\nBuilding seed filter...  [0.000953s]\\nSearching alignments...  [0.001039s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000407s]\\nBuilding query index...  [0.015401s]\\nBuilding seed filter...  [0.000969s]\\nSearching alignments...  [0.001047s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000435s]\\nBuilding query index...  [0.013206s]\\nBuilding seed filter...  [0.000928s]\\nSearching alignments...  [0.001046s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.00038s]\\nBuilding query index...  [0.013619s]\\nBuilding seed filter...  [0.00085s]\\nSearching alignments...  [0.00091s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000484s]\\nBuilding query index...  [0.010697s]\\nBuilding seed filter...  [0.001s]\\nSearching alignments...  [0.001033s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000452s]\\nBuilding query index...  [0.010476s]\\nBuilding seed filter...  [0.000821s]\\nSearching alignments...  [0.000865s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000435s]\\nBuilding query index...  [0.009296s]\\nBuilding seed filter...  [0.000867s]\\nSearching alignments...  [0.000868s]\\nDeallocating buffers...  [4e-06s]\\nComputing alignments...  [0.011083s]\\nDeallocating reference...  [4e-06s]\\nLoading reference sequences...  [1.5e-05s]\\nDeallocating buffers...  [0.000334s]\\nDeallocating queries...  [0.000155s]\\nLoading query sequences...  [1.1e-05s]\\nClosing the input file...  [4.2e-05s]\\nClosing the output file...  [0.002172s]\\nClosing the database file...  [3.6e-05s]\\nDeallocating taxonomy...  [5e-06s]\\nTotal time = 0.314811s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000153s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000781s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000151s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000423s]\\nOpening the output file...  [8.4e-05s]\\nLoading query sequences...  [0.020101s]\\nMasking queries...  [0.083307s]\\nBuilding query seed set...  [0.018953s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.013993s]\\nAllocating buffers...  [2.1e-05s]\\nLoading reference sequences...  [4.6e-05s]\\nBuilding reference histograms...  [0.000282s]\\nAllocating buffers...  [3.9e-05s]\\nInitializing temporary storage...  [0.015567s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.000946s]\\nBuilding query index...  [0.013192s]\\nBuilding seed filter...  [0.001592s]\\nSearching alignments...  [0.001139s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.00049s]\\nBuilding query index...  [0.019178s]\\nBuilding seed filter...  [0.001201s]\\nSearching alignments...  [0.001102s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000764s]\\nBuilding query index...  [0.017498s]\\nBuilding seed filter...  [0.001086s]\\nSearching alignments...  [0.00095s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000564s]\\nBuilding query index...  [0.015515s]\\nBuilding seed filter...  [0.001037s]\\nSearching alignments...  [0.00102s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000463s]\\nBuilding query index...  [0.010489s]\\nBuilding seed filter...  [0.000939s]\\nSearching alignments...  [0.000955s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000416s]\\nBuilding query index...  [0.011745s]\\nBuilding seed filter...  [0.001179s]\\nSearching alignments...  [0.000867s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000427s]\\nBuilding query index...  [0.011994s]\\nBuilding seed filter...  [0.00103s]\\nSearching alignments...  [0.000882s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.00037s]\\nBuilding query index...  [0.011431s]\\nBuilding seed filter...  [0.000955s]\\nSearching alignments...  [0.000826s]\\nDeallocating buffers...  [3e-06s]\\nComputing alignments...  [0.011168s]\\nDeallocating reference...  [5e-06s]\\nLoading reference sequences...  [2.7e-05s]\\nDeallocating buffers...  [0.000642s]\\nDeallocating queries...  [0.00043s]\\nLoading query sequences...  [1.4e-05s]\\nClosing the input file...  [3e-05s]\\nClosing the output file...  [0.000272s]\\nClosing the database file...  [2.7e-05s]\\nDeallocating taxonomy...  [1e-05s]\\nTotal time = 0.304069s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000194s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000983s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000192s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000395s]\\nOpening the output file...  [0.000103s]\\nLoading query sequences...  [0.01036s]\\nMasking queries...  [0.0603s]\\nBuilding query seed set...  [0.018555s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.008589s]\\nAllocating buffers...  [2.4e-05s]\\nLoading reference sequences...  [7e-05s]\\nBuilding reference histograms...  [0.0002s]\\nAllocating buffers...  [5.8e-05s]\\nInitializing temporary storage...  [0.016172s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.00094s]\\nBuilding query index...  [0.007679s]\\nBuilding seed filter...  [0.000788s]\\nSearching alignments...  [0.001005s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000463s]\\nBuilding query index...  [0.010612s]\\nBuilding seed filter...  [0.000799s]\\nSearching alignments...  [0.000911s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000402s]\\nBuilding query index...  [0.007743s]\\nBuilding seed filter...  [0.000739s]\\nSearching alignments...  [0.000821s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000416s]\\nBuilding query index...  [0.007104s]\\nBuilding seed filter...  [0.000727s]\\nSearching alignments...  [0.00086s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000389s]\\nBuilding query index...  [0.009163s]\\nBuilding seed filter...  [0.00108s]\\nSearching alignments...  [0.000921s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000397s]\\nBuilding query index...  [0.011494s]\\nBuilding seed filter...  [0.001007s]\\nSearching alignments...  [0.00096s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000434s]\\nBuilding query index...  [0.010346s]\\nBuilding seed filter...  [0.000716s]\\nSearching alignments...  [0.000897s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.00039s]\\nBuilding query index...  [0.006518s]\\nBuilding seed filter...  [0.000708s]\\nSearching alignments...  [0.000843s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.009051s]\\nDeallocating reference...  [5e-06s]\\nLoading reference sequences...  [1.5e-05s]\\nDeallocating buffers...  [1.3e-05s]\\nDeallocating queries...  [0.000168s]\\nLoading query sequences...  [1e-05s]\\nClosing the input file...  [4.3e-05s]\\nClosing the output file...  [0.000285s]\\nClosing the database file...  [4.5e-05s]\\nDeallocating taxonomy...  [3.3e-05s]\\nTotal time = 0.214399s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000185s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000891s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [8e-05s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000455s]\\nOpening the output file...  [6.1e-05s]\\nLoading query sequences...  [0.015417s]\\nMasking queries...  [0.067726s]\\nBuilding query seed set...  [0.017135s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.018127s]\\nAllocating buffers...  [3.7e-05s]\\nLoading reference sequences...  [5.1e-05s]\\nBuilding reference histograms...  [0.00027s]\\nAllocating buffers...  [2e-05s]\\nInitializing temporary storage...  [0.016862s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001025s]\\nBuilding query index...  [0.010157s]\\nBuilding seed filter...  [0.001182s]\\nSearching alignments...  [0.000968s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000436s]\\nBuilding query index...  [0.009822s]\\nBuilding seed filter...  [0.000944s]\\nSearching alignments...  [0.000893s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000451s]\\nBuilding query index...  [0.016401s]\\nBuilding seed filter...  [0.001373s]\\nSearching alignments...  [0.00085s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000375s]\\nBuilding query index...  [0.011894s]\\nBuilding seed filter...  [0.00139s]\\nSearching alignments...  [0.001099s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000506s]\\nBuilding query index...  [0.010825s]\\nBuilding seed filter...  [0.00084s]\\nSearching alignments...  [0.000915s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000499s]\\nBuilding query index...  [0.015088s]\\nBuilding seed filter...  [0.000838s]\\nSearching alignments...  [0.000965s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000551s]\\nBuilding query index...  [0.011601s]\\nBuilding seed filter...  [0.000886s]\\nSearching alignments...  [0.00085s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000415s]\\nBuilding query index...  [0.009161s]\\nBuilding seed filter...  [0.001081s]\\nSearching alignments...  [0.000914s]\\nDeallocating buffers...  [4e-06s]\\nComputing alignments...  [0.013936s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [2.2e-05s]\\nDeallocating buffers...  [0.000503s]\\nDeallocating queries...  [0.000433s]\\nLoading query sequences...  [1.4e-05s]\\nClosing the input file...  [1.9e-05s]\\nClosing the output file...  [0.000909s]\\nClosing the database file...  [1.8e-05s]\\nDeallocating taxonomy...  [3.6e-05s]\\nTotal time = 0.270002s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000177s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000694s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000156s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000227s]\\nOpening the output file...  [0.000131s]\\nLoading query sequences...  [0.01686s]\\nMasking queries...  [0.062702s]\\nBuilding query seed set...  [0.014947s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.010126s]\\nAllocating buffers...  [2.6e-05s]\\nLoading reference sequences...  [4.7e-05s]\\nBuilding reference histograms...  [0.000198s]\\nAllocating buffers...  [2e-05s]\\nInitializing temporary storage...  [0.015221s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001117s]\\nBuilding query index...  [0.009568s]\\nBuilding seed filter...  [0.000778s]\\nSearching alignments...  [0.001084s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000458s]\\nBuilding query index...  [0.008785s]\\nBuilding seed filter...  [0.000778s]\\nSearching alignments...  [0.000888s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000434s]\\nBuilding query index...  [0.008921s]\\nBuilding seed filter...  [0.000754s]\\nSearching alignments...  [0.000862s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000388s]\\nBuilding query index...  [0.008479s]\\nBuilding seed filter...  [0.000806s]\\nSearching alignments...  [0.000864s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000357s]\\nBuilding query index...  [0.008044s]\\nBuilding seed filter...  [0.000851s]\\nSearching alignments...  [0.001089s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000401s]\\nBuilding query index...  [0.012708s]\\nBuilding seed filter...  [0.000751s]\\nSearching alignments...  [0.000865s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000391s]\\nBuilding query index...  [0.009935s]\\nBuilding seed filter...  [0.000975s]\\nSearching alignments...  [0.001025s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000396s]\\nBuilding query index...  [0.010932s]\\nBuilding seed filter...  [0.000753s]\\nSearching alignments...  [0.000958s]\\nDeallocating buffers...  [4e-06s]\\nComputing alignments...  [0.010489s]\\nDeallocating reference...  [8e-06s]\\nLoading reference sequences...  [1.8e-05s]\\nDeallocating buffers...  [0.000415s]\\nDeallocating queries...  [0.000106s]\\nLoading query sequences...  [1.4e-05s]\\nClosing the input file...  [2e-05s]\\nClosing the output file...  [0.000282s]\\nClosing the database file...  [1.4e-05s]\\nDeallocating taxonomy...  [1.3e-05s]\\nTotal time = 0.228591s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000178s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.011287s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000229s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000425s]\\nOpening the output file...  [8.8e-05s]\\nLoading query sequences...  [0.014462s]\\nMasking queries...  [0.062249s]\\nBuilding query seed set...  [0.020012s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.012465s]\\nAllocating buffers...  [2.4e-05s]\\nLoading reference sequences...  [4.4e-05s]\\nBuilding reference histograms...  [0.000193s]\\nAllocating buffers...  [2.1e-05s]\\nInitializing temporary storage...  [0.018575s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001222s]\\nBuilding query index...  [0.01008s]\\nBuilding seed filter...  [0.001218s]\\nSearching alignments...  [0.001257s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000692s]\\nBuilding query index...  [0.01202s]\\nBuilding seed filter...  [0.001112s]\\nSearching alignments...  [0.001159s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000621s]\\nBuilding query index...  [0.012773s]\\nBuilding seed filter...  [0.000809s]\\nSearching alignments...  [0.001016s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000401s]\\nBuilding query index...  [0.009858s]\\nBuilding seed filter...  [0.000769s]\\nSearching alignments...  [0.00094s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000377s]\\nBuilding query index...  [0.010375s]\\nBuilding seed filter...  [0.000816s]\\nSearching alignments...  [0.000968s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000351s]\\nBuilding query index...  [0.011253s]\\nBuilding seed filter...  [0.001002s]\\nSearching alignments...  [0.000927s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000462s]\\nBuilding query index...  [0.01158s]\\nBuilding seed filter...  [0.001107s]\\nSearching alignments...  [0.001225s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000559s]\\nBuilding query index...  [0.010161s]\\nBuilding seed filter...  [0.000893s]\\nSearching alignments...  [0.00146s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.009597s]\\nDeallocating reference...  [5e-06s]\\nLoading reference sequences...  [2e-05s]\\nDeallocating buffers...  [6e-06s]\\nDeallocating queries...  [0.000201s]\\nLoading query sequences...  [1.3e-05s]\\nClosing the input file...  [1.7e-05s]\\nClosing the output file...  [0.000215s]\\nClosing the database file...  [2.6e-05s]\\nDeallocating taxonomy...  [2.5e-05s]\\nTotal time = 0.249684s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000262s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001364s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000177s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.00045s]\\nOpening the output file...  [0.000115s]\\nLoading query sequences...  [0.018785s]\\nMasking queries...  [0.102642s]\\nBuilding query seed set...  [0.024189s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.013448s]\\nAllocating buffers...  [2.2e-05s]\\nLoading reference sequences...  [5.7e-05s]\\nBuilding reference histograms...  [0.000187s]\\nAllocating buffers...  [1.5e-05s]\\nInitializing temporary storage...  [0.024586s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001239s]\\nBuilding query index...  [0.012563s]\\nBuilding seed filter...  [0.000896s]\\nSearching alignments...  [0.001026s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000392s]\\nBuilding query index...  [0.014239s]\\nBuilding seed filter...  [0.000946s]\\nSearching alignments...  [0.000911s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.0004s]\\nBuilding query index...  [0.012532s]\\nBuilding seed filter...  [0.000903s]\\nSearching alignments...  [0.000934s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000436s]\\nBuilding query index...  [0.009722s]\\nBuilding seed filter...  [0.00097s]\\nSearching alignments...  [0.000964s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000408s]\\nBuilding query index...  [0.012881s]\\nBuilding seed filter...  [0.000987s]\\nSearching alignments...  [0.001075s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000416s]\\nBuilding query index...  [0.010782s]\\nBuilding seed filter...  [0.000866s]\\nSearching alignments...  [0.000844s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000378s]\\nBuilding query index...  [0.010758s]\\nBuilding seed filter...  [0.00086s]\\nSearching alignments...  [0.000878s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000395s]\\nBuilding query index...  [0.009429s]\\nBuilding seed filter...  [0.000856s]\\nSearching alignments...  [0.000839s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.01183s]\\nDeallocating reference...  [5e-06s]\\nLoading reference sequences...  [1.4e-05s]\\nDeallocating buffers...  [0.000532s]\\nDeallocating queries...  [0.000271s]\\nLoading query sequences...  [2.1e-05s]\\nClosing the input file...  [4.2e-05s]\\nClosing the output file...  [0.000251s]\\nClosing the database file...  [2.1e-05s]\\nDeallocating taxonomy...  [1.4e-05s]\\nTotal time = 0.311031s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000163s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000917s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000171s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000414s]\\nOpening the output file...  [7e-05s]\\nLoading query sequences...  [0.014092s]\\nMasking queries...  [0.079779s]\\nBuilding query seed set...  [0.020718s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.014645s]\\nAllocating buffers...  [4e-05s]\\nLoading reference sequences...  [6e-05s]\\nBuilding reference histograms...  [0.000248s]\\nAllocating buffers...  [2.7e-05s]\\nInitializing temporary storage...  [0.016719s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001712s]\\nBuilding query index...  [0.014058s]\\nBuilding seed filter...  [0.001062s]\\nSearching alignments...  [0.001098s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000401s]\\nBuilding query index...  [0.013197s]\\nBuilding seed filter...  [0.000939s]\\nSearching alignments...  [0.001039s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000439s]\\nBuilding query index...  [0.011983s]\\nBuilding seed filter...  [0.000925s]\\nSearching alignments...  [0.000995s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000399s]\\nBuilding query index...  [0.010303s]\\nBuilding seed filter...  [0.000945s]\\nSearching alignments...  [0.000894s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.00049s]\\nBuilding query index...  [0.01064s]\\nBuilding seed filter...  [0.001143s]\\nSearching alignments...  [0.001005s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.00042s]\\nBuilding query index...  [0.016167s]\\nBuilding seed filter...  [0.001159s]\\nSearching alignments...  [0.001059s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000553s]\\nBuilding query index...  [0.01129s]\\nBuilding seed filter...  [0.00106s]\\nSearching alignments...  [0.000935s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000433s]\\nBuilding query index...  [0.009977s]\\nBuilding seed filter...  [0.001087s]\\nSearching alignments...  [0.000922s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.010982s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [1.7e-05s]\\nDeallocating buffers...  [0.000414s]\\nDeallocating queries...  [0.000281s]\\nLoading query sequences...  [1.4e-05s]\\nClosing the input file...  [1.5e-05s]\\nClosing the output file...  [0.000229s]\\nClosing the database file...  [1.7e-05s]\\nDeallocating taxonomy...  [1.4e-05s]\\nTotal time = 0.278932s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000114s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000719s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000136s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000458s]\\nOpening the output file...  [8.2e-05s]\\nLoading query sequences...  [0.016786s]\\nMasking queries...  [0.072494s]\\nBuilding query seed set...  [0.018972s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.01201s]\\nAllocating buffers...  [2.9e-05s]\\nLoading reference sequences...  [5.3e-05s]\\nBuilding reference histograms...  [0.000218s]\\nAllocating buffers...  [2.2e-05s]\\nInitializing temporary storage...  [0.024391s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001005s]\\nBuilding query index...  [0.010508s]\\nBuilding seed filter...  [0.000874s]\\nSearching alignments...  [0.00093s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000384s]\\nBuilding query index...  [0.009853s]\\nBuilding seed filter...  [0.000965s]\\nSearching alignments...  [0.001042s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000417s]\\nBuilding query index...  [0.015626s]\\nBuilding seed filter...  [0.000799s]\\nSearching alignments...  [0.00084s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000403s]\\nBuilding query index...  [0.008652s]\\nBuilding seed filter...  [0.000793s]\\nSearching alignments...  [0.00089s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.00043s]\\nBuilding query index...  [0.010598s]\\nBuilding seed filter...  [0.001s]\\nSearching alignments...  [0.000901s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000403s]\\nBuilding query index...  [0.00961s]\\nBuilding seed filter...  [0.000831s]\\nSearching alignments...  [0.000946s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000407s]\\nBuilding query index...  [0.010263s]\\nBuilding seed filter...  [0.000763s]\\nSearching alignments...  [0.000874s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000428s]\\nBuilding query index...  [0.009704s]\\nBuilding seed filter...  [0.000866s]\\nSearching alignments...  [0.001051s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.018205s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [1.7e-05s]\\nDeallocating buffers...  [0.000462s]\\nDeallocating queries...  [0.000432s]\\nLoading query sequences...  [1.4e-05s]\\nClosing the input file...  [4.8e-05s]\\nClosing the output file...  [0.00024s]\\nClosing the database file...  [3.5e-05s]\\nDeallocating taxonomy...  [1.2e-05s]\\nTotal time = 0.269372s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000155s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000844s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000184s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000783s]\\nOpening the output file...  [0.000108s]\\nLoading query sequences...  [0.018481s]\\nMasking queries...  [0.0839s]\\nBuilding query seed set...  [0.024214s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.014593s]\\nAllocating buffers...  [4e-05s]\\nLoading reference sequences...  [4.1e-05s]\\nBuilding reference histograms...  [0.000187s]\\nAllocating buffers...  [1.5e-05s]\\nInitializing temporary storage...  [0.029871s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001091s]\\nBuilding query index...  [0.012882s]\\nBuilding seed filter...  [0.001245s]\\nSearching alignments...  [0.001058s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000556s]\\nBuilding query index...  [0.015832s]\\nBuilding seed filter...  [0.001355s]\\nSearching alignments...  [0.00111s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000616s]\\nBuilding query index...  [0.016246s]\\nBuilding seed filter...  [0.001055s]\\nSearching alignments...  [0.000845s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000409s]\\nBuilding query index...  [0.011138s]\\nBuilding seed filter...  [0.001041s]\\nSearching alignments...  [0.001019s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000508s]\\nBuilding query index...  [0.010374s]\\nBuilding seed filter...  [0.000877s]\\nSearching alignments...  [0.000883s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000391s]\\nBuilding query index...  [0.011515s]\\nBuilding seed filter...  [0.001199s]\\nSearching alignments...  [0.000843s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000394s]\\nBuilding query index...  [0.012168s]\\nBuilding seed filter...  [0.000902s]\\nSearching alignments...  [0.000885s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000379s]\\nBuilding query index...  [0.013137s]\\nBuilding seed filter...  [0.001066s]\\nSearching alignments...  [0.000961s]\\nDeallocating buffers...  [1.8e-05s]\\nComputing alignments...  [0.017288s]\\nDeallocating reference...  [7e-06s]\\nLoading reference sequences...  [1.8e-05s]\\nDeallocating buffers...  [0.000447s]\\nDeallocating queries...  [0.000573s]\\nLoading query sequences...  [3.1e-05s]\\nClosing the input file...  [2.6e-05s]\\nClosing the output file...  [0.000207s]\\nClosing the database file...  [1.5e-05s]\\nDeallocating taxonomy...  [1.7e-05s]\\nTotal time = 0.316396s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000119s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000811s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000174s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000408s]\\nOpening the output file...  [8.7e-05s]\\nLoading query sequences...  [0.011685s]\\nMasking queries...  [0.045877s]\\nBuilding query seed set...  [0.012901s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.007652s]\\nAllocating buffers...  [3e-05s]\\nLoading reference sequences...  [6.8e-05s]\\nBuilding reference histograms...  [0.000246s]\\nAllocating buffers...  [2.9e-05s]\\nInitializing temporary storage...  [0.015546s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001309s]\\nBuilding query index...  [0.0083s]\\nBuilding seed filter...  [0.000697s]\\nSearching alignments...  [0.000898s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000511s]\\nBuilding query index...  [0.00733s]\\nBuilding seed filter...  [0.000645s]\\nSearching alignments...  [0.000863s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000515s]\\nBuilding query index...  [0.007312s]\\nBuilding seed filter...  [0.000627s]\\nSearching alignments...  [0.000852s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000414s]\\nBuilding query index...  [0.007361s]\\nBuilding seed filter...  [0.000725s]\\nSearching alignments...  [0.000901s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.0005s]\\nBuilding query index...  [0.00844s]\\nBuilding seed filter...  [0.000993s]\\nSearching alignments...  [0.001033s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000452s]\\nBuilding query index...  [0.009098s]\\nBuilding seed filter...  [0.000761s]\\nSearching alignments...  [0.000832s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000431s]\\nBuilding query index...  [0.008684s]\\nBuilding seed filter...  [0.000694s]\\nSearching alignments...  [0.000858s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000416s]\\nBuilding query index...  [0.00689s]\\nBuilding seed filter...  [0.000811s]\\nSearching alignments...  [0.000918s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.012998s]\\nDeallocating reference...  [3.1e-05s]\\nLoading reference sequences...  [1.6e-05s]\\nDeallocating buffers...  [6e-06s]\\nDeallocating queries...  [0.000153s]\\nLoading query sequences...  [1.4e-05s]\\nClosing the input file...  [1.9e-05s]\\nClosing the output file...  [0.000538s]\\nClosing the database file...  [1.8e-05s]\\nDeallocating taxonomy...  [4e-06s]\\nTotal time = 0.191082s\\nReported 19 pairwise alignments, 19 HSPs.\\n19 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000187s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 19\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000843s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000133s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.00068s]\\nOpening the output file...  [0.000103s]\\nLoading query sequences...  [0.015097s]\\nMasking queries...  [0.071385s]\\nBuilding query seed set...  [0.019626s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.012377s]\\nAllocating buffers...  [3.3e-05s]\\nLoading reference sequences...  [5.3e-05s]\\nBuilding reference histograms...  [0.000253s]\\nAllocating buffers...  [2.1e-05s]\\nInitializing temporary storage...  [0.027587s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001164s]\\nBuilding query index...  [0.009589s]\\nBuilding seed filter...  [0.001203s]\\nSearching alignments...  [0.001101s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000536s]\\nBuilding query index...  [0.016426s]\\nBuilding seed filter...  [0.001047s]\\nSearching alignments...  [0.001076s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.00048s]\\nBuilding query index...  [0.014875s]\\nBuilding seed filter...  [0.001227s]\\nSearching alignments...  [0.000966s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000515s]\\nBuilding query index...  [0.009501s]\\nBuilding seed filter...  [0.00198s]\\nSearching alignments...  [0.001182s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000666s]\\nBuilding query index...  [0.009817s]\\nBuilding seed filter...  [0.00083s]\\nSearching alignments...  [0.000892s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000519s]\\nBuilding query index...  [0.013498s]\\nBuilding seed filter...  [0.000792s]\\nSearching alignments...  [0.000881s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000488s]\\nBuilding query index...  [0.011956s]\\nBuilding seed filter...  [0.000795s]\\nSearching alignments...  [0.00088s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000418s]\\nBuilding query index...  [0.010601s]\\nBuilding seed filter...  [0.001167s]\\nSearching alignments...  [0.001144s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.012137s]\\nDeallocating reference...  [2e-05s]\\nLoading reference sequences...  [3.1e-05s]\\nDeallocating buffers...  [0.000561s]\\nDeallocating queries...  [0.000418s]\\nLoading query sequences...  [2.1e-05s]\\nClosing the input file...  [1.3e-05s]\\nClosing the output file...  [0.000204s]\\nClosing the database file...  [1.6e-05s]\\nDeallocating taxonomy...  [2.1e-05s]\\nTotal time = 0.280516s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000172s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000902s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.00012s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000425s]\\nOpening the output file...  [9.5e-05s]\\nLoading query sequences...  [0.011998s]\\nMasking queries...  [0.053226s]\\nBuilding query seed set...  [0.015283s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.010683s]\\nAllocating buffers...  [3.5e-05s]\\nLoading reference sequences...  [5.5e-05s]\\nBuilding reference histograms...  [0.000268s]\\nAllocating buffers...  [2.6e-05s]\\nInitializing temporary storage...  [0.029417s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001161s]\\nBuilding query index...  [0.010738s]\\nBuilding seed filter...  [0.00077s]\\nSearching alignments...  [0.000959s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000474s]\\nBuilding query index...  [0.012247s]\\nBuilding seed filter...  [0.000839s]\\nSearching alignments...  [0.000978s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000614s]\\nBuilding query index...  [0.012168s]\\nBuilding seed filter...  [0.000798s]\\nSearching alignments...  [0.000972s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000493s]\\nBuilding query index...  [0.011194s]\\nBuilding seed filter...  [0.000926s]\\nSearching alignments...  [0.000974s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000618s]\\nBuilding query index...  [0.010973s]\\nBuilding seed filter...  [0.000776s]\\nSearching alignments...  [0.001122s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000544s]\\nBuilding query index...  [0.010683s]\\nBuilding seed filter...  [0.001043s]\\nSearching alignments...  [0.001309s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000691s]\\nBuilding query index...  [0.008511s]\\nBuilding seed filter...  [0.000884s]\\nSearching alignments...  [0.000906s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.001202s]\\nBuilding query index...  [0.010384s]\\nBuilding seed filter...  [0.000873s]\\nSearching alignments...  [0.001133s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.010227s]\\nDeallocating reference...  [1.2e-05s]\\nLoading reference sequences...  [3.7e-05s]\\nDeallocating buffers...  [9e-06s]\\nDeallocating queries...  [0.000242s]\\nLoading query sequences...  [1.2e-05s]\\nClosing the input file...  [0.000119s]\\nClosing the output file...  [0.000237s]\\nClosing the database file...  [1.4e-05s]\\nDeallocating taxonomy...  [5e-06s]\\nTotal time = 0.242621s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000139s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000805s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000133s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000574s]\\nOpening the output file...  [0.00013s]\\nLoading query sequences...  [0.018995s]\\nMasking queries...  [0.058436s]\\nBuilding query seed set...  [0.01294s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.011164s]\\nAllocating buffers...  [3.3e-05s]\\nLoading reference sequences...  [5.7e-05s]\\nBuilding reference histograms...  [0.000235s]\\nAllocating buffers...  [2.7e-05s]\\nInitializing temporary storage...  [0.018716s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001413s]\\nBuilding query index...  [0.010125s]\\nBuilding seed filter...  [0.0009s]\\nSearching alignments...  [0.001176s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.001297s]\\nBuilding query index...  [0.010326s]\\nBuilding seed filter...  [0.000967s]\\nSearching alignments...  [0.001016s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000433s]\\nBuilding query index...  [0.0109s]\\nBuilding seed filter...  [0.000774s]\\nSearching alignments...  [0.001292s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000481s]\\nBuilding query index...  [0.010327s]\\nBuilding seed filter...  [0.000848s]\\nSearching alignments...  [0.000999s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000484s]\\nBuilding query index...  [0.011005s]\\nBuilding seed filter...  [0.000839s]\\nSearching alignments...  [0.001207s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000472s]\\nBuilding query index...  [0.01223s]\\nBuilding seed filter...  [0.000945s]\\nSearching alignments...  [0.001129s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000437s]\\nBuilding query index...  [0.008675s]\\nBuilding seed filter...  [0.00075s]\\nSearching alignments...  [0.001064s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000401s]\\nBuilding query index...  [0.00814s]\\nBuilding seed filter...  [0.000833s]\\nSearching alignments...  [0.001054s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.011773s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [2e-05s]\\nDeallocating buffers...  [8e-06s]\\nDeallocating queries...  [0.000202s]\\nLoading query sequences...  [1.5e-05s]\\nClosing the input file...  [1.6e-05s]\\nClosing the output file...  [0.000222s]\\nClosing the database file...  [1.6e-05s]\\nDeallocating taxonomy...  [1.7e-05s]\\nTotal time = 0.23818s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000491s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000877s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000759s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000256s]\\nOpening the output file...  [0.00015s]\\nLoading query sequences...  [0.016674s]\\nMasking queries...  [0.063297s]\\nBuilding query seed set...  [0.014103s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.008913s]\\nAllocating buffers...  [2.2e-05s]\\nLoading reference sequences...  [5.2e-05s]\\nBuilding reference histograms...  [0.000215s]\\nAllocating buffers...  [3.2e-05s]\\nInitializing temporary storage...  [0.024329s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001857s]\\nBuilding query index...  [0.011354s]\\nBuilding seed filter...  [0.000825s]\\nSearching alignments...  [0.001049s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000522s]\\nBuilding query index...  [0.008817s]\\nBuilding seed filter...  [0.000822s]\\nSearching alignments...  [0.000979s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000472s]\\nBuilding query index...  [0.010737s]\\nBuilding seed filter...  [0.000893s]\\nSearching alignments...  [0.001008s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000517s]\\nBuilding query index...  [0.011716s]\\nBuilding seed filter...  [0.001059s]\\nSearching alignments...  [0.001194s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000467s]\\nBuilding query index...  [0.010371s]\\nBuilding seed filter...  [0.000948s]\\nSearching alignments...  [0.001122s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000494s]\\nBuilding query index...  [0.008946s]\\nBuilding seed filter...  [0.00094s]\\nSearching alignments...  [0.001164s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000637s]\\nBuilding query index...  [0.01135s]\\nBuilding seed filter...  [0.000795s]\\nSearching alignments...  [0.000975s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.00055s]\\nBuilding query index...  [0.008934s]\\nBuilding seed filter...  [0.000877s]\\nSearching alignments...  [0.001083s]\\nDeallocating buffers...  [3.6e-05s]\\nComputing alignments...  [0.009701s]\\nDeallocating reference...  [1.6e-05s]\\nLoading reference sequences...  [2.2e-05s]\\nDeallocating buffers...  [6e-06s]\\nDeallocating queries...  [0.00022s]\\nLoading query sequences...  [1.5e-05s]\\nClosing the input file...  [1.7e-05s]\\nClosing the output file...  [0.000237s]\\nClosing the database file...  [1.7e-05s]\\nDeallocating taxonomy...  [1.7e-05s]\\nTotal time = 0.243986s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000192s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000923s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000162s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000218s]\\nOpening the output file...  [0.000106s]\\nLoading query sequences...  [0.025125s]\\nMasking queries...  [0.16663s]\\nBuilding query seed set...  [0.028953s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.023194s]\\nAllocating buffers...  [6.6e-05s]\\nLoading reference sequences...  [6.3e-05s]\\nBuilding reference histograms...  [0.000275s]\\nAllocating buffers...  [3.2e-05s]\\nInitializing temporary storage...  [0.020664s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001331s]\\nBuilding query index...  [0.016264s]\\nBuilding seed filter...  [0.001257s]\\nSearching alignments...  [0.001215s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.0007s]\\nBuilding query index...  [0.017034s]\\nBuilding seed filter...  [0.001199s]\\nSearching alignments...  [0.001167s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000524s]\\nBuilding query index...  [0.021683s]\\nBuilding seed filter...  [0.001221s]\\nSearching alignments...  [0.001055s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000526s]\\nBuilding query index...  [0.014995s]\\nBuilding seed filter...  [0.001135s]\\nSearching alignments...  [0.00119s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000486s]\\nBuilding query index...  [0.014576s]\\nBuilding seed filter...  [0.001137s]\\nSearching alignments...  [0.001324s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000505s]\\nBuilding query index...  [0.015848s]\\nBuilding seed filter...  [0.001342s]\\nSearching alignments...  [0.001104s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000503s]\\nBuilding query index...  [0.01477s]\\nBuilding seed filter...  [0.001425s]\\nSearching alignments...  [0.001223s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000747s]\\nBuilding query index...  [0.019563s]\\nBuilding seed filter...  [0.001241s]\\nSearching alignments...  [0.001348s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.018853s]\\nDeallocating reference...  [8e-06s]\\nLoading reference sequences...  [2.5e-05s]\\nDeallocating buffers...  [0.000709s]\\nDeallocating queries...  [0.000808s]\\nLoading query sequences...  [1.6e-05s]\\nClosing the input file...  [4.5e-05s]\\nClosing the output file...  [0.000286s]\\nClosing the database file...  [2e-05s]\\nDeallocating taxonomy...  [5e-05s]\\nTotal time = 0.448665s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000196s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000981s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000158s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000392s]\\nOpening the output file...  [9.5e-05s]\\nLoading query sequences...  [0.015435s]\\nMasking queries...  [0.102905s]\\nBuilding query seed set...  [0.025281s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.014534s]\\nAllocating buffers...  [4e-05s]\\nLoading reference sequences...  [6.3e-05s]\\nBuilding reference histograms...  [0.000242s]\\nAllocating buffers...  [1.8e-05s]\\nInitializing temporary storage...  [0.020637s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.00145s]\\nBuilding query index...  [0.015794s]\\nBuilding seed filter...  [0.000924s]\\nSearching alignments...  [0.00106s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000548s]\\nBuilding query index...  [0.015554s]\\nBuilding seed filter...  [0.000937s]\\nSearching alignments...  [0.001052s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000487s]\\nBuilding query index...  [0.013013s]\\nBuilding seed filter...  [0.000898s]\\nSearching alignments...  [0.000998s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000472s]\\nBuilding query index...  [0.010505s]\\nBuilding seed filter...  [0.001195s]\\nSearching alignments...  [0.001159s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.00045s]\\nBuilding query index...  [0.009507s]\\nBuilding seed filter...  [0.00102s]\\nSearching alignments...  [0.001205s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.00059s]\\nBuilding query index...  [0.013047s]\\nBuilding seed filter...  [0.001363s]\\nSearching alignments...  [0.001152s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000533s]\\nBuilding query index...  [0.012753s]\\nBuilding seed filter...  [0.001053s]\\nSearching alignments...  [0.001258s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.00054s]\\nBuilding query index...  [0.012135s]\\nBuilding seed filter...  [0.001007s]\\nSearching alignments...  [0.001143s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.012079s]\\nDeallocating reference...  [8e-06s]\\nLoading reference sequences...  [2.2e-05s]\\nDeallocating buffers...  [0.000447s]\\nDeallocating queries...  [0.000116s]\\nLoading query sequences...  [1.8e-05s]\\nClosing the input file...  [3.6e-05s]\\nClosing the output file...  [0.000206s]\\nClosing the database file...  [1.8e-05s]\\nDeallocating taxonomy...  [1.3e-05s]\\nTotal time = 0.318861s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000192s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000913s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000142s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000734s]\\nOpening the output file...  [0.000108s]\\nLoading query sequences...  [0.016266s]\\nMasking queries...  [0.073732s]\\nBuilding query seed set...  [0.022985s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.01264s]\\nAllocating buffers...  [2.7e-05s]\\nLoading reference sequences...  [4.9e-05s]\\nBuilding reference histograms...  [0.000299s]\\nAllocating buffers...  [1.6e-05s]\\nInitializing temporary storage...  [0.026405s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001676s]\\nBuilding query index...  [0.015665s]\\nBuilding seed filter...  [0.000885s]\\nSearching alignments...  [0.001177s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000648s]\\nBuilding query index...  [0.009832s]\\nBuilding seed filter...  [0.000904s]\\nSearching alignments...  [0.000958s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000477s]\\nBuilding query index...  [0.014572s]\\nBuilding seed filter...  [0.00134s]\\nSearching alignments...  [0.001261s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.00064s]\\nBuilding query index...  [0.01367s]\\nBuilding seed filter...  [0.001324s]\\nSearching alignments...  [0.001265s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.001626s]\\nBuilding query index...  [0.011378s]\\nBuilding seed filter...  [0.001084s]\\nSearching alignments...  [0.00121s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000739s]\\nBuilding query index...  [0.010865s]\\nBuilding seed filter...  [0.001307s]\\nSearching alignments...  [0.001301s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000528s]\\nBuilding query index...  [0.010965s]\\nBuilding seed filter...  [0.000889s]\\nSearching alignments...  [0.001046s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000543s]\\nBuilding query index...  [0.009506s]\\nBuilding seed filter...  [0.00169s]\\nSearching alignments...  [0.001175s]\\nDeallocating buffers...  [1.9e-05s]\\nComputing alignments...  [0.010068s]\\nDeallocating reference...  [2.4e-05s]\\nLoading reference sequences...  [0.000105s]\\nDeallocating buffers...  [9e-06s]\\nDeallocating queries...  [0.000292s]\\nLoading query sequences...  [1.6e-05s]\\nClosing the input file...  [1.9e-05s]\\nClosing the output file...  [0.000246s]\\nClosing the database file...  [1.4e-05s]\\nDeallocating taxonomy...  [5e-06s]\\nTotal time = 0.288056s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000226s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000828s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000184s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000466s]\\nOpening the output file...  [0.000141s]\\nLoading query sequences...  [0.030229s]\\nMasking queries...  [0.071369s]\\nBuilding query seed set...  [0.021242s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.017485s]\\nAllocating buffers...  [4e-05s]\\nLoading reference sequences...  [5.3e-05s]\\nBuilding reference histograms...  [0.000336s]\\nAllocating buffers...  [2.5e-05s]\\nInitializing temporary storage...  [0.02235s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001489s]\\nBuilding query index...  [0.018399s]\\nBuilding seed filter...  [0.00156s]\\nSearching alignments...  [0.001201s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000609s]\\nBuilding query index...  [0.016344s]\\nBuilding seed filter...  [0.001636s]\\nSearching alignments...  [0.001704s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000578s]\\nBuilding query index...  [0.015475s]\\nBuilding seed filter...  [0.000964s]\\nSearching alignments...  [0.001019s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.00054s]\\nBuilding query index...  [0.011517s]\\nBuilding seed filter...  [0.00105s]\\nSearching alignments...  [0.001209s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000597s]\\nBuilding query index...  [0.011391s]\\nBuilding seed filter...  [0.001202s]\\nSearching alignments...  [0.00126s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.00074s]\\nBuilding query index...  [0.014068s]\\nBuilding seed filter...  [0.001275s]\\nSearching alignments...  [0.001391s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000531s]\\nBuilding query index...  [0.016168s]\\nBuilding seed filter...  [0.002606s]\\nSearching alignments...  [0.001356s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000514s]\\nBuilding query index...  [0.013639s]\\nBuilding seed filter...  [0.001293s]\\nSearching alignments...  [0.00122s]\\nDeallocating buffers...  [9e-06s]\\nComputing alignments...  [0.012907s]\\nDeallocating reference...  [2.5e-05s]\\nLoading reference sequences...  [4e-05s]\\nDeallocating buffers...  [2.9e-05s]\\nDeallocating queries...  [0.000239s]\\nLoading query sequences...  [1.8e-05s]\\nClosing the input file...  [6.4e-05s]\\nClosing the output file...  [0.000294s]\\nClosing the database file...  [5.7e-05s]\\nDeallocating taxonomy...  [2.6e-05s]\\nTotal time = 0.324367s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000191s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001108s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000166s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000269s]\\nOpening the output file...  [0.000126s]\\nLoading query sequences...  [0.015086s]\\nMasking queries...  [0.072691s]\\nBuilding query seed set...  [0.019233s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.013865s]\\nAllocating buffers...  [3.3e-05s]\\nLoading reference sequences...  [7.7e-05s]\\nBuilding reference histograms...  [0.000271s]\\nAllocating buffers...  [2.7e-05s]\\nInitializing temporary storage...  [0.019303s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001354s]\\nBuilding query index...  [0.017842s]\\nBuilding seed filter...  [0.001527s]\\nSearching alignments...  [0.001646s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000646s]\\nBuilding query index...  [0.014258s]\\nBuilding seed filter...  [0.000899s]\\nSearching alignments...  [0.001232s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000542s]\\nBuilding query index...  [0.013397s]\\nBuilding seed filter...  [0.00106s]\\nSearching alignments...  [0.001045s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000613s]\\nBuilding query index...  [0.012898s]\\nBuilding seed filter...  [0.001603s]\\nSearching alignments...  [0.001132s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000839s]\\nBuilding query index...  [0.013225s]\\nBuilding seed filter...  [0.001587s]\\nSearching alignments...  [0.001164s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000728s]\\nBuilding query index...  [0.013596s]\\nBuilding seed filter...  [0.001184s]\\nSearching alignments...  [0.001225s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000586s]\\nBuilding query index...  [0.012354s]\\nBuilding seed filter...  [0.001097s]\\nSearching alignments...  [0.001192s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000494s]\\nBuilding query index...  [0.011657s]\\nBuilding seed filter...  [0.001055s]\\nSearching alignments...  [0.001074s]\\nDeallocating buffers...  [8e-06s]\\nComputing alignments...  [0.02062s]\\nDeallocating reference...  [7e-06s]\\nLoading reference sequences...  [3.5e-05s]\\nDeallocating buffers...  [0.000387s]\\nDeallocating queries...  [0.000359s]\\nLoading query sequences...  [1.6e-05s]\\nClosing the input file...  [2.9e-05s]\\nClosing the output file...  [0.000294s]\\nClosing the database file...  [1.4e-05s]\\nDeallocating taxonomy...  [2.2e-05s]\\nTotal time = 0.299394s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000228s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001415s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000137s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000474s]\\nOpening the output file...  [0.000107s]\\nLoading query sequences...  [0.015302s]\\nMasking queries...  [0.090637s]\\nBuilding query seed set...  [0.023131s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.016115s]\\nAllocating buffers...  [4.4e-05s]\\nLoading reference sequences...  [6.2e-05s]\\nBuilding reference histograms...  [0.000268s]\\nAllocating buffers...  [3.1e-05s]\\nInitializing temporary storage...  [0.028415s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001654s]\\nBuilding query index...  [0.010915s]\\nBuilding seed filter...  [0.001158s]\\nSearching alignments...  [0.001116s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000557s]\\nBuilding query index...  [0.008845s]\\nBuilding seed filter...  [0.000917s]\\nSearching alignments...  [0.001086s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000679s]\\nBuilding query index...  [0.009727s]\\nBuilding seed filter...  [0.000999s]\\nSearching alignments...  [0.001082s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000661s]\\nBuilding query index...  [0.010191s]\\nBuilding seed filter...  [0.00086s]\\nSearching alignments...  [0.001025s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000827s]\\nBuilding query index...  [0.011849s]\\nBuilding seed filter...  [0.001187s]\\nSearching alignments...  [0.001336s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000669s]\\nBuilding query index...  [0.009868s]\\nBuilding seed filter...  [0.001188s]\\nSearching alignments...  [0.0011s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000567s]\\nBuilding query index...  [0.014907s]\\nBuilding seed filter...  [0.001298s]\\nSearching alignments...  [0.00142s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000495s]\\nBuilding query index...  [0.012182s]\\nBuilding seed filter...  [0.001057s]\\nSearching alignments...  [0.001272s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.01918s]\\nDeallocating reference...  [3e-05s]\\nLoading reference sequences...  [0.000106s]\\nDeallocating buffers...  [4.8e-05s]\\nDeallocating queries...  [0.000219s]\\nLoading query sequences...  [1.5e-05s]\\nClosing the input file...  [2.8e-05s]\\nClosing the output file...  [0.00025s]\\nClosing the database file...  [1.4e-05s]\\nDeallocating taxonomy...  [1.8e-05s]\\nTotal time = 0.308878s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000157s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001278s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000181s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000483s]\\nOpening the output file...  [9.1e-05s]\\nLoading query sequences...  [0.014571s]\\nMasking queries...  [0.07433s]\\nBuilding query seed set...  [0.019976s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.01794s]\\nAllocating buffers...  [5.2e-05s]\\nLoading reference sequences...  [7.3e-05s]\\nBuilding reference histograms...  [0.000556s]\\nAllocating buffers...  [4.4e-05s]\\nInitializing temporary storage...  [0.020987s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001606s]\\nBuilding query index...  [0.013778s]\\nBuilding seed filter...  [0.001172s]\\nSearching alignments...  [0.001376s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000525s]\\nBuilding query index...  [0.013974s]\\nBuilding seed filter...  [0.001024s]\\nSearching alignments...  [0.001156s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000618s]\\nBuilding query index...  [0.014083s]\\nBuilding seed filter...  [0.001115s]\\nSearching alignments...  [0.001175s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000567s]\\nBuilding query index...  [0.012889s]\\nBuilding seed filter...  [0.001212s]\\nSearching alignments...  [0.00125s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000574s]\\nBuilding query index...  [0.012315s]\\nBuilding seed filter...  [0.001025s]\\nSearching alignments...  [0.001167s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000625s]\\nBuilding query index...  [0.014454s]\\nBuilding seed filter...  [0.000965s]\\nSearching alignments...  [0.001053s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000596s]\\nBuilding query index...  [0.013575s]\\nBuilding seed filter...  [0.001026s]\\nSearching alignments...  [0.001046s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000565s]\\nBuilding query index...  [0.009806s]\\nBuilding seed filter...  [0.000978s]\\nSearching alignments...  [0.001198s]\\nDeallocating buffers...  [8e-06s]\\nComputing alignments...  [0.012664s]\\nDeallocating reference...  [2.8e-05s]\\nLoading reference sequences...  [2.2e-05s]\\nDeallocating buffers...  [0.000595s]\\nDeallocating queries...  [0.000654s]\\nLoading query sequences...  [2.4e-05s]\\nClosing the input file...  [2.8e-05s]\\nClosing the output file...  [0.000406s]\\nClosing the database file...  [2.7e-05s]\\nDeallocating taxonomy...  [2e-05s]\\nTotal time = 0.294386s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000191s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.00093s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000188s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000413s]\\nOpening the output file...  [8.8e-05s]\\nLoading query sequences...  [0.01526s]\\nMasking queries...  [0.074533s]\\nBuilding query seed set...  [0.020949s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.010085s]\\nAllocating buffers...  [2.6e-05s]\\nLoading reference sequences...  [6e-05s]\\nBuilding reference histograms...  [0.000306s]\\nAllocating buffers...  [4.1e-05s]\\nInitializing temporary storage...  [0.019988s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.00142s]\\nBuilding query index...  [0.011484s]\\nBuilding seed filter...  [0.001316s]\\nSearching alignments...  [0.001227s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000525s]\\nBuilding query index...  [0.014459s]\\nBuilding seed filter...  [0.001123s]\\nSearching alignments...  [0.002152s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000773s]\\nBuilding query index...  [0.015529s]\\nBuilding seed filter...  [0.000865s]\\nSearching alignments...  [0.001039s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.00076s]\\nBuilding query index...  [0.009842s]\\nBuilding seed filter...  [0.001366s]\\nSearching alignments...  [0.001235s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000488s]\\nBuilding query index...  [0.010163s]\\nBuilding seed filter...  [0.001026s]\\nSearching alignments...  [0.001197s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000608s]\\nBuilding query index...  [0.012503s]\\nBuilding seed filter...  [0.001014s]\\nSearching alignments...  [0.001087s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000743s]\\nBuilding query index...  [0.009629s]\\nBuilding seed filter...  [0.000982s]\\nSearching alignments...  [0.000951s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000469s]\\nBuilding query index...  [0.008932s]\\nBuilding seed filter...  [0.000887s]\\nSearching alignments...  [0.001278s]\\nDeallocating buffers...  [7e-06s]\\nComputing alignments...  [0.010642s]\\nDeallocating reference...  [3.6e-05s]\\nLoading reference sequences...  [2.3e-05s]\\nDeallocating buffers...  [1e-05s]\\nDeallocating queries...  [0.000234s]\\nLoading query sequences...  [1.6e-05s]\\nClosing the input file...  [2e-05s]\\nClosing the output file...  [0.000224s]\\nClosing the database file...  [2.1e-05s]\\nDeallocating taxonomy...  [1.6e-05s]\\nTotal time = 0.271766s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000228s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000943s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.00016s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.00042s]\\nOpening the output file...  [0.000127s]\\nLoading query sequences...  [0.014474s]\\nMasking queries...  [0.07573s]\\nBuilding query seed set...  [0.023036s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.012784s]\\nAllocating buffers...  [4e-05s]\\nLoading reference sequences...  [6.5e-05s]\\nBuilding reference histograms...  [0.000339s]\\nAllocating buffers...  [2.7e-05s]\\nInitializing temporary storage...  [0.018409s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001272s]\\nBuilding query index...  [0.009997s]\\nBuilding seed filter...  [0.000909s]\\nSearching alignments...  [0.001219s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000494s]\\nBuilding query index...  [0.010489s]\\nBuilding seed filter...  [0.000886s]\\nSearching alignments...  [0.001145s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000832s]\\nBuilding query index...  [0.009489s]\\nBuilding seed filter...  [0.000818s]\\nSearching alignments...  [0.001099s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000524s]\\nBuilding query index...  [0.010686s]\\nBuilding seed filter...  [0.000931s]\\nSearching alignments...  [0.001113s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000505s]\\nBuilding query index...  [0.009346s]\\nBuilding seed filter...  [0.000946s]\\nSearching alignments...  [0.001412s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000538s]\\nBuilding query index...  [0.009342s]\\nBuilding seed filter...  [0.000975s]\\nSearching alignments...  [0.001153s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000573s]\\nBuilding query index...  [0.010099s]\\nBuilding seed filter...  [0.000861s]\\nSearching alignments...  [0.001042s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000582s]\\nBuilding query index...  [0.009946s]\\nBuilding seed filter...  [0.001093s]\\nSearching alignments...  [0.001088s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.011897s]\\nDeallocating reference...  [1.5e-05s]\\nLoading reference sequences...  [4e-05s]\\nDeallocating buffers...  [1.1e-05s]\\nDeallocating queries...  [0.000267s]\\nLoading query sequences...  [1.7e-05s]\\nClosing the input file...  [2.6e-05s]\\nClosing the output file...  [0.00026s]\\nClosing the database file...  [1.5e-05s]\\nDeallocating taxonomy...  [5e-06s]\\nTotal time = 0.265639s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000294s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001065s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000163s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000475s]\\nOpening the output file...  [0.000118s]\\nLoading query sequences...  [0.015095s]\\nMasking queries...  [0.058553s]\\nBuilding query seed set...  [0.019988s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.009603s]\\nAllocating buffers...  [3.7e-05s]\\nLoading reference sequences...  [5.9e-05s]\\nBuilding reference histograms...  [0.000195s]\\nAllocating buffers...  [2.5e-05s]\\nInitializing temporary storage...  [0.019418s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001529s]\\nBuilding query index...  [0.013065s]\\nBuilding seed filter...  [0.00136s]\\nSearching alignments...  [0.001528s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000652s]\\nBuilding query index...  [0.013678s]\\nBuilding seed filter...  [0.001105s]\\nSearching alignments...  [0.001265s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000607s]\\nBuilding query index...  [0.014521s]\\nBuilding seed filter...  [0.003254s]\\nSearching alignments...  [0.002405s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000984s]\\nBuilding query index...  [0.013018s]\\nBuilding seed filter...  [0.00131s]\\nSearching alignments...  [0.001134s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000522s]\\nBuilding query index...  [0.011519s]\\nBuilding seed filter...  [0.001s]\\nSearching alignments...  [0.001207s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000553s]\\nBuilding query index...  [0.013034s]\\nBuilding seed filter...  [0.000948s]\\nSearching alignments...  [0.001061s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.003087s]\\nBuilding query index...  [0.019912s]\\nBuilding seed filter...  [0.002731s]\\nSearching alignments...  [0.0012s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000566s]\\nBuilding query index...  [0.012899s]\\nBuilding seed filter...  [0.007372s]\\nSearching alignments...  [0.00319s]\\nDeallocating buffers...  [4.3e-05s]\\nComputing alignments...  [0.013057s]\\nDeallocating reference...  [1e-05s]\\nLoading reference sequences...  [3.4e-05s]\\nDeallocating buffers...  [1e-05s]\\nDeallocating queries...  [0.00033s]\\nLoading query sequences...  [3.1e-05s]\\nClosing the input file...  [3.2e-05s]\\nClosing the output file...  [0.000399s]\\nClosing the database file...  [3.6e-05s]\\nDeallocating taxonomy...  [2.4e-05s]\\nTotal time = 0.292368s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000217s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000999s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000188s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000411s]\\nOpening the output file...  [0.000128s]\\nLoading query sequences...  [0.01208s]\\nMasking queries...  [0.058314s]\\nBuilding query seed set...  [0.019957s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.014871s]\\nAllocating buffers...  [2.6e-05s]\\nLoading reference sequences...  [4.9e-05s]\\nBuilding reference histograms...  [0.000228s]\\nAllocating buffers...  [2.1e-05s]\\nInitializing temporary storage...  [0.019257s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001811s]\\nBuilding query index...  [0.01142s]\\nBuilding seed filter...  [0.000952s]\\nSearching alignments...  [0.001171s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000463s]\\nBuilding query index...  [0.008641s]\\nBuilding seed filter...  [0.000805s]\\nSearching alignments...  [0.001023s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000687s]\\nBuilding query index...  [0.009525s]\\nBuilding seed filter...  [0.000944s]\\nSearching alignments...  [0.001035s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.00064s]\\nBuilding query index...  [0.012321s]\\nBuilding seed filter...  [0.001102s]\\nSearching alignments...  [0.001229s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.002159s]\\nBuilding query index...  [0.012962s]\\nBuilding seed filter...  [0.001726s]\\nSearching alignments...  [0.001215s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000593s]\\nBuilding query index...  [0.009493s]\\nBuilding seed filter...  [0.001984s]\\nSearching alignments...  [0.001236s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000744s]\\nBuilding query index...  [0.012372s]\\nBuilding seed filter...  [0.001176s]\\nSearching alignments...  [0.001165s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000611s]\\nBuilding query index...  [0.009292s]\\nBuilding seed filter...  [0.000872s]\\nSearching alignments...  [0.001111s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.011666s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [2.2e-05s]\\nDeallocating buffers...  [7e-06s]\\nDeallocating queries...  [0.000227s]\\nLoading query sequences...  [1.8e-05s]\\nClosing the input file...  [4.1e-05s]\\nClosing the output file...  [0.000226s]\\nClosing the database file...  [1.9e-05s]\\nDeallocating taxonomy...  [1.5e-05s]\\nTotal time = 0.252047s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000253s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.002902s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000109s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000213s]\\nOpening the output file...  [8.4e-05s]\\nLoading query sequences...  [0.01049s]\\nMasking queries...  [0.051783s]\\nBuilding query seed set...  [0.013713s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.00854s]\\nAllocating buffers...  [3.7e-05s]\\nLoading reference sequences...  [5.7e-05s]\\nBuilding reference histograms...  [0.000292s]\\nAllocating buffers...  [2.8e-05s]\\nInitializing temporary storage...  [0.017274s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001092s]\\nBuilding query index...  [0.008834s]\\nBuilding seed filter...  [0.000756s]\\nSearching alignments...  [0.001018s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000481s]\\nBuilding query index...  [0.007396s]\\nBuilding seed filter...  [0.000779s]\\nSearching alignments...  [0.001028s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000483s]\\nBuilding query index...  [0.007406s]\\nBuilding seed filter...  [0.000796s]\\nSearching alignments...  [0.000976s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000517s]\\nBuilding query index...  [0.0068s]\\nBuilding seed filter...  [0.000723s]\\nSearching alignments...  [0.000985s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000467s]\\nBuilding query index...  [0.006573s]\\nBuilding seed filter...  [0.00077s]\\nSearching alignments...  [0.001066s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000452s]\\nBuilding query index...  [0.008151s]\\nBuilding seed filter...  [0.000898s]\\nSearching alignments...  [0.001092s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000504s]\\nBuilding query index...  [0.012254s]\\nBuilding seed filter...  [0.000796s]\\nSearching alignments...  [0.000979s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000507s]\\nBuilding query index...  [0.011077s]\\nBuilding seed filter...  [0.000875s]\\nSearching alignments...  [0.0011s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.01085s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [1.8e-05s]\\nDeallocating buffers...  [7e-06s]\\nDeallocating queries...  [0.000181s]\\nLoading query sequences...  [2.6e-05s]\\nClosing the input file...  [3e-05s]\\nClosing the output file...  [0.000166s]\\nClosing the database file...  [1e-05s]\\nDeallocating taxonomy...  [1.3e-05s]\\nTotal time = 0.203022s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000214s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.00094s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000174s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000413s]\\nOpening the output file...  [0.00011s]\\nLoading query sequences...  [0.013289s]\\nMasking queries...  [0.066174s]\\nBuilding query seed set...  [0.016381s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.010595s]\\nAllocating buffers...  [2.6e-05s]\\nLoading reference sequences...  [4.6e-05s]\\nBuilding reference histograms...  [0.000198s]\\nAllocating buffers...  [1.7e-05s]\\nInitializing temporary storage...  [0.019344s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001263s]\\nBuilding query index...  [0.009013s]\\nBuilding seed filter...  [0.000864s]\\nSearching alignments...  [0.001219s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000481s]\\nBuilding query index...  [0.008866s]\\nBuilding seed filter...  [0.00086s]\\nSearching alignments...  [0.001415s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000829s]\\nBuilding query index...  [0.012646s]\\nBuilding seed filter...  [0.000917s]\\nSearching alignments...  [0.001006s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000506s]\\nBuilding query index...  [0.008432s]\\nBuilding seed filter...  [0.000917s]\\nSearching alignments...  [0.001147s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000565s]\\nBuilding query index...  [0.008459s]\\nBuilding seed filter...  [0.00099s]\\nSearching alignments...  [0.001162s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000486s]\\nBuilding query index...  [0.008899s]\\nBuilding seed filter...  [0.000822s]\\nSearching alignments...  [0.00109s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000501s]\\nBuilding query index...  [0.009312s]\\nBuilding seed filter...  [0.001349s]\\nSearching alignments...  [0.001226s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000446s]\\nBuilding query index...  [0.012378s]\\nBuilding seed filter...  [0.001474s]\\nSearching alignments...  [0.00139s]\\nDeallocating buffers...  [7e-06s]\\nComputing alignments...  [0.013144s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [2.1e-05s]\\nDeallocating buffers...  [9e-06s]\\nDeallocating queries...  [0.000247s]\\nLoading query sequences...  [1.2e-05s]\\nClosing the input file...  [5.1e-05s]\\nClosing the output file...  [0.000221s]\\nClosing the database file...  [4e-05s]\\nDeallocating taxonomy...  [1.2e-05s]\\nTotal time = 0.243102s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000204s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001347s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000149s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000502s]\\nOpening the output file...  [0.000139s]\\nLoading query sequences...  [0.012257s]\\nMasking queries...  [0.083152s]\\nBuilding query seed set...  [0.016321s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.011876s]\\nAllocating buffers...  [2.6e-05s]\\nLoading reference sequences...  [4.9e-05s]\\nBuilding reference histograms...  [0.000255s]\\nAllocating buffers...  [1.8e-05s]\\nInitializing temporary storage...  [0.01888s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001305s]\\nBuilding query index...  [0.010931s]\\nBuilding seed filter...  [0.001021s]\\nSearching alignments...  [0.001193s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000582s]\\nBuilding query index...  [0.015097s]\\nBuilding seed filter...  [0.001419s]\\nSearching alignments...  [0.001258s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.00092s]\\nBuilding query index...  [0.013694s]\\nBuilding seed filter...  [0.00099s]\\nSearching alignments...  [0.001036s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000513s]\\nBuilding query index...  [0.012883s]\\nBuilding seed filter...  [0.001005s]\\nSearching alignments...  [0.001015s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000495s]\\nBuilding query index...  [0.010925s]\\nBuilding seed filter...  [0.001486s]\\nSearching alignments...  [0.001383s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000489s]\\nBuilding query index...  [0.013599s]\\nBuilding seed filter...  [0.001144s]\\nSearching alignments...  [0.001148s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000515s]\\nBuilding query index...  [0.009289s]\\nBuilding seed filter...  [0.00091s]\\nSearching alignments...  [0.001096s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000503s]\\nBuilding query index...  [0.01177s]\\nBuilding seed filter...  [0.000996s]\\nSearching alignments...  [0.000988s]\\nDeallocating buffers...  [1.8e-05s]\\nComputing alignments...  [0.010155s]\\nDeallocating reference...  [7e-06s]\\nLoading reference sequences...  [1.5e-05s]\\nDeallocating buffers...  [6e-06s]\\nDeallocating queries...  [0.00021s]\\nLoading query sequences...  [2.3e-05s]\\nClosing the input file...  [2.4e-05s]\\nClosing the output file...  [0.000309s]\\nClosing the database file...  [3.9e-05s]\\nDeallocating taxonomy...  [6e-06s]\\nTotal time = 0.277382s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000217s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001156s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000204s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000285s]\\nOpening the output file...  [0.000143s]\\nLoading query sequences...  [0.014668s]\\nMasking queries...  [0.064146s]\\nBuilding query seed set...  [0.013411s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.00997s]\\nAllocating buffers...  [2.6e-05s]\\nLoading reference sequences...  [4.4e-05s]\\nBuilding reference histograms...  [0.00022s]\\nAllocating buffers...  [2.7e-05s]\\nInitializing temporary storage...  [0.043537s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001161s]\\nBuilding query index...  [0.009018s]\\nBuilding seed filter...  [0.000965s]\\nSearching alignments...  [0.001022s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000559s]\\nBuilding query index...  [0.014557s]\\nBuilding seed filter...  [0.001088s]\\nSearching alignments...  [0.001051s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000565s]\\nBuilding query index...  [0.012078s]\\nBuilding seed filter...  [0.00077s]\\nSearching alignments...  [0.001099s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000502s]\\nBuilding query index...  [0.009964s]\\nBuilding seed filter...  [0.000785s]\\nSearching alignments...  [0.001124s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000501s]\\nBuilding query index...  [0.010041s]\\nBuilding seed filter...  [0.000862s]\\nSearching alignments...  [0.001118s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000477s]\\nBuilding query index...  [0.010174s]\\nBuilding seed filter...  [0.000973s]\\nSearching alignments...  [0.001002s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000504s]\\nBuilding query index...  [0.010756s]\\nBuilding seed filter...  [0.000837s]\\nSearching alignments...  [0.000965s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000483s]\\nBuilding query index...  [0.009122s]\\nBuilding seed filter...  [0.000846s]\\nSearching alignments...  [0.001043s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.01154s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [2.3e-05s]\\nDeallocating buffers...  [8e-06s]\\nDeallocating queries...  [0.0003s]\\nLoading query sequences...  [1.7e-05s]\\nClosing the input file...  [1.6e-05s]\\nClosing the output file...  [0.000251s]\\nClosing the database file...  [2.1e-05s]\\nDeallocating taxonomy...  [1.7e-05s]\\nTotal time = 0.266173s\\nReported 20 pairwise alignments, 20 HSPs.\\n20 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000186s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 20\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.00102s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000144s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000239s]\\nOpening the output file...  [0.000242s]\\nLoading query sequences...  [0.01278s]\\nMasking queries...  [0.06364s]\\nBuilding query seed set...  [0.012999s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.016359s]\\nAllocating buffers...  [3.8e-05s]\\nLoading reference sequences...  [0.000106s]\\nBuilding reference histograms...  [0.000294s]\\nAllocating buffers...  [1.8e-05s]\\nInitializing temporary storage...  [0.024244s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001279s]\\nBuilding query index...  [0.009686s]\\nBuilding seed filter...  [0.001008s]\\nSearching alignments...  [0.001119s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.00054s]\\nBuilding query index...  [0.01246s]\\nBuilding seed filter...  [0.000744s]\\nSearching alignments...  [0.000917s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000495s]\\nBuilding query index...  [0.009421s]\\nBuilding seed filter...  [0.001214s]\\nSearching alignments...  [0.001329s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000573s]\\nBuilding query index...  [0.011634s]\\nBuilding seed filter...  [0.000909s]\\nSearching alignments...  [0.00116s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000546s]\\nBuilding query index...  [0.012819s]\\nBuilding seed filter...  [0.001085s]\\nSearching alignments...  [0.001255s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000646s]\\nBuilding query index...  [0.013325s]\\nBuilding seed filter...  [0.001327s]\\nSearching alignments...  [0.001233s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000518s]\\nBuilding query index...  [0.012569s]\\nBuilding seed filter...  [0.00111s]\\nSearching alignments...  [0.001486s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000606s]\\nBuilding query index...  [0.011304s]\\nBuilding seed filter...  [0.001596s]\\nSearching alignments...  [0.001096s]\\nDeallocating buffers...  [7e-06s]\\nComputing alignments...  [0.011293s]\\nDeallocating reference...  [8e-06s]\\nLoading reference sequences...  [3.3e-05s]\\nDeallocating buffers...  [8e-06s]\\nDeallocating queries...  [0.000319s]\\nLoading query sequences...  [1.9e-05s]\\nClosing the input file...  [2.4e-05s]\\nClosing the output file...  [0.000314s]\\nClosing the database file...  [3.5e-05s]\\nDeallocating taxonomy...  [5e-06s]\\nTotal time = 0.261509s\\nReported 20 pairwise alignments, 20 HSPs.\\n20 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000184s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 20\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001039s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000192s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000241s]\\nOpening the output file...  [0.000119s]\\nLoading query sequences...  [0.01428s]\\nMasking queries...  [0.066148s]\\nBuilding query seed set...  [0.022394s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.012007s]\\nAllocating buffers...  [4.3e-05s]\\nLoading reference sequences...  [6.6e-05s]\\nBuilding reference histograms...  [0.000249s]\\nAllocating buffers...  [4.1e-05s]\\nInitializing temporary storage...  [0.018604s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001292s]\\nBuilding query index...  [0.00995s]\\nBuilding seed filter...  [0.000903s]\\nSearching alignments...  [0.001231s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000575s]\\nBuilding query index...  [0.009757s]\\nBuilding seed filter...  [0.000922s]\\nSearching alignments...  [0.001139s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000494s]\\nBuilding query index...  [0.014186s]\\nBuilding seed filter...  [0.000949s]\\nSearching alignments...  [0.001118s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000466s]\\nBuilding query index...  [0.009155s]\\nBuilding seed filter...  [0.001129s]\\nSearching alignments...  [0.001236s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000484s]\\nBuilding query index...  [0.008659s]\\nBuilding seed filter...  [0.00116s]\\nSearching alignments...  [0.001094s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000527s]\\nBuilding query index...  [0.010095s]\\nBuilding seed filter...  [0.000821s]\\nSearching alignments...  [0.0011s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000476s]\\nBuilding query index...  [0.009001s]\\nBuilding seed filter...  [0.000784s]\\nSearching alignments...  [0.000937s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000494s]\\nBuilding query index...  [0.010864s]\\nBuilding seed filter...  [0.000908s]\\nSearching alignments...  [0.001226s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.01246s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [1.8e-05s]\\nDeallocating buffers...  [5e-06s]\\nDeallocating queries...  [0.000218s]\\nLoading query sequences...  [1.4e-05s]\\nClosing the input file...  [1.9e-05s]\\nClosing the output file...  [0.000195s]\\nClosing the database file...  [1.1e-05s]\\nDeallocating taxonomy...  [1.4e-05s]\\nTotal time = 0.251845s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000169s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000779s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000588s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.001143s]\\nOpening the output file...  [0.000122s]\\nLoading query sequences...  [0.011652s]\\nMasking queries...  [0.06846s]\\nBuilding query seed set...  [0.021463s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.010832s]\\nAllocating buffers...  [2.9e-05s]\\nLoading reference sequences...  [5.3e-05s]\\nBuilding reference histograms...  [0.000255s]\\nAllocating buffers...  [2e-05s]\\nInitializing temporary storage...  [0.02696s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001276s]\\nBuilding query index...  [0.01047s]\\nBuilding seed filter...  [0.000876s]\\nSearching alignments...  [0.000996s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000582s]\\nBuilding query index...  [0.008886s]\\nBuilding seed filter...  [0.001153s]\\nSearching alignments...  [0.001042s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000477s]\\nBuilding query index...  [0.01457s]\\nBuilding seed filter...  [0.000982s]\\nSearching alignments...  [0.001211s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000554s]\\nBuilding query index...  [0.008139s]\\nBuilding seed filter...  [0.000863s]\\nSearching alignments...  [0.00105s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000528s]\\nBuilding query index...  [0.010545s]\\nBuilding seed filter...  [0.000977s]\\nSearching alignments...  [0.001061s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000674s]\\nBuilding query index...  [0.012504s]\\nBuilding seed filter...  [0.001173s]\\nSearching alignments...  [0.001094s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000597s]\\nBuilding query index...  [0.009256s]\\nBuilding seed filter...  [0.000845s]\\nSearching alignments...  [0.001077s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000853s]\\nBuilding query index...  [0.008075s]\\nBuilding seed filter...  [0.00088s]\\nSearching alignments...  [0.001014s]\\nDeallocating buffers...  [7e-06s]\\nComputing alignments...  [0.010531s]\\nDeallocating reference...  [4.8e-05s]\\nLoading reference sequences...  [2.2e-05s]\\nDeallocating buffers...  [9e-06s]\\nDeallocating queries...  [0.000195s]\\nLoading query sequences...  [1.1e-05s]\\nClosing the input file...  [1.3e-05s]\\nClosing the output file...  [0.000176s]\\nClosing the database file...  [1.1e-05s]\\nDeallocating taxonomy...  [4e-06s]\\nTotal time = 0.258609s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000181s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000994s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000172s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000268s]\\nOpening the output file...  [0.000113s]\\nLoading query sequences...  [0.013867s]\\nMasking queries...  [0.057532s]\\nBuilding query seed set...  [0.020724s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.010275s]\\nAllocating buffers...  [3.4e-05s]\\nLoading reference sequences...  [6.6e-05s]\\nBuilding reference histograms...  [0.000231s]\\nAllocating buffers...  [2.5e-05s]\\nInitializing temporary storage...  [0.016382s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.00114s]\\nBuilding query index...  [0.011696s]\\nBuilding seed filter...  [0.00101s]\\nSearching alignments...  [0.001283s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000603s]\\nBuilding query index...  [0.009951s]\\nBuilding seed filter...  [0.00079s]\\nSearching alignments...  [0.001051s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000447s]\\nBuilding query index...  [0.007875s]\\nBuilding seed filter...  [0.000848s]\\nSearching alignments...  [0.001045s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000454s]\\nBuilding query index...  [0.009711s]\\nBuilding seed filter...  [0.000871s]\\nSearching alignments...  [0.001026s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000439s]\\nBuilding query index...  [0.011157s]\\nBuilding seed filter...  [0.000895s]\\nSearching alignments...  [0.000962s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000491s]\\nBuilding query index...  [0.01157s]\\nBuilding seed filter...  [0.000757s]\\nSearching alignments...  [0.000988s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000456s]\\nBuilding query index...  [0.007764s]\\nBuilding seed filter...  [0.00075s]\\nSearching alignments...  [0.001052s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000507s]\\nBuilding query index...  [0.009762s]\\nBuilding seed filter...  [0.00071s]\\nSearching alignments...  [0.001027s]\\nDeallocating buffers...  [1e-05s]\\nComputing alignments...  [0.009617s]\\nDeallocating reference...  [1.9e-05s]\\nLoading reference sequences...  [2.2e-05s]\\nDeallocating buffers...  [7e-06s]\\nDeallocating queries...  [0.000268s]\\nLoading query sequences...  [2.6e-05s]\\nClosing the input file...  [2e-05s]\\nClosing the output file...  [0.000205s]\\nClosing the database file...  [1.4e-05s]\\nDeallocating taxonomy...  [5e-06s]\\nTotal time = 0.230358s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000232s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.005051s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000262s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000458s]\\nOpening the output file...  [8.4e-05s]\\nLoading query sequences...  [0.010962s]\\nMasking queries...  [0.057283s]\\nBuilding query seed set...  [0.015739s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.009222s]\\nAllocating buffers...  [3.4e-05s]\\nLoading reference sequences...  [5.9e-05s]\\nBuilding reference histograms...  [0.000218s]\\nAllocating buffers...  [2.5e-05s]\\nInitializing temporary storage...  [0.017834s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001194s]\\nBuilding query index...  [0.008448s]\\nBuilding seed filter...  [0.000862s]\\nSearching alignments...  [0.001012s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000526s]\\nBuilding query index...  [0.010179s]\\nBuilding seed filter...  [0.000891s]\\nSearching alignments...  [0.00103s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000607s]\\nBuilding query index...  [0.010779s]\\nBuilding seed filter...  [0.001181s]\\nSearching alignments...  [0.001295s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000555s]\\nBuilding query index...  [0.011977s]\\nBuilding seed filter...  [0.00101s]\\nSearching alignments...  [0.001075s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000482s]\\nBuilding query index...  [0.007091s]\\nBuilding seed filter...  [0.000798s]\\nSearching alignments...  [0.001069s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000465s]\\nBuilding query index...  [0.010807s]\\nBuilding seed filter...  [0.001496s]\\nSearching alignments...  [0.001103s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.00053s]\\nBuilding query index...  [0.017869s]\\nBuilding seed filter...  [0.000967s]\\nSearching alignments...  [0.001201s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000531s]\\nBuilding query index...  [0.012404s]\\nBuilding seed filter...  [0.001429s]\\nSearching alignments...  [0.001317s]\\nDeallocating buffers...  [2.4e-05s]\\nComputing alignments...  [0.01158s]\\nDeallocating reference...  [8e-06s]\\nLoading reference sequences...  [2.1e-05s]\\nDeallocating buffers...  [9e-06s]\\nDeallocating queries...  [0.00024s]\\nLoading query sequences...  [1.7e-05s]\\nClosing the input file...  [2.1e-05s]\\nClosing the output file...  [0.000313s]\\nClosing the database file...  [1.6e-05s]\\nDeallocating taxonomy...  [4e-06s]\\nTotal time = 0.237977s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000208s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000749s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000203s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.00023s]\\nOpening the output file...  [0.000166s]\\nLoading query sequences...  [0.013671s]\\nMasking queries...  [0.064618s]\\nBuilding query seed set...  [0.015588s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.010563s]\\nAllocating buffers...  [3.8e-05s]\\nLoading reference sequences...  [0.00011s]\\nBuilding reference histograms...  [0.000246s]\\nAllocating buffers...  [2.2e-05s]\\nInitializing temporary storage...  [0.034667s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001195s]\\nBuilding query index...  [0.008803s]\\nBuilding seed filter...  [0.000991s]\\nSearching alignments...  [0.001213s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000558s]\\nBuilding query index...  [0.008154s]\\nBuilding seed filter...  [0.001256s]\\nSearching alignments...  [0.003405s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000785s]\\nBuilding query index...  [0.014908s]\\nBuilding seed filter...  [0.000967s]\\nSearching alignments...  [0.001287s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000682s]\\nBuilding query index...  [0.007691s]\\nBuilding seed filter...  [0.000862s]\\nSearching alignments...  [0.00089s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000584s]\\nBuilding query index...  [0.007677s]\\nBuilding seed filter...  [0.001066s]\\nSearching alignments...  [0.001037s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000531s]\\nBuilding query index...  [0.007941s]\\nBuilding seed filter...  [0.000798s]\\nSearching alignments...  [0.001113s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000498s]\\nBuilding query index...  [0.008507s]\\nBuilding seed filter...  [0.000849s]\\nSearching alignments...  [0.001003s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000494s]\\nBuilding query index...  [0.007306s]\\nBuilding seed filter...  [0.000757s]\\nSearching alignments...  [0.000956s]\\nDeallocating buffers...  [4e-06s]\\nComputing alignments...  [0.009659s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [1.9e-05s]\\nDeallocating buffers...  [8e-06s]\\nDeallocating queries...  [0.000238s]\\nLoading query sequences...  [1.4e-05s]\\nClosing the input file...  [1.6e-05s]\\nClosing the output file...  [0.000217s]\\nClosing the database file...  [1.4e-05s]\\nDeallocating taxonomy...  [4.8e-05s]\\nTotal time = 0.246421s\\nReported 20 pairwise alignments, 20 HSPs.\\n20 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.00014s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 20\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.009788s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000183s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000144s]\\nError: Error detecting input file format. First line seems to be blank.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs... No such file or directory\\n [0.000107s]\\nError: Error opening file /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond/Fischerella_NHAJ01.matches.daa\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000168s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.0001s]\\nError: Error detecting input file format. First line seems to be blank.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs... No such file or directory\\n [8.8e-05s]\\nError: Error opening file /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond/Fischerella_NHAJ01.matches.daa\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000103s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [6.9e-05s]\\nError: Error detecting input file format. First line seems to be blank.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs... No such file or directory\\n [0.00013s]\\nError: Error opening file /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond/Fischerella_NHAJ01.matches.daa\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000149s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [9.6e-05s]\\nError: Error detecting input file format. First line seems to be blank.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs... No such file or directory\\n [0.000152s]\\nError: Error opening file /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond/Fischerella_NHAJ01.matches.daa\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.00022s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000137s]\\nError: Error detecting input file format. First line seems to be blank.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs... No such file or directory\\n [0.0001s]\\nError: Error opening file /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond/Fischerella_NHAJ01.matches.daa\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000184s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000395s]\\nOpening the output file...  [0.000108s]\\nLoading query sequences...  [0.010801s]\\nMasking queries...  [0.057941s]\\nBuilding query seed set...  [0.020979s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.014956s]\\nAllocating buffers...  [2.7e-05s]\\nLoading reference sequences...  [5.2e-05s]\\nBuilding reference histograms...  [0.000233s]\\nAllocating buffers...  [2.9e-05s]\\nInitializing temporary storage...  [0.017862s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001211s]\\nBuilding query index...  [0.011644s]\\nBuilding seed filter...  [0.000937s]\\nSearching alignments...  [0.001222s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000527s]\\nBuilding query index...  [0.011336s]\\nBuilding seed filter...  [0.000877s]\\nSearching alignments...  [0.000962s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000525s]\\nBuilding query index...  [0.010589s]\\nBuilding seed filter...  [0.000968s]\\nSearching alignments...  [0.001089s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000485s]\\nBuilding query index...  [0.008351s]\\nBuilding seed filter...  [0.000762s]\\nSearching alignments...  [0.001064s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000556s]\\nBuilding query index...  [0.007942s]\\nBuilding seed filter...  [0.000993s]\\nSearching alignments...  [0.001329s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000539s]\\nBuilding query index...  [0.007919s]\\nBuilding seed filter...  [0.000953s]\\nSearching alignments...  [0.00121s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000459s]\\nBuilding query index...  [0.008314s]\\nBuilding seed filter...  [0.001069s]\\nSearching alignments...  [0.000878s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000512s]\\nBuilding query index...  [0.007019s]\\nBuilding seed filter...  [0.000825s]\\nSearching alignments...  [0.001007s]\\nDeallocating buffers...  [4e-06s]\\nComputing alignments...  [0.013527s]\\nDeallocating reference...  [7e-06s]\\nLoading reference sequences...  [2.1e-05s]\\nDeallocating buffers...  [8e-06s]\\nDeallocating queries...  [0.000245s]\\nLoading query sequences...  [2.7e-05s]\\nClosing the input file...  [1.8e-05s]\\nClosing the output file...  [0.000274s]\\nClosing the database file...  [1.9e-05s]\\nDeallocating taxonomy...  [5e-06s]\\nTotal time = 0.233347s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000155s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.012098s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000154s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000246s]\\nOpening the output file...  [0.000129s]\\nLoading query sequences...  [0.015044s]\\nMasking queries...  [0.068819s]\\nBuilding query seed set...  [0.022955s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.018489s]\\nAllocating buffers...  [3.5e-05s]\\nLoading reference sequences...  [0.000191s]\\nBuilding reference histograms...  [0.001764s]\\nAllocating buffers...  [2.2e-05s]\\nInitializing temporary storage...  [0.020998s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001262s]\\nBuilding query index...  [0.015266s]\\nBuilding seed filter...  [0.001107s]\\nSearching alignments...  [0.001172s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000497s]\\nBuilding query index...  [0.012413s]\\nBuilding seed filter...  [0.000913s]\\nSearching alignments...  [0.000972s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000435s]\\nBuilding query index...  [0.01285s]\\nBuilding seed filter...  [0.000935s]\\nSearching alignments...  [0.001144s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000457s]\\nBuilding query index...  [0.010786s]\\nBuilding seed filter...  [0.000836s]\\nSearching alignments...  [0.000991s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000528s]\\nBuilding query index...  [0.009339s]\\nBuilding seed filter...  [0.00087s]\\nSearching alignments...  [0.000972s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000398s]\\nBuilding query index...  [0.009515s]\\nBuilding seed filter...  [0.001005s]\\nSearching alignments...  [0.001113s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000474s]\\nBuilding query index...  [0.010039s]\\nBuilding seed filter...  [0.001695s]\\nSearching alignments...  [0.00105s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000463s]\\nBuilding query index...  [0.008836s]\\nBuilding seed filter...  [0.001148s]\\nSearching alignments...  [0.000988s]\\nDeallocating buffers...  [1.6e-05s]\\nComputing alignments...  [0.014803s]\\nDeallocating reference...  [7e-06s]\\nLoading reference sequences...  [2.2e-05s]\\nDeallocating buffers...  [0.000297s]\\nDeallocating queries...  [2.9e-05s]\\nLoading query sequences...  [1.2e-05s]\\nClosing the input file...  [1.4e-05s]\\nClosing the output file...  [0.001366s]\\nClosing the database file...  [1.5e-05s]\\nDeallocating taxonomy...  [4e-06s]\\nTotal time = 0.277667s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000205s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000911s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000151s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000393s]\\nOpening the output file...  [8.6e-05s]\\nLoading query sequences...  [0.012817s]\\nMasking queries...  [0.062711s]\\nBuilding query seed set...  [0.021177s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.012885s]\\nAllocating buffers...  [2.7e-05s]\\nLoading reference sequences...  [4.9e-05s]\\nBuilding reference histograms...  [0.000227s]\\nAllocating buffers...  [1.7e-05s]\\nInitializing temporary storage...  [0.027117s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001039s]\\nBuilding query index...  [0.010443s]\\nBuilding seed filter...  [0.000797s]\\nSearching alignments...  [0.001013s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000427s]\\nBuilding query index...  [0.008452s]\\nBuilding seed filter...  [0.000848s]\\nSearching alignments...  [0.000985s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000458s]\\nBuilding query index...  [0.010444s]\\nBuilding seed filter...  [0.000954s]\\nSearching alignments...  [0.001049s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.00045s]\\nBuilding query index...  [0.012866s]\\nBuilding seed filter...  [0.000911s]\\nSearching alignments...  [0.001082s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.0005s]\\nBuilding query index...  [0.008299s]\\nBuilding seed filter...  [0.000913s]\\nSearching alignments...  [0.000981s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000463s]\\nBuilding query index...  [0.008811s]\\nBuilding seed filter...  [0.000889s]\\nSearching alignments...  [0.00096s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000502s]\\nBuilding query index...  [0.008581s]\\nBuilding seed filter...  [0.001077s]\\nSearching alignments...  [0.000951s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000508s]\\nBuilding query index...  [0.007994s]\\nBuilding seed filter...  [0.001002s]\\nSearching alignments...  [0.000928s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.010564s]\\nDeallocating reference...  [1.7e-05s]\\nLoading reference sequences...  [1.6e-05s]\\nDeallocating buffers...  [0.000277s]\\nDeallocating queries...  [0.000281s]\\nLoading query sequences...  [1.3e-05s]\\nClosing the input file...  [4e-05s]\\nClosing the output file...  [0.000148s]\\nClosing the database file...  [1.1e-05s]\\nDeallocating taxonomy...  [1.5e-05s]\\nTotal time = 0.245861s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000149s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000734s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000229s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000257s]\\nOpening the output file...  [0.000109s]\\nLoading query sequences...  [0.010175s]\\nMasking queries...  [0.059402s]\\nBuilding query seed set...  [0.016043s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.009235s]\\nAllocating buffers...  [4.9e-05s]\\nLoading reference sequences...  [5.4e-05s]\\nBuilding reference histograms...  [0.000251s]\\nAllocating buffers...  [3.5e-05s]\\nInitializing temporary storage...  [0.017478s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001097s]\\nBuilding query index...  [0.008332s]\\nBuilding seed filter...  [0.00079s]\\nSearching alignments...  [0.000986s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000475s]\\nBuilding query index...  [0.007522s]\\nBuilding seed filter...  [0.000867s]\\nSearching alignments...  [0.00099s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000474s]\\nBuilding query index...  [0.008165s]\\nBuilding seed filter...  [0.00105s]\\nSearching alignments...  [0.000963s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000452s]\\nBuilding query index...  [0.006934s]\\nBuilding seed filter...  [0.000755s]\\nSearching alignments...  [0.000974s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000472s]\\nBuilding query index...  [0.007227s]\\nBuilding seed filter...  [0.000932s]\\nSearching alignments...  [0.00124s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000575s]\\nBuilding query index...  [0.012442s]\\nBuilding seed filter...  [0.001099s]\\nSearching alignments...  [0.000946s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.00049s]\\nBuilding query index...  [0.008166s]\\nBuilding seed filter...  [0.000729s]\\nSearching alignments...  [0.000983s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000534s]\\nBuilding query index...  [0.006815s]\\nBuilding seed filter...  [0.000756s]\\nSearching alignments...  [0.000908s]\\nDeallocating buffers...  [4e-06s]\\nComputing alignments...  [0.01443s]\\nDeallocating reference...  [5e-06s]\\nLoading reference sequences...  [1.5e-05s]\\nDeallocating buffers...  [5e-06s]\\nDeallocating queries...  [0.000213s]\\nLoading query sequences...  [4.3e-05s]\\nClosing the input file...  [1.6e-05s]\\nClosing the output file...  [0.000299s]\\nClosing the database file...  [1.8e-05s]\\nDeallocating taxonomy...  [4e-06s]\\nTotal time = 0.21492s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000168s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000859s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000103s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000192s]\\nOpening the output file...  [0.000128s]\\nLoading query sequences...  [0.015327s]\\nMasking queries...  [0.058463s]\\nBuilding query seed set...  [0.019922s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.009778s]\\nAllocating buffers...  [2.9e-05s]\\nLoading reference sequences...  [4.9e-05s]\\nBuilding reference histograms...  [0.000249s]\\nAllocating buffers...  [2.2e-05s]\\nInitializing temporary storage...  [0.019371s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.002901s]\\nBuilding query index...  [0.010211s]\\nBuilding seed filter...  [0.000854s]\\nSearching alignments...  [0.001086s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000513s]\\nBuilding query index...  [0.012064s]\\nBuilding seed filter...  [0.000799s]\\nSearching alignments...  [0.000968s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000428s]\\nBuilding query index...  [0.010136s]\\nBuilding seed filter...  [0.001282s]\\nSearching alignments...  [0.001012s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000422s]\\nBuilding query index...  [0.009673s]\\nBuilding seed filter...  [0.00119s]\\nSearching alignments...  [0.0011s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000486s]\\nBuilding query index...  [0.012147s]\\nBuilding seed filter...  [0.000853s]\\nSearching alignments...  [0.00111s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000546s]\\nBuilding query index...  [0.009381s]\\nBuilding seed filter...  [0.000797s]\\nSearching alignments...  [0.000988s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000409s]\\nBuilding query index...  [0.011695s]\\nBuilding seed filter...  [0.000781s]\\nSearching alignments...  [0.001029s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000429s]\\nBuilding query index...  [0.008732s]\\nBuilding seed filter...  [0.000877s]\\nSearching alignments...  [0.001003s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.013658s]\\nDeallocating reference...  [1e-05s]\\nLoading reference sequences...  [2.1e-05s]\\nDeallocating buffers...  [9e-06s]\\nDeallocating queries...  [0.000243s]\\nLoading query sequences...  [3.2e-05s]\\nClosing the input file...  [3e-05s]\\nClosing the output file...  [0.000222s]\\nClosing the database file...  [1.5e-05s]\\nDeallocating taxonomy...  [6e-06s]\\nTotal time = 0.245173s\\nReported 17 pairwise alignments, 17 HSPs.\\n17 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000218s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 17\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.007025s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000179s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000229s]\\nOpening the output file...  [0.000123s]\\nLoading query sequences...  [0.012248s]\\nMasking queries...  [0.07284s]\\nBuilding query seed set...  [0.020651s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.011542s]\\nAllocating buffers...  [3.6e-05s]\\nLoading reference sequences...  [6.3e-05s]\\nBuilding reference histograms...  [0.000356s]\\nAllocating buffers...  [2.3e-05s]\\nInitializing temporary storage...  [0.019763s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.002438s]\\nBuilding query index...  [0.017331s]\\nBuilding seed filter...  [0.001501s]\\nSearching alignments...  [0.001165s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000504s]\\nBuilding query index...  [0.014787s]\\nBuilding seed filter...  [0.001402s]\\nSearching alignments...  [0.00104s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000521s]\\nBuilding query index...  [0.014513s]\\nBuilding seed filter...  [0.001092s]\\nSearching alignments...  [0.001092s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000479s]\\nBuilding query index...  [0.009712s]\\nBuilding seed filter...  [0.000975s]\\nSearching alignments...  [0.000906s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000494s]\\nBuilding query index...  [0.011298s]\\nBuilding seed filter...  [0.000894s]\\nSearching alignments...  [0.00109s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000468s]\\nBuilding query index...  [0.01064s]\\nBuilding seed filter...  [0.0009s]\\nSearching alignments...  [0.001175s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000538s]\\nBuilding query index...  [0.009443s]\\nBuilding seed filter...  [0.000965s]\\nSearching alignments...  [0.001015s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000625s]\\nBuilding query index...  [0.008627s]\\nBuilding seed filter...  [0.000846s]\\nSearching alignments...  [0.000892s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.011129s]\\nDeallocating reference...  [4e-06s]\\nLoading reference sequences...  [1.7e-05s]\\nDeallocating buffers...  [0.000311s]\\nDeallocating queries...  [2.6e-05s]\\nLoading query sequences...  [1.1e-05s]\\nClosing the input file...  [3e-05s]\\nClosing the output file...  [0.000174s]\\nClosing the database file...  [1e-05s]\\nDeallocating taxonomy...  [4e-06s]\\nTotal time = 0.270555s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000196s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.00086s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.00016s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000276s]\\nOpening the output file...  [0.000113s]\\nLoading query sequences...  [0.009913s]\\nMasking queries...  [0.056182s]\\nBuilding query seed set...  [0.017344s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.010025s]\\nAllocating buffers...  [3.8e-05s]\\nLoading reference sequences...  [4.5e-05s]\\nBuilding reference histograms...  [0.000261s]\\nAllocating buffers...  [1.6e-05s]\\nInitializing temporary storage...  [0.019042s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001204s]\\nBuilding query index...  [0.008278s]\\nBuilding seed filter...  [0.000874s]\\nSearching alignments...  [0.001281s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000472s]\\nBuilding query index...  [0.010542s]\\nBuilding seed filter...  [0.000809s]\\nSearching alignments...  [0.001119s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000459s]\\nBuilding query index...  [0.008495s]\\nBuilding seed filter...  [0.000747s]\\nSearching alignments...  [0.000971s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000496s]\\nBuilding query index...  [0.012211s]\\nBuilding seed filter...  [0.002988s]\\nSearching alignments...  [0.002053s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000679s]\\nBuilding query index...  [0.012297s]\\nBuilding seed filter...  [0.001834s]\\nSearching alignments...  [0.001199s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000698s]\\nBuilding query index...  [0.011081s]\\nBuilding seed filter...  [0.000827s]\\nSearching alignments...  [0.001027s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000527s]\\nBuilding query index...  [0.011281s]\\nBuilding seed filter...  [0.000949s]\\nSearching alignments...  [0.001004s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000505s]\\nBuilding query index...  [0.009542s]\\nBuilding seed filter...  [0.000818s]\\nSearching alignments...  [0.000973s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.010299s]\\nDeallocating reference...  [1.7e-05s]\\nLoading reference sequences...  [1.7e-05s]\\nDeallocating buffers...  [7e-06s]\\nDeallocating queries...  [0.000143s]\\nLoading query sequences...  [2.3e-05s]\\nClosing the input file...  [1.4e-05s]\\nClosing the output file...  [0.000184s]\\nClosing the database file...  [1.1e-05s]\\nDeallocating taxonomy...  [1.3e-05s]\\nTotal time = 0.233775s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000126s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000818s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000157s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000484s]\\nOpening the output file...  [0.000166s]\\nLoading query sequences...  [0.011556s]\\nMasking queries...  [0.057618s]\\nBuilding query seed set...  [0.013525s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.013075s]\\nAllocating buffers...  [3.8e-05s]\\nLoading reference sequences...  [5.8e-05s]\\nBuilding reference histograms...  [0.000241s]\\nAllocating buffers...  [2.4e-05s]\\nInitializing temporary storage...  [0.02182s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001192s]\\nBuilding query index...  [0.008555s]\\nBuilding seed filter...  [0.000792s]\\nSearching alignments...  [0.001017s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000481s]\\nBuilding query index...  [0.009303s]\\nBuilding seed filter...  [0.000838s]\\nSearching alignments...  [0.001026s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000459s]\\nBuilding query index...  [0.009115s]\\nBuilding seed filter...  [0.000884s]\\nSearching alignments...  [0.000963s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000496s]\\nBuilding query index...  [0.008378s]\\nBuilding seed filter...  [0.000793s]\\nSearching alignments...  [0.001042s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000446s]\\nBuilding query index...  [0.010158s]\\nBuilding seed filter...  [0.001023s]\\nSearching alignments...  [0.001099s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000474s]\\nBuilding query index...  [0.012504s]\\nBuilding seed filter...  [0.000784s]\\nSearching alignments...  [0.00103s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000504s]\\nBuilding query index...  [0.008245s]\\nBuilding seed filter...  [0.000821s]\\nSearching alignments...  [0.000962s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000517s]\\nBuilding query index...  [0.008605s]\\nBuilding seed filter...  [0.000775s]\\nSearching alignments...  [0.000946s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.013761s]\\nDeallocating reference...  [1.5e-05s]\\nLoading reference sequences...  [1.7e-05s]\\nDeallocating buffers...  [6e-06s]\\nDeallocating queries...  [0.000178s]\\nLoading query sequences...  [1e-05s]\\nClosing the input file...  [1.4e-05s]\\nClosing the output file...  [0.000176s]\\nClosing the database file...  [1e-05s]\\nDeallocating taxonomy...  [3e-06s]\\nTotal time = 0.228301s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000239s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000969s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000131s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000265s]\\nOpening the output file...  [0.000109s]\\nLoading query sequences...  [0.010628s]\\nMasking queries...  [0.059874s]\\nBuilding query seed set...  [0.019528s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.010523s]\\nAllocating buffers...  [3.7e-05s]\\nLoading reference sequences...  [6e-05s]\\nBuilding reference histograms...  [0.000246s]\\nAllocating buffers...  [3e-05s]\\nInitializing temporary storage...  [0.018217s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001213s]\\nBuilding query index...  [0.011088s]\\nBuilding seed filter...  [0.001018s]\\nSearching alignments...  [0.001207s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000527s]\\nBuilding query index...  [0.012249s]\\nBuilding seed filter...  [0.000898s]\\nSearching alignments...  [0.001115s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000443s]\\nBuilding query index...  [0.012898s]\\nBuilding seed filter...  [0.001096s]\\nSearching alignments...  [0.001173s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000469s]\\nBuilding query index...  [0.012302s]\\nBuilding seed filter...  [0.001035s]\\nSearching alignments...  [0.001228s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000531s]\\nBuilding query index...  [0.014546s]\\nBuilding seed filter...  [0.00134s]\\nSearching alignments...  [0.001523s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000462s]\\nBuilding query index...  [0.010757s]\\nBuilding seed filter...  [0.00078s]\\nSearching alignments...  [0.000937s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000482s]\\nBuilding query index...  [0.010406s]\\nBuilding seed filter...  [0.000794s]\\nSearching alignments...  [0.001047s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.00048s]\\nBuilding query index...  [0.007371s]\\nBuilding seed filter...  [0.000839s]\\nSearching alignments...  [0.00118s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.01082s]\\nDeallocating reference...  [5e-06s]\\nLoading reference sequences...  [1.7e-05s]\\nDeallocating buffers...  [5e-06s]\\nDeallocating queries...  [0.00022s]\\nLoading query sequences...  [1.4e-05s]\\nClosing the input file...  [2.3e-05s]\\nClosing the output file...  [0.000189s]\\nClosing the database file...  [1.1e-05s]\\nDeallocating taxonomy...  [1.4e-05s]\\nTotal time = 0.24553s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000221s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000759s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.00022s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000216s]\\nOpening the output file...  [0.000114s]\\nLoading query sequences...  [0.014729s]\\nMasking queries...  [0.082455s]\\nBuilding query seed set...  [0.01524s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.010549s]\\nAllocating buffers...  [0.000114s]\\nLoading reference sequences...  [5.9e-05s]\\nBuilding reference histograms...  [0.000409s]\\nAllocating buffers...  [2.5e-05s]\\nInitializing temporary storage...  [0.019703s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001278s]\\nBuilding query index...  [0.009775s]\\nBuilding seed filter...  [0.000995s]\\nSearching alignments...  [0.00129s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000644s]\\nBuilding query index...  [0.013466s]\\nBuilding seed filter...  [0.001385s]\\nSearching alignments...  [0.001357s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000664s]\\nBuilding query index...  [0.012574s]\\nBuilding seed filter...  [0.001063s]\\nSearching alignments...  [0.001088s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000545s]\\nBuilding query index...  [0.008042s]\\nBuilding seed filter...  [0.000926s]\\nSearching alignments...  [0.00094s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000443s]\\nBuilding query index...  [0.01045s]\\nBuilding seed filter...  [0.00095s]\\nSearching alignments...  [0.001106s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000429s]\\nBuilding query index...  [0.008775s]\\nBuilding seed filter...  [0.001041s]\\nSearching alignments...  [0.001094s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000551s]\\nBuilding query index...  [0.010997s]\\nBuilding seed filter...  [0.001193s]\\nSearching alignments...  [0.00099s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000489s]\\nBuilding query index...  [0.010663s]\\nBuilding seed filter...  [0.000861s]\\nSearching alignments...  [0.000997s]\\nDeallocating buffers...  [4e-06s]\\nComputing alignments...  [0.009241s]\\nDeallocating reference...  [1e-05s]\\nLoading reference sequences...  [2.1e-05s]\\nDeallocating buffers...  [0.000352s]\\nDeallocating queries...  [0.000133s]\\nLoading query sequences...  [1.6e-05s]\\nClosing the input file...  [1.6e-05s]\\nClosing the output file...  [0.000181s]\\nClosing the database file...  [1.8e-05s]\\nDeallocating taxonomy...  [2e-05s]\\nTotal time = 0.264495s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000193s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001368s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000229s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000461s]\\nOpening the output file...  [9.4e-05s]\\nLoading query sequences...  [0.013631s]\\nMasking queries...  [0.078391s]\\nBuilding query seed set...  [0.017606s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.015368s]\\nAllocating buffers...  [4.4e-05s]\\nLoading reference sequences...  [6.4e-05s]\\nBuilding reference histograms...  [0.000307s]\\nAllocating buffers...  [2e-05s]\\nInitializing temporary storage...  [0.020243s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001101s]\\nBuilding query index...  [0.009328s]\\nBuilding seed filter...  [0.000849s]\\nSearching alignments...  [0.000979s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000441s]\\nBuilding query index...  [0.009964s]\\nBuilding seed filter...  [0.001118s]\\nSearching alignments...  [0.001155s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000546s]\\nBuilding query index...  [0.016176s]\\nBuilding seed filter...  [0.000821s]\\nSearching alignments...  [0.000972s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000432s]\\nBuilding query index...  [0.008543s]\\nBuilding seed filter...  [0.000836s]\\nSearching alignments...  [0.001031s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000435s]\\nBuilding query index...  [0.00841s]\\nBuilding seed filter...  [0.000861s]\\nSearching alignments...  [0.001003s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000469s]\\nBuilding query index...  [0.009074s]\\nBuilding seed filter...  [0.000858s]\\nSearching alignments...  [0.000948s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000483s]\\nBuilding query index...  [0.011453s]\\nBuilding seed filter...  [0.00085s]\\nSearching alignments...  [0.000998s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000433s]\\nBuilding query index...  [0.008278s]\\nBuilding seed filter...  [0.000799s]\\nSearching alignments...  [0.00096s]\\nDeallocating buffers...  [4e-06s]\\nComputing alignments...  [0.011125s]\\nDeallocating reference...  [1.8e-05s]\\nLoading reference sequences...  [2.1e-05s]\\nDeallocating buffers...  [0.000357s]\\nDeallocating queries...  [0.000101s]\\nLoading query sequences...  [1.1e-05s]\\nClosing the input file...  [3.9e-05s]\\nClosing the output file...  [0.000218s]\\nClosing the database file...  [3.3e-05s]\\nDeallocating taxonomy...  [1.4e-05s]\\nTotal time = 0.26023s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000185s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000784s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000149s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000396s]\\nOpening the output file...  [7.2e-05s]\\nLoading query sequences...  [0.013716s]\\nMasking queries...  [0.070346s]\\nBuilding query seed set...  [0.015795s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.01227s]\\nAllocating buffers...  [2.6e-05s]\\nLoading reference sequences...  [4.3e-05s]\\nBuilding reference histograms...  [0.000243s]\\nAllocating buffers...  [1.9e-05s]\\nInitializing temporary storage...  [0.026152s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.00128s]\\nBuilding query index...  [0.015156s]\\nBuilding seed filter...  [0.001033s]\\nSearching alignments...  [0.001336s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000526s]\\nBuilding query index...  [0.01239s]\\nBuilding seed filter...  [0.00135s]\\nSearching alignments...  [0.00124s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000573s]\\nBuilding query index...  [0.017632s]\\nBuilding seed filter...  [0.000977s]\\nSearching alignments...  [0.001278s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.00063s]\\nBuilding query index...  [0.010964s]\\nBuilding seed filter...  [0.000911s]\\nSearching alignments...  [0.000996s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000455s]\\nBuilding query index...  [0.009342s]\\nBuilding seed filter...  [0.00101s]\\nSearching alignments...  [0.001139s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000601s]\\nBuilding query index...  [0.010834s]\\nBuilding seed filter...  [0.001025s]\\nSearching alignments...  [0.000997s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000516s]\\nBuilding query index...  [0.016303s]\\nBuilding seed filter...  [0.001709s]\\nSearching alignments...  [0.000957s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000478s]\\nBuilding query index...  [0.012085s]\\nBuilding seed filter...  [0.000964s]\\nSearching alignments...  [0.001076s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.013267s]\\nDeallocating reference...  [7e-06s]\\nLoading reference sequences...  [2e-05s]\\nDeallocating buffers...  [0.000473s]\\nDeallocating queries...  [0.0005s]\\nLoading query sequences...  [1.8e-05s]\\nClosing the input file...  [1.9e-05s]\\nClosing the output file...  [0.000244s]\\nClosing the database file...  [1.5e-05s]\\nDeallocating taxonomy...  [1.6e-05s]\\nTotal time = 0.282926s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000159s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.00068s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000145s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000243s]\\nOpening the output file...  [0.000112s]\\nLoading query sequences...  [0.012561s]\\nMasking queries...  [0.050152s]\\nBuilding query seed set...  [0.010907s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.008696s]\\nAllocating buffers...  [4.4e-05s]\\nLoading reference sequences...  [8.6e-05s]\\nBuilding reference histograms...  [0.000265s]\\nAllocating buffers...  [2.2e-05s]\\nInitializing temporary storage...  [0.016935s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001344s]\\nBuilding query index...  [0.007428s]\\nBuilding seed filter...  [0.000882s]\\nSearching alignments...  [0.001153s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000492s]\\nBuilding query index...  [0.007591s]\\nBuilding seed filter...  [0.000814s]\\nSearching alignments...  [0.001056s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000461s]\\nBuilding query index...  [0.007606s]\\nBuilding seed filter...  [0.000787s]\\nSearching alignments...  [0.00112s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000544s]\\nBuilding query index...  [0.006801s]\\nBuilding seed filter...  [0.000787s]\\nSearching alignments...  [0.001169s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000447s]\\nBuilding query index...  [0.006951s]\\nBuilding seed filter...  [0.000824s]\\nSearching alignments...  [0.001121s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000482s]\\nBuilding query index...  [0.007533s]\\nBuilding seed filter...  [0.000745s]\\nSearching alignments...  [0.000976s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000461s]\\nBuilding query index...  [0.007472s]\\nBuilding seed filter...  [0.000908s]\\nSearching alignments...  [0.001026s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000491s]\\nBuilding query index...  [0.008809s]\\nBuilding seed filter...  [0.000873s]\\nSearching alignments...  [0.001297s]\\nDeallocating buffers...  [2.1e-05s]\\nComputing alignments...  [0.014444s]\\nDeallocating reference...  [8e-06s]\\nLoading reference sequences...  [2.1e-05s]\\nDeallocating buffers...  [7e-06s]\\nDeallocating queries...  [0.000239s]\\nLoading query sequences...  [1.9e-05s]\\nClosing the input file...  [2e-05s]\\nClosing the output file...  [0.000245s]\\nClosing the database file...  [1.4e-05s]\\nDeallocating taxonomy...  [6e-06s]\\nTotal time = 0.196879s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000177s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000944s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000143s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000298s]\\nOpening the output file...  [0.005554s]\\nLoading query sequences...  [0.014809s]\\nMasking queries...  [0.064815s]\\nBuilding query seed set...  [0.01363s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.012979s]\\nAllocating buffers...  [3.7e-05s]\\nLoading reference sequences...  [5.7e-05s]\\nBuilding reference histograms...  [0.000271s]\\nAllocating buffers...  [2.8e-05s]\\nInitializing temporary storage...  [0.127161s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001255s]\\nBuilding query index...  [0.009845s]\\nBuilding seed filter...  [0.000869s]\\nSearching alignments...  [0.001213s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000519s]\\nBuilding query index...  [0.012343s]\\nBuilding seed filter...  [0.000915s]\\nSearching alignments...  [0.001038s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000547s]\\nBuilding query index...  [0.011875s]\\nBuilding seed filter...  [0.000775s]\\nSearching alignments...  [0.000932s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000472s]\\nBuilding query index...  [0.008197s]\\nBuilding seed filter...  [0.000743s]\\nSearching alignments...  [0.001008s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.00042s]\\nBuilding query index...  [0.007221s]\\nBuilding seed filter...  [0.000814s]\\nSearching alignments...  [0.00102s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000473s]\\nBuilding query index...  [0.010751s]\\nBuilding seed filter...  [0.000792s]\\nSearching alignments...  [0.00111s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000487s]\\nBuilding query index...  [0.00984s]\\nBuilding seed filter...  [0.000783s]\\nSearching alignments...  [0.001062s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000458s]\\nBuilding query index...  [0.006952s]\\nBuilding seed filter...  [0.000848s]\\nSearching alignments...  [0.001041s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.012365s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [1.6e-05s]\\nDeallocating buffers...  [6e-06s]\\nDeallocating queries...  [0.000231s]\\nLoading query sequences...  [1.1e-05s]\\nClosing the input file...  [1.6e-05s]\\nClosing the output file...  [0.000223s]\\nClosing the database file...  [1e-05s]\\nDeallocating taxonomy...  [8e-06s]\\nTotal time = 0.350601s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000226s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001005s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000156s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000457s]\\nOpening the output file...  [0.000168s]\\nLoading query sequences...  [0.012043s]\\nMasking queries...  [0.047733s]\\nBuilding query seed set...  [0.015205s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.008585s]\\nAllocating buffers...  [2.6e-05s]\\nLoading reference sequences...  [4.5e-05s]\\nBuilding reference histograms...  [0.000227s]\\nAllocating buffers...  [2.1e-05s]\\nInitializing temporary storage...  [0.017152s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.000999s]\\nBuilding query index...  [0.008177s]\\nBuilding seed filter...  [0.000789s]\\nSearching alignments...  [0.001005s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000485s]\\nBuilding query index...  [0.006392s]\\nBuilding seed filter...  [0.000749s]\\nSearching alignments...  [0.001092s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000469s]\\nBuilding query index...  [0.008674s]\\nBuilding seed filter...  [0.000706s]\\nSearching alignments...  [0.000918s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000455s]\\nBuilding query index...  [0.006121s]\\nBuilding seed filter...  [0.000738s]\\nSearching alignments...  [0.00104s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000445s]\\nBuilding query index...  [0.005909s]\\nBuilding seed filter...  [0.000739s]\\nSearching alignments...  [0.001036s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000414s]\\nBuilding query index...  [0.007062s]\\nBuilding seed filter...  [0.000784s]\\nSearching alignments...  [0.001134s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000472s]\\nBuilding query index...  [0.008093s]\\nBuilding seed filter...  [0.000769s]\\nSearching alignments...  [0.001188s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000475s]\\nBuilding query index...  [0.010143s]\\nBuilding seed filter...  [0.000977s]\\nSearching alignments...  [0.001211s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.009393s]\\nDeallocating reference...  [5e-06s]\\nLoading reference sequences...  [2.4e-05s]\\nDeallocating buffers...  [8e-06s]\\nDeallocating queries...  [0.000143s]\\nLoading query sequences...  [1.2e-05s]\\nClosing the input file...  [1.4e-05s]\\nClosing the output file...  [0.000189s]\\nClosing the database file...  [1.6e-05s]\\nDeallocating taxonomy...  [1.3e-05s]\\nTotal time = 0.192423s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000235s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001153s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000142s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000209s]\\nOpening the output file...  [0.000109s]\\nLoading query sequences...  [0.014933s]\\nMasking queries...  [0.075559s]\\nBuilding query seed set...  [0.016216s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.01208s]\\nAllocating buffers...  [3.8e-05s]\\nLoading reference sequences...  [8.3e-05s]\\nBuilding reference histograms...  [0.000239s]\\nAllocating buffers...  [4.1e-05s]\\nInitializing temporary storage...  [0.018614s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001237s]\\nBuilding query index...  [0.012251s]\\nBuilding seed filter...  [0.000989s]\\nSearching alignments...  [0.00104s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000491s]\\nBuilding query index...  [0.012182s]\\nBuilding seed filter...  [0.001215s]\\nSearching alignments...  [0.00092s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000501s]\\nBuilding query index...  [0.012267s]\\nBuilding seed filter...  [0.00093s]\\nSearching alignments...  [0.001053s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000485s]\\nBuilding query index...  [0.009377s]\\nBuilding seed filter...  [0.000956s]\\nSearching alignments...  [0.001115s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000492s]\\nBuilding query index...  [0.009545s]\\nBuilding seed filter...  [0.001039s]\\nSearching alignments...  [0.001156s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000493s]\\nBuilding query index...  [0.012125s]\\nBuilding seed filter...  [0.000896s]\\nSearching alignments...  [0.00102s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000418s]\\nBuilding query index...  [0.010339s]\\nBuilding seed filter...  [0.001018s]\\nSearching alignments...  [0.000963s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.0005s]\\nBuilding query index...  [0.009872s]\\nBuilding seed filter...  [0.000924s]\\nSearching alignments...  [0.0011s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.012119s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [1.9e-05s]\\nDeallocating buffers...  [0.000332s]\\nDeallocating queries...  [5.2e-05s]\\nLoading query sequences...  [1.4e-05s]\\nClosing the input file...  [1.7e-05s]\\nClosing the output file...  [0.002635s]\\nClosing the database file...  [1.9e-05s]\\nDeallocating taxonomy...  [1.6e-05s]\\nTotal time = 0.26385s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000254s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000956s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000188s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.0004s]\\nOpening the output file...  [9.8e-05s]\\nLoading query sequences...  [0.018604s]\\nMasking queries...  [0.095644s]\\nBuilding query seed set...  [0.031071s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.016444s]\\nAllocating buffers...  [5.1e-05s]\\nLoading reference sequences...  [6e-05s]\\nBuilding reference histograms...  [0.000221s]\\nAllocating buffers...  [2.5e-05s]\\nInitializing temporary storage...  [0.026887s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.00112s]\\nBuilding query index...  [0.011598s]\\nBuilding seed filter...  [0.001094s]\\nSearching alignments...  [0.001105s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000496s]\\nBuilding query index...  [0.015518s]\\nBuilding seed filter...  [0.00122s]\\nSearching alignments...  [0.001093s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000545s]\\nBuilding query index...  [0.014638s]\\nBuilding seed filter...  [0.001116s]\\nSearching alignments...  [0.001074s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000535s]\\nBuilding query index...  [0.010245s]\\nBuilding seed filter...  [0.000925s]\\nSearching alignments...  [0.001072s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000479s]\\nBuilding query index...  [0.011517s]\\nBuilding seed filter...  [0.001593s]\\nSearching alignments...  [0.001164s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000515s]\\nBuilding query index...  [0.012455s]\\nBuilding seed filter...  [0.001034s]\\nSearching alignments...  [0.001088s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000552s]\\nBuilding query index...  [0.012377s]\\nBuilding seed filter...  [0.001144s]\\nSearching alignments...  [0.001111s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000495s]\\nBuilding query index...  [0.010299s]\\nBuilding seed filter...  [0.000985s]\\nSearching alignments...  [0.001083s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.015454s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [1.5e-05s]\\nDeallocating buffers...  [0.000438s]\\nDeallocating queries...  [9.9e-05s]\\nLoading query sequences...  [2.2e-05s]\\nClosing the input file...  [5.1e-05s]\\nClosing the output file...  [0.000197s]\\nClosing the database file...  [4.9e-05s]\\nDeallocating taxonomy...  [2.8e-05s]\\nTotal time = 0.328751s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000138s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001528s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000144s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.00046s]\\nOpening the output file...  [0.000121s]\\nLoading query sequences...  [0.015124s]\\nMasking queries...  [0.076373s]\\nBuilding query seed set...  [0.022345s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.013486s]\\nAllocating buffers...  [2.3e-05s]\\nLoading reference sequences...  [4.6e-05s]\\nBuilding reference histograms...  [0.000209s]\\nAllocating buffers...  [2.6e-05s]\\nInitializing temporary storage...  [0.024468s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001361s]\\nBuilding query index...  [0.019025s]\\nBuilding seed filter...  [0.002617s]\\nSearching alignments...  [0.001681s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000757s]\\nBuilding query index...  [0.018663s]\\nBuilding seed filter...  [0.001262s]\\nSearching alignments...  [0.001488s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000672s]\\nBuilding query index...  [0.01786s]\\nBuilding seed filter...  [0.001155s]\\nSearching alignments...  [0.001202s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.00051s]\\nBuilding query index...  [0.014958s]\\nBuilding seed filter...  [0.001104s]\\nSearching alignments...  [0.001235s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000511s]\\nBuilding query index...  [0.014924s]\\nBuilding seed filter...  [0.001272s]\\nSearching alignments...  [0.001118s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.00046s]\\nBuilding query index...  [0.014442s]\\nBuilding seed filter...  [0.001167s]\\nSearching alignments...  [0.001156s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000532s]\\nBuilding query index...  [0.012861s]\\nBuilding seed filter...  [0.001017s]\\nSearching alignments...  [0.001398s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.00047s]\\nBuilding query index...  [0.012207s]\\nBuilding seed filter...  [0.001184s]\\nSearching alignments...  [0.001119s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.011844s]\\nDeallocating reference...  [2.4e-05s]\\nLoading reference sequences...  [2.3e-05s]\\nDeallocating buffers...  [0.00044s]\\nDeallocating queries...  [0.000375s]\\nLoading query sequences...  [1.7e-05s]\\nClosing the input file...  [2e-05s]\\nClosing the output file...  [0.000289s]\\nClosing the database file...  [1.5e-05s]\\nDeallocating taxonomy...  [5e-06s]\\nTotal time = 0.318716s\\nReported 20 pairwise alignments, 20 HSPs.\\n20 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.00023s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 20\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001002s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000148s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000233s]\\nOpening the output file...  [8.3e-05s]\\nLoading query sequences...  [0.014945s]\\nMasking queries...  [0.081028s]\\nBuilding query seed set...  [0.023002s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.015945s]\\nAllocating buffers...  [4.1e-05s]\\nLoading reference sequences...  [5.9e-05s]\\nBuilding reference histograms...  [0.000239s]\\nAllocating buffers...  [2.6e-05s]\\nInitializing temporary storage...  [0.018291s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001173s]\\nBuilding query index...  [0.015254s]\\nBuilding seed filter...  [0.001324s]\\nSearching alignments...  [0.001337s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000553s]\\nBuilding query index...  [0.015413s]\\nBuilding seed filter...  [0.001001s]\\nSearching alignments...  [0.001059s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000562s]\\nBuilding query index...  [0.01173s]\\nBuilding seed filter...  [0.000984s]\\nSearching alignments...  [0.001016s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000474s]\\nBuilding query index...  [0.012216s]\\nBuilding seed filter...  [0.001166s]\\nSearching alignments...  [0.001087s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000466s]\\nBuilding query index...  [0.013396s]\\nBuilding seed filter...  [0.001054s]\\nSearching alignments...  [0.001191s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000486s]\\nBuilding query index...  [0.014671s]\\nBuilding seed filter...  [0.001271s]\\nSearching alignments...  [0.000998s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.00049s]\\nBuilding query index...  [0.016123s]\\nBuilding seed filter...  [0.000976s]\\nSearching alignments...  [0.001124s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000589s]\\nBuilding query index...  [0.012334s]\\nBuilding seed filter...  [0.001318s]\\nSearching alignments...  [0.001187s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.012422s]\\nDeallocating reference...  [5e-06s]\\nLoading reference sequences...  [1.9e-05s]\\nDeallocating buffers...  [0.000364s]\\nDeallocating queries...  [9.2e-05s]\\nLoading query sequences...  [1.3e-05s]\\nClosing the input file...  [1.5e-05s]\\nClosing the output file...  [0.000153s]\\nClosing the database file...  [1.6e-05s]\\nDeallocating taxonomy...  [1.4e-05s]\\nTotal time = 0.302364s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.00016s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001194s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000114s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000419s]\\nOpening the output file...  [0.000122s]\\nLoading query sequences...  [0.010762s]\\nMasking queries...  [0.058404s]\\nBuilding query seed set...  [0.015832s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.00872s]\\nAllocating buffers...  [3.3e-05s]\\nLoading reference sequences...  [6e-05s]\\nBuilding reference histograms...  [0.000235s]\\nAllocating buffers...  [4.3e-05s]\\nInitializing temporary storage...  [0.017513s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.000996s]\\nBuilding query index...  [0.007441s]\\nBuilding seed filter...  [0.000713s]\\nSearching alignments...  [0.001037s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000416s]\\nBuilding query index...  [0.008781s]\\nBuilding seed filter...  [0.000867s]\\nSearching alignments...  [0.001098s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000498s]\\nBuilding query index...  [0.006649s]\\nBuilding seed filter...  [0.000713s]\\nSearching alignments...  [0.000978s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000464s]\\nBuilding query index...  [0.007743s]\\nBuilding seed filter...  [0.000695s]\\nSearching alignments...  [0.001008s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000484s]\\nBuilding query index...  [0.005895s]\\nBuilding seed filter...  [0.00073s]\\nSearching alignments...  [0.001s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000485s]\\nBuilding query index...  [0.007173s]\\nBuilding seed filter...  [0.000844s]\\nSearching alignments...  [0.001054s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000481s]\\nBuilding query index...  [0.010953s]\\nBuilding seed filter...  [0.000865s]\\nSearching alignments...  [0.000995s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000492s]\\nBuilding query index...  [0.005629s]\\nBuilding seed filter...  [0.000667s]\\nSearching alignments...  [0.001069s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.017154s]\\nDeallocating reference...  [7e-06s]\\nLoading reference sequences...  [2e-05s]\\nDeallocating buffers...  [8e-06s]\\nDeallocating queries...  [0.000352s]\\nLoading query sequences...  [2.7e-05s]\\nClosing the input file...  [5.8e-05s]\\nClosing the output file...  [0.001317s]\\nClosing the database file...  [5.7e-05s]\\nDeallocating taxonomy...  [1.4e-05s]\\nTotal time = 0.211414s\\nReported 27 pairwise alignments, 27 HSPs.\\n27 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000178s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001594s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000101s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000242s]\\nOpening the output file...  [9e-05s]\\nLoading query sequences...  [0.017874s]\\nMasking queries...  [0.097504s]\\nBuilding query seed set...  [0.030087s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.016561s]\\nAllocating buffers...  [2.5e-05s]\\nLoading reference sequences...  [5.1e-05s]\\nBuilding reference histograms...  [0.000231s]\\nAllocating buffers...  [1.6e-05s]\\nInitializing temporary storage...  [0.030937s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001238s]\\nBuilding query index...  [0.013734s]\\nBuilding seed filter...  [0.001126s]\\nSearching alignments...  [0.001103s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.00044s]\\nBuilding query index...  [0.014504s]\\nBuilding seed filter...  [0.001177s]\\nSearching alignments...  [0.001061s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000473s]\\nBuilding query index...  [0.014429s]\\nBuilding seed filter...  [0.00127s]\\nSearching alignments...  [0.001044s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000458s]\\nBuilding query index...  [0.018087s]\\nBuilding seed filter...  [0.001235s]\\nSearching alignments...  [0.001023s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000491s]\\nBuilding query index...  [0.019608s]\\nBuilding seed filter...  [0.001157s]\\nSearching alignments...  [0.001277s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000512s]\\nBuilding query index...  [0.015173s]\\nBuilding seed filter...  [0.001345s]\\nSearching alignments...  [0.001269s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000525s]\\nBuilding query index...  [0.01383s]\\nBuilding seed filter...  [0.001435s]\\nSearching alignments...  [0.00123s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000507s]\\nBuilding query index...  [0.016709s]\\nBuilding seed filter...  [0.001521s]\\nSearching alignments...  [0.001099s]\\nDeallocating buffers...  [7e-06s]\\nComputing alignments...  [0.015075s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [2e-05s]\\nDeallocating buffers...  [0.000505s]\\nDeallocating queries...  [0.000519s]\\nLoading query sequences...  [1.6e-05s]\\nClosing the input file...  [1.5e-05s]\\nClosing the output file...  [0.000201s]\\nClosing the database file...  [1.3e-05s]\\nDeallocating taxonomy...  [2.2e-05s]\\nTotal time = 0.361716s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000197s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001105s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000154s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000392s]\\nOpening the output file...  [0.000108s]\\nLoading query sequences...  [0.016609s]\\nMasking queries...  [0.090775s]\\nBuilding query seed set...  [0.024461s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.020957s]\\nAllocating buffers...  [2.6e-05s]\\nLoading reference sequences...  [6.2e-05s]\\nBuilding reference histograms...  [0.000209s]\\nAllocating buffers...  [2.5e-05s]\\nInitializing temporary storage...  [0.021779s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001197s]\\nBuilding query index...  [0.014908s]\\nBuilding seed filter...  [0.001021s]\\nSearching alignments...  [0.001114s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000458s]\\nBuilding query index...  [0.011799s]\\nBuilding seed filter...  [0.001888s]\\nSearching alignments...  [0.001125s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000459s]\\nBuilding query index...  [0.012211s]\\nBuilding seed filter...  [0.001212s]\\nSearching alignments...  [0.001057s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000476s]\\nBuilding query index...  [0.013006s]\\nBuilding seed filter...  [0.001085s]\\nSearching alignments...  [0.001039s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000583s]\\nBuilding query index...  [0.016448s]\\nBuilding seed filter...  [0.001128s]\\nSearching alignments...  [0.001065s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000577s]\\nBuilding query index...  [0.018707s]\\nBuilding seed filter...  [0.001195s]\\nSearching alignments...  [0.001029s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000555s]\\nBuilding query index...  [0.017924s]\\nBuilding seed filter...  [0.000974s]\\nSearching alignments...  [0.001053s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000472s]\\nBuilding query index...  [0.011337s]\\nBuilding seed filter...  [0.001093s]\\nSearching alignments...  [0.00097s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.022167s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [1.8e-05s]\\nDeallocating buffers...  [0.000397s]\\nDeallocating queries...  [9.7e-05s]\\nLoading query sequences...  [1.2e-05s]\\nClosing the input file...  [4.7e-05s]\\nClosing the output file...  [0.004333s]\\nClosing the database file...  [2.5e-05s]\\nDeallocating taxonomy...  [4.4e-05s]\\nTotal time = 0.343481s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000165s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.007522s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000116s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000211s]\\nOpening the output file...  [0.006342s]\\nLoading query sequences...  [0.014914s]\\nMasking queries...  [0.089392s]\\nBuilding query seed set...  [0.025627s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.019268s]\\nAllocating buffers...  [2.4e-05s]\\nLoading reference sequences...  [8.6e-05s]\\nBuilding reference histograms...  [0.000293s]\\nAllocating buffers...  [2.7e-05s]\\nInitializing temporary storage...  [0.020204s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001319s]\\nBuilding query index...  [0.015701s]\\nBuilding seed filter...  [0.001352s]\\nSearching alignments...  [0.001127s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000509s]\\nBuilding query index...  [0.020632s]\\nBuilding seed filter...  [0.001055s]\\nSearching alignments...  [0.001069s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.00057s]\\nBuilding query index...  [0.020085s]\\nBuilding seed filter...  [0.001421s]\\nSearching alignments...  [0.001285s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000555s]\\nBuilding query index...  [0.015456s]\\nBuilding seed filter...  [0.001217s]\\nSearching alignments...  [0.001182s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000631s]\\nBuilding query index...  [0.017819s]\\nBuilding seed filter...  [0.001507s]\\nSearching alignments...  [0.001496s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000662s]\\nBuilding query index...  [0.017663s]\\nBuilding seed filter...  [0.001044s]\\nSearching alignments...  [0.001008s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000459s]\\nBuilding query index...  [0.016797s]\\nBuilding seed filter...  [0.001101s]\\nSearching alignments...  [0.001107s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.00062s]\\nBuilding query index...  [0.012705s]\\nBuilding seed filter...  [0.001147s]\\nSearching alignments...  [0.001169s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.022046s]\\nDeallocating reference...  [7e-06s]\\nLoading reference sequences...  [3.9e-05s]\\nDeallocating buffers...  [0.000491s]\\nDeallocating queries...  [0.0003s]\\nLoading query sequences...  [1.7e-05s]\\nClosing the input file...  [5.9e-05s]\\nClosing the output file...  [0.000216s]\\nClosing the database file...  [4.5e-05s]\\nDeallocating taxonomy...  [2.3e-05s]\\nTotal time = 0.362805s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000199s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001003s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000285s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.0003s]\\nOpening the output file...  [0.000127s]\\nLoading query sequences...  [0.021098s]\\nMasking queries...  [0.134778s]\\nBuilding query seed set...  [0.029207s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.015707s]\\nAllocating buffers...  [4.2e-05s]\\nLoading reference sequences...  [6e-05s]\\nBuilding reference histograms...  [0.000194s]\\nAllocating buffers...  [3e-05s]\\nInitializing temporary storage...  [0.018131s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001119s]\\nBuilding query index...  [0.013278s]\\nBuilding seed filter...  [0.001006s]\\nSearching alignments...  [0.001079s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000466s]\\nBuilding query index...  [0.013363s]\\nBuilding seed filter...  [0.001004s]\\nSearching alignments...  [0.001055s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.00051s]\\nBuilding query index...  [0.01348s]\\nBuilding seed filter...  [0.001025s]\\nSearching alignments...  [0.001116s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000433s]\\nBuilding query index...  [0.01358s]\\nBuilding seed filter...  [0.001076s]\\nSearching alignments...  [0.001127s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000563s]\\nBuilding query index...  [0.016867s]\\nBuilding seed filter...  [0.001166s]\\nSearching alignments...  [0.001138s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000507s]\\nBuilding query index...  [0.018338s]\\nBuilding seed filter...  [0.001166s]\\nSearching alignments...  [0.001234s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000478s]\\nBuilding query index...  [0.013722s]\\nBuilding seed filter...  [0.001294s]\\nSearching alignments...  [0.001127s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000528s]\\nBuilding query index...  [0.017374s]\\nBuilding seed filter...  [0.001076s]\\nSearching alignments...  [0.001152s]\\nDeallocating buffers...  [4e-06s]\\nComputing alignments...  [0.013719s]\\nDeallocating reference...  [8e-06s]\\nLoading reference sequences...  [2.2e-05s]\\nDeallocating buffers...  [0.00059s]\\nDeallocating queries...  [0.000585s]\\nLoading query sequences...  [1.6e-05s]\\nClosing the input file...  [1.7e-05s]\\nClosing the output file...  [0.000264s]\\nClosing the database file...  [1.6e-05s]\\nDeallocating taxonomy...  [4e-06s]\\nTotal time = 0.38821s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000222s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001026s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.00022s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.00036s]\\nOpening the output file...  [0.000141s]\\nLoading query sequences...  [0.022933s]\\nMasking queries...  [0.093273s]\\nBuilding query seed set...  [0.023546s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.016278s]\\nAllocating buffers...  [5e-05s]\\nLoading reference sequences...  [5.8e-05s]\\nBuilding reference histograms...  [0.000215s]\\nAllocating buffers...  [2.9e-05s]\\nInitializing temporary storage...  [0.020833s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001223s]\\nBuilding query index...  [0.013714s]\\nBuilding seed filter...  [0.001023s]\\nSearching alignments...  [0.001205s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000543s]\\nBuilding query index...  [0.018864s]\\nBuilding seed filter...  [0.001311s]\\nSearching alignments...  [0.001023s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000458s]\\nBuilding query index...  [0.013217s]\\nBuilding seed filter...  [0.001012s]\\nSearching alignments...  [0.001059s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000495s]\\nBuilding query index...  [0.014453s]\\nBuilding seed filter...  [0.001007s]\\nSearching alignments...  [0.00101s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000507s]\\nBuilding query index...  [0.013552s]\\nBuilding seed filter...  [0.001088s]\\nSearching alignments...  [0.001002s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000478s]\\nBuilding query index...  [0.015658s]\\nBuilding seed filter...  [0.001609s]\\nSearching alignments...  [0.0011s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000519s]\\nBuilding query index...  [0.019533s]\\nBuilding seed filter...  [0.001345s]\\nSearching alignments...  [0.001065s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000574s]\\nBuilding query index...  [0.012527s]\\nBuilding seed filter...  [0.001042s]\\nSearching alignments...  [0.000984s]\\nDeallocating buffers...  [1.6e-05s]\\nComputing alignments...  [0.014341s]\\nDeallocating reference...  [7e-06s]\\nLoading reference sequences...  [3.4e-05s]\\nDeallocating buffers...  [0.000434s]\\nDeallocating queries...  [0.000381s]\\nLoading query sequences...  [1.7e-05s]\\nClosing the input file...  [5.7e-05s]\\nClosing the output file...  [0.000312s]\\nClosing the database file...  [3.7e-05s]\\nDeallocating taxonomy...  [1.8e-05s]\\nTotal time = 0.339309s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000204s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.00117s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000245s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000441s]\\nOpening the output file...  [0.000142s]\\nLoading query sequences...  [0.01845s]\\nMasking queries...  [0.095344s]\\nBuilding query seed set...  [0.026916s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.014196s]\\nAllocating buffers...  [3.7e-05s]\\nLoading reference sequences...  [6e-05s]\\nBuilding reference histograms...  [0.000223s]\\nAllocating buffers...  [1.7e-05s]\\nInitializing temporary storage...  [0.025356s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001266s]\\nBuilding query index...  [0.014193s]\\nBuilding seed filter...  [0.001075s]\\nSearching alignments...  [0.001196s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000614s]\\nBuilding query index...  [0.011913s]\\nBuilding seed filter...  [0.001296s]\\nSearching alignments...  [0.001005s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000547s]\\nBuilding query index...  [0.019413s]\\nBuilding seed filter...  [0.001058s]\\nSearching alignments...  [0.001063s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000582s]\\nBuilding query index...  [0.01488s]\\nBuilding seed filter...  [0.001227s]\\nSearching alignments...  [0.001095s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000539s]\\nBuilding query index...  [0.010923s]\\nBuilding seed filter...  [0.001116s]\\nSearching alignments...  [0.001189s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000559s]\\nBuilding query index...  [0.011959s]\\nBuilding seed filter...  [0.001063s]\\nSearching alignments...  [0.001078s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000624s]\\nBuilding query index...  [0.013724s]\\nBuilding seed filter...  [0.001451s]\\nSearching alignments...  [0.001085s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000482s]\\nBuilding query index...  [0.012682s]\\nBuilding seed filter...  [0.001541s]\\nSearching alignments...  [0.000987s]\\nDeallocating buffers...  [1e-05s]\\nComputing alignments...  [0.01314s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [2.2e-05s]\\nDeallocating buffers...  [0.000549s]\\nDeallocating queries...  [0.000634s]\\nLoading query sequences...  [1.7e-05s]\\nClosing the input file...  [2.4e-05s]\\nClosing the output file...  [0.000253s]\\nClosing the database file...  [1.8e-05s]\\nDeallocating taxonomy...  [1.7e-05s]\\nTotal time = 0.330876s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.00017s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000908s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000196s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000423s]\\nOpening the output file...  [0.000109s]\\nLoading query sequences...  [0.023513s]\\nMasking queries...  [0.112351s]\\nBuilding query seed set...  [0.033248s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.031834s]\\nAllocating buffers...  [8.7e-05s]\\nLoading reference sequences...  [9.6e-05s]\\nBuilding reference histograms...  [0.000479s]\\nAllocating buffers...  [2.5e-05s]\\nInitializing temporary storage...  [0.019486s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001275s]\\nBuilding query index...  [0.014694s]\\nBuilding seed filter...  [0.001113s]\\nSearching alignments...  [0.001144s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000481s]\\nBuilding query index...  [0.020998s]\\nBuilding seed filter...  [0.00112s]\\nSearching alignments...  [0.001138s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000662s]\\nBuilding query index...  [0.019699s]\\nBuilding seed filter...  [0.001487s]\\nSearching alignments...  [0.0012s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000489s]\\nBuilding query index...  [0.012701s]\\nBuilding seed filter...  [0.001409s]\\nSearching alignments...  [0.001074s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000596s]\\nBuilding query index...  [0.013647s]\\nBuilding seed filter...  [0.001278s]\\nSearching alignments...  [0.001411s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000504s]\\nBuilding query index...  [0.014614s]\\nBuilding seed filter...  [0.001143s]\\nSearching alignments...  [0.001152s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000558s]\\nBuilding query index...  [0.022795s]\\nBuilding seed filter...  [0.001453s]\\nSearching alignments...  [0.001158s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.00092s]\\nBuilding query index...  [0.01788s]\\nBuilding seed filter...  [0.001505s]\\nSearching alignments...  [0.001143s]\\nDeallocating buffers...  [1e-05s]\\nComputing alignments...  [0.016355s]\\nDeallocating reference...  [1.9e-05s]\\nLoading reference sequences...  [2.6e-05s]\\nDeallocating buffers...  [0.000685s]\\nDeallocating queries...  [0.000451s]\\nLoading query sequences...  [1.9e-05s]\\nClosing the input file...  [2.1e-05s]\\nClosing the output file...  [0.000251s]\\nClosing the database file...  [1.4e-05s]\\nDeallocating taxonomy...  [1.5e-05s]\\nTotal time = 0.40388s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000142s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001707s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000214s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000259s]\\nOpening the output file...  [0.000148s]\\nLoading query sequences...  [0.024801s]\\nMasking queries...  [0.10035s]\\nBuilding query seed set...  [0.019703s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.01843s]\\nAllocating buffers...  [2.6e-05s]\\nLoading reference sequences...  [5.2e-05s]\\nBuilding reference histograms...  [0.000387s]\\nAllocating buffers...  [1.7e-05s]\\nInitializing temporary storage...  [0.02235s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001402s]\\nBuilding query index...  [0.017058s]\\nBuilding seed filter...  [0.001466s]\\nSearching alignments...  [0.001267s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000587s]\\nBuilding query index...  [0.016291s]\\nBuilding seed filter...  [0.001126s]\\nSearching alignments...  [0.001136s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000503s]\\nBuilding query index...  [0.01411s]\\nBuilding seed filter...  [0.001245s]\\nSearching alignments...  [0.001012s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000446s]\\nBuilding query index...  [0.012735s]\\nBuilding seed filter...  [0.001807s]\\nSearching alignments...  [0.001041s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000516s]\\nBuilding query index...  [0.012652s]\\nBuilding seed filter...  [0.001264s]\\nSearching alignments...  [0.001187s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000537s]\\nBuilding query index...  [0.016967s]\\nBuilding seed filter...  [0.001095s]\\nSearching alignments...  [0.001106s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000519s]\\nBuilding query index...  [0.016184s]\\nBuilding seed filter...  [0.001064s]\\nSearching alignments...  [0.001097s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000442s]\\nBuilding query index...  [0.012104s]\\nBuilding seed filter...  [0.001101s]\\nSearching alignments...  [0.001098s]\\nDeallocating buffers...  [4e-06s]\\nComputing alignments...  [0.014742s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [1.6e-05s]\\nDeallocating buffers...  [0.000466s]\\nDeallocating queries...  [0.000523s]\\nLoading query sequences...  [1.9e-05s]\\nClosing the input file...  [4.9e-05s]\\nClosing the output file...  [0.000248s]\\nClosing the database file...  [2e-05s]\\nDeallocating taxonomy...  [3e-06s]\\nTotal time = 0.347155s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000163s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000768s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000141s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000373s]\\nOpening the output file...  [8.7e-05s]\\nLoading query sequences...  [0.023635s]\\nMasking queries...  [0.102786s]\\nBuilding query seed set...  [0.023636s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.021515s]\\nAllocating buffers...  [4e-05s]\\nLoading reference sequences...  [5.4e-05s]\\nBuilding reference histograms...  [0.000233s]\\nAllocating buffers...  [3e-05s]\\nInitializing temporary storage...  [0.019303s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001299s]\\nBuilding query index...  [0.017933s]\\nBuilding seed filter...  [0.00123s]\\nSearching alignments...  [0.001218s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000545s]\\nBuilding query index...  [0.020118s]\\nBuilding seed filter...  [0.001233s]\\nSearching alignments...  [0.001077s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000446s]\\nBuilding query index...  [0.02017s]\\nBuilding seed filter...  [0.001112s]\\nSearching alignments...  [0.000995s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.00053s]\\nBuilding query index...  [0.0141s]\\nBuilding seed filter...  [0.001187s]\\nSearching alignments...  [0.001119s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000505s]\\nBuilding query index...  [0.012644s]\\nBuilding seed filter...  [0.001281s]\\nSearching alignments...  [0.001006s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000416s]\\nBuilding query index...  [0.019145s]\\nBuilding seed filter...  [0.001422s]\\nSearching alignments...  [0.001182s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000571s]\\nBuilding query index...  [0.025049s]\\nBuilding seed filter...  [0.001282s]\\nSearching alignments...  [0.00104s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000502s]\\nBuilding query index...  [0.020322s]\\nBuilding seed filter...  [0.0012s]\\nSearching alignments...  [0.001155s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.016278s]\\nDeallocating reference...  [1.6e-05s]\\nLoading reference sequences...  [3.2e-05s]\\nDeallocating buffers...  [0.000624s]\\nDeallocating queries...  [0.000592s]\\nLoading query sequences...  [1.8e-05s]\\nClosing the input file...  [2.3e-05s]\\nClosing the output file...  [0.000214s]\\nClosing the database file...  [1.5e-05s]\\nDeallocating taxonomy...  [1.7e-05s]\\nTotal time = 0.383976s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000159s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000948s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000147s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000495s]\\nOpening the output file...  [9e-05s]\\nLoading query sequences...  [0.017766s]\\nMasking queries...  [0.099606s]\\nBuilding query seed set...  [0.026638s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.017974s]\\nAllocating buffers...  [3.3e-05s]\\nLoading reference sequences...  [5.5e-05s]\\nBuilding reference histograms...  [0.000278s]\\nAllocating buffers...  [2.3e-05s]\\nInitializing temporary storage...  [0.026261s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.00147s]\\nBuilding query index...  [0.014489s]\\nBuilding seed filter...  [0.001135s]\\nSearching alignments...  [0.00124s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000492s]\\nBuilding query index...  [0.013607s]\\nBuilding seed filter...  [0.001237s]\\nSearching alignments...  [0.001055s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000433s]\\nBuilding query index...  [0.013634s]\\nBuilding seed filter...  [0.001481s]\\nSearching alignments...  [0.001007s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000471s]\\nBuilding query index...  [0.0124s]\\nBuilding seed filter...  [0.00105s]\\nSearching alignments...  [0.001021s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000459s]\\nBuilding query index...  [0.012093s]\\nBuilding seed filter...  [0.001483s]\\nSearching alignments...  [0.00116s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000436s]\\nBuilding query index...  [0.013202s]\\nBuilding seed filter...  [0.001097s]\\nSearching alignments...  [0.001154s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000467s]\\nBuilding query index...  [0.013568s]\\nBuilding seed filter...  [0.001018s]\\nSearching alignments...  [0.001016s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000486s]\\nBuilding query index...  [0.012004s]\\nBuilding seed filter...  [0.001061s]\\nSearching alignments...  [0.000976s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.01303s]\\nDeallocating reference...  [1.1e-05s]\\nLoading reference sequences...  [2e-05s]\\nDeallocating buffers...  [0.000677s]\\nDeallocating queries...  [0.000516s]\\nLoading query sequences...  [1.5e-05s]\\nClosing the input file...  [1.7e-05s]\\nClosing the output file...  [0.000249s]\\nClosing the database file...  [1.9e-05s]\\nDeallocating taxonomy...  [6e-06s]\\nTotal time = 0.332991s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000162s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000685s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000147s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000513s]\\nOpening the output file...  [0.00013s]\\nLoading query sequences...  [0.022588s]\\nMasking queries...  [0.101856s]\\nBuilding query seed set...  [0.021462s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.01714s]\\nAllocating buffers...  [3.5e-05s]\\nLoading reference sequences...  [5.3e-05s]\\nBuilding reference histograms...  [0.000227s]\\nAllocating buffers...  [2.6e-05s]\\nInitializing temporary storage...  [0.019486s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001218s]\\nBuilding query index...  [0.017082s]\\nBuilding seed filter...  [0.001208s]\\nSearching alignments...  [0.001207s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000483s]\\nBuilding query index...  [0.015148s]\\nBuilding seed filter...  [0.001413s]\\nSearching alignments...  [0.001124s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000511s]\\nBuilding query index...  [0.01414s]\\nBuilding seed filter...  [0.001082s]\\nSearching alignments...  [0.001097s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000455s]\\nBuilding query index...  [0.0123s]\\nBuilding seed filter...  [0.001359s]\\nSearching alignments...  [0.001049s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000513s]\\nBuilding query index...  [0.012273s]\\nBuilding seed filter...  [0.001136s]\\nSearching alignments...  [0.001089s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.00044s]\\nBuilding query index...  [0.015583s]\\nBuilding seed filter...  [0.001048s]\\nSearching alignments...  [0.001102s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.00046s]\\nBuilding query index...  [0.013741s]\\nBuilding seed filter...  [0.001557s]\\nSearching alignments...  [0.001021s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000527s]\\nBuilding query index...  [0.012144s]\\nBuilding seed filter...  [0.00114s]\\nSearching alignments...  [0.00103s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.014851s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [3.5e-05s]\\nDeallocating buffers...  [0.000504s]\\nDeallocating queries...  [0.000528s]\\nLoading query sequences...  [2.3e-05s]\\nClosing the input file...  [4.6e-05s]\\nClosing the output file...  [0.000227s]\\nClosing the database file...  [3.8e-05s]\\nDeallocating taxonomy...  [1.7e-05s]\\nTotal time = 0.336921s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000212s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.00087s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000122s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000378s]\\nOpening the output file...  [0.000106s]\\nLoading query sequences...  [0.02052s]\\nMasking queries...  [0.106724s]\\nBuilding query seed set...  [0.026994s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.026867s]\\nAllocating buffers...  [4.8e-05s]\\nLoading reference sequences...  [6.1e-05s]\\nBuilding reference histograms...  [0.000308s]\\nAllocating buffers...  [2.5e-05s]\\nInitializing temporary storage...  [0.018719s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001091s]\\nBuilding query index...  [0.015566s]\\nBuilding seed filter...  [0.001188s]\\nSearching alignments...  [0.001208s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000433s]\\nBuilding query index...  [0.022019s]\\nBuilding seed filter...  [0.001117s]\\nSearching alignments...  [0.001195s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000464s]\\nBuilding query index...  [0.020636s]\\nBuilding seed filter...  [0.00115s]\\nSearching alignments...  [0.001113s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000458s]\\nBuilding query index...  [0.019758s]\\nBuilding seed filter...  [0.001164s]\\nSearching alignments...  [0.001175s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000564s]\\nBuilding query index...  [0.013548s]\\nBuilding seed filter...  [0.001268s]\\nSearching alignments...  [0.001146s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000507s]\\nBuilding query index...  [0.013775s]\\nBuilding seed filter...  [0.001139s]\\nSearching alignments...  [0.001021s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.00054s]\\nBuilding query index...  [0.01435s]\\nBuilding seed filter...  [0.001076s]\\nSearching alignments...  [0.001038s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.00051s]\\nBuilding query index...  [0.015347s]\\nBuilding seed filter...  [0.001142s]\\nSearching alignments...  [0.001082s]\\nDeallocating buffers...  [7e-06s]\\nComputing alignments...  [0.014788s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [2.1e-05s]\\nDeallocating buffers...  [0.000658s]\\nDeallocating queries...  [0.000455s]\\nLoading query sequences...  [1.4e-05s]\\nClosing the input file...  [1.8e-05s]\\nClosing the output file...  [0.000251s]\\nClosing the database file...  [5e-05s]\\nDeallocating taxonomy...  [4.8e-05s]\\nTotal time = 0.376796s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000183s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000877s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000166s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000373s]\\nOpening the output file...  [7.3e-05s]\\nLoading query sequences...  [0.021003s]\\nMasking queries...  [0.084589s]\\nBuilding query seed set...  [0.024504s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.015718s]\\nAllocating buffers...  [2.4e-05s]\\nLoading reference sequences...  [4.7e-05s]\\nBuilding reference histograms...  [0.000219s]\\nAllocating buffers...  [1.5e-05s]\\nInitializing temporary storage...  [0.039682s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001072s]\\nBuilding query index...  [0.01211s]\\nBuilding seed filter...  [0.001041s]\\nSearching alignments...  [0.001119s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000459s]\\nBuilding query index...  [0.012228s]\\nBuilding seed filter...  [0.001013s]\\nSearching alignments...  [0.001073s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000661s]\\nBuilding query index...  [0.012454s]\\nBuilding seed filter...  [0.001011s]\\nSearching alignments...  [0.000974s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.00048s]\\nBuilding query index...  [0.010897s]\\nBuilding seed filter...  [0.001001s]\\nSearching alignments...  [0.000963s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000521s]\\nBuilding query index...  [0.011323s]\\nBuilding seed filter...  [0.001168s]\\nSearching alignments...  [0.001183s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000498s]\\nBuilding query index...  [0.011799s]\\nBuilding seed filter...  [0.00124s]\\nSearching alignments...  [0.001087s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000506s]\\nBuilding query index...  [0.013382s]\\nBuilding seed filter...  [0.001012s]\\nSearching alignments...  [0.001132s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000453s]\\nBuilding query index...  [0.01072s]\\nBuilding seed filter...  [0.000922s]\\nSearching alignments...  [0.001076s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.01893s]\\nDeallocating reference...  [2.2e-05s]\\nLoading reference sequences...  [2.3e-05s]\\nDeallocating buffers...  [0.00053s]\\nDeallocating queries...  [0.000656s]\\nLoading query sequences...  [1.4e-05s]\\nClosing the input file...  [1.5e-05s]\\nClosing the output file...  [0.000187s]\\nClosing the database file...  [9e-06s]\\nDeallocating taxonomy...  [1.1e-05s]\\nTotal time = 0.324603s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000136s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.005336s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000146s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000239s]\\nOpening the output file...  [0.000123s]\\nLoading query sequences...  [0.016738s]\\nMasking queries...  [0.101455s]\\nBuilding query seed set...  [0.022075s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.014263s]\\nAllocating buffers...  [6.3e-05s]\\nLoading reference sequences...  [9.8e-05s]\\nBuilding reference histograms...  [0.000216s]\\nAllocating buffers...  [2.4e-05s]\\nInitializing temporary storage...  [0.020168s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001256s]\\nBuilding query index...  [0.016645s]\\nBuilding seed filter...  [0.001014s]\\nSearching alignments...  [0.001175s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000538s]\\nBuilding query index...  [0.012594s]\\nBuilding seed filter...  [0.001037s]\\nSearching alignments...  [0.001118s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.00054s]\\nBuilding query index...  [0.01354s]\\nBuilding seed filter...  [0.001136s]\\nSearching alignments...  [0.000949s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000434s]\\nBuilding query index...  [0.01122s]\\nBuilding seed filter...  [0.00106s]\\nSearching alignments...  [0.000992s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000548s]\\nBuilding query index...  [0.011999s]\\nBuilding seed filter...  [0.001014s]\\nSearching alignments...  [0.000993s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000477s]\\nBuilding query index...  [0.011904s]\\nBuilding seed filter...  [0.001s]\\nSearching alignments...  [0.00103s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000452s]\\nBuilding query index...  [0.012244s]\\nBuilding seed filter...  [0.001226s]\\nSearching alignments...  [0.001073s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000473s]\\nBuilding query index...  [0.011878s]\\nBuilding seed filter...  [0.000989s]\\nSearching alignments...  [0.000971s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.013771s]\\nDeallocating reference...  [8e-06s]\\nLoading reference sequences...  [2.2e-05s]\\nDeallocating buffers...  [0.000538s]\\nDeallocating queries...  [0.000319s]\\nLoading query sequences...  [1.5e-05s]\\nClosing the input file...  [7.3e-05s]\\nClosing the output file...  [0.005477s]\\nClosing the database file...  [1.7e-05s]\\nDeallocating taxonomy...  [3e-06s]\\nTotal time = 0.320652s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000171s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001273s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000124s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000192s]\\nOpening the output file...  [0.000108s]\\nLoading query sequences...  [0.016442s]\\nMasking queries...  [0.094565s]\\nBuilding query seed set...  [0.024246s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.014679s]\\nAllocating buffers...  [2.5e-05s]\\nLoading reference sequences...  [4.3e-05s]\\nBuilding reference histograms...  [0.000321s]\\nAllocating buffers...  [2.6e-05s]\\nInitializing temporary storage...  [0.016347s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001249s]\\nBuilding query index...  [0.014811s]\\nBuilding seed filter...  [0.001366s]\\nSearching alignments...  [0.00133s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000575s]\\nBuilding query index...  [0.017386s]\\nBuilding seed filter...  [0.001283s]\\nSearching alignments...  [0.001143s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000496s]\\nBuilding query index...  [0.01398s]\\nBuilding seed filter...  [0.001011s]\\nSearching alignments...  [0.001132s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000534s]\\nBuilding query index...  [0.011619s]\\nBuilding seed filter...  [0.001024s]\\nSearching alignments...  [0.001024s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000555s]\\nBuilding query index...  [0.011679s]\\nBuilding seed filter...  [0.001079s]\\nSearching alignments...  [0.000985s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000514s]\\nBuilding query index...  [0.012365s]\\nBuilding seed filter...  [0.001711s]\\nSearching alignments...  [0.001055s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000466s]\\nBuilding query index...  [0.013665s]\\nBuilding seed filter...  [0.00113s]\\nSearching alignments...  [0.000964s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000488s]\\nBuilding query index...  [0.011261s]\\nBuilding seed filter...  [0.001118s]\\nSearching alignments...  [0.000978s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.013908s]\\nDeallocating reference...  [8e-06s]\\nLoading reference sequences...  [2.2e-05s]\\nDeallocating buffers...  [0.00052s]\\nDeallocating queries...  [0.000335s]\\nLoading query sequences...  [1.7e-05s]\\nClosing the input file...  [1.8e-05s]\\nClosing the output file...  [0.000214s]\\nClosing the database file...  [1.4e-05s]\\nDeallocating taxonomy...  [5e-06s]\\nTotal time = 0.313439s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000221s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001242s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000169s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000424s]\\nOpening the output file...  [0.000127s]\\nLoading query sequences...  [0.021445s]\\nMasking queries...  [0.097195s]\\nBuilding query seed set...  [0.025122s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.016166s]\\nAllocating buffers...  [3.3e-05s]\\nLoading reference sequences...  [7e-05s]\\nBuilding reference histograms...  [0.000302s]\\nAllocating buffers...  [2.9e-05s]\\nInitializing temporary storage...  [0.025117s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001095s]\\nBuilding query index...  [0.01288s]\\nBuilding seed filter...  [0.001136s]\\nSearching alignments...  [0.001172s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000536s]\\nBuilding query index...  [0.012459s]\\nBuilding seed filter...  [0.001047s]\\nSearching alignments...  [0.001166s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000479s]\\nBuilding query index...  [0.01417s]\\nBuilding seed filter...  [0.001176s]\\nSearching alignments...  [0.001006s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000486s]\\nBuilding query index...  [0.01129s]\\nBuilding seed filter...  [0.001164s]\\nSearching alignments...  [0.001078s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000606s]\\nBuilding query index...  [0.011381s]\\nBuilding seed filter...  [0.001028s]\\nSearching alignments...  [0.001058s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.00046s]\\nBuilding query index...  [0.012605s]\\nBuilding seed filter...  [0.00115s]\\nSearching alignments...  [0.001148s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000663s]\\nBuilding query index...  [0.015269s]\\nBuilding seed filter...  [0.001022s]\\nSearching alignments...  [0.000845s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000458s]\\nBuilding query index...  [0.011388s]\\nBuilding seed filter...  [0.001073s]\\nSearching alignments...  [0.000921s]\\nDeallocating buffers...  [3e-06s]\\nComputing alignments...  [0.028292s]\\nDeallocating reference...  [5e-06s]\\nLoading reference sequences...  [1.7e-05s]\\nDeallocating buffers...  [0.000413s]\\nDeallocating queries...  [0.000394s]\\nLoading query sequences...  [1.3e-05s]\\nClosing the input file...  [2.3e-05s]\\nClosing the output file...  [0.004768s]\\nClosing the database file...  [1.1e-05s]\\nDeallocating taxonomy...  [1.2e-05s]\\nTotal time = 0.344736s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000131s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001293s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000125s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000393s]\\nOpening the output file...  [0.000139s]\\nLoading query sequences...  [0.010314s]\\nMasking queries...  [0.054357s]\\nBuilding query seed set...  [0.014141s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.00958s]\\nAllocating buffers...  [5.7e-05s]\\nLoading reference sequences...  [5.6e-05s]\\nBuilding reference histograms...  [0.000354s]\\nAllocating buffers...  [4.3e-05s]\\nInitializing temporary storage...  [0.018284s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001318s]\\nBuilding query index...  [0.009989s]\\nBuilding seed filter...  [0.001035s]\\nSearching alignments...  [0.001038s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.00045s]\\nBuilding query index...  [0.008815s]\\nBuilding seed filter...  [0.000771s]\\nSearching alignments...  [0.000938s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000461s]\\nBuilding query index...  [0.007925s]\\nBuilding seed filter...  [0.000774s]\\nSearching alignments...  [0.000974s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000466s]\\nBuilding query index...  [0.006948s]\\nBuilding seed filter...  [0.000723s]\\nSearching alignments...  [0.000942s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000468s]\\nBuilding query index...  [0.006917s]\\nBuilding seed filter...  [0.000779s]\\nSearching alignments...  [0.000971s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000477s]\\nBuilding query index...  [0.012084s]\\nBuilding seed filter...  [0.000786s]\\nSearching alignments...  [0.00087s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000537s]\\nBuilding query index...  [0.008775s]\\nBuilding seed filter...  [0.001383s]\\nSearching alignments...  [0.001097s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000679s]\\nBuilding query index...  [0.008609s]\\nBuilding seed filter...  [0.000804s]\\nSearching alignments...  [0.001152s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.011295s]\\nDeallocating reference...  [1.8e-05s]\\nLoading reference sequences...  [2.2e-05s]\\nDeallocating buffers...  [1.6e-05s]\\nDeallocating queries...  [0.000223s]\\nLoading query sequences...  [1.7e-05s]\\nClosing the input file...  [5.6e-05s]\\nClosing the output file...  [0.000237s]\\nClosing the database file...  [2.1e-05s]\\nDeallocating taxonomy...  [7e-06s]\\nTotal time = 0.211384s\\nReported 10 pairwise alignments, 10 HSPs.\\n10 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000169s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 10\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000963s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000143s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000403s]\\nOpening the output file...  [7.8e-05s]\\nLoading query sequences...  [0.024956s]\\nMasking queries...  [0.100311s]\\nBuilding query seed set...  [0.023465s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.017903s]\\nAllocating buffers...  [3.9e-05s]\\nLoading reference sequences...  [5.9e-05s]\\nBuilding reference histograms...  [0.000346s]\\nAllocating buffers...  [3e-05s]\\nInitializing temporary storage...  [0.029594s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001134s]\\nBuilding query index...  [0.015006s]\\nBuilding seed filter...  [0.001131s]\\nSearching alignments...  [0.001188s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000535s]\\nBuilding query index...  [0.020184s]\\nBuilding seed filter...  [0.00125s]\\nSearching alignments...  [0.001077s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000492s]\\nBuilding query index...  [0.016831s]\\nBuilding seed filter...  [0.001029s]\\nSearching alignments...  [0.001171s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000469s]\\nBuilding query index...  [0.012216s]\\nBuilding seed filter...  [0.001306s]\\nSearching alignments...  [0.001089s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000485s]\\nBuilding query index...  [0.011972s]\\nBuilding seed filter...  [0.00103s]\\nSearching alignments...  [0.001164s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000502s]\\nBuilding query index...  [0.013054s]\\nBuilding seed filter...  [0.001134s]\\nSearching alignments...  [0.001123s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000536s]\\nBuilding query index...  [0.013384s]\\nBuilding seed filter...  [0.001222s]\\nSearching alignments...  [0.001115s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.00043s]\\nBuilding query index...  [0.012001s]\\nBuilding seed filter...  [0.001111s]\\nSearching alignments...  [0.001031s]\\nDeallocating buffers...  [1.2e-05s]\\nComputing alignments...  [0.013164s]\\nDeallocating reference...  [5e-06s]\\nLoading reference sequences...  [1.6e-05s]\\nDeallocating buffers...  [0.000457s]\\nDeallocating queries...  [0.000624s]\\nLoading query sequences...  [1.9e-05s]\\nClosing the input file...  [1.8e-05s]\\nClosing the output file...  [0.000186s]\\nClosing the database file...  [1.8e-05s]\\nDeallocating taxonomy...  [1.2e-05s]\\nTotal time = 0.354696s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000212s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001138s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000111s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000251s]\\nOpening the output file...  [0.000108s]\\nLoading query sequences...  [0.016743s]\\nMasking queries...  [0.080385s]\\nBuilding query seed set...  [0.018747s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.014922s]\\nAllocating buffers...  [4.3e-05s]\\nLoading reference sequences...  [5.8e-05s]\\nBuilding reference histograms...  [0.000284s]\\nAllocating buffers...  [3e-05s]\\nInitializing temporary storage...  [0.019154s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001325s]\\nBuilding query index...  [0.014243s]\\nBuilding seed filter...  [0.001153s]\\nSearching alignments...  [0.001229s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000535s]\\nBuilding query index...  [0.017449s]\\nBuilding seed filter...  [0.001256s]\\nSearching alignments...  [0.001195s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000459s]\\nBuilding query index...  [0.013326s]\\nBuilding seed filter...  [0.000871s]\\nSearching alignments...  [0.00109s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000505s]\\nBuilding query index...  [0.014078s]\\nBuilding seed filter...  [0.00085s]\\nSearching alignments...  [0.001025s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000478s]\\nBuilding query index...  [0.010509s]\\nBuilding seed filter...  [0.001072s]\\nSearching alignments...  [0.001033s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000517s]\\nBuilding query index...  [0.010658s]\\nBuilding seed filter...  [0.001003s]\\nSearching alignments...  [0.001091s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000493s]\\nBuilding query index...  [0.013184s]\\nBuilding seed filter...  [0.001001s]\\nSearching alignments...  [0.000976s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000467s]\\nBuilding query index...  [0.009234s]\\nBuilding seed filter...  [0.000926s]\\nSearching alignments...  [0.000928s]\\nDeallocating buffers...  [2.3e-05s]\\nComputing alignments...  [0.01107s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [2e-05s]\\nDeallocating buffers...  [0.000483s]\\nDeallocating queries...  [8.4e-05s]\\nLoading query sequences...  [1.8e-05s]\\nClosing the input file...  [1.6e-05s]\\nClosing the output file...  [0.000259s]\\nClosing the database file...  [1.6e-05s]\\nDeallocating taxonomy...  [2.6e-05s]\\nTotal time = 0.288376s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000143s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000801s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000218s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000217s]\\nOpening the output file...  [0.000213s]\\nLoading query sequences...  [0.016969s]\\nMasking queries...  [0.075124s]\\nBuilding query seed set...  [0.023094s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.011557s]\\nAllocating buffers...  [2.5e-05s]\\nLoading reference sequences...  [4.7e-05s]\\nBuilding reference histograms...  [0.000288s]\\nAllocating buffers...  [1.6e-05s]\\nInitializing temporary storage...  [0.024466s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001274s]\\nBuilding query index...  [0.012394s]\\nBuilding seed filter...  [0.000946s]\\nSearching alignments...  [0.001204s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.00064s]\\nBuilding query index...  [0.015034s]\\nBuilding seed filter...  [0.000839s]\\nSearching alignments...  [0.001113s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000484s]\\nBuilding query index...  [0.013297s]\\nBuilding seed filter...  [0.00132s]\\nSearching alignments...  [0.001065s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.00059s]\\nBuilding query index...  [0.013278s]\\nBuilding seed filter...  [0.00167s]\\nSearching alignments...  [0.000903s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000474s]\\nBuilding query index...  [0.009145s]\\nBuilding seed filter...  [0.001008s]\\nSearching alignments...  [0.001047s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000427s]\\nBuilding query index...  [0.01361s]\\nBuilding seed filter...  [0.000989s]\\nSearching alignments...  [0.000957s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000606s]\\nBuilding query index...  [0.009298s]\\nBuilding seed filter...  [0.000911s]\\nSearching alignments...  [0.00104s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000601s]\\nBuilding query index...  [0.012242s]\\nBuilding seed filter...  [0.000987s]\\nSearching alignments...  [0.001188s]\\nDeallocating buffers...  [3e-06s]\\nComputing alignments...  [0.018434s]\\nDeallocating reference...  [5e-06s]\\nLoading reference sequences...  [1.8e-05s]\\nDeallocating buffers...  [0.000336s]\\nDeallocating queries...  [0.000106s]\\nLoading query sequences...  [2.2e-05s]\\nClosing the input file...  [5e-05s]\\nClosing the output file...  [0.000199s]\\nClosing the database file...  [1.6e-05s]\\nDeallocating taxonomy...  [7e-06s]\\nTotal time = 0.293383s\\nReported 22 pairwise alignments, 22 HSPs.\\n22 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000221s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.006569s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.00016s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000413s]\\nOpening the output file...  [8.5e-05s]\\nLoading query sequences...  [0.02319s]\\nMasking queries...  [0.097911s]\\nBuilding query seed set...  [0.018877s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.018105s]\\nAllocating buffers...  [5.4e-05s]\\nLoading reference sequences...  [5.5e-05s]\\nBuilding reference histograms...  [0.000276s]\\nAllocating buffers...  [2.7e-05s]\\nInitializing temporary storage...  [0.021006s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001198s]\\nBuilding query index...  [0.01619s]\\nBuilding seed filter...  [0.001504s]\\nSearching alignments...  [0.001244s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000551s]\\nBuilding query index...  [0.020013s]\\nBuilding seed filter...  [0.001268s]\\nSearching alignments...  [0.001083s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000611s]\\nBuilding query index...  [0.020283s]\\nBuilding seed filter...  [0.001101s]\\nSearching alignments...  [0.001087s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.00055s]\\nBuilding query index...  [0.01457s]\\nBuilding seed filter...  [0.001512s]\\nSearching alignments...  [0.00107s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000582s]\\nBuilding query index...  [0.012335s]\\nBuilding seed filter...  [0.001178s]\\nSearching alignments...  [0.00138s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.00051s]\\nBuilding query index...  [0.013149s]\\nBuilding seed filter...  [0.001071s]\\nSearching alignments...  [0.00107s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000497s]\\nBuilding query index...  [0.013298s]\\nBuilding seed filter...  [0.001235s]\\nSearching alignments...  [0.000956s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000536s]\\nBuilding query index...  [0.013734s]\\nBuilding seed filter...  [0.001255s]\\nSearching alignments...  [0.001019s]\\nDeallocating buffers...  [7e-06s]\\nComputing alignments...  [0.014328s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [1.8e-05s]\\nDeallocating buffers...  [0.000407s]\\nDeallocating queries...  [0.000356s]\\nLoading query sequences...  [1.4e-05s]\\nClosing the input file...  [1.8e-05s]\\nClosing the output file...  [0.000182s]\\nClosing the database file...  [1.8e-05s]\\nDeallocating taxonomy...  [2e-05s]\\nTotal time = 0.344659s\\nReported 23 pairwise alignments, 23 HSPs.\\n23 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.00019s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000796s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000114s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000477s]\\nOpening the output file...  [8.1e-05s]\\nLoading query sequences...  [0.020007s]\\nMasking queries...  [0.099786s]\\nBuilding query seed set...  [0.02567s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.015983s]\\nAllocating buffers...  [2.4e-05s]\\nLoading reference sequences...  [4.4e-05s]\\nBuilding reference histograms...  [0.000257s]\\nAllocating buffers...  [7.2e-05s]\\nInitializing temporary storage...  [0.019744s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001066s]\\nBuilding query index...  [0.016345s]\\nBuilding seed filter...  [0.001372s]\\nSearching alignments...  [0.001235s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.00053s]\\nBuilding query index...  [0.013896s]\\nBuilding seed filter...  [0.00145s]\\nSearching alignments...  [0.001057s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000583s]\\nBuilding query index...  [0.020658s]\\nBuilding seed filter...  [0.001116s]\\nSearching alignments...  [0.001071s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000502s]\\nBuilding query index...  [0.012532s]\\nBuilding seed filter...  [0.001132s]\\nSearching alignments...  [0.00109s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000477s]\\nBuilding query index...  [0.012701s]\\nBuilding seed filter...  [0.001353s]\\nSearching alignments...  [0.001113s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.0005s]\\nBuilding query index...  [0.019437s]\\nBuilding seed filter...  [0.001184s]\\nSearching alignments...  [0.001032s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000446s]\\nBuilding query index...  [0.015036s]\\nBuilding seed filter...  [0.001288s]\\nSearching alignments...  [0.00104s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.00045s]\\nBuilding query index...  [0.012285s]\\nBuilding seed filter...  [0.001637s]\\nSearching alignments...  [0.00101s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.014334s]\\nDeallocating reference...  [1.6e-05s]\\nLoading reference sequences...  [2.1e-05s]\\nDeallocating buffers...  [0.000515s]\\nDeallocating queries...  [0.000394s]\\nLoading query sequences...  [1.9e-05s]\\nClosing the input file...  [2e-05s]\\nClosing the output file...  [0.00018s]\\nClosing the database file...  [1.5e-05s]\\nDeallocating taxonomy...  [2.1e-05s]\\nTotal time = 0.34554s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000187s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.00128s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000181s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000235s]\\nOpening the output file...  [0.00016s]\\nLoading query sequences...  [0.018133s]\\nMasking queries...  [0.098512s]\\nBuilding query seed set...  [0.028449s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.028053s]\\nAllocating buffers...  [8.6e-05s]\\nLoading reference sequences...  [5.7e-05s]\\nBuilding reference histograms...  [0.000261s]\\nAllocating buffers...  [2.3e-05s]\\nInitializing temporary storage...  [0.022542s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001145s]\\nBuilding query index...  [0.014523s]\\nBuilding seed filter...  [0.001167s]\\nSearching alignments...  [0.001207s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000459s]\\nBuilding query index...  [0.021212s]\\nBuilding seed filter...  [0.001276s]\\nSearching alignments...  [0.001137s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000577s]\\nBuilding query index...  [0.021316s]\\nBuilding seed filter...  [0.001185s]\\nSearching alignments...  [0.001099s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000418s]\\nBuilding query index...  [0.015822s]\\nBuilding seed filter...  [0.001197s]\\nSearching alignments...  [0.001108s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000483s]\\nBuilding query index...  [0.0146s]\\nBuilding seed filter...  [0.001402s]\\nSearching alignments...  [0.001124s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000448s]\\nBuilding query index...  [0.014607s]\\nBuilding seed filter...  [0.0014s]\\nSearching alignments...  [0.001186s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000486s]\\nBuilding query index...  [0.019589s]\\nBuilding seed filter...  [0.001345s]\\nSearching alignments...  [0.001021s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000461s]\\nBuilding query index...  [0.012847s]\\nBuilding seed filter...  [0.00105s]\\nSearching alignments...  [0.001207s]\\nDeallocating buffers...  [1.5e-05s]\\nComputing alignments...  [0.015456s]\\nDeallocating reference...  [1.1e-05s]\\nLoading reference sequences...  [1.7e-05s]\\nDeallocating buffers...  [0.00044s]\\nDeallocating queries...  [0.000372s]\\nLoading query sequences...  [1.3e-05s]\\nClosing the input file...  [4.6e-05s]\\nClosing the output file...  [0.053848s]\\nClosing the database file...  [3.5e-05s]\\nDeallocating taxonomy...  [6e-06s]\\nTotal time = 0.426379s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000128s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.00079s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [9.8e-05s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000353s]\\nOpening the output file...  [0.000104s]\\nLoading query sequences...  [0.02236s]\\nMasking queries...  [0.098894s]\\nBuilding query seed set...  [0.026375s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.017776s]\\nAllocating buffers...  [6.4e-05s]\\nLoading reference sequences...  [7.6e-05s]\\nBuilding reference histograms...  [0.000268s]\\nAllocating buffers...  [2.8e-05s]\\nInitializing temporary storage...  [0.149652s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001258s]\\nBuilding query index...  [0.014926s]\\nBuilding seed filter...  [0.001106s]\\nSearching alignments...  [0.001144s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000478s]\\nBuilding query index...  [0.020236s]\\nBuilding seed filter...  [0.001097s]\\nSearching alignments...  [0.001192s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.00046s]\\nBuilding query index...  [0.021319s]\\nBuilding seed filter...  [0.001125s]\\nSearching alignments...  [0.001073s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.00043s]\\nBuilding query index...  [0.017237s]\\nBuilding seed filter...  [0.001035s]\\nSearching alignments...  [0.001035s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000453s]\\nBuilding query index...  [0.012913s]\\nBuilding seed filter...  [0.001107s]\\nSearching alignments...  [0.001136s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000487s]\\nBuilding query index...  [0.014631s]\\nBuilding seed filter...  [0.001097s]\\nSearching alignments...  [0.001073s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000417s]\\nBuilding query index...  [0.01504s]\\nBuilding seed filter...  [0.001245s]\\nSearching alignments...  [0.00125s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000497s]\\nBuilding query index...  [0.018499s]\\nBuilding seed filter...  [0.001161s]\\nSearching alignments...  [0.001093s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.105116s]\\nDeallocating reference...  [4e-06s]\\nLoading reference sequences...  [1.5e-05s]\\nDeallocating buffers...  [0.000443s]\\nDeallocating queries...  [0.000495s]\\nLoading query sequences...  [1.2e-05s]\\nClosing the input file...  [1.4e-05s]\\nClosing the output file...  [0.000189s]\\nClosing the database file...  [2.1e-05s]\\nDeallocating taxonomy...  [1.1e-05s]\\nTotal time = 0.580806s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000174s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000735s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000186s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000246s]\\nOpening the output file...  [8.3e-05s]\\nLoading query sequences...  [0.016614s]\\nMasking queries...  [0.096675s]\\nBuilding query seed set...  [0.026874s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.018434s]\\nAllocating buffers...  [2.8e-05s]\\nLoading reference sequences...  [4.4e-05s]\\nBuilding reference histograms...  [0.000201s]\\nAllocating buffers...  [2.5e-05s]\\nInitializing temporary storage...  [0.024692s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001279s]\\nBuilding query index...  [0.017038s]\\nBuilding seed filter...  [0.001418s]\\nSearching alignments...  [0.001097s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000591s]\\nBuilding query index...  [0.014184s]\\nBuilding seed filter...  [0.00115s]\\nSearching alignments...  [0.001047s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000498s]\\nBuilding query index...  [0.015728s]\\nBuilding seed filter...  [0.001161s]\\nSearching alignments...  [0.00106s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000449s]\\nBuilding query index...  [0.012255s]\\nBuilding seed filter...  [0.001127s]\\nSearching alignments...  [0.001024s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000493s]\\nBuilding query index...  [0.011814s]\\nBuilding seed filter...  [0.001044s]\\nSearching alignments...  [0.001056s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000438s]\\nBuilding query index...  [0.019528s]\\nBuilding seed filter...  [0.001162s]\\nSearching alignments...  [0.00131s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000574s]\\nBuilding query index...  [0.013964s]\\nBuilding seed filter...  [0.001026s]\\nSearching alignments...  [0.001053s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000431s]\\nBuilding query index...  [0.012805s]\\nBuilding seed filter...  [0.001011s]\\nSearching alignments...  [0.001035s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.013779s]\\nDeallocating reference...  [3.2e-05s]\\nLoading reference sequences...  [2e-05s]\\nDeallocating buffers...  [0.000552s]\\nDeallocating queries...  [0.000508s]\\nLoading query sequences...  [1.8e-05s]\\nClosing the input file...  [4e-05s]\\nClosing the output file...  [0.000243s]\\nClosing the database file...  [2.1e-05s]\\nDeallocating taxonomy...  [6e-06s]\\nTotal time = 0.340723s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000156s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000697s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000179s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000376s]\\nOpening the output file...  [9.1e-05s]\\nLoading query sequences...  [0.023246s]\\nMasking queries...  [0.102115s]\\nBuilding query seed set...  [0.020198s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.015574s]\\nAllocating buffers...  [5.2e-05s]\\nLoading reference sequences...  [6.2e-05s]\\nBuilding reference histograms...  [0.000256s]\\nAllocating buffers...  [2.7e-05s]\\nInitializing temporary storage...  [0.018956s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001086s]\\nBuilding query index...  [0.013608s]\\nBuilding seed filter...  [0.00132s]\\nSearching alignments...  [0.0012s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.00047s]\\nBuilding query index...  [0.017711s]\\nBuilding seed filter...  [0.001155s]\\nSearching alignments...  [0.000985s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000503s]\\nBuilding query index...  [0.01594s]\\nBuilding seed filter...  [0.001072s]\\nSearching alignments...  [0.001056s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000465s]\\nBuilding query index...  [0.012055s]\\nBuilding seed filter...  [0.001099s]\\nSearching alignments...  [0.000932s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000497s]\\nBuilding query index...  [0.012007s]\\nBuilding seed filter...  [0.001061s]\\nSearching alignments...  [0.001072s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000458s]\\nBuilding query index...  [0.0133s]\\nBuilding seed filter...  [0.001079s]\\nSearching alignments...  [0.000984s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000472s]\\nBuilding query index...  [0.013407s]\\nBuilding seed filter...  [0.001319s]\\nSearching alignments...  [0.001069s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000471s]\\nBuilding query index...  [0.012794s]\\nBuilding seed filter...  [0.001157s]\\nSearching alignments...  [0.0011s]\\nDeallocating buffers...  [9e-06s]\\nComputing alignments...  [0.015258s]\\nDeallocating reference...  [5e-06s]\\nLoading reference sequences...  [3.5e-05s]\\nDeallocating buffers...  [0.00048s]\\nDeallocating queries...  [0.000441s]\\nLoading query sequences...  [1.4e-05s]\\nClosing the input file...  [4.7e-05s]\\nClosing the output file...  [0.000195s]\\nClosing the database file...  [1.4e-05s]\\nDeallocating taxonomy...  [1.8e-05s]\\nTotal time = 0.331712s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000141s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000704s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000154s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000239s]\\nOpening the output file...  [8.7e-05s]\\nLoading query sequences...  [0.02155s]\\nMasking queries...  [0.103314s]\\nBuilding query seed set...  [0.028568s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.02479s]\\nAllocating buffers...  [4.6e-05s]\\nLoading reference sequences...  [7.3e-05s]\\nBuilding reference histograms...  [0.000241s]\\nAllocating buffers...  [2.8e-05s]\\nInitializing temporary storage...  [0.030966s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001155s]\\nBuilding query index...  [0.014906s]\\nBuilding seed filter...  [0.001052s]\\nSearching alignments...  [0.001158s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000581s]\\nBuilding query index...  [0.019281s]\\nBuilding seed filter...  [0.001178s]\\nSearching alignments...  [0.001043s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000474s]\\nBuilding query index...  [0.020406s]\\nBuilding seed filter...  [0.001227s]\\nSearching alignments...  [0.001128s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000577s]\\nBuilding query index...  [0.018863s]\\nBuilding seed filter...  [0.001177s]\\nSearching alignments...  [0.000977s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.00053s]\\nBuilding query index...  [0.017538s]\\nBuilding seed filter...  [0.001293s]\\nSearching alignments...  [0.001281s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000443s]\\nBuilding query index...  [0.021918s]\\nBuilding seed filter...  [0.001201s]\\nSearching alignments...  [0.001196s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000488s]\\nBuilding query index...  [0.017878s]\\nBuilding seed filter...  [0.001172s]\\nSearching alignments...  [0.001102s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000467s]\\nBuilding query index...  [0.013569s]\\nBuilding seed filter...  [0.001373s]\\nSearching alignments...  [0.001152s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.020015s]\\nDeallocating reference...  [9e-06s]\\nLoading reference sequences...  [2.1e-05s]\\nDeallocating buffers...  [0.000572s]\\nDeallocating queries...  [0.000457s]\\nLoading query sequences...  [1.3e-05s]\\nClosing the input file...  [3.1e-05s]\\nClosing the output file...  [0.006119s]\\nClosing the database file...  [2.3e-05s]\\nDeallocating taxonomy...  [6e-06s]\\nTotal time = 0.412848s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000242s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001395s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000175s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000146s]\\nOpening the output file...  [0.005375s]\\nLoading query sequences...  [0.028108s]\\nMasking queries...  [0.095441s]\\nBuilding query seed set...  [0.020855s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.016815s]\\nAllocating buffers...  [4.5e-05s]\\nLoading reference sequences...  [7.1e-05s]\\nBuilding reference histograms...  [0.000381s]\\nAllocating buffers...  [3.3e-05s]\\nInitializing temporary storage...  [0.019379s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001165s]\\nBuilding query index...  [0.015412s]\\nBuilding seed filter...  [0.001227s]\\nSearching alignments...  [0.001178s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000489s]\\nBuilding query index...  [0.018241s]\\nBuilding seed filter...  [0.001452s]\\nSearching alignments...  [0.001004s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000513s]\\nBuilding query index...  [0.016247s]\\nBuilding seed filter...  [0.001098s]\\nSearching alignments...  [0.001084s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000523s]\\nBuilding query index...  [0.012152s]\\nBuilding seed filter...  [0.001159s]\\nSearching alignments...  [0.001148s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000511s]\\nBuilding query index...  [0.012245s]\\nBuilding seed filter...  [0.001064s]\\nSearching alignments...  [0.001168s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.0005s]\\nBuilding query index...  [0.017853s]\\nBuilding seed filter...  [0.001118s]\\nSearching alignments...  [0.000935s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000517s]\\nBuilding query index...  [0.014045s]\\nBuilding seed filter...  [0.001019s]\\nSearching alignments...  [0.001024s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000545s]\\nBuilding query index...  [0.012272s]\\nBuilding seed filter...  [0.001408s]\\nSearching alignments...  [0.00103s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.015191s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [1.5e-05s]\\nDeallocating buffers...  [0.000417s]\\nDeallocating queries...  [0.000409s]\\nLoading query sequences...  [1.3e-05s]\\nClosing the input file...  [2e-05s]\\nClosing the output file...  [0.000212s]\\nClosing the database file...  [2.8e-05s]\\nDeallocating taxonomy...  [1.4e-05s]\\nTotal time = 0.350587s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000125s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000822s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000151s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000432s]\\nOpening the output file...  [0.000111s]\\nLoading query sequences...  [0.016761s]\\nMasking queries...  [0.092677s]\\nBuilding query seed set...  [0.021873s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.016179s]\\nAllocating buffers...  [4.2e-05s]\\nLoading reference sequences...  [5.2e-05s]\\nBuilding reference histograms...  [0.000176s]\\nAllocating buffers...  [1.6e-05s]\\nInitializing temporary storage...  [0.017481s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001161s]\\nBuilding query index...  [0.016914s]\\nBuilding seed filter...  [0.001076s]\\nSearching alignments...  [0.001128s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000463s]\\nBuilding query index...  [0.014034s]\\nBuilding seed filter...  [0.001065s]\\nSearching alignments...  [0.001056s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000427s]\\nBuilding query index...  [0.017171s]\\nBuilding seed filter...  [0.00109s]\\nSearching alignments...  [0.001106s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000458s]\\nBuilding query index...  [0.013284s]\\nBuilding seed filter...  [0.001089s]\\nSearching alignments...  [0.001153s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000503s]\\nBuilding query index...  [0.014519s]\\nBuilding seed filter...  [0.001138s]\\nSearching alignments...  [0.001257s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000499s]\\nBuilding query index...  [0.018565s]\\nBuilding seed filter...  [0.001125s]\\nSearching alignments...  [0.001046s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000482s]\\nBuilding query index...  [0.015705s]\\nBuilding seed filter...  [0.000998s]\\nSearching alignments...  [0.001054s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000453s]\\nBuilding query index...  [0.011859s]\\nBuilding seed filter...  [0.001037s]\\nSearching alignments...  [0.001037s]\\nDeallocating buffers...  [1.5e-05s]\\nComputing alignments...  [0.013954s]\\nDeallocating reference...  [7e-06s]\\nLoading reference sequences...  [1.6e-05s]\\nDeallocating buffers...  [0.000462s]\\nDeallocating queries...  [0.000193s]\\nLoading query sequences...  [1.3e-05s]\\nClosing the input file...  [4.3e-05s]\\nClosing the output file...  [0.000201s]\\nClosing the database file...  [1.8e-05s]\\nDeallocating taxonomy...  [5e-06s]\\nTotal time = 0.325982s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000129s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.00085s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.00012s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000226s]\\nOpening the output file...  [0.000107s]\\nLoading query sequences...  [0.020707s]\\nMasking queries...  [0.098146s]\\nBuilding query seed set...  [0.023854s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.015277s]\\nAllocating buffers...  [2.3e-05s]\\nLoading reference sequences...  [4.1e-05s]\\nBuilding reference histograms...  [0.00022s]\\nAllocating buffers...  [2.6e-05s]\\nInitializing temporary storage...  [0.031538s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001248s]\\nBuilding query index...  [0.015912s]\\nBuilding seed filter...  [0.001256s]\\nSearching alignments...  [0.001288s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000625s]\\nBuilding query index...  [0.020312s]\\nBuilding seed filter...  [0.001087s]\\nSearching alignments...  [0.001011s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000488s]\\nBuilding query index...  [0.014082s]\\nBuilding seed filter...  [0.001047s]\\nSearching alignments...  [0.001144s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000468s]\\nBuilding query index...  [0.013176s]\\nBuilding seed filter...  [0.001063s]\\nSearching alignments...  [0.001068s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000539s]\\nBuilding query index...  [0.01194s]\\nBuilding seed filter...  [0.001035s]\\nSearching alignments...  [0.000956s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.00043s]\\nBuilding query index...  [0.013744s]\\nBuilding seed filter...  [0.001546s]\\nSearching alignments...  [0.001123s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000517s]\\nBuilding query index...  [0.013702s]\\nBuilding seed filter...  [0.00124s]\\nSearching alignments...  [0.001091s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000523s]\\nBuilding query index...  [0.011954s]\\nBuilding seed filter...  [0.001201s]\\nSearching alignments...  [0.001056s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.013658s]\\nDeallocating reference...  [1.8e-05s]\\nLoading reference sequences...  [3.2e-05s]\\nDeallocating buffers...  [0.000418s]\\nDeallocating queries...  [0.000159s]\\nLoading query sequences...  [2.7e-05s]\\nClosing the input file...  [1.7e-05s]\\nClosing the output file...  [0.000245s]\\nClosing the database file...  [3.2e-05s]\\nDeallocating taxonomy...  [1e-05s]\\nTotal time = 0.343923s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000212s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001022s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000112s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000517s]\\nOpening the output file...  [9.5e-05s]\\nLoading query sequences...  [0.017112s]\\nMasking queries...  [0.092883s]\\nBuilding query seed set...  [0.025735s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.015748s]\\nAllocating buffers...  [2.7e-05s]\\nLoading reference sequences...  [4.4e-05s]\\nBuilding reference histograms...  [0.000211s]\\nAllocating buffers...  [1.6e-05s]\\nInitializing temporary storage...  [0.018628s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001246s]\\nBuilding query index...  [0.017848s]\\nBuilding seed filter...  [0.001391s]\\nSearching alignments...  [0.001186s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000495s]\\nBuilding query index...  [0.013272s]\\nBuilding seed filter...  [0.001136s]\\nSearching alignments...  [0.001069s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000429s]\\nBuilding query index...  [0.01369s]\\nBuilding seed filter...  [0.001065s]\\nSearching alignments...  [0.000962s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000507s]\\nBuilding query index...  [0.013652s]\\nBuilding seed filter...  [0.001536s]\\nSearching alignments...  [0.001133s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000464s]\\nBuilding query index...  [0.012535s]\\nBuilding seed filter...  [0.001159s]\\nSearching alignments...  [0.001096s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000449s]\\nBuilding query index...  [0.017127s]\\nBuilding seed filter...  [0.001003s]\\nSearching alignments...  [0.001031s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.00068s]\\nBuilding query index...  [0.014755s]\\nBuilding seed filter...  [0.001018s]\\nSearching alignments...  [0.001082s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.00044s]\\nBuilding query index...  [0.011769s]\\nBuilding seed filter...  [0.001056s]\\nSearching alignments...  [0.001011s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.013771s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [1.7e-05s]\\nDeallocating buffers...  [0.000466s]\\nDeallocating queries...  [0.00045s]\\nLoading query sequences...  [1.9e-05s]\\nClosing the input file...  [1.7e-05s]\\nClosing the output file...  [0.000246s]\\nClosing the database file...  [1.4e-05s]\\nDeallocating taxonomy...  [5e-06s]\\nTotal time = 0.324699s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000166s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000734s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000127s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000174s]\\nOpening the output file...  [0.000114s]\\nLoading query sequences...  [0.021323s]\\nMasking queries...  [0.102482s]\\nBuilding query seed set...  [0.02237s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.021759s]\\nAllocating buffers...  [4.1e-05s]\\nLoading reference sequences...  [9.3e-05s]\\nBuilding reference histograms...  [0.000279s]\\nAllocating buffers...  [2.7e-05s]\\nInitializing temporary storage...  [0.0254s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001151s]\\nBuilding query index...  [0.016373s]\\nBuilding seed filter...  [0.001105s]\\nSearching alignments...  [0.001101s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.00053s]\\nBuilding query index...  [0.014092s]\\nBuilding seed filter...  [0.001182s]\\nSearching alignments...  [0.001173s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000495s]\\nBuilding query index...  [0.014281s]\\nBuilding seed filter...  [0.001394s]\\nSearching alignments...  [0.001041s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000493s]\\nBuilding query index...  [0.012585s]\\nBuilding seed filter...  [0.001294s]\\nSearching alignments...  [0.001107s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000529s]\\nBuilding query index...  [0.013723s]\\nBuilding seed filter...  [0.001244s]\\nSearching alignments...  [0.001098s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000493s]\\nBuilding query index...  [0.013435s]\\nBuilding seed filter...  [0.001187s]\\nSearching alignments...  [0.001035s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000445s]\\nBuilding query index...  [0.021697s]\\nBuilding seed filter...  [0.001418s]\\nSearching alignments...  [0.001018s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000513s]\\nBuilding query index...  [0.012879s]\\nBuilding seed filter...  [0.00125s]\\nSearching alignments...  [0.001088s]\\nDeallocating buffers...  [1.7e-05s]\\nComputing alignments...  [0.016231s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [2e-05s]\\nDeallocating buffers...  [0.000596s]\\nDeallocating queries...  [0.000493s]\\nLoading query sequences...  [1.7e-05s]\\nClosing the input file...  [4.6e-05s]\\nClosing the output file...  [0.000224s]\\nClosing the database file...  [4.4e-05s]\\nDeallocating taxonomy...  [1.4e-05s]\\nTotal time = 0.355691s\\nReported 24 pairwise alignments, 24 HSPs.\\n24 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000124s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000863s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000102s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.00045s]\\nOpening the output file...  [7.9e-05s]\\nLoading query sequences...  [0.02381s]\\nMasking queries...  [0.111433s]\\nBuilding query seed set...  [0.021902s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.019452s]\\nAllocating buffers...  [2.8e-05s]\\nLoading reference sequences...  [5.2e-05s]\\nBuilding reference histograms...  [0.000307s]\\nAllocating buffers...  [1.8e-05s]\\nInitializing temporary storage...  [0.027743s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001106s]\\nBuilding query index...  [0.016924s]\\nBuilding seed filter...  [0.001193s]\\nSearching alignments...  [0.001231s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.0005s]\\nBuilding query index...  [0.014262s]\\nBuilding seed filter...  [0.00134s]\\nSearching alignments...  [0.001148s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000461s]\\nBuilding query index...  [0.014609s]\\nBuilding seed filter...  [0.00123s]\\nSearching alignments...  [0.00109s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000474s]\\nBuilding query index...  [0.0143s]\\nBuilding seed filter...  [0.001317s]\\nSearching alignments...  [0.001112s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000477s]\\nBuilding query index...  [0.015433s]\\nBuilding seed filter...  [0.001275s]\\nSearching alignments...  [0.001091s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.00047s]\\nBuilding query index...  [0.013939s]\\nBuilding seed filter...  [0.001165s]\\nSearching alignments...  [0.001014s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000453s]\\nBuilding query index...  [0.014644s]\\nBuilding seed filter...  [0.001064s]\\nSearching alignments...  [0.001047s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000475s]\\nBuilding query index...  [0.014095s]\\nBuilding seed filter...  [0.001162s]\\nSearching alignments...  [0.001148s]\\nDeallocating buffers...  [1.4e-05s]\\nComputing alignments...  [0.035849s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [2.1e-05s]\\nDeallocating buffers...  [0.000475s]\\nDeallocating queries...  [0.000473s]\\nLoading query sequences...  [1.8e-05s]\\nClosing the input file...  [1.8e-05s]\\nClosing the output file...  [0.000195s]\\nClosing the database file...  [1.1e-05s]\\nDeallocating taxonomy...  [1.2e-05s]\\nTotal time = 0.38513s\\nReported 27 pairwise alignments, 27 HSPs.\\n27 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000162s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000856s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000158s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000212s]\\nOpening the output file...  [0.00012s]\\nLoading query sequences...  [0.016429s]\\nMasking queries...  [0.09546s]\\nBuilding query seed set...  [0.028549s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.016503s]\\nAllocating buffers...  [3.3e-05s]\\nLoading reference sequences...  [4.2e-05s]\\nBuilding reference histograms...  [0.000219s]\\nAllocating buffers...  [2.6e-05s]\\nInitializing temporary storage...  [0.03164s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.00119s]\\nBuilding query index...  [0.014171s]\\nBuilding seed filter...  [0.001059s]\\nSearching alignments...  [0.0011s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000454s]\\nBuilding query index...  [0.012834s]\\nBuilding seed filter...  [0.001073s]\\nSearching alignments...  [0.001035s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000528s]\\nBuilding query index...  [0.013838s]\\nBuilding seed filter...  [0.001315s]\\nSearching alignments...  [0.001177s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000468s]\\nBuilding query index...  [0.01253s]\\nBuilding seed filter...  [0.001084s]\\nSearching alignments...  [0.001036s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000577s]\\nBuilding query index...  [0.011733s]\\nBuilding seed filter...  [0.00111s]\\nSearching alignments...  [0.001077s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000465s]\\nBuilding query index...  [0.013024s]\\nBuilding seed filter...  [0.001062s]\\nSearching alignments...  [0.001003s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.00049s]\\nBuilding query index...  [0.013488s]\\nBuilding seed filter...  [0.001079s]\\nSearching alignments...  [0.001088s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000463s]\\nBuilding query index...  [0.011773s]\\nBuilding seed filter...  [0.001083s]\\nSearching alignments...  [0.000965s]\\nDeallocating buffers...  [1.5e-05s]\\nComputing alignments...  [0.0129s]\\nDeallocating reference...  [8e-06s]\\nLoading reference sequences...  [2.4e-05s]\\nDeallocating buffers...  [0.000561s]\\nDeallocating queries...  [0.000514s]\\nLoading query sequences...  [3e-05s]\\nClosing the input file...  [2.9e-05s]\\nClosing the output file...  [0.000235s]\\nClosing the database file...  [1.5e-05s]\\nDeallocating taxonomy...  [4e-06s]\\nTotal time = 0.335558s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000127s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000825s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000121s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000237s]\\nOpening the output file...  [9.3e-05s]\\nLoading query sequences...  [0.021769s]\\nMasking queries...  [0.097142s]\\nBuilding query seed set...  [0.019072s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.015003s]\\nAllocating buffers...  [3.5e-05s]\\nLoading reference sequences...  [5.7e-05s]\\nBuilding reference histograms...  [0.000226s]\\nAllocating buffers...  [2.8e-05s]\\nInitializing temporary storage...  [0.031329s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001134s]\\nBuilding query index...  [0.013213s]\\nBuilding seed filter...  [0.001111s]\\nSearching alignments...  [0.001228s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000579s]\\nBuilding query index...  [0.01279s]\\nBuilding seed filter...  [0.00107s]\\nSearching alignments...  [0.001021s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000491s]\\nBuilding query index...  [0.013747s]\\nBuilding seed filter...  [0.001013s]\\nSearching alignments...  [0.00104s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000557s]\\nBuilding query index...  [0.011719s]\\nBuilding seed filter...  [0.001029s]\\nSearching alignments...  [0.001199s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000532s]\\nBuilding query index...  [0.013294s]\\nBuilding seed filter...  [0.001254s]\\nSearching alignments...  [0.001207s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000586s]\\nBuilding query index...  [0.01283s]\\nBuilding seed filter...  [0.001012s]\\nSearching alignments...  [0.0009s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000482s]\\nBuilding query index...  [0.013021s]\\nBuilding seed filter...  [0.001056s]\\nSearching alignments...  [0.000997s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000444s]\\nBuilding query index...  [0.01165s]\\nBuilding seed filter...  [0.001346s]\\nSearching alignments...  [0.001136s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.021678s]\\nDeallocating reference...  [4e-06s]\\nLoading reference sequences...  [1.7e-05s]\\nDeallocating buffers...  [0.000428s]\\nDeallocating queries...  [0.000324s]\\nLoading query sequences...  [1.5e-05s]\\nClosing the input file...  [5.2e-05s]\\nClosing the output file...  [0.00645s]\\nClosing the database file...  [3.3e-05s]\\nDeallocating taxonomy...  [1.3e-05s]\\nTotal time = 0.340129s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000201s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000972s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000104s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000223s]\\nOpening the output file...  [8e-05s]\\nLoading query sequences...  [0.018696s]\\nMasking queries...  [0.115019s]\\nBuilding query seed set...  [0.025812s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.019352s]\\nAllocating buffers...  [3.9e-05s]\\nLoading reference sequences...  [5.4e-05s]\\nBuilding reference histograms...  [0.000212s]\\nAllocating buffers...  [2.9e-05s]\\nInitializing temporary storage...  [0.026047s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001199s]\\nBuilding query index...  [0.018344s]\\nBuilding seed filter...  [0.001169s]\\nSearching alignments...  [0.001155s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000495s]\\nBuilding query index...  [0.014536s]\\nBuilding seed filter...  [0.00123s]\\nSearching alignments...  [0.001087s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.0005s]\\nBuilding query index...  [0.014343s]\\nBuilding seed filter...  [0.001103s]\\nSearching alignments...  [0.001121s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000512s]\\nBuilding query index...  [0.012785s]\\nBuilding seed filter...  [0.00185s]\\nSearching alignments...  [0.001084s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000503s]\\nBuilding query index...  [0.012623s]\\nBuilding seed filter...  [0.001076s]\\nSearching alignments...  [0.001107s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000477s]\\nBuilding query index...  [0.013751s]\\nBuilding seed filter...  [0.001178s]\\nSearching alignments...  [0.001065s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000511s]\\nBuilding query index...  [0.013905s]\\nBuilding seed filter...  [0.001191s]\\nSearching alignments...  [0.001009s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000503s]\\nBuilding query index...  [0.012522s]\\nBuilding seed filter...  [0.001048s]\\nSearching alignments...  [0.001084s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.015483s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [1.9e-05s]\\nDeallocating buffers...  [0.000524s]\\nDeallocating queries...  [0.000331s]\\nLoading query sequences...  [1.3e-05s]\\nClosing the input file...  [1.4e-05s]\\nClosing the output file...  [0.00024s]\\nClosing the database file...  [1.3e-05s]\\nDeallocating taxonomy...  [1.5e-05s]\\nTotal time = 0.359525s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000218s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001044s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.00014s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000626s]\\nOpening the output file...  [8.2e-05s]\\nLoading query sequences...  [0.018761s]\\nMasking queries...  [0.109021s]\\nBuilding query seed set...  [0.0237s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.020402s]\\nAllocating buffers...  [3.5e-05s]\\nLoading reference sequences...  [5.5e-05s]\\nBuilding reference histograms...  [0.000294s]\\nAllocating buffers...  [2.6e-05s]\\nInitializing temporary storage...  [0.03051s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001177s]\\nBuilding query index...  [0.017083s]\\nBuilding seed filter...  [0.001265s]\\nSearching alignments...  [0.001161s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000555s]\\nBuilding query index...  [0.015491s]\\nBuilding seed filter...  [0.001445s]\\nSearching alignments...  [0.001073s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000551s]\\nBuilding query index...  [0.015329s]\\nBuilding seed filter...  [0.001339s]\\nSearching alignments...  [0.00105s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000451s]\\nBuilding query index...  [0.013498s]\\nBuilding seed filter...  [0.001424s]\\nSearching alignments...  [0.001236s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000475s]\\nBuilding query index...  [0.018039s]\\nBuilding seed filter...  [0.001667s]\\nSearching alignments...  [0.0012s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000518s]\\nBuilding query index...  [0.01471s]\\nBuilding seed filter...  [0.001281s]\\nSearching alignments...  [0.00104s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000469s]\\nBuilding query index...  [0.014716s]\\nBuilding seed filter...  [0.001367s]\\nSearching alignments...  [0.001169s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000485s]\\nBuilding query index...  [0.016641s]\\nBuilding seed filter...  [0.001414s]\\nSearching alignments...  [0.001244s]\\nDeallocating buffers...  [2e-05s]\\nComputing alignments...  [0.015932s]\\nDeallocating reference...  [8e-06s]\\nLoading reference sequences...  [2.2e-05s]\\nDeallocating buffers...  [0.000633s]\\nDeallocating queries...  [0.000617s]\\nLoading query sequences...  [1.8e-05s]\\nClosing the input file...  [1.7e-05s]\\nClosing the output file...  [0.000236s]\\nClosing the database file...  [1.4e-05s]\\nDeallocating taxonomy...  [1.6e-05s]\\nTotal time = 0.372942s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000187s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.00094s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000164s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000421s]\\nOpening the output file...  [0.000137s]\\nLoading query sequences...  [0.018746s]\\nMasking queries...  [0.089712s]\\nBuilding query seed set...  [0.023925s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.01706s]\\nAllocating buffers...  [3.5e-05s]\\nLoading reference sequences...  [5.7e-05s]\\nBuilding reference histograms...  [0.000232s]\\nAllocating buffers...  [2.6e-05s]\\nInitializing temporary storage...  [0.085086s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001217s]\\nBuilding query index...  [0.012222s]\\nBuilding seed filter...  [0.001429s]\\nSearching alignments...  [0.001217s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000485s]\\nBuilding query index...  [0.012836s]\\nBuilding seed filter...  [0.00117s]\\nSearching alignments...  [0.001093s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000592s]\\nBuilding query index...  [0.017953s]\\nBuilding seed filter...  [0.001082s]\\nSearching alignments...  [0.001009s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000485s]\\nBuilding query index...  [0.011852s]\\nBuilding seed filter...  [0.001263s]\\nSearching alignments...  [0.001089s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000558s]\\nBuilding query index...  [0.010772s]\\nBuilding seed filter...  [0.001286s]\\nSearching alignments...  [0.001117s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000467s]\\nBuilding query index...  [0.012172s]\\nBuilding seed filter...  [0.001053s]\\nSearching alignments...  [0.000989s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.00046s]\\nBuilding query index...  [0.017133s]\\nBuilding seed filter...  [0.001107s]\\nSearching alignments...  [0.001154s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000461s]\\nBuilding query index...  [0.010539s]\\nBuilding seed filter...  [0.000994s]\\nSearching alignments...  [0.001014s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.012813s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [2e-05s]\\nDeallocating buffers...  [0.000384s]\\nDeallocating queries...  [4.5e-05s]\\nLoading query sequences...  [1.2e-05s]\\nClosing the input file...  [3.5e-05s]\\nClosing the output file...  [0.000184s]\\nClosing the database file...  [1.8e-05s]\\nDeallocating taxonomy...  [1.5e-05s]\\nTotal time = 0.379014s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000159s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000862s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000103s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000405s]\\nOpening the output file...  [0.000114s]\\nLoading query sequences...  [0.016001s]\\nMasking queries...  [0.087107s]\\nBuilding query seed set...  [0.018644s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.01426s]\\nAllocating buffers...  [3.8e-05s]\\nLoading reference sequences...  [5.4e-05s]\\nBuilding reference histograms...  [0.000223s]\\nAllocating buffers...  [2.7e-05s]\\nInitializing temporary storage...  [0.016026s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001169s]\\nBuilding query index...  [0.013705s]\\nBuilding seed filter...  [0.00131s]\\nSearching alignments...  [0.001199s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000484s]\\nBuilding query index...  [0.020223s]\\nBuilding seed filter...  [0.001092s]\\nSearching alignments...  [0.001083s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.00053s]\\nBuilding query index...  [0.012543s]\\nBuilding seed filter...  [0.001037s]\\nSearching alignments...  [0.000948s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.00044s]\\nBuilding query index...  [0.011019s]\\nBuilding seed filter...  [0.000981s]\\nSearching alignments...  [0.001025s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000499s]\\nBuilding query index...  [0.011185s]\\nBuilding seed filter...  [0.000999s]\\nSearching alignments...  [0.001244s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000546s]\\nBuilding query index...  [0.013023s]\\nBuilding seed filter...  [0.001469s]\\nSearching alignments...  [0.000998s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000465s]\\nBuilding query index...  [0.01512s]\\nBuilding seed filter...  [0.001047s]\\nSearching alignments...  [0.001079s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000515s]\\nBuilding query index...  [0.012144s]\\nBuilding seed filter...  [0.001124s]\\nSearching alignments...  [0.001008s]\\nDeallocating buffers...  [2.8e-05s]\\nComputing alignments...  [0.013854s]\\nDeallocating reference...  [4e-06s]\\nLoading reference sequences...  [3e-05s]\\nDeallocating buffers...  [0.000422s]\\nDeallocating queries...  [0.000304s]\\nLoading query sequences...  [1.3e-05s]\\nClosing the input file...  [1.6e-05s]\\nClosing the output file...  [0.000194s]\\nClosing the database file...  [1.3e-05s]\\nDeallocating taxonomy...  [5e-06s]\\nTotal time = 0.300282s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000163s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000885s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000159s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000238s]\\nOpening the output file...  [0.000108s]\\nLoading query sequences...  [0.014491s]\\nMasking queries...  [0.093875s]\\nBuilding query seed set...  [0.018512s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.013808s]\\nAllocating buffers...  [3.5e-05s]\\nLoading reference sequences...  [5.6e-05s]\\nBuilding reference histograms...  [0.000249s]\\nAllocating buffers...  [2.5e-05s]\\nInitializing temporary storage...  [0.024876s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001226s]\\nBuilding query index...  [0.015085s]\\nBuilding seed filter...  [0.001151s]\\nSearching alignments...  [0.001374s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.0005s]\\nBuilding query index...  [0.011611s]\\nBuilding seed filter...  [0.001056s]\\nSearching alignments...  [0.001206s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000506s]\\nBuilding query index...  [0.013934s]\\nBuilding seed filter...  [0.000948s]\\nSearching alignments...  [0.001133s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000439s]\\nBuilding query index...  [0.010422s]\\nBuilding seed filter...  [0.000996s]\\nSearching alignments...  [0.000968s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000447s]\\nBuilding query index...  [0.010383s]\\nBuilding seed filter...  [0.000991s]\\nSearching alignments...  [0.001027s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000471s]\\nBuilding query index...  [0.011988s]\\nBuilding seed filter...  [0.00104s]\\nSearching alignments...  [0.000996s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000479s]\\nBuilding query index...  [0.0135s]\\nBuilding seed filter...  [0.001369s]\\nSearching alignments...  [0.001307s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000564s]\\nBuilding query index...  [0.017277s]\\nBuilding seed filter...  [0.00157s]\\nSearching alignments...  [0.001434s]\\nDeallocating buffers...  [5e-06s]\\nComputing alignments...  [0.014358s]\\nDeallocating reference...  [9e-06s]\\nLoading reference sequences...  [2.4e-05s]\\nDeallocating buffers...  [0.000557s]\\nDeallocating queries...  [3.9e-05s]\\nLoading query sequences...  [1.6e-05s]\\nClosing the input file...  [1.9e-05s]\\nClosing the output file...  [0.000253s]\\nClosing the database file...  [3.4e-05s]\\nDeallocating taxonomy...  [5e-06s]\\nTotal time = 0.310518s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000195s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001513s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000375s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000454s]\\nOpening the output file...  [0.000133s]\\nLoading query sequences...  [0.020942s]\\nMasking queries...  [0.104306s]\\nBuilding query seed set...  [0.027399s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.01681s]\\nAllocating buffers...  [3.6e-05s]\\nLoading reference sequences...  [6.2e-05s]\\nBuilding reference histograms...  [0.00026s]\\nAllocating buffers...  [2.5e-05s]\\nInitializing temporary storage...  [0.028563s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001352s]\\nBuilding query index...  [0.015845s]\\nBuilding seed filter...  [0.001703s]\\nSearching alignments...  [0.001345s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.00087s]\\nBuilding query index...  [0.013966s]\\nBuilding seed filter...  [0.00146s]\\nSearching alignments...  [0.001142s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000565s]\\nBuilding query index...  [0.014599s]\\nBuilding seed filter...  [0.00186s]\\nSearching alignments...  [0.001198s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000677s]\\nBuilding query index...  [0.017799s]\\nBuilding seed filter...  [0.001553s]\\nSearching alignments...  [0.001208s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000537s]\\nBuilding query index...  [0.018346s]\\nBuilding seed filter...  [0.001886s]\\nSearching alignments...  [0.001189s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.00045s]\\nBuilding query index...  [0.019114s]\\nBuilding seed filter...  [0.001097s]\\nSearching alignments...  [0.001102s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000569s]\\nBuilding query index...  [0.014549s]\\nBuilding seed filter...  [0.001507s]\\nSearching alignments...  [0.001125s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000951s]\\nBuilding query index...  [0.013638s]\\nBuilding seed filter...  [0.001196s]\\nSearching alignments...  [0.001132s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.013432s]\\nDeallocating reference...  [2.1e-05s]\\nLoading reference sequences...  [3.4e-05s]\\nDeallocating buffers...  [0.0006s]\\nDeallocating queries...  [4e-05s]\\nLoading query sequences...  [1.6e-05s]\\nClosing the input file...  [5e-05s]\\nClosing the output file...  [0.000293s]\\nClosing the database file...  [3.9e-05s]\\nDeallocating taxonomy...  [1.5e-05s]\\nTotal time = 0.371272s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000185s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000938s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000162s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000427s]\\nOpening the output file...  [0.000113s]\\nLoading query sequences...  [0.02422s]\\nMasking queries...  [0.110805s]\\nBuilding query seed set...  [0.029989s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.017756s]\\nAllocating buffers...  [3.8e-05s]\\nLoading reference sequences...  [5.9e-05s]\\nBuilding reference histograms...  [0.00026s]\\nAllocating buffers...  [2.6e-05s]\\nInitializing temporary storage...  [0.020784s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001298s]\\nBuilding query index...  [0.013155s]\\nBuilding seed filter...  [0.001197s]\\nSearching alignments...  [0.001552s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000754s]\\nBuilding query index...  [0.013593s]\\nBuilding seed filter...  [0.001045s]\\nSearching alignments...  [0.00109s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000573s]\\nBuilding query index...  [0.012832s]\\nBuilding seed filter...  [0.001273s]\\nSearching alignments...  [0.001021s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000598s]\\nBuilding query index...  [0.013086s]\\nBuilding seed filter...  [0.0011s]\\nSearching alignments...  [0.001203s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000485s]\\nBuilding query index...  [0.012771s]\\nBuilding seed filter...  [0.001254s]\\nSearching alignments...  [0.001397s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000859s]\\nBuilding query index...  [0.014183s]\\nBuilding seed filter...  [0.001174s]\\nSearching alignments...  [0.001194s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000702s]\\nBuilding query index...  [0.013266s]\\nBuilding seed filter...  [0.001052s]\\nSearching alignments...  [0.001068s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000497s]\\nBuilding query index...  [0.011968s]\\nBuilding seed filter...  [0.001254s]\\nSearching alignments...  [0.000992s]\\nDeallocating buffers...  [6e-06s]\\nComputing alignments...  [0.015454s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [2.2e-05s]\\nDeallocating buffers...  [0.000604s]\\nDeallocating queries...  [0.000559s]\\nLoading query sequences...  [3.3e-05s]\\nClosing the input file...  [2e-05s]\\nClosing the output file...  [0.000224s]\\nClosing the database file...  [3.5e-05s]\\nDeallocating taxonomy...  [3.1e-05s]\\nTotal time = 0.35268s\\nReported 23 pairwise alignments, 23 HSPs.\\n23 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000208s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.001239s]\\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000151s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000212s]\\nOpening the output file...  [8.1e-05s]\\nLoading query sequences...  [0.020646s]\\nMasking queries...  [0.091404s]\\nBuilding query seed set...  [0.025436s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.025459s]\\nAllocating buffers...  [3.8e-05s]\\nLoading reference sequences...  [6.3e-05s]\\nBuilding reference histograms...  [0.000364s]\\nAllocating buffers...  [4.9e-05s]\\nInitializing temporary storage...  [0.022351s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001353s]\\nBuilding query index...  [0.024172s]\\nBuilding seed filter...  [0.002138s]\\nSearching alignments...  [0.001568s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000806s]\\nBuilding query index...  [0.019107s]\\nBuilding seed filter...  [0.001292s]\\nSearching alignments...  [0.000985s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000496s]\\nBuilding query index...  [0.014s]\\nBuilding seed filter...  [0.001485s]\\nSearching alignments...  [0.001284s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000705s]\\nBuilding query index...  [0.018543s]\\nBuilding seed filter...  [0.001111s]\\nSearching alignments...  [0.00138s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000579s]\\nBuilding query index...  [0.017872s]\\nBuilding seed filter...  [0.001585s]\\nSearching alignments...  [0.001392s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000715s]\\nBuilding query index...  [0.019391s]\\nBuilding seed filter...  [0.001294s]\\nSearching alignments...  [0.001242s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000594s]\\nBuilding query index...  [0.015182s]\\nBuilding seed filter...  [0.001641s]\\nSearching alignments...  [0.001232s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000489s]\\nBuilding query index...  [0.015522s]\\nBuilding seed filter...  [0.001164s]\\nSearching alignments...  [0.001056s]\\nDeallocating buffers...  [9e-06s]\\nComputing alignments...  [0.015836s]\\nDeallocating reference...  [5e-06s]\\nLoading reference sequences...  [1.9e-05s]\\nDeallocating buffers...  [0.000615s]\\nDeallocating queries...  [0.000275s]\\nLoading query sequences...  [1.8e-05s]\\nClosing the input file...  [2.3e-05s]\\nClosing the output file...  [0.000325s]\\nClosing the database file...  [1.9e-05s]\\nDeallocating taxonomy...  [4.3e-05s]\\nTotal time = 0.376223s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000185s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.000973s]\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nScoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)\\nTemporary directory: /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond\\nOpening the database...  [0.000255s]\\n#Target sequences to report alignments for: 25\\nOpening the input file...  [0.000372s]\\nOpening the output file...  [0.000149s]\\nLoading query sequences...  [0.019831s]\\nMasking queries...  [0.100081s]\\nBuilding query seed set...  [0.026929s]\\nAlgorithm: Double-indexed\\nBuilding query histograms...  [0.017796s]\\nAllocating buffers...  [2.6e-05s]\\nLoading reference sequences...  [5.4e-05s]\\nBuilding reference histograms...  [0.000238s]\\nAllocating buffers...  [1.9e-05s]\\nInitializing temporary storage...  [0.018041s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 0.\\nBuilding reference index...  [0.001324s]\\nBuilding query index...  [0.013041s]\\nBuilding seed filter...  [0.001127s]\\nSearching alignments...  [0.001252s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 1.\\nBuilding reference index...  [0.000429s]\\nBuilding query index...  [0.012324s]\\nBuilding seed filter...  [0.001044s]\\nSearching alignments...  [0.00115s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 2.\\nBuilding reference index...  [0.000557s]\\nBuilding query index...  [0.015605s]\\nBuilding seed filter...  [0.001604s]\\nSearching alignments...  [0.001223s]\\nProcessing query chunk 0, reference chunk 0, shape 0, index chunk 3.\\nBuilding reference index...  [0.000595s]\\nBuilding query index...  [0.015849s]\\nBuilding seed filter...  [0.000976s]\\nSearching alignments...  [0.001046s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 0.\\nBuilding reference index...  [0.000453s]\\nBuilding query index...  [0.010696s]\\nBuilding seed filter...  [0.000998s]\\nSearching alignments...  [0.001159s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 1.\\nBuilding reference index...  [0.000509s]\\nBuilding query index...  [0.011825s]\\nBuilding seed filter...  [0.001027s]\\nSearching alignments...  [0.001015s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 2.\\nBuilding reference index...  [0.000475s]\\nBuilding query index...  [0.018077s]\\nBuilding seed filter...  [0.00193s]\\nSearching alignments...  [0.001339s]\\nProcessing query chunk 0, reference chunk 0, shape 1, index chunk 3.\\nBuilding reference index...  [0.000581s]\\nBuilding query index...  [0.017198s]\\nBuilding seed filter...  [0.001132s]\\nSearching alignments...  [0.001218s]\\nDeallocating buffers...  [8e-06s]\\nComputing alignments...  [0.014956s]\\nDeallocating reference...  [6e-06s]\\nLoading reference sequences...  [3.9e-05s]\\nDeallocating buffers...  [0.000527s]\\nDeallocating queries...  [0.000428s]\\nLoading query sequences...  [1.5e-05s]\\nClosing the input file...  [6.3e-05s]\\nClosing the output file...  [0.004414s]\\nClosing the database file...  [3.2e-05s]\\nDeallocating taxonomy...  [5e-06s]\\nTotal time = 0.34456s\\nReported 21 pairwise alignments, 21 HSPs.\\n21 queries aligned.\\n'\n",
      "b'diamond v0.9.24.125 | by Benjamin Buchfink <buchfink@gmail.com>\\nLicensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>\\nCheck http://github.com/bbuchfink/diamond for updates.\\n\\n#CPU threads: 8\\nLoading subject IDs...  [0.000232s]\\nScoring parameters: (Matrix=custom Lambda=0.267 K=0.041 Penalties=11/1)\\nDB sequences = 21\\nDB sequences used = 21\\nDB letters = 3334\\n#Target sequences to report alignments for: 25\\nGenerating output...  [0.002267s]\\n'\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "105"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "def run_diamond(database,input_fasta,output_path):\n",
    "    output_file = os.path.join(output_path, os.path.splitext(os.path.basename(input_fasta))[0] + \".matches\")\n",
    "    diamond_cmd = \"diamond blastp -d %s -q %s -a %s\"%(database,input_fasta,output_file)\n",
    "    p = subprocess.Popen(diamond_cmd, shell=True, stdin=subprocess.PIPE, stdout=subprocess.PIPE,\n",
    "                         stderr=subprocess.STDOUT, close_fds=True,cwd=\"./\")\n",
    "    output = p.stdout.read()\n",
    "    print(output)\n",
    "\n",
    "def view_diamond(diamond_matches,m8_output):\n",
    "    view_cmd = \"diamond view -a %s -o %s\"%(diamond_matches,m8_output)\n",
    "    p = subprocess.Popen(view_cmd, shell=True, stdin=subprocess.PIPE, stdout=subprocess.PIPE,\n",
    "                         stderr=subprocess.STDOUT, close_fds=True,cwd=\"./\")\n",
    "    output = p.stdout.read()\n",
    "    print(output)\n",
    "    \n",
    "outdir = \"/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond/\"\n",
    "dbfile = \"/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond/PAL_hkg\"\n",
    "prokka_list = glob.glob(\"/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/*/*.faa\")\n",
    "scaffold_list = glob.glob(\"/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/top_100/*.fasta\")\n",
    "\n",
    "strain_list = []\n",
    "for item in scaffold_list:\n",
    "    strain = os.path.basename(item).split(\".\")[0]\n",
    "    strain_list.append(strain)\n",
    "    \n",
    "count = 0\n",
    "    \n",
    "for prokka_faa in natsorted(prokka_list):\n",
    "    strain = os.path.basename(prokka_faa).split(\".\")[0]\n",
    "    if strain in strain_list:\n",
    "        count += 1\n",
    "        run_diamond(dbfile,\n",
    "                    prokka_faa,\n",
    "                    outdir)\n",
    "        view_diamond(\"%s%s.matches.daa\"%(outdir,strain),\"%s%s.m8\"%(outdir,strain))\n",
    "\n",
    "count"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Selecting genes (the highest %ID homolog per PAL housekeeping gene) and adding to existent aligments"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {
    "scrolled": false
   },
   "outputs": [],
   "source": [
    "diamond_columns = [\"qseqid\",\"sseqid\",\"pident\",\"length\",\"mismatch\",\"gapopen\",\n",
    "                   \"qstart\",\"qend\",\"sstart\",\"send\",\"evalue\",\"bitscore\"]\n",
    "                    #retrived from https://www.biostars.org/p/166013/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "93\n",
      "20.79\n"
     ]
    }
   ],
   "source": [
    "def get_hkg_dict(m8_file):\n",
    "    prokka_hkg = {}\n",
    "    diamond_df = pd.read_csv(m8_file,sep=\"\\t\",names=diamond_columns).sort_values(by=\"sseqid\")\n",
    "    for i,r in diamond_df.iterrows():\n",
    "        if diamond_df.qseqid.loc[i] not in list(prokka_hkg.values()):\n",
    "            subset = diamond_df[diamond_df.sseqid == diamond_df.sseqid.loc[i]]\n",
    "            max_id = subset.loc[subset['pident'].idxmax()][\"pident\"]\n",
    "            max_qseq = subset.loc[subset['pident'].idxmax()][\"qseqid\"]\n",
    "            max_sseq = subset.loc[subset['pident'].idxmax()][\"sseqid\"]\n",
    "            prokka_hkg[max_qseq] = max_sseq\n",
    "    assert(len(prokka_hkg.keys()) == len(np.unique(diamond_df.sseqid.values)))\n",
    "    return prokka_hkg\n",
    "\n",
    "def extract_genes(m8_file,prokka_folder,hkg_folder,prokka_hkg):\n",
    "    strain = os.path.basename(m8_file).split(\".\")[0]\n",
    "    prokka_faa = \"%s%s/%s.faa\"%(prokka_folder,strain,strain)\n",
    "    input_handle  = open(prokka_faa,\"r\")\n",
    "    for seq_record in SeqIO.parse(input_handle, \"fasta\"):\n",
    "        if int(seq_record.id) in prokka_hkg:\n",
    "            hkg_homolog = str(prokka_hkg[int(seq_record.id)].split(\"_Moorea\")[0])\n",
    "            output_handle = open(\"%s%s.faa\"%(hkg_folder,hkg_homolog), \"a\")\n",
    "            new_id = \"%s\"%(strain)\n",
    "            output_handle.write(\"\\n\\n>%s %s\\n%s\\n\" % (\n",
    "               new_id,\n",
    "               \"\",\n",
    "               seq_record.seq))\n",
    "            output_handle.close()\n",
    "    input_handle.close()\n",
    "\n",
    "m8_list = glob.glob(\"/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/diamond/*.m8\")\n",
    "prokka_folder = \"/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/prokka/\"\n",
    "hkg_folder = \"/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/\"\n",
    "\n",
    "complete_count = 0\n",
    "dict_len = []\n",
    "\n",
    "for m8_file in m8_list:\n",
    "    prokka_hkg = get_hkg_dict(m8_file)\n",
    "    dict_len.append(len(prokka_hkg))\n",
    "    if len(prokka_hkg) == 21:\n",
    "        complete_count += 1\n",
    "        extract_genes(m8_file,prokka_folder,hkg_folder,prokka_hkg)\n",
    "\n",
    "print(complete_count)\n",
    "print(np.average(dict_len))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "100"
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(m8_list)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Running MUSCLE and TrimAl**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#conda install -c bioconda muscle\n",
    "#conda install -c bioconda trimal"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/11_rpsC.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/10_rpsB.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/4_rplF.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/15_rpsH.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/20_rpsO.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/18_rpsL.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/19_rpsM.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/9_rplV.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/2_rplC.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/6_rplM.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/8_rplR.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/7_rplN.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/3_rplE.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/13_rpsE.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/12_rpsD.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/17_rpsK.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/16_rpsI.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/14_rpsG.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/1_rplA.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/5_rplK.faa',\n",
       " '/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/21_rpsQ.faa']"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "input_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [],
   "source": [
    "for fasta_file in input_list:\n",
    "    muscle_in = fasta_file\n",
    "    muscle_out = os.path.join(os.path.dirname(fasta_file),os.path.basename(fasta_file).strip(\".faa\")+\".ali.faa\")\n",
    "    trim_out = os.path.join(os.path.dirname(fasta_file),os.path.basename(fasta_file).strip(\".faa\")+\".trimmed.faa\")\n",
    "    subprocess.call(\"muscle -in %s -out %s\"%(muscle_in,muscle_out),shell=True)\n",
    "    subprocess.call(\"trimal -in %s -out %s -gt 0.8 -st 0.001 -cons 70\"%(muscle_out,trim_out),shell=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10_rpsB.ali.faa     15_rpsH.trimmed.faa \u001b[31m20_rpsO.faa\u001b[m\u001b[m         6_rplM.ali.faa\r\n",
      "\u001b[31m10_rpsB.faa\u001b[m\u001b[m         16_rpsI.ali.faa     20_rpsO.trimmed.faa \u001b[31m6_rplM.faa\u001b[m\u001b[m\r\n",
      "10_rpsB.trimmed.faa \u001b[31m16_rpsI.faa\u001b[m\u001b[m         21_rpsQ.ali.faa     6_rplM.trimmed.faa\r\n",
      "11_rpsC.ali.faa     16_rpsI.trimmed.faa \u001b[31m21_rpsQ.faa\u001b[m\u001b[m         7_rplN.ali.faa\r\n",
      "\u001b[31m11_rpsC.faa\u001b[m\u001b[m         17_rpsK.ali.faa     21_rpsQ.trimmed.faa \u001b[31m7_rplN.faa\u001b[m\u001b[m\r\n",
      "11_rpsC.trimmed.faa \u001b[31m17_rpsK.faa\u001b[m\u001b[m         2_rplC.ali.faa      7_rplN.trimmed.faa\r\n",
      "12_rpsD.ali.faa     17_rpsK.trimmed.faa \u001b[31m2_rplC.faa\u001b[m\u001b[m          8_rplR.ali.faa\r\n",
      "\u001b[31m12_rpsD.faa\u001b[m\u001b[m         18_rpsL.ali.faa     2_rplC.trimmed.faa  \u001b[31m8_rplR.faa\u001b[m\u001b[m\r\n",
      "12_rpsD.trimmed.faa \u001b[31m18_rpsL.faa\u001b[m\u001b[m         3_rplE.ali.faa      8_rplR.trimmed.faa\r\n",
      "13_rpsE.ali.faa     18_rpsL.trimmed.faa \u001b[31m3_rplE.faa\u001b[m\u001b[m          9_rplV.ali.faa\r\n",
      "\u001b[31m13_rpsE.faa\u001b[m\u001b[m         19_rpsM.ali.faa     3_rplE.trimmed.faa  \u001b[31m9_rplV.faa\u001b[m\u001b[m\r\n",
      "13_rpsE.trimmed.faa \u001b[31m19_rpsM.faa\u001b[m\u001b[m         4_rplF.ali.faa      9_rplV.trimmed.faa\r\n",
      "14_rpsG.ali.faa     19_rpsM.trimmed.faa \u001b[31m4_rplF.faa\u001b[m\u001b[m          PAL_core_genes.faa\r\n",
      "\u001b[31m14_rpsG.faa\u001b[m\u001b[m         1_rplA.ali.faa      4_rplF.trimmed.faa  \u001b[34mdiamond\u001b[m\u001b[m\r\n",
      "14_rpsG.trimmed.faa \u001b[31m1_rplA.faa\u001b[m\u001b[m          5_rplK.ali.faa\r\n",
      "15_rpsH.ali.faa     1_rplA.trimmed.faa  \u001b[31m5_rplK.faa\u001b[m\u001b[m\r\n",
      "\u001b[31m15_rpsH.faa\u001b[m\u001b[m         20_rpsO.ali.faa     5_rplK.trimmed.faa\r\n"
     ]
    }
   ],
   "source": [
    "!ls ./pnas-phylogenomics-top_100/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Concatenating processed aligments**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/1_rplA.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/2_rplC.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/3_rplE.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/4_rplF.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/5_rplK.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/6_rplM.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/7_rplN.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/8_rplR.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/9_rplV.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/10_rpsB.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/11_rpsC.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/12_rpsD.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/13_rpsE.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/14_rpsG.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/15_rpsH.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/16_rpsI.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/17_rpsK.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/18_rpsL.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/19_rpsM.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/20_rpsO.trimmed.faa\n",
      "/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/21_rpsQ.trimmed.faa\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'Obscuribacter_phosphatis_(Mle1_12)': 'TKRRKKLDFEH-NQVAEEAIALKKENSVKFQTVEVHFKLGVNPKLNDQQVRSTVNLPGGTGKVRVAVVAKGDKLEASGADHFGAEELVEKIGAGLDFDLVATPDAMAMLSKMGKILGPKGLMPNPKDGTVTDVGTVELKAGKVSFRAEKDGIVQMIGKLSFDARLLNLAAVVDQVQKVKPSSVKGTYIKSYVSTTMGPGVKLDLNTLQDLSMTLGVMGKKVGMTQVFDENGLAVPVTVVKLGDNIVTEVKTKDKHGYQAVQVGGFKVAEKKLNKPELGGFKKKELAPMQPLKEFRLKDACAYNVGEALKEEILSAGMRVDVRGRSIGKGFQGTIKRYNAGRGPMSHGSKFHRSMGSIGAGTTPGRVFRGLHMPGHMGDDNTCARRIKVVKVDTEKGLLLVKGSIPGCDGALVVVTPSKKWN-----DIKYKQVAHLKQFYQNEMQVPRVVKVVINMGLGEAANNAKAIDSGVKELAIAGQKPVVNRARKSIATFKLRKGQPVGASVTLRGKRMYDFLAKLIHVALPRIRDFRGLSSKSFDGRGNFSVGIKEQLIFPEINYEQIDAVRGMDISIVTTAPTDQEAHALLAAMGMPFK-MSRIGKAPITVPAGVTVQFADNKVSVKGPKGQLTKEFRPEIAFKQEGDTITVLRKSEDRSVR--SLHGLSRTLLANMVKGVSTGFDKTLEIVGVGYRASMDGKKLVMQLGYSHSVEIDAPEGITIAVGKGNVITVSGADKQAVGDMSSFIRGKRPPEVYKGKGVKYQGEYIRRKAGKAGKKK---VKKVVGKVKLQIQAGKANPAPPIGPALGQHGVNIMQFCKEYNAKTQDKAGSVVPVEITVYEDRSFDFVLKTPPAAELIKKAANVEKGSAAPNKNKVGTLSRAKVTEIAKVKMPDLNATTLEAAESMIQGTARNMGITIVD-KTCPEVTRWLVVDAGKTLGRLAAEVALLRGKHKPFTPHVDCGDFVVVVNAEKIVSGKKTDKFYRRHSGFPGGFKEESLHLRRRPIAIIEAVRGMLPHTRLGDQQFTKLKVYAGAHPHAQKPAHLPKHVTMIQQTRLTCADNTGARELMCIRVLGGSNRYAVGDIVGVVKDAPNMVKKSDVVRCVVVRVTVKRPDGTIRFDDNAAVIVQDGNPKGTRVFGPIARELRDKNFTKIISLAPEVLNKQDRKASITRHRRIRRVGCERPRLCVYRSNHIYAQIIDDTLSASTLDDLKKASNESAVGELIARAKEKGIQVVFDRGGYIYHGRVAAVAEAAREAGLDF------SKVAKWVRSPKVRRVVNEIRGKEVAKARVMLRFMPYAAARVVEKLLHSAVSNLVHSQNEADFKVVEAFCDGPMERFQPRARGRAYPILKRSSHITLKLSD---------------------------------------LPVASMRQ-----LLEAGVHFGHQTRRWNPKMRPYIY-GERNGIYIIDLEQTTRALDRACEFVRKAASEKKNIIFVGTKKQAAEIVEEEAKRC-GAHFVNRRWLGGMLTNFETIRLRINRLKELEEMRDTGDFYRRGKKEQSMLNRELTKLEKSLGGIKTMRGKPDLLFIIDQKRELIAVSEAKKI--GIATVGIIDTNC--DPDGIDYVIPGNDDSIRSIKLITSKIADSILEGKQ--GQAYSSPAEKAWKEPEPQMEHADA-------PDAFGDEAKALGQKVNPVGFRVGIHRG----------WSSNWFVNKKDVAPLIEEDH--RIRTFLKKE-L-----TGAGVSKVDISRKADQIQIEIF-------TARPGVVVGKGG-----------------------------------------------------------------------AGIETLS----------------------AKIRGMIN-------RPGLDVKISILE-------------------------------------VNRVDLEAKLVAESIAQQLEKRVAFRRAMKQAMQRCMRA-GAVGVKIMVAG-----------------------------RLGGAEIARTE-----WAKEG---RIPLQTLRADIDYATAEATTL-------------------------------------------MGIIGVKVWVF-----------------RGELEPGQWS-------PP--------NIKSQSERSQEGSR--EARGPVKSGRRRKAGLARYTDSVCKLRNEGFLKGTTTKCERRRYGPGQHGQRKKQSEYALQLREKQKVRQTYGVLESQFSAYFEQATRKPTGLLQLLERLDNLVHRSGIVPSRAQARQLIRHGHFTVNGKVVSIPSYQVRPGEVISVSDKAKKVIKILETSPAAAHPWMEVSINLASMINSREDLDQSIKEALIVEYYSR--EEAPSAAGDGVRRGRETKRE---RVESEWEEKIIQVRRVTKVVKGGKKLSFRAVVAVGNGKGQVGIGVGKAAEVISAIQKGVVDAKKSLVNVPLV-GSTIPHQIVGKRGSSRILLKPAAKGTGVIAGGAARAILELAGVGDVLSKSLGSRSPLNVARATIDGLKGLRTFEEAARLRG--ISIRQLFAMSRRKRIPADYNSQLVSRFVNRMMQGKRSTASLFYEALLDRTKEPLDFNKAVKNVTPLVEVKARRVGGSTYQVPVEVKTRGFALATQWIINNSRKRGRTFARLSELFDAANGTGASVKKKDETHKMAEANKAFAHYRYMPTDPISDMFTRIRNAQASAVMPASKQKVAIAELLRREGFISETAGSEQMRIVLKYGGKGEVVIGLKRVSKPGLRVYRQNMPRVYGGLGIAIVSTSQGLMTDRQARKSNLGGEVVAEVW-SVIQYIGTGRRKSATARVRLVPGTGQVINSRMDYVGGRDLRIFTSFHAADAMGRFDVHVLVGGGIAGQIGAIRHGIARALSEAAPQNRVLREGLLTRDPRVKERKKYGRRARKRFQFSKRMAKKARTKKERRNVLQGVVIASTFNNTIVSSTDGQVIAWASAGTVGFKGAKKGTPFAAQQAAESVAKRSMDQGMRQVEVLVKGPGAGRETAIRALQSGLEITLIKDVTPIPHNGCRPPKRRRVVPTINQLIRERTYKTKSPALKCPQRRGVCTQVKTTTPKKPNSALRKIARVRLTNGIEVTAYIPGVGHNLQEHSVVLIRGGRVKDLPGVRYHIVRGALDTAGVKDRMQGRSKYGAKRPKMARIAGVDLPRNKRTLIALTYIHGIGRTSAKVCVGIADHRRIQDLSEELIRVENYQVEGDLRRELNVKRLEINYRGQRHRRGLPTRGQRTKTNARTRRGKTVAGKKQAGMALLARKQEIISNQNHTKDTGSPEVQVAMLTERINQLTEHLKHKDHSRRGLLMMVGKRRRLLQYLMDERYRLIKLGLRRMPRKVLVGKVVSNKMDKTVVVAVENRFPHRKYEKQMVETRKFKAHDPENKCQMGDVVRIEECPPVSKDKKWQVIEITGHSISN-----------',\n",
       " 'Cyanobacterium_sp._(Zagget_111_MP)': 'TKKQKKMQMGF-VQAAVDAIKLQEISKSKFETVECHMRLGIDVRQADQQIRSTVVLPAGLGKVKVCVIAKGAKAEAVGADFAGAEEIIDKISGGFEFDLIATPDCMGMLGKLGRVLGPKGLMPNPKTGTVTDIAAVDAKAGKVEYRADKQGMIHCIGKIQFAGDLVNYATLADAILKAKPSSAKGTFVKSYLSTTMGPGIKLDPKTFAAEVMTLGVIGKKVGMTQIFDEQGLAIPVTVIKVDETVVTQVKTVETDGYNAIQVGTVAAKEKHLTKAQLGHFKKNNLSNYRHLQEFRVENPQDYKVGDKVELSVLDNVEKVDVTGKSIGKGFQGTVKRWNFGRGPMAHGSKNHREPGSIGAGTTPSRVIKGKRMAGNMGNERVTITKLKLVKVDSANGLVLVKGSVPGCEGRLVTIVPTRKWN----VNLKYEDVPAMEKFYTNDHQVPKLVKIVLNMGVGEASHNSKIAEAIEAQLTIAGQKAVVTKAKKSIASYKLREGMPVGAMATLRGERMYDFLQKLICVVLPRIRDFRGISAKSFDGRGNYTLGLKEQALFPEITYEEVDLVKGMNVSVITTAETDDEARELLRLLGMPFKSMSRIGKKPIEIPQGVEVKIEGQVVTAKGPLGTEVVNVRPEIEVKIEDNHVVLAKVGTSRETD--ALYGLSRTLVANAIHGVKDGFVKKLEINGVGYRAQMQGTTLNMALGYSHPVDIVPPQGITITVDANTKITVKGSNKQMVGDIAALIRSKRPPEVYKGKGIKYEGEYIRRKAGKAGKK---MAKKVVGKIKLQIEAGKANPSPPVGPALGQHGVNIMDFCKQFNAKTQDKAGYIIPVVIDVYEDKSFTFIIKSPPAPVLIKKALNISKGSAAPNKDKVAEITRAQLEEIAKIKMDDLNATTMDAAVKMIAGSCRAMGVTVKD-KTSAEVERWYVIDAGEVLGRLAVRIAILRGKNKPYTPNVDTGDFVIVINAEKVVTGKKTDKIYLHHTGYPGGLKSASFELMKDPRIVIEAVRGMLQHNTLGAEQFNKLKVYVGSHPHAQKPVL----EKMIQETRLEVADNTGAKQLLCIRVMGSSNKFAVGDIVASVKDAPNMVKKGEVVRAVVVRTKIKRNDGVIRFDENAAVIINDGNPRGTRVFGPVARELRDKGYMKIVSLAPEVIKQRQRKPLQKRHKSIRKLGTEFPRLAVYRSTHIYAQLIDDTIAASSNDDLKELGNEAAVGEAIAKAKKAGVCVVFDRGGFLYHGRVAALADAAREAGLMF------AKAQTDIKTAKLRRVINEVRGKAVVEAQDMLRFMPYFAARVVEKNLKAAVANAREQGVAAELKISQIYADSVYKRARTRAQGRMYSRMKRTSHLTLVVSDA--------------------------------------MPTASMIE-----LLEAGVHFGHQTQRWNPKMKPYIY-GARNGIYVIDLRKTTDLLDEAYNLIRDYAAKGKNVLFVGTKKQAAEVVAEEAARS-GAYYINRRWLGGMLTNFETIRGRVNKLKEMEEFISNGYIDKLPKKEVAQINRQLAKLSKTLGGIKEMRGLPDVIFIVDQAKEEIAIKEANKL--GIPVVCLADTNA--NPDGINYIIPGNDDAIRSIKLITSKLADAVLEGKQLRENKANEAKKEKISAED-------------AGVTEEVKEETVMGQKTHPTGFRVGIIQP----------WVSTWFAKVKDYAVFVAEDA--KIRKFVKKK-L-----YAAGVSRILIARKAQNITITVV-------TAKPGIVVGRGG-----------------------------------------------------------------------QGIDELR----------------------QDVTKHIG----------KSVVINVVE-------------------------------------VARIDADAQLVAESIAQQLEKRIAFRRAMKQAMQRTMRS-GVQGIKVMVSG-----------------------------RLGGAEIARSE-----WAKEG---RIPLQTLRADVDYGFTEADTI-------------------------------------------MGKIGVKVWIF-----------------KGNLMPGEMA---DQNIKL------------KSGKD-NSDK--GGRRPRRARNTTAESMSRYTGPNNKLRNKKDLS--NRKLSMRQTASGQHGARKKLSEYAIHLAEKQKIRFTYLVSEKQFVRYYNEAARGVTGLLQLLERLDNILYRAGFGITRKQTRQIVNHGHVLVNGRKVDIPSYIVKAGDVITVKSKSSKYLNVIGMIDMACAPWMTIDKALITFDREREELDPEIKEQLVIEYYSK--NVTEAAETVSEEVKSETTEE---KPQSEWTETVVQISRVTKVVKGGKKLSFRAIVIVGNKKGQVGIGCAKASEVIIAIQKAIADGRKNVINVPIFKT-TIPHPITGKSGAGAVMLRPASAGTGIIAGGAVRSVLELAGIENILSKSLGSKSPLNAANATIEALKSLTPFNDVAKKRG--LTMAELLNMSRRRRIPADYNSVDISKFINRIMRGKKSLAKIFYATMIERTNDPIEFQKAMTNATPLLEVKARRIGGSTYQVPIEVKDRGFALSSSWLIESAKKRGKSFIKLTELIDASNGQGAACKKREDTHKMAEANKAFAHYRYMTTDPIADMLTRIRNASHSTVVPASKLKIQLAKLLKEEGYIDEVEAGFQISITLKYEANGRPVIKLERISKPGLRHYSKKLQKVLGGMGVAIVSTPKGLLTDRKARKENVGGEVLCYVY-AKEIIMATGRRKTSIAVVKLVKGNGRIVNGRLTYIGNRPIEVYRPLVLTETQDRYDVRVKAGGGVVSQCGAIRHGISRALVKSNEEFRVLKEGLLTRDPRVKERKKYGRRARKRFQFSKRMAR-K-TKKEKKNVLQGVVIQSTFNNTIVTSTDGNAIAWATAGASGFKGARKGTPFAAQSAAELVAKKSIDQGMKKVEVHIKGPGSGRETAIRAIQAGLEITLIKDVTPIPHNGCRPPKKRRVMPTINQLVRERQDKTKSPALMCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTNGFEVTSYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIVRGTLDTAGVANRAQSRSKYGAKRDSMARLVGVDLPRNKRMEVALTYIFGIGPTRAKILTGISPDLRTDDLTDDILRLSNYHIEGDLRRIMNIKRLEIGYRGLRHKRKLPCRGQRTKTNARTQRGKPVAKKK---MSINLNKQEIVKFGKDEKDTGSSAVQVAMLTQKISELTEHLKNKDATKRGLLMMVGKRKRLLAYVKNDEYRLIKLKIRGMPKKEKQGIVVSNKMDKTVVVKVADYEPHPKYKKIIESNKHYKAHDAENACSEGDVVRIVETRPISADKRWTVAEIVEKAR-------------',\n",
       " 'YS2_(Zagget_bin_221)': 'TKKQKKMQMGF-EQSAVETIKLKEISKSKFETVECHMRLGIDVRQADQQIRSTIVLPAGLGKVRVCVIAKGPKAEAAGAEYAGAEEIVDKIAGGFEFDLIATPDCMGMLGKLGRVLGPKGLMPNPKTGTVTDIAAVDAKAGKVEFRADKQGMIHCIGKVEFSEDLTNFATLVDAILRAKPSAAKGTFVKSYLSTTMGPGLKLDPKNIAGEVMTLGVIGKKLGMTQIFDEQGLAIPVTVIKVDETVVTQVKTVETDGYNAIQVGTVPAKEKHLTKAQLGHFKKNNLSNYRHLQEFRVENPQDYKVGDKIELSVLENVEKVDVTGKSIGKGFQGTVKRWNFGRGPMAHGSKNHREPGSIGAGTTPSRVIKGKRMAGNMGNERVTITKLKVVKVDSANNLVLVKGSVPGCEGRLVTLVPTRKWN----VSLKYEEVKALEKFYTNEHQVPALHKIVLNMGVGEASHNSKIAETIEAQLTIAGQKAVVTKAKKSIASYKLREGMPVGAMVTLRGERMYDFLQKLICVVLPRIRDFRGISAKSFDGRGNYTLGLKEQALFPEITYEEVDLVKGMNVSIITTANTDDEARELLRLLGMPFKLMSRIGKLPIAIPAGVEVKIEGHTVTAKGPKGTEVVTVRPEINVAVEDNHVVVTRVDDSREAR--SLHGLSRTLIANAIKGVSEGFEKKLEIVGVGYRANMEGSNLNMALGYSHPVIIVPPAGITISVEANTKVSVKGTNKQLVGDVAAEIRGKRPPEVYKGKGVKYEGEYIRRKAGKAGKK---MAKKVTGKIKLQIEAGKANPSPPVGPALGQHGVNIMDFCKQFNARTESQAGYIIPVVIDVYEDRSFTFVTKSPPAPVLIKKALGLTRGSAAPNKDKVGEITQAQLEEIAKTKMADLNATTLEAAVEMIKGSCRAMGVTVKA-KTSAEVERWYLIDADEILGRLAVRVAILRGKNKPYTPNVDTGDFVIVINADKVVTGNKTDKIYYHHTKYPGGLKSASFELQKDGRLAIEAVKGMLQHNTLGAEQFNKLKVYVGSHPHAQKPIL----EKMIQETRLEVADNTGAKELLCIRVLGSSNKFGVGDIVATVKDAPNLVKKGEVVKAVIVRTKIKRNDGVIRFDDNAAVIINDGNPRGTRVFGPVARELRDKGYMKIVSLAPEVIKQENRKAKQKRHRSLRKLGTEFPRLAVYRSTHIYAQLIDDTLSASSNDDLKELGNEAAVGETIAKAKKAGVCVVFDRGGFLYHGRVAALADAAREAGLCF------AKAQTDIKTAKLRRVINEVRGKAVKEAQDMLRFMPYFAARVVEKNLKAAIANAREKGVTPELRISEIFADSVYKRARTRAQGRMYSRLKRTSHLTLVVSDK--------------------------------------MPVASMIE-----LLEAGVHFGHQTQRWNPKMKPYIY-GARNGIYVIDLRKTTDLLDAAYAAVRDFAARGKNVLFVGTKKQAADVVAEEAKRA-GAYYINRRWLGGMLTNFETIRSRVNKLRELEDFIASGHIEKLPKKEVAQINRQVAKLSKTLGGIKEMRGLPDIIFIVDQQKELIAIKEANKL--NIPVICLADTNA--NPEGIDYIIPGNDDAIRSVKLIVSRLADAVLEGKQLRENNANTTAKEEIKAED---GVVES-----------------MGQKTHPTGFRVGIIQP----------WASTWFAKIREYGAFVAEDA--KIRKFVKKK-L-----YAAGVSKILIARKAQNITVTVV-------TAKPGIVVGRGG-----------------------------------------------------------------------QGIEELK----------------------QAVSKYIG----------KAVVINVVE-------------------------------------VARIDADAQLVAEAIAQQLEKRVAFRRAMKQAMQRTMRS-GVQGIKVMVSG-----------------------------RLGGAEIARSE-----WAKEG---RIPLQTLRADVDYGFTEADTI-------------------------------------------MGKIGVKVWIF-----------------KGNLMPGEKADMNIKAKSG-------------KENNNNAER--GQRRPRR----NNKAMARYTGPKNKLRNWGDLS--NRKLSMRQTASGQHGARKKLSEYALHLNEKQKIRLTYLVSEKQFAKYYQEATRGVTGLLQLLERLDNVLFRAGFGVTRSQSRQIVNHGHVLVNGKKVDIPSYLVKAGDVITVRSKSSDFIKVIESIDVACAPWMTIDRALITFDREREELDPDFKEQLVIEYYSK--EVAVEENAADDRQSRETTEE---KPQSEWTERVVQIRRVTKVVKGGKKLSFRAVVIVGNKKGQVGVGCAKASEVIIAIQKAIADGRKNLVTVPIFKT-TIPHPITGRSGSGEVMLRPASQGTGIIAGGSVRLVLELAGIENILSKSLGSKSPLNAAYATIEALKNLTPFSEVAARRG--LTLAQLLNMSRRKRVPADYNSVDISKFINRIMRGKKSIAKIFYATMIERTGKALEFQKALTNATPLLEVKARRIGGSTYQVPIEVKDRGFALSSSWLIEAAKKRGKSFIKLTELIDASNGQGSACKKREDTHRMAEANKAFAHYRY-MTDPIADMLTRIRNASLAEVMPSSKLKVELAKLLKSEGFIDAVEEDFKLTITLKYDAKNKPVIKLQRVSKPGLRTYSKQIPQVLGGLGVAIVSTSKGLMTDRKARKENIGGEVLCYVYM--IV--ATGRRKTSVAVVKLVKGTGRVINGKLAYIGNRPIEINRPLVLTETKEQYDVRVKAGGGIVSQAGAIRHGIARALVKTNEEFKVLREGLLTRDPRVKERKKYGRKARKRFQFSKRMAR-K-TKKEKKNVLQGVVIQSTFNNTIVTSTDGNAIAWATAGASGFKGARKGTPFAAQSAAELVAKKSIDQGMKKVEVHIKGPGSGRETAIRAIQAGLEITLIKDVTPIPHNGCRPPKKRRVMPTINQLVRERTDKTKSPALMCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTNGFEVTSYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIVRGTLDTAGVANRAQSRSKYGAKKNQMARLAGVDLPRNKRMEIALTYIFGIGPTRAKILTKISPDLRTDDLTDDILRLSAYNIEGDLRRIMNIKRLEIGYRGMRHKRNLPCRGQRTKTNARTRRGKPIAGKKK--MSINLNKQEIVKFGANEKDTGSVSVQIALMSKKISELTEHLKNKDATKRGLLMMVGKRKRLLSYLKNEGYRLIKLGIRGMPKKEKQGIVISNKMDKTVVVKVASYDPHPKYKKIIESNKNYKAHDENNVCNEGDVVRIVETRPLSSGKCWTVAEVVEQAK-------------',\n",
       " 'SM1_D11_(SM2F09_MP_sspace)': 'KQNSKRSQVCMEGLIKLSALNLRKIPSAKFESIELAFHLGIDTKQADQQIRGTAILPAGTGKIKVAVIAQGEFAQAAGADLVGDKDLLKDIESGIEFDLLATLDMMKDVGKLGRILGPRGLMPNPKDGTVTDIVAVNIKKGRINFRAEKEGVVHLVGKKSFSDDLINIKEAFLTLQKMKPSSAKGVFFKSYIKSTMSPAFELDASKII----MLGLMTKKRGMTQIFDEEGSAIPVTVLEVLENVIIQKKSNEKDGYNAIQIGSVEEEKR-LTKAEVNHCKKKELKVFRELKEFRVDFIQKYNVGDVLNEEVLQKGALLEATSRPKGKGTTGGIKRWHHHRRPMSHGAKHHRHIGSAGAGTTPGRVFKGKHMPGR-DSNDVTIPNLELVSYNSEHGFILVKGAVAGAKNSLVILKAAKKVGLNSA-RKEYEQIPNLKKFYTNDLQVPKLLKIVINRGVAEDSKTGNILEEMVKDITVSGQKPLVNKSKKSIATFKLREGMPNGISVTLRKDLMYSFLDRFISIASPRIRDFRGFKIKS-DRRGNFTIGIADQRIFVELE-NRTN--KGFNITFVTSANTDDEGRALLEEMGFPFSNMSRIGKKSIKLPKGVSAEQQDCKIIIKGPKGNLVVEMIEGFILQKEEDNLSLAIKDENQRKKLNAFHGLYGSLLQNAVVGVSEGFSKTLNLEGVGYRANLAGNVLNLAMGYSHPVQFTAPQDIKLEVPNQTTIIVSGIDKQEVGQVAANIREIRKPEPYKGKGIKYKDETVRRKAGKTGKK---VSQPLFKQIKLQIPAGQANPSPPIGPALGQAGVNIMEFCKQFNDATKTKPGLVVPVIITVTKTRTFTFILKQPPASILIKEELKLKSGSARPNQIKVGKITVEQLRKIAEQKMPDLNANDADAAAKIIAGSAKSMGLTIEGYKTFVTRQRWYVIDADKVLGRLACEVAILRGKRKAFTPHCDNGDFVIIINAEHILTGKDEDNIYYKHSGYIGHLKTETLELLRRPEQVFIAVSRMLPKATLRSLLLKKLKVYAGDHPHAQNPIIISNI--MIQETWLEVADNSGAKKLECIRVLSGEGAVGVGDIVAAVKSAPKSVKKGSVVKAVIVRVRKRRKNGVISFDENAAVLINKLEPIGTRIFGPIDRGLREFGFAKILSLAPDVVANLTKKEKRKVKIRIRKIGDEKPRLAVHKSGHIYAQIIDDTLFASSLSEFKSSGSEAAIGKLIAIAKDKKISVVFDRGGFLYHGRISELADSARENGLNFKSQDKDVSENTVIKSVKLARVTNVIRGLSVAEAMNFLYFCPYISKESVAKLLKSATANA-LNGLDLKLYISEIYTNAWLKRMRAASKGKGSPRKKAYSKMSIKLSIE-------------------MNFLLYSQKCIMNLDINVLDETAELTEEIAKRFLESGVHFGQVMKNRSPKMDEYVYGIAKNGIQIINLDEMWNCLKKAGTLLRKLSAEDKNILFVGTNKKAVSKVLEEFSESHNLNYMTNRWLGGTLTNPIT-RNRIGYLRELEGIEQSGLINSMTAKEKSFLNKKLKKLKKNLGGLKKIKGPIHAIVIIDPMYEINASLEAFKKSNNLSILALADTDCRFSPDEFNIVIPCNVSSICSLKEVMSILVAYIEEGRK--EAAVRK---SNIAKAKTTIQVQSQVSQIKQENQEENMVKGEMGQKKPARANRLGVVAE----------HESTWFSEFGNFKDHLEEDW--KIRELIKNK-YG----EKAGISRIKINRKADEIDVEIY-------SARPANIFGRDN-----------------------------------------------------------------------QKIEELK-------------------------NRLLG-------EISSKIKIYVVE--------------------------------------ANLN-DPKILAERIAYDLERRIAFRKACKNVISKAMASPEVIGIKVMVSG-----------------------------RLGGAEIARRE-----WYLEG---KVPLHTLHSNIDNGFARAITT-------------------------------------------YGIIGVQVVIH--------------------------------------------------------RHNFQKHNRKH--NRSRNKMARYVKKKKKVRKFGSVNGL-PLTVNRTSRPGMHGKREKKSEYAKGLFNKNSLRASYGISEKQFRNYFTKAAK-DKGLLQILERLDNLVYRLGIAPTLPAARQLVVHGHIQVEKEKVDKPSYLVKVGQKVRLNPKKKKLIIE-EALAKAEYIYVSFDKNVGCFARSESEIPSQVELAKIIEFSSRKINNNNNRNPENSERRA-RQER---DKDSEWKEFVVQISRVTKVVKGGKNLSFRALVVIGDERGSFGVGLGKAAEVVDAVKKAIENAKKDVVKI-LFDQGSIPHNILGHFNASELHLFKARKGTGLIAAGMVKTALELAGLHNVVCKIHGSTNPANVLFALKDAILKLKTRGQIRQLRNNIAPKPSVLIMSRKKRKILEYHNLLLSKFIRMLMWGKLSVANSVYTALLNKFKSAINFQKSLDNIRPLVKVKARRVGGSTYQVPIEVSEAGFSIAMRWIITAARNKNADIAKLFEFSEAYEKRGDAFRKREETHKMAEANRAFAHYRIMTSDPIADMLTKIRNSKKKEVIGYSKIKEGIIDAFLRLGFINEVEETRKIKVFLKYVNR-KPVIKIIRNSKPSCREYTSLIKKKKAGIAVFILSTSKGVLSKAEAQEKGIGGEKLCTIWM----YNGTGRRKRAIARVFIKPGSGKIVNNKVNYFCNNLFLIKEPLVRTNNLNKFDILVNVGGGIKGQSDAVKMGLARALLMFNLSNRPLKAGLLTRDSREVERKHYYKKARKSPQYSKRMAVK---KKAKQKFQNGLVINSTFNNTLVSITNGDVLSWSSAGACGFKGSRKGTPFAAQSASNNAAKTATSVGLKSVIIYISGPGAGRESAIRAIESGIEIQSIKDITPIPHNGCRPPKRRRVMPTLNQLVNRRKRASKSPALQNPAKAGVCTRVYTTTPKKPNSALRKVARVKLSNGIEVTAYIPGEGHNLQEHSMVLIRGGRVKDLPGVRYHIIRGSHDTTGVAKRKRSRSKYGAKKKQ-MKIAGIDIPDKKRILIALTYIFGIGLSSSKILLKINTEIKTKDLSPQVIQITHIKIEGDLRRVSNIARMSISYRGIRHREGLPVRGQRSRNKGGKRRDR-----KKAR-MLQKQKREILERQRHTLDTGSPEAQIALFSARITDIMHHLQNNDSSLRGLQQLVSKRKRQMKYLANERYKLIELGIRGMQDKDQSGKVVSTKMEKTITVLIERQIQDKKYKKIVKRRKKLLAHVPDGVCAEGDFVTIESCRPISKNKSFILKSINK----------------',\n",
       " 'Gloeobacter_kilaueensis_JS1': 'AKISKRLTLQVRQTYPQRAIMAIELATAKFETIETHIRLGINPKYADQQVRATVVLPRGTGKIRIAVLAKGEKVEAAGADISGSEELIERIQGGMDFDMIATPDIMPQVARLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHIFGKKSFTEALLNLKAVQEAIDRAKPSGAKGRYWRTHIKSTMGPAIEIDVNILRDLKMSLGILGRKLGMTQIFDEEGRAIPVTVVEAGPCPVTQLKSEATDGYTAVQVGFGTVREKVLTKPEVGHCKKAGLQPVRHLREFAVEDTSSYSLGQLIGVDIFEAGQLVDVVGTSIGKGFAGGQKRHHFGRGPMAHGSKNHRAPGSIGAGTTPGRVFPGKKMPGRMGNERVTVRRLTVVRVIAERNLLLIKGGLPGVEGGLLIISPAKRVG-R--ARLKYREVPRLEQFYSNLMQVPKISKIVVNRGLGEAAQNAKALDSSLAEIAITGQKPVITRAKKAIAGFKLRAGVPVGMTVTLRSQRMYEFLDRLVSVALPRIRDFRGVSPKAFDGRGNYTLGLREQLIFPEIEYDSIDKIRGMDITICTTAQSDEEGRALLAALGMPFRQMSRIGKQPIAIPPRVEVNVDGQHVTVKGPKGTLERTLPETLAVVRENGQILVNRRGDSRRAR--EQHGLGRTLVANMVTGVTTGFTKPMQIAGVGYRVALTGRKLTISAGFSHPVEVELPPGIDVEVDQKNNFVIKGIDKQAVGDLAASIRAIRPPEPYKGKGIRYAAEKVSLKAGKSGKK---MAKKVTAKIKLALPAGKANPAPPVGPALGQHGVNIMMFCKEYNARTADQVGTVIPVEITVFEDRSFTFVLKTPPASVLLTKAAGVEKGSGKPNQLKAGSITNAQLRQIAEQKLPDLNANDVEAAMRIIAGTARNMGIAIADTKTLPAIERWLVVDAGQHLGRLASVVALLRGKHRPFTPHLDTGDFVIVINAEKVVTGRKSQKLYRRNSGRPGGMKVETFHLQRLPERIVEAVWGMLPHTRLGRQQYTKLKVYKGAHPHAQKPAFLSTVLGMIQQTRLSVADNTGARELLCIRVLGGGNRYAVGDIVAVVKEAPNMVKKSEVVRAVVVRTRIRRENGTIRFDDNAAVIIDNGNPRGTRVFGPVARELRERNFTKIISLAPEVLMKVDRKSAQRRHQRIRKLGTEQPRLAVYRSNHIYAQVIDDTLAASTLEVLRTETTEAAVGRLVARAKEKGISVIFDRGGKLYHGRVKAVAEAAREAGLEF-MAD--VKAARFIRSPKVRRILDQIRGRSYQDALIILEFMPHAATEPIKKVLQSAVANAEHNSLDRRLVVSTAFADGPLKRFRAGDRGRARPVRKRTSHITVAVRSE--------------------------------------MSVVTLPQ-----MLEAGVHFGHQTRRWNPKMRRYIF-TDRNGIHIIDLTQTAHLLDEAYSYLREAADQGKKILFVGTKRQAAPVIAQEAKRC-GMFWVNQRWLGGMLTNWETIRTSVDQLKELEAMEANGQFDLLPKKEASVQRKKLERLTKYLGGLKNMRRKPDILLVVDQRREQNAVMEARRL--NIPIVSLLDTNC--DPDLTDVGIPANDDAIRSIRLIVGKLADAIYEGRH--GQL------EDYTEEEATPPAAEA--AGPGEVLYSTLDEDEMGQKVHPVGLRLGIIRD----------HRSRWFAKAGEYPALLSEDH--KIRKYIEKK-L-----ASSGIAEIDIERKADQVEITIR-------TARPGVVVGRGG-----------------------------------------------------------------------SGIDELR----------------------ASLEKLLG---------RRQIRINVAD-------------------------------------VQRVDAEAGLLAEYIAGQLERRVAFRRAVRQAIQRAQRA-GVQGIKVAVSG-----------------------------RLNGAEIARSE-----WTREG---RIPLHTLRADIDYAYRIAQTT-------------------------------------------YGTLGVKVWIF-----------------RGEILPGSTT-----AAPE----------------EQAPKPQRQRRKKIQYEERTEE-MARYTGPVCKLRAVGYLKGISPKCERRNTRPGVHGQRQKLSEYAIRLKEKQKARWTYGILEKQFRRTFSDAGRGNTGFLELLERLDNVIYRLGFATSRPQARQWVNHGHFAVNGRRVSIPSYRVRPGDQIAVLP-SAAFVKGLEVASLPHPPWLEVNRTIATVKAEREDIDTPVQELLIVEFYSR--MPPQQPRGRRGRGQGRAERV---QAESEFQERVVQIRRVTKVVKGGKKLSFRAVIVVGDGNGRVGVGVGKANDVIGAVKKGVSDARKALIRVPLNKINSIPHPMSGTSGAANVFLKPAAGGTGVIAGGAVRTVLELAGIKNILAKSLGSKSPLNNARAAADALSRLRTLSEVAQERD--VPVSNLWSMSRRLRPVADYASRLVTMMTNKLMQGKKATARILYASLIERTGEPLDFNQAVLNCTPRIEVKARRVGGATYQVPMEVRERGTSLALRWLVQFARKRGKSMVKLSELLDASNDTGSAVRKREETHRMAEANKAFAHYRY-MNDTLADMLTRIRNAKHDVVVPSTRMTRSVAEVLKAEGFIDQDGEGDRLLLTLKYAGRQRPIIRLKRVSRPGLRVYANKLPRVLGGIGIAIISTSQGVMTDRDARRVGVGGEVLCYVSM--AVYWGTGRRKSAIARVRLVPGTGELINGRGDYLHFRQLVARAPLETLGLENQYNIVVNAGGGVAGQAGAIRLGIARALCELSPDNRPLKEGYLSRDPRAKERRKYGLKARKAPQFSKRMAKRGGQKKVRRNVPSGVAIQSTFNNTIVTISDGDVISWASAGSSGFKGAKKGTPFAAQQAAESAGRRAIDSGMRQCEVMVSGPGAGRETAIRALQVGLEITLIRDVTPIPHNGCRPPKRRRVMPTIAQLIRERQRKTKSPALRCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGVGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKNRMQSRSKYGAKRPKMARIAGVDIPREKRVEVALTYIFGIGPARALILTGISPDLRVRDLSDQVLRIEKYQVEGDLRREINIKRLDIGLRGRRHRLGLPVRGQRTRTNARTRRGRTVAGKKKASMPLTQRKQELIGYQVHETDTGSPEIQVAMLSDRINQLTEHLRHKDSSRRGLLKLIGRRKQLLAYLAEERYRLVRLGLRRMPRKEKVGLVVSDAMQKTVVVAVENRVPHPKYKKIVVRTRKFKAHDEENRCKTGDRVRILESPPISKTKRWVVTDILNEARNP-----------',\n",
       " 'Gloeobacter_violaceus_PCC_7421_(re-annotation)': 'VQLSKRLKLQVRQTYPQKALLLKQTANAKFETAETHIRLGINPKYADQQVRSTVVLPRGTGKIRIAVLAKGEKVEAAGADIAGSEELIERIQGGMEFDMIATPDIMPQVARLGKLLGPRGLMPSPKGGTVTDLVAIEFKAGKLEFRADRTGIVHIFGKVSFTEALMNLKSVQDAIDRAKPSGAKGRYWRTHIKSTMGPSIEIDINALRDLKMSLGILGRKLGMTQIFDEEGRAIPVTVVEAGPCPVTQVKSEATDGYTAVQLGFGTAREKVLTRPEVGHCKKAGLAPVRHLREFRLPDSSQYTPGQQITVDLFAAGQLVDVVGTSIGKGFAGGQKRHHFGRGPMAHGSKNHRAPGSIGAGTTPGRVFPGKRMPGRMGNERVTVRKLTVVRVIPERNVILIQGGLPGVEGGLLMISPAKSVG-RAKTRMKYREVPRLEQFYTNLMQVPKISKIVINRGLGEASQNAKALDGSIAEVAIAGQKPVVTRAKKAIAGFKLRAGVPVGVTVTLRARRMYEFLDRLVSVALPRIRDFRGVSPKAFDGRGNYTLGLREQLIFPEIEYDSVDKIRGMDITICTTAKTDEEGRALLAALGMPFR-MSRIGKLPIAIPPKVEVTLDGRRVVVKGPKGTLDLTLPDSVEVVREDGRLLVTRRGESRRAR--EQHGLGRTLVANMVTGVTTGFTKPMQIAGVGYRVALTGRKLTINAGFSHPIEIELPAGIDIEVDPKNNFVIKGFDKQAVGDLAAKIRDIRPPEPYKGKGIRYTAEKILLKAGKSGKK---MAKKVSAKIKLALPAGKANPAPPVGPALGQHGVNIMMFCKEYNARTADQVGMVIPVEITVFEDRSFTFVLKTPPASVLLTKAAGVEKGSGKPNKEKAGTITGAQLRQIAEQKLPDLNANDVEAAMKIIAGTARNMGIVIADTKTLPEIERWLVVDAGQHLGRLASVVALIRGKHRPFTPHLDTGDFVIVVNAEKVVTGKKSQKLYRRSSGRPGGMKVETFHLQRLPERIVEAIWGMLPHTRLGRQQFTKLKVYKGPHPHAQKPEFLSTVLGMIQQSRLSVADNTGARELLCIRVLGGGNRYAVGDVVAVVKDAPNMVKKSEVVRAVVVRTRIRRDNGVIRFDDNAAVIIDNGNPRGTRVFGPVARELREKNFTKIISLAPEVLMKVDRKTAHRRHQRIRKIGTEQPRLAVYRSNHIYAQVIDDTLAASTLEPLRSETTEAAVGRLVARAKEKGIAVVFDRGGKLYHGRVKAVADAAREAGLDF-MAD--VKAARFIRSPKVRRILDQIRGRSYQDALIILQFMPHAATEPIKKVLESAAANAEHNSLDRRLVVSTAFADGPLKRFRAGDRGRARPVRKRTSHITVAVRSE--------------------------------------MSVVSLPQ-----MLEAGVHFGHQTRRWNPKMRRYIF-TDRNGIHIIDLTQTAHLLDEAYSYLREASDQGKKILFVGTKRQAAPVIAQEAKRC-GMFWVNQRWLGGMLTNWDTIRTSVDQLKELETMEANGTLDLLPKKEASVTRKKLERLTKYLGGLKNMRRKPDILLVVDQRREQNAVMEARRL--NIPIVSLLDTNC--DPDLTDVGIPANDDAIRSIRLIVGKLADAIYEGRH--GQLEDFSEEEPVPAAAVAVESEDK--GEVFDDEEE------MGQKVHPIGLRLGIIRD----------HRSRWFAKAGEYPALLAEDH--KIRAYIVKK-L-----ASGGIADVDIERKADQVEITIR-------TARPGVVVGRGG-----------------------------------------------------------------------AGIDELK----------------------ANLEKLI---------GRRQIRINVVE-------------------------------------VQRVDAEAALIAEYIVGQLERRVAFRRAVRQAIQRAQRA-GVQGIKVMVAG-----------------------------RLNGAEIARSE-----WTREG---RIPLHTLRADIDYAYKIAQTT-------------------------------------------YGTLGVKVWIF-----------------RGEILPGAGG---PAAAPE--------------EVAAQPKPQRRRRKKIQYEERTDEQMARYTGPVCKLRAVGYLKGISPKCERRNFRPGVHGQRQKLSEYAIRLKEKQKARWSYGVLEKQFRRTFSDASRGNTGFLELLERLDNVVYRLGFATSRAQARQWVNHGHFAVNGRRVSIPSYRVRPGDAITVMPNA--FVKALEAASLPQAPWLEANRTLATVKAEREDIDTPVQELLIVEFYSR--GGPGPEQAPEDPDREAERAQ----AETEFQERVVQIRRVTKVVKGGKKLSFRAVIVVGDGNGRVGVGVGKANDVIGAVKKGVSDARKALIRVPLNKINSIPHPMSGSSGAANVFLKPASGGTGVIAGGAVRTVLELAGIKNVLAKSLGSKSPLNNARAAADALSRLRTLSEVAGERG--VPVSNLWSMSRRLRPVADYASRLVTMMTNKLMKGKKATARILYSALVERTGEPLDFNQAVLNCTPRIEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRKRGKSMVKLSELMDAANDTGSAVRKREETHRMAEANKAFAHYRYMANDTLADMLTRIRNAKHDIVVPSTRMTRSVAEVLKMEGLIEQDGEGDRLLLTLKYAGRQRPLINLKRVSRPGLRVYANKLPRVLGGIGIAIISTSQGVMTDRDARRAGVGGEVLCYVFVQAAVYWGTGRRKCAIARVRLVPGSGELINGRGDYLHFRQLMAKAPLETLGLENQYNIIVNAGGGVAGQAGAIRLGIARALCELSPDNRPLKEGYLSRDPRAKERRKYGLKARKAPQFSKRMAKRGGQKKVRRNVPSGVAIQSTFNNTIVTIADGDVISWASAGSSGFKGAKKGTPFAAQQAAESAGRRAIDSGMRQCEVMVSGPGAGRETAIRALQVGLEITLIRDVTPIPHNGCRPPKRRRVMPTIAQLIRERQRKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGVGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKNRMQSRSKYGTKRPKMARIAGVDIPREKRVEIALTYIFGIGPSRATILTGINPNLRVRELTDQVLRIEKYQVEGDLRREINIKRLDIGLRGRRHRLGLPVRGQRTRTNARTRRGRTVAGKKKAAMPLTQRKQELIGYQVHETDTGSPEVQVAMLSDRINQLTEHLRHKDSSRRGLLKLIGRRKQLLAYLAEDRYRLVRLGLRRMPRKEKIGLVVSDAMQKTVVVAVENRVPHPKYKKIVVRTRKFKAHDEENRCKVGDRVRILESPPLSKTKRWVVLDILDEARNP-----------',\n",
       " 'Synechococcus_sp._RCC307_(re-annotation)': 'KSISKRFKATVEPRYPLEAVLVKANATAKFETIEAHVRLGIDPKYTDQQLRTTVALPKGTGRIRIAVITRGEKVEAAGAELVGDEELVDSIAGGMDFDLIATPDMMPKVAKLGRVLGPRGLMPNPKTGTVTDVTAIEFKAGKLEFRADRSGIVHVFGKASFSDDLLNLKSVQETVDRNKPSGAKGRYWKSYITSTMGPSVEVDFSALQDLSMSIGILGKKLGMSQFFDDEGRAVPVTVIEAGPCRITQVKTPTNDGYSAVQVGFGDVREKLVNQPAKGHLKKSGEDLLRHLKEYRVDSTDGVELGSSVTVDAFEPGQKVDVSGDTMGRGFSGYQKRHGFSRGPMTHGSKNHREPGSTGAGTTPGRIYPGKRMAGRYGGTKTTVKGLVVLKVDSERNLLVVKGSVPGKPGALLSIRPAVRVGTA---SLKYEAIPKLKDLFSNLHQVPKVVKVTVNRGLGEAAQNAKSLEASVEEIAITGQKPVVTRAKKAIAGFKIRQGMPIGVAVTLRGDRMYAFLERLINLALPRIRDFRGVSEKSFDGRGNYTLGVREQIIFPEISFDKIDAIRGMDITIVTTARNDEEGQALLREMGMPFRNMSRIGKNPVPIPEKVSVEISGLTVKVKGPKGELERVLPDGVSVSQADNAVTVSPSDTSRRSR--ERHGLCRTLVANMVEGVSKGFSKKLEIIGVGYRAQVKGKKLVVSAGYSHPVEMDAPEGVTFAVENNTLVTVSGADKELVGNEAAKVRGIRPPEPYKGKGIKYQGERILRKAGKTGKK---MAKKVTAVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKAGFVIPVEISVFEDRSFTFITKTPPASVLIVKAAGIEKGSGESAKGSVGSITRAQLEEIANTKLPDLNCSSVESAMRIIEGTARNMGVAVKDMKTVPTVERWYVVDANQCLGRLATEVAVLRGKNKPYTPHLDTGDFVVVINADKVVTGNKSDKLYRRHSGRPGGMKVETFALQRIPERIVEAIKGMLPHNALGRQLFRKLKVYKGAHPHAQQPQYLNPSASMIQETFLNVADNSGAKRIQCIRVL-GTNRYAVGDIVATVKDAPNMVKKSDIVKAVVVRTKMRRETGAIRFDDNAAVIINDKNPRGTRVFGPVARELRERSFTKIVSLAPEVIMSTPRKEQQKRHRRLRHLGTERPRLAVYRSNHIYAQVIDDTLAASSLDDLRTLGNDASVGQLVARAIAKGIQVVFDRGGNLYHGRVKALAEAAREAGLQFMATTASALAGRFVRSVKVRRVLDQIRGRSYRDALIMLEFMPYRSTGPITKVLRSAVANAEHNGLDPSLVVSMASADGPMKRYRPRAQGRAYQIKKQTCHISIGVTPA--------------------------------------MAVVTLSE-----MMEAGAHFGHQTRRWNPKMQRYIH-CARNGVHIIDLVQTAVCMNNAYKWVRSAARSGKRFLFVGTKKQAAEVVALEASRC-GASYVNQRWLGGMLTNWSTMRARIERLKDLERMESSGAIAMRPKKEASVLRRELDRLQKYLGGLKNMRRLPDVVVLIDQKREYNAVLECQKL--DIPVVSMLDTNC--DPDLCDVPIPCNDDAVRSVQLIIGRLADAINEGRH--GSNDQG---------------DEG-----------------MGHKINPTGLRLGITQE----------HRSRWYAPARTYPTLLQEDE--RIRKFVNKK-Y-----ASAGISSVLIARKADQLEVELK-------TARPGVLVGRQG-----------------------------------------------------------------------SGIEELR----------------------AGIQKTLG-------DKSRQVRINVVE-------------------------------------VERVDADAFLLAEYIAQQLEKRVAFRRVIRMAVQRAQRA-GVLGLKIQVGG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYATKVASTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLPGQKQ-----QPPV----------------GAAPRRR-ASRRPQEFEDRSNEEMSRYRGARLRVRRLGDLPGLTRKAAKRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSERQLVRYVKKARAGSTGLLKLLERLDNVCFRLGFGPTIPGSRQLVNHGHVTVNGRVVDIASYQCKAGDVIAIREKSKKLAENLEFPGLANIPHLELEKKLAKVVGEREWVALEINELLVVEYYSRKVDVPAAAEGQGQGQGE-RRGRGRDERDSEWQERVIQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVKKGVADGKKHLVKVPLTRSNSIPTISTGRDGAASVLMRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALRDLRTHQDTAKERG--ITLEQIYSMSRRKRPIPDFNSRLASMMVARLMKGKKSTARILSDAFIERTGDPLEFETAVRNSTPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVNFSRSRGRSMAKLAELMDAANEAGSAVRKREETHKMAEANKAFAHYRYMATDPISDMLTRIRNARHQTTIPLSRMNRSIAKVLEQEGFIQSEGEGQKLVLSLKYSGKQKPTISVQRVSKPGLRIYTNRLPKVLGGLGVAIISTSKGVMSDRDARKEGVGGEVLCYVYM-RVVYWGTGRRKTSVARVRVVPGDGTIINGRGDYLNYNPYLVRAPLDTLGLSKEYDILVNVGGGLTGQADAIKQGAARALCELSPDNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAKKK-GKKAKRNVPNGVAIQSTFNNTIVSITDGEVIAWSSAGASGFKGARKGTPFAAQTAAEAAARRALEQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRAKRRRVMPTIQQLIRERSRKTKSPALRCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKAPKVARISGVDIPREKRVEIALTYIYGVGLTRSQILTGVSPDTRVKDLSDDVLRAETFTLEGDLRREMAMKRLDIGLRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKK--MPLNTKKQELINHQTHATDTGSVEVQVAMLSERITQLTGHLQNHDSSRQGLMKMLGRRKSLLGYLKSERYDLIKLGIRGMAVKERVGTVVSDKMEKTVVVAVENRFPHPIYKKTVSRTTRYKVHDEDNRCQVGDRVRITETRPLSRSKRWAVAEIMTTKSGS-----------',\n",
       " 'Prochlorococcus_marinus_marinus_CCMP1375_(re-annotation)': 'TKISKRMKLKVEDKYPLEAILVKENANAKFETIEAHIRLGIDPKYTDQQIRTTVSLPKGTGQVRIAVIAKGEKVEAAGADLAGEEELIDSISKGMNFDLISTPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLISIEFKAGKLEFRADRAGIVHVFGKASFSEDLLNLKVLHEAIDRNKPSGAKGRYWKSYITSTMGPSVEIDIAALQDSKMSIGILGKKLGMSQFFDEKGRSIPVTLVEAGPCRITQLKSSATDGYSAVQIAFGAVREKLINKPFQGHLAKSGDDLLRYMSEYRVKDVGEFQLGAQITVGDFEEGQKVDVSGTSMGRGFSGYQKRHGFSRGPMSHGSKNHREPGSTGAGTTPGRIYPGKRMAGRYGGKKITTRGLMIMKVDSDKNLLVIKGSLPGKPGSLLNIRPAKCVGSKGG-SLKYETIPKLKDLFSNLHQVPKVVKINVNRGLGEAAQNSKALEASLSEVSITGQKALVTRAKKAIAGFKIRQGMPIGCAVTLRGERMYAFLERLINLALPRIRDFRGVSPKSFDGRGNFTLGVKEQLIFPEISFDKIDAIRGMDITIVTSARTDEEGRALLKEMGMPFRNMSRIGKKPIPVPEKVAVTLDGLLVTVKGPKGELTRTLPEGVTIDQTDGLIIVSADSEKRKSR--ERHGLSRTLVANMIEGVNNGYSKQLEIVGVGSRAQVKGKTLVVSAGYSHPVEVIPPEGITFKVENNTNVLVSGIDKELVGNEAAKIRAIRPPEPYKGKGIKYLGERILRKAGKSGKK---MAKKITAVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKAGFVIPVEISVFEDRSFTFITKTPPASVLITKAAGIAKGSGDSAKGQAGSINRAQLEEIAKTKLPDLNCNNIESAMKVIAGTARNMGVSVSDMKTTPSINRWYLVDANQTLGRLATEVAVLRGKNKPFTPHLDTGDFVIIVNAEKIVTGKKMQKLYRRHSGRPGGMKVETFSLQRIPERIVEAIKGMLPHNALGRQLFRKLKVYKGSHPHAQNPQLITTNELMIQETYLTVADNSGAKRLQCIRVL-GSNRYAVGDIVAAVKDAPNMVKKSDVVKAVVVRTKLRRETGSIRFDDNAAVLINDKNPRGTRVFGPVARELRERNYTKIVSLAPEVISTLSRKQKQKRHKRLRNLGTQRPRLAVFRSNHIYAQVIDDTLAASTLEEFLSSATAASVGEMLARALGKGIQVVFDRGGSLYHGRVKALAEAARQAGLTFMS-DLPAQAGRFIRSAKVRRVLDQIRGRTYRDALIMLEFMPYRSTGPITKVLRSAVANAENNGMDPSLIIKTASADGPMKRYRPRAQGRAFAIKKQTCHISIAVAPQ--------------------------------------MAVVTLSE-----MMEAGAHFGHQTRRWNPKMSRYIY-CARNGVHIIDLVKTAVCMNNAYKWTRNAAKSGKRFLFVGTKKQASEVVAQEAARC-GASYVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEASVLRHELERLQKYLGGLKGMKRLPDVVVLVDQRRETNAVLEARKL--DIPLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLVLGRIADAINEGRH--GSNDQR--AQKYS----------------------------MGHKIHPNGLRLGITQE----------HRSRWYASSKTYPLLLQEDD--RIRVFIQKK-Y-----GAAGISDVLIARKADQLEVELK-------TARPGVIVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTIG-------DRSRQVRINVVE-------------------------------------IERVDADAHLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVGG-----------------------------RLNGAEIARSE-----WTREG---RVPLHTLRAEIDYANKTANTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLSKEEQ-----PLPV----------------GASPRRK-GNRRPQQFEDRSNDGMSRYRGPRLRIRRLGDLPGLTRKAAKRSNPPGQHGNRRKRSEYAIRLEEKQKLRFNYGISERQLVRYVKKARAGSTGLLKLLERLDNVCFRLGFGPTIPGSRQLVNHGHVTVNGKTLDIASYQCKSGDTIAIRERSKKLAENLEFPGLANVPHLELEKKMAKVTGDREWVAIEINELLVVEYYSRKV-VPAAAEGQ-QQEQK-RRNQ---DRDSEWQERVIQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVADGKKNLVRVPLTPNSSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSLLRTHKATAKERG--ISLEQIYSMSRRKRPVPDFNNRLATMMVARLMKGKKSTAKILSQAFIERTGDPIEFETAIKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVNFSRSRGRSMAKLAELMDAANEAGNAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHENTIPASRMSRSIAKVLQNEGFIQSEGEGKTLVLELKYSGKHRPTISMKRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARKQGVGGEVLCYVYMKAVVYWGTGRRKTSVARVRLIPGTGKIINGRGDYLNFNPYLVKAPLHTLGLGEAYDVLVNVGGGLTGQSDAIKQGAARALCELSVDNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMATKKTGKKSKRNVPNGVVIQSTFNNTIVSITDGEVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALDQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRAKRRRVMPTIQQLIRERTRKTKSPALRCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLLRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQARSKYGAKAPKVARIAGIDLPRDKRVEIALTYIYGIGLTRAQILTGVNPDIRVKDLEDDVLRTESFTIEGDLRREMALKRLDIGLRGRRHRMSLPVRGQRTRTNARTRRGRTVAGRKK--MTLNTAKQKIINHQTHGTDTGSVEVQVAMLSERINQLSKHLQNHDSSRQGLLKMIGQRKRLLNYVKSSRYSLVKLGIRGMALKERLGTVVSDKMDKTVVVAVVNRFPHSIYQKTVSRTTRYKAHDEENNCKVGDRVRITETRPLSATKRWSVAEVLRTTKQE-----------',\n",
       " 'Cyanobium_gracile_PCC_6307': 'TKVSKRFTLKVEDRYPLEALLVKDNATAKFETIEAHVRLGIDPKYTDQQLRTTVALPHGTGQIRIAVIARGEKVEAAGADLAGDDELVDQIAGGIDFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLGAIEFKAGKLEFRADRAGIVHVFGKASFDAKLLNLKALQETIDRNKPSGAKGRYWRSYLASTMGPSVQVDFAALQDIKMSIGILGKKLGMSQFFDEEGKSIPVTLIEAGPCRITQLKSEATDGYTAVQLGFGETRDKLVNKPAKGHLAKSGETPLRHLKEYRLEAIDGLELGAAVTVGDFAPGQKVDVSGDTIGRGFSGYQKRHGFSRGPMSHGSKNHREPGSTGAGTTPGRVYPGKRMAGRYGGKQITTRGLTILKVDAERNLLVVKGSVPGKPGALLDIRPARRVGAN---SLKYETIPKLKDLLSNVHEVPKVVKVTVNRGLGEAAQNAKALEASITELAITGQKVVVTRAKKAIAGFKIRQGMPIGVAVTLRGERMYAFLERLIHLALPRIRDFRGVSPKSFDGRGNYTLGIREQIIFPEISFDKIDAIRGMDVTIVTNARNDEEGRALLREMGMPFRTMSRIGKAPIPIPDKVTISLDGLAVTVKGPKGELSRVLPEGVSVSQEGGTVVVSPTSSHRRSR--ERHGLSRTLVANMVEGVSQGFTRKLEIVGVGYRAQVQGRKLVVSAGYSHPVEMVPPEGVTFSVENNTSVFVSGPDKELVGNEAAKIRGIRPPEPYKGKGIKYEGERILRKAGKTGKK---MAKKVVAVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQEKAGYVIPVEISVFEDRSFTFITKTPPASVLISKAAGIDKGAATSAKGAVGAISRSQLEEIAKTKLPDLNCTSVESAMRIIEGTARNMGVAVKDMKTPPSLERWYLVDADQTLGRLASEVAVLRGKNKPFTPHLDTGDFVVVINADKVVSGNKTQKIYRRHSGRPGGMKTETFHLQRLPERIIEAIKGMLPHNALGRQLFRKLKVYKGAHPHAQQPQLLDPAAAMIQETFLNVADNSGAKRIQCIRVL-GTNRYAVGDIVAAVKDAPNMVKKSDVVKAVVVRTRLRRDTGAIRFDDNAAVILGDNNPKGTRVFGPVARELRERNFTKIVSLAPEVISSLSRKQQQKRHRRLRLLGTDRPRLAVFRSNHIYAQVIDDTLAASTLDDLRTVGNDASVGQLVARALAKGIQVVFDRGGNLYHGRVKALADAAREAGLQF-MATTTALAGRFIRSVKVRRVLDQIRGRTYRDALIMLEFMPYRSTGPITKVLRSAVANAEHNGLDPALVISQASADGPLKRFRPRAQGRAYAIKKQTCHISIAVAAT--------------------------------------MAVVTLSE-----MMEAGAHFGHQTRRWNPKMSRYIY-CARNGVHIIDLVQTAVCMNNAYKWTRSAARSGKRFLFVGTKKQASEVIAQEAARC-GASYVNQRWLGGMLTNWSTMRARIDRLKDLERMESSGAIAMRPKKEAAVLRRELDRLQKYLGGLKNMRRLPDVVVLVDQRRETNAVLECRKL--DIPLVSMLDTNC--DPDLCDVPIPCNDDAVRSVQLVLGRLADAINEGRH--GAQDQR-------GGD-----DDA-----------------MGHKIHPTGLRLGITQD----------HRSRWYAPSKTYPVLLQEDD--RIRSFINKK-Y-----AAAGISDVLIARKADQLEVELK-------TARPGVLVGRQG-----------------------------------------------------------------------SGIEELR----------------------TGIQKTLG-------DAHRQVRINVVE-------------------------------------VERVDADAFLLAEYIAQQLEKRVAFRRVMRMAVQRAQRA-GVLGLKLQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYATKVATTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLPGQQD-----QLPV----------------GGAPKRR-ANRRPQQFEDRSNEEMSRYRGPRLRIRRLGDLPGLTRKAAKRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGISERQLVRYVKKARAGSTGLLKLLERLDNVCFRLGLGPTVPGARQLVNHGHVTVNGRVVDIPSYQCKAGDVVAIRERSKALAENLAFPGLANIPHLEFDKKMAKVGGEREWVALEINELLVVEFYSRKVDVPAAAEGQQEQRRGNRRGQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNERGQVGVGVGKAGDVIGAVRKGVADGKKHLVKVPLTRTSSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMQALEGLRTHKETAKERG--ISLEQIYSMSRRKRPVPDFNSRLASMIVARLMKGKKSTARILSDAFIERTGDPLDFETAVRNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVNFSRARGRSMAKLAELMDAANEAGSAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRLRNARHERTIPASRLSRSIANVLHQEGFIAAEGEGQRLVLELKYSGKHRPTISVQRVSKPGLRIYKNRLPKVLGGLGVAIISTSRGVMSDRDARKQGVGGEVLCYVYM-RVVYWGTGRRKTAVARVRVVPGSGSVINGRGDYLNYNPYLVKAPLQTLGLEAQYDLLVNVGGGLTGQADAIKQGAARALCDLSPDNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAKKKTGKKAKRNVPNGVAIQSTFNNTIVSITDGEVVSWSSAGASGFKGARKGTPFAAQTAAEAAARRALEQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRAKRRRVMPTIQQLIRERTRKTKSPALRCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDTSGVKDRRQSRSKYGAKTPKMARIAGVDIPRDKRVEISLTYVYGIGLTRARILSGVNPDIRVKDLSDDVLRADSFVLEGDLRREMALKRLDIGLRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKK--MPLTTQKQELINHQAHGTDTGSVEVQVAMLSERIQQLSGHLQNHDSSRQGLLKMIGRRKRLLGYLRSTRYALIKLGIRGMALKERVGTVVSDKMDKTVVVAVENRFPHPIYQKTVRRTTRYKAHDEANSCRVGDRVRITETRPLSRTKRWAVAEVLSTTSAG-----------',\n",
       " 'Prochlorococcus_sp._MIT9211_(re-annotation)': 'TKISKRMALKIEDRYPLEAILVKESSTAKFETIEAHVRLGIDPKYTDQQIRTTVTLPNGTGQIRIAVIARGEKVEAAGADLAGEEELVDSISKGMGFDLIATPDMMPKVAKLGRVLGPRGLMPNPKTGTVTDLVAIEFKAGKLEFRADKAGIIHVFGKASFNDALLNLKTLQETIDRNKPSGAKGRFWKTYITSTMGPSIEVDIAALQDISMSIGILGKKLGMSQFFDDKGRSVPVTLIEAGPCRITQLKSLATDGYSAVQIGYGFTREKLVNKPFKGHLSKSGEGFLRHLHEYRVDDLGVYELGAKITVGNFEAGQKVDVSGNSMGRGFAGYQKRHSFSRGPMSHGSKNHRQPGSTGAGTTPGRIYPGKRMAGRYGGKKITTRGLTIMKIDSDRNLLVVKGSVPGKPGSLLNIRPAQRVGLTGG-SLKYETIPKLKDLLSNIHQVPKVVKVNVNRGLGEAAQNSKTLEASLSEMAITGQKALVTRAKKAIAGFKIREGMPIGCTVTLRGERMYAFLERLINLALPRIRDFRGVSPKSFDGRGNYTLGVKEQLIFPEISFDKIDTIRGMDITIVTSARSDEEGRALLKALGFPFRTMSRIGKQPIPVPEKVAVELDGLSLTVKGPKGELSRTLPEGVSISQVDNSIVVSAINSKRKSR--ERHGLSRSLVANMVEGVSKGYSKKLEIVGVGSRAQVKGKNLIVSAGYSHPVEVIPPDGITFVVENNTNVTVSGIDKELVGNEAAKIRAIRPPEPYKGKGIKYAGERIIRKAGKSGKK---MAKKIVAVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTQDKAGFVIPVEISVFEDRSFTFITKTPPASVLITKAAGIEKGSGDSAKGQVGTINRAQLEEIAKTKLPDLNCSNIESAMRVIEGTARNMGVSVKDMKTTPAIDRWYLVDANQTLGRLASKIAVLRGKNKPFTPHLDTGDFVVVVNAEKIISGKKQQKLYRRHSGRPGGMKVETFSLQRIPERIVEAVKGMLPHNSLGRQLFRKLKVYKGAHPHAQEPKLLDTNGLMIQETFLTVADNSGAKKLQCIRVL-GSNRYAVGDIVAAVKDAPNMVKKSDVVKAVVVRTKLRRETGSIRFDDNAAVLINDKNPRGTRVFGPVARELRERNFTKIVSLAPEVIATLSKKQQQKRHKRLRHLGTHRPRLAVFRSNHIYAQVIDDTISASTLDDLRELGSDASVGALLARALAKGIQVVFDRGGNLYHGRVKALAKSAREAGLKFMVESSSAQAGRYIRSAKVRRVLDQIRGRTYRDALIMLEFMPYRSTGPITKVLRSAVANAENNGLDPALVITRATADAPMKRYRPRAQGRAFAIKKQTCHISISVAPE--------------------------------------MAVVTLSE-----MMEAGAHFGHQTRRWNPKMSRYIY-CARNGVHIIDLVKTAVCMNSAYKWTRNAAKSGKRFLFVGTKKQASEVVALEANRC-GASYVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEASVLRHELERLQKYLGGLKGMKRLPDVVVLVDQRRETNAVLEARKL--DIPLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLILGRLADAINEGRH--GSNEHR-------GGQ----------------------RLKMGHKIHPNGLRLGITQE----------HRSRWYASSKTYPTLLQEDD--RIRGFIQKK-Y-----ASAGISDVLIARKADQLEVELK-------TARPGVIVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTIG-------DRSRQVRINVVE-------------------------------------IERVDADAHLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVGG-----------------------------RLNGAEIARSE-----WTREG---RVPLHTLRAEIDYANKTANTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLSKEDQ-----PLPV----------------GASPRRK-GSRRPQQFEDRSNDGMSRYRGPRLRIRRLGDLPGLTRKAAKRSNPPGQHGQRRKRSEYAIRLEEKQKLRFNYGISERQLVRYVKKARSGSTGLLKLLERLDNVCFRLGFGPTIPGSRQLVNHGHVTVNGKVLDIASYQCKPGDVISIREKSKKLAENLEFPGLANVPHLEFEKKMAKITGEREWVAIEINELLVVEYYSRKVAVPAAAEGQQQQQRK-GRNQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVADGKKNLVKVPLTRNSSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALAELRTHKGTAKERG--ISLEQIYSMSRRKRPVPDFNNRLATMMVARLMKGKKSTARILSDAFIERTGDPIEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVNFSRSRGRSMAKLAELMDAANEAGNAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHETTIPASRMSRSIAKVLQKEGFIQSEGEGKTLVLELKYSGKHRPTISMQRVSKPGLRIYKNRLPKILGGLGVAIISTSKGVMSDRDARKQGVGGEVLCYVYMRTVVYWGTGRRKTSVARVRLIPGTGKIINGRGDYLNFNPYLVKAPLQTLGLSDSYDVLVNVGGGLTGQADAIKQGAARALCELSADNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMATKKTGKKSKRNVPNGVVIQSTFNNTIVSITDGEVVSWSSAGASGFKGARKGTPFAAQTAAEAAARRALEQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTIQQLIRERTRKTKSPALRCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLLRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKVPKVARIAGIDIPREKRVEVALTYIYGVGLTRAKILAGVNPDIRVKDLDDDVLRVESFTLEGDLRREMALKRLDIGLRGRRHRMSLPVRGQRTRTNARTRRGRTVAGRKK--MTLNTEKQKLINHQNHGTDTGSAEVQVAMLSERISKLSNHLQNHDSSRQGLLKMIGQRKRLLNYLRSKRYTIIKLKLRGMALKERLGTVVSDKMDKTVVVAVENRFPHPIYQKIVSRTARYKVHDAENTCKVGDKVRITETRPLSASKRWTVAEVLRSTHEI-----------',\n",
       " 'Prochlorococcus_marinus_str._MIT_9303': 'PKLSKRITLKVEDRYPIEAILVKENATAKFETIEAHVRLGIDPKYTDQQLRTTVALPQGTGQVRIAVITRGEKLEAAGAELAGDDDLVESIGKGMDFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLAAIEFKAGKLEFRADRAGIVHVFGKASFSDALLNLKTLQETIDRNKPSGAKGRYWKSYITSTMGPSVEVDVTALQDIEMSIGILGKKLGMSQFFDDQGRAIPVTLIEAGPCRITQLKTSDIDGYAAVQIGFGDTREKLINKPSKGHLTKSGEVLLKHLREYRVEGLEGLELGAAITVGSFEAGQKVDVSGDTMGRGFSGYQKRHGFSRGPMSHGSKNHREPGSTGAGTTPGRIYPGKRMAGRYGGKKRTTRGLTILKVDSNRNLLVVKGSVPGKPGALLNIRPAKRVGAQGG-SLKYETIPKLKDLLSNIHEVPKVLKITVNRGLGEAAQNAKSLEASITELAITGQKVVVTRAKKAIAGFKIRQGMPIGCAVTLRGERMYAFLERLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVSFDKIDAIRGMDITIVTSARTDEEGQSLLREMGMPFRNMSRIGKNPIPIPDKVAVTLDGLAVSVKGPKGELNRTLPEGVSVSQVENTIVVTPTSQKRKSR--ERHGLCRSLVANMVEGVSKGYTRKLELIGVGSRAQVKGKKLVVSAGFSHPVEMDPPEGITFAVENNTNVTVSGADKELVGNEAAKIRAIRPPEPYKGKGIRYEGERILRKAGKSGKK---MAKKVVSVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKAGLVIPVEISVFEDRSFTFITKTPPASVLITKAAGIEKGSGESAHGKVGSLSRSQLEEIAKTKLPDLNCTSIESAMRIIEGTARNMGVSISDMKTVPSIERWFLVDANQTLGRLATEVAVLRGKNKPFTPHLDTGDFVVVVNADKIVSGNKNQKLYRRHSGRPGGMKVETFALQRLPERIVEAIKGMLPHNALGRQLFRKLKVYRGPHPHAQRPQLLNPAASMIQESFLTVADNSGAKRIQCIRVL-GSNRYAVGDIVAAVKDAPNMVKKSEVVKAVVVRTKLRRETGSIRFDDNAAVLINDKNPRGTRVFGPVARELRERNFTKIVSLAPEVISNLSRKQQQKRHRRLRHLGTQRPRLAVFRSNHIYAQVIDDTLAASTLDDLRTLGSDASVGDLVARALAKGIQVVFDRGGNLYHGRVKSLADAAREAGLQFMTTSSPAKAGRFIRSVKVRRVLDQIRGRTYRDALIMLEFMPYRSTGPITKVLRSAVANAEHNGLDPALVIAQASADGPMKRYRPRAQGRAFAIKKQTCHISIAVAAD--------------------------------------MAVVTLSE-----MMEAGAHFGHQTRRWNPKMSRYIY-CARNGVHIIDLVQTAICMNNAYKWTRSSARSGKRFLFVGTKKQASEVVALEATRC-GASYVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEASVLRRELERLQKYLGGLKGMRRLPDVVVLVDQRRETNAVLEARKL--DIPLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLVLGRLADAINEGRH--GTNEQR-------GAD---DNDD------------------MGNKIHPTGLRLGITQE----------HRSRWYATSKMYPILLQEDD--RIRRFIHKK-Y-----GAAGISDVLIARKADQLEVELK-------TARPGVLVGRQG-----------------------------------------------------------------------SGIEELR----------------------TGIQKTIG-------DHSRQVRINVVE-------------------------------------VERVDADAFLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVGG-----------------------------RLNGAEIARNE-----WTREG---RVPLHTLRAEIDYATKVASTT-------------------------------------------YGVLGIKVWIF-----------------KGEV-LGDEA----QSMPV----------------GASPRRR-GNRRPQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAAKRSHPPGQHGQRRKRSEYAIRLEEKQKLRFNYGISERQLVRYVKKARAGSTGLLKLLERLDNVCFRLGFGPTVPGARQLVNHGHVTVNGRVLDIASYQCKAGDVVAIRERSKKLAENLEFPGLANVPHIELDKKMAKIISEREWVALEINELLVVEYYSRKVDVPAAAEGQHQEQQQ-RRGQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNERGQVGVGVGKAGDVIGAVRKGVADGKKHLVKVPLTRHNSIPTLSNGREGAANVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALASLRTHKETAKERG--ISLEQIYSMSRRKRPVPDFNNRLATMMISRLMKGKKSTARILAQAFIERTGDPIEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVNFSRARGRSMSKLAELMDAANEAGSAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHQTTVPASRMSRSIAKVLQQEGFIESEGEGLTLVLELKYSGKHRPTISMQRVSKPGLRIYKNRLPKVLGGLGMAIISTSKGVMSDRDARKQGVGGEVLCYVYM-SVVYWGTGRRKTSVARVRLVPGTGTIINGRGDYLNFNPYLVKAPLQTLGLNEQYDVLVNVGGGLTGQADAIKQGAARALCELSADNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAKKKTGKKTKRNVPNGVAIQSTFNNTIVSITDGEVIAWSSAGASGFKGARKGTPFAAQTAAEAAARRALEQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTIQQLIRERTRKTKSPALRCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDTAGVKDRSQSRSKYGAKASKMARIAGVDIPRDKRVEVALTYIYGIGLTRAKILSGVNPDIRVKDLEDDVLRLEAFTIEGDLRREMALKRLDIGLRGRRHRMSLPVRGQRTRTNARTRRGRTVAGKKK--MSLDTEKQQLINNQTHGTDTGSVEVQVAMLSERITKLSSHLQNHDSSRQGLLKMIGRRKRLLSYVRSQRYNLIKLGIRGMALKERLGTVVSDKMDKTVVVAVENRFPHPIYQKTVSRTTRYKAHDAGNTCRIGDRVRITETRPISRSKRWTVAEVLSHSPKA-----------',\n",
       " 'Prochlorococcus_marinus_str._MIT_9313': 'PKLSKRITLKVEDRYPLEAILVKENATAKFETIEAHVRLGIDPKYTDQQLRTTVALPQGTGQVRIAVISRGEKLEAAGAELAGDDDLVESIGKGMDFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLAAIEFKAGKLEFRADRAGIVHVFGKASFSDALLNLKTLQETIDRNKPSGAKGRYWKSYITSTMGPSVEVDVTALQDLEMSIGILGKKLGMSQFFDDQGRAIPVTLIEAGPCRITQLKTSEIDGYAAVQIGFGDTREKLINKPSKGHLTKSGEVLLKHLREYRVEGLEGLELGAAITVGSFEAGQKVDVSGDTMGRGFAGYQKRHGFSRGPMSHGSKNHREPGSTGAGTTPGRIYPGKRMAGRYGGKKRTTRGLTILKVDSNRNLLVVKGSVPGKPGALLNIRPAKRVGAQGG-SLKYETIPKLKDLLSNIHEVPKVLKITVNRGLGEAAQNAKSLEASITELAITGQKVVVTRAKKAIAGFKIRQGMPIGCAVTLRGERMYAFLERLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEISFDKIDAIRGMDITIVTSARTDEEGQSLLREMGMPFRNMSRIGKNPIPIPDKVAVTLDGLAVSVKGPKGELSRTLPEGVSVSQVENTIVVTPTSQKRKSR--ERHGLCRSLVANMVEGVSKGYTRKLELIGVGSRAQVKGKKLVVSAGFSHPVEMDPPEGVTFAVENNTNVTVSGADKELVGNEAAKIRAIRPPEPYKGKGIRYEGERILRKAGKSGKK---MAKKVVSVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKAGLVIPVEISVFEDRSFTFITKTPPASVLITKAAGIEKGSGESAHGKVGSLSRSQLEEIAKTKLPDLNCTSIESAMRIIEGTARNMGVSISDMKTVPSIERWFLVDANQTLGRLATEVAVLRGKNKPFTPHLDTGDFVVVVNADKIVSGNKNQKLYRRHSGRPGGMKVETFALQRLPERIVEAIKGMLPHNALGRQLFRKLKVYRGPHPHAQRPQLLNPAASMIQESFLTVADNSGAKRIQCIRVL-GSNRYAVGDIVAAVKDAPNMVKKSEVVKAVVVRTKLRRDTGSIRFDDNAAVLINDKNPRGTRVFGPVARELRERNFTKIVSLAPEVISNLSRKQQQKRHRRLRHLGTQRPRLAVFRSNHIYAQVIDDTLAASTLDDLRTLGSDASVGDLVARALAKGIQVVFDRGGNLYHGRVKSLADAAREAGLQFMTTSSPAKAGRFIRSVKVRRVLDQIRGRTYRDALIMLEFMPYRSTGPITKVLRSAVANAEHNGLDPALVIAQASADGPMKRYRPRAQGRAFAIKKQTCHISIAVAAD--------------------------------------MAVVTLSE-----MMEAGAHFGHQTRRWNPKMSRYIY-CARNGVHIIDLVQTAICMNNAYKWTRSSARSGKRFLFVGTKKQASEVVALEATRC-GASYVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEASVLRRELERLQKYLGGLKGMRRLPDVVVLVDQRRETNAVLEARKL--DIPLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLVLGRLADAINEGRH--GTNDQR-------GAD---DSDD------------------MGHKIHPTGLRLGITQE----------HRSRWYATSKMYPILLQEDD--RIRRFIHKK-Y-----GAAGISDVLIARKADQLEVELK-------TARPGVLVGRQG-----------------------------------------------------------------------SGIEELR----------------------TGIQKTIG-------DHSRQVRINVVE-------------------------------------VERVDADAFLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVGG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRAEIDYATKVASTT-------------------------------------------YGVLGIKVWIF-----------------KGEV-LGDEA----QTMPV----------------GASPRRR-GNRRPQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAAKRSHPPGQHGQRRKRSEYAIRLEEKQKLRFNYGISERQLVRYVKKARAGSTGLLKLLERLDNVCFRLGFGPTVPGARQLVNHGHVTVNGRVLDIASYQCKAGDVVAIRERSKKLAENLEFPGLANVPHIELDKKMAKIISEREWVALEINELLVVEYYSRKVDVPAAAEGQ-QEQQQ-RRGQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNERGQVGVGVGKAGDVIGAVRKGVADGKKHLVKVPLTRHNSIPTLSNGRDGAANVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALASLRTHKETAKERG--ISLEQIYSMSRRKRPVPDFNNRLATMMISRLMKGKKSTARILAQAFIERTGDPIEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVNFSRARGRSMSKLAELMDAANEAGSAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHQTTVPASRMSRSIAKVLQQEGFIESEGEGRTLVLELKYSGKHRPTISMQRVSKPGLRIYKNRLPKVLGGLGMAIISTSKGVMSDRDARKQGVGGEVLCYVYM-SVVYWGTGRRKTSVARVRLVPGSGTIINGRGDYLNFNPYLVKAPLQTLGLNEQYDVLVNVGGGLTGQADAIKQGAARALCELSADNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAKKKTGKKTKRNVPNGVAIQSTFNNTIVSIADGEVIAWSSAGASGFKGARKGTPFAAQTAAEAAARRALEQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTIQQLIRERTRKTKSPALRCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDTAGVKDRSQSRSKYGAKASKMARIAGVDIPRDKRVEVALTYIYGIGLTRAKILSDVNPDIRVKDLEDDVLRLEAFTIEGDLRREMALKRLDIGLRGRRHRMSLPVRGQRTRTNARTRRGRTVAGKKK--MSLDTEKQQLINNQTHGTDTGSVEVQVAMLSERITKLSSHLQNHDSSRQGLLKMIGRRKRLLSYVRSQRYNLIKLGIRGMALKERLGTVVSDKMDKTVVVAVENRFPHPIYQKTVSRTTRYKAHDAGNTCRVGDRVRITETRPLSRSKRWTVAEVLSHSPKA-----------',\n",
       " 'Synechococcus_sp._WH7803_(re-annotation)': 'PKTSKRLALKIEERYPLEAILVKENATAKFETVEAHARLGIDPKYTDQQLRTTVALPHGTGQVRIAVITRGEKVEAAGAELAGDEDLVETISKGMDFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSGNLLNLKTLQETIDRNKPSGAKGRYWKSYVTSTMGPSVEVDIAALQDINMSIGILGKKLGMSQLFDEQGKAVPVTLIEAGPCRITQLKTTETDGYSAVQIGFGETRDKLVNKPAKGHLAKSGEGVLRHLSEYRVDDVNGLELGGAVTVSDFEAGQKVDVSGDTMGRGFAGYQKRHGFSRGPMTHGSKNHREPGSTGAGTTPGRIYPGKRMAGRYGGKKTTTRGLTILKVDSDRNLLVVKGSVPGKPGALLNIRPANRVGAKGG-SLKYETIPKLKDLLSNVHEVPKVVKVTVNRGLGEAAQNAKSLEASVNELAITGQKVVVTRAKKAIAGFKIRQGMPIGCAVTLRGERMYAFLERLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQIIFPEISFDKIDAIRGMDITIVTTARTDEEGRALLREMGMPFRNMSRIGKSPIPIPDKVSVSLDGLAVTVKGPKGELKRTLPDGVTVNQVDNTIVVAPTSEKRQSR--ERHGLCRTLVANMIEGVSKGYSKKLEIVGVGSRAQVKGKTLVVSAGYSHPVEMLAPEGITFAVENNTNVTVSGTDKELVGNEAAKIRAIRPPEPYKGKGIKYAGERILRKAGKSGKK---MAKKVTAVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKAGFVIPVEISVFEDRSFTFITKTPPASVLITKAAGIEKGSGESAKGSVGSIKRSQLEEIAKTKLPDLNCTSIESAMRIIEGTARNMGVAITDMKTVPSIDRWYLVDANQTLGRLATEVAVLRGKNKAFTPHLDTGDFVVVVNADKIVSGNKQQKLYRRHSGRPGGMKVETFHLQRLPERIVEAIKGMLPHNALGRQMFRKLKVYKGAHPHAQQPKLLDPAATMIQESFLTVADNSGAKRIQCIRVL-GTNRYAVGDIVAAVKDAPNMVKKSDVVKAVVVRTKLRRDTGSIRFDDNAAVIINDKNPKGTRVFGPVARELRERNFTKIVSLAPEVISTLSRKQQQKRHRRLRHLGTDRPRLAVFRSNHIYAQVIDDTLSASTLDDLRTLGTDASVGELVARALAKGIQVVFDRGGNLYHGRVKALADAAREAGLQFMTTSSPAQAGRFIRSVKVRRVLDQIRGRTYRDALIMLEFMPYRSTGPITKVLRSAVANAEHNGLDPSLVISQASADGPMKRYRPRAQGRAYAIKKQTCHISIAVASD--------------------------------------MAVVTLAE-----MMEAGAHFGHQTRRWNPKMSRYIY-CARNGVHIIDLVQTAVCMNNAYKWVRSAARSGKRFLFVGTKKQASEVVAQEALRC-GGSYVNQRWLGGMLTNWTTMKARIDRLKDLERMEASGAIAMRPKKEGAVLRRELDRLQKYLGGLKNMRRLPDVVVLVDQRRETNAVLEARKL--DIPLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLVLSRLADAINEGRH--GSNEQR-------GGD---------GSEG------------MGHKIHPTGLRLGITQE----------HRSRWYASSKSYPTLLKEDD--RIRKFIHKK-Y-----GSAGISDVLIARKADQLEVELK-------TARPGVLVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTVG-------DPNRQVRINVVE-------------------------------------VERVDADAFLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYATKVASTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLGDEAQ----QQLPV----------------GATPRRR-AGRRPQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAAKRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSERQLVRYVKKARAGSTGLLKLLERLDNVCFRLGFGPTVPGARQLVNHGHVTVNGRVTDIASYQCKPGDVIAIRERSKQLAENLEFPGLANVPHLELDKKLAKVVGEREWVALEINELLVVEYYSRKVDVPAAAEGQ-QEQRR-RRNQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNERGQVGVGVGKAGDVIGAVRKGVADGKKHLVKVPLTRHNSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALDSLRTHKETAKERG--ISLEQIYSMSRRKRPVPDFNNRLATMLVARLMKGKKSTARILSDAFIERTGDPIEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVSFSRARGRSMAKLAELMDAANEAGSAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHQNTVPASRMSRSIAKVLQQEGFIESEGEGHTLVLELKYSGKHRPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARKQGVGGEVLCYVY--SVVYWGTGRRKTSVARVRLVPGNGTIINGRGDYLNYNPYLVRAPLQTLGLLTEYDVLVNVGGGLTGQADAIKQGAARALCELSADNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAKKKSGKKAKRNVPNGVAIQSTFNNTIVSITDGEVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALDQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTIQQLIRERKAKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIGGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKTPKVARIAGVDIPRDKRIEVALTYIYGIGLTRAQILTGVNPDTRVKDLEDDVLRTEAFTIEGDLRREMALKRLDIGVRGRRHRMSLPVRGQRTRTNARTRRGRTVAGKKK--MSLDTEKQELINHQTHATDTGSAEVQVAMLTERISKLSSHLQNHDSSRQGLLKMIGRRKRLLGYVRSKRYSLIKLGIRGMALKERVGTVVSDKMDKTVVVAVENRFPHPIYQKTVSRTTRYKAHDENNSVRVGDRVRITETRPLSRHKRWAVAEVLSHSPKA-----------',\n",
       " 'Synechococcus_sp._WH_8020': 'PKLSKRLALKIEDRYPLEAILVKENATAKFETMEAHVRLGIDPKYTDQQLRTTVALPQGTGQVRIAVITRGEKVEAAGAELSGDEDLVEAISKGMNFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFTEALLNLKTLQETIDRNKPSGAKGRYWKSYVTSTMGPSVEVDIALLQDIEMSIGILGKKLGMSQFFDEQGKAVPVTLIEAGPCRITQLKSSETDGYQAVQIGFGEIREKLINKPAKGHLAKSGEDLVRHLCEYRVDDLDGIQLGGAVTVGDFAAGQKVDVSGDTMGRGFAGLQKRHGFSRGPMTHGSKNHRQPGSIGAGTTPGRIYPGKRMSGRYGGKKITTRGLTILKIDSDRNLLVVKGSVPGKPGSLLNIRPANRVGAKGG-SLKYETIPKLKDLLSNIHEVPKVLKVTVNRGLGEAATNAKSLEASVNELAITGQKVVITRAKKAIAAFKIRQGMPIGCAVTLRGDRMYAFLERFINLALPRIRDFRGVSPKSFDGRGNYTVGVREQIIFPEISFDKIDAIRGMDITIVTSARTDEEGRALLREMGMPFRNMSRIGKNPVPIPDKVNVTLNGLAVTVKGPKGELKRTLPIGVSVNQVDNSIVVEPTSTKRSSR--ERHGLCRTLVANMVIGVSQGYSKKLEIVGVGSRAQVKGKTLVVSAGYSHPVEVVPPEGITFTVENNTNVTVSGTDKELVGNEAAKIRAIRPPEPYKGKGIKYAGERILRKAGKSGKK---MAKKVVAVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKAGYVIPVEISVFEDRSFTFITKTPPASVLITKAAGIQKGSGESAKGSVGSINRSQLEEIAKTKLPDLNCTSVESAMRIIEGTARNMGVAVSDMKTVPSIDRWYLVDANQTLGRLATEVAVLRGKNKAFTPHLDTGDFVIVVNADKIVSGKKQQKLYRRHSGRPGGMKVETFHLQRLPERIVEAIKGMLPHNALGRQLFRKLKVYKGTHPHAQQPTLLDPAASMIQETFLTVADNSGAKRIQCIRVL-GTNRYAVGDIVATVKDAPNMVKKSDIVKAVVVRTKMRRDTGSIRFDDNAAVIINDKNPKGTRVFGPVARELRERSFTKIVSLAPEVISTLSRKQQQKRHRRLRHLGTNRPRLAVFRSNHIYAQLIDDTLSASTLDDLRSLGSDASVGDLVARAIAKGIQVVFDRGGNLYHGRVKALADAAREAGLQFMTTSSPAQAGRFIRSVKVRRVLDQIRGRTYRDALIMLEFMPYRSTGPITKVLRSAVANAEHNGLDPALVISQASADGPMKRYRPRAQGRAFAIKKQTCHISIAVAAD--------------------------------------MAVVTLAE-----MMEAGAHFGHQTRRWNPKMSRYIY-CARNGVHIIDLVQTAVCMNNAYKWVRSAARSGKRFLFVGTKKQASEVVAQEALRC-GSSYVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEGAVLRRELERLQKYLGGLKNMRRIPDVVVLVDQRRETNAVLEARKL--DISLVSMLDTNC--DPDLCEVPIPCNDDAVRSIQLVLSRLADAINEGRH--GGQDGR--------------GDDG-----------------MGHKIHPTGLRLGITQE----------HRSRWYASSKSYPALLQEDD--RIRKFIHKK-Y-----GSAGISDVLIARKADQLEVELK-------TARPGVLVGRQG-----------------------------------------------------------------------SGIEDLR----------------------SGIQKTVG-------DSSRQVRINVVE-------------------------------------VERVDADAFLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRAEIDYATKVASTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLSDDSQ----QQIPV----------------GANPRRR-AGRRPQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAAKRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSERQLVRYVKKARAGSTGLLKLLERLDNVCFRIGFGPTVPGARQLVNHGHVTVNGRVTDIASYQCKAGDVIAIRERSKLLAENLQFPGLANVPHLELDKKLAKVIGEREWVALEINELLVVEYYSRKVDVPAAAEGQGQQQEQ-RRGQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNERGQVGVGVGKAGDVIGAVRKGVADGKKHLVKVPLTRHNSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALQLLRTHKETAKERG--ISLEQIYSMSRRKRPIPDFNNRLATMMVARLMKGKKSTARILSDAFIERTGDPVEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVNFSRARGRSMAKLAELMDAANEAGNAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHESTVPASRMSRSIAKVLQQEGFIESEGEGSTLVLGLKYSGKHRPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARKQGVGGEVLCYVYM-TVVYWGTGRRKTSVARVRLVPGNGTIINGRGDYLNYNPYLVKAPLQTLGLSTEYDVLVNVGGGLTGQADAIKQGAARALCELSVDNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAKKKSGKKAKRNVPNGVAIQSTFNNTIVSITDGEVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALEQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRSKRRRVMPTIQQLIRERKAKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIGGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKAPKVARIAGVDIPRDKRIEVALTYIYGIGSTRAKILAGVNPDIRVKDLEDDVLRTESFTIEGDLRREMALKRLDIGLRGRRHRMSLPVRGQRTRTNARTRRGRTVAGKKK--MSLDTEKQQLINHQTHATDTGSAEVQVAMLSERISKLSSHLQNHDSSRQGLLKMIGRRKRLLGYVRSQRYSLIKLGIRGMALKERVGTVVSDKMDKTVVVAVENRFPHPIYQKTVSRTTRYKAHDEDNACCVGDRVRITETRPLSRHKRWAIAEVLSQSPKA-----------',\n",
       " 'Synechococcus_sp._CC9311': 'PKLSKRLALKIEDRYPLEAILVKENATAKFETMEAHVRLGIDPKYTDQQLRTTVALPQGTGQVRIAVITRGEKVEAAGAELSGDEDLVEAISKGMNFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFTEALLNLKTLQETIDRNKPSGAKGRYWKSYVTSTMGPSVEVDIALLQDIEMSIGILGKKLGMSQFFDEQGKAVPVTLIEAGPCRITQLKSSETDGYQAVQIGFGEIREKLINKPAKGHLAKSGEDLVRHLCEYRVDDLDGIQLGGAVTVGDFAAGQKVDVSGDTMGRGFAGLQKRHGFSRGPMTHGSKNHRQPGSIGAGTTPGRIYPGKRMSGRYGGKKITTRGLTILKIDSERNLLVVKGSVPGKPGSLLNIRPANRVGAKGG-SLKYETIPKLKDLLSNIHEVPKVLKVTVNRGLGEAATNAKSLEASVNELAITGQKVVITRAKKAIAAFKIRQGMPIGCAVTLRGDRMYAFLERFINLALPRIRDFRGVSPKSFDGRGNYTVGVREQIIFPEISFDKIDAIRGMDITIVTSARTDEEGRALLREMGMPFRNMSRIGKNPIPIPDKVNVTLSGLAVTVKGPKGELKRTLPSGVSVNQVDNSIVVAPTSTKRSSR--ERHGLCRTLVANMVIGVSQGYSKKLEIVGVGSRAQVKGKTLVVSAGYSHPVEVVPPEGITFTVENNTNVTVSGTDKELVGNEAAKIRAIRPPEPYKGKGIKYAGERILRKAGKSGKK---MAKKVVAVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKAGYVIPVEISVFEDRSFTFITKTPPASVLITKAAGIQKGSGESAKGSVGSINRSQLEEIAKTKLPDLNCTSVESAMRVIEGTARNMGVAISDMKTVPSIDRWYLVDANQTLGRLATEVAVLRGKNKAFTPHLDTGDFVIVVNADKIVSGKKQQKLYRRHSGRPGGMKVETFHLQRLPERIVEAIKGMLPHNALGRQLFRKLKVYKGTHPHAQQPKLLDPAASMIQETFLTVADNSGAKRIQCIRVL-GTNRYAVGDIVATVKDAPNMVKKSDIVKAVVVRTKMRRDTGSIRFDDNAAVIINDKNPKGTRVFGPVARELRERSFTKIVSLAPEVISTLSRKQQQKRHRRLRHLGTNRPRLAVFRSNHIYAQLIDDTLSASTLDDLRSLCSDASVGDLVARAIAKGIQVVFDRGGNLYHGRVKALADAAREAGLQFMTTSSPAQAGRFIRSVKVRRVLDQIRGRTYRDALIMLEFMPYRSTGPITKVLRSAVANAEHNGLDPALIISQATADGPMKRYRPRAQGRAYAIKKQTCHISIAVAAD--------------------------------------MAVVTLAE-----MMEAGAHFGHQTRRWNPKMSRYIY-CARNGVHIIDLVQTAVCMNNAYKWVRSAARSGKRFLFVGTKKQASEVVAQEALRC-GSSYVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEGAVLRRELERLQKYLGGLKNMRRIPDVVVLVDQRRETNAVLEARKL--DISLVSMLDTNC--DPDLCEVPIPCNDDAVRSIQLVLSRLADAINEGRH--GGQDGR----GDDGQG-------------------------MGHKIHPTGLRLGITQE----------HRSRWYASSKSYPALLQEDD--RIRKFIHKK-Y-----GSAGISDVLIARKADQLEVELK-------TARPGVLVGRQG-----------------------------------------------------------------------SGIEDLR----------------------SGIQKTVG-------DSSRQVRINVVE-------------------------------------VERVDADAFLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRAEIDYATKVASTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLSDDSQ----QQIPV----------------GANPRRR-AGRRPQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAAKRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSERQLVRYVKKARAGSTGLLKLLERLDNVCFRIGFGPTVPGARQLVNHGHVTVNGRVTDIASYQCKAGDVIAIRERSKLLAENLQFPGLANVPHLELDKKLAKVIGEREWVALEINELLVVEYYSRKVDVPAAAEGQGQQQEQ-RRGQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNERGQVGVGVGKAGDVIGAVRKGVADGKKHLVKVPLTRHNSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALQLLRTHKETAKERG--ISLEQIYSMSRRKRPIPDFNNRLATMMVARLMKGKKSTARILSDAFIERTGDPVEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVNFSRARGRSMAKLAELMDAANEAGSAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHESTVPASRMSRSIAKVLQQEGFIESEGEGRTLVLGLKYSGKHRPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARKQGVGGEVLCYVYM-TVVYWGTGRRKTSVARVRLVPGNGTIINGRGDYLNYNPYLVKAPLQTLGLSTEYDVLVNVGGGLTGQADAIKQGAARALCELSVDNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAKKKSGKKAKRNVPNGVAIQSTFNNTIVSITDGEVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALEQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRSKRRRVMPTIQQLIRERKAKTKSPALRCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIGGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKAPKMARIAGVDIPRDKRIEVALTYIYGIGSTRAKILAGVNPDIRVKDLEDDVLRTESFTIEGDLRREMALKRLDIGLRGRRHRMSLPVRGQRTRTNARTRRGRTVAGKKK--MSLDTEKQQLINHQTHATDTGSAEVQVAMLSERISKLSSHLQNHDSSRQGLLKMIGRRKRLLGYVRSQRYSLIKLGIRGMALKERVGTVVSDKMDKTVVVAVENRFPHPIYQKTVSRTTRYKAHDEDNACRVGDRVRITETRPLSRHKRWAIAEVLSQSPKA-----------',\n",
       " 'Synechococcus_sp._WH_8016': 'PKLSKRLALKIEDRYPLEAILVKENATAKFETMEAHVRLGIDPKYTDQQLRTTVALPQGTGQVRIAVITRGEKVEAAGAELSGDEDLVEAISKGMNFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFKEALLNLKTLQETIDRNKPSGAKGRYWKSYVTSTMGPSVEVDIALLQDIEMSIGILGKKLGMSQFFDEQGKAVPVTLIEAGPCRITQLKSSDTDGYQAVQIGFGDIREKLINKPAKGHLAKSGEDLLRHLCEYRVDDLDGIQVGGAITVGDFTAGQKVDVSGDTMGRGFAGLQKRHGFSRGPMTHGSKNHRQPGSIGAGTTPGRIYPGKRMSGRYGGKKITTRGLTILKIDSDRNLLVVKGSVPGKPGSLLNIRPANRVGAKGG-SLKYETIPKLKDLLSNIHEVPKVLKVTVNRGLGEAATNAKSLEASVNELAITGQKVVITRAKKAIAAFKIRQGMPIGCAVTLRGDRMYAFLERFINLALPRIRDFRGVSPKSFDGRGNYTVGVREQIIFPEISFDKIDAIRGMDITIVTSARTDEEGRALLREMGMPFRNMSRIGKNPIPIPDKVNVTLDGLAVTVKGPKGELKRTLPVGVSVNQVDNSIVVAPTSTKRSSR--ERHGLCRTLVANMVIGVSQGYSKKLEIVGVGSRAQVKGKTLVVSAGYSHPVEMVPPEGITFAVEGNTNVTVSGTDKELVGNEAAKIRAIRPPEPYKGKGIKYAGERILRKAGKSGKK---MAKKVVAVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKAGYVIPVEISVFEDRSFTFITKTPPASVLITKAAGIPKGSGESAKGSVGSINRSQLEEIAKTKLPDLNCTSVESAMRIIEGTARNMGVAVSDMKTVPSIDRWYLVDANQTLGRLATEVAVLRGKNKAFTPHLDTGDFVIVVNADKIVSGKKQQKLYRRHSGRPGGMKVETFHLQRLPERIVEAIKGMLPHNALGRQLFRKLKVYKGTHPHAQQPKLLDPAASMIQETFLTVADNSGAKRIQCIRVL-GTNRYAVGDIVATVKDAPNMVKKSDIVKAVVVRTKMRRDTGSIRFDDNAAVIINDKNPKGTRVFGPVARELRERSFTKIVSLAPEVISTLSRKQQQKRHRRLRHLGTDRPRLAVFRSNHIYAQLIDDTLSASTLDDLRSLGSDASVGDLVARAIAKGIQVVFDRGGNLYHGRVKALADAAREAGLQFMTTSSPAQAGRFIRSVKVRRVLDQIRGRTYRDALIMLEFMPYRSTGPITKVLRSAVANAEHNGLDPALIISQATADGPMKRYRPRAQGRAYAIKKQTCHISIAVAAD--------------------------------------MAVVTLAE-----MMEAGAHFGHQTRRWNPKMSRYIY-CARNGVHIIDLVQTAVCMNNAYKWVRSAARSGKRFLFVGTKKQASEVVAQEALRC-GSSYVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEGAVLRRELERLQKYLGGLKNMRRIPDVVVLVDQRRETNAVLEARKL--DISLVSMLDTNC--DPDLCEVPIPCNDDAVRSIQLVLSRLADAINEGRH--GGQDGR----GDDGQG-------------------------MGHKIHPTGLRLGITQE----------HRSRWYASSKNYPALLQEDD--RIRKFIHKK-Y-----GSAGISDVLIARKADQLEVELK-------TARPGVLVGRQG-----------------------------------------------------------------------SGIEDLR----------------------SGIQKTVG-------DSSRQVRINVVE-------------------------------------VERVDADAFLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRAEIDYATKVASTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLSDDSQ----QQIPV----------------GANPRRR-AGRRPQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAAKRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSERQLVRYVKKARAGSTGLLKLLERLDNVCFRIGFGPTVPGARQLVNHGHVTVNGRVTDIASYQCKAGDVIAIRERSKLLAENLQFPGLANVPHLELDKKLAKVVGEREWVALEINELLVVEYYSRKVDVPAAAEGQGQQQEQ-RRGQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNERGQVGVGVGKAGDVIGAVRKGVADGKKHLVKVPLTRHNSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALQLLRTHKETAKERG--ISLEQIYSMSRRKRPIPDFNNRLATMMVARLMKGKKSTARILSDAFIERTGDPVEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVNFSRARGRSMAKLAELMDAANEAGSAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHESTVPASRMSRSIAKVLQQEGFIESEGEGRTLVLELKYSGKHRPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARKQGVGGEVLCYVYM-AVVYWGTGRRKTSVARVRLVPGNGTIINGRGDYLNYNPYLVKAPLQTLGLSTEYDVLVNVGGGLTGQADAIKQGAARALCELSVDNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAKKKSGKKAKRNVPNGVAIQSTFNNTIVSITDGEVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALEQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRSKRRRVMPTIQQLIRERKAKTKSPALRCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIGGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKSPKMARIAGVDIPRDKRIEVALTYIYGIGSTRAKILAGVNPDIRVKDLEDDVLRTESFTIEGDLRREMALKRLDIGLRGRRHRMSLPVRGQRTRTNARTRRGRTVAGKKK--MSLDTEKQQLINHQTHATDTGSAEVQVAMLSERISKLSSHLQNHDSSRQGLLKMIGRRKRLLGYVRSQRYSLIKLGIRGMALKERVGTVVSDKMDKTVVVAVENRFPHPIYQKTVSRTTRYKAHDEDNACRVGDRVRITETRPLSRHKRWAIAEVLSQSPKA-----------',\n",
       " 'Synechococcus_sp._CC9902': 'PKLSKRLALKIEDRYPLEAILVKDNANAKFETMEAHVRLGIDPKYTDQQLRTTVALPNGTGQVRIAVVTSGEKVAAAGAELAGDEDLVEAISKGMNFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLAAIDFKAGKLEFRADRTGIVHVFGKASFTDALLNLKTLQETIDRNKPSGAKGRYWKSYVTSTMGPSVEVDFSALQEIGMSIGILGKKLGMSQLFDEQGRAVPVTLIEAGPCRITQLKNDDTDGYAAVQIGFGETREKLINKPSKGHLTKSGEDLLRHLREYRVDNVDGLELGGSITVSDFEAGQKVDVSGDTMGRGFAGLQKRHGFSRGPMTHGSKNHRQPGSIGAGTTPGRIYPGKRMSGRYGGKKTTTRGLTILRIDSDRNLLVVKGSVPGKPGALLNIKPAVRVGAKGG-SLKYETIPKLKDLLTNIHEVPKVMKVTINRGLGEAATNAKSLEASVKELAITGQKVVVTRAKKAIAGFKIRQGMPIGCAVTLRGDRMYAFLERLINLALPRIRDFRGVSAKSFDGRGNYTLGVREQIIFPEISFDKIDAIRGMDITIVTTARSDEEGRALLSEMGMPFQNMSRIGKNPVPVPDKVTVSLDGLTVKVKGPKGELERTLPEGVSVSQEANTIVVAPTSTKRISR--ERHGLSRTLVANMIEGVHNGYSKALEIVGVGSRAQVKGTTLVVSAGYSHPVEMPAPAGITFKVENNTRVIVSGIDKELVGNEAAKVRAIRPPEPYKGKGIKYEGEHILRKAGKSGKK---MAKKVVAVIKLALQAGKANPAPPVGPALGQHGVNIMMFCKEYNARTQDKAGLVIPVEISVFEDRSFTFITKTPPASVLITKAAGIEKGSGDSAKGSVGSISRAQLEEIAKTKLPDLNCSSVDSAMRIIEGTARNMGVAVSDMKTLPSIDRWYVVDANQTLGRLATEVAILRGKNNPFTPHLDTGDFVVVVNAEKIVSGKKQQKLYRRHSGRPGGMKVETFSLQRIPERIVEAIKGMLPHNALGRQMYRKLKVYKGTHPHAQKPQLLNPSATMIQESFLTVADNSGAKRIQCIRVL-GTNRYAVGDIVAAVKDAPNMVKKSDVVKAVVVRTKLRRETGSIRFDDNAAVIINDKNPKGTRVFGPVARELRERSFTKIVSLAPEVISKLSRKQQQKRHRRLRHIGTNRPRLAVFRSNHIYAQLIDDTLSASTVDELRSLASDASVGALVARAIAKGIQVVFDRGGNLYHGRIKALADAAREAGLQFMTSSTSAQAGRFIRSVKVRRVLDQIRGRTYRDALIMLEFMPYRSTGPITKVLRSAVANAENNGLDPALVISSASADGPMKRYRPRAQGRAFQIKKQTCHISIAVAVD--------------------------------------MAVVTLSE-----MMEAGAHFGHQTRRWNPKMSRYIY-CARNGVHIIDLVKTAVCMNNAYKWTRSAARSGKRFLFVGTKKQASEVVALEAARC-GAAYVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEGAVLRRELERLQKYLGGLKTMRRLPDVVVLVDQRRESNAVLEARKL--DIPLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLVLGRLADAINEGRH--GSNEQR-------GGD---DYEG------------------MGHKIHPTGLRLGITQE----------HRSRWYASSKTYPSLLQEDD--RIRKFIHKK-Y-----GSAGISDVLIARKADQLEVELK-------TARPGVLVGRQG-----------------------------------------------------------------------SGIEDLR----------------------AGIQKTVG-------DKSRQVRINVVE-------------------------------------VERVDGDAFLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYATKVGKTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLNEQSQ-----TMPV----------------GANPRRR-ASRRPQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAAKRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSERQLVRYVKKARAGSTGLLKLLERLDNICFRLGFGPTVPGSRQLVNHGHVTVNGRVTDIASYQVKPGDVLAIREKSKQLAENLAFPGLSNIPHLELDKKLAKCTGEREWVALEINELLVVEYYSRKVDVPAAAQGQ-QQEQR-RRGQ---DRDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVADGKKHLVKVPLTRHSSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSSLRTHKETAKERG--ISLEQIYSMSRRKRPIPDFNSRLATMMVARLMQGKKSTARILSDAFIERTGDPLEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVSFSRARGRSMSKLAELMDAANEAGSAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHETTVPASRMTRSIAKVLQQEGFIENEGEGRALVLSLKYSGKHRPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARREGVGGEVLCYVY--SVVYWGTGRRKTSVARVRLVPGNGTIINGRGDYLNYNPYIVKAPLETLGLSTEYDVLVNVGGGLTGQSGAIKQGAARALCELSADNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAKKKSGKKAKRNVPNGVAIQSTFNNTIVSISDGEIIAWSSAGASGFKGARKGTPFAAQTAAEAAARRALEQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTIQQLIRERKVKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIGGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDTSGVKDRRQSRSKYGAKAPKMARIAGVDIPRDKRVEVSLTYIYGIGLTRAKILAGVNPDIRVKDLEDDILRTEAFTIEGDLRREMALKRLDIGVRGRRHRMSLPVRGQRTRTNARTRRGRTVAGKKK--MSLDTEKQQLINHQTHGTDTGSAEVQVAMLSERINRLSGHLQNHDSSRQGLLKMIGRRKRLLNYMRSQRYSTIKLGIRGMAVKERVGTVVSDKMEKTVVVAVETRFPHPIYQKTVSRTTRYKAHDEDNSCRVGDRVRITETRPMSRQKRWAIAEVLSHSPKA-----------',\n",
       " 'Synechococcus_sp._CC9616': 'TKISKRLALKIEDRYPLEAILVKDNATAKFETIEAHVRLGIDPKYTDQQLRTTVALPNGTGQVRIAVVTRGEKVEAAGAELAGEEELVETIAKGMNFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLEAIEFKAGKLEFRADRTGIVHVFGKASFSEALLNLKTLQETIDRNKPSGAKGRYWKSYVTSTMGPSVEVDFSALQDIEMSIGILGKKLGMSQFFDEQGKAVPVTLIEAGPCRITQLKSNDTDGYAAVQIGFGGTREKLINRPAKGHLSKSGDELLRHLREYRVDAVEGLELGGAITVGDFETGQKVDVSGDTMGRGFAGYQKRHGFSRGPMTHGSKNHREPGSTGAGTTPGRIYPGKRMAGRFGGKKITTRGLTILKIDSDRNLLVVKGSVPGKPGALLNIRPAKRVG-KGG-SLKYETIPKLKDLLKNVHEVPKVLKVTVNRGLGEAAQNAKSLEASVGELAITGQKVMVTRAKKAIAAFKIRQGMPIGCAVTLRGERMYAFLERLINLALPRIRDFRGVNPKSFDGRGNYTLGVREQLIFPEISFDKIDAIRGMDITIVTTARTDEEGRALLREMGMPFRNMSRIGKNPVPVPDKVTVSLDGLTVTVKGPKGELQRTLPEGVSVSQVDSTIVVSPTSEKRQSR--ERHGLCRALVANMIEGVSNGYSKSLEIIGVGSRAQVKGKTLVVSAGYSHPVEMDAPEGITFKVENNTKVIVSGIDKELVGNEAAKVRAIRPPEPYKGKGIRYEGEWILRKAGKSGKK---MAKKVTAVIKLALQAGKANPAPPVGPALGQHGVNIMMFCKEYNARTQDKAGYVIPVEISVYEDRSFTFITKTPPASVLITKAAGIQKGSGESAKGSVGSIKRAQLEEIAKTKLPDLNCTSIESAMRIIEGTARNMGVSISDMKTLPSIERWYLVDANQTLGRLATEVAVLRGKNNPFTPHLDTGDFVVVVNADKVVSGKKLQKLYRRHSGRPGGMKTETFALQRIPERIVEAIKGMLPHNALGRQMFRKLKVYKGSHPHAQKPQLLNPSASMIQESFLTVADNSGAKRIQCIRVL-GTNRYAVGDIVATVKDAPNMVKKSDVVKAVVVRTKLRRDTGSIRFDDNAAVIINDNNPKGTRVFGPVARELRDRSFTKIVSLAPEVISSLSRKQQQKRHRRLRDLGTSRPRLAVFRSNHIYAQLIDDTLSASTIDELRTLGCDASVGELVARAIAKGIQVVFDRGGNLYHGRIKALANAAREAGLQFMTTSSTAQAGRFIRSVKVRRVLDQIRGRTYRDALIMLEFMPYRSTGPITKVLRSAVANAEHNGLDPALVISSASADGPMKRYRPRAQGRAYQIKKQTCHISIAVAAD--------------------------------------MAVVTLSE-----MMEAGAHFGHQTRRWNPKMSRYIY-CARNGVHIIDLVQTAVCMNNAYKWTRSAARSGKRFLFVGTKKQASEVVALEAARC-GAAFVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEGAVLRRELERLQKYLGGLKTMRRLPDVVVLVDQRRESNAVLEARKL--DIPLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLVLGRLADAINEGRH--GHQDQR-------GGD---DAEG------------------MGHKINPTGLRLGITQE----------HRSRWYAPSKSYPALLQEDD--RIRKFIHKK-Y-----GSAGISDVLIARKADQLEVELK-------TARPGVLVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTVG-------DRSRQVRINVVE-------------------------------------VERVDGDAFLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYATKVASTT-------------------------------------------YGVLGIKVWVF-----------------KGEV-LGDEA----PPMPV----------------GASPRRR-ASRRPQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAAKRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGISERQLVRYVKKARAGSTGLLKLLERLDNVCFRMGFGPTVPGARQLVNHGHVTVNGRVTDIASYQCKVGDVIAIRERSKRLAENLEFPGLANIPHLDLDKKLAKVTGEREWVALEINELLVVEYYSRKVDVPAAAEGQ-QQEQR-RRGQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVADGKKHLVKVPLTRHNSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSLLRTHKETAKERG--ISLEQIYSMSRRKRPVPDFNSRLATMMVARLMKGKKSTARILSDAFIERTGDPLEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVSFSRSRGRSMAKLAELMDAANEAGSAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHESTIPASRMTRSIAKVLQQEGFIESEGEGRTLVLALKYSGKHRPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARREGVGGEVLCYVY--SVVYWGTGRRKTSVARVRLVPGNGTIINGRGDYLNYNPYIVKAPLETLGLTTEYDVLVNVGGGLTGQAGAIKQGAARALCELSADNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAKKKSGKKAKRNVPNGVAIQSTFNNTIVSITDGEVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALDQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTIQQLIRERKAKTKSPALRCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIGGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKAPKMARIAGVDIPRDKRVEVALTYIYGIGPNRARILSGVNPDIRVKDLDDDVLRTESFTIEGDLRREMALKRLDIGLRGRRHRMSLPVRGQRTRTNARTRRGRTVAGKKK--MSLDTEKQQLINHQTHGTDTGSPEVQVAMLTERISRLSSHLQNHDSSRQGLLKMIGRRKRLLSYMRSQRYTIIKLGIRGMALKERVGTVVSDKMEKTVVVAVESRFPHPIYQKTVSRTTRYKAHDEGNTCRVGDRVRITETRPMSRHKRWAIAEVLSHSPKA-----------',\n",
       " 'Synechococcus_sp._WH_8109': 'PKISKRLALKIEDRYPLEAILVKDNANAKFETMEAHVRLGIDPKYTDQQLRTTVALPNGTGQVRIAVVTRGEKVEAAGAELAGEEDLVESISKGMDFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLEAIESKAGKLEFRADRTGIVHVFGKASFSDALLNLKTLQETIDRNKPSGSKGRYWKSYVTSTMGPSVEVDFSALQDIEMSIGILGKKLGMSQFFDEQGRAVPVTLIEAGPCRITQLKNDDTDGYSAVQIGFGETREKLINKPAQGHLNKSGEGLLRHLREYRVDSVEGLEIGGAITVGDFEAGQKVDVSGDTVGRGFAGYQKRHGFSRGPMTHGSKNHREPGSTGAGTTPGRIYPGKRMAGRYGGKKITTRGLTILKVDSEHNLLVVKGSVPGKPGALLNIRPALRVGAKGG-SLKYETIPKLKDLLTNIHEVPKVVKVTVNRGLGEAAANAKSLEASVNELAITGQKVVVTRAKKAIAGFKIRQGMPIGCAVTLRGDRMYAFLERLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQIIFPEISFDKIDAIRGMDITIVTTARSDEEGRALLREMGMPFQNMSRIGKNPVPVPEKVTVSLDGLTVKVKGPKGELERTLPEGVSVSQDNNTIVVSPTSTKRISR--ERHGLSRTLVANMIEGVSNGYSKALEIVGVGSRAQVKGKTLVVSAGYSHPVEMEAPEGITFKVENNTRVIVSGIDKELVGNEAAKVRAIRPPEPYKGKGIKYEGERILRKAGKSGKK---MAKKVVAVIKLALDAGKANPAPPVGPALGQHGVNIMMFCKEYNARTQDKAGYVIPVEISVFEDRSFTFITKTPPASVLITKAAKIQKGSGESAKGSVGSISRAQLEEIAKTKLPDLNCTSVESAMRIIEGTARNMGVSISDMKTLPSIDRWYVVDANQTLGRLATEVAVLRGKNNPFTPHLDTGDFVVIVNAEKIVSGRKQQKLYRRHSGRPGGMKVETFALQRIPERIVEAIKGMLPHNALGRQMFRKLKVYKGTHPHAQKPQLLNPSASMIQESYLTVADNSGAKRIQCIRVL-GTNRYAVGDIVAAVKDAPNMVKKSDVVKAVVVRTKMRRETGSIRFDDNAAVIINDKNPKGTRVFGPVARELRERSFTKIVSLAPEVISQISRKQQQKRHRRLRHIGTDRPRLAVFRSNHIYAQVIDDTLSASTVDELRTLASDASVGELVARAIAKGIQVVFDRGGNLYHGRIKALADAAREAGLQF---------------MKVRRVLDQIRGRSYRDALIMLEFMPYRSTGPITKVLRSAVANAEHNGLDPSLVISSATADGPMKRYRPRAQGRAYQIKKQTCHISIAVAAD--------------------------------------MAVVTLAE-----MMEAGAHFGHQTRRWNPKMSRYIY-CARNGVHIIDLVQTAVCMNNAYKWTRSAARSGKRFLFVGTKKQASEVVALEAARC-GASYVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEGAVLRRELERLQKYLGGLKNMRRLPDVVVLVDQRRESNAVLEARKL--DIPLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLILGRLADAINEGRH--GSNDQR-------GGD-----SEG-----------------MGHKINPTGLRLGITQE----------HRSRWYASSKNYPALLQEDD--RIRKFIHKK-Y-----GSAGISDVLIARKADQLEVELK-------TARPGVLVGRQG-----------------------------------------------------------------------SGIEDLR----------------------SGIQKTIG-------DSSRQVRINVVE-------------------------------------VERVDGDAFLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYATKVASTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLSEQAQ-----PMPV----------------GAAPRRR-ASRRPQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAAKRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSERQLVRYVKKARAGSTGLLKLLERLDNVCFRLGFGPTVPGARQLVNHGHVTVNGRVTDIASYQCKPGDVIAIRERSKKLAENLEFPGLANVPHLELDKKLAKVTAEREWVALEINELLVVEYYSRKVDVPAAAEGQQQQEQR-RRGQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVADGKKHLVKVPLTRHNSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSLLRTHKETAKERG--ISLEQIYSMSRRKRPIPDFNSRLATMMVVRLMQGKKSTARILSDAFIERTGDPLEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVSFSRARGRSMAKLAELMDAANEAGNAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHETTIPASRMTRSIAKVLQQEGFIESEGEGRTLVLALKYSGKHRPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARREGVGGEVLCYVY--SVVYWGTGRRKTSVARVRLVPGNGTIINGRGDYLNYNPYIVKAPLETLGLSSEYDILVNVGGGLTGQSGAIKQGAARALCELSADNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAKKKSGKKAKRNVPNGVAIQSTFNNTIVSITDGEVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALDQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTIQQLIRERKAKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIGGVGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKAPKMARIAGVDIPRDKRIEVSLTYIYGVGLTRSQILTGVNPDIRVKDLEDDLLRMEEYTLEGDLRREMALKRLDIGVRGRRHRMSLPVRGQRTRTNARTRRGRTVAGKKK--MSLDTEKQQLINHQTHGTDTGSAEVQVAMLSERINRLSSHLQNHDSSRQGLLKMIGRRKRLLSYMRSQRYATIKLGIRGMAVKERIGTVVSDKMEKTVVVAVESRFPHPIYQKTVSRTTRYKAHDEDNTCRVGDRVRITETRPMSRQKRWAIAEVLSHSPKA-----------',\n",
       " 'Synechococcus_sp._CC9605': 'PKISKRLALKIEDRYPLEAILVKDNANAKFETMEAHVRLGIDPKYTDQQLRTTVALPNGTGQVRIAVVTRGEKVEAAGAELAGEEDLVESISKGMDFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLEAIEFKAGKLEFRADRTGIVHVFGKASFSDALLNLKTLQETIDRNKPSGAKGRYWKSYVTSTMGPSVEVDFSALQDIEMSIGILGKKLGMSQFFDEQGRAVPVTLIEAGPCRITQLKNDDTDGYSAVQIGFGETREKLINKPAQGHLNKSGEGLLRHLREYRVDSVEGLELGGAITVGDFEAGQKVDVSGDTVGRGFAGYQKRHGFSRGPMTHGSKNHREPGSTGAGTTPGRIYPGKRMAGRYGGKKITTRGLTILKVDSEHNLLVVKGSVPGKPGALLNIRPALRVGANGG-SLKYETIPKLKDLLTNIHEVPKVLKVTVNRGLGEAAANAKSLEASVNELAITGQKVVVTRAKKAIAGFKIRQGMPIGCAVTLRGDRMYAFLERLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQIIFPEISFDKIDAIRGMDITIVTTARSDEEGRALLREMGMPFQNMSRIGKNPVPVPEKVTVSLDGLTVKVKGPKGELERTLPEGVSVSQDNNTIVVSPTSTKRISR--ERHGLSRTLVANMIEGVSNGYSKALEIVGVGSRAQVKGKTLVVSAGYSHPVEMEAPEGITFKVENNTRVIVSGIDKELVGNEAAKVRAIRPPEPYKGKGIKYEGERILRKAGKSGKK---MAKKVVAVIKLALDAGKANPAPPVGPALGQHGVNIMMFCKEYNARTQDKAGYVIPVEISVFEDRSFTFITKTPPASVLITKAAKIQKGSGESAKGSVGSISRAQLEEIAKTKLPDLNCTSIESAMRIIEGTARNMGVSISDMKTLPSIDRWYVVDANQTLGRLATEVAVLRGKTNPFTPHLDTGDFVVVVNAEKIVSGRKQQKLYRRHSGRPGGMKVETFALQRIPERIVEAIKGMLPHNALGRQMFRKLKVYKGTHPHAQKPQLLNPSASMIQESYLTVADNSGAKRIQCIRVL-GTNRYAVGDIVAAVKDAPNMVKKSDVVKAVVVRTKMRRETGSIRFDDNAAVIINDKNPKGTRVFGPVARELRERSFTKIVSLAPEVISQISRKQQQKRHRRLRHIGTDRPRLAVFRSNHIYAQVIDDTLSASTVDELRTLASDASVGELVARAMAKGIQVVFDRGGNLYHGRIKALADAAREAGLQFMTSSTPAQAGRFIRSVKVRRVLDQIRGRTYRDALIMLEFMPYRSTGPITKVLRSAVANAEHNGLDPSLVISSATADGPMKRYRPRAQGRAYQIKKQTCHISIAVAAD--------------------MLVVQVPGRWRRTRNPQAMAVVTLAE-----MMEAGAHFGHQTRRWNPKMSRYIY-CARNGVHIIDLVQTAVCMNNAYKWTRSAARSGKRFLFVGTKKQASEVVALEAARC-GASYVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEGAVLRRELERLQKYLGGLKNMRRLPDVVVLVDQRRESNAVLEARKL--DIPLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLILGRLADAINEGRH--GSNDQR-------GGDSEG----------------------MGHKINPTGLRLGITQE----------HRSRWYASSKNYPALLQEDD--RIRKFIHKK-Y-----GSAGISDVLIARKADQLEVELK-------TARPGVLVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTIG-------DSSRQVRINVVE-------------------------------------VERVDGDAFLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYATKVASTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLSEQAQ-----PMPV----------------GAAPRRR-ASRRPQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAAKRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSERQLVRYVKKARAGSTGLLKLLERLDNVCFRLGFGPTVPGARQLVNHGHVTVNGRVTDIASYQCKPGDVIAIRERSKKLAENLEFPGLANVPHLELDKKLAKVTAEREWVALEINELLVVEYYSRKVDVPAAAEGQ-QQQEQ-RRGQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVADGKKHLVKVPLTRHNSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSLLRTHKETAKERG--ISLEQIYSMSRRKRPIPDFNSRLATMMVVRLMQGKKSTARILSDAFIERTGDPLEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVSFSRARGRSMAKLAELMDAANEAGNAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHETTIPASRMTRSIAKVLQQEGFIESEGEGRTLVLALKYSGKHRPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARREGVGGEVLCYVY--SVVYWGTGRRKTSVARVRLVPGNGTIINGRGDYLNYNPYIVKAPLETLGLSSEYDILVNVGGGLTGQSGAIKQGAARALCELSADNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAKKKSGKKAKRNVPNGVAIQSTFNNTIVSITDGEVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALDQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTIQQLIRERKAKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIGGVGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKAPKMARIAGVDIPRDKRIEVSLTYIYGVGLTRSQILTGVNPDIRVKDLEDDLLRLENYTVEGDLRREMALKRLDIGLRGRRHRMSLPVRGQRTRTNARTRRGRTVAGKKK--MSLDTEKQQLINHQTHGTDTGSAEVQVAMLSERINRLSRHLQNHDSSRQGLLKMIGRRKRLLSYMRSQRYATIKLGIRGMAVKERIGTVVSDKMEKTVVVAVESRFPHPIYQKTVSRTTRYKAHDEDNTCRVGDRVRITETRPMSRQKRWAIAEVLSHSPKA-----------',\n",
       " 'Synechococcus_sp._KORDI-52_(genome_sequencing)': 'PKISKRLALKIEDRYPLEAILVKDNANAKFETMEAHVRLGIDPKYTDQQLRTTVALPNGTGQVRIAVVTRGEKVEAAGAELAGEEDLVESISKGMDFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLEAIEFKAGKLEFRADRTGIVHVFGKASFSDALLNLKTLQETIDRNKPSGAKGRYWKSYVTSTMGPSVEVDFSALQDIEMSIGILGKKLGMSQFFDEQGRAVPVTLIEAGPCRITQLKNDDTDGYSAVQIGFGETREKLINKPAQGHLNKSGEGLLRHLREYRVDSVEGLELGGAITVGDFEAGQKVDVSGDTVGRGFSGYQKRHGFSRGPMTHGSKNHREPGSTGAGTTPGRIYPGKRMAGRYGGKKITTRGLTILKVDSEHNLLVVKGSVPGKPGALLNIRPALRVGAKGG-SLKYETIPKLKDLLTNIHEVPKVVKVTVNRGLGEAAANAKSLEASVNELAITGQKVVVTRAKKAIAGFKIRQGMPIGCAVTLRGDRMYAFLERLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQIIFPEISFDKIDAIRGMDITIVTTARSDEEGRALLREMGMPFQNMSRIGKNPVPVPEKVTVALDGLTVKVKGPKGELERTLPEGVSVSQDNNTVVVSPTSTKRISR--ERHGLSRTLVANMIEGVNNGYSKALEIVGVGSRAQVKGKTLVVSAGYSHPVELEAPEGITFKVENNTKVIVSGIDKELVGNEAAKVRAIRPPEPYKGKGIKYEGERILRKAGKSGKK---MAKKVVAVIKLALDAGKANPAPPVGPALGQHGVNIMMFCKEYNARTQDKAGYVIPVEISVFEDRSFTFITKTPPASVLITKAAKIQKGSGESAKGSVGSISRAQLEEIAKTKLPDLNCTSVESAMRIIEGTARNMGVSISDMKTLPSIDRWYVVDANQTLGRLATEVAVLRGKNNPFTPHLDTGDFVVVVNAEKIVSGRKQQKLYRRHSGRPGGMKVETFALQRIPERIVEAIKGMLPHNALGRQMFRKLKVYKGTHPHAQKPQLLNPSASVIQESYLTVADNSGAKRIQCIRVL-GTNRYAVGDIVAAVKDAPNMVKKSDVVKAVVVRTKMRRETGSIRFDDNAAVIINDKNPKGTRVFGPVARELRERSFTKIVSLAPEVISKLSRKQQQKRHRRLRHIGTDRPRLAVFRSNHIYAQVIDDTLSASTVDELRTLGSDASVGELVARAIAKGIQVVFDRGGNLYHGRVKALADAAREAGLQFMTSSTPAQAGRFIRSVKVRRVLDQIRGRTYRDALIMLEFMPYRSTGPITKVLRSAVANAEHNGLDPSLVISSATADGPMKRYRPRAQGRAYQIKKQTCHISIAVAAD--------------------------------------MAVVTLAE-----MMEAGAHFGHQTRRWNPKMSRYIY-CARNGVHIIDLVQTAVCMNNAYKWTRSAARSGKRFLFVGTKKQASEVVALEAARC-GASYVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEGAVLRRELERLQKYLGGLKNMRRLPDVVVLVDQRRESNAVLEARKL--DIPLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLILGRLADAINEGRH--GSNDQR-------GGDSEG----------------------MGHKINPTGLRLGITQE----------HRSRWYASSKNYPALLQEDD--RIRKFIHKK-Y-----GSAGISDVLIARKADQLEVELK-------TARPGVLVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTVG-------DSSRQVRINVVE-------------------------------------VERVDGDAFLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYATKVASTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLSEQAQ-----PMPV----------------GAAPRRR-ASRRPQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAAKRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSERQLVRYVKKARAGSTGLLKLLERLDNVCFRLGFGPTVPGARQLVNHGHVTVNGRVTDIASYQCKPGDVIAIRERSKKLAENLEFPGLANVPHLELDKKLAKVTGEREWVALEINELLVVEYYSRKVDVPAAAEGQQQQEQR-RRGQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVADGKKHLVKVPLTRHNSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSLLRTHKETAKERG--ISLEQIYSMSRRKRPIPDFNSRLATMMVVRLMQGKKSTARILSDAFIERTGDPLEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVSFSRARGRSMAKLAELMDAANEAGNAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHETTIPASRMTRSIAKVLQQEGFIESEGEGRTLVLALKYSGKHRPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARREGVGGEVLCYVY--SVVYWGTGRRKTSVARVRLVPGNGTIINGRGDYLNYNPYLVKAPLETLGLSSEYDILVNVGGGLTGQSGAIKQGAARALCELSADNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMPKKKSGKKAKRNVPNGVAIQSTFNNTIVSITDGEVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALDQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTIQQLIRERKAKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIGGVGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKAPKVARIAGVDIPRDKRIEVSLTYIYGVGLTRSQILTGVNPDIRVKDLEDDLLRMEEYTLEGDLRREMALKRLDIGLRGRRHRMSLPVRGQRTRTNARTRRGRTVAGKKK--MSLDTEKQQLINHQTHGTDTGSAEVQVAMLSERINRLSSHLQNHDSSRQGLLKMIGRRKRLLSYMRSQRYATIKLGIRGMAVKERVGTVVSDKMEKTVVVAVESRFPHPIYQKTVSRTTRYKAHDEDNTCRVGDRVRITETRPMSRQKRWAIAEVLSHSPKA-----------',\n",
       " 'Synechococcus_sp._WH_8102': 'PKLSKRLALKIEDRYPLEAILVKDNATAKFETMEAHVRLGIDPKYTDQQLRTTVALPNGTGQVRIAVVTRGEKVEAAGAELAGDEDLVETIAKGMEFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFTEALLNLKTLQETIDRNKPSGAKGRYWKSYVTSTMGPSVEVDFSALQDIEMSIGILGKKLGMSQFFDEQGRAVPVTLIEAGPCRITQLKSSDTDGYAAVQIGFGDTREKLINKPAKGHLNKTGETLLRHLREYRVDGLDGFELGGSITVGDFEAGQKVDVSGDTIGRGFSGYQKRHGFSRGPMTHGSKNHREPGSIGPGTTPGRIYPGKRMAGRYGGKKITTRGLTILKVDSERNLLVVKGSVPGKPGALLNIRPALRVGAKGG-SLKYETIPKLKDLLTNIHEVPKVVKVTVNRGLGEAAANAKSLEASVNELAITGQKVVVTRAKKAIAAFKIRQGMPIGCAVTLRGDRMYAFLERLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQIIFPEISFDKIDAIRGMDITIVTTARSDEEGRALLREMGMPFQNMSRIGKNPVPVPDKVTVSLDGLTVKVKGPKGELERTLPDGVSVSQDNNCIVVAPSTSKRFSR--ERHGLCRTLVANMIEGVNNGYSKSLEIVGVGSRAQVKGKTLVVSAGYSHPVEMEPPEGITFKVENNTKVIVSGIDKELVGNEAAKVRAIRPPEPYKGKGIKYEGERIMRKAGKSGKK---MAKKVTAVIKLALQAGKANPAPPVGPALGQHGVNIMMFCKEYNARTQDKAGFVIPVEISVYEDRSFTFITKTPPASVLITKAAKIEKGSGESAKGNVGSINRAQLEEIAKTKLPDLNCTSVESAMRIIEGTARNMGVSISDMKTLPSLERWYLVDANKTLGRLATEVAVLRGKNNPFTPHLDTGDFVVVVNAEKIVSGSKTQKLYRRHSGRPGGMKTETFALQRIPERIVEAIKGMLPHNALGRQMFRKLKVYKGSHPHAQKPQLLNPSASMIQESYLSVADNSGAKRIQCIRVL-GTNRYAVGDIVAAVKDAPNMVKKSDVVKAVVVRTKLRRDTGSIRFDDNAAVIINDNNPKGTRVFGPVARELRERNFTKIVSLAPEVISKLSRKQQQKRHRRLRHLGTDRPRLAVFRSNHIYAQVIDDTLSASTVDELRALGSDASVGELVARAIAKGISVVFDRGGNLYHGRIKALADAAREAGLQFMTTSSTAQAGRFIRSVKVRRVLDQIRGRTYRDALIMLEFMPYRSTGPITKVLRSAVANAEHNGLDPALVISSASADGPMKRYRPRAQGRAFQIKKQTCHISIAVAAD--------------------------------------MAVVTLAE-----MMEAGAHFGHQTRRWNPKMSRYIY-CARNGVHIIDLVQTAVCMNNAYKWTRSAARSGKRFLFVGTKKQASEVVALEAARC-GAAYVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEGAVLRRELERLQKYLGGLKTMRRLPDVVVLVDQRRESNAVLEARKL--DIPLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLVLGRLADAINEGRH--GNNEQR-------GGG---DAEG------------------MGHKINPTGLRLGITQE----------HRSRWYASSKSYPALLQEDD--RIRKFIHKK-Y-----GSAGISDVLIARKADQLEVELK-------TARPGVLVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTVG-------DRSRQVRINVVE-------------------------------------VERVDGDAFLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYATKVASTT-------------------------------------------YGVLGIKVWVF-----------------KGEV-LGDEA----PLIPV----------------GASPRRR-ASRRPQQFEDRSNEGMARYRGPRLRIRRLGDLPGLTRKAAKRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSERQLVRYVKKARAGSTGLLKLLERLDNVCFRLGFGPTVPGARQLVNHGHVTVNGRVTDIASYQCKPGDVVAIRERSKQLAENLEFPGLANVPHLELDKKLAKVTGEREWVALEINELLVVEYYSRKVDVPAAAEGQ-QQEQR-RRNQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVADGKKHLVKVPLTRHNSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSLLRTHKETAKERG--ISLEQIYSMSRRKRPVPDFNSRLATMMVSRLMQGKKSTARILSDAFIERTGDPLEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVSFSRARGRSMAKLAELMDAANEAGSAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHETTIPASRMTRSIAKVLQQEGFIESEGEGRTLVLSLKYSGKHRPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARREGVGGEVLCYVY--SVVYWGTGRRKTSVARVRLVPGNGTIINGRGDYLNYNPYIVKAPLETLGLGTEYDILVNVGGGLTGQSGAIKQGAARALCELSADNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAKKKTGKKAKRNVPNGVAIQSTFNNTIVSITDGEVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALDQGMRQIEVLVKGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTIQQLIRERKAKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIGGVGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKTPKMARIAGVDIPRDKRVEVSLTYIYGVGPTRARILTGVSPDIRVKDLEDDLLRADDFTLEGDLRREMALKRLDIGVRGRRHRMSLPVRGQRTRTNARTRRGRTVAGKKK--MSLDTEKQQLINHQTHGTDTGSAEVQVAMLSERISRLSSHLQNHDSSRQGLLKMIGRRKRLLSYMRSQRYTIIKLGIRGMAVKERVGTVVSDKMEKTVVVAVESRFPHPIYQKTVSRTTRYKAHDEDNSCRVGDRVRITETRPMSRHKRWAIAEVLSHSPKADKSAGSTAPAP',\n",
       " 'Synechococcus_sp._KORDI-100_(genome_sequencing)': 'PKLSKRLALKIDDRYPLEAILVRENATAKFETMEAHVRLGIDPKYTDQQLRTTVALPNGTGQVRIAVVTRGEKVEAAGAELAGEEELVETIAKGMDFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEALLNLKTLQETIDRNKPSGAKGRYWKSYVTSTMGPSVEVDYSALQDIQMSIGILGKKLGMSQFFDEQGKAVPVTLIEAGPCRITQLKSNDTDGYAAVQIGFGDTREKLINRPAKGHLSKSGDELLRHLREYRVDAVDGLELGGAITVGDFETGQKVDVSGDTMGRGFAGYQKRHGFSRGPMTHGSKNHREPGSTGAGTTPGRIYPGKRMAGRFGGKKITTRGLTILKIDSDRNLLVVKGSVPGKPGSLLNIRPAKRVGDKGG-SLKYETIPKLKDLLKNIHEVPKVLKVTVNRGLGEAAQNAKSLEASVGELAITGQKVVVTRAKKAIAGFKIRQGMPIGCAVTLRGDRMYAFLERLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEISFDKIDAIRGMDITIVTTARNDEEGRALLREMGMPFRNMSRIGKNPVPVPDKVTVSLDGLTIKVKGPKGELQRTLPDGVSVSQVDNTIVVKPSTDKRFSR--ERHGLCRTLVANMVEGVSKGYSKSLEIVGVGSRAQVKGKKLVVSAGYSHPVEVDPPEGITFAVENNTKVTVSGTDKELVGNEAAKVRSIRPPEPYKGKGIRYEGERILRKAGKSGKK---MAKKVTAVIKLALQAGKANPAPPVGPALGQHGVNIMMFCKEYNARTQDKAGYVIPVEISVYEDRSFTFITKTPPASVLITKAAGIQKGSGQSAKGSVGSIKRAQLEEIAKTKLPDLNCTSIESAMRIIEGTARNMGVSISDMKTLPSIERWYLVDANQTLGRLATEVAVLRGKNNPFTPHLDTGDFVVVVNADKVVSGKKQQKLYRRHSGRPGGMKTETFALQRIPERIVEAIKGMLPHNALGRQMFRKLKVYKGSHPHAQKPQLLNPSASVIQESFLTVADNSGAKRIQCIRVL-GTNRYAVGDIVAAVKDAPNMVKKSDVVKAVVVRTKLRRDTGSIRFDDNAAVIINDKNPKGTRVFGPVARELRDRNFTKIVSLAPEVISSLSRKQQQKRHRRLRDLGTSRPRLAVFRSNHIYAQLIDDTLSASTVDELRALGCDASVGELVARAIAKGIQVVFDRGGNLYHGRIKALANAAREAGLQFMTTSTTAKAGRFIRSVKVRRVLDQIRGRTYRDALIMLEFMPYRSTGPITKVLRSAVANAEHNGLDPALVISSASADGPMKRYRPRAQGRAYQIKKQTCHISIAVAAD--------------------------------------MAVVTLAE-----MMEAGAHFGHQTRRWNPKMSRYIY-CARNGVHIIDLVQTAVCMNNAYKWTRSAARSGKRFLFVGTKKQASEVVALEAARC-GAAYVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEGAVLRRELERLQKYLGGLKTMRRLPDVVVLVDQRRESNAVLEARKL--DIPLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLVLGRLADAINEGRH--GHHDQR-------GGD---DADG------------------MGHKINPTGLRLGITQE----------HRSRWYAPSKSYPALLQEDD--RIRKFIHKK-Y-----GSAGISDVLIARKADQLEVELK-------TARPGVLVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTVG-------DHSRQVRINVVE-------------------------------------VERVDGDAFLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYATKVASTT-------------------------------------------YGVLGIKVWVF-----------------KGEV-LGDEA----PPMPV----------------GASPRRR-ASRRPQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAAKRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGISERQLVRYVKKARAGSTGLLKLLERLDNVCFRMGFGPTVPGARQLVNHGHVTVNGRVTDIASYQCKVGDVIAIRERSKRLAENLEFPGLANIPHLDLDKKLAKVTGEREWVALEINELLVVEYYSRKVDVPAAAEGQ-QQEQR-RRGQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVADGKKHLVKVPLTRHNSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSLLRTHKETAKERG--ISLEQIYSMSRRKRPVPDFNSRLATMMVARLMKGKKSTARILSDAFIERTGDPLEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVSFSRSRGRSMAKLAELMDAANESGSAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHERTIPASRMTRSIAKVLQQEGFIESEGEGRTLVLALKYSGKHRPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARREGVGGEVLCYVY--SVVYWGTGRRKTSVARVRLVPGSGTIINGRGDYLNYNPYIVKAPLETLGLSTEYDVLVNVGGGLTGQAGAIKQGAARALCELSADNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAKKKSGKKAKRNVPNGVAIQSTFNNTIVSITDGEVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALDQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTIQQLIRERKAKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIGGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKAPKLARIAGVDIPRDKRIEVALTYIYGIGPTRAKILTGVNPDTRVKDLEDDVLRTEGYTIEGDLRREMALKRLDIGLRGRRHRMSLPVRGQRTRTNARTRRGRTVAGKKK--MSLDTEKQQLINHQTHGTDTGSPEVQVAMLTERISRLSSHLQNHDSSRQGLLKMIGRRKRLLSYMRSQRYSLIKLGIRGMALKERVGTVVSDKMEKTVVVAVESRFPHPIYQKTVSRTTRYKAHDEGNSCRVGDRVRITETRPMSRHKRWAIAEVLSHSPKA-----------',\n",
       " 'Synechococcus_sp._KORDI-49_(genome_sequencing)': 'PKISKRVALKIEDRYPLEAILVRENATAKFETMEAHVRLGIDPKYTDQQLRTTVALPNGTGQVRIAVVTRGEKVEAAGAELAGDEDLVETIAKGMNFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFTEALLNLKTLQETIDRNKPSGAKGRYWKSYVTSTMGPSVEVDFSALQDIQMSIGILGKKLGMSQFFDEQGRAVPVTLIEAGPCRITQLKSDDTDGYAAVQIGFGETREKLINKPSKGHLSKSGDDLLRHLREYRVDAVDGLELGGSITVSDFEAGQKVDVSGDTIGRGFSGFQKRHGFSRGPMTHGSKNHREPGSIGPGTTPGRIYPGKKMAGRYGGKKITTRGLTILKVDSDRNLLVVKGSVPGKPGALLNIRPAVRVGANGG-SLKYETIPKLKDLLTNIHEVPKVIKVTVNRGLGEAAQNAKSLEASVNELAITGQKVVVTRAKKAIAGFKIRQGMPIGCAVTLRGDRMYAFLERLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQIIFPEISFDKIDAIRGMDITIVTTARSDEEGRALLREMGMPFRNMSRIGKNPVPLPDKVTVSLDGLTVKVKGPKGELERTLPEGVSVSQEENTIVVTPSTTKRFSR--ERHGLCRTLVANMVEGVSNGYSKSLEIVGVGSRAQVKGKTLVVSAGYSHPVEMEPPDGITFKVENNTKVIVSGIDKELVGNEAAKVRAIRPPEPYKGKGIKYEGERILRKAGKSGKK---MAKKVTAVIKLALQAGKANPAPPVGPALGQHGVNIMMFCKEYNARTQDKAGFVIPVEISVYEDRSFTFITKTPPASVLITKAAKIDKGSGESAKGNVGSINRAQLEEIAKTKLPDLNCNSVESAMRIIEGTARNMGVSISDMKTLPSIDRWYVVDANQTLGRLATEVAVLRGKNNPFTPHLDTGDFVIVVNAEKIVSGRKQQKLYRRHSGRPGGMKVETFALQRIPERIVEAIKGMLPHNALGRQMFRKLKVYKGSHPHAQQPQLLNPSASVIQETFLTVADNSGAKRIQCIRVL-GTNRYAVGDIVAAVKDAPNMVKKSDVVKAVVVRTKLRRDTGSIRFDDNAAVIINDKNPKGTRVFGPVARELRDRSFTKIVSLAPEVISTTSRKQQQKRHRRLRHLGTNRPRLAVFRSNHIYAQVINDTLSASTVDELRALGSDASVGELVARAIAQGIQVVFDRGGNLYHGRIKALADAAREAGLQFMTTSTPAQAGRFIRSVKVRRVLDQIRGRTYRDALIMLEFMPYRSTGPITKVLRSAVANAEHNGLDPALVISSASADGPMKRYRPRAQGRAYQIKKQTCHISIAVAAD--------------------------------------MAVVTLAE-----MMEAGAHFGHQTRRWNPKMSRYIY-CARNGVHIIDLVQTAVCMNNAYKWTRSAARSGKRFLFVGTKKQASEVVAQEAARC-GAAFVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEGAVLRRELERLQKYLGGLKSMRRLPDVVVLVDQRRESNAVLEARKL--DIPLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLVLGRLADAINEGRH--GNNDQR-------GGD---DGEA------------------MGHKINPTGLRLGITQE----------HRSRWYAPSKSYPALLQEDD--RIRKFIHKK-Y-----GSAGISDVLIARKADQLEVELK-------TARPGVLVGRQG-----------------------------------------------------------------------SGIEDLR----------------------SGIQKTVG-------DQSRQVRINVVE-------------------------------------VERVDGDAFLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYATKVASTT-------------------------------------------YGVLGIKVWVF-----------------KGEV-LGDEA----PQIPV----------------GATPRRR-ASRRPQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAAKRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSERQLVRYVKKARAGSTGLLKLLERLDNVCFRLGFGPTVPGARQLVNHGHVMVNGRVTDIASYQCKTGDVIAIRERSKKLAENLEFPGLANIPHMELDKKLAKIVGEREWVALEINELLVVEYYSRKVDVPAAAEGQ-QQEQR-RRNQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVADGKKHLVKVPLTRHSSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSLLRTHKETAKERG--ISLEQIYSMSRRKRPIPDFNSRLASMMVARLMQGKKSTARILSDAFIERTGDPLEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVSFSRSRGRSMAKLAELMDAANEAGSAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHETTIPASRMTRSIAKVLQQEGFIESEGEGRTLVLALKYSGKHRPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARREGVGGEVLCYVY--TVVYWGTGRRKTSVARVRLVPGNGTIINGRGDYLNYNPYIVKAPLETLGLGSEYDVLVNVGGGLTGQAGAIKQGAARALCELSADNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAKKKSGKKAKRNVPNGVAIQSTFNNTIVSITDGEVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALEQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTIQQLIRERKAKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIGGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDTSGVKDRRQSRSKYGAKTPKVARIAGVDIPRDKRIEVALTYIYGIGPTRARILSGVNPDIRVKDLEDDVLRTEAYTIEGDLRREMALKRLDIGVRGRRHRMSLPVRGQRTRTNARTRRGRTVAGKKK--MSLDTEKQQLINHQTHGTDTGSAEVQVAMLSERINRLSGHLQNHDSSRQGLLKMIGRRKRLLSYVRSQRYSIIKLGIRGMAVKERVGTVVSDKMEKTVVVAVESRFPHPIYQKTVSRTTRYKAHDEDNTCRVGDRVRITETRPMSRHKRWTIAEVLSHSPKA-----------',\n",
       " 'Prochlorococcus_sp._MIT0801': 'KNFSKRMTLKVEERYPIEAILVKENANAKFETIEAHIRLGIDPKYTDQQLRTTVALPSGTGQIRIAVITRGEKVEAAGADLAGEEDLVDSINKGMNFDLISTPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLEAIEFKAGKLEFRADKAGIVHVFGKASFSDALLNLKTLQATIEKNKPSGAKGKFWRSFITSTMGPSIEVDINELQDLQMSLGILGKKLGMSQLFDDQGRAVPVTLIEAGPCRITQLKSADTDGYASVQIGFQLIREKLINKPSKGHLAKSGNDLLRHLREYRVENSSEFELGASITVDDFEKGQKVDVSGDTMGRGFAGYQKRHGFSRGPMSHGSKNHRLPGSIGAGTTPGRVYPGKRMAGRMGGKKVTTRGLEILKIDTNHNLLVVKGSVPGKPGSLLNIRPAKRVGQQGG-SLKYETIPKLKDLLKNVHQVPKVQKVTLNRGLGEAATNSKALEASLNEMAISGQKALVTRAKKAIATFKIRQGMPIGCAVTLRGDRMYAFLERFINLALPRIRDFRGVSPKSFDGRGNYTIGVKEQLIFPEITFDKVDTIRGMDITIVTSASSDEQGKALLTEMGMPFRKMSRTGKKPISLPEKVDVKLEGLSITVKGPKGELQRTLPNGVSLSKEENYIFVNPINEKRQSR--EMHGLCRSLVANMVEGVSNGFTKKLEIIGVGSRAQVKGNTLVVSAGYSHPVEVVPPEGITFKVENNTNVIVTGPDKELVGNEAAKIRAIRPPEPYKGKGIKYEGEQIIRKAGKSGKT---MAKKVVALIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNSRTQDKAGFVIPVEISVFEDRSFTFITKTPPASVLITKAAGIAKGSGESAKGSAGSISTSQLEEIAKTKLPDLNCSSIESAMKVIEGTAKNMGVSIKDMKTVPSIDRWFLVDANQTLGRLATEVAVLRGKTKPFTPHLDTGDFVIVVNAEKIVTGKKDQKLYRRHSGRPGGMKVETFALQRIPERIVEAIKGMLPHTRLGRQLFTKLKVYKGSHPHAQEPKLLNSESVMIQETFLTVADNSGAKRLQCIRVL-GSNRYAVGDIVASVKDAPNMVKKSDVVKAVVVRTRMRRETGSIRFDDNAAVLINDQNPRGTRVFGPVARELRERNFTKIVSLAPEVISKLSRKQQQKRHKRLRNLGTSRPRLAVFRSNHIYAQIIDDTIAASTLDDLKALASDASVGELVAKALSKGIQVIFDRGGNIYHGRVKALADAARVAGLNF-MTNSSAQAGRYIRSAKVRRVLDQIRGRTYRDALIMLEFMPYRSTEPITKVLRSAVANAENNGLDPSLMISTATADGPMKRYRPRAQGRAFAIKKQTCHISIAVSAS--------------------------LGGGEPGTHKKNMAVVSLSE-----MMEAGAHFGHQTRRWNPKMSRYIF-SARNGVHIIDLVKTAVCMNSAYKWTRGAARSGKRFLFVGTKKQASEVVAQEAIRC-GASYVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEGAVLRRELERLQKYLGGLKGMRRLPDVVILVDQRRETNAVLEARKL--DIPLVSMLDTNC--DPDLCEIPIPCNDDAVRSVQLVLGRLADAINEGRH--GPNE-------------------------------------MGHKIHPTGIRLGITQE----------HRSKWYAPSKTYPLLLQEDD--RIRTFIKKK-Y-----GAAGISDVLIARKADQLEVELK-------TARPGVIVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTIG-------DRNRQVRINVVE-------------------------------------VEKVDADAYLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVGG-----------------------------RLNGAEIARSE-----WTREG---RVPLHTLRAEIDYATKVATTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLPKEEQ-----PLPV----------------GSSPRRR-GNRRPPQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAAKRSHPPGQHGQRRKRSEYAIRLEEKQKLRFNYGISERQLVRYVKKARAGSTGLLKLLERLDNVCFRLGFGPTIPGSRQLVNHGHVTVNGKITDIASYQCKAGDVIAIRDNSKQLAQNLEFPGLANVPHLELDKKLAKISADREWVAIEINELLVVEYYSRKVDATPAADGRENRRN--RRGQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVADGKKHLVRVPLTRNSSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSELRTHKATAKERG--ISLEQIYSMSRRKRPVPDFNNRLASMMVHRLMKGKKSTAKILSDAFIERTGDPIEFETAVKNVTPLVEVRARRVGGATYQVPMEVRERGIAMALRWLVNFSRSRGRSMSKLAELMDAANEAGNAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHEKTVPASKMSLSIAKVLQREGFIENEGEGRKLILGLKYTGKHRPIISMQRVSKPGLRIYKNRLPKVLGGLGIAIISTSNGVMSDRDARKQGVGGEVLCYVCM-KVVYWGTGRRKTSVARVRLTPGKGEIINGRGDYLNFNPYIVKAPLKTLGLSESYDVLVNVGGGLTGQSDAIKQGAARALCTLSLDNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMATKKSGKKSKRNVPNGVVIQSTFNNTIVSITDGEVISWSSAGASGFKGARKGTPFAAQTAAELAARRALEQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTIQQLIRERKTKTKSPALRCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIGGIGHNLQEHSVVLLRGGRVKDLPGVRYHIVRGTLDTAGVKDRKQSRSKYGAKSPKVARISGIDIPREKRVEVALTYIYGIGLTRAQILSGVNPDIRVKDLEDDILRTEEFTVEGDLRREMALKRLDIGVRGRRHRMSLPVRGQRTRTNARTRRGRTVAGRKK--MSLSTEKQNLINHQVHPTDTGSVEVQVAMLTTRISKLSTHLQNHDSSRQGLLKMIGQRKRLLGYVKSKRYTLIKLAIRGMALKEMVGTVVSDKMQKTVVVAVENRFPHPIYQKIISRTTRYKAHDAENHCKVGDRVRIKESPPISAHKRWTVTDVLVQGMKS-----------',\n",
       " 'Prochlorococcus_marinus_str._NATL2A': 'KNFSKRMTLKVEERYPIEAILVKENANAKFETIEAHIRLGIDPKYTDQQLRTTVALPSGTGQIRIAVVTRGEKVEAAGADLAGEEDLVDSINKGMNFDLISTPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLEAIEFKAGKLEFRADKAGIVHVFGKASFSEALLNLKTLQTTIEKNKPSGAKGKFWRSFITSTMGPSVEVDINELQDLQMSLGILGKKLGMSQLFDDQGRAVPVTLIEAGPCRITQLKSADTDGYAAVQIGFQLIREKLINKPSKGHLAKSGNDLLRHLREYRVENSSEFELGASITVDDFEKGQKVDISGDTMGRGFAGYQKRHGFSRGPMSHGSKNHRLPGSIGAGTTPGRVYPGKRMAGRMGGKKVTTRALEILKIDTNHNLLVVKGSVPGKPGSLLNIRPAKRVGQQGG-SLKYETIPKLKDLLKNVHQVPKVQKVTLNRGLGEAATNSKALEASLKEMAISGQKALVTRAKKAIATFKIRQGMPIGCAVTLRGDRMYAFLERFINLALPRIRDFRGVSPKSFDGRGNYTIGVKEQLIFPEITFDKVDTIRGMDITIVTSASSDEQGKALLSEMGMPFRKMSRTGKKPISLPEKVDVKFEGLSITVKGPKGELKRTLPNGVSLSKDENFIFVKPINEKRQSR--EMHGLCRSLVANMVEGVSNGFTKKLEIVGVGSRAQVKGKTLVVSAGYSHPVEVVPPEGITFKVENNTNVIVTGPDKELVGNEAAKIRAIRPPEPYKGKGIKYQGELIIRKAGKSGKT---MAKKVVALIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNSRTQDKAGFVIPVEISVFEDRSFSFITKTPPASVLITKAAGIAKGSGESAKGSAGSISTSQLEEIAKTKLPDLNCSSIESAMKVIEGTAKNMGVSIKDMKTVPSIDRWFLVDANQTLGRLATEVAVLRGKTKPFTPHLDTGDFVIVVNAEKIVTGKKDQKLYRRHSGRPGGMKVETFALQRIPERIVEAIKGMLPHTRLGRQLFTKLKVYKGSHPHAQEPKLLNSESVMIQETFLTVADNSGAKRLQCIRVL-GSNRYAVGDIVASVKDAPNMVKKSDVVKAVVVRTRMRRETGSIRFDDNAAVLINDQNPRGTRVFGPVARELRERNFTKIVSLAPEVISKLSRKQQQKRHKRLRNLGTSRPRLAVFRSNHIYAQIIDDTIAASTLDDLKALASDASVGELVAKALSKGIQVIFDRGGNIYHGRVKALAEAARVAGLNF-MTTSSAQAGRYIRSAKVRRVLDQIRGRTYRDALIMLEFMPYRSTEPITKVLRSAVANAENNGLDPSLMISTATADGPMKRYRPRAQGRAFAIKKQTCHISISVSAS---------------------------------------------------MMEAGAHFGHQTRRWNPKMSRYIY-SARNGVHIIDLVKTAVCMNSAYKWTRGAARSGKRFLFVGTKKQASEVVAQEAIRC-GASYVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEGAVLRRELERLQKYLGGLKGMRRLPDVVVLVDQRRETNAVLEARKL--DIPLVSMLDTNC--DPDLCEIPIPCNDDAVRSVQLVLGRLADAINEGRH--GPNE-------------------------------------MGHKIHPTGIRLGITQE----------HRSKWYAPSKTYPLLLQEDD--RIRTFIKKK-Y-----LAAGISDVLIARKADQLEVELK-------TARPGVIVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTIG-------DRNRQVRINVVE-------------------------------------VEKVDADAYLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVGG-----------------------------RLNGAEIARSE-----WTREG---RVPLHTLRAEIDYATKVASTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLPKEEQ-----PLPV----------------GSSPRRR-GNRRPQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAAKRSHPPGQHGQRRKRSEYAIRLEEKQKLRFNYGISERQLVRYVKKARAGSTGLLKLLERLDNVCFRLGFGPTIPGSRQLVNHGHVTVNGRITDIASYQCKAGDVIAIRDNSKQLAQNLEFPGLANVPHLELDKKLAKISADREWVAIEINELLVVEYYSRKVSAPPAADGRQENRRS-RRGQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVADGKKHLVRVPLTRNSSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSELRTHKATAKERG--ISLEQIYSMSRRKRPVPDFNNRLASMMVHRLMKGKKSTAKILSDAFIERTGDPIEFETAVKNVTPLVEVRARRVGGATYQVPMEVRERGIAMALRWLVNFSRSRGRSMAKLAELMDAANEAGNAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHEKTVPASRMSLSIAKVLQSEGFIENEGEGRKLILGLKYTGKHRPIISMQRVSKPGLRIYKNRLPKVLGGLGIAIISTSNGVMSDRDARKQGVGGEVLCYVCM-KVVYWGTGRRKTSVARVRLTPGKGEIINGRGDYLNFNPYIVKAPLKTLGLSESYDVLVNVGGGLTGQSDAIKQGAARALCTLSLDNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMATKKSGKKSKRNVPNGVVIQSTFNNTIVSITDGEVISWSSAGASGFKGARKGTPFAAQTAAELAARRALEQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTIQQLIRERKTKTKSPALRCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIGGIGHNLQEHSVVLLRGGRVKDLPGVRYHIVRGSLDTAGVKDRRQSRSKYGAKSPKMARISGIDIPREKRVEVALTYIYGIGLTRAQILSGVNPDIRVKDLEDDILRTEEFTLEGDLRREMALKRLDIGVRGRRHRMSLPVRGQRTRTNARTRRGRTVAGRKK--MSLGTEKQNLINHQVHPTDTGSAEVQVAMLTTRISKLSTHLQNHDSSRQGLLKMIGQRKRLLGYVRSKRYTLIKLAIRGMALKEMVGTVVSDKMQKTVVVAVENRFPHPIYQKIISRTTRYKAHDAENHCKVGDRVRIKESPPISAHKRWTVTDVLVKGMKS-----------',\n",
       " 'Prochlorococcus_marinus_str._NATL1A': 'KNFSKRMTLKVEERYPIEAILVKENANAKFETIEAHIRLGIDPKYTDQQLRTTVALPSGTGQIRIAVVTRGEKVEAAGADLAGEEDLVDSINKGMNFDLISTPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDLEAIEFKAGKLEFRADKAGIVHVFGKASFSEALLNLKTLQTTIEKNKPSGAKGKFWRSFITSTMGPSVEVDINELQDLQMSLGILGKKLGMSQLFDDQGRAVPVTLIEAGPCRITQLKSADTDGYAAVQIGFQLIREKLINKPSKGHLAKSGNDLLRHLREYRVENSSEFELGASITVDDFEKGQKVDVSGDTMGRGFAGYQKRHGFSRGPMSHGSKNHRLPGSIGAGTTPGRVYPGKRMAGRMGGKKVTTRALEILKIDTNHNLLVVKGSVPGKPGSLLNIRPAKRVGQQGG-SLKYETIPKLKDLLKNVHQVPKVQKVTLNRGLGEAATNSKALEASLKEMAISGQKALVTRAKKAIATFKIRQGMPIGCSVTLRGDRMYAFLERFINLALPRIRDFRGVSPKSFDGRGNYTIGVKEQLIFPEITFDKVDTIRGMDITIVTSASSDEQGKALLSEMGMPFRKMSRTGKKPISLPEKVDVKFEGLSITVKGPKGELQRTLPNGVSLSKDENFIFVKPINEKRQSR--EMHGLCRSLVANMVEGVSNGFTKKLEIVGVGSRAQVKGKTLVVSAGYSHPVEVVPPEGITFKVENNTNVIVTGPDKELVGNEAAKIRAIRPPEPYKGKGIKYQGELIIRKAGKSGKT---MAKKVVALIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNSRTQDKAGFVIPVEISVFEDRSFSFITKTPPASVLITKAAGIAKGSGESAKGSAGSISTSQLEEIAKTKLPDLNCSSIESAMKVIEGTAKNMGVSIKDMKTVPSIDRWFLVDANQTLGRLATEVAVLRGKTKPFTPHLDTGDFVIVVNAEKIVTGKKDQKLYRRHSGRPGGMKVETFALQRIPERIVEAIKGMLPHTRLGRQLFTKLKVYKGSHPHAQEPKLLNSESVMIQETFLTVADNSGAKRLQCIRVL-GSNRYAVGDIVASVKDAPNMVKKSDVVKAVVVRTRMRRETGSIRFDDNAAVLINDQNPRGTRVFGPVARELRERNFTKIVSLAPEVISKLSRKQQQKRHKRLRNLGTSRPRLAVFRSNHIYAQIIDDTIAASTLDDLKALASDASVGELVAKALSKGIQVIFDRGGNIYHGRVKALAEAARVAGLNFMTTSSTAQAGRYIRSAKVRRVLDQIRGRTYRDALIMLEFMPYRSTEPITKVLRSAVANAENNGLDPSLMISTATADGPMKRYRPRAQGRAFAIKKQTCHISISVSAS---------------------------------------------------MMEAGAHFGHQTRRWNPKMSRYIY-SARNGVHIIDLVKTAVCMNSAYKWTRGAARSGKRFLFVGTKKQASEVVAQEAIRC-GASYVNQRWLGGMLTNWTTMKARIDRLKDLERMESSGAIAMRPKKEGAVLRRELERLQKYLGGLKGMRRLPDVVVLVDQRRETNAVLEARKL--DIPLVSMLDTNC--DPDLCEIPIPCNDDAVRSVQLVLGRLADAINEGRH--GPNE-------------------------------------MGHKIHPTGIRLGITQE----------HRSKWYAPSKTYPLLLQEDD--RIRTFIKKK-Y-----VAAGISDVLIARKADQLEVELK-------TARPGVIVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTIG-------DRNRQVRINVVE-------------------------------------VEKVDADAYLLAEYIAQQLEKRVAFRRTIRMAVQRAQRA-GVLGLKIQVGG-----------------------------RLNGAEIARSE-----WTREG---RVPLHTLRAEIDYATKVASTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLPKEEQ-----PLPV----------------GSSPRRR-GNRRPQQFEDRSNESMSRYRGPRLRIRRLGDLPGLTRKAAKRSHPPGQHGQRRKRSEYAIRLEEKQKLRFNYGISERQLVRYVKKARAGSTGLLKLLERLDNVCFRLGFGPTIPGSRQLVNHGHVTVNGKITDIASYQCKAGDVIAIRDNSKQLAQNLEFPGLANVPHLELDKKLAKITADREWVAIEINELLVVEYYSRKVGAPTAADGRQENRRS-RRGQ---ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVADGKKHLVRVPLTRNSSIPTLSNGRDGAASVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSELRTHKATAKERG--ISLEQIYSMSRRKRPVPDFNNRLASMMVHRLMKGKKSTAKILSDAFIERTGDPIEFETAVKNVTPLVEVRARRVGGATYQVPMEVRERGIAMALRWLVNFSRSRGRSMAKLAELMDAANEAGNAVRKREETHKMAEANKAFAHYRYMAHDPISDMLTRIRNARHEKTVPASRMSLSIAKVLQSEGFIENEGEGRKLILGLKYTGKHRPIISMQRVSRPGLRIYKNRLPKVLGGLGIAIISTSNGVMSDRDARKQGVGGEVLCYVCM-KVVYWGTGRRKTSVARVRLTPGKGEIINGRGDYLNFNPYIVKAPLKTLGLSESYDVLVNVGGGLTGQSDAIKQGAARALCTLSLDNRSLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMATKKTGKKSKRNVPNGVVIQSTFNNTIVSITDGEVISWSSAGASGFKGARKGTPFAAQTAAELAARRALEQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTIQQLIRERKTKTKSPALRCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIGGIGHNLQEHSVVLLRGGRVKDLPGVRYHIVRGTLDTAGVKDRRQSRSKYGAKSPKMARISGIDIPREKRVEVALTYIYGIGLTRAQILSGVNPDIRVKDLDDDILRTEEFTLEGDLRREMALKRLDIGVRGRRHRMSLPVRGQRTRTNARTRRGRTVAGRKK--MSLGTEKQNLINHQVHPTDTGSVEVQVAMLTTRISKLSTHLQNHDSSRQGLLKMIGQRKRLLGYVRSKRYTLIKLAIRGMALKEMVGTVVSDKMQKTVVVAVENRFPHPIYQKIISRTTRYKAHDAENHCKVGDRVRIKESPPISAHKRWTVTDVLVKGMKS-----------',\n",
       " 'Prochlorococcus_marinus_str._MIT_9215': 'KKLSKRMALKIEDRYPLEALIIKENANAKFETIEAHIRLGIDPKYTDQQLRTTVALPHGTGQIRIAVITSGENVKAAGADLFGEEDLVESINKGMEFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDIGAIEFKAGKLEFRADKAGIVHVFGKASFTEALFNLKTLQESIDKNKPSGAKGKYWKSYVTSTMGPSVQVDINAVQDYQMSIGILGKKLGMSQLFDDKGNAVPVTLIEAGPCRVTQLKTNALDGYTAIQIGYGVSKEKHISKPEKGHLLKSGEELLKHLKEYRVEETSSYEIGNQITVKNFEVGQKVDISGKSMGRGFSGYQKRHGFSRGPMSHGSKNHRAPGSTGAGTTPGRIYPGKRMAGRYGGKQITTKGLLVLKIDDQKNLLVVKGSVPGKPGSIVNIKPNNVVGGEKS-TLKYESIPKLKELLKNIHQVPKVVKVNVNRGLGEAASNSKALEASLNEMAITGQKALVTRAKKAIAGFKIREGMPIGCTVTLRGDRMYSFLERFINLALPRIRDFRGVNPKSFDGRGNYTVGVKEQLIFPEISFDKIDSIRGMDITIVTSARSDQEGKALLQELGMPFSNMSRIGKTPVLIPDKVTVDFDGLIVTVKGPKGELKRQMPEGVCFDKKDNTVVVSPTTSKIFSR--QRHGLCRALIANMVEGVSQGFSKKLEIVGVGSRAQVKGKNLVVSAGYSHPIEMIPPDGITYKVESNTNVTVSGIDKEIVGNEAAKIRSIRPPEPYKGKGIKYHDEIILRKAGKSGKK---MAKKIVAVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKAGFVIPVEISVFEDRSFTFITKTPPASVLITKAAGIEKGSGESAKGSVGNISKAQLEEIAKTKLPDLNCSSVESAMKVIEGTARNMGVSITDMKTTPTIERWFLVDANKTLGRLATEIALLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKSQKLYRRHSGRPGGMKIEKFSLKRIPERIIEAVKGMLPHNSLGRQQFKKLKVYKGAHPHAQNPVL------MIQETYLTVADNSGAKRLQCIRVL-GSNRYAVGDIVATVKDAPNMVKKSEVVKAVIVRTKLRRNTGSIRFDDNAAVLINDKNPKGTRVFGPVARELRDKNYTKIVSLAPEVITKLSRKLQQKRHRRLRFLGDTRPRLSVFRSNHIYAQVIDDTISASTVDELRESPDNSSVGKLLARAIKKGVQVIFDRGGNIYHGRVKALADAAREAGLEFMTKTSEAIAGNYIRSAKVRRVLDQIRGRSYRDALIMLEFMPYRSTDPITKVLRSAVANAEHNGMDPSLIISSAWANGPMKRYRPRAQGRAFSIKKQTCHISISVESN--------------------------------------MAVVSLSE-----MMEAGAHFGHQTRRWNPKMSKYIY-CARNGVHIIDLVKTALCMNNAYKWTRNAAKSGKRFLFVGTKKQASDVVAQEATRC-GAAYVNQRWLGGMLTNWTTMKARIERLKDLERMESSGSIAMRPKKEAAVLRRELERLQKYLGGLKGMRRLPDVVVLVDQRRESNAVLEARKL--DISLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLILGRLADAINEGRK--GSNAER--------KN-------------------------MGHKIHPSGLRLGITQE----------HRSKWFATSKTYPILLQEDF--KIRTFIEKK-Y-----GAAGISDVLIARKADQLELELK-------TARPGVIVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTIG-------DRTRQVRINVVE-------------------------------------VERVDADAFLLAEYIAQQLEKRVAFRRTIRMALQRAQRA-GVLGLKIQVGG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRAEIDYATREANTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLPKEEQ-----TIPV----------------GANPKRK-ASRRPQQFEDRSNENMSRYRGPRLRVRRLGELPGLTRKASKKSNPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSEKQLVRYVKKARAGSTGLLRLLERLDNVCFRLGFGGTIPGSRQLVNHGHVTVNGKVLDIAGYQCKSGDVIGIKENSKKLVENIEFPGLANVPHLDLDKKLGKINGDREWVALEINELLVVEYYSRKVNVPGAIPVEQKKNNR-SKNL---ERDSDWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVSDGKKNLVRVPLTPNNSIPTLSKGRDGAANVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSQLRTHKSASRERG--ISLEQLYSMSRRKRPVPDFNSRLASMMISRLMKGKKSTARILSDAFIERTGNAVEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVTFSRARGKSMSKLAELMDAANETGSAVKKREDTHKMAEANKAFAHYRYMSHDPISDMLTRIRNAKHTTTIPGSKMSLSIAKVLQKEGFIENEGEGKSIILGLKYSGKNKPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARKQGIGGEVLCYVYMKKAVYWGTGRRKTSVARVRLIPGNGLIINGRGDYLNFNPHLVKAPLQTLGLENSYDILVNVGGGLTGQADAIKQGAARALCELSPDNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAAKKTGKKSKRNVPNGVVIQSTFNNTIVSISDGHVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALDQGMRQIEVLVRGPGAGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTISQLVGERTKKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLLRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKAPKMARIAGIDIPREKRVEIALTYVYGIGLTRSKILTGVNPDTRVKDLSDDVLRTEEFTLEGDLRREMALKRLDIGVRGRRHRMSLPVRGQRTRTNARTRRGRTVAGRKK--MSLDTEKQKLIEHQVHPTDTGSAEVQVAMLSKRISKLSDHLQNHDSSRQGLLKMIGKRKRLLSYIKNQRYQLVKIGIRGMALKERIGTVVSDKMDKTVVVAVINRYPHPTYKKIVSRTTRYKAHDPENTCVTGDRVKIRETRPLSAQKRWAIEEILNKTNQA-----------',\n",
       " 'Prochlorococcus_marinus_str._MIT_9202_scf_1109798535001_genomic_scaffold': 'KKLSKRMALKIEDRYPLEALIIKENANAKFETIEAHIRLGIDPKYTDQQLRTTVALPHGTGQIRIAVITSGENVKAAGADLFGEEDLVESINKGMEFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDIGAIEFKAGKLEFRADKAGIVHVFGKASFTDALFNLKTLQESIDKNKPSGAKGKYWKSYVTSTMGPSVQVDINAVQDYQMSIGILGKKLGMSQLFDDKGNAVPVTLIEAGPCRVTQLKTNALDGYTAIQIGYGVSKEKHISKPEKGHLLKSGEELLKHLKEYRVEETSSYEIGNQITVKNFEVGQKVDISGKSMGRGFSGYQKRHGFSRGPMSHGSKNHRAPGSTGAGTTPGRIYPGKRMAGRYGGKQITTKGLLVLKIDDQKNLLVVKGSVPGKPGSIVNIKPNNVVGGEKS-TLKYESIPKLKELLKNIHQVPKVVKVNVNRGLGEAASNSKALEASLNEMAITGQKALVTRAKKAIAGFKIREGMPIGCTVTLRGDRMYSFLERFINLALPRIRDFRGVNPKSFDGRGNYTVGVKEQLIFPEISFDKIDSIRGMDITIVTSARSDQEGKALLQELGMPFSNMSRIGKTPVLIPDKVTVDFDGLIVTVKGPKGELKRQMPEGVCFDKKDNAVVVSPTTSKIFSR--QRHGLCRALIANMVEGVSQGFSKKLEIVGVGSRAQVKGKNLVVSAGYSHPIEMIPPDGITYKVESNTNVTVSGIDKEIVGNEAAKIRSIRPPEPYKGKGIKYHDEVILRKAGKSGKK---MAKKIVAVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKAGFVIPVEISVFEDRSFTFITKTPPASVLITKAAGIEKGSGESAKGSVGNISKAQLEEIAKTKLPDLNCSSVESAMKVIEGTARNMGVSITDMKTTPTIERWFLVDANKTLGRLATEIALLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKSQKLYRRHSGRPGGMKIEKFSLKRIPERIIEAVKGMLPHNSLGRQQFKKLKVYKGVHPHAQNPVL------MIQETYLTVADNSGAKRLQCIRVL-GSNRYAVGDIVATVKDAPNMVKKSEVVKAVIVRTKLRRNTGSIRFDDNAAVLINDKNPKGTRVFGPVARELRDKNYTKIVSLAPEVITKLSRKLQQKRHRRLRFLGDTRPRLSVFRSNHIYAQVIDDTISASTVDELRESPDNSSVGKLLARAIKKGVQVIFDRGGNIYHGRVKALADAAREAGLEFMTKTPEAIAGNYIRSAKVRRVLDQIRGRSYRDALIMLEFMPYRSTDPITKVLRSAVANAEHNGMDPSLIISSAWANGPMKRYRPRAQGRAFSIKKQTCHISISVESN--------------------------------------MAVVSLSE-----MMEAGAHFGHQTRRWNPKMSKYIY-CARNGVHIIDLVKTALCMNDAYKWTRNAAKSGKRFLFVGTKKQASDVVAQEATRC-GAAYVNQRWLGGMLTNWTTMKARIERLKDLERMESSGSIAMRPKKEAAVLRRELERLQKYLGGLKSMRRLPDVVVLVDQRRESNAVLEARKL--DISLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLILGRLADAINEGRK--GSNAER--------KN-------------------------MGHKIHPSGLRLGITQE----------HRSKWFATSKTYPILLQEDF--KIRTFIEKK-Y-----GAAGISDVLIARKADQLELELK-------TARPGVIVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTIG-------DRTRQVRINVVE-------------------------------------VERVDADAFLLAEYIAQQLEKRVAFRRTIRMALQRAQRA-GVLGLKIQVGG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRAEIDYATREANTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLPKEEQ-----TIPV----------------GANPKRK-ASRRPQQFEDRSSEN-------------MGELPGLTRKASKKSNPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSEKQLVRYVKKARAGSTGLLRLLERLDNVCFRLGFGGTIPGSRQLVNHGHVTVNGKVLDIAGYQCKSGDVIGIKENSKKLVENIEFPGLANVPHLDLDKKLGKINGDREWVALEINELLVVEYYSRKVNVPGAIPVEQKKNNR-SKNL---ERDSDWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVSDGKKNLVRVPLTPNNSIPTLSKGRDGAANVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSQLRTHKSASRERG--ISLEQLYSMSRRKRPVPDFNSRLASMMISRLMKGKKSTARILSDAFIERTGNAVEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVTFSRARGKSMSKLAELMDAANETGSAVKKREDTHKMAEANKAFAHYRYMSHDPISDMLTRIRNAKHTTTIPGSKMSLSIAKVLQKEGFIENEGEGKSIILGLKYSGKNKPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARKQGIGGEVLCYVYMKKAVYWGTGRRKTSVARVRLIPGNGLIINGRGDYLNFNPHLVKAPLQTLGLENSYDILVNVGGGLTGQADAIKQGAARALCELSPDNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAAKKTGKKSKRNVPNGVVIQSTFNNTIVSITDGHVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALDQGMRQIEVLVRGPGAGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTISQLVGERTKKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLLRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKAPKMARIAGIDIPREKRVEIALTYVYGIGLTRSKILTGVNPDTRVKDLSDDVLRTEEFTLEGDLRREMALKRLDIGVRGRRHRMSLPVRGQRTRTNARTRRGRTVAGRKK--MSLDTEKQKLIEHQVHPTDTGSVEVQVAMLSKRISKLSDHLQNHDSSRQGLLKMIGKRKRLLSYIKNQRYQLVKIGIRGMALKERIGTVVSDKMDKTVVVAVINRYPHPTYKKIVSRTTRYKAHDPENTCVTGDRVKIRETRPLSAHKRWAIEEILNKTNQA-----------',\n",
       " 'Prochlorococcus_marinus_str._MIT_9312': 'KKLSKRMALKIEDRYPLEALIIKENANAKFETIEAHIRLGIDPKYTDQQLRTTVALPHGTGQIKIAVITSGENVKAAGADLFGEEDLVESINKGMDFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDIAAIEFKAGKLEFRADKAGIVHVFGKASFTEALFNLKTLQESIDKNKPSGSKGKYWKSYVTSTMGPSVQVDISAVQDYQMSIGILGKKLGMSQLFDDKGNAVPVTLIEAGPCRVTQLKTTTLDGYSAVQIGYGLSKDKHINKPEKGHLLKSGEELLKHLKEYRVEETTSYEIGNQITVKNFEVGQKVDISGKSMGRGFAGYQKRHGFSRGPMSHGSKNHRAPGSTGAGTTPGRIYPGKRMAGRYGGKQITTKGLLVLKIDDQKNLLVIKGSVPGKPGSIVNIKPNNVVGGQKS-TLKYESIPKLKDLLKNIHQVPKVVKVNVNRGLGEAASNSKSLEASLNEMAITGQKALVTRAKKAIAGFKIREGMPIGCTVTLRGDRMYSFLERFINLALPRIRDFRGVNPKSFDGRGNYTVGVKEQLIFPEISFDKIDSIRGMDITIVTSARSDQEGKALLQELGMPFSNMSRIGKTPVLIPEKVTVDLDGLIVTVKGPKGELKRLMPEGVSFDKKDNTVVVSPTTNKIHSR--QRHGLCRALIANMVEGVTQGFSKKLEIVGVGSRAQVKGKNLVVSAGYSHPVEMIPPDGITYKVESNTNVTVSGIDKEIVGNEAAKIRSIRPPEPYKGKGIKYHDERILRKAGKSGKK---MAKKIVAVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTQDKAGFVIPVEISVFEDRSFTFITKTPPAAVLITKAAGIDKGSGESSKGSVGNISKAQLEEIAKTKLPDLNCSSVESAMKVIEGTARNMGVSITDMKTTPTIERWFLVDADKTLGRLATEIAVLRGKNKPFTPHLDTGDFVIVVNAEKIVSGKKSQKLYRRHSGRPGGMKTEKFSLQRIPERIIEAVKGMLPHNSLGRQQFKKLKVYKGAHPHAQNPVL------MIQETYLTVADNSGAKRLQCIRVL-GSNRYAVGDIVATVKDAPNMVKKSEVVKAVIVRTKLRRNTGSIRFDDNAAVLINDKNPKGTRVFGPVARELRDKNYTKIVSLAPEVITKLSRKLQQKRHRRLRSVGDTRPRLSVFRSNHIYAQVIDDTISASTVDELRESPDNSSVGKLLARAIKKGIQVIFDRGGNLYHGRVKALADAAREAGLEFMTNTSEAIAGNYVRSAKVRRVLDQIRGRSYRDALIMLEFMPYRSTDPITKVLRSAVANAEHNGMDPSLIISSAWANGPMKRYRPRAQGRAFSIKKQTCHISISVEST--------------------------------------MAVVSLSE-----MMEAGAHFGHQTRRWNPKMSKYIY-CARNGVHIIDLVKTALCMNNAYKWTRNAAKGGKRFLFVGTKKQASDVVAQEATRC-GAAYVNQRWLGGMLTNWTTMKARIERLKDLERMESSGSIAMRPKKEAAVLRRELERLQKYLGGLKGMRRLPDVVVLVDQRRETNAVLEARKL--DISLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLILGRLADAINEGRK--ASNNEK---------N-------------------------MGHKIHPSGLRLGITQE----------HRSKWFATSKTYPILLQEDF--KIRTFIQKK-Y-----GAAGISDVLIARKADQLELELK-------TARPGVIVGRQG-----------------------------------------------------------------------SGIEELR----------------------AGIQKTIG-------DNTRQVRINVVE-------------------------------------VERVDADAFLLAEYISQQLEKRVAFRRTIRMALQRAQRA-GVLGLKIQVGG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRAEIDYATREANTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLPKEEK-----TIPV----------------GASPKRK-AGRRPQQFEDRSNENMSRYRGPRLRVRRLGELPGLTRKASKKSNPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSEKQLVRYVKKARAGSTGLLRLLERLDNVCFRLGFGGTIPGSRQLVNHGHVTVNGKVLDIAGYQCKSGDVIGIKENSKKLVENIEFPGLANVPHLDLDKKLGKITGDREWVALEINELLVVEYYSRKVKVPGAIPGEQKKNNR-SKNL---ERDSDWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVSDGKKNLVRVPLTPNNSIPTLSKGRDGAANVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSQLRTHKSASRERG--ISLEQLYSMSRRKRPVPDFNSRLASMMISRLMKGKKSTARILSDAFIERTGDAVEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVTFSRARGKSMSKLAELMDAANETGSSVKKREDTHKMAEANKAFAHYRYMSHDPISDMLTRIRNAKHTTTIPGSKMSLSIAKVLQKEGFIDNEGEGKSIILSLKYSGKNKPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARKQGIGGEVLCYVYMKKAVYWGTGRRKTSVARVRLIPGNGQIINGRGDYLNFNPHLIKAPLQTLGLENSYDILVNVGGGLTGQADAIKQGAARALCELSPDNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAAKKTGKKSKRNVPNGVVIQSTFNNTIVSITDGHVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALDQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTISQLIGERTRKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLLRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKAPKMARIAGIDIPREKRVEIALTYVYGIGLTRSKILAGVNPDIRVKDLSDDVLRTEEFTLEGDLRREMALKRLDIGVRGRRHRMSLPVRGQRTRTNARTRRGRTVAGRKK--MSLDTEKQKLIEHQVHATDTGSVEVQVAMLSKRISKLSEHLQNHDASRQGLLKMIGKRKRLLSYIKNQKYQLVKIGIRGMALKERIGTVVSDKMDKTVVVAVINRYPHPTYKKIVSRTTRYKAHDPENTCALGDRVKIRETRPLSAHKRWAIQEILNKTSQT-----------',\n",
       " 'Prochlorococcus_sp._MIT0604': 'KKLSKRMALKIEDRYPLEALIIKENANAKFETIEAHIRLGIDPKYTDQQLRTTVALPHGTGQIKIAVITSGENVKAAGADLFGEEDLVESINKGMEFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDIAAIEFKAGKLEFRADKAGIVHVFGKASFAEALFNLKTLQESIDKNKPSGAKGKYWKTYVTSTMGPSVQVDINAIQDYQMSIGILGKKLGMSQLFDDKGNSVPVTLIEAGPCRVTQLKTTALDGYTAVQIGYGLSKEKHLSKPEKGHLLKSGEKLLKHLKEYRVEETSSYEIGKQITVKNFEVGQKVDISGKSMGRGFAGYQKRHGFSRGPMSHGSKNHRAPGSTGAGTTPGRIYPGKRMAGRYGGKQITTKGLLVLKIDDQKNLLVVKGSVPGKPGSIVNIKPNNVVGGEKS-TLKYESIPKLKDLLKNIHQVPKVVKVNVNRGLGEAASNSKALEASLNEMAITGQKALVTRAKKAIAGFKIREGMPIGCTVTLRGDRMYSFLERFINLALPRIRDFRGVNPKSFDGRGNYTVGVKEQLIFPEISFDKIDSIRGMDITIVTSAKSDQEGKALLQELGMPFSNMSRIGKTPVLIPEKVTVDFDGLTVTVKGPKGELKRLMPEGVSFDKKDNTVVVSPTTTKIHSR--QRHGLCRALIANMVEGVTQGFSKKLEIVGVGSRAQVKGKNLVVSAGYSHPVEMIPPDGITYKVESNTNVTVSGIDKEIVGNEAAKIRSIRPPEPYKGKGIKYHDERILRKAGKSGKK---MAKKIVAVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKAGFVIPVEISVFEDRSFTFITKTPPASVLITKAAGIEKGSGESAKGSVGNISKAQLEEIAKTKLPDLNCSSVESAMKVIEGTARNMGVSITDMKTTPTIERWFLVDADKTLGRLATEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKSQKLYRRHSGRPGGMKIEKFSLQRIPERIIEAVKGMLPHNSLGRQQFKKLKVYKGAHPHAQNPVL------MIQETYLTVADNSGAKRLQCIRVL-GSNRYAVGDIVATVKDAPNMVKKSEVVKAVIVRTKLRRNTGSIRFDDNAAVLINDKNPKGTRVFGPVARELRDKNYTKIVSLAPEVITKLSRKLQQKRHRRLRFLGDTRPRLSVFRSNHIYAQVIDDTISASTVDELRESSDNSSVGKLLARAIKKGIQVIFDRGGNLYHGRVKALADAAREAGLEFMTKTPEAIAGNYVRSAKVRRVLDQIRGRSYRDALIMLEFMPYRSTDPITKVLRSAVANAEHNGMDPSLVISSAWANGPMKRYRPRAQGRAFSIKKQTCHISISVEST--------------------------------------MAVVSLSE-----MMEAGAHFGHQTRRWNPKMSKYIY-CARNGVHIIDLVKTALCMNNAYKWTRNAAKSGKRFLFVGTKKQASDVVAQEATRC-GAAYVNQRWLGGMLTNWTTMKARIERLKDLERMESSGSIAMRPKKEAAVLRRELERLQKYLGGLKGMRRLPDVVVLVDQRRESNAVLEARKL--DISLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLILGRLADAINEGRK--GSSSER--------KN-------------------------MGHKIHPSGLRLGITQE----------HRSKWFATSKTYPILLQEDF--KIRTFIQKK-Y-----GAAGISDVLIARKADQLELELK-------TARPGVIVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTIG-------DRTRQVRINVVE-------------------------------------VERVDADAFLLAEYIAQQLEKRVAFRRTIRMALQRAQRA-GVLGLKIQVGG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRAEIDYATREANTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLPKEEQ-----TIPV----------------GASPKRK-ASRRPQQFEDRSNENMSRYRGPRLRVRRLGELPGLTRKASKKSNPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSEKQLVRYVKKARAGSTGLLRLLERLDNVCFRLGFGGTIPGSRQLVNHGHVTVNGKVLDIAGYQCKSGDVIGIKENSKKLVENIEFPGLANVPHLDLDKKLGKINGDREWVALEINELLVVEYYSRKVNVPGAMPVEQKKNNR-SKNL---ERDSDWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVSDGKKNLVRVPLTPNNSIPTLSKGRDGAANVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSQLRTHKSASRERG--ISLEQLYSMSRRKRPIPDFNSRLASMMISRLMKGKKSTARILSDAFIERTGNAVEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVTFSRARGKSMSKLAELMDAANETGSAVKKREDTHKMAEANKAFAHYRYMSHDPISDMLTRIRNAKHTTTIPGSKMSLSIAKVLQKEGFIDNEGEGKSIILGLKYSGKNKPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARKQGIGGEVLCYVYMKKAVYWGTGRRKTSVARVRLIPGNGLIINGRGDYLNFNPHLIKAPLQTLGLENSYDILVNVGGGLTGQADAIKQGAARALCELSPDNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAAKKTGKKSKRNVPNGVVIQSTFNNTIVSITDGHVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALDQGMRQIEVLVRGPGAGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTISQLIGERTRKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLLRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKAPKVARIAGIDIPREKRVEIALTYVYGIGLTRSKILTGVNPDTRVKDLSDDVLRTEEFTLEGDLRREMALKRLDIGVRGRRHRMSLPVRGQRTRTNARTRRGRTVAGRKK--MSLDTEKQKLIEHQVHPTDTGSAEVQVAMLSKRISKLSDHLQNHDASRQGLLKMIGKRKRLLTYIKNQKYQLVKIGIRGMALKERIGTVVSDKMDKTVVVAVINRYPHPTYKKIVSRTTRYKAHDPENTCVLGDRVKIRETRPLSAHKRWAIEEILNKTSQA-----------',\n",
       " 'Prochlorococcus_marinus_str._AS9601': 'KKLSKRMALKIEDRYPLEALIIKENANAKFETIEAHIRLGIDPKYTDQQLRTTVVLPHGTGQIKIAVITSGENVKAAGADLFGEEDLVESINKGMEFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDIAAIEFKAGKLEFRADKAGIVHVFGKASFTEALFNLKTLQESIDKNKPSGAKGKYWKTYVTSTMGPSVQLDINAVQDYQMSIGILGKKLGMSQLFDDKGNSVPVTLIEAGPCRVTQLKTTALDGYTAVQIGYGLSKEKHLSKPEKGHLLKSGEELLKHLKEYRVEETSSYEIGKQITVKNFEVGQKVDISGKSMGRGFAGYQKRHGFSRGPMSHGSKNHRAPGSTGAGTTPGRIYPGKRMAGRYGGKQITTKGLLVLKIDDQKNLLVVKGSVPGKPGSIINIKPNNVVGS----TLKYESIPKLKDLLKNIHQVPKVVKVNVNRGLGEAASNSKALEASLNEMAITGQKALVTRAKKAIAGFKIREGMPIGCTVTLRGDRMYSFLERFINLALPRIRDFRGVNPKSFDGRGNYTVGVKEQLIFPEISFDKIDSIRGMDITIVTSAKSDQEGKALLQELGMPFSNMSRIGKTPVLIPEKVTVDFDGLTVTVKGPKGELKRLMPEGVSFDKKDNTVVVSPTTTKIHSR--QRHGLCRALIANMVEGVTQGFSKKLEIVGVGSRAQVKGKNLVVSAGYSHPVEMIPPDGITYKVESNTNVTVSGIDKEIVGNEAAKIRSIRPPEPYKGKGIKYHDERILRKAGKSGKK---MAKKIVAVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKAGFVIPVEISVFEDRSFTFITKTPPASVLITKAAGIEKGSGESAKGSVGNISKAQLEEIAKTKLPDLNCSSVESAMKVIEGTARNMGVSITDMKTTPTIERWFLVDADKTLGRLATEIAVLRGKNKPYTPHLDTGDFVIVVNAEKVVTGKKSQKLYRRHSGRPGGMKIEKFSLQRIPERIIEAVKGMLPHNSLGRQQFKKLKVYKGAHPHAQNPVL------MIQETYLTVADNSGAKRLQCIRVL-GSNRYAVGDIVATVKDAPNMVKKSEVVKAVIVRTKLRRNTGSIRFDDNAAVLINDKNPKGTRVFGPVARELRDKNYTKIVSLAPEVITKLSRKLQQKRHRRLRYLGDTRPRLSVYRSNHIYAQVIDDTISASTVDELKESPDNSSVGKLLARAIKKGIQVIFDRGGNLYHGRVKALADAAREAGLEFMTKTPEAIAGNYVRSAKVRRVLDQIRGRSYRDALIMLEFMPYRSTDPITKVLRSAVANAEHNGMDPSLVISSAWANGPMKRYRPRAQGRAFSIKKQTCHISISVEST--------------------------------------MAVVSLSE-----MMEAGAHFGHQTRRWNPKMSKYIY-CARNGVHIIDLVKTALCMNNAYKWTRNAAKSGKRFLFVGTKKQASDVVAQEATRC-GAAYVNQRWLGGMLTNWTTMKARIERLKDLERMESSGSIAMRPKKEAAVLRRELERLQKYLGGLKGMRRLPDVVVLVDQRRESNAVLEARKL--DISLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLILGRLADAINEGRK--GSNAER--------KN-------------------------MGHKIHPSGLRLGITQE----------HRSKWFATSKTYPILLQEDF--KIRTFIQKK-Y-----GAAGISDVLIARKADQLELELK-------TARPGVIVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTIG-------DRTRQVRINVVE-------------------------------------VERVDADAFLLAEYIAQQLEKRVAFRRTIRMALQRAQRA-GVLGLKIQVGG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRAEIDYATREANTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLPKEEQ-----TIPV----------------GASPKRK-ASRRPQQFEDRSNENMSRYRGPRLRVRRLGELPGLTRKASKKSNPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSEKQLVRYVKKARAGSTGLLRLLERLDNVCFRLGFGGTIPGSRQLVNHGHVTVNGKVLDIAGYQCKSGDVIGIKENSKKLVENIEFPGLANVPHLDLDKKLGKINGDREWVALEINELLVVEYYSRKVNVPAATPVE-QKKN--SKNL---ERDSDWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVSDGKKNLVRVPLTPNNSIPTLSLGSDGAANVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSQLRTHKSVSRERG--ISLEQLYSMSRRKRPVPDFNSRLASMMISRLMKGKKSTARILSDAFIERTGNAVEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVTFSRARGKSMSKLAELMDAANETGSSVKKREDTHKMAEANKAFAHYRYMSHDPISDMLTRIRNAKHTTTIPGSKMSLSIAKVLQKEGFIDNEGEGKSIILGLKYSGKNKPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARKQGIGGEVLCYVYMKKAVYWGTGRRKTSVARVRLIPGNGLIINGRGDYLNFNPHLIKAPLQTLGLENSYDILVNVGGGLTGQADAIKQGAARALCELSPDNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAAKKTGKKSKRNVPNGVVIQSTFNNTIVSITDGHVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALDQGMRQIEVLVRGPGAGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTISQLVGERTKKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLLRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKAPKMARIAGIDIPREKRVEIALTYVYGIGLTRSKILTGVNPDTRVKDLSDDVLRTEEFTLEGDLRREMALKRLDIGVRGRRHRMSLPVRGQRTRTNARTRRGRTVAGRKK--MSLDTEKQKLIEHQVHPTDTGSVEVQVAMLSKRISKLSDHLQNHDASRQGLLKMIGKRKRLLSYLKNQKYQLVKIGIRGMALKERIGTVVSDKMDKTVVVAVINRYPHPTYKKIVSRTTRYKAHDPENTCVLGDRVKIRETRPLSAHKRWAIEEILNKTSQA-----------',\n",
       " 'Prochlorococcus_marinus_str._MIT_9301': 'KKLSKRMALKIEDRYPLEALIIKENANAKFETIEAHIRLGIDPKYTDQQLRTTVALPHGTGQIKIAVITSGENVKAAGADLFGEEDLVESINKGMEFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDIAAIEFKAGKLEFRADKAGIVHVFGKASFTEALFNLKTLQESIDKNKPSGAKGKYWKTYVTSTMGPSVQVDINAVQDYQMSIGILGKKLGMSQLFDEKGNSVPVTLIEAGPCRITQLKTTALDGYTAVQIGYGLSKDKHISKPEKGHLLKSGEELLKHLKEYRVEETSSYEIGNQITVKNFEVGQKVDISGKSMGRGFAGYQKRHGFSRGPMSHGSKNHRAPGSTGAGTTPGRIYPGKRMAGRYGGKQITTKGLLVLKIDDQKNLLVVKGSVPGKPGSIINIKPNNIVGGEKS-TLKYESIPKLKDLLKNIHQVPKVVKVNVNRGLGEAASNSKALEASLNEMAITGQKALVTRAKKAIAGFKIREGMPIGCTVTLRGDRMYSFLERFINLALPRIRDFRGVNPKSFDGRGNYTVGVKEQLIFPEISFDKIDSIRGMDITIVTSARSDQEGKALLQELGMPFSNMSRIGKTPVLIPDKVSVDFDGLTVTVKGPKGELKRQMPEGVSFDKKDNTVIVSPTTTKIFSR--QRHGLCRALIANMVKGVSQGFSKKLEIVGVGSRAQVKGKNLVVSAGYSHPIEMIPPDGITYKVESNTNVTVSGIDKEIVGNEAAKIRSIRPPEPYKGKGIKYHDERILRKAGKSGKK---MAKKIVAVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKAGFVIPVEISVFEDRSFTFITKTPPASVLITKAAGIEKGSGESAKGSVGNISKAQLEEIAKTKLPDLNCSSVESAMKVIEGTARNMGVSITDMKTTPTIERWFLVDADKTLGRLATEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKSQKLYRRHSGRPGGMKIEKFSLQRIPERIIEAVKGMLPHNSLGRQQFKKLKVYKGAHPHAQNPVL------MIQETYLTVADNSGAKRLQCIRVL-GSNRYAVGDIVATVKDAPNMVKKSEVVKAVIVRTKLRRNTGSIRFDDNAAVLINDKNPKGTRVFGPVARELRDKNYTKIVSLAPEVITKLSRKLQQKRHRRLRFLGDTRPRLSVFRSNHIYAQVIDDTISASTVDELRESPDNSSVGKLLARAIKKGIQVIFDRGGNLYHGRVKALADAAREAGLEFMTTTPEAVAGNYVRSAKVRRVLDQIRGRSYRDALIMLEFMPYRSTDPITKVLRSAVANAEHNGMDPSLVISSAWANGPMKRYRPRAQGRAFSIKKQTCHISISVEST--------------------------------------MAVVSLSE-----MMEAGAHFGHQTRRWNPKMSKYIY-CARNGVHIIDLVKTALCMNNAYKWTRNAAKSGKRFLFVGTKKQASDVVAQEATRC-GAAYVNQRWLGGMLTNWTTMKARIERLKDLERMESSGSIAMRPKKEAAVLRRELERLQKYLGGLKGMRRLPDVVVLVDQRRESNAVLEARKL--DISLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLILGRLADAINEGRK--GSNAER--------KN-------------------------MGHKIHPSGLRLGITQE----------HRSKWFATSKTYPILLQEDF--KIRTFIQKK-Y-----GAAGISDVLIARKADQLELELK-------TARPGVIVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTIG-------DRTRQVRINVVE-------------------------------------VERVDADAFLLAEYIAQQLEKRVAFRRTIRMALQRAQRA-GVLGLKIQVGG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRAEIDYATREANTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLPKEEQ-----TIPV----------------GASPKRK-ASRRPQQFEDRSNENMSRYRGPRLRVRRLGELPGLTRKASKKSNPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSEKQLVRYVKKARAGSTGLLRLLERLDNVCFRLGFGGTIPGSRQLVNHGHVTVNGKVLDIAGYQCKSGDVIGIKENSKKLVENIEFPGLANVPHLDLDKKLGKINGDREWVALEINELLVVEYYSRKVNVPGATPVEQKKNNR-SKNL---ERDSDWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVSDGKKNLVRVPLTPNNSIPTLSLGRDGAANVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSQLRTHKSASKERG--ISLEQLYSMSRRKRPVPDFNSRLASMMISRLMKGKKSTARILSDAFIERTGNAVEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVTFSRARGKSMSKLAELMDAANETGSAVKKREDTHKMAEANKAFAHYRYMSHDPISDMLTRIRNAKHTTTIPSSKMSLSIAKVLQKEGFIDNEGEGKSIILGLKYSGKNKPTISMQRVSKPGLRIYKNRLPKVLGGLGVAIISTSKGVMSDRDARKQGIGGEVLCYVYMKKAVYWGTGRRKTSVARVRLIPGNGLIINGRGDYLNFNPHLIKAPLQTLGLENSYDMYVNVGGGLTGQADAIKQGAARALCELSPDNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMPAKKTGKKSKRNVPNGVVIQSTFNNTIVSITDGHVISWSSAGASGFKGARKGTPFAAQTAAEAAARRALDQGMRQIEVLVRGPGAGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTISQLVGERTKKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLLRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKAPKMARIAGIDIPREKRVEIALTYVYGIGLTRSKILTGVNPDTRVKDLSDDVLRTEEFTLEGDLRREMALKRLDIGVRGRRHRMSLPVRGQRTRTNARTRRGRTVAGRKK--MSLDTEKQKLIEHQVHPTDTGSAEVQVAMLSKRISKLSDHLQNHDASRQGLLKMIGKRKRLLSYLKNQKYQLVKIGIRGMALKERIGTVVSDKMDKTVVVAVINRYPHPTYKKIVSRTTRYKAHDPENTCVLGDRVKIRETRPLSAHKRWAIEEILNKTSQA-----------',\n",
       " 'Prochlorococcus_marinus_str._MIT_9515': 'KKLSKRMTLKIEDRYPLEALIIKENANAKFETIEAHIRLGIDPKYTDQQLRTTVALPHGTGQIKIAVITSGENVKAAGADLFGEEDLVESINKGMDFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDIAAIEFKAGKLEFRADKAGIVHVFGKASFTNALFNLKTLQESIDKNKPSGAKGKYWKTYVTSTMGPSVQVDINALQDYQMSIGILGKKLGMSQLFDDDGNAVPVTLIEAGPCRVTQLKTQSLDGYTAIQIGYGVSKDKHLSKPEKGHLLKSGKELLKHLKEFKVEENSSYEIGKEITVTNFEVGQKVDISGKSMGRGFSGYQKRHGFSRGPMSHGSKNHRAPGSTGAGTTPGRIYPGKRMAGRYGGKKITTKGLLVVKIDDQKNLLVVKGSVPGKPGSIVNIRPNNTVGGTKS-TLKYETIPKLKDLLKNIHQVPKVIKVNVNRGLGEAASNSKALEASLNEMAITGQKALVTRSKKAIAGFKIREGMAIGCTVTLRGDRMYSFLERFINLALPRIRDFRGVNPKSFDGRGNYTLGVKEQLIFPEISFDKIDSIRGMDITIVTSASNDQEGKALLKELGMPFS-MSRIGKSPVQIPDKVSVDINGLTITVKGPKGELKRLMPEGVDFVQKENQIVVTPSTTKRYSR--ERHGLCRTLISNMVQGVTDGYSKKLEIVGVGSRAQVKGKTLVVSAGYSHPVEMTPPDGITYKVESNTNVTVSGIDKEIVGNEAAKIRSIRPPEPYKGKGIKYQDERIIRKAGKSGKK---MAKKIVAVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKAGFVIPVEISVFEDRSFTFITKTPPASVLITKAAGIEKGSGESSKGSVGNISKSQLEEIAKTKLPDLNCTSIESAMKVIEGTARNMGISITEMKTTPTIERWFLVDADKTLGRLATEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKSQKLYRRHSGRPGGMKVEKFSLQRIPERIIEAVKGMLPHNSLGRQQFKKLKVYKGSHPHAQNPVL------MIQETYLTVADNSGAKRLQCIRVL-GSNRYAVGDVVASVKDAPNMVKKSDVVKAVIVRTRLRRNTGSIRFDDNAAVLINDKNPKGTRVFGPVARELRDKNFTKIVSLAPEVIAKISRKLQQKRHKRLRYLGNIRPRLAVFRSNHIYAQVIDDTISASTVDELKESSNSSSVGKLLARAIKKGIQVIFDRGGNLYHGRVKALADAARDAGLNFMIKKSEAIAGKYIRSAKVRRVLDQIRGKSYRDALIMLEFMPYRSTDPITKVLRSAVANAEHNGMEPSLVISSASADGPMKRFRPRAQGRAFSIKKQTCHISISVESN--------------------------------------MAVVSLSE-----MMEAGAHFGHQTRRWNPKMSKYIY-CARNGVHIIDLVKTALCMNNAYKWTRNAAKSGKRFLFVGTKKQASDVVAQEAVRC-GAAYVNQRWLGGMLTNWSTMKARIERLKDLERMESSGAIAMRPKKEAAVLRRELERLQKYLGGLKGMRRLPDVVVLVDQRRESNAVLEARKL--DISLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLILGRLADAINEGRK--GSNDQR--------KV-------------------------MGNKINPTGLRLGITQE----------HRSKWFATSKTYPILLQEDY--KIRNFIQKK-Y-----SSAGISDVLIARKADQLELELK-------TARPGVIVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTIG-------DRTRQVRINVVE-------------------------------------VERVDADAYLLAEYIAQQLEKRVAFRRTIRMALQRAQRA-GVLGLKIQVGG-----------------------------RLNGAEIARSE-----WTREG---RVPLHTLRAEVDYALREANTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLPKEEQ-----TIPV----------------GAIPRRK-GSRKPQQFEDRSNENMSRYRGPRLRVRRLGELPGLTRKASKKSNPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSERQLVRYVKKARAGSTGLLRLLERLDNVCFRLGFGGTIPGSRQLVNHGHVTVNGKVLDIAGYQCKSGDVISIKEKSKKLVENIEFPGLANVPHIELDKKLGKINGDREWVALEINELLVVEYYSRKVNLPPSNPND-QRKGN-SKN----ERDSEWQERVVQIRRVFKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVSDGKKHLVRVPLTPNNSIPTLSKGRDGAANVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSQLRTHKSASRERG--ISLEQLYSMSRRKRPVPDFNSRLASMMISRLMKGKKSTAKILSDAFIERTGNAVEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVTFSRGRGKSMSKLAELMDAANETGSAVKKREDTHKMAEANKAFAHYRYMSHDPISDMLTRIRNAKHTSTIPASRMILSIAKVLQKEGFIDNEGEGESIVLGLKYSGKNRPTISMQRVSKPGLRVYKNKLPKVLGGLGVAIVSTSKGVMSDRDARKQGIGGEVLCYVYMKKAVYWGTGRRKTSVARVRLIPGNGQIINGRGDYLNFNPHLVKAPLQTLGLENSYDIFVNVGGGLTGQADAIKQGAARALCDLSPDNRPLKEGHLSRDPRAKERRKYGLKARKAPQFSKRMAAKKTGKKSKKNVPNGVVIQSTFNNTIVSISDGHVISWSSAGASGFKGARKGTPFAAQTAAEAAAKRALDQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTISQLIGERTRKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLLRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKAPKMARIAGIDIPREKRVEIALTYIYGIGLTRSKILTGVNPDIRVKDLSDDVLRAEDFTVEGDLRREMALKRLDIGVRGRRHRMSLPVRGQRTRTNARTRRGRTVAGRKK--MTLDTEKQKLIEHQVHPTDTGSVEIQVAMISERITKLSEHLQNHDASRQGLLKMIGKRKRLLSYIKDKNYQLIKIGIRGMALKERIGTVVSDKMDKTVVVAVINRYPHPTYKKIVSKTTRYKAHDPENSCAMGDRVKIKETRPLSAHKRWAIEEILNKTIKN-----------',\n",
       " 'Prochlorococcus_marinus_subsp._pastoris_str._CCMP1986': 'KKLSKRMKLKIEDRYPLEALIVKENANAKFETIEAHIRLGIDPKYTDQQLRTTVALPNGTGQIKIAVITSGENVKAAGADLFGEEDLVESINKGMDFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDIAAIEFKAGKLEFRADKAGIVHVFGKASFTNALFNLKTLQESIDKNKPSGAKGKYWRSYLTSTMGPSVQVDINALQDYQMSIGILGKKLGMSQLFDKDGNAVPVTLIEAGPCRVTQLKTQPLDGYTAIQIGYGVSKDKHLSKPEKGHLLKSGEILLKHLKEYRVEENSSYEIGKEITVTNFEVGQKVDISGKSMGRGFSGYQKRHGFSRGPMSHGSKNHRAPGSTGAGTTPGRIYPGKRMAGRYGGKKITTKGLLVVKIDDQKNLLVVKGSVPGKPGSIVNIKPNNVVGGAKS-TLKYEAIPKLKDLLKNIHQVPKVIKVNVNRGLGEAASNSKALEASLNEMAITGQKALVTRSKKAIAGFKIREGMAIGCTVTLRGDRMYSFLERFINLALPRIRDFRGVNPKSFDGRGNYTLGVKEQLIFPEISFDKIDSIRGMDITIVTSASTDQEGKALLKELGMPFS-MSRIGKSPVQIPEKVSVDIKGLSITVKGPKGELKRLMPEGVNFDQKENQIVVTPATTKRYSR--ERHGLCRTLISNMVQGVTEGYEKKLEIVGVGSRAQVKGKNLVVSAGYSHPVEMTPPDGITYKVESNTNVTVSGIDKEIVGNEAAKIRSIRPPEPYKGKGIKYQDERIIRKAGKSGKK---MAKKIVAVIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKAGFVIPVEISVFEDRSFTFITKTPPASVLITKAAGIEKGAGESSKGSVGNISKSQLEEIAKTKLPDLNCTSIESAMKVIEGTARNMGVSITEMKTTPTIERWFLVDADKTLGRLSTEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKSQKLYRRHSGRPGGMKVEKFSLQRIPERIIEAVKGMLPHNSLGRQQFKKLKVYKGSHPHAQNPVL------MIQETYLTVADNSGAKRLQCIRVL-GSNRYAVGDVVASVKDAPNMVKKSDVVKAVIVRTRLRRNTGSIRFDDNAAVLINDKNPKGTRVFGPVARELRDKNFTKIVSLAPEVIAKISRKLQQKRHKRLRYLGSARPRLAVFRSNHIYAQVIDDTISASTVDELKEKSNSSSVGKLLARAMKKGVEVIFDRGGNLYHGRVKALADAARDAGLNFMTKTPEAIAGKYIRSAKVRRVLDQIRGKSYRDALIMLEFMPYRSTDPITKVLRSAVANAEHNGMDPSLVISSASADGPMKRFRPRAQGRAFSIKKQTCHISISVESN--------------------------------------MAVVSLSE-----MMEAGAHFGHQTRRWNPKMSKYIY-CARNGVHIIDLVKTALCMNNAYKWTRNAAKSGKRFLFVGTKKQASDVVAQEAVRC-GAAYVNQRWLGGMLTNWTTMKARIERLKDLERMESSGAIAMRPKKEAAVLRRELERLQKYLGGLKGMRRLPDVVVLVDQRRESNAVLEARKL--DISLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLILGRLADAINEGRK--GSNDQR--------KN-------------------------MGNKINPTGFRLGITQE----------HRSKWFATSKTYPTLLQEDD--KIRTFIQKK-Y-----SSAGISDVLIARKADQLELELK-------TARPGVIVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTIG-------DRTRQVRINVVE-------------------------------------VERVDADAYLLAEYIAQQLEKRVAFRRTIRMALQRAQRA-GVLGLKVQVGG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRAEVDYALREANTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLPKEEQ-----TIPV----------------GAIPRRK-GSRKPQQFEDRSNENMSRYRGPRLRVRRLGELPGLTRKASKKSNPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSERQLVRYVKKARAGSTGLLRLLERLDNVCFRLGFGGTIPGSRQLVNHGHVTINGKVLDIAGYQCKPGDVISIKENSKKLVENIEFPGLANVPHIELDKKLGKINGDREWVALEINELLVVEYYSRKVNPPSSNANE-QRRGN-SKN----ERDSEWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVSDGKKHLVRVPLTPNNSIPTLSKGRDGAANVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSQLRTHKSASRERG--ISLEQLYSMSRRKRPVPDFNSRLASMMISRLMKGKKSTAKILSDAFIERTGNAVEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVTFSRGRGKSMSKLAELMDAANETGSAVKKREDTHKMAEANKAFAHYRYMSHDPISDMLTRIRNAKHTSTIPASKMSLSIAKVLQKEGFIDNEGEGKSIVLGLKYSGKNKPTISMQRVSKPGLRVYKNKLPKVLGGLGVAIVSTSKGVMSDRDARKQGIGGEVLCYVYMKKAVYWGTGRRKTSVARVRLIPGNGQIINGRGDYLNFNPHLVKAPLLTLGLENSYDIFVNVGGGLTGQADAIKQGAARALCGLSPDNRPLKEGHLSRDPRSKERKKYGLKARKAGQFSKRMAAKKTGKKSKKNVPNGVVIQSTFNNTIVSITDGHVISWSSAGASGFKGARKGTPFAAQTAAEAAAKRALDQGMRQIKVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTISQLIGERTRKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLLRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKAPKMARIAGIDIPREKRVEIALTYIYGVGLTRSKILTGVNPDIRVKDLSDDVLRTEDFTVEGDLRREMAMKRLDIGVRGRRHRMSLPVRGQRTRTNARTRRGRTVAGRKK--MTLDTEKQKLIEHQVHATDTGSVEVQVAMLSERISKLSDHLQNHDASRQGLLKMIGKRKRLLSYIKNQNYQLIKIGIRGMALKERIGTVVSDKMDKTVVVAVINRYPHPTYKKIVSKTTRYKAHDPENSCVLGDRVKIKETRPLSAHKRWAIEEILNKTIMS-----------',\n",
       " 'Prochlorococcus_marinus_bv._HNLC2_Ga0082359': 'KKLSKRMALKIEDRYAKDALLVKENANAKFETIEAHIRLGIDPKYTDQQLRTTVALPNGTGQIKIAVITSGENVKAAGADLFGEEDLVEDINKGMDFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDIDAIEFKAGKLEFRADKAGIVHVFGKASFSEALFNLKTLQEAIDKNKPSGAKGKYWKTYISSTMGPSVQVDIGGLQEFLMSVGILGKKLGMSQLFDDKGNAVPVTLIQAGPCRVTQLKSLPVDGYSAVQIGFGLSKEKLLNKPKKGHLAKSGDDLLKHLKEYRVEETAQYEIGSQITVDNFEVGQKVDISGKSMGRGFAGYQKRHGFSRGPMSHGSKNHRQPGSTGAGTTPGRIYPGKRMAGRYGGKKITTKGLIVVKVDNEKNLIVVKGSVPGKPGSLLNIRPNNKVGGNKS-TLKYESIPKLKDLLKNIHQVPKVVKVNINRGLGEAAQNSKALEASLNEMAITGQKALVTRAKKAIAGFKIRQGMPIGCSVTLRGDRMYSFLERFINLALPRIRDFRGVNPKSFDGRGNYTIGVKEQLIFPEISFDKIDSIRGMDITIVTSATTDEEGKSLLKELGMPFSNMSRIGKNPVPIPDKVSIDIQGLSITVKGPKGELKRVMPSGVNFDKTDQEVIVSPSTSKRFSR--ERFGLCRTLISNMVQGVSEGFTKKLEIVGVGSRAQVKGKNLVVSAGYSHPIEMVPPDGITYKVESNTNVIVSGIDKEVVGNEAAKIRSIRPPEPYKGKGIKYHDERIIRKAGKSGKK---MAKKVVALIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKAGFVIPVEISVFEDRSFTFITKTPPASVLITKAAGIEKGAGESAKGSVGQISKSQLEEIAKTKLPDLNCTSIESAMKVIEGTARNMGVSISDMKTTPTIKRWFLVDADKTLGRLATEIAILRGKNKPFTPHLDTGDFVIVVNAEKIVTGKKSQKLYRRHSGRPGGMKIEKFTLQRIPERIIEAVKGMLPHNSLGRQQFRKLKVYKGPHPHAQDPVL------MIQETYLTVADNSGAKRLQCIRVL-GSNRYAVGDIVASVKDAPNMVKKSDVVKAVIVRTKLRRNTGSIRFDDNAAVLINDKNPKGTRVFGPVARELRDRNFTKIVSLAPEVITSLSRKKQQKRHKRLRFLGTDRPRLSVFRSNHIYAQVIDDTIAASTVDELKKSLNAASVGSLLARAIKKGIQVVFDRGGNIYHGRVKALAEAARKGGLKFMTKTPDAKAGNYVRSAKVRRVLDQIRGKSYRDALIMLEFMPYRSTEPITKVLRSAVANAEHNGMDPSLIINSASADGPMKRFRPRAQGRAFAIKKQTCHITISVQSN--------------------------------------MAVVTLSE-----MMEAGAHFGHQTRRWNPKMSKYIY-CARNGVHIIDLVKTAVCMNDAYKWARSAAKSGKRFLFVGTKKQASDVVAQEASRC-GAAYVNQRWLGGMLTNWTTMKARIERLKDLERMESSGAIAMRPKKEAAVLRRELERLQKYLGGLKNMRRLPDVVVLVDQRRETNAVLEARKL--DISLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLVLGRLADAINEGRK--GSNDQR--------KN-------------------------MGHKINPSGLRLGITQE----------HRSKWFASSKTYPLLLQEDH--KIRTFIQKK-Y-----GSAGISDVLIARKADQLELELK-------TARPGVIVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTIG-------DRTRQVRINVVE-------------------------------------VERVDADAFLLAEYIAQQLEKRVAFRRTIRMALQRAQRA-GVQGLKIQVGG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRAEIDYATREANTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLSKEEQ-----TIPV----------------GSNSKRK-GGRMPQQFEDRSNENMSRYRGPRLRVRRLGELPGLTRKASKKSNPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSERQLVRYVKKARAGSTGLLRLLERLDNVCFRLGFGPTIPGSRQLVNHGHVTVNGKVLDIAGYQCKTGDVISIKENSKKLVENIEFPGLANVPHIDLDKKLGKINGDREWVALEINELLVVEYYSRKVKINSVNQDE-PKRNS-SKNL---ERDSDWQERVVQIRRVSKTVKGGKKMSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVADGKKNLVRVPLTQTNSIPTLSLGRDGAANVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSQLRTHKSVARERG--ITLEQLYSMSRRKRPVPDFNSRLASMMISRLMKGKKSTARILSDAFIERTGNPVEFETAVKNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVTFSRARGKSMAKLAELMDAANEAGSAVKKREDTHKMAEANKAFAHYRYMSHDPISDMLTRIRNARHTTTIPASKMSRSIALVLQKEGFIENEGEGKSLILGLKYSGKNKPTISMQRVSKPGLRIYKNKLPKVLGGLGVAIISTSKGVMSDRDARKQGIGGEVLCYVYMKKAVYWGTGRRKTSVARVRLIPGNGQIINGRGDYLNFNPHLVKAPLQTLGLENSYDIFVNVGGGLTGQADAIKQGAARALCELSPDNRPLKEGHLSRDPRAKERRKYGLKARKAPQYSKRMPAKKTGKKSKRNVPNGVVIQSTFNNTIVSITDGHVISWSSAGASGFKGARKGTPFAAQTAAESAARRALDQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTIQQLISERTRKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLLRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQARSKYGAKAPKVARIAGIDIPREKRVEISLTYIYGIGLTRSKILTGVNPDTRVKDLSDDVLRTETFTVEGDLRREMALKRLDIGIRGRRHRMSLPVRGQRTRTNARTRRGRTVAGRKK--MPLDTEKQKLIEHQIHGTDTGSVEIQVAMLSERISKLSDHLQNHDASRQGLLKMIGKRKRMLSYIKNKTYLLVKLGIRGMALKERVGTVVSDKMDKTVVVAVINRYPHPTYKKIVSRTKRYQAHDPENICTTGDRVRIRETRPLSANKRWAIDEILNKSLVG-----------',\n",
       " 'Prochlorococcus_marinus_bv._HNLC1_Ga0082358': 'KKLSKRMELKIEDRYAQEALIVKENANAKFETIEAHIRLGIDPKYTDQQLRTTVALPNGTGQIKIAVITSGENVKAAGADLFGEEDLVESINKGMDFDLIATPDMMPKVAKLGRVLGPRGLMPNPKAGTVTDIAAIEFKAGKLEFRADKAGIVHVFGKASFSNQLFNLKTLQEAIDKNKPSGAKGKYWKTYLSSTMGPSIQIDIGALQDYQ------------------------------------------------------------------------------------------------------------------MGRGFSGYQKRHGFSRGPMSHGSKNHRQPGSTGAGTTPGRIYPGKRMAGRYGGKNITTKGLLVVKVDDEKNLIVVKGSVPGKPGSLLNIKPNNKVGNNGG-TLKYDAIPKLKDLLKNIHQVPKVIKVNINRGLGEAAQNSKALEASLNEMAITGQKALVTRAKKAIAGFKIRQGMPIGCTVTLRGDRMYSFLERFINLALPRIRDFRGVNPKSFDGRGNYTIGVKEQLIFPEISFDKIDSIRGMDITIVTSASTDEEGKALLKELGMPFSNMSRIGKSPVLIPEKVSVDIDGLTVTVKGPKGELKRLMPEGVNFNKKENEIFVTPSTSKRFSR--ERHGLCRTLIANMVKGVSEGFVKQLEIVGVGSRAQVKGKNLVVSAGYSHPVEIIPPEGITFKVESNTNVFVSGIDKEIVGNEAAKIRSIRPPEPYKGKGIKYHDERIIRKAGKSGKK---MAKKVVALIKLALQAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKAGFVIPVEISVFEDRSFTFITKTPPASVLITKAAGIEKGAGESAKGAVGKISKSQLEEIAKTKLPDLNCTSIESAMKVIEGTARNMGVSISDMKTTPTIKRWFLVDAGQTLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKIVSGKKSQKLYRRHSGRPGGMKIEKFTLRRIPERIIEAVKGMLPHNSLGRQQFKKLKVYKGSHPHAQNPVL------MIQETYLTVADNSGAKRLQCIRVL-GSNRYAVGDIVASVKDAPNMVKKSDVVKAVIVRTKLRRNTGSIRFDDNAAVLINDKNPKGTRVFGPVARELRDRNFTKIVSLAPEVINTISRKQQQKRHRRLRFLGTTKPRLSVFRSNHIYAQVIDDTISASTIDEFKIDTNNSSVGSLLARAIKKGVEVVFDRGGKIYHGRVKALADAARKAGLKFMITTQDAVAGNYIRSAKVRRVLDQIRGKSYRDALIMLEFMPYRSTEPITKVLRSAVANAEHNGMDPSLIINSAAANGPMKRYRPRAQGRAFSIKKQSCHITISVQSS---------------------------------------------------------------------------------------------------MDRSAAKSGKRFLFVGTKKQASEVVAQEASRC-GAAYVNQRWLGGMLTNWTTMKARIERLKDLERMESSGAIAMRPKKEAAVLRRELERLQKYLGGLKNMRRLPDVVVLVDQRRESNAVLEARKL--DISLVSMLDTNC--DPDLCEVPIPCNDDAVRSVQLILGRLADAINEGRK--GSNDQR--------KN-------------------------MGHKISPTGLRLGITQE----------HRSKWYASSKTYPTLLQEDH--KIRTFVQKK-Y-----GAAGISDVLIARKADQLELELK-------TARPGVIVGRQG-----------------------------------------------------------------------SGIEELR----------------------SGIQKTIG-------DRTRQVRINVVE-------------------------------------VERVDADAFLLAEYIAQQLEKRVAFRRTIRMALQRAQRA-GVLGLKIQVGG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRAEIDYATKEANTT-------------------------------------------YGVLGIKVWVF-----------------KGEVLSKEEQ-----SIPV----------------GTTNKRK-GSRVPQQFEDRSNENMSRYRGPRLRVRRLGELPGLTRKASKKSNPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSERQLVRYVKKARAGSTGLLKLLERLDNVCFRLGFGPTIPGSRQLVNHGHVTVNGKVLDIAGYQCKPNDIISIKENSKKLVENIEFPGLANVPHLELDKKLGKVINDREWVAIEINELLVVEYYSRKVSTANQDEPKKNNRKRDSKNL---ERDSDWQERVVQIRRVSKTVKGGKKMSFRAIIVVGNEKGQVGVGVGKAGDVIGAVRKGVADGKKNLVRVPLTRNNSIPTLSLGRDGAANVLIRPAAPGTGVIAGGSIRTVLELAGIKNVLAKRLGSKTPLNNARAAMVALSQLRTHKSVSRERG--ISLEQLYSMSRRKRQVPDFNSRLASMMISRLMKGKKSTARILSDAFIERTGDPVDFETAVKNATPLVEVRARRVGGATYQVPMEVRERGIAMALRWLVTFSRARGKSMAKLAELMDAANETGSAVKKREDTHKMAEANKAFAHYRYMSHDPISDMLTRIRNARHSTTIPASKMSRSIAKVLQKEGFIENEGEGKSLIIGLKYSGKNRPTISMQRVSKPGLRIYKNKLPRVLGGLGVAIISTSKGIMSDRDARKQGIGGEVLCYVYMKKAVYWGTGRRKTSVARVRLIPGNGQIINGRGDYLNFNPHIVKAPLQTLGLETSYDIFVNV-------------------------------------WRWFDWSSWS-KTRSC------MAAKKTGKKSKRNVPNGVVIQSTFNNTIVSITDGHVISWSSAGASGFKGARKGTPFAAQTAAESAARRALDQGMRQIEVLVRGPGSGRETAIRALQAGLEITLIRDVTPLPHNGCRRPKRRRVMPTIQQLISERKRKTKSPALKCPERRGVCTRVYTSTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLLRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQARSKYGAKAPKVARIAGIDIPREKRVEIALTYIYGIGLTRSKILTGVNPDTRVKDLADDVLRTEAYTVEGDLRREMALKRLDIGVRGRRHRMSLPVRGQRTRTNARTRRGRTVAGRKK--MTLDTEKQKLIDHQIHPTDTGSVELQVAMLSERISKLSDHLQNQDASRQGLLKMIGKRKRMLSYIKDNKYALIKIGIRGMALKERVGTVVSDKMDKTVVVAVINRYPHPTYKKIVSRTTRYKAHDPENECNTGDRVRIKETRPLSAQKRWAIDEILNKQAKV-----------',\n",
       " 'Synechococcus_sp._PCC_7336': 'KKLSKRMRLKVQLTYPLAALLLKETATAKFEAAEVHVRLGIDPKYADQQIRTTVVLPKGTGQVRVGVIARGEKVEAAGADVVGSEELIEEISNGLDFDLVATPDMMPKVAKLGRILGPRKLMPSPKAGTVTDLGAIEFKGGKLEFRADRTGIVHIFGKSSFPEDLLNLKTFQEAIDRHRPSGAKGKYWRSYVCSTMGPSIEVDISALRDLTMAVGIFGKKLGMTQIFDDEGLAVPVTVVEAGPCTVTQIKTKDTDGYTAIQIGYGAAKR--LNKPEMGHLAKAGITPLKHLIEYRLDEPEIYELGQQLTVEAFAAGQLVDISGTTIGRGFTGYQKRHNFRRGPMSHGSKNHRQPGSTGAGTTPGRVYPGKLMAGRDGGKRTTIRKLTVVRVDTERNLLLIKGAVPGVEGGLLEIRPAVKVG-R--MRLKYEKLPQLQELYKNIHQVPKLEKIVVNRGLGEASQNAKALEASLAEISITGQRPVITRAKKAIASFKIREGMPVGLKVTLRREKMYAFLDRLLNVGLPRIRDFRGVSPKGFDGRGNYTIGVREQLIFPEISYDQIDQIRGMDISIVTTATDDNAGRALLKAFGMPFA-MSRIGKQPIPIPDKVTVKIEGQDVAVKGPKGELQRTIPAKVTVVLDGGEVKVAPVDSTRVGR--QMHGLCRTLVANMVEGVSQGFEKRLEIQGVGYRAQAKGTTLVLNVGYSNPVEIPAPEGIQLAVEGNTNVIVTGIDKELVGNTAATIRAVRPPEPYKGKGIRYAGEYILRKAGKSGK----MPRKVVAIIKLAIEAGKANPAPPIGPALGQHGVNIMMFCKEYNAKTADQAGTVVPVEISVYEDRSFTFILKTPPAAVLIKKAAGIERGSGMSHTEKVGSITQAQLRQIAETKLPDLNTNNVEAAMCIIAGTARNMGVTVAEMKTVPELEPWYVVDADQRLGRLATAIATLRGKNKPFTPHMDTGDFVIVVNAEKIVTGRKEQKLYRRHSGRPGGMKVETFHLQRLPERIIEAVKGMLPHNALGRRLFTKLKVYAGPHPHAQQPELINTIPGMIQETMLNVADNTGARRIQCIRVPGSTGRYAVGDIVAVVKDAPNMVKKSDVVKAVVVRTVIRRPNGTIRFDDNAAVLINDQNPRGTRVFGPVARELREKNFTKIVSLAPEVLMKLSRKQNRRRHQRIRKVGTACPRLAVYRSNNIYAQVIDDTLASSTLDELRQLGNTAAVGKQVARAIAAGIQVVFDRGGKLYHGRVKALADAAREAGLEFMATDTLVKAARTVRSPKVRRVLNQIRGRSYADALIILEFMPYAACEPVRKVLRSAVANAEHNGLDATLYVSTAYADGPLKRFRPRAQGRAYQIQKKTCHITVSVSPE--------------------------------------MAVVSLAD-----LLEAGVHFGHQTRRWNPKMRPYIF-TERNGIHIIDLVQTAGYMEEAYQYVRTAAEQGKRFLFVGTKRQAAAIVAQEALRC-GSYYINQRWLGGMLTNWNTIKTRTDRLKELEEMQSSGIMDRLPKKEGSMRRREMEKLQKYLGGIKTMRRIPDVVITIDQRREMNAVQECRKL--DIPIVSLLDSNC--DPDYADVFVPANDDAIRSIKLIVGKLADAIYEGRH--GQLETMEEF-EYEGAMDVEDFEDDAAEDSAAEDSEVAAEDSMGQKIHPTGFRLGITKT----------HRSRWFASPDRYPALLQEDH--KIRAYLRKN-L-----TNAGLADIAIERKADQIDLEVH-------TSRPGVVVGRGG-----------------------------------------------------------------------SGIEALR----------------------VGLLKELG--------GTRTVRINVVE-------------------------------------VTRVDAEAALIAESIAQQLERRVSFRRVVRQSIQRAQRA-GVEGIKIQISG-----------------------------RLNGAEIARSE-----WTREG---RIPLHTLRANIDYAEHDAKTT-------------------------------------------FGIIGVKVWVF-----------------KGEILPGQEL--ETRPAPT-----------------QQPKRR-SQRRRPTFEDRAAEGMSRFTGPRLKVRRFGDLPGLTRKRPKNTNPPGVHGARKKKSEYAIRLEEKQKVRFNYGVTEKQMVRYMKRARRGSTGLLQMLERLDCIAFRLGMAPTIPAARQAVNHGHICVNGRKVSIPSYECKAGDVITVRDNSRKLIAYAEYPGLF-LPYLEFDKALGKIKEPREAVCAPVNELLVVEYYSRKI------MAERRSRKK-SRET---REASEWQERVVQIRRVTKVVKGGKSLSFRAVVVVGNEKGQAGVGVGKANDVIGAVRKAVADGKKHLITVPLTSSQSLPHRMTGEAGGARVLVMPAAPGTGVIAGGAVRTVLELAGVRNALAKQLGSSNPLNNARATIDALSLLRTFGSVAEERG--IEVSRLYSMSRRRREVPDFGSRLVSMLIRKLMVGKASIASIVYDAMMERTGEPLEVENAIRNATPMVEVKARRVGGATYQVPMEVRDRGTTLALRWLTSFSRARGKSMSKLAELMDAANETGNTIRKREETHRMAEANKAFAHYRYAHNDTISDMLTRIRNARKDAVLPSTKMTRNIARVLTEEGFVGEEDARGKIVLSLKYKGKHRPIIKLQRVSRPGLRVYKNKLPRVLGGIGIAIISTSSGIMTDRDARRSGIGGEVLCYVYM-KAVYWGTGRRKTAIARVRLVPGEGNLINKRGNYLQFRELVAKAPLEVLGLEGSYDILVNAGGGVTGQADAIKLGVARALCDLDPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARQR--KKAKRNVPNGVAIQSTFNNTIVTITDGEAISWASAGSSGFKGAKKGTPFAAQTAAESAARTAIDNGMRQIEVMVSGPGSGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKQKSKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQGRSKYGAKRPKMARIAGVDIPREKRVEVALTYIYGIGPSRARILAGVNPDTRARDLQDEVLRVEDYQVEGDLRREMNIKRLDIGYRGRRHRTGLPLRGQRTRTNARTRKGRTVAGKKK--MALLQRKQELISYQVHETDTGSPEVQVALLTDRINSLTTHLQNKDASRRGLMTIIGKRKRLLGYIQSERYFITRLNIRVKRRKERIGLVVSDKMDKTIVVAVESRISHSKYGKIVVRTHKYKAHDADNRCKQGDRVRIAETRPLSKTKRWEVADILTPGAKS-----------',\n",
       " \"Synechococcus_sp._JA-2-3B'a(2-13)\": 'RKLSKRMQLKVKPIYPEAALLMKETATAKFEAAEVHIRLGIDPKYADQQLRTTVVLPRGTGQIRVAVIARGEKVEAAGADRVGYEDLIEEISKGMDFDLIATPDVMPQVAKLGRLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEYRADRTGIVHLFGKCKFPEALLNLKAVQESIDRNRPSGAKGKYWRTHVSATMGPSIEVDINALRELKMALGILGRKVGMTQIFSPEGLAIPVTVVQAGPCTVTQIKTQATDGYNAIQLGYLPTAEKHLTRAQRGHLTKAGVELLRHLREFRVDQPEAYQLGQKITVEMFSPGQLVDVAGTSIGRGFAGYQKRHHFGRGPMSHGSKNHRRPGSIGAGTTPGRVFPGMRMAGRLGGGRVTTRKLQLVQVDPERDLLVIKGCVPGVEGGLLEITPAKQVGKR--MRLQYTVIPKLKELYTNIHQVPRLEKIVINRGLGEASQNAKALESSIAEISITGQRPVITRAKKAIASFKIRKGMPVGLMVTLRRERMYAFLDRLINVALPRIRDFRGVSPKAFDGRGNYTLGIREQLIFPEINYDSVDQLRGMDISIVTTAKTDEEGRALLKAFGMPFA-MSRIGRRPIALPAKVEIQIDGQHIAVKGPKGQLSRSLPPLITVQQNAQMLTVSRLNDSRPAR--QLHGLCRTLVANMVDGVSKGFERRLELVGVGYRAAIQGSKLVLNVGYSHPVEIPFPPGIQIAVEGNNIIVVSGTDKELVGNTAARIRAVRPPEPYKGKGIRYLGEQVRRKAGKSGKAKK-MAKKLVAVVKLAIQAGKATPAPPIGPALGQHGVNIMAFCKEYNAKTADQAGMVVPVEISIFEDRSFTFVLKTPPASVLIKKALGIESGSSEPHKTKVGSLTRAQLRQIAEQKLPDLNAHDVEAAMKIIAGTARSMGVTVVDMKTLPTLAPWYLVDANQRLGRLAAAVALLRGKHKPFTPHLDTGDYVIVINAEKVVTGKKTQKLYRRHSGRPGGMKVETFQLQRLPERVVEAVKGMLPHTRLGRRQFTKLKVYVGPHPHAQQPQYLNTIPGMIQQTMLTVADNTGARKMMCIRVLGSSGRYALGDIIGVVKDAPNMVKKSDVVKAVVIRTVVRRPDGCIRFDDNAAVIINDGNPRGTRVFGPVARELREKNFTKIISLAPEVLMKTSRKITQRRHRRIRKVGTERPRLAVFRSHHIYAQVIDDTLSASTLDELREFTTEAAVGRSVARALQMGIRVVFDRGGKLYHGRVQALAEAAREAGLQF-MADSSVKAARYIRSPKVRRVLDQIRGRTYADALILLEFMPYAACEPVRKVLRSAVANAEHNGLDPRLVISQAYADGPLKRFRPRAQGRAYPIRKRTCHITIAVRPE--------------------------------------MSVVSLPQ-----LLEAGVHFGHKASRWNPKMRPYIF-TERNGIHIIDLVQTARYLNEAYEYVRDAADRGWRFLFVGTKRQAAGIIAQEARRC-GSYYVNQRWLGGMLTNWATIKTRIDRLKEIEEMESSGLLDRLPKQEASRLRRELARLEKYLGGIKTMRKLPDAVIIVDQRREANAVQECIKL--NIPIISLLDTNC--DPDLSDIFIPSNDDAIRAIKLIVGKLADAIYEGRH--GQLDTVEEDYDYEGAM---DLDDD-ILED-MEDEEEGAEEGMGQKIHPTGFRLGVIKE----------HRSRWFADPARYPALLQEDD--QIRTYLTKQ-L-----SSAGLADIQIERKADQIDLEIR-------AARPGVVVGRGG-----------------------------------------------------------------------SSLETLR----------------------QGLQKELG--------GERTIRINVVE-------------------------------------VTRADAEATLLAENIAQQLERRIAFRRIVRQVIQRAQRA-GVQGIKIQIAG-----------------------------RLNGAEIARTE-----WTREG---RIPLHTLRADIDYAEHLAHTT-------------------------------------------FGIIGVKVWVF-----------------KGEVLPGQER--HEQKFPL-----------------QQPKRR-QQRRRPTFEDRAQEAMSRYTGPRLKIRRFGDLPGLTRKRPKNTNPPGMHGARKKKSEYAIRLEEKQKVRFNYGLSERQMIRYMRKARRGSTGLLQMLERLDCIVFRLGMAPTIPAARQLVSHGHIEVNGQKVTIPSYGCKVGDVITVKNKSRKLVAYAEYPGLF-LPYLEFDKKLGRIKEPREQICAPVNELLVVEFYSRKL------MAE-QRERRSKKNR--EEQPTEWQERVIQIRRVTKVVKGGKKLSFRAVVVVGNERGQVGVGVGKANDVIGAVRKGVVDGKKHLIEVPLTKGLSVPHPMQGAGGGATVLVMPAAPGTGVIAGGAVRTVLELAGVRNSLAKQLGSSNPLNNARATIDALRRMRTFKQVAQDRD--IDVRRLLGMSRRRRDVPDYGSRLVTMLIRKLMMGKASLARILYEALVERTGDPLPLENAVRNATPLVEVKARRVGGATYQVPVEVRERGTSLALRWLVTFSRTRGRTMAKLAELIDAANETGSAIRRREEVHRMAEANKAFAHYRYAHNDTIADMLTRIRNARHESVVPATRMTRSIARVLHEEGFVEKEGEGQAVVLRLKYKGKGRPIIGLKRVSRPGLRVYRNKLPRVLGGIGIAIISTSNGIMTDREARKQGIGGEVLCLVYT-RAVYWGTGRRKTAVARVRLVPGTGKIINDRGDYLQYQELLVKGPLEILGLENSYDILVRAGGGVHGQADAIKLGVARALCEVDPANRPLKEGYLKRDPRAVERKKYGLKARKAPQYSKRMARRR--RKVKKNIPNGVAIQSTFNNTIVTITDGDVVSWASAGSTGFKGAKKGTPFAAQMAAESAARQAMENGMRQIEVMVSGPGAGRETAIRALQTGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERARKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDAAGVKDRKQGRSKYGAKRPKMARIAGVDIPREKRVEIALTYIYGIGLTRSRILTGINPDTRTRDLTDEVLRVEEYQVEGDLRREMNIKRLDIGYRGRRHRLGLPVRGQRTRTNARTRKGRTVAGKKKAPMALLQEKQQIIEYRLHDTDTGSAEVQVALLTSRINQLSQHLQNKDNSRRGLLMMIGRRKRLLNYIASDRFRLARLNIRVMAVREKIGIVVSNKMQKTVVVAVENRVAHPKYGKIVVKTKKFKAHDEENRCQEGDLVRITETRPLSRTKRWQVAEILREVKKL-----------',\n",
       " 'Synechococcus_sp._JA-3-3Ab': 'RKLSKRMQLKVKPIYPAEALLMKETATAKFEAAEVHIRLGIDPKYADQQLRTTVVLPRGTGQIRVAVIARGEKVEAAGADRVGFEDLIEEIGKGIDFDLIATPDVMPQVAKLGRLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEYRADKTGIVHLFGKCKFPEALLNLKAVQESIDRNRPPGAKGKYWRAHVCATMGPSIEVDINALRELKMALGILGRKVGMTQIFSPEGLAIPVTVVQAGPCTVTQIKTQATDGYNAVQLGYWPTAEKHLTRAQRGHLSKAGVELLRHLREFRVDQPEAYQLGQKITVEIFSPGQLLDVAGTSIGRGFAGYQKRHHFGRGPMSHGSKNHRRPGSVGAGTTPGRVFPGLRMAGRMGGCRVTIRKLQLVQVDPERNLLIIKGSVPGVEGGLLEITPAKQVGKR--MRLQYTVIPKLKELYTNIHQVPRLEKIVINRGLGDASQNAKALESSIAEISITGQRPVITRAKKAIASFKIRKGMPVGLMVTLRRERMYAFLDRLINIALPRIRDFRGVSPKSFDGRGNYTLGIREQLIFPEINYDSVDQIRGMDISIVTTAETDEEGRALLKEFGMPFA-MSRIGKRPIALPAKVEVQIEGQQISVKGPKGELSRSLPPLIAVHQEAQTLKVSRVNESRPAR--QLHGLCRTLVANMVDGVSRGFERRLELVGVGYRASTQGNKLILNVGYSHPVEIPFPPGIQIAVEGNNVIVVSGIDKELVGNTAARIRAVRPPEPYKGKGIRYVGEQVRRKAGKSGKAKK-MAKKLVAVVKLAIQAGKATPAPPIGPALGQHGVNIMAFCKEYNAKTADQAGMVVPVEISIYEDRSFTFVLKTPPASVLIKKALGIESGSSEPHKVKVGSLTRAQLRQIAEQKLPDLNARDVEAAMKIIAGTARSMGVTIVDMTTVPTLTPWYLVDANQRLGRLAVVVALLRGKHKPFTPHLDTGDYVIVINAEKVVTGKKTQKLYRRHSGRPGGMKVETFQLQRLPERVIEAVKGMLPHTRLGRRQFTKLKVYAGPHPHAQQPQYIHTIPGMIQQTMLTVADNTGARKLMCIRVLGSSGRYALGDIVGVVKDAPNMVKKSDVVKAVVIRTVVRRPDGCIRFDDNAAVIINDGNPRGTRVFGPVARELREKNFTKIISLAPEVLMKITRKAAQRRHRRIRKVGTERPRLAVFRSHHIYAQVIDDTLSASTLDELREFTTEAAVGRSVARALQAGIQVVFDRGGKLYHGRVQALAEAAREAGLQF-MADPSIKAARYIRSPKVRRVLNQIRGRTYADALILLEFMPYAACEPVRKVLRSAVANAEHNGLDPRLVISKAYADGPLKRFRPRAQGRAYPIRKRTCHITIAVRPE--------------------------------------MSVVSLPQ-----LLEAGVHFGHKASRWNPKMRPYIF-AERNGIHIIDLVQTARYLNEAYEYVREGAEKGWRFLFVGTKRQAAGIIAHEATRC-GSYYVNQRWLGGMLTNWATIKTRIDRLKEIEEMQSSGLLDRLPKQEAARLKRELAKLEKYLGGIKTMRKLPDAVIIVDQRREANAVQECIKL--KIPIISLLDTNC--DPDLSDVFIPSNDDAIRAIKLIVGKLADAIYEGRH--GQLDTVDEY-DYEGAM-DMDLEDD-ILEDVEDEEEGDEQGEMGQKIHPTGFRLGVIKE----------HRSRWFADPARYPALLREDD--LIRAYLTKQ-L-----SSAGLADIQIERKADQIDLEIR-------AARPGVVVGRGG-----------------------------------------------------------------------SGLDALR----------------------QGLQKELG-------SPERTIRINVVE-------------------------------------VTRADAEAVLLAENIAQQLERRIAFRRIVRQVIQRAQRA-GVQGIKIQIAG-----------------------------RLNGAEIARTE-----WTREG---RIPLHTLRADIDYADHIAQTT-------------------------------------------FGVIGVKVWVF-----------------KGEVLPGQER--PEQKVPL-----------------QQPKRR-QQRRRPTFEDRAVEAMSRYTGPRLKIRRFGDLPGLTRKRPKNTNPPGMHGARKKKSEYAIRLEEKQKVRLNYGVSERQMLRYMRKARRGSTGLLQMLERLDCIVFRLGMAPTIPAARQLVSHGHIEVNGRKVTIPSYGCKVGDVITVRNKSRKLVAYAEYPGLF-LPYLEFDKKLGRIKEPREQICAPVNELLVVEFYSRKL------MAE-QRERRSKKSR--EEQTSEWQERVIQIRRVTKVVKGGKKLSFRAVVVVGNERGQVGVGVGKANDVIGAVRKGVVDGKKHLIEVPLTKGLSVPHPMRGSGGGATVLVMPAAPGTGVIAGGAVRTVLELAGVRNSLAKQLGSSNPLNNARATIDALRRMRTFEQVAEDRG--IDVRRLFGMSRRRRDVPDYGSRLVTMLIHKLMMGKASLARILYDALVERTGDPLPLENAVRNATPLVEVKARRVGGATYQVPVEVRERGTSLALRWLVTFSRTRGRTMSKLAEIIDAANETGNAIRRREEVHRMAEANKAFAHYRYAHNDTIADMLTRIRNARHESVVPATRMTRSIARVLHEEGFVDKEGEGHALVLRLKYKGKGRPIIGLKRVSRPGLRVYCNKLPRVLGGIGIAIISTSNGIMTDRDARKQGIGGEVLCYVYMSRAVYWGTGRRKTAVARVRLVPGTGKLINDRGDYLQYQELLVKAPLELLGLENSYDILVRAGGGVHGQADAIKLGVARALCEVDPANRPLKEGYLKRDPRAVERKKYGLKARKAPQYSKRMARRR--RKVKKNVPNGVAIQSTFNNTIVTITDGDVVSWASAGSTGFKGAKKGTPFAAQMAAESAARQAMENGMRQIEVMVSGPGAGRETAIRALQTGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERVRKTKAPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDAAGVKDRKQGRSKYGAKRPKMARIAGVDIPRDKRVEIALTYIYGIGLTRSRILTGINPDTRTRDLTDEVLRVEEYQVEGDLRREMNIKRLDIGYRGRRHRLNLPVRGQRTRTNARTRKGRTVAGKKKAPMALLQEKQEIIEYRLHDTDTGSAEVQVALLTSRINQLSQHLQNKDNSRRGLMMMIGRRKRLLNYIASDRFRLARLNIRVMPAREKIGVVVSNKMQKTVVVAVENRVAHPKYGKIVVKTKKFKAHDEENRCQEGDLVRITETRPLSRTKRWRVAEIIREVKKL-----------',\n",
       " 'Chamaesiphon_sp._PCC_6605': 'KKYSRRYQLKIETRYPLEAMLLRETATAKFESAEVHVRLGIDPKYTDQQLRTTVALPKGTGQIRVAVIARGEKVEAAGADLAGSEELIAEIQGGMDFDLIATPDMMPQIARLGRVLGPRGLMPSPKAGTVADLGAIEFKAGKLEYRADKSGIVHIFGKAAFDEDLVNLKAFQESIDRNRPAGAKGRYWRSYVSATMGPSIEVDINSLRDMKMSIGILGTKLGMTQIFDETGRAIPVTVVQAGPCTVTQIKTKATDGYGSVQIGFDEVKEKALNRPELGHLKKSEAPPLRHLQEFRVDNPDNWTLGQEIKPDIFTVGQIIDVTGNSIGKGFAGWQKRWNFARGPMSHGSKNHREPGSTGAGTTPGRVYPGKKMAGRLGGKRITIKKLVVVQIDVERNVLLIEGSVPGKPGALLRIVPEVKVG-K--MQLKYQEIPKLEQFYTNLLQVPKVIKVTVNRGLGEAATNAKALESSIAEIAITGQKPVVTRAKKAIAGFKIRAGMPVGVMVTLRGDRMYAFLDRFINLTLPRIRDFRGISPKSFDGRGNYSIGIREQLIFPEIEYDQIDQIRGMDISIITTADNDEAGRALLKEMGMPFRSMSRIGKRPIPLPAKVTVTLDGQSVVVKGPKGELSRVLPSGVVVMMEGDTVVVNRANESRPAR--QQHGLCRTLVANMVEGVSTGFQKKLEIQGVGYRAAVDGRILTLNVGYSLPVKIDPPAGITIEVEKNTSVTISGIDKELVGNTAAKIRDVRPPEVYKGKGIRYEGEVVRRKAGKSGKK---MAKKVVAIVKLALNAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTAEQTGTVIPVEISIFEDRSFTFVLKTPPASVLIKKAAKIETGSNEPNKKKVGQITTTQLQEIAQIKLPDLNANDIDAAMKIVAGTARNMGVAIVDVKTLPTADAWYIIDADQRLGRLASQIAVLRGKNKPFTPHIDTGDFVIVINAEKVVTGKKTQKIYRRHSGRPGGMKTEVFKLQRIPERIIEAVKGMLPKNSMGKHLFTKLNVYAGTHPHAQQPELINTTPGMIQESYLTVADNSGARKIMCIRVLGAGNTYGVGDIIAVVKDAPNMVKKSEVVRAVIVRTKMRRESGSIRFDDNAAVLVNEGNPRGTRVFGPVARELRDKSFTKIVSLAPEVLMKLTRKEAQRRHRRVRGVGSERPRLAIFRSLHIYAQVIDDTLAASSLEELKSLGNEAAVGKLIARAKAKGILVVFDRGGNLYHGRVQALADAAREAGLNFMAIDNKIKAAKYIRSPKVRRVLDQIRGRSYREALIILEFMPYKACEEIIKVLRSAVANAEHNGMNPALVVSTAFADGPLKRFRPRAQGRAYKIRKPTCHITVAVAE---------------------------------------MPVVTLAQ-----MMESGVHFGHQTRRWNPKMAPYIF-TARNGVHIIDLVQTAQLMEEAYTYMRTAAEQGKRFLFVGTKRQAAGIIAQEASRC-GAAYINQRWLGGMLTNWTTIKSRVERLKDLERREASGALDLLPKKEASVLRREMAKLQKYLGGIKNMRKVPDIVIVVDQRREYNAMLECQKL--NIPIVSLLDTNC--DPDLVDVAIPANDDAIRSVKLILGKLADAIYEGRH--GQLEGADPYDGYEDVEDDFD-EEG-FAEA--ADEG---SEGMGQKIHPVGFRLGITQE----------HRSRWYADTRKYSEVLQEDY--IIRQHIEKD----ATLATAGISKVRIERKADQIELEIH-------TARPGVVVGRQG-----------------------------------------------------------------------SGIEVIR----------------------QALLKKLG--------TDRQIRINVIE-------------------------------------IAKVDADAALIAEFIAQQLSRRVSFRRVVRQAIQRAQRA-EVKGIKIQVGG-----------------------------RLNGAEIARTE-----QAREG---SVPLHTLRADIDYASKTADTT-------------------------------------------YGIIGVKVWVF-----------------KGEIIPGQEN----LEVPS----------------AAQPKRRQQQRRRQQFEDRSDEGMSRYRGPRLRIRRLGPLPGLTRKDARRAYPPGQHGQRKKRSEYAIRLEEKQKLRMNYGVTEKQMLRYVRKARRGSTGLLQLLERLDNTVFRLAMAPTIPGARQLVNHGHVTVNGRTVDIASYQCRPGDVIGVRNRSRKLVENLQYPGLANTPHLEFDKNLGKVTGEREWVALQVNELLIVEYYSRQA------MAQEQQERRKGKKE---EKEITYQERVIQIRRVSKVVKGGKKLSFRAIVVIGNEKGEVGVGVGKASDVIGAVRKGVADGKKHLITIPLTKSNTIPHPIVGGGGGANVMMRPAAPGTGVIAGGAVRTVLELAGVKNILAKQLGSGNPLNNARAAVNALKTLRSFNQVAEDRG--VTVADLFAMSRRKRQVPDYNSRLVNMMVRRIMYGKKSVARIIYDAMIERTGDPLEFEKAVKNATPLVEVKGRRVGGATYQVPMEVKERGTALAMRWLINYSRSRGKSMAKLAELMDAANETGSAIRKREETHRMAEANKAFAHYRYMAHDTISDMLTRIRNAKHQTTLPATKLTRSIATVLRQEGYVEEEGEGDKLVLTLRYKGKNRPIIALQRVSKPGLRVYSNKLPKVLGGIGIAVISTSSGIMTDREARKQGLGGEVLCYVWM-RAVYWGTGRRKSSIARVRLVPGTGKLVNGRGEYFNFQSYIARSPLETLGLESEYDILVNAGGGLTGQSDAVKLGVARALCELDPNNRPLKEGYLTRDPRCVERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNTIVTISDGDCISWASSGSSGFKGAKKGTPFAAQTAAESAGKRAMDQGMRQLEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRARKVKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIFGIGLTRAKVLTGVNPDTRVKDLSEDVLRVEDYQIEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMTLTQRKQEIAAYQLHATDTGSSDLQVAMLTERINRLSEHLKNSDASRRGLLKMIGQRKRLLAFIQNQRYQLASLGIRRMAVKERVGIVVSTKMQKTAVVSVENRSAHGKYGKVVVRTKKYKVHDEENQCKDGDRVRITETRPLSRTKRWTLAEILSTKA-------------',\n",
       " 'Halothece_sp._PCC_7418': 'KQPSRRMRLKVEDRYPLSALLLKETATAKFETAEAHVRLGIDPKYSDQQIRTTVTFPKGTGQVRVAVITRGEKVEAAGAEVVGSEELIQEIQNGMDFDVMATPDMMPQVAKLGRVLGPKGLMPSPKGGTVTELEAIEFKAGKQEFRADRTGIVHVFGKVSFSEDLLNLKALQETLDRNRPSGAKGKYWRSYVSASMGPAIEVDIAGLRDYK-----MGRKLGMTQIFDEEGIAIPVTVVEVGPCIVTQIKTPSTDGYSAVQLGYKTVKEKALTKPELGHLKKVNTPSLRHLKEYRVDDPESYQLGQSLNADLFSEDQLVDVRGTSIGRGFAGYQKRHGFARGPMSHGSKNHRLPGSIGPGTTPGRVYPGSRMAGRMGGKTVTTRKLKVVRVDSEKNVLLIKGTIPGKPGGLLSITPSNLVGAKASMRLKYDTIPQLKEFYSNVHEIPKLVKVTVNRGLGEASQNAKALELSVQELAITGQKPVVTRAKKAIAGFKVRQGMPVGVMVTLRSERMYAFLDRLINVALPRIRDFRGISPRSFDGRGNYNLGLREQILFPEIEYDNIDQIRGMDIAIITTAKTDEEGRALLKAMGMPFRQMSRIGKLPIPIPQKVEITVDGKQVKVKGPKGELEQTMPDFVTIEQEDNQILVKRVNDSRKAR--ERHGLARTLVANMVHGVSQGFERRLQIQGVGYRAQVQGRNLILNVGYSHPVEIAPPEGIQINVEKNTEIIISGINKEIVGNLTAKIRAVRPPEPYKQKGIRFMGEQVRQKAGKAGKK---MAKKVVALIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTADQAGMVIPVEISVYEDRSFTFTLKTPPAAVLLKKAAGIEKGSSEPNRQTVGSITRAQLQEIAETKMPDLNSNDIEAAMKVVEGTARNMGITVKDVQTLPTLEKWYVIDAGYRLGRLASEIAIIRGKNKPYTPHMDTGDFVIVINADKVVSGNKNEKLYRRHSGRPGGMKVETFQLQRIPERIIEAVKGMLPKNRLGRQLATKLKVYAGPHPHAQKPVI---TPSMIQETILNVADNSGARKIQCLRVMRAGNPYGIGDIIGVVKDAPNMVKKSDIIRAVIVRTRERRDSGSIRFDDNAAVIINDNNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKNNRKELKQRHRRIRKVGTERPRLAVFRSHHIYAQVIDDTLAASTVDNLRQLTTEAGVGTLVARALEQGIQVVFDRGGNIYHGRVRSLADAAREAGLEFMPVDTSTKAARYIRSPKVRRVLDQIRGRSYREALIMLEFMPYRACDPVLKALRSAAANAEHNGLDRALVISQAYADGPLRRFRPRAQGRAYQIRKPTCHITIAVAPE--------------------------------------MSVITLRE-----LLESGVHFGHQTRRWHPAMRPYIY-TARNGVHIIDLVQTAQLMEEAYSYLQDASAKGQKVLFVGTKRQAAGIIAQEASRC-GAFYVNQRWLGGMLTNWETIKTRVERLKELERLKESGAMARRPKKEAAVLGRELEKLQKYLGGIKMMRRLPDIVVIVDQRREHNAIMECQKL--GIPIISMLDTNC--DPALADLPIPANDDAIRSVKLIVGKLANAIYEGRH--GELDMEAEYDEFDEGI---TYQEEDDEEESEADSEMAVGESMGQKIHPTGLRLGITKE----------HLSCWYADSKRYPELLQEDF--RIRNYIDQT-L-----NNAGISATHIERKADQISLQIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIESLR----------------------VGLQQLLG--------GNRQISINVSE-------------------------------------VSQVDADAVLIGEYITQQLERRVSFRRVVRQAMQRAQRA-EVQGIKIQISG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYRTAETI-------------------------------------------YGILGVKVWVF-----------------KGEILPEDEE----QPSEV----------------GPTPRR---QPRRQKFEDRSEEKMSRYRGPRLRVRRLGDLPGLTRKSPRKAYPPGQHGQRRKRSEYAIRLEEKQKLRYNYGVTEKQLMRYMKKARRGSTGLLEMLERLDNTVFRMGMAGTIPAARQLVNHGHILVNGRVMDIASYQCRPGDVIAVRDRSRQLVENMEYPGLANLPHLEFDKTYGKVNGEREWVALQVNELLVVEYYSRKV------MAK-----R-NQSK---KEDSQFQERVIQIRRVSKVVKGGKKLSFRAIVVVGNDRGQVGVGVGKAGDVIGAVRKGVADAKKQLVDVPITKGNSIPHRADGFAGGAKVMVRPAAPGTGVIAGGAVRTVLELSGVKNILAKQLGSSNPLNNARAAIDALASLRTFKEVANERG--VPLEQIFSMSRRKSPVPDYNSRLVSMMIRRLMMGKKSVARILYDAFVEKTNDPLEFEQAVRNVTPLVQVKARRVGGATYQVPMEVRERGTSLALRWLSQYSRSRGKTMARFAEIIDAANETGSTIRKREETHKMAEANKAFAHYRYMATDTISDMLTRIRNAQHETTVPYNKMNRNIARVLKEEGFISEEGEGKKVLISLRYQGRNRPIIRLTRVSRPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARKRGIGGEILCYVWMNRVVYWGTGRRKSSVARVRLVPGNGQIVNTRGDYFNYNPYIIKSPLETLGLENDYDILVNAGGGLTGQAEAVRLGVARALCKLDPENRPLKEGYLSRDPRAKERRKYGLKARKAPQYSKRMARKKSSRKTKRNVPNGVAIQSTFNNTLVTISDGDVISWASAGSSGFKGAKKGTPFAAQTAADQAGRRATEQGMRQVEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERTQKTKSPALKCPQRRGVCTRVYTATPKKPNSALRKVARVRLTSGYEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIVRGTLDTTGVKDRRQGRSKYGTKRPKMARIAGVDLPRDKRVEIALTYIYGIGFSRARILTGVNPDTRAKDLDVEELRIENYQVEGDLRREMNIKRLDIGYRGRRHRSGLPVRGQRTRTNARTRRGRAIAGKKKPPMSLTQKKQEIINYQVHETDTGSTEVQIAILTKRITRLTEHLKNKDSSRRGLLKLIGHRRSLLSYLQDQRYQLIRLGIRGMTTKERIGVVVSNKMQKTVVVAVENRAPSRKYGKTVVTTKRYKVHDENNDCQEGDRVRIVETRPLSKTKRWTVASIIESASGQ-----------',\n",
       " 'Dactylococcopsis_salina_PCC_8305': 'KKLSRRMRLKVEDRYPQAALLLKATATAKFETAEAHIRLGIDPKYSDQQIRTTVAFPKGTGQVRIAVITRGEKVEAAGADVVGSEELIQEMQNGMDFDVMATPDMMPQVAKLGRVLGPKGLMPSPKGGTVTELDAIEFKAGKQEFRADRTGIVHVFGRVSFSEDLLNLKALQETLDRNRPSGAKGKYWRSYVSASMGPAIEVDIAGLRDYK-----MGRKLGMTQIFDEQGTAIPVTVIEVGPCTVTQIKTPSTDGYSAVQLGYRNTKEKLLTKPELGHLKKADAPNLSHLKEYRVQDTSEYELGQSLNAELFSEVELVDVRGTSTGRGFASNHKRHGFARGPMTHGSKNHRLPGSIGPGTTPGRVYPGKRMSGRMGGKTVTTRKLKVVRVDSEKNVILVKGSIPGKSGGLLSITPSNIVG-KA-MRLKYETIPQLQEFYSNVHEIPKVVKVTVNRGLGEASQNAKALELSVKEIAITGQKPVVTRAKKAIAGFKVREGMPVGVMVTLRDQRMYAFLDRLINVSLPRIRDFRGISPRSFDGRGNYNLGLREQILFPEIEYDSIDQIRGMDIAIITTAKTDEEGRALLKAMGMPFRQMSRIGKLPIPIPKKVEITLDGKLVKVKGPKGELEQTMPDFVTIEQEEDRILVHRVNDSRKAR--ERHGLARTLVANMVHGVSQGFERRLQIQGVGYRAQVQGRNLILNVGYSHPVEIVPPDGIQVAVEKNTEIIINGINKELVGNLTAKIRAVRPPEPYKGKGIRFLGEQVRQKAGKSGKK---MAKKVVALIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTADQAGMVIPVEISVFEDRSFTFTLKTPPAAVLLKKAAGIEKGSSEPNRQTAGAVSREQLREIAETKMPDLNANDIEAAMKIVEGTARNMGITVKDLQTQPTLEEWYVIDAGYRVGRLATKIAIIRGKNKPFTPHMDTGDFVIVINAEKVLSGNKNDKLYRRHSGRPGGMKVETFQLQRIPERIIEAVKGMLPKNRLGRQIFQKLKVYSGPHPHAQKPII---TPSMIQETILNVADNSGARKIQCLRVMRAGNAYGIGDIIGVVKDAPNMVKKSDVIRAVIVRTRERRDSGTIRFDDNAAVLINDNNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKNNRRELKQRHRRIRKVGTQRPRLAVFRSHHIYAQVIDDTIAASTVDNLRQLTTEASVGNLVARALEQGIQVVFDRGGNIYHGRVRSLADAAREAGLEFMPVDTTTKAARYIRSPKVRRVLDQIRGCSYREALIMLEFMPYRACDPVLKVLRSAAANAEHNGLDRALVISKATADGPLRRFRPRAQGRAYQIRKPTCHIMIAVAPE--------------------------------------MAVISLRE-----LLESGVHFGHQTRRWHPAMRPYIY-TARNGVHIIDLVQTAQLMEEAYSYLQDASANGQKVLFVGTKRQAAGIIAQEASRC-GAFYVNQRWLGGMLTNWETIKTRVERLKELERLKESGAMARRPKKEAAVLGRELEKLQKYLGGIKMMRRLPDIVVIVDQRREHNAIMECEKL--GIPIVSMLDTNC--DPGLADIPIPANDDAIRSVKLIVGKLADAIYEGRH--GQLDQEAEYDELEEGIGEYEYEEE---EEDSEMATVGSGEEMGQKIHPTGLRLGITKE----------HQSCWYADSRRYPELLQEDY--KIRQYVQKT-L-----NNAGISATRIERKADQITLQIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIESLR----------------------VGLQELLG--------GNRQISINVSE-------------------------------------VSQVDADAVLIGEYITQQLERRVSFRRVVRQAMQRAQRA-EVKGIKIQISG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAATI-------------------------------------------YGILGVKVWIF-----------------KGEILPEDEE---QQQADL----------------NVVPRR---QPRRQKFEDRSEENMSRYRGPRLRIRRLGDLPGLTRKSPRKAYPPGQHGQRRKRSEYAIRLEEKQKLRYNYGVTEKQLMGYMKKARRGSTGLLEFLERLDNTVFRMGMAGTIPAARQLVNHGHILVNGRVMDIASYQCRPGDVIAVRDRSRQLVENMEYPGLANLPHLEFDKTYGKVNSEREWVALQVNELLVVEYYSRKV------MAKRNQGKK--------GDDSQYQERVIQIRRVSKVVKGGKKLSFRAIVVVGNDRGQVGVGVGKAGDVIGAVRKGVADAKKQLVDIPITKGNSIPHRADGFAGGAKVMIRPAAPGTGVIAGGAVRTVLELSGVKNILAKQLGSSNPLNNARAAIDALSSLRTFKEVADERD--VSLEQIFSMSRRKPPVPDYNSRLVSMMIRRLMMGKKSVARILYDAFVEKTNDPLDFEQAVRNVTPLVQVKARRVGGATYQVPMEVRERGTSLALRWLSQYSRNRGKTMARFAEIMDAANETGSTIRKREETHKMAEANKAFAHYRYMPTDTISDMLTRIRNAQHETTVPFNKMNRNIARVLKEEGFISEEGEGKKVLISLRYQGRNRPIIKLQRVSRPGLRVYSNKLPRVLGGIGIAIVSTSSGIMTDREARKRRIGGEVLCYVWMNRVVYWGTGRRKSSIARVRLIPGTGEIVNNRGDYFNYNPYVVKAPLETLGLESDYDILVNAGGGLTGQAEAVRLGVARALCKLDPENRPLKEGYMSRDPRSKERKKYGLKARKAPQFSKRMARKKSSRKTKKNVPSGVAVQSTFNNTLVTISDGDVISWASAGSSGFKGAKKGTPFAAQTAADQAGRRATEQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERTQKTKSPALKCPQRRGVCTRVYTATPKKPNSALRKVARVRLTSGYEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIVRGTLDTTGVKDRRQGRSKYGTKRPKMARISGVELPRDKRVETALTYIYGIGLSTSQILTGINPDTRAKDLDVEELRIENYQVEGDLRREMNIKRLDIGYRGRRHRLGLPVRGQRTRTNARTRRGRAIAGKKKPPMSLTQRKQEIINYQVHETDTGSTEVQIAILTKRITRLTEHLKNKDSSRRGLLKLIGHRRSLLSYLQNERYQLIRLGIRGMTTKERMGVVVSNKMQKTVVVAVENRSPSRKYGKTVVTTKRYKVHDENNDCQEGDRVRIRETRPLSKTKRWTIAAIIESASGQ-----------',\n",
       " 'Pseudanabaena_sp._PCC_7367': 'KKQSKRFLAKVEERYPLEALLLKETATAKFEAAEAHIRLGIDPKYADQQLRTTVVLPQGTGQVRVAVIARGEKVEAAGADIVGSEELIEDISKGMDFDVIATPDVMPQVAKLGRLLGPRGLMPSPKGGTVTDLEAVDFKGGKIEFRADRTGIVHVFGKANFEQDLLNLKALQETIDRNRPSGAKGKYWQSYLSASMGPAIELDVSALRDLKMSVGILGTKLGITQVFDDEGNVVPVTVVQAGPCKITQIKTEVKEGYNAIQVGFGNTREKLLTKPELGHLAKSQTSPVKHLHEYRVDDTGTYDLGQEIDVSIFNDGDLIDVSGVSVGKGFSGYQKRHNFRRGPMSHGSKNHRQPGSTGAGTTPGRVYPGKKMAGRLGGVKVTIRKLTVFQVDSEKNILLVRGSVPGKPGALLNITPANIVGKK--ARFLYTQAPKLEQFYTNSHQVPKFNKVVVNRGLGEASQNAKALESSITEISITGQKPVVTRARKAIAGFKLREGMPVGVMVTLRRERMYAFLDRLINFSLPRIRDFRGISAKSFDGRGNYTLGVREQLIFPEISYDSIDQIRGMDISIITTANTDEEGRALLAAMGMPFR-MSRIGKLPIPIPKKVQITIAGQQVTVKGPKGELQRTLTDYVDIEQEGEEILVKQRRESRICR--QQHGLSRTLIANMVEGVTNGFERRLQIQGVGYRAQLQGKNLILNVGYSNPVQVEPPDGIQIQVENNTNVIVTGINKEVVGNTAAQIRAVRPPEVYKGKGIRYLGENVRRKAGKAGKK---MARKVVAIIKLAIQAGKANPAPPIGPALGQHGVNIMMFCKEYNAKTADKAGTVIPVEISVYEDRSFTFILKTPPASELIKKAAGINSGSSEPNRVKVGSITKAQLKEIAETKMPDLNANDIDAAMNIVAGTASNMGVTIAEHRTVPDVERWYVVDAGKRLGRLAVVIAILRGKNKPYTPHADNGDFVIVLNADQVVTGRKTQKLYRRHSGRPGGMKEETFKLIRIPERIIEAVKGMLPKNSLGRQLFRKLKVYKGDHPHAQQPQLVNTIPGMIQETILNVADNSGAKKLLCIRVIAGGNKVGVGDIVATVKDAPNMVKKSDVVKAVIVRTRINRDSGSIRFDDNAAVIVNDGNPRGTRVFGPVARELRDKNYTKIISLAPEVLMKGTRKQARSRHKRIRKMGTAKPRLSIFRSNHIVAQLIDDTLAASTLEEIKQLTNAAAVGELIARAIAKGLNVVFDRGGKLYHGRVRALAEAARSGGLEFMLLETDVKAAKYVRAPKARRVLDQIRGRSYREALVILEFMPYRACEPITKVLRSAVANAEHNGMNPVLVINQAFADGPFRRFRPRAQGRAFKIKKPTCHITITVKPE--------------------------------------MPVVSLAE-----LLESGVHFGHQTRRWNPKMDSYIF-TERNGVHIIDLVQTAQCMEQAYQYMRQASEQGKKVLFVGTKRQAAGIIAQEAARC-GGHYVNQRWLGGMLTNWTTIKTRIERLKDLLRREESGAMDRMPKKEAARARRELDKLQKYLGGLKSMRKPPDIVVIVDQKREYNAVQECQKL--KIPIVSLLDTNC--DPDTADIPIPANDDAIRSVKLIISRLADAIYEGKH--GQIDEV-DY-DSAGNPEDFAYVETAEEAATAVQETATVEEAMGQKVHPNGLRVGVIRP----------HLSRWFADGDRYPILASEDF--EIRKYITKT-L-----DNAGVADVRIERKADQVDIEIR-------TARPGVVVGRGG-----------------------------------------------------------------------SGIDQLR----------------------QGLLKHLGKVGTKEKTGTSQIRINVVE-------------------------------------VARVDAEAALIGEYIAQQIERRVSFRRVVRQAIQRAQRA-GVEGIKVQVGG-----------------------------RLNGAEIARTE-----WVREG---RVPLHTIRADIDYAYRTASTV-------------------------------------------YGIIGVKVWIF-----------------KGEIEPGQE-----AAAPA----------------QPQPRRRQQQRRRPQFEDRGAEGMARYRGPRLRIRRLGELPGLTRKSPRHNYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLVRYMKKARKGSTGLLEQLERLDNTVFRMGMAPTIPGARQLVNHGHITVNGKVVSIPSYQCVPGEEIGVKNNSRKMVENLQYPGLANVPHLEFDKKLGKVNSEREWVALQINELLVVEYYSR--------MAERERKGK-RKGR--AEKDSEWQERVVQVRRVTKVVKGGKKLSFRAIVIIGNEKGQVGVGVGKAGDVIGAVKKGVADGKKHLVTVPLTKSNSIPHPSYGVGSGASVMIRPASPGTGVIAGGAVRTVLELAGVKNILAKQLGSKNPLNNARAAVNALDSLRTFKEVAQSRD--LTLEQLFSMSRRKRPTPDYNSRLVSMLIKRMMRGKLSVSNLVYKAFVERTGAPLEFDRAIRNATPLVEVKARRVGGATYQVPMEVRDRGVALALRWLVGYSRSRGRGMVKLAEIMDAANETGATIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHSVTIPATKMTNNIARVLKQEGYISEEGEGDRLMVALRYRGKHRPIIKLQRVSRPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRKGVGGEVLCYVWMSRVVYWGTGRRKRSVARVRVVPGSGNVVNGKGDYLQFNPYIAKAPLEVLGLENDYDILVNAGGGVTGQAEAIRMGVARALCELAPENRPLKEGYLTRDPRAKERKKYGLKARKAPQFSKRMARRRGSRKQKRNVPNGIAIQSTFNNTIVTITDGDVISWASAGSSGFKGAKKGTPYAAQTASEGAARRAIDNGMRQVEVMVSGPGSGRETAVRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERTQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQGRSKYGAKRPKMARIAGVDIPRDKRIEVALTYIYGIGLTSSRILTGINPDTRSKDLSDEVLRVENYQVEGDLRRELSIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALLQEKQEIFGYQLHPTDTGSSDVQVALLTERINQLSAHLKHKDSSRRSLLKMIGQRKRLLAYVRDNHYKLIRLGIRGMAVKEKVGQVVSTKMQKTVVVAVENRSAHPKYGKIVVQTKRFKAHDGEEKCQEGDRVRIRETRPLSRTKRWEVIEILSTISAQ-----------',\n",
       " 'Synechococcus_sp._PCC_7502': 'KKASKRVRAKVELRYPSEALLLKDTATAKFESAEAHVRLGIDPKYADQQLRTTVALPKGTGMVKVAVIARGEKVEAAGADISGSEDLIDEISKGLDFDLIATPDMMPQVAKLGRLLGPRGLMPSPKGGTVTDLAAIEFKGGKIEFRADRTGIVHVFGKANFTEDLLNLKALQEVIDRNRPSGAKGRYWRSYVSSTMGPAIEVDISSLRDLKMSVGILGTKLGMTQVFDSNGNAVPITVVQAGPCTITQVKTPSKEGYSAIQIGYGQTRAKLVSKPELGHLKKSDSLPVKHLQEYRVADTSSYHLGQELNVSQFSEGQVVDVIGTSIGKGFAGYQKRHNFKRGPMAHGSKNHRAPGSTGAGTTPGRVYPGKRMAGRLGGKQITTKKLILVQVDTEKNLLLIKGALPGKPGALVNIVPAKVVG-KRKMRFKYSQAPKLEQFYTNVHQVPKFEKVVINRGLGEAAQNSKSLEASLVEIAIAGQKPVVTRAKKAIAGFKVRQGVPVGMMVTLRRERMYAFLDRLINFSLPRIRDFRGVSPKSFDGRGNYTLGIREQLIFPEISYDSIDQVRGLDISIITTAKTDEEGRALLKAMGMPFRQMSRIGKRPINIPPKVSVAIAGQDVTVKGAKGELHRTLSPKIEIVQDGNNLQVNRRDDSRSAR--QLHGLSRTLVANMIEGVSNGFQTRLEIQGVGYRAQVQGTTLVLNVGYSIPVQINPPSGIDIKVEGTTNVIVSGINKEVVGNVAAQIRSVRPPEVYKGKGIRYAGEVVRRKAGKTGKK---MAKKVTAIIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNARTADQVGTVIPVEISVFEDRSFSFVLKTPPASILIQKAAGLSKGSGEPNRKKVGSITRNQLREIAQIKLPDLNANDIDAAMNIIEGTAKNMGVTITAQRTLPNPDRWYVVDAGQRLGRLAVAIAILRGKNKPYTPHLDTGDFVIVINAEKVVTGKKTQKVYKRHSGRPGGMKEETFKLIRVPERIIEAVKGMLPKNTLGRQLFTKLNVYAGAHPHAQQPQLINTIPGMIQESYLNVADNSGAKKLLCIRVLAGGSKFAVGDIIATVKDAPNMVKKSDVVRAVVVRTRIRRDSGSIRFDDNAAVIINEGNPKGTRVFGPVARELRDKNFTKIISLAPEVLMKTSHKEARSRHKRIRKLGTERPRLSVFRSNHIVVQIIDDTLAASTLEELKTLTNEASVGELIARAIAKGIQVVFDRGGNLYHGRIEALAEAARASGLVFMILSENVPAVKFIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPILKLLRSAAANAEHNGFDRALVVSQAYADGSLKRFRPRAQGRAYQILKPTCSITITVKPE--------------------------------------MSVVSLAQ-----LLESGVHFGHQTRRWNPKMEPYIF-TERNGVHIIDLVQTAQYMEEAYAYMRNASEQGRKVLFVGTKRQAAGIVAQEALRC-GSHFVNQRWLGGMLTNWTTIKTRIDRLKDLERRDESGALDRLPKKEASMLRREMEKLQKYLGGLKAMRKIPDVVVIVDQKREYNAVQECQKL--GIPIVSILDTNC--DPDTVDVPIPGNDDAIRSVKLIISKLADAIYEGRH--GQV------DEYDAVAADSDYDPD-YVDDIADEVT---EEDMGQKIHPTGLRLGIVQP----------HLSKWFADTDRYPILAAEDY--EIRRHIEKT-L-----SSAGISRILIDRKADQADIEIR-------TARPGVVVGRQG-----------------------------------------------------------------------TGIDQLR----------------------KGLFEYLTKVGSIKKSGTSQIRINVVE-------------------------------------VAKVDSEAPLIAEYIAQQLERRVSFRRIVRQAIQRAQRA-GIEGIKVQIGG-----------------------------RLNGAEIARSE-----WVREG---RVPLHTLRADIDYSYKTAQTI-------------------------------------------YGVIGVKVWVF-----------------KGEIIPGREE----STLQQ-----------------EQPRRR-QQRRRTQFEDRSGVGMSRYRGPRLRIRRLGELPGLSRKQPKHAYPPGQHGQRKKKSEYGVRLEEKQKLRFNYGLTERQLLRYVRQARRGSTGLLQLLERLDNTVFRTGMAPTIPAARQLVRHGHITINGKVVSIPSYGCRPGEVLGVKDSSRKLVENLEFPGLANTPHLEFEKKLAKINGEREWIALQVNELLVVEYYSRQADAGTPEGGREDKRGKRNARG---EKESEWQERVVQIRRVTKVVKGGKKLSFRAIVIVGNEKGQVGVGVGKASDVIGAVKKGVADGKKHLIEVPITKSNSIPHPSHGVGGGAQVMIRPASAGTGVIAGGAVRTVLELAGVKNILAKQLGSGNPLNNARAAINALSTLRTFSEVAKARD--LTLENLYTMSRRKRITPDYSSRLVNMLIRRLMVGKFSLAGIVYDAFVERTGPAIEFDRAVRNATPLVEVKARRVGGATYQVPMEVRDRGVALALRWLVQYSRSRGRSMSKLAELMDAANETGSAVRKREETHRMAEANKAFAHYKYMANDTISDMLTRIRNAKNQVTIPATKMTQSIAQVLRQEGYIDEEGEGDRLMLSLKYKGKNRPIIMLKRVSRPGLRVYSNKLPKVLGGIGIAIVSTSKGIMTDREARKQGVGGEILCFVYMSRAVYWGTGRRKTAVARVRLVPGSGTIVNGKGDYLQFNAYIVKAPLETLGLENEYDVIVNAGGGVTGQADAIKLGVARALCELSPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARRRTGRKQKRNVPSGVAIQSTFNNTIITIADGDVISWASAGSSGFKGAKKGTPFAAQTAAEGAARRAMDQGMRQIEVMVSGPGSGRETAVRALQTGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERNQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKGRMQGRSKYGAKRPKMARIAGVDLPRDKRIEIGLTYIFGIGLPSAQILTNVNPDTRVKDLSDEILRVENYQVEGDLRRELNIKRLDIGYRGRRHRLGLPLRGQRTKTNARTRKGRTVAGKKKAPMALLQRKQELLSYQIHPTDTGSSQVQVAMLTERINQLSTHLKHKDSSRRSLLKLIGQRKRLLAYIKSTDYKLIRLGVRGMAIKEKVGLVVSNKMDKTVVVAVENRSSHGKYGKIVVQTRRFKAHDELNACKEGDQVRIQETRPLSRTKRWQVVNVINSTST------------',\n",
       " 'Planktothrix_agardhii_CYA126/8': 'RQESRRLRLKVEDRYPLEAILLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVSLPKGTGQIRVAVLAKGEKVEAAGADVVGSEDLINDIQKGLDFDLIATPDMMPQVAKLGRLLGPKGLMPSPKGGTVTDVAAIEFKAGKLEFRADRTGIVHVFGKTAFSADLLNLSTLQECIDRNRPSGAKGRYWRSFVASTMGPSIEVDITALRELKMSVGILGTKVGMTQIFEESGKAIPVTVIQAGPCIVTQIKTKQTDGYTAIQLGYGEVPDRKLSNAEQGHLAKSGGTCVRHLKEYRVSEPESFQLGQQLKADIFTQGQLVDVSGTSMGRGFSGYQKRHNFKRGPMAHGSKNHRQPGSIGPGTTPGRVYPGKRMAGHYGASQVTTRKLIVVRIDPERNLLLVKGAVPGKPGGLLSIAPANIVGGK--MKLKYDKIPKMEQFYENIHQVPKIIKVTINRGLGEASQNAKALESSLNELAITGQKPVVTRATKAIAGFKVRKGMPVGVMVTLRSERMYAFLDRLINLSLPRIRDFRGVSGKSFDGRGNYSLGVREQLIFPEISYDSIDQIRGLDISIVTTAKTDEEGRALLKEMGMPFRSMSRIGKRPISVPKQVTLTINGQQVIVKGPKGELSRVLPPEVVVEQDGETVNVKRASESRTAR--QRHGLCRTLVFNMVEGVSNGFERRLEIQGVGYRAAVQGTNLVLNVGYSKPVEMIPPEGIRVAVENNTNVIISGIDKEIVGNMAAKVRAVRPPEPYKGKGIRYQGEVVRRKAGKTGKGGKKMAKKVVAVIKLAITAGKANPAPPIGPALGQHGVNIMAFCKEYNARTADKVGLVVPVEISVFEDRSFTFVLKTPPASVLIRKAAGVERGSGEPNRKKVGQITQAQLREIAETKMPDLNANDVEAAMKIVAGTARNMGVTVVDSKTLPSLEPWYVIDADQRLGRLATEVAIIRGKNKPYTPHIDTGDFVIIVNAEKIVTGKKSQKLYRRHSGRPGGMKIETFKLQRIPERIIEAIKGMLPKNALGRNLFTKLKVYAGPHPHAQKPELINTIPTMIQQTILNVADNSGARKLMCIRVLKGGNRYGIGDIIAVVKDAPNMVKRSDVVRAVIVRTRLRRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRQESDRRHRRVRKVGTERPRLAVFRSNHIYVQVIDDTLAASTVEDFKSDGTDASIGQLIARAVAKGIKVVFDRGGYIYHGRVKALADAAREAGLDFMAIDTSVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRSCEPILKVLRSAVANAEHNGLDPALVISQAYADGPIKRFRPRAQGRAYQIRKPTCHIVVAVAPE--------------------------------------MAVISLAE-----MLESGVHFGHQTRRWNPKMSPYIY-TERNGVHIIDLVQTAELMEEAYDYVRSSSEDGKKFLFVGTKRQAAGIIASEAARC-GGYYVNQRWLGGMLTNWATIKTRVDRLKTLEDRESSGYFDYLPKKEASVLRRELTKLRKYLGGIKAMRRVPDVVILVDQRREYNAVMECRKL--DIPIVSLLDTNC--DPDLADIHIPANDDAIRSIKLIVGKLADAIYEGRH--GQGAEIEDY-DYEA-----EYEED-EVEDLDSDSE---SEEMGQKINPIGFRLGITQE----------HRSRWFADSKQYPELLQEDH--TIREYVRKE-L-----NNAGISKVRIERKADQVDLEIH-------TARPGVVVGRGG-----------------------------------------------------------------------AGIETLR----------------------VGLQKKLG--------NNRQIRINVVE-------------------------------------VAKVDADATLIAEYIAQQLEKRVSFRRVVRQAITRAQKA-GIEGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTALTI-------------------------------------------YGILGVKVWVF-----------------KGEIIPGQEE---LANSPT-----------------SQPRRN-RQPRRQKYDDRSNEGMARYRGPRLRIRRLGDLPGLTRKSPRKAYPPGQHGQRRKRSEYAVRLEEKQKLRYNYGVSERQLLRYVRKARRGSTGLLQTLERLDNTIFRLGMAPTIPAARQLVSHGHVTVNGSRVNIPSYQCRPGELIGVRNTSRELVKNLQYPGLANIPHLELDKQLAKVNSEREWVALQINELLVVEFYSRQA------MAEQQQQRKGSRTK---EKEVEWQERVVQIRRVTKVVKGGKKLSFRAVVIVGNEKGQVGVGVGKAADVIGAVKKGVADGRKNVVEVPLTKSNSIPHPINGAGGGAQVMMRPASPGTGVIAGGAVRTVLELAGVRNILAKQLGSSNPLNNARATIDALQSLRTLADVAKERG--VPIEHLYAMSRRKRPIVDYNSRLISMMVRRIMKGKKSIARIIYQALIERTGEPLEFEKAIRNATPLVEVKARRVGGATYQVPMEVRERGTALALRWLIQYARARGRSMSRLAELMDAANETGSAVRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNSRHQTTIPSTRMTRSIAQVLKQEGFIDEEGEGKKLVVSLKYNSKTKPTIKLQRVSRPGLRVYRNKLPRVLGGIGVAIISTSSGVMTDREARRQGVGGEVLCYIWMTRAVYWGTGRRKSSVARVRLVPGTGKLINGKGDYLQYNQYLAKSPLETLGLENDYDILVNAGGGLTGQAESIRLGVARALCELNPDNRPLKEGYLTRDPRATERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGLAIQSTFNNTIVTITDGEVISWASAGSTGFKGAKKGTPFAAQTAAENAGRRANDQGMRQVEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRARQKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGYEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQGRSKYGAKRPKMARIAGVDLPRDKRVEIGLTYIFGIGLSRSQILTGVNPDTRIKDLNDDILRIQLYQVEGDLRREMNIKRLDIGYRGRRHRLGLPVRGQRTRTNARTRRGRTVAGKKKAAMALTQRKQEIMSHQTHETDTGSCEVQVAMLTERISKLSEHLKNKDASRRGLLQMISRRKSLLGFLQDPRYQLIRLGIRGMAVKERVGLVVSNKMDKTVVVTVENRSPHPKYGKIVVKTKRYKAHDENNQCQEGDRVRIAETRPLSKTKRWNVVEILGGNDKT-----------',\n",
       " 'Synechococcus_sp._PCC_7335_scf_1103496006895_genomic_scaffold': '------MQLKVAELYPLEAVLLKETATAKFETAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVVARGEKVEAAGATIAGSEELIEDISKGMDFDLVATPDMMPKVAKLGRVLGPRGLMPSPKGGTVTELAAIEFKAGKLEFRADKTGIVHVFGKVDFSEDLLNLKALQETIDRNRPSGAKGRFWRSSVSSTMGPSISVDVSALRDLKMSVGILGTKLGMTQVFDETGNAIPVTVIQAGPCVVTQIKTEKTDGYSSVQIGYGEVKEKALNRPKLGHLKKSGASPLRHLREYRVSDTGEFELGQALAADRFEAGQTIDVSGISIGKGFAGNQKRNNFRRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKKMAGRMGSEKVTIRKLSVVRVDSDRNLLLVKGGVPGKAGTLLNITPANIVG-QG-MSLKYDKAPKLEQFYTNPHQVPKLAKVTVNRGLGEASQNAKALESSINELAITGQRPVVTRAKKAIAGFKIREGMPVGVKVTLRADRMYAFTNRLINLTLPRIRDFRGVSPRSFDGRGNYTLGLREQLIFPEVDYDSVDQIRGMDITIVTTATSDEEGRALLKELGMPFRNMSRIGKQPVPIPKGVTVTIDGQMLTAKGSKGELSREFPPEIAIEQTGEEVVVTRVNESRNAR--QRHGLVRTLVANMVEGVSKGYEKKLEIQGVGYRASLQGQKLVMALGYSHPIEFDPPSGITFAVEKNTNVTVTGIDKEIVGNTAAKVRAARPPEPYKGKGVRYAGEQIRRKAGKAGK----MAKKVVALIKLAIPAGKANPAPPIGPALGQHGVNIMAFCKDYNSRTADKAGLIVPVEISVFEDRSFTFILKTPPASVLIKKAAKIEKGSGNPRTEKVGSITRAQLQEIAETKMPDLNANDIEMAMNIIEGTARNMGVEVAGSKTLTEIERWYVVDADQRLGRLATEVAVLRGKKKPYAPHMDTGDFVVVVNADKVVTGKKEQKLYRRHSGRPGGMKTETFQLQRIPERIVEAVKGMLPKNALGRKLFTKLKVYAGPHPHAQSPQLVNT--TMIQETFLNVADNSGARKLMCIRVL-GGRQYAVGDIVAVVKDAPNMVKKSDVVKAVVVRTRIRRASGSIRFDDNAAVLINDGNPRGTRVFGPVARELRDKSFTKIVSLAPEVLMKTTRRESRRRHSRVRRVGTERPRLAIFRSNHIYAQIIDDTLSASTLDD--ATGNDAAVGKLIARAIAKDIQVVFDRGGKLYHGRVAALADAAREAGLKFIDIDSDVKAAKFVRSPKVRRVLDQIRGRSYREALIMLEFMPYKSCEPILKVLRSAVANAEHNGLDPVLYISEAYADGPLKRFRPRAQGRAYQIRKPTCHIAIAVAPE--------------------------------------MPVVSLSD-----LLESGVHFGHQTRRWNPKMDEYIF-TSRNGVHIIDLVQTATLLEEAYQYVLKSAESGKSFLFVGTKRQAAGIVAQEASRC-GSYYVNQRWLGGMLTNWSTIKTRVDRLKDLERMEQSGALAMRPKKEGSMLRRELDKLQKYLGGIKTMRRIPDAVIIVDVRREYNAVQECQKL--GLPIISLLDTNC--DPDVVDVPIPGNDDAIRAIKLVIGRLADAIYEGST-GGNTASGADIDGYDGAD---------YNDELNANAADEPSADMGQKIHPIGLRLGITQE----------HRSRWYADKNRYPELLEEDH--NIREYVIKN-L-----ANAGISDVHIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------AGIESLR----------------------SGLQKQIK-------DDSRQIRINVVE-------------------------------------VSKVDADAGLICEYITQQLERRVSFRRVVRQAIQRAQRA-GVEGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRANIDYAYRTAQTT-------------------------------------------YGILGVKTWVF-----------------KGEVIPGQES----QPEES----------------RAQPRRR-QPRRRQQFEDRSNDSMSRYRGPRLRVRRLGDLPGLTRKSARKAYPPGQHGQRKKKSEYAIRLEEKQKLRFNYGVTERQLLRYVKKARRGSTGILQQLERLDNTIFRLGMGPTIPAARQVVNHGHITVNGRVVSIPSYQCRPGDVIGVRGKSKKLVTNLEFPGLANIPHLDFDKKLAKVTGEREWIALNINELLVVEYYSRQA---MADNRDNRRNKK-NKNR---EKDSEWKERVVQIRRVTKVVKGGKKLSFRAIVVVGNEKGQVGVGVGKASDVIGAVKKGVADGKKHLVEVPLNKAYSIPHPMTGRAGGARVFMRPAAPGTGVIAGGAVRIVLELAGVRNILAKQLGSDNPLNNARAAAAAISSLRTLSGVAEERG--IPVEKLYAMSRRKRTVPDYNSRLVSMMSRRIMLGKKSVAKILYGAFIERTGDPLEFEQAVRNTTPLVEVKARRVGGATYQVPMEVRERGTALSLRWITQFSRSRGRTMAKLAELIDAANETGGAVRKREETHRMAEANKAFAHYRYMANDTISDMLTRIRNARHQTVIPSTKMTRSIATVLKEEGFIDLDGEGDRITITLKYKGKTRPIIKLSRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGVMTDRDARRQGIGGEVLCYIYM-RAVYWGTGRRKSSVARVRLVPGNGTLINKKGEYLQANAYLAKAPLETLGLEGEYDILVNVGGGLTGQADSIKLGVARALCMLDPDNRPLKEGYLTRDPRQKERKKYGLKARKAGQFSKRMAKRKTGRKQKKNIPNGVAIQSTFNNTIVTITDGNAVSWASAGSSGFKGAKKGTPFAAQTAADGAARRAIDQGMRQCEVQVSGPGSGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERTKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKKPKMARIAGVDLPRDKRIEIGLTYIYGVGLSRSQILTGVDPNIRVKDLADDVLRVEDYLIEGDLRRVMNIKRLDIGYRGRRHRMGLPVRGQRTRTNSRTRRSKTIAGKKKPP----------------------------MLTDRINTLSKHLQNKDSSRRGLLRMIGQRKRLLAYLMDERYRLIKLGIRRMAAKERVGLVVSNKMDKTVVVAVESRVQHKKYDKIMVRTQRYKAHDENNTCKEGDRVKISETRPMSRHKRWVVADILSSSDL------------',\n",
       " 'Synechocystis_sp._PCC_6803_(updateJune2012)': 'KKLSKRMQAKVDSKYPLEAMLLKETATAKFETAEAHIRLGIDPKYSDQQIRTTVSLPKGTGQVRVAVLARGEKVEAAGADIAGSEELIEEIQKGMDFDLIATPDMMPKIARLGKQLGPRGLMPSPKGGTVTDLAAVEFKAGKLEFRADRTGIVHVFGKASFSDDLLNLKALQETIDRNRPSGAKGRFWRTFVSSSMGPSIPVDINALRDLKVSIGILGTKLGMTQIFDESGISIPVTVVQAGPCPVTQVKTQDTDGYNAIQVGFLPVKEKALSKPELGHLKKSNTDPMRHLKEYRLTDAPNLQPGDAVTADIFQAGDLVDVAGQSMGRGFAGYQKRHNFRRGNMTHGSKNHRLPGSTGAGTTPGRVYPGKRMAGQYGASQVTVRRLTVVRVDAERNLLIIKGALPGKPGTLLNITPAKTVG-RG-MRLKYETIPKLEEFYKNIHQVPKLTKVTVNRGLGEASQNAKALESSLTELAITGQKPVVTRARKAIAGFKIREGMPVGVMVTLRSERMYAFLDRLINLALPRIRDFRGISPNSFDGRGNYSLGIREQLIFPEIDYDTIDQIRGMDVSIITSAQTDEEGRALLKALGMPFR-MSRIGKRPIPLPAKVSVDIQGSHLSVKGPKGSLERQLPEKVIVAQEGETITVTRQDESRTAR--ERHGLVRTLVANMVDGVAQGFERRLEIQGVGYRAQAQGNKLTLNVGYSKPVEMTMPQGIEVKVENNTQVIVSGIDKELLGNTAAKIRAVRPPEPYKGKGIRYQGEYVRRKAGKTGKK---MAKKVVALIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTADKPGMIIPVEISVFEDRSFTFILKTPPASVLIRKAAGVEKGSSEPNKNKVASITREQLREIAQTKLPDLNANDIDAAMNIIEGTARNMGITVNSMKTLPNLDHWYVIDAGQRLGRLATEVAILRGKNKPFTPHMDTGDFVIIINAEKIVTGRKEQKLYRRHSGRPGGMKEETFKLQRLPERIVEAVRGMLPKNSLGRKLFTKLKVYAGPHPHAQQPELINTIPAVIQQTYLNVADNSGARKLMCLRVLGTGNTYGIGDIIAVVKDAPNMIKKSDVVRAVIVRTKLRRASGSIRFDDNAAVIINEGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKSTRKSAQRRHRRLRHLGTERPRLAVFRSNHIYAQVIDDTLAASTLDDLKKLTTEASVGKLVARAIAKGIQVVFDRGGKLYHGRVKALAEAAREAGLNFTKLDTTVKAARYVRSPKVRRVLDQIRGRSYREALIILEFMPYKACEPVLKVLRSAVANAEHNGLEPALVVSQAFADGPLRRFRPRAQGRAYQIRKPTCHITVAVAPA--------------------------------------MPVVSLAD-----LLESGVHFGHQTRRWNPRMDQYIY-TARNGVHIIDLVQTAQLMEDAYEYVRSSTINGKKFLFVGTKRQAAGIISQEAQRC-GAHYVNQRWLGGMLTNWETIRKRVDRLKELEALEASGGIDRRGKKEGSMLRRELGKLQKYLGGIKNMRKLPDVVVIVDQRREHNAIHECQKL--GIPIIAMLDTNC--DPDVVDVPIPANDDAIRSIKLIVGKLADAIYEGRH--GQPDVSDDYEEFDEGL-DGDNLEV-EAAEEAAEAEVATPEAMGQKIHPVGFRLGITKD----------HKSCWYADPKRYPELLQEDH--KIRQYIEKT-L-----NNAGISDIRIERKAEQIELGIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIEQLR----------------------EGLQKLLG--------SARQIRVNVIE-------------------------------------VPNADADAALMAEYIGQQLERRVSFRRVVRQALQRAERA-EVKGIKIQVSG-----------------------------RLNGAEIARTE-----WVREG---RVPLHTLRADIDYAYRTALTT-------------------------------------------YGILGIKVWIF-----------------KGEVIPGQEA---AIVAPP-----------------SQPRR---KSRRQQFDDRSQDGMSRYRGPRLRIRRLGELPGLSRKSPRRAYPPGQHGQRRKRSEYAIRLEEKQKLRLNYGITEKQLVRYVKKARRGSTGLLELLERLDNTVFRLGMAGTIPGARQLVCHGHITVNGQVVDIPSYQCRPGDIVSVRDRSRKLVENMEFPGLANVPHLEFDKTFGKVNSDREWVALQINELLVIEYYSRKA------MAK-RRKT--SREK---KEDTNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNETGQVGVGVGKAGDVIGAVRKGVADGKKQLIEVPLTKSNSITHITNGVSGGAKVVVRPAAPGTGVIAGGAVRTVLELAGVKNILAKQLGSNNPLNNARAAINALETLRTFSEVAEERG--VSVEHLYTMSRRKRPVPDYNSTLLSMTIRRVMRGKKSLASIVYNALVEKTGDPLEFEKAIKNLTPLVEVKARRVGGATYQVPMEVRARGTALALRWLVHFSRARGRTMEKLAEIMDAANETGAAIKKREETHRMAEANKAFAHYRYMATDTISDMLTRIRNARHSTTVPTTKMTLSIAKVLKSEGFIDSEGEGNKLVLTLKYKGKTRPLITLQRVSKPGLRVYSPKIPRVLGGIGIAIVSTSHGIMTDREARRQGIGGEILCYIWMSKVVYWGTGRRKAAIARVRLVPGQGEVVNGKGEYFNRIAYILKAPLETLGLEGEYNILVNAGGGLTGQADAVKLGVARALCQLSPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARRKTGKKAKKNVPSGVAIQSTFNNTIVTISDGDVISWASAGSSGFKGAKKGTPYAAQTAADSAARRAMEQGMRQLEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERQKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIVRGTLDATGVKDRKQGRSKYGTKREKVARIAGVDLPRDKRVEIALTYLYGIGLSRSHILTGVSPDVRVKDLSDDALRIDNYEIEGDLRREMNIKRLDIGYRGRRHRQGLPVRGQRTRTNARTRRGRTVAGKKKTPMSLTQRKQELMTYQAHETDTGSADLQVAFLTERITQLTGHLKNKDASRRGLLKMIGRRKRLLSFINEERYQLIRLGIRRMAIKERVGIVVSNKMDKTVVVAVESRSPHPKYGKIVVKTKKFKAHDEENQCQEGDKVRIQETRPLSKTKRWQVINIMSHSS-------------',\n",
       " 'Trichodesmium_erythraeum_IMS101_(Genome_Extraction)': 'KKISRRLQLKVEERYPKEALLLKETATAKFEAAEAHVRLGIDPKYTDQQLRTTVALPKGTGQIRIAVIAKGEKVEAAGADLAGSEELIEQISKGMEFDLIATPDVMPQVAKLGRLLGPRGLMPSPKGGTVTDLTAIEFKAGKLEFRADRSGIVHVFGKTSFSEDLLNLKALQECIDRNRPSGAKGRYWRTFVSATMGPSIEIDVSSLQDLKVVVGILGTKLGMTQVFEETGKAIPVTVVEAGPCVVTQIKTEQTDGYTAVQIGYGEAKNKTLTNAQEGHLVKSGGTPLRYLREYRVDNTGNFELGQQIKVDLFEAGQLVDVTGKSIGKGFAGYQKRHNFRRGPMAHGSKNHRLPGSTGPGTTPGRVYPGKKMAGHLGNVRVTTRKLKIVRIDPERNLLLIKGSIPGKSGTLISIAPANIVGQK--MTLKYKKIPKLDQFYKNIHQVPKLVKVTINRGLGEASQNAKALDSSIKEIAITGQKPVVTRAKKAIAGFKIRQGMPVGVMVTLRSQRMYEFIERLINLALPRIRDFRGINPKSFDGRGNYSLGVKEQLIFPEIEYDKIDQIRGMDISIITTAKTDEEGRALLKEMGMPFRNMSRIGKLPIPIPKKVTITIEGQKVTVKGPLGELSRVLPPEIAVKEQENTIIVKPQEQSRRAR--QRYGLCRTLVANMVEGVSKGYEKRLVIQGVGYRAQVQGKTLVLNVGYSKPVEMPAPEGINIAVENNTNVIVNGINKELVGNTAAKIRAVRPPEVYKGKGVRYAGEVIKLKAGKSGKK---MAKKVVAVIKLAITAGKANPSPPIGPALGQHGVNIMAFCKEYNAKTADQAGTVVPVEISVYEDRSFTFVLKTPPAAVLIKKAAGVESGSSEPNRTKVGSITQAQLQEIAQTKMPDLNANNIEAAMKIVAGTARNMGITISDMKTLPIEGKWYVIDAGQRLGRLATAIAIIRGKNKPYTPHMDTGDFVIVVNAEKVVTGKKSQKLYRRHSGRPGGMKTEVFKLQRLPQRIIEAVFGMLPKNSLGRKLHNKLKVYPGSHPHAQKPELIQTIPGMIQQSLLNVADNSGARKLMCIRVVGGGNRYAIGDIIAVVKDAPNMVKKSDVVRAVVVRTKLRRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKYTRKDSIKRHRRVRKVGTERPRLSVFRSNHIYAQVIDDTLAASTLEEVKSIGNDASIGKLIARSLEKGIKVVFDRGGNLYHGRVKALAEAAREAGLDFMPIDTNVKAARYIRSPKVRRVLDQIRGISYREALIILEFMPYRSCVPIRKVLCSAVANAENNGIDPDLVISKAFADGPLPRYRPRAQGRAYKIRKPTCHITVAVASD--------------------------------------MPVVSLAQ-----LLESGVHFGHQTRRWNPRMSQYIF-TERNGVHIIDLVQTAQLMDEAYNYIRTASEQGKKFLFVGTKRQAAGIIAQEATRC-GGYYINQRWLGGMLTNWTTIKTRVERLKDLERREESKALDLLPKKEASVLRREMAKLQKYLGGIKNMRRLPDGVIMVDQRREYNAVQECQKL--NIPIVSLLDTNC--DPTQVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLEADEEYDDYEGAEGDLDLDSA-EEES-LEDNN---EEEMGQKIHPTGFRLGITQE----------HRSRWYADPKNYPDTLQEDH--KIRQYVKAT-L-----ANAGISQIRVERKAEQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------AGIDSLR----------------------VGLQKLLG--------SNRQIRINVVE-------------------------------------VTKVDTDAMLIGEYIAQQLEKRVSFRRVVRQAITRAQKA-GVEGIKIQVSG-----------------------------RVNGAEIARTE-----VTREG---SVPLHTLRANIDYAYCTAKTI-------------------------------------------YGILGIKIWVF-----------------KGEIIPGQEE---IANAKV-----------------NQPKRR-QQKRRQQYDDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRRKRSEYAIRLEEKQKLLYNYGVCERQLLRYVKKARRGSTGLLELLERLDNTVFRLGMAPTIPSARQLVNHGHVTVNGKVVSIASYQCKPGEVISVKEKSKEMVKNLQYPGLANIPHLEFDKKLGKVNGEREWVALQINELLIVEYYSRQA----MAEQQ-QRRKKSSRTK---EKEIEWQERVVQIRRVTKVVKGGKNLSFRAIVVIGNERGQVGVGVGKAADVIGAVRKGVADGKKHIIEVPLTKSNSIPHPVNGFGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVQNILAKQLGSNNPLNNARAAVNALDALRTLADVAQERE--VPIENLYVMSRRRRPIPDYNSRLVTMIVRRIMRGKKSVANIVYDAFIERTDDPLEFEKAVRNLTPLVEVKARRVGGATYQVPMEVRDRGTTLALRWLVQYSRSRGKTMAKLAELMDAANETGSAIRKREETHKMAEANKAFAHYRYMANDTISDMLTRIRNSQHTTTVPATKMTRSIAKVLKDEGFIEEQGEGKKLVIFLKYKGKNRPIIYLKRVSKPGLRVYKNKLPRVLGGIGIAIISTSSGIMTDREARKRGIGGEVLCYVWMSRAMYWGTGRRKSSVASVRLVPGTGKLINGKGEYLQYNHYLVKAPLDTLGFENEYDILVKAGGGLTGQAESIRLGVARALCQLDPENRPLKEGYLTRDPRAKERKKYGLKARKASQYSKRMARKKTGKKQKKNVPNGVAIQSTFNNTIVTIADGEVISWASAGSSGFKGAKKGTPFAAQTAAESAARRANDQGMRQVQVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRAREKQTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQSRSKYGAKKPKVARIAGVDLPRDKRIEIGLTYIYGIGLSRSQILTDVNPDTRVKDLSDDVIRVQNYEVEGDLRREMSIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKASMALKQRKQEIIGYQTHETDTGSADVQVAMLTEKITKLSAHLRNQDSSQRGLLKMIGKRKRLLTYIQNERYKLIRLGIRGMAAKERVGVVVSNKMHKTIVVAIENRSAHPRYGKIVVKTKKYKVHDEENKCNEGDRVRIRETRPLSRTKRWELSEIIGSKATL-----------',\n",
       " 'Pleurocapsa_sp._PCC_7319': 'KKMSRRMRLKVENKYPLEALLLKETATAKFETAEAHIKLGIDPKYTDQQLRTTVSLPKGTGQVRVAVIARGEKVEAAGAEIYGSEELIEDIQKGMDFDLIATPDMMPKVAKLGRVLGPKGLMPSPKGGTVTDLKAVAFKAGKLEFRADKTGIVHVFGKASFSEDLLNLKALQETVDRNRPSGAKGRYWRTYVSASMGPSIQVDVSALQDLSMSVGILGTKLGMTQVFDDTGNAIPVTVVQAGPCVVTQIKTKETDGYSSIQIGYGEVKEKALSKPEIGHLSKAGANPLRHLKEYRIEDTASYELGKTLDSEIFNTDTTVDISGKTMGRGFAGYQKRHNFKRGLMSHGSKNHRAPGSTGAGTTPGRVYPGKKMAGRYGGKQVTTRKLKVVKVDAEQNLLLIKGAVPGKPGALLSISPSNIVG----MPLKYEDIPKLEQFFSNVHQVPKVVKIVVNRGLGEASQNAKALESSINEMMITGQKPVVTRAKKAIAGFKIRQGMPVGVMVTLRGERMYAFLERLICLALPRIRDFRGISPTSFDGRGNYSLGVREQLMFPEIDYDSIDQIRGMDISIVTSANNDEEGRALLKEMGMPFRNMSRIGKQPIPIPDKVSAQISGQSVTIKGPKGTLSREIPEPIEVKQEGNTIVVNPTNDSRMGR--QRHGLCRTLVANMVDGVSQGFQKRLLIQGVGYRAQAQGKKLTLNVGYSKPVEIEMPEGIDVAVENRTEVIISGISKEVVGNVAAKIRAVRPPEPYKGKGIRYFDEQVKRKAGKTGK----MPRKVVALIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTADKAGLVIPVEISVFEDRSFTFILKTPPASVLIKKAAGIDKGASEPNKQKVASITRDQLKEIAQTKLPDLNANDIDAAMKIVAGTAKNMGVAIAEMKTLPTLEQWYVVDAGQRLGRLATEIAILRGKNKPFTPHMDTGDFVIVVNAEKVVTGRKEQKLYRRHSGRPGGMKTETFQLQRIPNRIIEAVKGMLPKNVMGRKLFTKLKVYAGPHPHAQTPELINTIPAMIQQTYLNVADNSGARKLMCLRVIGKGNPYAIGDVVAVVKDAPNMVKKSEVVKAVIVRTKVNRPSGSIRFDDNAAVIINQGSPRGTRVFGPVARELREKNFTKIVSLAPEVIMKLSRRESTRRHRRVRKVGTERPRLCVFRSNHIYAQVIDDTLAAS--DNVKSSGTEVSVGKLVARALEKGIKVVFDRGGNLYHGRVKALADAARESGLDFMAIDTNVKAAKYIRSPKVRRVLDQIRGRQYREALIILEFMPYKACEPILKTLRSAVANAEHNGLDPALVISKAFADGPLKRYRPRAQGRAYQIRKPTCHITVAVAPT--------------------------------------MPVVSLAE-----LLESGVHFGHQTRRWNPRMSQYIY-TARNGVHIIDLVQTAQLMEEAYGYMRSSSERGKRVLFVGTKRQAAGIIAQEASRC-GAFYVNQRWLGGMLTNWETIKTRVERLKELEHLEESGALDKRPKKEASVLRREMSKLQKYLGGIKSMRKVPDIVVIVDQKREHNAISECQKL--GIPVVSILDTNC--DPLIVDVPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDNQDDYEEFDEAIEEPSEEEV-EAEISTPESESGDSEEMGQKIHPTGFRLGITKD----------HQSCWYADTKRYPELLQEDR--LIREYVNKE-L-----SNAGISQVKIERKADQIDLAIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIEKLR----------------------TELQAKVG--------NNRQIRINVIE-------------------------------------VPRVDADAALIAEYICQQLERRVSFRRVVRQAVQRAQRA-EVLGIKIQVSG-----------------------------RLNGAEIARTE-----WVREG---RVPLHTLRADIDYSYRTALTI-------------------------------------------YGILGVKVWVF-----------------KGEIIPGQEE--AAAAVPS-----------------QAPRR---QRRRQKFDDRSE--MSRYRGPRLRIRRLGELPGLSRKSPRKAYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVTEKQLIRYVRKARRGSTGLLELLERLDNTIFRMGMAGTIPAARQLVNHGHILVNDRVVNIASYQCRPGDVIKVRDRSRKLVENMEYPGLANLPHLEFDKTFGKVNNEREWVALQVNELLVVEYYSRMA------MAKSKSRKK-GRAK---EKDSNWQERVVQIRRVSKVVKGGKKLSFRAIVVVGNEKGQVGVGVGKAADVIGAVRKGVADAKKQLVEVSLTKASSIVHVTNGVAGGAKVFMRPAAPGTGVIAGGAVRTVLELAGVKNILAKQLGSNSPLNNARAAIDALESLRTFAEVARERD--IPLEQLYAMSRRKRPIPDYNSRLLSMTIRRIMRGKQSLARIMYDAMIEKTGEAIEFEQAIKNLTPLVEVKARRVGGATYQVPMEVRGRGTTLALRWLIQFSRSRGRTMAKLAEIMDAANETGGAMKKREETHRMAEANKAFAHYRYMATDTISDMLTRIRNAKHQTTVPSTRMTRAIANVLQKEGFIGEEGQEQKIVISLKYKGRNRPIITLKRVSKPGLRVYSRKIPRVLGGIGIAIVSTSRGVMTDREARSQGVGGEILCYVWM-KVVYWATGRRKSSIARVRLVPGEGKLVNGKGEYFNRIPYLIKGPLETLGFESEYDILVNAGGGLTGQADAVKLGVARALCKQDPNNRPLKEGYLTRDPRCKERKKYGLKARKAPQFSKRMAQRRSGKKNKKNVPNGVAIQSTFNNTIVTISDGDVISWSSAGASGFKGAKKGTPFAAQTAADSAGRRAMDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERQKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIVRGTLDTTGVKDRAQGRSKYGTKRPKMARISGVDLPRDKRVEVALTYIFGVGLSRAQAITGVNPDTRVRDLSDDVLRIENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPLRGQRTRTNARTRRGRTVAGKKKAPMTLTQRKQELMAYQIHETDTGSADLQIAVLTERINQLTAHLKNKDSSRRGLLKMIGRRKGLLAYIQDERYQLIRLGIRRMAVKERVGEVVSDKMDKTVVVAIENRSPHPKYGKIVVRTKKYKAHDAENQCQIGDRVRIRETRPLSKTKRWTVEEIIESPKS------------',\n",
       " 'Arthrospira_platensis_NIES-39': 'KKVSRRLQLKVEDRYPLEGLLLKETATAKFESVEAHIRLGIDPKYTDQQLRTTVTLPKGTGQVRVAVLAKGDKVEAAGADLVGSDDLIAEIQKGLDFDLIATPDMMPQVAKLGRLLGPKGLMPSPKGGTVTDLTAIEFKAGKLEFRSDRTGIVHVFGKVSFSEDLLNLSALQECIDRNRPSGAKGRYWRSYVCSTMGPAIQLDINALRDLKMSVGILGTKLGMTQVFDETGKAIPVTVVQAGPCTITQVKTEATDGYTAIQLGYGEVPNRKLSHAEQGHLTKSNSPLVRHLKEYRLESTDNFELGQEVKVDAFEPGQIVDVTGISMGRGFAGNQKRNNFRRGPMAHGSKNHRLPGSIGPGTTPGRVYPGKKMAGRYGGKQITTRKLTIVRVDLERNLLLIKGSIPGKTGGLLNIKPANLVGGK--MPLKYETIPKLEQFYENIHQVPKVVKVTINRGLGEASQNAKALESSVNELAITGQKPVVTRAKKAIAGFKIRQGMPVGVMVTLRSDRMYAFMERLIHLALPRIRDFRGISPKSFDGRGNYSLGVKEQLIFPEIDYDRIDQIRGLDISIVTTAQTDEEGRALLKEMGMPFRNMSRIGKRPIPIPAKVTLTLDGQKVTVKGPKGELSRVLPNEVILSLEGDTLIVKRRDESLVAR--QRHGLCRTLVANMVDGVSQGFERRLEIQGVGYRAQVQGKNLILNVGYSKPVEMVPPEGCSVAVENNTNVIVSGINKELVGNMAAKIRAVRPPEPYKGKGIRYAGEQVRRKAGKAGKK---MAKKVVAVIKLAINAGKANPAPPIGPALGQHGVNIMMFCKEYNAKTADKVGLVIPVEISVYEDRSFTFVLKTPPASVLIKKAAGIEKGSGEPNRVRVGTINRSQLREIAEVKMPDLNANDVEAAMKIVEGTARNMGIAVSDMKTLPSIERWYVIDADQRLGRLATEIAIIRGKNKPYTPHMDTGDFVIVVNAQKVVTGKKEQKIYRRHSGRPGGMKTETFQLQRVPERIIEAVKGMLPKNALGRQLFTKLKVYSGPHPHAQEPELIDTIPTMIQQSILNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVRAVIVRSRLRRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRNDSQRRHRRVRKVGTERPRLAVFRSHHIYVQVIDDTLAASTLDEITALGTDASVGQLIARSLSKGIRVVFDRGGYIYHGRVKALADAAREAGLEFMAVSTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACTPVLKVLRSAVANAEHNGLDPALIVSQAFADGPLKRYRPRAQGRAYPIRKPTCHITVTVAP---------------------------------------MPVISLAD-----MLESGVHFGHQTRRWNPRMAPYIF-TERNGVHIIDLVQTAQLMDDAYDYVRSASEEGKKFLFVGTKRQAAGIIAQEAARC-GGYYVNQRWLGGMLTNWVTIKTRVDRLKDLEARENSGQLDYLPKKEAAVLRREMGKLQKYLGGLKGMRRVPDAVIMVDQRREYNAVMECRKL--GIPIIALLDTNC--DPDLADIHIPANDDAIRAIKLIIGKLADGIYEGRH--GVSAVG-A-DDYDDAD---DYDDEEFGDDYESDDDSDSSDDMGQKIHPIGFRLGVTQE----------HKSRWFADAGQYPQLLQEDY--TIRQYIQKN-L-----SNAGISDVRIERKADQIDLEVL-------TARPGVVVGRGG-----------------------------------------------------------------------TGIDSLR----------------------QGLQKELG--------SNRQIRINVVE-------------------------------------VSRVDADATLIAENIAAQLEKRVSFRRVVRQAITRAQKA-GIEGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTALTI-------------------------------------------YGILGVKVWVF-----------------KGEIIPGQEE----TPAPN----------------TRAPKSR-RTPRRQKYDDRSSDTMSRYRGPRLRVRRLGDLPGLTRKTARRAYPPGQHGQRRKRSEYAVRLEEKQKLRFNYGLSERQLLRYVRKARRGSTGLLQYLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGKVVDIASYQCRPGEIIGVRNRSREMVKNLQYPGLANVPHLELDKNLATVNGEREWVALSINELLVVEYYSRMA------MAEQQRRKKNTRTK---EKESEWQERVVQIRRVTKVVKGGKKLSFRAIVIVGNEKGQVGVGVGKAADVIGAVKKGVADGRKNVVEIPLTKSNSLPHPANGVGGGAQVIVRPAAPGTGVIAGGAVRTVLELAGVKNVLAKQLGSSNPLNNARATINALSGLRTLSDVAKERG--IPIEQLYAMSRRKRPVPDYNSRLVSMMVRRIMRGKKSVANIVYDALIERTGDPLEFEKAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLIHFSRTRGRSMARLAELMDAANETGSAVRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNSRHQTTIPSTKMTRSIAKVLQDEGFINEEGAGKKLVVSLKYNTKTRPTITLKRVSRPGLRVYKNKLPRVLGGIGIAIISTSRGVMTDREARLQGLGGEVLCYVWMTKAVYWGTGRRKSAIARVRLVPGSGQVINGRGEYLQFNSYLTKAPLETLGLENDYDVLVNVGGGLTGQAESIRLGVARALCELSPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNTIVSITDGEVISWASAGSTGFKGAKKGTPFAAQTAAESAGRRASDQGMRQIEVMVSGPGSGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRARDKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRNGRSKYGAKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSKILTGVNPDTRVKDLSDDVLRIQNYQVEGDLRREMSIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAAMALTQDKQKIMAHQAHETDTGSSDLQVAMLTERINRLSQHLKNHDASRRGLLQIIGRRKRLLAYINDQRYRLIRLGIRGMAVKERVGVVVSNKMQKTVVVAIENRSPHPKYGKIVVNTKRYKAHDEENTCQEGDRVRIRETRPLSRTKRWNVVEIINKTNQ------------',\n",
       " 'Arthrospira_platensis_C1_(Draft2_circular_genome_)': 'KKVSRRLKLKVEDRYPLEGLLLKETATAKFESVEAHIRLGIDPKYTDQQLRTTVTLPKGTGQVRVAVLAKGDKVEAAGADLVGSDDLIAEIQKGLDFDLIATPDMMPQVAKLGRLLGPKGLMPSPKGGTVTDLTAIEFKAGKLEFRSDRTGIVHVFGKVSFSEDLLNLSALQECIDRNRPSGAKGRYWRSYVCSTMGPAIQLDINALRDLK-------------------------------------MKTEATDGYTAIQLGYGEVPNRKLSHAEQGHLTKSNSPLVRHLKEYRLESTDNFELGQEIKVDTFELGQMVDVTGISMGRGFAGNQKRNNFRRGPMAHGSKNHRLPGSIGPGTTPGRVYPGKKMAGRYGGKQITTRKLTIVRIDLERNLLLIKGSIPGKPGGLLNIKPANLVGGK--MPLKYETIPKLEQFYENIHQVPKVVKVTINRGLGEASQNAKALESSVNELAITGQKPVVTRAKKAIAGFKIRQGMPVGVMVTLRSDRMYAFMERLIHLALPRIRDFRGISPKSFDGRGNYSLGVKEQLIFPEIDYDRIDQIRGLDISIVTTAQTDEEGRALLKEMGMPFRNMSRIGKRPIPIPAKVTLTLDGQKVTVKGPKGELSRVLPNEVILSLEEDTLIVKRRDESRVAR--QRHGLCRTLVANMVDGVSQGFERRLEIQGVGYRAQVQGKNLILNVGYSKPVEMVPPEGCSVAVENNTNVIVSGINKELVGNMAAKIRAVRPPEPYKGKGIRYAGEQVRRKAGKAGKK---MAKKVVAVIKLAINAGKANPAPPIGPALGQHGVNIMMFCKEYNAKTADKVGLVIPVEISVYEDRSFTFVLKTPPASVLIKKAAGIEKGSGEPNRVRVGTINRSQLREIAEVKMPDLNANDLEAAMKIVEGTARNMGIAVADMKTLPSIERWYVIDADQRLGRLATEIAIIRGKNKPYTPHMDTGDFVIVVNADKVVTGKKEQKIYRRHSGRPGGMKTETFQLQRVPERIIEAVKGMLPKNALGRQLFTKLKVYSGPHPHAQDPELIDTIPTMIQQSILNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVRAVIVRSRLRRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRNDSQRRHRRVRKVGTERPRLAVFRSHHIYVQVIDDTLAASTLDEITALGTDASVGQLIARSLSKGIRVVFDRGGYIYHGRVKALADAAREAGLEFMASSNSVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACTPVLKVLRSAVANAEHNGLDPALIVSQAFADGPLKRYRPRAQGRAYPIRKPTCHITVTVAE-IIVKLSQLAHPRVRVSVGNLPTDRRLGGGLTRKGEKLIMPVISLAD-----MLESGVHFGHQTRRWNPRMAPYIF-TERNGVHIIDLVQTAQLMDDAYDYVRSASEQGKKFLFVGTKRQAAGIIAQEAARC-GGYYVNQRWLGGMLTNWVTIKTRVDRLKDLEARENSGQLDYLPKKEAAVLRREMGKLQKYLGGLKGMRRVPDAVIMVDQRREYNAVMECRKL--GIPIIALLDTNC--DPDLADIHIPANDDAIRAIKLIVGKLADGIYEGCH--GVSAVG-ADDDYDDAD---DYDDEEFGDDYESDDDSDDSDDMGQKIHPIGFRLGVTQE----------HKSRWFADASQYPQLLQEDH--TIRKYIQKN-L-----SNAGISDVRIERKADQIDLEVL-------TARPGVVVGRGG-----------------------------------------------------------------------AGIDSLR----------------------QGLQKELG--------SNRQIRINVVE-------------------------------------VSRVDADATLIAENIAAQLEKRVSFRRVVRQAITRAQKA-GIEGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTALTI-------------------------------------------YGILGVKVWVF-----------------KGEIIPGQEE----TPAPN----------------TRAPKSR-RTPIRQKYDDRSSDTMSRYRGPRLRVRRLGDLPGLTRKTARRAYPPGQHGQRRKRSEYAVRLEEKQKLRFNYGLSERQLLRYVRKARRGSTGLLQYLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGKVVDIASYQCRPGETIGVRNRSREMVKNLQYPGLANVPHLELDKNLATVNGEREWVALSINELLVVEYYSRMA------MAEQRRKKN-TRTK---EKESEWQERVVQIRRVTKVVKGGKKLSFRAIVIVGNEKGQVGVGVGKAADVIGAVKKGVADGRKNVVEIPLTKSNSLPHPANGVGGGAQVIVRPAAPGTGVIAGGAVRTVLELAGVKNVLAKQLGSSNPLNNARATINALSGLRTLSDVAKERG--IPIEQLYAMSRRKRPVPDYNSRLVSMMVRRIMRGKKSVANIVYDALIERTGDPLEFEKAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLIHFSRTRGRSMARLAELMDAANETGSAVRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNSRHQTTIPSTKMTRSIAKVLQDEGFINEEGVGKKLVVSLKYNTKTRPTITLKRVSRPGLRVYKNKLPRVLGGIGIAIISTSRGVMTDREARLQGLGGEVLCYVWMTKAVYWGTGRRKSAIARVRLVPGSGQVINGRGEYLQFNTYLTKAPLETLGLENDYDVLVNVGGGLTGQAESIRLGVARALCELSPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNTIVSITDGEVISWASAGSTGFKGAKKGTPFAAQTAAESAGRRASDQGMRQIEVMVSGPGSGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRARDKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRNGRSKYGAKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSKILTGVNPDTRVKDLTDDVLRIQNYQVEGDLRREMSIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAAMALTQDKQKIMAHQAHETDTGSSDLQVAMLTERINRLSQHLKNHDASRRGLLQIIGRRKRLLAYINDQRYRLIRLGIRGMAVKERVGVVVSNKMQKTVVVAIENRSPHPKYGKIVVNTKRYKAHDEDNTCQEGDRVRIRETRPLSRTKRWNVVEIINKNNQ------------',\n",
       " 'Leptolyngbya_sp._PCC_7376': 'AKLSRRFRAKVQDRYPLEALLLKETATAKFETAEAHIRLGIDPKYTDQQIRTTVGFPKGTGQVRVAVITSGEQVVASGADLVGSEELIEDIQKGMDFELIATPDMMPKVARLGRQLGPRGLMPSPKGGTVTDVAAIEFKAGKQEFRADRAGIVHVFGKSSFSEDLLNLKTLQETIDRNRPSGAKGRYWRSYVSASMGPSIEVDINSLRDLVMSLGILGTKLGMTQIFDETGAAIPVTVVQAGPCTVTQVKTSDTDGYASVQIGYGDVKEKALSKPELGHLSKAGATPLRHLKEYRLDDVSSYELGQSITADIFEAGQTVDVSGNTIGRGFAGYQKRHNFKRGPMTHGSKNHRQPGSTGAGTTPGRVYPGKRMAGRYGGTKITVRKLEVVRVDGERNLLLIKGAVPGKAGNLLSIAPSNIVGQK--MRLKYDTIPKLEQFYKNIHEVPKVIKISVNRGLGEAASNAKAMESSRAELAITGQRPVITRAKKAIAGFKIRQGMPVGVMVTLRSEKMYAFLDRLVSLALPRIRDFRGISPKSFDGRGNYSLGIREQLIFPEIDYDSIDQIRGMDISIITTAKTDEEGRALLKAMGMPFRSMSRIGKQPVPIPDKVTVAVNGLSVSVKGPKGSLERTLPEKVTIAQEDNTVVISPVDSSRAAR--ERHGLCRTLVANMVEGVSNGYSKSLEIIGVGYRAQVKGNTLILNVGYSKPVEMVMPEGITAAMEKNTQVIISGIDKEVVGNTAAKVRAVRPPEPYKGKGIRYQGEYVRRKAGKAGK----MAKKVVTLIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTSDKAGMVIPVEISVYEDRSFTFILKTPPASVLIRKAAGVEKGSGQPNKDKVGSITRAQLKEIAEIKMPDLNANDIEAAMNIVEGTARNMGVTLADMKTLPNLDQWYVVDADQRLGRLATEIATLRGKNKPYTPHMDTGDFVVVINAEKVVTGRKEQKLYRRHSGRPGGMKVETFQLQRIPERIIEAVKGMLPKNALGRRLFTKLKVYAGAHPHAQQPEMINTTPGMIQQTYLNVADNSGARKLMCLRVLSTGNRYGIGDIIAVVKEAPNMVKKSDIVRAVIVRTRLRRVSGSIRFDDNAAVIINEGNPKGTRVFGPVARELRDKNFTKIVSLAPEVIMKATRKQLQRRHYRIRRVGSDRPRLAIFRSHHIYAQVIDDTLAASTLDDLRALGNAASVGSLIARALEKGIKVVFDRGGNLYHGRVKALAEAAREAGLDFMSIDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRSCEPILKVLRSAVANAEHNGLDPALIVSTAFADGPLKRYRPRAQGRAFQIRKPTCHITVAVAPT--------------------------------------MPVVSLAE-----LLESGVHFGHQTRRWNPKMSQYIY-TSRNGVHIIDLVQTAQLVEEAYAYMKDASENGKRVLFVGTKRQAAGIIAQEAKRC-GAFYINQRWLGGMLTNWETIKTRVERLKELESLQDSGNLDRRPKKEASMLRRELDKLQKYLGGIKNMRKIPDIVVVIDQRREHNAIQECRKL--NIPIVSLLDTNC--DPHTVDLPIPANDDAIRSIKLIVGKLADAIYSGRH--GQPVKGENYEEFEEAIEEIEVEAVETVEATPPAEVAESSETMGQKIHPVGFRLGITKE----------HLSRWYADSKQYPELLQEDY--KIRQYVDAN-L-----NNAGISKVLIERKADQVDLEIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIEALR----------------------TGLQQALG--------DQRQIRINVVE-------------------------------------VARVDADAALIAEYIVQQLERRVSFRRVVRQAIQRAQRA-EVQGIKIQVSG-----------------------------RLNGAEIARTE-----WVREG---RVPLHTLRADIDYSYKTAQTI-------------------------------------------YGILGIKVWIF-----------------KGEIIPGQED----IPPAN---------------AAPMPRR---KARRQQFEDRSEQ-MARYRGARLRIRRLGELPGLTRKSARREYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSEKQLIRYVRKARRGSTGILQQLERLDNTIFRLGMAGTIPAARQLVNHGHIMVNGRRVDIPSYQCRPGDLITVRDRSRKLVENMEFPGLANVPHLEFDKNLGKVNGEREWVALQVNELLVVEYYSRMA------MAK-RRKS--SKNK---DKDSNWQERVIQIRRVSKVVKGGKKLSFRAIVVIGNETGKVGVGVGKAGDVIGAVRKGVADAKKHVVDVPLTKTNTVTHRVNGIAGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVKNILAKQLGSKSPLNNARATIDALGNLRSFSSVAQERG--VSIERIYAMSRRKREVPDYNNRLLSMTIRRVMSGKKSLARIVYDALIDRTGDPIEFEEAIRNLTPLVEVKARRVGGATYQVPMEVRGRGTALALRWLIGYARQRGKTMAKLAELMDAANETGGAIKKREDTHRMAEANKAFAHYRYMANDTISDMLTRIRNAQHENTVPVTRMTRSIAAVLKDEGFISTEGEGRKLVISLKYKGRNRPIIKLQRVSKPGLRVYSNKLPRVLGGIGIAIISTSKGVMTDRQARKEGIGGEILCYVW-QRAVYWGTGRRKASIARVRLIPGSGNVVNGRGEYFNRIAYLLKSPLETLGLEGEYDIFVNAGGGLTGQADAVKLGVARALCELDPDNRPLKEGYLTRDPRCKERKKYGLKARKAPQFSKRMARKKGGKKVKKNIPNGVAIQSTFNNTIVTISDGDAVSWSSAGASGFKGAKKGTPFAAQTAADNAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIVRGTLDATGVKDRRQSRSKYGAKRPKMARIAGVDLPRDKRVEVALTYVYGIGLSKAQVITGVNPDTRVRDLSDDALRITNYQIEGDLRREMNIKRLDIGYRGRRHRQGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQELMGYQIHETDTGSADLQVAFLTTRINQLTEHLKNKDSSRRGLLKMIGKRKRLLQFIKDARYQLIRLGIRRMATKERVGLVVSDKMDKTVVVAVENRSPHPKYKKIVVKTKRFKAHDEENQCKVGDRVRIRETPPTSKTKRWAIAEILK----------------',\n",
       " 'Synechococcus_sp._PCC_7002': 'KKLSRRLRAKVEDRYPLEALLLKETATAKFETAEAHIRLGIDPKYTDQQLRTTVAFPKGTGQVRVAVITSGEQVVASGADLVGSEELIEDIQKGMDFELIATPDMMPKVARLGRQLGPRGLMPSPKGGTVTDVAAIAFKAGKQEFRADRAGIVHVFGKSSFAEDLLNLKALQETIDRNRPSGAKGRYWRSYVSASMGPSIQVDINALRDLFMSLGILGTKLGMTQIFDETGTAIPVTVVQAGPCTVTQIKTSETDGYTSVQIGYGDVKEKNLTKPQLGHLKKADAAPLRHLKEYRLDDVSGYELGQAITANIFEAGQVVDVSGNTIGRGFAGYQKRHNFKRGNMTHGSKNHRLPGSTGAGTTPGRVYPGKRMAGRYGGTKITVRKLEVVRVDEEKNLLLIKGAIPGKAGNLLSIAPSNIVGQK--MRLKYDSIPKLEQFYSNVHEVPKLVKISVNRGLGEASSNAKAMESSIKELSITGQRPVITRAKKAIAGFKIREGMPVGVMVTLRSEKMYSFLDRLISLALPRIRDFRGISPKSFDGRGNYSLGIREQLIFPEIDYDSIDQIRGMDISIITTARTDEEGRALLREMGMPFRSMSRIGKKPIPIPDKVTVTLNGANVAVKGPKGSLERTLPEKVTVTQEDNTIVVSPVDNSRTAR--ERHGLCRTLVANMIEGVSQGYSKQLEIIGVGYRAQVQGNKLTLNVGYSKPVEMQMPEGVTAAMEKNTQVIISGIDKEVVGNTAAKVRAVRPPEPYKGKGIRYQGEYIRRKAGKAGKK---MAKKVVTVIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTSDQAGMIIPVEISVYEDRSFTFILKTPPASVLIRKAAGVEKGSAQPNKQKAGSISRAQLQEIAQTKMPDLNANDIEAAMNIVAGTARNMGITVTDMKTLPTLDQWYVVDADQRLGRLASEVAVLRGKNKAYTPHMDTGDFVIVVNAEKVVTGRKEQKLYRRHSGRPGGMKVETFQLQRIPERIIEAVKGMLPKNALGRRLFTKLKVYAGTHPHAQQPELINTIPGMIQQTYLNVADNSGARKLMCLRVLSTGNRYGIGDIIAVVKEAPNMVKKSDVVRAVIVRTRLRRASGSIRFDDNAAVIINEGNPKGTRVFGPVARELRDKNFTKIVSLAPEVIMKATRKQLQRRHFRIRRVGTDRPRLAVFRSHHIYAQVIDDTLAASTLDDLKGLGNSASVGNLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMTIDTQVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAVANAEHNGLDPALVVSAAFADGPLKRFRPRAQGRAYQIRKPTCHITVAVAPA--------------------------------------MPVVSLAE-----LLESGVHFGHQTRRWNPRMAPYIY-TSRNGVHIIDLVQTAQLVEQAYTYMKEASENGKHVLFVGTKRQAAGIIAQEAKRC-GAFYINQRWLGGMLTNWETIKTRVERLKELEALEESGNLARRPKKEASMLRRELDKLQKYLGGIKNMRKIPDIVVIIDQRREHNAIQECQKL--GIPIVSLLDTNC--DPQTVDLPIPANDDAIRSIKLIVGKLADAIYAGRH--GQPVDDGDYGDFDEAIDEY-------------ADETDSESEMGQKIHPVGFRLGITKE----------HLSRWYADPKQYPALVQEDY--KIRQHIDAN-L-----NNAGISKVLIERKADQVDLEIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIEALR----------------------TGLQDVLG--------DQRQIRINVVE-------------------------------------VNRVDADAALIAEYIVQQLERRVSFRRVVRQAVQRAQRA-EVQGIKIQVSG-----------------------------RLNGAEIARTE-----WVREG---RVPLHTLRADIDYSYKTAQTI-------------------------------------------YGILGIKVWIF-----------------KGEIIPGQEE----TSAAN---------------AAPLPRR---KSRRQQFEDRSEQ-MARYRGARLRIRRLGELPGLTRKSARREYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSEKQLIRYVRKARRGSTGILQLLERLDNTVFRLGMAGTIPGARQLVNHGHILVNGRVVDIPSYQCRPGDVITVRDRSRKLIENMEFPGLANIPHLEFDKTLGKVNGEREWVALQINELLVVEYYSRMA------MA--KRRKS-SKNK---EKETNWQERVIQIRRVSKVVKGGKKLSFRAIVVIGNETGKVGVGVGKAGDVIGAVRKGVSDAKKHVVDVPLTKTNTITHRINGVAGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVKNILAKQLGSSSPLNNARATVDALGNLRSFSSVAQERG--VSIERIYAMSRRKRVVPDYNNRLLSMTIRRVMKGKKSLARIVYDALIERTGDPIEFETAIRNLTPLVEVKARRVGGATYQVPMEVRGRGTALALRWLIGYARQRGKSMAKLAELMDAANETGGAIKKREDTHRMAEANKAFAHYRYMANDTISDMLTRIRNAQHDNTVPMTRMTRNIAEVLKDEGFINTEGEGEKLVISLKYKGRNRPIIKLQRVSKPGLRVYSNKLPRVLGGIGIAIISTSKGVMTDRQARKEGIGGEILCYVWMQRPIYWGTGRRKASIARVRLVPGSGKVVNNRGEYFNRIPYLLKSPLETLGLESEYDILVNAGGGLTGQADAVKLGVARALCELDPDNRPLKEGYLTRDPRCKERKKYGLKARKAPQFSKRMARKKGSKKVKKNIPNGVAIQSTFNNTIVTISDGDAVSWSSAGASGFKGAKKGTPFAAQTAADNAARRAMEQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIVRGTLDATGVKDRKQGRSKYGAKRPKMARIAGVDLPRDKRIEIALTYVYGIGLSRAQIITGVNPDTRVRDLSDDALRVTNYQVEGDLRREMNIKRLDIGYRGRRHRQGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQELMGYQVHETDTGSADLQVAFLTERINQLTEHLKNKDSSRRGLLKMIGKRKRLLQFIKDERYQLIRLGIRRMATKERVGMVVSDKMDKTVIVAVENRSPHPKYKKIVVKTKRFKAHDEENKCKVGDRVRIRETPPTSKTKRWTIAEILNQ---------------',\n",
       " 'Acaryochloris_marina_MBIC11017': 'KKVSRRMRLQVEDRYPLEALLLCKTATAKFESAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRIAVVARGEKVEAAGADVVGSEELITEIQQGMDFDLIATPDMMPQVAKVGRILGPRGLMPSPKAGTVTDLPAIEFKAGKLEFRADRTGIVHVFGKAKFSQDLLNLKALQDTIDRNRPSGAKGRYWRSYVSATMGPSIEVDISSLRDLNMSVGILGTKLGMTQVFDEAGRAIPVTVVQAGPCPVTQIKTAQTDGYTAIQIGYGSTTEKALTRPELGHLKKSGSTPVRHLQEYRLADVSEYELGQSISADLFSEGQLVDISGTSIGRGFAGYQKRHNFRRGPMGHGSKNHRAPGSTGAGTTPGRIYPGKRMAGQMGNVKKTIRKLTVVRVDAERNVLLIKGSVPGKPGALLNIQPATIVG-ES-MRLKYDTAPKLEQFYKNAHEIPKVTKVSVNRGLGEASQNAKALERSLEELAITGQRPVVTRAKKAIAGFKIRQGMPVGVMVTLRGERMYAFLDRLVNLALPRIRDFRGVSPKSFDGRGNYTLGLREQLIFPEVDYDSIDQIRGMDISIITTAKTDEEGRALLKVLGMPFRNMSRIGKVPIPIPDKVNVTIKGQAVTVKGPKGELSREITPEVAIEQADNLVTVTRRNESRIAR--QRHGLSRTLIANMVEGVSKGFEKKLQIQGVGYRAQVQGRNLILNVGYSNPVTIEPPEGVQVAVESNTNVIVSGINKEVVGNTAARIRAVRPPEPYKGKGIRYADEYVRRKVGKAGKK---MAKKVVAIIKLAIAAGKANPAPPIGPALGQHGVNIMMFCKEYNAKTSDQAGLVVPVEISVYEDRSFTFILKTPPASVLIKKAAGIEKGSGEPNAVQVGQITKAQLQEIAQTKMPDLNANDIEAAMNIVAGTARNMGVAVVDMKTLPTADKWYVVDANQRLGRLATQIAILRGKNKPYTPFMNTGDFVIVINAEKVVTGNKEQKLYRRHSGRPGGMKTETFHLQRIPERIIEAVKGMLPKNALGRQLFRSLKVYAGPHPHAQKPELIQTIPGMIQETTLNVADNSGAKRLLCIRVIGGGNRYGVGDIIATVKDAPNMVKKSDVVRAVIVRTRICRESGSIRFDDNAAVLINDGNPRGTRVFGPVARELRDKSFTKIVSLAPEVLMKTTRRDARSRHQRVRKVGTERPRLAVFRSNHIYAQVIDDTLAASTVEDLKS-GTDASVGKLVARAIEKGIAVVFDRGGNLYHGRVKALADAAREAGLEFMTTQLKAYAAKYIRSPKVRRVLDQIRGRSYREALIVLEFMPYRACEPVLKVLRSAVANAETNGLDPVLVVSHAYADGPLKRFRPRAQGRAYQIQKPTCHITIAVSAS--------------------------------------MPVISLAQ-----MLESGVHFGHQARRWNPKMDPYIF-TERNGVHIIDLVQTAQLMDEAYHYVRSASESGKKFLFVGTKRQAAGIIAQEAARC-GGYYVNQRWLGGMLTNWTTIKTRIERLKDLERRYESGIFDLLPKQEASMLRRELDKLQKYLGGLKQMHKIPDVVVIIDIRREYNAVQECQKL--GLPIVSLLDTNC--DPDWVDIPIPANDDAIRSIKLIIGKLADAIYEGRH--GQVSAE----TFEGDD---DFEDEPTAEAEAPAEDAATAEAMGQKIHPTGFRLGITQD----------HWSRWFADTDRYPELLQEDF--KVRNYVNKN-L-----SNAGIAGVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------SGIESLR----------------------VGLQKELG--------DDRPIRINVVE-------------------------------------VARVDAEATLIAEYIVQQLVRRVSFRRVVRQAIQRAQRA-GVEGIKIQVGG-----------------------------RLNGAEIARSE-----WTREG---RVPLHTLRANIDYSYRTASTT-------------------------------------------YGILGVKVWVF-----------------KGEVIPGADE----QPTNR----------------EPQQRRRQQQRRRQQFEDRSE--MSRYRGPRLRVRRLGELPGLSRKTPRRAYPPGQHGQRKKRSEYAAQLEEKQKLRFNYGLSERQLLRYVRKARRGSTGLLQLLERLDNTIFRLGMAPTIPAARQLVNHGHVLVNGRVVSIASYQCRPGDVIQSRDRSRKLIEHMQFPGLANIPHLDFDKTLGKVNGEREWIALEINELLVVEYYSRKG------MAKENRRKN-NRNK---EKDSTWQERVVQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVRKGVADGKKQLVEVPLTRSNSIPHPMIGRGGAAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSDNPLNNARATLHALASLRTFSEVAGERG--IEVEKLYAMSRRKRPVPDYNSRLISMTIRRLMAGKKSLARILYDALIERTNDPLEFETAVRNVTPLVEVKARRVGGATYQVPMEVRDRGIALALRWIIRFSRQRGRSMVKLAELIDAANETGAAIRKREETHRMAEANKAFAHYRYMANDTIGDMLTRIRNARHQTTVMSTRMTRSVASVLKDEGFISEEGEGERLVITLKYRGKNRPIITLQRISKPGLRVYSNRLPRVLGGIGIAIISTSSGIMTDRDARQTGVGGEVLCYVWMSAVAYRGTGRRKSSIVRVRLVPGSGVMINGKGDYLQFNPYIAKAPLESLGLENDYDVLVNAGGGLTGQADAIKLGVARALCELDPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMAPKRSGRKQKKHVPNGVAIQSTFNNTIVTISDGDVVSWASAGSSGFKGAKKGTPFAAQTASESAARRAMDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQGRSKYGAKRPKMARIAGVDLPRDKRIEIGLTYIYGIGLTRSQILTGVNPDTRVRELGDDVLRVAGYQVEGDLRREMNVKRLDIGYRGRRHRAGLPVRGQRTRTNSRTRRSKTVAGKKKAPMALLQKKHEIINYQVHETDTGSADVQVAMLTERISKLSEHLKNKDSSRRGLLKMIGQRKRLLTYIMDAHYRLIRLGIRGMAIKERIGVVVSDKMDKTVVVAVSNRVPHKKYGKIVGQTRRYKAHDEENACHVGDRVRIRESRPLSRTKRWAVTDVLVAANRP-----------',\n",
       " 'Oscillatoria_sp._PCC_7112': 'KKVSRRLQLKVEDRYPLEALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVSLPKGTGQVRIAVIAKGEKVEAAGADIAGSEELIDEIQKGMDFDLIATPDVMPLVAKLGRLLGPRGLMPSPKGGTVTDIAAIEFKGGKLEFRADRTGIVHVFGKAAFSEDLLNLHALQETVDRNRPSGAKGRYWRSYVSATMGPSIEVDINALRDTKMSVGILGTKLGMTQVFDDEGRAIPVTVIQAGPCTVTQIKTKQTDGYSAVQVGYNEVKPKALNKPELGHLAKSGANPLRHLQEYRLQDTSSFELGQQIKADTFTPGQIVDVIGTNIGKGFAGFQKRHNFKRGPMSHGSKNHREPGSIGPGTTPGRVYPGKRMAGRLGNVQVTVRKLTIVRVDAEQNLLLIKGAVPGKAGALVNVVPEKKVG-R--MKLKYEKIPKLDQFYTNIHQVPKLVKVTVNRGLGEASQNAKALDSSIDEIGITGQKPVVTRAKKAIAGFKIRKGVPVGVMVTLRADRMYDFLDRLINLALPRIRDFRGISPKSFDGRGNYSLGLREQLIFPEIEYDRIDQIRGMDISIITTANTDEEGRALLKEMGMPFRNMSRIGKRPISIPNKVTITLDGQKVAVKGPKGELSRVIPAEIVMGIEGETLLVTRRDESRVAR--QLHGLCRTLVANMVEGVSQGFQRRLEIIGTGYRAQVQGRNLILNVGYSKPVEMPPPEGITMAVEGNTNVIVSGIDKELVGNTAARIRAVRPPEVYKGKGIRYSGEVIRRKAGKTGKTGKKMAKKVVAVIKLAITAGKANPAPPIGPALGQHGVNIMMFCKEYNARTADQAGLVVPVEISVFEDRSFTFILKTPPASVLIQKAAGIAKGSGEPNRKQVGTITQAQLREIAETKMPDLNANDVEAAMKIVAGTAKNMGVAIADMKTVPSLEHWYVVDADQRLGRLATEVAYLRGKHKPYTPSMDTGDFVIVINADKIVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIVEAIRGMLPHTSLGRQLFTKLKVYAGPHPHAQKPELVHTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGVGDIIAVVKDAPNMVKKSDVVRAVIVRTRLRRDSGSIRFDDNAAVIINDNNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRKESQRRHRRVRKVGTERPRLAVFRSEHIYAQVIDDTLAASTLDELKSLGNSASVGQLIARCSSAGIKVVFDRGGNLYHGRVKALAEAAREAGLDFMAIDTTIKAARYIRSPKVRRVLDQIRGKSYQEALILLEFMPYRACEPIVKVLRSAVANAEHNGLDKSLVVSQAYADGPLKRFRPRAQGRAYQIRKPTCHITVVVAPE--------------------------------------MPVVSLAQ-----MLESGVHFGHQTRRWNPKMSPYIF-TERNGVHIIDLVQTAQLMEDAYEYMRVASEKGKKFLFVGTKRQAAGIVAQEAQRC-GAYYVNQRWLGGMLTNWTTIKSRVERLKDLERREEIGALDLLPKKEASVLRREMAKLQKYLGGIKSMRKIPDGVVLVDQRREYNAVLECQKL--GIPIVSLLDTNC--DPDLVDIAIPANDDAIRSIKLIVGKLADAIYEGRH--GQQGGDYDYGGYEGAEYDVEIEEL-DPDYVDPDDAAEEGEEMGQKIHPIGFRLGITQE----------HRSRWFADPKNYPELLQEDY--KIRKYVRKT-L-----NNAGISSVKIERKADQIDLEVF-------TARPGVVVGRGG-----------------------------------------------------------------------AGIETLR----------------------VGLQQQLG--------SNRQIRINVVE-------------------------------------VQRVDADATLIAEYIAQQLERRVSFRRVVRQAITRAQKV-GIQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAQTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEQ----EAAPN----------------AAVPRTK-QKRRRQNFEDRSNEGMSRYRGPRLRIRRLKELPGLTRKTPRRDYPPGQHGQRKKLSEYAVRLQEKQKLRFNYGVTERQLLRYVRKARRGSTGLLQFLERLDNTVFRMGMAPTIPSARQLVNHGHITVNDREVNIASYQCKPGDVIAVKNRSRTMVENLKDPGLAHLPHLEFDKKLGKVNGEREWIALQINELLVVEYYSRQAMAEQKEQRT-KKKSSSNKTR---EKESEWQERVVQIRRVTKVVKGGKKLSFRAIVIIGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEVPLTKSNSIPHPINGDGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNILAKQLGSGNPLNNARAAVNALSTLRTLSEVAQERG--IPVENLYGMSRRKRPVPDYNSRLVSMMVRRIMQGKKSVASIVYDALIERTGEPLDFEKAVKNATPLVEVKARRVGGATYQVPMEVRDRGTTLALRWLIRFARARGRSMAKLAELMDAANETGSAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNSRHQTTIPATKMTRSIAQVLKSEGFIEEEEEGKRLVLSLKYKGKTRPIIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQGLGGEVLCYVWMSRAVYWGTGRRKRSVARVRLVPGTGILVNGKGDYLQFNPYLAKAPLETLGLENEYDILVKVGGGLTGQADSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARGKSGKKQKRNVPNGVGIQSTFNNTIVTITDGEVISWASAGSSGFKGAKKGTPFAAQTAAESAARRANDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRARQKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGAKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDTRVKDLTEDVLRVETYQIEGDLRREMNIKRLDIGYRGRRHRLGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIMGFQTHETDTGSADVQVAMLTDRITKLSAHLKNKDASRRGLMMMISRRKRLLAYIQNDRYKLIRLGIRGMAIKERVGVVVSDKMDKTVVVAIENRSSHTKYGKIVVRTKRYKAHDEENQCKSGDRVRITETRPLSKTKRWTVAEIFGAKNG------------',\n",
       " 'Spirulina_major_PCC_6313': 'KKLSRRLQAKVEDRYPTEALLLKETATAKFETAEVHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIGRGEKVEAAGADLFGSEELIEEIQKGLDFDLIATPDMMPKVAKLGRMLGPKGLMPSPKAGTVTDVAAIEFKAGKLEFRADRTGIVHVFGKASFPEDLLNLKALQETVDRNRPSGAKGRYWRSYVSASMGPSIQVDISGLRDLKMSVGILGTKLGMTQIFDESGASIPVTVIQAGPCTITQIKQKQTDGYTAVQIGYQEVKEKALTKPELGHLSKANAAPLRHLKEYRLDDVSSFELGQALSADIFTPGQLVDVSGKTIGRGFAGYQKRHNFRRGPMSHGSKNHRLPGSTGPGTTPGRIFPGKKMAGQYGNTKVTIRKLQVVRVDTEKNLLLVRGAIPGKPGNVLSIVPANIVG----MRLKYDTIPKLEQFYTNVHQVPKVHKITLNRGLGEASQNAKALESSLSELAISGQRPVVTRAKKAIAGFKIREGMPVGVMVTLRSDRMYAFLDRLISLALPRIRDFRGISPKSFDGRGNYSLGVREQLIFPEIDYDSIDQIRGMDISIVTTANTDEEGRALLKEMGMPFRQMSRIGKRPITVPDKVTVTIDGNHVAVKGPKGQLERQIPGMVDIHQDGNTITVVRHNESRKCR--ERHGLSRTLIANMIEGVSNGYEKQLIIQGVGYRAQMQGTTLVLSVGYSHPVEIQPPEGVQFAVEKNTQVVVSGINKEIVGNVTAQIRAVRPPEVYKGKGIRYKGEVVRRKAGKAGKK---MAKKVVGLVKLALPAGKAAATPPVGPALGQHGVNIMAFCKEYNAKTADKAGLIIPVEISIYEDRSFTFILKTPPASVLIKKEAGIDKGAAQPNTQTVGSITRDQLRSIAEIKLPDLNANDIDAAMKIVAGTAKNMGVAITDMKTLPTVEKWYVVDADQRLGRLATEIAVLRGKNKTFTPHMDTGDYVIVVNAEKVVTGNKSQKLYRRHSGRPGGMKTETFHLQRIPERIVEAVKGMLPKNALGRKLFTNLKVYAGPHQHAQQPELIKTHA-MIQQTYLNVADNSGARKLMCLRVLGKGNRYGIGDIIAVVKEAPNQVKKSDVVTAVIVRTKLRRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDRNYTKIVSLAPEVIMKTTRKAQHRRHRRLRKVGTTRPRLAVFRSNHIYAQLIDDTLAASSLDELKTLGNDASVGTLIARAKAKGIHVVFDRGGNLYHGRVKALAEAAREAGLDFMAVDTRVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACGPILKVLRSAVANAEHNGLDPSLMVSSAYADGPLRRFRARAQGRAYQIQRPTCHITVAVAPD--------------------------------------MAVVSLAS-----LLESGVHFGHQTRRWNPKMDPYIY-TARNGVHIIDLVQTAQLMGEAYEYMRTAAEKGKKVLFVGTKRQAAGIIAQESARC-GAYFINQRWLGGMLTNWETIRTRIDRLKELEHLEESGNLDKRPKKEASVLRRELMKLQKYLGGIKNMRRPPDIVVIVDQRREYNAILECQKL--NIPIVSMLDTNC--DPDLVDIPVPANDDAIRSIKLIVGRLADAIYEGRH--GQLDVDDDYEEFDDELAEGEFDDEASGEEVNTEPATSDSEEMGQKVHPTGFRLGVIKE----------HRSRWFADRKRYPELLQEDH--KIREHVQAN-L-----SNAGISDVRIERKADQIDLEIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIEALR----------------------TGLQQLLG--------GTRQIRVNVIE-------------------------------------VARVDADSVLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-EVQGIKIQVGG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYRTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE--LAAATPS-----------------QQPAR--RQRRNRQFEDRSE--MSRYRGPRLRIRRLGELPGLSRKSSRRDYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGITEKQMVNYMKKARKGSTGLLQLLERLDNTVFRLGMAGTIPGARQLVNHGHLMVNGRVVDIASYQCRPGDVLKVRDRSKNLVKNMEYPGLANVPHLEYNKLMGKVTGEREWVALQINELLVVEYYSRKV------MAKTKGKKG-NRTK----KEELFQERVIQIRRVSKVVKGGKKLSFRAIVVVGDENGKVGVGVGKAADVIGAVRKGVADGKKQLVDVPLNKANSITHPARGVAGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVKNILAKQLGSSSPLNNARAAVDALETLRTFRDVAQERN--VPIEKLYIMSRRKRPVPDHNSRLVSMTARRLMRGKESLARLIYDAFIERTGDPLEFEQAIRNITPLVEVKARRVGGATYQVPMEVRNRGITLALRWLATYSRSRGRTMAKLAEIMDAANETGGAIRKREETHRMAEANKAFAHYRYMANDTISDMLTRIRNARHSTTVPSTRMTRDIARVLKDEGFIDEEGEGKKLTLSLKYAGRTRPIITLKRISKPGLRVYKNKLPRVLGGIGIAIISTSSGIMTDREARQRGIGGEVLCYIWM-RAVYWGTGRRKSSVARVRLVPGTGQIVNDRGDYFNYNPYLVKAPLETLGLEAEYDILVRAGGGLTGQSDAVKLGVARALCKLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQFSKRMARKKGGKKTKRNVPNGVAITSTFNNTIVTISDGDVISWASSGSSGFKGAKKGTPFAAQTAADSAARRAIEQGMRQLEVMVSGPGAGRETAIRALQSGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIVRGTLDATGVKDRRQSRSKYGTKRPKMARIAGIDLPRDKRIEIGLTYIYGIGLSRSHILTGVNPDTRVKDLADDVLRIENYQIEGDLRREMNIKRLDIGYRGRRHRQGLPVRGQRTRTNARTRRGRTVAGKKKAAMTITQRKQQLIVYQAHETDTGCSDIQIAVMTERINNLTTHLKNKDSSRRGLLKLIGKRRRLLNYIQDAHYQLIRLGIRRMAVKERVGLVVSNKMDKTVVVAIENRAPHPKYGKIVVKTKKYKAHDEDNKCNEGDRVRIRESRPLSRTKRWTVTEVLEITNR------------',\n",
       " 'Moorea_producens-heterotrophic_bacteria_co-culture_from_Jamaica_Moorea_producens_JHB_22AUG96-1_(Final_Draft_of_Linear_Chromosome_and_2_Linear_Plasmid_Scaffolds)': 'KKPSRRLQLKVEERYPIEALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVALPKGTGQIRVAVIARGEKVEAAGADIAGSEELIAEIQNGMDFDLIATPDMMPKVARLGRLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKVSFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSASMGPSIEVDIAILRELKVSVGILGTKLGMTQVFDDEGRAIPVTIVQAGPCTVTQIKTNQTDGYNAIQIGYDQVKPKAITKPEIGHLAKSSAPPLRHLREYRVDDTSKFELGQPVNTDMFSIGQIVDVSGKSIGRGFAGYQKRHNFRRGPMAHGSKNHRLPGSTGAGTTPGRTYPGKKMAGRYGGTKVTIRKLEVVKIDAERNLLAIKGAVPGKPGTLLSIAPTNKVG-G--MGLKYETIPKLEQFYKNIHQVPKLVKVSLNRGLGEASQNAKALESSVNEIAITGQKPVVTRAKQAIAGFKIRAGMPVGVTVTLRSERMYSFLERLINLALPRIRDFRGLSPRSFDGRGNYTLGVREQLIFPEVDYDSIDQIRGMDITIVTTANTDEEGRALLKEMGMPFRKMSRIGKRPIPIPSKVTVTIDGQQVKVKGSKGELDWVLPDTIKAEQVGETVQVSRREETRTAR--ESHGLSRTLVANMIEGVSKGFEKRLDIQGVGYRAQVQGRNLILNVGYSKPVEIVPPEGIQVKVENNTNVIVSGINKEVVGNTAAKIRAVRPPEVYKGKGIRYAGEIVRRKAGKAGKK---MAKKVVALIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTADQAGMVIPVEISVFEDRSFTFILKTPPASVLIKKAAGIEKGSAQPNRTRVGSLTRSQLQEIAQTKMPDLNANDIEAAMKIVEGTARNMGVAITDMKTIPSIEKWYVVDANQRLGRLASEIAILRGKNKPYTPHMDTGDFVIVVNAEKVVTGRKDQKLYRRHSGRPGGMKTETFQLQRIPERIIEAVKGMLPKNSLGRQLFKKLKVYTGQHPHAQQPQLIKTIPGMIQESYLNVADNSGARKIMCIRVIGAGNRYGVGDIIGVVKDAPNMVKKSDVVRAVIVRTRLRRESGSIRFDDNAAVIINDGNPRGTRVFGPVARELREKNFTKIISLAPEVLMKLTRKESRHRHRRVRKVGTERPRLAVFRSNHIYVQVIDDTIAASTLEALRSLGTEASVGQLVARALEKGLKVVFDRGGYLYHGRVKALAEAAREAGLDFMAIDTTTKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACQPVLKVLRSAVANAEHNGIDPALYITKAFADGPLKRYRPRAQGRAYQIRKPTCHITVAVSPD--------------------------------------MPVVSLAE-----MLESGVHFGHQTRRWNPKMDPYIY-TARNGVHIIDLVQTAQLMEDAYHYIRTSSEQGKRFLFVGTKRQAAGIIAQEASRC-GAYYVNQRWLGGMLTNWETIKTRVERLKDLERRETSGALDLLPKKEGSMLRRELGKLQKYLGGIKMMRKVPDIVIIIDQKREYNAILECEKL--SLPIVSLLDTNC--DPDLADIPIPANDDAIRSIKLIIGKLADAIYEGRH--GELDTAEEY-EYEEFEEGLEIQET-EQSESVDQGDNQTEQEVGQKIHPIGFRLGITQE----------HRSRWFADSKRYPLLLQEDY--QIRQYVEKN-L-----NNAGISDIRIERKADQIDLEIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIESLR----------------------VGLQKALH--------NTRQIRINVVE-------------------------------------VSRVDADATLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-DVQGIKIQVGG-----------------------------RLNGAEIARSE-----WTREG---RVPLHTLRANIDYSYKTAQTI-------------------------------------------YGILGIKVWVF-----------------KGEVIPGQEE----KPAPN----------------TAQPRRR-QPKRRQQFEDRSND-MSRYRGPRLRVRRLGDLPGLSRKTPRRAYPPGQHGQRRKRSEYAIRLEEKQKLRYNYGVTEKQLIRYVRKARRGSTGLLQLLERLDNTVFRMGMGGTIPAARQLVNHGHVTVNGQVVNIPSYQCRPGDVIAVRNRSRRLVENLEFPGLANLPHLEFDKTFGKVNGDREWVALQINELLVVEYYSRKV------MAN-RRKN--NRSK---EKESNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNEKGQVGVGVGKASDVIGAVRKGVADAKKQLIDVPLTKTNSIPHPSRGRAVGAKVIMRPAAPGTGVIAGGAVRTVLELAGVKNVLAKQLGSNNPLNNARAAINSLDSLRTFSEVAQERD--IPLEQIYAMSRRKRPVPDYNSRLICMLGCRLMRGKKSLAGIIYDAMIERTGDPIEFEKAVRNLTPLVEVKARRVGGATYQVPMEVKERGTTLALRWLIKFSRSRGRSMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMPNDTIADMLTRIRNARHQTTVQSTKMTRAIAQVLKEEGFIDEEGEGKKLVVSLKYKGNNRPIIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSNGIMTDREARRRGIGGEVLCYVWMARAVYWGTGRRKSSVARVRLVPGSGQLVNKRGDYFNFNPYLLKAPLETLGLENEYDILVNAGGGLTGQSDAVRLGVARALCELDPENRPLKEGYLTRDPRSKERKKYGLKARKAPQYSKRMARKKGGKKQKRNVPNGIAIQSTFNNTIVTISDGDVVSWASAGSSGFKGAKKGTPFAAQTAADSAARRAIDQGMRQVEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLISERKKKTKSPALKCPQRRGVCTRVFTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQGRSKYGTKRPKVARIASVDLPRDKRVEIGLTYIYGIGLSRSKILTGVNPDTRVKDLTDEVLRVENYQVEGDLRRQMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTIAGKKKAPMSLTQRKTEIINYQVHETDTGSAEVQVAILTERINTLSLHLQHKDSSRRGLLKMIGRRKRLLSYIRDQRYQLIRLGIRGMAAKERVGLVVSDKMDKTVVVAIENRSPHPKYGKIVVRTKRYKAHDEENKCKEGDRVRIRETRPLSRTKRWMVAEIVSSSSK------------',\n",
       " 'Moorea_bouillonii-heterotrophic_bacteria_co-culture_from_Papua_New_Guinea_Moorea_bouillonii_PNG_19MAY05-8_(One_linear_chromosome_and_12_unmapped_contigs)': 'KKPSRRLQLKVEERYPIEALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVALPKGTGQIRVAVIARGEKVEAAGADIAGSEELIAEIQNGMDFDLIATPDMMPKVARLGRLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQESIDRHRPSGAKGRYWRTYVSASMGPSIEVDIAILRELKVSVGILGTKLGMTQVFDDEGRAIPVTVVKAGPCTVTQIKTNQTDGYNAIQIGYDQVKPKAITKPEIGHLAKSSAPPLRHLREYRVDDTSNFELGQPVNTDMFSIGQIIDVSGKTIGRGFAGYQKRHNFRRGPMAHGSKNHRLPGSTGAGTTPGRTYPGKKMAGRYGGTKVTIRKLELVKIDAERNLLLIKGAVPGKPGTLLSIAPTNKVG-R--MGLKYETIPKLEQFYKNIHQVPKLVKVSLNRGLGEASQNAKALESSVSEIAITGQKPVVTRAKQAIAGFKIRAGMPVGVTVTLRSERMYSFLERLINLALPRIRDFRGLSPRSFDGRGNYTLGVREQLIFPEVDYDSIDQIRGMDITIVTTANTDEEGRALLKEMGMPFRKMSRIGKRPIPIPSKVTVTIDGQQVKVKGSKGELDWVLPDTIKAEQVGETVEVSRREETRTAR--ESHGLSRTLVANMIEGVSKGFEKRLDIQGVGYRAQVKGRNLILNVGYSQPVEIVPPEGIEVKVENNTNVIVSGINKEVVGNTAAKIRAVRPPEVYKGKGIRYAGEIVRRKAGKAGKK---MAKKVVALIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTADQAGMVIPVEISVFEDRSFTFILKTPPASVLIKKAAGIEKGSAQPNRTKAGSLTRSQLQEIAQTKMPDLNANDIEAAMKIVEGTARNMGVAITDMKTIPSIEKWYVVDANQRLGRLASEIAILRGKNKPYTPHMDTGDFVIVVNAEKVVTGRKEQKLYRRHSGRPGGMKTETFQLQRIPERIIEAVKGMLPKNSLGRQLFKKLKVYTGQHPHAQQPQLIKTIPGMIQESYLNVADNSGARKIMCIRVIGAGNRYGVGDIIGVVKDAPNMVKKSDVVRAVIVRTRLRRESGSIRFDDNAAVIINDGNPRGTRVFGPVARELREKNFTKIISLAPEVLMKLTRKESRHRHRRVRKVGTERPRLAVFRSNHIYVQVIDDTIAASTLEALRSLGTEASVGQLVARALEKGLKVVFDRGGYLYHGRVKALAEAAREAGLDFMAIDTTTKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACQPVLKVLRSAVANAEHNGIDPALYITKAFADGPLKRYRPRAQGRAYQIRKPTCHITVAVSPD--------------------------------------MPVVSLAE-----MLESGVHFGHQTRRWNPKMDPYIY-TARNGVHIIDLVQTAQLMEDAYHYIRTSSEQGKRFLFVGTKRQAAGIIAQEASRC-GAYYVNQRWLGGMLTNWETIKTRVERLKDLERRETSGALDLLPKKEGSMLRRELGKLQKYLGGIKMMRKVPDIVIIIDQKREYNAILECEKL--SLPIVSLLDTNC--DPDLADIPIPANDDAIRSIKLIIGKLADAIYEGRH--GQLDAAYEYEEFEEGLEEFEIQET-EQSESVDQGDNQTEEEVGQKIHPIGFRLGITQE----------HRSRWFADSKRYPLLLQEDY--QIRQYVDKN-L-----NNAGISDIRIERKADQIDLEIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIESLR----------------------VGLQKALG--------NNRQIRINVVE-------------------------------------VSRVDADATLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-DVQGIKVQVGG-----------------------------RLNGAEIARSE-----WTREG---RVPLHTLRANIDYSYKTAQTI-------------------------------------------YGILGIKVWVF-----------------KGEVIPGQEE----KPTPN----------------TAQPRRR-QPKRRQQFEDRSND-MSRYRGPRLRVRRLGDLPGLSRKTPRRAYPPGQHGQRRKRSEYAIRLEEKQKLRYNYGVTEKQLIRYVRKARRGSTGLLQLLERLDNTVFRMGMGGTIPAARQLVNHGHVTVNGQVVNIASYQCRPGDVIAVRNRSRRLVENLEFPGLANLPHLEFDKTFGKVNGDREWVALQINELLVVEYYSRKV------MAN-RRKS--NRSK---EKESNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNEKGQVGVGVGKASDVIGAVRKGVADAKKQLIDVPLTKTNSIPHPSRGRAVGAKVIMRPAAPGTGVIAGGAVRTVLELAGVKNVLAKQLGSNNPLNNARAAINSLDSLRTFSEVAQERD--IPLEQIYAMSRRKRPVPDYNSRLICMLGCRLMRGKKSLARIIYDAMIERTGDPIEFEKAVRNLTPLVEVKARRVGGATYQVPMEVKERGTTLALRWLIKFSRSRGRSMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMPNDTIADMLTRIRNARHQTTVQSTKMTRAIAQVLKEEGFIDEEGEGKKLVVSLKYKGNNRPIISLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSNGIMTDREARRRGIGGEVLCYVWMARAVYWGTGRRKSSVARVRLVPGSGQLVNKRGDYFNFNPYLLKAPLETLGLENEYDILVNAGGGLTGQSDAVRLGVARALCELDPENRPLKEGYLTRDPRSKERKKYGLKARKAPQYSKRMARKKGGKKQKRNVPNGIAIQSTFNNTIVTISDGDVVSWASAGSSGFKGAKKGTPFAAQTAADSAARRAIDQGMRQVEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLISERKKKTKSPALKCPQRRGVCTRVFTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQGRSKYGTKRPKVARIASVDLPRDKRVEIGLTYIYGIGLSRSKILTGVNPDTRVKDLTDEVLRVENYQVEGDLRRQMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTIAGKKKAPMSLTQRKTEIINYQVHETDTGSAEVQVAILTERINTLSLHLQHKDSSRRGLLKMIGRRKRLLSYIRDQRYKLIRLGIRGMAAKERVGLVVSDKMDKTVVVAIENRSPHPKYGKIVVRTKRYKAHDEENKCKEGDRVRIRETRPLSRTKRWMVAEIVSSSSK------------',\n",
       " 'Moorea_producens-heterotrophic_bacteria_co-culture_from_Palmyra_Atoll_Moorea_producens_PAL_15AUG08-1_(Complete_Genome': 'KKPSRRLQLKVEERYPIEALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVALPKGTGQIRVAVIARGEKVEAAGADIAGSEELIAEIQNGMDFDLIATPDMMPKVARLGRLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSASMGPSIEVDIAILRELKVSVGILGTKLGMTQVFDDEGRAIPVTVVKAGPCTVTQIKTNQTDGYNAIQIGYDQVKPKAITKPEIGHLAKSSAPPLRHLREYRVDDSSKFELGQPVNTDMFSIGQIVDVSGKTIGRGFAGYQKRHNFRRGPMAHGSKNHRLPGSTGAGTTPGRTYPGKKMAGRYGGTKVTIRKLEVVKIDAERNLLLIKGAVPGKPGTLLSIAPTNKVG-R--MGLKYETIPKLEQFYKNIHQVPKLVKVSLNRGLGEASQNAKALESSVSEIAITGQKPVVTRAKQAIAGFKIRAGMPVGVTVTLRSERMYSFLERLINLALPRIRDFRGLSPRSFDGRGNYTLGVREQLIFPEVDYDSIDQIRGMDITIVTTANTDEEGRALLKEMGMPFRKMSRIGKRPIPIPSKVTVTIDGQQVKVKGSKGELDWVLPDTIKAEQVGETVEVSRREETRTAR--ESHGLSRTLVANMIEGVSKGFEKRLDIQGVGYRAQVKGRNLILNVGYSQPVEIVPPEGIEVKVENNTNVIVSGINKEVVGNTAAKIRAVRPPEVYKGKGIRYAGEIVRRKAGKAGKK---MAKKVVALIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTADQAGMVIPVEISVFEDRSFTFILKTPPASVLIKKAAGIEKGSAQPNRTRVGTLTRSQLQEIAQTKMPDLNANDIEAAMKIVEGTARNMGVAITDMKTIPSIEKWYVVDANQRLGRLASEIAILRGKNKPYTPHMDTGDFVIVVNAEKVVTGRKEQKLYRRHSGRPGGMKTETFQLQRIPERIIEAVKGMLPKNSLGRQLFKKLKVYTGQHPHAQQPQLIKTIPGMIQESYLNVADNSGARKIMCIRVIGAGNRYGVGDIIGVVKDAPNMVKKSDVVRAVIVRTRLRRESGSIRFDDNAAVIINDGNPRGTRVFGPVARELREKNFTKIISLAPEVLMKLTRKESRHRHRRVRKVGTERPRLAVFRSNHIYVQVIDDTIAASTLEALRSLGTEASVGQLVARALEKGLKVVFDRGGYLYHGRVKALAEAAREAGLDFMAIDTTTKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACQPVLKVLRSAVANAEHNGIDPALYITKAFADGPLKRYRPRAQGRAYQIRKPTCHITVAVSPD--------------------------------------MPVVSLAE-----MLESGVHFGHQTRRWNPKMDPYIY-TARNGVHIIDLVQTAQLMEDAYHYIRTSSEQGKRFLFVGTKRQAAGIIAQEASRC-GAYYVNQRWLGGMLTNWETIKTRVERLKDLERRETSGALDLLPKKEGSMLRRELGKLQKYLGGIKMMRKVPDIVIIIDQKREYNAILECEKL--SLPIVSLLDTNC--DPDLADIPIPANDDAIRSIKLIIGKLADAIYEGRH--GQLDTAEEY-EYEEFEEGLEIQET-EQSESVDQGDNQTEQEVGQKIHPIGFRLGITQE----------HRSRWFADSKRYPLLLQEDY--QIRQYVEKN-L-----NNAGISDIRIERKADQIDLEIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIESLR----------------------VGLQKALG--------NNRQIRINVVE-------------------------------------VSRVDADATLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-DVQGIKVQVGG-----------------------------RLNGAEIARSE-----WTREG---RVPLHTLRANIDYSYKTAQTI-------------------------------------------YGILGIKVWVF-----------------KGEVIPGQEE----KPAPN----------------TAQPRRR-QPKRRQQFEDRSND-MSRYRGPRLRVRRLGDLPGLSRKTPRRAYPPGQHGQRRKRSEYAIRLEEKQKLRYNYGVTEKQLIRYVRKARRGSTGLLQLLERLDNTVFRMGMGGTIPAARQLVNHGHVTVNGQVVNIASYQCRPGDVIAVRNRSRRLVENLEFPGLANLPHLEFDKTFGKVNGDREWVALQINELLVVEYYSRKV------MAN-RRKS--NRSK---EKESNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNEKGQVGVGVGKASDVIGAVRKGVADAKKQLIDVPLTKTNSIPHPSRGRAVGAKVIMRPAAPGTGVIAGGAVRTVLELAGVKNVLAKQLGSNNPLNNARAAINSLDSLRTFSEVAQERD--IPLEQIYAMSRRKRPIPDYNSRLICMLGCRLMRGKKSLARIIYDAMIERTGDPIEFEKAVRNLTPLVEVKARRVGGATYQVPMEVKERGTTLALRWLIKFSRTRGRSMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMPNDTIADMLTRIRNARHQTTVQSTKMTRAIAQVLKEEGFIDEEGEGKKLVVSLKYKGNNRPIISLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSNGIMTDREARRRGIGGEVLCYVWMARAVYWGTGRRKSSVARVRLVPGSGQLVNKRGDYFNFNPYLLKAPLETLGLENEYDILVNAGGGLTGQSDAVRLGVARALCELDPENRPLKEGYLTRDPRSKERKKYGLKARKAPQYSKRMARKKGGKKQKRNVPNGIAIQSTFNNTIVTISDGDVVSWASAGSSGFKGAKKGTPFAAQTAADSAARRAIDQGMRQVEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLISERKKKTKSPALKCPQRRGVCTRVFTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQGRSKYGTKRPKVARIASVDLPRDKRVEIGLTYIYGIGLSRSKILTGVNPDTRVKDLTDEVLRVENYQVEGDLRRQMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTIAGKKKAPMSLTQRKTEIINYQVHETDTGSAEVQVAILTERINTLSLHLQHKDSSRRGLLKMIGRRKRLLSYIRDQRYKLIRLGIRGMSAKERVGLVVSDKMDKTVVVAIENRSPHPKYGKIVVRTKRYKAHDEENKCKEGDRVRIRETRPLSRTKRWMVAEIVSSS--------------',\n",
       " 'Moorea_producens-heterotrophic_bacteria_co-culture_from_Cura?ao_Moorea_producens_3L_(One_linear_chromosome_and_78_unmapped_contigs)': 'KKPSRRLQLKVEERYPIEALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVALPKGTGQIRVAVIARGEKVEASGADIAGSEELIAEIQNGMDFDLIATPDMMPKVARLGRLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKVSFSEDLLNLKALQETIDRHRPSGAKGRYWRTYVSASMGPSIEVDIAILRELKVSVGILGTKLGMTQVFDEEGRAIPVTVVKAGPCTVTQIKTNQTDGYNAIQIGYDQVKPKAINKPEIGHLAKSSAPPLRHLREYRVDDTSNFELGQPVNTDMFSIGQIVDVSGKSIGRGFAGYQKRHNFRRGPMAHGSKNHRLPGSTGAGTTPGRTYPGKKMAGRYGGTKVTIRKLEVVKIDQERNLLAIKGAVPGKPGTLLSIAPTNKVG-R--MGLKYETIPKLEQFYKNIHQVPKLVKVSLNRGLGEASQNAKALESSVNEIAITGQKPVVTRAKQAIAGFKIRAGMPVGVTVTLRSERMYSFLERLINLALPRIRDFRGLSPRSFDGRGNYTLGVREQLIFPEVDYDSIDQIRGMDITIVTTANTDEEGRALLKEMGMPFRKMSRIGKRPIPIPSKVTVTIDGQQVKVKGSKGELDWVLPDTIKAEQVGETVQVSRREETRTAR--ESHGLSRTLVANMIEGVSKGFEKRLDIQGVGYRAQVQGRNLILNVGYSKPVEIVPPEGIQVKVENNTNVIVSGINKEVVGNTAAKIRAVRPPEVYKGKGIRYAGEIVRRKAGKAGKK---MAKKVVALIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTADQAGMVIPVEISVFEDRSFTFILKTPPASVLIKKAAGIEKGSAQPNRTKAGSLTRSQLQEIAQTKMPDLNANDIEAAMKIVEGTARNMGVAITDMKTIPSIEKWYVVDANQRLGRLASEIAILRGKNKPYTPHMDTGDFVIVVNAEKVVTGRKDQKLYRRHSGRPGGMKTETFELQRIPERIIEAVKGMLPKNSLGRQLFKKLKVYTGQHPHAQQPQLIKTIPGMIQESYLNVADNSGARKIMCIRVIGAGNRYGVGDIIGVVKDAPNMVKKSDVVRAVIVRTRLRRESGSIRFDDNAAVIINDGNPRGTRVFGPVARELREKNFTKIISLAPEVLMKLTRKESRHRHRRVRKVGTERPRLAVFRSNHIYVQVIDDTIAASTLEALRSLGTEASVGQLVARALEKGLKVVFDRGGYLYHGRVKALAEAAREAGLDFMAIDTTTKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACQPVLKVLRSAVANAEHNGIDPALYITKAFADGPLKRYRPRAQGRAYQIRKPTCHITVAVSPD--------------------------------------MPVVSLAE-----MLESGVHFGHQTRRWNPKMDPYIY-TARNGVHIIDLVQTAQLMEDAYHYIRTSSEQGKRFLFVGTKRQAAGIIAQEASRC-GAYYVNQRWLGGMLTNWETIKTRVERLKDLERRETSGALDLLPKKEGSMLRRELGKLQKYLGGIKMMRKVPDIVIIIDQKREYNAILECEKL--SLPIVSLLDTNC--DPDLADIPIPANDDAIRSIKLIIGKLADAIYEGRH--GELDTAYEYEEFEEALEEFEIQET-EQSESVDQGDNQTEQEVGQKIHPIGFRLGITQE----------HRSRWFADSKRYPLLLQEDY--QIRQYVDKN-L-----NNAGISDIRIERKADQIDLEIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIESLR----------------------VGLQKSLG--------NNRQIRINVVE-------------------------------------VSRVDADATLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-DVQGIKVQVGG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRANIDYSYKTAQTI-------------------------------------------YGILGIKVWVF-----------------KGEVIPGQEE----KPTPN----------------TAQPRRR-QPKRRQQFEDRSND-MARYRGPRLRVRRLGDLPGLSRKTPRRAYPPGQHGQRRKRSEYAIRLEEKQKLRYNYGVTEKQLIRYVRKARRGSTGLLQLLERLDNTVFRMGMGGTIPAARQLVSHGHVTVNGTVVNIPSYQCRPGDVIAVRNRSRRLVENLEFPGLANLPHLEFDKTFGKVNGDREWVALQINELLVVEYYSRKV------MAN-RRKN--NRSK---EKESNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNEKGQVGVGVGKASDVIGAVRKGVADAKKQLIDVPLTKTNSIPHPSRGRAVGAKVIMRPAAPGTGVIAGGAVRTVLELAGVKNVLAKQLGSNNPLNNARAAINSLDSLRTFSEVAQERD--IPLEQIYAMSRRKRPVPDYNSRLICMLGCRLMRGKKSLARIIYDAMIERTGDPIEFEKAVRNLTPLVEVKARRVGGATYQVPMEVKERGTTLALRWLIKFSRSRGRSMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMPNDTIADMLTRIRNARHQTTVQSTKMTRAIAQVLKEEGFIDEEGEGKKLVVSLKYKGNNRPIIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSNGIMTDREARRRGIGGEVLCYVWMARAVYWGTGRRKSSVARVRLVPGSGQLVNKRGDYFNFNPYLLKAPLETLGLENEYDILVNAGGGLTGQSDAVRLGVARALCELDPENRPLKEGYLTRDPRSKERKKYGLKARKAPQYSKRMARKKGGKKQKRNVPNGIAIQSTFNNTIVTISDGDVVSWASAGSSGFKGAKKGTPFAAQTAADSAARRAIDQGMRQVEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLISERKKKTKSPALKCPQRRGVCTRVFTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQGRSKYGTKRPKVARIASVDLPRDKRVEIGLTYIYGIGLSRSKILTGVNPDTRVKDLTDEVLRVENYQVEGDLRRQMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTIAGKKKAPMSLTQRKTEIINYQVHETDTGSAEVQVAILTERINTLSLHLQHKDSSRRGLLKMIGRRKRLLSYIRDQRYQLIRLGIRGMAAKERVGLVVSDKMNKTVVVAIENRSPHPKYGKIVVRTKRYKAHDEENKCKEGDRVRIRETRPLSRTKRWMVAEIVSSSSK------------',\n",
       " 'Calothrix_sp._336/3_null_replaces_82363': 'RKISKRLQLKVEDKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVITRGEKVEAAGADIVGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEYRADKTGIVHVFGKAAFAEDLLNLKALQESIDRNRPSGAKGRYWRTYISATMGPAIRIDVGALRELKVSVGILGTKLGMTQVFDEGGVAIPVTVIQAGPCTVTQVKTQQTDGYSAIQVGFGEVKPKALSKPKLGHLAKSSAPSLRHLREYRIDNASEYNLGQEIKADIFSAGEIVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRLAGRMGGCRVTVKKLTVVRVDTERNLILIKGAVPGKPGTLLNIVPTKKVG-K--MRLKYETIPKLQQFYTNVHQAPKLVKVTVNRGLGEAASNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGDRMYAFLDRLINLALPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDRIDQIRGMDISIITTAKTDEEGRALLKEMGMPFRQMSRIGKRPIAVPAKVEIKIDGTKIVVKGPKGELSRELPTQVSVSQEGQTLTVERKDDSRVSR--QMHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLVMNMGYSHQVQIVPPEGIQFAVENNTNVIVTGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIERGSNEPNKKKVGSITTEQLRQIAQTKMPDLNANDIDAAMKIIAGTARNMGVTITDDKTLPSIERWYVVDADQRLGRLATEIAVLRGKNKPYTPHMDTGDFVIVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERIIEAVKGMLPKNSLGRNLFTKLKVYAGPHPHAQQPKLVNTIPGVIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMVKKSDVVEAVIVRTRIRRDSGTIRFDDNAAVIINDGNPKGTRVFGPVARELRDKSFTKIVSLAPEVLMKLTRRESIRRHRRVRKVGSERPRLAVFRSHHIYAQVIDDTISASTLEDLKSLGNEASVGKLIARAKEKGIQIVFDRGGNLYHGRIKALAEAAREAGLDFMAVDTKVKAAKFVRSPKVRRVLDQIRGRSYREALIILEFMPYRACDPVLKLLRSAAANAEHNGLDRALKITQAYADGPLKRFQPRAQGRAYQIRKPTCHITLAVATA--------------------------------------MAVVSLAQ-----MMESGVHFGHQTRRWNPKMDPYIY-TARNGVHIIDLVQTAQLMEEAYAYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVERLKDLERREETGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECQKL--SIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEDDYEDYEGAEEEFDYEET-LTDSLIPDEE---EEEVGQKIHPVGFRLGITQE----------HQSRWFAEPSRYPELLQEDY--KLRNYIESK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------GNRQIRINVVE-------------------------------------VQKVDADAYLIAEFIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVGG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPS-------------GPREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKIRLNYGVTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPGARQLVSHCHVTVNGRVVNIASYQCRPGDVVAVRDKSRKLVENLQYPGLANLPHLEFDKKMGKVNSEREWVALQVNELLVVEYYSRQA------MAT-NRRKT-NRAK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGTGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSGNPLNNARAAVNALSTLRTFAEVAEERG--IPLENLYIMSRRRRPVPDYNSRLISMIIRRIMRGKKSIARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRDRGTTLALRWLVQYSRQRGRTMAKLAELMDAANESGSAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARQQTTVPATKMTRSIAKVLREEGFIDEEGEGKKLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSATARVRLVPGSGQLVNGKGDYFQYNAYLIKGPLETLGLESEYDILVKAGGGLTGQSDAIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKGGKKTKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRKRRQKTKSPALKCPQRRGVCTRVFTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIVRGTLDTAGVKDRKQGRSKYGTKRPKVARIAGVDLPRDKRVEIGLTYIYGIGLSRSKILTGVNPDTRVKELSDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIMTYQVHETDTGSADLQIAMLTARINRLSEHLQNKDSSRRGLLKMIGQRKRLLSYIIDERYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTRRYKAHDEDNQCKVGDRVRIQETRPLSKTKRWQVTEVLNK---------------',\n",
       " 'Oscillatoria_acuminata_PCC_6304': 'KKISRRLQLKVEERYPLEALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADLAGSEDLIAEIQQGMDFDLIATPDMMPQVAKLGKLLGPRGLMPSPKGGTVTDLGAIEFKAGKLEFRADRTGIVHVFGKAAFPEDLLNLKALQESIDRNRPSGAKGRYWRSYVTSTMGPSIEVDISALRDLKMAVGILGTKLGMTQVFDEEGRAIPVTVVQAGPCTVTQIKSKQTDGYSAIQVGYGETTTKAINKPELGHLVKSSAPPLRHLREYRIDDVSSFELGQQMTVEGFTANQIVDVTGITIGRGFAGYQKRHNFKRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRMAGRLGGSQITTRKLEIVRVDAEKNLLLIKGALPGKPGALLNIKPANYVG-RAKMQLKYETIPKLEQFYTNIHQVPKVVKITINRGLGEASSNAKALESSLSEIAIAGQKPVVTRAKKAIAGFKIRQGMPVGMMVTLRGDRMYSFLDRLISLALPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDRIDQIRGMDISIITTATTDEEGRALLKEMGMPFKGMSRIGKRPINIPAKVVVSIDGQKVSVKGPKGELSRELPAEVIVAQEGETIEVKRRDESRAAR--QRHGLCRTLVANMVQGVSEGFQKRLEIQGVGYRAQVQGRNLILNVGYSNPVEINPPEGISVAVENNTNVIISGINKEIVGNTAAKIRDVRPPEPYKGKGIRYAGEQVRRKAGKTGKK---MAKKVVAVIKLAINAGKANPAPPIGPALGQHGVNIMMFCKEYNARTADQAGLVVPVEISVFEDRSFTFILKTPPASVLIRKAAGIERGSGEPNRKKVAEISRDQLREIAQTKMPDLNANDIEAAMKIVEGTARNMGVTVGDMKTVPSIEQWYVIDANQRLGRLAVEIAILRGKNKPYTPHMDTGDFVIVVNAEKIVTGKKSQKLYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLYTKLNVYAGSHPHAQKPELINTIPGMIQESYLNVADNSGAKRLMCIRVL-GGNRYAVGDIIAVVKDAPNMIKKSDVVRAVVVRTRLRRDSGSIRFDDNAAVIVNDGNPKGTRVFGPVARELRDKNYTKIVSLAPEVLMKLTRSESERRHRRVRRVGTERPRLSVFRSHHIYAQVIDDTLAASTLDALKALGTDASVGKLIARAREKGIKVVFDRGGNLYHGRVKALAEAAREAGLEFMAIDTSVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACDPIRKVLRSAAANAEHNGLDPALWVSQAYADGPLKRFRPRAQGRAYMIRKPTCHITVAVAPD--------------------------------------MGVVSLAQ-----LLESGVHFGHQTRRWNPKMSPYIF-TERNGVHIIDLVQTAQLMEEAYDYMRSASERGEKVLFVGTKRQAAGIIAQEASRC-GAYYVNQRWLGGMLTNWTTIKTRADRLKELERMEDTGAMDRRPKKEASMLRRELSKLQKYLGGIKNMRKIPDIVVIVDQRREYNAVQECQKL--DIPIVSLLDTNC--DPDFTDIPIPGNDDAIRSIKLIVGKLADAIYEGRH--GEVDSSEEF-DYEGVEGEYEYDEE-GEEAYQPDSEAESDEKMGQKIHPTGFRLGVTKE----------HRSRWFADSNRYPELLEEDY--KIREYVHST-L-----NNAGISDVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------SGIESLR----------------------TGLQDMLG--------SNRQIRINVVE-------------------------------------VARVDADATLIAEYIASQLERRVSFRRVVRQAIQRAQKA-GVEGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYRTASTI-------------------------------------------YGILGVKVWTF-----------------KGEIIPGQEL----EATAA---------------ASAGPRPRGGKKRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLSRKTPKRAYPPGQHGQRKKRSEYAVRLEEKQKLRFNYGLSEKQLLRYVRRARRGSTGLLQLLERLDNTIFRMGMAPTIPAARQLVNHGHLTVNGRVVDIPSYNCRPGDQIVVRDRSRQMVQNLQSPGLANMPHLEFDKKLGKVNGEREWIALQINELLVVEYYSRQA------MAE-RRKKN-ARTR---EKETDWQERVVQIRRVTKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVISAVKKGVADGKKHLIEVPLTKANSIPHRVNGSATGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNILAKQLGSNNPLNNARAAIDALVSLRTLSEVADERG--IAIDKLYAMSRRKRSVPDYNSRLVGMMVRRVMKGKKSVANIVYNAMIERTGDALEFEKAVKNVTPLVEVKGRRVGGATYQVPMEVRDRGTALALRWIIHFSRTRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQKTVLSTNMTRSIAKVLQEEGFIEEEGEGDKLAIALKYKGKNRPIIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDRDARRQGVGGEVLCYVWMSRVMYWGTGRRKSSVARVRLVPGNGTVINGKAEYLQYNAYLTKAPLETLGLENEYDVLVNVGGGLTGQAESIRLGVARALCELDPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGRKQKRNVPNGVGIQSTFNNTIVTITDGEVISWATSGSSGFKGAKKGTPFAAQTAAENAARRAVEQGMRQIEVMVSGPGSGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIREREKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQGRSKYGAKRPKMARIAGVDLPRDKRVEIGLRYIYGIGPTRAQILTGVNPDTRVKDLTDEVLRLENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMTLTQRKHELINYQIHETDTGSADVQVAMLTTRINQLSAHLKNKDSSRRGLLKMIGQRKRLLAYILDARYRLIRLGIRGMAVKERVGVVVSDKMDKTVVVAIENRSPHPKYGKIVVKTKRYKAHDEENKCKEGDRVRIQETRPLSRTKHWQVIEIINSATQI-----------',\n",
       " 'Geitlerinema_sp._PCC_7407': 'KKVSRRLQLKVEDRYPLEALLLKETSTAKFESAEAHIRLGLDPKYTDQQLRTTVALPKGTGQVRIAVIARGEKVEAAGADLVGSEELINDIQKGMDFDLIATPDMMPQVAKLGRQLGPRGLMPSPKGGTVTDLEAIEFKAGKLEFRADRTGIVHVFGKASFAEDLLNLKALQECIDRNRPSGAKGRYWRSYVSSTMGPSIEIDVNSLRDLKMSVGILGTKLGMTQVFDEAGRAIPVTVIQAGPCIVTQVKTKETDGYNAIQLGYNEVKEKALSKPELGHLKKAEAPALRHLKEYRADDSSQFQLGQSLGADTFSAGQIVDVTGTSIGKGFAGYQKRHNFKRGPMAHGSKNHRLPGSTGAGTTPGRVYPGKRMAGRLGGSQVTIKKLTVVRVDAERNLILIQGAVPGKTGALLSIAPAKLVG-RA-TRIKYQTVPKLAQFYQNIHQVPKVIKVTVNRGLGEAAQNAKALESSLTEIAITGQKPVVTRAKQAIAGFKIRQGMPIGIMVTLRRERMNAFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEISYDDIDQIRGFDISIVTTANTDEEGRALLKELGMPFRNMSRIGKRPISIPAKVTVSISGQSITVKGPKGELSRTFPAGVVFEQDGETILVKRRDESRVAR--ERHGLCRTLLSNMVEGVSQGFQKRLEIQGVGYRAQVQGRNLILNVGYSHQVNIEPPEGVQLAVENNTNVIVSGFDKEAVGNIAASIRAVRPPEPYKGKGIRYAGEVVRRKAGKAGKK---MAKKVVSVIKLAIQAGKANPAPPIGPALGQHGVNIMAFCKEYNARTADQAGMVIPVEISVYEDRSFTFILKTPPASVLIRKAAGVERGSGNPNSQKVGSISRDQLREIAQTKMPDLNANDVEAAMKIIEGTARNMGVTIKDMKTLPSRVDWHVIDADQRLGRLASEVAILRGKHKPYTPSMDTGDHVIVINADKIVTGRKTQKLYRRHSGRPGGMKVETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGAHPHAQKPQLIQTVPGMIQETYLNVADNSGARKLMCIRVL-GGNRYAVGDIIAVVKDAPNMIKKSDVVRAVIVRTRLRRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKMSRKESRRRHQRVRKLGTERPRLAVFRSNHIYVQAIDDTISASTVEALRSFGTEASVGKLVARLIEKGVQVVFDRGGNLYHGRVKALAEAAREAGLDFMAVDTRVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPILKVLRSAVANAEHNGLDPALFVSKAFADGPLKRFRPRAQGRAYQIRRPTCHVTVVVSPK--------------------------------------MPVVSLAQ-----LLEAGVHFGHQTRRWNPSMAPYIY-TARNGVHIIDLVQTARCMEDAYQFMRTSAEQGKKFLFVGTKRQAAGIIAQEAQRC-GAHYVNQRWLGGMLTNWATIKTRIERLKELERREETGALDLLPKKEASVLRRELEKLRKYLGGLRSMRKLPDIVVIVDQRREYNAVLECQKI--GVPIVALLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEEDYEDYEGAEEDYDYEES-ASGD---DEG---ADNMGQKIHPTGFRLGVTKE----------HRSRWFADFDRYPELLQEDS--KIRSYVQKN-L-----SNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------SGIENLR----------------------TGLQDLLH-------NGDRQIRINVVE-------------------------------------VARVDADAALISEYIAQQLERRVSFRRVVRQAMQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAQTT-------------------------------------------YGVLGIKTWVF-----------------KGEIIPGQEE----VAAPP----------------SNQPRRRQSQRRRQQFEDRSNDNMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKKSEYAIRLEEKQKLRYNYGITERQLVRYVRKARSGSTGLLQLLERLDNTIFRMGLAPTIPAARQLVNHGHVMVNGRVVSIPSYNCRPGDAIAIRNRSRKLAENLQSPGLAHVPHLEFDKKMGKVNSEREWVALQVNELLIVEFYSRKV------MTN-RRKS--NRSR---EKETDWQERVVQIRRVTKVVKGGKKLSFRAIVIVGNERGQVGVGVGKASDVIGAVRKGVADGKKHLVDVPLSKSNSIPHPVKGAAGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNILAKQLGSSNPLNNARAAVDALATLRTFSEVAEERG--VSIEKLYIMSRRKRSVPDYNSRLVSMMIRRIMHGKKSTARIVYDAFIDRVGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRDRGIALALRWLIQYARQRGRAMSRLAELMDAANETGSAIRKREETHKMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKLTRSIAQVLKDEGFIDEEGDEKRLVISLKYRGRNRPIIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDRDARRQGLGGEVLCYVWMARAMYWGTGRRKSSVARVRLVPGSGQIINGRGTYLQFNPYLAKAPLETLGLENEYDILVNVGGGLTGQADSIRLGVARALCELDPENRPLKEGYLTRDPRAKERKKYGLKARKAPQFSKRMARKKTGKKQKRNVPSGVAIQSTFNNTIVTITAGEVISWASAGSSGFKGAKKGTPFAAQTASESAARRATDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQGRSKYGAKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVSPDTRVKDLSDEVLRIEGYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKATMALLQRKQEIINYQTHETDTGSADVQVAMLTERINRLSLHLRNKDASRQGLLKMIGRRKRLLAYILDTRYRLIRLGIRGMATKERVGLVISDKMDKTVVVAVENRAPHPKYGKIVVKTKHYKAHDEENRCRTGDRVRISETRPLSRTKRWIVTDIITTATNV-----------',\n",
       " 'Synechococcus_sp._UTEX_2973': 'RKVSRRLQLKVEDRYPLAALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVALPKGTGQIRVAVIARGEKVEAAGADIAGSEELIEEISKGLDFDLIATPDVMPQVAKLGRQLGPRGLMPSPKGGTVTDLEAIEFKAGKLEFRADRTGIVHVFGKASFSDDLLNLKALQETIDRNRPSGAKGRYWRSYISATMGPAIEVDINALRDLKVSIGILGTKLGMTQIFDESGKAVPVTVIQAGPCPITQIKTVATDGYNAIQIGFLEVREKQLSKPELGHLSKAGAPPLRHLLEYRVPSTDGLELGQALTADRFEAGQKVDVQGHTIGRGFTGYQKRHGFARGPMSHGSKNHRLPGSTGAGTTPGRVYPGKRMAGRSGNDKTTIRGLTVVRVDADRNLLLVKGSVPGKPGALLNITPATVVG-QQA-SLKYDIIPKLKEFYTNIHQVPKVVKVTINRGLGEGAQNAKALESSLAEIAIAGQKPVVTRAKKAIAGFKIRAGMPVGLMVTLRGDRRDAFLERLINLSLPRIRDFRGISPRSFDGRGNYTLGLREQLIFPEITYDSIDQIRGMDITIVTSANSDEEGRALLRELGMPFRNVSRIGKRPIPIPAKVSVAIDGRLVSVKGPKGELSRELPSGVVVTQEDGNIIVSRADESRLAR--QRHGLSRTLVANLVEGVDSGFQKRLEIIGVGYRAQVQGTTLILNVGYSNPVQIEPPEGIQFVVENNTNVVVSGISKEVVGNTAARIRAVRPPEPYKGKGIRYAGEVVLRKAGKTGKK---MAKKVVALIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKVGLVIPVEISVFEDRSFTFILKTPPASVLIAKAAGIERGSGNPNKTKVGKITTAQLREIAETKLPDLNTKNVEAAMRIVEGTARNMGVTIADMKTLPAIERWYVVDADQRLGRLATEIAVLRGKHKPYTPHMDTGDFVIVINADKVVTGRKSQKLYRRHSGRPGGMKVETFHLQRLPERIIEAVKGMLPKNALGRQLFTKLKVYRGAHPHAQQPELIQTFAGMIQETYLNVADNSGARKLMCIRVL-GSNRYAVGDIIAVVKDAPNMVKKSDVVRAVVVRTKLRRDSGVIRFDDNAAVIINDGNPRGTRVFGPVARELRDRNFTKIVSLAPEVIMKVSRRESRRRHRRVRTIGTARPRLSVFRSNHIYAQVIDDTLAASSLDDLRQLGNQASVGKLIARAQAKGVTVVFDRGGNLYHGRVKALAEAAREAGLEFMAVDTAVKAARYIRSPKVRRVLDQLRGRSYREALILLEFMPYKSCEPILKVLRSAVANAEHNGLDPTLVISQAYADGPLKRFRPRAQGRAYQIRKQTCHITIAVAP---------------------------------------MAIISLAE-----MMESGVHFGHQTRRWNPRMAPYIY-TARNGVHIIDLVKTAQCVETAYRWVRTNAEQGKRFLFVGTKRQAAGIIAEEATRC-GSYYINQRWLGGMLTNWATIKGRVDRLKELERMEETGALALRPKKEAAVLRRELERLQKYLGGIKNMRRLPDAVVIIDQKREYNAVQECQKL--GIPIVAMLDTNC--DPDVVDVPIPGNDDAIRSVKLIVSKLADAIYEARH--GAAPVAEEY-DYDGAE---EYEDD-------ADEA------VGQKINPVGFRLGVTQE----------HRSRWFADPNRYPQLLQEDK--KIRDYVRKN-L-----SNAGIADIRVERKADQVELEIR-------TARPGVVVGRGG-----------------------------------------------------------------------AGIDTLR----------------------EGLQALLK-------DSSRQIRINVIE-------------------------------------VERVDADAALLGEYIAQQLERRVAFRRCVRQAIQRAQRA-GVQGIKIQVAG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYTTATTT-------------------------------------------YGILGIKVWVF-----------------RGEIIPGQED---AAPSNV-----------------GQPRRRNQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAARRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGLSERQLVRYVKKARRGSTGLLQLLERLDNLVFRLGFAPTIPGARQLVNHGHVTVNGRVVDIASYNCRPGEVIGVRQRSRKLVTNLEYPGLANVPHLDFDKKLAKVTGEREWVALQINELLVVEYYSRKVMAEEKDKKQSSRRRN-NRRT---EKESEWQERVVQIRRVTKVVKGGKKLSFRAIVIVGNERGQVGVGVGKASDVIGAVRKGVADGKKHLVNVPLTRDQSIPHPSTGEGGAAQVLIRPAAPGTGVIAGGAVRTVLELAGVKNVLAKRLGSKSPLNNARAALEALSSLRTFQEVAQSRD--IPVEQLY-MSRRKRSVPDFNSRLASMMVARLMDGKKSLARILYSAFIERTGDPLEFEQAVRNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVQYSRQRGKSMAKLAELMDAANETGSSVRKREETHKMAEANKAFAHYRYMANDPIADMLTRIRNARHATTVPASRLARSIAEVLKREGFIDEEGEGQRLVLSLKYKGKNQPIIALKRVSKPGLRVYSNRLPRVLGGIGIAIISTSQGIMTDRDARRQGVGGEVLCYVWM-RVVYWGTGRRKTAVARVRLVPGSGKLINGRGDYLNYNPYLTRLPLETLGLEGDYDILANVGGGLTGQADAIRLGVARALCELAPENRPLKEGFLSRDPRRKERKKYGLKARKAPQFSKRMARKKSGRKQKRNVPSGVAIQSTFNNTIVSIADGEVISWASAGSSGFKGAKKGTPFAAQTAAEAAARRAIDQGMRQLEVMVSGPGSGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERTKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKRPKVARIAGVDLPRDKRAEIGLTYIYGIGPTRAKILTGVSPDTRIKDLNDEVLRVEDYQVEGDLRREMSIKRLDIGVRGRRHRAGLPVRGQRTRTNARTRRGRTVAGKKKATMSLTQRKQEIFEYQIHPTDTGSADVQVAVLSERISRLSQHLQNKDASRTGLLRLIGQRKRLLAYILDDRYKLIRLGIRGMAVKERVGVVVSDKMDKTVVVAIEDRTAHPKYGKIVVRTKRYKAHDEDNRAKTGDRVRIQETRPLSRTKRWTVAEILESVGA------------',\n",
       " 'Synechococcus_elongatus_PCC_7942': 'RKVSRRLQLKVEDRYPLAALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVALPKGTGQIRVAVIARGEKVEAAGADIAGSEELIEEISKGLDFDLIATPDVMPQVAKLGRQLGPRGLMPSPKGGTVTDLEAIEFKAGKLEFRADRTGIVHVFGKASFSDDLLNLKALQETIDRNRPSGAKGRYWRSYISATMGPAIEVDINALRDLKMSIGILGTKLGMTQIFDESGKAVPVTVIQAGPCPITQIKTVATDGYNAIQIGFLEVREKQLSKPELGHLSKAGAPPLRHLLEYRVPSTDGLELGQALTADRFEAGQKVDVQGHTIGRGFTGYQKRHGFARGPMSHGSKNHRLPGSTGAGTTPGRVYPGKRMAGRSGNDKTTIRGLTVVRVDADRNLLLVKGSVPGKPGALLNITPATVVG-QQA-SLKYDIIPKLKEFYTNIHQVPKVVKVTINRGLGEGAQNAKALESSLAEIAIAGQKPVVTRAKKAIAGFKIRAGMPVGLMVTLRGDRRDAFLERLINLSLPRIRDFRGISPRSFDGRGNYTLGLREQLIFPEITYDSIDQIRGMDITIVTSANSDEEGRALLRELGMPFRNMSRIGKRPIPIPAKVSVAIDGRLVSVKGPKGELSRELPSGVVVTQEDGNIIVSRADESRLAR--QRHGLSRTLVANLVEGVDSGFQKRLEIIGVGYRAQVQGTTLILNVGYSNPVQIEPPEGIQFVVENNTNVVVSGISKEVVGNTAARIRAVRPPEPYKGKGIRYAGEVVLRKAGKTGKK---MAKKVVALIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKVGLVIPVEISVFEDRSFTFILKTPPASVLIAKAAGIERGSGNPNKTKVGKITTAQLREIAETKLPDLNTKNVEAAMRIVEGTARNMGVTIADMKTLPAIERWYVVDADQRLGRLATEIAVLRGKHKPYTPHMDTGDFVIVINADKVVTGRKSQKLYRRHSGRPGGMKVETFHLQRLPERIIEAVKGMLPKNALGRQLFTKLKVYRGAHPHAQQPELIQTFAGMIQETYLNVADNSGARKLMCIRVL-GSNRYAVGDIIAVVKDAPNMVKKSDVVRAVVVRTKLRRDSGVIRFDDNAAVIINDGNPRGTRVFGPVARELRDRNFTKIVSLAPEVIMKVSRRESRRRHRRVRTIGTARPRLSVFRSNHIYAQVIDDTLAASSLDDLRQLGNQASVGKLIARAQAKGVTVVFDRGGNLYHGRVKALAEAAREAGLEFMAVDTAVKAARYIRSPKVRRVLDQLRGRSYREALILLEFMPYKSCEPILKVLRSAVANAEHNGLDPTLVISQAYADGPLKRFRPRAQGRAYQIRKQTCHITIAVAP---------------------------------------MAIISLAE-----MMESGVHFGHQTRRWNPRMAPYIY-TARNGVHIIDLVKTAQCVETAYRWVRTNAEQGKRFLFVGTKRQAAGIIAEEATRC-GSYYINQRWLGGMLTNWATIKGRVDRLKELERMEETGALALRPKKEAAVLRRELERLQKYLGGIKNMRRLPDAVVIIDQKREYNAVQECQKL--GIPIVAMLDTNC--DPDVVDVPIPGNDDAIRSVKLIVSKLADAIYEARH--GAAPVAEEY-DYDGAE--DEYEDD-ADEA------------MGQKINPVGFRLGVTQE----------HRSRWFADPNRYPQLLQEDK--KIRDYVRKN-L-----SNAGIADIRVERKADQVELEIR-------TARPGVVVGRGG-----------------------------------------------------------------------AGIDTLR----------------------EGLQALLK-------DSSRQIRINVIE-------------------------------------VERVDADAALLGEYIAQQLERRVAFRRCVRQAIQRAQRA-GVQGIKIQVAG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYTTATTT-------------------------------------------YGILGIKVWVF-----------------RGEIIPGQED---AAPSNV-----------------GQPRRRNQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAARRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGLSERQLVRYVKKARRGSTGLLQLLERLDNLVFRLGFAPTIPGARQLVNHGHVTVNGRVVDIASYNCRPGEVIGVRQRSRKLVTNLEYPGLANVPHLDFDKKLAKVTGEREWVALQINELLVVEYYSRKVMAEEKDKKQSSRRRN-NRRT---EKESEWQERVVQIRRVTKVVKGGKKLSFRAIVIVGNERGQVGVGVGKASDVIGAVRKGVADGKKHLVNVPLTRDQSIPHPSTGEGGAAQVLIRPAAPGTGVIAGGAVRTVLELAGVKNVLAKRLGSKSPLNNARAALEALSSLRTFQEVAQSRD--IPVEQLY-MSRRKRSVPDFNSRLASMMVARLMDGKKSLARILYSAFIERTGDPLEFEQAVRNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVQYSRQRGKSMAKLAELMDAANETGSSVRKREETHKMAEANKAFAHYRYMANDPIADMLTRIRNARHATTVPASRLARSIAEVLKREGFIDEEGEGQRLVLSLKYKGKNQPIIALKRVSKPGLRVYSNRLPRVLGGIGIAIISTSQGIMTDRDARRQGVGGEVLCYVWM-RVVYWGTGRRKTAVARVRLVPGSGKLINGRGDYLNYNPYLTRLPLETLGLEGDYDILANVGGGLTGQADAIRLGVARALCELAPENRPLKEGFLSRDPRRKERKKYGLKARKAPQFSKRMARKKSGRKQKRNVPSGVAIQSTFNNTIVSIADGEVISWASAGSSGFKGAKKGTPFAAQTAAEAAARRAIDQGMRQLEVMVSGPGSGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERTKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKRPKMARIAGVDLPRDKRAEIGLTYIYGIGPTRAKILTGVSPDTRIKDLNDEVLRVEDYQVEGDLRREMSIKRLDIGVRGRRHRAGLPVRGQRTRTNARTRRGRTVAGKKKATMSLTQRKQEIFEYQIHPTDTGSADVQVAVLSERISRLSQHLQNKDASRTGLLRLIGQRKRLLAYILDDRYKLIRLGIRGMAVKERVGVVVSDKMDKTVVVAIEDRTAHPKYGKIVVRTKRYKAHDEDNRAKTGDRVRIQETRPLSRTKRWTVAEILESVGA------------',\n",
       " 'Synechococcus_leopoliensis_UTEX_625a': 'RKVSRRLQLKVEDRYPLAALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVALPKGTGQIRVAVIARGEKVEAAGADIAGSEELIEEISKGLDFDLIATPDVMPQVAKLGRQLGPRGLMPSPKGGTVTDLEAIEFKAGKLEFRADRTGIVHVFGKASFSDDLLNLKALQETIDRNRPSGAKGRYWRSYISATMGPAIEVDINALRDLKVSIGILGTKLGMTQIFDESGKAVPVTVIQAGPCPITQIKTVATDGYNAIQIGFLEVREKQLSKPELGHLSKAGAPPLRHLLEYRVPSTDGLELGQALTADRFEAGQKVDVQGHTIGRGFTGYQKRHGFARGPMSHGSKNHRLPGSTGAGTTPGRVYPGKRMAGRSGNDKTTIRGLTVVRVDADRNLLLVKGSVPGKPGALLNITPATVVG-QQA-SLKYDIIPKLKEFYTNIHQVPKVVKVTINRGLGEGAQNAKALESSLAEIAIAGQKPVVTRAKKAIAGFKIRAGMPVGLMVTLRGDRRDAFLERLINLSLPRIRDFRGISPRSFDGRGNYTLGLREQLIFPEITYDSIDQIRGMDITIVTSANSDEEGRALLRELGMPFRNVSRIGKRPIPIPAKVSVAIDGRLVSVKGPKGELSRELPSGVVVTQEDGNIIVSRADESRLAR--QRHGLSRTLVANLVEGVDSGFQKRLEIIGVGYRAQVQGTTLILNVGYSNPVQIEPPEGIQFVVENNTNVVVSGISKEVVGNTAARIRAVRPPEPYKGKGIRYAGEVVLRKAGKTGKK---MAKKVVALIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKVGLVIPVEISVFEDRSFTFILKTPPASVLIAKAAGIERGSGNPNKTKVGKITTAQLREIAETKLPDLNTKNVEAAMRIVEGTARNMGVTIADMKTLPAIERWYVVDADQRLGRLATEIAVLRGKHKPYTPHMDTGDFVIVINADKVVTGRKSQKLYRRHSGRPGGMKVETFHLQRLPERIIEAVKGMLPKNALGRQLFTKLKVYRGAHPHAQQPELIQTFAGMIQETYLNVADNSGARKLMCIRVL-GSNRYAVGDIIAVVKDAPNMVKKSDVVRAVVVRTKLRRDSGVIRFDDNAAVIINDGNPRGTRVFGPVARELRDRNFTKIVSLAPEVIMKVSRRESRRRHRRVRTIGTARPRLSVFRSNHIYAQVIDDTLAASSLDDLRQLGNQASVGKLIARAQAKGVTVVFDRGGNLYHGRVKALAEAAREAGLEFMAVDTAVKAARYIRSPKVRRVLDQLRGRSYREALILLEFMPYKSCEPILKVLRSAVANAEHNGLDPTLVISQAYADGPLKRFRPRAQGRAYQIRKQTCHITIAVAP---------------------------------------MAIISLAE-----MMESGVHFGHQTRRWNPRMAPYIY-TARNGVHIIDLVKTAQCVETAYRWVRTNAEQGKRFLFVGTKRQAAGIIAEEATRC-GSYYINQRWLGGMLTNWATIKGRVDRLKELERMEETGALALRPKKEAAVLRRELERLQKYLGGIKNMRRLPDAVVIIDQKREYNAVQECQKL--GIPIVAMLDTNC--DPDVVDVPIPGNDDAIRSVKLIVSKLADAIYEARH--GAAPVAEEY-DYDGAE--DEYEDD-ADEA------------VGQKINPVGFRLGVTQE----------HRSRWFADPNRYPQLLQEDK--KIRDYVRKN-L-----SNAGIADIRVERKADQVELEIR-------TARPGVVVGRGG-----------------------------------------------------------------------AGIDTLR----------------------EGLQALLK-------DSSRQIRINVIE-------------------------------------VERVDADAALLGEYIAQQLERRVAFRRCVRQAIQRAQRA-GVQGIKIQVAG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYTTATTT-------------------------------------------YGILGIKVWVF-----------------RGEIIPGQED---AAPSNV-----------------GQPRRRNQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAARRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGLSERQLVRYVKKARRGSTGLLQLLERLDNLVFRLGFAPTIPGARQLVNHGHVTVNGRVVDIASYNCRPGEVIGVRQRSRKLVTNLEYPGLANVPHLDFDKKLAKVTGEREWVALQINELLVVEYYSRKVMAEEKDKKQSSRRRN-NRRT---EKESEWQERVVQIRRVTKVVKGGKKLSFRAIVIVGNERGQVGVGVGKASDVIGAVRKGVADGKKHLVNVPLTRDQSIPHPSTGEGGAAQVLIRPAAPGTGVIAGGAVRTVLELAGVKNVLAKRLGSKSPLNNARAALEALSSLRTFQEVAQSRD--IPVEQLY-MSRRKRSVPDFNSRLASMMVARLMDGKKSLARILYSAFIERTGDPLEFEQAVRNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVQYSRQRGKSMAKLAELMDAANETGSSVRKREETHKMAEANKAFAHYRYMANDPIADMLTRIRNARHATTVPASRLARSIAEVLKREGFIDEEGEGQRLVLSLKYKGKNQPIIALKRVSKPGLRVYSNRLPRVLGGIGIAIISTSQGIMTDRDARRQGVGGEVLCYVWM-RVVYWGTGRRKTAVARVRLVPGSGKLINGRGDYLNYNPYLTRLPLETLGLEGDYDILANVGGGLTGQADAIRLGVARALCELAPENRPLKEGFLSRDPRRKERKKYGLKARKAPQFSKRMARKKSGRKQKRNVPSGVAIQSTFNNTIVSIADGEVISWASAGSSGFKGAKKGTPFAAQTAAEAAARRAIDQGMRQLEVMVSGPGSGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERTKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKRPKVARIAGVDLPRDKRAEIGLTYIYGIGPTRAKILTGVSPDTRIKDLNDEVLRVEDYQVEGDLRREMSIKRLDIGVRGRRHRAGLPVRGQRTRTNARTRRGRTVAGKKKATMSLTQRKQEIFEYQIHPTDTGSADVQVAVLSERISRLSQHLQNKDASRTGLLRLIGQRKRLLAYILDDRYKLIRLGIRGMAVKERVGVVVSDKMDKTVVVAIEDRTAHPKYGKIVVRTKRYKAHDEDNRAKTGDRVRIQETRPLSRTKRWTVAEILESVGA------------',\n",
       " 'Synechococcus_elongatus_PCC_6301_(re-annotation)': 'RKVSRRLQLKVEDRYPLAALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVALPKGTGQIRVAVIARGEKVEAAGADIAGSEELIEEISKGLDFDLIATPDVMPQVAKLGRQLGPRGLMPSPKGGTVTDLEAVEFKAGKLEFRADRTGIVHVFGKASFSDDLLNLKALQETIDRNRPSGAKGRYWRSYISATMGPAIEVDINALRDPKVSIGILGTKLGMTQIFDESGKAVPVTVIQAGPCPITQIKTVATDGYNAIQIGFLEVREKQLSKPELGHLSKAGAPPLRHLLEYRVPSTDGLELGQALTADRFEAGQKVDVQGHTIGRGFTGYQKRHGFARGPMSHGSKNHRLPGSTGAGTTPGRVYPGKRMAGRSGNDKTTIRGLTVVRVDADRNLLLVKGSVPGKPGALLNITPATVVG-QQA-SLKYDIIPKLKEFYTNIHQVPKVVKVTINRGLGEGAQNAKALESSLAEIAIAGQKPVVTRAKKAIAGFKIRAGMPVGLMVTLRGDRRDAFLERLINLSLPRIRDFRGISPRSFDGRGNYTLGLREQLIFPEITYDSIDQIRGMDITIVTSANSDEEGRALLRELGMPFRNVSRIGKRPIPIPAKVSVAIDGRLVSVKGPKGELSRELPSGVVVTQEDGNIIVSRADESRLAR--QRHGLSRTLVANLVEGVDSGFQKRLEIIGVGYRAQVQGTTLILNVGYSNPVQIEPPEGIQFVVENNTNVVVSGISKEVVGNTAARIRAVRPPEPYKGKGIRYAGEVVLRKAGKTGKK---MAKKVVALIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNARTQDKVGLVIPVEISVFEDRSFTFILKTPPASVLIAKAAGIERGSGNPNKTKVGKITTAQLREIAETKLPDFNTKNVEAAMRIVEGTARNMGVTIADMKTLPAIERWYVVDADQRLGRLATEIAVLRGKHKPYTPHMDTGDFVIVINADKVVTGRKSQKLYRRHSGRPGGMKVETFHLQRLPERIIEAVKGMLPKNALGRQLFTKLKVYRGAHPHAQQPELIQTFAGMIQETYLNVADNSGARKLMCIRVL-GSNRYAVGDIIAVVKDAPNMVKKSDVVRAVVVRTKLRRDSGVIRFDDNAAVIINDGNPRGTRVFGPVARELRDRNFTKIVSLAPEVIMKVSRRESRRRHRRVRTIGTARPRLSVFRSNHIYAQVIDDTLAASSLDDLRQLGNQASVGKLIARAQAKGVTVVFDRGGNLYHGRVKALAEAAREAGLEFMAVDTAVKAARYIRSPKVRRVLDQLRGRSYREALILLEFMPYKSCEPILKVLRSAVANAEHNGLDPTLVISQAYADGPLKRFRPRAQGRAYQIRKQTCHITIAVAP---------------------------------------MAIISLAE-----MMESGVHFGHQTRRWNPRMAPYIY-TARNGVHIIDLVKTAQCVETAYRWVRTNAEQGKRFLFVGTKRQAAGIIAEEATRC-GSYYINQRWLGGMLTNWATIKGRVDRLKELERMGETGALALRPKKEAAVLRRELERLQKYLGGIKNMRRLPDAVVIIDQKREYNAVQECQKL--GIPIVAMLDTNC--DPDVVDVPIPGNDDAIRSVKLIVSKLADAIYEARH--GAAPVAEEY-DYDGAE---EYEDD-------ADEA------VGQKINPVGFRLGVTQE----------HRSRWFADPNRYPQLLQEDK--KIRDYVRKN-L-----SNAGIADIRVERKADQVELEIR-------TARPGVVVGRGG-----------------------------------------------------------------------AGIDTLR----------------------EGLQALLK-------DSSRQIRINVIE-------------------------------------VERVDADAALLGEYIAQQLERRVAFRRCVRQAIQRAQRA-GVQGIKIQVAG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYTTATTT-------------------------------------------YGILGIKVWVF-----------------RGEIIPGQED---AAPSNV-----------------GQPRRRNQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKAARRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGLSERQLVRYVKKARRGSTGLLQLLERLDNLVFRLGFAPTIPGARQLVNHGHVTVNGRVVDIASYNCRPGEVIGVRQRSRKLVTNLEYPGLANVPHLDFDKKLAKVTGEREWVALQINELLVVEYYSRKVMAEEKDKKQSSRRRN-NRRT---EKESEWQERVVQIRRVTKVVKGGKKLSFRAIVIVGNERGQVGVGVGKASDVIGAVRKGVADGKKHLVNVPLTRDQSIPHPSTGEGGAAQVLIRPAAPGTGVIAGGAVRTVLELAGVKNVLAKRLGSKSPLNNARAALEALSSLRTFQEVAQSRD--IPVEQLY-MSRRKRSVPDFNSRLASMMVARLMDGKKSLARILYSAFIERTGDPLEFEQAVRNATPLVEVRARRVGGATYQVPMEVRERGTAMALRWLVQYSRQRGKSMAKLAELMDAANETGSSVRKREETHKMAEANKAFAHYRYMANDPIADMLTRIRNARHATTVPASRLARSIAEVLKREGFIDEEGEGQRLVLSLKYKGKNQPIIALKRVSKPGLRVYSNRLPRVLGGIGIAIISTSQGIMTDRDARRQGVGGEVLCYVWM-RVVYWGTGRRKTAVARVRLVPGSGKLINGRGDYLNYNPYLTRLPLETLGLEGDYDILANVGGGLTGQADAIRLGVARALCELAPENRPLKEGFLSRDPRRKERKKYGLKARKAPQFSKRMARKKSGRKQKRNVPSGVAIQSTFNNTIVSIADGEVISWASAGSSGFKGAKKGTPFAAQTAAEAAARRAIDQGMRQLEVMVSGPGSGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERTKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQSRSKYGAKRPKVARIAGVDLPRDKRAEIGLTYIYGIGPTRAKILTGVSPDTRIKDLNDEVLRVEDYQVEGDLRREMSIKRLDIGVRGRRHRAGLPVRGQRTRTNARTRRGRTVAGKKKATMSLTQRKQEIFEYQIHPTDTGSADVQVAVLSERISRLSQHLQNKDASRTGLLRLIGQRKRLLAYILDDRYKLIRLGIRGMAVKERVGVVVSDKMDKTVVVAIEDRTAHPKYGKIVVRTKRYKAHDEDNRAKTGDRVRIQETRPLSRTKRWTVAEILESVGA------------',\n",
       " 'Crinalium_epipsammum_PCC_9333': 'KKVSRRFQLRVEDRYPTEALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEANGADIAGSEELIEDIQKGMDFDLIATPDMMPQVAKLGRVLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSQDLLNLKALQETIDRNRPSGAKGRYWRSYVAATMGPSIEVDVNALRDLKMSVGILGTKLGMTQVFDEAGKAIPVTVVQAGPCTVTQVKTKETDGYTAIQLGYKQVREKTLSKPELGHLAKSGAAPLRHLQEYRLDDASNFQLGQELKADLFSNGEVVDVIGTSIGRGFAGYQKRHNFKRGPMSHGSKNHRQPGSIGPGTTPGRVYPGKKMAGRLGGVQVTIRKLTVVRVDAERNLLLIKGAVPGKPGALLNIVPENKVG-R--MRLKYETIPKLEQFYANIHQVPKLIKVTVNRGLGEASSNAKALESSLSEIAITGQKPVVTRAKKAIAAFKLRQGMPVGVMVTLRSDRMYAFVDRLINLALPRIRDFRGISPKSFDGRGNYSLGIREQLIFPEVEYDRIDQIRGMDISIITTANTDEEGRALLKEMGMPFRSMSRIGKRPINIPGKVTVAIEGQHVGVKGPKGELSRVLPAEVTVEQEGDTLVVKRRNESRPAR--QLHGLSRTLVANMVDGVSEGFQKRLEIQGVGYRAQVQGRNLILNVGYSKPVEITPPDGIQVAVENNTNVVVSGIDKEVVGNTAAKIRDIRPPEPYKGKGIRYAGEVVRRKAGKAGKK---MAKKVVAVVKLALNAGKANPAPPVGPALGQHGVNIMAFCKEYNARTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLIRKAAGIERGSNEPNKKKVGSLSRAQLQEIAQTKMPDLNANDIDAAVKIIEGTARNMGVTINDMKTLPTLEQWYVVDADQRLGRLASEIAILRGKNKPYTPHMDTGDFVIVLNAEKVVTGKKTQKVYRRHSGRPGGMKTETFKLQRIPERIIEAVRGMLPKNSLGRQLFTKLKVYEGSHPHAQQPQLINTIPGMIQESYLNVADNSGARKLMCIRVLGSGNRYGVGDIIAVVKDAPNMVKKSDVVRAVIVRTKMRRESGSIRFDDNAAVLINEGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRKESRRRHKRVRQIGTERPRLAVFRSNHIYVQVIDDTLAASTVEDLKSLSNQAAVGKLIARALAKGIQVVFDRGGNLYHGRVKALADAAREGGLEFMAVKNTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLNVLRSAVANAEHNNLDPALVVSQAFADGPLKRFRPRAQGRAYKIRKPTCHITVAVAED--------------------------------------MAVVSLAQ-----MMESGVHFGHQTRRWNPKMAPYIY-TSRNGVHIIDLVQTAQLMEEAYSYMRTAAEQGKKFLFVGTKRQAAGIIAQEASRC-GAAYINQRWLGGMLTNWTTIKTRVERLKDLERREETGALDLLPKKEASVLRREMAKLQKYLGGIKNMRKVPDIVLIVDQKREYNAVLECQKL--SLPIVSLLDTNC--DPDVVDIPVPANDDAIRSIKLIIGRLADAIYEGRH--GELEAEYEDEDYEGSDEDAEYEEL-TGDYPVDDEE---TEEMGQKIHPVGFRLGITQE----------HRSRWFADAKHYPELLQEDH--KIRQYVEKT-L-----SNAGISQVRIERKADQIELEVH-------TARPGVVVGRGG-----------------------------------------------------------------------TGIESLR----------------------TGLQTALG--------GNRSIRINVVE-------------------------------------VARVDAEAGLIGEYIAQQLERRVSFRRVVRQAIQRAQRA-EVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYRTAKTI-------------------------------------------YGILGVKVWIF-----------------KGEIIPGQED----QAAAN----------------TAQPKRR-QQRRRQQFEDRSSEGMSRYRGPRLRIRRLGDLPGLTRKTARRAYAPGQHGQRKKRSEYAIRLEEKQKLRFNYGVTERQLLRYVRRARRGSTGLLQLLERLDNTVFRMGMAPTIPAARQLVNHGHVTVNGRVVDIASYQCRAGEVISVRNRSRQRVQNLQNPGLANVPHLEFDKNLGKVNSEREWVALTINELLVVEYYSRQA------MAK-ERRKS-SRAK---EKEITFQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVRKGVADGKKHLIDVPLTKANSIPHPINGTGGGAKVMMRPAAPGTGVIAGGAVRTVLELSGVRNILAKQLGSNNPLNNARAAVNALSTLRTLSEVAEERG--IPLENLYAMSRRKRPVPDYNSRLVSMIIRRIMRGKKSIAGIIYDALIERTGDPLEFEKAVRNATPLVEVKGRRVGGATYQVPMEVSNRGTTLAMRWLINFARQRGRSMARLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTIPATKMTRSIAKVLKDEGFIEEEGEGKKLVVSLKYKGKNRPIVALKRISKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQGLGGEVLCYVWMSRAVYWGTGRRKSSVARVRLVPGSGQMINGRGEYFQFNSYIAKAPLETLGLENEYDILVNAGGGLTGQSDSVRLGVARALCELDPDNRPLKEGYLTRDPRATERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNTIITIADGDVISWASSGSSGFKGAKKGTPFAAQTAAEGAARRAMDQGMRQIEVMVSGPGAGRETAIRSLQSGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRARKAKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTSGVKDRRQGRSKYGAKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDEVLRIEDYQIEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMSLTQRKQEIMTYQVHETDTGSAEVQVAMMTERINRLSTHLKNKDASRRGLLTIIGQRKRLLAYIQDQNYQLIRLGIRGMAVKERVGLVVSDKMDKTVVVAIENRSSHTKYGKIVVRTKRYKAHDEENKCKEGDRVRIQETRPLSRTKRWVVAEILNSTSQS-----------',\n",
       " 'Oscillatoriales_sp._JSC-12': 'AKVSKRLRLKVEDRYPLEAVLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVALPKGTGQIRVAVIARGEKVEASGADIAGSEELIDEIQKGMDFDLIATPDMMPQVAKLGKLLGPRGLMPSPKGGTVTDLGAIEFKAGKVEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRSYVCATMGPSIQVDINGLRDLKMSVGILGTKLGMTQVFDESGKAIPVTVIQAGPCTVTQVKTKETDGYAAIQVGFREVAQKALTKPELGHLVKSNSSPVRHLREYRLDNSGEFELGQQITVERFTAGQTVDVVGTSIGRGFAGYQKRHNFKRGPMAHGSKNHRLPGSTGAGTTPGRVYPGKRMAGQLGNTRVTIRKLTIVRVDAERNLLLIKGAVPGKPGALVNVLPAKIVG-RS-MRLKYEKIPKLEQFYTNIHQVPKLVKITVNRGLGEASQNAKALEASLAEIAITGQKPVVTRAKKAIAGFKLRQGMPVGIMVTLRSDRMYAFLDRLINLALPRIRDFRGISPKSFDGRGNYTLGIREQLIFPEIEYDSIDQIRGMDVSIITTANTDEEGRALLKEFGMPFRSMSRIGKRPISIPNKVTVVIDGQSVTVKGPKGELSRVLPSEVEVVQEGETVLVNRRDDSRVAR--QRHGLCRTLVANMVEGVSQGFQRRLEITGVGYRAQVQGKNLTLNMGYSHPVQIEPPEGIQFAVENNTNVIVTGIDKEIVGNTAARIRAVRPPEPYKGKGIRYTGEVIKRKAGKAGKK---MAKKVVAVIKLALPAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTSDQVGMVVPVEISVYEDRSFTFVLKTPPASVLLQKAAGIDKGSGEPNRKKVGAITRAQLREIAQTKMPDLNANDVEAAMKIIEGTARNMGVTIKDMKTLPAVERWYIVDANQRLGRLATEVAVLRGKNKPYTPSMDTGDFVIIVNAEKIVTGKKTQKLYRRHSGRPGGMKVETFKLQRIPERIIEAVKGMLPKNALGRQLFTKLKVYAGPHPHAQQPQLIITIPGMIQQSYLNVADNSGARKLMCIRVLGGGNRYAIGDIVAVVKDAPNMVKKSDVVMAVVVRTRLRRESGSIRFDDNAAVIINDRNPRGTRVFGPVARELRDKEFTKIVSLAPEVLMKASRRESRRRHARVRKVGTERPRLAVFRSNHIYVQLIDDTISASTLDDLKSLGNEASVGKLIARSLAKGIQVVFDRGGNLYHGRVKALADAAREAGLEFMAVDTAVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPILKVLRSAAANAEHNGYDRALIVSTAYADGPLKRYRPRAQGRAYQIRKPTCHITIAVAPG--------------------------------------MPVVSLAQ-----LLESGVHFGHQTRRWNPKMDPYIY-TARNGVHIIDLVQTAKLMDDAYNYVRNASEQGKKFLFIGTKRQAAGIVAQEASRC-GAYYVNQRWLGGMLTNWATIKTRVDRLKELERREETGALDLLPKKEASVLRRELEKLQKYLGGIKAMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSLLDTNC--DPDVVDIPIPANDDAIRSIKLIVSKLADAIYEGRH--GQLEVEDDYEDYEGAEEEYDYDDS---DSFADDDD--ETEEMGQKIHPTGLRLGITQE----------HRSRWFADEHRYAELLQEDF--KIRHFFQKNPLKLKGLESPGISQIRIERKADQIDLEVH-------TARPGVVVGRGG-----------------------------------------------------------------------QGIEELR----------------------SGLQKVLG-------GGDRQIRINVVE-------------------------------------VARVDADAALIGEYIAQQLERRVSFRRVVRQAIQRAQKA-GIEGIKIQVSG-----------------------------RLNGAEIARPE-----WTREG---RVPLHTLRADIDYAYCTASTI-------------------------------------------YGTLGIKVWVF-----------------KGEIIPGQEE----RPAPT----------------NQQPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKQARRAYPPGQHGQRKKRSEYAVRLEEKQKLRFNYGLTERQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTVNGRVVDIPSYNCRPGDVIGVREHSRKLVENLQYPGLANLPHLEFDKKLGKVNGDREWVALQINELLVVEYYSRQA------MAK-GRKNA--KAR---EKETDWQERVVQIRRVTKVVKGGKKLSFRAIVVVGNERGQVGVGVGKAADVIGAVKKGVADGKKHLVEVPLTKNNSIPHPATGEGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVKNVLAKQLGSDNPLNNARAAVEALATLRTFSEVAEERG--IALEKLYAMSRRKRPVPDYNSRLVTMMARRLMQGKKSLARIIYEAFIQRTGDPIGFEQAVKNATPLVEVKARRVGGATYQVPVEVRDRGTALALRWLIQFARQRGKTMAKLAELMDASNQTGSAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRSIAEVLKDEGFIEEEGEGKKLVISLKYKGKGKPIISLKRVSKPGFRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQGLGGEVLCYVWM-RVMYFGTGRRKSAIARVRLVPGNGQLINGRGDYLQFNPYLAKAPLETLGLENEYDVLVNAGGGLTGQADAVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQFSKRPRQKRTGRKQKKNVPNGVAIQSTFNNTIVTISDGEVISWASAGSSGFKGAKKGTPFAAQTAAENAARRASDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERTKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGAKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLTRSKVLTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRLGLPVRGQRTRTNARTRRGRTVAGKKKAPMALLQRKQEIISYQIHETDTGSADVQVAMLTARINRLSEHLKNKDSSRMGLLKMIGQRKRLLAYIQDDRYKLIKLGIRGMAVKERVGLVVSDKMEKTVVVAVENRSAHPKYGKIVAKTKRYKAHDEDNKCKIGDRVRIQETRPLSRTKRWQVIDILTTATQA-----------',\n",
       " 'Synechococcus_sp._PCC_6312': 'AKVSKRLRLKVEDRYPLDALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDEIQKGMDFDLIATPDMMPQVAKVGRILGPRGLMPSPKAGTVTDLPAIEFKAGKLEFRADRTGIVHVFGKASFPEDLLNLKALQETIDRNRPSGAKGRFWRSYVAATMGPSIQVDISGLRDLKMAIGILGTKLGMTQIFDDSGKAVPITVVQAGPCPVTQIKTKATDGYTAIQLGYGETKEKALSKPEMGHLSKSESNPLRHLQEYRLEDVSSYQLGQPITVDIFEAGKLVDVVGTSIGRGFAGYQKRHNFKRGPMAHGSKNHRLPGSTGAGTTPGRVYPGKRMAGQLGNCRVTVRKLQIVRVDAERNLLLIKGALPGKPGALVNILPATIVGQK--MRMKYETAPQLQQFYSNVHQVPKVVKVTVNRGLGEAAQNAKSLEATLAEVAITGQKPVVTRAKKAIAGFKIRQGMPVGVMVTLRADRMYAFLDRLINLALPRIRDFRGINPKSFDGRGNFTLGLREQLIFPEVNYDDIDQIRGMDISIITTAKTDEEGRALLKALGMPFRNMSRIGKRPIPLPDKVSLALAGQEVTVKGPKGQLSRVLPAEVEVLQEGSTILVSRRNESRPAR--QRHGLCRTLVANMVEGVAQGFSKRLEIQGIGYRANLQGTKLVLSVGYSHPVEIDPPTGITFELEDNTLILVSGIDKEQVGNLAAQIRAVRPPEPYKGKGIRYAGEAVRRKVGKTGKK---MAKKVTAIIKLAIQAGKANPAPPIGPALGQHGVNIMMFCKEYNARTADQVGTVIPVEISVFEDRSFTFILKTPPASVLIQKAAGIERGSGEPNKKKVGQLTRSQLQGIAQQKMPDLNANDIEAAMRIIEGTARNMGVTIVDLKTLPELDPWYVIDADQRLGRLATEAAVLRGKNKPYTPHMDAGDFLVIINAEKVVTGKKSQKVYRRHSGRPGGMKTETFQLQRIPERIIEAIKGMLPKNSLGRKLFTKLKVYAGPHPHAQKPELISTIPGMIQETYLNVADNSGAKKLLCIRVLGGGNRYGVGDIIATVKDAPNMVKKSDVVRAVIVRTRIRRDSGSIRFDDNAAVLINEGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKQSRTEAQSRHRRIRKVGTARPRLAVFRSHHIYAQVIDDTLAASTVEELKEFGSQAAVGQSLARALAAGIEVVFDRGGNLYHGRVQALADAAREAGLEFMATPTISKAARYVRSPKVRRVLDQLRGRSYREALIILRFMPYRACDPITKVLRSAAANSEHNGINPALIVYEAFADGPLKRFRPRAQGRAYQIRKPTCHITISVAES--------------------------------------MSVLSLAQ-----LLEAGVHFGHQARRWNPKMSRYIF-TVRNGVHIIDLVQTAQLVDEAYNYVREASESGKKFLFVGTKRQAAGIVHQEAARC-GGYFINQRWLGGMLTNWTTIKTRVDRLKELERMEETGQIALRPKQEAAVLRRELARLQKYLGGIKNMRRLPDVAVIVDVKREYNAVQECQKL--GIPIVALLDTNC--DPDQVDIPVPANDDAIRSIKLIVGKLADAIYEGRH--GQVEEA---DLGDGAEEDYDAEDE-TLDSVEEESDADDSKDMGQKINPIGFRLGITQD----------HRSRWYADASRYPELLQEDH--RIRQYIEKT-L-----AGASIADVRIERKADQVELQVR-------TARPGVVVGKGG-----------------------------------------------------------------------AGIENLR----------------------LSLKKLLP--------PGRSVKVNVVE-------------------------------------VTRVDAEAGLLAEYITQQLERRVAFRRAVRQAIQRAQRA-GVEGIKIQVAG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYRTAKTI-------------------------------------------YGILGVKVWIF-----------------KGEVLPGQQP----DPDFG----------------REQPRQR-PPRQRRNFEDRSSRNMARYRGPRLRIRRLGELAGLTRKSPKRSYPPGQHGQRKKKSEYATRLEEKQKLRFNYGVSERQLVRYVRKARRGSTGLLQLLERLDNTVFRMGMAPTIPAARQLVNHGHITVNGRAVSIPSYQVRPGEIIGVRQRSRQLVENLQSPGLANLPHLEFDKTFGKVNGEREWVALQVNELLVVEYYSRKV------MAS-RRKG--SRKA---EKETDWQERVVQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKAADVIGAVKKGVADGKKHLVEIPITKANSIPHPTTGEGGAARVLIRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSSNPLNNARAAIEALASLRTLAEVAEERE--ISIQQLYSMSRRKRPTPDYNSRLISMLIQRIMLGKKSIARIVYDAMVERTGEALQFEKAIKNATPLVEVKARRVGGATYQVPMEVRDRGTALALRWLVQFSRKRGRSMVKLAEVMDAANETGSTIRKREETHKMAEANKAFAHYRYMANDTISDMLTRIRNARHQTTIPATRMTRNIAQVLKAEGFIDKVGEGKQLVVGLKYRGKQRPIIALKRVSKPGLRVYANQLPRVLGGIGIAIISTSSGIMTDREARKQGIGGEVLCYVWMSRAVYLGTGRRKSSIARVRLVPGTGQLINDRGEYLNFNPYIAKAPLETLGLESSYDILVNAGGGLTGQADSIRLGVARALCQLDGENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMAPKRSGRKQKRNVPSGVAIQSTFNNTIISITDGEVISWASAGSSGFKGAKKGTPFAAQTAADSAARRAIDQGMRQIEVMVSGPGSGRETAIRALQSGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGAKQPKMARIAGVDLPRDKRVEIALTYIYGIGLTRSQILTGVNPDTRVKDLVDDVLRIDEYDVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAAMVLLQRKQEIINYQVHGTDTGSADVQVAILTERIKRLSEHLKNKDASRRGLLKMIGQRKRLLSYIADGHYRLIKLGIRGMAVKERVGVVVSDKMEKTVVVAVENRSSHPKYGKIVVKTNRYKAHDEENRCKVGDRVRIQETRPLSRTKRWTVIEILTATISA-----------',\n",
       " 'Rivularia_sp._PCC_7116': 'NKLSRRMKLKVEDKYPIEALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQIRVAVIARGEKVEAAGADVVGSEELITEIQQGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKVSFTEDLLNLKALQESIDRNRPSGAKGRYWRTYVSATMGPSIRVDTNALRDIKMSVGILGTKVGMTQVFDDAGAAIPVTVVQAGPCTVTQIKTQQTDGYSAIQLAYGEVKEKALNKPKLGHLAKASAQPLRHLEEYRAQNTDDYSLGQQVSADIFEAGQLVDIAGTSIGRGFAGNQKRNNFGRGPMSHGSKNHREPGSIGAGTTPGRVYPGKRMAGRLGGKRITVRKLKVVRVDAEKNLLLIKGAIPGKPGALVNIVPAKEVK----MGLKYETIPNLKQFYTNIHQVPKVIKVTVNRGLGEASQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIREGMPVGVMVTLRSSKMYNFLDRLINLSLPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDKIDQIRGMDISIITTARDDEEGRALLKEMGMPFRQMSRIGKQPIAVPAKVQIKIEGASISVKGPKGELSRELPTQVSVSQEGDTLTVSRQDDSRVCR--QMHGLSRSLVSNMVEGVSQGFQRKLQLQGVGYRAQVKGKDLVLNVGYSHPVEITPPDGVQFAIEDKLTIVVSGYDKEIVGNTSARIRAVRPPEPYKGKGIRYENEVVRRKAGKAGKGGKKMAKKVTAIIKLALQAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADKAGMVIPVEISVFEDRSFTFILKTPPASVLITKAAKIDKGSAEPQKVKVGSITKDQLREIAETKMPDLNANDIEAAMKIIAGTARNMGVTVSDMKTLPTVERWYVVDADQRLGRLATEIAVLRGKNKPFTPHMDAGDFVVVVNAEKVVTGKKSQKLYRRHSGRPGGMKTETFHLQRLPERIIEAVKGMLPKNNLGRQLFKKLKVYAGSHPHAQLPKLIDTIPGMIQQSYLNVADNSGARKLMCIRVLGAGNSYGIGDIIAVVKDAPNMIKKSDVVQAVIVRTKVNRDSGTIRFDDNAAVIINDGNPRGTRVFGPVARELREKNFTKIVSLAPEVLMKFTRKQSQRRHRRIRKVGSERPRLAVYRSNNIYAQVIDDTLTASSIEELKSLGNQASVGKTLARALEKGIQVVFDRGGNLYHGRVKALADAAREGGLNFMATDTKVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACDPVLKVLRSAAANAEHNGLDRALIVSTAFADGPFKRYQPRAQGRAYQIRKPTCHITVAVSAA--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYAYMRTQAEQGKKFLFVGTKRQAAGIIAQESSRC-GAHYINQRWLGGMLTNWTTIKTRVDRLKDLERREESGALDLLPKKEASMLRRELSKLQKYLGGIKTMRKIPDVVIIVDQRREYNAVQECEKL--GLPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGYH--GQVSSDDDYEDYEGAEDDYDYEDSEFSESLSEGGEQEGESTMGQKINPIGFRLGITKE----------HQSRWFADPSRYPELLQEDH--KLRQYIDNK-LGRNAQNNAGISEVKIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIEALR----------------------TGLQKELG--------GNRQIRINVVE-------------------------------------VQRVDADAFLIGEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRANIDYAYVTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEVIPGEEE----APTSN-------------PRNEREPRRR-QNRRRPSFEDRSNENMSRYRGPRLRIRRLGELPGLSRKTARRAYPPGQHGQRKKRSEYAIRLEEKQKLKFNYGLTEKQMVRYIRKARRGSTGLLQFLERLDNTIFRLGMAPTIPAARQLVNHGHVQVNGREVNIASYQCRPGDLIGVKNKSRQLVENLQYPGLANLPHLEFDKNLGKVNGEREWVALSVNELLVVEYYSRKA------MAN-NRRKT-SRAK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLVDIPITKSNSIPHPTNGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNILAKQLGSNNPLNNARAAMNALGVLRNFKEVAEDRD--VSIEKLYAMSRRRRAVPDYNSRLISMTIRRIMRGKKSLARIVYDAMIERTGDPTEFERAIRNLTPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRQRGRTMAKLAELMDASNETGSAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTQSIAKVLQEEGFIDETGEGSSLVISLKYKGKNRPIITLKRVSRPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDRQARQQNVGGEVLCYVWMSRAVYWGTGRRKSSVARVRLVPGNGKMINGKGDYFQFNPYIIKAPLETLGLENEYDILVRAGGGLTGQADAVRLGVARALCQLDPDNRPLKEGYLSRDPRSKERKKYGLKARKAPQYSKRMARKKGGRKQKRNVPNGIGIQSTFNNSIVTITDGDVVSWASAGSSGFKGAKKGTPFAAQTAAESAARRAMDQGMRQIEVMVSGPGAGRETAIRAIQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIFGIGLTSAKILTGVNPDTRVKELSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAP-MLTQKKQELISYQVHETDTGSSEVQVAMLTERINRLSKHLQNKDSSRRGLLKMIGQRKRLLSYIQDERYKLISLGIRGMAIKERVGLVVSDKMEKTVVVAVENRDSHQKYGKIVVTTRRYKAHDEENNCKEGDRVRIQETRPLSKTKRWQVKEVLSSRNN------------',\n",
       " 'Cyanothece_sp._PCC_7425': 'RKLSRRLRLKVEDRYPLAALLLKETATAKFESAEAHVRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADLVGSEELIDAIQKGMDFDLIATPDMMPQVAKVGRILGPRGLMPSPKAGTVTDLTAIDFKAGKLEFRADRSGIVHVFGKATFAPDLLNLKALQETIDRNRPSGAKGRYWRSYVSATMGPSIQVDISALRDLKMAVGILGTKLGMTQIFDQERRAIPVTVVQAGPCPITQIKTKQTDGYAAIQVAFGEVRAKSLSKAELGHLAKSNTAPMRHLHEYRTGEVDQFQVGQVLKADLFSAGQVVDVTGTSIGRGFAGYQKRHNFKRGPMAHGSKNHRLPGSTGAGTTPGRVYPGKRMAGQLGSARTTIRQLTIVRVDPERNLVLIKGAVPGKPGALVSITPANSVG-RK-MRLKYENAPKLEQFYKNIHQVPKLVKITINRGLGEAAQNAKSLEASLNEIAIAGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRSDRMYAFFDRLINLALPRIRDFRGVNPKSFDGRGNYSLGLREQLIFPEISYDSIDQMRGMDVSIITTATTDEEGRALLKVMGMPFR-MSRIGKRPIPIPDKVAVTIDGSTVKVKGPKGELSQVLPPEVVVVQENSSLLVNRRDDSRPAR--QRHGLSRTLVANMVKGVTEGFRKNMQIVGVGYRLQVTGNKLAITAGYSHPVEIDLPKGISVEVDQKSNFGVIGIDKQAVGDVAALIRAVRPLEPYKGKGIRYATETLRLKAGKTGKK---MAKKVVAIIKLAITAGKANPAPPIGPALGQHGVNIMMFCKEYNARTADQAGLVIPVEISVYEDRSFTFILKTPPASVLIQKAAGIERGSGEPNKKKVGKITTAQLREIAQTKLPDLNANDVDAAMRIVAGTARNMGVTIVDMKTLPSLEPWYVVDAEQHLGRLATQIAILRGKTKPYTPSMDTGDFVIVVNAEKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNTLGRNLFTKLKVYAGPHPHAQKPELLKTIAGMIQESYLNVADNSGAKKLLCIRVLGGGNRYGVGDIIATVKDAPNMVKKSDVVRAVIVRTRIRRESGSIRFDDNAAVLINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLNRVESRSRHRRVRKVGTDRPRLAVFRSHHIYAQVIDDTLAASSLEEFKTYGTAASIGKLIARSLAKGIKVVFDRGGKLYHGRVKALAEAAREAGLEFVLLEQPAYAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACDPILKVLRSAVANAEHNGLNPTLVVSQAYADGPLKRFRPRAQGRAYPIRKPTCHITIAVSPS--------------------------------------MPVLSLAQ-----MLEAGVHFGHQARRWNPKMSPYIF-TERNGVHIIDLVQTAQLMDEAYEYVRSASEEGKRFLFIGTKRQAAGLVAHEAQRC-GSYFINQRWLGGMLTNWTTIKTRVERLKELERRQETGALDLLPKQEASMLRREMEKLQKYLGGIKQMRKLPDVVLIVDVRREYNAVQECHKL--GIPIVSLLDTNC--DPDEVDIPIPANDDAIRSIKLIVGKLADGIYEGRH--GQLDTEEDYEDYEGAE---DYDEE-----FEPEEE--ETEEMGQKIHPTGFRLGVTQE----------HRSRWFADPERYPELLQEDH--KIRKFIATK-DGNLDLTSAGIAEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------SGIELLR----------------------TRLQQHLG---------GRSIRVNVVE-------------------------------------VQRVDAEAGLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GIEGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYRTARTV-------------------------------------------YGILGIKVWVF-----------------KGEVIPGQEE-------PP-------------SNREAQPRQRHQQKKRQQFEDRSNEDMARYRGPRLRVRRLGDLPGLTRKASRRAYPPGQHGQRKKRSEYAMRLEEKQKLRFNYGLTERQLVRYVRRARRGSTGLLQLLERLDNIIFRLGMAPTIPAARQLVNHGHVTVNGRVVSIPSYQCRPGNEIGIRNKSRRLAENLKFPGLANLPHLELDKKLAKVNSEREWVALQVNELLVVEYYSRKV------MAN-RRKT--ARKE---ERDTNWQERVVQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVRKGVADGKKHLVDVPITKSNSIPHPTSGIGGAAKVLIRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSSNPLNNARAALNALDSLRTFSQVAEERG--IAVESLYVMSRRKRTIPDFNSRLVNMMIRRIMLGKKSVARIIYGAFIER-GDPLEFERAVKNSTPLVEVKARRVGGATYQVPMEVRERGVALALRWLVRFSRQRGRSMAKLAELMDAANETGNTIRKREETHKMAEANKAFAHYRYMASDTIGDMLTRIRNARHPTTVPATRMTRSIAQVLKNEGFIDEEGDGKLLVISLKYRGKQRPTIALKRISKPGLRVYSNRLPRVLGGIGIAIISTSSGIMTDRDARRSGVGGEVLCYVWMSRATYIGTGRRKSSVARVRLVPGTGQLINGKGEYLQFNPYLAKAPLESLGLENDYDILVNTGGGLTGQADSIRLGVARALCQLDPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRPQQKRGGRKQKRNVPNGVAIQSTFNNTIVTISDGEVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGAKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLTRSQILTGVNPDTRIRELTDDILRVEAYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALLQRKQEIISYQTHETDTGSADVQVAMLTERINKLSTHLQNKDSSRRGLLQMIGRRKRLLAYILNERYRLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTRRYKAHDEEEKCQVGDRVRIQETRPLSRTKRWVVTDILTSANRT-----------',\n",
       " 'Chroococcidiopsis_thermalis_PCC_7203': 'KKVSRRLRLKVEDRYPIEALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEELISEIQNGMDFDLIATPDVMPQVARLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKAAFPEDLLNLKALQETIDRNRPSGAKGRYWRTFVSATMGPAIEVEISGLRDLKMSIGILGTKLGMTQVFDETGKAIPVTVVQAGPCVVTQIKTKQTDGYTAIQVGFKEVKPKALSKPELGHLAKSSAPALRHLHEYRLDNLSEYTLGQQITADIFNSGQIVDVSGNSIGRGFAGYQKRHNFGRGPMSHGSKNHRLPGSIGAGTTPGRVYPGKRMAGRLGGSQVTIRKLTVVRVDAERNLLLIKGAVPGKPGALLSIVPTKKVG-KG-AKLKYEKIPQLEQFYKNVHQVPKLVKITINRGLGEAAQNAKAMESSLTELAIAGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGDRMYAFLDRFVNLALPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTANTDEEGRALLKAFGMPFRQMSRIGKRPITIPSKVTVTINGSHVAVKGPKGELARVLPSAVTVEQEGDTILVKRQDDSRLSR--QLHGLSRTLVANMVDGVSQGFQKRLEIQGVGYRAQVQGRNLVLSLGYSHPVQIEPPEGIQLAVENNTNVIVSGFDKELVGNVAAQIRDKRPPEPYKGKGVRYAGEAVRRKAGKAGKK---MAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNARTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAGIERGSNEPNKKRVGSISTAQLREIAQTKLPDLNANDIDAAMKIVAGTAKNMGVNVTEMKTIPTINQWYLVDADQRLGRLATEIAILRGKNKPFTPHMDTGDFVIVVNAEKIVTGKKSQKVYRRHSGRPGGMKTETFQLQRIPERIVEAVKGMLPKNSLGRQLFTKLKVYAGAHPHAQKPEIIQTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMVKKSDVVRAVIVRTRLRRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNYTKIVSLAPEVLMKLSRQESEKRHRRIRVVGSERPRLAVFRSHHIYAQVIDDTLAASTLDDLKSLKTEASVGKLIARAKAKGIQVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTIKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAVANAEHNGLEPSLKITQAYADGPLKRFRPRAQGRAYQIRKPTCHITIAVAA---------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSQYIY-TARNGVHIIDLVQTAQLMDEAYNYVRQSAEQGKKFLFVGTKRQAAGIVAQEALRC-GASYVNQRWLGGMLTNWTTIKTRAERLKELERREESGALDLLPKKEASVLRRELAKLQKYLGGIKNMRKVPDVVIIVDQRREYNAVLECQKL--GIEIVSMLDTNC--DPDLVDVPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEEDYEDYEGAEEDFDYDES-YTDAVIPNEE---EEEMGQKIHPVGFRLGVTKE----------HHSRWFAEPQRYPEILQEDY--KLRQYIEQK-LGRYAANNAGIAEVRIERKADQIELELR-------TARPGVVVGRGG-----------------------------------------------------------------------AGIEALR----------------------NGLQELLG--------SQRQIRINVVE-------------------------------------VAQVDADAFLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GIQGIRVQVGG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWAF-----------------KGEIIPGQEP----QPTPG----------------TNQPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRVRRLGDLPGLTRKNARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRRARRGSTGLLQLLERLDNTVFRMGMAPTIPAARQLVNHGHVMVNGREVNIASYQCRPGEVIGIKNKSRQLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-GRRKA-NRTK---EKETTFQERVIQIRRVSKVVKGGKKLSFRAIVAVGNERGQVGIGVGKAGDVIGAVKKGVADGKKHLIDVPLTKSNSIPHPTNGTGGGAKVLMRPAAPGTGVIAGGAVRTVLELAGVRNILAKQLGSNNPLNNARAAVNALATLRTFAEVAEERG--IPVEKLYIMSRRKRPISDYNSRLISMMMRRVMRGKKSLARIIYDAMIERTGDPLEFEKAVRNATPLVEVKARRVGGATYQVPMEVRERGTALALRWLIQFSRQRGRSMSKLAELMDAANETGSSIRKREETHRMADANKAFAHYRYMANDTIADMLTRIRNARHQTTIPSTKMTRNLARVLQEEGFIQEEAEGKRLVVTLKYKGKNRPIVALKRVSRPGLRVYSNKLPRVLGGIGIAIISTSHGIMTDREARRQGLGGEVLCYIWMNRVMYWGTGRRKASVARVRLVPGNGQLVNGKGDYFQFNAYLAKAPLETLGLENEYDILVNAGGGLTGQADSIRLGVARALCQLEPENRHLKEGYLTRDPRAKERKKYGLKARKAPQFSKRMARKKSGKKQKRNVPNGMAIQSTFNNSIVTITDGDAISWASAGSSGFKGAKKGTPFAAQTAAESAARRAVDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERKKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARISGVDLPRDKRIEIGLTYIYGIGLTRSKILTGVNPDTRVKDLGDDVLRIENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQELISYQVHDTDTGSTEVQIAMLTERISRLSEHLRNKDSSRRGLLKLIGQRKRLLSFLQDQKYQLLRLGIRGMAVKERIGLVVSDKMQKTVVVAVENRSPHPKYAKIVVRTRRYKAHDEENKCKEGDRVRIRETRPLSRTKRWEVIEILDSPTTA-----------',\n",
       " 'Thermosynechococcus_sp._NK55a': 'RKLSRRLRLKVEDRYPLEALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVALPKGTGQIRVAVIARGEKVEAAGADLVGSEELIDEIQKGMDFDLIATPDMMPQVAKVGRILGPRGLMPSPKAGTVTDLPAIEFKAGKVEFRADRTGIVHVFGKASFSEDLLNLKALQESIDRNRPSGAKGRYWRSYVAATMGPSIQVDINALRELKVTVGILGTKLGMTQIFDEAGRSVPITVVQAGPCPITQIKTPQTDGYTAIQVAYGEVREKNLSRPERGHLNKSQTPPMRHLREFRLEDVSAYQLGQAITVDIFSPGQLVDVRGTSIGRGFAGYQKRHNFKRGPMAHGSKNHRLPGSTGAGTTPGRVFPGKRMAGRMGNTAVTIRKLQVVRVDPERNLILIKGALPGKPGALVSITPAKVVG-RK-MRLKYKTVPQLQQFYKNIHQVPKIVKVTVNRGLGEAAQNAKALEATLAEIAITGQKPVVTRAKKAIAGFKIRKGMPVGVAVTLRSDRMYAFLDRLINLALPRIRDFRGVNPKSFDGRGNYTLGLREQLIFPEVNYDDIDQIRGMDVSIITTANTDEEGRALLKAMGMPFRNMSRIGKRPIPLPKNVTLTLEGQQVTVKGPKGQLTRVFPPEVNVAQEGETIVVRRRDDSRPAK--ERHGLCRTLLANMVEGVSQGFTKKLEIQGVGYRAQLQGKTLVLSMGYSHPVEIVPPEGITLEIEDNTIVLVSGIDKELVGNTSARIRAVRPPEPYKGKGIRYMGEFVRRKVGKTGKK---MAKKVVAVIKLAIQAGKANPAPPIGPALGQHGVNIMMFCKEYNARTADQVGMVVPVEISVYEDRSFTFVLKTPPASVLIQKAAGIEKGSGEPNKKKVGSITRAQLREIAEKKMPDLNANDIEAAMRIIEGTARNMGVTITDTKTLPDLAPWYVIDADQRLGRLAAEIAILRGKNKAYTPHMDTGDFVIVINAEKVVTGRKSQKLYRRHSGRPGGMKIETFQLQRIPERIIEAVKGMLPKNSLGRKLFTKLKVYAGPHPHAQKPQLINTIPGMIQETYLNVADNSGAKKLLCIRVLGGGNRYGVGDIIATVKDAPNMVKKSDVVRAVIVRTRIRRESGSIRFDDNAAVLINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKQTRIAAQSRHQRIRKIGTDRPRLAVFRSHHIYAQVIDDTLAASSLEELRQLGTAASVGRLIARAKAAGIRVVFDRGGNIYHGRVKALAEAAREAGLDFVSTSSPARAAKYVRSPKVRRVLDQLRGRTYRDALIMLRFMPYRACEPITKVLRSAAANATHNGLDPALVISQAYADGPLKRFRPRAQGRAYQIRKPTCHITIAVAPD--------------------------------------MAVLSLAQ-----MLEAGVHFGHQARRWNPRMAPYIF-TVRNGVHIIDLVQTAQLVDEAYNYIRNAAEKGKRFLFIGTKRQAAGIVEQEALRC-GSYYVNQRWLGGMLTNWNTIKTRVDRLKELESMEENGLIDLRPKQEASALRRELARLQKYLGGIKQMRRLPDVAIIVDVKREYNAVAECHKL--GIPIVALLDTNC--DPTQVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLEEPANLDEDDDGM---TASDD--GDADIPDDSDA----VGQKIHPIGFRLGITQE----------HRSRWFADSDRYPELLQEDH--RIRTFINKQ-L-----ANAGIAEVRIERKADQVELQIR-------TARPGVVVGKGG-----------------------------------------------------------------------QGIEELR----------------------KQLRQMLP--------SNRTIKVNVVE-------------------------------------VSRVDAEASLLAEYITQQLERRVAFRRAVRQAIQRAQRA-GIEGIKVQVAG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYRTARTI-------------------------------------------YGILGVKVWIF-----------------KGEVLPGQTE-----AVPR-----------------EATRRS-PQRRLPQFENRSN--MARYRGPRLRIRRLGELAGLTRKVPKRSYPPGQHGQRKKRSEYALRLDEKQKLRFNYGVSERQLVRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHILVNGRTVSIPSYQCRPGDVITVRDNSRRLVENLQNPGLANLPHLELDKTLGRVTGERQWVALEVNELLVVEYYSRKV------MAN-RRKN--PRKV---EKETDWQERVVQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKAADVIGAVRKGVADGKKHLIDVPITKSNSIPHPTFGEGGAARVMIRPAAPGTGVIAGGSVRTVLELAGVRNVLAKQLGSSNPLNNARAALEALAALRTFQEVAEERE--IPIENLYSMSRRKRPTPDYNNVLVNMLIQRVMRGKKSLARIVYDAMVERTGDAVQFERAVKNATPLVEVKARRVGGATYQVPMEVRDRGVSLALRWLVQFSRKRGRSMSKLAELMDAANETGSTIRKREETHKMAEANKAFAHYRYMANDTIGDMLTRIRNARHQTTIPATRMTRSIAQVLKAEGFIDEEGDGKRLVVSLKYRGKQRPIIALKRVSKPGLRVYANRLPRVLGGIGIAIISTSSGIMTDREARKQGIGGEVLCYVWMSRVVYLGTGRRKSAVARVRLIPGSGQLINGRGAYLQNNPYLVKAPLETLGLENSYDIYVNAGGGLTGQADAIRLGIARALCQLDIENRPLKEGYLTRDPRAKERRKYGLKARKAPQYSKRMAPKRSGRKQKRNVPSGVAIQSTFNNTIVSITDGEVIAWASAGSSGFKGAKKGTPFAAQTAADNAARRAIDQGMRQVEVMVSGPGSGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRV----------------------------------------------VARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGAKRPKMARIAGVDLPRDKRIEIALTYIYGIGLTRSKILTGVNPDTRTRDLTDDILRIDEYQVEGDLRREMNIKRLDIGYRGRRHRLGLPVRGQRTRTNARTRRGRTVAGKKKPAMALQHVKQEIINYQIHGTDTGSADVQVAILTERIKQLSEHLKNKDASRRGLLKIIGRRKRLLAYIHDQRYHLIRLGIRGMAVKERVGVVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDENNEAKVGDRVRIRETRPLSRTKRWVIAEILSPRTA------------',\n",
       " 'Thermosynechococcus_elongatus_BP-1_(re-annotation)': 'RKLSRRLRLKVEDRYPLEALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVALPKGTGQIRVAVIARGEKVEAAGADVVGSEELIDEIQKGMDFDLIATPDMMPQVAKVGRILGPRGLMPSPKAGTVTDLPAIEFKAGKVEFRADRSGIVHVFGKASFSEDLLNLKALQESIDRNRPSGAKGRYWRSYVAATMGPSIQVDINALRELKVTVGILGTKLGMTQIFDEAGRSVPITVVQAGPCPITQIKTPQTDGYTAIQVAYGEVREKNLSRPERGHLNKSQTPPMRHLREFRLEDVSAYQLGQAITVDIFSPGQLVDVRGTSIGRGFAGYQKRHNFKRGPMAHGSKNHRLPGSTGAGTTPGRVFPGKRMAGRMGNTAVTIRKLQVMRVDPERNLILIKGALPGKPGALVSITPAKVVG-RK-MRLKYKTVPQLQQFYKNIHQVPKIVKVTVNRGLGEAAQNAKALEATLAEIAITGQKPVVTRAKKAIAGFKIRKGMPVGVAVTLRSERMYAFLDRLINLALPRIRDFRGVNPKSFDGRGNYTLGLREQLIFPEVNYDDIDQIRGMDVSIITTANTDEEGRALLKAMGMPFRNMSRIGKRPIPLPKNVTLTLDGQQVTVKGPKGQLSRVFPPEVNVVQEGEAIVVKRRDDSRPAK--ERHGLCRTLLANMVEGVSQGFTKKLEIQGVGYRAQLQGKTLVLSMGYSHPVEIVPPEGITLEIEDNTIVLVSGIDKELVGNTSARIRAVRPPEPYKGKGIRYMGEFVRRKVGKTGKK---MAKKVVAVIKLAIQAGKANPAPPIGPALGQHGVNIMMFCKEYNARTADQVGMVVPVEISVYEDRSFTFVLKTPPASVLIQKAAGIEKGSGEPNKKKVGSITRAQLREIAEKKMPDLNANDIEAAMRIIEGTARNMGVTITDTKTLPDLAPWYVIDADQRLGRLAAEIAILRGKNKAYTPHMDTGDFVIVINAEKVVTGKKSQKLYRRHSGRPGGMKIETFQLQRIPERIIEAVKGMLPKNSLGRKLFTKLKVYAGPHPHAQKPQLINTIPGMIQETYLNVADNSGAKKLLCIRVLGGSNRYGVGDIIATVKDAPNMVKKSDVVRAVIVRTRIRRESGSIRFDDNAAVLINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKQTRTAAQSRHQRIRKVGTDRPRLAVFRSHHIYAQVIDDTLAASSLEELRQLGTAASVGRLIARAKAAGIRVVFDRGGNIYHGRVKALADAAREGGLDFVSTSSPARAAKYVRSPKVRRVLDQLRGRTYRDALIMLRFMPYRACEPITKVLRSAAANATHNGLDPALVISQAYADGPLKRFRPRAQGRAYQIRKPTCHITIAVAPD--------------------------------------MAVLSLAQ-----MLEAGVHFGHQARRWNPRMAPYIF-TVRNGVHIIDLVQTAQLVEEAYNYIRNAAEKGKRFLFIGTKRQAAGIVEQEALRC-GSYYVNQRWLGGMLTNWTTIKTRVDRLKELESMEESGLIDLRPKQEASALRRELARLQKYLGGIKQMRRLPDIAIIVDVKREYNAVAECHKL--GIPIVALLDTNC--DPTQVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLEEPEADADEDDNG---MTTSD-DGDADIPDDSDA----VGQKIHPIGFRLGITQD----------HRSRWYADSDRYPELLQEDH--RIRTFINQQ-L-----ANAGIAEVRIERKADQVELQIR-------TARPGVVVGKGG-----------------------------------------------------------------------QGIEELR----------------------KQLRQMLP--------ANRTIKVNVVE-------------------------------------VNRVDAEASLLAEYITQQLERRVAFRRAVRQAIQRAQRA-GIEGIKVQVAG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYRTARTI-------------------------------------------YGILGVKVWIF-----------------KGEVLPGQTE-----AVPR-----------------EATRRS-PQRRLPQFENRSN--MARYRGPRLRIRRLGELAGLTRKVPKRSYPPGQHGQRKKRSEYALRLEEKQKLRFNYGVSERQLVRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHILVNGRNVSIPSYQCRPGDVITVRDNSRRLVENLQNPGLANLPHLELDKTLGRVTGERQWVALEVNELLVVEYYSRKV------MAN-RRKN--PRKV---EKETDWQERVVQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKAADVIGAVRKGVADGKKHLIDVPITKSNSIPHPTFGEGGAARVMIRPAAPGTGVIAGGSVRTVLELAGVRNVLAKQLGSSNPLNNARAALEALAALRTFQEVAEERE--IPIENLYSMSRRKRPTPDYNNVLVNMLIQRVMRGKKSLARIVYEAMVERTGDALQFERAVKNATPLVEVKARRVGGATYQVPMEVRDRGISLALRWLVQFSRKRGRSMSKLAELMDAANETGSTIRKREETHKMAEANKAFAHYRYMANDTIGDMLTRIRNARHQTTIPATRMTRSIAQVLKAEGFIDEEGDGKRLVVSLKYRGKQRPIIALKRVSKPGLRVYANRLPRVLGGIGIAIISTSNGIMTDREARKQGIGGEVLCYVWMSRVVYLGTGRRKSAVARVRLIPGSGQLINGRGAYLQNNPYLVKAPLETLGLENSYDIYVNAGGGLTGQADAIRLGIARALCQLDIENRPLKEGYLTRDPRAKERRKYGLKARKAPQYSKRMAPKRSGRKQKRNVPSGVAIQSTFNNTIVSITDGEVIAWASAGSSGFKGAKKGTPFAAQTAADNAARRAIDQGMRQIEVMVSGPGSGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKRKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGAKRPKMARIAGVDLPRDKRIEIALTYIYGIGLTRSKILTGVNPDTRTRDLTDDILRIDEYQVEGDLRREMNIKRLDIGYRGRRHRLGLPVRGQRTRTNARTRRGRTVAGKKKPAMALQHVKQEIINYQIHGTDTGSTDVQVAILTERIKQLSEHLKNKDASRRGLLKIIGRRKRLLAYLYDQRYQLIRLGIRGMAVKERVGVVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDENNEAKVGDRVRIRETRPLSRTKRWVIAEILSPRTA------------',\n",
       " 'Microcoleus_sp._PCC_7113': 'RKLSRRLQLKVEDRYPLEALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADLVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGRLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSSTMGPSIQVDINALRDMKMSVGILGTKLGMTQIFDEEGKAIPVTVVQAGPCTVTQIKTKQTDGYTSVQIGYDEVKPKALNKPELGHLAKSSAPPLRHLREYRLDDTASFELGQALKADIFTAGQIVDVAGTSIGRGFAGYQKRHNFKRGPMAHGSKNHRLPGSTGAGTTPGRTFPGKRMAGRLGGTRVTIRKLQVVRVDTERNLLLIKGAVPGKPGALLSIAPTNKVG-R--MQLKYETIPKLEQFYTNIHQVPKLVKVTVNRGLGEASQNAKALESSVNELAITGQKPVVTRAKKAIAGFKIRQGMPVGVMVTLRGERMYSFLNRLINLSLPRIRDFRGVSPKSFDGRGNYTLGIREQLIFPEVEYDSIDQIRGMDISIITTANTDEEGRALLKEMGMPFRTMSRIGKRPITVPGKVTVNIDGQHVAVKGPKGELSRILPEQVAIEQDGETLRVVRRDESRAAR--QRHGLSRTLVANMVEGVSQGFEKRLSIQGVGYRAQVQGRNLILNVGYSKPVEIVPPDGIQVAVENNVNVIVSGINKEIVGNTAARIRAVRPPEVYKGKGIRYANEVVRRKAGKAGKK---MAKKVVAMIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTADQAGMVVPVEISVFEDRSFTFILKTPPASVLIQKAAGIPKGASQPNKQKVGTITRAQLQEIAQTKMPDLNANDIEAAMKIVEGTARNMGVAIADMKTLPSLEKWYVVDANQRLGRLASEIAILTGKNKPYTPHMDTGDFVIVLNAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERIIEAVKGMLPKSSLGKSLFTKLKVYAGDHPHAQKPQLIQTIPGMIQETYLNVADNSGARKLQCIRVLGAGNRYGVGDIIAVVKDAPNMVKKSDVVRAVIVRTRLRRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRKESRRRHRRVRKVGTERPRLAVFRSNHIYVQLIDDTLAASTLENLKTLGTEASVGKLIARSIEKGIRVVFDRGGNLYHGRVQALAEAAREAGLDFMAIDTTTKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPILKVLRSAVANAEHNGLDPALVITQAFADGPLKRFRPRAQGRAYQIRKPTCHITVAVSPK--------------------------------------MPVISLAE-----MLESGVHFGHQTRRWNPKMEPYIY-TSRNGVHIIDLVQTAQLMEEAYSYTRTASEQGKKFLFVGTKRQAAGIIAQEASRC-GAYYVNQRWLGGMLTNWETIKTRVERLKELERREETGALDLLPKKEASVLRRELGKLQKYLGGIKLMRKVPDVVVIVDQRREYNAILECEKL--GLTLVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLEAEEEYEDYEGAMDDVDEEDI-EIDDVVAEDG---EGSMGQKIHPIGFRLGITQE----------HRSRWFADTKRYPEILQEDH--KIRQFVEQK-L-----NNAGIANIRIERKADQVDLEIH-------TARPGVVVGRGG-----------------------------------------------------------------------AGIENLR----------------------VGLQEALG--------GHRQIRINVVE-------------------------------------VARVDADAELIAEYIAQQLERRVSFRRVVRQAIQRAQRA-DVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRANIDFAYRQAKTI-------------------------------------------YGILGVKVWVF-----------------KGEVIPGQEE----QPAQP----------------TTQPRRR-QQRRRQQFEDRSNE-MSRYRGPRLRVRRLGDLPGLTRKTPRRAYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGITEKQLLRYVRKARRGSTGLLQMLERLDNTVFRMGMAGTIPAARQLVNHGHVTVNGRVVNIPSYQCRPGEVIAVKNTSKRLVQNLEYPGLANLPHLEFDKNLGKVNSDREWVALQINELLVVEFYSRKA------MAT-RRKS--NRAK---EKETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVRKGVADGKKQLIDVPLTKSNSIPHPSNGAAVGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNILAKQLGSNNPLNNARAAVNALDSLRTFSEVAEERG--VPIENLYAMSRRKRPIPDYNNRLISMMVRRIMRGKKSIARIIYDAMIERTGDPLDFESAVKNVTPLVEVKARRVGGATYQVPMEVRDRGTTLALRWLIRFSRERGRSMAKLAELMDAANGTGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTIPSTKMTRSVAKVLKDEGFIDEEGEGKKLVVSLKYKGKNRPIIAMKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQGIGGEVLCYVWMTRAVYWGTGRRKSSVARVRLVPGNGELVNGRGDYFQFNAYLLKAPLETLGLENEYDILVNAGGGLTGQADSIRLGVARALCELDPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNTIVTIADGDVVSWASAGSSGFKGAKKGTPFAAQTAADSAARRAADQGMRQIEVMVSGPGAGRETAIRAIQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRRQGRSKYGTKRPKMARIAGIDLPRDKRVEIGLTYIYGIGLSRSKILTGVNPDTRVKDLSDEVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRAVAGKKKPPMSLTQRKHEIIEYQVHETDTGSAEVQIAMLTERINRLSEHLKNKDSSRRGLLQLIGRRKRLLAYMRSQRYPLIRLGIRGMAVKERVGVVVSDKMDKTVVVAVENRSPHPKYGKIVVRTKRYKAHDEENQCKEGDRVRIHETRPLSRTKRWKVAEILTHTSLA-----------',\n",
       " 'Calothrix_sp._PCC_6303': 'KKLSRRMQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGNVRVAVITRGEKVEAAGADIVGSEELIDDIAKGMDFDLIATPDVMPQVAKLGKMLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKVSFSEDLLNLKALQETIDRNRPSGAKGRYWRTYISSTMGPSIRLDINALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVVQAGPCTVTQVKTKQTDGYSAIQVGYREVKPKALSKPKLGHLAKSSAPALRHLLEYRIDNADEYSLGQLITADIFSAGQIVDVAGTSIGKGFAGNQKRNNFGRGPMSHGSKNHREPGSIGAGTTPGRVYPGKRMAGRLGGTSVTIKKLTVIRVDTERNLLLIKGAIPGKPGALVNVVPANLVG-RQKMRLKYETIPKLKQFYSNVHQVPKVIKITVNRGLGEAASNAKALEASINEVAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGDRMYNFLDRLINLSLPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDKIDQIRGMDISIITTAKNDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVTINGSNVQVKGPKGELSRDLPANIVLTQEGEILTVSPIDDSRSSR--QLHGLFRTLISNMVEGVSTGFQRRLEIQGVGYRAAVQGRNLVLNMGYSHQVQIPPPDGVQFAVENNTNVIVSGYDKEIVGNTAAQIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGTVIPVEISVFEDRSFTFVLKTPPASILIKKAAKIEAGSNEPNKKKVGQITREQLQQIAQTKLPDLNANDIDAAMKIIEGTAKNMGVTIADAKTLPNPNQWYVIDADQRLGRLATEIAILRGKNKPYTPHLDTGDFVIIVNAEKIVTGKKTQKLYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQQPKLVNTIPGMIQQSYLNVADNSGARKLMCIRVLGAGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRIRRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELREKSFTKIVSLAPEVLMKLTRRESVRRHKRVRKVGSERPRLAVFRSNHIYAQVIDDTLAASTLEDLKTLGNEASVGKLIARSLAAGIKVVFDRGGNLYHGRIKALADAAREAGLEFMATDTVVKAARYISSPKVRRVLDQIRGRTYREALILLEFMPYRSCDPILKLLRSAAANAEHNGMDRALVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITLAVAAT--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMNPYIY-TSRNGVHIIDLVQTAQLMDEAYNYMRSQAESGKKFLFVGTKRQAAGIIAQEAARC-GSHYINQRWLGGMLTNWTTIKTRVDRLKDLERREENGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKIPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIIGKLADAIYEGRH--GQLDPEEEYDDYDGSD---EYDDTDYSEYKTPDDEE-----MGQKIHPVGFRLGITQE----------HQSRWFASPSRYPELLQEDY--KLRKYIDEK-LGRYAQNNAGICQVKIERKADQIDLEVQ-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQQLLG--------GTRQIRINVVE-------------------------------------VQKVDANAFLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GIQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEI----EAAPA--------------TREREPRRRNQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLSRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGVTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVLINGREVNIASYQCRPGEVISVRNNSKKLVENLQYPGLANLPHLEFDKKMGKVNGEREWIALQVNELLVVEYYSRQA------MAT-GGRKKTNRTK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDVPMTKANSIPHPINGTGGGAKVLMRPASPGTGVIAGGAVRTVLELAGVRNILAKQLGSNNPLNNARAAVNALSTLRTFAEVAEDRG--VPIESLYAMSRRRRQVPDYNSRLVSMTIRRIMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMARLAELMDAANETGSAIKKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIARVLREEGFIEEEGEGKKLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNVGGEVLCYVWMARAMYFGTGRRKRAIARVRLVPGSGKLVNKKGEYFQFNTYLIKAPLETLGMENDYDILVIAGGGLTGQSDAIRLGVARALCQLDPGNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKAKRNVPNGIAIQSTFNNSIVTIADGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAMDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKELSDDVLRIENYQIEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALNQQKQEIFAYQIHETDTGSADVQVAMLTERISLLSKHLQNKDSSRRGLLKLIGQRKSLLAYIQSEHYQLIRLGIRGMAVKERMGVVVSDKMQKTVVVAVQNRAPHPRYGKIVVKTQRYKAHDEDNKCKIGDRVRIQETRPLSKTKRWQVLEIITN---------------',\n",
       " 'Gloeocapsa_sp._PCC_7428': 'KKESRRMQLKVEDRYPIEALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADVVGSEELIDEIQQGMDFDLIATPDVMPQVARLGRLLGPRGLMPSPKGGTVTDVTAIDFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTFVSATMGPSIQVDINALRDMKMAVGILGTKLGMTQVFDEAGKAIPVTVVQAGPCTVTQIKTKQTDGYAAIQVGYKEVKPKALNKPLLGHLAKSSAPSLRHLREYHLDNSSEYNLGQEIKADIFTAGQMVDVIGTSIGRGFAGYQKRHNFGRGPMSHGSKNHRLPGSIGAGTTPGRVYPGKKMAGRLGGKRVTIPQLTVVKVDSERNILLIKGAVPGKPGALLNIVPAKR-G-SQKMRLKYEKIPQLEQFYTNIHQVPKLVKVTVNRGLGEAAQNAKALESSINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGLMVTLRGERMYAFVDRLVNLALPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVDYDSIDQIRGMDISIITTAKNDEEGRALLKAMGMPFRQMSRIGKRPIAIPAKVQVTLDGSQVTVKGPKGELSRELPSAVTIEQEGDTLLVKRKDDSRTAR--QMHGLARTLVANMVDGVSQGFQRRLEIQGVGYRAAVQGRNLTLNVGYSHPVQIEPPEGIQIAVENNTNVIVTGFDKELVGNTAAKIRDVRPPEPYKGKGIRYSGEFVRRKAGKAGKK---MAKKVVAVIKLALTAGKANPAPPVGPALGQHGVNIMMFCKEYNARTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLISKAAGIERGSNEPNKKKVGKISRAQLQEIAQTKMPDLNANDIEAAMKIVAGTARNMGVTVVDMKTVPTIERWYVVDADQRLGRLATEIAILRGKNTPFTPHMDTGGFVIVVNAEKVVTGKKSQKLYRRHSGRPGGMKTETFQLQRLPERIVEAVKGMLPKNSLGRQLFTKLKVYAGAHPHAQQPKIIQTIPGMIQQSYLNVADNSGARKLMCIRVMGAGNRYGVGDIIAVVKDAPNMVKKSDVVRAVIVRTRLRRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLDRRESKIRHRRIRKVGSERPRLAVFRSHHIYAQVIDDTLSASSLETLKQLGTEASVGKLIARSLEKGIRVVFDRGGNLYHGRVKALAEAAREAGLDFMASDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACDPILKVLRSAVANAEHNGLNPALKITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVATA--------------------------------------MPVVSLAQ-----MMEAGVHFGHQTRRWNPKMAPYIY-TSRNGVHIIDLVQTAQLMDEAYQYTRTAAEQGKKFLFVGTKRQAAGIIAQEAARC-GSYYVNQRWLGGMLTNWTTIKTRAERLKDLERREESGALDLLPKKEASVLRREMAKLQKYLGGIKMMRKIPDVVVIVDQRREYNAVQECQKL--GIQIVSMLDTNC--DPDVVDVPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEY-DYEGAEEDLDYDES-YTDSLLPDEEAEAEDEMGQKIHPVGFRLGVTQE----------HQSRWFAEPARYPEILKEDH--KLRNYVEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------TGIESLR----------------------TGLQELLG--------GNRQIRINVVE-------------------------------------VQQVDADAYLIAEYIAQQLERRVSFRRVVRQTIQRAQKA-GIQGIRIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYTTAQTI-------------------------------------------YGILGIKVWIF-----------------KGEIIPGQEP----VQTPS----------------SNQPRRRQQQRRRQQFEDRSNEEMSRYRGPRLRVRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAVRLEEKQKLRFNYGLTEKQLLRYVRRARRGSTGLLQLLERLDNTVFRMGMAPTIPAARQLVNHGHITVNGRVVDIASYQCRPGDEIKVRDRSRKLVENLQYPGLANLPHLEFDKKLGRVNSEREWVALQVNELLVVEYYSRQA------MATGGRRKS-NRVK---EKETNWQERVIQIRRVSKVVKGGKKLSFRAVVAVGNERGQVGIGVGKAGDVIGAVKKGVADGKKHLVDVPLTNSNSIPHPINGSGGGAQVMMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTFSEVAEERG--VPIENLYAMSRRKRPVPDYNSRLISMMMRRVMRGKKSLARIIYDAMIERTGDPIEFERAVRNATPLVEVKARRVGGATYQVPMEVRDRGTTLALRWLIQYARQRGRSMAKLAELMDAANETGSAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTIPATKMTRSIAQVLREEGFIDEEGEGKRLVISLKYKGKNRPLIALKRISKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQGLGGEVLCYVWMSRVMYRGTGRRKSSVARVRLVPGDGQMVNGKGDYFQFNPYLAKAPLETLGLENEYNILVTAGGGLTGQSDSIRLGVARALCQLDPENRPLKEGYLTRDPRAKERKKYGLKARKAPQFSKRMARKKSGKKQKRNVPNGIAIQSTFNNSIVTIADGDVISWASAGSSGFKGAKKGTPYAAQTAAESAARQATDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERSKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLTRSKVLTGVNPDTRVKDLSDDVLRIENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQELISYQLHETDTGSSAVQIAMLTERINRLSEHLRNKDSSRRGLLKLIGQRKRLLSYLQDEQYQLIRLGIRGMAIKERVGVVVSDKMEKTVVVAVENRAPHPKYGKIVVQTRRYKAHDEANECKEGDRVRIQETRPLSRTKRWVVKEILSSATTS-----------',\n",
       " 'Calothrix_sp._PCC_7507': 'KKVSRRLRLKVEDRYPIEALLLQATATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQIRVAVIARGEKVEAAGADIAGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKAAFAEDLLNLKALQETIDRNRPAGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGRAIPVTVIQAGPCTVTQVKTKQTDGYAAIQVGFGAVKPKALNRPLLGHLAKSSAPALRHLSEYHTDAVGDYALGQEIKADIFSAGQIVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTVVRVDAERNLLLIKGAIPGKPGALVSIVPANIVG-RQKARLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFYDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPIPIPAKVQVAIDGTKVVVKGPKGELSRSLPVNVIVSQEGEILHVTRRDETRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIVPPEGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYTGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTIADMKTLPSLQHWYIVDADQRLGRLASEIAVLRGKNKPFTPHMDTGDFVIVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRLPERIVEAIKGMLPKNSLGKHLFTKLKVYAGPHPHAQQPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMVKKSDVVEAVIVRTRINRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRKVGSERPRLAIFRSNHIYAQVIDDTLAASTVEELKSLGNEASVGKLIARALEKGIKVVFDRGGNLYHGRIKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGRSYREALIILEFMPYRACDPILKVLRSAAANAEHNGLDRALIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAA--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDGAYNYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEDDYEDYDSGEYDEDYEESEYTDSVIPDEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPERYPELLQEDY--KLRQYIEKK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TTLPP------APRDRERGDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHITVNGRVVNIASYQTRPGEEIAVRDKSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-GRRKA-NRAK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGTGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--VPIENLYIMSRRKRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRDRGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLRDEGFVEEEEEGKKLVIALKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSSVARVRLVPGEGKLVNGKGVYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKELSDDVLRIENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQELISYQVHETDTGSSEVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYIQSEKYQLIRLGIRGMAIKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVKTQRYKVHDEENKSKVGDRVRIQETRPLSKTKRWQITEILNTKATS-----------',\n",
       " 'Anabaena_sp._wa102': 'KKVSRRLQLKVEDKYPLEALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQIRVAVIARGEKVEAAGADIAGSEELIEDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDVAAIEFKAGKLEYRADRTGIVHVFGKTAFSEDLLNLKALQESIDRNRPSGAKGRYWRSYISSTMGPSIAIDISALRELKVSVGILGTKLGMTQIFDEAGVSIPVTVVKAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPAVRHLNEYRTDSAGDYALGQEIKADIFSAGEIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGTRVTIRKLTVVRVDAERNLILIKGAIPGKPGALVSVVPATIVG-KK-ARLKYETIPKLNQFYTNVHQVPKVIKVTINRGLGEAAQNAKSLEASISEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLINLTLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDRIDQVRGLDISIITTAKTDEEGRALLKEMGMPFRQMSRIGKRPITIPAKVQVAIDGVKVVVKGPKGELSRQLPNNVILSLDGETLLVTRRDETRTSR--QMHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLEGRNLVLNMGYSHKVQIEPPEGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGSISRTQLREIAQTKLPDLNANDIDAAMNIVEGTAKNMGVTVTDSKTLPSLERWYLVDADKRLGRLASEVAILRGKRKAYTPHLDTGDFVIIINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRLPERILEAIKGMLPKNSLGKQLFTKLKVYAGPHPHAQQPKIISTIPGVIQQSYLNVADNSGAKKLMCIRVLGAGNRYGVGDIIAVVKESPNMVKKSDVVEAVIVRTRISRDSGCIRFDDNAAVIINEGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRKESQRRHRRVRKVGSERPRLAIFRSNHIYAQVIDDTIAASTVEELKSVGNDASVGKLVARALEKGIKVVFDRGGNLYHGRIKALAEAAREAGLDFMATDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYGACDPILKVLRSAAANAEHNGLDRALVISQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVSAT--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMAPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRTQSEQGKKFLFVGTKRQAAGIIAQEALRC-GSHYINQRWLGGMLTNWATIKTRAERLKDLERREESGALDLLPKKEASMLRREMTKLQKYLGGIKNMRKVPDIVVIVDQKREYNAVQECEKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIIGKLADAIYEGRH--GQLEAEEDYDDY--ADEDYEYDES-STES-----E---DEAVGQKIHPVGFRLGITQE----------HQSRWFAEPDRYPELLQEDY--KLRQYIEQK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------LGLQGALG--------GNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVGG-----------------------------RLNGAEIARSE-----WTREG---RVPLHTLRADIDYSGCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEQ----APAQP---ARERDRDRDRGDRDREPRRRQQQRRRQQFEDRSNESMSRYRGPRLRIRRLGELPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRMNYGLTEKQMLRYVRKARRGSTGLLQLLERLDNTVFRMGMAPTIPAARQLVNHGHVTVNGKPVNIASYQCRPGEEVAVRNRSKKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MVT-GRRKA-SRTK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGIRNILAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IAIEKLYIMSRRRRAVPDYNSRLISMMMRRIMRGKKSIARIVYDALIERTGSPLEFEKAVRNATPLVEVKARRVGGATYQVPMEVRDRGTTLALRWLVQYSRQRGRTMARLAELMDAANETGSAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGEGKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCHVWMSRAVYWGTGRRKASVARVRLVPGEGKMVNGKGDYFQFNPYLIKAPLETLGLESEYDIVVKAGGGLTGQADSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRATDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKLPKVARISGVDLPRDKRIEIGLTYIYGIGLTRSHILTGVNPDTRVKELSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQELISYQVHETDTGSADVQIAMLTDRINRLSQHLQNKDSSRRGLLKMIGQRKRLLAYIQSEKYKLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVQTRRYKVHDEENTCKTGDRVRIQETRPLSKTKRWQVAEILNTKAT------------',\n",
       " 'Anabaena_sp._90': 'KKVSRRLQLKVEDRYPLEALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQIRVAVIARGEKVEAAGADIAGSEELIEDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDVAAIEFKAGKLEYRADRTGIVHVFGKTAFSEDLLNLKALQESIDRNRPSGAKGRYWRSYISSTMGPSIAIDISALRELKMSVGILGTKLGMTQIFDEAGVSIPVTVIKAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKSLNRPLLGHLAKSSAPPVRHLNEYRTDSTGDYALGQEIKADIFSVGEIVDVVGTSIGRGFAGNQKRNNFARGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGTRVTIRKLTVIRVDAERNLIIIKGAIPGKPGALVSVVPATIVG-KK-ARLKYETIPKLTQFYTNVHQVPKVIKITINRGLGESAQNAKSLEASISEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLINLTLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDRIDQVRGLDISIITTAKTDEEGRALLKEMGMPFRQMSRIGKRPITIPAKVQVAIDGVKVVVKGPKGELSRQLPKNVIVSQEGETLIVTRRDETRTSR--QMHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGSNLVLNMGYSHQVQIEPPAGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLIRKAAKIERGSSEPNKTKVGSISRTQLREIAQTKLPDLNANDIDAAMNIVEGTAKNMGVTITDMKTLPSIERWYLVDADKRLGRLATEVAILRGKKKPFTPHMDTGDFVIIINAEKVVTGKKTQKVYRRHSGRPGGMKTETFKLQRLPERILEAIKGMLPKNSLGKQLFTKLKVYAGPHPHAQLPKIISTIPGMIQQSYLNVADNSGAKKLMCIRVLGAGNRYGVGDIIAVVKESPNMVKKSDVVEAVIVRTRISRDSGAIRFDDNAAVIINEGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRKESQRRHRRVRKVGSERPRLSIFRSNHIYAQLIDDTIAASTVEELKSIGNDASVGKLVARALEKGIKVVFDRGGNLYHGRVKALAEAAREAGLDFMATDTTVKAARYIRSPKVRRVLDQIRGRSYREALILLEFMPYGACDPILKVLRSAAANAEHNGLDRSLVISQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVSAT--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMAPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRTQSEQGKKFLFVGTKRQAAGIIAQEALRC-GSHYINQRWLGGMLTNWATIKTRAERLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKNMRKVPDIVVIVDQKREYNAVQECEKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLEAEEDYDDY--ADEDYEYDES-STES-----E---GEAMGQKIHPVGFRLGITQE----------HQSRWFAEPDRYPELLQEDY--KLRQYIEQK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------LGLQGALG--------GNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVGG-----------------------------RLNGAEIARSE-----WTREG---RVPLHTLRADIDYSGCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEQ----APAQP-----AKDRERGDRDREREPRRRQQQRRRQQFEDRSNEAMSRYRGPRLRIRRLGELPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRMNYGLTEKQMLRYVRKARRGSTGLLQLLERLDNTVFRMGMAPTIPAARQLVNHGHVTVNGKPVDIASYQCRPGEEVAVRNRSKKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MVT-GRRKA-SRTK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGIRNILAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--ISIEKLYIMSRRRRTVPDYNSRLISMMMRRIMRGKKSIARIVYDALIERTGSPLEFEKAVRNATPLVEVKARRVGGATYQVPMEVRDRGTTLALRWLVQYSRQRGRTMARLAELMDAANETGSAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRNIAKVLREEGFIEEEGEGKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCHVWMSRAVYWGTGRRKCSVASVRLVPGEGKMVNGKGDYFQFNPYLIKAPLETLGLESEYDILVKAGGGLTGQADSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRATDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKLPKMARISGVDLPRDKRVEIGLTYIYGIGLTRSHILTGVNPDIRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISFQVHETDTGSADVQIAMLTDRINKLSQHLQNKDSSRRGLLKMIGQRKSLLAYIQSEKYKLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVQTRRYKVHDEENTCKIGDRVRIQETRPLSKTKRWQVKEILNIKAT------------',\n",
       " 'Dolichospermum_circinale_ACBU02': 'KKVSRRLRLKVEDRYPLEGLLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQIRVAVIARGEKVEAAGADIAGSEELIEQIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIGAIEFKAGKLEYRADRTGIVHVFGKAGFSEDLLNLKALQESIDRNRPSGAKGRYWRSYVSSTMGPSIAIDINALRELKVSVGILGTKLGMTQIFDEEGVSIPVTVIQAGPCTVTQVKTKQTDGYAAIQVGYGEVKPKALNRPLLGHLAKSSAPAVRHLNEYRTDSAGDYALGQEIKADIFSAGEIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGTRVTIRKLTVVRVDTERNLIIIKGAIPGKPGALVNVLPAKKVG-KK-ARLKYETIPKLSQFYTNVHQVPKVTKITVNRGLGEAAQNAKSLEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLINLTLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDKIDQVRGLDISIITTAKTDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVAIDGVKVVVKGPKGELSRQLPNNVIVSQEGETLLVTRRDETRTSR--QMHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLVLNMGYSHQVQIEPPEGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKLVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLIRKAAKIERGSEQPNKKKVGSISRNQLREIAQTKLPDLNANDIDAAMKIVAGTAKNMGVTITDSKTLPSLERWYLVDADKRLGRLASEVAILRGKKKAYTPHLDTGDFVIIINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRLPERILEAIKGMLPKNSLGKQLFTKLKVYAGPHPHAQQPKIISTIPGVIQQSYLNVADNSGAKKLMCIRVLGAGNRYGVGDIIAVVKESPNMVKKSDVVEAVIVRTRINRDSGTIRFDDNAAVIINEGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRKVGSERPRLAIFRSNHIYAQVIDDTIAASTVEELKSIGNEASVGKLVARALEKGIKVVFDRGGNLYHGRIKALAEAAREAGLDFMATDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYGACEPILKVLRSAAANAEHNGLDRGLVISQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVSAT--------------------------------------MPVVSLAQ-----LMESGVHFGHQTRRWNPKMAPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRTQAEQGKKFLFVGTKRQAAGIIAQEALRC-GSHYINQRWLGGMLTNWATIKTRAERLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGN--------------------------------------------------------------------------------------------------------------------------VGQKIHPVGFRLGITQE----------HQSRWFAEPDRYPELLQEDY--KLRQYIEQK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------LGLQGALG--------GNRQIRINVVE-------------------------------------IQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVGG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSGCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEQ----APVQP-----------------ARERD--RDRDRDR-GDREREPMSRYRGPRLRIRRLGELPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRMNYGLTEKQMLRYVRKARRGSTGLLQLLERLDNTVFRMGMAPTIPAARQLVNHGHVTVNGRVVNIASYQCRAGEEVAVRNRSKKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA--------------------------------VIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGIRNILAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IAIEKLYIMSRRRRPVPDYNSRLISMMMRRLMRGKKSIARIVYDALIERTGSPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRDRGTTLALRWLVQYSRQRGRTMARLAELMDAANETGSAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNAKHQTTVPATKMTRSIAKVLREEGFIEEEGEGKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCHVWMSRAVYWGTGRRKSSVARVRLVPGAGKMVNGKGEYFQFNQYLIKAPLETLGXENEYDIVVKAGGGLTGQADSIRLGVARALCELDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGIAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRATDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRQKVARIAGVDLPRDKRVEIGLTYIYGIGLTRSHILTGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTDRINQLSKHLQNKDSSRRGLLKMIGQRKRLLAYIQSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVKTRRYKVHDEENTCKIGDRVRIQETRPLSKTKRWQVTEILNTKAKA-----------',\n",
       " 'Fischerella_JAHHIM01': 'KKISRRLQLKVEDKYPLEALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADVVGSEELINEIQQGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDVAAIEFKAGKLEFRADRTGIVHVFGKVAFSEDLLNLKALQETIDRNRPSGAKGRYWRSYVASTMGPSIKVDISALRDLKVSVGILGTKLGMTQVFDDAGVSIPVTVVQAGPCTVTQVKTKQTDGYAAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDTPSDYALGQEIKADIFSAGQLVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGTRVTIRKLTVVRVDPERNLLLIKGAIPGKPGSLINILPAKKVG-K--ARLKYETIPKLNQFYTNVHQVPKLVKITINRGLGEAAQNAKALEASLNEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRSERMYAFLDRLISLSLPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKTDEEGRALLKEMGMPFRQMSRIGKRPITIPAKVQVTIDGTKVMVKGPKGELSRDLPANVIVSQEGEILQVNRRDDTRVSR--QMHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLVLNMGFSHQVQIVPPEGVQFAVENNTNVIVTGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGSITRAQLQEIAQTKMPDLNANDIEAAMNIVAGTAKNMGVTIADSKTLPSLERWYLVDADKRLGRLASEIAILRGKRKAYTPHLDTGDFVIVINAEKVVTGKKSQKLYRRHSGRPGGMKTETFKLQRLPERIVEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTFLNVADNSGAKKLMCIRVLGAGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRIRRDSGTIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRKVGSERPRLAVFRSNHIYAQVIDDTLAASTIEELKSLGTEASVGKLIARLLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAIDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKLLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITLAVAASSYGKLEGLSNLAHPVIRVFPFREIQLDGGLTRIGEKY-MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYAYMRSQSEQGKKFLFVGTKRQAAGIIAQEAARC-GSHYINQRWLGGMLTNWATIKTRADRLKDLERREENGALDLLPKKEASMLRREMTKLQKYLGGIKNMRKVPDVVVIVDQRREYNAVQECQKL--GLPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDTEEDYEDYDGAEYDEDYDES-YAESSSDDEE---PENMGQKIHPVGFRLGITQE----------HQSRWFAVPDRYPELLQEDH--KLRQYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQELLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIGEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TPAPP-------------TTRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRAYPPGQHGQRKKRSEYAVRLEEKQKLRLNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAIRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQANREHLIMAT-GRRKA-NRTK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPMTKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIERLYIMSRRRRPVPDYNSRLVSMIIRRVMRGKKSLARLVYDALIERTGAPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTVADMLTRIRNARHQTTVPATKMTRSIAKVLQEEGFIESEEEGKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQSLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGEGKLVNGKGEYFQFNAYLIRNPLETLGLENEYDILVKAGGGLTGQADSIRLGVARALCQLDPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKGGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARISGVDLPRDKRVEIGLTYIYGIGLTRSLILTGVNPDTRVKDLSDDILRIENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIMSYQVHETDTGSAEVQVAMLSERISRLSLHLQNKDSSRRGLLKMIGQRKRLLAYIQSEKYQLIRLGIRGMAIKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTRRYKAHDAESKCKVGDRVRIQETRPLSKTKRWQVKEILNTKATT-----------',\n",
       " 'Nodularia_spumigena_CCY9414': 'KKISRRFQLKVEDRYPLEALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVTLPKGTGQVRVAVIARGEKVEAAGADVVGSDELINEIQQGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDVAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVTSTMGPSIKVDINALRDLK-----------MTQVFDEAGVSIPVTVVQAGPCTVTQVKTKQTDGYSAIQLGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSPSDYALGQELKADIFSEGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGTRVTIRKLTVVRVDLERNLLLIKGAIPGKPGSLVNILPAKKVG-K--PRLKYETIPKLNQFYTNVHQVPKVIKITINRGLGEAAQNAKSLEASLNEIAITGQKPVVTRAKKAIAGFKIREGMPVGIMVTLRGERMYAFLDRLISLSLPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKNDEEGRALLKEMGMPFRQMSRIGKQPITIPAKVEVTIDGPKVVVKGPKGQLSRTLSANVIVSQEGGILNVTRRDETRVSR--QMHGLSRTLVANMVEGVSQGFKRRLEIQGVGYRAQLQGRNLVLNIGYSHQVNIVPPDGVEFLVETNTNIVVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGSITKAQLQEIAQTKMPDLNANDIEAAMRIVEGTANNMGVTISDTKTLPTLERWYVVDADKRLGRLATEIAILRGKKKAFTPHMDTGDFVIVINAEKIVTGKKTQKLYRRHSGRPGGMKTETFKLQRLPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQQPKLINTIPGMIQQTYLNVADNSGAKKIMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVIEAVIVRTRIRRDSGTIRFDDNAAVIINDGNPRGTRVFGPVARELREKSFTKIVSLAPEVLMKLTRRESQRRHRRVRKVGTERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGTDASVGKLIARLLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAIDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKLLRSAAANAEHNGLDRTLVITQAYANGPLKRFQPRAQGRAYQIRKPTCHITVAVADA--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSQSEQGKKFLFVGTKRQAAGIIAQEAARC-GSHYINQRWLGGMLTNWATIKTRADRLKDLERREENGALDLLPKKEASMLRREMTKLQKYLGGIKNMRKVPDVVVIVDQRREYNAVQECQKL--GLPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRG--GTLEGEDDYEDYDGSEYDDDYDES-YAQG-SSDES---SDEMGQKIHPVGFRLGITRE----------HQSRWFAVPERYPELLQEDY--KLRQYIEKK-LGKLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------LGLQEVLG--------GNRQIRINVVE-------------------------------------VQKVDADASLIGEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----IPAPP-STRDRGDR-----DREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLSRKTARRAYPPGQHGQRKKRSEYAVRLEEKQKIRMNYGLTEKQLLRYVRKARRGSTGLLQMLERLDNTVFRLGMAPTIPAARQLVNHGHVTVNGKRVDIASYQCRPGEVVAIRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-GRRKA-NRTK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPMTKSNSIPHPIDGVGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNILAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIEKLYI-----------------MIIRRVMRGKKSLARLVYDAMIERTGAPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRDRGTTLALRWLVNFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTVADMLTRIRNARHQTTVPATKMTRSIAKVLQEEGFVEAEGEGKRLVISLKYKGKNRPLIALKRVSKPGLRVYSTKLPRVLGGIGIAIISTSSGIMTDREARRQSLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGEGKLVNGKGEYFQFNAYLIKNPLETLGLENEYDILVKAGGGLTGQADSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKR-----------------MAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAMDQGMRQIEVMVSGPGAGRETAIRAIQAGLEITLIRDITPIPHNGCRPPKRRRV---------------------------------------------------------------------------MIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARISGVDLPRDKRVEIGLTYIYGIGLTRSLILTGVNPDTRVKDLSDEVLRIENYQVEGDLRREMSIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIMSYQVHETDTGSAEVQVAMLSDRISRLSLHLQNKDSSRRGLLKMIGQRKRLLSYINSEKYQLIRLGIRGMAIKERVGLVVSDKMQKTVVVAIENRSPHPKYGKIMVKTRRYKVHDEENQCKVGDRVRIQETRPLSKTKRWQVKDILNTKATM-----------',\n",
       " \"'Nostoc_azollae'_0708_chromosome\": 'KKISRRLQLKVEDRYPLEALMLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDQIQKGMDFDLIATPDVMPMVAKLGKLLGPRGLMPSPKGGTVTDVGAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKIDISALRDLQMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYCAIQVGYGVVKPKVLNRPLLGHLAKSSAPALRHLNEYRTDASGDYALGQELKADIFSAGQIVDVIGTSIGRGFAGNQKRNHFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGTRVTIRKLTIVRLDADRNLILIKGAIPGKPGALVNIVPTNKVG-R--ARLKYETIPKLQQFYTNVHQVPKVIKITVNRGLGEAAQNAKALEASLNEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLVSLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKEMGMPFRQMSRIGKCPITVPAKVQVAIDGSKVVVKGPKGELSRNLPPHVTVSQEGETLLVTRRDDTRTSR--QMHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIEPPDGIQFAVEGTINVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYVGEVVRRKAGKTGGKGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFAFVLKTPPASVLIRKAAKIERGSNEPNKKKVGSISKAQLKEIAQTKLPDLNANDIDAAMNIVAGTAKNMGVTITDSKTLPSLERWYVVDADKRLGRLASEIAVLRGKRKAYTPHLDTGDFVVVINAEKVVTGKKTQKFYRRHSGRPGGMKTETFKLQRLPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVNTIPGMIQQSYLNVADNSGARKLMCIRVLGGGNRYGVGDIIAVVKDAPNMVKKSDVVEAVIVRTRINRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRIKALAEAAREAGLDFMAIDTTVQAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRSLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAT--------------------------------------MAVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSQAEQGKKFLFVGTKRQAAGIIAQEAARC-GSHYINQRWLGGMLTNWATIKTRAERLKDLERREENGALDLLPKKEGSMLRREMTKLQKYLGGIKNMRKVPDVVIIVDQRREYNAVQECEKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLEAEGDYEDYEGAEDDYDYDES-YTDTLIPDEE---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPDRYPELLQEDH--KLRKYIDQK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQELLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAQTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TPAQP-------------TTREREPRRRQQQRRRQQFEDRSNEAMSRYRGPRLRIRRLGELPGLTRKSARRAYAPGQHGQRKKRSEYAIRLEEKQKLRLNYGLTEKQMLRYVRKARRGSTGLLQLLERLDNTVFRMGMAPTIPGARQLVNHGHVTVNGRVVNIASYQCRPGEEIAVRDNSKKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAA-GRRKA-NRAK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTFAEVAEDRG--IAIEKLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGSALEFERAVRNATPLVEVKARRVGGATYQVPMEVRDRGTTLALRWLVQFSRSRGRTMARLAELMDAANETGSAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRNIAKVLQDEGFIESEGEGKQLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGEGNFVNGKGEYFQFNAYLIKAPLETLGLESEYDILVKAGGGLTGQADSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQVGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARISGVDLPRDKRIEIGLTYIYGIGLSRSQILTGVNPDTRVKDLNDDILRIENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQELISFQVHETDTGSADVQIAMLTDRINRLSQHLQNKDSSRRGLLKMIGQRKRLLGYIQNEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRSPHPKYGKIVVQTRRYKAHDEDNHCKVGDRVRIQETRPLSKTKRWQVTEILNAKATT-----------',\n",
       " 'Anabaena_cylindrica_PCC_7122': 'KKLSRRLQLKVEDRYPVEALLLKETATAKFEAVEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIEDIQKGMDFDLIATPDVMPMVAKLGKLLGPRGLMPSPKGGTVTDVAAIEFKAGKLEFRADRTGIVHVFGKASFAEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKIDISALRDLKMSVGILGTKLGMTQIFDDAGRAIPVTVIQAGPCTVTQVKTKQTDGYAAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYRTDASGDYALGQELKADIFSEGQIVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGTRVTIRKLTVVRVDAERNLILIKGAIPGKPGALVNVVPTNKVGNK--ARLKYETIPKLNQFYTNVHQVPKVVKITINRGLGEAAQNAKSLEASLNEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLVSLSLPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVAIDGSKVVVKGPKGELSRVLPSNVIVSQEGETLLVTRRDETRTSR--QLHGLSRTLVANMVEGVSLGFQRRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIEPPEGIQFAVEGTTNVIVSGYDKEIVGNTAAKVRGVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGSITRVQLREIAQTKLPDLNANDIDAAMNIVEGTAKNMGVTITDSKTLPTLERWYVVDADKRLGRLASEIAILRGKKKPYTPHMDTGDFVVVINAEKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQQPKLINTIPGMIQQTYLNVADNSGAKKLMCIRVLGAGNRYGVGDIIAVVKESPNMVKKSDVVEAVIVRTRINRDSGCIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRKVGSERPRLAVFRSNHIYVQVIDDTLAASTVEELKSLGNNASVGKLIARSLEKGIKVVFDRGGNLYHGRIKALAEAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAT--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSQSEQGKKFLFVGTKRQAAGIIAQEAARC-GSHYINQRWLGGMLTNWATIKTRADRLKDLERREENGALDLLPKKEASMLRREMTKLQKYLGGIKNMRKVPDVVIIVDQRREYNAVQECEKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLEAEGDYDDYEGAEGDYDYDESEYTDAVIPDEE---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPDRYPELLQEDH--KLRKYIEQK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVGG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAQTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TPSAP---------------TTREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSPRRAYPPGQHGQRKKRSEYAIRLEEKQKLRLNYGLTEKQMLRYVRKARRGSTGLLQFLERLDNTVFRMGMAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEEIAVRNKSKKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-GRRKT-NRAK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTFAEVAEDRG--IPIESLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRDRGTTLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRNIAKVLRDEGFIESEGEGQQLVIALKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAVISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGEGKFVNGKGEYFQFNAYLIKAPLETLGLESEYDILVKAGGGLTGQADSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAGRRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARISGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRIENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQELISYQVHETDTGSADVQIAMLTDRINRLSQHLQNKDSSRRGLLKMIGQRKRLLSYIQSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVRTQRYKVHDEDNHCKVGDRVRIQETRPLSKTKRWQVTEVLNTKATT-----------',\n",
       " 'Anabaena_sp._PCC_7108': 'KKISRRLELKVEDRYPVEALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFAEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKIDISALRDLKMSVGILGTKLGMTQIFDDAGRAIPVTVIQAGPCTVTQVKTKQTDGYFAVQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYRIDNSGDYALGQEIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGTRVTIRKLTVVRVDAERNLILIKGAIPGKPGALVNVLPAKIVGSK--ARLKYETIPKLNQFYTNVHQVPKVVKITINRGLGEAAQNAKALEASLNEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVAIDGTSVVIKGPKGELSRILPANVTVSQEGETLLVTRRDETRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIEPPDGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGSITRAQLREIAQTKLPDLNANDIDAAMNIVEGTAKNMGVTVADSKTLPSLERWYVVDADKRLGRLASEIAVLRGKRKAYTPHMDTGDFVVVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRLPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQQPKLISTIPGMIQQTYLNVADNSGAKKLMCIRVLGAGNRYGVGDIIAVVKESPNMVKKSDVVEAVIVRTRINRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRKVGSERPRLAVFRSNHIYVQIIDDTLAASTLEELKSLGNNASVGKLIARSLEKGIKVVFDRGGNLYHGRIKALAEAAREAGLDFMATDTTVKAARYIRSAKVRRVLDQIRGRSYREALIILEFMPYRACDPVLKVLRSAAANAEHNGLDRALVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAA--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRTQSEQGKKFLFVGTKRQAAGIIAQEAARC-GSHYINQRWLGGMLTNWATIKTRADRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKNMRKVPDVVIIVDQKREYNAVQECEKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLEAEEDYEDYSEED---DYDESEYADAVVPDEE---EEAMGQKIHPVGFRLGITQE----------HQSRWFAVPDRYPELLQEDH--KLRKYIEQK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQEMLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAQTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TPPPP---------ATRDRDRDRDPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRMNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRMGMAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEEVAVRNKSKKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTFAEVAEDRG--IAIENLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYEALIERTGGALEFEKAVRNATPLVEVKARRVGGATYQVPMEVRDRGTTLALRWLVQYSRARGRTMARLAELMDAANETGSAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRNIAKVLQNEGFIEEEGEGKLLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVASVRLVPGEGKLVNGKGVYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPNGIAIQSTFNNSIVSITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARISGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRIENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQELISFQVHETDTGSADVQIAMLTDRINRLSQHLQNKDSSRRGLLKMIGQRKRLLSYIQSDKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTQRYKVHDEENNCKVGDRVRIQETRPLSKTKRWQITEILNTKSTT-----------',\n",
       " 'Cylindrospermum_stagnale_PCC_7417': 'KKISRRLQLKVEDRYPLEALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKAAFAEDLLNLKALQETIDRNRPSGAKGRYWRTYIAATMGPSIRIDINALRDFKMSVGILGTKLGMTQIFDEAGRAIPVTIIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYRTDSSSDYALGQAIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRVTISKLTVVRVDAERNLLLIKGAIPGKPGALVSIVPATLVG-RAKARLKYETIPKLNQFYTNVHQVPKVVKITVNRGLGEAAQNAKSLEASLNEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLVSLALPRIRDFRGVSAKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKNDEEGRALLKEMGMPFRQMSRIGKRPITIPAKVQVAIDGTKVVVKGPKGELSRNLPVNVTVSQSGETLLVTRRDETRTSR--QLHGLSRTLVANMVEGVSSGFQRRLEIQGVGYRAQLQGSNLLLNMGYSHQVQIAPPEGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIEHGSNEPNKKKVGNITRAQLREIAQTKLPDLNANDIDAAMNIVEGTAKNMGVTVSDMKTLPSLERWYVVDADKRLGRLASEIAILRGKNKPYTPHLDTGDFVVVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRLPERILEAVKGMLPKNSLGKHLFTKLKVYAGPHPHAQQPKLINTIPGMIQQTYLNVADNSGARKLMCIRILGGGNRYGVGDIIAVVKDAPNMVKKSDVVEAVIVRTRINRDSGCIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRKVGSERPRLAVFRSNHIYAQIIDDTLAASTIEELKSLGNEASVGKLIARSLEKGIKVVFDRGGNLYHGRIKALADAAREAGLDFMATDTTAKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAT--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDAAYNYMRSQSEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRADRLKDLERREETGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECQKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLEGEEDYEEYEGDEY--DYEET-YTDALIPDGE---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPDRYPELLQEDY--KLRQYIEQK-LGRLAPNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIEALR----------------------TGLQELLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TPPPP---------TTRDRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLSRKSPRRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRMGIAPTIPAARQLVNHGHITVNGRIVNIASYQCRAGEEIAVKNKSRKLVENLQYPGLANIPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQAFTRALIMAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPINGTGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNILAKQLGSNNPLNNARAAVNALSTLRTFAEVAEDRG--VPIENLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRARGRTMAKLAELMDAANETGSAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRNIAKVLQNEGFIEVEEEGKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYIWMSRAVYWGTGRRKCSVARVRLVPGEGKLVNGKGVYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPNGIGIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARISGVDLPRDKRVEIGLTYIYGIGLSRSQILTSVNPDTRVKDLTDDVLRIENYQVEGDLRREMSIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQELISFQVHETDTGSADVQVAMLTDRINRLSEHLQNKDSSRRGLLKMIGQRKRLLSYIQSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVKTQRYKVHDEDNKCKVGDRVRIQETRPLSKTKHWQVTEILNAKATT-----------',\n",
       " 'Nostoc_JAIVFY01': 'AKISRRLQLKVEDKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIGAIEFKAGKLEFRADRTGIVHVFGKATFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKIDVTALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYFAIQVGYGEVKPKSLNRPLLGHLAKSSAPALRHLNEYHTESSGDYALGQEIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDAERNLLLIKGAIPGKAGALVSVVPAKVVG----ARLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSVDQVRGMDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTNIVVKGPKGELSRTLTPNVSLSQEGDILQVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIEPPDGIQFAVEGTTNVIVSGYDKEIVGNTAAKVRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSISKAQLQEIAQTKLPDLNANDIDAAMKIVAGTAKNMGITVTDTKTLPTLERWYIVDADKRLGRLASEIAVLRGKKKPYTPHLDTGDFVIVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVSTIPGMIQQTYLNVADNSGARKLMCIRILGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVSRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQIIDDTIAASTVEELKSLCNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAEKCANLVWLTNLAHLRIRVFPGWEMHLDGGLTRIGVRKYMPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--SIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEDDYEDYDGSEYDDDYEETEYTDAVIPDEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDY--KLRQYIEQK-LGRQAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEP----DPLPP----ASRDRERDPRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGRVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSKTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVMMRPASPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIESLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEEEGKHLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGTYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENKCKVGDRVRIQETRPLSKTKRWKITEVLNVKPT------------',\n",
       " 'Fischerella_JAHHHS01': 'AKISRRLQLKVEDKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIGAIEFKAGKLEFRADRTGIVHVFGKATFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKIDVTALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYFAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTESSGDYALGQEIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDAERNLLLIKGAIPGKPGALVSVVPAKVVG----ARLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSVDQVRGMDISIITTATNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTNIVVKGPKGELSRTLTPNVSLSQEGEILHVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIEPPEGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSISKAQLQEIAQTKLPDLNANDIDAAMKIVAGTAKNMGITVTDTKTLPTLERWYIVDADKRLGRLASEIAVLTGKKKPYTPHLDTGDFVIVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVSTIPGMIQQTYLNVADNSGARKLMCIRILGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVSRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQIIDDTIAASTVEELKSLCNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAEKCANLVWLTNLAHLRIRVFPGWEMHLDGGLTRIGVRKYMPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEDDYEDYDGSEYDDDYEETEYTDAVIPDEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDH--KLRQYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TPLPP----ASRDREREPRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGRVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVMMRPASPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIESLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEEEGKHLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGSGQLVNGKGNYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENKCKVGDRVRIQETRPLSKTKRWKITEVLNVKPT------------',\n",
       " 'Nostoc_JACJQM01': 'AKISRRLQLKVEDKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIGAIEFKAGKLEFRADRTGIVHVFGKATFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKIDVTALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYFAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTESSGDYALGQEIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDAERNLLLIKGAIPGKPGALVSVVPAKVVG----ARLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSVDQVRGMDISIITTATNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTNIVVKGPKGELSRTLTPNVSLSQEGEILHVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIEPPDGIQFAVEGTTNVIVSGYDKEIVGNTAAKVRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSISKAQLQEIAQTKLPDLNANDIDAAMKIVAGTAKNMGITVTDTKTLPTLERWYIVDADKRLGRLASEIAVLRGKKKPYTPHLDTGDFVIVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVSTIPGMIQQTYLNVADNSGARKLMCIRILGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVSRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQIIDDTIAASTVEELKSLCNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAEKCANLVWLTNLAHLRIRVFPGWEMHLDGGLTRIGVRKYMPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--SIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEDDYEDYDGSEYDDDYEETEYTDAVIPDEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDY--KLRQYIEQK-LGRQAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEP----DPLPP----ASRDRERDPRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGRVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSKTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVMMRPASPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIESLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEEEGKHLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGTYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENKCKVGDRVRIQETRPLSKTKRWKITEVLNVKPT------------',\n",
       " 'Nostoc_NOLK01': 'AKISRRLQLKVEDKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKTTFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKIDVTALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYFAIQVGFGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDAERNLLLIKGAIPGKPGALVSVVPAKVVG----ARLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRSERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSVDQVRGMDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTNIVVKGPKGELSRTLRDNVSLSQEGEILHVNRRDETRTSK--QLHGLSRTLVANMVEGVSLGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIEPPEGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSITKAQLQEIAQTKLPDLNANDIDAAMKIVAGTAKNMGITVKDTKTLPSIERWYIVDADKRLGRLASEIAVLRGKKKPYTPHLDTGDFVIVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRVPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQQPKLVSTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVSRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQVIDDTIAASTLEDLKSLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAEECANLVWLTNLAHLRIRVFPSWEMHLDGGLTRIGVRKYMPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEDDYEDYDGSEYDDDYEET-YTDAVIPDEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDY--KLRQYIEQK-LGRQAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEP----DPLPP----ASRDRERDPRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGMHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPASPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYEALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEEEGKHLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGNYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENKCKVGDRVRIQETRPLSKTKRWKITEVLNVKPA------------',\n",
       " 'Nostoc_punctiforme_PCC_73102': 'AKISRRLQLKVEDKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKTTFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKIDVTALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYFAIQVGFGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSEGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDAERNLLLIKGAIPGKPGALVSVVPAKVVG----ARLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSVDQVRGMDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAINGTNIVVKGPKGELSRTLRDNVSLSQEGEILHVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIEPPDGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSITKAQLQEIAQTKLPDLNANDIDAAMKIVAGTAKNMGITVKDTKTLPTLERWYIVDADKRLGRLASEIAVLRGKKKPYTPHLDTGDFVIVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVSTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVSRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQVIDDTIAASTLEELKSLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEDDYEDYDGSEYDDDYEETEYTDAVIPDEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDY--KLRQYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEP----DPLPP----ASRDRERDPRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYEALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEEEGKHLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGNYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENKCKVGDRVRIQETRPLSKTKRWKITEVLNIKPA------------',\n",
       " 'Nostoc_JACKZN01': 'AKISRRLQLKVEDKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKTTFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKIDVTALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYFAIQVGFGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDAERNLLLIKGAIPGKPGALVSVVPAKVVG----ARLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSVDQVRGMDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVEVAINGTNIVVKGPKGELSRTLRDNVSLSQEGETLHVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIEPPDGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSITTAQLQEIAQTKLPDLNANDIDAAMKIVAGTAKNMGITVKDTKTLPTLERWYIVDADKRLGRLASEVAVLRGKKKPYTPHLDTGDFVIIINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVSTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVSRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQVIDDTIAASTLEELKSLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAEECANLVWLTNLAHLRIRVFPDWEIHLDGGLTRIGVEKYMPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEDDYEDYDGSEYDDDYEETEYTDAVIPEEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDY--KLRQYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEP----DPLPP----ASRDRERDPRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGMHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYEALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEEEGKHLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGNYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENKCKVGDRVRIQETRPLSKTKRWKITEVLNVKPA------------',\n",
       " 'Nostoc_JACJSH01': 'AKISRRLQLKVEDKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKTTFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKIDVTALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYFAIQVGFGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDAERNLLLIKGAIPGKPGALVSVVPAKVVG----ARLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSVDQVRGMDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTNIVVKGPKGELSRTLRDNVSLSQEGEILHVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIEPPDGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSITKAQLQEIAQTKLPDLNANDIDAAMKIVAGTAKNMGITVKDTKTLPTLERWYIVDADKRLGRLASEIAVLRGKKKPYTPHLDTGDFVIVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVNTIPGMIQQTYLNVADNSGARKLMCIRILGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVSRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQVIDDTIAASTLEELKSLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDTEDDYEDYDGSEYDDDYEETEYTDAVIPEEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDY--KLRQYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEP----DPLPP----ASRDRERDPRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYEALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEEEGKHLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGNYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKITEVLNIKPA------------',\n",
       " 'Nostoc_JADEXT01': 'AKISRRLQLKVEDKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKTTFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKIDVTALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYFAIQVGFGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDAERNLLLIKGAIPGKPGALVSILPAKVVG----ARLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSVDQVRGMDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTNIVVKGPKGELSRTLRDNVSLSQEGEVLQVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIEPPDGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSITKAQLQEIAQTKLPDLNANDIDAAMKIVAGTAKNMGITVKDTKTLPTLERWYIVDADKRLGRLASEIAVLRGKKKPYTPHLDTGDFVIVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVSTIPGMIQQTYLNVADNSGARKLMCIRILGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVSRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQVIDDTIAASTLEELKSLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEDDYEDYDGSEYDDDYEETEYTDAVIPEEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDY--KLRQYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TPLPP----ASRERERDPRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGETIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYEALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEEEGKHLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGNYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENKCKVGDRVRIQETRPLSKTKRWKITEVLNIKPA------------',\n",
       " 'Nostoc_JACKZS01': 'AKISRRLQLKVEDKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKTTFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKIDVTALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYFAIQVGFGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDAERNLLLIKGAIPGKPGALVSVVPAKVVG----ARLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSVDQVRGMDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVEVAINGTNIVVKGPKGELSRTLRDNVSLSQEGETLHVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIEPPDGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSITTAQLQEIAQTKLPDLNANDIDAAMKIVAGTAKNMGITVKDTKTLPTLERWYIVDADKRLGRLASEVAVLRGKKKPYTPHLDTGDFVIIINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVSTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVSRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQVIDDTIAASTLEELKSLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAEECANLVWLTNLAHLRIRVFPDWEIHLDGGLTRIGVEKYMPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEDDYEDYDGSEYDDDYEETEYTDAVIPEEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDY--KLRQYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEP----DPLPP----ASRDRERDPRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGMHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYEALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEEEGKHLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGNYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENKCKVGDRVRIQETRPLSKTKRWKITEVLNVKPA------------',\n",
       " 'Nostoc_JACKZR01': 'AKISRRLQLKVEDKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKTTFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKIDVTALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYFAIQVGFGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDAERNLLLIKGAIPGKPGALVSVVPAKVVG----ARLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSVDQVRGMDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVEVAINGTNIVVKGPKGELSRTLRDNVSLSQEGETLHVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIEPPDGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSITTAQLQEIAQTKLPDLNANDIDAAMKIVAGTAKNMGITVKDTKTLPTLERWYIVDADKRLGRLASEVAVLRGKKKPYTPHLDTGDFVIIINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVSTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVSRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQVIDDTIAASTLEELKSLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAEECANLVWLTNLAHLRIRVFPDWEIHLDGGLTRIGVEKYMPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEDDYEDYDGSEYDDDYEETEYTDAVIPEEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDY--KLRQYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEP----DPLPP----ASRDRERDPRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGMHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYEALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEEEGKHLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGNYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENKCKVGDRVRIQETRPLSKTKRWKITEVLNVKPA------------',\n",
       " 'Nostoc_JABXKH01': 'AKISRRLQLKVEDKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIGAIEFKAGKLEFRADRTGIVHVFGKTTFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKIDVTALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYFAIQVGFGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQEIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDAERNLLLIKGAIPGKPGALVSVVPAKIVG----ARLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSVDQVRGMDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAINGTNIVVKGPKGELSRTLRDNVSLSQEGEILHVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIEPPEGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSITQAQLQEIAQTKLPDLNANDIDAAMKIVAGTAKNMGITVKDTKTLPTLERWYIVDADKRLGRLASEIAVLRGKKKPYTPHLDTGDFVIVINVEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGSHPHAQKPKLVSTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVSRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQVIDDTIAASTLEDLKSLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEDDYEDYDGSEYDDDYEETEYTDAVIPEEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDY--KLRQYIEQK-LGRQAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEP----DPLPP----ASRDRERDPRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGRVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPASPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYEALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEAEEEGKHLVISLKYKGKTRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGNYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENKCKVGDRVRIQETRPLSKTKRWKITEVLNVKPA------------',\n",
       " 'Nostoc_JABXKG01': 'AKISRRLQLKVEDKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIGAIEFKAGKLEFRADRTGIVHVFGKTTFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKIDVTALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYFAIQVGFGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQEIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDAERNLLLIKGAIPGKPGALVSVVPAKIVG----ARLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSVDQVRGMDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAINGTNIVVKGPKGELSRTLRDNVSLSQEGEILHVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIEPPEGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSITQAQLQEIAQTKLPDLNANDIDAAMKIVAGTAKNMGITVKDTKTLPTLERWYIVDADKRLGRLASEIAVLRGKKKPYTPHLDTGDFVIVINVEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGSHPHAQKPKLVSTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVSRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQVIDDTIAASTLEDLKSLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEDDYEDYDGSEYDDDYEET-YTDAVIPEEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDY--KLRQYIEQK-LGRQAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEP----DPLPP----ASRDRERDPRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGRVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPASPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYEALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEAEEEGKHLVISLKYKGKTRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGNYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENKCKVGDRVRIQETRPLSKTKRWKITEVLNVKPA------------',\n",
       " 'Nostoc_JABXKF01': 'PKISRRLQLKVEDKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADLVGSEELIDEIQKGLDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKTTFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVSIPVTVIQAGPCTVTQVKTKETDGYFAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDRSSDYALGQEIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDPERNLLLIKGAIPGKPGALVSVVPAKIVG----ARLKYETIPKLDQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQVRGMDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTNIVVKGPKGELSRTLAANVSLSQEGEILQVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLVLNMGYSHQVQIEPPEGIQFAVEGPTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSITKAQLQEIAQTKLPDLNANDIDAAMKIVAGTAKNMGVTVTDTKTLPSLERWYIVDADKRLGRLASEIAVLRGKKKPYTPHLDTGDFVIVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKRLFTKLKVYAGPHPHAQKPKLVSTIPGMIQQTYLNVADNSGARKLMCIRILGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVNRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQIIDDTIAASTVEELKSLSNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAE-----------------------MHLDGGLTRIGVEKYMPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEDDYEDYDGSEYDDDYEETEYTDAVIPDEE---AEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDH--KLRQYIEQK-LGRQAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TPLPP----ASRDREREPREREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHSHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQINELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPASPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKLTRSIAKVLREEGFIEEEEEGKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGTYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQIITGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKITEILNVKPT------------',\n",
       " 'Nostoc_JABXKL01': 'PKISRRLQLKVEEKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADLVGSEELIDEIQKGLDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKTTFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVSIPVTVIQAGPCTVTQVKTKETDGYFAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQEIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDPERNLLLIKGAIPGKPGALVSVVPAKIVG----PRLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQVRGMDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTNIVVKGPKGELSRTLRDNVSLSQEGEILHVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIEPPDGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSITKAQLQEIAQTKLPDLNANDIDAAMKIVAGTAKNMGVTVTDTKTLPTLERWYIVDADKRLGRLASEIAVLRGKKKPYTPHLDTGDFVIVINVEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVSTIPGMIQQTYLNVADNSGARKLMCIRILGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVSRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQIIDDTIAASTVEELKSLCNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEDDYEDYDGSEYDDDYEETEYTDAVIPDEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDY--KLRQYIEQK-LGRQAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TPLPP----ASRDRERDPRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDVPITKSNSIPHPIDGVGGGAKVIMRPASPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLAEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKLTRSIAKVLREEGFIEEEEEGKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGNYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQIITGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKITEVLNVKPT------------',\n",
       " 'Nostoc_JABXKO01': 'PKISRRLHLKVEEKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKTTFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVSIPVTVIQAGPCTVTQVKTKETDGYFAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQEIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDPERNLLLIKGAIPGKPGALVSVVPAKIVG----PRLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQVRGMDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTNIVVKGPKGELSRTLRDNVSLSQEGEILHVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIEPPDGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSITKAQLQEIAQTKLPDLNANDIEAAMKIVAGTAKNMGVTVTDTKTLPTLERWYIVDADKRLGRLASEIAVLRGKKKPYTPHLDTGDFVIVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVNTIPGMIQQTYLNVADNSGARKLMCIRILGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVNRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQLIDDTIAASTVEELKSLCNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAE---------------------MEIHLDGGLTRIGVENYMPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEDDYEDYDGSEYDDDYEETEYTDAVIPEEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDY--KLRQYIEQK-LGRQAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEP----DPLPP----ASRDRERDPRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVMMRPASPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKLTRSIAKVLREEGFIEEEEEGKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGSGQLVNGKGNYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDIRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKITEVLNVKPT------------',\n",
       " 'Nostoc_JABXKN01': 'PKISRRLQLKVEEKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKTTFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVSIPVTVIQAGPCTVTQVKTKETDGYFAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQEIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDPERNLLLIKGAIPGKPGALVSVVPAKIVG----PRLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQVRGMDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTNIVVKGPKGELSRTLRDNVSLSQEGEILHVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIEPPDGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSITKAQLQEIAQTKLPDLNANDIEAAMKIVAGTAKNMGVTVTDTKTLPTLERWYIVDADKRLGRLASEIAVLRGKKKPYTPHLDTGDFVIVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVSTIPGMIQQTYLNVADNSGARKLMCIRILGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVNRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQLIDDTIAASTVEELKSLCNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAE---------------------MEIHLDGGLTRIGVENYMPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEDDYEDYDGSEYDDDYEETEYTDAVIPEEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDY--KLRQYIEQK-LGRQAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEP----DPLPP----ASRDRERDPRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVMMRPASPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKLTRSIAKVLREEGFIEEEEEGKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGNYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDIRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKITEVLNVKPT------------',\n",
       " 'Nostocales_JADEXV01': 'PKISRRLQLKVEEKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADLVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKTTFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVSIPVTVIQAGPCTVTQVKTKETDGYFAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQEIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDPERNLLLIKGAIPGKPGALVSVVPAKTVG----PRLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQVRGMDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTNIVVKGPKGELSRTLRDNVSLSQEGEILHVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIEPPDGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSITKAQLQEIAQTKLPDLNANDIEAAMKIVAGTAKNMGVTVTDTKTLPTLERWYIVDADKRLGRLASEIAVLRGKKKPYTPHLDTGDFVIVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVNTIPGMIQQTYLNVADNSGARKLMCIRILGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVNRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQLIDDTIAASTVEELKSLCNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAEEYVNLVWLTNLAHLRIRVFPDWEIHLDGGLTRIGVEKYMPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEDDYEDYDGSEYDDDYEETEYTDAVIPDEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDY--KLRQYIEQK-LGRQAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEP----DPLPP----ASRDRERDPRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLAEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKLTRSIAKVLREEGFIEEEEEGKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGNYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKITEVLNVKPT------------',\n",
       " 'Nostoc_LXQE01': 'PKISRRLQLKVEEKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKTTFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVSIPVTVIQAGPCTVTQVKTKETDGYFAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQEIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDPERNLLLIKGAIPGKPGALVSVVPAKIVG----PRLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQVRGMDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTNIVVKGPKGELSRTLRDNVSLSQEGEILHVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIEPPDGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSITKAQLQEIAQTKLPDLNANDIEAAMKIVAGTAKNMGVTVTDTKTLPTLERWYIVDADKRLGRLASEIAVLRGKKKPYTPHLDTGDFVIVINVEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVSTIPGMIQQTYLNVADNSGARKLMCIRILGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVNRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQLIDDTIAASTVEELKSLCNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAE---------------------MEIHLDGGLTRIGVENYMPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEDDYEDYDGSEYDDDYEETEYTDAVIPEEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDY--KLRQYIEQK-LGRQAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEP----DPLPP----ANRDRERDPRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVMMRPASPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKLTRSIAKVLREEGFIEEEEEGKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGNYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDIRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKITEVLNVKPT------------',\n",
       " 'Nostoc_JAIVFR01': 'PKISRRLQLKVEDKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLGAIEFKAGKLEFRADRTGIVHVFGKATFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYFAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTESSGDYALGQEIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDTERNLLLIKGAIPGKPGALVSIVPAKTVG----ARLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEISITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQVRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGANIVVKGPKGELSRTLTPNVSLSQEGDILQVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIEPPEGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIQKAAKIEKGSNEPNKKKVGSISKAQLQEIAQTKLPDLNANDIDAAMKIVAGTAKNMGVTITDTKTLPSLERWYIVDADKRLGRLASEIAVLRGKKKPYTPHLDTGDFVIVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVNRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQVIDDTIAASTVEELKSLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYDGSEYDEDYEEGEYTDAIIPDEE---TDEMGQKIHPVGFRLGITQE----------HQSRWFAVPDRYPELLQEDH--KLRQYIDQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TPPPP-ASRERERDPRDRGDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEEIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNGEREWVALQVNELLVVEYYSRQAIKRAEIMAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVMMRPASPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIESLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEEEGKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGTYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQIITGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENKCKVGDRVRIQETRPLSKTKRWKITEILNVKPT------------',\n",
       " 'Nostoc_WBKL01': 'PKISRRLQLKVEDKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLGAIEFKAGKLEFRADRTGIVHVFGKTTFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYFAIQVGFGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSGDYALGQQIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTIVRVDAERNLLLIKGAIPGKPGALVSILPAKVVG----ARLKYETIPKLNQFYTNVHQVPKLVKVTINRGLGEAAQNAKSLEASINEIAVTGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFFDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQVRGMDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTNIVVKGPKGELSRTLAANISLSQEGEILQVNRRDETRTSK--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQLQGRNLVLNMGYSHQVQIEPPEGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKVEKGSNEPNKKKVGSISKAQLQEIAQTKLPDLNANDIDAAMKIVAGTAKNMGVTITDTKTLPSLERWYIVDADKRLGRLASEIAVLRGKKKPYTPHLDTGDFVIVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVSTIPGMIQQTYLNVADNSGARKLMCIRILGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVSRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESNRRHRRVRKVGSERPRLAVFRSNHIYAQLIDDTISASTVEDLKSLCNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATNTTVKAARFIRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEEDYEDYDGGEYDEDYEETEYTDAVIPDEE---TEEMGQKIHPVGFRLGITHE----------HQSRWFAVPDRYPELLQEDH--KLRQYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQSIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TPLPP-----ATRDPRDRGDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLAEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQYSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEEEGKHLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKNAVARVRLVPGTGQLVNGKGNYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPSGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRAQIITGVNPDTRVKDLSDDVLRIENYQVEGDLRRELNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYISSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENKCKVGDRVRIQETRPLSKTKRWKITEVLNVKPT------------',\n",
       " 'Fremyella_diplosiphon_UTEX_481': 'RKISRRLQLKVEDRYPLEALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADVVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSSTMGPSIKVDISALRDLKVSVGILGTKLGMTQIFDDAGVAIPVTVVQAGPCTVTQVKTKQTDGYAAIQVGFSEVKPKALNKPLLGHLAKSSAPALRHLNEYRLDTASDYALGQEIKADIFSAGQIVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVNIVPAKQVG-K--ARLKYETIPKLNQFYTNVHQVPKLVKITVNRGLGEAASNAKALDASLNEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTKVVVKGPKGELSRELPAHVSVSQEGETLLVTRRDETRTSR--QMHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIEPPDGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIDAAMKIVEGTAKNMGVTIADMKTLPALERWYIVDADQRLGRLASEIAVLRGKKKAYTPHLDTGDFVIVINAEKIVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERIVEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVNTIPGVIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMVKKSDVVEAVIVRTRINRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESTRRHRRIRKVGSERPRLAIFRSNHIYAQVIDDTLSASTVEELKSLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRIQALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGRSYREALIILEFMPYRACDPVLKVLRSAAANAEHNGLDRALVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAA--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYTYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWTTIKTRVDRLKDLERREETGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECQKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEDDYEDYEGGEEDYDYEET-LTGSYASEED---EEEVGQKIHPVGFRLGITQE----------HQSRWFAVPDKYPELLQEDY--KLRQYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQELLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE---AAPAPT------------SREREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAVRLEEKQKLRLNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEEIAVRNKSRKLAENLQYPGLANLPHLEFDKKLGKVNGEREWVALQVNELLVVEYYSRQA------MAT-GRRKA-NRAK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--VPIENLYI-----------------MIMRRVMRGKKSLARIVYDALIERTGPALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKLTRSIAKVLRDEGFIEEEGEGKRLVIALKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWISRAVYWGTGRRKSAVARVRLIPGSGQLVNGKGNYFQFNAYLIKGPLETLGLENEYDILVKAGGGLTGQADSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKVARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRIENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQELISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGHRKRLLSYVQSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTRRYKAHDEDNQCKVGDRVRIQETRPLSKTKRWQVTEILNAKATT-----------',\n",
       " 'Chlorogloeopsis_sp._PCC_7702': 'RKKSRRFQLKVEDRYPLEALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQIRVAVIARGEKVEAAGADVVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLTAIEFKAGKVEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRSYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFSDEGVAIPVTVIQVGPCTVTQIKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPPLRYLREYRTDNASDYSLGQQIKADIFSVGQIVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRAPGRLGGSRVTIKKLTVVRVDPERNILLIKGAVPGKPGGLLNIVPAKKVG-K--MRLKYETIPKLKQFYTNIHQVPKLVKITVNRGLGEAAQNAKALESSLNEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIAVTLRGERMYAFLDRLINLSLPRIRDFRGVSPKSFDGRGNYTLGIREQLIFPEVDYDSIDQIRGMDISIITTAKTDEEGRALLKEMGMPFRQMSRIGKRPITIPPKVQVTVDGTKVSVKGPKGELSRQLPSNVTISQQGEILTVNRCDDSRTSR--QMHGLCRTLVANMIEGVSKGFERRLEIQGVGYRAQLQGRNLILNVGYSHQVQIAPPDGIQFAVENNTNVIVSGYDKEVVGNTAARIRAVRPPEPYKGKGIRYAGELVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVVPVEISVYEDRSFTFVLKTPPASVLISKAAKIEKGSSEPNKKKVGSITREQLRQIAQTKMPDLNANDIDAAMKIIEGTAKNMGVTIVDEKTLPSLTRWYVVDANQCLGRLASEIAILRGKNKPYTPHLDTGDFVIVVNAEKVVTGKKTQKIYRRHSGRPGGMKTETFDLQRLPERIIEAVKGMLPKNRLGRQLFGKLKVYAGPHPHAQKPQIISSILGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMVKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRIESQRRHRRIRKVGSERPRLAVFRSHHIYAQVIDDTLAASSLEELRSLGNQASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALAEAAREAGLDFMATDTIVRAARYIRSPKVRRVLDQIRGRSYREALILLEFMPYRACEPVLTLLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITIAVAPA--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMAPYIY-TARNGVHIIDLVQTAQLMEEAYAYMRSQAEQGKKFLFVGTKRQAAGIIAQEATRC-GAHYINQRWLGGMLTNWTTIKTRVERLKELERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDLVDIPIPANDDAIRSIKLIIGKLADAIYEGRH--GQLEVEEEYEEYEGAEEDYEEEES-YTDSLIPDDE---DEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRQYIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIEALR----------------------TGLQQMLG--------NNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTV-------------------------------------------YGILGVKVWVF-----------------KGEIIPGQDE----TPPPT-------------STREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRYNYGLTERQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVSHGHVTVNGRVVNIPSYECRPGEVIGVRDKSRKLVENLQYPGLANLPHLEFDKALGKVNGEREWVALNINELLVVEYYSRQA------MAT-NRRKA-NRAK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPVNGTGGGAKVLMRPAAPGTGVIAGGAVRTVLELAGVKNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--IPVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYEALIQKTGDPLEFEKAVRNTTPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRQRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRLRNARHQTTVPSTKMTRSIAKVLRDEGFIDQEGEGKRLVISLKYKGKNRPIIALKRISKPGLRVYSNKLPRVLGGIGVAIISTSNGIMTDREARRKNLGGEVLCYIWMGRAIYWGTGRRKSAVARVRLIPGTGQLVNGKGDYFQFNPYLIKAPLETLGLENEYDIVVKAGGGLTGQSDAIRLGVARALCQLDPDNRPLKEGYLTRDPRSKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGIAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSKILTGVNPDTRVKDLSDDILRIENYQVEGDLRREMNIKRLDIGYRGRRHRLGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNERYQLIRLGIRGMAVKQRVGLVVSDKMQKTVVVAVENRAPHPKYKKIIVKTRRYKAHDEENRCRVGDRVRIQETRPLSKTKRWQVIEILTTKNS------------',\n",
       " 'Nostoc_sp._PCC_7524': 'QKVSRRLRLKVEDKYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEASGADVVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRSYVSATMGPSIRVDVAALRDYKMSVGILGTKLGMTQIFDEAGVAIPVTVVQVGPCVVTQVKTQQTDGYAAIQVGYGAVKPKALNKPLLGHLAKSSAPALRHLQEYHADNSSEYALGQEIKADIFSAGQIVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDTERNLLLIKGSVPGKPGALLNIVPAKQVG-K--ARLKYETIPKLQQFYTNVHQVPKLVKVSVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLTLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEIEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITVPAKVQVTVDGSKVIVKGPKGELSRDLPTHVTVAQDGQTLLVTRKDDSRTSR--QMHGLSRTLVANMVEGVSQGFQKRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPDGIQFAVENNTNVIVSGYDKEVVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGSITTAQLREIAQTKLPDLNANDIDAAMKIVAGTAKNMGVTVQDMKTLPTLERWYVVDADKRLGRLASEIAILRGKNKAYTPHLDTGDFVIVVNAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRLPERIVEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGVGDIIAVVKDAPNMVKKSDVVEAVIVRTRINRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRKESQRRHRRIRKVGSERPRLSVFRSNHIYAQVIDDTLAASTVEELKSLGTEASIGKLIARSLEKGIKVVFDRGGNLYHGRIKALAEAAREAGLDFMATDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRATEPVLKLLRSAAANAEHNGLDRTLVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITLAVAAN--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEDAYSYMRSQAEQGKKFLFVGTKRQAAGIIAQEAARC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREENGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECQKL--AIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEEDYEFDES-YSD-LQSDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPDRYPELLQEDH--KLRKYIEQK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIEALR----------------------TGLQELLG--------NNRQIRINVVE-------------------------------------VQKVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GIQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TPAQP-------------ATREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRCNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRMGMAPTIPAARQLVNHGHVTVNGRVVDIASYQTRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-GRRKA-NRAK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IAIEKLYIMSRRRRPVSDYNSRLVSMIIRRVMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGITLALRWLVQFSRSRGRTMARLAELLDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIARVLQEEGFIETEEEGKRLVISLKYKGKNHPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSSVARVRLVPGSGQLVNGKGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPNGKAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLTRSQILTGVNPDTRVKELSDDVLRIENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTARINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYIQSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEENKCKVGDRVRIQETRPLSKTKRWQVTEILNTKATT-----------',\n",
       " 'Nostoc_sp._PCC_7107': 'KKVSRRLRLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDQIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKAGFAEDLLNLKALQETIDRNRPSGAKGRYWRTYISATMGPAIKVDITALRDLKVSVGILGTKLGMTQVFDEAGAAIPVTVIQAGPCTVTQVKTKQTDGYAAIQLGYGEVKPKSLNKPLLGHLAKSSAPALRHLSEFRTDNSGEYTLGQEIKADIFSAGQIVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGTRVTISKLTVVRVDAERNLLLIKGAIPGKPGALVTVVPAKKVG-K--ARLKYETIPKLTQFYTNVHQVPKVIKVTVNRGLGEAAQNAKSLEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLTLPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQITIDGTKVVVKGPKGELSRDLPVHVTVSQEGETLQVVRKDDSRTSR--QMHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLVLNMGFSHQVNIEPPDGIQFAVEGNTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGGKGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGSITQAQLREIAQTKLPDLNANDIDAAMNIVAGTARNMGVTITEMKTLPSLEHWYIVDADQRLGRLASEIAILRGKKKPYTPHIDTGDFVIVINAEKIVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERIVEAIKGMLPKNSLGKHLFTKLKVYAGPHPHAQQPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRIHRDSGTIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRKVGSERPRLAVFRSNHIYAQVIDDTLAASTLDEIKALGNEASVGKLIARALEKGIKVVFDRGGNLYHGRIKTLAEAAREAGLDFMATNTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACDPVLKLIRSAAANAEHNGLDRALVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAT--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVERLKDLERREENGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECQKL--SIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEEEYEDYEGGEEDYEYDESEYSDSLIPDEE---EEEMGQKIHPVGFRLGITQE----------HQSRWYADPERYPELLQEDH--KLRKYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLISEYIAQQLERRVSFRRVVRQAIQRAQRA-GIQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TQPQP-------------ATREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRMNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHCHVTVNGRVVNIASYQCRPGEVIAVRDKSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-GRRKT-NRAK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLAEVAEDRG--IAIDKLYIMSRRRRPVPDYNSRLVSMMIRRIMRGKKSLARIVYEALIERTGAALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANESGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLQEEGFIESEGEGKRLVVALKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGSGQLVNGKGDYFQFNAYLIKAPLETLGLESEYDILVKAGGGLTGQADSIRLGVARALCQLDPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGLAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRIENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIITFQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGHRKRLLAYLQSEKYQLIRLGIRGMAIKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVKTRRYKAHDEENKCKVGDRVRIQETRPLSKTKRWQVTEILNTKNS------------',\n",
       " 'Nostoc_piscinale_CENA21': 'KKVSRRLRLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKAAFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDITALRDLKVSVGILGTKLGMTQIFDEAGVAIPVTVVQAGPCTVTQVKTKQTDGYAAIQLGYGEVKPKALNKPVLGHLAKSSAPALRHLSEFRTDNSGEYTLGQEIKADIFSAGQIVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGTRVTISKLTVVRVDAERNLLLIKGAIPGKPGSLVSVVPAKKVG-K--ARLKYETIPKLNQFYTNIHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGAKMYAFLDRLISLTLPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPSKVQVTIDGTKVVVKGPKGELSRDLPVQVTVNQEGETLQVVRKDDSRTSR--QMHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLVLNMGFSHQVHIEPPDGIQFAVENNTNVIVSGYDKEVVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGGKGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGSITKAQLREIAQTKLPDLNANDIDAAMNIVAGTARNMGVTITEMKTLPALEHWYIVDADLRLGRLATEIAILRGKKKPYTPHIDTGDFVIVINAEKIVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPQLISTIPGVIQQTYLNVADNSGARKLMCIRVLGAGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRIRRDSGTIRFDDNAAVIINEGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTLDELKALGNEASVGKLIARALEKGIKVVFDRGGNLYHGRIKTLADAAREAGLDFMATNTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKLIRSAAANAEHNGLDRALVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAT--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEDAYTYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWSTIKTRVERLKDLERREENGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECQKL--SIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYDDYEGAEEDYDYDESDYSDSLLPDDE--DEDEVGQKIHPVGFRLGITQE----------HQSRWYADPERYPELLQEDY--KLRNYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------GTRQIRINVVE-------------------------------------VQRVDADAYLISEYIAQQLERRVSFRRVVRQAIQRAQRA-GIQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE---TQPQPA-------------ATREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRAYPPGQHGQRKKRSEYAVRLEEKQKLRCNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTVNGRVVNIPSYQCRPGEVIAVRDKSRKLVENLQYPGLANLPHLEFDKKLGKVNGEREWVALQVNELLVVEYYSRQA------MAT-GRRKA-NRAK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLAEVAEDRG--IAIDKLYIMSRRRRPVPDYNSRLVSMMIRRIMRGKKSLARIVYDALIERTGAAIEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLQEEGFIETEGEGKRLVISLKYKGKNRPLIALKRISKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGSGQLVNGKGDYFQFNAYLIKAPLETLGLETEYDILVKAGGGLTGQADSIRLGVARALCQLDPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGLAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKVARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTSVNPDTRVKDLSDDVLRIENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISFQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGHRKRLLAYLQNEKYQLIRLGIRGMAIKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVKTRRYKAHDEENKCKVGDRVRIQETRPLSKTKRWQVTEILNTKNS------------',\n",
       " 'Nostoc_SZNH01': 'KKVSRRLQLKVEEKYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQIRVAVIARGEKVEAAGADIAGSEELINDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTFVSATMGPSIKVDISALRDLKMSVGILGTKLGMTQVFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVSVVPAKRVG----ARLKYETIPKLSQFFTNVHQVPKLVKITLNRGLGEAAQNAKALEASINEIGITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQVRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTKVVVKGPKGELSRNLPTNVMVSQEGEILQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKIVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGTVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVNRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTIAASTVEELKXLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVQALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMTKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEEDYED-EGAEYDEDYEEG-YTDSLIPDDD---EDDMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP------AVRERDRGEREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRAGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQINELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--ITIENLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATRMTRNIAKVLREEGFIEEEGDEKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGQLVNGKGNYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTSVNPDTRVKDLSDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLGYIQSEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKVTEILNVKPT------------',\n",
       " 'Desmonostoc_JACERF01': 'KKVSRRLQLKVEDRYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVAATMGPSIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSGDYALGQQIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGTRVTIRKLTIVRVDPERNLLLIKGAIPGKPGALVSVVPAKKVG-K--ARLKYETIPKLQQFYTNVHQVPKLVKITINRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFLDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQVRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITVPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEILQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIEPPEGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSIERWYIVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRLPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVNRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQIIDDTIAASTVEELKSLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGPEEYANLVWLTNLAHLRIRVFPYWEIRLDGGLTRIGVKKYMPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWTTIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYDGTEYDEDYEEGEYTDTLIPDEE---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDY--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP----AARERDRDRGDREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNGEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDEKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYIWMSRAVYWGTGRRKSAVARVRLVPGTGQLVNGKGNYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKELSDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYIQSEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKITEILNVKPT------------',\n",
       " 'Nostoc_JACJSJ01': 'KKVSRRLQLKVEDRYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVAATMGPSIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSSGDYALGQQIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGTRVTIRKLTIVRVDPERNLLLIKGAIPGKPGALVSVVPAKKVG-K--ARLKYETIPKLQQFYTNVHQVPKLVKITINRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRAERMYAFLDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQVRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITVPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEILQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIEPPEGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSIERWYIVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVINAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRLPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVNRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQIIDDTIAASTVEELKSLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGPE-----------------MFPYWEIRLDGGLTRIGVKKYMPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWTTIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYDGTEYDEDYEEGEYTDTLIPDEE---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDY--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP----AARERDRDRGDREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEVIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNGEREWVALQVNELLVVEYYSRQA------MAT-ERKSRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIQNLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDEKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYIWMSRAVYWGTGRRKSAVARVRLVPGTGQLVNGKGNYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKELSDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYIQSEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKITEILNVKPT------------',\n",
       " 'Nostoc_JACJTU01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADLAGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKATFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVAATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSPSDYALGQQIKADIFSAGEIVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDPERNLLLIKGAIPGKPGALVNVVPAKKVG-K--ARLKYETIPKLQQFFTNVHQVPKLVKITINRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQVRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIDAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVNRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTIAASTVEELKSLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYDGTEYDEEYEESDYTDTLIPDDD--EEEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP---AARDRERGDRGDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRAGETIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNGEREWVALQVNELLVVEYYSRQA------MAT-ERKGRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDEKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKELSDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKITEILNVKPA------------',\n",
       " 'Nostoc_JACJTK01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADLAGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKATFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVAATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLNEYHTDSPSDYALGQQIKADIFSAGEIVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDPERNLLLIKGAIPGKPGALVNVVPAKKVG-K--ARLKYETIPKLQQFFTNVHQVPKLVKITINRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQVRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIDAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVNRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTIAASTVEELKSLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYDGTEYDEEYEESDYTDTLIPDDD--EEEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP---AARDRERGDRGDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRAGETIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNGEREWVALQVNELLVVEYYSRQA------MAT-ERKGRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDEKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKELSDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKITEILNVKPA------------',\n",
       " 'Nostoc_LWTK01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQVGYAEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQIVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDPERNLLLIKGAIPGKPGALVSVVPAKQVG-K--ARLKYETIPKLSQFFTNVHQVPKLVKITVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLVSLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAILRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYRRHSGRPGGMKTETFKLQRLPERILEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVNRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTIAASTVEELKSLGNDASVGKLIARSLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEEDYDDYDGAEYDEEYEESDYTDTLIPDDD--EEEEMGQKIHPVGFRLGITQE----------HQSRWFAVPDRYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP-AAR-DRGDRGDR-DRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRAGETIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNGEREWVALQVNELLVVEYYSRQA------MAT-ERKGRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDEKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKELSDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKITEILNVKPT------------',\n",
       " 'Nostoc_JACJTC01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVSVVPAKQVG-K--ARLKYETIPKLSQFFTNVHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPISIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLVLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNIAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVKRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRSLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEYEEDYEEGDYTDSLIPDDD---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP-AARSDRGDRSDRGDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEEIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKTRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMMIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDDKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLTEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKVTEILNVKPT------------',\n",
       " 'Nostoc_VRZE01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVSVVPAKQVG-K--ARLKYETIPKLSQFFTNVHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPISIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLVLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNIAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVKRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRSLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEYEEDYEEGDYTDSLIPDDD---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP-AARSDRGDRSDRGDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEEIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKTRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMMIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDDKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLTEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKVTEILNVKPT------------',\n",
       " 'Nostoc_LAHC01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQIGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVSVVPAKQVG-K--ARLKYETIPKLSQFFTNVHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPISIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLLLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNIAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVKRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRSLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEYEEDYEEGDYTDTLIPDDD---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP-AARSDRGDRSDRGDREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEEIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKTRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMMIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDDKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLNDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLTEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKVTEILNVKPT------------',\n",
       " 'Nostoc_LAHB01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQIGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVSVVPAKQVG-K--ARLKYETIPKLSQFFTNVHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPISIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLLLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNIAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKK---------MLEKPTQHRQWAPP------------------------------------------WVNMAKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVKRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRSLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEYEEDYEEGDYTDTLIPDDD---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP-AARSDRGDRSDRGDREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEEIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKTRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMMIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDDKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLNDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLTEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKVTEILNVKPT------------',\n",
       " 'Nostoc_LAHI01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQIGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVSVVPAKQVG-K--ARLKYETIPKLSQFFTNVHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPISIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLLLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNIAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVKRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRSLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEYEEDYEEGDYTDTLIPDDD---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP-AARSDRGDRSDRGDREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEEIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKTRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMMIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDDKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLNDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLTEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKVTEILNVKPT------------',\n",
       " 'Nostoc_LAHE01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQIGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVSVVPAKQVG-K--ARLKYETIPKLSQFFTNVHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPISIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLLLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNIAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVKRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRSLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEYEEDYEEGDYTDTLIPDDD---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP-AARSDRGDRSDRGDREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEEIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKTRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMMIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDDKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLNDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLTEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKVTEILNVKPT------------',\n",
       " 'Nostoc_LAGY01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQIGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVSVVPAKQVG-K--ARLKYETIPKLSQFFTNVHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPISIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLLLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNIAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKXALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVKRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRSLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEYEEDYEEGDYTDTLIPDDD---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP-AARSDRGDRSDRGDREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEEIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKTRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMMIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDDKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLNDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLTEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKVTEILNVKPT------------',\n",
       " 'Nostoc_LAGX01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQIGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVSVVPAKQVG-K--ARLKYETIPKLSQFFTNVHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPISIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLLLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNIAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVKRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRSLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEYEEDYEEGDYTDTLIPDDD---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP-AARSDRGDRSDRGDREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEEIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKTRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMMIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDDKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLNDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLTEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKVTEILNVKPT------------',\n",
       " 'Nostoc_LAHD01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQIGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVSVVPAKQVG-K--ARLKYETIPKLSQFFTNVHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPISIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLLLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNIAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVKRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRSLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEYEEDYEEGDYTDTLIPDDD---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP-AARSDRGDRSDRGDREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEEIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKTRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMMIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDDKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLNDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLTEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKVTEILNVKPT------------',\n",
       " 'Nostoc_LAHH01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQIGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVSVVPAKQVG-K--ARLKYETIPKLSQFFTNVHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPISIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLLLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNIAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVKRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRSLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEYEEDYEEGDYTDTLIPDDD---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP-AARSDRGDRSDRGDREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEEIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKTRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMMIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDDKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLNDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLTEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKVTEILNVKPT------------',\n",
       " 'Nostoc_LAHG01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQIGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVSVVPAKQVG-K--ARLKYETIPKLSQFFTNVHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPISIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLLLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNIAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKK---------------------------------------------------MVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVKRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRSLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEYEEDYEEGDYTDTLIPDDD---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP-AARSDRGDRSDRGDREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEEIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKTRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMMIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDDKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLNDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLTEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKVTEILNVKPT------------',\n",
       " 'Nostoc_LAHF01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQIGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVSVVPAKQVG-K--ARLKYETIPKLSQFFTNVHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPISIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLLLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNIAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKK-----------------------MGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVKRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRSLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEYEEDYEEGDYTDTLIPDDD---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP-AARSDRGDRSDRGDREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEEIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKTRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMMIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDDKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLNDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLTEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKVTEILNVKPT------------',\n",
       " 'Nostoc_LAGZ01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQIGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVSVVPAKQVG-K--ARLKYETIPKLSQFFTNVHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPISIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLLLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNIAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVKRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRSLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEYEEDYEEGDYTDTLIPDDD---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP-AARSDRGDRSDRGDREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEEIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKTRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMMIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDDKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLNDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLTEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKVTEILNVKPT------------',\n",
       " 'Nostoc_LAHJ01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQIGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVSVVPAKQVG-K--ARLKYETIPKLSQFFTNVHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPISIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLLLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNIAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIXAAMKXVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVKRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRSLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEYEEDYEEGDYTDTLIPDDD---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP-AARSDRGDRSDRGDREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEEIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKTRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMMIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDDKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLNDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLTEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKVTEILNVKPT------------',\n",
       " 'Nostoc_LAHA01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQIGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVSVVPAKQVG-K--ARLKYETIPKLSQFFTNVHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPISIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLLLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNIAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVKRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRSLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEYEEDYEEGDYTDTLIPDDD---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP-AARSDRGDRSDRGDREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEEIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKTRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMMIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDDKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLNDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLTEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKVTEILNVKPT------------',\n",
       " 'Nostoc_JACKZU01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDDIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPAIKVDISALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYAAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGTRVTIRKLTVVRVDAERNLLLIKGAIPGKPGALVSVVPAKQVG-K--ARLKYETIPKLSQFFTNVHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLSLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGLDISIITTAKNDEEGRALLKELGMPFRQMSRIGKRPISIPAKVQVAIDGTKVVVKGPKGELSRNLPANVIVSQEGEVLQVTRRDDTRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQIQGRNLVLNMGYSHQVQIEPPEGVQFAVEGTTNVIVSGYDKEIVGNIAAKIRAVRAPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTAKNMGVTVTDIKTLPSLERWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLISTIPGMIQQTYLNVADNSGARKLMCIRVLGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVKRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNEASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRSLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAE--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMEDAYSYMRTQAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEYEEDYEEGDYTDTLIPDDD---EEEMGQKIHPVGFRLGITQE----------HQSRWFAVPERYPELLQEDH--KLRTYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQGLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TAPPP-AARSDRGDRSDRGDREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGLAPTIPAARQLVNHGHITVNGRVVNIASYQCRPGEEIAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-ERKTRTKRAK---KEETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLSEVAEDRG--IPIENLYIMSRRRRPVSDYNSRLVSMMIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEGDDKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGKLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLNDDVLRIENYEVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLTEHLQNKDSSRRGLLKLIGQRKRLLAYIQNEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTQRYKVHDEENQCKVGDRVRIQETRPLSKTKRWKVTEILNVKPT------------',\n",
       " 'Nostoc_WVID01': 'KKLSRRLQLKVEDRYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADLVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVAATMGPSIKLDINALRDFKMSVGILGTKLGMTQIFDAAGVSIPVTVIQAGPCTVTQIKTKQTDGYSAIQVGYGEVKPKALNSPLLGHLAKSSAPALRHLNEYHTDSSSDYALGQQIKADIFSEGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLIVVKVDSERNLLLIKGAIPGKPGALVNVLPAKRVG-K--ARLKYETIPKLNQFYTNVHQVPKLVKVTVNRGLGEAAQNAKSLEASINEIAIAGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFVDRLISLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKNDEEGRALLKEMGMPFRQMSRIGKRPITVPTKVQVAIDGTKVVVKGPKGELSRDLPTAVTVSQEGETLQVTRRDDTKTSR--QMHGLCRTLVANMVEGVSSGFQRRLEIQGVGYRAQVQGRNLILNMGYSHQVQIVPPEGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEEVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIEKGSNEPNKKKVGSITKAQLQEIAQTKLPDLNANDIEAAMKIVAGTAKNMGVAIADMKTLPSLERWYVVDADQRLGRLATEIAILRGKKKAYTPHLDTGDFVIVVNAEKIVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIVEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLVNTIPGMIQQTYLNVADNSGARKLMCIRILGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVNRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELREKNFTKIVSLAPEVLMKLTRKESQRRHRRVRRVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARLLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATETTVKAARFLRSAKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAE-----------------MFPSWRLRLGGGLTRIGVQKYMPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDTEEDYEDYDGGEYDEDYEET-YTDTVIPDEE---E--MGQKIHPVGFRLGITHE----------HQSRWFAVPERYPELLQEDH--KLRQYIEKK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQELLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GIQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TPPPP---------TTRDRDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVVAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNGEREWVALQVNELLVVEYYSRQA------MAT-GRRKT-NRTK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTFAEVAEDRG--IAIEHLYIMSRRKRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRDRGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATRMTRSIAKVLREEGFIEAEEEGKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLIPGNGQLVNGKGTYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSIRLGVARALCQLDPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRIEIGLTYIYGIGLSRSQILTGVNPDIRVKELSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSAEVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKSLLAYIQSEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTRRYKAHDEENKCKVGDRVRIQETRPLSKTKRWQVTEVLNVKSI------------',\n",
       " 'Westiellopsis_JAHHHK01': 'KKLSRRLQLKVEDRYPLEALLLKDTATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADLVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVAATMGPSIKLDINALRDFKMSVGILGTKLGMTQIFDAAGVAIPVTVIQAGPCTVTQVKTKQTDGYTAIQVGYHEVKPKALNKPLLGHLAKSSAPALRHLNEYHTDGSGDYTLGQQIKADIFSEGQIVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGKRITIRKLTVVKVDAERNLLLIKGAIPGKPGALVNVMPAKQVG-K--ARLKYETIPKLNQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASINEIAIAGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFVDRLISLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKNDEEGRALLKEMGMPFRQMSRIGKRPITVPTKVQVAIDGTKVVVKGPKGELSRDLPNAVTVSQEGEILQITRRDDTKTSR--QMHGLCRTLVANMVEGVSSGFQRRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIVPPEGIQFAVEGTTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEEVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIEKGSNEPNKKKVGSITKAQLQEIAQTKLPDLNANDIEAAMKIVAGTAKNMGVGIADMKTLPSLERWYVVDADQRLGRLASEIAILRGKKKAYTPHLDTGDFVIVVNAEKIITGKKTQKIYHRHSGRPGGMKTETFKLQRLPERIVEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLVSTIPGMIQQTYLNVADNSGARKLMCIRILGGGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRVNRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELREKNFTKIVSLAPEVLMKLTRRESQRRHRRVRKVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNDASVGKLIARLLEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGRSYREALIILEFMPYRATEPILKVLRSAAANAEHNGLDRTLIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAAT-----------------MFPSWELRLGGGLTRIGVQKYMPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAQLMDNAYNYMRSHAEQGKKFLFVGTKRQAAGIIAQEASRC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREETGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDIVVIVDQRREYNAVQECQKL--NIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEEDYEDYDGSEYDEDYDET-YTDTVIPDEE---EE-PGQEETPPGQKFGTHPDGETVTSKPKTKKERDILFPNPYQPPVKQDFMDKIDAYISKE-IEIVTESQTNIVRIPMLIYVNSIDWDSRDTPDFQSTFRMGQIVGCFGSWDTVEALQADPSVISVEASKPGLGWESNSQNLSLDSNTDSSAYSTISSIKADTIHQIPEKGDKALIAIIDSGIDLLHKTFLDSTENSTRILAIWDQTDDTGPSPNIG-------QINQYIRDNIVPDQLGRDPNGHGTHVASIAAGKPVGEFCGGVAPEAKIIVVKIDGAALQYIKYVAAQ--EKLPVVVNVSQGINAAAHD-GREGISIPQPGYLVVKSAGNERNTKLHTRINVLPRQRHFFRWQSTPISRSEDLIQFWFDSSMNLEFCLKTPSGHFNQSICSANLTEIPEPINLPEGNTCHISYKRLHRTNGKSLLQIIINKGEGQYIQRGWWQLEVWNISKNAKNNVIHAWIEIGVRGTVIPGTAE----NVITVSSVDIDDKNSFFNLSQDYGPTL--DNREKPDMSQRFQEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRLNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGIAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVVAVRDRSRKLVENLQYPGLANLPHLEFDKKLGKVNGEREWVALQVNELLVVEYYSRQA------MAT-GRRKT-NRTK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTFAEVAEDRG--IAIEHLYIMSRRKRPVSDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGGALEFERAVRNATPLVEVKARRVGGATYQVPMEVRDRGTTLALRWLVQFSRSRGRTMAKLAELMDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLREEGFIEEEEEGKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLIPGSGQLVNGKGTYFQFNAYLIRAPLETLGLENEYDILVKAGGGLTGQADSIRLGVARALCQLDPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRIEIGLTYIYGIGLSRSQILTGVNPDIRVKELSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSAEVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKSLLAYIQSEKYLLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTRRYKAHDEENKCKVGDRVRIQETRPLSKTKRWQVTEVLNVKSI------------',\n",
       " 'Anabaena_variabilis_ATCC_29413': 'KKVSRRLRLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADVFGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKMLGPRGLMPSPKGGTVTDIAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIRVDINALRDYKMSVGILGTKLGMTQIFDEAGVAIPVTVVQVGPCVVTQVKSKQTDGYAAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLKEYHTDSSSDYALGQEIKADIFSAGELVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGTRVTIRKLTVIRVDAERNLLLIKGAVPGKPGSLLSIVPAKKVG-K--ARLKYETIPKLQQFYTNVHQVPKLVKITVNRGLGEAAQNAKSLEASLTEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFFDRLISLSLPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEIEYDSIDQIRGLDISIITTAKNDEEGRALLKEFGMPFRQMSRIGKRPITVPAKVQVTIDGTKVVVKGPKGELSRELPVNVSVSQEGETLQVTRRDETRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLILNMGYSHQVQIEPPDGIQFAVENNTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGTITTAQLREIAQTKLPDLNANDIDAAMKIVAGTARNMGVTVTDMKTLPSLERWYVVDADKRLGRLASEIAILRGKNKAYTPHLDTGDFVIVVNAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRLPERIVEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRIRRDSGTIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRKVGSERPRLAVFRSNHIYAQVIDDTLSASTVEEVKSLGNEASIGKLIARSLEKGIKVVFDRGGNLYHGRIKALAEAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPVLTLLRSAAANAEHNGLDRALVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITLAVAAG--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAHLMDEAYNYMRSQAEQGKKFLFVGTKRQAAGIIAQEAARC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECQKL--SIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEDDYEYDET-YTDSVIPDDE---EEAMGQKIHPVGFRLGITQE----------HQSRWFAEPSRYPELLQEDH--KLRQYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIEALR----------------------TGLQTLLG--------GNRQIRINVVE-------------------------------------VQRVDADAYLIAEFIAQQLERRVSFRRVVRQAIQRAQKA-GIQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEV----AAPPP----STRDRDRDRGDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRLNYGLTEKQLLRYVRRARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEEIAVRDKSRKLVENLQYPGLANLPHLEFDKKLGKVNGEREWVALQVNELLVVEYYSRQA------MAT-GRRKA-NRTK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLAEVAEDRG--IAIEKLYIMSRRRRPVPDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMARLAELLDAANESGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLQEEGFIESEGDEKRLVIALKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGQLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIINYQVHGTDTGSTDVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGHRKRLLAYLQSEKYQLIRLGIRGMAIKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVKTRRYKAHDEDNKCKVGDRVRIQETRPLSKTKRWQVAEILNTKA-------------',\n",
       " 'Nostoc_sp._PCC_7120_(re-annotation)': 'KKVSRRLRLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADVVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKMLGPRGLMPSPKGGTVTDVAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIRVDINALRDYKVSVGILGTKLGMTQIFDEAGVAIPVTVVQVGPCVVTQVKSKQTDGYAAIQVGYGEVKPKALNRPLLGHLAKSSAPALRHLKEYHTDGSSDYALGQEIKADIFSVGELVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGTRVTIRKLTVIRVDAERNLLLIKGAVPGKPGALLSIVPAKKVG-K--ARLKYETIPKLQQFYTNVHQVPKLVKITVNRGLGEAAQNAKSLEASLTEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFFDRLISLSLPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEIEYDSIDQIRGLDISIITTAKNDEEGRALLKEFGMPFRQMSRIGKRPITVPAKVQVTIDGTKVVVKGPKGELSRELPANVSVSQEGETLQVTRRDETRTSR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIEPPDGIQFVVENNTNVIVSGYDKEIVGNTAAKVRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGTITTAQLREIAQTKLPDLNANDIDAAMKIVAGTARNMGVTVTDMKTLPSLERWYVVDADKRLGRLASEIAILRGKNKAYTPHLDTGDFVIVVNAKKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRLPERIVEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLIHTIPGVIQQTYLNVADNSGARKLMCIRVLGAGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRIRRDSGTIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESERRHRRVRKVGSERPRLAVFRSNHIYAQVIDDTLAASTVEELKSLGNEASIGKLIARSLEKGIKVVFDRGGNLYHGRIKALAEAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGLSYREALIILEFMPYRATEPVLTLLRSAAANAEHNGLDRALVITQAYADGPLKRFQPRAQGRAYQIRKPTCHITLAVAAG--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TSRNGVHIIDLVQTAHLMDEAYNYMRSQAEQGKKFLFVGTKRQAAGIIAQEAARC-GSHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECQKL--SIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGSEEDYDYDET-YADSVIPEDG---EEAVGQKIHPVGFRLGITQE----------HQSRWFAEPSRYPELLQEDH--KLRQYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIEALR----------------------TGLQTLLG--------GNRQIRINVVE-------------------------------------VQRVDADAYLIAEFIAQQLERRVSFRRVVRQAIQRAQKA-GIQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEV----ATPPP----SPRDRDRDRGDRDREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRLNYGLTEKQLLRYVRRARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEEIAVRDKSRKLVENLQYPGLANLPHLEFDKKLGKVNSEREWVALQVNELLVVEYYSRQA------MAT-GRRKA-NRTK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGVGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTLAEVAEDRG--IAIEKLYIMSRRRRPVPDYNSRLVSMIIRRIMRGKKSLARIVYDALIERTGNALEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTTLALRWLVQFSRSRGRTMARLAELLDAANESGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLQEEGFIESEGDEKRLVIALKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNVGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGTGQLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKVARIAGVDLPRDKRVEIGLTYIYGIGLSRSQIITGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIINYQVHGTDTGSTDVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGHRKRLLAYLQSEKYQLIRLGIRGMAIKERVGLVVSDKMQKTVVVAIENRAPHPKYGKIVVKTRRYKAHDEDNKCKVGDRVRIQETRPLSKTKRWQVAEILNTKA-------------',\n",
       " 'Mastigocladus_NGJL01': '--------YKVQDKY------------------------------------------------WQXXXXRGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAXXEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLK---------------------MVPKTIERLVPXX--------------------------------------------------------------------------------------------------------------XXGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQ----------------------------------------------------------------------------MVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMXXXX--------------------------------------------------IRYR-----------------MAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTXXXAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMYALX-------------LGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTXXXXXXXXX-----------------------------------------------------------------MIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDXXXXXXXXX---------------------------------------------XXXXVAL--KQC---------------------------------------GKSPMS--W--------------------------------MGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGIS-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------XXXXD---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------MSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRXXXX----------------------------------------------------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGXXXXXXXW--------------------------------------------QVCVMF---------------------------------------------------------------------------MEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAI-----------------------XXXXMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKR-------------------------------------------------------------------------MRQIEVMVSGPGAGRETAIRALQAGLEIXXIRDITPIPHNGCRPPKRRRV-------------------------MAVGLRTYXXXX--------------------------------------------------VRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Amazonocrinis_JAECZC01': 'KKISRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVALPKGTGQVRVAVIARGEKVEAAGADIAGSEELIDQIQKGMEFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIRVDISALRDLKMSVGILGTKLGMTQVFDEAGVAIPVTVIQAGPCTVTQIKTKQTDGYAAIQVGFGEIKPKALNKPLLGHLAKSSAPALRHLSEYHTDSSSDYALGQEIKADIFSAGQIVDVVGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRMAGRLGGSRVTIRKLTVVRVDAERNLLLIKGAIPGKPGSLVNVLPAKKVG-K--ARLKYETIPKLNQFYTNVHQVPKLVKITVNRGLGEAAQNAKALEASLNEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLISLTLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEIEYDNIDQVRGLDISITTTAKNDEEGRALLKELGMPFRQMSRIGKRPITIPAKVQVVIDGTKVVVKGPKGELSRNLPTAVTISQEGETLQVNRRDDSRTAR--QLHGLSRTLVANMVEGVSQGFQRRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPDGIQFAVENNTNVIVSGYDKEVVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAKIERGSNEPNKKKVGKITRAQLQEIAQTKLPDLNANDIEAAMKIVEGTARNMGVTVTDMKTLPSLERWYVVDADQRLGRLASEVAILRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKVYRRHSGRPGGMKTETFKLQRLPERIVEAVKGMLPKNSLGKQLFTKLKVYAGPHPHAQKPKLVNTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGIGDIIAVVKDAPNMVKKSDVVEAVIVRTRIRRDSGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRVRKVGSERPRLAVFRSNHIYAQVIDDTLAASTVEDLKSLGNEASVGKLIARSLEKGIKVVFDRGGNLYHGRIKALADAAREAGLDFMATDTTVKAARFIRSPKVRRVLDQIRGRSYREALIILEFMPYRATEPVLKLLRSAAANAEHNGLDRALIITQAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVGAD--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEDAYTYMRTQAEQGKKFLFVGTKRQAAGIIAQEAARC-GSHYINQRWLGGMLTNWATIKTRADRLKDLERREESGALDLLPKKEASMLRREMAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECQKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEDYEDYEGAEEDYDYDESDYSDSLIPDDE---DEEMGQKIHPVGFRLGITQE----------HQSRWFAVPDRYPELLQEDH--KLRNYIEQK-LGRLAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIEALR----------------------TGLQELLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GIQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYTTAKTV-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TPPPP-------------AAREREPRRRQQQRRRQQFEDRSNEGMSRYRGPRLRIRRLGDLPGLTRKSARRAYPPGQHGQRKKRSEYAIRLEEKQKLRMNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTVNGRVVNIASYQVRPGEVIAVRERSKKLVENLQYPGLANLPHLEFDKKLGKVNGEREWVALQVNELLVVEYYSRQA------MAT-GRRKA-NRTK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPIDGIGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALSTLRTFAEVAEDRG--IPIENLYIMSRRKRPVPDYNSRLVSMIIRRIMRGKKSLARIVYDALVERTGPALEFERAVRNATPLVEVKARRVGGATYQVPMEVRDRGTTLALRWLVQFSRSRGRTMARLAELLDAANETGNAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPATKMTRSIAKVLRDEGFIEEEGEGKRLVISLKYKGKNRPLIALKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAVYWGTGRRKSAVARVRLVPGSGQLVNGKGDYFQFNAYLIKAPLETLGLENEYDILVKAGGGLTGQADSVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGMAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRIENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSLHLQNKDSSRRGLLKLIGQRKRLLAYIQSEKYQLIRLGIRGMAIKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVQTRRYKAHDEENKCKVGDRVRIQETRPLSKTKRWQVTEILNTKATA-----------',\n",
       " 'Fischerella_ALVT01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADVVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPGLRHLKEYRIDNSGDYSLGQEIKADIFSAGQTVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSIGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDPERNLLLIKGAVPGKPGALVDIVPAKVVG----MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAIAGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLINLALPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTANTDEEGRALLKEMGMPFRQMSRIGKRPITVPPKVQVTLDNSKVVVKGPKGELSRELPVYVTVSQDGEILTVTRRDDSRISR--QMHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLILNMGYSHQVQIAPPEGVQFAVENNTNVIVSGYDKEVVGNTAAKIRSVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGTVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITQEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDLKTLPSLTQWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVINAEKVVTGKKTQKVYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGVIQQTYLNVADNSGARKLMCIRILGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRINRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAVFRSHHIYAQVIDDTLAASTVEEVKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAIDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGFDRALVITKAFADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPA--------------------------------------MAVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKELERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEEYEGAEEDYDYEESDYADSFIPDEE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPGRYPELLQEDH--KLRHFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIEALR----------------------TGLQNLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLISEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----TPPPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEVIGVRDRSKKLVENLQYPGLANLPHLEFDKTLGKVNGEREWIALSINELLVVEYYSRQA------MAT-NRRKT-NRAK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPVDGTGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALIALRTFSEVAEDRG--VAVENLYIMSRRKRPVADYNSRLVSMMIRRVMRGKKSLARIVYEALIARIGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFARQRGRTMAKLAELMDAANETGSAIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRSIAKVLRDEGFIDEEGEGKRLVISLKYKGKNRPIIALKRISKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGSGQLVNGKGDYFQFNPYLLKAPLETLGLENEYDILVKAGGGLTGQSSAIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDILRVENYQVEGDLRRELNIKRLEIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADIQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLAYIQDEKYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEDNRCKIGDRVRIQETRPLSKTKRWQVTEILNTKNS------------',\n",
       " 'Hapalosiphon_LIRN01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADLVGSEELIEEIQKGMNFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDITALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYRIDNSGDYTLGQEIKADIFSAGQKVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTIKKLTVVRVDSDRNLLLIKGAVPGKPGSLVNIVPAKVVG----MRLKYETISNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLINLTLPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKTDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVTFDNTKVVVKGPKGELSRELPANVILSQEGEILTVNRRDESRVSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVMNMGYSHQVQIVPPEGVQFAVENNTNVIVSGYDKEVVGNTAARIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGTVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKRKVGSITKDQLRQIAETKMPDLNANDIEAAMKIIAGTAKNMGVTVADMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKIYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQSYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRINRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTLAASTVEELKSLGNKASVGKLLARALEKGIKVVFDRGGNLYHGRIKALADAAREAGLDFMDVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGFDRALIITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELSKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFSDSFVPDEE---EEEMGQKIHPIGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQELLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWIF-----------------KGEIIPGQEE----APPPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAVRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEAIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRQA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPVDGAGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTALRTFSEVAEDRG--IALENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYAAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRSIAKVLQEEGYIEEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLIPGSGQLVNGKGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRRELNIKRLEIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKLEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQSEHYQLIRLGIRGMAVKERIGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVAEILTTKNT------------',\n",
       " 'Fischerella_ALVX01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADLVGSEELIEEIQKGMNFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDITALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYRIDNSGDYTLGQEIKADIFSAGQKVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTIKKLTVVRVDSDRNLLLIKGAVPGKPGSLVNIVPAKVVG----MRLKYETISNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLINLTLPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKTDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVTFDNAKVVVKGPKGELSRELPANVILSQEGEILTVNRRDESRVSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVMNMGYSHQVQIVPPEGVQFAVENNTNVIVSGYDKEVVGNTAARIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKGGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGTVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKRKVGSITKDQLRQIAETKMPDLNANDIEAAMKIIAGTAKNMGVTVADMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKIYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQSYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRINRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTLAASTVEELKSLGNKASVGKLLARALEKGIKVVFDRGGNLYHGRIKALADAAREAGLDFMDVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGFDRALIITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELSKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFSDSFVPDEE---EEEMGQKIHPIGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQELLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWIF-----------------KGEIIPGQEE----APPPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAVRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTVNGRVVNIASYQCRPGEAIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRQA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPVDGAGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTALRTFSEVAEDRG--IALENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYAAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRSIAKVLQEEGYIEEDGEGKRLVIALKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLIPGSGQLVNGKGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRRELNIKRLEIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKLEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQSEHYQLIRLGIRGMAVKERIGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVAEILTTKNT------------',\n",
       " 'Nostoc_JAHHHW01': 'KRVSRRLQLKVEDRYPLDALLLKQTATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADLVGSEELIEEIQKGMNFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDVTALRDLKMSVGILGTKLGMTQIFDDAGIAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSASALRHLNEYRIDNSSEYTPGQEIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFARGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTIKKLTVVRVDSDRNLLLIKGAVPGKPGSLVNIVPAKVVG----MRLKYETISNLKQFYTNVHQVPKLVKVTINRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRSERMYAFLDRLINLTLPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKTDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVTFDNAKVVVKGPKGELSRELPANVILSQEGEILTVNRRDESRVSR--QLHGLFRTLVSNMVEGVSKGFERRLEIQGVGYRAQVQGRNLVMNMGYSHQVQIVPPEGVQFAVENNTNVIISGYDKEVVGNTAARIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGTVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDRGSSEPNKRKVGSITQEQLRQIAQTKMPDLNANDVEAAMKIIAGTAKNMGVTVTDMKTLPSITHWYVVDADKRLGRLASEIAVLRGKNKPFTPHMDTGDFVIVVNAEKVVTGKKTQKIYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQSYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRINRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELREKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSFGNKASVGKLIARALEKGIKVVFDRGGNLYHGRIKALADAAREAGLDFMDVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAHADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEDAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELSKLQKYLGGIKIMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDAEEEYEDYEGAEEEYDYEESDFSDSFVPDEE---EQEMGQKIHPIGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIDSLR----------------------TGLQNLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWIF-----------------KGEIIPGQEE----TPPPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAVRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTINGRVVNIPSYQCRPGEVIAVRNKSRKLVENLQYPGLANLPHLEFDKKMGKVNSEREWIALSINELLVVEYYSRQA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPVDGAGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTALRTFSEVAEDRG--IPLENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYAAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKITRSIAKVLQEEGYIEEDGEGKRLVIALKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRHNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGKGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRATDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRRELNIKRLEIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKLEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQSEHYQLIRLGIRGMAVKERIGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVAEILTTKNT------------',\n",
       " 'Fischerella_CENA71': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEELIEEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKIDITALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPAVRHLNEYRIDNSGDYTLGQEIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTIKKLTVVRVDSDRNLLLIKGAVPGKPGSLVNIVPAKVVG----MRLKYETISNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLINLTLPRIRDFRGISAKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKTDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVTFDNAKIVVKGPKGELSRELPANVILSQEGEILTVNRRDESRVSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVMNMGYSHQVQIVPPEGVQFAVENNTNVIVSGYDKEVVGNTAAKIRAVRPPEPYKGKGIRYAGEIVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGTVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKRQVGSITQEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKTQKIYHRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQSYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMVKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTLAASTVEELKSLGNKASVGKLIARALEKGIKVVFDRGGNLYHGRIKALADAAREAGLDFMDVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPILKVLRSAAANAEHNGLDRALIITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFSDSFIPDEE---EEEMGQKIHPIGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIDSLR----------------------TGLQELLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWIF-----------------KGEIIPGQEE----TPPPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAVRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTINGRVVNIPSYQCRPGELIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWIALSINELLVVEYYSRQA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPVDGAGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTALRTFSEVAEDRG--IPLENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIQRTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTISDMLTRIRNARHQTTVPSTKMTRSIAKVLQEEGYIEEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGSGQLVNGKGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKLEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQSEHYQLIRLGIRGMAVKERIGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVAEILTTKNT------------',\n",
       " 'Mastigocladus_MNPM02': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEELIEEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDITALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPAVRHLNEYRIDNSGEYTLGQEIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFARGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTIKKLTVVRVDSDRNLLLIKGAVPGKPGSLVNIVPAKVVG----MRLKYETISNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLINLTLPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKTDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVTFDNTKVVVKGPKGELSRELPANVILSQEGEILTVNRRDESRVSR--QLHGLFRTLVSNMVEGVSKGFERRLEIQGVGYRAQVQGRNLVMNMGYSHQVQIVPPEGVQFAVENNTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGTVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKRKVGSITQEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDANKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKIVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGVIQQSYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRINRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTLAASTVEELKSLGNKASVGKLLARALEKGIKVVFDRGGNLYHGRIKALADAAREAGLDFMDVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPILKVLRSAAANAEHNGFDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFSDSFIPDEE---EEEMGQKIHPIGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIDSLR----------------------SGLQELLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWIF-----------------KGEIIPGQEE----TPPPP-------------STREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAVRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTINGRVVNIASYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNAEREWIALSINELLVVEYYSRQA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPVDGAGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALTALRTFSEVAEDRG--IPLENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYAAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTISDMLTRIRNARHQTTVPSTKMTRSIAKVLQEEGYIEEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGKGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRRELNIKRLEIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKLEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQSEHYQLIRLGIRGMALKERIGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVAEILTTKNT------------',\n",
       " 'Westiellopsis_NAPS01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEELIEEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDITALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPAVRHLNEYRIDNSGEYTLGQEIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFARGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTIKKLTVVRVDSDRNLLLIKGAVPGKPGSLVNIVPAKVVG----MRLKYETISNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLINLTLPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKTDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVTFDNTKVVVKGPKGELSRELPANVILSQEGEILTVNRRDESRVSR--QLHGLFRTLVSNMVEGVSKGFERRLEIQGVGYRAQVQGRNLVMNMGYSHQVQIVPPEGVQFAVENNTNVIVSGYDKEVVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGTVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKRKVGSITQEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDANKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKIVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGVIQQSYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRINRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTLAASTVEELKSLGNKASVGKLLARALEKGIKVVFDRGGNLYHGRIKALADAAREAGLDFMDVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPILKVLRSAAANAEHNGFDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFSDSFIPDEE---EEEMGQKIHPIGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIDSLR----------------------SGLQELLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWIF-----------------KGEIIPGQEE----TPPPP-------------STREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAVRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTINGRVVNIASYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNAEREWIALSINELLVVEYYSRQA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPVDGAGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALTALRTFSEVAEDRG--IPLENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYAAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTISDMLTRIRNARHQTTVPSTKMTRSIAKVLQEEGYIEEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGKGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRRELNIKRLEIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKLEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQSEHYQLIRLGIRGMAVKERIGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVAEILTTKNT------------',\n",
       " 'Fischerella_ALVS01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEELIEEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDITALRDLKMSVGILGTKLGMTQIFDEAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPAVRHLNEYRIDNSGEYTLGQEIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFARGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTIKKLTVVRVDSDRNLLLIKGAVPGKPGSLVNIVPAKVVG----MRLKYETISNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLINLTLPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKTDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVTFDNTKVVVKGPKGELSRELPANVILSQEGEILTVNRRDESRVSR--QLHGLFRTLVSNMVEGVSKGFERRLEIQGVGYRAQVQGRNLVMNMGYSHQVQIVPPEGVQFAVENNTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQVGTVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKRKVGSITQEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDANKRLGRLASEIAVLRGKNKPFTPHLDTGDFVIVVNAEKIVTGKKTQKLYRRHSGRPGGMKTETFKLQRIPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGVIQQSYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRINRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTLAASTVEELKSLGNKASVGKLLARALEKGIKVVFDRGGNLYHGRIKALADAAREAGLDFMDVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPILKVLRSAAANAEHNGFDRSLVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFSDSFIPDEE---EEEMGQKIHPIGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIDSLR----------------------SGLQELLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWIF-----------------KGEIIPGQEE----TPPPP-------------STREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAVRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTINGRVVNIASYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNAEREWIALSINELLVVEYYSRQA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPVDGAGGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAVNALTALRTFSEVAEDRG--IPLENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYAAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTISDMLTRIRNARHQTTVPSTKMTRSIAKVLQEEGYIEEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGKGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRRELNIKRLEIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKLEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQSEHYQLIRLGIRGMALKERIGLVVSDKMQKTVVVAIENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVAEILTTKNT------------',\n",
       " 'Fischerella_AJLJ01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEELIEEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDITALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLNEYRIDNSGDYTLGQEIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFRRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTIKKLTVVRVDSDRNLLLIKGAVPGKPGSLVNIVPAKVVG----MRLKYETISNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLINLTLPRIRDFRGISPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKTDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVTFDNAKVVVKGPKGELSRELPANVILSQEGEILTVDRRDESRVSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVMNMGYSHQVQIVPPEGVQFAVENNTNVIVSGYDKEVVGNTAAKIRAVRPPEPYKGKGIRYAGEVVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGTVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSNEPNKKKVGSITQEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPFTPHMDTGDFVIVVNAEKVVTGKKTQKLYRRHSGRPGGMKTETFKLQRLPERIVEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQSYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRINRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTLAASTLEELKSLGNKASVGKLIARALEKGIKVVFDRGGNLYHGRIKALADAAREAGLDFMDVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPILKVLRSAAANAEHNGLDRSLVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFSDSFIPDEE---EEEMGQKIHPIGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQELLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWIF-----------------KGEIIPGQEE----TPPPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAVRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTINGRVVNIPSYQCRPGEVIAVKNKSRKLVENLQYPGLANLPHLEFDKKMGKVNSEREWIALSINELLVVEYYSRQA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPVDGAGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTALRTFSEVAEDRG--IPLENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYAAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRSIAKVLQEEGYIEEDGEGKRLVIALKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGKGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRRELNIKRLEIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKLEIISYQVHDTDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQSEHYQLIRLGIRGMAVKERIGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVAEILTTKNT------------',\n",
       " 'Fischerella_JACJRZ01': 'KRVSRRLQLKVEDRYPLDALLLKQTATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGIIGTKLGMTQIFDDAGVAIPVTVVQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQEIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLIKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLINLTLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKNDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVTFDNAKVSVKGPKGELSRELPANVILSQDGEILTVNRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQIQGRNLVLNMGYSHQVQIAPPEGVQFAVENNTNVIVSGYDKEVVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITQEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRIKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEDK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APPPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNSEREWVALSINELLVVEYYSRQA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPVDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--IAVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGKGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NRQW01': 'KRVSRRLQLKVEDRYPLDALLLKQTATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGIIGTKLGMTQIFDDAGVAIPVTVVQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQEIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLIKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLINLTLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKNDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVTFDNAKVSVKGPKGELSRELPANVILSQDGEILTVNRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQIQGRNLVLNMGYSHQVQIAPPEGVQFAVENNTNVIVSGYDKEVVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITQEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRIKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEDK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APPPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNSEREWVALSINELLVVEYYSRQA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPVDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--IAVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGKGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_AJLK01': 'KRVSRRLQLKVEDRYPLDALLLKQTATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGIIGTKLGMTQIFDDAGVAIPVTVVQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQEIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLIKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYAFLDRLINLTLPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKNDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVTFDNAKVSVKGPKGELSRELPANVILSQDGEILTVNRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQIQGRNLVLNMGYSHQVQIAPPEGVQFAVENNTNVIVSGYDKEVVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITQEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRIKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEDK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APPPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNSEREWVALSINELLVVEYYSRQA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIDIPITKSNSIPHPVDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--IAVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGKGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQVAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NHAG01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQRLWTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NGZU01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NHAE01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_JACYLX01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDKLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-AARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NMQB01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGSVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NGZY01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NHAH01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NGZX01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASQIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NMQC01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_JACYLY01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDKLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NGZT01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NGAA01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNERYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_JACYMA01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDKLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NMQH01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NGZS01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NQMD01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGAAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------------------------------------------------------------------------------------------------------------------MRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NHAB01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NQME01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDKLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSANVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NMQI01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQEIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNLIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKRKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTITDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEDK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWIF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNSEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NMQE01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQEIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYAYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--IPVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGKLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDILRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NMQF01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGAAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NHAA01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NQMF01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_AGIZ01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NMQJ01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQEIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPSLKQFYTNIHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYAYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--IPVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGKLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDILRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_AJLL01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNSEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NMQK01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSVPALRHLKEYRLDNSGDYALGQEIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLIKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKEIVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTINGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNSEREWVALSINELLVVEYYSRQA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGKGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NMQG01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NGZW01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NHAK01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_MRCA01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQEIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKRKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTITDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEDK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYSYCTAKTI-------------------------------------------YGILGIKVWIF-----------------KGEIIPGQEE----APPPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNSEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NGZV01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NHAF01': 'KRVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVKTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NMQA01': 'KKVSRRLQLKVEDRYPLDALLLKETATAKFEAAEAHIRLGIDPKYTDQQLRTTVVLPKGTGQVRVAVIARGEKVEAAGADIVGSEDLIDEIQKGMDFDLIATPDVMPQVAKLGKLLGPRGLMPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQEIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPSLKQFYTNIHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPIKVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASEIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--IAVENLYIMSRRKRPVPDYNSRLVSMMIRRVMRGKKSLARIVYDAMIERTGDPLEFERAVRNATPLVEVKARRVGGATYQVPMEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Fischerella_NGZZ01': '-------------------------------------------------------------------------------------------------------------------------MPSPKGGTVTDLAAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSATMGPSIKVDINALRDLKMSVGILGTKLGMTQIFDDAGVAIPVTVIQAGPCTVTQVKTKQTDGYSAIQVGYGEVKPKALNKPLLGHLAKSSAPALRHLKEYRLDNSGDYALGQQIKADIFSAGQQVDVIGTSIGRGFAGNQKRNNFGRGPMSHGSKNHRAPGSTGAGTTPGRVYPGKRAPGRLGGSRTTVRKLTVVRVDSERNLLLVKGAVPGKPGSLLNILPAKVVG-K--MRLKYETIPNLKQFYTNVHQVPKLVKVTVNRGLGEAAQNAKALEASLSEIAITGQKPVVTRAKKAIAGFKIRQGMPVGIMVTLRGERMYSFLDRLINLALPRIRDFRGVSPKSFDGRGNYTLGVREQLIFPEVEYDSIDQIRGMDISIITTAKSDEEGRALLKEMGMPFRQMSRIGKRPITVPAKVQVNFDNAKVTVKGPKGELSRELPANVILSQDGETLTVSRRDESRTSR--QLHGLFRTLVANMVEGVSKGFERRLEIQGVGYRAQVQGRNLVLNMGYSHQVQIAPPEGIQFAVENNTNVIVSGYDKELVGNTAAKIRAVRPPEPYKGKGIRYAGEAVRRKAGKTGKSGKKMAKKVVAVIKLALNAGKANPAPPVGPALGQHGVNIMMFCKEYNAKTADQAGMVIPVEISVYEDRSFTFVLKTPPASVLITKAAKIDKGSSEPNKKKVGSITKEQLRQIAQTKMPDLNANDIEAAMKIIAGTAKNMGVTVTDMKTLPSITRWYVVDADKRLGRLASQIAVLRGKNKPYTPHMDTGDFVIVVNADKVVTGKKTQKIYRRHSGRPGGMKTETFKLQRLPERIIEAVKGMLPKNSLGRQLFTKLKVYAGPHPHAQKPKLINTIPGMIQQTYLNVADNSGARKLMCIRVLGAGNRYGVGDIIAVVKDAPNMIKKSDVVEAVIVRTRIKRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVLMKLTRRESQRRHRRIRKVGSERPRLAIFRSHHIYAQVIDDTIAASTVEELKSLGNQASVGKLIARALEKGIKVVFDRGGNLYHGRVKALADAAREAGLDFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPVLKVLRSAAANAEHNGLDRALVITKAYADGPLKRFQPRAQGRAYQIRKPTCHITVAVAPS--------------------------------------MPVVSLAQ-----MMESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMEEAYTYMRSQAEQGKKFLFVGTKRQAAGIIAQEASRC-GAHYINQRWLGGMLTNWATIKTRVDRLKDLERREESGALDLLPKKEASMLRRELAKLQKYLGGIKTMRKVPDVVVIVDQRREYNAVQECIKL--GIPIVSMLDTNC--DPDVVDIPIPANDDAIRSIKLIVGKLADAIYEGRH--GQLDVEEEYEDYEGAEEEYDYEESDFADSFIPDDE---EEEMGQKIHPVGFRLGITQE----------HQSRWFADPSRYPELLQEDH--KLRNFIEEK-LGRYAQNNAGISEVRIERKADQIDLEVR-------TARPGVVVGRGG-----------------------------------------------------------------------QGIESLR----------------------TGLQDLLG--------SNRQIRINVVE-------------------------------------VQRVDADAYLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-GVQGIKVQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTAKTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE----APAPP-------------TTREREPRRR-QQRRRQQFEDRSNEGMSRYRGPRLRIRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGLTEKQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHITVNGRVVNIPSYQCRPGEVIAVRNRSRKLVENLQYPGLANLPHLEFDKKMGKVNGEREWVALSINELLVVEYYSRKA------MAT-NRRKT-NRVK---KEETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKASDVIGAVKKGVADGKKHLIEIPITKSNSIPHPIDGTGGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVRNVLAKQLGSNNPLNNARAAINALTTLRTFSEVAEDRG--ITVENLYI--------------------------------------------------------------------------MEVRERGTSLALRWLVQFSRSRGRTMARLAELMDAANETGSSIRKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHQTTVPSTKMTRNIAKVLREEGYIDEDGEGKRLVISLKYKGKNRPIITLKRVSKPGLRVYSNKLPRVLGGIGIAIISTSSGIMTDREARRQNLGGEVLCYVWMSRAMYWGTGRRKSAVARVRLVPGTGQLVNGRGDYFQFNPYLIKAPLETLGLENEYDILVKAGGGLTGQANSIRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKRNVPNGVAIQSTFNNSIVTITDGDVISWASAGSSGFKGAKKGTPFAAQTAAESAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERRQKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVMIRGGRVKDLPGVRYHIIRGTLDTAGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYIYGIGLSRSQILTGVNPDTRVKDLSDDVLRVENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIISYQVHETDTGSADVQIAMLTERINRLSEHLQNKDSSRRGLLKLIGQRKRLLSYIQNEHYQLIRLGIRGMAVKERVGLVVSDKMQKTVVVAVENRAPHPKYGKIVVNTRRYKAHDEENRCKIGDRVRIQETRPLSKTKRWQVTEILTTKNT------------',\n",
       " 'Synechocystis_sp._PCC_6308': 'KKVSRRYKAKVEARYPLEALLLKETATAKFETAEVHIRLGIDPKYTDQQLRTTVSLPKGTGQVRVAVITRGEQVEAAGADIVGEEDLIDEIQKGMDFDLIATPDMMPKIAKLGRALGPKGLMPSPKGGTVTNLAAIEFKAGKLEFRADRTGIVHVFGKSSFSEDLLNLKAIQETVDRQRPSGAKGRYWRTFVSSSMGPSIELDVNALRDLKMSLGVLGTKLGMTTIFDETGNAIPVTVVQAGPCRVTQIKTKETDGYTAVQLGYSELPDRKLTSPEDGHLRNKGLPALRHLKEYRLDDVSSFEIGQIIGADLFEEGALVDVGGKTIGRGFAGYQKRHNFKRGNMTHGSKNHRLPGSTGAGTTPGRVYPGKRMAGRYGNTNVKIRKLVVAKIDSDRNLLLIKGAIPGKTGNLVSITPALIVGQK---RLKYETIPKLEQFYTNIHQVPKVSKIVINRGLGEASQNAKALESSLSELAITGQKPVVTRAKKAIAGFKVRQGMPVGAMVTLRSDKMYAFLERLISLALPRIRDFRGVSPKSFDGRGNYSLGVKEQLIFPEISYDTIDQIRGFDISIITTANTDEEGRALLKEMGMPFRKMSRIGKRPITIPAKVEIKIDGQTVNVTGPKGNLSRDLPSLITIVQEGQELLVTRDNDSRKAR--ERHGLCRTLVSNMVDGVNQGFEKKLEIQGVGYRAQAQGAKLTLNVGYSKPVEMEMPQGINVAVNTNTEVVVSGIDKELVGNVAAKIRAVRPPEVYKGKGIRYAGEYVRRKVGKSGKK---MAKKVVALIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTASQVGMIIPVEISVFEDRSFTFILKTPPASELIKTAVGIKSGSKEPNRNKVAKITKAQLQEIAQTKMPDLNANDIEAAMKIIAGTARNMGVTVVDMKTVPDIEKWFIVDADQRLGRLATEIAVLRGKNKPFTPHLDTGDFVIVVNAEKVVTGKKEQKLYRRHSGRPGGMKTETFKLQRIPERIIEAVKGMLPKNSLGRTLFTKLKVYTGSHPHAQKPELINTIPGMIQQSYLNVADNSGARKIMCLRVLSTGNTYGIGDIIAVVKDAPNMVKKSDIVRAVIVRTKLRRDSGSIRFDDNAAVIINDNNPRGTRVFGPVARELRDRNFTKIISLAPEVLMTINRKDLQRRHLRIRKVGTERPRLAVFRSNHIYAQIIDDTLSASTVDELKELTNEASVGKLVARALAKGIQVVFDRGGNLYHGRVKALADAARESGLNFPEVDTSVKAARYIRSPKVRRVLDQIRGRSYSEALIILEFMPYRACEPIIKVLRSAVANAEHNGLNPSLVVSTAFADGPLKRFRPRAQGRAYQIRKRTCHITVAVAP---------------------------------------MSVVTLKE-----LLDSGVHFGHQTRRWNPKMAPYIY-TARNGVHIIDLVQTAQLMDEAYNYIKKAAESGKKVLFVGTKRQAAGIIAQEASRC-GAFYINQRWLGGMLTNWETIKSRVARLKDLEELEESGALDRRPKKEASVLRRELNKLQKYLGGIKDMRRVPDVVVIIDIRREHNAIKECEKL--NLPVVSMLDTNC--NPELVDFPIPANDDAIRSIKLIIGKLADAIYEGRY--GKAVAEGRFDEFEESLGEPDYDED-YEDE-EYDEDADGEEEMGQKVHPIGFRLGYTKE----------HSSCWYADAKFYPQLLQEDH--KIRQYIEKN-L-----GNASIAEIKIDRKADQIDLSIH-------TARPGVVVGKGG-----------------------------------------------------------------------QGIEKLR----------------------TDLQALLK--------NQRQFRVNVIE-------------------------------------VANVDASAPLMAEFITSQLERRVSFRRVVRQAIQRAQKA-EVLGIKIQVSG-----------------------------RLNGAEIARTE-----WIREG---RVPLHTLRADIDYSYKTASTI-------------------------------------------YGILGIKVWIF-----------------KGEIIPGEED----LRANN---------------TKAKQGR--KKPRRQKFEDRSE--MARYTGPRLRVRRLGDLPGLTRKNARRQYPPGQHGQRRKRSEYAIRLEEKQKLRYNYGVSESQLVRYVKKARRGSTGLLQLLERLDNTVFRLGMAPTIPGARQLVNHGHIMVNGKVVDIASYQCRPGDVINVRKRSVNIVKNLLNPGLANLPHLEFDKNLGKVNGEREWIALNINELLVVEFYSRKA------MAK-EREQE-QRKE---KKDNPWQERVVQIRRVSKVVKGGKKLSFRAIVVVGNEKGQVGVGVGKAGDVINAVRKAVSDGKKHLITINLNKADSITHVSNGAAGGAKVFMRPAAPGTGVIAGGAVRTVLELAGIKNILAKQQGSNNPLNNARAAVNALQQIRSFSQVAKERE--IPIEQIYSMSRRVVEVPDYNSRLVNMTIRRIMKGKKSLARILYDALIERTGEPMEFEKAIKNLTPLVEVKARRVGGATYQVPMEVRSRGSSLALRWLIAFSRKRGKTMSKLAEIMDAANETGGAIKKREETHKMAEANKAFAHYRYMPHDTISDMLTRIRNARHSTVIPSTRMTRSIAEVLKSEGFISEEGEGSKIVVSLKYKGRNRPIINIRRVSTPGLRVYSKKLPRVLGGIGVAIVSTSHGIMTDREARKQNIGGEVLCYIW-SKVVYLGTGRRKSSVARVRLVPGTGAVVNNRGSYFQQIQYILKAPLETLGLENEYDILVNSGGGLTGQSDAVKLGVARALCQLDPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKRGTKKQKKNIPSGVAIKSTFNNTIVTITDGNVISWATAGSSGFKGAKKGTPFAAQTAADSAARTAMDNGMRQVDVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERQKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIVRGTLDATGVKDRRQGRSKYGAKRPKMARISGIDLPRDKRVEIGLTYLYGIGLTSSQILTGVNPDTRVRDLSDDVLRIDNFQIEGDLRREMNIKRLDIGYRGRRHRQGLPLRGQRTRTNARTRRGRAIAGKKKAPMSLTQKKQEIISYQVHETDTGSSNLQVALLTERISQLTEHLKNKDSSRRGLLKLIGQRKSLLGYIKSQAYQLIQLGIRGMAVKERVGVVVSNKMDKTAVVAIENRSPHPKYGKIVVKTKKYKAHDPENKCQEGDRVKIRETRPLSKTKRWELAEILISKAI------------',\n",
       " 'Cyanobacterium_stanieri_PCC_7202': 'KKLSRRYKAKVEPKYPLEALLLKETATAKFESAEVHIRLGIDPKYTDQQLRTTVSLPKGTGLVKVAVITRGEKVEAAGADICGEEELIDEIQKGMDFDLIATPDMMPKIAKLGRVLGPKGLMPSPKGGTVTDLPAIEFKAGKLEFRADRTGIVHVFGKSSFSEDLLNLKALQETVDRNRPSGAKGRYWRSFVSSSMGPSIEVDINSLRDLKMSLGILGTKLGMTTIFDEAGTAIPVTVVQAGPCRVTQIKTKEKDGYVALQLGYDELPDRKLTSAEDGHLSTKGLPALRHLKEYRLDDVSSYELGQTIGADFFNEGDLVDVGGKTIGRGFAGYQKRHNFKRGNMTHGSKNHRLPGSTGAGTTPGRTYPGKRMAGRYGNTNIKVRKLTVVKVDSDRNLLLIKGAIPGKTGNLVSITPATIVG-KQ--RLKYETIPKLEQFYTNIHQVPKITKVVINRGLGEASQNAKALESSLSELAITGQKPVVTRAKKAIAGFKIRQGMPVGAMVTLRSDKMYAFLDRLISLALPRIRDFRGVSPKSFDGRGNYSLGVREQLIFPEIEYDSIDQIRGFDISIITTANTDEEGRALLKEMGMPFRKMSRIGKRPIAIPAKVNVDINGQNVSVSGPKGNLARELPSLVTVVQEGETIQVNRENDSRKAR--ERHGLCRTLVANMIDGVSQGFEKKLEIQGVGYRAQAQGSKLTLNVGYSKPVEMQMPEGISVAVNSNTEVVVSGIDKELVGNMAAKIRAVRPPEVYKGKGIRYAGEYVRRKVGKAGKK---MAKKVVALIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAQTSNQAGMIIPVEISVYEDRSFTFILKTPPASVLIKKAVGIDKGSKEPNREKVAQITEDQLREIAQTKMPDLNANDIEAAMKIIAGTARNMGVTVAEMKTVPDIDKWFVVDANQRLGRLATEIAVLRGKNKPFTPHLDTGDFVIVVNAEKIVTGNKSQKLYRRHSGRPGGMKTETFKLQRIPERIIEAVKGMLPKNSLGRSLFTKLKVYTGPHPHAQQPELIQTIPAMIQQSYLNVADNSGARKIMCLRVLSTGNTYGIGDIIAVVKDAPNMVKKSDIVRAVIVRTKLRRESGSIRFDDNAAVIINDNNPRGTRVFGPVARELRDRNFTKIISLAPEVLMTINRKNLQRRHLRIRKVGTERPRLAVFRSNHIYAQLIDDTLSASTLDELRDLSNDASVGKLVARALSKGIKVVFDRGGNLYHGRVKALADAARETGLNFIEVDTSVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPIIKVLRSAVANAEHNGLDPTLVVSTAYADGPLKRYRPRAQGRAYQIRKRTCHITVAVAP---------------------------------------MPVVSLKE-----LLDSGVHFGHQTRRWNPKMSQYIY-TARNGVHIIDLVQTAQLMDEAYNYVKKAAESGKKVLFVGTKRQAAGIIAQEAARC-GAFYINQRWLGGMLTNWETIKARVAHLKELEALEESGALDRRPKKEASVLRRELTKLRKYLGGIKDMNRVPDLVIIVDIRREHNAIKECQKL--NLPVVSMLDTNC--NPELVDLPIPANDDAIRSIKLIIGKLADAIYEGRY--GKAVAEDEFESVQTIEDDEDYDDE-----DYSDTEEDGEEEMGQKVHPVGFRLGITKE----------HLSCWYADPKEYPQLLQEDH--LIRQYIDKN-L-----SNASIAEVKIDRKADQIDLSIH-------TARPGVVVGKGG-----------------------------------------------------------------------AGIEKLR----------------------TDLQGLLK--------NQRQFRVNVIE-------------------------------------VANVDANAALMAEFIASQLERRVSFRRVVRQALQRAEKA-EVKGIKIQVSG-----------------------------RLNGAEIARSE-----WIREG---RVPLHTLRADIDYCYRTASTI-------------------------------------------YGILGIKVWIF-----------------KGEVIPGEEN----LETPA-------------------PRGRERQPRRQKFEDRSE--MSRYTGPRLRVRRLGDLPGLTRKNARRQYPPGQHGQRRKRSEYAIRLEEKQKLRYNYGVSEKQLVRYVKKARRGSTGLLQLLERLDNTVFRLGMAPTIPAARQLVNHGHVTVNGGVVDIASYQCRPGDVIAIRDRSRKIVENLSNPGLANLPHLEFDKTYGKVNGEREWVALNINELLVIEYYSRKA------MAK-EREQK-RKRR---EKDNSWQERVVQIRRVSKVVKGGKKLSFRAIVVVGNEKGQVGVGVGKAGDVINAVRKAVTDGKKHLITVKINKASSITHVSNGVAGGAKVFMRPAAPGTGVIAGGAVRTVLELAGIKNILAKQQGSNNPLNNARAAVNALEQIRSFSEAAQERD--IPLEQIYSMSRRVVDVPDYNSRLVSMTIRRIMKGKKSLARILYDAMIERTGDPMEFEQAIKNLTPLVEVKARRVGGATYQVPMEVRSRGSSLALRWLIQFSRKRGRTMSKLAEIMDAANETGGAIKKREETHKMAEANKAFAHYRYMPHDTISDMLTRIRNARHPTVIPSTRMTRSIAEVLKTEGFITEEGEGKKIVLSLKYKGRNRPIINIRRVSTPGLRVYSKKLPRVLGGIGVAIISTSHGIMTDREARKQGIGGEILCYIW-SKVVYLGTGRRKASIARVRLVPGSGAVVNGKAVYFQQIQYIVKAPLETLGLESEYDILVKAGGGLTGQSDAVKLGVARALCQLDPENRPLKEGYLTRDPRAKERKKYGLKARKAPQFSKRMAKKRGGKKQKKNIPSGTAIKSTFNNTIVTITDGNVISWATAGSSGFKGAKKGTPFAAQTAADSAARTAMDNGMKQVEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERQKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIVRGTLDATGVKDRKQGRSKYGAKRPKMARISGIDLPRDKRVEIGLTYLYGIGLVTSQILTGVNPDTRVKDLSDDILRIENYQIEGDLRREMNIKRLDIGYRGRRHRQGLPVRGQRTKTNARTRRGRAIAGKKKAPMSLTQKKQEIISYQVHETDTGSSDLQVALLTARITQLTEHLKNKDSSRRGLLKIIGKRKRLLGYIKNQGYQLIKLGIRGMAQKERVGVVVSNKMDKTAVVAVENRSPHPKYGKIVVKTKKYKAHDPENQCQEGDRVRIRETRPLSKTKRWELAEIISTKAI------------',\n",
       " 'Cyanobacterium_sp._PCC_10605': 'KKLSRRFKAKVEAKYPLEALLLKETATAKFETAEVHIRLGIDPKYTDQQLRTTVSLPKGTGQVRVAVITRGEKVEAGGADLVGDEDLIDEIQKGMDFDLIATPDMMPKIAKLGRVLGPKGLMPSPKGGTVTDLVAIEFKAGKLEFRADRSGIVHVFGKASFSNDLLNLKAIQETVDRNRPSGAKGRYWRTFVSSSMGPSIEVDINSLRDMKMSLGVLGTKLGMTTVFDETGNAIPVTVIQAGPCRVTQIKTPEKDGYTALQIGYDELPDRKLTSPEAGHLATKGLPALRHLKEYRLDDVSGYELGQTIGADLFSEGQLVDVGGKTIGRGFAGYQKRHNFKRGNMTHGSKNHRLPGSTGAGTTPGRTYPGKRMAGRYGNTQIKVRKLTVVKIDSERNLILVKGSIPGKPGNLVSVTPATIVG-Q---KLKYETIPKLEQFYTNIHQVPKVTKVVINRGLGEASQNAKALESSLSELAITGQKPVVTRAKKAIAGFKIRQGMPVGAMVTLRSDKMYAFLERLINLSLPRIRDFRGISPKSFDGRGNYSLGVKEQLIFPEIEYDSIDQIRGFDISIITTANTDEEGRALLKEMGMPFRKMSRIGKRPITIPAKVEVTIDGQLVQVKGPKGDLSRTLPALVTLTQEGQEIKVVRDNDSRKAR--ERHGLCRTLVDNMIQGVSQGFEKKLEIQGVGYRAQAQGSKLTLNVGYSKPVEMEMPQGISVAVNNNTEVVVSGIDKELVGNVAAKIRAVRPPEVYKGKGIRYSGEYVRRKVGKAGKK---MAKKVVALIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTSNQPGMIIPVEISVFEDRSFTFILKTPPASVLIKKAAGIEKGSKEPNRAKVAQITKAQLEEIAQTKMQDLNANDIDAAMKIIAGTARNMGVTVVDMKTVPSIEKWYVVDANQRLGRLATEIAVLRGKNKAYTPHLDTGDYVIVVNAEKVVTGRKEQKLYRRHSGRPGGMKVETFQLQRIPERIIEAVKGMLPKNSLGRSLFTKLKVYTGPHPHAQQPELVNTIAGMIQQSYLNVADNSGARKIMCLRVLSTGNTYGIGDIIAVVKDAPNMVKKSDIVRAVIVRTKLRRESGSIRFDDNAAVIINDNNPRGTRVFGPVARELRDKNFTKIISLAPEVIMTINRKNLKRRHLRIRKVGTERPRLAVFRSNHIYAQIIDDTLAASTLDELRELTNDASVGKLIARALSKGIKVVFDRGGNLYHGRVKALADAARESGLNFIEVDTSVKAARYIRSPKVRRVLNQIRGRSYREALILLEFMPYRACEPIVKVLRSAVANAEHNGLDPNLVISQAFADGPLKRYRPRAQGRAYQIRKRTCHITVAVAP---------------------------------------MSVVSLKE-----LLDSGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLLEEAYAYLKKAAENGKKVLFVGTKRQAAGIIAQEAARC-GAFYINQRWLGGMLTNWETIKSRVGRLKELEELEESGAINLRPKKEASVLRRELTKLQRYLGGIKDMRRVPDIVVIIDIRREHNAVKECEKL--GLSVISMLDTNC--NPELVDYPIPANDDAIRSIKLIIGKLADAIYEGRY--GKAAAEDEFESVSAEA-EYDYDDDDYDDEDYEDDSDSGEEEMGQKVHPIGFRLGITKE----------HSSCWYADKRNYPKLLQEDH--KIRQYIDKN-L-----ANASIADVKIERKADQIDLSIH-------TARPGVVVGKGG-----------------------------------------------------------------------QGIEKIR----------------------TDLQALLK--------NQRQFRVNVIE-------------------------------------VTNVDASAALMAEFLTSQLERRVSFRRVVRQAIQRAQKA-EVLGIKIQVSG-----------------------------RLNGAEIARTE-----WIREG---RVPLHTLRADIDYSYKTASTI-------------------------------------------YGILGVKVWIF-----------------KGEIIPGEEN----LNLTA---------------NRNKPSR--KKPRRQKFEDRSE--MARYTGPRLRVRRLGDLPGLTRKNARRSYPPGQHGQRRKRSEYAIRLEEKQKLRYNYGVSEKQLVRYVKKARRGSTGLLQLLERLDNTVFRLGMAPTIPSARQLVNHGHIMVNGQVVDIASYQCRPGDIISVRNRSTNIVKNLLNPGLANLPHLEFDKNLGKVNGEREWVALNINELLVIEYYSRKA------MAK-EQERK-RNKE---KKDNSWQERVVQIRRVSKVVKGGKKLSFRAIVVVGNEKGQVGVGVGKAGDVINAVRKAVSDGKKHIVNVNINKSNSITHVSTGMAGGAQVFMRPAAPGTGVIAGGAVRTVLELAGIKNILAKQQGSNNPLNNARAAINALQQIRSFSSVAKERD--IPLEQIFSMSRRTVEVPDYNSRLVNMTVRRIMKGKKSLAKILYDALIERTGEPMEFEKAIKNLTPLVEVKARRVGGATYQVPMEVRGRGTSLALRWLTQFARKRGKSMSKLAEIMDAANETGAAIKKREETHKMAEANKAFAHYRYMPHDTISDMLTRIRNARHSTVIPSTRMTRNIADVLKEEGFIGEEGEGKKIVVSLKYKGRNRPVINIRRVSTPGLRVYSKKLPRVLGGIGVAIISTSSGIMTDREARKQGIGGEILCYVW-SQVVYLGTGRRKASVARVRLVPGTGAVVNNRGSYFQLIPYILKAPLETLGLENEYDIIVNAGGGLTGQSDAVKLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQFSKRMAKKRGGKKQKKNIPSGTAIKSTFNNTIVTITDGNVISWATAGSSGFKGAKKGTPFAAQTAADSAARVAMDNGMKQVEVMVSGPGAGRETAIRALQAGLEITLIRDITPIPHNGCRPPKRRRVMPTIQQLIRERTKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIVRGTLDATGVKDRKQGRSKYGAKRPKMARISGIDLPRDKRVEIGLTYLYGIGLTTSQILTGVNPDTRVRDLSDDVLRIENYQIEGDLRREMNIKRLDIGYRGRRHRQGLPVRGQRTKTNARTRRGRAIAGKKK-PMSLTQKKQEIIGYQIHETDTGSSALQVALLSARISQLTEHLKNKDSSRRGLLKLIGQRKRLLGYIKRQDYQLIKLGIRGMAVKERVGVVVSNKMDKTAVVAVENRSPHPKYRKIVVKTKKYKAHDEQNQCLEGDRVRIRETRPLSKTKRWEIAEIISRKAL------------',\n",
       " 'Spirulina_sp._PCC_9445': 'KKISRRLQAKVEKRYPLEALLLKETATAKFETAEAHIRLGIDPKYTDQQLRTTVSFPKGTGQVRVAVIARGEKVEAAGADIAGSEELIEEIQKGMDFDVIATPDMMPQVAKLGRLLGPRGLMPSPKAGTVTDLAAIEFKAGKQEFRADRTGIVHVFGKASFSEDLLNLKALQEVIDRNRPSGAKGRYWRSYVSASMGPAIEVDINALRDLKMSFGILGTKLGMTQIFDETGKAIPVTVIQAGPCTVTQIKTPETDGYTAVQVGYEEVRPKALTKPELGHLEKSGAGPLRHLTEYRVSDSNTFELGQAIKADIFSPGQLIDVHGTSIGRGFAGYQKRHNFRRGPMSHGSKNHREPGSTGPGTTPGRIFPGKRMAGRYGGTKVTIRKLEVVRVDAERNLLLVKGSVPGKPGALVSITPANIVG-Q--MRLKYETIPKLEQFYTNIHQVPKVVKITVNRGLGEASQNAKALESSINELAITGQKPVITKAKKAIAGFKIRQGMPVGVMVTLRSDRMYAFLDRLISLALPRIRDFRGISPKSFDGRGNYSLGVREQLIFPEIDYDSIDQIRGMDISIITTAKTDEEGRALLKEMGMPFRNMSRIGKRPIEIPDKVTVTINGSHVAVKGPKGQLERVLPSQVSVEQEDKVVRVLRKDESRLSR--QRHGLSRTLVANMVEGVSKGYEKRLEIQGVGYRAQVQGKKLILNVGYSKPVEVDPPDGIQLTLEKNTEVIVSGINKEIVGNVAAKIRAIRPPEVYKGKGIRYKGEFVRRKAGKAGKK---MAKKVVALVKLALPAGKAAATPPVGPALGQHGVNIMAFVKEYNAKTADQAGMVVPVEISIYEDRSFTFILKTPPASVLIKKAAGIEKGASQPNKQVVATITQDQLRKIAETKMPDLNANDVEAAMRIIAGTAKNMGVKVAAMKTLPTLDQWYVVDADQRLGRLASEIAILRGKNKAFTPHMDTGDFVIVVNAEKVVTGKKSQKLYRRHSGRPGGMKTETFQLQRIPERIVEAVKGMLPKNSLGRKLFTKLKVYTGPHKHAQQPELINTQPTMIQQTYLNVADNSGARKLMCLRVLGTGNRYGIGDIIAVVKDAPNQVKKSDIVRAVIVRTRLRRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNFTKIVSLAPEVIMKRTRKASQRRHRRLRTVGTERPRLAVFRSHHIYAQVIDDTLAASSLEELKELGTDASVGAAIARSLAKGIQVVFDRGGNLYHGRVKALADAAREGGLNFMAVETQVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPITKVLRSAVANAEHNGYDPRLVVTQAYANGPLRRFRARAQGRAYQIHKPTCHITVAVAPE--------------------------------------MAVVSLAN-----LLESGVHFGHQTRRWNPKMDPYIY-TSRNGVHIIDLVQTAELMEEAYEYMRNAASQGKKFMFVGTKRQAAGIIAQEAARC-NSYYVNQRWLGGMLTNWETIKHRIDHLKELERLEETGSIEKRPKKEGSVLRRELSKLRKYLGGIKSMRRPPDVVLIVDQRREYNAILECQKL--HIPIVSLLDTNC--DPDLVDIPVPANDDAIRSIKLIVGRLADAILEGRS--GGKDVIEEI-DYSGEI---DYEDD-FEDDDGEDDD---SEEMGQKIHPTGFRLGVTKE----------HKSCWYADTKRYPELLQEDH--QIRQYVEKN-L-----NNAGISDVRIERKADQIDLEIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIESLR----------------------TGLQEHLG--------GNRQIRVNVIE-------------------------------------VSRVDADGVLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-EVQGIKIQVSG-----------------------------RLNGAEIARTE-----WTREG---RVPLHTLRADIDYAYCTARTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE--APSAAPA-----------------QTPRR---QRRRQQFEDRSE--MSRYRGPRLRIRRLGELPGLTRKSSRRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSERQLIRYMRQARRGSTGLLQLLERLDNTVFRLGMAGTIPAARQLVNHGHVMVNGRVVDIASYQCRAGDVVTVRNKSRRLVENMEYPGLANLPHLEFDKTMGKVNGEREWVALNINELLVVEYYSRKV------MPKGKKKGR-KNQK---EQENAFQERVIQIRRVSKVVKGGKKLSFRAIVVVGNERGQVGVGVGKAADVIGAVRKGVADGKKQLVDISLTKSNSITHPARGVAGGAKVMMRPAAPGTGVIAGGAVRTVLELAGVKNILAKQLGSNNPLNNARAAVDALQTLRTFKDVAEERG--IPVENLYIMSRRKREVPDYNSRLISMTVRRIMRGKKSLARLVYDALVERTGDPIEFEQAIRNLTPLVEVKARRVGGATYQVPMEVRNRGTTLALRWLSQYSRQRGRTMAKLAEIMDAANETGGAIRKREETHRMADANKAFAHYRYMANDTISDMLTRIRNARHPTTIPSTRVTRDIAKVLKAEGFIDEEGEGKRLVLTLKYKGRTRPIIALKRISKPGLRVYKNKLPRVLGGIGIAIISTSSGIMTDREARQRGIGGEVLCYVWMNRAMYWGTGRRKSAIARVRLVPGSGKLVNNRGEHFNQNPYLAKAPLETLGLENEYDILVNAGGGLTGQSDAVRLGVARALCQLDPDNRPLKEGYLTRDPRAKERKKYGLKARKAPQFSKRMARKKGGKKQKRNVPSGIAITSTFNNTIVTISDGDVISWASSGSSGFKGAKKGTPFAAQTAADSAAKRAIEQGMRQLEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDATGVKDRRQGRSKYGTKRPKMARIAGIDLPRDKRVEIGLTYIYGIGLSRSQVLTGVNPDTRVRDLSDDVLRIENFQVEGDLRREMNIKRLDIGYRGRRHRLGLPVRGQRTRTNARTRRGRTVAGKKKAAMSLTQLKQEIISYQVHETDTGSSDVQVAMLTKRINQLTEHLKNKDSSRRGLLKLIGRRKRLLAYIQDEHYQLIRLGIRRMAAKERVGMVISNKMDKTIVVAVENRSPHPKYGKIVVKTKHYKAHDEENQCQEGDRVRIQETRPYSKTKRWILREIMNRSVV------------',\n",
       " 'Stanieria_cyanosphaera_PCC_7437': 'KKLSRRMRAKVQDKYPLEALLLKETATAKFETAEAHIRLGIDPKYTDQQLRTTVTFPKGTGQVRIAVIARGEKVEAAGAEIVGSEELIDEIQKGMDFDLIATPDMMPKVARLGRMLGPKGLMPSPKGGTVTDLPAIDFKGGKQEFRADRTGIVHVFGKVSFSEDLLNLKALQETVDRNRPSGAKGRYWRSFVSSSMGPSIQVDISGLRDLKMAVGILGTKLGMTQIFDETGSAIPITVVQAGPCVITQIKTKATDGYNSIQIGYGEVKEKALNKPELGHLKKSGATPLRHLKEYQIDDVSSFELGQSINAELFSAGEIVDVTGTTIGRGFAGYQKRHNFKRGSMSHGSKNHRLPGSTGAGTTPGRVYPGKRMAGRYGASQVTIRKLTVVEIDSEKNLLLIKGAVPGKAGALLSIAPSNIVG-KK-MKLKYEQIPKLQQFYQNIHQVPKVIKVVVNRGLGEASQNAKALESSINELGITGQKPVVTRAKKAIAGFKIRKGMPVGVMVTLRGERMYAFLDRLINLALPRIRDFRGVSPKSFDGRGNYSLGVREQLIFPEIDYDSIDQIRGMDISIITTANTDEEGRALLKEMGMPFRNMSRIGKRPIPIPNKVTVNIQERYVTVKGPKGTLERELPSLVSVKQEGETLLVVPEDESRKAR--ERHGLSRTLVANMVSGVADGFEKRLQIQGVGYRAQAQGKKLTLNVGYSKPVEMTMPDGIQVAVEKNTEITVSGIDKEVVGNVAAKIRAVRPPEPYKGKGIRYLGEVVRRKAGKTGKK---MPRKVVAIIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNARTADKVGLVIPVEISVFEDRSFTFILKTPPASVLIRKAAGIERGASQPNKQTVATITQAQLREIAETKMPDLNANDIEAAMKIVAGTAKNMGVAIAEMKTLPTLEQWYVVDAEKRLGRLASEIAILRGKNKPFTPHMDTGDFVIVVNADKVVTGRKSQKLYRRHSGRPGGMKVETFQLQRIPERIIEAVKGMLPKNSLGRKLFTKLKVYSGSHPHAQQPELINTIPVMIQQTYLNVADNSGARKLMCLRVLGTGNRYGIGDIIAVVKDAPNMVKKSDIVRAVIVRTKLRRESGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKNFTKIVSLAPEVIMKSSRKQLQRRHQRIRKVGTERPRLSVFRSNHVYAQIIDDTLAASTLDGLKTLRTEASVGKLVARALEQGIKVVFDRGGNLYHGRIKALAEAAREAGLDFMAIDSNVKAARYVRSPKVRRVLDQIRGRQYREALIILEFMPYKACEPILKVLRSAVANAEHNGLDPSLIVSQAYADGPFKRYRPRAQGRAYQIRKPTCHITVVVAPT--------------------------------------MPVVSLAE-----LLESGVHFGHQTRRWNPKMSTYIY-TARNGVHIIDLVQTAQLIEEAYDYMRSSSEQGKRVLFVGTKRQAAGIIAQEASRC-GAYYVNQRWLGGMLTNWETIRTRVERLKELERLEETGALDKRPKKEASVLRRELSKLQKYLSGIKLMRKVPDIVVIIDQKREHNAISECQKL--GIPIVSILDTNC--DPELVDIPIPANDDAIRSIKLIIGKLADAIYEGRH--GQLDTAEEDEEFEEGMDGEDYPDAEYDEEYEEEYEENGEEEMGQKIHPVGFRLGITKD----------HQSCWYADTKRYPELLQEDH--LIRNYLAKE-L-----NNAGISQIRIERKADQIDLAIH-------TARPGVVVGRGG-----------------------------------------------------------------------TGIEKLR----------------------TELQEAVG--------NNRQIRINVIE-------------------------------------VARVDADAALIAEYITQQLERRVSFRRVVRQAIQRAQRA-EVQGIKIQVSG-----------------------------RLNGAEIARTE-----WVREG---RVPLHTLRADIDYAYRTALTI-------------------------------------------YGILGVKVWVF-----------------KGEIIPGQED----IPAAV---------------PNQAPRR---QRRRQKFEDRSE--MSRYRGPRLRVRRLGGLPGLSRKSPRKTYPPGQHGQRKKRSEYAVRLEEKQKLRYNYGVTEKQLIRYVRKARRGSTGLLEMLERLDNTIFRLGMAGTIPGARQLVNHGHITVNGRVVNIASYQCRPGDVIAVRDRSRRLVENMEYPGLANLPHLEFDKTFGKVNGEREWIALQINELLVIEYYSRMA------MAKTQSRKKRNRAK---EKETNWQERVVQIRRVSKVVKGGKKLSFRAIVVVGNEKGQVGVGVGKAADVIGAVRKGVADAKKQLIEVPLTKSNSITHVINGAAGGARVFMRPAAPGTGVIAGGAVRTVLELAGVKNILAKQLGSNSPLNNARAAIDALETLRAFSEVAKERG--ITLEQIYAMSRRKRPVPDYNSRLVNMTVRRVMSGKQSLARIVYDALISRTGDPLEFEQAIKNITPLVEVKARRVGGATYQVPMEVKSRGTTLALRWLVHFARARGRTMAKLAEIMDAANETGGAIKKREETHRMAEANKAFAHYRYMANDTISDMLTRIRNAKHPTTVPATNMTRSIAQVLASEGFIGEEGEGKKLVITLKYKGKNRPIITLKRVSKPGLRVYSRKLPRVLGGIGIAIISTSHGVMTDREARRQGIGGEILCYVWMQRVVYWGTGRRKSSIARVRLVPGNGQIVNNKGEYFNRIPYLIKAPLETLGFENEYDILVNAGGGLTGQADAVKLGVARALCQQDPNNRPLKEGYLTRDPRSKERKKYGLKARKAPQYSKRMAQKKSGKKQKKNVPNGVAIQSTFNNTIVTIADGDVVSWASAGSSGFKGAKKGTPFAAQTAADSAAKRAMEQGMRQIEVMVSGPGAGRETAIRAIQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERQKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIVRGTLDATGVKDRRQGRSKYGTKRPKMARVSGVDLPRDKRVEVALTYIYGIGLSRSQILTGVNPDTRVRDLSDDVLRIENYQVEGDLRREMNIKRLDIGYRGRRHRMGLPVRGQRTRTNARTRRGKTVAGKKKPPMSLTQRKQELMSYQIHETDTGSADLQVAILTERINQLTQHLKNKDSSRRGLLKMIGRRRNLLSYIQDERYQLIRLGIRRMAVKERVGLVVSDKMDKTVVVAIENRSPHPKYGKIVVKTQKYKVHDENNQCKIGDRVKIRETRPLSRTKRWAIAEILEAKD-------------',\n",
       " 'Cyanobacterium_UCYN-A': 'KKLSKRIKAKIQKKYPIEALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVSLPNGTGQIRVAVIAKGEKVEAAGADIVGSEELIEEIQKGMDFELIATPDMMPKVAKLGRALGPRGLMPSPKGGTVTDLGAIEFKSGKLEFRADRTGIVHVFGKASFSKDLLNLKALQDTVDKQRPSGAKGRYWRSYVSASMGPSIQIDINALRDLKMAIGLLGTKLGMTQIFDETGAAIPVTVMKVGPCTITQIKTSDTDGYNAIQIGYSEVKEKALVKPELGHLKKAGASPLRHLKEYRVDDIQPYQLGETVNVDLFNTGDLVDVTGTSMGRGFAGYQKRHNFKRGNMTHGSKSHRLPGSTGAGTTPGRVYPGKKMAGQYGASKVTTRRLKVVQIDAECNLLLIKGAIPGKPGGLLNITPATIVG-K--VRLKYETIVKLQQFYTNVHQVPKVLKVNINRGLGEASQNAKALESSIREFEITGQKPIITRAKTAIAGFKIREGMPVGVMVTLRSEKMYSFLDRLINIALPRIRDFRGISPKSFDGRGNYSLGIREQLIFPEIDYDTIDQIRGMDVSIITTAQTDEEGRALLKEMGMPFR-MSRIGKRPITLPDKVTIDIKGQYITVKGPKGSLERELPSRVQLNKEGETLMVQRQDDSLISR--ERHGLCRTLVANMVHGVSAGFEKRLSIQGVGYRAQAKGTKLTLNVGYSKPVEIAMPNGINVAVENNTLVIISGIDKEIVGNIAAKIRAVRPPEPYKGKGIRYLNEFVVRKAGKSGKK---MAKKVVALIKLALPAGKANPAPPVGPALGQRGVNIMAFCKEYNAKTSDKAGMIIPVEISVYEDRSFTFVLKTPPASVLILKAAGIEKGSSQPDKEKVGSITYNQLKDIAQTKMPDLNANDIDAAMNIVEGTARNMGVLIKDMKTTPIIEQWYVIDAKQRLGRLACEIAILRGKKKPYTPHLDTGDFIIVINAEKIVTGRKHQKLYRRHSGRPGGMKVETFKLLRIPERIIEAVKGMLPKNTLGRQLFTKLKVYAGPHPHAQKPELLNTIPNMIQQTYLNVADNSGARKLMCLRVLGTGNRYGIGDIVAVVKDAPNMVKRSDIVKAVIVRTRINRENGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKSYTKIVSLAPEVLMKLTRRESKRRHQRIRKVGTNCPRLAVFRSNHIYAQIIDDTLAASTLEGLKEASSEASVGKLVARLIAKGIQVVFDRGGNLYHGRVKALAEAAREAGLKFMALDMTAKASKYIRSPKVRRVLDQIRGRSYREALIILEFMPYKACEPIIKLLRSAVANAEHNGLDPALFISKAYADGPLKRFRPRAQGRAYQIRKPTCHITIAVASE--------------------------------------MPVVSLSE-----LLESGVHFGHQTRRWNPKMSQYIY-TARNGVHIIDLVQTAQLMEDAYQYVRNGSDKGKRFLFVGTKRQAAGIIAQEASRC-GANYVNQRWLGGMLTNWETIKTRVERLKELEAMEESGQIEKRPKKEGAVLRRELGKLQKYLGGIKTMRRPPDVVVIVDQRREYNAIQECQKL--GIPMISLLDTNC--DPDYADIPIPANDDAIRSIKLILGKLADAIDEGSH--RRVNAN---DDYETSDESFEESTS-VSSSLEGENIEEENQAMGQKIHPIGFRLGITKE----------HKSRWYADKRRYPELLQEDL--AVRKHIEKN-L-----SNAGIADIRIERKADQIDLAIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIEQLR----------------------TALQKNLG--------ENRQIRINVIE-------------------------------------VSRVDADAALIAEYITQQLERRVSFRRVVRQAIQRAQRA-EVKGIKIQVGG-----------------------------RLNGAEIARSE-----WVREG---RVPLHTLRADIDYSYRTAQTT-------------------------------------------YGILGVKVWVF-----------------KGEIIPGQEE-QTTTSAPP------------------TPKR----NRRKQFEDRSNEEMSRYRGPRLRVRRLGELPGLSRKNPRRTYPPGQHGQRRKRSEYAVRLEEKQKLLYNYGVTEKQLIRYMKKARRGSTGLLELLERLDNTVFRLGMAGTIPGARQLVNHCHVLVNGRTVNIASYQCRAGDIISVRNNSRRMVENMQYPGLANLPHLEFDKTFGKVNDKREWVALQINELLIVEYYSRKV------MAK-NRKN--NRDK---AKETNWQERVIQIRRVSKVVKGGKKLSFRAVVIIGNEKGQVGVGVGKAGDVIGAVRKGVADGKKQLVDVSITKSSSITHITKGVAGGARVIVRPAAPGTGVIAGGAVRTVLELAGLKNILAKQLGSNNPLNNARAAVDALEGLRTLSEVAQERD--VPLEHLYAMSRRRKPVPDYSSCLLNMTIRRVMKGKKSVAGIVYEAMIERTGEPLEFEQAIKNLTPLVEVKARRVGGATYQVPMEVRSRGTTLALRWLIRYSRIRGRTMAKLAEIVDAFNETGAAMKKRDETHRMADANKAFAHYRYMANDTISDMLTRIRNARHPTTVPTTRMTRSIAQVLKEEGFISEEGEGQKLVISLKYKARGRPIITLRRVSKPGLRVYSHKLPRVLGGIGVAIISTSKGIMTDREARRQNVGGEVLCYIWMEKAVYWGTGRRKSSIARVRLIPGTGEIVNKKGNYFNQIAYIIKAPLEKLGLDKEYDILVNAGGGLTGQADAVKLGVARALCELNPENRTLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKGGKKQKKNVPNGVAIKSTFNNTIVTISDGDVISWSSAGASGFKGAKKGTPFAAQTAADSAARRAIDNGMRQIEVMVSGPGAGRETAIRALQSGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDAQGVKDRKQGRSKYGAKRPKMARISGVDLPRDKRIEIGLTYLFGIGLSRSQILTGVNPDIRVRDLSDDLLRIDNFQIEGDLRREMNIKRLDIGYRGRRHRQGLPVRGQRTRTNGRTRRGRTVAGKKKAPMSLNPQKQTLMSYQIHETDTGSADLQVAILTERIVQLTSHLKNKDASRRGLLKMIGRRRRLLAYIKSNRYQLIRLGIRRMAVKEKVGVVVSAQMDKTIVVAVENRSPHPKYGKIIVKTKKFKAHDQENKCQ---RRR------------------------------------',\n",
       " 'Cyanothece_sp._ATCC_51142_chromosome_circular': 'PKLSKRIKAKVQDRYPLEALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVSLPKGTGQIRVAVIARGEKVEAAGADIVGSEELIDEIQKGMDFELIATPDMMPKVAKLGRLLGPRGLMPSPKGGTVTDLEAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRQRPSGAKGRYWRSFVSASMGPSIQIDISALRDLKMAVGLLGTKLGMTQIFEESGLAIPVTVVQAGPCTITQIKTAETDGYSAIQIGYLEVKEKALTKPELGHLKKVEASPLRHLKEYRVDDTATYTLGEAVKADIFNAGDLVDIAGTSMGRGFAGYQKRHNFKRGSMTHGSKNHRLPGSTGAGTTPGRVYPGKKMAGQYGAVKTTTRHLQVVRVDAERNLLLIKGAVPGKPGGLLNITPAKIVG-K--ARLKYETIPKLQQFYSNIHQVPKIAKITINRGLGEASQNAKALESSISELEITGQKPVVTRAKKAIAGFKIREGMPVGVMVTLRSEKMYAFLDRLINLALPRIRDFRGISPKSFDGRGNYSLGIREQLIFPEIDYDTIDQIRGMDVSIITTAQTDEEGRALLKEMGMPFR-MSRIGKKPIPLPNKVTIDIKGQHIAVKGPKGSLELDLPSEVTVNQEGETVEVQRVDDSRTAR--ERHGLFRTLVANMIQGVSQGFEKRLNIQGVGYRAQAQGSKLTLNVGYSKPVEMTMPQGINVAVENNTQVVVSGIDKEIVGNIAAKIRGVRPPEPYKGKGIRYQDEYVRRKAGKAGKK---MAKKVVALIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTSDQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAGIEKGSAQPNNQKVGSISRDQLKEIAQTKMPDLNANDIDAAMNIVEGTARNMGVTINDMKTTPGLDQWYVIDADQRLGRLACEVAILRGKNKPYTPHMDTGDFVIIINADKVVTGRKTQKLYRRHSGRPGGMKIETFKLQRIPERIIEAVKGMLPKNALGRQLFTKLKVYPGPHPHAQKPELLNTIPNMIQQTYLNVADNSGARKLMCLRVLGTGNRYGIGDIVAVVKDAPNMIKRSDIVTAVIVRTRVNRASGSIRFDDNAAVIINDGNPRGTRVFGPVARELRDKSYTKIVSLAPEVLMKTTRKESKRRHRRIRKVGTDCPRLAVFRSNHIYAQIIDDTLAASTLEDLRNLGTEASVGKLVARGMAKGIQVVFDRGGNLYHGRVKALADAAREAGLQFMAVDTTAKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACEPILKLLRSAVANAEHNGLDPTLVVSQAYADGPLRRYRPRAQGRAYQIRKPTCHITIAVAPE--------------------------------------MAVVSLED-----LLNAGVHFGHQTRRWNPKMSQYIY-TARNGVHIIDLVQTAQLMEDAYQYVRNNADKGKRFLFVGTKRQAAGIIAQEASRC-GANYVNQRWLGGMLTNWETIKTRVERLKELEAMEESGQIARRPKKEASVLRRELGKLQKYLGGIKTMRRPPDVVVIVDQRREYNAIQECQKL--GIPMISLLDTNC--DPDYADIPIPANDDAIRSIKLILGKLADAIYEGRH--GQLDSDDDYEEFDESLAEGDYDD--YDEEDSETVSSQGEEEMGQKIHPIGFRLGVTKE----------HKSCWYADKRRYPELLQEDL--AIRQHVEKN-L-----SNAGIADIRIERKADQIDLAIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIEQLR----------------------TGLQQTLG--------ENRQIRINVIE-------------------------------------VSRVDADAALIAEYITQQLERRVSFRRVVRQAIQRAQRA-EVKGIKIQVGG-----------------------------RLNGAEIARTE-----WVREG---RVPLHTLRADIDYAYRTAQTI-------------------------------------------YGILGVKVWVF-----------------KGEIIPGQDEQAMAAPAPT-------------------PRK---KRRPQQFEDRSNEEMSRYRGPRLRVRRLGELPGLSRKTPRRAYPPGQHGQRRKRSEYAIRLEEKQKLLYNYGVTEKQLVRYMKKARRDSTGLLELLERLDNTVFRLGMAGTIPGARQLVNHGHVIVNGRVVNIASYQCRPGDVISVRNKSRRMVENMQYPGLANLPHLEFDKTLGKVNDKREWVALQINELLVVEYYSRKV------MAK-SRKS--NRDK---SKDSNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNEKGQVGVGVGKAGDVIGAVRKGVADGKKQLIDVSLTKSSSITHVAKGVSGGAKVIVRPAAPGTGVIAGGAVRTVLELAGLKNILAKQLGSNNPLNNARAVINALEGLRTFSEVAQERG--VPLEHLYTMSRRRKPVPDYNSCLLNMTIRRVMKGKKSVAGIVYDAMIERTGEPLEFERAMKNLTPLVEVKARRVGGATYQVPMEVRARGTTLALRWLIRYSRIRGRSMAKLAEIMDAANETGAAMKKRDETHRMADANKAFAHYRYMANDIISDMLTRIRNARHPTTVPTTKMTRSIAQVLKEEGFIGEEGEGKKLVISLKYKSRGRPIITLQRVSKPGLRVYSNKLPRVLGGIGIAIISTSKGIMTDREARRQNVGGEVLCYVWMKKVVYWGTGRRKSAVARVRLVPGSGEIVNKKGNYFNRIPYLIKGPLETLGLENDYDILVNAGGGLTGQADAVKLGVARALCELAPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKR-----------------MAIQSTFNNTIVTISDGDVISWSSAGASGFKGAKKGTPFAAQTAADSAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDAQGVKDRKQGRSKYGTKKPKMARISGVDLPRDKRVEIGLTYLYGIGLSRSQILTGVNPDTRVRDLSDDVLRIENYQIEGDLRREMNIKRLDIGYRGRRHRQGLPVRGQRTRTNGRTRRGRTVAGKKKAPMSLTAEKQVLMSYQVHETDTGSADLQVAILTKRITQLTDHLKNKDASRRGLLKMIGRRRRLLGYINDNRYQLIRLGIRRMAVKERVGVVVSTKMDKTAVVAVENRSPHPKYGKIVVKTKKFKAHDPENQCQEGDRVRIRETRPLSKTKRWEIKTILTAH--------------',\n",
       " 'Cyanothece_sp._PCC_8801': 'KKLSKRFKAKVQNQYPLEALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVSLPKGTGQVRVAVIARGEKVEAAGADIVGSEELIEEIFKGMDFELIATPDMMPKVAKLGRLLGPRGLMPSPKGGTVTDLTAIEFKAGKLEFRADRTGIVHVFGKASFTEDLLNLKALQETVDRNRPSGAKGRYWRSFVSASMGPSIAVDINALRDLKMAVGILGTKLGMTQIFDETGMAIPVTVVQAGPCTVTQVKTPDTDGYTAIQVGYHEVKEKALTKAEIGHLKKIDAPPLRHLKEYRLDDSSQYQLGDAIKADIFNPGDLVDVSGKSMGRGFAGYQKRHNFKRGNMTHGSKNHRLPGSTGAGTTPGRVYPGKRMAGQYGATQVTIRHLTVVRVDSDRNLILVKGAIPGKPGTLLNITPAKTVG-K--MRLKYDTIPKLEQFYKNPHEVPKVVKITVNRGLGEASQNAKALESSINELSITGQKPVVTRAKKAIAGFKIRQGMPVGVMVTLRSDRMYAFLDRLINLALPRIRDFRGISAKSFDGRGNYSLGIREQLIFPEIDYDTIDQIRGMDVSIITTAQTDEEGRALLKELGMPFR-MSRIGKRPIPIPNKVTIDIQGQHVAVKGPKGSLERTLPDKVNVVQENDTLLVQREDDSRTAR--ERHGLTRTLVANMVEGVSKGFEKRLSIQGVGYRAQVQGTKLTLNVGYSKPVEMTMPAGIQVAVENNTLVIVSGIDKEIVGNVSAEIRAVRPPEVYKGKGIRYSDEFVRRKAGKTGKK---MAKKVVAMIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAGVERGSNQPNKTKVGSITQDQLREIAQTKMPDLNANDIEAAMKIVAGTARNMGVTIKEMKTTPTLDQWYVIDADQRLGRLASEVAILRGKNKPFTPHMDTGDFVIIVNAEKVVTGKKSQKLYRRHSGRPGGMKVETFKLQRLPERIVEAVKGMLPKNSLGRKLFTKLKVYTGPHPHAQQPELVNTIPTMIQQTYLNVADNSGARKLMCLRVLGTGNRYGIGDIIAVVKDAPNMIKRSEIVRAVIVRTRVRRASGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNYTKIVSLAPEVLMKLSRKESQRRHRRIRKVGTNCPRLAVFRSNHIYAQIIDDTIAASTVEDLKKLGTEASVGKLVARALAQGIQVVFDRGGNLYHGRVKALADAAREAGLQFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACDPILKVLRSAAANAEHNGLDRALVVSQAYADGPLRRYRPRAQGRAYQIRKPTCHITVAVAPT--------------------------------------MPVVSLAE-----LLESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMENAYEYVRKSSEQGKRFLFIGTKRQAAGIIAQEASRC-GANYVNQRWLGGMLTNWETIKGRVERLKELESMEENGAISRRPKKEAAVLRRELGKLDKYLGGIKTMRKLPDVVVIVDQRREYNAISECQKL--GIPIISMLDTNC--DPDFADIPIPANDDAIRSIKLILGKLADAIYEGRH--GQLDSEQDYEDFDESISD-EYDD--YEDEEEQDLEVDSEDEMGQKIHPIGFRLGITAD----------HKSCWYADSKRYPETLQEDR--QIRQYITKN-L-----SNAGISDIRIERKADQVDLAIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIEQLR----------------------LGLQQDLG--------GNRQIRINVIE-------------------------------------VQRVDADAMLIAEYITQQLERRVSFRRVVRQAIQRAQRA-EVKGIKIQVSG-----------------------------RLNGAEIARTE-----WVREG---RVPLHTLRADIDYAYRTAQTI-------------------------------------------YGILGVKVWVF-----------------KGEIIPGQEE----IPMPV---------------PSQTPRR---QRRRQQFEDRSGEEMSRYRGPRLRVRRLGELPGLSRKNPRRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGVTEGQLIRYVKKARRGSTGLLELLERLDNTVFRLGMAGTIPAARQLVNHGHITVNGKVVDIPSYQCRPGEVIAVRNQSRRLVENMQYPGLANLPHLEFDKTLGKVNAEREWIALQINELLVVEYYSRKA------MAK-RRKS--SRTK---AKETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNENGQVGVGVGKAGDVIGAVRKGVADGKKQLIDVSLTKASSITHITRGVSGGAKVIVRPAAPGTGVIAGGAVRTVLELAGVKNILAKQLGSSNPLNNARAVINALEGLRTFSEVAQERG--VPLEHLYTMSRRRRPVPDYNSRLISMTIRRVMKGKKSVAGIMYDALIEKTGDPLEFEKAIKNLTPLVEVKARRVGGATYQVPMEVRSRGTTLALRWLVHYARIRGRTMSKLAEIMDAANETGGAMKKRDETHRMAEANKAFAHYRYMANDTISDMLTRIRNARHPTTVPTTKMTRSIAQVLKEEGFIGEEGEGQKLVLTLKYKDKNRPIIMLKRVSKPGLRVYSNKLPRVLGGIGIAIVSTSKGIMTDREARRQNVGGEILCYIWM-KVVYWGTGRRKSAIARVRLVPGSGAIVNNKGNYFNRIAYIIKAPLETLGLENDYDILVNAGGGLTGQADAVKLGVARALCELAPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKKNVPNGIAIQSTFNNTIVTIADGDVVSWASAGSSGFKGAKKGTPFAAQTAADAAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDAQGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYLFGIGLSRSQILTGVNPDTRVRDLTDEVLRIENYQIEGDLRREMNIKRLDIGYRGRRHRQGLPVRGQRTRTNARTRRGRTVAGKKKAPMSLSAQKQEVITYQVHETDTGSADLQVALLTTRISQLTGHLQNKDASRRGLLKMIGRRRRLLSYINDQRYQLIRLGIRRMAVKERVGVVVSNKMDKTVVVAIENRSPHPKYGKIVVKTKKFKAHDQENQCQEGDRVRIQETRPLSKTKRWVVAEILTSH--------------',\n",
       " 'Cyanothece_sp._PCC_8802': 'KKLSKRFKAKVQNQYPLEALLLKETATAKFESAEAHIRLGIDPKYTDQQLRTTVSLPKGTGQVRVAVIARGEKVEAAGADIVGSEELIEEIFKGMDFELIATPDMMPKVAKLGRLLGPRGLMPSPKGGTVTDLTAIEFKAGKLEFRADRTGIVHVFGKASFTEDLLNLKALQETVDRNRPSGAKGRYWRSFVSASMGPSIAVDINALRDLKMAVGILGTKLGMTQIFDETGMAIPVTVVQAGPCTVTQVKTPDTDGYTAIQVGYHEVKEKALTKAEIGHLKKIDAPPLRHLKEYRLDDSSQYQLGDAIKADIFNPGDLVDVSGKSMGRGFAGYQKRHNFKRGNMTHGSKNHRLPGSTGAGTTPGRVYPGKRMAGQYGATQVTIRHLTVVRVDSDRNLILVKGAIPGKPGTLLNITPAKTVG-K--MRLKYDTIPKLEQFYKNPHEVPKVVKITVNRGLGEASQNAKALESSINELSITGQKPVVTRAKKAIAGFKIRQGMPVGVMVTLRSDRMYAFLDRLINLALPRIRDFRGISAKSFDGRGNYSLGIREQLIFPEIDYDTIDQIRGMDVSIITTAQTDEEGRALLKELGMPFR-MSRIGKRPIPIPNKVTIDIQGQHVAVKGPKGSLERTLPDKVNVVQENDTLLVQREDDSRTAR--ERHGLTRTLVANMVEGVSKGFEKRLSIQGVGYRAQVQGTKLTLNVGYSKPVEMTMPAGIQVAVENNTLVIVSGIDKEIVGNVSAEIRAVRPPEVYKGKGIRYSDEFVRRKAGKTGKK---MAKKVVAMIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTADQAGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAGVERGSNQPNKTKVGSITQDQLREIAQTKMPDLNANDIEAAMKIVAGTARNMGVTIKEMKTTPTLDQWYVIDADQRLGRLASEVAILRGKNKPFTPHMDTGDFVIIVNAEKVVTGKKSQKLYRRHSGRPGGMKVETFKLQRLPERIVEAVKGMLPKNSLGRKLFTKLKVYTGPHPHAQQPELVNTIPTMIQQTYLNVADNSGARKLMCLRVLGTGNRYGIGDIIAVVKDAPNMIKRSEIVRAVIVRTRVRRASGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNYTKIVSLAPEVLMKLSRKESQRRHRRIRKVGTNCPRLAVFRSNHIYAQIIDDTIAASTVEDLKKLGTEASVGKLVARALAQGIQVVFDRGGNLYHGRVKALADAAREAGLQFMAVDTTVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACDPILKVLRSAAANAEHNGLDRALVVSQAYADGPLRRYRPRAQGRAYQIRKPTCHITVAVAPT--------------------------------------MPVVSLAE-----LLESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAQLMENAYEYVRKSSEQGKRFLFIGTKRQAAGIIAQEASRC-GANYVNQRWLGGMLTNWETIKGRVERLKELESMEENGAISRRPKKEAAVLRRELGKLDKYLGGIKTMRKLPDVVVIVDQRREYNAISECQKL--GIPIISMLDTNC--DPDFADIPIPANDDAIRSIKLILGKLADAIYEGRH--GQLDSEQDYEDFDESISD-EYDD--YEDEEEQDLEVDSEDEMGQKIHPIGFRLGITAD----------HKSCWYADSKRYPETLQEDR--QIRQYITKN-L-----SNAGISDIRIERKADQVDLAIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIEQLR----------------------LGLQQDLG--------GNRQIRINVIE-------------------------------------VQRVDADAMLIAEYITQQLERRVSFRRVVRQAIQRAQRA-EVKGIKIQVSG-----------------------------RLNGAEIARTE-----WVREG---RVPLHTLRADIDYAYRTAQTI-------------------------------------------YGILGVKVWVF-----------------KGEIIPGQEE----IPMPV---------------PSQTPRR---QRRRQQFEDRSGEEMSRYRGPRLRVRRLGELPGLSRKNPRRAYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGVTEGQLIRYVKKARRGSTGLLELLERLDNTVFRLGMAGTIPAARQLVNHGHITVNGKVVDIPSYQCRPGEVIAVRNQSRRLVENMQYPGLANLPHLEFDKTLGKVNAEREWIALQINELLVVEYYSRKA------MAK-RRKG--SRTK---AKETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNENGQVGVGVGKAGDVIGAVRKGVADGKKQLIDVSLTKASSITHITRGVSGGAKVIVRPAAPGTGVIAGGAVRTVLELAGVKNILAKQLGSSNPLNNARAVINALEGLRTFSEVAQERG--VPLEHLYTMSRRRRPVPDYNSRLISMTIRRVMKGKKSVAGIMYDALIEKTGDPLEFEKAIKNLTPLVEVKARRVGGATYQVPMEVRSRGTTLALRWLVHYARIRGRTMSKLAEIMDAANETGGAMKKRDETHRMAEANKAFAHYRYMANDTISDMLTRIRNARHPTTVPTTKMTRSIAQVLKEEGFIGEEGEGQKLVLTLKYKDKNRPIIMLKRVSKPGLRVYSNKLPRVLGGIGIAIVSTSKGIMTDREARRQNVGGEILCYIWM-KVVYWGTGRRKSAIARVRLVPGSGAIVNDKGNYFNRIAYIIKAPLETLGLENDYDILVNAGGGLTGQADAVKLGVARALCELAPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKKNVPNGIAIQSTFNNTIVTIADGDVVSWASAGSSGFKGAKKGTPFAAQTAADAAARRAIDQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDAQGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIGLTYLFGIGLSRSQILTGVNPDTRVRDLTDEVLRIENYQIEGDLRREMNIKRLDIGYRGRRHRQGLPVRGQRTRTNARTRRGRTVAGKKKAPMSLSAQKQEVITYQVHETDTGSADLQVALLTTRISQLTGHLQNKDASRRGLLKMIGRRRRLLSYINDQRYQLIRLGIRRMAVKERVGVVVSNKMDKTVVVAIENRSPHPKYGKIVVKTKKFKAHDQENQCQEGDRVRIQETRPLSKTKRWVVAEILTSH--------------',\n",
       " 'Microcystis_panniformis_FACHB-1757': 'KKVSRRLRAKVENKYPLEALLLKELATAKFETAEAHIRLGIDPKYTDQQLRTTVSLPKGTGRVRVAVITRGEQVEAAGADIVGSEELIEEIAKGMDFELIATPDMMPKVAKLGRQLGPRGLMPSPKGGTVTDLTAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETVDRNRPSGAKGRYWRTYVSSSMGPSIQVDISTLRDLKVSIGILGTKLGMTQIFDKTGVAIPVTVVQAGPCPVTQVKTKKTDGYESIQVGYKTVKEKALNKPLLGHLAKAGVSPLRHLIEYRIEDASAYTLGQEITAAIFQEGDLVDVAGTTIGRGFSGYQKRHNFKRGNMTHGSKNHRLPGSTGAGTTPGRVFPGKRMAGQYGSTQVTIRKLSVVKIDSERNLILIKGAVPGKPGTLLNITPAKKFG-K--MKLKYETIPKLEQFYTNIHQVPKVIKITVNRGLGEASQNAKALESSKEELSITGQQPVVTRAKKAIAGFKIREGMPVGVMVTLRGDRMYAFLDRLINLALPRIRDFRGISGNSFDGRGNYSLGIREQLIFPEIEYDKIDQIRGMDISIITTAKNDEEGRALLKEMGMPFR-MSRIGKRPIPIPNKVTVDIDGATVTVKGPKGTLQRTLPTAVTINKDGETLLVTRQDDSRTAR--ERHGLCRTLVANMVEGVATGFQKRLDIQGVGYRAQAQGSKLVLNVGYSKPVEMEMPDGVSVAVENNTQVIVSGIDKEAVGNIAAKIREVRPPEPYKGKGIRYLGEVVRRKVGKAGKGGKKMAKKVVAIIKLALPAGKANPAPPVGPALGQHGVNIMAFCKDYNAKTADQAGMIIPVEISVFEDRSFTFVLKTPPASVLIRKAAGVERGSNEPNKKFVATITRDQLREIAQTKLPDLNANDIEAAMNIVAGTARNMGVKISDMKTLPTLEQWYVIDADQRLGRLATEIAILRGKNKAFTPHLDTGDFVIVINAEKVVTGKKQQKVYRRDSGRPGGMKVESFKLQRIPERIIEAVKGMLPKNSLGRKLFTKLKVYAGAHPHAQQPELINTIPAVIQQTYLNVADNSGARKLMCLRVLGTGNTYGIGDIIAVVKDAPNMVKKSDVVTAVIVRTRVRRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNYTKIVSLAPEVLMKLSRKESRRRHQRVRKIGTERPRLSVFRSNHIYAQIIDDTLAASTLETLRGLTTEASVGKLVARALERGIQVVFDRGGNLYHGRVKALAEAARSAGLNFMTIDTSVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACDPILKVLRSAVANAENNGLDPALVVSQAFADGPLKRFRPRAQGRAYQIRKPTCHITVAVAPK--------------------------------------MPVVSLAE-----LLESGVHFGHQTRRWNPKMSQYIY-TARNGVHIIDLVQTAQLIEEAYEFVRGEADRGKRFLFIGTKRQAAAIIKQEALRS-GSHFVNQRWLGGMLTNWETIRGRVERLKELEELENSGALDKRPKKEASVLRRELGKLEKYLGGIKTMRRLPDLVIVVDQRREYNAIQECQKL--GIPIISLLDTNC--DPDLVDVPIPANDDAIRSVKLILGKISDAIIEGRR-GGQAAVE-EYEDYEDET---EYEEEDYSQYFASGDDDN----MGQKIHPLGFRLGVIKD----------HKSCWYADAKRYPELVQEDR--RIREYVEKN-L-----ANAGIADIRIERKADQVDISIH-------TARPGVVVGRGG-----------------------------------------------------------------------TGIEQLR----------------------LGLQKTLG--------SQRQIRINVIE-------------------------------------VARVDADANLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-EVKGIKIQVSG-----------------------------RLNGAEIARTE-----WVREG---RVPLHTLRADIDYSYKTASTI-------------------------------------------YGILGVKVWIF-----------------KGEIIPGQEE--IAAAVPA-----------------QAPRR--QQRRRQQFEDRSSEGMSRYRGPRLRVRRLGELPGLTRKNARRAYAPGQHGQRKKRSEYAIRLEEKQKLRFNYGVSEKQLVRYVRKARRGSTGLLQLLERLDNTVFRLGMAGTIPGARQLVNHGHVTVNGRVVDIASYQCRPGDVIGVRNQSQDLVKNMEYPGLANLPHLEFDKTLGKVNGEREWIALSINELLVVEYYSRKV------MAK-RRKG--NREK---EKETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNENGQVGVGVGKAGDVIGAVRKGVADGKKQLIEVSLTKASSITHLTRGASGGAQVLMRPAAPGTGVIAGGAVRTVLELAGVKNILAKQLGSDNPLNNARAAVNALETLRTFSEVAKDRG--VSIEHLYTMSRRKRPIPDYNSCLVSMTIRRIMRGKKSLANIVYDALVERTGEPLEFEKAVKNATPLVEVKARRVGGATYQVPMEVRNRGSTLALRWLVHYARTRGKTMAKLAEIMDAANETGGTIKKREETHRMAEANKAFAHYRYMSNDTISDMLTRIRNARQTTTIPTTRMTRSIAQVLKNEGFIDEEGEGKSLVLSLKYKGKNRPIITLTRVSKPGLRVYSNKLPRVLGGIGIAIVSTSRGIMTDREARKQNVGGEVLCYIWMKKVVYWGTGRRKSAVASVRLVPGTGAIVNGRGEHFNRIPYIIKAPLETLGLENEYDIIVKAGGGLTGQADAVKLGVARALCQLAPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKKNIPTGVAIQSTFNNTIVTIADGDVISWASAGSSGFKGAKKGTPFAAQTAADNAARRAIEQGMRQLEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKRKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRAVGSKIYP-------------------ESDITSFAAPSMPVARIAGVDLPRDKRVEIALTYLYGVGLSRSQVLTGVNPDTRVKDLSDDVLRIENYQIEGDLRREMNIKRLDIGYRGRRHRLGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQKKQELISYQAHETDTGSSELQVAFLTERINQLTEHLKNKDASRRGLLQMIGRRRGLLTYIQDQRYQLIRLGIRRMAVKERVGVVVSDKMDKTVVVAIENRSPHPKYGKIVVKTQKFKAHDEENQAKQGDRVRIRETRPLSKTKRWQVAEILTDN--------------',\n",
       " 'Microcystis_aeruginosa_NIES-2549': 'KKVSRRLRAKVENKYPLEALLLKELATAKFETAEAHIRLGIDPKYTDQQLRTTVSLPKGTGRVRVAVITRGEKVEAAGADIVGSEELIEEIAKGMDFELIATPDMMPKVAKLGRQLGPRGLMPSPKGGTVTDLTAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETVDRNRPSGAKGRYWRTYVSSSMGPSIQVDISTLRDLKVSIGILGTKLGMTQIFDKTGVAIPVTVVQAGPCPVTQVKTKKTDGYESIQVGYKTVKEKALNKPLLGHLAKAGVSPLRHLIEYRLEDASAYTLGQEITAAIFQEGDLVDVAGTTIGRGFSGYQKRHNFKRGNMTHGSKNHRLPGSTGAGTTPGRVFPGKRMAGQYGSTQVTIRKLSVVKIDSERNLILIKGAVPGKPGTLLNITPAKKFG-K--MRLKYETIPKLEQFYTNIHQVPKVIKITVNRGLGEASQNAKALESSKEELSITGQQPVVTRAKKAIAGFKIREGMPVGVMVTLRGDRMYAFLDRLINLALPRIRDFRGISGNSFDGRGNYSLGIREQLIFPEIEYDKIDQIRGMDISIITTAKNDEEGRALLKEMGMPFR-MSRIGKRPIPIPNKVTVDIDGATVTVKGPKGTLQRTLPTAVAIDKDGETLLVTRQDDSRTAR--ERHGLCRTLVANMVEGVATGFQKRLDIQGVGYRAQAQGSKLVLTVGYSKQVEMEMPDGVSVAVENNTQVIVSGIDKEAVGNIAAKIREVRPPEPYKGKGIRYLGEVVRRKVGKAGKGGKKMAKKVVAIIKLALPAGKANPAPPVGPALGQHGVNIMAFCKDYNAKTADQAGMIIPVEISVFEDRSFTFVLKTPPASVLIRKAAGVERGSNEPNKKFVATITRDQLREIAQTKMPDLNANDIEAAMNIVAGTARNMGVKISDMKTLPTLEQWYVIDADQRLGRLATEIAILRGKNKAFTPHLDTGDFVIVINAEKVVTGKKQQKVYRRDSGRPGGMKVESFKLQRIPERIIEAVKGMLPKNSLGRKLFTKLKVYAGAHPHAQQPELINTIPAVIQQTYLNVADNSGARKLMCLRVLGTGNTYGIGDIIAVVKDAPNMVKKSDVVTAVIVRTRVRRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNYTKIVSLAPEVLMKLSRKESRRRHQRVRKIGTERPRLSVFRSNHIYAQIIDDTLAASTLETLRGLTTEASVGKLVARALERGIQVVFDRGGNLYHGRVKALAEAARSAGLNFMTIDTSVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACDPILKVLRSAVANAENNGLDPALVVSQAFADGPLKRFRPRAQGRAYQIRKPTCHITVAVAPK--------------------------------------MPVVSLAE-----LLESGVHFGHQTRRWNPKMSQYIY-TARNGVHIIDLVQTAQLIEEAYEFVRGEADRGKRFLFIGTKRQAAGIIKQEALRS-GSHFVNQRWLGGMLTNWETIRGRVERLKELEELENSGALDKRPKKEASVLRRELGKLEKYLGGIKTMRRLPDLVVVVDQRREYNAIQECQKL--GIPIISLLDTNC--DPDLVDVPIPANDDAIRSVKLILGKISDAIIEGRR-GGQAAVE-EYEDYEDET---EYEQEDYSQYFASGDDDN----MGQKIHPLGFRLGVIKD----------HKSCWYADAKRYPELVQEDR--RIREYVEKN-L-----ANAGIADIRIERKADQVDISIH-------TARPGVVVGRGG-----------------------------------------------------------------------TGIEQLR----------------------LGLQKALG--------GQRQIRINVIE-------------------------------------VARVDADANLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-EVKGIKIQVSG-----------------------------RLNGAEIARTE-----WVREG---RVPLHTLRADIDYSYKTASTI-------------------------------------------YGILGVKVWIF-----------------KGEIIPGQEE--IAAAVPA-----------------QAPRR--QQRRRQQFEDRSSEGMSRYRGPRLRVRRLGELPGLTRKNARRAYAPGQHGQRKKRSEYAIRLEEKQKLRFNYGVSEKQLVRYVRKARRGSTGLLQLLERLDNTVFRLGMAGTIPGARQLVNHGHVTVNGRVVDIASYQCRPGDVIGVRNQSQDLVKNMEYPGLANLPHLEFDKTLGKVNGEREWIALSINELLVVEYYSRKV------MAK-RRKG--NREK---EKETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNENGQVGVGVGKAGDVIGAVRKGVADGKKQLIEVSLTKASSITHLTRGASGGAQVLMRPAAPGTGVIAGGAVRTVLELAGVKNILAKQLGSDNPLNNARAAVNALETLRTFSEVAKDRG--VSVEHLYTMSRRKRPIPDYNSCLVSMTIRRIMRGKKSLANIVYDALVERTGEPLEFEKAVKNATPLVEVKARRVGGATYQVPMEVRNRGSTLALRWLVHYARTRGKTMAKLAEIMDAANETGGTIKKREETHRMAEANKAFAHYRYMSNDTISDMLTRIRNARQTTTIPTTRMTRSIAQVLKDEGFIDEEGEGKSLVLSLKYKGKNRPIITLTRVSKPGLRVYSNKLPRVLGGIGIAIVSTSKGIMTDREARKQNVGGEVLCYIWMKKVVYWGTGRRKSAVASVRLVPGTGAIVNGRGEHFNRIPYIIKAPLETLGLENEYDIIVKAGGGLTGQADAVKLGVARALCQLAPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKKNIPTGVAIQSTFNNTIVTIADGDVISWASAGSSGFKGAKKGTPFAAQTAADNAARRAIEQGMRQLEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKRKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDATGVKNRQKARSKYGTKRPKVARIAGVDLPRDKRVEIALTYLYGVGLSRSQVLTGVNPDTRVKDLSDDVLRIENYQIEGDLRREMNIKRLDIGYRGRRHRLGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQKKQELISYQAHETDTGSSELQVAFLTERINQLTEHLKNKDASRRGLLQMIGRRRGLLTYIQDQRYQLIRLGIRRMAVKERVGVVVSDKMDKTVVVAIENRSPHPKYGKIVVKTQKFKAHDPENQAKQGDRVRIRETRPLSKTKRWQVAEILTDN--------------',\n",
       " 'Microcystis_aeruginosa_NIES-843': 'KKVSRRLRAKVENKYPLEALLLKELATAKFETAEAHIRLGIDPKYTDQQLRTTVSLPKGTGRVRVAVITRGEKVEAAGADIVGSEELIEEIAKGMDFELIATPDMMPKVAKLGRQLGPRGLMPSPKGGTVTDLTAIEFKAGKLEFRADRTGIVHVFGKASFSEDLLNLKALQETVDRNRPSGAKGRYWRTYVSSSMGPSIQVDISALRDLKMSIGILGTKLGMTQIFDKTGVAIPVTVVQAGPCPVTQVKTKKTDGYESIQVGYKTVKEKALNKPLLGHLAKAGVSPLRHLIEYRLEDASAYTLGQEITADIFQEGDLVDVAGTTIGRGFSGYQKRHNFKRGNMTHGSKNHRLPGSTGAGTTPGRVFPGKRMAGQYGSTQVTIRKLSVVKIDSERNLILIKGAVPGKPGTLLNITPAKKFG-K--MKLKYETIPKLEQFYTNIHQVPKVIKITVNRGLGEASQNAKALESSKAELSITGQQPVVTRAKKAIAGFKIREGMPVGVMVTLRGDRMYAFLDRLINLALPRIRDFRGISGNSFDGRGNYSLGIREQLIFPEIEYDKIDQIRGMDISIITTAKNDEEGRALLKEMGMPFR-MSRIGKRPIPIPNKVTVDIDGATVTVKGPKGTLQRTLPTAVAINKDGETLLVTRQDDSRTAR--ERHGLCRTLVANMVEGVATGFQKRLDIQGVGYRAQAQGSKLVLNVGYSKPVEMEMPDGVSVAVENSTQVIVSGIDKEAVGNTAAKIREVRPPEPYKGKGIRYLGEVVRRKVGKAGKGGKKMAKKVVAIIKLALPAGKANPAPPVGPALGQHGVNIMAFCKDYNAKTADQAGMIIPVEISVFEDRSFTFVLKTPPASVLIRKAAGVERGSNEPNKKFVATITRDQLREIAQTKMPDLNANDIEAAMNIVAGTARNMGVKISDMKTLPTLEQWYVIDADQRLGRLATEIAILRGKNKAFTPHLDTGDFVIVINAEKVVTGKKQQKVYRRDSGRPGGMKVESFKLQRIPERIIEAVKGMLPKNSLGRKLFTKLKVYAGAHPHAQQPELINTIPAMIQQTYLNVADNSGARKLMCLRVLGTGNTYGIGDIIAVVKDAPNMVKKSDVVTAVIVRTRVRRDSGSIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNYTKIVSLAPEVLMKLSRKESRRRHQRVRKIGTERPRLSVFRSNHIYAQIIDDTLAASTLETLRGLTTEASVGKLVARALDKGIQVVFDRGGNLYHGRVKALAEAARSAGLNFMTIDTSVKAARYIRSPKVRRVLDQIRGRSYREALIILEFMPYRACDPILKVLRSAVANAEHNGLDPALVVSQAFADGPLKRFRPRAQGRAYQIRKPTCHITVAVAPK--------------------------------------MPVVSLAE-----LLESGVHFGHQTRRWNPKMSQYIY-TARNGVHIIDLVQTAQLIEEAYEFVRGEADRGKRFLFIGTKRQAAAIIKQEALRS-GSHFVNQRWLGGMLTNWETIRGRVERLKELEELENSGALDKRPKKEASVLRRELGKLEKYLGGIKTMRRLPDLVVVVDQRREYNAIQECQKL--GIPIISLLDTNC--DPDLVDVPIPANDDAIRSVKLILGKISDAIIEGRR-GGQAAVE-EYEDYDNET---EYEEEDYSQYFASGDDDN----MGQKIHPLGFRLGVIKD----------HKSCWYADAKRYPELLQEDR--RIRQYVEKN-L-----ANAGIADIRIERKADQVDISIH-------TARPGVVVGRGG-----------------------------------------------------------------------TGIEQLR----------------------LGLQKALG--------GQRQIRINVIE-------------------------------------VARVDADANLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-EVKGIKIQVSG-----------------------------RLNGAEIARTE-----WVREG---RVPLHTLRADIDYSYKTASTI-------------------------------------------YGILGVKVWIF-----------------KGEIIPGQEE--IAAAVPA-----------------QAPRR--QQRRRQQFEDRSSEGMSRYRGPRLRVRRLGELPGLTRKNARRAYAPGQHGQRKKRSEYAIRLEEKQKLRFNYGVSEKQLIRYVRKARRGSTGLLQLLERLDNTVFRLGMAGTIPGARQLVNHGHVTVNGRVVDIASYQCRPGDVIGVRNQSQDLVKNMEYPGLANLPHLEFDKTLGKVNGEREWIALSINELLVVEYYSRKV------MAK-RRKG--NREK---EKETTWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNENGQVGVGVGKAGDVIGAVRKGVADGKKQLIEVSLTKASSITHLTRGASGGAQVIMRPAAPGTGVIAGGAVRTVLELAGVKNILAKQLGSDNPLNNARAAVNALETLRTFSEVAKDRG--VSIEHLYTMSRRKRPIPDYNSCLVSMTIRRIMRGKKSLANIVYDALVERTGEPLEFEKAVKNATPLVEVKARRVGGATYQVPMEVRNRGSTLALRWLVHYARTRGKTMAKLAEIMDAANETGGTIKKREETHRMAEANKAFAHYRYMSNDTISDMLTRIRNARQTTTIPTTRMTRSIAQVLKDEGFIDEEGEGKSLVLSLKYKGKNRPIITLTRVSKPGLRVYSNKLPRVLGGIGIAIVSTSKGIMTDREARKQNVGGEVLCYIWMKKVVYWGTGRRKSAVAAVRLVPGTGAIVNGRGEHFNRIPYIIKAPLETLGLENEYDIIVKAGGGLTGQADAVKLGVARALCQLAPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKTGKKQKKNIPTGVAIQSTFNNTIVTIADGDVISWASAGSSGFKGAKKGTPFAAQTAADNAARRAIEQGMRQLEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERKRKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDATGVKNRQKARSKYGTKRPKMARIAGVDLPRDKRVEIALTYLYGVGLSRSQVLTGVNPDTRVKDLSDDVLRIENYQIEGDLRREMNIKRLDIGYRGRRHRLGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQKKQELISYQAHETDTGSSELQVAFLTERINQLTEHLKNKDASRRGLLQMIGRRRGLLTYIQDQRYQLIRLGIRRMAVKERVGVVVSDKMDKTVVVAIENRSPHPKYGKIVVKTQKFKAHDAENQAKQGDRVRIRETRPLSKTKRWEVAEILTDS--------------',\n",
       " 'Pleurocapsa_sp._PCC_7327': 'KKVSRRLRAKVEDKYPLEALLLKKTATAKFETAEAHIRLGIDPKYTDQQLRTTVTFPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQGGLDFELIATPDMMPKVAKLGRLLGPRGLMPSPKGGTVTDLAAIEFKAGKQEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRSFVSASMGPSIQVDINALRDLKMALGILGTKLGMTQIFDETGIAIPVTVVQAGPCPITQIKTPETDGYNSIQVGYQEVKEKALSKPEIGHLSKSGASPLRHLKEYRLDEPSQYQLGSAIKVDIFSVGDLVDVAGKTIGRGFAGYQKRHNFKRGSMTHGSKNHRLPGSTGAGTTPGRVFPGKRMAGRYGNTQVTIRRLQVVRVDAERNLLLIKGAVPGKPGSLLSITPAKKVG-K--MRLKYETIPKLEQFYSNIHEVPKVVKVTVNRGLGEASQNAKALESSISELAITGQKPIVTRAKKAIAGFKIRQGMPVGVMVTLRSERMYAFLDRLINLALPRIRDFRGINPNSFDGRGNYSLGIREQLIFPEIDYDSIDQIRGMDISIITTAKTDEEGRALLKAMGMPFR-MSRIGKRPIAIPNKVTVEINGLNVVVKGPKGTLERTLPPQVKVEQEGNTLKVLRRDESRVAR--ERHGLSRTLVANMVEGVSQGFQKRLDIQGVGYRAQAQGSKLTLNVGYSKPVEMMMPKDIQVAVENNTQVVISGIDKEIVGNIAAQIRGVRPPEPYKGKGIRYQGEFVRRKAGKTGKK---MAKKVVALIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTADQAGMIIPVEISVFEDRSFTFVLKTPPASVLIRKAAGIERGSNEPNKKIVATITRDQLRQIAETKMPDLNANDIEAAMKIVAGTAKNMGVNIKETKTLPTIEPWYVIDADQRLGRLASEIAILRGKNKPFTPHMDTGDFAIVINAEKVVTGKKKQKVYRRHSGRPGGMKTETFQLQRIPERIIEAVKGMLPKNSLGRKLFTKLKVYAGSHPHAQKPELINTIPTMIQQTYLNVADNSGARKLMCLRVLGTGNTYGIGDIIAVVKDAPNMIKKSDVVRAVIVRTRLRRESGTIRFDDNAAVIINEGNPRGTRVFGPVARELRDKNYTKIVSLAPEVIMKPTRVELKRRHKRLRKVGTERPRLAVFRSHHIYAQIIDDTLAASTLETLKTLGTEASVGKLVARALEKGIKVVFDRGGNLYHGRVKALAEAAREAGLSFMAVDTTVRAARFVRSPKVRRVLDQIRGRSYREALIVLEFMPYKACEPVLKALRSAVANAEHNGLDPALVVSKAYADGPLRRYRPRAQGRAYQIRKPTCHITVAVAPD--------------------------------------MPVVSLAE-----LLESGVHFGHQTRRWNPKMSQYIY-TARNGVHIIDLVQTAQLMEEAYDYVRKSAEAGKRFLFIGTKRQAAGIIAQEASRC-GANYVNQRWLGGMLTNWETIKSRLERLKELEALEENGSIAKRPKKEASVLRRELAKLQKYLSGLKTMRKLPDIAVIVDLRREYNAIRECQKL--GIPIVSLLDTNC--DPDVVDVPIPANDDAIRSIKLVIGKLADAIYEGRH--GQLDTDEDYEEFEESIEETDYEEEDYSDLYANEDEDEN---MGQKIHPIGFRLGITKE----------HKSRWYADTKRYPELLQEDR--RIRQYVDKN-L-----SNAGIADIRIERKADQIDLAIH-------TARPGVVVGRGG-----------------------------------------------------------------------TGIEALR----------------------VGLQEELG--------GTRQIRINVIE-------------------------------------VPRVDADATLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-EVKGIKVQVSG-----------------------------RLNGAEIARTE-----WIREG---RVPLHTLRADIDYSYRTAQTI-------------------------------------------YGILGIKVWVF-----------------KGEIIPGQEE--PATPPPA-----------------QAPRR---QRRRQQFEDRSNE-MSRYRGPRLRVRRLGELPGLTRKSARRSYPPGQHGQRKKRSEYAIRLEEKQKLRFNYGVTERQLLRYVRKARRGSTGLLQLLERLDNTVFRLGMAGTIPAARQLVCHGHITVNGKVVDIPSYQCRPGDVIAVRDRSRRLVQNMEYPGLANLPHLEFDKTLGKVNSEREWVALSINELLVVEYYSRQA------MAK-RQKS--SRTK---TKETNWKERVIQIRRVSKVVKGGKKLSFRAIVVVGNESGQVGVGVGKAGDVIGAVRKGVADGKKQLIEVPLTKASSITHITRGASGGAKVIMRPAAPGTGVIAGGAVRTVLELAGVKNILAKQLGSNNPLNNARAAVDALSTLRTFSEVAQDRG--VPVEHLYTMSRRRRTIPDYNSRLVNMTIRRVMKGKKSLASIVYDALIERTSDPIEFEKAIKNLTPLVEVKARRVGGATYQVPMEVRERGTTLALRWLINYSRNRGKTMAKLAEIMDAANETGGAIKKREETHRMAEANKAFAHYRYMAIDTISDMLTRIRNARHPTTVPITKMTRSIAKVLKDEGFIDEEGEGNKLVISLKYKGKNRPLITLKRVSKPGLRVYSGKLPRVLGGIGIAIVSTSKGIMTDREARKQKVGGEVLCYIWMQKVVYWGTGRRKSSVARVRLVPGSGQLVNDKGEYFNRIPYLIKAPLETLGLENEYDILVKAGGGLTGQSDAVKLGVARALCQLAPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKSGKKQKKNVPNGIAIQSTFNNTIVTISDGDVISWASAGSSGFKGAKKGTPFAAQTAADQAARRAIEQGMRQIEVMVSGPGAGRETAIRALQSGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERNKKTKSPALKCPQRRGVCTRVFTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDATGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIALTYLYGIGLSRSQILTGVNPDIRVKDLTDDILRIENYQIEGDLRREMNIKRLDIGYRGRRHRLGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQELMTYQAHETDTGSADLQVALLTERINQLNSHLQNKDASRRGLLKMIGRRKSLLAYIQDNRYQLIRLGIRRMAVKERVGVVVSNKMEKTVIIAVENRSPHPKYGKIVVKTKKFKAHDEENRCREGDRVRIRETRPLSRTKRWIVTEILNTDTP------------',\n",
       " 'Cyanothece_sp._PCC_7822_chromosome': 'KKISRRLKAKVENKYPLEALLLKETATAKFETAEAHVRLGIDPKYTDQQLRTTVTFPKGTGQVRVAVIARGEKVEAGGADIVGSEELIDEIQNGMDFELIATPDMMPKVAKLGRLLGPRGLMPSPKGGTVTDLAAIEFKAGKQEFRADRTGIVHVFGKASFSEDLLNLKALQETIDRNRPSGAKGRYWRTYVSASMGPSIQVDVNSLRDLKMSVGIIGTKLGMTQIFDETGIAIPVTVIHAGPCPITQIKTPETDGYSSIQIGYGEVKEKALNKPLLGHLKKSESIPLKNLKEYRLDNPEQYQLGDALKVDIFQPGDIVDVAGKTIGRGFAGYQKRHNFKRGSMTHGSKNHRLPGSTGAGTTPGRVYPGKRMAGRYGGTQVTIRKLTVVQVDPERNLILIKGAVPGKPGNLLSITPAKKVG-K--MRLKYETIPKLEQFYKNIHQVPKVIKVTVNRGLGEASQNAKALESSINELSITGQKPVVTRAKKAIAGFKIRKGMPVGVMVTLRQERMYAFLDRLINLALPRIRDFRGISGNSFDGRGNYSLGIREQLIFPEIDYDKIEQIRGMDISIITTARTDEEGRALLTEMGMPFR-MSRIGKRPIEIPNKVTVNIEGSHVAVKGPKGELERTLPSMVNVELTDNTLRVVRTDDSRVAR--QRHGLCRTLVFNMVEGVSNGFEKRLEIQGVGYRAAAAGSKLTLNVGYSKPVEMDMPQGITVAVEKNTEITISGIDKEIVGNIAAKIRAVRPPEPYKGKGIRYKGELVRRKAGKTGKK---MAKKIVAIIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNAKTSDQAGMVVPVEISVFEDRSFTFILKTPPASVLIRKAAGVERGSNEPNKKNVGSITNDQLREIAQTKLPDLNANDIEAAMKIVAGTARNMGISIKDMKTLPTLDKWYLIDAGQRLGRLASEIAILRGKNKPYTPHLDTGDFVIVINAEKVVTGKKQQKLYRRHSGRPGGMKLETFKLQRLPERIVEAVKGMLPKNTLGRQLFTKLKVYDGPHPHAQKPELINTIPGMIQQTYLNVADNSGARKLMCLRVLGTGNRYGIGDIIAVVKDAPNMVKKSDVVKAVIVRTRLRRSSGTIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNYTKIVSLAPEVLMKRTRKESQRRHRRIRTVGSERPRLAVFRSNHIYAQVIDDTLAASSLDELKTLGNDASVGKLVARALAKGIAVVFDRGGNLYHGRVKALAEAAREAGLDFMAVDTTVKAARYVRSPKVRRVLDQIRGRTYQEALIILEFMPYRACDPILKVLRSAVANAEHNGLDRALVISKAYADGPLRRYRARAQGRAYQIRKPTCHITISVAPT--------------------------------------MSVVTLNE-----LLESGVHFGHQTRRWNPKMAPYIY-TARNGVHIIDLVQTAELIEEAYDYMRKAADQGKKFLFVGTKRQAAGIIAQEALRC-GSNFVNQRWLGGMLTNWETIRTRAERLKELEEMEESGAISRRPKKEAAVLRRELFKLQKYLGGIKTMRRIPDVVVIIDLKREYNAVQECQKL--GIPIVALLDTNC--DPDYVDVPIPGNDDAIRSIKLIIGKLADAIYEGRH--GEVAPDEDYEDFDENLEN-DYEEENYSQASKYSDDDESDR-MGQKIHPVGFRLGITKE----------HKSRWYADKHNYPELLQEDR--KIRSYIDAN-L-----SNAGISDVRIERKADQVDIEIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIESLR----------------------LGLQTALG--------GHRQIRINVIE-------------------------------------VSRVDADATLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-EVKGIKVQVSG-----------------------------RLNGAEIARTE-----WIREG---RVPLHTLRADIDFSYRTAKTI-------------------------------------------YGILGVKVWVF-----------------KGEIIPGQEE----TPPPP--------------SNAPAPRR---QRRRQQFEDRSNE-MSRYRGPRLRVRRLGELPGLTRKNARRSYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSEKQLIRYVRKARRGSTGLLQLLERLDNTIFRLGMAGTIPAARQLVNHGHILVNDQVVDIASYQCRPGDVIKVKNQSRKLVENMEYPGLANLPHLEFDKTFGKVNGEREWIALNINELLVVEFYSRQA------MAK-RRKS--SRTK---EKETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNETGQVGVGVGKAGDVIGAVRKGVADGKKQLIEVPLTKASSIPHLTNGASGGASVIMRPAAPGTGVIAGGAVRTVLELAGVKNILAKQLGSDNPLNNARAAINALETLRTFSEVAKERD--LPIEHIYSMSRRRRPIPDFNSYLVNMTIRRVMRGKKSLATIVYDALVERTGDALEFEQAVKNITPLVEVKARRVGGATYQVPMEVRGRGTTLALRWLISFSRNRGRTMAKLAEIMDAANETGGAIKKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHPTTVPTTKMTRSIALVLKQEGFIDSEGEGAKLLISLKYKDKNRPIITLKRVSKPGLRVYAPKLPRVLGGIGIAIVSTSKGIMTDREARRQGIGGEVLCYIWMRRVVYWGTGRRKTAIARVRLIPGTGQIVNGKGDYFNQIAYIIKAPLETLGLENDYDILVKAGGGLTGQADAVKLGVARALCQLAPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKGGKKQKKNVPNGIAIQSTFNNTIVTIADGDVISWASAGSSGFKGAKKGTPFAAQTAADNAARRAMEQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERQKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDSTGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIALTYLYGIGLSRSQILTGVNPDTRVKDLSDDTLRIENYQIEGDLRREMNIKRLDIGYRGRRHRQGLPVRGQRTRTNARTRRGRTVAGKKKAPMTLTQEKQEIINYQAHGTDTGSADLQVALLTERINQLTTHLQNKDASRRGLLQLIGRRRRLLAYIQDPRYQLIRLGIRRMAVKERVGVVVSDKMDKTVVVAIENRTRHPKYRKIVVQTKKYKAHDEENQCKTGDRVRIQETRPLSKTKRWIVAEILTHVAGT-----------',\n",
       " 'Cyanothece_sp._PCC_7424': 'KKLSRRFKAKVQDKYPLEALLLKETATAKFETAEVHVRLGIDPKYTDQQLRTTVTFPKGTGQVRVAVIARGEKVEAAGADIVGSEELIDEIQNGMDFELIATPDMMPKVAKLGRLLGPRGLMPSPKGGTVTDLAAIEFKAGKQEFRADRTGIVHVFGKASFTEDLLNLKALQETIDRNRPSGAKGRYWRTFVSASMGPSIQVDVNSLRDLKMSVGILGTKLGMTQIFDETGIAIPVTVIHAGPCPITQVKTPATDGYNSIQIGYGEVKEKALNKPKLGHLKKSASTPVRHLKEYRLENAADYQLGDGIKVDLFKPGDLVDVAGKTIGRGFAGYQKRHNFKRGSMTHGSKNHRLPGSTGAGTTPGRVYPGKRMAGRYGGTQVTIRKLTVVQIDTERNLILIKGAVPGKPGNLLSITPAKTVG-K--MRLKYETIPKLEQFYTNIHQVPKVIKVTVNRGLGDASQNAKALESSIAELAITGQKPVVTRARKAIAGFKIRKGMPVGVMVTLRSERMYAFLDRLISLALPRIRDFRGISGNSFDGRGNYSLGIREQLIFPEIDYDKIDQIRGMDISIITTAQTDEEGRALLTEMGMPFR-MSRIGKRPITIPNKVTVNIDGSHVAVKGPKGELQRTLPTLVIVEQDEGVLRVVRQDDSRTAR--QRHGLCRTLVSNMVEGVSKGFEKRLEIQGVGYRAAAQGSKLTLNVGYSKPVEMDMPSGIGVAVEKNTEITITGIDKEIVGNVAAKIRAVRPPEPYKGKGIRYKGEVVRRKAGKTGKK---MAKKIVAIIKLALPAGKANPAPPVGPALGQHGVNIMAFCKEYNGKTSDQVGMVIPVEISVFEDRSFTFVLKTPPASVLIRKAAGVERGSNEPNKKNVGSITQDQLREIAQTKMPDLNANDIEAAMKIVAGTARNMGISVKDMKTLPTLDKWYVIDAGQRLGRLASEIAILRGKNKPYTPHLDTGDFVIVINAEKVVTGKKQQKVYHRHSGRPGGMKVETFKLQRLPERIVEAVKGMLPKNSLGRQLFTKLKVYSGSHPHAQKPELINTIPGMIQQTYLNVADNSGARKLMCLRVLGTGNRYGIGDIIAVVKDAPNMVKKSDVVKAVIVRTRLRRASGTIRFDDNAAVIINDGNPKGTRVFGPVARELRDKNYTKIVSLAPEVLMKRTRTESQRRHSRIRKVGTSRPRLAVFRSNHIYAQVIDDTLAASTLDDLKGFGTEASVGKLVARALAKGIKVVFDRGGNLYHGRIRALAQAAREAGLDFIAVDTTVKAARYVRSPKVRRVLDQIRGRTYREALIILEFMPYKACDVILKLLRSAVANAEHNGLDPTLVVSKAYADGPLRRFRARAQGRAYQIRKPTCHITLFVAPE--------------------------------------MSVVSLNE-----LLESGVHFGHQTRRWNPKMSPYIY-TARNGVHIIDLVQTAELMEEAYDYMRKASDQGKKFLFVGTKRQAAGIIAQEALRC-GSNYVNQRWLGGMLTNWETIRTRAERLKELEEMEESGALARRPKKEAAVLRREMYKLQKYLGGIKTMRRIPDVVIIIDLKREYNAVQECQKL--GIPIVALLDTNC--DPDYVDVPIPANDDAIRSIKLIIGKLADAIYEGRH--GEVASDEDYEEFDESL-E-EDDYEQQQDWYSSDDE---DEEMGQKIHPVGFRLGITKE----------HKSRWFADKYHYPELLQEDR--KIRQYVEKN-L-----SNAGISDIRIERKADQVDIEIH-------TARPGVVVGRGG-----------------------------------------------------------------------SGIEALR----------------------VGLQNALG--------GHRQIRINVVE-------------------------------------VARVDADATLIAEYIAQQLERRVSFRRVVRQAIQRAQRA-EVKGIKVQVSG-----------------------------RLNGAEIARTE-----WIREG---RVPLHTLRADIDFAYRTAKTI-------------------------------------------YGILGVKIWVF-----------------KGEIIPGQEE---QAPAQP----------------ATTPKR---QRRRQQFEDRSNE-MARYRGPRLRVRRLGELPGLTRKNARRAYPPGQHGQRRKRSEYAIRLEEKQKLRFNYGVSEKQLIRYVRRARRGSTGLLQLLERLDNTVFRLGMAGTIPAARQLVNHGHILVNDRVVDIASYQCRPGDVIKVRDQSRKLVQNMEYPGLANLPHLEFDKTLGKVNGEREWIALNINELLVVEYYSRQA------MAK-RRKS--SRTK---EKETNWQERVIQIRRVSKVVKGGKKLSFRAIVVVGNETGQVGVGVGKAGDVIGAVRKGVADGKKQLIEVPLTKASSIPHLTNGASGGASVIMRPAAPGTGVIAGGAVRTVLELAGVKNILAKQLGSDNPLNNARAAINALETLRTFSEVAKERD--VPLEHIYSMSRRKRPIPDYNSYLINMTIRRVMRGKKSLANIIYDALVERTGDPLEFEQAVKNATPLVEVKARRVGGATYQVPMEVRGRGTTLALRWLIQFSRSRGRTMAKLAEIMDAANETGGAIKKREETHRMAEANKAFAHYRYMANDTIADMLTRIRNARHPTTVPTTKMTRSIAQVLKQEGFIDSEGEGGKLVISLKYKGKTRPIITLKRVSKPGLRVYSPKLPRVLGGIGIAIVSTSQGIMTDREARRQGVGGEVLCYIWMKRAVYWGTGRRKTSVARVRLIPGTGQVVNGKGDYFNRIAYLIKAPLETLGLENEYDILVKAGGGLTGQADAVKLGVARALCELAPENRPLKEGYLTRDPRAKERKKYGLKARKAPQYSKRMARKKGGKKQKKNVPNGIAIQSTFNNTIVTIADGDVISWASAGSSGFKGAKKGTPFAAQTAADNAARRAMEQGMRQIEVMVSGPGAGRETAIRALQAGLEITLIRDVTPIPHNGCRPPKRRRVMPTIQQLIRERQKKTKSPALKCPQRRGVCTRVYTTTPKKPNSALRKVARVRLTSGFEVTAYIPGIGHNLQEHSVVLIRGGRVKDLPGVRYHIIRGTLDATGVKDRKQGRSKYGTKRPKMARIAGVDLPRDKRVEIALTYLYGIGLSRSKILTGVNPDTRVKDLSDDALRIENYQIEGDLRREMNIKRLDIGYRGRRHRQGLPVRGQRTRTNARTRRGRTVAGKKKAPMALTQRKQEIINYQAHGTDTGSADLQVALLTERINQLTTHLQNKDASRRGLLQLIGRRRGLLSYIQDQRYQLIRLGIRRMAVKERVGVVVSDKMNKTVVVAIENRSPHPKYRKIVVQTKKYKAHDEDNQCKIGDRVRIQETRPLSKTKRWIVAEILNTTPTS-----------'}"
      ]
     },
     "execution_count": 46,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ali_list = natsorted(glob.glob(\"/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/*.trimmed.faa\"))\n",
    "\n",
    "sequences_dict = {}\n",
    "\n",
    "for item in ali_list:\n",
    "    print(item)\n",
    "    for record in (SeqIO.parse(item, \"fasta\")):\n",
    "        if record.id not in sequences_dict:\n",
    "            sequences_dict[record.id] = '%s'%record.seq\n",
    "        else:\n",
    "            sequences_dict[record.id] = '%s'%sequences_dict[record.id]+'%s'%record.seq\n",
    "            \n",
    "sequences_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n",
      "3487\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "<function TextIOWrapper.close()>"
      ]
     },
     "execution_count": 47,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "seq = []\n",
    "\n",
    "for key in sequences_dict:\n",
    "    seq.append(SeqRecord(Seq(\"%s\"%sequences_dict[key],IUPAC.protein), id = \"%s\"%key, description=\"\"))\n",
    "\n",
    "for record in seq:\n",
    "    print(len(record))\n",
    "    \n",
    "handle = open(\"/Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/concat_genes-220221.faa\", \"w\")\n",
    "SeqIO.write(seq,handle,\"fasta\")\n",
    "handle.close"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">Obscuribacter_phosphatis_(Mle1_12)\r\n",
      ">Cyanobacterium_sp._(Zagget_111_MP)\r\n",
      ">YS2_(Zagget_bin_221)\r\n",
      ">SM1_D11_(SM2F09_MP_sspace)\r\n",
      ">Gloeobacter_kilaueensis_JS1\r\n",
      ">Gloeobacter_violaceus_PCC_7421_(re-annotation)\r\n",
      ">Synechococcus_sp._RCC307_(re-annotation)\r\n",
      ">Prochlorococcus_marinus_marinus_CCMP1375_(re-annotation)\r\n",
      ">Cyanobium_gracile_PCC_6307\r\n",
      ">Prochlorococcus_sp._MIT9211_(re-annotation)\r\n",
      ">Prochlorococcus_marinus_str._MIT_9303\r\n",
      ">Prochlorococcus_marinus_str._MIT_9313\r\n",
      ">Synechococcus_sp._WH7803_(re-annotation)\r\n",
      ">Synechococcus_sp._WH_8020\r\n",
      ">Synechococcus_sp._CC9311\r\n",
      ">Synechococcus_sp._WH_8016\r\n",
      ">Synechococcus_sp._CC9902\r\n",
      ">Synechococcus_sp._CC9616\r\n",
      ">Synechococcus_sp._WH_8109\r\n",
      ">Synechococcus_sp._CC9605\r\n",
      ">Synechococcus_sp._KORDI-52_(genome_sequencing)\r\n",
      ">Synechococcus_sp._WH_8102\r\n",
      ">Synechococcus_sp._KORDI-100_(genome_sequencing)\r\n",
      ">Synechococcus_sp._KORDI-49_(genome_sequencing)\r\n",
      ">Prochlorococcus_sp._MIT0801\r\n",
      ">Prochlorococcus_marinus_str._NATL2A\r\n",
      ">Prochlorococcus_marinus_str._NATL1A\r\n",
      ">Prochlorococcus_marinus_str._MIT_9215\r\n",
      ">Prochlorococcus_marinus_str._MIT_9202_scf_1109798535001_genomic_scaffold\r\n",
      ">Prochlorococcus_marinus_str._MIT_9312\r\n",
      ">Prochlorococcus_sp._MIT0604\r\n",
      ">Prochlorococcus_marinus_str._AS9601\r\n",
      ">Prochlorococcus_marinus_str._MIT_9301\r\n",
      ">Prochlorococcus_marinus_str._MIT_9515\r\n",
      ">Prochlorococcus_marinus_subsp._pastoris_str._CCMP1986\r\n",
      ">Prochlorococcus_marinus_bv._HNLC2_Ga0082359\r\n",
      ">Prochlorococcus_marinus_bv._HNLC1_Ga0082358\r\n",
      ">Synechococcus_sp._PCC_7336\r\n",
      ">Synechococcus_sp._JA-2-3B'a(2-13)\r\n",
      ">Synechococcus_sp._JA-3-3Ab\r\n",
      ">Chamaesiphon_sp._PCC_6605\r\n",
      ">Halothece_sp._PCC_7418\r\n",
      ">Dactylococcopsis_salina_PCC_8305\r\n",
      ">Pseudanabaena_sp._PCC_7367\r\n",
      ">Synechococcus_sp._PCC_7502\r\n",
      ">Planktothrix_agardhii_CYA126/8\r\n",
      ">Synechococcus_sp._PCC_7335_scf_1103496006895_genomic_scaffold\r\n",
      ">Synechocystis_sp._PCC_6803_(updateJune2012)\r\n",
      ">Trichodesmium_erythraeum_IMS101_(Genome_Extraction)\r\n",
      ">Pleurocapsa_sp._PCC_7319\r\n",
      ">Arthrospira_platensis_NIES-39\r\n",
      ">Arthrospira_platensis_C1_(Draft2_circular_genome_)\r\n",
      ">Leptolyngbya_sp._PCC_7376\r\n",
      ">Synechococcus_sp._PCC_7002\r\n",
      ">Acaryochloris_marina_MBIC11017\r\n",
      ">Oscillatoria_sp._PCC_7112\r\n",
      ">Spirulina_major_PCC_6313\r\n",
      ">Moorea_producens-heterotrophic_bacteria_co-culture_from_Jamaica_Moorea_producens_JHB_22AUG96-1_(Final_Draft_of_Linear_Chromosome_and_2_Linear_Plasmid_Scaffolds)\r\n",
      ">Moorea_bouillonii-heterotrophic_bacteria_co-culture_from_Papua_New_Guinea_Moorea_bouillonii_PNG_19MAY05-8_(One_linear_chromosome_and_12_unmapped_contigs)\r\n",
      ">Moorea_producens-heterotrophic_bacteria_co-culture_from_Palmyra_Atoll_Moorea_producens_PAL_15AUG08-1_(Complete_Genome\r\n",
      ">Moorea_producens-heterotrophic_bacteria_co-culture_from_Cura?ao_Moorea_producens_3L_(One_linear_chromosome_and_78_unmapped_contigs)\r\n",
      ">Calothrix_sp._336/3_null_replaces_82363\r\n",
      ">Oscillatoria_acuminata_PCC_6304\r\n",
      ">Geitlerinema_sp._PCC_7407\r\n",
      ">Synechococcus_sp._UTEX_2973\r\n",
      ">Synechococcus_elongatus_PCC_7942\r\n",
      ">Synechococcus_leopoliensis_UTEX_625a\r\n",
      ">Synechococcus_elongatus_PCC_6301_(re-annotation)\r\n",
      ">Crinalium_epipsammum_PCC_9333\r\n",
      ">Oscillatoriales_sp._JSC-12\r\n",
      ">Synechococcus_sp._PCC_6312\r\n",
      ">Rivularia_sp._PCC_7116\r\n",
      ">Cyanothece_sp._PCC_7425\r\n",
      ">Chroococcidiopsis_thermalis_PCC_7203\r\n",
      ">Thermosynechococcus_sp._NK55a\r\n",
      ">Thermosynechococcus_elongatus_BP-1_(re-annotation)\r\n",
      ">Microcoleus_sp._PCC_7113\r\n",
      ">Calothrix_sp._PCC_6303\r\n",
      ">Gloeocapsa_sp._PCC_7428\r\n",
      ">Calothrix_sp._PCC_7507\r\n",
      ">Anabaena_sp._wa102\r\n",
      ">Anabaena_sp._90\r\n",
      ">Dolichospermum_circinale_ACBU02\r\n",
      ">Fischerella_JAHHIM01\r\n",
      ">Nodularia_spumigena_CCY9414\r\n",
      ">'Nostoc_azollae'_0708_chromosome\r\n",
      ">Anabaena_cylindrica_PCC_7122\r\n",
      ">Anabaena_sp._PCC_7108\r\n",
      ">Cylindrospermum_stagnale_PCC_7417\r\n",
      ">Nostoc_JAIVFY01\r\n",
      ">Fischerella_JAHHHS01\r\n",
      ">Nostoc_JACJQM01\r\n",
      ">Nostoc_NOLK01\r\n",
      ">Nostoc_punctiforme_PCC_73102\r\n",
      ">Nostoc_JACKZN01\r\n",
      ">Nostoc_JACJSH01\r\n",
      ">Nostoc_JADEXT01\r\n",
      ">Nostoc_JACKZS01\r\n",
      ">Nostoc_JACKZR01\r\n",
      ">Nostoc_JABXKH01\r\n",
      ">Nostoc_JABXKG01\r\n",
      ">Nostoc_JABXKF01\r\n",
      ">Nostoc_JABXKL01\r\n",
      ">Nostoc_JABXKO01\r\n",
      ">Nostoc_JABXKN01\r\n",
      ">Nostocales_JADEXV01\r\n",
      ">Nostoc_LXQE01\r\n",
      ">Nostoc_JAIVFR01\r\n",
      ">Nostoc_WBKL01\r\n",
      ">Fremyella_diplosiphon_UTEX_481\r\n",
      ">Chlorogloeopsis_sp._PCC_7702\r\n",
      ">Nostoc_sp._PCC_7524\r\n",
      ">Nostoc_sp._PCC_7107\r\n",
      ">Nostoc_piscinale_CENA21\r\n",
      ">Nostoc_SZNH01\r\n",
      ">Desmonostoc_JACERF01\r\n",
      ">Nostoc_JACJSJ01\r\n",
      ">Nostoc_JACJTU01\r\n",
      ">Nostoc_JACJTK01\r\n",
      ">Nostoc_LWTK01\r\n",
      ">Nostoc_JACJTC01\r\n",
      ">Nostoc_VRZE01\r\n",
      ">Nostoc_LAHC01\r\n",
      ">Nostoc_LAHB01\r\n",
      ">Nostoc_LAHI01\r\n",
      ">Nostoc_LAHE01\r\n",
      ">Nostoc_LAGY01\r\n",
      ">Nostoc_LAGX01\r\n",
      ">Nostoc_LAHD01\r\n",
      ">Nostoc_LAHH01\r\n",
      ">Nostoc_LAHG01\r\n",
      ">Nostoc_LAHF01\r\n",
      ">Nostoc_LAGZ01\r\n",
      ">Nostoc_LAHJ01\r\n",
      ">Nostoc_LAHA01\r\n",
      ">Nostoc_JACKZU01\r\n",
      ">Nostoc_WVID01\r\n",
      ">Westiellopsis_JAHHHK01\r\n",
      ">Anabaena_variabilis_ATCC_29413\r\n",
      ">Nostoc_sp._PCC_7120_(re-annotation)\r\n",
      ">Mastigocladus_NGJL01\r\n",
      ">Amazonocrinis_JAECZC01\r\n",
      ">Fischerella_ALVT01\r\n",
      ">Hapalosiphon_LIRN01\r\n",
      ">Fischerella_ALVX01\r\n",
      ">Nostoc_JAHHHW01\r\n",
      ">Fischerella_CENA71\r\n",
      ">Mastigocladus_MNPM02\r\n",
      ">Westiellopsis_NAPS01\r\n",
      ">Fischerella_ALVS01\r\n",
      ">Fischerella_AJLJ01\r\n",
      ">Fischerella_JACJRZ01\r\n",
      ">Fischerella_NRQW01\r\n",
      ">Fischerella_AJLK01\r\n",
      ">Fischerella_NHAG01\r\n",
      ">Fischerella_NGZU01\r\n",
      ">Fischerella_NHAE01\r\n",
      ">Fischerella_JACYLX01\r\n",
      ">Fischerella_NMQB01\r\n",
      ">Fischerella_NGZY01\r\n",
      ">Fischerella_NHAH01\r\n",
      ">Fischerella_NGZX01\r\n",
      ">Fischerella_NMQC01\r\n",
      ">Fischerella_JACYLY01\r\n",
      ">Fischerella_NGZT01\r\n",
      ">Fischerella_NGAA01\r\n",
      ">Fischerella_JACYMA01\r\n",
      ">Fischerella_NMQH01\r\n",
      ">Fischerella_NGZS01\r\n",
      ">Fischerella_NQMD01\r\n",
      ">Fischerella_NHAB01\r\n",
      ">Fischerella_NQME01\r\n",
      ">Fischerella_NMQI01\r\n",
      ">Fischerella_NMQE01\r\n",
      ">Fischerella_NMQF01\r\n",
      ">Fischerella_NHAA01\r\n",
      ">Fischerella_NQMF01\r\n",
      ">Fischerella_AGIZ01\r\n",
      ">Fischerella_NMQJ01\r\n",
      ">Fischerella_AJLL01\r\n",
      ">Fischerella_NMQK01\r\n",
      ">Fischerella_NMQG01\r\n",
      ">Fischerella_NGZW01\r\n",
      ">Fischerella_NHAK01\r\n",
      ">Fischerella_MRCA01\r\n",
      ">Fischerella_NGZV01\r\n",
      ">Fischerella_NHAF01\r\n",
      ">Fischerella_NMQA01\r\n",
      ">Fischerella_NGZZ01\r\n",
      ">Synechocystis_sp._PCC_6308\r\n",
      ">Cyanobacterium_stanieri_PCC_7202\r\n",
      ">Cyanobacterium_sp._PCC_10605\r\n",
      ">Spirulina_sp._PCC_9445\r\n",
      ">Stanieria_cyanosphaera_PCC_7437\r\n",
      ">Cyanobacterium_UCYN-A\r\n",
      ">Cyanothece_sp._ATCC_51142_chromosome_circular\r\n",
      ">Cyanothece_sp._PCC_8801\r\n",
      ">Cyanothece_sp._PCC_8802\r\n",
      ">Microcystis_panniformis_FACHB-1757\r\n",
      ">Microcystis_aeruginosa_NIES-2549\r\n",
      ">Microcystis_aeruginosa_NIES-843\r\n",
      ">Pleurocapsa_sp._PCC_7327\r\n",
      ">Cyanothece_sp._PCC_7822_chromosome\r\n",
      ">Cyanothece_sp._PCC_7424\r\n"
     ]
    }
   ],
   "source": [
    "!grep '>' /Users/tiagoferreiraleao/Dropbox/tiago-NAS/schneider/pnas-phylogenomics-top_100/concat_genes-220221.faa"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
